--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl OptInPhoneNumberInput {
     /// Creates a new builder-style object to manufacture [`OptInPhoneNumberInput`](crate::operation::opt_in_phone_number::OptInPhoneNumberInput).
-    pub fn builder() -> crate::operation::opt_in_phone_number::builders::OptInPhoneNumberInputBuilder {
-        crate::operation::opt_in_phone_number::builders::OptInPhoneNumberInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::opt_in_phone_number::builders::OptInPhoneNumberInputBuilder {
+        super::super::super::operation::opt_in_phone_number::builders::OptInPhoneNumberInputBuilder::default()
     }
 }

@@ -52,8 +52,8 @@
     /// Consumes the builder and constructs a [`OptInPhoneNumberInput`](crate::operation::opt_in_phone_number::OptInPhoneNumberInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::opt_in_phone_number::OptInPhoneNumberInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::opt_in_phone_number::OptInPhoneNumberInput {
+    ) -> ::std::result::Result<super::super::super::operation::opt_in_phone_number::OptInPhoneNumberInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::opt_in_phone_number::OptInPhoneNumberInput {
             phone_number: self.phone_number,
         })
     }
