--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>The end time for the history query in Unix epoch format.</p>
     pub end_time: ::std::option::Option<i64>,
     /// <p>An array of execution statuses to filter the history results. Only executions with the specified statuses are returned.</p>
-    pub execution_statuses: ::std::option::Option<::std::vec::Vec<crate::types::ExecutionStatus>>,
+    pub execution_statuses: ::std::option::Option<::std::vec::Vec<super::types::ExecutionStatus>>,
     /// <p>The maximum number of history records to return. Valid range is 1 to 1000.</p>
     pub max_results: ::std::option::Option<i32>,
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
@@ -32,7 +32,7 @@
     /// <p>An array of execution statuses to filter the history results. Only executions with the specified statuses are returned.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.execution_statuses.is_none()`.
-    pub fn execution_statuses(&self) -> &[crate::types::ExecutionStatus] {
+    pub fn execution_statuses(&self) -> &[super::types::ExecutionStatus] {
         self.execution_statuses.as_deref().unwrap_or_default()
     }
     /// <p>The maximum number of history records to return. Valid range is 1 to 1000.</p>
@@ -46,8 +46,8 @@
 }
 impl GetScheduledQueryHistoryInput {
     /// Creates a new builder-style object to manufacture [`GetScheduledQueryHistoryInput`](crate::operation::get_scheduled_query_history::GetScheduledQueryHistoryInput).
-    pub fn builder() -> crate::operation::get_scheduled_query_history::builders::GetScheduledQueryHistoryInputBuilder {
-        crate::operation::get_scheduled_query_history::builders::GetScheduledQueryHistoryInputBuilder::default()
+    pub fn builder() -> super::operation::get_scheduled_query_history::builders::GetScheduledQueryHistoryInputBuilder {
+        super::operation::get_scheduled_query_history::builders::GetScheduledQueryHistoryInputBuilder::default()
     }
 }

@@ -58,7 +58,7 @@
     pub(crate) identifier: ::std::option::Option<::std::string::String>,
     pub(crate) start_time: ::std::option::Option<i64>,
     pub(crate) end_time: ::std::option::Option<i64>,
-    pub(crate) execution_statuses: ::std::option::Option<::std::vec::Vec<crate::types::ExecutionStatus>>,
+    pub(crate) execution_statuses: ::std::option::Option<::std::vec::Vec<super::types::ExecutionStatus>>,
     pub(crate) max_results: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
 }
@@ -113,7 +113,7 @@
     /// To override the contents of this collection use [`set_execution_statuses`](Self::set_execution_statuses).
     ///
     /// <p>An array of execution statuses to filter the history results. Only executions with the specified statuses are returned.</p>
-    pub fn execution_statuses(mut self, input: crate::types::ExecutionStatus) -> Self {
+    pub fn execution_statuses(mut self, input: super::types::ExecutionStatus) -> Self {
         let mut v = self.execution_statuses.unwrap_or_default();
         v.push(input);
         self.execution_statuses = ::std::option::Option::Some(v);
@@ -120,12 +120,12 @@
         self
     }
     /// <p>An array of execution statuses to filter the history results. Only executions with the specified statuses are returned.</p>
-    pub fn set_execution_statuses(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ExecutionStatus>>) -> Self {
+    pub fn set_execution_statuses(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ExecutionStatus>>) -> Self {
         self.execution_statuses = input;
         self
     }
     /// <p>An array of execution statuses to filter the history results. Only executions with the specified statuses are returned.</p>
-    pub fn get_execution_statuses(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ExecutionStatus>> {
+    pub fn get_execution_statuses(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ExecutionStatus>> {
         &self.execution_statuses
     }
     /// <p>The maximum number of history records to return. Valid range is 1 to 1000.</p>
@@ -160,10 +160,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_scheduled_query_history::GetScheduledQueryHistoryInput,
+        super::operation::get_scheduled_query_history::GetScheduledQueryHistoryInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_scheduled_query_history::GetScheduledQueryHistoryInput {
+        ::std::result::Result::Ok(super::operation::get_scheduled_query_history::GetScheduledQueryHistoryInput {
             identifier: self.identifier,
             start_time: self.start_time,
             end_time: self.end_time,
