--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteSchedulingPolicyInput {
     /// Creates a new builder-style object to manufacture [`DeleteSchedulingPolicyInput`](crate::operation::delete_scheduling_policy::DeleteSchedulingPolicyInput).
-    pub fn builder() -> crate::operation::delete_scheduling_policy::builders::DeleteSchedulingPolicyInputBuilder {
-        crate::operation::delete_scheduling_policy::builders::DeleteSchedulingPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_scheduling_policy::builders::DeleteSchedulingPolicyInputBuilder {
+        super::super::super::operation::delete_scheduling_policy::builders::DeleteSchedulingPolicyInputBuilder::default()
     }
 }

@@ -46,9 +46,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_scheduling_policy::DeleteSchedulingPolicyInput,
+        super::super::super::operation::delete_scheduling_policy::DeleteSchedulingPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_scheduling_policy::DeleteSchedulingPolicyInput { arn: self.arn })
+        ::std::result::Result::Ok(super::super::super::operation::delete_scheduling_policy::DeleteSchedulingPolicyInput { arn: self.arn })
     }
 }
