--- original
+++ modified
@@ -60,8 +60,8 @@
 }
 impl GetUserAttributeVerificationCodeInput {
     /// Creates a new builder-style object to manufacture [`GetUserAttributeVerificationCodeInput`](crate::operation::get_user_attribute_verification_code::GetUserAttributeVerificationCodeInput).
-    pub fn builder() -> crate::operation::get_user_attribute_verification_code::builders::GetUserAttributeVerificationCodeInputBuilder {
-        crate::operation::get_user_attribute_verification_code::builders::GetUserAttributeVerificationCodeInputBuilder::default()
+    pub fn builder() -> super::operation::get_user_attribute_verification_code::builders::GetUserAttributeVerificationCodeInputBuilder {
+        super::operation::get_user_attribute_verification_code::builders::GetUserAttributeVerificationCodeInputBuilder::default()
     }
 }

@@ -171,11 +171,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_user_attribute_verification_code::GetUserAttributeVerificationCodeInput,
+        super::operation::get_user_attribute_verification_code::GetUserAttributeVerificationCodeInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_user_attribute_verification_code::GetUserAttributeVerificationCodeInput {
+            super::operation::get_user_attribute_verification_code::GetUserAttributeVerificationCodeInput {
                 access_token: self.access_token,
                 attribute_name: self.attribute_name,
                 client_metadata: self.client_metadata,
