--- original
+++ modified
@@ -166,8 +166,8 @@
 }
 impl PutDeliverySourceInput {
     /// Creates a new builder-style object to manufacture [`PutDeliverySourceInput`](crate::operation::put_delivery_source::PutDeliverySourceInput).
-    pub fn builder() -> crate::operation::put_delivery_source::builders::PutDeliverySourceInputBuilder {
-        crate::operation::put_delivery_source::builders::PutDeliverySourceInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_delivery_source::builders::PutDeliverySourceInputBuilder {
+        super::super::super::operation::put_delivery_source::builders::PutDeliverySourceInputBuilder::default()
     }
 }

@@ -471,8 +471,8 @@
     /// Consumes the builder and constructs a [`PutDeliverySourceInput`](crate::operation::put_delivery_source::PutDeliverySourceInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_delivery_source::PutDeliverySourceInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::put_delivery_source::PutDeliverySourceInput {
+    ) -> ::std::result::Result<super::super::super::operation::put_delivery_source::PutDeliverySourceInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::put_delivery_source::PutDeliverySourceInput {
             name: self.name,
             resource_arn: self.resource_arn,
             log_type: self.log_type,
