--- original
+++ modified
@@ -20,9 +20,9 @@
 }
 impl DeleteServiceLinkedConfigurationRecorderInput {
     /// Creates a new builder-style object to manufacture [`DeleteServiceLinkedConfigurationRecorderInput`](crate::operation::delete_service_linked_configuration_recorder::DeleteServiceLinkedConfigurationRecorderInput).
-    pub fn builder() -> crate::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderInputBuilder
+    pub fn builder() -> super::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderInputBuilder
     {
-        crate::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderInputBuilder::default()
+        super::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderInputBuilder::default()
     }
 }

@@ -66,11 +66,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_service_linked_configuration_recorder::DeleteServiceLinkedConfigurationRecorderInput,
+        super::operation::delete_service_linked_configuration_recorder::DeleteServiceLinkedConfigurationRecorderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::delete_service_linked_configuration_recorder::DeleteServiceLinkedConfigurationRecorderInput {
+            super::operation::delete_service_linked_configuration_recorder::DeleteServiceLinkedConfigurationRecorderInput {
                 service_principal: self.service_principal,
                 arn: self.arn,
             },
