--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl DeleteEmailIdentityPolicyInput {
     /// Creates a new builder-style object to manufacture [`DeleteEmailIdentityPolicyInput`](crate::operation::delete_email_identity_policy::DeleteEmailIdentityPolicyInput).
-    pub fn builder() -> crate::operation::delete_email_identity_policy::builders::DeleteEmailIdentityPolicyInputBuilder {
-        crate::operation::delete_email_identity_policy::builders::DeleteEmailIdentityPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::delete_email_identity_policy::builders::DeleteEmailIdentityPolicyInputBuilder {
+        super::operation::delete_email_identity_policy::builders::DeleteEmailIdentityPolicyInputBuilder::default()
     }
 }

@@ -73,10 +73,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_email_identity_policy::DeleteEmailIdentityPolicyInput,
+        super::operation::delete_email_identity_policy::DeleteEmailIdentityPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_email_identity_policy::DeleteEmailIdentityPolicyInput {
+        ::std::result::Result::Ok(super::operation::delete_email_identity_policy::DeleteEmailIdentityPolicyInput {
             email_identity: self.email_identity,
             policy_name: self.policy_name,
         })
