--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
+        input: super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
     ) -> ::std::result::Result<
-        crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationOutput,
+        super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
+            super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError>()
+                err.downcast::<super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationOutput>()
+                .downcast::<super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
+        input: super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("DeleteBucketAnalyticsConfiguration")
                 .build()
                 .expect("required fields set"),
@@ -142,14 +142,14 @@
                 DeleteBucketAnalyticsConfigurationEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
+                super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
+                super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
             >::new())
             .with_retry_classifier(
                 ::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                    crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
+                    super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError,
                 >::builder()
                 .transient_errors({
                     let mut transient_errors: Vec<&'static str> = ::aws_runtime::retries::classifiers::TRANSIENT_ERRORS.into();
@@ -229,21 +229,21 @@
         let body = response.body().bytes().expect("body loaded");
         #[allow(unused_mut)]
         let mut force_error = false;
-        ::tracing::debug!(extended_request_id = ?crate::s3_request_id::RequestIdExt::extended_request_id(response));
-        if matches!(crate::rest_xml_unwrapped_errors::body_is_error(body), Ok(true)) {
+        ::tracing::debug!(extended_request_id = ?super::super::s3_request_id::RequestIdExt::extended_request_id(response));
+        if matches!(super::super::rest_xml_unwrapped_errors::body_is_error(body), Ok(true)) {
             force_error = true;
         }
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 204 || force_error {
-            crate::protocol_serde::shape_delete_bucket_analytics_configuration::de_delete_bucket_analytics_configuration_http_error(
+            super::super::protocol_serde::shape_delete_bucket_analytics_configuration::de_delete_bucket_analytics_configuration_http_error(
                 status, headers, body,
             )
         } else {
-            crate::protocol_serde::shape_delete_bucket_analytics_configuration::de_delete_bucket_analytics_configuration_http_response(
+            super::super::protocol_serde::shape_delete_bucket_analytics_configuration::de_delete_bucket_analytics_configuration_http_response(
                 status, headers, body,
             )
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -256,16 +256,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput>()
+            .downcast::<super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
+                _input: &super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -273,7 +273,7 @@
                 ::std::result::Result::Ok(())
             }
             fn uri_query(
-                _input: &crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
+                _input: &super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
                 mut output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 let mut query = ::aws_smithy_http::query::Writer::new(output);
@@ -293,13 +293,13 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
+                input: &super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
                 uri_base(input, &mut uri)?;
                 uri_query(input, &mut uri)?;
-                let builder = crate::protocol_serde::shape_delete_bucket_analytics_configuration::ser_delete_bucket_analytics_configuration_headers(
+                let builder = super::super::protocol_serde::shape_delete_bucket_analytics_configuration::ser_delete_bucket_analytics_configuration_headers(
                     input, builder,
                 )?;
                 ::std::result::Result::Ok(builder.method("DELETE").uri(uri))
@@ -336,16 +336,16 @@
             .downcast_ref::<DeleteBucketAnalyticsConfigurationInput>()
             .ok_or("failed to downcast to DeleteBucketAnalyticsConfigurationInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_endpoint(cfg.load::<::aws_types::endpoint_config::EndpointUrl>().map(|ty| ty.0.clone()))
-            .set_force_path_style(cfg.load::<crate::config::ForcePathStyle>().map(|ty| ty.0))
-            .set_use_arn_region(cfg.load::<crate::config::UseArnRegion>().map(|ty| ty.0))
-            .set_disable_multi_region_access_points(cfg.load::<crate::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
-            .set_accelerate(cfg.load::<crate::config::Accelerate>().map(|ty| ty.0))
-            .set_disable_s3_express_session_auth(cfg.load::<crate::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
+            .set_force_path_style(cfg.load::<super::super::config::ForcePathStyle>().map(|ty| ty.0))
+            .set_use_arn_region(cfg.load::<super::super::config::UseArnRegion>().map(|ty| ty.0))
+            .set_disable_multi_region_access_points(cfg.load::<super::super::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
+            .set_accelerate(cfg.load::<super::super::config::Accelerate>().map(|ty| ty.0))
+            .set_disable_s3_express_session_auth(cfg.load::<super::super::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
             .set_use_s3_express_control_endpoint(Some(true))
             .set_bucket(Some(
                 _input
@@ -378,7 +378,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-DeleteBucketAnalyticsConfigurationError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl DeleteBucketAnalyticsConfigurationError {
     /// Creates the `DeleteBucketAnalyticsConfigurationError::Unhandled` variant from any error type.
@@ -385,7 +385,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -393,7 +393,7 @@

     /// Creates the `DeleteBucketAnalyticsConfigurationError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -448,26 +448,26 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl crate::s3_request_id::RequestIdExt for crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError {
+impl super::super::s3_request_id::RequestIdExt for super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError {
     fn extended_request_id(&self) -> Option<&str> {
         self.meta().extended_request_id()
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError {
+impl ::aws_types::request_id::RequestId for super::super::operation::delete_bucket_analytics_configuration::DeleteBucketAnalyticsConfigurationError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::delete_bucket_analytics_configuration::_delete_bucket_analytics_configuration_input::DeleteBucketAnalyticsConfigurationInput;
+pub use super::super::operation::delete_bucket_analytics_configuration::_delete_bucket_analytics_configuration_input::DeleteBucketAnalyticsConfigurationInput;

-pub use crate::operation::delete_bucket_analytics_configuration::_delete_bucket_analytics_configuration_output::DeleteBucketAnalyticsConfigurationOutput;
+pub use super::super::operation::delete_bucket_analytics_configuration::_delete_bucket_analytics_configuration_output::DeleteBucketAnalyticsConfigurationOutput;

 mod _delete_bucket_analytics_configuration_input;

