--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::delete_object_tagging::DeleteObjectTaggingInput,
+        input: super::operation::delete_object_tagging::DeleteObjectTaggingInput,
     ) -> ::std::result::Result<
-        crate::operation::delete_object_tagging::DeleteObjectTaggingOutput,
+        super::operation::delete_object_tagging::DeleteObjectTaggingOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::delete_object_tagging::DeleteObjectTaggingError,
+            super::operation::delete_object_tagging::DeleteObjectTaggingError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::delete_object_tagging::DeleteObjectTaggingError>()
+                err.downcast::<super::operation::delete_object_tagging::DeleteObjectTaggingError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::delete_object_tagging::DeleteObjectTaggingOutput>()
+                .downcast::<super::operation::delete_object_tagging::DeleteObjectTaggingOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::delete_object_tagging::DeleteObjectTaggingInput,
+        input: super::operation::delete_object_tagging::DeleteObjectTaggingInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("DeleteObjectTagging")
                 .build()
                 .expect("required fields set"),
@@ -126,9 +126,9 @@
                             .with_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(DeleteObjectTaggingTelemetryInputCaptureInterceptor))
 .with_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(::aws_smithy_runtime::client::stalled_stream_protection::StalledStreamProtectionInterceptor::default()))
 .with_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(DeleteObjectTaggingEndpointParamsInterceptor))
-                            .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<crate::operation::delete_object_tagging::DeleteObjectTaggingError>::new())
-.with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<crate::operation::delete_object_tagging::DeleteObjectTaggingError>::new())
-.with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<crate::operation::delete_object_tagging::DeleteObjectTaggingError>::builder().transient_errors({
+                            .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<super::operation::delete_object_tagging::DeleteObjectTaggingError>::new())
+.with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<super::operation::delete_object_tagging::DeleteObjectTaggingError>::new())
+.with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<super::operation::delete_object_tagging::DeleteObjectTaggingError>::builder().transient_errors({
                                             let mut transient_errors: Vec<&'static str> = ::aws_runtime::retries::classifiers::TRANSIENT_ERRORS.into();
                                             transient_errors.push("InternalError");
                                             ::std::borrow::Cow::Owned(transient_errors)
@@ -215,11 +215,11 @@
         }
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 204 || force_error {
-            crate::protocol_serde::shape_delete_object_tagging::de_delete_object_tagging_http_error(status, headers, body)
+            super::protocol_serde::shape_delete_object_tagging::de_delete_object_tagging_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_delete_object_tagging::de_delete_object_tagging_http_response(status, headers, body)
+            super::protocol_serde::shape_delete_object_tagging::de_delete_object_tagging_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -232,16 +232,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::delete_object_tagging::DeleteObjectTaggingInput>()
+            .downcast::<super::operation::delete_object_tagging::DeleteObjectTaggingInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::delete_object_tagging::DeleteObjectTaggingInput,
+                _input: &super::operation::delete_object_tagging::DeleteObjectTaggingInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -260,7 +260,7 @@
                 ::std::result::Result::Ok(())
             }
             fn uri_query(
-                _input: &crate::operation::delete_object_tagging::DeleteObjectTaggingInput,
+                _input: &super::operation::delete_object_tagging::DeleteObjectTaggingInput,
                 mut output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 let mut query = ::aws_smithy_http::query::Writer::new(output);
@@ -274,13 +274,13 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::delete_object_tagging::DeleteObjectTaggingInput,
+                input: &super::operation::delete_object_tagging::DeleteObjectTaggingInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
                 uri_base(input, &mut uri)?;
                 uri_query(input, &mut uri)?;
-                let builder = crate::protocol_serde::shape_delete_object_tagging::ser_delete_object_tagging_headers(input, builder)?;
+                let builder = super::protocol_serde::shape_delete_object_tagging::ser_delete_object_tagging_headers(input, builder)?;
                 ::std::result::Result::Ok(builder.method("DELETE").uri(uri))
             }
             let mut builder = update_http_builder(&input, ::http_1x::request::Builder::new())?;
@@ -315,16 +315,16 @@
             .downcast_ref::<DeleteObjectTaggingInput>()
             .ok_or("failed to downcast to DeleteObjectTaggingInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_endpoint(cfg.load::<::aws_types::endpoint_config::EndpointUrl>().map(|ty| ty.0.clone()))
-            .set_force_path_style(cfg.load::<crate::config::ForcePathStyle>().map(|ty| ty.0))
-            .set_use_arn_region(cfg.load::<crate::config::UseArnRegion>().map(|ty| ty.0))
-            .set_disable_multi_region_access_points(cfg.load::<crate::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
-            .set_accelerate(cfg.load::<crate::config::Accelerate>().map(|ty| ty.0))
-            .set_disable_s3_express_session_auth(cfg.load::<crate::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
+            .set_force_path_style(cfg.load::<super::config::ForcePathStyle>().map(|ty| ty.0))
+            .set_use_arn_region(cfg.load::<super::config::UseArnRegion>().map(|ty| ty.0))
+            .set_disable_multi_region_access_points(cfg.load::<super::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
+            .set_accelerate(cfg.load::<super::config::Accelerate>().map(|ty| ty.0))
+            .set_disable_s3_express_session_auth(cfg.load::<super::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
             .set_bucket(Some(
                 _input
                     .bucket
@@ -345,90 +345,6 @@
 // The get_* functions below are generated from JMESPath expressions in the
 // operationContextParams trait. They target the operation's input shape.

-#[allow(unreachable_code, unused_variables)]
-#[cfg(test)]
-mod delete_object_tagging_test {
-
-    /// S3 clients should escape special characters in Object Keys
-    /// when the Object Key is used as a URI label binding.
-    ///
-    /// Test ID: S3EscapeObjectKeyInUriLabel
-    #[::tokio::test]
-    #[::tracing_test::traced_test]
-    async fn s3_escape_object_key_in_uri_label_request() {
-        let (http_client, request_receiver) = ::aws_smithy_http_client::test_util::capture_request(None);
-        let config_builder = crate::config::Config::builder()
-            .with_test_defaults()
-            // TODO(https://github.com/smithy-lang/smithy-rs/issues/4177):
-            //  Until the incorrect separation is addressed, we need to rely on this workaround.
-            .allow_no_auth()
-            .endpoint_url("https://s3.us-west-2.amazonaws.com");
-
-        let mut config_builder = config_builder;
-        config_builder.set_region(Some(crate::config::Region::new("us-east-1")));
-
-        let config = config_builder.http_client(http_client).build();
-        let client = crate::Client::from_conf(config);
-        let result = client
-            .delete_object_tagging()
-            .set_bucket(::std::option::Option::Some("mybucket".to_owned()))
-            .set_key(::std::option::Option::Some("my key.txt".to_owned()))
-            .send()
-            .await;
-        let _ = dbg!(result);
-        let http_request = request_receiver.expect_request();
-        let expected_query_params = &["tagging"];
-        ::aws_smithy_protocol_test::assert_ok(::aws_smithy_protocol_test::validate_query_string(&http_request, expected_query_params));
-        let body = http_request.body().bytes().expect("body should be strict");
-        // No body.
-        ::pretty_assertions::assert_eq!(&body, &::bytes::Bytes::new());
-        let uri: ::http_1x::Uri = http_request.uri().parse().expect("invalid URI sent");
-        ::pretty_assertions::assert_eq!(http_request.method(), "DELETE", "method was incorrect");
-        ::pretty_assertions::assert_eq!(uri.path(), "/my%20key.txt", "path was incorrect");
-        ::pretty_assertions::assert_eq!(uri.host().expect("host should be set"), "mybucket.s3.us-west-2.amazonaws.com");
-    }
-
-    /// S3 clients should preserve an Object Key representing a path
-    /// when the Object Key is used as a URI label binding, but still
-    /// escape special characters.
-    ///
-    /// Test ID: S3EscapePathObjectKeyInUriLabel
-    #[::tokio::test]
-    #[::tracing_test::traced_test]
-    async fn s3_escape_path_object_key_in_uri_label_request() {
-        let (http_client, request_receiver) = ::aws_smithy_http_client::test_util::capture_request(None);
-        let config_builder = crate::config::Config::builder()
-            .with_test_defaults()
-            // TODO(https://github.com/smithy-lang/smithy-rs/issues/4177):
-            //  Until the incorrect separation is addressed, we need to rely on this workaround.
-            .allow_no_auth()
-            .endpoint_url("https://s3.us-west-2.amazonaws.com");
-
-        let mut config_builder = config_builder;
-        config_builder.set_region(Some(crate::config::Region::new("us-east-1")));
-
-        let config = config_builder.http_client(http_client).build();
-        let client = crate::Client::from_conf(config);
-        let result = client
-            .delete_object_tagging()
-            .set_bucket(::std::option::Option::Some("mybucket".to_owned()))
-            .set_key(::std::option::Option::Some("foo/bar/my key.txt".to_owned()))
-            .send()
-            .await;
-        let _ = dbg!(result);
-        let http_request = request_receiver.expect_request();
-        let expected_query_params = &["tagging"];
-        ::aws_smithy_protocol_test::assert_ok(::aws_smithy_protocol_test::validate_query_string(&http_request, expected_query_params));
-        let body = http_request.body().bytes().expect("body should be strict");
-        // No body.
-        ::pretty_assertions::assert_eq!(&body, &::bytes::Bytes::new());
-        let uri: ::http_1x::Uri = http_request.uri().parse().expect("invalid URI sent");
-        ::pretty_assertions::assert_eq!(http_request.method(), "DELETE", "method was incorrect");
-        ::pretty_assertions::assert_eq!(uri.path(), "/foo/bar/my%20key.txt", "path was incorrect");
-        ::pretty_assertions::assert_eq!(uri.host().expect("host should be set"), "mybucket.s3.us-west-2.amazonaws.com");
-    }
-}
-
 /// Error type for the `DeleteObjectTaggingError` operation.
 #[non_exhaustive]
 #[derive(::std::fmt::Debug)]
@@ -440,7 +356,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-DeleteObjectTaggingError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl DeleteObjectTaggingError {
     /// Creates the `DeleteObjectTaggingError::Unhandled` variant from any error type.
@@ -447,7 +363,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -455,7 +371,7 @@

     /// Creates the `DeleteObjectTaggingError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -510,26 +426,26 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl crate::s3_request_id::RequestIdExt for crate::operation::delete_object_tagging::DeleteObjectTaggingError {
+impl super::s3_request_id::RequestIdExt for super::operation::delete_object_tagging::DeleteObjectTaggingError {
     fn extended_request_id(&self) -> Option<&str> {
         self.meta().extended_request_id()
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::delete_object_tagging::DeleteObjectTaggingError {
+impl ::aws_types::request_id::RequestId for super::operation::delete_object_tagging::DeleteObjectTaggingError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::delete_object_tagging::_delete_object_tagging_input::DeleteObjectTaggingInput;
+pub use super::operation::delete_object_tagging::_delete_object_tagging_input::DeleteObjectTaggingInput;

-pub use crate::operation::delete_object_tagging::_delete_object_tagging_output::DeleteObjectTaggingOutput;
+pub use super::operation::delete_object_tagging::_delete_object_tagging_output::DeleteObjectTaggingOutput;

 mod _delete_object_tagging_input;

