--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::list_objects::ListObjectsInput,
+        input: super::operation::list_objects::ListObjectsInput,
     ) -> ::std::result::Result<
-        crate::operation::list_objects::ListObjectsOutput,
+        super::operation::list_objects::ListObjectsOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::list_objects::ListObjectsError,
+            super::operation::list_objects::ListObjectsError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::list_objects::ListObjectsError>()
+                err.downcast::<super::operation::list_objects::ListObjectsError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::list_objects::ListObjectsOutput>()
+                .downcast::<super::operation::list_objects::ListObjectsOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::list_objects::ListObjectsInput,
+        input: super::operation::list_objects::ListObjectsInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("ListObjects")
                 .build()
                 .expect("required fields set"),
@@ -133,13 +133,13 @@
                 ListObjectsEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::list_objects::ListObjectsError,
+                super::operation::list_objects::ListObjectsError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::list_objects::ListObjectsError,
+                super::operation::list_objects::ListObjectsError,
             >::new())
             .with_retry_classifier(
-                ::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<crate::operation::list_objects::ListObjectsError>::builder()
+                ::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<super::operation::list_objects::ListObjectsError>::builder()
                     .transient_errors({
                         let mut transient_errors: Vec<&'static str> = ::aws_runtime::retries::classifiers::TRANSIENT_ERRORS.into();
                         transient_errors.push("InternalError");
@@ -234,11 +234,11 @@
         }
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_list_objects::de_list_objects_http_error(status, headers, body)
+            super::protocol_serde::shape_list_objects::de_list_objects_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_list_objects::de_list_objects_http_response(status, headers, body)
+            super::protocol_serde::shape_list_objects::de_list_objects_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -251,16 +251,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::list_objects::ListObjectsInput>()
+            .downcast::<super::operation::list_objects::ListObjectsInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::list_objects::ListObjectsInput,
+                _input: &super::operation::list_objects::ListObjectsInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -268,7 +268,7 @@
                 ::std::result::Result::Ok(())
             }
             fn uri_query(
-                _input: &crate::operation::list_objects::ListObjectsInput,
+                _input: &super::operation::list_objects::ListObjectsInput,
                 mut output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 let mut query = ::aws_smithy_http::query::Writer::new(output);
@@ -301,13 +301,13 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::list_objects::ListObjectsInput,
+                input: &super::operation::list_objects::ListObjectsInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
                 uri_base(input, &mut uri)?;
                 uri_query(input, &mut uri)?;
-                let builder = crate::protocol_serde::shape_list_objects::ser_list_objects_headers(input, builder)?;
+                let builder = super::protocol_serde::shape_list_objects::ser_list_objects_headers(input, builder)?;
                 ::std::result::Result::Ok(builder.method("GET").uri(uri))
             }
             let mut builder = update_http_builder(&input, ::http_1x::request::Builder::new())?;
@@ -342,16 +342,16 @@
             .downcast_ref::<ListObjectsInput>()
             .ok_or("failed to downcast to ListObjectsInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_endpoint(cfg.load::<::aws_types::endpoint_config::EndpointUrl>().map(|ty| ty.0.clone()))
-            .set_force_path_style(cfg.load::<crate::config::ForcePathStyle>().map(|ty| ty.0))
-            .set_use_arn_region(cfg.load::<crate::config::UseArnRegion>().map(|ty| ty.0))
-            .set_disable_multi_region_access_points(cfg.load::<crate::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
-            .set_accelerate(cfg.load::<crate::config::Accelerate>().map(|ty| ty.0))
-            .set_disable_s3_express_session_auth(cfg.load::<crate::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
+            .set_force_path_style(cfg.load::<super::config::ForcePathStyle>().map(|ty| ty.0))
+            .set_use_arn_region(cfg.load::<super::config::UseArnRegion>().map(|ty| ty.0))
+            .set_disable_multi_region_access_points(cfg.load::<super::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
+            .set_accelerate(cfg.load::<super::config::Accelerate>().map(|ty| ty.0))
+            .set_disable_s3_express_session_auth(cfg.load::<super::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
             .set_bucket(Some(
                 _input
                     .bucket
@@ -373,124 +373,12 @@
 // The get_* functions below are generated from JMESPath expressions in the
 // operationContextParams trait. They target the operation's input shape.

-#[allow(unreachable_code, unused_variables)]
-#[cfg(test)]
-mod list_objects_test {
-
-    /// This test validates that parsing respects whitespace
-    /// Test ID: KeysWithWhitespace
-    #[::tokio::test]
-    #[::tracing_test::traced_test]
-    async fn keys_with_whitespace_response() {
-        let expected_output = crate::operation::list_objects::ListObjectsOutput::builder()
-            .set_max_keys(::std::option::Option::Some(1000))
-            .set_is_truncated(::std::option::Option::Some(false))
-            .set_marker(::std::option::Option::Some("".to_owned()))
-            .set_name(::std::option::Option::Some("bucketname".to_owned()))
-            .set_prefix(::std::option::Option::Some("".to_owned()))
-            .set_contents(::std::option::Option::Some(vec![
-                crate::types::Object::builder()
-                    .set_key(::std::option::Option::Some("    ".to_owned()))
-                    .set_last_modified(::std::option::Option::Some(::aws_smithy_types::DateTime::from_fractional_secs(
-                        1626452453, 0_f64,
-                    )))
-                    .set_e_tag(::std::option::Option::Some("\"etag123\"".to_owned()))
-                    .set_size(::std::option::Option::Some(0))
-                    .set_owner(::std::option::Option::Some(
-                        crate::types::Owner::builder()
-                            .set_id(::std::option::Option::Some("owner".to_owned()))
-                            .build(),
-                    ))
-                    .set_storage_class(::std::option::Option::Some(
-                        "STANDARD"
-                            .parse::<crate::types::ObjectStorageClass>()
-                            .expect("static value validated to member"),
-                    ))
-                    .build(),
-                crate::types::Object::builder()
-                    .set_key(::std::option::Option::Some(" a ".to_owned()))
-                    .set_last_modified(::std::option::Option::Some(::aws_smithy_types::DateTime::from_fractional_secs(
-                        1626451330, 0_f64,
-                    )))
-                    .set_e_tag(::std::option::Option::Some("\"etag123\"".to_owned()))
-                    .set_size(::std::option::Option::Some(0))
-                    .set_owner(::std::option::Option::Some(
-                        crate::types::Owner::builder()
-                            .set_id(::std::option::Option::Some("owner".to_owned()))
-                            .build(),
-                    ))
-                    .set_storage_class(::std::option::Option::Some(
-                        "STANDARD"
-                            .parse::<crate::types::ObjectStorageClass>()
-                            .expect("static value validated to member"),
-                    ))
-                    .build(),
-            ]))
-            .build();
-        let mut http_response = ::aws_smithy_runtime_api::http::Response::try_from(::http_1x::response::Builder::new()
-        .status(200)
-                    .body(::aws_smithy_types::body::SdkBody::from("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<ListBucketResult\n\txmlns=\"http://s3.amazonaws.com/doc/2006-03-01/\">\n\t<Name>bucketname</Name>\n\t<Prefix></Prefix>\n\t<Marker></Marker>\n\t<MaxKeys>1000</MaxKeys>\n\t<IsTruncated>false</IsTruncated>\n\t<Contents>\n\t\t<Key>    </Key>\n\t\t<LastModified>2021-07-16T16:20:53.000Z</LastModified>\n\t\t<ETag>&quot;etag123&quot;</ETag>\n\t\t<Size>0</Size>\n\t\t<Owner>\n\t\t\t<ID>owner</ID>\n\t\t</Owner>\n\t\t<StorageClass>STANDARD</StorageClass>\n\t</Contents>\n\t<Contents>\n\t\t<Key> a </Key>\n\t\t<LastModified>2021-07-16T16:02:10.000Z</LastModified>\n\t\t<ETag>&quot;etag123&quot;</ETag>\n\t\t<Size>0</Size>\n\t\t<Owner>\n\t\t\t<ID>owner</ID>\n\t\t</Owner>\n\t\t<StorageClass>STANDARD</StorageClass>\n\t</Contents>\n</ListBucketResult>\n"))
-                    .unwrap()
-                    ).unwrap();
-        use ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugin;
-        use ::aws_smithy_runtime_api::client::ser_de::DeserializeResponse;
-
-        let op = crate::operation::list_objects::ListObjects::new();
-        let config = op.config().expect("the operation has config");
-        let de = config
-            .load::<::aws_smithy_runtime_api::client::ser_de::SharedResponseDeserializer>()
-            .expect("the config must have a deserializer");
-
-        // Build a config bag with the protocol for schema-based deserialization
-        #[allow(unused_mut)]
-        let mut test_cfg = ::aws_smithy_types::config_bag::ConfigBag::base();
-
-        let parsed = de.deserialize_streaming(&mut http_response);
-        let parsed = parsed.unwrap_or_else(|| {
-            let http_response = http_response.map(|body| {
-                ::aws_smithy_types::body::SdkBody::from(::bytes::Bytes::copy_from_slice(&::aws_smithy_protocol_test::decode_body_data(
-                    body.bytes().unwrap(),
-                    ::aws_smithy_protocol_test::MediaType::from("application/xml"),
-                )))
-            });
-            de.deserialize_nonstreaming_with_config(&http_response, &test_cfg)
-        });
-        let parsed = parsed
-            .expect("should be successful response")
-            .downcast::<crate::operation::list_objects::ListObjectsOutput>()
-            .unwrap();
-        ::pretty_assertions::assert_eq!(parsed.is_truncated, expected_output.is_truncated, "Unexpected value for `is_truncated`");
-        ::pretty_assertions::assert_eq!(parsed.marker, expected_output.marker, "Unexpected value for `marker`");
-        ::pretty_assertions::assert_eq!(parsed.next_marker, expected_output.next_marker, "Unexpected value for `next_marker`");
-        ::pretty_assertions::assert_eq!(parsed.contents, expected_output.contents, "Unexpected value for `contents`");
-        ::pretty_assertions::assert_eq!(parsed.name, expected_output.name, "Unexpected value for `name`");
-        ::pretty_assertions::assert_eq!(parsed.prefix, expected_output.prefix, "Unexpected value for `prefix`");
-        ::pretty_assertions::assert_eq!(parsed.delimiter, expected_output.delimiter, "Unexpected value for `delimiter`");
-        ::pretty_assertions::assert_eq!(parsed.max_keys, expected_output.max_keys, "Unexpected value for `max_keys`");
-        ::pretty_assertions::assert_eq!(
-            parsed.common_prefixes,
-            expected_output.common_prefixes,
-            "Unexpected value for `common_prefixes`"
-        );
-        ::pretty_assertions::assert_eq!(
-            parsed.encoding_type,
-            expected_output.encoding_type,
-            "Unexpected value for `encoding_type`"
-        );
-        ::pretty_assertions::assert_eq!(
-            parsed.request_charged,
-            expected_output.request_charged,
-            "Unexpected value for `request_charged`"
-        );
-    }
-}
-
 /// Error type for the `ListObjectsError` operation.
 #[non_exhaustive]
 #[derive(::std::fmt::Debug)]
 pub enum ListObjectsError {
     /// <p>The specified bucket does not exist.</p>
-    NoSuchBucket(crate::types::error::NoSuchBucket),
+    NoSuchBucket(super::types::error::NoSuchBucket),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -498,7 +386,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-ListObjectsError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl ListObjectsError {
     /// Creates the `ListObjectsError::Unhandled` variant from any error type.
@@ -505,7 +393,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -513,7 +401,7 @@

     /// Creates the `ListObjectsError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -576,26 +464,26 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl crate::s3_request_id::RequestIdExt for crate::operation::list_objects::ListObjectsError {
+impl super::s3_request_id::RequestIdExt for super::operation::list_objects::ListObjectsError {
     fn extended_request_id(&self) -> Option<&str> {
         self.meta().extended_request_id()
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::list_objects::ListObjectsError {
+impl ::aws_types::request_id::RequestId for super::operation::list_objects::ListObjectsError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::list_objects::_list_objects_input::ListObjectsInput;
+pub use super::operation::list_objects::_list_objects_input::ListObjectsInput;

-pub use crate::operation::list_objects::_list_objects_output::ListObjectsOutput;
+pub use super::operation::list_objects::_list_objects_output::ListObjectsOutput;

 mod _list_objects_input;

