--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::put_bucket_acl::PutBucketAclInput,
+        input: super::super::operation::put_bucket_acl::PutBucketAclInput,
     ) -> ::std::result::Result<
-        crate::operation::put_bucket_acl::PutBucketAclOutput,
+        super::super::operation::put_bucket_acl::PutBucketAclOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::put_bucket_acl::PutBucketAclError,
+            super::super::operation::put_bucket_acl::PutBucketAclError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::put_bucket_acl::PutBucketAclError>()
+                err.downcast::<super::super::operation::put_bucket_acl::PutBucketAclError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::put_bucket_acl::PutBucketAclOutput>()
+                .downcast::<super::super::operation::put_bucket_acl::PutBucketAclOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::put_bucket_acl::PutBucketAclInput,
+        input: super::super::operation::put_bucket_acl::PutBucketAclInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("PutBucketAcl")
                 .build()
                 .expect("required fields set"),
@@ -103,7 +103,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::orchestrator::Metadata::new("PutBucketAcl", "S3"));
-        cfg.store_put(crate::s3_express::checksum::provide_default_checksum_algorithm());
+        cfg.store_put(super::super::s3_express::checksum::provide_default_checksum_algorithm());
         let mut signing_options = ::aws_runtime::auth::SigningOptions::default();
         signing_options.double_uri_encode = false;
         signing_options.content_sha256_header = true;
@@ -134,9 +134,9 @@
                 PutBucketAclEndpointParamsInterceptor,
             ))
             .with_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(
-                crate::http_request_checksum::RequestChecksumInterceptor::new(
+                super::super::http_request_checksum::RequestChecksumInterceptor::new(
                     |input: &::aws_smithy_runtime_api::client::interceptors::context::Input| {
-                        let input: &crate::operation::put_bucket_acl::PutBucketAclInput = input.downcast_ref().expect("correct type");
+                        let input: &super::super::operation::put_bucket_acl::PutBucketAclInput = input.downcast_ref().expect("correct type");
                         let checksum_algorithm = input.checksum_algorithm();
                         let checksum_algorithm = checksum_algorithm.map(|algorithm| algorithm.as_str());
                         (checksum_algorithm.map(|s| s.to_string()), true)
@@ -166,7 +166,7 @@
                         // From the httpChecksum trait
                         let http_checksum_required = true;

-                        let is_presigned_req = cfg.load::<crate::presigning::PresigningMarker>().is_some();
+                        let is_presigned_req = cfg.load::<super::super::presigning::PresigningMarker>().is_some();

                         // If the request is presigned we do not set a default.
                         // If the RequestChecksumCalculation is WhenSupported and the user has not set a checksum value or algo
@@ -194,13 +194,13 @@
                 ),
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::put_bucket_acl::PutBucketAclError,
+                super::super::operation::put_bucket_acl::PutBucketAclError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::put_bucket_acl::PutBucketAclError,
+                super::super::operation::put_bucket_acl::PutBucketAclError,
             >::new())
             .with_retry_classifier(
-                ::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<crate::operation::put_bucket_acl::PutBucketAclError>::builder()
+                ::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<super::super::operation::put_bucket_acl::PutBucketAclError>::builder()
                     .transient_errors({
                         let mut transient_errors: Vec<&'static str> = ::aws_runtime::retries::classifiers::TRANSIENT_ERRORS.into();
                         transient_errors.push("InternalError");
@@ -310,11 +310,11 @@
         }
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_put_bucket_acl::de_put_bucket_acl_http_error(status, headers, body)
+            super::super::protocol_serde::shape_put_bucket_acl::de_put_bucket_acl_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_put_bucket_acl::de_put_bucket_acl_http_response(status, headers, body)
+            super::super::protocol_serde::shape_put_bucket_acl::de_put_bucket_acl_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -327,16 +327,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::put_bucket_acl::PutBucketAclInput>()
+            .downcast::<super::super::operation::put_bucket_acl::PutBucketAclInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::put_bucket_acl::PutBucketAclInput,
+                _input: &super::super::operation::put_bucket_acl::PutBucketAclInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -344,7 +344,7 @@
                 ::std::result::Result::Ok(())
             }
             fn uri_query(
-                _input: &crate::operation::put_bucket_acl::PutBucketAclInput,
+                _input: &super::super::operation::put_bucket_acl::PutBucketAclInput,
                 mut output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 let mut query = ::aws_smithy_http::query::Writer::new(output);
@@ -353,13 +353,13 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::put_bucket_acl::PutBucketAclInput,
+                input: &super::super::operation::put_bucket_acl::PutBucketAclInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
                 uri_base(input, &mut uri)?;
                 uri_query(input, &mut uri)?;
-                let builder = crate::protocol_serde::shape_put_bucket_acl::ser_put_bucket_acl_headers(input, builder)?;
+                let builder = super::super::protocol_serde::shape_put_bucket_acl::ser_put_bucket_acl_headers(input, builder)?;
                 ::std::result::Result::Ok(builder.method("PUT").uri(uri))
             }
             let mut builder = update_http_builder(&input, ::http_1x::request::Builder::new())?;
@@ -367,7 +367,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_put_bucket_acl_input::ser_access_control_policy_http_payload(&input.access_control_policy)?,
+            super::super::protocol_serde::shape_put_bucket_acl_input::ser_access_control_policy_http_payload(&input.access_control_policy)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -400,16 +400,16 @@
             .downcast_ref::<PutBucketAclInput>()
             .ok_or("failed to downcast to PutBucketAclInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_endpoint(cfg.load::<::aws_types::endpoint_config::EndpointUrl>().map(|ty| ty.0.clone()))
-            .set_force_path_style(cfg.load::<crate::config::ForcePathStyle>().map(|ty| ty.0))
-            .set_use_arn_region(cfg.load::<crate::config::UseArnRegion>().map(|ty| ty.0))
-            .set_disable_multi_region_access_points(cfg.load::<crate::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
-            .set_accelerate(cfg.load::<crate::config::Accelerate>().map(|ty| ty.0))
-            .set_disable_s3_express_session_auth(cfg.load::<crate::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
+            .set_force_path_style(cfg.load::<super::super::config::ForcePathStyle>().map(|ty| ty.0))
+            .set_use_arn_region(cfg.load::<super::super::config::UseArnRegion>().map(|ty| ty.0))
+            .set_disable_multi_region_access_points(cfg.load::<super::super::config::DisableMultiRegionAccessPoints>().map(|ty| ty.0))
+            .set_accelerate(cfg.load::<super::super::config::Accelerate>().map(|ty| ty.0))
+            .set_disable_s3_express_session_auth(cfg.load::<super::super::config::DisableS3ExpressSessionAuth>().map(|ty| ty.0))
             .set_use_s3_express_control_endpoint(Some(true))
             .set_bucket(Some(
                 _input
@@ -442,7 +442,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-PutBucketAclError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl PutBucketAclError {
     /// Creates the `PutBucketAclError::Unhandled` variant from any error type.
@@ -449,7 +449,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -457,7 +457,7 @@

     /// Creates the `PutBucketAclError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -512,26 +512,26 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl crate::s3_request_id::RequestIdExt for crate::operation::put_bucket_acl::PutBucketAclError {
+impl super::super::s3_request_id::RequestIdExt for super::super::operation::put_bucket_acl::PutBucketAclError {
     fn extended_request_id(&self) -> Option<&str> {
         self.meta().extended_request_id()
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::put_bucket_acl::PutBucketAclError {
+impl ::aws_types::request_id::RequestId for super::super::operation::put_bucket_acl::PutBucketAclError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::put_bucket_acl::_put_bucket_acl_input::PutBucketAclInput;
+pub use super::super::operation::put_bucket_acl::_put_bucket_acl_input::PutBucketAclInput;

-pub use crate::operation::put_bucket_acl::_put_bucket_acl_output::PutBucketAclOutput;
+pub use super::super::operation::put_bucket_acl::_put_bucket_acl_output::PutBucketAclOutput;

 mod _put_bucket_acl_input;

