--- original
+++ modified
@@ -62,8 +62,8 @@
 }
 impl StartLiveTailInput {
     /// Creates a new builder-style object to manufacture [`StartLiveTailInput`](crate::operation::start_live_tail::StartLiveTailInput).
-    pub fn builder() -> crate::operation::start_live_tail::builders::StartLiveTailInputBuilder {
-        crate::operation::start_live_tail::builders::StartLiveTailInputBuilder::default()
+    pub fn builder() -> super::operation::start_live_tail::builders::StartLiveTailInputBuilder {
+        super::operation::start_live_tail::builders::StartLiveTailInputBuilder::default()
     }
 }

@@ -190,8 +190,8 @@
     /// Consumes the builder and constructs a [`StartLiveTailInput`](crate::operation::start_live_tail::StartLiveTailInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::start_live_tail::StartLiveTailInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::start_live_tail::StartLiveTailInput {
+    ) -> ::std::result::Result<super::operation::start_live_tail::StartLiveTailInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::start_live_tail::StartLiveTailInput {
             log_group_identifiers: self.log_group_identifiers,
             log_stream_names: self.log_stream_names,
             log_stream_name_prefixes: self.log_stream_name_prefixes,
