--- original
+++ modified
@@ -34,8 +34,8 @@
 }
 impl DeleteFunctionCodeSigningConfigInput {
     /// Creates a new builder-style object to manufacture [`DeleteFunctionCodeSigningConfigInput`](crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigInput).
-    pub fn builder() -> crate::operation::delete_function_code_signing_config::builders::DeleteFunctionCodeSigningConfigInputBuilder {
-        crate::operation::delete_function_code_signing_config::builders::DeleteFunctionCodeSigningConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_function_code_signing_config::builders::DeleteFunctionCodeSigningConfigInputBuilder {
+        super::super::super::operation::delete_function_code_signing_config::builders::DeleteFunctionCodeSigningConfigInputBuilder::default()
     }
 }

@@ -95,11 +95,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigInput,
+        super::super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigInput {
+            super::super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigInput {
                 function_name: self.function_name,
             },
         )
