--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct UpdateReputationEntityPolicyInput {
     /// <p>The type of reputation entity. Currently, only <code>RESOURCE</code> type entities are supported.</p>
-    pub reputation_entity_type: ::std::option::Option<crate::types::ReputationEntityType>,
+    pub reputation_entity_type: ::std::option::Option<super::types::ReputationEntityType>,
     /// <p>The unique identifier for the reputation entity. For resource-type entities, this is the Amazon Resource Name (ARN) of the resource.</p>
     pub reputation_entity_reference: ::std::option::Option<::std::string::String>,
     /// <p>The Amazon Resource Name (ARN) of the reputation management policy to apply to this entity. This is an Amazon Web Services Amazon SES-managed policy.</p>
@@ -13,7 +13,7 @@
 }
 impl UpdateReputationEntityPolicyInput {
     /// <p>The type of reputation entity. Currently, only <code>RESOURCE</code> type entities are supported.</p>
-    pub fn reputation_entity_type(&self) -> ::std::option::Option<&crate::types::ReputationEntityType> {
+    pub fn reputation_entity_type(&self) -> ::std::option::Option<&super::types::ReputationEntityType> {
         self.reputation_entity_type.as_ref()
     }
     /// <p>The unique identifier for the reputation entity. For resource-type entities, this is the Amazon Resource Name (ARN) of the resource.</p>
@@ -27,8 +27,8 @@
 }
 impl UpdateReputationEntityPolicyInput {
     /// Creates a new builder-style object to manufacture [`UpdateReputationEntityPolicyInput`](crate::operation::update_reputation_entity_policy::UpdateReputationEntityPolicyInput).
-    pub fn builder() -> crate::operation::update_reputation_entity_policy::builders::UpdateReputationEntityPolicyInputBuilder {
-        crate::operation::update_reputation_entity_policy::builders::UpdateReputationEntityPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::update_reputation_entity_policy::builders::UpdateReputationEntityPolicyInputBuilder {
+        super::operation::update_reputation_entity_policy::builders::UpdateReputationEntityPolicyInputBuilder::default()
     }
 }

@@ -36,7 +36,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct UpdateReputationEntityPolicyInputBuilder {
-    pub(crate) reputation_entity_type: ::std::option::Option<crate::types::ReputationEntityType>,
+    pub(crate) reputation_entity_type: ::std::option::Option<super::types::ReputationEntityType>,
     pub(crate) reputation_entity_reference: ::std::option::Option<::std::string::String>,
     pub(crate) reputation_entity_policy: ::std::option::Option<::std::string::String>,
 }
@@ -43,17 +43,17 @@
 impl UpdateReputationEntityPolicyInputBuilder {
     /// <p>The type of reputation entity. Currently, only <code>RESOURCE</code> type entities are supported.</p>
     /// This field is required.
-    pub fn reputation_entity_type(mut self, input: crate::types::ReputationEntityType) -> Self {
+    pub fn reputation_entity_type(mut self, input: super::types::ReputationEntityType) -> Self {
         self.reputation_entity_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of reputation entity. Currently, only <code>RESOURCE</code> type entities are supported.</p>
-    pub fn set_reputation_entity_type(mut self, input: ::std::option::Option<crate::types::ReputationEntityType>) -> Self {
+    pub fn set_reputation_entity_type(mut self, input: ::std::option::Option<super::types::ReputationEntityType>) -> Self {
         self.reputation_entity_type = input;
         self
     }
     /// <p>The type of reputation entity. Currently, only <code>RESOURCE</code> type entities are supported.</p>
-    pub fn get_reputation_entity_type(&self) -> &::std::option::Option<crate::types::ReputationEntityType> {
+    pub fn get_reputation_entity_type(&self) -> &::std::option::Option<super::types::ReputationEntityType> {
         &self.reputation_entity_type
     }
     /// <p>The unique identifier for the reputation entity. For resource-type entities, this is the Amazon Resource Name (ARN) of the resource.</p>
@@ -90,10 +90,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_reputation_entity_policy::UpdateReputationEntityPolicyInput,
+        super::operation::update_reputation_entity_policy::UpdateReputationEntityPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_reputation_entity_policy::UpdateReputationEntityPolicyInput {
+        ::std::result::Result::Ok(super::operation::update_reputation_entity_policy::UpdateReputationEntityPolicyInput {
             reputation_entity_type: self.reputation_entity_type,
             reputation_entity_reference: self.reputation_entity_reference,
             reputation_entity_policy: self.reputation_entity_policy,
