--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::QueueSnapshotUtilizationDetail>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::QueueSnapshotUtilizationDetail>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::QueueSnapshotUtilizationDetailBuilder::default();
+            let mut builder = super::types::builders::QueueSnapshotUtilizationDetailBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -23,7 +23,7 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                         "totalCapacityUsage" => {
                             builder = builder.set_total_capacity_usage(
-                                crate::protocol_serde::shape_queue_snapshot_capacity_usage_list::de_queue_snapshot_capacity_usage_list(
+                                super::protocol_serde::shape_queue_snapshot_capacity_usage_list::de_queue_snapshot_capacity_usage_list(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -32,7 +32,7 @@
                         }
                         "fairshareUtilization" => {
                             builder = builder.set_fairshare_utilization(
-                                crate::protocol_serde::shape_fairshare_utilization_detail::de_fairshare_utilization_detail(
+                                super::protocol_serde::shape_fairshare_utilization_detail::de_fairshare_utilization_detail(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -41,7 +41,7 @@
                         }
                         "quotaShareUtilization" => {
                             builder = builder.set_quota_share_utilization(
-                                crate::protocol_serde::shape_quota_share_utilization_detail::de_quota_share_utilization_detail(
+                                super::protocol_serde::shape_quota_share_utilization_detail::de_quota_share_utilization_detail(
                                     tokens,
                                     _value,
                                     depth + 1,
