--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::terminate_service_job::TerminateServiceJobOutput,
-    crate::operation::terminate_service_job::TerminateServiceJobError,
+    super::operation::terminate_service_job::TerminateServiceJobOutput,
+    super::operation::terminate_service_job::TerminateServiceJobError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::terminate_service_job::TerminateServiceJobError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::terminate_service_job::TerminateServiceJobError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::terminate_service_job::TerminateServiceJobError::unhandled(generic)),
+        None => return Err(super::operation::terminate_service_job::TerminateServiceJobError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ClientException" => crate::operation::terminate_service_job::TerminateServiceJobError::ClientException({
+        "ClientException" => super::operation::terminate_service_job::TerminateServiceJobError::ClientException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ClientExceptionBuilder::default();
-                output = crate::protocol_serde::shape_client_exception::de_client_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::terminate_service_job::TerminateServiceJobError::unhandled)?;
+                let mut output = super::types::error::builders::ClientExceptionBuilder::default();
+                output = super::protocol_serde::shape_client_exception::de_client_exception_json_err(_response_body, output)
+                    .map_err(super::operation::terminate_service_job::TerminateServiceJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "ServerException" => crate::operation::terminate_service_job::TerminateServiceJobError::ServerException({
+        "ServerException" => super::operation::terminate_service_job::TerminateServiceJobError::ServerException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServerExceptionBuilder::default();
-                output = crate::protocol_serde::shape_server_exception::de_server_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::terminate_service_job::TerminateServiceJobError::unhandled)?;
+                let mut output = super::types::error::builders::ServerExceptionBuilder::default();
+                output = super::protocol_serde::shape_server_exception::de_server_exception_json_err(_response_body, output)
+                    .map_err(super::operation::terminate_service_job::TerminateServiceJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,7 +50,7 @@
             }
             tmp
         }),
-        _ => crate::operation::terminate_service_job::TerminateServiceJobError::generic(generic),
+        _ => super::operation::terminate_service_job::TerminateServiceJobError::generic(generic),
     })
 }

@@ -60,12 +60,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::terminate_service_job::TerminateServiceJobOutput,
-    crate::operation::terminate_service_job::TerminateServiceJobError,
+    super::operation::terminate_service_job::TerminateServiceJobOutput,
+    super::operation::terminate_service_job::TerminateServiceJobError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::terminate_service_job::builders::TerminateServiceJobOutputBuilder::default();
+        let mut output = super::operation::terminate_service_job::builders::TerminateServiceJobOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -72,11 +72,11 @@
 }

 pub fn ser_terminate_service_job_input(
-    input: &crate::operation::terminate_service_job::TerminateServiceJobInput,
+    input: &super::operation::terminate_service_job::TerminateServiceJobInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_terminate_service_job_input::ser_terminate_service_job_input_input(&mut object, input)?;
+    super::protocol_serde::shape_terminate_service_job_input::ser_terminate_service_job_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
