--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::vec::Vec<crate::types::EksContainerEnvironmentVariable>>,
+    Option<::std::vec::Vec<super::super::types::EksContainerEnvironmentVariable>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -26,7 +26,7 @@
                         break;
                     }
                     _ => {
-                        let value = crate::protocol_serde::shape_eks_container_environment_variable::de_eks_container_environment_variable(
+                        let value = super::super::protocol_serde::shape_eks_container_environment_variable::de_eks_container_environment_variable(
                             tokens,
                             _value,
                             depth + 1,
