--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::JobDetail>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::JobDetail>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::JobDetailBuilder::default();
+            let mut builder = super::super::types::builders::JobDetailBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -52,7 +52,7 @@
                         "status" => {
                             builder = builder.set_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::JobStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::JobStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -71,7 +71,7 @@
                             );
                         }
                         "attempts" => {
-                            builder = builder.set_attempts(crate::protocol_serde::shape_attempt_details::de_attempt_details(
+                            builder = builder.set_attempts(super::super::protocol_serde::shape_attempt_details::de_attempt_details(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -92,7 +92,7 @@
                             );
                         }
                         "retryStrategy" => {
-                            builder = builder.set_retry_strategy(crate::protocol_serde::shape_retry_strategy::de_retry_strategy(
+                            builder = builder.set_retry_strategy(super::super::protocol_serde::shape_retry_strategy::de_retry_strategy(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -113,7 +113,7 @@
                             );
                         }
                         "dependsOn" => {
-                            builder = builder.set_depends_on(crate::protocol_serde::shape_job_dependency_list::de_job_dependency_list(
+                            builder = builder.set_depends_on(super::super::protocol_serde::shape_job_dependency_list::de_job_dependency_list(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -128,10 +128,10 @@
                         }
                         "parameters" => {
                             builder =
-                                builder.set_parameters(crate::protocol_serde::shape_parameters_map::de_parameters_map(tokens, _value, depth + 1)?);
+                                builder.set_parameters(super::super::protocol_serde::shape_parameters_map::de_parameters_map(tokens, _value, depth + 1)?);
                         }
                         "container" => {
-                            builder = builder.set_container(crate::protocol_serde::shape_container_detail::de_container_detail(
+                            builder = builder.set_container(super::super::protocol_serde::shape_container_detail::de_container_detail(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -139,10 +139,10 @@
                         }
                         "nodeDetails" => {
                             builder =
-                                builder.set_node_details(crate::protocol_serde::shape_node_details::de_node_details(tokens, _value, depth + 1)?);
+                                builder.set_node_details(super::super::protocol_serde::shape_node_details::de_node_details(tokens, _value, depth + 1)?);
                         }
                         "nodeProperties" => {
-                            builder = builder.set_node_properties(crate::protocol_serde::shape_node_properties::de_node_properties(
+                            builder = builder.set_node_properties(super::super::protocol_serde::shape_node_properties::de_node_properties(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -149,7 +149,7 @@
                             )?);
                         }
                         "arrayProperties" => {
-                            builder = builder.set_array_properties(crate::protocol_serde::shape_array_properties_detail::de_array_properties_detail(
+                            builder = builder.set_array_properties(super::super::protocol_serde::shape_array_properties_detail::de_array_properties_detail(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -156,10 +156,10 @@
                             )?);
                         }
                         "timeout" => {
-                            builder = builder.set_timeout(crate::protocol_serde::shape_job_timeout::de_job_timeout(tokens, _value, depth + 1)?);
+                            builder = builder.set_timeout(super::super::protocol_serde::shape_job_timeout::de_job_timeout(tokens, _value, depth + 1)?);
                         }
                         "tags" => {
-                            builder = builder.set_tags(crate::protocol_serde::shape_tagris_tags_map::de_tagris_tags_map(
+                            builder = builder.set_tags(super::super::protocol_serde::shape_tagris_tags_map::de_tagris_tags_map(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -170,11 +170,11 @@
                         }
                         "platformCapabilities" => {
                             builder = builder.set_platform_capabilities(
-                                crate::protocol_serde::shape_platform_capability_list::de_platform_capability_list(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_platform_capability_list::de_platform_capability_list(tokens, _value, depth + 1)?,
                             );
                         }
                         "eksProperties" => {
-                            builder = builder.set_eks_properties(crate::protocol_serde::shape_eks_properties_detail::de_eks_properties_detail(
+                            builder = builder.set_eks_properties(super::super::protocol_serde::shape_eks_properties_detail::de_eks_properties_detail(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -181,7 +181,7 @@
                             )?);
                         }
                         "eksAttempts" => {
-                            builder = builder.set_eks_attempts(crate::protocol_serde::shape_eks_attempt_details::de_eks_attempt_details(
+                            builder = builder.set_eks_attempts(super::super::protocol_serde::shape_eks_attempt_details::de_eks_attempt_details(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -188,7 +188,7 @@
                             )?);
                         }
                         "ecsProperties" => {
-                            builder = builder.set_ecs_properties(crate::protocol_serde::shape_ecs_properties_detail::de_ecs_properties_detail(
+                            builder = builder.set_ecs_properties(super::super::protocol_serde::shape_ecs_properties_detail::de_ecs_properties_detail(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -202,7 +202,7 @@
                         }
                         "consumableResourceProperties" => {
                             builder = builder.set_consumable_resource_properties(
-                                crate::protocol_serde::shape_consumable_resource_properties::de_consumable_resource_properties(
+                                super::super::protocol_serde::shape_consumable_resource_properties::de_consumable_resource_properties(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -218,7 +218,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::job_detail_correct_errors(builder).build()))
+            Ok(Some(super::super::serde_util::job_detail_correct_errors(builder).build()))
         }
         _ => Err(::aws_smithy_json::deserialize::error::DeserializeError::custom(
             "expected start object or null",
