--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ServiceJobPreemptionSummary>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ServiceJobPreemptionSummary>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ServiceJobPreemptionSummaryBuilder::default();
+            let mut builder = super::types::builders::ServiceJobPreemptionSummaryBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,7 +31,7 @@
                             }
                             "recentPreemptedAttempts" => {
                                 builder = builder.set_recent_preempted_attempts(
-                                    crate::protocol_serde::shape_service_job_recent_preempted_attempt_list::de_service_job_recent_preempted_attempt_list(tokens, _value, depth + 1)?
+                                    super::protocol_serde::shape_service_job_recent_preempted_attempt_list::de_service_job_recent_preempted_attempt_list(tokens, _value, depth + 1)?
                                 );
                             }
                             _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
