--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::JobDefinition>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::JobDefinition>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::JobDefinitionBuilder::default();
+            let mut builder = super::types::builders::JobDefinitionBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -65,10 +65,10 @@
                         }
                         "parameters" => {
                             builder =
-                                builder.set_parameters(crate::protocol_serde::shape_parameters_map::de_parameters_map(tokens, _value, depth + 1)?);
+                                builder.set_parameters(super::protocol_serde::shape_parameters_map::de_parameters_map(tokens, _value, depth + 1)?);
                         }
                         "retryStrategy" => {
-                            builder = builder.set_retry_strategy(crate::protocol_serde::shape_retry_strategy::de_retry_strategy(
+                            builder = builder.set_retry_strategy(super::protocol_serde::shape_retry_strategy::de_retry_strategy(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -75,7 +75,7 @@
                             )?);
                         }
                         "containerProperties" => {
-                            builder = builder.set_container_properties(crate::protocol_serde::shape_container_properties::de_container_properties(
+                            builder = builder.set_container_properties(super::protocol_serde::shape_container_properties::de_container_properties(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -82,10 +82,10 @@
                             )?);
                         }
                         "timeout" => {
-                            builder = builder.set_timeout(crate::protocol_serde::shape_job_timeout::de_job_timeout(tokens, _value, depth + 1)?);
+                            builder = builder.set_timeout(super::protocol_serde::shape_job_timeout::de_job_timeout(tokens, _value, depth + 1)?);
                         }
                         "nodeProperties" => {
-                            builder = builder.set_node_properties(crate::protocol_serde::shape_node_properties::de_node_properties(
+                            builder = builder.set_node_properties(super::protocol_serde::shape_node_properties::de_node_properties(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -92,7 +92,7 @@
                             )?);
                         }
                         "tags" => {
-                            builder = builder.set_tags(crate::protocol_serde::shape_tagris_tags_map::de_tagris_tags_map(
+                            builder = builder.set_tags(super::protocol_serde::shape_tagris_tags_map::de_tagris_tags_map(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -103,11 +103,11 @@
                         }
                         "platformCapabilities" => {
                             builder = builder.set_platform_capabilities(
-                                crate::protocol_serde::shape_platform_capability_list::de_platform_capability_list(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_platform_capability_list::de_platform_capability_list(tokens, _value, depth + 1)?,
                             );
                         }
                         "ecsProperties" => {
-                            builder = builder.set_ecs_properties(crate::protocol_serde::shape_ecs_properties::de_ecs_properties(
+                            builder = builder.set_ecs_properties(super::protocol_serde::shape_ecs_properties::de_ecs_properties(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -114,7 +114,7 @@
                             )?);
                         }
                         "eksProperties" => {
-                            builder = builder.set_eks_properties(crate::protocol_serde::shape_eks_properties::de_eks_properties(
+                            builder = builder.set_eks_properties(super::protocol_serde::shape_eks_properties::de_eks_properties(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -123,13 +123,13 @@
                         "containerOrchestrationType" => {
                             builder = builder.set_container_orchestration_type(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::OrchestrationType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::OrchestrationType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "consumableResourceProperties" => {
                             builder = builder.set_consumable_resource_properties(
-                                crate::protocol_serde::shape_consumable_resource_properties::de_consumable_resource_properties(
+                                super::protocol_serde::shape_consumable_resource_properties::de_consumable_resource_properties(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -145,7 +145,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::job_definition_correct_errors(builder).build()))
+            Ok(Some(super::serde_util::job_definition_correct_errors(builder).build()))
         }
         _ => Err(::aws_smithy_json::deserialize::error::DeserializeError::custom(
             "expected start object or null",
