--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::terminate_job::TerminateJobOutput, crate::operation::terminate_job::TerminateJobError> {
+) -> std::result::Result<super::operation::terminate_job::TerminateJobOutput, super::operation::terminate_job::TerminateJobError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::terminate_job::TerminateJobError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::terminate_job::TerminateJobError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::terminate_job::TerminateJobError::unhandled(generic)),
+        None => return Err(super::operation::terminate_job::TerminateJobError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ClientException" => crate::operation::terminate_job::TerminateJobError::ClientException({
+        "ClientException" => super::operation::terminate_job::TerminateJobError::ClientException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ClientExceptionBuilder::default();
-                output = crate::protocol_serde::shape_client_exception::de_client_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::terminate_job::TerminateJobError::unhandled)?;
+                let mut output = super::types::error::builders::ClientExceptionBuilder::default();
+                output = super::protocol_serde::shape_client_exception::de_client_exception_json_err(_response_body, output)
+                    .map_err(super::operation::terminate_job::TerminateJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "ServerException" => crate::operation::terminate_job::TerminateJobError::ServerException({
+        "ServerException" => super::operation::terminate_job::TerminateJobError::ServerException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServerExceptionBuilder::default();
-                output = crate::protocol_serde::shape_server_exception::de_server_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::terminate_job::TerminateJobError::unhandled)?;
+                let mut output = super::types::error::builders::ServerExceptionBuilder::default();
+                output = super::protocol_serde::shape_server_exception::de_server_exception_json_err(_response_body, output)
+                    .map_err(super::operation::terminate_job::TerminateJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,7 +47,7 @@
             }
             tmp
         }),
-        _ => crate::operation::terminate_job::TerminateJobError::generic(generic),
+        _ => super::operation::terminate_job::TerminateJobError::generic(generic),
     })
 }

@@ -56,10 +56,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::terminate_job::TerminateJobOutput, crate::operation::terminate_job::TerminateJobError> {
+) -> std::result::Result<super::operation::terminate_job::TerminateJobOutput, super::operation::terminate_job::TerminateJobError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::terminate_job::builders::TerminateJobOutputBuilder::default();
+        let mut output = super::operation::terminate_job::builders::TerminateJobOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -66,11 +66,11 @@
 }

 pub fn ser_terminate_job_input(
-    input: &crate::operation::terminate_job::TerminateJobInput,
+    input: &super::operation::terminate_job::TerminateJobInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_terminate_job_input::ser_terminate_job_input_input(&mut object, input)?;
+    super::protocol_serde::shape_terminate_job_input::ser_terminate_job_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
