--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::EksContainerDetail>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::EksContainerDetail>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::EksContainerDetailBuilder::default();
+            let mut builder = super::types::builders::EksContainerDetailBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -43,14 +43,14 @@
                             );
                         }
                         "command" => {
-                            builder = builder.set_command(crate::protocol_serde::shape_string_list::de_string_list(tokens, _value, depth + 1)?);
+                            builder = builder.set_command(super::protocol_serde::shape_string_list::de_string_list(tokens, _value, depth + 1)?);
                         }
                         "args" => {
-                            builder = builder.set_args(crate::protocol_serde::shape_string_list::de_string_list(tokens, _value, depth + 1)?);
+                            builder = builder.set_args(super::protocol_serde::shape_string_list::de_string_list(tokens, _value, depth + 1)?);
                         }
                         "env" => {
                             builder = builder.set_env(
-                                crate::protocol_serde::shape_eks_container_environment_variables::de_eks_container_environment_variables(
+                                super::protocol_serde::shape_eks_container_environment_variables::de_eks_container_environment_variables(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -59,7 +59,7 @@
                         }
                         "resources" => {
                             builder = builder.set_resources(
-                                crate::protocol_serde::shape_eks_container_resource_requirements::de_eks_container_resource_requirements(
+                                super::protocol_serde::shape_eks_container_resource_requirements::de_eks_container_resource_requirements(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -82,12 +82,12 @@
                         }
                         "volumeMounts" => {
                             builder = builder.set_volume_mounts(
-                                crate::protocol_serde::shape_eks_container_volume_mounts::de_eks_container_volume_mounts(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_eks_container_volume_mounts::de_eks_container_volume_mounts(tokens, _value, depth + 1)?,
                             );
                         }
                         "securityContext" => {
                             builder = builder.set_security_context(
-                                crate::protocol_serde::shape_eks_container_security_context::de_eks_container_security_context(
+                                super::protocol_serde::shape_eks_container_security_context::de_eks_container_security_context(
                                     tokens,
                                     _value,
                                     depth + 1,
