--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::deregister_job_definition::DeregisterJobDefinitionOutput,
-    crate::operation::deregister_job_definition::DeregisterJobDefinitionError,
+    super::super::operation::deregister_job_definition::DeregisterJobDefinitionOutput,
+    super::super::operation::deregister_job_definition::DeregisterJobDefinitionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled(
+            return Err(super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ClientException" => crate::operation::deregister_job_definition::DeregisterJobDefinitionError::ClientException({
+        "ClientException" => super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::ClientException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ClientExceptionBuilder::default();
-                output = crate::protocol_serde::shape_client_exception::de_client_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ClientExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_client_exception::de_client_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "ServerException" => crate::operation::deregister_job_definition::DeregisterJobDefinitionError::ServerException({
+        "ServerException" => super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::ServerException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServerExceptionBuilder::default();
-                output = crate::protocol_serde::shape_server_exception::de_server_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServerExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_server_exception::de_server_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,7 +54,7 @@
             }
             tmp
         }),
-        _ => crate::operation::deregister_job_definition::DeregisterJobDefinitionError::generic(generic),
+        _ => super::super::operation::deregister_job_definition::DeregisterJobDefinitionError::generic(generic),
     })
 }

@@ -64,12 +64,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::deregister_job_definition::DeregisterJobDefinitionOutput,
-    crate::operation::deregister_job_definition::DeregisterJobDefinitionError,
+    super::super::operation::deregister_job_definition::DeregisterJobDefinitionOutput,
+    super::super::operation::deregister_job_definition::DeregisterJobDefinitionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::deregister_job_definition::builders::DeregisterJobDefinitionOutputBuilder::default();
+        let mut output = super::super::operation::deregister_job_definition::builders::DeregisterJobDefinitionOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -76,11 +76,11 @@
 }

 pub fn ser_deregister_job_definition_input(
-    input: &crate::operation::deregister_job_definition::DeregisterJobDefinitionInput,
+    input: &super::super::operation::deregister_job_definition::DeregisterJobDefinitionInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_deregister_job_definition_input::ser_deregister_job_definition_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_deregister_job_definition_input::ser_deregister_job_definition_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
