--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetDeliveryInput {
     /// Creates a new builder-style object to manufacture [`GetDeliveryInput`](crate::operation::get_delivery::GetDeliveryInput).
-    pub fn builder() -> crate::operation::get_delivery::builders::GetDeliveryInputBuilder {
-        crate::operation::get_delivery::builders::GetDeliveryInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_delivery::builders::GetDeliveryInputBuilder {
+        super::super::super::operation::get_delivery::builders::GetDeliveryInputBuilder::default()
     }
 }

@@ -42,7 +42,7 @@
         &self.id
     }
     /// Consumes the builder and constructs a [`GetDeliveryInput`](crate::operation::get_delivery::GetDeliveryInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::get_delivery::GetDeliveryInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_delivery::GetDeliveryInput { id: self.id })
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::get_delivery::GetDeliveryInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_delivery::GetDeliveryInput { id: self.id })
     }
 }
