--- original
+++ modified
@@ -46,8 +46,8 @@
 }
 impl RevokeGrantInput {
     /// Creates a new builder-style object to manufacture [`RevokeGrantInput`](crate::operation::revoke_grant::RevokeGrantInput).
-    pub fn builder() -> crate::operation::revoke_grant::builders::RevokeGrantInputBuilder {
-        crate::operation::revoke_grant::builders::RevokeGrantInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::revoke_grant::builders::RevokeGrantInputBuilder {
+        super::super::super::operation::revoke_grant::builders::RevokeGrantInputBuilder::default()
     }
 }

@@ -135,8 +135,8 @@
         &self.dry_run
     }
     /// Consumes the builder and constructs a [`RevokeGrantInput`](crate::operation::revoke_grant::RevokeGrantInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::revoke_grant::RevokeGrantInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::revoke_grant::RevokeGrantInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::revoke_grant::RevokeGrantInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::revoke_grant::RevokeGrantInput {
             key_id: self.key_id,
             grant_id: self.grant_id,
             dry_run: self.dry_run,
