--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl GetUserInput {
     /// Creates a new builder-style object to manufacture [`GetUserInput`](crate::operation::get_user::GetUserInput).
-    pub fn builder() -> crate::operation::get_user::builders::GetUserInputBuilder {
-        crate::operation::get_user::builders::GetUserInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_user::builders::GetUserInputBuilder {
+        super::super::super::operation::get_user::builders::GetUserInputBuilder::default()
     }
 }

@@ -46,7 +46,7 @@
         &self.user_name
     }
     /// Consumes the builder and constructs a [`GetUserInput`](crate::operation::get_user::GetUserInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::get_user::GetUserInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_user::GetUserInput { user_name: self.user_name })
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::get_user::GetUserInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_user::GetUserInput { user_name: self.user_name })
     }
 }
