--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetDomainPermissionsPolicyInput {
     /// Creates a new builder-style object to manufacture [`GetDomainPermissionsPolicyInput`](crate::operation::get_domain_permissions_policy::GetDomainPermissionsPolicyInput).
-    pub fn builder() -> crate::operation::get_domain_permissions_policy::builders::GetDomainPermissionsPolicyInputBuilder {
-        crate::operation::get_domain_permissions_policy::builders::GetDomainPermissionsPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::get_domain_permissions_policy::builders::GetDomainPermissionsPolicyInputBuilder {
+        super::operation::get_domain_permissions_policy::builders::GetDomainPermissionsPolicyInputBuilder::default()
     }
 }

@@ -66,10 +66,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_domain_permissions_policy::GetDomainPermissionsPolicyInput,
+        super::operation::get_domain_permissions_policy::GetDomainPermissionsPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_domain_permissions_policy::GetDomainPermissionsPolicyInput {
+        ::std::result::Result::Ok(super::operation::get_domain_permissions_policy::GetDomainPermissionsPolicyInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
         })
