--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListSamlProviderTagsInput {
     /// Creates a new builder-style object to manufacture [`ListSamlProviderTagsInput`](crate::operation::list_saml_provider_tags::ListSamlProviderTagsInput).
-    pub fn builder() -> crate::operation::list_saml_provider_tags::builders::ListSamlProviderTagsInputBuilder {
-        crate::operation::list_saml_provider_tags::builders::ListSamlProviderTagsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_saml_provider_tags::builders::ListSamlProviderTagsInputBuilder {
+        super::super::super::operation::list_saml_provider_tags::builders::ListSamlProviderTagsInputBuilder::default()
     }
 }

@@ -96,9 +96,9 @@
     /// Consumes the builder and constructs a [`ListSamlProviderTagsInput`](crate::operation::list_saml_provider_tags::ListSamlProviderTagsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_saml_provider_tags::ListSamlProviderTagsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_saml_provider_tags::ListSamlProviderTagsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_saml_provider_tags::ListSamlProviderTagsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_saml_provider_tags::ListSamlProviderTagsInput {
             saml_provider_arn: self.saml_provider_arn,
             marker: self.marker,
             max_items: self.max_items,
