--- original
+++ modified
@@ -13,9 +13,9 @@
     /// <li>
     /// <p><code>EXTERNAL</code> – Configure DKIM for the identity by using Bring Your Own DKIM (BYODKIM).</p></li>
     /// </ul>
-    pub signing_attributes_origin: ::std::option::Option<crate::types::DkimSigningAttributesOrigin>,
+    pub signing_attributes_origin: ::std::option::Option<super::types::DkimSigningAttributesOrigin>,
     /// <p>An object that contains information about the private key and selector that you want to use to configure DKIM for the identity for Bring Your Own DKIM (BYODKIM) for the identity, or, configures the key length to be used for <a href="https://docs.aws.amazon.com/ses/latest/DeveloperGuide/easy-dkim.html">Easy DKIM</a>.</p>
-    pub signing_attributes: ::std::option::Option<crate::types::DkimSigningAttributes>,
+    pub signing_attributes: ::std::option::Option<super::types::DkimSigningAttributes>,
 }
 impl PutEmailIdentityDkimSigningAttributesInput {
     /// <p>The email identity.</p>
@@ -29,18 +29,18 @@
     /// <li>
     /// <p><code>EXTERNAL</code> – Configure DKIM for the identity by using Bring Your Own DKIM (BYODKIM).</p></li>
     /// </ul>
-    pub fn signing_attributes_origin(&self) -> ::std::option::Option<&crate::types::DkimSigningAttributesOrigin> {
+    pub fn signing_attributes_origin(&self) -> ::std::option::Option<&super::types::DkimSigningAttributesOrigin> {
         self.signing_attributes_origin.as_ref()
     }
     /// <p>An object that contains information about the private key and selector that you want to use to configure DKIM for the identity for Bring Your Own DKIM (BYODKIM) for the identity, or, configures the key length to be used for <a href="https://docs.aws.amazon.com/ses/latest/DeveloperGuide/easy-dkim.html">Easy DKIM</a>.</p>
-    pub fn signing_attributes(&self) -> ::std::option::Option<&crate::types::DkimSigningAttributes> {
+    pub fn signing_attributes(&self) -> ::std::option::Option<&super::types::DkimSigningAttributes> {
         self.signing_attributes.as_ref()
     }
 }
 impl PutEmailIdentityDkimSigningAttributesInput {
     /// Creates a new builder-style object to manufacture [`PutEmailIdentityDkimSigningAttributesInput`](crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput).
-    pub fn builder() -> crate::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesInputBuilder {
-        crate::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesInputBuilder::default()
+    pub fn builder() -> super::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesInputBuilder {
+        super::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesInputBuilder::default()
     }
 }

@@ -49,8 +49,8 @@
 #[non_exhaustive]
 pub struct PutEmailIdentityDkimSigningAttributesInputBuilder {
     pub(crate) email_identity: ::std::option::Option<::std::string::String>,
-    pub(crate) signing_attributes_origin: ::std::option::Option<crate::types::DkimSigningAttributesOrigin>,
-    pub(crate) signing_attributes: ::std::option::Option<crate::types::DkimSigningAttributes>,
+    pub(crate) signing_attributes_origin: ::std::option::Option<super::types::DkimSigningAttributesOrigin>,
+    pub(crate) signing_attributes: ::std::option::Option<super::types::DkimSigningAttributes>,
 }
 impl PutEmailIdentityDkimSigningAttributesInputBuilder {
     /// <p>The email identity.</p>
@@ -76,7 +76,7 @@
     /// <p><code>EXTERNAL</code> – Configure DKIM for the identity by using Bring Your Own DKIM (BYODKIM).</p></li>
     /// </ul>
     /// This field is required.
-    pub fn signing_attributes_origin(mut self, input: crate::types::DkimSigningAttributesOrigin) -> Self {
+    pub fn signing_attributes_origin(mut self, input: super::types::DkimSigningAttributesOrigin) -> Self {
         self.signing_attributes_origin = ::std::option::Option::Some(input);
         self
     }
@@ -87,7 +87,7 @@
     /// <li>
     /// <p><code>EXTERNAL</code> – Configure DKIM for the identity by using Bring Your Own DKIM (BYODKIM).</p></li>
     /// </ul>
-    pub fn set_signing_attributes_origin(mut self, input: ::std::option::Option<crate::types::DkimSigningAttributesOrigin>) -> Self {
+    pub fn set_signing_attributes_origin(mut self, input: ::std::option::Option<super::types::DkimSigningAttributesOrigin>) -> Self {
         self.signing_attributes_origin = input;
         self
     }
@@ -98,21 +98,21 @@
     /// <li>
     /// <p><code>EXTERNAL</code> – Configure DKIM for the identity by using Bring Your Own DKIM (BYODKIM).</p></li>
     /// </ul>
-    pub fn get_signing_attributes_origin(&self) -> &::std::option::Option<crate::types::DkimSigningAttributesOrigin> {
+    pub fn get_signing_attributes_origin(&self) -> &::std::option::Option<super::types::DkimSigningAttributesOrigin> {
         &self.signing_attributes_origin
     }
     /// <p>An object that contains information about the private key and selector that you want to use to configure DKIM for the identity for Bring Your Own DKIM (BYODKIM) for the identity, or, configures the key length to be used for <a href="https://docs.aws.amazon.com/ses/latest/DeveloperGuide/easy-dkim.html">Easy DKIM</a>.</p>
-    pub fn signing_attributes(mut self, input: crate::types::DkimSigningAttributes) -> Self {
+    pub fn signing_attributes(mut self, input: super::types::DkimSigningAttributes) -> Self {
         self.signing_attributes = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains information about the private key and selector that you want to use to configure DKIM for the identity for Bring Your Own DKIM (BYODKIM) for the identity, or, configures the key length to be used for <a href="https://docs.aws.amazon.com/ses/latest/DeveloperGuide/easy-dkim.html">Easy DKIM</a>.</p>
-    pub fn set_signing_attributes(mut self, input: ::std::option::Option<crate::types::DkimSigningAttributes>) -> Self {
+    pub fn set_signing_attributes(mut self, input: ::std::option::Option<super::types::DkimSigningAttributes>) -> Self {
         self.signing_attributes = input;
         self
     }
     /// <p>An object that contains information about the private key and selector that you want to use to configure DKIM for the identity for Bring Your Own DKIM (BYODKIM) for the identity, or, configures the key length to be used for <a href="https://docs.aws.amazon.com/ses/latest/DeveloperGuide/easy-dkim.html">Easy DKIM</a>.</p>
-    pub fn get_signing_attributes(&self) -> &::std::option::Option<crate::types::DkimSigningAttributes> {
+    pub fn get_signing_attributes(&self) -> &::std::option::Option<super::types::DkimSigningAttributes> {
         &self.signing_attributes
     }
     /// Consumes the builder and constructs a [`PutEmailIdentityDkimSigningAttributesInput`](crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput).
@@ -119,11 +119,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput,
+        super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput {
+            super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput {
                 email_identity: self.email_identity,
                 signing_attributes_origin: self.signing_attributes_origin,
                 signing_attributes: self.signing_attributes,
