--- original
+++ modified
@@ -6,7 +6,7 @@
     /// <p>The global table name.</p>
     pub global_table_name: ::std::option::Option<::std::string::String>,
     /// <p>A list of Regions that should be added or removed from the global table.</p>
-    pub replica_updates: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaUpdate>>,
+    pub replica_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicaUpdate>>,
 }
 impl UpdateGlobalTableInput {
     /// <p>The global table name.</p>
@@ -16,14 +16,14 @@
     /// <p>A list of Regions that should be added or removed from the global table.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replica_updates.is_none()`.
-    pub fn replica_updates(&self) -> &[crate::types::ReplicaUpdate] {
+    pub fn replica_updates(&self) -> &[super::super::super::types::ReplicaUpdate] {
         self.replica_updates.as_deref().unwrap_or_default()
     }
 }
 impl UpdateGlobalTableInput {
     /// Creates a new builder-style object to manufacture [`UpdateGlobalTableInput`](crate::operation::update_global_table::UpdateGlobalTableInput).
-    pub fn builder() -> crate::operation::update_global_table::builders::UpdateGlobalTableInputBuilder {
-        crate::operation::update_global_table::builders::UpdateGlobalTableInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_global_table::builders::UpdateGlobalTableInputBuilder {
+        super::super::super::operation::update_global_table::builders::UpdateGlobalTableInputBuilder::default()
     }
 }

@@ -32,7 +32,7 @@
 #[non_exhaustive]
 pub struct UpdateGlobalTableInputBuilder {
     pub(crate) global_table_name: ::std::option::Option<::std::string::String>,
-    pub(crate) replica_updates: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaUpdate>>,
+    pub(crate) replica_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicaUpdate>>,
 }
 impl UpdateGlobalTableInputBuilder {
     /// <p>The global table name.</p>
@@ -55,7 +55,7 @@
     /// To override the contents of this collection use [`set_replica_updates`](Self::set_replica_updates).
     ///
     /// <p>A list of Regions that should be added or removed from the global table.</p>
-    pub fn replica_updates(mut self, input: crate::types::ReplicaUpdate) -> Self {
+    pub fn replica_updates(mut self, input: super::super::super::types::ReplicaUpdate) -> Self {
         let mut v = self.replica_updates.unwrap_or_default();
         v.push(input);
         self.replica_updates = ::std::option::Option::Some(v);
@@ -62,19 +62,19 @@
         self
     }
     /// <p>A list of Regions that should be added or removed from the global table.</p>
-    pub fn set_replica_updates(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaUpdate>>) -> Self {
+    pub fn set_replica_updates(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicaUpdate>>) -> Self {
         self.replica_updates = input;
         self
     }
     /// <p>A list of Regions that should be added or removed from the global table.</p>
-    pub fn get_replica_updates(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ReplicaUpdate>> {
+    pub fn get_replica_updates(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicaUpdate>> {
         &self.replica_updates
     }
     /// Consumes the builder and constructs a [`UpdateGlobalTableInput`](crate::operation::update_global_table::UpdateGlobalTableInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_global_table::UpdateGlobalTableInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_global_table::UpdateGlobalTableInput {
+    ) -> ::std::result::Result<super::super::super::operation::update_global_table::UpdateGlobalTableInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_global_table::UpdateGlobalTableInput {
             global_table_name: self.global_table_name,
             replica_updates: self.replica_updates,
         })
