--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_metric_filter::DeleteMetricFilterOutput,
-    crate::operation::delete_metric_filter::DeleteMetricFilterError,
+    super::super::operation::delete_metric_filter::DeleteMetricFilterOutput,
+    super::super::operation::delete_metric_filter::DeleteMetricFilterError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::delete_metric_filter::DeleteMetricFilterError::unhandled(generic)),
+        None => return Err(super::super::operation::delete_metric_filter::DeleteMetricFilterError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidParameterException" => crate::operation::delete_metric_filter::DeleteMetricFilterError::InvalidParameterException({
+        "InvalidParameterException" => super::super::operation::delete_metric_filter::DeleteMetricFilterError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "OperationAbortedException" => crate::operation::delete_metric_filter::DeleteMetricFilterError::OperationAbortedException({
+        "OperationAbortedException" => super::super::operation::delete_metric_filter::DeleteMetricFilterError::OperationAbortedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationAbortedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_operation_aborted_exception::de_operation_aborted_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
+                let mut output = super::super::types::error::builders::OperationAbortedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_operation_aborted_exception::de_operation_aborted_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::delete_metric_filter::DeleteMetricFilterError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::delete_metric_filter::DeleteMetricFilterError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,14 +65,14 @@
             }
             tmp
         }),
-        "ServiceUnavailableException" => crate::operation::delete_metric_filter::DeleteMetricFilterError::ServiceUnavailableException({
+        "ServiceUnavailableException" => super::super::operation::delete_metric_filter::DeleteMetricFilterError::ServiceUnavailableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceUnavailableExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::ServiceUnavailableExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
+                    super::super::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::delete_metric_filter::DeleteMetricFilterError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -81,7 +81,7 @@
             }
             tmp
         }),
-        _ => crate::operation::delete_metric_filter::DeleteMetricFilterError::generic(generic),
+        _ => super::super::operation::delete_metric_filter::DeleteMetricFilterError::generic(generic),
     })
 }

@@ -91,12 +91,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_metric_filter::DeleteMetricFilterOutput,
-    crate::operation::delete_metric_filter::DeleteMetricFilterError,
+    super::super::operation::delete_metric_filter::DeleteMetricFilterOutput,
+    super::super::operation::delete_metric_filter::DeleteMetricFilterError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_metric_filter::builders::DeleteMetricFilterOutputBuilder::default();
+        let mut output = super::super::operation::delete_metric_filter::builders::DeleteMetricFilterOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -103,11 +103,11 @@
 }

 pub fn ser_delete_metric_filter_input(
-    input: &crate::operation::delete_metric_filter::DeleteMetricFilterInput,
+    input: &super::super::operation::delete_metric_filter::DeleteMetricFilterInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_delete_metric_filter_input::ser_delete_metric_filter_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_delete_metric_filter_input::ser_delete_metric_filter_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
