--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ConfigurationTemplate>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::ConfigurationTemplate>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ConfigurationTemplateBuilder::default();
+            let mut builder = super::super::types::builders::ConfigurationTemplateBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -45,17 +45,17 @@
                         "deliveryDestinationType" => {
                             builder = builder.set_delivery_destination_type(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::DeliveryDestinationType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::DeliveryDestinationType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "defaultDeliveryConfigValues" => {
                             builder = builder.set_default_delivery_config_values(
-                                    crate::protocol_serde::shape_configuration_template_delivery_config_values::de_configuration_template_delivery_config_values(tokens, _value, depth + 1)?
+                                    super::super::protocol_serde::shape_configuration_template_delivery_config_values::de_configuration_template_delivery_config_values(tokens, _value, depth + 1)?
                                 );
                         }
                         "allowedFields" => {
-                            builder = builder.set_allowed_fields(crate::protocol_serde::shape_allowed_fields::de_allowed_fields(
+                            builder = builder.set_allowed_fields(super::super::protocol_serde::shape_allowed_fields::de_allowed_fields(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -62,7 +62,7 @@
                             )?);
                         }
                         "allowedOutputFormats" => {
-                            builder = builder.set_allowed_output_formats(crate::protocol_serde::shape_output_formats::de_output_formats(
+                            builder = builder.set_allowed_output_formats(super::super::protocol_serde::shape_output_formats::de_output_formats(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -77,11 +77,11 @@
                         }
                         "allowedFieldDelimiters" => {
                             builder = builder.set_allowed_field_delimiters(
-                                crate::protocol_serde::shape_allowed_field_delimiters::de_allowed_field_delimiters(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_allowed_field_delimiters::de_allowed_field_delimiters(tokens, _value, depth + 1)?,
                             );
                         }
                         "allowedSuffixPathFields" => {
-                            builder = builder.set_allowed_suffix_path_fields(crate::protocol_serde::shape_record_fields::de_record_fields(
+                            builder = builder.set_allowed_suffix_path_fields(super::super::protocol_serde::shape_record_fields::de_record_fields(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -89,7 +89,7 @@
                         }
                         "deliverySourceConfiguration" => {
                             builder = builder.set_delivery_source_configuration(
-                                crate::protocol_serde::shape_delivery_source_configuration_schemas::de_delivery_source_configuration_schemas(
+                                super::super::protocol_serde::shape_delivery_source_configuration_schemas::de_delivery_source_configuration_schemas(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -98,7 +98,7 @@
                         }
                         "s3TablesIntegration" => {
                             builder = builder.set_s3_tables_integration(
-                                crate::protocol_serde::shape_s3_tables_integration::de_s3_tables_integration(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_s3_tables_integration::de_s3_tables_integration(tokens, _value, depth + 1)?,
                             );
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
