--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::DeliverySource>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::DeliverySource>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::DeliverySourceBuilder::default();
+            let mut builder = super::types::builders::DeliverySourceBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -37,7 +37,7 @@
                         }
                         "resourceArns" => {
                             builder =
-                                builder.set_resource_arns(crate::protocol_serde::shape_resource_arns::de_resource_arns(tokens, _value, depth + 1)?);
+                                builder.set_resource_arns(super::protocol_serde::shape_resource_arns::de_resource_arns(tokens, _value, depth + 1)?);
                         }
                         "service" => {
                             builder = builder.set_service(
@@ -54,11 +54,11 @@
                             );
                         }
                         "tags" => {
-                            builder = builder.set_tags(crate::protocol_serde::shape_tags::de_tags(tokens, _value, depth + 1)?);
+                            builder = builder.set_tags(super::protocol_serde::shape_tags::de_tags(tokens, _value, depth + 1)?);
                         }
                         "deliverySourceConfiguration" => {
                             builder = builder.set_delivery_source_configuration(
-                                crate::protocol_serde::shape_delivery_source_configuration::de_delivery_source_configuration(
+                                super::protocol_serde::shape_delivery_source_configuration::de_delivery_source_configuration(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -68,7 +68,7 @@
                         "status" => {
                             builder = builder.set_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::DeliverySourceStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::DeliverySourceStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -75,7 +75,7 @@
                         "statusReason" => {
                             builder = builder.set_status_reason(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::DeliverySourceStatusReason::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::DeliverySourceStatusReason::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
