--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::AnomalyDetector>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::AnomalyDetector>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::AnomalyDetectorBuilder::default();
+            let mut builder = super::types::builders::AnomalyDetectorBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -36,7 +36,7 @@
                             );
                         }
                         "logGroupArnList" => {
-                            builder = builder.set_log_group_arn_list(crate::protocol_serde::shape_log_group_arn_list::de_log_group_arn_list(
+                            builder = builder.set_log_group_arn_list(super::protocol_serde::shape_log_group_arn_list::de_log_group_arn_list(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -45,7 +45,7 @@
                         "evaluationFrequency" => {
                             builder = builder.set_evaluation_frequency(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::EvaluationFrequency::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::EvaluationFrequency::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -59,7 +59,7 @@
                         "anomalyDetectorStatus" => {
                             builder = builder.set_anomaly_detector_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::AnomalyDetectorStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::AnomalyDetectorStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
