--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::Anomaly>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::Anomaly>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::AnomalyBuilder::default();
+            let mut builder = super::super::types::builders::AnomalyBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -90,18 +90,18 @@
                         "state" => {
                             builder = builder.set_state(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::State::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::State::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "histogram" => {
-                            builder = builder.set_histogram(crate::protocol_serde::shape_histogram::de_histogram(tokens, _value, depth + 1)?);
+                            builder = builder.set_histogram(super::super::protocol_serde::shape_histogram::de_histogram(tokens, _value, depth + 1)?);
                         }
                         "logSamples" => {
-                            builder = builder.set_log_samples(crate::protocol_serde::shape_log_samples::de_log_samples(tokens, _value, depth + 1)?);
+                            builder = builder.set_log_samples(super::super::protocol_serde::shape_log_samples::de_log_samples(tokens, _value, depth + 1)?);
                         }
                         "patternTokens" => {
-                            builder = builder.set_pattern_tokens(crate::protocol_serde::shape_pattern_tokens::de_pattern_tokens(
+                            builder = builder.set_pattern_tokens(super::super::protocol_serde::shape_pattern_tokens::de_pattern_tokens(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -108,7 +108,7 @@
                             )?);
                         }
                         "logGroupArnList" => {
-                            builder = builder.set_log_group_arn_list(crate::protocol_serde::shape_log_group_arn_list::de_log_group_arn_list(
+                            builder = builder.set_log_group_arn_list(super::super::protocol_serde::shape_log_group_arn_list::de_log_group_arn_list(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -144,7 +144,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::anomaly_correct_errors(builder).build().map_err(|err| {
+            Ok(Some(super::super::serde_util::anomaly_correct_errors(builder).build().map_err(|err| {
                 ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err)
             })?))
         }
