--- original
+++ modified
@@ -4,27 +4,27 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::put_delivery_source::PutDeliverySourceOutput, crate::operation::put_delivery_source::PutDeliverySourceError>
+) -> std::result::Result<super::operation::put_delivery_source::PutDeliverySourceOutput, super::operation::put_delivery_source::PutDeliverySourceError>
 {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled(generic)),
+        None => return Err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConflictException" => crate::operation::put_delivery_source::PutDeliverySourceError::ConflictException({
+        "ConflictException" => super::operation::put_delivery_source::PutDeliverySourceError::ConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+                let mut output = super::types::error::builders::ConflictExceptionBuilder::default();
+                output = super::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -33,13 +33,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::put_delivery_source::PutDeliverySourceError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::put_delivery_source::PutDeliverySourceError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -48,16 +48,16 @@
             }
             tmp
         }),
-        "ServiceQuotaExceededException" => crate::operation::put_delivery_source::PutDeliverySourceError::ServiceQuotaExceededException({
+        "ServiceQuotaExceededException" => super::operation::put_delivery_source::PutDeliverySourceError::ServiceQuotaExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
+                let mut output = super::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+                .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -66,14 +66,14 @@
             }
             tmp
         }),
-        "ServiceUnavailableException" => crate::operation::put_delivery_source::PutDeliverySourceError::ServiceUnavailableException({
+        "ServiceUnavailableException" => super::operation::put_delivery_source::PutDeliverySourceError::ServiceUnavailableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceUnavailableExceptionBuilder::default();
+                let mut output = super::types::error::builders::ServiceUnavailableExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+                    super::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
+                        .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -82,13 +82,13 @@
             }
             tmp
         }),
-        "ThrottlingException" => crate::operation::put_delivery_source::PutDeliverySourceError::ThrottlingException({
+        "ThrottlingException" => super::operation::put_delivery_source::PutDeliverySourceError::ThrottlingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottlingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+                let mut output = super::types::error::builders::ThrottlingExceptionBuilder::default();
+                output = super::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -97,13 +97,13 @@
             }
             tmp
         }),
-        "ValidationException" => crate::operation::put_delivery_source::PutDeliverySourceError::ValidationException({
+        "ValidationException" => super::operation::put_delivery_source::PutDeliverySourceError::ValidationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+                let mut output = super::types::error::builders::ValidationExceptionBuilder::default();
+                output = super::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -112,7 +112,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_delivery_source::PutDeliverySourceError::generic(generic),
+        _ => super::operation::put_delivery_source::PutDeliverySourceError::generic(generic),
     })
 }

@@ -121,13 +121,13 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::put_delivery_source::PutDeliverySourceOutput, crate::operation::put_delivery_source::PutDeliverySourceError>
+) -> std::result::Result<super::operation::put_delivery_source::PutDeliverySourceOutput, super::operation::put_delivery_source::PutDeliverySourceError>
 {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_delivery_source::builders::PutDeliverySourceOutputBuilder::default();
-        output = crate::protocol_serde::shape_put_delivery_source::de_put_delivery_source(_response_body, output)
-            .map_err(crate::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
+        let mut output = super::operation::put_delivery_source::builders::PutDeliverySourceOutputBuilder::default();
+        output = super::protocol_serde::shape_put_delivery_source::de_put_delivery_source(_response_body, output)
+            .map_err(super::operation::put_delivery_source::PutDeliverySourceError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -134,11 +134,11 @@
 }

 pub fn ser_put_delivery_source_input(
-    input: &crate::operation::put_delivery_source::PutDeliverySourceInput,
+    input: &super::operation::put_delivery_source::PutDeliverySourceInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_delivery_source_input::ser_put_delivery_source_input_input(&mut object, input)?;
+    super::protocol_serde::shape_put_delivery_source_input::ser_put_delivery_source_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -145,12 +145,12 @@

 pub(crate) fn de_put_delivery_source(
     _value: &[u8],
-    mut builder: crate::operation::put_delivery_source::builders::PutDeliverySourceOutputBuilder,
+    mut builder: super::operation::put_delivery_source::builders::PutDeliverySourceOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::put_delivery_source::builders::PutDeliverySourceOutputBuilder,
+    super::operation::put_delivery_source::builders::PutDeliverySourceOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -160,7 +160,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "deliverySource" => {
-                    builder = builder.set_delivery_source(crate::protocol_serde::shape_delivery_source::de_delivery_source(
+                    builder = builder.set_delivery_source(super::protocol_serde::shape_delivery_source::de_delivery_source(
                         tokens,
                         _value,
                         depth + 1,
