--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::test_metric_filter::TestMetricFilterOutput, crate::operation::test_metric_filter::TestMetricFilterError> {
+) -> std::result::Result<super::super::operation::test_metric_filter::TestMetricFilterOutput, super::super::operation::test_metric_filter::TestMetricFilterError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::test_metric_filter::TestMetricFilterError::unhandled(generic)),
+        None => return Err(super::super::operation::test_metric_filter::TestMetricFilterError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidParameterException" => crate::operation::test_metric_filter::TestMetricFilterError::InvalidParameterException({
+        "InvalidParameterException" => super::super::operation::test_metric_filter::TestMetricFilterError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,14 +32,14 @@
             }
             tmp
         }),
-        "ServiceUnavailableException" => crate::operation::test_metric_filter::TestMetricFilterError::ServiceUnavailableException({
+        "ServiceUnavailableException" => super::super::operation::test_metric_filter::TestMetricFilterError::ServiceUnavailableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceUnavailableExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::ServiceUnavailableExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
+                    super::super::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -48,7 +48,7 @@
             }
             tmp
         }),
-        _ => crate::operation::test_metric_filter::TestMetricFilterError::generic(generic),
+        _ => super::super::operation::test_metric_filter::TestMetricFilterError::generic(generic),
     })
 }

@@ -57,12 +57,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::test_metric_filter::TestMetricFilterOutput, crate::operation::test_metric_filter::TestMetricFilterError> {
+) -> std::result::Result<super::super::operation::test_metric_filter::TestMetricFilterOutput, super::super::operation::test_metric_filter::TestMetricFilterError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::test_metric_filter::builders::TestMetricFilterOutputBuilder::default();
-        output = crate::protocol_serde::shape_test_metric_filter::de_test_metric_filter(_response_body, output)
-            .map_err(crate::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
+        let mut output = super::super::operation::test_metric_filter::builders::TestMetricFilterOutputBuilder::default();
+        output = super::super::protocol_serde::shape_test_metric_filter::de_test_metric_filter(_response_body, output)
+            .map_err(super::super::operation::test_metric_filter::TestMetricFilterError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -69,11 +69,11 @@
 }

 pub fn ser_test_metric_filter_input(
-    input: &crate::operation::test_metric_filter::TestMetricFilterInput,
+    input: &super::super::operation::test_metric_filter::TestMetricFilterInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_test_metric_filter_input::ser_test_metric_filter_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_test_metric_filter_input::ser_test_metric_filter_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -80,12 +80,12 @@

 pub(crate) fn de_test_metric_filter(
     _value: &[u8],
-    mut builder: crate::operation::test_metric_filter::builders::TestMetricFilterOutputBuilder,
+    mut builder: super::super::operation::test_metric_filter::builders::TestMetricFilterOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::test_metric_filter::builders::TestMetricFilterOutputBuilder,
+    super::super::operation::test_metric_filter::builders::TestMetricFilterOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -95,7 +95,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "matches" => {
-                    builder = builder.set_matches(crate::protocol_serde::shape_metric_filter_matches::de_metric_filter_matches(
+                    builder = builder.set_matches(super::super::protocol_serde::shape_metric_filter_matches::de_metric_filter_matches(
                         tokens,
                         _value,
                         depth + 1,
