--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::TriggerHistoryRecord>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::TriggerHistoryRecord>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::TriggerHistoryRecordBuilder::default();
+            let mut builder = super::super::types::builders::TriggerHistoryRecordBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,7 +31,7 @@
                         "executionStatus" => {
                             builder = builder.set_execution_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ExecutionStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::ExecutionStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -51,7 +51,7 @@
                         }
                         "destinations" => {
                             builder = builder.set_destinations(
-                                crate::protocol_serde::shape_scheduled_query_destination_list::de_scheduled_query_destination_list(
+                                super::super::protocol_serde::shape_scheduled_query_destination_list::de_scheduled_query_destination_list(
                                     tokens,
                                     _value,
                                     depth + 1,
