--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyOutput,
-    crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError,
+    super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyOutput,
+    super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled(generic)),
+        None => return Err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConflictException" => crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ConflictException({
+        "ConflictException" => super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
+                let mut output = super::types::error::builders::ConflictExceptionBuilder::default();
+                output = super::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -36,14 +36,14 @@
             tmp
         }),
         "ResourceNotFoundException" => {
-            crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ResourceNotFoundException({
+            super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ResourceNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                    let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
+                        super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                            .map_err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -54,14 +54,14 @@
             })
         }
         "ServiceUnavailableException" => {
-            crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ServiceUnavailableException({
+            super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ServiceUnavailableException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceUnavailableExceptionBuilder::default();
+                    let mut output = super::types::error::builders::ServiceUnavailableExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
+                        super::protocol_serde::shape_service_unavailable_exception::de_service_unavailable_exception_json_err(_response_body, output)
+                            .map_err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -71,13 +71,13 @@
                 tmp
             })
         }
-        "ValidationException" => crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ValidationException({
+        "ValidationException" => super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::ValidationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
+                let mut output = super::types::error::builders::ValidationExceptionBuilder::default();
+                output = super::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -86,7 +86,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::generic(generic),
+        _ => super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::generic(generic),
     })
 }

@@ -96,14 +96,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyOutput,
-    crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError,
+    super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyOutput,
+    super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_delivery_destination_policy::builders::PutDeliveryDestinationPolicyOutputBuilder::default();
-        output = crate::protocol_serde::shape_put_delivery_destination_policy::de_put_delivery_destination_policy(_response_body, output)
-            .map_err(crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
+        let mut output = super::operation::put_delivery_destination_policy::builders::PutDeliveryDestinationPolicyOutputBuilder::default();
+        output = super::protocol_serde::shape_put_delivery_destination_policy::de_put_delivery_destination_policy(_response_body, output)
+            .map_err(super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -110,11 +110,11 @@
 }

 pub fn ser_put_delivery_destination_policy_input(
-    input: &crate::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyInput,
+    input: &super::operation::put_delivery_destination_policy::PutDeliveryDestinationPolicyInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_delivery_destination_policy_input::ser_put_delivery_destination_policy_input_input(&mut object, input)?;
+    super::protocol_serde::shape_put_delivery_destination_policy_input::ser_put_delivery_destination_policy_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -121,12 +121,12 @@

 pub(crate) fn de_put_delivery_destination_policy(
     _value: &[u8],
-    mut builder: crate::operation::put_delivery_destination_policy::builders::PutDeliveryDestinationPolicyOutputBuilder,
+    mut builder: super::operation::put_delivery_destination_policy::builders::PutDeliveryDestinationPolicyOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::put_delivery_destination_policy::builders::PutDeliveryDestinationPolicyOutputBuilder,
+    super::operation::put_delivery_destination_policy::builders::PutDeliveryDestinationPolicyOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -136,7 +136,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "policy" => {
-                    builder = builder.set_policy(crate::protocol_serde::shape_policy::de_policy(tokens, _value, depth + 1)?);
+                    builder = builder.set_policy(super::protocol_serde::shape_policy::de_policy(tokens, _value, depth + 1)?);
                 }
                 _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
             },
