--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::Import>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::Import>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ImportBuilder::default();
+            let mut builder = super::types::builders::ImportBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -38,7 +38,7 @@
                         "importStatus" => {
                             builder = builder.set_import_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ImportStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::ImportStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -50,7 +50,7 @@
                             );
                         }
                         "importStatistics" => {
-                            builder = builder.set_import_statistics(crate::protocol_serde::shape_import_statistics::de_import_statistics(
+                            builder = builder.set_import_statistics(super::protocol_serde::shape_import_statistics::de_import_statistics(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -58,7 +58,7 @@
                         }
                         "importFilter" => {
                             builder =
-                                builder.set_import_filter(crate::protocol_serde::shape_import_filter::de_import_filter(tokens, _value, depth + 1)?);
+                                builder.set_import_filter(super::protocol_serde::shape_import_filter::de_import_filter(tokens, _value, depth + 1)?);
                         }
                         "creationTime" => {
                             builder = builder.set_creation_time(
