--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::OpenSearchApplication>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::OpenSearchApplication>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::OpenSearchApplicationBuilder::default();
+            let mut builder = super::types::builders::OpenSearchApplicationBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -43,7 +43,7 @@
                             );
                         }
                         "status" => {
-                            builder = builder.set_status(crate::protocol_serde::shape_open_search_resource_status::de_open_search_resource_status(
+                            builder = builder.set_status(super::protocol_serde::shape_open_search_resource_status::de_open_search_resource_status(
                                 tokens,
                                 _value,
                                 depth + 1,
