--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl StopConfigurationRecorderInput {
     /// Creates a new builder-style object to manufacture [`StopConfigurationRecorderInput`](crate::operation::stop_configuration_recorder::StopConfigurationRecorderInput).
-    pub fn builder() -> crate::operation::stop_configuration_recorder::builders::StopConfigurationRecorderInputBuilder {
-        crate::operation::stop_configuration_recorder::builders::StopConfigurationRecorderInputBuilder::default()
+    pub fn builder() -> super::operation::stop_configuration_recorder::builders::StopConfigurationRecorderInputBuilder {
+        super::operation::stop_configuration_recorder::builders::StopConfigurationRecorderInputBuilder::default()
     }
 }

@@ -46,10 +46,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::stop_configuration_recorder::StopConfigurationRecorderInput,
+        super::operation::stop_configuration_recorder::StopConfigurationRecorderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::stop_configuration_recorder::StopConfigurationRecorderInput {
+        ::std::result::Result::Ok(super::operation::stop_configuration_recorder::StopConfigurationRecorderInput {
             configuration_recorder_name: self.configuration_recorder_name,
         })
     }
