--- original
+++ modified
@@ -46,8 +46,8 @@
 }
 impl PutProvisionedConcurrencyConfigInput {
     /// Creates a new builder-style object to manufacture [`PutProvisionedConcurrencyConfigInput`](crate::operation::put_provisioned_concurrency_config::PutProvisionedConcurrencyConfigInput).
-    pub fn builder() -> crate::operation::put_provisioned_concurrency_config::builders::PutProvisionedConcurrencyConfigInputBuilder {
-        crate::operation::put_provisioned_concurrency_config::builders::PutProvisionedConcurrencyConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_provisioned_concurrency_config::builders::PutProvisionedConcurrencyConfigInputBuilder {
+        super::super::super::operation::put_provisioned_concurrency_config::builders::PutProvisionedConcurrencyConfigInputBuilder::default()
     }
 }

@@ -139,11 +139,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_provisioned_concurrency_config::PutProvisionedConcurrencyConfigInput,
+        super::super::super::operation::put_provisioned_concurrency_config::PutProvisionedConcurrencyConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_provisioned_concurrency_config::PutProvisionedConcurrencyConfigInput {
+            super::super::super::operation::put_provisioned_concurrency_config::PutProvisionedConcurrencyConfigInput {
                 function_name: self.function_name,
                 qualifier: self.qualifier,
                 provisioned_concurrent_executions: self.provisioned_concurrent_executions,
