--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl GetSuppressedDestinationInput {
     /// Creates a new builder-style object to manufacture [`GetSuppressedDestinationInput`](crate::operation::get_suppressed_destination::GetSuppressedDestinationInput).
-    pub fn builder() -> crate::operation::get_suppressed_destination::builders::GetSuppressedDestinationInputBuilder {
-        crate::operation::get_suppressed_destination::builders::GetSuppressedDestinationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_suppressed_destination::builders::GetSuppressedDestinationInputBuilder {
+        super::super::super::operation::get_suppressed_destination::builders::GetSuppressedDestinationInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_suppressed_destination::GetSuppressedDestinationInput,
+        super::super::super::operation::get_suppressed_destination::GetSuppressedDestinationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_suppressed_destination::GetSuppressedDestinationInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_suppressed_destination::GetSuppressedDestinationInput {
             email_address: self.email_address,
             tenant_name: self.tenant_name,
         })
