--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl ListConfigurationSetsInput {
     /// Creates a new builder-style object to manufacture [`ListConfigurationSetsInput`](crate::operation::list_configuration_sets::ListConfigurationSetsInput).
-    pub fn builder() -> crate::operation::list_configuration_sets::builders::ListConfigurationSetsInputBuilder {
-        crate::operation::list_configuration_sets::builders::ListConfigurationSetsInputBuilder::default()
+    pub fn builder() -> super::operation::list_configuration_sets::builders::ListConfigurationSetsInputBuilder {
+        super::operation::list_configuration_sets::builders::ListConfigurationSetsInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`ListConfigurationSetsInput`](crate::operation::list_configuration_sets::ListConfigurationSetsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_configuration_sets::ListConfigurationSetsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_configuration_sets::ListConfigurationSetsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_configuration_sets::ListConfigurationSetsInput {
+        ::std::result::Result::Ok(super::operation::list_configuration_sets::ListConfigurationSetsInput {
             next_token: self.next_token,
             page_size: self.page_size,
         })
