--- original
+++ modified
@@ -5,20 +5,20 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AccountRecoverySettingType {
     /// <p>The list of options and priorities for user message delivery in forgot-password operations. Sets or displays user pool preferences for email or SMS message priority, whether users should fall back to a second delivery method, and whether passwords should only be reset by administrators.</p>
-    pub recovery_mechanisms: ::std::option::Option<::std::vec::Vec<crate::types::RecoveryOptionType>>,
+    pub recovery_mechanisms: ::std::option::Option<::std::vec::Vec<super::types::RecoveryOptionType>>,
 }
 impl AccountRecoverySettingType {
     /// <p>The list of options and priorities for user message delivery in forgot-password operations. Sets or displays user pool preferences for email or SMS message priority, whether users should fall back to a second delivery method, and whether passwords should only be reset by administrators.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.recovery_mechanisms.is_none()`.
-    pub fn recovery_mechanisms(&self) -> &[crate::types::RecoveryOptionType] {
+    pub fn recovery_mechanisms(&self) -> &[super::types::RecoveryOptionType] {
         self.recovery_mechanisms.as_deref().unwrap_or_default()
     }
 }
 impl AccountRecoverySettingType {
     /// Creates a new builder-style object to manufacture [`AccountRecoverySettingType`](crate::types::AccountRecoverySettingType).
-    pub fn builder() -> crate::types::builders::AccountRecoverySettingTypeBuilder {
-        crate::types::builders::AccountRecoverySettingTypeBuilder::default()
+    pub fn builder() -> super::types::builders::AccountRecoverySettingTypeBuilder {
+        super::types::builders::AccountRecoverySettingTypeBuilder::default()
     }
 }

@@ -26,7 +26,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AccountRecoverySettingTypeBuilder {
-    pub(crate) recovery_mechanisms: ::std::option::Option<::std::vec::Vec<crate::types::RecoveryOptionType>>,
+    pub(crate) recovery_mechanisms: ::std::option::Option<::std::vec::Vec<super::types::RecoveryOptionType>>,
 }
 impl AccountRecoverySettingTypeBuilder {
     /// Appends an item to `recovery_mechanisms`.
@@ -34,7 +34,7 @@
     /// To override the contents of this collection use [`set_recovery_mechanisms`](Self::set_recovery_mechanisms).
     ///
     /// <p>The list of options and priorities for user message delivery in forgot-password operations. Sets or displays user pool preferences for email or SMS message priority, whether users should fall back to a second delivery method, and whether passwords should only be reset by administrators.</p>
-    pub fn recovery_mechanisms(mut self, input: crate::types::RecoveryOptionType) -> Self {
+    pub fn recovery_mechanisms(mut self, input: super::types::RecoveryOptionType) -> Self {
         let mut v = self.recovery_mechanisms.unwrap_or_default();
         v.push(input);
         self.recovery_mechanisms = ::std::option::Option::Some(v);
@@ -41,17 +41,17 @@
         self
     }
     /// <p>The list of options and priorities for user message delivery in forgot-password operations. Sets or displays user pool preferences for email or SMS message priority, whether users should fall back to a second delivery method, and whether passwords should only be reset by administrators.</p>
-    pub fn set_recovery_mechanisms(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::RecoveryOptionType>>) -> Self {
+    pub fn set_recovery_mechanisms(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::RecoveryOptionType>>) -> Self {
         self.recovery_mechanisms = input;
         self
     }
     /// <p>The list of options and priorities for user message delivery in forgot-password operations. Sets or displays user pool preferences for email or SMS message priority, whether users should fall back to a second delivery method, and whether passwords should only be reset by administrators.</p>
-    pub fn get_recovery_mechanisms(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::RecoveryOptionType>> {
+    pub fn get_recovery_mechanisms(&self) -> &::std::option::Option<::std::vec::Vec<super::types::RecoveryOptionType>> {
         &self.recovery_mechanisms
     }
     /// Consumes the builder and constructs a [`AccountRecoverySettingType`](crate::types::AccountRecoverySettingType).
-    pub fn build(self) -> crate::types::AccountRecoverySettingType {
-        crate::types::AccountRecoverySettingType {
+    pub fn build(self) -> super::types::AccountRecoverySettingType {
+        super::types::AccountRecoverySettingType {
             recovery_mechanisms: self.recovery_mechanisms,
         }
     }
