--- original
+++ modified
@@ -5,30 +5,30 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AccountTakeoverActionsType {
     /// <p>The action that you assign to a low-risk assessment by threat protection.</p>
-    pub low_action: ::std::option::Option<crate::types::AccountTakeoverActionType>,
+    pub low_action: ::std::option::Option<super::super::types::AccountTakeoverActionType>,
     /// <p>The action that you assign to a medium-risk assessment by threat protection.</p>
-    pub medium_action: ::std::option::Option<crate::types::AccountTakeoverActionType>,
+    pub medium_action: ::std::option::Option<super::super::types::AccountTakeoverActionType>,
     /// <p>The action that you assign to a high-risk assessment by threat protection.</p>
-    pub high_action: ::std::option::Option<crate::types::AccountTakeoverActionType>,
+    pub high_action: ::std::option::Option<super::super::types::AccountTakeoverActionType>,
 }
 impl AccountTakeoverActionsType {
     /// <p>The action that you assign to a low-risk assessment by threat protection.</p>
-    pub fn low_action(&self) -> ::std::option::Option<&crate::types::AccountTakeoverActionType> {
+    pub fn low_action(&self) -> ::std::option::Option<&super::super::types::AccountTakeoverActionType> {
         self.low_action.as_ref()
     }
     /// <p>The action that you assign to a medium-risk assessment by threat protection.</p>
-    pub fn medium_action(&self) -> ::std::option::Option<&crate::types::AccountTakeoverActionType> {
+    pub fn medium_action(&self) -> ::std::option::Option<&super::super::types::AccountTakeoverActionType> {
         self.medium_action.as_ref()
     }
     /// <p>The action that you assign to a high-risk assessment by threat protection.</p>
-    pub fn high_action(&self) -> ::std::option::Option<&crate::types::AccountTakeoverActionType> {
+    pub fn high_action(&self) -> ::std::option::Option<&super::super::types::AccountTakeoverActionType> {
         self.high_action.as_ref()
     }
 }
 impl AccountTakeoverActionsType {
     /// Creates a new builder-style object to manufacture [`AccountTakeoverActionsType`](crate::types::AccountTakeoverActionsType).
-    pub fn builder() -> crate::types::builders::AccountTakeoverActionsTypeBuilder {
-        crate::types::builders::AccountTakeoverActionsTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::AccountTakeoverActionsTypeBuilder {
+        super::super::types::builders::AccountTakeoverActionsTypeBuilder::default()
     }
 }

@@ -36,56 +36,56 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AccountTakeoverActionsTypeBuilder {
-    pub(crate) low_action: ::std::option::Option<crate::types::AccountTakeoverActionType>,
-    pub(crate) medium_action: ::std::option::Option<crate::types::AccountTakeoverActionType>,
-    pub(crate) high_action: ::std::option::Option<crate::types::AccountTakeoverActionType>,
+    pub(crate) low_action: ::std::option::Option<super::super::types::AccountTakeoverActionType>,
+    pub(crate) medium_action: ::std::option::Option<super::super::types::AccountTakeoverActionType>,
+    pub(crate) high_action: ::std::option::Option<super::super::types::AccountTakeoverActionType>,
 }
 impl AccountTakeoverActionsTypeBuilder {
     /// <p>The action that you assign to a low-risk assessment by threat protection.</p>
-    pub fn low_action(mut self, input: crate::types::AccountTakeoverActionType) -> Self {
+    pub fn low_action(mut self, input: super::super::types::AccountTakeoverActionType) -> Self {
         self.low_action = ::std::option::Option::Some(input);
         self
     }
     /// <p>The action that you assign to a low-risk assessment by threat protection.</p>
-    pub fn set_low_action(mut self, input: ::std::option::Option<crate::types::AccountTakeoverActionType>) -> Self {
+    pub fn set_low_action(mut self, input: ::std::option::Option<super::super::types::AccountTakeoverActionType>) -> Self {
         self.low_action = input;
         self
     }
     /// <p>The action that you assign to a low-risk assessment by threat protection.</p>
-    pub fn get_low_action(&self) -> &::std::option::Option<crate::types::AccountTakeoverActionType> {
+    pub fn get_low_action(&self) -> &::std::option::Option<super::super::types::AccountTakeoverActionType> {
         &self.low_action
     }
     /// <p>The action that you assign to a medium-risk assessment by threat protection.</p>
-    pub fn medium_action(mut self, input: crate::types::AccountTakeoverActionType) -> Self {
+    pub fn medium_action(mut self, input: super::super::types::AccountTakeoverActionType) -> Self {
         self.medium_action = ::std::option::Option::Some(input);
         self
     }
     /// <p>The action that you assign to a medium-risk assessment by threat protection.</p>
-    pub fn set_medium_action(mut self, input: ::std::option::Option<crate::types::AccountTakeoverActionType>) -> Self {
+    pub fn set_medium_action(mut self, input: ::std::option::Option<super::super::types::AccountTakeoverActionType>) -> Self {
         self.medium_action = input;
         self
     }
     /// <p>The action that you assign to a medium-risk assessment by threat protection.</p>
-    pub fn get_medium_action(&self) -> &::std::option::Option<crate::types::AccountTakeoverActionType> {
+    pub fn get_medium_action(&self) -> &::std::option::Option<super::super::types::AccountTakeoverActionType> {
         &self.medium_action
     }
     /// <p>The action that you assign to a high-risk assessment by threat protection.</p>
-    pub fn high_action(mut self, input: crate::types::AccountTakeoverActionType) -> Self {
+    pub fn high_action(mut self, input: super::super::types::AccountTakeoverActionType) -> Self {
         self.high_action = ::std::option::Option::Some(input);
         self
     }
     /// <p>The action that you assign to a high-risk assessment by threat protection.</p>
-    pub fn set_high_action(mut self, input: ::std::option::Option<crate::types::AccountTakeoverActionType>) -> Self {
+    pub fn set_high_action(mut self, input: ::std::option::Option<super::super::types::AccountTakeoverActionType>) -> Self {
         self.high_action = input;
         self
     }
     /// <p>The action that you assign to a high-risk assessment by threat protection.</p>
-    pub fn get_high_action(&self) -> &::std::option::Option<crate::types::AccountTakeoverActionType> {
+    pub fn get_high_action(&self) -> &::std::option::Option<super::super::types::AccountTakeoverActionType> {
         &self.high_action
     }
     /// Consumes the builder and constructs a [`AccountTakeoverActionsType`](crate::types::AccountTakeoverActionsType).
-    pub fn build(self) -> crate::types::AccountTakeoverActionsType {
-        crate::types::AccountTakeoverActionsType {
+    pub fn build(self) -> super::super::types::AccountTakeoverActionsType {
+        super::super::types::AccountTakeoverActionsType {
             low_action: self.low_action,
             medium_action: self.medium_action,
             high_action: self.high_action,
