--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl AttributeType {
     /// Creates a new builder-style object to manufacture [`AttributeType`](crate::types::AttributeType).
-    pub fn builder() -> crate::types::builders::AttributeTypeBuilder {
-        crate::types::builders::AttributeTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::AttributeTypeBuilder {
+        super::super::types::builders::AttributeTypeBuilder::default()
     }
 }

@@ -80,8 +80,8 @@
     /// Consumes the builder and constructs a [`AttributeType`](crate::types::AttributeType).
     /// This method will fail if any of the following fields are not set:
     /// - [`name`](crate::types::builders::AttributeTypeBuilder::name)
-    pub fn build(self) -> ::std::result::Result<crate::types::AttributeType, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::AttributeType {
+    pub fn build(self) -> ::std::result::Result<super::super::types::AttributeType, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::AttributeType {
             name: self.name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "name",
