--- original
+++ modified
@@ -46,8 +46,8 @@
 }
 impl ClientSecretDescriptorType {
     /// Creates a new builder-style object to manufacture [`ClientSecretDescriptorType`](crate::types::ClientSecretDescriptorType).
-    pub fn builder() -> crate::types::builders::ClientSecretDescriptorTypeBuilder {
-        crate::types::builders::ClientSecretDescriptorTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::ClientSecretDescriptorTypeBuilder {
+        super::super::types::builders::ClientSecretDescriptorTypeBuilder::default()
     }
 }

@@ -118,8 +118,8 @@
         &self.client_secret_create_date
     }
     /// Consumes the builder and constructs a [`ClientSecretDescriptorType`](crate::types::ClientSecretDescriptorType).
-    pub fn build(self) -> crate::types::ClientSecretDescriptorType {
-        crate::types::ClientSecretDescriptorType {
+    pub fn build(self) -> super::super::types::ClientSecretDescriptorType {
+        super::super::types::ClientSecretDescriptorType {
             client_secret_id: self.client_secret_id,
             client_secret_value: self.client_secret_value,
             client_secret_create_date: self.client_secret_create_date,
