--- original
+++ modified
@@ -5,26 +5,26 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct CompromisedCredentialsRiskConfigurationType {
     /// <p>Settings for the sign-in activity where you want to configure compromised-credentials actions. Defaults to all events.</p>
-    pub event_filter: ::std::option::Option<::std::vec::Vec<crate::types::EventFilterType>>,
+    pub event_filter: ::std::option::Option<::std::vec::Vec<super::super::types::EventFilterType>>,
     /// <p>Settings for the actions that you want your user pool to take when Amazon Cognito detects compromised credentials.</p>
-    pub actions: ::std::option::Option<crate::types::CompromisedCredentialsActionsType>,
+    pub actions: ::std::option::Option<super::super::types::CompromisedCredentialsActionsType>,
 }
 impl CompromisedCredentialsRiskConfigurationType {
     /// <p>Settings for the sign-in activity where you want to configure compromised-credentials actions. Defaults to all events.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.event_filter.is_none()`.
-    pub fn event_filter(&self) -> &[crate::types::EventFilterType] {
+    pub fn event_filter(&self) -> &[super::super::types::EventFilterType] {
         self.event_filter.as_deref().unwrap_or_default()
     }
     /// <p>Settings for the actions that you want your user pool to take when Amazon Cognito detects compromised credentials.</p>
-    pub fn actions(&self) -> ::std::option::Option<&crate::types::CompromisedCredentialsActionsType> {
+    pub fn actions(&self) -> ::std::option::Option<&super::super::types::CompromisedCredentialsActionsType> {
         self.actions.as_ref()
     }
 }
 impl CompromisedCredentialsRiskConfigurationType {
     /// Creates a new builder-style object to manufacture [`CompromisedCredentialsRiskConfigurationType`](crate::types::CompromisedCredentialsRiskConfigurationType).
-    pub fn builder() -> crate::types::builders::CompromisedCredentialsRiskConfigurationTypeBuilder {
-        crate::types::builders::CompromisedCredentialsRiskConfigurationTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::CompromisedCredentialsRiskConfigurationTypeBuilder {
+        super::super::types::builders::CompromisedCredentialsRiskConfigurationTypeBuilder::default()
     }
 }

@@ -32,8 +32,8 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct CompromisedCredentialsRiskConfigurationTypeBuilder {
-    pub(crate) event_filter: ::std::option::Option<::std::vec::Vec<crate::types::EventFilterType>>,
-    pub(crate) actions: ::std::option::Option<crate::types::CompromisedCredentialsActionsType>,
+    pub(crate) event_filter: ::std::option::Option<::std::vec::Vec<super::super::types::EventFilterType>>,
+    pub(crate) actions: ::std::option::Option<super::super::types::CompromisedCredentialsActionsType>,
 }
 impl CompromisedCredentialsRiskConfigurationTypeBuilder {
     /// Appends an item to `event_filter`.
@@ -41,7 +41,7 @@
     /// To override the contents of this collection use [`set_event_filter`](Self::set_event_filter).
     ///
     /// <p>Settings for the sign-in activity where you want to configure compromised-credentials actions. Defaults to all events.</p>
-    pub fn event_filter(mut self, input: crate::types::EventFilterType) -> Self {
+    pub fn event_filter(mut self, input: super::super::types::EventFilterType) -> Self {
         let mut v = self.event_filter.unwrap_or_default();
         v.push(input);
         self.event_filter = ::std::option::Option::Some(v);
@@ -48,32 +48,32 @@
         self
     }
     /// <p>Settings for the sign-in activity where you want to configure compromised-credentials actions. Defaults to all events.</p>
-    pub fn set_event_filter(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EventFilterType>>) -> Self {
+    pub fn set_event_filter(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::EventFilterType>>) -> Self {
         self.event_filter = input;
         self
     }
     /// <p>Settings for the sign-in activity where you want to configure compromised-credentials actions. Defaults to all events.</p>
-    pub fn get_event_filter(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EventFilterType>> {
+    pub fn get_event_filter(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::EventFilterType>> {
         &self.event_filter
     }
     /// <p>Settings for the actions that you want your user pool to take when Amazon Cognito detects compromised credentials.</p>
     /// This field is required.
-    pub fn actions(mut self, input: crate::types::CompromisedCredentialsActionsType) -> Self {
+    pub fn actions(mut self, input: super::super::types::CompromisedCredentialsActionsType) -> Self {
         self.actions = ::std::option::Option::Some(input);
         self
     }
     /// <p>Settings for the actions that you want your user pool to take when Amazon Cognito detects compromised credentials.</p>
-    pub fn set_actions(mut self, input: ::std::option::Option<crate::types::CompromisedCredentialsActionsType>) -> Self {
+    pub fn set_actions(mut self, input: ::std::option::Option<super::super::types::CompromisedCredentialsActionsType>) -> Self {
         self.actions = input;
         self
     }
     /// <p>Settings for the actions that you want your user pool to take when Amazon Cognito detects compromised credentials.</p>
-    pub fn get_actions(&self) -> &::std::option::Option<crate::types::CompromisedCredentialsActionsType> {
+    pub fn get_actions(&self) -> &::std::option::Option<super::super::types::CompromisedCredentialsActionsType> {
         &self.actions
     }
     /// Consumes the builder and constructs a [`CompromisedCredentialsRiskConfigurationType`](crate::types::CompromisedCredentialsRiskConfigurationType).
-    pub fn build(self) -> crate::types::CompromisedCredentialsRiskConfigurationType {
-        crate::types::CompromisedCredentialsRiskConfigurationType {
+    pub fn build(self) -> super::super::types::CompromisedCredentialsRiskConfigurationType {
+        super::super::types::CompromisedCredentialsRiskConfigurationType {
             event_filter: self.event_filter,
             actions: self.actions,
         }
