--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl DeviceSecretVerifierConfigType {
     /// Creates a new builder-style object to manufacture [`DeviceSecretVerifierConfigType`](crate::types::DeviceSecretVerifierConfigType).
-    pub fn builder() -> crate::types::builders::DeviceSecretVerifierConfigTypeBuilder {
-        crate::types::builders::DeviceSecretVerifierConfigTypeBuilder::default()
+    pub fn builder() -> super::types::builders::DeviceSecretVerifierConfigTypeBuilder {
+        super::types::builders::DeviceSecretVerifierConfigTypeBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.salt
     }
     /// Consumes the builder and constructs a [`DeviceSecretVerifierConfigType`](crate::types::DeviceSecretVerifierConfigType).
-    pub fn build(self) -> crate::types::DeviceSecretVerifierConfigType {
-        crate::types::DeviceSecretVerifierConfigType {
+    pub fn build(self) -> super::types::DeviceSecretVerifierConfigType {
+        super::types::DeviceSecretVerifierConfigType {
             password_verifier: self.password_verifier,
             salt: self.salt,
         }
