--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The device key, for example <code>us-west-2_EXAMPLE-a1b2c3d4-5678-90ab-cdef-EXAMPLE22222</code>.</p>
     pub device_key: ::std::option::Option<::std::string::String>,
     /// <p>Metadata about a user's device, like name and last-access source IP.</p>
-    pub device_attributes: ::std::option::Option<::std::vec::Vec<crate::types::AttributeType>>,
+    pub device_attributes: ::std::option::Option<::std::vec::Vec<super::super::types::AttributeType>>,
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
     pub device_create_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The date and time when the item was modified. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -23,7 +23,7 @@
     /// <p>Metadata about a user's device, like name and last-access source IP.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.device_attributes.is_none()`.
-    pub fn device_attributes(&self) -> &[crate::types::AttributeType] {
+    pub fn device_attributes(&self) -> &[super::super::types::AttributeType] {
         self.device_attributes.as_deref().unwrap_or_default()
     }
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -41,8 +41,8 @@
 }
 impl DeviceType {
     /// Creates a new builder-style object to manufacture [`DeviceType`](crate::types::DeviceType).
-    pub fn builder() -> crate::types::builders::DeviceTypeBuilder {
-        crate::types::builders::DeviceTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeviceTypeBuilder {
+        super::super::types::builders::DeviceTypeBuilder::default()
     }
 }

@@ -51,7 +51,7 @@
 #[non_exhaustive]
 pub struct DeviceTypeBuilder {
     pub(crate) device_key: ::std::option::Option<::std::string::String>,
-    pub(crate) device_attributes: ::std::option::Option<::std::vec::Vec<crate::types::AttributeType>>,
+    pub(crate) device_attributes: ::std::option::Option<::std::vec::Vec<super::super::types::AttributeType>>,
     pub(crate) device_create_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) device_last_modified_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) device_last_authenticated_date: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -76,7 +76,7 @@
     /// To override the contents of this collection use [`set_device_attributes`](Self::set_device_attributes).
     ///
     /// <p>Metadata about a user's device, like name and last-access source IP.</p>
-    pub fn device_attributes(mut self, input: crate::types::AttributeType) -> Self {
+    pub fn device_attributes(mut self, input: super::super::types::AttributeType) -> Self {
         let mut v = self.device_attributes.unwrap_or_default();
         v.push(input);
         self.device_attributes = ::std::option::Option::Some(v);
@@ -83,12 +83,12 @@
         self
     }
     /// <p>Metadata about a user's device, like name and last-access source IP.</p>
-    pub fn set_device_attributes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::AttributeType>>) -> Self {
+    pub fn set_device_attributes(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::AttributeType>>) -> Self {
         self.device_attributes = input;
         self
     }
     /// <p>Metadata about a user's device, like name and last-access source IP.</p>
-    pub fn get_device_attributes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::AttributeType>> {
+    pub fn get_device_attributes(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::AttributeType>> {
         &self.device_attributes
     }
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -134,8 +134,8 @@
         &self.device_last_authenticated_date
     }
     /// Consumes the builder and constructs a [`DeviceType`](crate::types::DeviceType).
-    pub fn build(self) -> crate::types::DeviceType {
-        crate::types::DeviceType {
+    pub fn build(self) -> super::super::types::DeviceType {
+        super::super::types::DeviceType {
             device_key: self.device_key,
             device_attributes: self.device_attributes,
             device_create_date: self.device_create_date,
