--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl EmailMfaConfigType {
     /// Creates a new builder-style object to manufacture [`EmailMfaConfigType`](crate::types::EmailMfaConfigType).
-    pub fn builder() -> crate::types::builders::EmailMfaConfigTypeBuilder {
-        crate::types::builders::EmailMfaConfigTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::EmailMfaConfigTypeBuilder {
+        super::super::types::builders::EmailMfaConfigTypeBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.subject
     }
     /// Consumes the builder and constructs a [`EmailMfaConfigType`](crate::types::EmailMfaConfigType).
-    pub fn build(self) -> crate::types::EmailMfaConfigType {
-        crate::types::EmailMfaConfigType {
+    pub fn build(self) -> super::super::types::EmailMfaConfigType {
+        super::super::types::EmailMfaConfigType {
             message: self.message,
             subject: self.subject,
         }
