--- original
+++ modified
@@ -36,7 +36,7 @@
     /// <p>Before Amazon Cognito can email your users, it requires additional permissions to call Amazon SES on your behalf. When you update your user pool with this option, Amazon Cognito creates a <i>service-linked role</i>, which is a type of role in your Amazon Web Services account. This role contains the permissions that allow you to access Amazon SES and send email messages from your email address. For more information about the service-linked role that Amazon Cognito creates, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/using-service-linked-roles.html">Using Service-Linked Roles for Amazon Cognito</a> in the <i>Amazon Cognito Developer Guide</i>.</p>
     /// </dd>
     /// </dl>
-    pub email_sending_account: ::std::option::Option<crate::types::EmailSendingAccountType>,
+    pub email_sending_account: ::std::option::Option<super::types::EmailSendingAccountType>,
     /// <p>Either the sender’s email address or the sender’s name with their email address. For example, <code>testuser@example.com</code> or <code>Test User <testuser@example.com></testuser@example.com></code>. This address appears before the body of the email.</p>
     pub from: ::std::option::Option<::std::string::String>,
     /// <p>The set of configuration rules that can be applied to emails sent using Amazon Simple Email Service. A configuration set is applied to an email by including a reference to the configuration set in the headers of the email. Once applied, all of the rules in that configuration set are applied to the email. Configuration sets can be used to apply the following types of rules to emails:</p>
@@ -91,7 +91,7 @@
     /// <p>Before Amazon Cognito can email your users, it requires additional permissions to call Amazon SES on your behalf. When you update your user pool with this option, Amazon Cognito creates a <i>service-linked role</i>, which is a type of role in your Amazon Web Services account. This role contains the permissions that allow you to access Amazon SES and send email messages from your email address. For more information about the service-linked role that Amazon Cognito creates, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/using-service-linked-roles.html">Using Service-Linked Roles for Amazon Cognito</a> in the <i>Amazon Cognito Developer Guide</i>.</p>
     /// </dd>
     /// </dl>
-    pub fn email_sending_account(&self) -> ::std::option::Option<&crate::types::EmailSendingAccountType> {
+    pub fn email_sending_account(&self) -> ::std::option::Option<&super::types::EmailSendingAccountType> {
         self.email_sending_account.as_ref()
     }
     /// <p>Either the sender’s email address or the sender’s name with their email address. For example, <code>testuser@example.com</code> or <code>Test User <testuser@example.com></testuser@example.com></code>. This address appears before the body of the email.</p>
@@ -119,8 +119,8 @@
 }
 impl EmailConfigurationType {
     /// Creates a new builder-style object to manufacture [`EmailConfigurationType`](crate::types::EmailConfigurationType).
-    pub fn builder() -> crate::types::builders::EmailConfigurationTypeBuilder {
-        crate::types::builders::EmailConfigurationTypeBuilder::default()
+    pub fn builder() -> super::types::builders::EmailConfigurationTypeBuilder {
+        super::types::builders::EmailConfigurationTypeBuilder::default()
     }
 }

@@ -130,7 +130,7 @@
 pub struct EmailConfigurationTypeBuilder {
     pub(crate) source_arn: ::std::option::Option<::std::string::String>,
     pub(crate) reply_to_email_address: ::std::option::Option<::std::string::String>,
-    pub(crate) email_sending_account: ::std::option::Option<crate::types::EmailSendingAccountType>,
+    pub(crate) email_sending_account: ::std::option::Option<super::types::EmailSendingAccountType>,
     pub(crate) from: ::std::option::Option<::std::string::String>,
     pub(crate) configuration_set: ::std::option::Option<::std::string::String>,
 }
@@ -203,7 +203,7 @@
     /// <p>Before Amazon Cognito can email your users, it requires additional permissions to call Amazon SES on your behalf. When you update your user pool with this option, Amazon Cognito creates a <i>service-linked role</i>, which is a type of role in your Amazon Web Services account. This role contains the permissions that allow you to access Amazon SES and send email messages from your email address. For more information about the service-linked role that Amazon Cognito creates, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/using-service-linked-roles.html">Using Service-Linked Roles for Amazon Cognito</a> in the <i>Amazon Cognito Developer Guide</i>.</p>
     /// </dd>
     /// </dl>
-    pub fn email_sending_account(mut self, input: crate::types::EmailSendingAccountType) -> Self {
+    pub fn email_sending_account(mut self, input: super::types::EmailSendingAccountType) -> Self {
         self.email_sending_account = ::std::option::Option::Some(input);
         self
     }
@@ -226,7 +226,7 @@
     /// <p>Before Amazon Cognito can email your users, it requires additional permissions to call Amazon SES on your behalf. When you update your user pool with this option, Amazon Cognito creates a <i>service-linked role</i>, which is a type of role in your Amazon Web Services account. This role contains the permissions that allow you to access Amazon SES and send email messages from your email address. For more information about the service-linked role that Amazon Cognito creates, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/using-service-linked-roles.html">Using Service-Linked Roles for Amazon Cognito</a> in the <i>Amazon Cognito Developer Guide</i>.</p>
     /// </dd>
     /// </dl>
-    pub fn set_email_sending_account(mut self, input: ::std::option::Option<crate::types::EmailSendingAccountType>) -> Self {
+    pub fn set_email_sending_account(mut self, input: ::std::option::Option<super::types::EmailSendingAccountType>) -> Self {
         self.email_sending_account = input;
         self
     }
@@ -249,7 +249,7 @@
     /// <p>Before Amazon Cognito can email your users, it requires additional permissions to call Amazon SES on your behalf. When you update your user pool with this option, Amazon Cognito creates a <i>service-linked role</i>, which is a type of role in your Amazon Web Services account. This role contains the permissions that allow you to access Amazon SES and send email messages from your email address. For more information about the service-linked role that Amazon Cognito creates, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/using-service-linked-roles.html">Using Service-Linked Roles for Amazon Cognito</a> in the <i>Amazon Cognito Developer Guide</i>.</p>
     /// </dd>
     /// </dl>
-    pub fn get_email_sending_account(&self) -> &::std::option::Option<crate::types::EmailSendingAccountType> {
+    pub fn get_email_sending_account(&self) -> &::std::option::Option<super::types::EmailSendingAccountType> {
         &self.email_sending_account
     }
     /// <p>Either the sender’s email address or the sender’s name with their email address. For example, <code>testuser@example.com</code> or <code>Test User <testuser@example.com></testuser@example.com></code>. This address appears before the body of the email.</p>
@@ -323,8 +323,8 @@
         &self.configuration_set
     }
     /// Consumes the builder and constructs a [`EmailConfigurationType`](crate::types::EmailConfigurationType).
-    pub fn build(self) -> crate::types::EmailConfigurationType {
-        crate::types::EmailConfigurationType {
+    pub fn build(self) -> super::types::EmailConfigurationType {
+        super::types::EmailConfigurationType {
             source_arn: self.source_arn,
             reply_to_email_address: self.reply_to_email_address,
             email_sending_account: self.email_sending_account,
