--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl HttpHeader {
     /// Creates a new builder-style object to manufacture [`HttpHeader`](crate::types::HttpHeader).
-    pub fn builder() -> crate::types::builders::HttpHeaderBuilder {
-        crate::types::builders::HttpHeaderBuilder::default()
+    pub fn builder() -> super::super::types::builders::HttpHeaderBuilder {
+        super::super::types::builders::HttpHeaderBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.header_value
     }
     /// Consumes the builder and constructs a [`HttpHeader`](crate::types::HttpHeader).
-    pub fn build(self) -> crate::types::HttpHeader {
-        crate::types::HttpHeader {
+    pub fn build(self) -> super::super::types::HttpHeader {
+        super::super::types::HttpHeader {
             header_name: self.header_name,
             header_value: self.header_value,
         }
