--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>A friendly name for the IdP.</p>
     pub provider_name: ::std::option::Option<::std::string::String>,
     /// <p>The type of IdP. Either SAML, OIDC, or a named social identity provider.</p>
-    pub provider_type: ::std::option::Option<crate::types::IdentityProviderTypeType>,
+    pub provider_type: ::std::option::Option<super::super::types::IdentityProviderTypeType>,
     /// <p>The scopes, URLs, and identifiers for your external identity provider. The following examples describe the provider detail keys for each IdP type. These values and their schema are subject to change. Social IdP <code>authorize_scopes</code> values must match the values listed here.</p>
     /// <dl>
     /// <dt>
@@ -78,7 +78,7 @@
         self.provider_name.as_deref()
     }
     /// <p>The type of IdP. Either SAML, OIDC, or a named social identity provider.</p>
-    pub fn provider_type(&self) -> ::std::option::Option<&crate::types::IdentityProviderTypeType> {
+    pub fn provider_type(&self) -> ::std::option::Option<&super::super::types::IdentityProviderTypeType> {
         self.provider_type.as_ref()
     }
     /// <p>The scopes, URLs, and identifiers for your external identity provider. The following examples describe the provider detail keys for each IdP type. These values and their schema are subject to change. Social IdP <code>authorize_scopes</code> values must match the values listed here.</p>
@@ -153,8 +153,8 @@
 }
 impl IdentityProviderType {
     /// Creates a new builder-style object to manufacture [`IdentityProviderType`](crate::types::IdentityProviderType).
-    pub fn builder() -> crate::types::builders::IdentityProviderTypeBuilder {
-        crate::types::builders::IdentityProviderTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::IdentityProviderTypeBuilder {
+        super::super::types::builders::IdentityProviderTypeBuilder::default()
     }
 }

@@ -164,7 +164,7 @@
 pub struct IdentityProviderTypeBuilder {
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) provider_name: ::std::option::Option<::std::string::String>,
-    pub(crate) provider_type: ::std::option::Option<crate::types::IdentityProviderTypeType>,
+    pub(crate) provider_type: ::std::option::Option<super::super::types::IdentityProviderTypeType>,
     pub(crate) provider_details: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     pub(crate) attribute_mapping: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     pub(crate) idp_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -201,17 +201,17 @@
         &self.provider_name
     }
     /// <p>The type of IdP. Either SAML, OIDC, or a named social identity provider.</p>
-    pub fn provider_type(mut self, input: crate::types::IdentityProviderTypeType) -> Self {
+    pub fn provider_type(mut self, input: super::super::types::IdentityProviderTypeType) -> Self {
         self.provider_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of IdP. Either SAML, OIDC, or a named social identity provider.</p>
-    pub fn set_provider_type(mut self, input: ::std::option::Option<crate::types::IdentityProviderTypeType>) -> Self {
+    pub fn set_provider_type(mut self, input: ::std::option::Option<super::super::types::IdentityProviderTypeType>) -> Self {
         self.provider_type = input;
         self
     }
     /// <p>The type of IdP. Either SAML, OIDC, or a named social identity provider.</p>
-    pub fn get_provider_type(&self) -> &::std::option::Option<crate::types::IdentityProviderTypeType> {
+    pub fn get_provider_type(&self) -> &::std::option::Option<super::super::types::IdentityProviderTypeType> {
         &self.provider_type
     }
     /// Adds a key-value pair to `provider_details`.
@@ -458,8 +458,8 @@
         &self.creation_date
     }
     /// Consumes the builder and constructs a [`IdentityProviderType`](crate::types::IdentityProviderType).
-    pub fn build(self) -> crate::types::IdentityProviderType {
-        crate::types::IdentityProviderType {
+    pub fn build(self) -> super::super::types::IdentityProviderType {
+        super::super::types::IdentityProviderType {
             user_pool_id: self.user_pool_id,
             provider_name: self.provider_name,
             provider_type: self.provider_type,
