--- original
+++ modified
@@ -21,7 +21,7 @@
     /// <p>Updated issuers have the format of <code>https://issuer-cognito-idp.\[region\].amazonaws.com/\[userPoolId\]</code>, where region is the primary Amazon Web Services Region of your user pool.</p>
     /// </dd>
     /// </dl>
-    pub r#type: ::std::option::Option<crate::types::IssuerType>,
+    pub r#type: ::std::option::Option<super::types::IssuerType>,
 }
 impl IssuerConfigurationType {
     /// <p>The type of issuer configuration. Determines the token issuing behavior for the user pool.</p>
@@ -41,14 +41,14 @@
     /// <p>Updated issuers have the format of <code>https://issuer-cognito-idp.\[region\].amazonaws.com/\[userPoolId\]</code>, where region is the primary Amazon Web Services Region of your user pool.</p>
     /// </dd>
     /// </dl>
-    pub fn r#type(&self) -> ::std::option::Option<&crate::types::IssuerType> {
+    pub fn r#type(&self) -> ::std::option::Option<&super::types::IssuerType> {
         self.r#type.as_ref()
     }
 }
 impl IssuerConfigurationType {
     /// Creates a new builder-style object to manufacture [`IssuerConfigurationType`](crate::types::IssuerConfigurationType).
-    pub fn builder() -> crate::types::builders::IssuerConfigurationTypeBuilder {
-        crate::types::builders::IssuerConfigurationTypeBuilder::default()
+    pub fn builder() -> super::types::builders::IssuerConfigurationTypeBuilder {
+        super::types::builders::IssuerConfigurationTypeBuilder::default()
     }
 }

@@ -56,7 +56,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct IssuerConfigurationTypeBuilder {
-    pub(crate) r#type: ::std::option::Option<crate::types::IssuerType>,
+    pub(crate) r#type: ::std::option::Option<super::types::IssuerType>,
 }
 impl IssuerConfigurationTypeBuilder {
     /// <p>The type of issuer configuration. Determines the token issuing behavior for the user pool.</p>
@@ -76,7 +76,7 @@
     /// <p>Updated issuers have the format of <code>https://issuer-cognito-idp.\[region\].amazonaws.com/\[userPoolId\]</code>, where region is the primary Amazon Web Services Region of your user pool.</p>
     /// </dd>
     /// </dl>
-    pub fn r#type(mut self, input: crate::types::IssuerType) -> Self {
+    pub fn r#type(mut self, input: super::types::IssuerType) -> Self {
         self.r#type = ::std::option::Option::Some(input);
         self
     }
@@ -97,7 +97,7 @@
     /// <p>Updated issuers have the format of <code>https://issuer-cognito-idp.\[region\].amazonaws.com/\[userPoolId\]</code>, where region is the primary Amazon Web Services Region of your user pool.</p>
     /// </dd>
     /// </dl>
-    pub fn set_type(mut self, input: ::std::option::Option<crate::types::IssuerType>) -> Self {
+    pub fn set_type(mut self, input: ::std::option::Option<super::types::IssuerType>) -> Self {
         self.r#type = input;
         self
     }
@@ -118,11 +118,11 @@
     /// <p>Updated issuers have the format of <code>https://issuer-cognito-idp.\[region\].amazonaws.com/\[userPoolId\]</code>, where region is the primary Amazon Web Services Region of your user pool.</p>
     /// </dd>
     /// </dl>
-    pub fn get_type(&self) -> &::std::option::Option<crate::types::IssuerType> {
+    pub fn get_type(&self) -> &::std::option::Option<super::types::IssuerType> {
         &self.r#type
     }
     /// Consumes the builder and constructs a [`IssuerConfigurationType`](crate::types::IssuerConfigurationType).
-    pub fn build(self) -> crate::types::IssuerConfigurationType {
-        crate::types::IssuerConfigurationType { r#type: self.r#type }
+    pub fn build(self) -> super::types::IssuerConfigurationType {
+        super::types::IssuerConfigurationType { r#type: self.r#type }
     }
 }
