--- original
+++ modified
@@ -25,7 +25,7 @@
     /// </ul>
     pub settings: ::std::option::Option<::aws_smithy_types::Document>,
     /// <p>An array of image files that you want to apply to roles like backgrounds, logos, and icons. Each object must also indicate whether it is for dark mode, light mode, or browser-adaptive mode.</p>
-    pub assets: ::std::option::Option<::std::vec::Vec<crate::types::AssetType>>,
+    pub assets: ::std::option::Option<::std::vec::Vec<super::types::AssetType>>,
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
     pub creation_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The date and time when the item was modified. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -63,7 +63,7 @@
     /// <p>An array of image files that you want to apply to roles like backgrounds, logos, and icons. Each object must also indicate whether it is for dark mode, light mode, or browser-adaptive mode.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.assets.is_none()`.
-    pub fn assets(&self) -> &[crate::types::AssetType] {
+    pub fn assets(&self) -> &[super::types::AssetType] {
         self.assets.as_deref().unwrap_or_default()
     }
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -77,8 +77,8 @@
 }
 impl ManagedLoginBrandingType {
     /// Creates a new builder-style object to manufacture [`ManagedLoginBrandingType`](crate::types::ManagedLoginBrandingType).
-    pub fn builder() -> crate::types::builders::ManagedLoginBrandingTypeBuilder {
-        crate::types::builders::ManagedLoginBrandingTypeBuilder::default()
+    pub fn builder() -> super::types::builders::ManagedLoginBrandingTypeBuilder {
+        super::types::builders::ManagedLoginBrandingTypeBuilder::default()
     }
 }

@@ -90,7 +90,7 @@
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) use_cognito_provided_values: ::std::option::Option<bool>,
     pub(crate) settings: ::std::option::Option<::aws_smithy_types::Document>,
-    pub(crate) assets: ::std::option::Option<::std::vec::Vec<crate::types::AssetType>>,
+    pub(crate) assets: ::std::option::Option<::std::vec::Vec<super::types::AssetType>>,
     pub(crate) creation_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) last_modified_date: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -192,7 +192,7 @@
     /// To override the contents of this collection use [`set_assets`](Self::set_assets).
     ///
     /// <p>An array of image files that you want to apply to roles like backgrounds, logos, and icons. Each object must also indicate whether it is for dark mode, light mode, or browser-adaptive mode.</p>
-    pub fn assets(mut self, input: crate::types::AssetType) -> Self {
+    pub fn assets(mut self, input: super::types::AssetType) -> Self {
         let mut v = self.assets.unwrap_or_default();
         v.push(input);
         self.assets = ::std::option::Option::Some(v);
@@ -199,12 +199,12 @@
         self
     }
     /// <p>An array of image files that you want to apply to roles like backgrounds, logos, and icons. Each object must also indicate whether it is for dark mode, light mode, or browser-adaptive mode.</p>
-    pub fn set_assets(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::AssetType>>) -> Self {
+    pub fn set_assets(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::AssetType>>) -> Self {
         self.assets = input;
         self
     }
     /// <p>An array of image files that you want to apply to roles like backgrounds, logos, and icons. Each object must also indicate whether it is for dark mode, light mode, or browser-adaptive mode.</p>
-    pub fn get_assets(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::AssetType>> {
+    pub fn get_assets(&self) -> &::std::option::Option<::std::vec::Vec<super::types::AssetType>> {
         &self.assets
     }
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -236,8 +236,8 @@
         &self.last_modified_date
     }
     /// Consumes the builder and constructs a [`ManagedLoginBrandingType`](crate::types::ManagedLoginBrandingType).
-    pub fn build(self) -> crate::types::ManagedLoginBrandingType {
-        crate::types::ManagedLoginBrandingType {
+    pub fn build(self) -> super::types::ManagedLoginBrandingType {
+        super::types::ManagedLoginBrandingType {
             managed_login_branding_id: self.managed_login_branding_id,
             user_pool_id: self.user_pool_id,
             use_cognito_provided_values: self.use_cognito_provided_values.unwrap_or_default(),
