--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl MessageTemplateType {
     /// Creates a new builder-style object to manufacture [`MessageTemplateType`](crate::types::MessageTemplateType).
-    pub fn builder() -> crate::types::builders::MessageTemplateTypeBuilder {
-        crate::types::builders::MessageTemplateTypeBuilder::default()
+    pub fn builder() -> super::types::builders::MessageTemplateTypeBuilder {
+        super::types::builders::MessageTemplateTypeBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.email_subject
     }
     /// Consumes the builder and constructs a [`MessageTemplateType`](crate::types::MessageTemplateType).
-    pub fn build(self) -> crate::types::MessageTemplateType {
-        crate::types::MessageTemplateType {
+    pub fn build(self) -> super::types::MessageTemplateType {
+        super::types::MessageTemplateType {
             sms_message: self.sms_message,
             email_message: self.email_message,
             email_subject: self.email_subject,
