--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct MfaOptionType {
     /// <p>The delivery medium to send the MFA code. You can use this parameter to set only the <code>SMS</code> delivery medium value.</p>
-    pub delivery_medium: ::std::option::Option<crate::types::DeliveryMediumType>,
+    pub delivery_medium: ::std::option::Option<super::types::DeliveryMediumType>,
     /// <p>The attribute name of the MFA option type. The only valid value is <code>phone_number</code>.</p>
     pub attribute_name: ::std::option::Option<::std::string::String>,
 }
 impl MfaOptionType {
     /// <p>The delivery medium to send the MFA code. You can use this parameter to set only the <code>SMS</code> delivery medium value.</p>
-    pub fn delivery_medium(&self) -> ::std::option::Option<&crate::types::DeliveryMediumType> {
+    pub fn delivery_medium(&self) -> ::std::option::Option<&super::types::DeliveryMediumType> {
         self.delivery_medium.as_ref()
     }
     /// <p>The attribute name of the MFA option type. The only valid value is <code>phone_number</code>.</p>
@@ -21,8 +21,8 @@
 }
 impl MfaOptionType {
     /// Creates a new builder-style object to manufacture [`MfaOptionType`](crate::types::MfaOptionType).
-    pub fn builder() -> crate::types::builders::MfaOptionTypeBuilder {
-        crate::types::builders::MfaOptionTypeBuilder::default()
+    pub fn builder() -> super::types::builders::MfaOptionTypeBuilder {
+        super::types::builders::MfaOptionTypeBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct MfaOptionTypeBuilder {
-    pub(crate) delivery_medium: ::std::option::Option<crate::types::DeliveryMediumType>,
+    pub(crate) delivery_medium: ::std::option::Option<super::types::DeliveryMediumType>,
     pub(crate) attribute_name: ::std::option::Option<::std::string::String>,
 }
 impl MfaOptionTypeBuilder {
     /// <p>The delivery medium to send the MFA code. You can use this parameter to set only the <code>SMS</code> delivery medium value.</p>
-    pub fn delivery_medium(mut self, input: crate::types::DeliveryMediumType) -> Self {
+    pub fn delivery_medium(mut self, input: super::types::DeliveryMediumType) -> Self {
         self.delivery_medium = ::std::option::Option::Some(input);
         self
     }
     /// <p>The delivery medium to send the MFA code. You can use this parameter to set only the <code>SMS</code> delivery medium value.</p>
-    pub fn set_delivery_medium(mut self, input: ::std::option::Option<crate::types::DeliveryMediumType>) -> Self {
+    pub fn set_delivery_medium(mut self, input: ::std::option::Option<super::types::DeliveryMediumType>) -> Self {
         self.delivery_medium = input;
         self
     }
     /// <p>The delivery medium to send the MFA code. You can use this parameter to set only the <code>SMS</code> delivery medium value.</p>
-    pub fn get_delivery_medium(&self) -> &::std::option::Option<crate::types::DeliveryMediumType> {
+    pub fn get_delivery_medium(&self) -> &::std::option::Option<super::types::DeliveryMediumType> {
         &self.delivery_medium
     }
     /// <p>The attribute name of the MFA option type. The only valid value is <code>phone_number</code>.</p>
@@ -63,8 +63,8 @@
         &self.attribute_name
     }
     /// Consumes the builder and constructs a [`MfaOptionType`](crate::types::MfaOptionType).
-    pub fn build(self) -> crate::types::MfaOptionType {
-        crate::types::MfaOptionType {
+    pub fn build(self) -> super::types::MfaOptionType {
+        super::types::MfaOptionType {
             delivery_medium: self.delivery_medium,
             attribute_name: self.attribute_name,
         }
