--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl NewDeviceMetadataType {
     /// Creates a new builder-style object to manufacture [`NewDeviceMetadataType`](crate::types::NewDeviceMetadataType).
-    pub fn builder() -> crate::types::builders::NewDeviceMetadataTypeBuilder {
-        crate::types::builders::NewDeviceMetadataTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::NewDeviceMetadataTypeBuilder {
+        super::super::types::builders::NewDeviceMetadataTypeBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.device_group_key
     }
     /// Consumes the builder and constructs a [`NewDeviceMetadataType`](crate::types::NewDeviceMetadataType).
-    pub fn build(self) -> crate::types::NewDeviceMetadataType {
-        crate::types::NewDeviceMetadataType {
+    pub fn build(self) -> super::super::types::NewDeviceMetadataType {
+        super::super::types::NewDeviceMetadataType {
             device_key: self.device_key,
             device_group_key: self.device_group_key,
         }
