--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl NotifyEmailType {
     /// Creates a new builder-style object to manufacture [`NotifyEmailType`](crate::types::NotifyEmailType).
-    pub fn builder() -> crate::types::builders::NotifyEmailTypeBuilder {
-        crate::types::builders::NotifyEmailTypeBuilder::default()
+    pub fn builder() -> super::types::builders::NotifyEmailTypeBuilder {
+        super::types::builders::NotifyEmailTypeBuilder::default()
     }
 }

@@ -88,8 +88,8 @@
     /// Consumes the builder and constructs a [`NotifyEmailType`](crate::types::NotifyEmailType).
     /// This method will fail if any of the following fields are not set:
     /// - [`subject`](crate::types::builders::NotifyEmailTypeBuilder::subject)
-    pub fn build(self) -> ::std::result::Result<crate::types::NotifyEmailType, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::NotifyEmailType {
+    pub fn build(self) -> ::std::result::Result<super::types::NotifyEmailType, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::NotifyEmailType {
             subject: self.subject.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "subject",
