--- original
+++ modified
@@ -55,8 +55,8 @@
 }
 impl PasswordPolicyType {
     /// Creates a new builder-style object to manufacture [`PasswordPolicyType`](crate::types::PasswordPolicyType).
-    pub fn builder() -> crate::types::builders::PasswordPolicyTypeBuilder {
-        crate::types::builders::PasswordPolicyTypeBuilder::default()
+    pub fn builder() -> super::types::builders::PasswordPolicyTypeBuilder {
+        super::types::builders::PasswordPolicyTypeBuilder::default()
     }
 }

@@ -178,8 +178,8 @@
         &self.temporary_password_validity_days
     }
     /// Consumes the builder and constructs a [`PasswordPolicyType`](crate::types::PasswordPolicyType).
-    pub fn build(self) -> crate::types::PasswordPolicyType {
-        crate::types::PasswordPolicyType {
+    pub fn build(self) -> super::types::PasswordPolicyType {
+        super::types::PasswordPolicyType {
             minimum_length: self.minimum_length,
             require_uppercase: self.require_uppercase.unwrap_or_default(),
             require_lowercase: self.require_lowercase.unwrap_or_default(),
