--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the IdP, for example <code>MySAMLProvider</code>.</p>
     pub provider_name: ::std::option::Option<::std::string::String>,
     /// <p>The type of the provider, for example <code>SAML</code>. Amazon Cognito supports SAML 2.0, OIDC, and social IdPs. User pools list supported social IdPs by name in this response parameter: Facebook, Google, Login with Amazon, and Sign in with Apple.</p>
-    pub provider_type: ::std::option::Option<crate::types::IdentityProviderTypeType>,
+    pub provider_type: ::std::option::Option<super::super::types::IdentityProviderTypeType>,
     /// <p>The date and time when the item was modified. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
     pub last_modified_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The date and time when the item was created. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -19,7 +19,7 @@
         self.provider_name.as_deref()
     }
     /// <p>The type of the provider, for example <code>SAML</code>. Amazon Cognito supports SAML 2.0, OIDC, and social IdPs. User pools list supported social IdPs by name in this response parameter: Facebook, Google, Login with Amazon, and Sign in with Apple.</p>
-    pub fn provider_type(&self) -> ::std::option::Option<&crate::types::IdentityProviderTypeType> {
+    pub fn provider_type(&self) -> ::std::option::Option<&super::super::types::IdentityProviderTypeType> {
         self.provider_type.as_ref()
     }
     /// <p>The date and time when the item was modified. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -33,8 +33,8 @@
 }
 impl ProviderDescription {
     /// Creates a new builder-style object to manufacture [`ProviderDescription`](crate::types::ProviderDescription).
-    pub fn builder() -> crate::types::builders::ProviderDescriptionBuilder {
-        crate::types::builders::ProviderDescriptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::ProviderDescriptionBuilder {
+        super::super::types::builders::ProviderDescriptionBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
 #[non_exhaustive]
 pub struct ProviderDescriptionBuilder {
     pub(crate) provider_name: ::std::option::Option<::std::string::String>,
-    pub(crate) provider_type: ::std::option::Option<crate::types::IdentityProviderTypeType>,
+    pub(crate) provider_type: ::std::option::Option<super::super::types::IdentityProviderTypeType>,
     pub(crate) last_modified_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) creation_date: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -63,17 +63,17 @@
         &self.provider_name
     }
     /// <p>The type of the provider, for example <code>SAML</code>. Amazon Cognito supports SAML 2.0, OIDC, and social IdPs. User pools list supported social IdPs by name in this response parameter: Facebook, Google, Login with Amazon, and Sign in with Apple.</p>
-    pub fn provider_type(mut self, input: crate::types::IdentityProviderTypeType) -> Self {
+    pub fn provider_type(mut self, input: super::super::types::IdentityProviderTypeType) -> Self {
         self.provider_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of the provider, for example <code>SAML</code>. Amazon Cognito supports SAML 2.0, OIDC, and social IdPs. User pools list supported social IdPs by name in this response parameter: Facebook, Google, Login with Amazon, and Sign in with Apple.</p>
-    pub fn set_provider_type(mut self, input: ::std::option::Option<crate::types::IdentityProviderTypeType>) -> Self {
+    pub fn set_provider_type(mut self, input: ::std::option::Option<super::super::types::IdentityProviderTypeType>) -> Self {
         self.provider_type = input;
         self
     }
     /// <p>The type of the provider, for example <code>SAML</code>. Amazon Cognito supports SAML 2.0, OIDC, and social IdPs. User pools list supported social IdPs by name in this response parameter: Facebook, Google, Login with Amazon, and Sign in with Apple.</p>
-    pub fn get_provider_type(&self) -> &::std::option::Option<crate::types::IdentityProviderTypeType> {
+    pub fn get_provider_type(&self) -> &::std::option::Option<super::super::types::IdentityProviderTypeType> {
         &self.provider_type
     }
     /// <p>The date and time when the item was modified. Amazon Cognito returns this timestamp in UNIX epoch time format. Your SDK might render the output in a human-readable format like ISO 8601 or a Java <code>Date</code> object.</p>
@@ -105,8 +105,8 @@
         &self.creation_date
     }
     /// Consumes the builder and constructs a [`ProviderDescription`](crate::types::ProviderDescription).
-    pub fn build(self) -> crate::types::ProviderDescription {
-        crate::types::ProviderDescription {
+    pub fn build(self) -> super::super::types::ProviderDescription {
+        super::super::types::ProviderDescription {
             provider_name: self.provider_name,
             provider_type: self.provider_type,
             last_modified_date: self.last_modified_date,
