--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>Your priority preference for using the specified attribute in account recovery. The highest priority is <code>1</code>.</p>
     pub priority: i32,
     /// <p>The recovery method that this object sets a recovery option for.</p>
-    pub name: crate::types::RecoveryOptionNameType,
+    pub name: super::types::RecoveryOptionNameType,
 }
 impl RecoveryOptionType {
     /// <p>Your priority preference for using the specified attribute in account recovery. The highest priority is <code>1</code>.</p>
@@ -16,14 +16,14 @@
         self.priority
     }
     /// <p>The recovery method that this object sets a recovery option for.</p>
-    pub fn name(&self) -> &crate::types::RecoveryOptionNameType {
+    pub fn name(&self) -> &super::types::RecoveryOptionNameType {
         &self.name
     }
 }
 impl RecoveryOptionType {
     /// Creates a new builder-style object to manufacture [`RecoveryOptionType`](crate::types::RecoveryOptionType).
-    pub fn builder() -> crate::types::builders::RecoveryOptionTypeBuilder {
-        crate::types::builders::RecoveryOptionTypeBuilder::default()
+    pub fn builder() -> super::types::builders::RecoveryOptionTypeBuilder {
+        super::types::builders::RecoveryOptionTypeBuilder::default()
     }
 }

@@ -32,7 +32,7 @@
 #[non_exhaustive]
 pub struct RecoveryOptionTypeBuilder {
     pub(crate) priority: ::std::option::Option<i32>,
-    pub(crate) name: ::std::option::Option<crate::types::RecoveryOptionNameType>,
+    pub(crate) name: ::std::option::Option<super::types::RecoveryOptionNameType>,
 }
 impl RecoveryOptionTypeBuilder {
     /// <p>Your priority preference for using the specified attribute in account recovery. The highest priority is <code>1</code>.</p>
@@ -52,17 +52,17 @@
     }
     /// <p>The recovery method that this object sets a recovery option for.</p>
     /// This field is required.
-    pub fn name(mut self, input: crate::types::RecoveryOptionNameType) -> Self {
+    pub fn name(mut self, input: super::types::RecoveryOptionNameType) -> Self {
         self.name = ::std::option::Option::Some(input);
         self
     }
     /// <p>The recovery method that this object sets a recovery option for.</p>
-    pub fn set_name(mut self, input: ::std::option::Option<crate::types::RecoveryOptionNameType>) -> Self {
+    pub fn set_name(mut self, input: ::std::option::Option<super::types::RecoveryOptionNameType>) -> Self {
         self.name = input;
         self
     }
     /// <p>The recovery method that this object sets a recovery option for.</p>
-    pub fn get_name(&self) -> &::std::option::Option<crate::types::RecoveryOptionNameType> {
+    pub fn get_name(&self) -> &::std::option::Option<super::types::RecoveryOptionNameType> {
         &self.name
     }
     /// Consumes the builder and constructs a [`RecoveryOptionType`](crate::types::RecoveryOptionType).
@@ -69,8 +69,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`priority`](crate::types::builders::RecoveryOptionTypeBuilder::priority)
     /// - [`name`](crate::types::builders::RecoveryOptionTypeBuilder::name)
-    pub fn build(self) -> ::std::result::Result<crate::types::RecoveryOptionType, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::RecoveryOptionType {
+    pub fn build(self) -> ::std::result::Result<super::types::RecoveryOptionType, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::RecoveryOptionType {
             priority: self.priority.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "priority",
