--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct RefreshTokenRotationType {
     /// <p>The state of refresh token rotation for the current app client.</p>
-    pub feature: crate::types::FeatureType,
+    pub feature: super::super::types::FeatureType,
     /// <p>When you request a token refresh with <code>GetTokensFromRefreshToken</code>, the original refresh token that you're rotating out can remain valid for a period of time of up to 60 seconds. This allows for client-side retries. When <code>RetryGracePeriodSeconds</code> is <code>0</code>, the grace period is disabled and a successful request immediately invalidates the submitted refresh token.</p>
     pub retry_grace_period_seconds: ::std::option::Option<i32>,
 }
 impl RefreshTokenRotationType {
     /// <p>The state of refresh token rotation for the current app client.</p>
-    pub fn feature(&self) -> &crate::types::FeatureType {
+    pub fn feature(&self) -> &super::super::types::FeatureType {
         &self.feature
     }
     /// <p>When you request a token refresh with <code>GetTokensFromRefreshToken</code>, the original refresh token that you're rotating out can remain valid for a period of time of up to 60 seconds. This allows for client-side retries. When <code>RetryGracePeriodSeconds</code> is <code>0</code>, the grace period is disabled and a successful request immediately invalidates the submitted refresh token.</p>
@@ -21,8 +21,8 @@
 }
 impl RefreshTokenRotationType {
     /// Creates a new builder-style object to manufacture [`RefreshTokenRotationType`](crate::types::RefreshTokenRotationType).
-    pub fn builder() -> crate::types::builders::RefreshTokenRotationTypeBuilder {
-        crate::types::builders::RefreshTokenRotationTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::RefreshTokenRotationTypeBuilder {
+        super::super::types::builders::RefreshTokenRotationTypeBuilder::default()
     }
 }

@@ -30,23 +30,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct RefreshTokenRotationTypeBuilder {
-    pub(crate) feature: ::std::option::Option<crate::types::FeatureType>,
+    pub(crate) feature: ::std::option::Option<super::super::types::FeatureType>,
     pub(crate) retry_grace_period_seconds: ::std::option::Option<i32>,
 }
 impl RefreshTokenRotationTypeBuilder {
     /// <p>The state of refresh token rotation for the current app client.</p>
     /// This field is required.
-    pub fn feature(mut self, input: crate::types::FeatureType) -> Self {
+    pub fn feature(mut self, input: super::super::types::FeatureType) -> Self {
         self.feature = ::std::option::Option::Some(input);
         self
     }
     /// <p>The state of refresh token rotation for the current app client.</p>
-    pub fn set_feature(mut self, input: ::std::option::Option<crate::types::FeatureType>) -> Self {
+    pub fn set_feature(mut self, input: ::std::option::Option<super::super::types::FeatureType>) -> Self {
         self.feature = input;
         self
     }
     /// <p>The state of refresh token rotation for the current app client.</p>
-    pub fn get_feature(&self) -> &::std::option::Option<crate::types::FeatureType> {
+    pub fn get_feature(&self) -> &::std::option::Option<super::super::types::FeatureType> {
         &self.feature
     }
     /// <p>When you request a token refresh with <code>GetTokensFromRefreshToken</code>, the original refresh token that you're rotating out can remain valid for a period of time of up to 60 seconds. This allows for client-side retries. When <code>RetryGracePeriodSeconds</code> is <code>0</code>, the grace period is disabled and a successful request immediately invalidates the submitted refresh token.</p>
@@ -66,8 +66,8 @@
     /// Consumes the builder and constructs a [`RefreshTokenRotationType`](crate::types::RefreshTokenRotationType).
     /// This method will fail if any of the following fields are not set:
     /// - [`feature`](crate::types::builders::RefreshTokenRotationTypeBuilder::feature)
-    pub fn build(self) -> ::std::result::Result<crate::types::RefreshTokenRotationType, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::RefreshTokenRotationType {
+    pub fn build(self) -> ::std::result::Result<super::super::types::RefreshTokenRotationType, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::RefreshTokenRotationType {
             feature: self.feature.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "feature",
