--- original
+++ modified
@@ -25,8 +25,8 @@
 }
 impl RiskExceptionConfigurationType {
     /// Creates a new builder-style object to manufacture [`RiskExceptionConfigurationType`](crate::types::RiskExceptionConfigurationType).
-    pub fn builder() -> crate::types::builders::RiskExceptionConfigurationTypeBuilder {
-        crate::types::builders::RiskExceptionConfigurationTypeBuilder::default()
+    pub fn builder() -> super::types::builders::RiskExceptionConfigurationTypeBuilder {
+        super::types::builders::RiskExceptionConfigurationTypeBuilder::default()
     }
 }

@@ -79,8 +79,8 @@
         &self.skipped_ip_range_list
     }
     /// Consumes the builder and constructs a [`RiskExceptionConfigurationType`](crate::types::RiskExceptionConfigurationType).
-    pub fn build(self) -> crate::types::RiskExceptionConfigurationType {
-        crate::types::RiskExceptionConfigurationType {
+    pub fn build(self) -> super::types::RiskExceptionConfigurationType {
+        super::types::RiskExceptionConfigurationType {
             blocked_ip_range_list: self.blocked_ip_range_list,
             skipped_ip_range_list: self.skipped_ip_range_list,
         }
