--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The name of your user pool attribute. When you create or update a user pool, adding a schema attribute creates a custom or developer-only attribute. When you add an attribute with a <code>Name</code> value of <code>MyAttribute</code>, Amazon Cognito creates the custom attribute <code>custom:MyAttribute</code>. When <code>DeveloperOnlyAttribute</code> is <code>true</code>, Amazon Cognito creates your attribute as <code>dev:MyAttribute</code>. In an operation that describes a user pool, Amazon Cognito returns this value as <code>value</code> for standard attributes, <code>custom:value</code> for custom attributes, and <code>dev:value</code> for developer-only attributes..</p>
     pub name: ::std::option::Option<::std::string::String>,
     /// <p>The data format of the values for your attribute. When you choose an <code>AttributeDataType</code>, Amazon Cognito validates the input against the data type. A custom attribute value in your user's ID token is always a string, for example <code>"custom:isMember" : "true"</code> or <code>"custom:YearsAsMember" : "12"</code>.</p>
-    pub attribute_data_type: ::std::option::Option<crate::types::AttributeDataType>,
+    pub attribute_data_type: ::std::option::Option<super::super::types::AttributeDataType>,
     /// <note>
     /// <p>You should use <a href="https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_UserPoolClientType.html#CognitoUserPools-Type-UserPoolClientType-WriteAttributes">WriteAttributes</a> in the user pool client to control how attributes can be mutated for new use cases instead of using <code>DeveloperOnlyAttribute</code>.</p>
     /// </note>
@@ -20,9 +20,9 @@
     /// <p>Specifies whether a user pool attribute is required. If the attribute is required and the user doesn't provide a value, registration or sign-in will fail.</p>
     pub required: ::std::option::Option<bool>,
     /// <p>Specifies the constraints for an attribute of the number type.</p>
-    pub number_attribute_constraints: ::std::option::Option<crate::types::NumberAttributeConstraintsType>,
+    pub number_attribute_constraints: ::std::option::Option<super::super::types::NumberAttributeConstraintsType>,
     /// <p>Specifies the constraints for an attribute of the string type.</p>
-    pub string_attribute_constraints: ::std::option::Option<crate::types::StringAttributeConstraintsType>,
+    pub string_attribute_constraints: ::std::option::Option<super::super::types::StringAttributeConstraintsType>,
 }
 impl SchemaAttributeType {
     /// <p>The name of your user pool attribute. When you create or update a user pool, adding a schema attribute creates a custom or developer-only attribute. When you add an attribute with a <code>Name</code> value of <code>MyAttribute</code>, Amazon Cognito creates the custom attribute <code>custom:MyAttribute</code>. When <code>DeveloperOnlyAttribute</code> is <code>true</code>, Amazon Cognito creates your attribute as <code>dev:MyAttribute</code>. In an operation that describes a user pool, Amazon Cognito returns this value as <code>value</code> for standard attributes, <code>custom:value</code> for custom attributes, and <code>dev:value</code> for developer-only attributes..</p>
@@ -30,7 +30,7 @@
         self.name.as_deref()
     }
     /// <p>The data format of the values for your attribute. When you choose an <code>AttributeDataType</code>, Amazon Cognito validates the input against the data type. A custom attribute value in your user's ID token is always a string, for example <code>"custom:isMember" : "true"</code> or <code>"custom:YearsAsMember" : "12"</code>.</p>
-    pub fn attribute_data_type(&self) -> ::std::option::Option<&crate::types::AttributeDataType> {
+    pub fn attribute_data_type(&self) -> ::std::option::Option<&super::super::types::AttributeDataType> {
         self.attribute_data_type.as_ref()
     }
     /// <note>
@@ -50,18 +50,18 @@
         self.required
     }
     /// <p>Specifies the constraints for an attribute of the number type.</p>
-    pub fn number_attribute_constraints(&self) -> ::std::option::Option<&crate::types::NumberAttributeConstraintsType> {
+    pub fn number_attribute_constraints(&self) -> ::std::option::Option<&super::super::types::NumberAttributeConstraintsType> {
         self.number_attribute_constraints.as_ref()
     }
     /// <p>Specifies the constraints for an attribute of the string type.</p>
-    pub fn string_attribute_constraints(&self) -> ::std::option::Option<&crate::types::StringAttributeConstraintsType> {
+    pub fn string_attribute_constraints(&self) -> ::std::option::Option<&super::super::types::StringAttributeConstraintsType> {
         self.string_attribute_constraints.as_ref()
     }
 }
 impl SchemaAttributeType {
     /// Creates a new builder-style object to manufacture [`SchemaAttributeType`](crate::types::SchemaAttributeType).
-    pub fn builder() -> crate::types::builders::SchemaAttributeTypeBuilder {
-        crate::types::builders::SchemaAttributeTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::SchemaAttributeTypeBuilder {
+        super::super::types::builders::SchemaAttributeTypeBuilder::default()
     }
 }

@@ -70,12 +70,12 @@
 #[non_exhaustive]
 pub struct SchemaAttributeTypeBuilder {
     pub(crate) name: ::std::option::Option<::std::string::String>,
-    pub(crate) attribute_data_type: ::std::option::Option<crate::types::AttributeDataType>,
+    pub(crate) attribute_data_type: ::std::option::Option<super::super::types::AttributeDataType>,
     pub(crate) developer_only_attribute: ::std::option::Option<bool>,
     pub(crate) mutable: ::std::option::Option<bool>,
     pub(crate) required: ::std::option::Option<bool>,
-    pub(crate) number_attribute_constraints: ::std::option::Option<crate::types::NumberAttributeConstraintsType>,
-    pub(crate) string_attribute_constraints: ::std::option::Option<crate::types::StringAttributeConstraintsType>,
+    pub(crate) number_attribute_constraints: ::std::option::Option<super::super::types::NumberAttributeConstraintsType>,
+    pub(crate) string_attribute_constraints: ::std::option::Option<super::super::types::StringAttributeConstraintsType>,
 }
 impl SchemaAttributeTypeBuilder {
     /// <p>The name of your user pool attribute. When you create or update a user pool, adding a schema attribute creates a custom or developer-only attribute. When you add an attribute with a <code>Name</code> value of <code>MyAttribute</code>, Amazon Cognito creates the custom attribute <code>custom:MyAttribute</code>. When <code>DeveloperOnlyAttribute</code> is <code>true</code>, Amazon Cognito creates your attribute as <code>dev:MyAttribute</code>. In an operation that describes a user pool, Amazon Cognito returns this value as <code>value</code> for standard attributes, <code>custom:value</code> for custom attributes, and <code>dev:value</code> for developer-only attributes..</p>
@@ -93,17 +93,17 @@
         &self.name
     }
     /// <p>The data format of the values for your attribute. When you choose an <code>AttributeDataType</code>, Amazon Cognito validates the input against the data type. A custom attribute value in your user's ID token is always a string, for example <code>"custom:isMember" : "true"</code> or <code>"custom:YearsAsMember" : "12"</code>.</p>
-    pub fn attribute_data_type(mut self, input: crate::types::AttributeDataType) -> Self {
+    pub fn attribute_data_type(mut self, input: super::super::types::AttributeDataType) -> Self {
         self.attribute_data_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The data format of the values for your attribute. When you choose an <code>AttributeDataType</code>, Amazon Cognito validates the input against the data type. A custom attribute value in your user's ID token is always a string, for example <code>"custom:isMember" : "true"</code> or <code>"custom:YearsAsMember" : "12"</code>.</p>
-    pub fn set_attribute_data_type(mut self, input: ::std::option::Option<crate::types::AttributeDataType>) -> Self {
+    pub fn set_attribute_data_type(mut self, input: ::std::option::Option<super::super::types::AttributeDataType>) -> Self {
         self.attribute_data_type = input;
         self
     }
     /// <p>The data format of the values for your attribute. When you choose an <code>AttributeDataType</code>, Amazon Cognito validates the input against the data type. A custom attribute value in your user's ID token is always a string, for example <code>"custom:isMember" : "true"</code> or <code>"custom:YearsAsMember" : "12"</code>.</p>
-    pub fn get_attribute_data_type(&self) -> &::std::option::Option<crate::types::AttributeDataType> {
+    pub fn get_attribute_data_type(&self) -> &::std::option::Option<super::super::types::AttributeDataType> {
         &self.attribute_data_type
     }
     /// <note>
@@ -161,36 +161,36 @@
         &self.required
     }
     /// <p>Specifies the constraints for an attribute of the number type.</p>
-    pub fn number_attribute_constraints(mut self, input: crate::types::NumberAttributeConstraintsType) -> Self {
+    pub fn number_attribute_constraints(mut self, input: super::super::types::NumberAttributeConstraintsType) -> Self {
         self.number_attribute_constraints = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the constraints for an attribute of the number type.</p>
-    pub fn set_number_attribute_constraints(mut self, input: ::std::option::Option<crate::types::NumberAttributeConstraintsType>) -> Self {
+    pub fn set_number_attribute_constraints(mut self, input: ::std::option::Option<super::super::types::NumberAttributeConstraintsType>) -> Self {
         self.number_attribute_constraints = input;
         self
     }
     /// <p>Specifies the constraints for an attribute of the number type.</p>
-    pub fn get_number_attribute_constraints(&self) -> &::std::option::Option<crate::types::NumberAttributeConstraintsType> {
+    pub fn get_number_attribute_constraints(&self) -> &::std::option::Option<super::super::types::NumberAttributeConstraintsType> {
         &self.number_attribute_constraints
     }
     /// <p>Specifies the constraints for an attribute of the string type.</p>
-    pub fn string_attribute_constraints(mut self, input: crate::types::StringAttributeConstraintsType) -> Self {
+    pub fn string_attribute_constraints(mut self, input: super::super::types::StringAttributeConstraintsType) -> Self {
         self.string_attribute_constraints = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the constraints for an attribute of the string type.</p>
-    pub fn set_string_attribute_constraints(mut self, input: ::std::option::Option<crate::types::StringAttributeConstraintsType>) -> Self {
+    pub fn set_string_attribute_constraints(mut self, input: ::std::option::Option<super::super::types::StringAttributeConstraintsType>) -> Self {
         self.string_attribute_constraints = input;
         self
     }
     /// <p>Specifies the constraints for an attribute of the string type.</p>
-    pub fn get_string_attribute_constraints(&self) -> &::std::option::Option<crate::types::StringAttributeConstraintsType> {
+    pub fn get_string_attribute_constraints(&self) -> &::std::option::Option<super::super::types::StringAttributeConstraintsType> {
         &self.string_attribute_constraints
     }
     /// Consumes the builder and constructs a [`SchemaAttributeType`](crate::types::SchemaAttributeType).
-    pub fn build(self) -> crate::types::SchemaAttributeType {
-        crate::types::SchemaAttributeType {
+    pub fn build(self) -> super::super::types::SchemaAttributeType {
+        super::super::types::SchemaAttributeType {
             name: self.name,
             attribute_data_type: self.attribute_data_type,
             developer_only_attribute: self.developer_only_attribute,
