--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl SoftwareTokenMfaSettingsType {
     /// Creates a new builder-style object to manufacture [`SoftwareTokenMfaSettingsType`](crate::types::SoftwareTokenMfaSettingsType).
-    pub fn builder() -> crate::types::builders::SoftwareTokenMfaSettingsTypeBuilder {
-        crate::types::builders::SoftwareTokenMfaSettingsTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::SoftwareTokenMfaSettingsTypeBuilder {
+        super::super::types::builders::SoftwareTokenMfaSettingsTypeBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.preferred_mfa
     }
     /// Consumes the builder and constructs a [`SoftwareTokenMfaSettingsType`](crate::types::SoftwareTokenMfaSettingsType).
-    pub fn build(self) -> crate::types::SoftwareTokenMfaSettingsType {
-        crate::types::SoftwareTokenMfaSettingsType {
+    pub fn build(self) -> super::super::types::SoftwareTokenMfaSettingsType {
+        super::super::types::SoftwareTokenMfaSettingsType {
             enabled: self.enabled.unwrap_or_default(),
             preferred_mfa: self.preferred_mfa.unwrap_or_default(),
         }
