--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl StringAttributeConstraintsType {
     /// Creates a new builder-style object to manufacture [`StringAttributeConstraintsType`](crate::types::StringAttributeConstraintsType).
-    pub fn builder() -> crate::types::builders::StringAttributeConstraintsTypeBuilder {
-        crate::types::builders::StringAttributeConstraintsTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::StringAttributeConstraintsTypeBuilder {
+        super::super::types::builders::StringAttributeConstraintsTypeBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.max_length
     }
     /// Consumes the builder and constructs a [`StringAttributeConstraintsType`](crate::types::StringAttributeConstraintsType).
-    pub fn build(self) -> crate::types::StringAttributeConstraintsType {
-        crate::types::StringAttributeConstraintsType {
+    pub fn build(self) -> super::super::types::StringAttributeConstraintsType {
+        super::super::types::StringAttributeConstraintsType {
             min_length: self.min_length,
             max_length: self.max_length,
         }
