--- original
+++ modified
@@ -36,8 +36,8 @@
 }
 impl UserPoolClientDescription {
     /// Creates a new builder-style object to manufacture [`UserPoolClientDescription`](crate::types::UserPoolClientDescription).
-    pub fn builder() -> crate::types::builders::UserPoolClientDescriptionBuilder {
-        crate::types::builders::UserPoolClientDescriptionBuilder::default()
+    pub fn builder() -> super::types::builders::UserPoolClientDescriptionBuilder {
+        super::types::builders::UserPoolClientDescriptionBuilder::default()
     }
 }

@@ -93,8 +93,8 @@
         &self.client_name
     }
     /// Consumes the builder and constructs a [`UserPoolClientDescription`](crate::types::UserPoolClientDescription).
-    pub fn build(self) -> crate::types::UserPoolClientDescription {
-        crate::types::UserPoolClientDescription {
+    pub fn build(self) -> super::types::UserPoolClientDescription {
+        super::types::UserPoolClientDescription {
             client_id: self.client_id,
             user_pool_id: self.user_pool_id,
             client_name: self.client_name,
