--- original
+++ modified
@@ -45,8 +45,8 @@
 }
 impl UsernameConfigurationType {
     /// Creates a new builder-style object to manufacture [`UsernameConfigurationType`](crate::types::UsernameConfigurationType).
-    pub fn builder() -> crate::types::builders::UsernameConfigurationTypeBuilder {
-        crate::types::builders::UsernameConfigurationTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::UsernameConfigurationTypeBuilder {
+        super::super::types::builders::UsernameConfigurationTypeBuilder::default()
     }
 }

@@ -120,8 +120,8 @@
     /// Consumes the builder and constructs a [`UsernameConfigurationType`](crate::types::UsernameConfigurationType).
     /// This method will fail if any of the following fields are not set:
     /// - [`case_sensitive`](crate::types::builders::UsernameConfigurationTypeBuilder::case_sensitive)
-    pub fn build(self) -> ::std::result::Result<crate::types::UsernameConfigurationType, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::UsernameConfigurationType {
+    pub fn build(self) -> ::std::result::Result<super::super::types::UsernameConfigurationType, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::UsernameConfigurationType {
             case_sensitive: self.case_sensitive.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "case_sensitive",
