--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl WebAuthnMfaSettingsType {
     /// Creates a new builder-style object to manufacture [`WebAuthnMfaSettingsType`](crate::types::WebAuthnMfaSettingsType).
-    pub fn builder() -> crate::types::builders::WebAuthnMfaSettingsTypeBuilder {
-        crate::types::builders::WebAuthnMfaSettingsTypeBuilder::default()
+    pub fn builder() -> super::super::types::builders::WebAuthnMfaSettingsTypeBuilder {
+        super::super::types::builders::WebAuthnMfaSettingsTypeBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.enabled
     }
     /// Consumes the builder and constructs a [`WebAuthnMfaSettingsType`](crate::types::WebAuthnMfaSettingsType).
-    pub fn build(self) -> crate::types::WebAuthnMfaSettingsType {
-        crate::types::WebAuthnMfaSettingsType {
+    pub fn build(self) -> super::super::types::WebAuthnMfaSettingsType {
+        super::super::types::WebAuthnMfaSettingsType {
             enabled: self.enabled.unwrap_or_default(),
         }
     }
