--- original
+++ modified
@@ -49,8 +49,8 @@
 }
 impl WebAuthnCredentialDescription {
     /// Creates a new builder-style object to manufacture [`WebAuthnCredentialDescription`](crate::types::WebAuthnCredentialDescription).
-    pub fn builder() -> crate::types::builders::WebAuthnCredentialDescriptionBuilder {
-        crate::types::builders::WebAuthnCredentialDescriptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::WebAuthnCredentialDescriptionBuilder {
+        super::super::types::builders::WebAuthnCredentialDescriptionBuilder::default()
     }
 }

@@ -167,8 +167,8 @@
     /// - [`relying_party_id`](crate::types::builders::WebAuthnCredentialDescriptionBuilder::relying_party_id)
     /// - [`authenticator_transports`](crate::types::builders::WebAuthnCredentialDescriptionBuilder::authenticator_transports)
     /// - [`created_at`](crate::types::builders::WebAuthnCredentialDescriptionBuilder::created_at)
-    pub fn build(self) -> ::std::result::Result<crate::types::WebAuthnCredentialDescription, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::WebAuthnCredentialDescription {
+    pub fn build(self) -> ::std::result::Result<super::super::types::WebAuthnCredentialDescription, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::WebAuthnCredentialDescription {
             credential_id: self.credential_id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "credential_id",
