--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteEmailTemplateInput {
     /// Creates a new builder-style object to manufacture [`DeleteEmailTemplateInput`](crate::operation::delete_email_template::DeleteEmailTemplateInput).
-    pub fn builder() -> crate::operation::delete_email_template::builders::DeleteEmailTemplateInputBuilder {
-        crate::operation::delete_email_template::builders::DeleteEmailTemplateInputBuilder::default()
+    pub fn builder() -> super::operation::delete_email_template::builders::DeleteEmailTemplateInputBuilder {
+        super::operation::delete_email_template::builders::DeleteEmailTemplateInputBuilder::default()
     }
 }

@@ -45,9 +45,9 @@
     /// Consumes the builder and constructs a [`DeleteEmailTemplateInput`](crate::operation::delete_email_template::DeleteEmailTemplateInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_email_template::DeleteEmailTemplateInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::delete_email_template::DeleteEmailTemplateInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_email_template::DeleteEmailTemplateInput {
+        ::std::result::Result::Ok(super::operation::delete_email_template::DeleteEmailTemplateInput {
             template_name: self.template_name,
         })
     }
