--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteEventSourceMappingInput {
     /// Creates a new builder-style object to manufacture [`DeleteEventSourceMappingInput`](crate::operation::delete_event_source_mapping::DeleteEventSourceMappingInput).
-    pub fn builder() -> crate::operation::delete_event_source_mapping::builders::DeleteEventSourceMappingInputBuilder {
-        crate::operation::delete_event_source_mapping::builders::DeleteEventSourceMappingInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_event_source_mapping::builders::DeleteEventSourceMappingInputBuilder {
+        super::super::super::operation::delete_event_source_mapping::builders::DeleteEventSourceMappingInputBuilder::default()
     }
 }

@@ -45,9 +45,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_event_source_mapping::DeleteEventSourceMappingInput,
+        super::super::super::operation::delete_event_source_mapping::DeleteEventSourceMappingInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_event_source_mapping::DeleteEventSourceMappingInput { uuid: self.uuid })
+        ::std::result::Result::Ok(super::super::super::operation::delete_event_source_mapping::DeleteEventSourceMappingInput { uuid: self.uuid })
     }
 }
