--- original
+++ modified
@@ -4,7 +4,7 @@
  *  SPDX-License-Identifier: Apache-2.0
  */

-use crate::endpoint_lib::diagnostic::DiagnosticCollector;
+use super::endpoint_lib::diagnostic::DiagnosticCollector;

 pub(crate) fn is_valid_host_label(label: &str, allow_dots: bool, e: &mut DiagnosticCollector) -> bool {
     let bytes = label.as_bytes();
@@ -89,7 +89,7 @@
         assert_eq!(is_valid_host_label("a-b-c", false), true);
     }

-    use crate::endpoint_lib::diagnostic::DiagnosticCollector;
+    use super::endpoint_lib::diagnostic::DiagnosticCollector;
     use proptest::prelude::*;
     proptest! {
         #[test]
