--- original
+++ modified
@@ -90,7 +90,7 @@

 macro_rules! ite {
     ($b:expr, $true_val:expr, $false_val:expr) => {{
-        use crate::endpoint_lib::ite::Ite;
+        use super::super::endpoint_lib::ite::Ite;
         let b = $b;
         let true_val = $true_val;
         let false_val = $false_val;
@@ -99,39 +99,3 @@
 }

 pub(crate) use ite;
-
-#[cfg(test)]
-mod test {
-
-    #[test]
-    fn base_cases() {
-        // All types optional
-        let a = "a";
-        let some_a = Some("a");
-        let b = "b";
-        let some_b = Some("b");
-        let none: Option<&str> = None;
-
-        // Option, Option
-        assert_eq!(ite!(true, some_a, some_b), Some("a"));
-        assert_eq!(ite!(false, some_a, some_b), Some("b"));
-        assert_eq!(ite!(false, some_a, none), None);
-        assert_eq!(ite!(true, some_a, none), Some("a"));
-
-        // Option, Value
-        assert_eq!(ite!(true, some_a, b), Some("a"));
-        assert_eq!(ite!(false, some_a, b), Some("b"));
-        assert_eq!(ite!(true, none, b), None);
-        assert_eq!(ite!(false, none, b), Some("b"));
-
-        // Value, Option
-        assert_eq!(ite!(true, a, some_b), Some("a"));
-        assert_eq!(ite!(false, a, some_b), Some("b"));
-        assert_eq!(ite!(false, a, none), None);
-        assert_eq!(ite!(true, a, none), Some("a"));
-
-        // Value, Value
-        assert_eq!(ite!(true, a, b), "a");
-        assert_eq!(ite!(false, a, b), "b");
-    }
-}
