--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl CheckIfPhoneNumberIsOptedOutInput {
     /// Creates a new builder-style object to manufacture [`CheckIfPhoneNumberIsOptedOutInput`](crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutInput).
-    pub fn builder() -> crate::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutInputBuilder {
-        crate::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutInputBuilder::default()
+    pub fn builder() -> super::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutInputBuilder {
+        super::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutInputBuilder::default()
     }
 }

@@ -53,10 +53,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutInput,
+        super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutInput {
+        ::std::result::Result::Ok(super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutInput {
             phone_number: self.phone_number,
         })
     }
