--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl DomainEntryPoint {
     /// Creates a new builder-style object to manufacture [`DomainEntryPoint`](crate::types::DomainEntryPoint).
-    pub fn builder() -> crate::types::builders::DomainEntryPointBuilder {
-        crate::types::builders::DomainEntryPointBuilder::default()
+    pub fn builder() -> super::types::builders::DomainEntryPointBuilder {
+        super::types::builders::DomainEntryPointBuilder::default()
     }
 }

@@ -65,8 +65,8 @@
         &self.external_connection_name
     }
     /// Consumes the builder and constructs a [`DomainEntryPoint`](crate::types::DomainEntryPoint).
-    pub fn build(self) -> crate::types::DomainEntryPoint {
-        crate::types::DomainEntryPoint {
+    pub fn build(self) -> super::types::DomainEntryPoint {
+        super::types::DomainEntryPoint {
             repository_name: self.repository_name,
             external_connection_name: self.external_connection_name,
         }
