--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The Amazon Resource Name (ARN) of the domain.</p>
     pub arn: ::std::option::Option<::std::string::String>,
     /// <p>The current status of a domain.</p>
-    pub status: ::std::option::Option<crate::types::DomainStatus>,
+    pub status: ::std::option::Option<super::super::types::DomainStatus>,
     /// <p>A timestamp that represents the date and time the domain was created.</p>
     pub created_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The ARN of an Key Management Service (KMS) key associated with a domain.</p>
@@ -37,7 +37,7 @@
         self.arn.as_deref()
     }
     /// <p>The current status of a domain.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::DomainStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::DomainStatus> {
         self.status.as_ref()
     }
     /// <p>A timestamp that represents the date and time the domain was created.</p>
@@ -63,8 +63,8 @@
 }
 impl DomainDescription {
     /// Creates a new builder-style object to manufacture [`DomainDescription`](crate::types::DomainDescription).
-    pub fn builder() -> crate::types::builders::DomainDescriptionBuilder {
-        crate::types::builders::DomainDescriptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::DomainDescriptionBuilder {
+        super::super::types::builders::DomainDescriptionBuilder::default()
     }
 }

@@ -75,7 +75,7 @@
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) owner: ::std::option::Option<::std::string::String>,
     pub(crate) arn: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::DomainStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::DomainStatus>,
     pub(crate) created_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) encryption_key: ::std::option::Option<::std::string::String>,
     pub(crate) repository_count: ::std::option::Option<i32>,
@@ -126,17 +126,17 @@
         &self.arn
     }
     /// <p>The current status of a domain.</p>
-    pub fn status(mut self, input: crate::types::DomainStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::DomainStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The current status of a domain.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::DomainStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::DomainStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>The current status of a domain.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::DomainStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::DomainStatus> {
         &self.status
     }
     /// <p>A timestamp that represents the date and time the domain was created.</p>
@@ -210,8 +210,8 @@
         &self.s3_bucket_arn
     }
     /// Consumes the builder and constructs a [`DomainDescription`](crate::types::DomainDescription).
-    pub fn build(self) -> crate::types::DomainDescription {
-        crate::types::DomainDescription {
+    pub fn build(self) -> super::super::types::DomainDescription {
+        super::super::types::DomainDescription {
             name: self.name,
             owner: self.owner,
             arn: self.arn,
