--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct PackageVersionDescription {
     /// <p>The format of the package version.</p>
-    pub format: ::std::option::Option<crate::types::PackageFormat>,
+    pub format: ::std::option::Option<super::super::types::PackageFormat>,
     /// <p>The namespace of the package version. The package component that specifies its namespace depends on its type. For example:</p>
     /// <ul>
     /// <li>
@@ -33,17 +33,17 @@
     /// <p>A timestamp that contains the date and time the package version was published.</p>
     pub published_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>Information about licenses associated with the package version.</p>
-    pub licenses: ::std::option::Option<::std::vec::Vec<crate::types::LicenseInfo>>,
+    pub licenses: ::std::option::Option<::std::vec::Vec<super::super::types::LicenseInfo>>,
     /// <p>The revision of the package version.</p>
     pub revision: ::std::option::Option<::std::string::String>,
     /// <p>A string that contains the status of the package version.</p>
-    pub status: ::std::option::Option<crate::types::PackageVersionStatus>,
+    pub status: ::std::option::Option<super::super::types::PackageVersionStatus>,
     /// <p>A <a href="https://docs.aws.amazon.com/codeartifact/latest/APIReference/API_PackageVersionOrigin.html">PackageVersionOrigin</a> object that contains information about how the package version was added to the repository.</p>
-    pub origin: ::std::option::Option<crate::types::PackageVersionOrigin>,
+    pub origin: ::std::option::Option<super::super::types::PackageVersionOrigin>,
 }
 impl PackageVersionDescription {
     /// <p>The format of the package version.</p>
-    pub fn format(&self) -> ::std::option::Option<&crate::types::PackageFormat> {
+    pub fn format(&self) -> ::std::option::Option<&super::super::types::PackageFormat> {
         self.format.as_ref()
     }
     /// <p>The namespace of the package version. The package component that specifies its namespace depends on its type. For example:</p>
@@ -91,7 +91,7 @@
     /// <p>Information about licenses associated with the package version.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.licenses.is_none()`.
-    pub fn licenses(&self) -> &[crate::types::LicenseInfo] {
+    pub fn licenses(&self) -> &[super::super::types::LicenseInfo] {
         self.licenses.as_deref().unwrap_or_default()
     }
     /// <p>The revision of the package version.</p>
@@ -99,18 +99,18 @@
         self.revision.as_deref()
     }
     /// <p>A string that contains the status of the package version.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::PackageVersionStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::PackageVersionStatus> {
         self.status.as_ref()
     }
     /// <p>A <a href="https://docs.aws.amazon.com/codeartifact/latest/APIReference/API_PackageVersionOrigin.html">PackageVersionOrigin</a> object that contains information about how the package version was added to the repository.</p>
-    pub fn origin(&self) -> ::std::option::Option<&crate::types::PackageVersionOrigin> {
+    pub fn origin(&self) -> ::std::option::Option<&super::super::types::PackageVersionOrigin> {
         self.origin.as_ref()
     }
 }
 impl PackageVersionDescription {
     /// Creates a new builder-style object to manufacture [`PackageVersionDescription`](crate::types::PackageVersionDescription).
-    pub fn builder() -> crate::types::builders::PackageVersionDescriptionBuilder {
-        crate::types::builders::PackageVersionDescriptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::PackageVersionDescriptionBuilder {
+        super::super::types::builders::PackageVersionDescriptionBuilder::default()
     }
 }

@@ -118,7 +118,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct PackageVersionDescriptionBuilder {
-    pub(crate) format: ::std::option::Option<crate::types::PackageFormat>,
+    pub(crate) format: ::std::option::Option<super::super::types::PackageFormat>,
     pub(crate) namespace: ::std::option::Option<::std::string::String>,
     pub(crate) package_name: ::std::option::Option<::std::string::String>,
     pub(crate) display_name: ::std::option::Option<::std::string::String>,
@@ -127,24 +127,24 @@
     pub(crate) home_page: ::std::option::Option<::std::string::String>,
     pub(crate) source_code_repository: ::std::option::Option<::std::string::String>,
     pub(crate) published_time: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) licenses: ::std::option::Option<::std::vec::Vec<crate::types::LicenseInfo>>,
+    pub(crate) licenses: ::std::option::Option<::std::vec::Vec<super::super::types::LicenseInfo>>,
     pub(crate) revision: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::PackageVersionStatus>,
-    pub(crate) origin: ::std::option::Option<crate::types::PackageVersionOrigin>,
+    pub(crate) status: ::std::option::Option<super::super::types::PackageVersionStatus>,
+    pub(crate) origin: ::std::option::Option<super::super::types::PackageVersionOrigin>,
 }
 impl PackageVersionDescriptionBuilder {
     /// <p>The format of the package version.</p>
-    pub fn format(mut self, input: crate::types::PackageFormat) -> Self {
+    pub fn format(mut self, input: super::super::types::PackageFormat) -> Self {
         self.format = ::std::option::Option::Some(input);
         self
     }
     /// <p>The format of the package version.</p>
-    pub fn set_format(mut self, input: ::std::option::Option<crate::types::PackageFormat>) -> Self {
+    pub fn set_format(mut self, input: ::std::option::Option<super::super::types::PackageFormat>) -> Self {
         self.format = input;
         self
     }
     /// <p>The format of the package version.</p>
-    pub fn get_format(&self) -> &::std::option::Option<crate::types::PackageFormat> {
+    pub fn get_format(&self) -> &::std::option::Option<super::super::types::PackageFormat> {
         &self.format
     }
     /// <p>The namespace of the package version. The package component that specifies its namespace depends on its type. For example:</p>
@@ -294,7 +294,7 @@
     /// To override the contents of this collection use [`set_licenses`](Self::set_licenses).
     ///
     /// <p>Information about licenses associated with the package version.</p>
-    pub fn licenses(mut self, input: crate::types::LicenseInfo) -> Self {
+    pub fn licenses(mut self, input: super::super::types::LicenseInfo) -> Self {
         let mut v = self.licenses.unwrap_or_default();
         v.push(input);
         self.licenses = ::std::option::Option::Some(v);
@@ -301,12 +301,12 @@
         self
     }
     /// <p>Information about licenses associated with the package version.</p>
-    pub fn set_licenses(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::LicenseInfo>>) -> Self {
+    pub fn set_licenses(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::LicenseInfo>>) -> Self {
         self.licenses = input;
         self
     }
     /// <p>Information about licenses associated with the package version.</p>
-    pub fn get_licenses(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::LicenseInfo>> {
+    pub fn get_licenses(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::LicenseInfo>> {
         &self.licenses
     }
     /// <p>The revision of the package version.</p>
@@ -324,36 +324,36 @@
         &self.revision
     }
     /// <p>A string that contains the status of the package version.</p>
-    pub fn status(mut self, input: crate::types::PackageVersionStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::PackageVersionStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>A string that contains the status of the package version.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::PackageVersionStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::PackageVersionStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>A string that contains the status of the package version.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::PackageVersionStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::PackageVersionStatus> {
         &self.status
     }
     /// <p>A <a href="https://docs.aws.amazon.com/codeartifact/latest/APIReference/API_PackageVersionOrigin.html">PackageVersionOrigin</a> object that contains information about how the package version was added to the repository.</p>
-    pub fn origin(mut self, input: crate::types::PackageVersionOrigin) -> Self {
+    pub fn origin(mut self, input: super::super::types::PackageVersionOrigin) -> Self {
         self.origin = ::std::option::Option::Some(input);
         self
     }
     /// <p>A <a href="https://docs.aws.amazon.com/codeartifact/latest/APIReference/API_PackageVersionOrigin.html">PackageVersionOrigin</a> object that contains information about how the package version was added to the repository.</p>
-    pub fn set_origin(mut self, input: ::std::option::Option<crate::types::PackageVersionOrigin>) -> Self {
+    pub fn set_origin(mut self, input: ::std::option::Option<super::super::types::PackageVersionOrigin>) -> Self {
         self.origin = input;
         self
     }
     /// <p>A <a href="https://docs.aws.amazon.com/codeartifact/latest/APIReference/API_PackageVersionOrigin.html">PackageVersionOrigin</a> object that contains information about how the package version was added to the repository.</p>
-    pub fn get_origin(&self) -> &::std::option::Option<crate::types::PackageVersionOrigin> {
+    pub fn get_origin(&self) -> &::std::option::Option<super::super::types::PackageVersionOrigin> {
         &self.origin
     }
     /// Consumes the builder and constructs a [`PackageVersionDescription`](crate::types::PackageVersionDescription).
-    pub fn build(self) -> crate::types::PackageVersionDescription {
-        crate::types::PackageVersionDescription {
+    pub fn build(self) -> super::super::types::PackageVersionDescription {
+        super::super::types::PackageVersionDescription {
             format: self.format,
             namespace: self.namespace,
             package_name: self.package_name,
