--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the allowed repository.</p>
     pub repository_name: ::std::option::Option<::std::string::String>,
     /// <p>The origin configuration restriction type of the allowed repository.</p>
-    pub origin_restriction_type: ::std::option::Option<crate::types::PackageGroupOriginRestrictionType>,
+    pub origin_restriction_type: ::std::option::Option<super::super::types::PackageGroupOriginRestrictionType>,
 }
 impl PackageGroupAllowedRepository {
     /// <p>The name of the allowed repository.</p>
@@ -15,14 +15,14 @@
         self.repository_name.as_deref()
     }
     /// <p>The origin configuration restriction type of the allowed repository.</p>
-    pub fn origin_restriction_type(&self) -> ::std::option::Option<&crate::types::PackageGroupOriginRestrictionType> {
+    pub fn origin_restriction_type(&self) -> ::std::option::Option<&super::super::types::PackageGroupOriginRestrictionType> {
         self.origin_restriction_type.as_ref()
     }
 }
 impl PackageGroupAllowedRepository {
     /// Creates a new builder-style object to manufacture [`PackageGroupAllowedRepository`](crate::types::PackageGroupAllowedRepository).
-    pub fn builder() -> crate::types::builders::PackageGroupAllowedRepositoryBuilder {
-        crate::types::builders::PackageGroupAllowedRepositoryBuilder::default()
+    pub fn builder() -> super::super::types::builders::PackageGroupAllowedRepositoryBuilder {
+        super::super::types::builders::PackageGroupAllowedRepositoryBuilder::default()
     }
 }

@@ -31,7 +31,7 @@
 #[non_exhaustive]
 pub struct PackageGroupAllowedRepositoryBuilder {
     pub(crate) repository_name: ::std::option::Option<::std::string::String>,
-    pub(crate) origin_restriction_type: ::std::option::Option<crate::types::PackageGroupOriginRestrictionType>,
+    pub(crate) origin_restriction_type: ::std::option::Option<super::super::types::PackageGroupOriginRestrictionType>,
 }
 impl PackageGroupAllowedRepositoryBuilder {
     /// <p>The name of the allowed repository.</p>
@@ -49,22 +49,22 @@
         &self.repository_name
     }
     /// <p>The origin configuration restriction type of the allowed repository.</p>
-    pub fn origin_restriction_type(mut self, input: crate::types::PackageGroupOriginRestrictionType) -> Self {
+    pub fn origin_restriction_type(mut self, input: super::super::types::PackageGroupOriginRestrictionType) -> Self {
         self.origin_restriction_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The origin configuration restriction type of the allowed repository.</p>
-    pub fn set_origin_restriction_type(mut self, input: ::std::option::Option<crate::types::PackageGroupOriginRestrictionType>) -> Self {
+    pub fn set_origin_restriction_type(mut self, input: ::std::option::Option<super::super::types::PackageGroupOriginRestrictionType>) -> Self {
         self.origin_restriction_type = input;
         self
     }
     /// <p>The origin configuration restriction type of the allowed repository.</p>
-    pub fn get_origin_restriction_type(&self) -> &::std::option::Option<crate::types::PackageGroupOriginRestrictionType> {
+    pub fn get_origin_restriction_type(&self) -> &::std::option::Option<super::super::types::PackageGroupOriginRestrictionType> {
         &self.origin_restriction_type
     }
     /// Consumes the builder and constructs a [`PackageGroupAllowedRepository`](crate::types::PackageGroupAllowedRepository).
-    pub fn build(self) -> crate::types::PackageGroupAllowedRepository {
-        crate::types::PackageGroupAllowedRepository {
+    pub fn build(self) -> super::super::types::PackageGroupAllowedRepository {
+        super::super::types::PackageGroupAllowedRepository {
             repository_name: self.repository_name,
             origin_restriction_type: self.origin_restriction_type,
         }
