--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl UpstreamRepository {
     /// Creates a new builder-style object to manufacture [`UpstreamRepository`](crate::types::UpstreamRepository).
-    pub fn builder() -> crate::types::builders::UpstreamRepositoryBuilder {
-        crate::types::builders::UpstreamRepositoryBuilder::default()
+    pub fn builder() -> super::types::builders::UpstreamRepositoryBuilder {
+        super::types::builders::UpstreamRepositoryBuilder::default()
     }
 }

@@ -46,8 +46,8 @@
     /// Consumes the builder and constructs a [`UpstreamRepository`](crate::types::UpstreamRepository).
     /// This method will fail if any of the following fields are not set:
     /// - [`repository_name`](crate::types::builders::UpstreamRepositoryBuilder::repository_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::UpstreamRepository, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::UpstreamRepository {
+    pub fn build(self) -> ::std::result::Result<super::types::UpstreamRepository, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::UpstreamRepository {
             repository_name: self.repository_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "repository_name",
