--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl UpstreamRepositoryInfo {
     /// Creates a new builder-style object to manufacture [`UpstreamRepositoryInfo`](crate::types::UpstreamRepositoryInfo).
-    pub fn builder() -> crate::types::builders::UpstreamRepositoryInfoBuilder {
-        crate::types::builders::UpstreamRepositoryInfoBuilder::default()
+    pub fn builder() -> super::super::types::builders::UpstreamRepositoryInfoBuilder {
+        super::super::types::builders::UpstreamRepositoryInfoBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.repository_name
     }
     /// Consumes the builder and constructs a [`UpstreamRepositoryInfo`](crate::types::UpstreamRepositoryInfo).
-    pub fn build(self) -> crate::types::UpstreamRepositoryInfo {
-        crate::types::UpstreamRepositoryInfo {
+    pub fn build(self) -> super::super::types::UpstreamRepositoryInfo {
+        super::super::types::UpstreamRepositoryInfo {
             repository_name: self.repository_name,
         }
     }
