--- original
+++ modified
@@ -30,15 +30,15 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the deployment package from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub s3_object_storage_mode: ::std::option::Option<crate::types::S3ObjectStorageMode>,
+    pub s3_object_storage_mode: ::std::option::Option<super::types::S3ObjectStorageMode>,
     /// <p>URI of a container image in the Amazon ECR registry. Do not use for a function defined with a .zip file archive.</p>
     pub image_uri: ::std::option::Option<::std::string::String>,
     /// <p>The instruction set architecture that the function supports. Enter a string array with one of the valid values (arm64 or x86_64). The default value is <code>x86_64</code>.</p>
-    pub architectures: ::std::option::Option<::std::vec::Vec<crate::types::Architecture>>,
+    pub architectures: ::std::option::Option<::std::vec::Vec<super::types::Architecture>>,
     /// <p>Set to true to publish a new version of the function after updating the code. This has the same effect as calling <code>PublishVersion</code> separately.</p>
     pub publish: ::std::option::Option<bool>,
     /// <p>Specifies where to publish the function version or configuration.</p>
-    pub publish_to: ::std::option::Option<crate::types::FunctionVersionLatestPublished>,
+    pub publish_to: ::std::option::Option<super::types::FunctionVersionLatestPublished>,
     /// <p>Set to true to validate the request parameters and access permissions without modifying the function code.</p>
     pub dry_run: ::std::option::Option<bool>,
     /// <p>Update the function only if the revision ID matches the ID that's specified. Use this option to avoid modifying a function that has changed since you last read it.</p>
@@ -84,7 +84,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the deployment package from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn s3_object_storage_mode(&self) -> ::std::option::Option<&crate::types::S3ObjectStorageMode> {
+    pub fn s3_object_storage_mode(&self) -> ::std::option::Option<&super::types::S3ObjectStorageMode> {
         self.s3_object_storage_mode.as_ref()
     }
     /// <p>URI of a container image in the Amazon ECR registry. Do not use for a function defined with a .zip file archive.</p>
@@ -94,7 +94,7 @@
     /// <p>The instruction set architecture that the function supports. Enter a string array with one of the valid values (arm64 or x86_64). The default value is <code>x86_64</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.architectures.is_none()`.
-    pub fn architectures(&self) -> &[crate::types::Architecture] {
+    pub fn architectures(&self) -> &[super::types::Architecture] {
         self.architectures.as_deref().unwrap_or_default()
     }
     /// <p>Set to true to publish a new version of the function after updating the code. This has the same effect as calling <code>PublishVersion</code> separately.</p>
@@ -102,7 +102,7 @@
         self.publish
     }
     /// <p>Specifies where to publish the function version or configuration.</p>
-    pub fn publish_to(&self) -> ::std::option::Option<&crate::types::FunctionVersionLatestPublished> {
+    pub fn publish_to(&self) -> ::std::option::Option<&super::types::FunctionVersionLatestPublished> {
         self.publish_to.as_ref()
     }
     /// <p>Set to true to validate the request parameters and access permissions without modifying the function code.</p>
@@ -139,8 +139,8 @@
 }
 impl UpdateFunctionCodeInput {
     /// Creates a new builder-style object to manufacture [`UpdateFunctionCodeInput`](crate::operation::update_function_code::UpdateFunctionCodeInput).
-    pub fn builder() -> crate::operation::update_function_code::builders::UpdateFunctionCodeInputBuilder {
-        crate::operation::update_function_code::builders::UpdateFunctionCodeInputBuilder::default()
+    pub fn builder() -> super::operation::update_function_code::builders::UpdateFunctionCodeInputBuilder {
+        super::operation::update_function_code::builders::UpdateFunctionCodeInputBuilder::default()
     }
 }

@@ -153,11 +153,11 @@
     pub(crate) s3_bucket: ::std::option::Option<::std::string::String>,
     pub(crate) s3_key: ::std::option::Option<::std::string::String>,
     pub(crate) s3_object_version: ::std::option::Option<::std::string::String>,
-    pub(crate) s3_object_storage_mode: ::std::option::Option<crate::types::S3ObjectStorageMode>,
+    pub(crate) s3_object_storage_mode: ::std::option::Option<super::types::S3ObjectStorageMode>,
     pub(crate) image_uri: ::std::option::Option<::std::string::String>,
-    pub(crate) architectures: ::std::option::Option<::std::vec::Vec<crate::types::Architecture>>,
+    pub(crate) architectures: ::std::option::Option<::std::vec::Vec<super::types::Architecture>>,
     pub(crate) publish: ::std::option::Option<bool>,
-    pub(crate) publish_to: ::std::option::Option<crate::types::FunctionVersionLatestPublished>,
+    pub(crate) publish_to: ::std::option::Option<super::types::FunctionVersionLatestPublished>,
     pub(crate) dry_run: ::std::option::Option<bool>,
     pub(crate) revision_id: ::std::option::Option<::std::string::String>,
     pub(crate) source_kms_key_arn: ::std::option::Option<::std::string::String>,
@@ -271,7 +271,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the deployment package from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn s3_object_storage_mode(mut self, input: crate::types::S3ObjectStorageMode) -> Self {
+    pub fn s3_object_storage_mode(mut self, input: super::types::S3ObjectStorageMode) -> Self {
         self.s3_object_storage_mode = ::std::option::Option::Some(input);
         self
     }
@@ -282,7 +282,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the deployment package from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn set_s3_object_storage_mode(mut self, input: ::std::option::Option<crate::types::S3ObjectStorageMode>) -> Self {
+    pub fn set_s3_object_storage_mode(mut self, input: ::std::option::Option<super::types::S3ObjectStorageMode>) -> Self {
         self.s3_object_storage_mode = input;
         self
     }
@@ -293,7 +293,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the deployment package from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn get_s3_object_storage_mode(&self) -> &::std::option::Option<crate::types::S3ObjectStorageMode> {
+    pub fn get_s3_object_storage_mode(&self) -> &::std::option::Option<super::types::S3ObjectStorageMode> {
         &self.s3_object_storage_mode
     }
     /// <p>URI of a container image in the Amazon ECR registry. Do not use for a function defined with a .zip file archive.</p>
@@ -315,7 +315,7 @@
     /// To override the contents of this collection use [`set_architectures`](Self::set_architectures).
     ///
     /// <p>The instruction set architecture that the function supports. Enter a string array with one of the valid values (arm64 or x86_64). The default value is <code>x86_64</code>.</p>
-    pub fn architectures(mut self, input: crate::types::Architecture) -> Self {
+    pub fn architectures(mut self, input: super::types::Architecture) -> Self {
         let mut v = self.architectures.unwrap_or_default();
         v.push(input);
         self.architectures = ::std::option::Option::Some(v);
@@ -322,12 +322,12 @@
         self
     }
     /// <p>The instruction set architecture that the function supports. Enter a string array with one of the valid values (arm64 or x86_64). The default value is <code>x86_64</code>.</p>
-    pub fn set_architectures(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Architecture>>) -> Self {
+    pub fn set_architectures(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::Architecture>>) -> Self {
         self.architectures = input;
         self
     }
     /// <p>The instruction set architecture that the function supports. Enter a string array with one of the valid values (arm64 or x86_64). The default value is <code>x86_64</code>.</p>
-    pub fn get_architectures(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Architecture>> {
+    pub fn get_architectures(&self) -> &::std::option::Option<::std::vec::Vec<super::types::Architecture>> {
         &self.architectures
     }
     /// <p>Set to true to publish a new version of the function after updating the code. This has the same effect as calling <code>PublishVersion</code> separately.</p>
@@ -345,17 +345,17 @@
         &self.publish
     }
     /// <p>Specifies where to publish the function version or configuration.</p>
-    pub fn publish_to(mut self, input: crate::types::FunctionVersionLatestPublished) -> Self {
+    pub fn publish_to(mut self, input: super::types::FunctionVersionLatestPublished) -> Self {
         self.publish_to = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies where to publish the function version or configuration.</p>
-    pub fn set_publish_to(mut self, input: ::std::option::Option<crate::types::FunctionVersionLatestPublished>) -> Self {
+    pub fn set_publish_to(mut self, input: ::std::option::Option<super::types::FunctionVersionLatestPublished>) -> Self {
         self.publish_to = input;
         self
     }
     /// <p>Specifies where to publish the function version or configuration.</p>
-    pub fn get_publish_to(&self) -> &::std::option::Option<crate::types::FunctionVersionLatestPublished> {
+    pub fn get_publish_to(&self) -> &::std::option::Option<super::types::FunctionVersionLatestPublished> {
         &self.publish_to
     }
     /// <p>Set to true to validate the request parameters and access permissions without modifying the function code.</p>
@@ -403,9 +403,9 @@
     /// Consumes the builder and constructs a [`UpdateFunctionCodeInput`](crate::operation::update_function_code::UpdateFunctionCodeInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_function_code::UpdateFunctionCodeInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::update_function_code::UpdateFunctionCodeInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_function_code::UpdateFunctionCodeInput {
+        ::std::result::Result::Ok(super::operation::update_function_code::UpdateFunctionCodeInput {
             function_name: self.function_name,
             zip_file: self.zip_file,
             s3_bucket: self.s3_bucket,
