--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::batch_get_metric_data::BatchGetMetricDataInput,
+        input: super::super::operation::batch_get_metric_data::BatchGetMetricDataInput,
     ) -> ::std::result::Result<
-        crate::operation::batch_get_metric_data::BatchGetMetricDataOutput,
+        super::super::operation::batch_get_metric_data::BatchGetMetricDataOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::batch_get_metric_data::BatchGetMetricDataError,
+            super::super::operation::batch_get_metric_data::BatchGetMetricDataError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::batch_get_metric_data::BatchGetMetricDataError>()
+                err.downcast::<super::super::operation::batch_get_metric_data::BatchGetMetricDataError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::batch_get_metric_data::BatchGetMetricDataOutput>()
+                .downcast::<super::super::operation::batch_get_metric_data::BatchGetMetricDataOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::batch_get_metric_data::BatchGetMetricDataInput,
+        input: super::super::operation::batch_get_metric_data::BatchGetMetricDataInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("BatchGetMetricData")
                 .build()
                 .expect("required fields set"),
@@ -133,13 +133,13 @@
                 BatchGetMetricDataEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::batch_get_metric_data::BatchGetMetricDataError,
+                super::super::operation::batch_get_metric_data::BatchGetMetricDataError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::batch_get_metric_data::BatchGetMetricDataError,
+                super::super::operation::batch_get_metric_data::BatchGetMetricDataError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::batch_get_metric_data::BatchGetMetricDataError,
+                super::super::operation::batch_get_metric_data::BatchGetMetricDataError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -161,11 +161,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_batch_get_metric_data::de_batch_get_metric_data_http_error(status, headers, body)
+            super::super::protocol_serde::shape_batch_get_metric_data::de_batch_get_metric_data_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_batch_get_metric_data::de_batch_get_metric_data_http_response(status, headers, body)
+            super::super::protocol_serde::shape_batch_get_metric_data::de_batch_get_metric_data_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -178,16 +178,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::batch_get_metric_data::BatchGetMetricDataInput>()
+            .downcast::<super::super::operation::batch_get_metric_data::BatchGetMetricDataInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::batch_get_metric_data::BatchGetMetricDataInput,
+                _input: &super::super::operation::batch_get_metric_data::BatchGetMetricDataInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -196,7 +196,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::batch_get_metric_data::BatchGetMetricDataInput,
+                input: &super::super::operation::batch_get_metric_data::BatchGetMetricDataInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -207,7 +207,7 @@
             builder = _header_serialization_settings.set_default_header(builder, ::http_1x::header::CONTENT_TYPE, "application/json");
             builder
         };
-        let body = ::aws_smithy_types::body::SdkBody::from(crate::protocol_serde::shape_batch_get_metric_data::ser_batch_get_metric_data_input(
+        let body = ::aws_smithy_types::body::SdkBody::from(super::super::protocol_serde::shape_batch_get_metric_data::ser_batch_get_metric_data_input(
             &input,
         )?);
         if let Some(content_length) = body.content_length() {
@@ -241,7 +241,7 @@
             .downcast_ref::<BatchGetMetricDataInput>()
             .ok_or("failed to downcast to BatchGetMetricDataInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -264,13 +264,13 @@
 #[derive(::std::fmt::Debug)]
 pub enum BatchGetMetricDataError {
     /// <p>The input you provided is invalid.</p>
-    BadRequestException(crate::types::error::BadRequestException),
+    BadRequestException(super::super::types::error::BadRequestException),
     /// <p>The request couldn't be processed because an error occurred with the Amazon SES API v2.</p>
-    InternalServiceErrorException(crate::types::error::InternalServiceErrorException),
+    InternalServiceErrorException(super::super::types::error::InternalServiceErrorException),
     /// <p>The resource you attempted to access doesn't exist.</p>
-    NotFoundException(crate::types::error::NotFoundException),
+    NotFoundException(super::super::types::error::NotFoundException),
     /// <p>Too many requests have been made to the operation.</p>
-    TooManyRequestsException(crate::types::error::TooManyRequestsException),
+    TooManyRequestsException(super::super::types::error::TooManyRequestsException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -278,7 +278,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-BatchGetMetricDataError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl BatchGetMetricDataError {
     /// Creates the `BatchGetMetricDataError::Unhandled` variant from any error type.
@@ -285,7 +285,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -293,7 +293,7 @@

     /// Creates the `BatchGetMetricDataError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -380,21 +380,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::batch_get_metric_data::BatchGetMetricDataError {
+impl ::aws_types::request_id::RequestId for super::super::operation::batch_get_metric_data::BatchGetMetricDataError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::batch_get_metric_data::_batch_get_metric_data_input::BatchGetMetricDataInput;
+pub use super::super::operation::batch_get_metric_data::_batch_get_metric_data_input::BatchGetMetricDataInput;

-pub use crate::operation::batch_get_metric_data::_batch_get_metric_data_output::BatchGetMetricDataOutput;
+pub use super::super::operation::batch_get_metric_data::_batch_get_metric_data_output::BatchGetMetricDataOutput;

 mod _batch_get_metric_data_input;

