--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
+        input: super::super::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
     ) -> ::std::result::Result<
-        crate::operation::send_custom_verification_email::SendCustomVerificationEmailOutput,
+        super::super::operation::send_custom_verification_email::SendCustomVerificationEmailOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::send_custom_verification_email::SendCustomVerificationEmailError,
+            super::super::operation::send_custom_verification_email::SendCustomVerificationEmailError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::send_custom_verification_email::SendCustomVerificationEmailError>()
+                err.downcast::<super::super::operation::send_custom_verification_email::SendCustomVerificationEmailError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::send_custom_verification_email::SendCustomVerificationEmailOutput>()
+                .downcast::<super::super::operation::send_custom_verification_email::SendCustomVerificationEmailOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
+        input: super::super::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("SendCustomVerificationEmail")
                 .build()
                 .expect("required fields set"),
@@ -136,13 +136,13 @@
                 SendCustomVerificationEmailEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::send_custom_verification_email::SendCustomVerificationEmailError,
+                super::super::operation::send_custom_verification_email::SendCustomVerificationEmailError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::send_custom_verification_email::SendCustomVerificationEmailError,
+                super::super::operation::send_custom_verification_email::SendCustomVerificationEmailError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::send_custom_verification_email::SendCustomVerificationEmailError,
+                super::super::operation::send_custom_verification_email::SendCustomVerificationEmailError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -217,11 +217,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_send_custom_verification_email::de_send_custom_verification_email_http_error(status, headers, body)
+            super::super::protocol_serde::shape_send_custom_verification_email::de_send_custom_verification_email_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_send_custom_verification_email::de_send_custom_verification_email_http_response(status, headers, body)
+            super::super::protocol_serde::shape_send_custom_verification_email::de_send_custom_verification_email_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -234,16 +234,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::send_custom_verification_email::SendCustomVerificationEmailInput>()
+            .downcast::<super::super::operation::send_custom_verification_email::SendCustomVerificationEmailInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
+                _input: &super::super::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -252,7 +252,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
+                input: &super::super::operation::send_custom_verification_email::SendCustomVerificationEmailInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -264,7 +264,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_send_custom_verification_email::ser_send_custom_verification_email_input(&input)?,
+            super::super::protocol_serde::shape_send_custom_verification_email::ser_send_custom_verification_email_input(&input)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -297,7 +297,7 @@
             .downcast_ref::<SendCustomVerificationEmailInput>()
             .ok_or("failed to downcast to SendCustomVerificationEmailInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -320,19 +320,19 @@
 #[derive(::std::fmt::Debug)]
 pub enum SendCustomVerificationEmailError {
     /// <p>The input you provided is invalid.</p>
-    BadRequestException(crate::types::error::BadRequestException),
+    BadRequestException(super::super::types::error::BadRequestException),
     /// <p>There are too many instances of the specified resource type.</p>
-    LimitExceededException(crate::types::error::LimitExceededException),
+    LimitExceededException(super::super::types::error::LimitExceededException),
     /// <p>The message can't be sent because the sending domain isn't verified.</p>
-    MailFromDomainNotVerifiedException(crate::types::error::MailFromDomainNotVerifiedException),
+    MailFromDomainNotVerifiedException(super::super::types::error::MailFromDomainNotVerifiedException),
     /// <p>The message can't be sent because it contains invalid content.</p>
-    MessageRejected(crate::types::error::MessageRejected),
+    MessageRejected(super::super::types::error::MessageRejected),
     /// <p>The resource you attempted to access doesn't exist.</p>
-    NotFoundException(crate::types::error::NotFoundException),
+    NotFoundException(super::super::types::error::NotFoundException),
     /// <p>The message can't be sent because the account's ability to send email is currently paused.</p>
-    SendingPausedException(crate::types::error::SendingPausedException),
+    SendingPausedException(super::super::types::error::SendingPausedException),
     /// <p>Too many requests have been made to the operation.</p>
-    TooManyRequestsException(crate::types::error::TooManyRequestsException),
+    TooManyRequestsException(super::super::types::error::TooManyRequestsException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -340,7 +340,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-SendCustomVerificationEmailError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl SendCustomVerificationEmailError {
     /// Creates the `SendCustomVerificationEmailError::Unhandled` variant from any error type.
@@ -347,7 +347,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -355,7 +355,7 @@

     /// Creates the `SendCustomVerificationEmailError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -466,21 +466,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::send_custom_verification_email::SendCustomVerificationEmailError {
+impl ::aws_types::request_id::RequestId for super::super::operation::send_custom_verification_email::SendCustomVerificationEmailError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::send_custom_verification_email::_send_custom_verification_email_input::SendCustomVerificationEmailInput;
+pub use super::super::operation::send_custom_verification_email::_send_custom_verification_email_input::SendCustomVerificationEmailInput;

-pub use crate::operation::send_custom_verification_email::_send_custom_verification_email_output::SendCustomVerificationEmailOutput;
+pub use super::super::operation::send_custom_verification_email::_send_custom_verification_email_output::SendCustomVerificationEmailOutput;

 mod _send_custom_verification_email_input;

