--- original
+++ modified
@@ -25,7 +25,7 @@
     /// </ul>
     pub model_id: ::std::option::Option<::std::string::String>,
     /// <p>Specifies whether to enable or disable the Bedrock trace. If enabled, you can see the full Bedrock trace.</p>
-    pub trace: ::std::option::Option<crate::types::Trace>,
+    pub trace: ::std::option::Option<super::super::super::types::Trace>,
     /// <p>The unique identifier of the guardrail that you want to use. If you don't provide a value, no guardrail is applied to the invocation.</p>
     /// <p>An error will be thrown in the following situations.</p>
     /// <ul>
@@ -40,9 +40,9 @@
     /// <p>The version number for the guardrail. The value can also be <code>DRAFT</code>.</p>
     pub guardrail_version: ::std::option::Option<::std::string::String>,
     /// <p>Model performance settings for the request.</p>
-    pub performance_config_latency: ::std::option::Option<crate::types::PerformanceConfigLatency>,
+    pub performance_config_latency: ::std::option::Option<super::super::super::types::PerformanceConfigLatency>,
     /// <p>Specifies the processing tier type used for serving the request.</p>
-    pub service_tier: ::std::option::Option<crate::types::ServiceTierType>,
+    pub service_tier: ::std::option::Option<super::super::super::types::ServiceTierType>,
     /// <p>Key-value pairs that you can use to filter invocation logs.</p>
     pub request_metadata: ::std::option::Option<::std::string::String>,
 }
@@ -77,7 +77,7 @@
         self.model_id.as_deref()
     }
     /// <p>Specifies whether to enable or disable the Bedrock trace. If enabled, you can see the full Bedrock trace.</p>
-    pub fn trace(&self) -> ::std::option::Option<&crate::types::Trace> {
+    pub fn trace(&self) -> ::std::option::Option<&super::super::super::types::Trace> {
         self.trace.as_ref()
     }
     /// <p>The unique identifier of the guardrail that you want to use. If you don't provide a value, no guardrail is applied to the invocation.</p>
@@ -98,11 +98,11 @@
         self.guardrail_version.as_deref()
     }
     /// <p>Model performance settings for the request.</p>
-    pub fn performance_config_latency(&self) -> ::std::option::Option<&crate::types::PerformanceConfigLatency> {
+    pub fn performance_config_latency(&self) -> ::std::option::Option<&super::super::super::types::PerformanceConfigLatency> {
         self.performance_config_latency.as_ref()
     }
     /// <p>Specifies the processing tier type used for serving the request.</p>
-    pub fn service_tier(&self) -> ::std::option::Option<&crate::types::ServiceTierType> {
+    pub fn service_tier(&self) -> ::std::option::Option<&super::super::super::types::ServiceTierType> {
         self.service_tier.as_ref()
     }
     /// <p>Key-value pairs that you can use to filter invocation logs.</p>
@@ -128,8 +128,8 @@
 }
 impl InvokeModelInput {
     /// Creates a new builder-style object to manufacture [`InvokeModelInput`](crate::operation::invoke_model::InvokeModelInput).
-    pub fn builder() -> crate::operation::invoke_model::builders::InvokeModelInputBuilder {
-        crate::operation::invoke_model::builders::InvokeModelInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::invoke_model::builders::InvokeModelInputBuilder {
+        super::super::super::operation::invoke_model::builders::InvokeModelInputBuilder::default()
     }
 }

@@ -141,11 +141,11 @@
     pub(crate) content_type: ::std::option::Option<::std::string::String>,
     pub(crate) accept: ::std::option::Option<::std::string::String>,
     pub(crate) model_id: ::std::option::Option<::std::string::String>,
-    pub(crate) trace: ::std::option::Option<crate::types::Trace>,
+    pub(crate) trace: ::std::option::Option<super::super::super::types::Trace>,
     pub(crate) guardrail_identifier: ::std::option::Option<::std::string::String>,
     pub(crate) guardrail_version: ::std::option::Option<::std::string::String>,
-    pub(crate) performance_config_latency: ::std::option::Option<crate::types::PerformanceConfigLatency>,
-    pub(crate) service_tier: ::std::option::Option<crate::types::ServiceTierType>,
+    pub(crate) performance_config_latency: ::std::option::Option<super::super::super::types::PerformanceConfigLatency>,
+    pub(crate) service_tier: ::std::option::Option<super::super::super::types::ServiceTierType>,
     pub(crate) request_metadata: ::std::option::Option<::std::string::String>,
 }
 impl InvokeModelInputBuilder {
@@ -246,17 +246,17 @@
         &self.model_id
     }
     /// <p>Specifies whether to enable or disable the Bedrock trace. If enabled, you can see the full Bedrock trace.</p>
-    pub fn trace(mut self, input: crate::types::Trace) -> Self {
+    pub fn trace(mut self, input: super::super::super::types::Trace) -> Self {
         self.trace = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether to enable or disable the Bedrock trace. If enabled, you can see the full Bedrock trace.</p>
-    pub fn set_trace(mut self, input: ::std::option::Option<crate::types::Trace>) -> Self {
+    pub fn set_trace(mut self, input: ::std::option::Option<super::super::super::types::Trace>) -> Self {
         self.trace = input;
         self
     }
     /// <p>Specifies whether to enable or disable the Bedrock trace. If enabled, you can see the full Bedrock trace.</p>
-    pub fn get_trace(&self) -> &::std::option::Option<crate::types::Trace> {
+    pub fn get_trace(&self) -> &::std::option::Option<super::super::super::types::Trace> {
         &self.trace
     }
     /// <p>The unique identifier of the guardrail that you want to use. If you don't provide a value, no guardrail is applied to the invocation.</p>
@@ -315,31 +315,31 @@
         &self.guardrail_version
     }
     /// <p>Model performance settings for the request.</p>
-    pub fn performance_config_latency(mut self, input: crate::types::PerformanceConfigLatency) -> Self {
+    pub fn performance_config_latency(mut self, input: super::super::super::types::PerformanceConfigLatency) -> Self {
         self.performance_config_latency = ::std::option::Option::Some(input);
         self
     }
     /// <p>Model performance settings for the request.</p>
-    pub fn set_performance_config_latency(mut self, input: ::std::option::Option<crate::types::PerformanceConfigLatency>) -> Self {
+    pub fn set_performance_config_latency(mut self, input: ::std::option::Option<super::super::super::types::PerformanceConfigLatency>) -> Self {
         self.performance_config_latency = input;
         self
     }
     /// <p>Model performance settings for the request.</p>
-    pub fn get_performance_config_latency(&self) -> &::std::option::Option<crate::types::PerformanceConfigLatency> {
+    pub fn get_performance_config_latency(&self) -> &::std::option::Option<super::super::super::types::PerformanceConfigLatency> {
         &self.performance_config_latency
     }
     /// <p>Specifies the processing tier type used for serving the request.</p>
-    pub fn service_tier(mut self, input: crate::types::ServiceTierType) -> Self {
+    pub fn service_tier(mut self, input: super::super::super::types::ServiceTierType) -> Self {
         self.service_tier = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the processing tier type used for serving the request.</p>
-    pub fn set_service_tier(mut self, input: ::std::option::Option<crate::types::ServiceTierType>) -> Self {
+    pub fn set_service_tier(mut self, input: ::std::option::Option<super::super::super::types::ServiceTierType>) -> Self {
         self.service_tier = input;
         self
     }
     /// <p>Specifies the processing tier type used for serving the request.</p>
-    pub fn get_service_tier(&self) -> &::std::option::Option<crate::types::ServiceTierType> {
+    pub fn get_service_tier(&self) -> &::std::option::Option<super::super::super::types::ServiceTierType> {
         &self.service_tier
     }
     /// <p>Key-value pairs that you can use to filter invocation logs.</p>
@@ -357,8 +357,8 @@
         &self.request_metadata
     }
     /// Consumes the builder and constructs a [`InvokeModelInput`](crate::operation::invoke_model::InvokeModelInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::invoke_model::InvokeModelInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::invoke_model::InvokeModelInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::invoke_model::InvokeModelInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::invoke_model::InvokeModelInput {
             body: self.body,
             content_type: self.content_type,
             accept: self.accept,
