--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl GetRoleTemplateVersionInput {
     /// Creates a new builder-style object to manufacture [`GetRoleTemplateVersionInput`](crate::operation::get_role_template_version::GetRoleTemplateVersionInput).
-    pub fn builder() -> crate::operation::get_role_template_version::builders::GetRoleTemplateVersionInputBuilder {
-        crate::operation::get_role_template_version::builders::GetRoleTemplateVersionInputBuilder::default()
+    pub fn builder() -> super::operation::get_role_template_version::builders::GetRoleTemplateVersionInputBuilder {
+        super::operation::get_role_template_version::builders::GetRoleTemplateVersionInputBuilder::default()
     }
 }

@@ -71,10 +71,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_role_template_version::GetRoleTemplateVersionInput,
+        super::operation::get_role_template_version::GetRoleTemplateVersionInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_role_template_version::GetRoleTemplateVersionInput {
+        ::std::result::Result::Ok(super::operation::get_role_template_version::GetRoleTemplateVersionInput {
             template_arn: self.template_arn,
             minor_version: self.minor_version,
         })
