--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::vec::Vec<crate::types::RepositoryExternalConnectionInfo>>,
+    Option<::std::vec::Vec<super::types::RepositoryExternalConnectionInfo>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -26,7 +26,7 @@
                         break;
                     }
                     _ => {
-                        let value = crate::protocol_serde::shape_repository_external_connection_info::de_repository_external_connection_info(
+                        let value = super::protocol_serde::shape_repository_external_connection_info::de_repository_external_connection_info(
                             tokens,
                             _value,
                             depth + 1,
