--- original
+++ modified
@@ -6,8 +6,8 @@
 ) -> ::std::result::Result<
     Option<
         ::std::collections::HashMap<
-            crate::types::PackageGroupOriginRestrictionType,
-            ::std::collections::HashMap<crate::types::PackageGroupAllowedRepositoryUpdateType, ::std::vec::Vec<::std::string::String>>,
+            super::types::PackageGroupOriginRestrictionType,
+            ::std::collections::HashMap<super::types::PackageGroupAllowedRepositoryUpdateType, ::std::vec::Vec<::std::string::String>>,
         >,
     >,
     ::aws_smithy_json::deserialize::error::DeserializeError,
@@ -30,8 +30,8 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => {
                         let key = key
                             .to_unescaped()
-                            .map(|u| crate::types::PackageGroupOriginRestrictionType::from(u.as_ref()))?;
-                        let value = crate::protocol_serde::shape_package_group_allowed_repository_update::de_package_group_allowed_repository_update(
+                            .map(|u| super::types::PackageGroupOriginRestrictionType::from(u.as_ref()))?;
+                        let value = super::protocol_serde::shape_package_group_allowed_repository_update::de_package_group_allowed_repository_update(
                             tokens,
                             _value,
                             depth + 1,
