--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::PackageGroupOriginRestriction>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::PackageGroupOriginRestriction>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::PackageGroupOriginRestrictionBuilder::default();
+            let mut builder = super::super::types::builders::PackageGroupOriginRestrictionBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -27,7 +27,7 @@
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                         .map(|s| {
                                             s.to_unescaped()
-                                                .map(|u| crate::types::PackageGroupOriginRestrictionMode::from(u.as_ref()))
+                                                .map(|u| super::super::types::PackageGroupOriginRestrictionMode::from(u.as_ref()))
                                         })
                                         .transpose()?,
                                 );
@@ -37,7 +37,7 @@
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                         .map(|s| {
                                             s.to_unescaped()
-                                                .map(|u| crate::types::PackageGroupOriginRestrictionMode::from(u.as_ref()))
+                                                .map(|u| super::super::types::PackageGroupOriginRestrictionMode::from(u.as_ref()))
                                         })
                                         .transpose()?,
                                 );
@@ -44,7 +44,7 @@
                             }
                             "inheritedFrom" => {
                                 builder = builder.set_inherited_from(
-                                    crate::protocol_serde::shape_package_group_reference::de_package_group_reference(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_package_group_reference::de_package_group_reference(tokens, _value, depth + 1)?,
                                 );
                             }
                             "repositoriesCount" => {
