--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::collections::HashMap<crate::types::PackageGroupAllowedRepositoryUpdateType, ::std::vec::Vec<::std::string::String>>>,
+    Option<::std::collections::HashMap<super::types::PackageGroupAllowedRepositoryUpdateType, ::std::vec::Vec<::std::string::String>>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -25,8 +25,8 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => {
                         let key = key
                             .to_unescaped()
-                            .map(|u| crate::types::PackageGroupAllowedRepositoryUpdateType::from(u.as_ref()))?;
-                        let value = crate::protocol_serde::shape_repository_name_list::de_repository_name_list(tokens, _value, depth + 1)?;
+                            .map(|u| super::types::PackageGroupAllowedRepositoryUpdateType::from(u.as_ref()))?;
+                        let value = super::protocol_serde::shape_repository_name_list::de_repository_name_list(tokens, _value, depth + 1)?;
                         match value {
                             Some(value) => {
                                 map.insert(key, value);
