--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::PackageGroupDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::PackageGroupDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::PackageGroupDescriptionBuilder::default();
+            let mut builder = super::super::types::builders::PackageGroupDescriptionBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -71,7 +71,7 @@
                         }
                         "originConfiguration" => {
                             builder = builder.set_origin_configuration(
-                                crate::protocol_serde::shape_package_group_origin_configuration::de_package_group_origin_configuration(
+                                super::super::protocol_serde::shape_package_group_origin_configuration::de_package_group_origin_configuration(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -79,7 +79,7 @@
                             );
                         }
                         "parent" => {
-                            builder = builder.set_parent(crate::protocol_serde::shape_package_group_reference::de_package_group_reference(
+                            builder = builder.set_parent(super::super::protocol_serde::shape_package_group_reference::de_package_group_reference(
                                 tokens,
                                 _value,
                                 depth + 1,
