--- original
+++ modified
@@ -5,137 +5,137 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::disassociate_external_connection::DisassociateExternalConnectionOutput,
-    crate::operation::disassociate_external_connection::DisassociateExternalConnectionError,
+    super::operation::disassociate_external_connection::DisassociateExternalConnectionOutput,
+    super::operation::disassociate_external_connection::DisassociateExternalConnectionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled(generic)),
+        None => return Err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AccessDeniedException" => crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::AccessDeniedException({
+        "AccessDeniedException" => super::operation::disassociate_external_connection::DisassociateExternalConnectionError::AccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_access_denied_exception::de_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                let mut output = super::types::error::builders::AccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_access_denied_exception::de_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                 let output = output.meta(generic);
-                crate::serde_util::access_denied_exception_correct_errors(output)
+                super::serde_util::access_denied_exception_correct_errors(output)
                     .build()
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
             };
             tmp
         }),
-        "ConflictException" => crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::ConflictException({
+        "ConflictException" => super::operation::disassociate_external_connection::DisassociateExternalConnectionError::ConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                let mut output = super::types::error::builders::ConflictExceptionBuilder::default();
+                output = super::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                 let output = output.meta(generic);
-                crate::serde_util::conflict_exception_correct_errors(output)
+                super::serde_util::conflict_exception_correct_errors(output)
                     .build()
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
             };
             tmp
         }),
         "InternalServerException" => {
-            crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::InternalServerException({
+            super::operation::disassociate_external_connection::DisassociateExternalConnectionError::InternalServerException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InternalServerExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_internal_server_exception::de_internal_server_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                    let mut output = super::types::error::builders::InternalServerExceptionBuilder::default();
+                    output = super::protocol_serde::shape_internal_server_exception::de_internal_server_exception_json_err(_response_body, output)
+                        .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                     let output = output.meta(generic);
-                    crate::serde_util::internal_server_exception_correct_errors(output)
+                    super::serde_util::internal_server_exception_correct_errors(output)
                         .build()
-                        .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                        .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
                 };
                 tmp
             })
         }
         "ResourceNotFoundException" => {
-            crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::ResourceNotFoundException({
+            super::operation::disassociate_external_connection::DisassociateExternalConnectionError::ResourceNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                    let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                        super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                            .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                     let output = output.meta(generic);
-                    crate::serde_util::resource_not_found_exception_correct_errors(output)
+                    super::serde_util::resource_not_found_exception_correct_errors(output)
                         .build()
-                        .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                        .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
                 };
                 tmp
             })
         }
         "ServiceQuotaExceededException" => {
-            crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::ServiceQuotaExceededException({
+            super::operation::disassociate_external_connection::DisassociateExternalConnectionError::ServiceQuotaExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
+                    let mut output = super::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
+                    output = super::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                     let output = output.meta(generic);
-                    crate::serde_util::service_quota_exceeded_exception_correct_errors(output)
+                    super::serde_util::service_quota_exceeded_exception_correct_errors(output)
                         .build()
-                        .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                        .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
                 };
                 tmp
             })
         }
-        "ThrottlingException" => crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::ThrottlingException({
+        "ThrottlingException" => super::operation::disassociate_external_connection::DisassociateExternalConnectionError::ThrottlingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottlingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                let mut output = super::types::error::builders::ThrottlingExceptionBuilder::default();
+                output = super::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_throttling_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled(
+                    super::protocol_serde::shape_throttling_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
                 );
                 let output = output.meta(generic);
-                crate::serde_util::throttling_exception_correct_errors(output)
+                super::serde_util::throttling_exception_correct_errors(output)
                     .build()
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
             };
             tmp
         }),
-        "ValidationException" => crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::ValidationException({
+        "ValidationException" => super::operation::disassociate_external_connection::DisassociateExternalConnectionError::ValidationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+                let mut output = super::types::error::builders::ValidationExceptionBuilder::default();
+                output = super::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
                 let output = output.meta(generic);
-                crate::serde_util::validation_exception_correct_errors(output)
+                super::serde_util::validation_exception_correct_errors(output)
                     .build()
-                    .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
+                    .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?
             };
             tmp
         }),
-        _ => crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::generic(generic),
+        _ => super::operation::disassociate_external_connection::DisassociateExternalConnectionError::generic(generic),
     })
 }

@@ -145,14 +145,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::disassociate_external_connection::DisassociateExternalConnectionOutput,
-    crate::operation::disassociate_external_connection::DisassociateExternalConnectionError,
+    super::operation::disassociate_external_connection::DisassociateExternalConnectionOutput,
+    super::operation::disassociate_external_connection::DisassociateExternalConnectionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::disassociate_external_connection::builders::DisassociateExternalConnectionOutputBuilder::default();
-        output = crate::protocol_serde::shape_disassociate_external_connection::de_disassociate_external_connection(_response_body, output)
-            .map_err(crate::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
+        let mut output = super::operation::disassociate_external_connection::builders::DisassociateExternalConnectionOutputBuilder::default();
+        output = super::protocol_serde::shape_disassociate_external_connection::de_disassociate_external_connection(_response_body, output)
+            .map_err(super::operation::disassociate_external_connection::DisassociateExternalConnectionError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -160,12 +160,12 @@

 pub(crate) fn de_disassociate_external_connection(
     _value: &[u8],
-    mut builder: crate::operation::disassociate_external_connection::builders::DisassociateExternalConnectionOutputBuilder,
+    mut builder: super::operation::disassociate_external_connection::builders::DisassociateExternalConnectionOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::disassociate_external_connection::builders::DisassociateExternalConnectionOutputBuilder,
+    super::operation::disassociate_external_connection::builders::DisassociateExternalConnectionOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -175,7 +175,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "repository" => {
-                    builder = builder.set_repository(crate::protocol_serde::shape_repository_description::de_repository_description(
+                    builder = builder.set_repository(super::protocol_serde::shape_repository_description::de_repository_description(
                         tokens,
                         _value,
                         depth + 1,
