--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetCapacityProviderInput {
     /// Creates a new builder-style object to manufacture [`GetCapacityProviderInput`](crate::operation::get_capacity_provider::GetCapacityProviderInput).
-    pub fn builder() -> crate::operation::get_capacity_provider::builders::GetCapacityProviderInputBuilder {
-        crate::operation::get_capacity_provider::builders::GetCapacityProviderInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_capacity_provider::builders::GetCapacityProviderInputBuilder {
+        super::super::super::operation::get_capacity_provider::builders::GetCapacityProviderInputBuilder::default()
     }
 }

@@ -44,9 +44,9 @@
     /// Consumes the builder and constructs a [`GetCapacityProviderInput`](crate::operation::get_capacity_provider::GetCapacityProviderInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_capacity_provider::GetCapacityProviderInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_capacity_provider::GetCapacityProviderInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_capacity_provider::GetCapacityProviderInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_capacity_provider::GetCapacityProviderInput {
             capacity_provider_name: self.capacity_provider_name,
         })
     }
