--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
+        input: super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
     ) -> ::std::result::Result<
-        crate::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput,
+        super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
+            super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError>()
+                err.downcast::<super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput>()
+                .downcast::<super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
+        input: super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("AdminUpdateDeviceStatus")
                 .build()
                 .expect("required fields set"),
@@ -142,13 +142,13 @@
                 AdminUpdateDeviceStatusEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
+                super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
+                super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
+                super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -218,11 +218,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_admin_update_device_status::de_admin_update_device_status_http_error(status, headers, body)
+            super::super::protocol_serde::shape_admin_update_device_status::de_admin_update_device_status_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_admin_update_device_status::de_admin_update_device_status_http_response(status, headers, body)
+            super::super::protocol_serde::shape_admin_update_device_status::de_admin_update_device_status_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -235,16 +235,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput>()
+            .downcast::<super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
+                _input: &super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -253,7 +253,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
+                input: &super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -270,7 +270,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_admin_update_device_status::ser_admin_update_device_status_input(&input)?,
+            super::super::protocol_serde::shape_admin_update_device_status::ser_admin_update_device_status_input(&input)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -303,7 +303,7 @@
             .downcast_ref::<AdminUpdateDeviceStatusInput>()
             .ok_or("failed to downcast to AdminUpdateDeviceStatusInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -326,21 +326,21 @@
 #[derive(::std::fmt::Debug)]
 pub enum AdminUpdateDeviceStatusError {
     /// <p>This exception is thrown when Amazon Cognito encounters an internal error.</p>
-    InternalErrorException(crate::types::error::InternalErrorException),
+    InternalErrorException(super::super::types::error::InternalErrorException),
     /// <p>This exception is thrown when the Amazon Cognito service encounters an invalid parameter.</p>
-    InvalidParameterException(crate::types::error::InvalidParameterException),
+    InvalidParameterException(super::super::types::error::InvalidParameterException),
     /// <p>This exception is thrown when the user pool configuration is not valid.</p>
-    InvalidUserPoolConfigurationException(crate::types::error::InvalidUserPoolConfigurationException),
+    InvalidUserPoolConfigurationException(super::super::types::error::InvalidUserPoolConfigurationException),
     /// <p>This exception is thrown when a user isn't authorized.</p>
-    NotAuthorizedException(crate::types::error::NotAuthorizedException),
+    NotAuthorizedException(super::super::types::error::NotAuthorizedException),
     /// <p>This exception is thrown when an operation is not available in the current region or for the current user pool configuration. This can occur when attempting to perform operations that are not supported in secondary replica regions.</p>
-    OperationNotEnabledException(crate::types::error::OperationNotEnabledException),
+    OperationNotEnabledException(super::super::types::error::OperationNotEnabledException),
     /// <p>This exception is thrown when the Amazon Cognito service can't find the requested resource.</p>
-    ResourceNotFoundException(crate::types::error::ResourceNotFoundException),
+    ResourceNotFoundException(super::super::types::error::ResourceNotFoundException),
     /// <p>This exception is thrown when the user has made too many requests for a given operation.</p>
-    TooManyRequestsException(crate::types::error::TooManyRequestsException),
+    TooManyRequestsException(super::super::types::error::TooManyRequestsException),
     /// <p>This exception is thrown when a user isn't found.</p>
-    UserNotFoundException(crate::types::error::UserNotFoundException),
+    UserNotFoundException(super::super::types::error::UserNotFoundException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -348,7 +348,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-AdminUpdateDeviceStatusError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl AdminUpdateDeviceStatusError {
     /// Creates the `AdminUpdateDeviceStatusError::Unhandled` variant from any error type.
@@ -355,7 +355,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -363,7 +363,7 @@

     /// Creates the `AdminUpdateDeviceStatusError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -482,21 +482,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError {
+impl ::aws_types::request_id::RequestId for super::super::operation::admin_update_device_status::AdminUpdateDeviceStatusError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::admin_update_device_status::_admin_update_device_status_input::AdminUpdateDeviceStatusInput;
+pub use super::super::operation::admin_update_device_status::_admin_update_device_status_input::AdminUpdateDeviceStatusInput;

-pub use crate::operation::admin_update_device_status::_admin_update_device_status_output::AdminUpdateDeviceStatusOutput;
+pub use super::super::operation::admin_update_device_status::_admin_update_device_status_output::AdminUpdateDeviceStatusOutput;

 mod _admin_update_device_status_input;

