--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::confirm_device::ConfirmDeviceInput,
+        input: super::super::operation::confirm_device::ConfirmDeviceInput,
     ) -> ::std::result::Result<
-        crate::operation::confirm_device::ConfirmDeviceOutput,
+        super::super::operation::confirm_device::ConfirmDeviceOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::confirm_device::ConfirmDeviceError,
+            super::super::operation::confirm_device::ConfirmDeviceError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::confirm_device::ConfirmDeviceError>()
+                err.downcast::<super::super::operation::confirm_device::ConfirmDeviceError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::confirm_device::ConfirmDeviceOutput>()
+                .downcast::<super::super::operation::confirm_device::ConfirmDeviceOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::confirm_device::ConfirmDeviceInput,
+        input: super::super::operation::confirm_device::ConfirmDeviceInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("ConfirmDevice")
                 .build()
                 .expect("required fields set"),
@@ -132,13 +132,13 @@
                 ConfirmDeviceEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::confirm_device::ConfirmDeviceError,
+                super::super::operation::confirm_device::ConfirmDeviceError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::confirm_device::ConfirmDeviceError,
+                super::super::operation::confirm_device::ConfirmDeviceError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::confirm_device::ConfirmDeviceError,
+                super::super::operation::confirm_device::ConfirmDeviceError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -208,11 +208,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_confirm_device::de_confirm_device_http_error(status, headers, body)
+            super::super::protocol_serde::shape_confirm_device::de_confirm_device_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_confirm_device::de_confirm_device_http_response(status, headers, body)
+            super::super::protocol_serde::shape_confirm_device::de_confirm_device_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -225,16 +225,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::confirm_device::ConfirmDeviceInput>()
+            .downcast::<super::super::operation::confirm_device::ConfirmDeviceInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::confirm_device::ConfirmDeviceInput,
+                _input: &super::super::operation::confirm_device::ConfirmDeviceInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -243,7 +243,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::confirm_device::ConfirmDeviceInput,
+                input: &super::super::operation::confirm_device::ConfirmDeviceInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -259,7 +259,7 @@
             );
             builder
         };
-        let body = ::aws_smithy_types::body::SdkBody::from(crate::protocol_serde::shape_confirm_device::ser_confirm_device_input(&input)?);
+        let body = ::aws_smithy_types::body::SdkBody::from(super::super::protocol_serde::shape_confirm_device::ser_confirm_device_input(&input)?);
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
             request_builder = _header_serialization_settings.set_default_header(request_builder, ::http_1x::header::CONTENT_LENGTH, &content_length);
@@ -291,7 +291,7 @@
             .downcast_ref::<ConfirmDeviceInput>()
             .ok_or("failed to downcast to ConfirmDeviceInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -314,35 +314,35 @@
 #[derive(::std::fmt::Debug)]
 pub enum ConfirmDeviceError {
     /// <p>This exception is thrown when a user attempts to confirm a device with a device key that already exists.</p>
-    DeviceKeyExistsException(crate::types::error::DeviceKeyExistsException),
+    DeviceKeyExistsException(super::super::types::error::DeviceKeyExistsException),
     /// <p>This exception is thrown when WAF doesn't allow your request based on a web ACL that's associated with your user pool.</p>
-    ForbiddenException(crate::types::error::ForbiddenException),
+    ForbiddenException(super::super::types::error::ForbiddenException),
     /// <p>This exception is thrown when Amazon Cognito encounters an internal error.</p>
-    InternalErrorException(crate::types::error::InternalErrorException),
+    InternalErrorException(super::super::types::error::InternalErrorException),
     /// <p>This exception is thrown when Amazon Cognito encounters an invalid Lambda response.</p>
-    InvalidLambdaResponseException(crate::types::error::InvalidLambdaResponseException),
+    InvalidLambdaResponseException(super::super::types::error::InvalidLambdaResponseException),
     /// <p>This exception is thrown when the Amazon Cognito service encounters an invalid parameter.</p>
-    InvalidParameterException(crate::types::error::InvalidParameterException),
+    InvalidParameterException(super::super::types::error::InvalidParameterException),
     /// <p>This exception is thrown when Amazon Cognito encounters an invalid password.</p>
-    InvalidPasswordException(crate::types::error::InvalidPasswordException),
+    InvalidPasswordException(super::super::types::error::InvalidPasswordException),
     /// <p>This exception is thrown when the user pool configuration is not valid.</p>
-    InvalidUserPoolConfigurationException(crate::types::error::InvalidUserPoolConfigurationException),
+    InvalidUserPoolConfigurationException(super::super::types::error::InvalidUserPoolConfigurationException),
     /// <p>This exception is thrown when a user isn't authorized.</p>
-    NotAuthorizedException(crate::types::error::NotAuthorizedException),
+    NotAuthorizedException(super::super::types::error::NotAuthorizedException),
     /// <p>This exception is thrown when an operation is not available in the current region or for the current user pool configuration. This can occur when attempting to perform operations that are not supported in secondary replica regions.</p>
-    OperationNotEnabledException(crate::types::error::OperationNotEnabledException),
+    OperationNotEnabledException(super::super::types::error::OperationNotEnabledException),
     /// <p>This exception is thrown when a password reset is required.</p>
-    PasswordResetRequiredException(crate::types::error::PasswordResetRequiredException),
+    PasswordResetRequiredException(super::super::types::error::PasswordResetRequiredException),
     /// <p>This exception is thrown when the Amazon Cognito service can't find the requested resource.</p>
-    ResourceNotFoundException(crate::types::error::ResourceNotFoundException),
+    ResourceNotFoundException(super::super::types::error::ResourceNotFoundException),
     /// <p>This exception is thrown when the user has made too many requests for a given operation.</p>
-    TooManyRequestsException(crate::types::error::TooManyRequestsException),
+    TooManyRequestsException(super::super::types::error::TooManyRequestsException),
     /// <p>This exception is thrown when Amazon Cognito encounters a user name that already exists in the user pool.</p>
-    UsernameExistsException(crate::types::error::UsernameExistsException),
+    UsernameExistsException(super::super::types::error::UsernameExistsException),
     /// <p>This exception is thrown when a user isn't confirmed successfully.</p>
-    UserNotConfirmedException(crate::types::error::UserNotConfirmedException),
+    UserNotConfirmedException(super::super::types::error::UserNotConfirmedException),
     /// <p>This exception is thrown when a user isn't found.</p>
-    UserNotFoundException(crate::types::error::UserNotFoundException),
+    UserNotFoundException(super::super::types::error::UserNotFoundException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -350,7 +350,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-ConfirmDeviceError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl ConfirmDeviceError {
     /// Creates the `ConfirmDeviceError::Unhandled` variant from any error type.
@@ -357,7 +357,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -365,7 +365,7 @@

     /// Creates the `ConfirmDeviceError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -540,21 +540,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::confirm_device::ConfirmDeviceError {
+impl ::aws_types::request_id::RequestId for super::super::operation::confirm_device::ConfirmDeviceError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::confirm_device::_confirm_device_input::ConfirmDeviceInput;
+pub use super::super::operation::confirm_device::_confirm_device_input::ConfirmDeviceInput;

-pub use crate::operation::confirm_device::_confirm_device_output::ConfirmDeviceOutput;
+pub use super::super::operation::confirm_device::_confirm_device_output::ConfirmDeviceOutput;

 mod _confirm_device_input;

