--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl SelectAggregateResourceConfigInput {
     /// Creates a new builder-style object to manufacture [`SelectAggregateResourceConfigInput`](crate::operation::select_aggregate_resource_config::SelectAggregateResourceConfigInput).
-    pub fn builder() -> crate::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigInputBuilder {
-        crate::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigInputBuilder::default()
+    pub fn builder() -> super::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigInputBuilder {
+        super::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigInputBuilder::default()
     }
 }

@@ -130,10 +130,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::select_aggregate_resource_config::SelectAggregateResourceConfigInput,
+        super::operation::select_aggregate_resource_config::SelectAggregateResourceConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::select_aggregate_resource_config::SelectAggregateResourceConfigInput {
+        ::std::result::Result::Ok(super::operation::select_aggregate_resource_config::SelectAggregateResourceConfigInput {
             expression: self.expression,
             configuration_aggregator_name: self.configuration_aggregator_name,
             limit: self.limit,
