--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::GuardrailAutomatedReasoningFinding>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::GuardrailAutomatedReasoningFinding>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -37,50 +37,50 @@
                     }
                     variant = match key.as_ref() {
                             "valid" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::Valid(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_valid_finding::de_guardrail_automated_reasoning_valid_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::Valid(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_valid_finding::de_guardrail_automated_reasoning_valid_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'valid' cannot be null"))?
                                 ))
                             }
                             "invalid" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::Invalid(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_invalid_finding::de_guardrail_automated_reasoning_invalid_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::Invalid(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_invalid_finding::de_guardrail_automated_reasoning_invalid_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'invalid' cannot be null"))?
                                 ))
                             }
                             "satisfiable" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::Satisfiable(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_satisfiable_finding::de_guardrail_automated_reasoning_satisfiable_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::Satisfiable(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_satisfiable_finding::de_guardrail_automated_reasoning_satisfiable_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'satisfiable' cannot be null"))?
                                 ))
                             }
                             "impossible" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::Impossible(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_impossible_finding::de_guardrail_automated_reasoning_impossible_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::Impossible(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_impossible_finding::de_guardrail_automated_reasoning_impossible_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'impossible' cannot be null"))?
                                 ))
                             }
                             "translationAmbiguous" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::TranslationAmbiguous(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_translation_ambiguous_finding::de_guardrail_automated_reasoning_translation_ambiguous_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::TranslationAmbiguous(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_translation_ambiguous_finding::de_guardrail_automated_reasoning_translation_ambiguous_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'translationAmbiguous' cannot be null"))?
                                 ))
                             }
                             "tooComplex" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::TooComplex(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_too_complex_finding::de_guardrail_automated_reasoning_too_complex_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::TooComplex(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_too_complex_finding::de_guardrail_automated_reasoning_too_complex_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'tooComplex' cannot be null"))?
                                 ))
                             }
                             "noTranslations" => {
-                                Some(crate::types::GuardrailAutomatedReasoningFinding::NoTranslations(
-                                    crate::protocol_serde::shape_guardrail_automated_reasoning_no_translations_finding::de_guardrail_automated_reasoning_no_translations_finding(tokens, _value, depth + 1)?
+                                Some(super::super::types::GuardrailAutomatedReasoningFinding::NoTranslations(
+                                    super::super::protocol_serde::shape_guardrail_automated_reasoning_no_translations_finding::de_guardrail_automated_reasoning_no_translations_finding(tokens, _value, depth + 1)?
                                     .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'noTranslations' cannot be null"))?
                                 ))
                             }
                             _ => {
                                                                               ::aws_smithy_json::deserialize::token::skip_value(tokens)?;
-                                                                              Some(crate::types::GuardrailAutomatedReasoningFinding::Unknown)
+                                                                              Some(super::super::types::GuardrailAutomatedReasoningFinding::Unknown)
                                                                             }
                         };
                 }
