--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<crate::types::GuardrailAutomatedReasoningNoTranslationsFinding>,
+    Option<super::types::GuardrailAutomatedReasoningNoTranslationsFinding>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -19,7 +19,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::GuardrailAutomatedReasoningNoTranslationsFindingBuilder::default();
+            let mut builder = super::types::builders::GuardrailAutomatedReasoningNoTranslationsFindingBuilder::default();
             ::aws_smithy_json::deserialize::token::skip_to_end(tokens)?;
             Ok(Some(builder.build()))
         }
