--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::vec::Vec<crate::types::GuardrailAutomatedReasoningTranslation>>,
+    Option<::std::vec::Vec<super::types::GuardrailAutomatedReasoningTranslation>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -27,7 +27,7 @@
                     }
                     _ => {
                         let value =
-                            crate::protocol_serde::shape_guardrail_automated_reasoning_translation::de_guardrail_automated_reasoning_translation(
+                            super::protocol_serde::shape_guardrail_automated_reasoning_translation::de_guardrail_automated_reasoning_translation(
                                 tokens,
                                 _value,
                                 depth + 1,
