--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ReasoningContentBlockDelta>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ReasoningContentBlockDelta>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -36,18 +36,18 @@
                         ));
                     }
                     variant = match key.as_ref() {
-                        "text" => Some(crate::types::ReasoningContentBlockDelta::Text(
+                        "text" => Some(super::types::ReasoningContentBlockDelta::Text(
                             ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                 .map(|s| s.to_unescaped().map(|u| u.into_owned()))
                                 .transpose()?
                                 .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'text' cannot be null"))?,
                         )),
-                        "redactedContent" => Some(crate::types::ReasoningContentBlockDelta::RedactedContent(
+                        "redactedContent" => Some(super::types::ReasoningContentBlockDelta::RedactedContent(
                             ::aws_smithy_json::deserialize::token::expect_blob_or_null(tokens.next())?.ok_or_else(|| {
                                 ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'redactedContent' cannot be null")
                             })?,
                         )),
-                        "signature" => Some(crate::types::ReasoningContentBlockDelta::Signature(
+                        "signature" => Some(super::types::ReasoningContentBlockDelta::Signature(
                             ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                 .map(|s| s.to_unescaped().map(|u| u.into_owned()))
                                 .transpose()?
@@ -57,7 +57,7 @@
                         )),
                         _ => {
                             ::aws_smithy_json::deserialize::token::skip_value(tokens)?;
-                            Some(crate::types::ReasoningContentBlockDelta::Unknown)
+                            Some(super::types::ReasoningContentBlockDelta::Unknown)
                         }
                     };
                 }
