--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::GuardrailTraceAssessment>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::GuardrailTraceAssessment>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::GuardrailTraceAssessmentBuilder::default();
+            let mut builder = super::types::builders::GuardrailTraceAssessmentBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -23,16 +23,16 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                         "modelOutput" => {
                             builder =
-                                builder.set_model_output(crate::protocol_serde::shape_model_outputs::de_model_outputs(tokens, _value, depth + 1)?);
+                                builder.set_model_output(super::protocol_serde::shape_model_outputs::de_model_outputs(tokens, _value, depth + 1)?);
                         }
                         "inputAssessment" => {
                             builder = builder.set_input_assessment(
-                                crate::protocol_serde::shape_guardrail_assessment_map::de_guardrail_assessment_map(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_guardrail_assessment_map::de_guardrail_assessment_map(tokens, _value, depth + 1)?,
                             );
                         }
                         "outputAssessments" => {
                             builder = builder.set_output_assessments(
-                                crate::protocol_serde::shape_guardrail_assessment_list_map::de_guardrail_assessment_list_map(
+                                super::protocol_serde::shape_guardrail_assessment_list_map::de_guardrail_assessment_list_map(
                                     tokens,
                                     _value,
                                     depth + 1,
