--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ContentBlockDelta>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ContentBlockDelta>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -36,41 +36,41 @@
                         ));
                     }
                     variant = match key.as_ref() {
-                        "text" => Some(crate::types::ContentBlockDelta::Text(
+                        "text" => Some(super::types::ContentBlockDelta::Text(
                             ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                 .map(|s| s.to_unescaped().map(|u| u.into_owned()))
                                 .transpose()?
                                 .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'text' cannot be null"))?,
                         )),
-                        "toolUse" => Some(crate::types::ContentBlockDelta::ToolUse(
-                            crate::protocol_serde::shape_tool_use_block_delta::de_tool_use_block_delta(tokens, _value, depth + 1)?.ok_or_else(
+                        "toolUse" => Some(super::types::ContentBlockDelta::ToolUse(
+                            super::protocol_serde::shape_tool_use_block_delta::de_tool_use_block_delta(tokens, _value, depth + 1)?.ok_or_else(
                                 || ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'toolUse' cannot be null"),
                             )?,
                         )),
-                        "toolResult" => Some(crate::types::ContentBlockDelta::ToolResult(
-                            crate::protocol_serde::shape_tool_result_blocks_delta::de_tool_result_blocks_delta(tokens, _value, depth + 1)?
+                        "toolResult" => Some(super::types::ContentBlockDelta::ToolResult(
+                            super::protocol_serde::shape_tool_result_blocks_delta::de_tool_result_blocks_delta(tokens, _value, depth + 1)?
                                 .ok_or_else(|| {
                                     ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'toolResult' cannot be null")
                                 })?,
                         )),
-                        "reasoningContent" => Some(crate::types::ContentBlockDelta::ReasoningContent(
-                            crate::protocol_serde::shape_reasoning_content_block_delta::de_reasoning_content_block_delta(tokens, _value, depth + 1)?
+                        "reasoningContent" => Some(super::types::ContentBlockDelta::ReasoningContent(
+                            super::protocol_serde::shape_reasoning_content_block_delta::de_reasoning_content_block_delta(tokens, _value, depth + 1)?
                                 .ok_or_else(|| {
                                     ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'reasoningContent' cannot be null")
                                 })?,
                         )),
-                        "citation" => Some(crate::types::ContentBlockDelta::Citation(
-                            crate::protocol_serde::shape_citations_delta::de_citations_delta(tokens, _value, depth + 1)?.ok_or_else(|| {
+                        "citation" => Some(super::types::ContentBlockDelta::Citation(
+                            super::protocol_serde::shape_citations_delta::de_citations_delta(tokens, _value, depth + 1)?.ok_or_else(|| {
                                 ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'citation' cannot be null")
                             })?,
                         )),
-                        "image" => Some(crate::types::ContentBlockDelta::Image(
-                            crate::protocol_serde::shape_image_block_delta::de_image_block_delta(tokens, _value, depth + 1)?
+                        "image" => Some(super::types::ContentBlockDelta::Image(
+                            super::protocol_serde::shape_image_block_delta::de_image_block_delta(tokens, _value, depth + 1)?
                                 .ok_or_else(|| ::aws_smithy_json::deserialize::error::DeserializeError::custom("value for 'image' cannot be null"))?,
                         )),
                         _ => {
                             ::aws_smithy_json::deserialize::token::skip_value(tokens)?;
-                            Some(crate::types::ContentBlockDelta::Unknown)
+                            Some(super::types::ContentBlockDelta::Unknown)
                         }
                     };
                 }
