--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct UpdateTableInput {
     /// <p>An array of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, <code>AttributeDefinitions</code> must include the key element(s) of the new index.</p>
-    pub attribute_definitions: ::std::option::Option<::std::vec::Vec<crate::types::AttributeDefinition>>,
+    pub attribute_definitions: ::std::option::Option<::std::vec::Vec<super::super::super::types::AttributeDefinition>>,
     /// <p>The name of the table to be updated. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
     pub table_name: ::std::option::Option<::std::string::String>,
     /// <p>Controls how you are charged for read and write throughput and how you manage capacity. When switching from pay-per-request to provisioned capacity, initial provisioned capacity values must be set. The initial provisioned capacity values are estimated based on the consumed read and write capacity of your table and global secondary indexes over the past 30 minutes.</p>
@@ -15,9 +15,9 @@
     /// <li>
     /// <p><code>PROVISIONED</code> - We recommend using <code>PROVISIONED</code> for steady workloads with predictable growth where capacity requirements can be reliably forecasted. <code>PROVISIONED</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/provisioned-capacity-mode.html">Provisioned capacity mode</a>.</p></li>
     /// </ul>
-    pub billing_mode: ::std::option::Option<crate::types::BillingMode>,
+    pub billing_mode: ::std::option::Option<super::super::super::types::BillingMode>,
     /// <p>The new provisioned throughput settings for the specified table or index.</p>
-    pub provisioned_throughput: ::std::option::Option<crate::types::ProvisionedThroughput>,
+    pub provisioned_throughput: ::std::option::Option<super::super::super::types::ProvisionedThroughput>,
     /// <p>An array of one or more global secondary indexes for the table. For each index in the array, you can request one action:</p>
     /// <ul>
     /// <li>
@@ -29,17 +29,17 @@
     /// </ul>
     /// <p>You can create or delete only one global secondary index per <code>UpdateTable</code> operation.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.OnlineOps.html">Managing Global Secondary Indexes</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
-    pub global_secondary_index_updates: ::std::option::Option<::std::vec::Vec<crate::types::GlobalSecondaryIndexUpdate>>,
+    pub global_secondary_index_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalSecondaryIndexUpdate>>,
     /// <p>Represents the DynamoDB Streams configuration for the table.</p><note>
     /// <p>You receive a <code>ValidationException</code> if you try to enable a stream on a table that already has a stream, or if you try to disable a stream on a table that doesn't have a stream.</p>
     /// </note>
-    pub stream_specification: ::std::option::Option<crate::types::StreamSpecification>,
+    pub stream_specification: ::std::option::Option<super::super::super::types::StreamSpecification>,
     /// <p>The new server-side encryption settings for the specified table.</p>
-    pub sse_specification: ::std::option::Option<crate::types::SseSpecification>,
+    pub sse_specification: ::std::option::Option<super::super::super::types::SseSpecification>,
     /// <p>A list of replica update actions (create, delete, or update) for the table.</p>
-    pub replica_updates: ::std::option::Option<::std::vec::Vec<crate::types::ReplicationGroupUpdate>>,
+    pub replica_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicationGroupUpdate>>,
     /// <p>The table class of the table to be updated. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub table_class: ::std::option::Option<crate::types::TableClass>,
+    pub table_class: ::std::option::Option<super::super::super::types::TableClass>,
     /// <p>Indicates whether deletion protection is to be enabled (true) or disabled (false) on the table.</p>
     pub deletion_protection_enabled: ::std::option::Option<bool>,
     /// <p>Specifies the consistency mode for a new global table. This parameter is only valid when you create a global table by specifying one or more <a href="https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ReplicationGroupUpdate.html#DDB-Type-ReplicationGroupUpdate-Create">Create</a> actions in the <a href="https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateTable.html#DDB-UpdateTable-request-ReplicaUpdates">ReplicaUpdates</a> action list.</p>
@@ -51,7 +51,7 @@
     /// <p><code>STRONG</code>: Configures a new global table for multi-Region strong consistency (MRSC).</p></li>
     /// </ul>
     /// <p>If you don't specify this field, the global table consistency mode defaults to <code>EVENTUAL</code>. For more information about global tables consistency modes, see <a href="https://docs.aws.amazon.com/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes"> Consistency modes</a> in DynamoDB developer guide.</p>
-    pub multi_region_consistency: ::std::option::Option<crate::types::MultiRegionConsistency>,
+    pub multi_region_consistency: ::std::option::Option<super::super::super::types::MultiRegionConsistency>,
     /// <p>A list of witness updates for a MRSC global table. A witness provides a cost-effective alternative to a full replica in a MRSC global table by maintaining replicated change data written to global table replicas. You cannot perform read or write operations on a witness. For each witness, you can request one action:</p>
     /// <ul>
     /// <li>
@@ -61,11 +61,11 @@
     /// </ul>
     /// <p>You can create or delete only one witness per <code>UpdateTable</code> operation.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes">Multi-Region strong consistency (MRSC)</a> in the Amazon DynamoDB Developer Guide</p>
-    pub global_table_witness_updates: ::std::option::Option<::std::vec::Vec<crate::types::GlobalTableWitnessGroupUpdate>>,
+    pub global_table_witness_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalTableWitnessGroupUpdate>>,
     /// <p>Updates the maximum number of read and write units for the specified table in on-demand capacity mode. If you use this parameter, you must specify <code>MaxReadRequestUnits</code>, <code>MaxWriteRequestUnits</code>, or both.</p>
-    pub on_demand_throughput: ::std::option::Option<crate::types::OnDemandThroughput>,
+    pub on_demand_throughput: ::std::option::Option<super::super::super::types::OnDemandThroughput>,
     /// <p>Represents the warm throughput (in read units per second and write units per second) for updating a table.</p>
-    pub warm_throughput: ::std::option::Option<crate::types::WarmThroughput>,
+    pub warm_throughput: ::std::option::Option<super::super::super::types::WarmThroughput>,
     /// <p>Controls the settings replication mode for a global table replica. This attribute can be defined using UpdateTable operation only on a regional table with values:</p>
     /// <ul>
     /// <li>
@@ -73,16 +73,16 @@
     /// <li>
     /// <p><code>DISABLED</code>: Remove settings replication on a regional table. Settings replication needs to be defined to ENABLED again in order to create a Multi-Account Global Table using this table.</p></li>
     /// </ul>
-    pub global_table_settings_replication_mode: ::std::option::Option<crate::types::GlobalTableSettingsReplicationMode>,
+    pub global_table_settings_replication_mode: ::std::option::Option<super::super::super::types::GlobalTableSettingsReplicationMode>,
     /// <p>A list of vector indexes to be added to or removed from the table. You can add or remove one vector index for each <code>UpdateTable</code> operation.</p>
     /// <p>To add a vector index, specify <code>IndexName</code>, <code>VectorAttribute</code>, <code>Dimensions</code>, <code>DistanceFunction</code>, and <code>Projection</code>. To remove a vector index, specify only the <code>IndexName</code>.</p>
-    pub vector_index_updates: ::std::option::Option<::std::vec::Vec<crate::types::VectorIndexUpdate>>,
+    pub vector_index_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::VectorIndexUpdate>>,
 }
 impl UpdateTableInput {
     /// <p>An array of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, <code>AttributeDefinitions</code> must include the key element(s) of the new index.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.attribute_definitions.is_none()`.
-    pub fn attribute_definitions(&self) -> &[crate::types::AttributeDefinition] {
+    pub fn attribute_definitions(&self) -> &[super::super::super::types::AttributeDefinition] {
         self.attribute_definitions.as_deref().unwrap_or_default()
     }
     /// <p>The name of the table to be updated. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
@@ -96,11 +96,11 @@
     /// <li>
     /// <p><code>PROVISIONED</code> - We recommend using <code>PROVISIONED</code> for steady workloads with predictable growth where capacity requirements can be reliably forecasted. <code>PROVISIONED</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/provisioned-capacity-mode.html">Provisioned capacity mode</a>.</p></li>
     /// </ul>
-    pub fn billing_mode(&self) -> ::std::option::Option<&crate::types::BillingMode> {
+    pub fn billing_mode(&self) -> ::std::option::Option<&super::super::super::types::BillingMode> {
         self.billing_mode.as_ref()
     }
     /// <p>The new provisioned throughput settings for the specified table or index.</p>
-    pub fn provisioned_throughput(&self) -> ::std::option::Option<&crate::types::ProvisionedThroughput> {
+    pub fn provisioned_throughput(&self) -> ::std::option::Option<&super::super::super::types::ProvisionedThroughput> {
         self.provisioned_throughput.as_ref()
     }
     /// <p>An array of one or more global secondary indexes for the table. For each index in the array, you can request one action:</p>
@@ -116,27 +116,27 @@
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.OnlineOps.html">Managing Global Secondary Indexes</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.global_secondary_index_updates.is_none()`.
-    pub fn global_secondary_index_updates(&self) -> &[crate::types::GlobalSecondaryIndexUpdate] {
+    pub fn global_secondary_index_updates(&self) -> &[super::super::super::types::GlobalSecondaryIndexUpdate] {
         self.global_secondary_index_updates.as_deref().unwrap_or_default()
     }
     /// <p>Represents the DynamoDB Streams configuration for the table.</p><note>
     /// <p>You receive a <code>ValidationException</code> if you try to enable a stream on a table that already has a stream, or if you try to disable a stream on a table that doesn't have a stream.</p>
     /// </note>
-    pub fn stream_specification(&self) -> ::std::option::Option<&crate::types::StreamSpecification> {
+    pub fn stream_specification(&self) -> ::std::option::Option<&super::super::super::types::StreamSpecification> {
         self.stream_specification.as_ref()
     }
     /// <p>The new server-side encryption settings for the specified table.</p>
-    pub fn sse_specification(&self) -> ::std::option::Option<&crate::types::SseSpecification> {
+    pub fn sse_specification(&self) -> ::std::option::Option<&super::super::super::types::SseSpecification> {
         self.sse_specification.as_ref()
     }
     /// <p>A list of replica update actions (create, delete, or update) for the table.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replica_updates.is_none()`.
-    pub fn replica_updates(&self) -> &[crate::types::ReplicationGroupUpdate] {
+    pub fn replica_updates(&self) -> &[super::super::super::types::ReplicationGroupUpdate] {
         self.replica_updates.as_deref().unwrap_or_default()
     }
     /// <p>The table class of the table to be updated. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn table_class(&self) -> ::std::option::Option<&crate::types::TableClass> {
+    pub fn table_class(&self) -> ::std::option::Option<&super::super::super::types::TableClass> {
         self.table_class.as_ref()
     }
     /// <p>Indicates whether deletion protection is to be enabled (true) or disabled (false) on the table.</p>
@@ -152,7 +152,7 @@
     /// <p><code>STRONG</code>: Configures a new global table for multi-Region strong consistency (MRSC).</p></li>
     /// </ul>
     /// <p>If you don't specify this field, the global table consistency mode defaults to <code>EVENTUAL</code>. For more information about global tables consistency modes, see <a href="https://docs.aws.amazon.com/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes"> Consistency modes</a> in DynamoDB developer guide.</p>
-    pub fn multi_region_consistency(&self) -> ::std::option::Option<&crate::types::MultiRegionConsistency> {
+    pub fn multi_region_consistency(&self) -> ::std::option::Option<&super::super::super::types::MultiRegionConsistency> {
         self.multi_region_consistency.as_ref()
     }
     /// <p>A list of witness updates for a MRSC global table. A witness provides a cost-effective alternative to a full replica in a MRSC global table by maintaining replicated change data written to global table replicas. You cannot perform read or write operations on a witness. For each witness, you can request one action:</p>
@@ -166,15 +166,15 @@
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes">Multi-Region strong consistency (MRSC)</a> in the Amazon DynamoDB Developer Guide</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.global_table_witness_updates.is_none()`.
-    pub fn global_table_witness_updates(&self) -> &[crate::types::GlobalTableWitnessGroupUpdate] {
+    pub fn global_table_witness_updates(&self) -> &[super::super::super::types::GlobalTableWitnessGroupUpdate] {
         self.global_table_witness_updates.as_deref().unwrap_or_default()
     }
     /// <p>Updates the maximum number of read and write units for the specified table in on-demand capacity mode. If you use this parameter, you must specify <code>MaxReadRequestUnits</code>, <code>MaxWriteRequestUnits</code>, or both.</p>
-    pub fn on_demand_throughput(&self) -> ::std::option::Option<&crate::types::OnDemandThroughput> {
+    pub fn on_demand_throughput(&self) -> ::std::option::Option<&super::super::super::types::OnDemandThroughput> {
         self.on_demand_throughput.as_ref()
     }
     /// <p>Represents the warm throughput (in read units per second and write units per second) for updating a table.</p>
-    pub fn warm_throughput(&self) -> ::std::option::Option<&crate::types::WarmThroughput> {
+    pub fn warm_throughput(&self) -> ::std::option::Option<&super::super::super::types::WarmThroughput> {
         self.warm_throughput.as_ref()
     }
     /// <p>Controls the settings replication mode for a global table replica. This attribute can be defined using UpdateTable operation only on a regional table with values:</p>
@@ -184,7 +184,7 @@
     /// <li>
     /// <p><code>DISABLED</code>: Remove settings replication on a regional table. Settings replication needs to be defined to ENABLED again in order to create a Multi-Account Global Table using this table.</p></li>
     /// </ul>
-    pub fn global_table_settings_replication_mode(&self) -> ::std::option::Option<&crate::types::GlobalTableSettingsReplicationMode> {
+    pub fn global_table_settings_replication_mode(&self) -> ::std::option::Option<&super::super::super::types::GlobalTableSettingsReplicationMode> {
         self.global_table_settings_replication_mode.as_ref()
     }
     /// <p>A list of vector indexes to be added to or removed from the table. You can add or remove one vector index for each <code>UpdateTable</code> operation.</p>
@@ -191,14 +191,14 @@
     /// <p>To add a vector index, specify <code>IndexName</code>, <code>VectorAttribute</code>, <code>Dimensions</code>, <code>DistanceFunction</code>, and <code>Projection</code>. To remove a vector index, specify only the <code>IndexName</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.vector_index_updates.is_none()`.
-    pub fn vector_index_updates(&self) -> &[crate::types::VectorIndexUpdate] {
+    pub fn vector_index_updates(&self) -> &[super::super::super::types::VectorIndexUpdate] {
         self.vector_index_updates.as_deref().unwrap_or_default()
     }
 }
 impl UpdateTableInput {
     /// Creates a new builder-style object to manufacture [`UpdateTableInput`](crate::operation::update_table::UpdateTableInput).
-    pub fn builder() -> crate::operation::update_table::builders::UpdateTableInputBuilder {
-        crate::operation::update_table::builders::UpdateTableInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_table::builders::UpdateTableInputBuilder {
+        super::super::super::operation::update_table::builders::UpdateTableInputBuilder::default()
     }
 }

@@ -206,22 +206,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct UpdateTableInputBuilder {
-    pub(crate) attribute_definitions: ::std::option::Option<::std::vec::Vec<crate::types::AttributeDefinition>>,
+    pub(crate) attribute_definitions: ::std::option::Option<::std::vec::Vec<super::super::super::types::AttributeDefinition>>,
     pub(crate) table_name: ::std::option::Option<::std::string::String>,
-    pub(crate) billing_mode: ::std::option::Option<crate::types::BillingMode>,
-    pub(crate) provisioned_throughput: ::std::option::Option<crate::types::ProvisionedThroughput>,
-    pub(crate) global_secondary_index_updates: ::std::option::Option<::std::vec::Vec<crate::types::GlobalSecondaryIndexUpdate>>,
-    pub(crate) stream_specification: ::std::option::Option<crate::types::StreamSpecification>,
-    pub(crate) sse_specification: ::std::option::Option<crate::types::SseSpecification>,
-    pub(crate) replica_updates: ::std::option::Option<::std::vec::Vec<crate::types::ReplicationGroupUpdate>>,
-    pub(crate) table_class: ::std::option::Option<crate::types::TableClass>,
+    pub(crate) billing_mode: ::std::option::Option<super::super::super::types::BillingMode>,
+    pub(crate) provisioned_throughput: ::std::option::Option<super::super::super::types::ProvisionedThroughput>,
+    pub(crate) global_secondary_index_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalSecondaryIndexUpdate>>,
+    pub(crate) stream_specification: ::std::option::Option<super::super::super::types::StreamSpecification>,
+    pub(crate) sse_specification: ::std::option::Option<super::super::super::types::SseSpecification>,
+    pub(crate) replica_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicationGroupUpdate>>,
+    pub(crate) table_class: ::std::option::Option<super::super::super::types::TableClass>,
     pub(crate) deletion_protection_enabled: ::std::option::Option<bool>,
-    pub(crate) multi_region_consistency: ::std::option::Option<crate::types::MultiRegionConsistency>,
-    pub(crate) global_table_witness_updates: ::std::option::Option<::std::vec::Vec<crate::types::GlobalTableWitnessGroupUpdate>>,
-    pub(crate) on_demand_throughput: ::std::option::Option<crate::types::OnDemandThroughput>,
-    pub(crate) warm_throughput: ::std::option::Option<crate::types::WarmThroughput>,
-    pub(crate) global_table_settings_replication_mode: ::std::option::Option<crate::types::GlobalTableSettingsReplicationMode>,
-    pub(crate) vector_index_updates: ::std::option::Option<::std::vec::Vec<crate::types::VectorIndexUpdate>>,
+    pub(crate) multi_region_consistency: ::std::option::Option<super::super::super::types::MultiRegionConsistency>,
+    pub(crate) global_table_witness_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalTableWitnessGroupUpdate>>,
+    pub(crate) on_demand_throughput: ::std::option::Option<super::super::super::types::OnDemandThroughput>,
+    pub(crate) warm_throughput: ::std::option::Option<super::super::super::types::WarmThroughput>,
+    pub(crate) global_table_settings_replication_mode: ::std::option::Option<super::super::super::types::GlobalTableSettingsReplicationMode>,
+    pub(crate) vector_index_updates: ::std::option::Option<::std::vec::Vec<super::super::super::types::VectorIndexUpdate>>,
 }
 impl UpdateTableInputBuilder {
     /// Appends an item to `attribute_definitions`.
@@ -229,7 +229,7 @@
     /// To override the contents of this collection use [`set_attribute_definitions`](Self::set_attribute_definitions).
     ///
     /// <p>An array of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, <code>AttributeDefinitions</code> must include the key element(s) of the new index.</p>
-    pub fn attribute_definitions(mut self, input: crate::types::AttributeDefinition) -> Self {
+    pub fn attribute_definitions(mut self, input: super::super::super::types::AttributeDefinition) -> Self {
         let mut v = self.attribute_definitions.unwrap_or_default();
         v.push(input);
         self.attribute_definitions = ::std::option::Option::Some(v);
@@ -236,12 +236,12 @@
         self
     }
     /// <p>An array of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, <code>AttributeDefinitions</code> must include the key element(s) of the new index.</p>
-    pub fn set_attribute_definitions(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::AttributeDefinition>>) -> Self {
+    pub fn set_attribute_definitions(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::AttributeDefinition>>) -> Self {
         self.attribute_definitions = input;
         self
     }
     /// <p>An array of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, <code>AttributeDefinitions</code> must include the key element(s) of the new index.</p>
-    pub fn get_attribute_definitions(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::AttributeDefinition>> {
+    pub fn get_attribute_definitions(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::AttributeDefinition>> {
         &self.attribute_definitions
     }
     /// <p>The name of the table to be updated. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
@@ -266,7 +266,7 @@
     /// <li>
     /// <p><code>PROVISIONED</code> - We recommend using <code>PROVISIONED</code> for steady workloads with predictable growth where capacity requirements can be reliably forecasted. <code>PROVISIONED</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/provisioned-capacity-mode.html">Provisioned capacity mode</a>.</p></li>
     /// </ul>
-    pub fn billing_mode(mut self, input: crate::types::BillingMode) -> Self {
+    pub fn billing_mode(mut self, input: super::super::super::types::BillingMode) -> Self {
         self.billing_mode = ::std::option::Option::Some(input);
         self
     }
@@ -277,7 +277,7 @@
     /// <li>
     /// <p><code>PROVISIONED</code> - We recommend using <code>PROVISIONED</code> for steady workloads with predictable growth where capacity requirements can be reliably forecasted. <code>PROVISIONED</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/provisioned-capacity-mode.html">Provisioned capacity mode</a>.</p></li>
     /// </ul>
-    pub fn set_billing_mode(mut self, input: ::std::option::Option<crate::types::BillingMode>) -> Self {
+    pub fn set_billing_mode(mut self, input: ::std::option::Option<super::super::super::types::BillingMode>) -> Self {
         self.billing_mode = input;
         self
     }
@@ -288,21 +288,21 @@
     /// <li>
     /// <p><code>PROVISIONED</code> - We recommend using <code>PROVISIONED</code> for steady workloads with predictable growth where capacity requirements can be reliably forecasted. <code>PROVISIONED</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/provisioned-capacity-mode.html">Provisioned capacity mode</a>.</p></li>
     /// </ul>
-    pub fn get_billing_mode(&self) -> &::std::option::Option<crate::types::BillingMode> {
+    pub fn get_billing_mode(&self) -> &::std::option::Option<super::super::super::types::BillingMode> {
         &self.billing_mode
     }
     /// <p>The new provisioned throughput settings for the specified table or index.</p>
-    pub fn provisioned_throughput(mut self, input: crate::types::ProvisionedThroughput) -> Self {
+    pub fn provisioned_throughput(mut self, input: super::super::super::types::ProvisionedThroughput) -> Self {
         self.provisioned_throughput = ::std::option::Option::Some(input);
         self
     }
     /// <p>The new provisioned throughput settings for the specified table or index.</p>
-    pub fn set_provisioned_throughput(mut self, input: ::std::option::Option<crate::types::ProvisionedThroughput>) -> Self {
+    pub fn set_provisioned_throughput(mut self, input: ::std::option::Option<super::super::super::types::ProvisionedThroughput>) -> Self {
         self.provisioned_throughput = input;
         self
     }
     /// <p>The new provisioned throughput settings for the specified table or index.</p>
-    pub fn get_provisioned_throughput(&self) -> &::std::option::Option<crate::types::ProvisionedThroughput> {
+    pub fn get_provisioned_throughput(&self) -> &::std::option::Option<super::super::super::types::ProvisionedThroughput> {
         &self.provisioned_throughput
     }
     /// Appends an item to `global_secondary_index_updates`.
@@ -320,7 +320,7 @@
     /// </ul>
     /// <p>You can create or delete only one global secondary index per <code>UpdateTable</code> operation.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.OnlineOps.html">Managing Global Secondary Indexes</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
-    pub fn global_secondary_index_updates(mut self, input: crate::types::GlobalSecondaryIndexUpdate) -> Self {
+    pub fn global_secondary_index_updates(mut self, input: super::super::super::types::GlobalSecondaryIndexUpdate) -> Self {
         let mut v = self.global_secondary_index_updates.unwrap_or_default();
         v.push(input);
         self.global_secondary_index_updates = ::std::option::Option::Some(v);
@@ -339,7 +339,7 @@
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.OnlineOps.html">Managing Global Secondary Indexes</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
     pub fn set_global_secondary_index_updates(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::GlobalSecondaryIndexUpdate>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalSecondaryIndexUpdate>>,
     ) -> Self {
         self.global_secondary_index_updates = input;
         self
@@ -355,13 +355,13 @@
     /// </ul>
     /// <p>You can create or delete only one global secondary index per <code>UpdateTable</code> operation.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.OnlineOps.html">Managing Global Secondary Indexes</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
-    pub fn get_global_secondary_index_updates(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::GlobalSecondaryIndexUpdate>> {
+    pub fn get_global_secondary_index_updates(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalSecondaryIndexUpdate>> {
         &self.global_secondary_index_updates
     }
     /// <p>Represents the DynamoDB Streams configuration for the table.</p><note>
     /// <p>You receive a <code>ValidationException</code> if you try to enable a stream on a table that already has a stream, or if you try to disable a stream on a table that doesn't have a stream.</p>
     /// </note>
-    pub fn stream_specification(mut self, input: crate::types::StreamSpecification) -> Self {
+    pub fn stream_specification(mut self, input: super::super::super::types::StreamSpecification) -> Self {
         self.stream_specification = ::std::option::Option::Some(input);
         self
     }
@@ -368,7 +368,7 @@
     /// <p>Represents the DynamoDB Streams configuration for the table.</p><note>
     /// <p>You receive a <code>ValidationException</code> if you try to enable a stream on a table that already has a stream, or if you try to disable a stream on a table that doesn't have a stream.</p>
     /// </note>
-    pub fn set_stream_specification(mut self, input: ::std::option::Option<crate::types::StreamSpecification>) -> Self {
+    pub fn set_stream_specification(mut self, input: ::std::option::Option<super::super::super::types::StreamSpecification>) -> Self {
         self.stream_specification = input;
         self
     }
@@ -375,21 +375,21 @@
     /// <p>Represents the DynamoDB Streams configuration for the table.</p><note>
     /// <p>You receive a <code>ValidationException</code> if you try to enable a stream on a table that already has a stream, or if you try to disable a stream on a table that doesn't have a stream.</p>
     /// </note>
-    pub fn get_stream_specification(&self) -> &::std::option::Option<crate::types::StreamSpecification> {
+    pub fn get_stream_specification(&self) -> &::std::option::Option<super::super::super::types::StreamSpecification> {
         &self.stream_specification
     }
     /// <p>The new server-side encryption settings for the specified table.</p>
-    pub fn sse_specification(mut self, input: crate::types::SseSpecification) -> Self {
+    pub fn sse_specification(mut self, input: super::super::super::types::SseSpecification) -> Self {
         self.sse_specification = ::std::option::Option::Some(input);
         self
     }
     /// <p>The new server-side encryption settings for the specified table.</p>
-    pub fn set_sse_specification(mut self, input: ::std::option::Option<crate::types::SseSpecification>) -> Self {
+    pub fn set_sse_specification(mut self, input: ::std::option::Option<super::super::super::types::SseSpecification>) -> Self {
         self.sse_specification = input;
         self
     }
     /// <p>The new server-side encryption settings for the specified table.</p>
-    pub fn get_sse_specification(&self) -> &::std::option::Option<crate::types::SseSpecification> {
+    pub fn get_sse_specification(&self) -> &::std::option::Option<super::super::super::types::SseSpecification> {
         &self.sse_specification
     }
     /// Appends an item to `replica_updates`.
@@ -397,7 +397,7 @@
     /// To override the contents of this collection use [`set_replica_updates`](Self::set_replica_updates).
     ///
     /// <p>A list of replica update actions (create, delete, or update) for the table.</p>
-    pub fn replica_updates(mut self, input: crate::types::ReplicationGroupUpdate) -> Self {
+    pub fn replica_updates(mut self, input: super::super::super::types::ReplicationGroupUpdate) -> Self {
         let mut v = self.replica_updates.unwrap_or_default();
         v.push(input);
         self.replica_updates = ::std::option::Option::Some(v);
@@ -404,26 +404,26 @@
         self
     }
     /// <p>A list of replica update actions (create, delete, or update) for the table.</p>
-    pub fn set_replica_updates(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ReplicationGroupUpdate>>) -> Self {
+    pub fn set_replica_updates(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicationGroupUpdate>>) -> Self {
         self.replica_updates = input;
         self
     }
     /// <p>A list of replica update actions (create, delete, or update) for the table.</p>
-    pub fn get_replica_updates(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ReplicationGroupUpdate>> {
+    pub fn get_replica_updates(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::ReplicationGroupUpdate>> {
         &self.replica_updates
     }
     /// <p>The table class of the table to be updated. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn table_class(mut self, input: crate::types::TableClass) -> Self {
+    pub fn table_class(mut self, input: super::super::super::types::TableClass) -> Self {
         self.table_class = ::std::option::Option::Some(input);
         self
     }
     /// <p>The table class of the table to be updated. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn set_table_class(mut self, input: ::std::option::Option<crate::types::TableClass>) -> Self {
+    pub fn set_table_class(mut self, input: ::std::option::Option<super::super::super::types::TableClass>) -> Self {
         self.table_class = input;
         self
     }
     /// <p>The table class of the table to be updated. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn get_table_class(&self) -> &::std::option::Option<crate::types::TableClass> {
+    pub fn get_table_class(&self) -> &::std::option::Option<super::super::super::types::TableClass> {
         &self.table_class
     }
     /// <p>Indicates whether deletion protection is to be enabled (true) or disabled (false) on the table.</p>
@@ -449,7 +449,7 @@
     /// <p><code>STRONG</code>: Configures a new global table for multi-Region strong consistency (MRSC).</p></li>
     /// </ul>
     /// <p>If you don't specify this field, the global table consistency mode defaults to <code>EVENTUAL</code>. For more information about global tables consistency modes, see <a href="https://docs.aws.amazon.com/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes"> Consistency modes</a> in DynamoDB developer guide.</p>
-    pub fn multi_region_consistency(mut self, input: crate::types::MultiRegionConsistency) -> Self {
+    pub fn multi_region_consistency(mut self, input: super::super::super::types::MultiRegionConsistency) -> Self {
         self.multi_region_consistency = ::std::option::Option::Some(input);
         self
     }
@@ -462,7 +462,7 @@
     /// <p><code>STRONG</code>: Configures a new global table for multi-Region strong consistency (MRSC).</p></li>
     /// </ul>
     /// <p>If you don't specify this field, the global table consistency mode defaults to <code>EVENTUAL</code>. For more information about global tables consistency modes, see <a href="https://docs.aws.amazon.com/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes"> Consistency modes</a> in DynamoDB developer guide.</p>
-    pub fn set_multi_region_consistency(mut self, input: ::std::option::Option<crate::types::MultiRegionConsistency>) -> Self {
+    pub fn set_multi_region_consistency(mut self, input: ::std::option::Option<super::super::super::types::MultiRegionConsistency>) -> Self {
         self.multi_region_consistency = input;
         self
     }
@@ -475,7 +475,7 @@
     /// <p><code>STRONG</code>: Configures a new global table for multi-Region strong consistency (MRSC).</p></li>
     /// </ul>
     /// <p>If you don't specify this field, the global table consistency mode defaults to <code>EVENTUAL</code>. For more information about global tables consistency modes, see <a href="https://docs.aws.amazon.com/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes"> Consistency modes</a> in DynamoDB developer guide.</p>
-    pub fn get_multi_region_consistency(&self) -> &::std::option::Option<crate::types::MultiRegionConsistency> {
+    pub fn get_multi_region_consistency(&self) -> &::std::option::Option<super::super::super::types::MultiRegionConsistency> {
         &self.multi_region_consistency
     }
     /// Appends an item to `global_table_witness_updates`.
@@ -491,7 +491,7 @@
     /// </ul>
     /// <p>You can create or delete only one witness per <code>UpdateTable</code> operation.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes">Multi-Region strong consistency (MRSC)</a> in the Amazon DynamoDB Developer Guide</p>
-    pub fn global_table_witness_updates(mut self, input: crate::types::GlobalTableWitnessGroupUpdate) -> Self {
+    pub fn global_table_witness_updates(mut self, input: super::super::super::types::GlobalTableWitnessGroupUpdate) -> Self {
         let mut v = self.global_table_witness_updates.unwrap_or_default();
         v.push(input);
         self.global_table_witness_updates = ::std::option::Option::Some(v);
@@ -508,7 +508,7 @@
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes">Multi-Region strong consistency (MRSC)</a> in the Amazon DynamoDB Developer Guide</p>
     pub fn set_global_table_witness_updates(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::GlobalTableWitnessGroupUpdate>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalTableWitnessGroupUpdate>>,
     ) -> Self {
         self.global_table_witness_updates = input;
         self
@@ -522,35 +522,35 @@
     /// </ul>
     /// <p>You can create or delete only one witness per <code>UpdateTable</code> operation.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables_HowItWorks.html#V2globaltables_HowItWorks.consistency-modes">Multi-Region strong consistency (MRSC)</a> in the Amazon DynamoDB Developer Guide</p>
-    pub fn get_global_table_witness_updates(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::GlobalTableWitnessGroupUpdate>> {
+    pub fn get_global_table_witness_updates(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::GlobalTableWitnessGroupUpdate>> {
         &self.global_table_witness_updates
     }
     /// <p>Updates the maximum number of read and write units for the specified table in on-demand capacity mode. If you use this parameter, you must specify <code>MaxReadRequestUnits</code>, <code>MaxWriteRequestUnits</code>, or both.</p>
-    pub fn on_demand_throughput(mut self, input: crate::types::OnDemandThroughput) -> Self {
+    pub fn on_demand_throughput(mut self, input: super::super::super::types::OnDemandThroughput) -> Self {
         self.on_demand_throughput = ::std::option::Option::Some(input);
         self
     }
     /// <p>Updates the maximum number of read and write units for the specified table in on-demand capacity mode. If you use this parameter, you must specify <code>MaxReadRequestUnits</code>, <code>MaxWriteRequestUnits</code>, or both.</p>
-    pub fn set_on_demand_throughput(mut self, input: ::std::option::Option<crate::types::OnDemandThroughput>) -> Self {
+    pub fn set_on_demand_throughput(mut self, input: ::std::option::Option<super::super::super::types::OnDemandThroughput>) -> Self {
         self.on_demand_throughput = input;
         self
     }
     /// <p>Updates the maximum number of read and write units for the specified table in on-demand capacity mode. If you use this parameter, you must specify <code>MaxReadRequestUnits</code>, <code>MaxWriteRequestUnits</code>, or both.</p>
-    pub fn get_on_demand_throughput(&self) -> &::std::option::Option<crate::types::OnDemandThroughput> {
+    pub fn get_on_demand_throughput(&self) -> &::std::option::Option<super::super::super::types::OnDemandThroughput> {
         &self.on_demand_throughput
     }
     /// <p>Represents the warm throughput (in read units per second and write units per second) for updating a table.</p>
-    pub fn warm_throughput(mut self, input: crate::types::WarmThroughput) -> Self {
+    pub fn warm_throughput(mut self, input: super::super::super::types::WarmThroughput) -> Self {
         self.warm_throughput = ::std::option::Option::Some(input);
         self
     }
     /// <p>Represents the warm throughput (in read units per second and write units per second) for updating a table.</p>
-    pub fn set_warm_throughput(mut self, input: ::std::option::Option<crate::types::WarmThroughput>) -> Self {
+    pub fn set_warm_throughput(mut self, input: ::std::option::Option<super::super::super::types::WarmThroughput>) -> Self {
         self.warm_throughput = input;
         self
     }
     /// <p>Represents the warm throughput (in read units per second and write units per second) for updating a table.</p>
-    pub fn get_warm_throughput(&self) -> &::std::option::Option<crate::types::WarmThroughput> {
+    pub fn get_warm_throughput(&self) -> &::std::option::Option<super::super::super::types::WarmThroughput> {
         &self.warm_throughput
     }
     /// <p>Controls the settings replication mode for a global table replica. This attribute can be defined using UpdateTable operation only on a regional table with values:</p>
@@ -560,7 +560,7 @@
     /// <li>
     /// <p><code>DISABLED</code>: Remove settings replication on a regional table. Settings replication needs to be defined to ENABLED again in order to create a Multi-Account Global Table using this table.</p></li>
     /// </ul>
-    pub fn global_table_settings_replication_mode(mut self, input: crate::types::GlobalTableSettingsReplicationMode) -> Self {
+    pub fn global_table_settings_replication_mode(mut self, input: super::super::super::types::GlobalTableSettingsReplicationMode) -> Self {
         self.global_table_settings_replication_mode = ::std::option::Option::Some(input);
         self
     }
@@ -573,7 +573,7 @@
     /// </ul>
     pub fn set_global_table_settings_replication_mode(
         mut self,
-        input: ::std::option::Option<crate::types::GlobalTableSettingsReplicationMode>,
+        input: ::std::option::Option<super::super::super::types::GlobalTableSettingsReplicationMode>,
     ) -> Self {
         self.global_table_settings_replication_mode = input;
         self
@@ -585,7 +585,7 @@
     /// <li>
     /// <p><code>DISABLED</code>: Remove settings replication on a regional table. Settings replication needs to be defined to ENABLED again in order to create a Multi-Account Global Table using this table.</p></li>
     /// </ul>
-    pub fn get_global_table_settings_replication_mode(&self) -> &::std::option::Option<crate::types::GlobalTableSettingsReplicationMode> {
+    pub fn get_global_table_settings_replication_mode(&self) -> &::std::option::Option<super::super::super::types::GlobalTableSettingsReplicationMode> {
         &self.global_table_settings_replication_mode
     }
     /// Appends an item to `vector_index_updates`.
@@ -594,7 +594,7 @@
     ///
     /// <p>A list of vector indexes to be added to or removed from the table. You can add or remove one vector index for each <code>UpdateTable</code> operation.</p>
     /// <p>To add a vector index, specify <code>IndexName</code>, <code>VectorAttribute</code>, <code>Dimensions</code>, <code>DistanceFunction</code>, and <code>Projection</code>. To remove a vector index, specify only the <code>IndexName</code>.</p>
-    pub fn vector_index_updates(mut self, input: crate::types::VectorIndexUpdate) -> Self {
+    pub fn vector_index_updates(mut self, input: super::super::super::types::VectorIndexUpdate) -> Self {
         let mut v = self.vector_index_updates.unwrap_or_default();
         v.push(input);
         self.vector_index_updates = ::std::option::Option::Some(v);
@@ -602,18 +602,18 @@
     }
     /// <p>A list of vector indexes to be added to or removed from the table. You can add or remove one vector index for each <code>UpdateTable</code> operation.</p>
     /// <p>To add a vector index, specify <code>IndexName</code>, <code>VectorAttribute</code>, <code>Dimensions</code>, <code>DistanceFunction</code>, and <code>Projection</code>. To remove a vector index, specify only the <code>IndexName</code>.</p>
-    pub fn set_vector_index_updates(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::VectorIndexUpdate>>) -> Self {
+    pub fn set_vector_index_updates(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::VectorIndexUpdate>>) -> Self {
         self.vector_index_updates = input;
         self
     }
     /// <p>A list of vector indexes to be added to or removed from the table. You can add or remove one vector index for each <code>UpdateTable</code> operation.</p>
     /// <p>To add a vector index, specify <code>IndexName</code>, <code>VectorAttribute</code>, <code>Dimensions</code>, <code>DistanceFunction</code>, and <code>Projection</code>. To remove a vector index, specify only the <code>IndexName</code>.</p>
-    pub fn get_vector_index_updates(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::VectorIndexUpdate>> {
+    pub fn get_vector_index_updates(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::VectorIndexUpdate>> {
         &self.vector_index_updates
     }
     /// Consumes the builder and constructs a [`UpdateTableInput`](crate::operation::update_table::UpdateTableInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::update_table::UpdateTableInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_table::UpdateTableInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::update_table::UpdateTableInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_table::UpdateTableInput {
             attribute_definitions: self.attribute_definitions,
             table_name: self.table_name,
             billing_mode: self.billing_mode,
