--- original
+++ modified
@@ -50,8 +50,8 @@
 }
 impl PutRolePolicyInput {
     /// Creates a new builder-style object to manufacture [`PutRolePolicyInput`](crate::operation::put_role_policy::PutRolePolicyInput).
-    pub fn builder() -> crate::operation::put_role_policy::builders::PutRolePolicyInputBuilder {
-        crate::operation::put_role_policy::builders::PutRolePolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_role_policy::builders::PutRolePolicyInputBuilder {
+        super::super::super::operation::put_role_policy::builders::PutRolePolicyInputBuilder::default()
     }
 }

@@ -148,8 +148,8 @@
     /// Consumes the builder and constructs a [`PutRolePolicyInput`](crate::operation::put_role_policy::PutRolePolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_role_policy::PutRolePolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::put_role_policy::PutRolePolicyInput {
+    ) -> ::std::result::Result<super::super::super::operation::put_role_policy::PutRolePolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::put_role_policy::PutRolePolicyInput {
             role_name: self.role_name,
             policy_name: self.policy_name,
             policy_document: self.policy_document,
