--- original
+++ modified
@@ -6,7 +6,7 @@
     /// <p>The global table name.</p>
     pub global_table_name: ::std::option::Option<::std::string::String>,
     /// <p>The Regions where the global table needs to be created.</p>
-    pub replication_group: ::std::option::Option<::std::vec::Vec<crate::types::Replica>>,
+    pub replication_group: ::std::option::Option<::std::vec::Vec<super::types::Replica>>,
 }
 impl CreateGlobalTableInput {
     /// <p>The global table name.</p>
@@ -16,14 +16,14 @@
     /// <p>The Regions where the global table needs to be created.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replication_group.is_none()`.
-    pub fn replication_group(&self) -> &[crate::types::Replica] {
+    pub fn replication_group(&self) -> &[super::types::Replica] {
         self.replication_group.as_deref().unwrap_or_default()
     }
 }
 impl CreateGlobalTableInput {
     /// Creates a new builder-style object to manufacture [`CreateGlobalTableInput`](crate::operation::create_global_table::CreateGlobalTableInput).
-    pub fn builder() -> crate::operation::create_global_table::builders::CreateGlobalTableInputBuilder {
-        crate::operation::create_global_table::builders::CreateGlobalTableInputBuilder::default()
+    pub fn builder() -> super::operation::create_global_table::builders::CreateGlobalTableInputBuilder {
+        super::operation::create_global_table::builders::CreateGlobalTableInputBuilder::default()
     }
 }

@@ -32,7 +32,7 @@
 #[non_exhaustive]
 pub struct CreateGlobalTableInputBuilder {
     pub(crate) global_table_name: ::std::option::Option<::std::string::String>,
-    pub(crate) replication_group: ::std::option::Option<::std::vec::Vec<crate::types::Replica>>,
+    pub(crate) replication_group: ::std::option::Option<::std::vec::Vec<super::types::Replica>>,
 }
 impl CreateGlobalTableInputBuilder {
     /// <p>The global table name.</p>
@@ -55,7 +55,7 @@
     /// To override the contents of this collection use [`set_replication_group`](Self::set_replication_group).
     ///
     /// <p>The Regions where the global table needs to be created.</p>
-    pub fn replication_group(mut self, input: crate::types::Replica) -> Self {
+    pub fn replication_group(mut self, input: super::types::Replica) -> Self {
         let mut v = self.replication_group.unwrap_or_default();
         v.push(input);
         self.replication_group = ::std::option::Option::Some(v);
@@ -62,19 +62,19 @@
         self
     }
     /// <p>The Regions where the global table needs to be created.</p>
-    pub fn set_replication_group(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Replica>>) -> Self {
+    pub fn set_replication_group(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::Replica>>) -> Self {
         self.replication_group = input;
         self
     }
     /// <p>The Regions where the global table needs to be created.</p>
-    pub fn get_replication_group(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Replica>> {
+    pub fn get_replication_group(&self) -> &::std::option::Option<::std::vec::Vec<super::types::Replica>> {
         &self.replication_group
     }
     /// Consumes the builder and constructs a [`CreateGlobalTableInput`](crate::operation::create_global_table::CreateGlobalTableInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_global_table::CreateGlobalTableInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_global_table::CreateGlobalTableInput {
+    ) -> ::std::result::Result<super::operation::create_global_table::CreateGlobalTableInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::create_global_table::CreateGlobalTableInput {
             global_table_name: self.global_table_name,
             replication_group: self.replication_group,
         })
