--- original
+++ modified
@@ -4,27 +4,27 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_object_acl::GetObjectAclOutput, crate::operation::get_object_acl::GetObjectAclError> {
+) -> std::result::Result<super::super::operation::get_object_acl::GetObjectAclOutput, super::super::operation::get_object_acl::GetObjectAclError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_object_acl::GetObjectAclError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::get_object_acl::GetObjectAclError::unhandled)?;
+    generic_builder = super::super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_object_acl::GetObjectAclError::unhandled(generic)),
+        None => return Err(super::super::operation::get_object_acl::GetObjectAclError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchKey" => crate::operation::get_object_acl::GetObjectAclError::NoSuchKey({
+        "NoSuchKey" => super::super::operation::get_object_acl::GetObjectAclError::NoSuchKey({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchKeyBuilder::default();
-                output = crate::protocol_serde::shape_no_such_key::de_no_such_key_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_object_acl::GetObjectAclError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchKeyBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_key::de_no_such_key_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_object_acl::GetObjectAclError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -33,7 +33,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_object_acl::GetObjectAclError::generic(generic),
+        _ => super::super::operation::get_object_acl::GetObjectAclError::generic(generic),
     })
 }

@@ -42,18 +42,18 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_object_acl::GetObjectAclOutput, crate::operation::get_object_acl::GetObjectAclError> {
+) -> std::result::Result<super::super::operation::get_object_acl::GetObjectAclOutput, super::super::operation::get_object_acl::GetObjectAclError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_object_acl::builders::GetObjectAclOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_object_acl::de_get_object_acl(_response_body, output)
-            .map_err(crate::operation::get_object_acl::GetObjectAclError::unhandled)?;
+        let mut output = super::super::operation::get_object_acl::builders::GetObjectAclOutputBuilder::default();
+        output = super::super::protocol_serde::shape_get_object_acl::de_get_object_acl(_response_body, output)
+            .map_err(super::super::operation::get_object_acl::GetObjectAclError::unhandled)?;
         output = output.set_request_charged(
-            crate::protocol_serde::shape_get_object_acl_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::get_object_acl::GetObjectAclError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
+            super::super::protocol_serde::shape_get_object_acl_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::super::operation::get_object_acl::GetObjectAclError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -60,7 +60,7 @@
 }

 pub fn ser_get_object_acl_headers(
-    input: &crate::operation::get_object_acl::GetObjectAclInput,
+    input: &super::super::operation::get_object_acl::GetObjectAclInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.request_payer {
@@ -91,8 +91,8 @@
 #[allow(unused_mut)]
 pub fn de_get_object_acl(
     inp: &[u8],
-    mut builder: crate::operation::get_object_acl::builders::GetObjectAclOutputBuilder,
-) -> std::result::Result<crate::operation::get_object_acl::builders::GetObjectAclOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::super::operation::get_object_acl::builders::GetObjectAclOutputBuilder,
+) -> std::result::Result<super::super::operation::get_object_acl::builders::GetObjectAclOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -111,7 +111,7 @@
             s if s.matches("Owner") /* Owner com.amazonaws.s3.synthetic#GetObjectAclOutput$Owner */ =>  {
                 let var_5 =
                     Some(
-                        crate::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -121,7 +121,7 @@
             s if s.matches("AccessControlList") /* Grants com.amazonaws.s3.synthetic#GetObjectAclOutput$Grants */ =>  {
                 let var_6 =
                     Some(
-                        crate::protocol_serde::shape_grants::de_grants(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_grants::de_grants(&mut tag, depth + 1)
                         ?
                     )
                 ;
