--- original
+++ modified
@@ -4,27 +4,27 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::head_bucket::HeadBucketOutput, crate::operation::head_bucket::HeadBucketError> {
+) -> std::result::Result<super::super::operation::head_bucket::HeadBucketOutput, super::super::operation::head_bucket::HeadBucketError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::head_bucket::HeadBucketError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::head_bucket::HeadBucketError::unhandled)?;
+    generic_builder = super::super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::head_bucket::HeadBucketError::unhandled(generic)),
+        None => return Err(super::super::operation::head_bucket::HeadBucketError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NotFound" => crate::operation::head_bucket::HeadBucketError::NotFound({
+        "NotFound" => super::super::operation::head_bucket::HeadBucketError::NotFound({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundBuilder::default();
-                output = crate::protocol_serde::shape_not_found::de_not_found_xml_err(_response_body, output)
-                    .map_err(crate::operation::head_bucket::HeadBucketError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotFoundBuilder::default();
+                output = super::super::protocol_serde::shape_not_found::de_not_found_xml_err(_response_body, output)
+                    .map_err(super::super::operation::head_bucket::HeadBucketError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -33,7 +33,7 @@
             }
             tmp
         }),
-        _ => crate::operation::head_bucket::HeadBucketError::generic(generic),
+        _ => super::super::operation::head_bucket::HeadBucketError::generic(generic),
     })
 }

@@ -42,39 +42,39 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::head_bucket::HeadBucketOutput, crate::operation::head_bucket::HeadBucketError> {
+) -> std::result::Result<super::super::operation::head_bucket::HeadBucketOutput, super::super::operation::head_bucket::HeadBucketError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::head_bucket::builders::HeadBucketOutputBuilder::default();
+        let mut output = super::super::operation::head_bucket::builders::HeadBucketOutputBuilder::default();
         output = output.set_access_point_alias(
-            crate::protocol_serde::shape_head_bucket_output::de_access_point_alias_header(_response_headers).map_err(|_| {
-                crate::operation::head_bucket::HeadBucketError::unhandled("Failed to parse AccessPointAlias from header `x-amz-access-point-alias")
+            super::super::protocol_serde::shape_head_bucket_output::de_access_point_alias_header(_response_headers).map_err(|_| {
+                super::super::operation::head_bucket::HeadBucketError::unhandled("Failed to parse AccessPointAlias from header `x-amz-access-point-alias")
             })?,
         );
         output = output.set_bucket_arn(
-            crate::protocol_serde::shape_head_bucket_output::de_bucket_arn_header(_response_headers)
-                .map_err(|_| crate::operation::head_bucket::HeadBucketError::unhandled("Failed to parse BucketArn from header `x-amz-bucket-arn"))?,
+            super::super::protocol_serde::shape_head_bucket_output::de_bucket_arn_header(_response_headers)
+                .map_err(|_| super::super::operation::head_bucket::HeadBucketError::unhandled("Failed to parse BucketArn from header `x-amz-bucket-arn"))?,
         );
         output = output.set_bucket_location_name(
-            crate::protocol_serde::shape_head_bucket_output::de_bucket_location_name_header(_response_headers).map_err(|_| {
-                crate::operation::head_bucket::HeadBucketError::unhandled(
+            super::super::protocol_serde::shape_head_bucket_output::de_bucket_location_name_header(_response_headers).map_err(|_| {
+                super::super::operation::head_bucket::HeadBucketError::unhandled(
                     "Failed to parse BucketLocationName from header `x-amz-bucket-location-name",
                 )
             })?,
         );
         output = output.set_bucket_location_type(
-            crate::protocol_serde::shape_head_bucket_output::de_bucket_location_type_header(_response_headers).map_err(|_| {
-                crate::operation::head_bucket::HeadBucketError::unhandled(
+            super::super::protocol_serde::shape_head_bucket_output::de_bucket_location_type_header(_response_headers).map_err(|_| {
+                super::super::operation::head_bucket::HeadBucketError::unhandled(
                     "Failed to parse BucketLocationType from header `x-amz-bucket-location-type",
                 )
             })?,
         );
         output = output.set_bucket_region(
-            crate::protocol_serde::shape_head_bucket_output::de_bucket_region_header(_response_headers).map_err(|_| {
-                crate::operation::head_bucket::HeadBucketError::unhandled("Failed to parse BucketRegion from header `x-amz-bucket-region")
+            super::super::protocol_serde::shape_head_bucket_output::de_bucket_region_header(_response_headers).map_err(|_| {
+                super::super::operation::head_bucket::HeadBucketError::unhandled("Failed to parse BucketRegion from header `x-amz-bucket-region")
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -81,7 +81,7 @@
 }

 pub fn ser_head_bucket_headers(
-    input: &crate::operation::head_bucket::HeadBucketInput,
+    input: &super::super::operation::head_bucket::HeadBucketInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.expected_bucket_owner {
