--- original
+++ modified
@@ -4,30 +4,30 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::restore_object::RestoreObjectOutput, crate::operation::restore_object::RestoreObjectError> {
+) -> std::result::Result<super::operation::restore_object::RestoreObjectOutput, super::operation::restore_object::RestoreObjectError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::restore_object::RestoreObjectError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::restore_object::RestoreObjectError::unhandled)?;
+    generic_builder = super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::restore_object::RestoreObjectError::unhandled(generic)),
+        None => return Err(super::operation::restore_object::RestoreObjectError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ObjectAlreadyInActiveTierError" => crate::operation::restore_object::RestoreObjectError::ObjectAlreadyInActiveTierError({
+        "ObjectAlreadyInActiveTierError" => super::operation::restore_object::RestoreObjectError::ObjectAlreadyInActiveTierError({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ObjectAlreadyInActiveTierErrorBuilder::default();
-                output = crate::protocol_serde::shape_object_already_in_active_tier_error::de_object_already_in_active_tier_error_xml_err(
+                let mut output = super::types::error::builders::ObjectAlreadyInActiveTierErrorBuilder::default();
+                output = super::protocol_serde::shape_object_already_in_active_tier_error::de_object_already_in_active_tier_error_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::restore_object::RestoreObjectError::unhandled)?;
+                .map_err(super::operation::restore_object::RestoreObjectError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -36,7 +36,7 @@
             }
             tmp
         }),
-        _ => crate::operation::restore_object::RestoreObjectError::generic(generic),
+        _ => super::operation::restore_object::RestoreObjectError::generic(generic),
     })
 }

@@ -45,23 +45,23 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::restore_object::RestoreObjectOutput, crate::operation::restore_object::RestoreObjectError> {
+) -> std::result::Result<super::operation::restore_object::RestoreObjectOutput, super::operation::restore_object::RestoreObjectError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::restore_object::builders::RestoreObjectOutputBuilder::default();
+        let mut output = super::operation::restore_object::builders::RestoreObjectOutputBuilder::default();
         output = output.set_request_charged(
-            crate::protocol_serde::shape_restore_object_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::restore_object::RestoreObjectError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
+            super::protocol_serde::shape_restore_object_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::operation::restore_object::RestoreObjectError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
             })?,
         );
         output = output.set_restore_output_path(
-            crate::protocol_serde::shape_restore_object_output::de_restore_output_path_header(_response_headers).map_err(|_| {
-                crate::operation::restore_object::RestoreObjectError::unhandled(
+            super::protocol_serde::shape_restore_object_output::de_restore_output_path_header(_response_headers).map_err(|_| {
+                super::operation::restore_object::RestoreObjectError::unhandled(
                     "Failed to parse RestoreOutputPath from header `x-amz-restore-output-path",
                 )
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -68,7 +68,7 @@
 }

 pub fn ser_restore_object_headers(
-    input: &crate::operation::restore_object::RestoreObjectInput,
+    input: &super::operation::restore_object::RestoreObjectInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.request_payer {
