--- original
+++ modified
@@ -3,18 +3,18 @@
 pub fn de_inventory_encryption(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::InventoryEncryption, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::InventoryEncryption, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::InventoryEncryption::builder();
+    let mut builder = super::types::InventoryEncryption::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("SSE-S3") /* SSES3 com.amazonaws.s3#InventoryEncryption$SSES3 */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_sses3::de_sses3(&mut tag, depth + 1)
+                        super::protocol_serde::shape_sses3::de_sses3(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -24,7 +24,7 @@
             s if s.matches("SSE-KMS") /* SSEKMS com.amazonaws.s3#InventoryEncryption$SSEKMS */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_ssekms::de_ssekms(&mut tag, depth + 1)
+                        super::protocol_serde::shape_ssekms::de_ssekms(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -38,7 +38,7 @@
 }

 pub fn ser_inventory_encryption(
-    input: &crate::types::InventoryEncryption,
+    input: &super::types::InventoryEncryption,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -48,7 +48,7 @@
     }
     if let Some(var_4) = &input.ssekms {
         let inner_writer = scope.start_el("SSE-KMS");
-        crate::protocol_serde::shape_ssekms::ser_ssekms(var_4, inner_writer)?
+        super::protocol_serde::shape_ssekms::ser_ssekms(var_4, inner_writer)?
     }
     scope.finish();
     Ok(())
