--- original
+++ modified
@@ -5,16 +5,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::write_get_object_response::WriteGetObjectResponseOutput,
-    crate::operation::write_get_object_response::WriteGetObjectResponseError,
+    super::super::operation::write_get_object_response::WriteGetObjectResponseOutput,
+    super::super::operation::write_get_object_response::WriteGetObjectResponseError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::write_get_object_response::WriteGetObjectResponseError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::write_get_object_response::WriteGetObjectResponseError::unhandled)?;
+    generic_builder = super::super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::write_get_object_response::WriteGetObjectResponseError::generic(generic))
+    Err(super::super::operation::write_get_object_response::WriteGetObjectResponseError::generic(generic))
 }

 #[allow(clippy::unnecessary_wraps)]
@@ -23,13 +23,13 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::write_get_object_response::WriteGetObjectResponseOutput,
-    crate::operation::write_get_object_response::WriteGetObjectResponseError,
+    super::super::operation::write_get_object_response::WriteGetObjectResponseOutput,
+    super::super::operation::write_get_object_response::WriteGetObjectResponseError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::write_get_object_response::builders::WriteGetObjectResponseOutputBuilder::default();
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        let mut output = super::super::operation::write_get_object_response::builders::WriteGetObjectResponseOutputBuilder::default();
+        output._set_extended_request_id(super::super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -36,7 +36,7 @@
 }

 pub fn ser_write_get_object_response_headers(
-    input: &crate::operation::write_get_object_response::WriteGetObjectResponseInput,
+    input: &super::super::operation::write_get_object_response::WriteGetObjectResponseInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.request_route {
