--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_restore_status(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::RestoreStatus, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::RestoreStatus, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::RestoreStatus::builder();
+    let mut builder = super::types::RestoreStatus::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("IsRestoreInProgress") /* IsRestoreInProgress com.amazonaws.s3#RestoreStatus$IsRestoreInProgress */ =>  {
