--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_metrics_configuration(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::MetricsConfiguration, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::MetricsConfiguration, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::MetricsConfiguration::builder();
+    let mut builder = super::types::MetricsConfiguration::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Id") /* Id com.amazonaws.s3#MetricsConfiguration$Id */ =>  {
@@ -27,7 +27,7 @@
             s if s.matches("Filter") /* Filter com.amazonaws.s3#MetricsConfiguration$Filter */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_metrics_filter::de_metrics_filter(&mut tag, depth + 1)
+                        super::protocol_serde::shape_metrics_filter::de_metrics_filter(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -37,13 +37,13 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::metrics_configuration_correct_errors(builder)
+    Ok(super::serde_util::metrics_configuration_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }

 pub fn ser_metrics_configuration(
-    input: &crate::types::MetricsConfiguration,
+    input: &super::types::MetricsConfiguration,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -54,7 +54,7 @@
     }
     if let Some(var_3) = &input.filter {
         let inner_writer = scope.start_el("Filter");
-        crate::protocol_serde::shape_metrics_filter::ser_metrics_filter(var_3, inner_writer)?
+        super::protocol_serde::shape_metrics_filter::ser_metrics_filter(var_3, inner_writer)?
     }
     scope.finish();
     Ok(())
