--- original
+++ modified
@@ -4,14 +4,14 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::upload_part_copy::UploadPartCopyOutput, crate::operation::upload_part_copy::UploadPartCopyError> {
+) -> std::result::Result<super::super::operation::upload_part_copy::UploadPartCopyOutput, super::super::operation::upload_part_copy::UploadPartCopyError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::upload_part_copy::UploadPartCopyError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::upload_part_copy::UploadPartCopyError::unhandled)?;
+    generic_builder = super::super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::upload_part_copy::UploadPartCopyError::generic(generic))
+    Err(super::super::operation::upload_part_copy::UploadPartCopyError::generic(generic))
 }

 #[allow(clippy::unnecessary_wraps)]
@@ -19,63 +19,63 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::upload_part_copy::UploadPartCopyOutput, crate::operation::upload_part_copy::UploadPartCopyError> {
+) -> std::result::Result<super::super::operation::upload_part_copy::UploadPartCopyOutput, super::super::operation::upload_part_copy::UploadPartCopyError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::upload_part_copy::builders::UploadPartCopyOutputBuilder::default();
+        let mut output = super::super::operation::upload_part_copy::builders::UploadPartCopyOutputBuilder::default();
         output = output.set_bucket_key_enabled(
-            crate::protocol_serde::shape_upload_part_copy_output::de_bucket_key_enabled_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_bucket_key_enabled_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse BucketKeyEnabled from header `x-amz-server-side-encryption-bucket-key-enabled",
                 )
             })?,
         );
-        output = output.set_copy_part_result(crate::protocol_serde::shape_upload_part_copy_output::de_copy_part_result_payload(
+        output = output.set_copy_part_result(super::super::protocol_serde::shape_upload_part_copy_output::de_copy_part_result_payload(
             _response_body,
         )?);
         output = output.set_copy_source_version_id(
-            crate::protocol_serde::shape_upload_part_copy_output::de_copy_source_version_id_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_copy_source_version_id_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse CopySourceVersionId from header `x-amz-copy-source-version-id",
                 )
             })?,
         );
         output = output.set_request_charged(
-            crate::protocol_serde::shape_upload_part_copy_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse RequestCharged from header `x-amz-request-charged",
                 )
             })?,
         );
         output = output.set_sse_customer_algorithm(
-            crate::protocol_serde::shape_upload_part_copy_output::de_sse_customer_algorithm_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_sse_customer_algorithm_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse SSECustomerAlgorithm from header `x-amz-server-side-encryption-customer-algorithm",
                 )
             })?,
         );
         output = output.set_sse_customer_key_md5(
-            crate::protocol_serde::shape_upload_part_copy_output::de_sse_customer_key_md5_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_sse_customer_key_md5_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse SSECustomerKeyMD5 from header `x-amz-server-side-encryption-customer-key-MD5",
                 )
             })?,
         );
         output = output.set_ssekms_key_id(
-            crate::protocol_serde::shape_upload_part_copy_output::de_ssekms_key_id_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_ssekms_key_id_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse SSEKMSKeyId from header `x-amz-server-side-encryption-aws-kms-key-id",
                 )
             })?,
         );
         output = output.set_server_side_encryption(
-            crate::protocol_serde::shape_upload_part_copy_output::de_server_side_encryption_header(_response_headers).map_err(|_| {
-                crate::operation::upload_part_copy::UploadPartCopyError::unhandled(
+            super::super::protocol_serde::shape_upload_part_copy_output::de_server_side_encryption_header(_response_headers).map_err(|_| {
+                super::super::operation::upload_part_copy::UploadPartCopyError::unhandled(
                     "Failed to parse ServerSideEncryption from header `x-amz-server-side-encryption",
                 )
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -82,7 +82,7 @@
 }

 pub fn ser_upload_part_copy_headers(
-    input: &crate::operation::upload_part_copy::UploadPartCopyInput,
+    input: &super::super::operation::upload_part_copy::UploadPartCopyInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.copy_source {
