--- original
+++ modified
@@ -5,19 +5,19 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_object_encryption::UpdateObjectEncryptionOutput,
-    crate::operation::update_object_encryption::UpdateObjectEncryptionError,
+    super::operation::update_object_encryption::UpdateObjectEncryptionOutput,
+    super::operation::update_object_encryption::UpdateObjectEncryptionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
+    generic_builder = super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled(
+            return Err(super::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled(
                 generic,
             ))
         }
@@ -25,13 +25,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AccessDenied" => crate::operation::update_object_encryption::UpdateObjectEncryptionError::AccessDenied({
+        "AccessDenied" => super::operation::update_object_encryption::UpdateObjectEncryptionError::AccessDenied({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AccessDeniedBuilder::default();
-                output = crate::protocol_serde::shape_access_denied::de_access_denied_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
+                let mut output = super::types::error::builders::AccessDeniedBuilder::default();
+                output = super::protocol_serde::shape_access_denied::de_access_denied_xml_err(_response_body, output)
+                    .map_err(super::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -40,13 +40,13 @@
             }
             tmp
         }),
-        "InvalidRequest" => crate::operation::update_object_encryption::UpdateObjectEncryptionError::InvalidRequest({
+        "InvalidRequest" => super::operation::update_object_encryption::UpdateObjectEncryptionError::InvalidRequest({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRequestBuilder::default();
-                output = crate::protocol_serde::shape_invalid_request::de_invalid_request_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidRequestBuilder::default();
+                output = super::protocol_serde::shape_invalid_request::de_invalid_request_xml_err(_response_body, output)
+                    .map_err(super::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -55,13 +55,13 @@
             }
             tmp
         }),
-        "NoSuchKey" => crate::operation::update_object_encryption::UpdateObjectEncryptionError::NoSuchKey({
+        "NoSuchKey" => super::operation::update_object_encryption::UpdateObjectEncryptionError::NoSuchKey({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchKeyBuilder::default();
-                output = crate::protocol_serde::shape_no_such_key::de_no_such_key_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchKeyBuilder::default();
+                output = super::protocol_serde::shape_no_such_key::de_no_such_key_xml_err(_response_body, output)
+                    .map_err(super::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,7 +70,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_object_encryption::UpdateObjectEncryptionError::generic(generic),
+        _ => super::operation::update_object_encryption::UpdateObjectEncryptionError::generic(generic),
     })
 }

@@ -80,20 +80,20 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_object_encryption::UpdateObjectEncryptionOutput,
-    crate::operation::update_object_encryption::UpdateObjectEncryptionError,
+    super::operation::update_object_encryption::UpdateObjectEncryptionOutput,
+    super::operation::update_object_encryption::UpdateObjectEncryptionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_object_encryption::builders::UpdateObjectEncryptionOutputBuilder::default();
+        let mut output = super::operation::update_object_encryption::builders::UpdateObjectEncryptionOutputBuilder::default();
         output = output.set_request_charged(
-            crate::protocol_serde::shape_update_object_encryption_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled(
+            super::protocol_serde::shape_update_object_encryption_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::operation::update_object_encryption::UpdateObjectEncryptionError::unhandled(
                     "Failed to parse RequestCharged from header `x-amz-request-charged",
                 )
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -100,7 +100,7 @@
 }

 pub fn ser_update_object_encryption_headers(
-    input: &crate::operation::update_object_encryption::UpdateObjectEncryptionInput,
+    input: &super::operation::update_object_encryption::UpdateObjectEncryptionInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.request_payer {
