--- original
+++ modified
@@ -3,21 +3,21 @@
 pub fn de_s3_key_filter(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::S3KeyFilter, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::S3KeyFilter, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::S3KeyFilter::builder();
+    let mut builder = super::types::S3KeyFilter::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("FilterRule") /* FilterRules com.amazonaws.s3#S3KeyFilter$FilterRules */ =>  {
                 let var_1 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::FilterRule>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::types::FilterRule>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_2 = builder.filter_rules.take().unwrap_or_default();
                             list_2.push(
-                                crate::protocol_serde::shape_filter_rule::de_filter_rule(&mut tag, depth + 1)
+                                super::protocol_serde::shape_filter_rule::de_filter_rule(&mut tag, depth + 1)
                                 ?
                             );
                             list_2
@@ -35,7 +35,7 @@
 }

 pub fn ser_s3_key_filter(
-    input: &crate::types::S3KeyFilter,
+    input: &super::types::S3KeyFilter,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -44,7 +44,7 @@
         for list_item_4 in var_3 {
             {
                 let inner_writer = scope.start_el("FilterRule");
-                crate::protocol_serde::shape_filter_rule::ser_filter_rule(list_item_4, inner_writer)?
+                super::protocol_serde::shape_filter_rule::ser_filter_rule(list_item_4, inner_writer)?
             }
         }
     }
