--- original
+++ modified
@@ -4,14 +4,14 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::delete_objects::DeleteObjectsOutput, crate::operation::delete_objects::DeleteObjectsError> {
+) -> std::result::Result<super::operation::delete_objects::DeleteObjectsOutput, super::operation::delete_objects::DeleteObjectsError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_objects::DeleteObjectsError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::delete_objects::DeleteObjectsError::unhandled)?;
+    generic_builder = super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::delete_objects::DeleteObjectsError::generic(generic))
+    Err(super::operation::delete_objects::DeleteObjectsError::generic(generic))
 }

 #[allow(clippy::unnecessary_wraps)]
@@ -19,18 +19,18 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::delete_objects::DeleteObjectsOutput, crate::operation::delete_objects::DeleteObjectsError> {
+) -> std::result::Result<super::operation::delete_objects::DeleteObjectsOutput, super::operation::delete_objects::DeleteObjectsError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_objects::builders::DeleteObjectsOutputBuilder::default();
-        output = crate::protocol_serde::shape_delete_objects::de_delete_objects(_response_body, output)
-            .map_err(crate::operation::delete_objects::DeleteObjectsError::unhandled)?;
+        let mut output = super::operation::delete_objects::builders::DeleteObjectsOutputBuilder::default();
+        output = super::protocol_serde::shape_delete_objects::de_delete_objects(_response_body, output)
+            .map_err(super::operation::delete_objects::DeleteObjectsError::unhandled)?;
         output = output.set_request_charged(
-            crate::protocol_serde::shape_delete_objects_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::delete_objects::DeleteObjectsError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
+            super::protocol_serde::shape_delete_objects_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::operation::delete_objects::DeleteObjectsError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -37,7 +37,7 @@
 }

 pub fn ser_delete_objects_headers(
-    input: &crate::operation::delete_objects::DeleteObjectsInput,
+    input: &super::operation::delete_objects::DeleteObjectsInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.mfa {
@@ -102,8 +102,8 @@
 #[allow(unused_mut)]
 pub fn de_delete_objects(
     inp: &[u8],
-    mut builder: crate::operation::delete_objects::builders::DeleteObjectsOutputBuilder,
-) -> std::result::Result<crate::operation::delete_objects::builders::DeleteObjectsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::operation::delete_objects::builders::DeleteObjectsOutputBuilder,
+) -> std::result::Result<super::operation::delete_objects::builders::DeleteObjectsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -122,10 +122,10 @@
             s if s.matches("Error") /* Errors com.amazonaws.s3.synthetic#DeleteObjectsOutput$Errors */ =>  {
                 let var_11 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::Error>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::types::Error>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_12 = builder.errors.take().unwrap_or_default();
                             list_12.push(
-                                crate::protocol_serde::shape_error::de_error(&mut tag, depth + 1)
+                                super::protocol_serde::shape_error::de_error(&mut tag, depth + 1)
                                 ?
                             );
                             list_12
@@ -139,10 +139,10 @@
             s if s.matches("Deleted") /* Deleted com.amazonaws.s3.synthetic#DeleteObjectsOutput$Deleted */ =>  {
                 let var_13 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::DeletedObject>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::types::DeletedObject>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_14 = builder.deleted.take().unwrap_or_default();
                             list_14.push(
-                                crate::protocol_serde::shape_deleted_object::de_deleted_object(&mut tag, depth + 1)
+                                super::protocol_serde::shape_deleted_object::de_deleted_object(&mut tag, depth + 1)
                                 ?
                             );
                             list_14
