--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_intelligent_tiering_filter(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::IntelligentTieringFilter, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::IntelligentTieringFilter, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::IntelligentTieringFilter::builder();
+    let mut builder = super::super::types::IntelligentTieringFilter::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Prefix") /* Prefix com.amazonaws.s3#IntelligentTieringFilter$Prefix */ =>  {
@@ -27,7 +27,7 @@
             s if s.matches("Tag") /* Tag com.amazonaws.s3#IntelligentTieringFilter$Tag */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_tag::de_tag(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_tag::de_tag(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -37,7 +37,7 @@
             s if s.matches("And") /* And com.amazonaws.s3#IntelligentTieringFilter$And */ =>  {
                 let var_3 =
                     Some(
-                        crate::protocol_serde::shape_intelligent_tiering_and_operator::de_intelligent_tiering_and_operator(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_intelligent_tiering_and_operator::de_intelligent_tiering_and_operator(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -51,7 +51,7 @@
 }

 pub fn ser_intelligent_tiering_filter(
-    input: &crate::types::IntelligentTieringFilter,
+    input: &super::super::types::IntelligentTieringFilter,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -62,11 +62,11 @@
     }
     if let Some(var_5) = &input.tag {
         let inner_writer = scope.start_el("Tag");
-        crate::protocol_serde::shape_tag::ser_tag(var_5, inner_writer)?
+        super::super::protocol_serde::shape_tag::ser_tag(var_5, inner_writer)?
     }
     if let Some(var_6) = &input.and {
         let inner_writer = scope.start_el("And");
-        crate::protocol_serde::shape_intelligent_tiering_and_operator::ser_intelligent_tiering_and_operator(var_6, inner_writer)?
+        super::super::protocol_serde::shape_intelligent_tiering_and_operator::ser_intelligent_tiering_and_operator(var_6, inner_writer)?
     }
     scope.finish();
     Ok(())
