--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_logging_enabled(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::LoggingEnabled, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::LoggingEnabled, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::LoggingEnabled::builder();
+    let mut builder = super::super::types::LoggingEnabled::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("TargetBucket") /* TargetBucket com.amazonaws.s3#LoggingEnabled$TargetBucket */ =>  {
@@ -27,7 +27,7 @@
             s if s.matches("TargetGrants") /* TargetGrants com.amazonaws.s3#LoggingEnabled$TargetGrants */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_target_grants::de_target_grants(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_target_grants::de_target_grants(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -50,7 +50,7 @@
             s if s.matches("TargetObjectKeyFormat") /* TargetObjectKeyFormat com.amazonaws.s3#LoggingEnabled$TargetObjectKeyFormat */ =>  {
                 let var_4 =
                     Some(
-                        crate::protocol_serde::shape_target_object_key_format::de_target_object_key_format(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_target_object_key_format::de_target_object_key_format(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -60,13 +60,13 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::logging_enabled_correct_errors(builder)
+    Ok(super::super::serde_util::logging_enabled_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }

 pub fn ser_logging_enabled(
-    input: &crate::types::LoggingEnabled,
+    input: &super::super::types::LoggingEnabled,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -80,7 +80,7 @@
         for list_item_6 in var_5 {
             {
                 let inner_writer = inner_writer.start_el("Grant");
-                crate::protocol_serde::shape_target_grant::ser_target_grant(list_item_6, inner_writer)?
+                super::super::protocol_serde::shape_target_grant::ser_target_grant(list_item_6, inner_writer)?
             }
         }
     }
@@ -90,7 +90,7 @@
     }
     if let Some(var_7) = &input.target_object_key_format {
         let inner_writer = scope.start_el("TargetObjectKeyFormat");
-        crate::protocol_serde::shape_target_object_key_format::ser_target_object_key_format(var_7, inner_writer)?
+        super::super::protocol_serde::shape_target_object_key_format::ser_target_object_key_format(var_7, inner_writer)?
     }
     scope.finish();
     Ok(())
