--- original
+++ modified
@@ -4,28 +4,28 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::copy_object::CopyObjectOutput, crate::operation::copy_object::CopyObjectError> {
+) -> std::result::Result<super::operation::copy_object::CopyObjectOutput, super::operation::copy_object::CopyObjectError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::copy_object::CopyObjectError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::copy_object::CopyObjectError::unhandled)?;
+    generic_builder = super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::copy_object::CopyObjectError::unhandled(generic)),
+        None => return Err(super::operation::copy_object::CopyObjectError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ObjectNotInActiveTierError" => crate::operation::copy_object::CopyObjectError::ObjectNotInActiveTierError({
+        "ObjectNotInActiveTierError" => super::operation::copy_object::CopyObjectError::ObjectNotInActiveTierError({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ObjectNotInActiveTierErrorBuilder::default();
+                let mut output = super::types::error::builders::ObjectNotInActiveTierErrorBuilder::default();
                 output =
-                    crate::protocol_serde::shape_object_not_in_active_tier_error::de_object_not_in_active_tier_error_xml_err(_response_body, output)
-                        .map_err(crate::operation::copy_object::CopyObjectError::unhandled)?;
+                    super::protocol_serde::shape_object_not_in_active_tier_error::de_object_not_in_active_tier_error_xml_err(_response_body, output)
+                        .map_err(super::operation::copy_object::CopyObjectError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -34,7 +34,7 @@
             }
             tmp
         }),
-        _ => crate::operation::copy_object::CopyObjectError::generic(generic),
+        _ => super::operation::copy_object::CopyObjectError::generic(generic),
     })
 }

@@ -43,76 +43,76 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::copy_object::CopyObjectOutput, crate::operation::copy_object::CopyObjectError> {
+) -> std::result::Result<super::operation::copy_object::CopyObjectOutput, super::operation::copy_object::CopyObjectError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::copy_object::builders::CopyObjectOutputBuilder::default();
+        let mut output = super::operation::copy_object::builders::CopyObjectOutputBuilder::default();
         output = output.set_bucket_key_enabled(
-            crate::protocol_serde::shape_copy_object_output::de_bucket_key_enabled_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_bucket_key_enabled_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse BucketKeyEnabled from header `x-amz-server-side-encryption-bucket-key-enabled",
                 )
             })?,
         );
-        output = output.set_copy_object_result(crate::protocol_serde::shape_copy_object_output::de_copy_object_result_payload(
+        output = output.set_copy_object_result(super::protocol_serde::shape_copy_object_output::de_copy_object_result_payload(
             _response_body,
         )?);
         output = output.set_copy_source_version_id(
-            crate::protocol_serde::shape_copy_object_output::de_copy_source_version_id_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_copy_source_version_id_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse CopySourceVersionId from header `x-amz-copy-source-version-id",
                 )
             })?,
         );
         output = output.set_expiration(
-            crate::protocol_serde::shape_copy_object_output::de_expiration_header(_response_headers)
-                .map_err(|_| crate::operation::copy_object::CopyObjectError::unhandled("Failed to parse Expiration from header `x-amz-expiration"))?,
+            super::protocol_serde::shape_copy_object_output::de_expiration_header(_response_headers)
+                .map_err(|_| super::operation::copy_object::CopyObjectError::unhandled("Failed to parse Expiration from header `x-amz-expiration"))?,
         );
         output = output.set_request_charged(
-            crate::protocol_serde::shape_copy_object_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
+            super::protocol_serde::shape_copy_object_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
             })?,
         );
         output = output.set_sse_customer_algorithm(
-            crate::protocol_serde::shape_copy_object_output::de_sse_customer_algorithm_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_sse_customer_algorithm_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse SSECustomerAlgorithm from header `x-amz-server-side-encryption-customer-algorithm",
                 )
             })?,
         );
         output = output.set_sse_customer_key_md5(
-            crate::protocol_serde::shape_copy_object_output::de_sse_customer_key_md5_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_sse_customer_key_md5_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse SSECustomerKeyMD5 from header `x-amz-server-side-encryption-customer-key-MD5",
                 )
             })?,
         );
         output = output.set_ssekms_encryption_context(
-            crate::protocol_serde::shape_copy_object_output::de_ssekms_encryption_context_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_ssekms_encryption_context_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse SSEKMSEncryptionContext from header `x-amz-server-side-encryption-context",
                 )
             })?,
         );
         output = output.set_ssekms_key_id(
-            crate::protocol_serde::shape_copy_object_output::de_ssekms_key_id_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_ssekms_key_id_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse SSEKMSKeyId from header `x-amz-server-side-encryption-aws-kms-key-id",
                 )
             })?,
         );
         output = output.set_server_side_encryption(
-            crate::protocol_serde::shape_copy_object_output::de_server_side_encryption_header(_response_headers).map_err(|_| {
-                crate::operation::copy_object::CopyObjectError::unhandled(
+            super::protocol_serde::shape_copy_object_output::de_server_side_encryption_header(_response_headers).map_err(|_| {
+                super::operation::copy_object::CopyObjectError::unhandled(
                     "Failed to parse ServerSideEncryption from header `x-amz-server-side-encryption",
                 )
             })?,
         );
         output = output.set_version_id(
-            crate::protocol_serde::shape_copy_object_output::de_version_id_header(_response_headers)
-                .map_err(|_| crate::operation::copy_object::CopyObjectError::unhandled("Failed to parse VersionId from header `x-amz-version-id"))?,
+            super::protocol_serde::shape_copy_object_output::de_version_id_header(_response_headers)
+                .map_err(|_| super::operation::copy_object::CopyObjectError::unhandled("Failed to parse VersionId from header `x-amz-version-id"))?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -119,7 +119,7 @@
 }

 pub fn ser_copy_object_headers(
-    input: &crate::operation::copy_object::CopyObjectInput,
+    input: &super::operation::copy_object::CopyObjectInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.acl {
