--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_object_version(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::ObjectVersion, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::ObjectVersion, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::ObjectVersion::builder();
+    let mut builder = super::super::types::ObjectVersion::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("ETag") /* ETag com.amazonaws.s3#ObjectVersion$ETag */ =>  {
@@ -27,11 +27,11 @@
             s if s.matches("ChecksumAlgorithm") /* ChecksumAlgorithm com.amazonaws.s3#ObjectVersion$ChecksumAlgorithm */ =>  {
                 let var_2 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::ChecksumAlgorithm>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::super::types::ChecksumAlgorithm>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_3 = builder.checksum_algorithm.take().unwrap_or_default();
                             list_3.push(
-                                Result::<crate::types::ChecksumAlgorithm, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                                    crate::types::ChecksumAlgorithm::from(
+                                Result::<super::super::types::ChecksumAlgorithm, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                                    super::super::types::ChecksumAlgorithm::from(
                                         ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                                     )
                                 )
@@ -48,8 +48,8 @@
             s if s.matches("ChecksumType") /* ChecksumType com.amazonaws.s3#ObjectVersion$ChecksumType */ =>  {
                 let var_4 =
                     Some(
-                        Result::<crate::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ChecksumType::from(
+                        Result::<super::super::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::ChecksumType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -77,8 +77,8 @@
             s if s.matches("StorageClass") /* StorageClass com.amazonaws.s3#ObjectVersion$StorageClass */ =>  {
                 let var_6 =
                     Some(
-                        Result::<crate::types::ObjectVersionStorageClass, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ObjectVersionStorageClass::from(
+                        Result::<super::super::types::ObjectVersionStorageClass, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::ObjectVersionStorageClass::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -146,7 +146,7 @@
             s if s.matches("Owner") /* Owner com.amazonaws.s3#ObjectVersion$Owner */ =>  {
                 let var_11 =
                     Some(
-                        crate::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -156,7 +156,7 @@
             s if s.matches("RestoreStatus") /* RestoreStatus com.amazonaws.s3#ObjectVersion$RestoreStatus */ =>  {
                 let var_12 =
                     Some(
-                        crate::protocol_serde::shape_restore_status::de_restore_status(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_restore_status::de_restore_status(&mut tag, depth + 1)
                         ?
                     )
                 ;
