--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_queue_configuration(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::QueueConfiguration, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::QueueConfiguration, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::QueueConfiguration::builder();
+    let mut builder = super::types::QueueConfiguration::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Id") /* Id com.amazonaws.s3#QueueConfiguration$Id */ =>  {
@@ -40,11 +40,11 @@
             s if s.matches("Event") /* Events com.amazonaws.s3#QueueConfiguration$Events */ =>  {
                 let var_3 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::Event>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::types::Event>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_4 = builder.events.take().unwrap_or_default();
                             list_4.push(
-                                Result::<crate::types::Event, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                                    crate::types::Event::from(
+                                Result::<super::types::Event, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                                    super::types::Event::from(
                                         ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                                     )
                                 )
@@ -61,7 +61,7 @@
             s if s.matches("Filter") /* Filter com.amazonaws.s3#QueueConfiguration$Filter */ =>  {
                 let var_5 =
                     Some(
-                        crate::protocol_serde::shape_notification_configuration_filter::de_notification_configuration_filter(&mut tag, depth + 1)
+                        super::protocol_serde::shape_notification_configuration_filter::de_notification_configuration_filter(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -71,13 +71,13 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::queue_configuration_correct_errors(builder)
+    Ok(super::serde_util::queue_configuration_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }

 pub fn ser_queue_configuration(
-    input: &crate::types::QueueConfiguration,
+    input: &super::types::QueueConfiguration,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -100,7 +100,7 @@
     }
     if let Some(var_8) = &input.filter {
         let inner_writer = scope.start_el("Filter");
-        crate::protocol_serde::shape_notification_configuration_filter::ser_notification_configuration_filter(var_8, inner_writer)?
+        super::protocol_serde::shape_notification_configuration_filter::ser_notification_configuration_filter(var_8, inner_writer)?
     }
     scope.finish();
     Ok(())
