--- original
+++ modified
@@ -2,11 +2,11 @@
 pub fn de_analytics_filter(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::AnalyticsFilter, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::AnalyticsFilter, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
-    let mut base: Option<crate::types::AnalyticsFilter> = None;
+    let mut base: Option<super::types::AnalyticsFilter> = None;
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Prefix") /* Prefix com.amazonaws.s3#AnalyticsFilter$Prefix */ =>  {
@@ -17,26 +17,26 @@
                     )
                     ?
                 ;
-                base = Some(crate::types::AnalyticsFilter::Prefix(tmp));
+                base = Some(super::types::AnalyticsFilter::Prefix(tmp));
             }
             ,
             s if s.matches("Tag") /* Tag com.amazonaws.s3#AnalyticsFilter$Tag */ =>  {
                 let tmp =
-                    crate::protocol_serde::shape_tag::de_tag(&mut tag, depth + 1)
+                    super::protocol_serde::shape_tag::de_tag(&mut tag, depth + 1)
                     ?
                 ;
-                base = Some(crate::types::AnalyticsFilter::Tag(tmp));
+                base = Some(super::types::AnalyticsFilter::Tag(tmp));
             }
             ,
             s if s.matches("And") /* And com.amazonaws.s3#AnalyticsFilter$And */ =>  {
                 let tmp =
-                    crate::protocol_serde::shape_analytics_and_operator::de_analytics_and_operator(&mut tag, depth + 1)
+                    super::protocol_serde::shape_analytics_and_operator::de_analytics_and_operator(&mut tag, depth + 1)
                     ?
                 ;
-                base = Some(crate::types::AnalyticsFilter::And(tmp));
+                base = Some(super::types::AnalyticsFilter::And(tmp));
             }
             ,
-            _unknown => base = Some(crate::types::AnalyticsFilter::Unknown),
+            _unknown => base = Some(super::types::AnalyticsFilter::Unknown),
         }
     }
     base.ok_or_else(|| ::aws_smithy_xml::decode::XmlDecodeError::custom("expected union, got nothing"))
@@ -43,24 +43,24 @@
 }

 pub fn ser_analytics_filter(
-    input: &crate::types::AnalyticsFilter,
+    input: &super::types::AnalyticsFilter,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     let mut scope_writer = writer.finish();
     match input {
-        crate::types::AnalyticsFilter::Prefix(inner) => {
+        super::types::AnalyticsFilter::Prefix(inner) => {
             let mut inner_writer = scope_writer.start_el("Prefix").finish();
             inner_writer.data(inner.as_str());
         }
-        crate::types::AnalyticsFilter::Tag(inner) => {
+        super::types::AnalyticsFilter::Tag(inner) => {
             let inner_writer = scope_writer.start_el("Tag");
-            crate::protocol_serde::shape_tag::ser_tag(inner, inner_writer)?
+            super::protocol_serde::shape_tag::ser_tag(inner, inner_writer)?
         }
-        crate::types::AnalyticsFilter::And(inner) => {
+        super::types::AnalyticsFilter::And(inner) => {
             let inner_writer = scope_writer.start_el("And");
-            crate::protocol_serde::shape_analytics_and_operator::ser_analytics_and_operator(inner, inner_writer)?
+            super::protocol_serde::shape_analytics_and_operator::ser_analytics_and_operator(inner, inner_writer)?
         }
-        crate::types::AnalyticsFilter::Unknown => {
+        super::types::AnalyticsFilter::Unknown => {
             return Err(::aws_smithy_types::error::operation::SerializationError::unknown_variant(
                 "AnalyticsFilter",
             ))
