--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_metrics_and_operator(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::MetricsAndOperator, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::MetricsAndOperator, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::MetricsAndOperator::builder();
+    let mut builder = super::super::types::MetricsAndOperator::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Prefix") /* Prefix com.amazonaws.s3#MetricsAndOperator$Prefix */ =>  {
@@ -27,10 +27,10 @@
             s if s.matches("Tag") /* Tags com.amazonaws.s3#MetricsAndOperator$Tags */ =>  {
                 let var_2 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::Tag>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::super::types::Tag>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_3 = builder.tags.take().unwrap_or_default();
                             list_3.push(
-                                crate::protocol_serde::shape_tag::de_tag(&mut tag, depth + 1)
+                                super::super::protocol_serde::shape_tag::de_tag(&mut tag, depth + 1)
                                 ?
                             );
                             list_3
@@ -61,7 +61,7 @@
 }

 pub fn ser_metrics_and_operator(
-    input: &crate::types::MetricsAndOperator,
+    input: &super::super::types::MetricsAndOperator,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -74,7 +74,7 @@
         for list_item_7 in var_6 {
             {
                 let inner_writer = scope.start_el("Tag");
-                crate::protocol_serde::shape_tag::ser_tag(list_item_7, inner_writer)?
+                super::super::protocol_serde::shape_tag::ser_tag(list_item_7, inner_writer)?
             }
         }
     }
