--- original
+++ modified
@@ -3,19 +3,19 @@
 pub fn de_inventory_schedule(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::InventorySchedule, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::InventorySchedule, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::InventorySchedule::builder();
+    let mut builder = super::super::types::InventorySchedule::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Frequency") /* Frequency com.amazonaws.s3#InventorySchedule$Frequency */ =>  {
                 let var_1 =
                     Some(
-                        Result::<crate::types::InventoryFrequency, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::InventoryFrequency::from(
+                        Result::<super::super::types::InventoryFrequency, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::InventoryFrequency::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -28,13 +28,13 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::inventory_schedule_correct_errors(builder)
+    Ok(super::super::serde_util::inventory_schedule_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }

 pub fn ser_inventory_schedule(
-    input: &crate::types::InventorySchedule,
+    input: &super::super::types::InventorySchedule,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
