--- original
+++ modified
@@ -3,18 +3,18 @@
 pub fn de_inventory_destination(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::InventoryDestination, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::InventoryDestination, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::InventoryDestination::builder();
+    let mut builder = super::super::types::InventoryDestination::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("S3BucketDestination") /* S3BucketDestination com.amazonaws.s3#InventoryDestination$S3BucketDestination */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_inventory_s3_bucket_destination::de_inventory_s3_bucket_destination(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_inventory_s3_bucket_destination::de_inventory_s3_bucket_destination(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -24,11 +24,11 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::inventory_destination_correct_errors(builder).build())
+    Ok(super::super::serde_util::inventory_destination_correct_errors(builder).build())
 }

 pub fn ser_inventory_destination(
-    input: &crate::types::InventoryDestination,
+    input: &super::super::types::InventoryDestination,
     writer: ::aws_smithy_xml::encode::ElWriter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
@@ -35,7 +35,7 @@
     let mut scope = writer.finish();
     if let Some(var_2) = &input.s3_bucket_destination {
         let inner_writer = scope.start_el("S3BucketDestination");
-        crate::protocol_serde::shape_inventory_s3_bucket_destination::ser_inventory_s3_bucket_destination(var_2, inner_writer)?
+        super::super::protocol_serde::shape_inventory_s3_bucket_destination::ser_inventory_s3_bucket_destination(var_2, inner_writer)?
     }
     scope.finish();
     Ok(())
