--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_object(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::Object, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::Object, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::Object::builder();
+    let mut builder = super::types::Object::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Key") /* Key com.amazonaws.s3#Object$Key */ =>  {
@@ -54,11 +54,11 @@
             s if s.matches("ChecksumAlgorithm") /* ChecksumAlgorithm com.amazonaws.s3#Object$ChecksumAlgorithm */ =>  {
                 let var_4 =
                     Some(
-                        Result::<::std::vec::Vec::<crate::types::ChecksumAlgorithm>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
+                        Result::<::std::vec::Vec::<super::types::ChecksumAlgorithm>, ::aws_smithy_xml::decode::XmlDecodeError>::Ok({
                             let mut list_5 = builder.checksum_algorithm.take().unwrap_or_default();
                             list_5.push(
-                                Result::<crate::types::ChecksumAlgorithm, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                                    crate::types::ChecksumAlgorithm::from(
+                                Result::<super::types::ChecksumAlgorithm, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                                    super::types::ChecksumAlgorithm::from(
                                         ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                                     )
                                 )
@@ -75,8 +75,8 @@
             s if s.matches("ChecksumType") /* ChecksumType com.amazonaws.s3#Object$ChecksumType */ =>  {
                 let var_6 =
                     Some(
-                        Result::<crate::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ChecksumType::from(
+                        Result::<super::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ChecksumType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -104,8 +104,8 @@
             s if s.matches("StorageClass") /* StorageClass com.amazonaws.s3#Object$StorageClass */ =>  {
                 let var_8 =
                     Some(
-                        Result::<crate::types::ObjectStorageClass, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ObjectStorageClass::from(
+                        Result::<super::types::ObjectStorageClass, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ObjectStorageClass::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -118,7 +118,7 @@
             s if s.matches("Owner") /* Owner com.amazonaws.s3#Object$Owner */ =>  {
                 let var_9 =
                     Some(
-                        crate::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
+                        super::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -128,7 +128,7 @@
             s if s.matches("RestoreStatus") /* RestoreStatus com.amazonaws.s3#Object$RestoreStatus */ =>  {
                 let var_10 =
                     Some(
-                        crate::protocol_serde::shape_restore_status::de_restore_status(&mut tag, depth + 1)
+                        super::protocol_serde::shape_restore_status::de_restore_status(&mut tag, depth + 1)
                         ?
                     )
                 ;
