--- original
+++ modified
@@ -5,16 +5,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::complete_multipart_upload::CompleteMultipartUploadOutput,
-    crate::operation::complete_multipart_upload::CompleteMultipartUploadError,
+    super::operation::complete_multipart_upload::CompleteMultipartUploadOutput,
+    super::operation::complete_multipart_upload::CompleteMultipartUploadError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled)?;
+    generic_builder = super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::complete_multipart_upload::CompleteMultipartUploadError::generic(
+    Err(super::operation::complete_multipart_upload::CompleteMultipartUploadError::generic(
         generic,
     ))
 }
@@ -25,57 +25,57 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::complete_multipart_upload::CompleteMultipartUploadOutput,
-    crate::operation::complete_multipart_upload::CompleteMultipartUploadError,
+    super::operation::complete_multipart_upload::CompleteMultipartUploadOutput,
+    super::operation::complete_multipart_upload::CompleteMultipartUploadError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::complete_multipart_upload::builders::CompleteMultipartUploadOutputBuilder::default();
-        output = crate::protocol_serde::shape_complete_multipart_upload::de_complete_multipart_upload(_response_body, output)
-            .map_err(crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled)?;
+        let mut output = super::operation::complete_multipart_upload::builders::CompleteMultipartUploadOutputBuilder::default();
+        output = super::protocol_serde::shape_complete_multipart_upload::de_complete_multipart_upload(_response_body, output)
+            .map_err(super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled)?;
         output = output.set_bucket_key_enabled(
-            crate::protocol_serde::shape_complete_multipart_upload_output::de_bucket_key_enabled_header(_response_headers).map_err(|_| {
-                crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
+            super::protocol_serde::shape_complete_multipart_upload_output::de_bucket_key_enabled_header(_response_headers).map_err(|_| {
+                super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
                     "Failed to parse BucketKeyEnabled from header `x-amz-server-side-encryption-bucket-key-enabled",
                 )
             })?,
         );
         output = output.set_expiration(
-            crate::protocol_serde::shape_complete_multipart_upload_output::de_expiration_header(_response_headers).map_err(|_| {
-                crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
+            super::protocol_serde::shape_complete_multipart_upload_output::de_expiration_header(_response_headers).map_err(|_| {
+                super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
                     "Failed to parse Expiration from header `x-amz-expiration",
                 )
             })?,
         );
         output = output.set_request_charged(
-            crate::protocol_serde::shape_complete_multipart_upload_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
+            super::protocol_serde::shape_complete_multipart_upload_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
                     "Failed to parse RequestCharged from header `x-amz-request-charged",
                 )
             })?,
         );
         output = output.set_ssekms_key_id(
-            crate::protocol_serde::shape_complete_multipart_upload_output::de_ssekms_key_id_header(_response_headers).map_err(|_| {
-                crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
+            super::protocol_serde::shape_complete_multipart_upload_output::de_ssekms_key_id_header(_response_headers).map_err(|_| {
+                super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
                     "Failed to parse SSEKMSKeyId from header `x-amz-server-side-encryption-aws-kms-key-id",
                 )
             })?,
         );
         output = output.set_server_side_encryption(
-            crate::protocol_serde::shape_complete_multipart_upload_output::de_server_side_encryption_header(_response_headers).map_err(|_| {
-                crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
+            super::protocol_serde::shape_complete_multipart_upload_output::de_server_side_encryption_header(_response_headers).map_err(|_| {
+                super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
                     "Failed to parse ServerSideEncryption from header `x-amz-server-side-encryption",
                 )
             })?,
         );
         output = output.set_version_id(
-            crate::protocol_serde::shape_complete_multipart_upload_output::de_version_id_header(_response_headers).map_err(|_| {
-                crate::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
+            super::protocol_serde::shape_complete_multipart_upload_output::de_version_id_header(_response_headers).map_err(|_| {
+                super::operation::complete_multipart_upload::CompleteMultipartUploadError::unhandled(
                     "Failed to parse VersionId from header `x-amz-version-id",
                 )
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -82,7 +82,7 @@
 }

 pub fn ser_complete_multipart_upload_headers(
-    input: &crate::operation::complete_multipart_upload::CompleteMultipartUploadInput,
+    input: &super::operation::complete_multipart_upload::CompleteMultipartUploadInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.checksum_crc32 {
@@ -301,9 +301,9 @@
 #[allow(unused_mut)]
 pub fn de_complete_multipart_upload(
     inp: &[u8],
-    mut builder: crate::operation::complete_multipart_upload::builders::CompleteMultipartUploadOutputBuilder,
+    mut builder: super::operation::complete_multipart_upload::builders::CompleteMultipartUploadOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::complete_multipart_upload::builders::CompleteMultipartUploadOutputBuilder,
+    super::operation::complete_multipart_upload::builders::CompleteMultipartUploadOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -363,8 +363,8 @@
             s if s.matches("ChecksumType") /* ChecksumType com.amazonaws.s3.synthetic#CompleteMultipartUploadOutput$ChecksumType */ =>  {
                 let var_42 =
                     Some(
-                        Result::<crate::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ChecksumType::from(
+                        Result::<super::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ChecksumType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
