--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_multipart_upload(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::MultipartUpload, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::MultipartUpload, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::MultipartUpload::builder();
+    let mut builder = super::types::MultipartUpload::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("UploadId") /* UploadId com.amazonaws.s3#MultipartUpload$UploadId */ =>  {
@@ -54,8 +54,8 @@
             s if s.matches("StorageClass") /* StorageClass com.amazonaws.s3#MultipartUpload$StorageClass */ =>  {
                 let var_4 =
                     Some(
-                        Result::<crate::types::StorageClass, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::StorageClass::from(
+                        Result::<super::types::StorageClass, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::StorageClass::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -68,7 +68,7 @@
             s if s.matches("Owner") /* Owner com.amazonaws.s3#MultipartUpload$Owner */ =>  {
                 let var_5 =
                     Some(
-                        crate::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
+                        super::protocol_serde::shape_owner::de_owner(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -78,7 +78,7 @@
             s if s.matches("Initiator") /* Initiator com.amazonaws.s3#MultipartUpload$Initiator */ =>  {
                 let var_6 =
                     Some(
-                        crate::protocol_serde::shape_initiator::de_initiator(&mut tag, depth + 1)
+                        super::protocol_serde::shape_initiator::de_initiator(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -88,8 +88,8 @@
             s if s.matches("ChecksumAlgorithm") /* ChecksumAlgorithm com.amazonaws.s3#MultipartUpload$ChecksumAlgorithm */ =>  {
                 let var_7 =
                     Some(
-                        Result::<crate::types::ChecksumAlgorithm, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ChecksumAlgorithm::from(
+                        Result::<super::types::ChecksumAlgorithm, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ChecksumAlgorithm::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -102,8 +102,8 @@
             s if s.matches("ChecksumType") /* ChecksumType com.amazonaws.s3#MultipartUpload$ChecksumType */ =>  {
                 let var_8 =
                     Some(
-                        Result::<crate::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ChecksumType::from(
+                        Result::<super::types::ChecksumType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ChecksumType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
