--- original
+++ modified
@@ -4,14 +4,14 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::delete_object::DeleteObjectOutput, crate::operation::delete_object::DeleteObjectError> {
+) -> std::result::Result<super::operation::delete_object::DeleteObjectOutput, super::operation::delete_object::DeleteObjectError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_object::DeleteObjectError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::delete_object::DeleteObjectError::unhandled)?;
+    generic_builder = super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::delete_object::DeleteObjectError::generic(generic))
+    Err(super::operation::delete_object::DeleteObjectError::generic(generic))
 }

 #[allow(clippy::unnecessary_wraps)]
@@ -19,26 +19,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::delete_object::DeleteObjectOutput, crate::operation::delete_object::DeleteObjectError> {
+) -> std::result::Result<super::operation::delete_object::DeleteObjectOutput, super::operation::delete_object::DeleteObjectError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_object::builders::DeleteObjectOutputBuilder::default();
+        let mut output = super::operation::delete_object::builders::DeleteObjectOutputBuilder::default();
         output = output.set_delete_marker(
-            crate::protocol_serde::shape_delete_object_output::de_delete_marker_header(_response_headers).map_err(|_| {
-                crate::operation::delete_object::DeleteObjectError::unhandled("Failed to parse DeleteMarker from header `x-amz-delete-marker")
+            super::protocol_serde::shape_delete_object_output::de_delete_marker_header(_response_headers).map_err(|_| {
+                super::operation::delete_object::DeleteObjectError::unhandled("Failed to parse DeleteMarker from header `x-amz-delete-marker")
             })?,
         );
         output = output.set_request_charged(
-            crate::protocol_serde::shape_delete_object_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::delete_object::DeleteObjectError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
+            super::protocol_serde::shape_delete_object_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::operation::delete_object::DeleteObjectError::unhandled("Failed to parse RequestCharged from header `x-amz-request-charged")
             })?,
         );
         output = output.set_version_id(
-            crate::protocol_serde::shape_delete_object_output::de_version_id_header(_response_headers).map_err(|_| {
-                crate::operation::delete_object::DeleteObjectError::unhandled("Failed to parse VersionId from header `x-amz-version-id")
+            super::protocol_serde::shape_delete_object_output::de_version_id_header(_response_headers).map_err(|_| {
+                super::operation::delete_object::DeleteObjectError::unhandled("Failed to parse VersionId from header `x-amz-version-id")
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -45,7 +45,7 @@
 }

 pub fn ser_delete_object_headers(
-    input: &crate::operation::delete_object::DeleteObjectInput,
+    input: &super::operation::delete_object::DeleteObjectInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.mfa {
