--- original
+++ modified
@@ -5,16 +5,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_object_retention::PutObjectRetentionOutput,
-    crate::operation::put_object_retention::PutObjectRetentionError,
+    super::super::operation::put_object_retention::PutObjectRetentionOutput,
+    super::super::operation::put_object_retention::PutObjectRetentionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_object_retention::PutObjectRetentionError::unhandled)?;
-    generic_builder = crate::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::put_object_retention::PutObjectRetentionError::unhandled)?;
+    generic_builder = super::super::s3_request_id::apply_extended_request_id(generic_builder, _response_headers);
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::put_object_retention::PutObjectRetentionError::generic(generic))
+    Err(super::super::operation::put_object_retention::PutObjectRetentionError::generic(generic))
 }

 #[allow(clippy::unnecessary_wraps)]
@@ -23,20 +23,20 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_object_retention::PutObjectRetentionOutput,
-    crate::operation::put_object_retention::PutObjectRetentionError,
+    super::super::operation::put_object_retention::PutObjectRetentionOutput,
+    super::super::operation::put_object_retention::PutObjectRetentionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_object_retention::builders::PutObjectRetentionOutputBuilder::default();
+        let mut output = super::super::operation::put_object_retention::builders::PutObjectRetentionOutputBuilder::default();
         output = output.set_request_charged(
-            crate::protocol_serde::shape_put_object_retention_output::de_request_charged_header(_response_headers).map_err(|_| {
-                crate::operation::put_object_retention::PutObjectRetentionError::unhandled(
+            super::super::protocol_serde::shape_put_object_retention_output::de_request_charged_header(_response_headers).map_err(|_| {
+                super::super::operation::put_object_retention::PutObjectRetentionError::unhandled(
                     "Failed to parse RequestCharged from header `x-amz-request-charged",
                 )
             })?,
         );
-        output._set_extended_request_id(crate::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
+        output._set_extended_request_id(super::super::s3_request_id::RequestIdExt::extended_request_id(_response_headers).map(str::to_string));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -43,7 +43,7 @@
 }

 pub fn ser_put_object_retention_headers(
-    input: &crate::operation::put_object_retention::PutObjectRetentionInput,
+    input: &super::super::operation::put_object_retention::PutObjectRetentionInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.request_payer {
