--- original
+++ modified
@@ -22,8 +22,8 @@
 impl PutEmailIdentityConfigurationSetAttributesInput {
     /// Creates a new builder-style object to manufacture [`PutEmailIdentityConfigurationSetAttributesInput`](crate::operation::put_email_identity_configuration_set_attributes::PutEmailIdentityConfigurationSetAttributesInput).
     pub fn builder(
-    ) -> crate::operation::put_email_identity_configuration_set_attributes::builders::PutEmailIdentityConfigurationSetAttributesInputBuilder {
-        crate::operation::put_email_identity_configuration_set_attributes::builders::PutEmailIdentityConfigurationSetAttributesInputBuilder::default()
+    ) -> super::operation::put_email_identity_configuration_set_attributes::builders::PutEmailIdentityConfigurationSetAttributesInputBuilder {
+        super::operation::put_email_identity_configuration_set_attributes::builders::PutEmailIdentityConfigurationSetAttributesInputBuilder::default()
     }
 }

@@ -68,11 +68,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_email_identity_configuration_set_attributes::PutEmailIdentityConfigurationSetAttributesInput,
+        super::operation::put_email_identity_configuration_set_attributes::PutEmailIdentityConfigurationSetAttributesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_email_identity_configuration_set_attributes::PutEmailIdentityConfigurationSetAttributesInput {
+            super::operation::put_email_identity_configuration_set_attributes::PutEmailIdentityConfigurationSetAttributesInput {
                 email_identity: self.email_identity,
                 configuration_set_name: self.configuration_set_name,
             },
