--- original
+++ modified
@@ -12,9 +12,9 @@
     /// <p>A description of the created repository.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>A list of upstream repositories to associate with the repository. The order of the upstream repositories in the list determines their priority order when CodeArtifact looks for a requested package version. For more information, see <a href="https://docs.aws.amazon.com/codeartifact/latest/ug/repos-upstream.html">Working with upstream repositories</a>.</p>
-    pub upstreams: ::std::option::Option<::std::vec::Vec<crate::types::UpstreamRepository>>,
+    pub upstreams: ::std::option::Option<::std::vec::Vec<super::types::UpstreamRepository>>,
     /// <p>One or more tag key-value pairs for the repository.</p>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::types::Tag>>,
 }
 impl CreateRepositoryInput {
     /// <p>The name of the domain that contains the created repository.</p>
@@ -36,20 +36,20 @@
     /// <p>A list of upstream repositories to associate with the repository. The order of the upstream repositories in the list determines their priority order when CodeArtifact looks for a requested package version. For more information, see <a href="https://docs.aws.amazon.com/codeartifact/latest/ug/repos-upstream.html">Working with upstream repositories</a>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.upstreams.is_none()`.
-    pub fn upstreams(&self) -> &[crate::types::UpstreamRepository] {
+    pub fn upstreams(&self) -> &[super::types::UpstreamRepository] {
         self.upstreams.as_deref().unwrap_or_default()
     }
     /// <p>One or more tag key-value pairs for the repository.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
 }
 impl CreateRepositoryInput {
     /// Creates a new builder-style object to manufacture [`CreateRepositoryInput`](crate::operation::create_repository::CreateRepositoryInput).
-    pub fn builder() -> crate::operation::create_repository::builders::CreateRepositoryInputBuilder {
-        crate::operation::create_repository::builders::CreateRepositoryInputBuilder::default()
+    pub fn builder() -> super::operation::create_repository::builders::CreateRepositoryInputBuilder {
+        super::operation::create_repository::builders::CreateRepositoryInputBuilder::default()
     }
 }

@@ -61,8 +61,8 @@
     pub(crate) domain_owner: ::std::option::Option<::std::string::String>,
     pub(crate) repository: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) upstreams: ::std::option::Option<::std::vec::Vec<crate::types::UpstreamRepository>>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub(crate) upstreams: ::std::option::Option<::std::vec::Vec<super::types::UpstreamRepository>>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::types::Tag>>,
 }
 impl CreateRepositoryInputBuilder {
     /// <p>The name of the domain that contains the created repository.</p>
@@ -128,7 +128,7 @@
     /// To override the contents of this collection use [`set_upstreams`](Self::set_upstreams).
     ///
     /// <p>A list of upstream repositories to associate with the repository. The order of the upstream repositories in the list determines their priority order when CodeArtifact looks for a requested package version. For more information, see <a href="https://docs.aws.amazon.com/codeartifact/latest/ug/repos-upstream.html">Working with upstream repositories</a>.</p>
-    pub fn upstreams(mut self, input: crate::types::UpstreamRepository) -> Self {
+    pub fn upstreams(mut self, input: super::types::UpstreamRepository) -> Self {
         let mut v = self.upstreams.unwrap_or_default();
         v.push(input);
         self.upstreams = ::std::option::Option::Some(v);
@@ -135,12 +135,12 @@
         self
     }
     /// <p>A list of upstream repositories to associate with the repository. The order of the upstream repositories in the list determines their priority order when CodeArtifact looks for a requested package version. For more information, see <a href="https://docs.aws.amazon.com/codeartifact/latest/ug/repos-upstream.html">Working with upstream repositories</a>.</p>
-    pub fn set_upstreams(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::UpstreamRepository>>) -> Self {
+    pub fn set_upstreams(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::UpstreamRepository>>) -> Self {
         self.upstreams = input;
         self
     }
     /// <p>A list of upstream repositories to associate with the repository. The order of the upstream repositories in the list determines their priority order when CodeArtifact looks for a requested package version. For more information, see <a href="https://docs.aws.amazon.com/codeartifact/latest/ug/repos-upstream.html">Working with upstream repositories</a>.</p>
-    pub fn get_upstreams(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::UpstreamRepository>> {
+    pub fn get_upstreams(&self) -> &::std::option::Option<::std::vec::Vec<super::types::UpstreamRepository>> {
         &self.upstreams
     }
     /// Appends an item to `tags`.
@@ -148,7 +148,7 @@
     /// To override the contents of this collection use [`set_tags`](Self::set_tags).
     ///
     /// <p>One or more tag key-value pairs for the repository.</p>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -155,19 +155,19 @@
         self
     }
     /// <p>One or more tag key-value pairs for the repository.</p>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
     /// <p>One or more tag key-value pairs for the repository.</p>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::types::Tag>> {
         &self.tags
     }
     /// Consumes the builder and constructs a [`CreateRepositoryInput`](crate::operation::create_repository::CreateRepositoryInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_repository::CreateRepositoryInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_repository::CreateRepositoryInput {
+    ) -> ::std::result::Result<super::operation::create_repository::CreateRepositoryInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::create_repository::CreateRepositoryInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             repository: self.repository,
