--- original
+++ modified
@@ -9,11 +9,11 @@
     /// <p>When you include a <code>ClientId</code> parameter, Amazon Cognito maps the configuration to the app client. When you include both <code>ClientId</code> and <code>UserPoolId</code>, Amazon Cognito maps the configuration to the app client only.</p>
     pub client_id: ::std::option::Option<::std::string::String>,
     /// <p>The configuration of automated reactions to detected compromised credentials. Includes settings for blocking future sign-in requests and for the types of password-submission events you want to monitor.</p>
-    pub compromised_credentials_risk_configuration: ::std::option::Option<crate::types::CompromisedCredentialsRiskConfigurationType>,
+    pub compromised_credentials_risk_configuration: ::std::option::Option<super::types::CompromisedCredentialsRiskConfigurationType>,
     /// <p>The settings for automated responses and notification templates for adaptive authentication with threat protection.</p>
-    pub account_takeover_risk_configuration: ::std::option::Option<crate::types::AccountTakeoverRiskConfigurationType>,
+    pub account_takeover_risk_configuration: ::std::option::Option<super::types::AccountTakeoverRiskConfigurationType>,
     /// <p>A set of IP-address overrides to threat protection. You can set up IP-address always-block and always-allow lists.</p>
-    pub risk_exception_configuration: ::std::option::Option<crate::types::RiskExceptionConfigurationType>,
+    pub risk_exception_configuration: ::std::option::Option<super::types::RiskExceptionConfigurationType>,
 }
 impl SetRiskConfigurationInput {
     /// <p>The ID of the user pool where you want to set a risk configuration. If you include <code>UserPoolId</code> in your request, don't include <code>ClientId</code>. When the client ID is null, the same risk configuration is applied to all the clients in the userPool. When you include both <code>ClientId</code> and <code>UserPoolId</code>, Amazon Cognito maps the configuration to the app client only.</p>
@@ -26,15 +26,15 @@
         self.client_id.as_deref()
     }
     /// <p>The configuration of automated reactions to detected compromised credentials. Includes settings for blocking future sign-in requests and for the types of password-submission events you want to monitor.</p>
-    pub fn compromised_credentials_risk_configuration(&self) -> ::std::option::Option<&crate::types::CompromisedCredentialsRiskConfigurationType> {
+    pub fn compromised_credentials_risk_configuration(&self) -> ::std::option::Option<&super::types::CompromisedCredentialsRiskConfigurationType> {
         self.compromised_credentials_risk_configuration.as_ref()
     }
     /// <p>The settings for automated responses and notification templates for adaptive authentication with threat protection.</p>
-    pub fn account_takeover_risk_configuration(&self) -> ::std::option::Option<&crate::types::AccountTakeoverRiskConfigurationType> {
+    pub fn account_takeover_risk_configuration(&self) -> ::std::option::Option<&super::types::AccountTakeoverRiskConfigurationType> {
         self.account_takeover_risk_configuration.as_ref()
     }
     /// <p>A set of IP-address overrides to threat protection. You can set up IP-address always-block and always-allow lists.</p>
-    pub fn risk_exception_configuration(&self) -> ::std::option::Option<&crate::types::RiskExceptionConfigurationType> {
+    pub fn risk_exception_configuration(&self) -> ::std::option::Option<&super::types::RiskExceptionConfigurationType> {
         self.risk_exception_configuration.as_ref()
     }
 }
@@ -54,8 +54,8 @@
 }
 impl SetRiskConfigurationInput {
     /// Creates a new builder-style object to manufacture [`SetRiskConfigurationInput`](crate::operation::set_risk_configuration::SetRiskConfigurationInput).
-    pub fn builder() -> crate::operation::set_risk_configuration::builders::SetRiskConfigurationInputBuilder {
-        crate::operation::set_risk_configuration::builders::SetRiskConfigurationInputBuilder::default()
+    pub fn builder() -> super::operation::set_risk_configuration::builders::SetRiskConfigurationInputBuilder {
+        super::operation::set_risk_configuration::builders::SetRiskConfigurationInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
 pub struct SetRiskConfigurationInputBuilder {
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) client_id: ::std::option::Option<::std::string::String>,
-    pub(crate) compromised_credentials_risk_configuration: ::std::option::Option<crate::types::CompromisedCredentialsRiskConfigurationType>,
-    pub(crate) account_takeover_risk_configuration: ::std::option::Option<crate::types::AccountTakeoverRiskConfigurationType>,
-    pub(crate) risk_exception_configuration: ::std::option::Option<crate::types::RiskExceptionConfigurationType>,
+    pub(crate) compromised_credentials_risk_configuration: ::std::option::Option<super::types::CompromisedCredentialsRiskConfigurationType>,
+    pub(crate) account_takeover_risk_configuration: ::std::option::Option<super::types::AccountTakeoverRiskConfigurationType>,
+    pub(crate) risk_exception_configuration: ::std::option::Option<super::types::RiskExceptionConfigurationType>,
 }
 impl SetRiskConfigurationInputBuilder {
     /// <p>The ID of the user pool where you want to set a risk configuration. If you include <code>UserPoolId</code> in your request, don't include <code>ClientId</code>. When the client ID is null, the same risk configuration is applied to all the clients in the userPool. When you include both <code>ClientId</code> and <code>UserPoolId</code>, Amazon Cognito maps the configuration to the app client only.</p>
@@ -103,7 +103,7 @@
         &self.client_id
     }
     /// <p>The configuration of automated reactions to detected compromised credentials. Includes settings for blocking future sign-in requests and for the types of password-submission events you want to monitor.</p>
-    pub fn compromised_credentials_risk_configuration(mut self, input: crate::types::CompromisedCredentialsRiskConfigurationType) -> Self {
+    pub fn compromised_credentials_risk_configuration(mut self, input: super::types::CompromisedCredentialsRiskConfigurationType) -> Self {
         self.compromised_credentials_risk_configuration = ::std::option::Option::Some(input);
         self
     }
@@ -110,7 +110,7 @@
     /// <p>The configuration of automated reactions to detected compromised credentials. Includes settings for blocking future sign-in requests and for the types of password-submission events you want to monitor.</p>
     pub fn set_compromised_credentials_risk_configuration(
         mut self,
-        input: ::std::option::Option<crate::types::CompromisedCredentialsRiskConfigurationType>,
+        input: ::std::option::Option<super::types::CompromisedCredentialsRiskConfigurationType>,
     ) -> Self {
         self.compromised_credentials_risk_configuration = input;
         self
@@ -118,11 +118,11 @@
     /// <p>The configuration of automated reactions to detected compromised credentials. Includes settings for blocking future sign-in requests and for the types of password-submission events you want to monitor.</p>
     pub fn get_compromised_credentials_risk_configuration(
         &self,
-    ) -> &::std::option::Option<crate::types::CompromisedCredentialsRiskConfigurationType> {
+    ) -> &::std::option::Option<super::types::CompromisedCredentialsRiskConfigurationType> {
         &self.compromised_credentials_risk_configuration
     }
     /// <p>The settings for automated responses and notification templates for adaptive authentication with threat protection.</p>
-    pub fn account_takeover_risk_configuration(mut self, input: crate::types::AccountTakeoverRiskConfigurationType) -> Self {
+    pub fn account_takeover_risk_configuration(mut self, input: super::types::AccountTakeoverRiskConfigurationType) -> Self {
         self.account_takeover_risk_configuration = ::std::option::Option::Some(input);
         self
     }
@@ -129,35 +129,35 @@
     /// <p>The settings for automated responses and notification templates for adaptive authentication with threat protection.</p>
     pub fn set_account_takeover_risk_configuration(
         mut self,
-        input: ::std::option::Option<crate::types::AccountTakeoverRiskConfigurationType>,
+        input: ::std::option::Option<super::types::AccountTakeoverRiskConfigurationType>,
     ) -> Self {
         self.account_takeover_risk_configuration = input;
         self
     }
     /// <p>The settings for automated responses and notification templates for adaptive authentication with threat protection.</p>
-    pub fn get_account_takeover_risk_configuration(&self) -> &::std::option::Option<crate::types::AccountTakeoverRiskConfigurationType> {
+    pub fn get_account_takeover_risk_configuration(&self) -> &::std::option::Option<super::types::AccountTakeoverRiskConfigurationType> {
         &self.account_takeover_risk_configuration
     }
     /// <p>A set of IP-address overrides to threat protection. You can set up IP-address always-block and always-allow lists.</p>
-    pub fn risk_exception_configuration(mut self, input: crate::types::RiskExceptionConfigurationType) -> Self {
+    pub fn risk_exception_configuration(mut self, input: super::types::RiskExceptionConfigurationType) -> Self {
         self.risk_exception_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>A set of IP-address overrides to threat protection. You can set up IP-address always-block and always-allow lists.</p>
-    pub fn set_risk_exception_configuration(mut self, input: ::std::option::Option<crate::types::RiskExceptionConfigurationType>) -> Self {
+    pub fn set_risk_exception_configuration(mut self, input: ::std::option::Option<super::types::RiskExceptionConfigurationType>) -> Self {
         self.risk_exception_configuration = input;
         self
     }
     /// <p>A set of IP-address overrides to threat protection. You can set up IP-address always-block and always-allow lists.</p>
-    pub fn get_risk_exception_configuration(&self) -> &::std::option::Option<crate::types::RiskExceptionConfigurationType> {
+    pub fn get_risk_exception_configuration(&self) -> &::std::option::Option<super::types::RiskExceptionConfigurationType> {
         &self.risk_exception_configuration
     }
     /// Consumes the builder and constructs a [`SetRiskConfigurationInput`](crate::operation::set_risk_configuration::SetRiskConfigurationInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::set_risk_configuration::SetRiskConfigurationInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::set_risk_configuration::SetRiskConfigurationInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::set_risk_configuration::SetRiskConfigurationInput {
+        ::std::result::Result::Ok(super::operation::set_risk_configuration::SetRiskConfigurationInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
             compromised_credentials_risk_configuration: self.compromised_credentials_risk_configuration,
