--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct DescribeEndpointsInput {}
 impl DescribeEndpointsInput {
     /// Creates a new builder-style object to manufacture [`DescribeEndpointsInput`](crate::operation::describe_endpoints::DescribeEndpointsInput).
-    pub fn builder() -> crate::operation::describe_endpoints::builders::DescribeEndpointsInputBuilder {
-        crate::operation::describe_endpoints::builders::DescribeEndpointsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_endpoints::builders::DescribeEndpointsInputBuilder {
+        super::super::super::operation::describe_endpoints::builders::DescribeEndpointsInputBuilder::default()
     }
 }

@@ -18,7 +18,7 @@
     /// Consumes the builder and constructs a [`DescribeEndpointsInput`](crate::operation::describe_endpoints::DescribeEndpointsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_endpoints::DescribeEndpointsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::describe_endpoints::DescribeEndpointsInput {})
+    ) -> ::std::result::Result<super::super::super::operation::describe_endpoints::DescribeEndpointsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::describe_endpoints::DescribeEndpointsInput {})
     }
 }
