--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetCsvHeaderInput {
     /// Creates a new builder-style object to manufacture [`GetCsvHeaderInput`](crate::operation::get_csv_header::GetCsvHeaderInput).
-    pub fn builder() -> crate::operation::get_csv_header::builders::GetCsvHeaderInputBuilder {
-        crate::operation::get_csv_header::builders::GetCsvHeaderInputBuilder::default()
+    pub fn builder() -> super::operation::get_csv_header::builders::GetCsvHeaderInputBuilder {
+        super::operation::get_csv_header::builders::GetCsvHeaderInputBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`GetCsvHeaderInput`](crate::operation::get_csv_header::GetCsvHeaderInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_csv_header::GetCsvHeaderInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_csv_header::GetCsvHeaderInput {
+    ) -> ::std::result::Result<super::operation::get_csv_header::GetCsvHeaderInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::get_csv_header::GetCsvHeaderInput {
             user_pool_id: self.user_pool_id,
         })
     }
