--- original
+++ modified
@@ -117,8 +117,8 @@
 }
 impl ListUsersInput {
     /// Creates a new builder-style object to manufacture [`ListUsersInput`](crate::operation::list_users::ListUsersInput).
-    pub fn builder() -> crate::operation::list_users::builders::ListUsersInputBuilder {
-        crate::operation::list_users::builders::ListUsersInputBuilder::default()
+    pub fn builder() -> super::operation::list_users::builders::ListUsersInputBuilder {
+        super::operation::list_users::builders::ListUsersInputBuilder::default()
     }
 }

@@ -325,8 +325,8 @@
         &self.filter
     }
     /// Consumes the builder and constructs a [`ListUsersInput`](crate::operation::list_users::ListUsersInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_users::ListUsersInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_users::ListUsersInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_users::ListUsersInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_users::ListUsersInput {
             user_pool_id: self.user_pool_id,
             attributes_to_get: self.attributes_to_get,
             limit: self.limit,
