--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl GetQueueUrlInput {
     /// Creates a new builder-style object to manufacture [`GetQueueUrlInput`](crate::operation::get_queue_url::GetQueueUrlInput).
-    pub fn builder() -> crate::operation::get_queue_url::builders::GetQueueUrlInputBuilder {
-        crate::operation::get_queue_url::builders::GetQueueUrlInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_queue_url::builders::GetQueueUrlInputBuilder {
+        super::super::super::operation::get_queue_url::builders::GetQueueUrlInputBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
         &self.queue_owner_aws_account_id
     }
     /// Consumes the builder and constructs a [`GetQueueUrlInput`](crate::operation::get_queue_url::GetQueueUrlInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::get_queue_url::GetQueueUrlInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_queue_url::GetQueueUrlInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::get_queue_url::GetQueueUrlInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_queue_url::GetQueueUrlInput {
             queue_name: self.queue_name,
             queue_owner_aws_account_id: self.queue_owner_aws_account_id,
         })
