--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl DisassociateExternalConnectionInput {
     /// Creates a new builder-style object to manufacture [`DisassociateExternalConnectionInput`](crate::operation::disassociate_external_connection::DisassociateExternalConnectionInput).
-    pub fn builder() -> crate::operation::disassociate_external_connection::builders::DisassociateExternalConnectionInputBuilder {
-        crate::operation::disassociate_external_connection::builders::DisassociateExternalConnectionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::disassociate_external_connection::builders::DisassociateExternalConnectionInputBuilder {
+        super::super::super::operation::disassociate_external_connection::builders::DisassociateExternalConnectionInputBuilder::default()
     }
 }

@@ -110,10 +110,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::disassociate_external_connection::DisassociateExternalConnectionInput,
+        super::super::super::operation::disassociate_external_connection::DisassociateExternalConnectionInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::disassociate_external_connection::DisassociateExternalConnectionInput {
+        ::std::result::Result::Ok(super::super::super::operation::disassociate_external_connection::DisassociateExternalConnectionInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             repository: self.repository,
