--- original
+++ modified
@@ -6,13 +6,13 @@
     /// <p>Limits the returned queries to only those for the specified log group.</p>
     pub log_group_name: ::std::option::Option<::std::string::String>,
     /// <p>Limits the returned queries to only those that have the specified status. Valid values are <code>Cancelled</code>, <code>Complete</code>, <code>Failed</code>, <code>Running</code>, and <code>Scheduled</code>.</p>
-    pub status: ::std::option::Option<crate::types::QueryStatus>,
+    pub status: ::std::option::Option<super::types::QueryStatus>,
     /// <p>Limits the number of returned queries to the specified number.</p>
     pub max_results: ::std::option::Option<i32>,
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
     pub next_token: ::std::option::Option<::std::string::String>,
     /// <p>Limits the returned queries to only the queries that use the specified query language.</p>
-    pub query_language: ::std::option::Option<crate::types::QueryLanguage>,
+    pub query_language: ::std::option::Option<super::types::QueryLanguage>,
 }
 impl DescribeQueriesInput {
     /// <p>Limits the returned queries to only those for the specified log group.</p>
@@ -20,7 +20,7 @@
         self.log_group_name.as_deref()
     }
     /// <p>Limits the returned queries to only those that have the specified status. Valid values are <code>Cancelled</code>, <code>Complete</code>, <code>Failed</code>, <code>Running</code>, and <code>Scheduled</code>.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::QueryStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::QueryStatus> {
         self.status.as_ref()
     }
     /// <p>Limits the number of returned queries to the specified number.</p>
@@ -32,14 +32,14 @@
         self.next_token.as_deref()
     }
     /// <p>Limits the returned queries to only the queries that use the specified query language.</p>
-    pub fn query_language(&self) -> ::std::option::Option<&crate::types::QueryLanguage> {
+    pub fn query_language(&self) -> ::std::option::Option<&super::types::QueryLanguage> {
         self.query_language.as_ref()
     }
 }
 impl DescribeQueriesInput {
     /// Creates a new builder-style object to manufacture [`DescribeQueriesInput`](crate::operation::describe_queries::DescribeQueriesInput).
-    pub fn builder() -> crate::operation::describe_queries::builders::DescribeQueriesInputBuilder {
-        crate::operation::describe_queries::builders::DescribeQueriesInputBuilder::default()
+    pub fn builder() -> super::operation::describe_queries::builders::DescribeQueriesInputBuilder {
+        super::operation::describe_queries::builders::DescribeQueriesInputBuilder::default()
     }
 }

@@ -48,10 +48,10 @@
 #[non_exhaustive]
 pub struct DescribeQueriesInputBuilder {
     pub(crate) log_group_name: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::QueryStatus>,
+    pub(crate) status: ::std::option::Option<super::types::QueryStatus>,
     pub(crate) max_results: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
-    pub(crate) query_language: ::std::option::Option<crate::types::QueryLanguage>,
+    pub(crate) query_language: ::std::option::Option<super::types::QueryLanguage>,
 }
 impl DescribeQueriesInputBuilder {
     /// <p>Limits the returned queries to only those for the specified log group.</p>
@@ -69,17 +69,17 @@
         &self.log_group_name
     }
     /// <p>Limits the returned queries to only those that have the specified status. Valid values are <code>Cancelled</code>, <code>Complete</code>, <code>Failed</code>, <code>Running</code>, and <code>Scheduled</code>.</p>
-    pub fn status(mut self, input: crate::types::QueryStatus) -> Self {
+    pub fn status(mut self, input: super::types::QueryStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Limits the returned queries to only those that have the specified status. Valid values are <code>Cancelled</code>, <code>Complete</code>, <code>Failed</code>, <code>Running</code>, and <code>Scheduled</code>.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::QueryStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::QueryStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>Limits the returned queries to only those that have the specified status. Valid values are <code>Cancelled</code>, <code>Complete</code>, <code>Failed</code>, <code>Running</code>, and <code>Scheduled</code>.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::QueryStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::QueryStatus> {
         &self.status
     }
     /// <p>Limits the number of returned queries to the specified number.</p>
@@ -111,24 +111,24 @@
         &self.next_token
     }
     /// <p>Limits the returned queries to only the queries that use the specified query language.</p>
-    pub fn query_language(mut self, input: crate::types::QueryLanguage) -> Self {
+    pub fn query_language(mut self, input: super::types::QueryLanguage) -> Self {
         self.query_language = ::std::option::Option::Some(input);
         self
     }
     /// <p>Limits the returned queries to only the queries that use the specified query language.</p>
-    pub fn set_query_language(mut self, input: ::std::option::Option<crate::types::QueryLanguage>) -> Self {
+    pub fn set_query_language(mut self, input: ::std::option::Option<super::types::QueryLanguage>) -> Self {
         self.query_language = input;
         self
     }
     /// <p>Limits the returned queries to only the queries that use the specified query language.</p>
-    pub fn get_query_language(&self) -> &::std::option::Option<crate::types::QueryLanguage> {
+    pub fn get_query_language(&self) -> &::std::option::Option<super::types::QueryLanguage> {
         &self.query_language
     }
     /// Consumes the builder and constructs a [`DescribeQueriesInput`](crate::operation::describe_queries::DescribeQueriesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_queries::DescribeQueriesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::describe_queries::DescribeQueriesInput {
+    ) -> ::std::result::Result<super::operation::describe_queries::DescribeQueriesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::describe_queries::DescribeQueriesInput {
             log_group_name: self.log_group_name,
             status: self.status,
             max_results: self.max_results,
