--- original
+++ modified
@@ -22,11 +22,11 @@
     /// </note>
     pub delivery_s3_key_prefix: ::std::option::Option<::std::string::String>,
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+    pub conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<super::super::super::types::ConformancePackInputParameter>>,
     /// <p>An object of type <code>TemplateSSMDocumentDetails</code>, which contains the name or the Amazon Resource Name (ARN) of the Amazon Web Services Systems Manager document (SSM document) and the version of the SSM document that is used to create a conformance pack.</p>
-    pub template_ssm_document_details: ::std::option::Option<crate::types::TemplateSsmDocumentDetails>,
+    pub template_ssm_document_details: ::std::option::Option<super::super::super::types::TemplateSsmDocumentDetails>,
     /// <p>The tags for the conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>,
 }
 impl PutConformancePackInput {
     /// <p>The unique name of the conformance pack you want to deploy.</p>
@@ -60,24 +60,24 @@
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.conformance_pack_input_parameters.is_none()`.
-    pub fn conformance_pack_input_parameters(&self) -> &[crate::types::ConformancePackInputParameter] {
+    pub fn conformance_pack_input_parameters(&self) -> &[super::super::super::types::ConformancePackInputParameter] {
         self.conformance_pack_input_parameters.as_deref().unwrap_or_default()
     }
     /// <p>An object of type <code>TemplateSSMDocumentDetails</code>, which contains the name or the Amazon Resource Name (ARN) of the Amazon Web Services Systems Manager document (SSM document) and the version of the SSM document that is used to create a conformance pack.</p>
-    pub fn template_ssm_document_details(&self) -> ::std::option::Option<&crate::types::TemplateSsmDocumentDetails> {
+    pub fn template_ssm_document_details(&self) -> ::std::option::Option<&super::super::super::types::TemplateSsmDocumentDetails> {
         self.template_ssm_document_details.as_ref()
     }
     /// <p>The tags for the conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::super::super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
 }
 impl PutConformancePackInput {
     /// Creates a new builder-style object to manufacture [`PutConformancePackInput`](crate::operation::put_conformance_pack::PutConformancePackInput).
-    pub fn builder() -> crate::operation::put_conformance_pack::builders::PutConformancePackInputBuilder {
-        crate::operation::put_conformance_pack::builders::PutConformancePackInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_conformance_pack::builders::PutConformancePackInputBuilder {
+        super::super::super::operation::put_conformance_pack::builders::PutConformancePackInputBuilder::default()
     }
 }

@@ -90,9 +90,9 @@
     pub(crate) template_body: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_s3_bucket: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_s3_key_prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
-    pub(crate) template_ssm_document_details: ::std::option::Option<crate::types::TemplateSsmDocumentDetails>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub(crate) conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<super::super::super::types::ConformancePackInputParameter>>,
+    pub(crate) template_ssm_document_details: ::std::option::Option<super::super::super::types::TemplateSsmDocumentDetails>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>,
 }
 impl PutConformancePackInputBuilder {
     /// <p>The unique name of the conformance pack you want to deploy.</p>
@@ -195,7 +195,7 @@
     /// To override the contents of this collection use [`set_conformance_pack_input_parameters`](Self::set_conformance_pack_input_parameters).
     ///
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub fn conformance_pack_input_parameters(mut self, input: crate::types::ConformancePackInputParameter) -> Self {
+    pub fn conformance_pack_input_parameters(mut self, input: super::super::super::types::ConformancePackInputParameter) -> Self {
         let mut v = self.conformance_pack_input_parameters.unwrap_or_default();
         v.push(input);
         self.conformance_pack_input_parameters = ::std::option::Option::Some(v);
@@ -204,27 +204,27 @@
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
     pub fn set_conformance_pack_input_parameters(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::super::types::ConformancePackInputParameter>>,
     ) -> Self {
         self.conformance_pack_input_parameters = input;
         self
     }
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub fn get_conformance_pack_input_parameters(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>> {
+    pub fn get_conformance_pack_input_parameters(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::ConformancePackInputParameter>> {
         &self.conformance_pack_input_parameters
     }
     /// <p>An object of type <code>TemplateSSMDocumentDetails</code>, which contains the name or the Amazon Resource Name (ARN) of the Amazon Web Services Systems Manager document (SSM document) and the version of the SSM document that is used to create a conformance pack.</p>
-    pub fn template_ssm_document_details(mut self, input: crate::types::TemplateSsmDocumentDetails) -> Self {
+    pub fn template_ssm_document_details(mut self, input: super::super::super::types::TemplateSsmDocumentDetails) -> Self {
         self.template_ssm_document_details = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object of type <code>TemplateSSMDocumentDetails</code>, which contains the name or the Amazon Resource Name (ARN) of the Amazon Web Services Systems Manager document (SSM document) and the version of the SSM document that is used to create a conformance pack.</p>
-    pub fn set_template_ssm_document_details(mut self, input: ::std::option::Option<crate::types::TemplateSsmDocumentDetails>) -> Self {
+    pub fn set_template_ssm_document_details(mut self, input: ::std::option::Option<super::super::super::types::TemplateSsmDocumentDetails>) -> Self {
         self.template_ssm_document_details = input;
         self
     }
     /// <p>An object of type <code>TemplateSSMDocumentDetails</code>, which contains the name or the Amazon Resource Name (ARN) of the Amazon Web Services Systems Manager document (SSM document) and the version of the SSM document that is used to create a conformance pack.</p>
-    pub fn get_template_ssm_document_details(&self) -> &::std::option::Option<crate::types::TemplateSsmDocumentDetails> {
+    pub fn get_template_ssm_document_details(&self) -> &::std::option::Option<super::super::super::types::TemplateSsmDocumentDetails> {
         &self.template_ssm_document_details
     }
     /// Appends an item to `tags`.
@@ -232,7 +232,7 @@
     /// To override the contents of this collection use [`set_tags`](Self::set_tags).
     ///
     /// <p>The tags for the conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::super::super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -239,20 +239,20 @@
         self
     }
     /// <p>The tags for the conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
     /// <p>The tags for the conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>> {
         &self.tags
     }
     /// Consumes the builder and constructs a [`PutConformancePackInput`](crate::operation::put_conformance_pack::PutConformancePackInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_conformance_pack::PutConformancePackInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::put_conformance_pack::PutConformancePackInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::put_conformance_pack::PutConformancePackInput {
+        ::std::result::Result::Ok(super::super::super::operation::put_conformance_pack::PutConformancePackInput {
             conformance_pack_name: self.conformance_pack_name,
             template_s3_uri: self.template_s3_uri,
             template_body: self.template_body,
