--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl ScheduleKeyDeletionInput {
     /// Creates a new builder-style object to manufacture [`ScheduleKeyDeletionInput`](crate::operation::schedule_key_deletion::ScheduleKeyDeletionInput).
-    pub fn builder() -> crate::operation::schedule_key_deletion::builders::ScheduleKeyDeletionInputBuilder {
-        crate::operation::schedule_key_deletion::builders::ScheduleKeyDeletionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::schedule_key_deletion::builders::ScheduleKeyDeletionInputBuilder {
+        super::super::super::operation::schedule_key_deletion::builders::ScheduleKeyDeletionInputBuilder::default()
     }
 }

@@ -120,9 +120,9 @@
     /// Consumes the builder and constructs a [`ScheduleKeyDeletionInput`](crate::operation::schedule_key_deletion::ScheduleKeyDeletionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::schedule_key_deletion::ScheduleKeyDeletionInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::schedule_key_deletion::ScheduleKeyDeletionInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::schedule_key_deletion::ScheduleKeyDeletionInput {
+        ::std::result::Result::Ok(super::super::super::operation::schedule_key_deletion::ScheduleKeyDeletionInput {
             key_id: self.key_id,
             pending_window_in_days: self.pending_window_in_days,
         })
