--- original
+++ modified
@@ -8,12 +8,12 @@
     /// <p>The description of the version.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>The function layer archive.</p>
-    pub content: ::std::option::Option<crate::types::LayerVersionContentInput>,
+    pub content: ::std::option::Option<super::super::super::types::LayerVersionContentInput>,
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architectures</a>.</p>
-    pub compatible_architectures: ::std::option::Option<::std::vec::Vec<crate::types::Architecture>>,
+    pub compatible_architectures: ::std::option::Option<::std::vec::Vec<super::super::super::types::Architecture>>,
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html">function runtimes</a>. Used for filtering with <code>ListLayers</code> and <code>ListLayerVersions</code>.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy">Runtime deprecation policy</a>.</p>
-    pub compatible_runtimes: ::std::option::Option<::std::vec::Vec<crate::types::Runtime>>,
+    pub compatible_runtimes: ::std::option::Option<::std::vec::Vec<super::super::super::types::Runtime>>,
     /// <p>The layer's software license. It can be any of the following:</p>
     /// <ul>
     /// <li>
@@ -35,13 +35,13 @@
         self.description.as_deref()
     }
     /// <p>The function layer archive.</p>
-    pub fn content(&self) -> ::std::option::Option<&crate::types::LayerVersionContentInput> {
+    pub fn content(&self) -> ::std::option::Option<&super::super::super::types::LayerVersionContentInput> {
         self.content.as_ref()
     }
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architectures</a>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.compatible_architectures.is_none()`.
-    pub fn compatible_architectures(&self) -> &[crate::types::Architecture] {
+    pub fn compatible_architectures(&self) -> &[super::super::super::types::Architecture] {
         self.compatible_architectures.as_deref().unwrap_or_default()
     }
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html">function runtimes</a>. Used for filtering with <code>ListLayers</code> and <code>ListLayerVersions</code>.</p>
@@ -48,7 +48,7 @@
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy">Runtime deprecation policy</a>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.compatible_runtimes.is_none()`.
-    pub fn compatible_runtimes(&self) -> &[crate::types::Runtime] {
+    pub fn compatible_runtimes(&self) -> &[super::super::super::types::Runtime] {
         self.compatible_runtimes.as_deref().unwrap_or_default()
     }
     /// <p>The layer's software license. It can be any of the following:</p>
@@ -66,8 +66,8 @@
 }
 impl PublishLayerVersionInput {
     /// Creates a new builder-style object to manufacture [`PublishLayerVersionInput`](crate::operation::publish_layer_version::PublishLayerVersionInput).
-    pub fn builder() -> crate::operation::publish_layer_version::builders::PublishLayerVersionInputBuilder {
-        crate::operation::publish_layer_version::builders::PublishLayerVersionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::publish_layer_version::builders::PublishLayerVersionInputBuilder {
+        super::super::super::operation::publish_layer_version::builders::PublishLayerVersionInputBuilder::default()
     }
 }

@@ -77,9 +77,9 @@
 pub struct PublishLayerVersionInputBuilder {
     pub(crate) layer_name: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) content: ::std::option::Option<crate::types::LayerVersionContentInput>,
-    pub(crate) compatible_architectures: ::std::option::Option<::std::vec::Vec<crate::types::Architecture>>,
-    pub(crate) compatible_runtimes: ::std::option::Option<::std::vec::Vec<crate::types::Runtime>>,
+    pub(crate) content: ::std::option::Option<super::super::super::types::LayerVersionContentInput>,
+    pub(crate) compatible_architectures: ::std::option::Option<::std::vec::Vec<super::super::super::types::Architecture>>,
+    pub(crate) compatible_runtimes: ::std::option::Option<::std::vec::Vec<super::super::super::types::Runtime>>,
     pub(crate) license_info: ::std::option::Option<::std::string::String>,
 }
 impl PublishLayerVersionInputBuilder {
@@ -114,17 +114,17 @@
     }
     /// <p>The function layer archive.</p>
     /// This field is required.
-    pub fn content(mut self, input: crate::types::LayerVersionContentInput) -> Self {
+    pub fn content(mut self, input: super::super::super::types::LayerVersionContentInput) -> Self {
         self.content = ::std::option::Option::Some(input);
         self
     }
     /// <p>The function layer archive.</p>
-    pub fn set_content(mut self, input: ::std::option::Option<crate::types::LayerVersionContentInput>) -> Self {
+    pub fn set_content(mut self, input: ::std::option::Option<super::super::super::types::LayerVersionContentInput>) -> Self {
         self.content = input;
         self
     }
     /// <p>The function layer archive.</p>
-    pub fn get_content(&self) -> &::std::option::Option<crate::types::LayerVersionContentInput> {
+    pub fn get_content(&self) -> &::std::option::Option<super::super::super::types::LayerVersionContentInput> {
         &self.content
     }
     /// Appends an item to `compatible_architectures`.
@@ -132,7 +132,7 @@
     /// To override the contents of this collection use [`set_compatible_architectures`](Self::set_compatible_architectures).
     ///
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architectures</a>.</p>
-    pub fn compatible_architectures(mut self, input: crate::types::Architecture) -> Self {
+    pub fn compatible_architectures(mut self, input: super::super::super::types::Architecture) -> Self {
         let mut v = self.compatible_architectures.unwrap_or_default();
         v.push(input);
         self.compatible_architectures = ::std::option::Option::Some(v);
@@ -139,12 +139,12 @@
         self
     }
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architectures</a>.</p>
-    pub fn set_compatible_architectures(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Architecture>>) -> Self {
+    pub fn set_compatible_architectures(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::Architecture>>) -> Self {
         self.compatible_architectures = input;
         self
     }
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architectures</a>.</p>
-    pub fn get_compatible_architectures(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Architecture>> {
+    pub fn get_compatible_architectures(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::Architecture>> {
         &self.compatible_architectures
     }
     /// Appends an item to `compatible_runtimes`.
@@ -153,7 +153,7 @@
     ///
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html">function runtimes</a>. Used for filtering with <code>ListLayers</code> and <code>ListLayerVersions</code>.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy">Runtime deprecation policy</a>.</p>
-    pub fn compatible_runtimes(mut self, input: crate::types::Runtime) -> Self {
+    pub fn compatible_runtimes(mut self, input: super::super::super::types::Runtime) -> Self {
         let mut v = self.compatible_runtimes.unwrap_or_default();
         v.push(input);
         self.compatible_runtimes = ::std::option::Option::Some(v);
@@ -161,13 +161,13 @@
     }
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html">function runtimes</a>. Used for filtering with <code>ListLayers</code> and <code>ListLayerVersions</code>.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy">Runtime deprecation policy</a>.</p>
-    pub fn set_compatible_runtimes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Runtime>>) -> Self {
+    pub fn set_compatible_runtimes(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::Runtime>>) -> Self {
         self.compatible_runtimes = input;
         self
     }
     /// <p>A list of compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html">function runtimes</a>. Used for filtering with <code>ListLayers</code> and <code>ListLayerVersions</code>.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy">Runtime deprecation policy</a>.</p>
-    pub fn get_compatible_runtimes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Runtime>> {
+    pub fn get_compatible_runtimes(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::Runtime>> {
         &self.compatible_runtimes
     }
     /// <p>The layer's software license. It can be any of the following:</p>
@@ -211,9 +211,9 @@
     /// Consumes the builder and constructs a [`PublishLayerVersionInput`](crate::operation::publish_layer_version::PublishLayerVersionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::publish_layer_version::PublishLayerVersionInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::publish_layer_version::PublishLayerVersionInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::publish_layer_version::PublishLayerVersionInput {
+        ::std::result::Result::Ok(super::super::super::operation::publish_layer_version::PublishLayerVersionInput {
             layer_name: self.layer_name,
             description: self.description,
             content: self.content,
