--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetUserPoolMfaConfigInput {
     /// Creates a new builder-style object to manufacture [`GetUserPoolMfaConfigInput`](crate::operation::get_user_pool_mfa_config::GetUserPoolMfaConfigInput).
-    pub fn builder() -> crate::operation::get_user_pool_mfa_config::builders::GetUserPoolMfaConfigInputBuilder {
-        crate::operation::get_user_pool_mfa_config::builders::GetUserPoolMfaConfigInputBuilder::default()
+    pub fn builder() -> super::operation::get_user_pool_mfa_config::builders::GetUserPoolMfaConfigInputBuilder {
+        super::operation::get_user_pool_mfa_config::builders::GetUserPoolMfaConfigInputBuilder::default()
     }
 }

@@ -44,9 +44,9 @@
     /// Consumes the builder and constructs a [`GetUserPoolMfaConfigInput`](crate::operation::get_user_pool_mfa_config::GetUserPoolMfaConfigInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_user_pool_mfa_config::GetUserPoolMfaConfigInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_user_pool_mfa_config::GetUserPoolMfaConfigInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_user_pool_mfa_config::GetUserPoolMfaConfigInput {
+        ::std::result::Result::Ok(super::operation::get_user_pool_mfa_config::GetUserPoolMfaConfigInput {
             user_pool_id: self.user_pool_id,
         })
     }
