--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteScheduledQueryInput {
     /// Creates a new builder-style object to manufacture [`DeleteScheduledQueryInput`](crate::operation::delete_scheduled_query::DeleteScheduledQueryInput).
-    pub fn builder() -> crate::operation::delete_scheduled_query::builders::DeleteScheduledQueryInputBuilder {
-        crate::operation::delete_scheduled_query::builders::DeleteScheduledQueryInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_scheduled_query::builders::DeleteScheduledQueryInputBuilder {
+        super::super::super::operation::delete_scheduled_query::builders::DeleteScheduledQueryInputBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
     /// Consumes the builder and constructs a [`DeleteScheduledQueryInput`](crate::operation::delete_scheduled_query::DeleteScheduledQueryInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_scheduled_query::DeleteScheduledQueryInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_scheduled_query::DeleteScheduledQueryInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_scheduled_query::DeleteScheduledQueryInput { identifier: self.identifier })
+        ::std::result::Result::Ok(super::super::super::operation::delete_scheduled_query::DeleteScheduledQueryInput { identifier: self.identifier })
     }
 }
