--- original
+++ modified
@@ -6,7 +6,7 @@
     /// <p>The name of organization conformance pack for which you want status details for member accounts.</p>
     pub organization_conformance_pack_name: ::std::option::Option<::std::string::String>,
     /// <p>An <code>OrganizationResourceDetailedStatusFilters</code> object.</p>
-    pub filters: ::std::option::Option<crate::types::OrganizationResourceDetailedStatusFilters>,
+    pub filters: ::std::option::Option<super::super::super::types::OrganizationResourceDetailedStatusFilters>,
     /// <p>The maximum number of <code>OrganizationConformancePackDetailedStatuses</code> returned on each page. If you do not specify a number, Config uses the default. The default is 100.</p>
     pub limit: ::std::option::Option<i32>,
     /// <p>The nextToken string returned on a previous page that you use to get the next page of results in a paginated response.</p>
@@ -18,7 +18,7 @@
         self.organization_conformance_pack_name.as_deref()
     }
     /// <p>An <code>OrganizationResourceDetailedStatusFilters</code> object.</p>
-    pub fn filters(&self) -> ::std::option::Option<&crate::types::OrganizationResourceDetailedStatusFilters> {
+    pub fn filters(&self) -> ::std::option::Option<&super::super::super::types::OrganizationResourceDetailedStatusFilters> {
         self.filters.as_ref()
     }
     /// <p>The maximum number of <code>OrganizationConformancePackDetailedStatuses</code> returned on each page. If you do not specify a number, Config uses the default. The default is 100.</p>
@@ -33,8 +33,8 @@
 impl GetOrganizationConformancePackDetailedStatusInput {
     /// Creates a new builder-style object to manufacture [`GetOrganizationConformancePackDetailedStatusInput`](crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput).
     pub fn builder(
-    ) -> crate::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusInputBuilder {
-        crate::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusInputBuilder::default()
+    ) -> super::super::super::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusInputBuilder {
+        super::super::super::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusInputBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
 #[non_exhaustive]
 pub struct GetOrganizationConformancePackDetailedStatusInputBuilder {
     pub(crate) organization_conformance_pack_name: ::std::option::Option<::std::string::String>,
-    pub(crate) filters: ::std::option::Option<crate::types::OrganizationResourceDetailedStatusFilters>,
+    pub(crate) filters: ::std::option::Option<super::super::super::types::OrganizationResourceDetailedStatusFilters>,
     pub(crate) limit: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
 }
@@ -64,17 +64,17 @@
         &self.organization_conformance_pack_name
     }
     /// <p>An <code>OrganizationResourceDetailedStatusFilters</code> object.</p>
-    pub fn filters(mut self, input: crate::types::OrganizationResourceDetailedStatusFilters) -> Self {
+    pub fn filters(mut self, input: super::super::super::types::OrganizationResourceDetailedStatusFilters) -> Self {
         self.filters = ::std::option::Option::Some(input);
         self
     }
     /// <p>An <code>OrganizationResourceDetailedStatusFilters</code> object.</p>
-    pub fn set_filters(mut self, input: ::std::option::Option<crate::types::OrganizationResourceDetailedStatusFilters>) -> Self {
+    pub fn set_filters(mut self, input: ::std::option::Option<super::super::super::types::OrganizationResourceDetailedStatusFilters>) -> Self {
         self.filters = input;
         self
     }
     /// <p>An <code>OrganizationResourceDetailedStatusFilters</code> object.</p>
-    pub fn get_filters(&self) -> &::std::option::Option<crate::types::OrganizationResourceDetailedStatusFilters> {
+    pub fn get_filters(&self) -> &::std::option::Option<super::super::super::types::OrganizationResourceDetailedStatusFilters> {
         &self.filters
     }
     /// <p>The maximum number of <code>OrganizationConformancePackDetailedStatuses</code> returned on each page. If you do not specify a number, Config uses the default. The default is 100.</p>
@@ -109,11 +109,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput,
+        super::super::super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput {
+            super::super::super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput {
                 organization_conformance_pack_name: self.organization_conformance_pack_name,
                 filters: self.filters,
                 limit: self.limit,
