--- original
+++ modified
@@ -25,7 +25,7 @@
     /// <p>The ARN of an IAM role that grants CloudWatch Logs permissions to deliver ingested log events to the destination stream. You don't need to provide the ARN when you are working with a logical destination for cross-account delivery.</p>
     pub role_arn: ::std::option::Option<::std::string::String>,
     /// <p>The method used to distribute log data to the destination. By default, log data is grouped by log stream, but the grouping can be set to random for a more even distribution. This property is only applicable when the destination is an Amazon Kinesis data stream.</p>
-    pub distribution: ::std::option::Option<crate::types::Distribution>,
+    pub distribution: ::std::option::Option<super::super::super::types::Distribution>,
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
     /// <p>If the log group uses either a log-group level or account-level transformer, and you specify <code>true</code>, the subscription filter will be applied on the transformed version of the log events instead of the original ingested log events.</p>
     pub apply_on_transformed_logs: ::std::option::Option<bool>,
@@ -67,7 +67,7 @@
         self.role_arn.as_deref()
     }
     /// <p>The method used to distribute log data to the destination. By default, log data is grouped by log stream, but the grouping can be set to random for a more even distribution. This property is only applicable when the destination is an Amazon Kinesis data stream.</p>
-    pub fn distribution(&self) -> ::std::option::Option<&crate::types::Distribution> {
+    pub fn distribution(&self) -> ::std::option::Option<&super::super::super::types::Distribution> {
         self.distribution.as_ref()
     }
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
@@ -88,8 +88,8 @@
 }
 impl PutSubscriptionFilterInput {
     /// Creates a new builder-style object to manufacture [`PutSubscriptionFilterInput`](crate::operation::put_subscription_filter::PutSubscriptionFilterInput).
-    pub fn builder() -> crate::operation::put_subscription_filter::builders::PutSubscriptionFilterInputBuilder {
-        crate::operation::put_subscription_filter::builders::PutSubscriptionFilterInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_subscription_filter::builders::PutSubscriptionFilterInputBuilder {
+        super::super::super::operation::put_subscription_filter::builders::PutSubscriptionFilterInputBuilder::default()
     }
 }

@@ -102,7 +102,7 @@
     pub(crate) filter_pattern: ::std::option::Option<::std::string::String>,
     pub(crate) destination_arn: ::std::option::Option<::std::string::String>,
     pub(crate) role_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) distribution: ::std::option::Option<crate::types::Distribution>,
+    pub(crate) distribution: ::std::option::Option<super::super::super::types::Distribution>,
     pub(crate) apply_on_transformed_logs: ::std::option::Option<bool>,
     pub(crate) field_selection_criteria: ::std::option::Option<::std::string::String>,
     pub(crate) emit_system_fields: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -216,17 +216,17 @@
         &self.role_arn
     }
     /// <p>The method used to distribute log data to the destination. By default, log data is grouped by log stream, but the grouping can be set to random for a more even distribution. This property is only applicable when the destination is an Amazon Kinesis data stream.</p>
-    pub fn distribution(mut self, input: crate::types::Distribution) -> Self {
+    pub fn distribution(mut self, input: super::super::super::types::Distribution) -> Self {
         self.distribution = ::std::option::Option::Some(input);
         self
     }
     /// <p>The method used to distribute log data to the destination. By default, log data is grouped by log stream, but the grouping can be set to random for a more even distribution. This property is only applicable when the destination is an Amazon Kinesis data stream.</p>
-    pub fn set_distribution(mut self, input: ::std::option::Option<crate::types::Distribution>) -> Self {
+    pub fn set_distribution(mut self, input: ::std::option::Option<super::super::super::types::Distribution>) -> Self {
         self.distribution = input;
         self
     }
     /// <p>The method used to distribute log data to the destination. By default, log data is grouped by log stream, but the grouping can be set to random for a more even distribution. This property is only applicable when the destination is an Amazon Kinesis data stream.</p>
-    pub fn get_distribution(&self) -> &::std::option::Option<crate::types::Distribution> {
+    pub fn get_distribution(&self) -> &::std::option::Option<super::super::super::types::Distribution> {
         &self.distribution
     }
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
@@ -283,9 +283,9 @@
     /// Consumes the builder and constructs a [`PutSubscriptionFilterInput`](crate::operation::put_subscription_filter::PutSubscriptionFilterInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_subscription_filter::PutSubscriptionFilterInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::put_subscription_filter::PutSubscriptionFilterInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::put_subscription_filter::PutSubscriptionFilterInput {
+        ::std::result::Result::Ok(super::super::super::operation::put_subscription_filter::PutSubscriptionFilterInput {
             log_group_name: self.log_group_name,
             filter_name: self.filter_name,
             filter_pattern: self.filter_pattern,
