--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>This parameter allows (through its <a href="http://wikipedia.org/wiki/regex">regex pattern</a>) a string of characters that can consist of any upper or lowercased letter or digit.</p>
     pub access_key_id: ::std::option::Option<::std::string::String>,
     /// <p>The status you want to assign to the secret access key. <code>Active</code> means that the key can be used for programmatic calls to Amazon Web Services, while <code>Inactive</code> means that the key cannot be used.</p>
-    pub status: ::std::option::Option<crate::types::StatusType>,
+    pub status: ::std::option::Option<super::super::super::types::StatusType>,
 }
 impl UpdateAccessKeyInput {
     /// <p>The name of the user whose key you want to update.</p>
@@ -24,14 +24,14 @@
         self.access_key_id.as_deref()
     }
     /// <p>The status you want to assign to the secret access key. <code>Active</code> means that the key can be used for programmatic calls to Amazon Web Services, while <code>Inactive</code> means that the key cannot be used.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::StatusType> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::super::types::StatusType> {
         self.status.as_ref()
     }
 }
 impl UpdateAccessKeyInput {
     /// Creates a new builder-style object to manufacture [`UpdateAccessKeyInput`](crate::operation::update_access_key::UpdateAccessKeyInput).
-    pub fn builder() -> crate::operation::update_access_key::builders::UpdateAccessKeyInputBuilder {
-        crate::operation::update_access_key::builders::UpdateAccessKeyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_access_key::builders::UpdateAccessKeyInputBuilder {
+        super::super::super::operation::update_access_key::builders::UpdateAccessKeyInputBuilder::default()
     }
 }

@@ -41,7 +41,7 @@
 pub struct UpdateAccessKeyInputBuilder {
     pub(crate) user_name: ::std::option::Option<::std::string::String>,
     pub(crate) access_key_id: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::StatusType>,
+    pub(crate) status: ::std::option::Option<super::super::super::types::StatusType>,
 }
 impl UpdateAccessKeyInputBuilder {
     /// <p>The name of the user whose key you want to update.</p>
@@ -81,24 +81,24 @@
     }
     /// <p>The status you want to assign to the secret access key. <code>Active</code> means that the key can be used for programmatic calls to Amazon Web Services, while <code>Inactive</code> means that the key cannot be used.</p>
     /// This field is required.
-    pub fn status(mut self, input: crate::types::StatusType) -> Self {
+    pub fn status(mut self, input: super::super::super::types::StatusType) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The status you want to assign to the secret access key. <code>Active</code> means that the key can be used for programmatic calls to Amazon Web Services, while <code>Inactive</code> means that the key cannot be used.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::StatusType>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::super::types::StatusType>) -> Self {
         self.status = input;
         self
     }
     /// <p>The status you want to assign to the secret access key. <code>Active</code> means that the key can be used for programmatic calls to Amazon Web Services, while <code>Inactive</code> means that the key cannot be used.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::StatusType> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::super::types::StatusType> {
         &self.status
     }
     /// Consumes the builder and constructs a [`UpdateAccessKeyInput`](crate::operation::update_access_key::UpdateAccessKeyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_access_key::UpdateAccessKeyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_access_key::UpdateAccessKeyInput {
+    ) -> ::std::result::Result<super::super::super::operation::update_access_key::UpdateAccessKeyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_access_key::UpdateAccessKeyInput {
             user_name: self.user_name,
             access_key_id: self.access_key_id,
             status: self.status,
