--- original
+++ modified
@@ -19,11 +19,11 @@
     /// <p>For more information about sending authorization, see the <a href="https://docs.aws.amazon.com/ses/latest/DeveloperGuide/sending-authorization.html">Amazon SES Developer Guide</a>.</p>
     pub feedback_forwarding_email_address_identity_arn: ::std::option::Option<::std::string::String>,
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub default_email_tags: ::std::option::Option<::std::vec::Vec<crate::types::MessageTag>>,
+    pub default_email_tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::MessageTag>>,
     /// <p>An object that contains the body of the message. You can specify a template message.</p>
-    pub default_content: ::std::option::Option<crate::types::BulkEmailContent>,
+    pub default_content: ::std::option::Option<super::super::super::types::BulkEmailContent>,
     /// <p>The list of bulk email entry objects.</p>
-    pub bulk_email_entries: ::std::option::Option<::std::vec::Vec<crate::types::BulkEmailEntry>>,
+    pub bulk_email_entries: ::std::option::Option<::std::vec::Vec<super::super::super::types::BulkEmailEntry>>,
     /// <p>The name of the configuration set to use when sending the email.</p>
     pub configuration_set_name: ::std::option::Option<::std::string::String>,
     /// <p>The ID of the multi-region endpoint (global-endpoint).</p>
@@ -33,7 +33,7 @@
     /// </note>
     pub tenant_name: ::std::option::Option<::std::string::String>,
     /// <p>An object that overrides, for the messages in this request only, settings that would otherwise apply to them. The overrides apply to every message in the request. Each setting that you don't override keeps the value that already applies.</p>
-    pub configuration_overrides: ::std::option::Option<crate::types::ConfigurationOverrides>,
+    pub configuration_overrides: ::std::option::Option<super::super::super::types::ConfigurationOverrides>,
 }
 impl SendBulkEmailInput {
     /// <p>The email address to use as the "From" address for the email. The address that you specify has to be verified.</p>
@@ -65,17 +65,17 @@
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.default_email_tags.is_none()`.
-    pub fn default_email_tags(&self) -> &[crate::types::MessageTag] {
+    pub fn default_email_tags(&self) -> &[super::super::super::types::MessageTag] {
         self.default_email_tags.as_deref().unwrap_or_default()
     }
     /// <p>An object that contains the body of the message. You can specify a template message.</p>
-    pub fn default_content(&self) -> ::std::option::Option<&crate::types::BulkEmailContent> {
+    pub fn default_content(&self) -> ::std::option::Option<&super::super::super::types::BulkEmailContent> {
         self.default_content.as_ref()
     }
     /// <p>The list of bulk email entry objects.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.bulk_email_entries.is_none()`.
-    pub fn bulk_email_entries(&self) -> &[crate::types::BulkEmailEntry] {
+    pub fn bulk_email_entries(&self) -> &[super::super::super::types::BulkEmailEntry] {
         self.bulk_email_entries.as_deref().unwrap_or_default()
     }
     /// <p>The name of the configuration set to use when sending the email.</p>
@@ -93,14 +93,14 @@
         self.tenant_name.as_deref()
     }
     /// <p>An object that overrides, for the messages in this request only, settings that would otherwise apply to them. The overrides apply to every message in the request. Each setting that you don't override keeps the value that already applies.</p>
-    pub fn configuration_overrides(&self) -> ::std::option::Option<&crate::types::ConfigurationOverrides> {
+    pub fn configuration_overrides(&self) -> ::std::option::Option<&super::super::super::types::ConfigurationOverrides> {
         self.configuration_overrides.as_ref()
     }
 }
 impl SendBulkEmailInput {
     /// Creates a new builder-style object to manufacture [`SendBulkEmailInput`](crate::operation::send_bulk_email::SendBulkEmailInput).
-    pub fn builder() -> crate::operation::send_bulk_email::builders::SendBulkEmailInputBuilder {
-        crate::operation::send_bulk_email::builders::SendBulkEmailInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::send_bulk_email::builders::SendBulkEmailInputBuilder {
+        super::super::super::operation::send_bulk_email::builders::SendBulkEmailInputBuilder::default()
     }
 }

@@ -113,13 +113,13 @@
     pub(crate) reply_to_addresses: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) feedback_forwarding_email_address: ::std::option::Option<::std::string::String>,
     pub(crate) feedback_forwarding_email_address_identity_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) default_email_tags: ::std::option::Option<::std::vec::Vec<crate::types::MessageTag>>,
-    pub(crate) default_content: ::std::option::Option<crate::types::BulkEmailContent>,
-    pub(crate) bulk_email_entries: ::std::option::Option<::std::vec::Vec<crate::types::BulkEmailEntry>>,
+    pub(crate) default_email_tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::MessageTag>>,
+    pub(crate) default_content: ::std::option::Option<super::super::super::types::BulkEmailContent>,
+    pub(crate) bulk_email_entries: ::std::option::Option<::std::vec::Vec<super::super::super::types::BulkEmailEntry>>,
     pub(crate) configuration_set_name: ::std::option::Option<::std::string::String>,
     pub(crate) endpoint_id: ::std::option::Option<::std::string::String>,
     pub(crate) tenant_name: ::std::option::Option<::std::string::String>,
-    pub(crate) configuration_overrides: ::std::option::Option<crate::types::ConfigurationOverrides>,
+    pub(crate) configuration_overrides: ::std::option::Option<super::super::super::types::ConfigurationOverrides>,
 }
 impl SendBulkEmailInputBuilder {
     /// <p>The email address to use as the "From" address for the email. The address that you specify has to be verified.</p>
@@ -215,7 +215,7 @@
     /// To override the contents of this collection use [`set_default_email_tags`](Self::set_default_email_tags).
     ///
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub fn default_email_tags(mut self, input: crate::types::MessageTag) -> Self {
+    pub fn default_email_tags(mut self, input: super::super::super::types::MessageTag) -> Self {
         let mut v = self.default_email_tags.unwrap_or_default();
         v.push(input);
         self.default_email_tags = ::std::option::Option::Some(v);
@@ -222,27 +222,27 @@
         self
     }
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub fn set_default_email_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::MessageTag>>) -> Self {
+    pub fn set_default_email_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::MessageTag>>) -> Self {
         self.default_email_tags = input;
         self
     }
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub fn get_default_email_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::MessageTag>> {
+    pub fn get_default_email_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::MessageTag>> {
         &self.default_email_tags
     }
     /// <p>An object that contains the body of the message. You can specify a template message.</p>
     /// This field is required.
-    pub fn default_content(mut self, input: crate::types::BulkEmailContent) -> Self {
+    pub fn default_content(mut self, input: super::super::super::types::BulkEmailContent) -> Self {
         self.default_content = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains the body of the message. You can specify a template message.</p>
-    pub fn set_default_content(mut self, input: ::std::option::Option<crate::types::BulkEmailContent>) -> Self {
+    pub fn set_default_content(mut self, input: ::std::option::Option<super::super::super::types::BulkEmailContent>) -> Self {
         self.default_content = input;
         self
     }
     /// <p>An object that contains the body of the message. You can specify a template message.</p>
-    pub fn get_default_content(&self) -> &::std::option::Option<crate::types::BulkEmailContent> {
+    pub fn get_default_content(&self) -> &::std::option::Option<super::super::super::types::BulkEmailContent> {
         &self.default_content
     }
     /// Appends an item to `bulk_email_entries`.
@@ -250,7 +250,7 @@
     /// To override the contents of this collection use [`set_bulk_email_entries`](Self::set_bulk_email_entries).
     ///
     /// <p>The list of bulk email entry objects.</p>
-    pub fn bulk_email_entries(mut self, input: crate::types::BulkEmailEntry) -> Self {
+    pub fn bulk_email_entries(mut self, input: super::super::super::types::BulkEmailEntry) -> Self {
         let mut v = self.bulk_email_entries.unwrap_or_default();
         v.push(input);
         self.bulk_email_entries = ::std::option::Option::Some(v);
@@ -257,12 +257,12 @@
         self
     }
     /// <p>The list of bulk email entry objects.</p>
-    pub fn set_bulk_email_entries(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::BulkEmailEntry>>) -> Self {
+    pub fn set_bulk_email_entries(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::BulkEmailEntry>>) -> Self {
         self.bulk_email_entries = input;
         self
     }
     /// <p>The list of bulk email entry objects.</p>
-    pub fn get_bulk_email_entries(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::BulkEmailEntry>> {
+    pub fn get_bulk_email_entries(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::BulkEmailEntry>> {
         &self.bulk_email_entries
     }
     /// <p>The name of the configuration set to use when sending the email.</p>
@@ -314,24 +314,24 @@
         &self.tenant_name
     }
     /// <p>An object that overrides, for the messages in this request only, settings that would otherwise apply to them. The overrides apply to every message in the request. Each setting that you don't override keeps the value that already applies.</p>
-    pub fn configuration_overrides(mut self, input: crate::types::ConfigurationOverrides) -> Self {
+    pub fn configuration_overrides(mut self, input: super::super::super::types::ConfigurationOverrides) -> Self {
         self.configuration_overrides = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that overrides, for the messages in this request only, settings that would otherwise apply to them. The overrides apply to every message in the request. Each setting that you don't override keeps the value that already applies.</p>
-    pub fn set_configuration_overrides(mut self, input: ::std::option::Option<crate::types::ConfigurationOverrides>) -> Self {
+    pub fn set_configuration_overrides(mut self, input: ::std::option::Option<super::super::super::types::ConfigurationOverrides>) -> Self {
         self.configuration_overrides = input;
         self
     }
     /// <p>An object that overrides, for the messages in this request only, settings that would otherwise apply to them. The overrides apply to every message in the request. Each setting that you don't override keeps the value that already applies.</p>
-    pub fn get_configuration_overrides(&self) -> &::std::option::Option<crate::types::ConfigurationOverrides> {
+    pub fn get_configuration_overrides(&self) -> &::std::option::Option<super::super::super::types::ConfigurationOverrides> {
         &self.configuration_overrides
     }
     /// Consumes the builder and constructs a [`SendBulkEmailInput`](crate::operation::send_bulk_email::SendBulkEmailInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::send_bulk_email::SendBulkEmailInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::send_bulk_email::SendBulkEmailInput {
+    ) -> ::std::result::Result<super::super::super::operation::send_bulk_email::SendBulkEmailInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::send_bulk_email::SendBulkEmailInput {
             from_email_address: self.from_email_address,
             from_email_address_identity_arn: self.from_email_address_identity_arn,
             reply_to_addresses: self.reply_to_addresses,
