--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>The unique identifier, or device key, of the device that you want to update the status for.</p>
     pub device_key: ::std::option::Option<::std::string::String>,
     /// <p>The configuration of the device secret verifier.</p>
-    pub device_secret_verifier_config: ::std::option::Option<crate::types::DeviceSecretVerifierConfigType>,
+    pub device_secret_verifier_config: ::std::option::Option<super::types::DeviceSecretVerifierConfigType>,
     /// <p>A friendly name for the device, for example <code>MyMobilePhone</code>.</p>
     pub device_name: ::std::option::Option<::std::string::String>,
 }
@@ -23,7 +23,7 @@
         self.device_key.as_deref()
     }
     /// <p>The configuration of the device secret verifier.</p>
-    pub fn device_secret_verifier_config(&self) -> ::std::option::Option<&crate::types::DeviceSecretVerifierConfigType> {
+    pub fn device_secret_verifier_config(&self) -> ::std::option::Option<&super::types::DeviceSecretVerifierConfigType> {
         self.device_secret_verifier_config.as_ref()
     }
     /// <p>A friendly name for the device, for example <code>MyMobilePhone</code>.</p>
@@ -43,8 +43,8 @@
 }
 impl ConfirmDeviceInput {
     /// Creates a new builder-style object to manufacture [`ConfirmDeviceInput`](crate::operation::confirm_device::ConfirmDeviceInput).
-    pub fn builder() -> crate::operation::confirm_device::builders::ConfirmDeviceInputBuilder {
-        crate::operation::confirm_device::builders::ConfirmDeviceInputBuilder::default()
+    pub fn builder() -> super::operation::confirm_device::builders::ConfirmDeviceInputBuilder {
+        super::operation::confirm_device::builders::ConfirmDeviceInputBuilder::default()
     }
 }

@@ -54,7 +54,7 @@
 pub struct ConfirmDeviceInputBuilder {
     pub(crate) access_token: ::std::option::Option<::std::string::String>,
     pub(crate) device_key: ::std::option::Option<::std::string::String>,
-    pub(crate) device_secret_verifier_config: ::std::option::Option<crate::types::DeviceSecretVerifierConfigType>,
+    pub(crate) device_secret_verifier_config: ::std::option::Option<super::types::DeviceSecretVerifierConfigType>,
     pub(crate) device_name: ::std::option::Option<::std::string::String>,
 }
 impl ConfirmDeviceInputBuilder {
@@ -89,17 +89,17 @@
         &self.device_key
     }
     /// <p>The configuration of the device secret verifier.</p>
-    pub fn device_secret_verifier_config(mut self, input: crate::types::DeviceSecretVerifierConfigType) -> Self {
+    pub fn device_secret_verifier_config(mut self, input: super::types::DeviceSecretVerifierConfigType) -> Self {
         self.device_secret_verifier_config = ::std::option::Option::Some(input);
         self
     }
     /// <p>The configuration of the device secret verifier.</p>
-    pub fn set_device_secret_verifier_config(mut self, input: ::std::option::Option<crate::types::DeviceSecretVerifierConfigType>) -> Self {
+    pub fn set_device_secret_verifier_config(mut self, input: ::std::option::Option<super::types::DeviceSecretVerifierConfigType>) -> Self {
         self.device_secret_verifier_config = input;
         self
     }
     /// <p>The configuration of the device secret verifier.</p>
-    pub fn get_device_secret_verifier_config(&self) -> &::std::option::Option<crate::types::DeviceSecretVerifierConfigType> {
+    pub fn get_device_secret_verifier_config(&self) -> &::std::option::Option<super::types::DeviceSecretVerifierConfigType> {
         &self.device_secret_verifier_config
     }
     /// <p>A friendly name for the device, for example <code>MyMobilePhone</code>.</p>
@@ -119,8 +119,8 @@
     /// Consumes the builder and constructs a [`ConfirmDeviceInput`](crate::operation::confirm_device::ConfirmDeviceInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::confirm_device::ConfirmDeviceInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::confirm_device::ConfirmDeviceInput {
+    ) -> ::std::result::Result<super::operation::confirm_device::ConfirmDeviceInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::confirm_device::ConfirmDeviceInput {
             access_token: self.access_token,
             device_key: self.device_key,
             device_secret_verifier_config: self.device_secret_verifier_config,
