--- original
+++ modified
@@ -33,8 +33,8 @@
 }
 impl DeleteUserAttributesInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserAttributesInput`](crate::operation::delete_user_attributes::DeleteUserAttributesInput).
-    pub fn builder() -> crate::operation::delete_user_attributes::builders::DeleteUserAttributesInputBuilder {
-        crate::operation::delete_user_attributes::builders::DeleteUserAttributesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_user_attributes::builders::DeleteUserAttributesInputBuilder {
+        super::super::super::operation::delete_user_attributes::builders::DeleteUserAttributesInputBuilder::default()
     }
 }

@@ -87,9 +87,9 @@
     /// Consumes the builder and constructs a [`DeleteUserAttributesInput`](crate::operation::delete_user_attributes::DeleteUserAttributesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_user_attributes::DeleteUserAttributesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_user_attributes::DeleteUserAttributesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_user_attributes::DeleteUserAttributesInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_user_attributes::DeleteUserAttributesInput {
             user_attribute_names: self.user_attribute_names,
             access_token: self.access_token,
         })
