--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::collections::HashMap<crate::types::MessageSystemAttributeName, ::std::string::String>>,
+    Option<::std::collections::HashMap<super::super::types::MessageSystemAttributeName, ::std::string::String>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -23,7 +23,7 @@
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => {
-                        let key = key.to_unescaped().map(|u| crate::types::MessageSystemAttributeName::from(u.as_ref()))?;
+                        let key = key.to_unescaped().map(|u| super::super::types::MessageSystemAttributeName::from(u.as_ref()))?;
                         let value = ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                             .map(|s| s.to_unescaped().map(|u| u.into_owned()))
                             .transpose()?;
