--- original
+++ modified
@@ -31,8 +31,8 @@
 }
 impl CreateEmailIdentityPolicyInput {
     /// Creates a new builder-style object to manufacture [`CreateEmailIdentityPolicyInput`](crate::operation::create_email_identity_policy::CreateEmailIdentityPolicyInput).
-    pub fn builder() -> crate::operation::create_email_identity_policy::builders::CreateEmailIdentityPolicyInputBuilder {
-        crate::operation::create_email_identity_policy::builders::CreateEmailIdentityPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::create_email_identity_policy::builders::CreateEmailIdentityPolicyInputBuilder {
+        super::super::super::operation::create_email_identity_policy::builders::CreateEmailIdentityPolicyInputBuilder::default()
     }
 }

@@ -100,10 +100,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::create_email_identity_policy::CreateEmailIdentityPolicyInput,
+        super::super::super::operation::create_email_identity_policy::CreateEmailIdentityPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::create_email_identity_policy::CreateEmailIdentityPolicyInput {
+        ::std::result::Result::Ok(super::super::super::operation::create_email_identity_policy::CreateEmailIdentityPolicyInput {
             email_identity: self.email_identity,
             policy_name: self.policy_name,
             policy: self.policy,
