--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListGroupPoliciesInput {
     /// Creates a new builder-style object to manufacture [`ListGroupPoliciesInput`](crate::operation::list_group_policies::ListGroupPoliciesInput).
-    pub fn builder() -> crate::operation::list_group_policies::builders::ListGroupPoliciesInputBuilder {
-        crate::operation::list_group_policies::builders::ListGroupPoliciesInputBuilder::default()
+    pub fn builder() -> super::operation::list_group_policies::builders::ListGroupPoliciesInputBuilder {
+        super::operation::list_group_policies::builders::ListGroupPoliciesInputBuilder::default()
     }
 }

@@ -96,8 +96,8 @@
     /// Consumes the builder and constructs a [`ListGroupPoliciesInput`](crate::operation::list_group_policies::ListGroupPoliciesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_group_policies::ListGroupPoliciesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_group_policies::ListGroupPoliciesInput {
+    ) -> ::std::result::Result<super::operation::list_group_policies::ListGroupPoliciesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_group_policies::ListGroupPoliciesInput {
             group_name: self.group_name,
             marker: self.marker,
             max_items: self.max_items,
