--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetDedicatedIpInput {
     /// Creates a new builder-style object to manufacture [`GetDedicatedIpInput`](crate::operation::get_dedicated_ip::GetDedicatedIpInput).
-    pub fn builder() -> crate::operation::get_dedicated_ip::builders::GetDedicatedIpInputBuilder {
-        crate::operation::get_dedicated_ip::builders::GetDedicatedIpInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_dedicated_ip::builders::GetDedicatedIpInputBuilder {
+        super::super::super::operation::get_dedicated_ip::builders::GetDedicatedIpInputBuilder::default()
     }
 }

@@ -45,7 +45,7 @@
     /// Consumes the builder and constructs a [`GetDedicatedIpInput`](crate::operation::get_dedicated_ip::GetDedicatedIpInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_dedicated_ip::GetDedicatedIpInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_dedicated_ip::GetDedicatedIpInput { ip: self.ip })
+    ) -> ::std::result::Result<super::super::super::operation::get_dedicated_ip::GetDedicatedIpInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_dedicated_ip::GetDedicatedIpInput { ip: self.ip })
     }
 }
