--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::EventDestination>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::EventDestination>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::EventDestinationBuilder::default();
+            let mut builder = super::super::types::builders::EventDestinationBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -33,7 +33,7 @@
                                 builder = builder.set_enabled(::aws_smithy_json::deserialize::token::expect_bool_or_null(tokens.next())?);
                             }
                             "MatchingEventTypes" => {
-                                builder = builder.set_matching_event_types(crate::protocol_serde::shape_event_types::de_event_types(
+                                builder = builder.set_matching_event_types(super::super::protocol_serde::shape_event_types::de_event_types(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -41,7 +41,7 @@
                             }
                             "KinesisFirehoseDestination" => {
                                 builder = builder.set_kinesis_firehose_destination(
-                                    crate::protocol_serde::shape_kinesis_firehose_destination::de_kinesis_firehose_destination(
+                                    super::super::protocol_serde::shape_kinesis_firehose_destination::de_kinesis_firehose_destination(
                                         tokens,
                                         _value,
                                         depth + 1,
@@ -50,11 +50,11 @@
                             }
                             "CloudWatchDestination" => {
                                 builder = builder.set_cloud_watch_destination(
-                                    crate::protocol_serde::shape_cloud_watch_destination::de_cloud_watch_destination(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_cloud_watch_destination::de_cloud_watch_destination(tokens, _value, depth + 1)?,
                                 );
                             }
                             "SnsDestination" => {
-                                builder = builder.set_sns_destination(crate::protocol_serde::shape_sns_destination::de_sns_destination(
+                                builder = builder.set_sns_destination(super::super::protocol_serde::shape_sns_destination::de_sns_destination(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -62,12 +62,12 @@
                             }
                             "EventBridgeDestination" => {
                                 builder = builder.set_event_bridge_destination(
-                                    crate::protocol_serde::shape_event_bridge_destination::de_event_bridge_destination(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_event_bridge_destination::de_event_bridge_destination(tokens, _value, depth + 1)?,
                                 );
                             }
                             "PinpointDestination" => {
                                 builder = builder.set_pinpoint_destination(
-                                    crate::protocol_serde::shape_pinpoint_destination::de_pinpoint_destination(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_pinpoint_destination::de_pinpoint_destination(tokens, _value, depth + 1)?,
                                 );
                             }
                             _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
@@ -80,7 +80,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::event_destination_correct_errors(builder).build().map_err(
+            Ok(Some(super::super::serde_util::event_destination_correct_errors(builder).build().map_err(
                 |err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err),
             )?))
         }
