--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationOutput,
-    crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError,
+    super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationOutput,
+    super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled(generic)),
+        None => return Err(super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "BadRequestException" => crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::BadRequestException({
+        "BadRequestException" => super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::BadRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
+                let mut output = super::super::types::error::builders::BadRequestExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::NotFoundException({
+        "NotFoundException" => super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,7 +65,7 @@
             }
             tmp
         }),
-        _ => crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::generic(generic),
+        _ => super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError::generic(generic),
     })
 }

@@ -75,12 +75,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationOutput,
-    crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationError,
+    super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationOutput,
+    super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_suppressed_destination::builders::DeleteSuppressedDestinationOutputBuilder::default();
+        let mut output = super::super::operation::delete_suppressed_destination::builders::DeleteSuppressedDestinationOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
