--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesOutput,
-    crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError,
+    super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesOutput,
+    super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled(generic)),
+        None => return Err(super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "BadRequestException" => crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::BadRequestException({
+        "BadRequestException" => super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::BadRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
+                let mut output = super::types::error::builders::BadRequestExceptionBuilder::default();
+                output = super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::NotFoundException({
+        "NotFoundException" => super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
+                let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -51,14 +51,14 @@
             tmp
         }),
         "TooManyRequestsException" => {
-            crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::TooManyRequestsException({
+            super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::TooManyRequestsException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                    let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
+                        super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                            .map_err(super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -68,7 +68,7 @@
                 tmp
             })
         }
-        _ => crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::generic(generic),
+        _ => super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError::generic(generic),
     })
 }

@@ -78,12 +78,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesOutput,
-    crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError,
+    super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesOutput,
+    super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_dedicated_ip_warmup_attributes::builders::PutDedicatedIpWarmupAttributesOutputBuilder::default();
+        let mut output = super::operation::put_dedicated_ip_warmup_attributes::builders::PutDedicatedIpWarmupAttributesOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -90,11 +90,11 @@
 }

 pub fn ser_put_dedicated_ip_warmup_attributes_input(
-    input: &crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput,
+    input: &super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_dedicated_ip_warmup_attributes_input::ser_put_dedicated_ip_warmup_attributes_input_input(&mut object, input)?;
+    super::protocol_serde::shape_put_dedicated_ip_warmup_attributes_input::ser_put_dedicated_ip_warmup_attributes_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
