--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_suppressed_destinations::ListSuppressedDestinationsOutput,
-    crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError,
+    super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsOutput,
+    super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled(generic)),
+        None => return Err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "BadRequestException" => crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::BadRequestException({
+        "BadRequestException" => super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::BadRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::BadRequestExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InvalidNextTokenException" => crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::InvalidNextTokenException({
+        "InvalidNextTokenException" => super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::InvalidNextTokenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidNextTokenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_next_token_exception::de_invalid_next_token_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidNextTokenExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_next_token_exception::de_invalid_next_token_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::NotFoundException({
+        "NotFoundException" => super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,13 +65,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -80,7 +80,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::generic(generic),
+        _ => super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::generic(generic),
     })
 }

@@ -90,14 +90,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_suppressed_destinations::ListSuppressedDestinationsOutput,
-    crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError,
+    super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsOutput,
+    super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_suppressed_destinations::de_list_suppressed_destinations(_response_body, output)
-            .map_err(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
+        let mut output = super::super::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_suppressed_destinations::de_list_suppressed_destinations(_response_body, output)
+            .map_err(super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -105,12 +105,12 @@

 pub(crate) fn de_list_suppressed_destinations(
     _value: &[u8],
-    mut builder: crate::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsOutputBuilder,
+    mut builder: super::super::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsOutputBuilder,
+    super::super::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -128,7 +128,7 @@
                 }
                 "SuppressedDestinationSummaries" => {
                     builder = builder.set_suppressed_destination_summaries(
-                        crate::protocol_serde::shape_suppressed_destination_summaries::de_suppressed_destination_summaries(
+                        super::super::protocol_serde::shape_suppressed_destination_summaries::de_suppressed_destination_summaries(
                             tokens,
                             _value,
                             depth + 1,
