--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesOutput,
-    crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError,
+    super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesOutput,
+    super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled(generic))
+            return Err(super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled(generic))
         }
     };

@@ -23,13 +23,13 @@
     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "BadRequestException" => {
-            crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::BadRequestException({
+            super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::BadRequestException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output).map_err(
-                        crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled,
+                    let mut output = super::types::error::builders::BadRequestExceptionBuilder::default();
+                    output = super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output).map_err(
+                        super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled,
                     )?;
                     let output = output.meta(generic);
                     output.build()
@@ -41,13 +41,13 @@
             })
         }
         "NotFoundException" => {
-            crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::NotFoundException({
+            super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::NotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output).map_err(
-                        crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled,
+                    let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                    output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output).map_err(
+                        super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled,
                     )?;
                     let output = output.meta(generic);
                     output.build()
@@ -59,15 +59,15 @@
             })
         }
         "TooManyRequestsException" => {
-            crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::TooManyRequestsException({
+            super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::TooManyRequestsException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                    let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                        super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
                             .map_err(
-                                crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled,
+                                super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled,
                             )?;
                     let output = output.meta(generic);
                     output.build()
@@ -78,7 +78,7 @@
                 tmp
             })
         }
-        _ => crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::generic(generic),
+        _ => super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::generic(generic),
     })
 }

@@ -88,18 +88,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesOutput,
-    crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError,
+    super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesOutput,
+    super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesOutputBuilder::default();
-        output = crate::protocol_serde::shape_put_email_identity_dkim_signing_attributes::de_put_email_identity_dkim_signing_attributes(
+            super::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesOutputBuilder::default();
+        output = super::protocol_serde::shape_put_email_identity_dkim_signing_attributes::de_put_email_identity_dkim_signing_attributes(
             _response_body,
             output,
         )
-        .map_err(crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled)?;
+        .map_err(super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -106,11 +106,11 @@
 }

 pub fn ser_put_email_identity_dkim_signing_attributes_input(
-    input: &crate::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput,
+    input: &super::operation::put_email_identity_dkim_signing_attributes::PutEmailIdentityDkimSigningAttributesInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_email_identity_dkim_signing_attributes_input::ser_put_email_identity_dkim_signing_attributes_input_input(
+    super::protocol_serde::shape_put_email_identity_dkim_signing_attributes_input::ser_put_email_identity_dkim_signing_attributes_input_input(
         &mut object,
         input,
     )?;
@@ -120,12 +120,12 @@

 pub(crate) fn de_put_email_identity_dkim_signing_attributes(
     _value: &[u8],
-    mut builder: crate::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesOutputBuilder,
+    mut builder: super::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesOutputBuilder,
+    super::operation::put_email_identity_dkim_signing_attributes::builders::PutEmailIdentityDkimSigningAttributesOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -137,12 +137,12 @@
                 "DkimStatus" => {
                     builder = builder.set_dkim_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::DkimStatus::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::DkimStatus::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "DkimTokens" => {
-                    builder = builder.set_dkim_tokens(crate::protocol_serde::shape_dns_token_list::de_dns_token_list(tokens, _value, depth + 1)?);
+                    builder = builder.set_dkim_tokens(super::protocol_serde::shape_dns_token_list::de_dns_token_list(tokens, _value, depth + 1)?);
                 }
                 "SigningHostedZone" => {
                     builder = builder.set_signing_hosted_zone(
