--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ReputationEntity>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ReputationEntity>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ReputationEntityBuilder::default();
+            let mut builder = super::types::builders::ReputationEntityBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,7 +31,7 @@
                         "ReputationEntityType" => {
                             builder = builder.set_reputation_entity_type(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ReputationEntityType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::ReputationEntityType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -43,7 +43,7 @@
                             );
                         }
                         "CustomerManagedStatus" => {
-                            builder = builder.set_customer_managed_status(crate::protocol_serde::shape_status_record::de_status_record(
+                            builder = builder.set_customer_managed_status(super::protocol_serde::shape_status_record::de_status_record(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -50,7 +50,7 @@
                             )?);
                         }
                         "AwsSesManagedStatus" => {
-                            builder = builder.set_aws_ses_managed_status(crate::protocol_serde::shape_status_record::de_status_record(
+                            builder = builder.set_aws_ses_managed_status(super::protocol_serde::shape_status_record::de_status_record(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -59,7 +59,7 @@
                         "SendingStatusAggregate" => {
                             builder = builder.set_sending_status_aggregate(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::SendingStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::SendingStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -66,7 +66,7 @@
                         "ReputationImpact" => {
                             builder = builder.set_reputation_impact(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::RecommendationImpact::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::RecommendationImpact::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
