--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsOutput,
-    crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError,
+    super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsOutput,
+    super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled(
+            return Err(super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "BadRequestException" => crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::BadRequestException({
+        "BadRequestException" => super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::BadRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
+                let mut output = super::types::error::builders::BadRequestExceptionBuilder::default();
+                output = super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,7 +54,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::generic(generic),
+        _ => super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::generic(generic),
     })
 }

@@ -64,14 +64,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsOutput,
-    crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError,
+    super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsOutput,
+    super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_multi_region_endpoints::builders::ListMultiRegionEndpointsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_multi_region_endpoints::de_list_multi_region_endpoints(_response_body, output)
-            .map_err(crate::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
+        let mut output = super::operation::list_multi_region_endpoints::builders::ListMultiRegionEndpointsOutputBuilder::default();
+        output = super::protocol_serde::shape_list_multi_region_endpoints::de_list_multi_region_endpoints(_response_body, output)
+            .map_err(super::operation::list_multi_region_endpoints::ListMultiRegionEndpointsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -79,12 +79,12 @@

 pub(crate) fn de_list_multi_region_endpoints(
     _value: &[u8],
-    mut builder: crate::operation::list_multi_region_endpoints::builders::ListMultiRegionEndpointsOutputBuilder,
+    mut builder: super::operation::list_multi_region_endpoints::builders::ListMultiRegionEndpointsOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::list_multi_region_endpoints::builders::ListMultiRegionEndpointsOutputBuilder,
+    super::operation::list_multi_region_endpoints::builders::ListMultiRegionEndpointsOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -94,7 +94,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "MultiRegionEndpoints" => {
-                    builder = builder.set_multi_region_endpoints(crate::protocol_serde::shape_multi_region_endpoints::de_multi_region_endpoints(
+                    builder = builder.set_multi_region_endpoints(super::protocol_serde::shape_multi_region_endpoints::de_multi_region_endpoints(
                         tokens,
                         _value,
                         depth + 1,
