--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_configuration_set::DeleteConfigurationSetOutput,
-    crate::operation::delete_configuration_set::DeleteConfigurationSetError,
+    super::operation::delete_configuration_set::DeleteConfigurationSetOutput,
+    super::operation::delete_configuration_set::DeleteConfigurationSetError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled(
+            return Err(super::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "BadRequestException" => crate::operation::delete_configuration_set::DeleteConfigurationSetError::BadRequestException({
+        "BadRequestException" => super::operation::delete_configuration_set::DeleteConfigurationSetError::BadRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
+                let mut output = super::types::error::builders::BadRequestExceptionBuilder::default();
+                output = super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -40,16 +40,16 @@
             tmp
         }),
         "ConcurrentModificationException" => {
-            crate::operation::delete_configuration_set::DeleteConfigurationSetError::ConcurrentModificationException({
+            super::operation::delete_configuration_set::DeleteConfigurationSetError::ConcurrentModificationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_json_err(
+                    let mut output = super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                    output = super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
+                    .map_err(super::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -59,13 +59,13 @@
                 tmp
             })
         }
-        "NotFoundException" => crate::operation::delete_configuration_set::DeleteConfigurationSetError::NotFoundException({
+        "NotFoundException" => super::operation::delete_configuration_set::DeleteConfigurationSetError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
+                let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -74,13 +74,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::delete_configuration_set::DeleteConfigurationSetError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::delete_configuration_set::DeleteConfigurationSetError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_configuration_set::DeleteConfigurationSetError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -89,7 +89,7 @@
             }
             tmp
         }),
-        _ => crate::operation::delete_configuration_set::DeleteConfigurationSetError::generic(generic),
+        _ => super::operation::delete_configuration_set::DeleteConfigurationSetError::generic(generic),
     })
 }

@@ -99,12 +99,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_configuration_set::DeleteConfigurationSetOutput,
-    crate::operation::delete_configuration_set::DeleteConfigurationSetError,
+    super::operation::delete_configuration_set::DeleteConfigurationSetOutput,
+    super::operation::delete_configuration_set::DeleteConfigurationSetError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_configuration_set::builders::DeleteConfigurationSetOutputBuilder::default();
+        let mut output = super::operation::delete_configuration_set::builders::DeleteConfigurationSetOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
