--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::MailFromAttributes>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::MailFromAttributes>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::MailFromAttributesBuilder::default();
+            let mut builder = super::super::types::builders::MailFromAttributesBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,7 +31,7 @@
                         "MailFromDomainStatus" => {
                             builder = builder.set_mail_from_domain_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::MailFromDomainStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::MailFromDomainStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -38,7 +38,7 @@
                         "BehaviorOnMxFailure" => {
                             builder = builder.set_behavior_on_mx_failure(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::BehaviorOnMxFailure::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::BehaviorOnMxFailure::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -51,7 +51,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::mail_from_attributes_correct_errors(builder).build().map_err(
+            Ok(Some(super::super::serde_util::mail_from_attributes_correct_errors(builder).build().map_err(
                 |err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err),
             )?))
         }
