--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsOutput,
-    crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError,
+    super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsOutput,
+    super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled(generic))
+            return Err(super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled(generic))
         }
     };

@@ -23,13 +23,13 @@
     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "BadRequestException" => {
-            crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::BadRequestException({
+            super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::BadRequestException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
+                    let mut output = super::types::error::builders::BadRequestExceptionBuilder::default();
+                    output = super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                        .map_err(super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -40,13 +40,13 @@
             })
         }
         "NotFoundException" => {
-            crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::NotFoundException({
+            super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::NotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
+                    let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                    output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                        .map_err(super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -57,15 +57,15 @@
             })
         }
         "TooManyRequestsException" => {
-            crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::TooManyRequestsException({
+            super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::TooManyRequestsException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                    let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                        super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
                             .map_err(
-                                crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled,
+                                super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled,
                             )?;
                     let output = output.meta(generic);
                     output.build()
@@ -76,7 +76,7 @@
                 tmp
             })
         }
-        _ => crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::generic(generic),
+        _ => super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::generic(generic),
     })
 }

@@ -86,18 +86,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsOutput,
-    crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError,
+    super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsOutput,
+    super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_configuration_set_event_destinations::de_get_configuration_set_event_destinations(
+            super::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsOutputBuilder::default();
+        output = super::protocol_serde::shape_get_configuration_set_event_destinations::de_get_configuration_set_event_destinations(
             _response_body,
             output,
         )
-        .map_err(crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
+        .map_err(super::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -105,12 +105,12 @@

 pub(crate) fn de_get_configuration_set_event_destinations(
     _value: &[u8],
-    mut builder: crate::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsOutputBuilder,
+    mut builder: super::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsOutputBuilder,
+    super::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -120,7 +120,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "EventDestinations" => {
-                    builder = builder.set_event_destinations(crate::protocol_serde::shape_event_destinations::de_event_destinations(
+                    builder = builder.set_event_destinations(super::protocol_serde::shape_event_destinations::de_event_destinations(
                         tokens,
                         _value,
                         depth + 1,
