--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::DomainDeliverabilityCampaign>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::DomainDeliverabilityCampaign>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::DomainDeliverabilityCampaignBuilder::default();
+            let mut builder = super::types::builders::DomainDeliverabilityCampaignBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -50,7 +50,7 @@
                             );
                         }
                         "SendingIps" => {
-                            builder = builder.set_sending_ips(crate::protocol_serde::shape_ip_list::de_ip_list(tokens, _value, depth + 1)?);
+                            builder = builder.set_sending_ips(super::protocol_serde::shape_ip_list::de_ip_list(tokens, _value, depth + 1)?);
                         }
                         "FirstSeenDateTime" => {
                             builder = builder.set_first_seen_date_time(::aws_smithy_json::deserialize::token::expect_timestamp_or_null(
@@ -101,7 +101,7 @@
                             );
                         }
                         "Esps" => {
-                            builder = builder.set_esps(crate::protocol_serde::shape_esps::de_esps(tokens, _value, depth + 1)?);
+                            builder = builder.set_esps(super::protocol_serde::shape_esps::de_esps(tokens, _value, depth + 1)?);
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
                     },
