--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::EmailAddressInsightsMailboxEvaluations>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::EmailAddressInsightsMailboxEvaluations>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::EmailAddressInsightsMailboxEvaluationsBuilder::default();
+            let mut builder = super::super::types::builders::EmailAddressInsightsMailboxEvaluationsBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -23,7 +23,7 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                         "HasValidSyntax" => {
                             builder = builder.set_has_valid_syntax(
-                                crate::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
+                                super::super::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -32,7 +32,7 @@
                         }
                         "HasValidDnsRecords" => {
                             builder = builder.set_has_valid_dns_records(
-                                crate::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
+                                super::super::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -41,7 +41,7 @@
                         }
                         "MailboxExists" => {
                             builder = builder.set_mailbox_exists(
-                                crate::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
+                                super::super::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -50,7 +50,7 @@
                         }
                         "IsRoleAddress" => {
                             builder = builder.set_is_role_address(
-                                crate::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
+                                super::super::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -59,7 +59,7 @@
                         }
                         "IsDisposable" => {
                             builder = builder.set_is_disposable(
-                                crate::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
+                                super::super::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -68,7 +68,7 @@
                         }
                         "IsRandomInput" => {
                             builder = builder.set_is_random_input(
-                                crate::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
+                                super::super::protocol_serde::shape_email_address_insights_verdict::de_email_address_insights_verdict(
                                     tokens,
                                     _value,
                                     depth + 1,
