--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_export_job::GetExportJobOutput, crate::operation::get_export_job::GetExportJobError> {
+) -> std::result::Result<super::operation::get_export_job::GetExportJobOutput, super::operation::get_export_job::GetExportJobError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_export_job::GetExportJobError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_export_job::GetExportJobError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_export_job::GetExportJobError::unhandled(generic)),
+        None => return Err(super::operation::get_export_job::GetExportJobError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "BadRequestException" => crate::operation::get_export_job::GetExportJobError::BadRequestException({
+        "BadRequestException" => super::operation::get_export_job::GetExportJobError::BadRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BadRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_export_job::GetExportJobError::unhandled)?;
+                let mut output = super::types::error::builders::BadRequestExceptionBuilder::default();
+                output = super::protocol_serde::shape_bad_request_exception::de_bad_request_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_export_job::GetExportJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::get_export_job::GetExportJobError::NotFoundException({
+        "NotFoundException" => super::operation::get_export_job::GetExportJobError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_export_job::GetExportJobError::unhandled)?;
+                let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_export_job::GetExportJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::get_export_job::GetExportJobError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::get_export_job::GetExportJobError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_export_job::GetExportJobError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_export_job::GetExportJobError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,7 +62,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_export_job::GetExportJobError::generic(generic),
+        _ => super::operation::get_export_job::GetExportJobError::generic(generic),
     })
 }

@@ -71,12 +71,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_export_job::GetExportJobOutput, crate::operation::get_export_job::GetExportJobError> {
+) -> std::result::Result<super::operation::get_export_job::GetExportJobOutput, super::operation::get_export_job::GetExportJobError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_export_job::builders::GetExportJobOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_export_job::de_get_export_job(_response_body, output)
-            .map_err(crate::operation::get_export_job::GetExportJobError::unhandled)?;
+        let mut output = super::operation::get_export_job::builders::GetExportJobOutputBuilder::default();
+        output = super::protocol_serde::shape_get_export_job::de_get_export_job(_response_body, output)
+            .map_err(super::operation::get_export_job::GetExportJobError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -84,12 +84,12 @@

 pub(crate) fn de_get_export_job(
     _value: &[u8],
-    mut builder: crate::operation::get_export_job::builders::GetExportJobOutputBuilder,
+    mut builder: super::operation::get_export_job::builders::GetExportJobOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_export_job::builders::GetExportJobOutputBuilder,
+    super::operation::get_export_job::builders::GetExportJobOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -111,7 +111,7 @@
                     )?);
                 }
                 "ExportDataSource" => {
-                    builder = builder.set_export_data_source(crate::protocol_serde::shape_export_data_source::de_export_data_source(
+                    builder = builder.set_export_data_source(super::protocol_serde::shape_export_data_source::de_export_data_source(
                         tokens,
                         _value,
                         depth + 1,
@@ -118,7 +118,7 @@
                     )?);
                 }
                 "ExportDestination" => {
-                    builder = builder.set_export_destination(crate::protocol_serde::shape_export_destination::de_export_destination(
+                    builder = builder.set_export_destination(super::protocol_serde::shape_export_destination::de_export_destination(
                         tokens,
                         _value,
                         depth + 1,
@@ -127,12 +127,12 @@
                 "ExportSourceType" => {
                     builder = builder.set_export_source_type(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::ExportSourceType::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::ExportSourceType::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "FailureInfo" => {
-                    builder = builder.set_failure_info(crate::protocol_serde::shape_failure_info::de_failure_info(tokens, _value, depth + 1)?);
+                    builder = builder.set_failure_info(super::protocol_serde::shape_failure_info::de_failure_info(tokens, _value, depth + 1)?);
                 }
                 "JobId" => {
                     builder = builder.set_job_id(
@@ -144,12 +144,12 @@
                 "JobStatus" => {
                     builder = builder.set_job_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::JobStatus::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::JobStatus::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "Statistics" => {
-                    builder = builder.set_statistics(crate::protocol_serde::shape_export_statistics::de_export_statistics(
+                    builder = builder.set_statistics(super::protocol_serde::shape_export_statistics::de_export_statistics(
                         tokens,
                         _value,
                         depth + 1,
