--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl DescribeSubscriptionFiltersInput {
     /// Creates a new builder-style object to manufacture [`DescribeSubscriptionFiltersInput`](crate::operation::describe_subscription_filters::DescribeSubscriptionFiltersInput).
-    pub fn builder() -> crate::operation::describe_subscription_filters::builders::DescribeSubscriptionFiltersInputBuilder {
-        crate::operation::describe_subscription_filters::builders::DescribeSubscriptionFiltersInputBuilder::default()
+    pub fn builder() -> super::operation::describe_subscription_filters::builders::DescribeSubscriptionFiltersInputBuilder {
+        super::operation::describe_subscription_filters::builders::DescribeSubscriptionFiltersInputBuilder::default()
     }
 }

@@ -108,10 +108,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_subscription_filters::DescribeSubscriptionFiltersInput,
+        super::operation::describe_subscription_filters::DescribeSubscriptionFiltersInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_subscription_filters::DescribeSubscriptionFiltersInput {
+        ::std::result::Result::Ok(super::operation::describe_subscription_filters::DescribeSubscriptionFiltersInput {
             log_group_name: self.log_group_name,
             filter_name_prefix: self.filter_name_prefix,
             next_token: self.next_token,
