--- original
+++ modified
@@ -33,7 +33,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by each class, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub log_group_class: ::std::option::Option<crate::types::LogGroupClass>,
+    pub log_group_class: ::std::option::Option<super::super::super::types::LogGroupClass>,
     /// <p>Use this array to filter the list of log groups returned. If you specify this parameter, the only other filter that you can choose to specify is <code>includeLinkedAccounts</code>.</p>
     /// <p>If you are using this operation in a monitoring account, you can specify the ARNs of log groups in source accounts and in the monitoring account itself. If you are using this operation in an account that is not a cross-account monitoring account, you can specify only log group names in the same account as the operation.</p>
     pub log_group_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -83,7 +83,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by each class, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn log_group_class(&self) -> ::std::option::Option<&crate::types::LogGroupClass> {
+    pub fn log_group_class(&self) -> ::std::option::Option<&super::super::super::types::LogGroupClass> {
         self.log_group_class.as_ref()
     }
     /// <p>Use this array to filter the list of log groups returned. If you specify this parameter, the only other filter that you can choose to specify is <code>includeLinkedAccounts</code>.</p>
@@ -96,8 +96,8 @@
 }
 impl DescribeLogGroupsInput {
     /// Creates a new builder-style object to manufacture [`DescribeLogGroupsInput`](crate::operation::describe_log_groups::DescribeLogGroupsInput).
-    pub fn builder() -> crate::operation::describe_log_groups::builders::DescribeLogGroupsInputBuilder {
-        crate::operation::describe_log_groups::builders::DescribeLogGroupsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_log_groups::builders::DescribeLogGroupsInputBuilder {
+        super::super::super::operation::describe_log_groups::builders::DescribeLogGroupsInputBuilder::default()
     }
 }

@@ -111,7 +111,7 @@
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
     pub(crate) limit: ::std::option::Option<i32>,
     pub(crate) include_linked_accounts: ::std::option::Option<bool>,
-    pub(crate) log_group_class: ::std::option::Option<crate::types::LogGroupClass>,
+    pub(crate) log_group_class: ::std::option::Option<super::super::super::types::LogGroupClass>,
     pub(crate) log_group_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
 }
 impl DescribeLogGroupsInputBuilder {
@@ -237,7 +237,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by each class, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn log_group_class(mut self, input: crate::types::LogGroupClass) -> Self {
+    pub fn log_group_class(mut self, input: super::super::super::types::LogGroupClass) -> Self {
         self.log_group_class = ::std::option::Option::Some(input);
         self
     }
@@ -252,7 +252,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by each class, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn set_log_group_class(mut self, input: ::std::option::Option<crate::types::LogGroupClass>) -> Self {
+    pub fn set_log_group_class(mut self, input: ::std::option::Option<super::super::super::types::LogGroupClass>) -> Self {
         self.log_group_class = input;
         self
     }
@@ -267,7 +267,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by each class, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn get_log_group_class(&self) -> &::std::option::Option<crate::types::LogGroupClass> {
+    pub fn get_log_group_class(&self) -> &::std::option::Option<super::super::super::types::LogGroupClass> {
         &self.log_group_class
     }
     /// Appends an item to `log_group_identifiers`.
@@ -296,8 +296,8 @@
     /// Consumes the builder and constructs a [`DescribeLogGroupsInput`](crate::operation::describe_log_groups::DescribeLogGroupsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_log_groups::DescribeLogGroupsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::describe_log_groups::DescribeLogGroupsInput {
+    ) -> ::std::result::Result<super::super::super::operation::describe_log_groups::DescribeLogGroupsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::describe_log_groups::DescribeLogGroupsInput {
             account_identifiers: self.account_identifiers,
             log_group_name_prefix: self.log_group_name_prefix,
             log_group_name_pattern: self.log_group_name_pattern,
