--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl GetKeyPolicyInput {
     /// Creates a new builder-style object to manufacture [`GetKeyPolicyInput`](crate::operation::get_key_policy::GetKeyPolicyInput).
-    pub fn builder() -> crate::operation::get_key_policy::builders::GetKeyPolicyInputBuilder {
-        crate::operation::get_key_policy::builders::GetKeyPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::get_key_policy::builders::GetKeyPolicyInputBuilder {
+        super::operation::get_key_policy::builders::GetKeyPolicyInputBuilder::default()
     }
 }

@@ -110,8 +110,8 @@
     /// Consumes the builder and constructs a [`GetKeyPolicyInput`](crate::operation::get_key_policy::GetKeyPolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_key_policy::GetKeyPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_key_policy::GetKeyPolicyInput {
+    ) -> ::std::result::Result<super::operation::get_key_policy::GetKeyPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::get_key_policy::GetKeyPolicyInput {
             key_id: self.key_id,
             policy_name: self.policy_name,
         })
