--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::confirm_subscription::ConfirmSubscriptionOutput,
-    crate::operation::confirm_subscription::ConfirmSubscriptionError,
+    super::super::operation::confirm_subscription::ConfirmSubscriptionOutput,
+    super::super::operation::confirm_subscription::ConfirmSubscriptionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled(generic)),
+        None => return Err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AuthorizationError" => crate::operation::confirm_subscription::ConfirmSubscriptionError::AuthorizationErrorException({
+        "AuthorizationError" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::AuthorizationErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                let mut output = super::super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,16 +35,16 @@
             }
             tmp
         }),
-        "FilterPolicyLimitExceeded" => crate::operation::confirm_subscription::ConfirmSubscriptionError::FilterPolicyLimitExceededException({
+        "FilterPolicyLimitExceeded" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::FilterPolicyLimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::FilterPolicyLimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_filter_policy_limit_exceeded_exception::de_filter_policy_limit_exceeded_exception_xml_err(
+                let mut output = super::super::types::error::builders::FilterPolicyLimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_filter_policy_limit_exceeded_exception::de_filter_policy_limit_exceeded_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -53,13 +53,13 @@
             }
             tmp
         }),
-        "InternalError" => crate::operation::confirm_subscription::ConfirmSubscriptionError::InternalErrorException({
+        "InternalError" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,13 +68,13 @@
             }
             tmp
         }),
-        "InvalidParameter" => crate::operation::confirm_subscription::ConfirmSubscriptionError::InvalidParameterException({
+        "InvalidParameter" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -83,13 +83,13 @@
             }
             tmp
         }),
-        "NotFound" => crate::operation::confirm_subscription::ConfirmSubscriptionError::NotFoundException({
+        "NotFound" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_found_exception::de_not_found_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -98,14 +98,14 @@
             }
             tmp
         }),
-        "ReplayLimitExceeded" => crate::operation::confirm_subscription::ConfirmSubscriptionError::ReplayLimitExceededException({
+        "ReplayLimitExceeded" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::ReplayLimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ReplayLimitExceededExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::ReplayLimitExceededExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_replay_limit_exceeded_exception::de_replay_limit_exceeded_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                    super::super::protocol_serde::shape_replay_limit_exceeded_exception::de_replay_limit_exceeded_exception_xml_err(_response_body, output)
+                        .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -114,16 +114,16 @@
             }
             tmp
         }),
-        "SubscriptionLimitExceeded" => crate::operation::confirm_subscription::ConfirmSubscriptionError::SubscriptionLimitExceededException({
+        "SubscriptionLimitExceeded" => super::super::operation::confirm_subscription::ConfirmSubscriptionError::SubscriptionLimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SubscriptionLimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_subscription_limit_exceeded_exception::de_subscription_limit_exceeded_exception_xml_err(
+                let mut output = super::super::types::error::builders::SubscriptionLimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_subscription_limit_exceeded_exception::de_subscription_limit_exceeded_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+                .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -132,7 +132,7 @@
             }
             tmp
         }),
-        _ => crate::operation::confirm_subscription::ConfirmSubscriptionError::generic(generic),
+        _ => super::super::operation::confirm_subscription::ConfirmSubscriptionError::generic(generic),
     })
 }

@@ -142,14 +142,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::confirm_subscription::ConfirmSubscriptionOutput,
-    crate::operation::confirm_subscription::ConfirmSubscriptionError,
+    super::super::operation::confirm_subscription::ConfirmSubscriptionOutput,
+    super::super::operation::confirm_subscription::ConfirmSubscriptionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::confirm_subscription::builders::ConfirmSubscriptionOutputBuilder::default();
-        output = crate::protocol_serde::shape_confirm_subscription::de_confirm_subscription(_response_body, output)
-            .map_err(crate::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
+        let mut output = super::super::operation::confirm_subscription::builders::ConfirmSubscriptionOutputBuilder::default();
+        output = super::super::protocol_serde::shape_confirm_subscription::de_confirm_subscription(_response_body, output)
+            .map_err(super::super::operation::confirm_subscription::ConfirmSubscriptionError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -158,8 +158,8 @@
 #[allow(unused_mut)]
 pub fn de_confirm_subscription(
     inp: &[u8],
-    mut builder: crate::operation::confirm_subscription::builders::ConfirmSubscriptionOutputBuilder,
-) -> std::result::Result<crate::operation::confirm_subscription::builders::ConfirmSubscriptionOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
+    mut builder: super::super::operation::confirm_subscription::builders::ConfirmSubscriptionOutputBuilder,
+) -> std::result::Result<super::super::operation::confirm_subscription::builders::ConfirmSubscriptionOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
 {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

