--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_phone_number_information(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::PhoneNumberInformation, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::PhoneNumberInformation, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::PhoneNumberInformation::builder();
+    let mut builder = super::super::types::PhoneNumberInformation::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("CreatedAt") /* CreatedAt com.amazonaws.sns#PhoneNumberInformation$CreatedAt */ =>  {
@@ -67,8 +67,8 @@
             s if s.matches("RouteType") /* RouteType com.amazonaws.sns#PhoneNumberInformation$RouteType */ =>  {
                 let var_5 =
                     Some(
-                        Result::<crate::types::RouteType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::RouteType::from(
+                        Result::<super::super::types::RouteType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::RouteType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -81,7 +81,7 @@
             s if s.matches("NumberCapabilities") /* NumberCapabilities com.amazonaws.sns#PhoneNumberInformation$NumberCapabilities */ =>  {
                 let var_6 =
                     Some(
-                        crate::protocol_serde::shape_number_capability_list::de_number_capability_list(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_number_capability_list::de_number_capability_list(&mut tag, depth + 1)
                         ?
                     )
                 ;
