--- original
+++ modified
@@ -2,7 +2,7 @@
 #[allow(unused_mut)]
 pub fn ser_tag(
     mut writer: ::aws_smithy_query::QueryValueWriter,
-    input: &crate::types::Tag,
+    input: &super::types::Tag,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
     let mut scope_1 = writer.prefix("Key");
@@ -21,12 +21,12 @@
 pub fn de_tag(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::Tag, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::Tag, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::Tag::builder();
+    let mut builder = super::types::Tag::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Key") /* Key com.amazonaws.sns#Tag$Key */ =>  {
@@ -58,7 +58,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::tag_correct_errors(builder)
+    Ok(super::serde_util::tag_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
