--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_platform_applications::ListPlatformApplicationsOutput,
-    crate::operation::list_platform_applications::ListPlatformApplicationsError,
+    super::super::operation::list_platform_applications::ListPlatformApplicationsOutput,
+    super::super::operation::list_platform_applications::ListPlatformApplicationsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::list_platform_applications::ListPlatformApplicationsError::unhandled(
+            return Err(super::super::operation::list_platform_applications::ListPlatformApplicationsError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AuthorizationError" => crate::operation::list_platform_applications::ListPlatformApplicationsError::AuthorizationErrorException({
+        "AuthorizationError" => super::super::operation::list_platform_applications::ListPlatformApplicationsError::AuthorizationErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "InternalError" => crate::operation::list_platform_applications::ListPlatformApplicationsError::InternalErrorException({
+        "InternalError" => super::super::operation::list_platform_applications::ListPlatformApplicationsError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,13 +54,13 @@
             }
             tmp
         }),
-        "InvalidParameter" => crate::operation::list_platform_applications::ListPlatformApplicationsError::InvalidParameterException({
+        "InvalidParameter" => super::super::operation::list_platform_applications::ListPlatformApplicationsError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -69,7 +69,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_platform_applications::ListPlatformApplicationsError::generic(generic),
+        _ => super::super::operation::list_platform_applications::ListPlatformApplicationsError::generic(generic),
     })
 }

@@ -79,14 +79,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_platform_applications::ListPlatformApplicationsOutput,
-    crate::operation::list_platform_applications::ListPlatformApplicationsError,
+    super::super::operation::list_platform_applications::ListPlatformApplicationsOutput,
+    super::super::operation::list_platform_applications::ListPlatformApplicationsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_platform_applications::builders::ListPlatformApplicationsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_platform_applications::de_list_platform_applications(_response_body, output)
-            .map_err(crate::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
+        let mut output = super::super::operation::list_platform_applications::builders::ListPlatformApplicationsOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_platform_applications::de_list_platform_applications(_response_body, output)
+            .map_err(super::super::operation::list_platform_applications::ListPlatformApplicationsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -95,9 +95,9 @@
 #[allow(unused_mut)]
 pub fn de_list_platform_applications(
     inp: &[u8],
-    mut builder: crate::operation::list_platform_applications::builders::ListPlatformApplicationsOutputBuilder,
+    mut builder: super::super::operation::list_platform_applications::builders::ListPlatformApplicationsOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::list_platform_applications::builders::ListPlatformApplicationsOutputBuilder,
+    super::super::operation::list_platform_applications::builders::ListPlatformApplicationsOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -125,7 +125,7 @@
             s if s.matches("PlatformApplications") /* PlatformApplications com.amazonaws.sns.synthetic#ListPlatformApplicationsOutput$PlatformApplications */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_list_of_platform_applications::de_list_of_platform_applications(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_list_of_platform_applications::de_list_of_platform_applications(&mut tag, depth + 1)
                         ?
                     )
                 ;
