--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_sms_sandbox_account_status::GetSmsSandboxAccountStatusOutput,
-    crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError,
+    super::super::operation::get_sms_sandbox_account_status::GetSmsSandboxAccountStatusOutput,
+    super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled(generic)),
+        None => return Err(super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AuthorizationError" => crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::AuthorizationErrorException({
+        "AuthorizationError" => super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::AuthorizationErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
+                let mut output = super::super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InternalError" => crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::InternalErrorException({
+        "InternalError" => super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "Throttled" => crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::ThrottledException({
+        "Throttled" => super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::ThrottledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttled_exception::de_throttled_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
+                let mut output = super::super::types::error::builders::ThrottledExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_throttled_exception::de_throttled_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,7 +65,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::generic(generic),
+        _ => super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::generic(generic),
     })
 }

@@ -75,16 +75,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_sms_sandbox_account_status::GetSmsSandboxAccountStatusOutput,
-    crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError,
+    super::super::operation::get_sms_sandbox_account_status::GetSmsSandboxAccountStatusOutput,
+    super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_sms_sandbox_account_status::builders::GetSmsSandboxAccountStatusOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_sms_sandbox_account_status::de_get_sms_sandbox_account_status(_response_body, output)
-            .map_err(crate::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
+        let mut output = super::super::operation::get_sms_sandbox_account_status::builders::GetSmsSandboxAccountStatusOutputBuilder::default();
+        output = super::super::protocol_serde::shape_get_sms_sandbox_account_status::de_get_sms_sandbox_account_status(_response_body, output)
+            .map_err(super::super::operation::get_sms_sandbox_account_status::GetSMSSandboxAccountStatusError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_sms_sandbox_account_status_output_output_correct_errors(output).build()
+        super::super::serde_util::get_sms_sandbox_account_status_output_output_correct_errors(output).build()
     })
 }

@@ -91,9 +91,9 @@
 #[allow(unused_mut)]
 pub fn de_get_sms_sandbox_account_status(
     inp: &[u8],
-    mut builder: crate::operation::get_sms_sandbox_account_status::builders::GetSmsSandboxAccountStatusOutputBuilder,
+    mut builder: super::super::operation::get_sms_sandbox_account_status::builders::GetSmsSandboxAccountStatusOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::get_sms_sandbox_account_status::builders::GetSmsSandboxAccountStatusOutputBuilder,
+    super::super::operation::get_sms_sandbox_account_status::builders::GetSmsSandboxAccountStatusOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
