--- original
+++ modified
@@ -2,7 +2,7 @@
 pub fn de_number_capability_list(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<::std::vec::Vec<crate::types::NumberCapability>, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<::std::vec::Vec<super::types::NumberCapability>, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
@@ -11,8 +11,8 @@
         match tag.start_el() {
             s if s.matches("member") /* member com.amazonaws.sns#NumberCapabilityList$member */ =>  {
                 out.push(
-                    Result::<crate::types::NumberCapability, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                        crate::types::NumberCapability::from(
+                    Result::<super::types::NumberCapability, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                        super::types::NumberCapability::from(
                             ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                         )
                     )
