--- original
+++ modified
@@ -2,7 +2,7 @@
 pub fn de_phone_number_information_list(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<::std::vec::Vec<crate::types::PhoneNumberInformation>, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<::std::vec::Vec<super::super::types::PhoneNumberInformation>, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
@@ -11,7 +11,7 @@
         match tag.start_el() {
             s if s.matches("member") /* member com.amazonaws.sns#PhoneNumberInformationList$member */ =>  {
                 out.push(
-                    crate::protocol_serde::shape_phone_number_information::de_phone_number_information(&mut tag, depth + 1)
+                    super::super::protocol_serde::shape_phone_number_information::de_phone_number_information(&mut tag, depth + 1)
                     ?
                 );
             }
