--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::publish_batch::PublishBatchOutput, crate::operation::publish_batch::PublishBatchError> {
+) -> std::result::Result<super::operation::publish_batch::PublishBatchOutput, super::operation::publish_batch::PublishBatchError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::publish_batch::PublishBatchError::unhandled(generic)),
+        None => return Err(super::operation::publish_batch::PublishBatchError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AuthorizationError" => crate::operation::publish_batch::PublishBatchError::AuthorizationErrorException({
+        "AuthorizationError" => super::operation::publish_batch::PublishBatchError::AuthorizationErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,16 +32,16 @@
             }
             tmp
         }),
-        "BatchEntryIdsNotDistinct" => crate::operation::publish_batch::PublishBatchError::BatchEntryIdsNotDistinctException({
+        "BatchEntryIdsNotDistinct" => super::operation::publish_batch::PublishBatchError::BatchEntryIdsNotDistinctException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BatchEntryIdsNotDistinctExceptionBuilder::default();
-                output = crate::protocol_serde::shape_batch_entry_ids_not_distinct_exception::de_batch_entry_ids_not_distinct_exception_xml_err(
+                let mut output = super::types::error::builders::BatchEntryIdsNotDistinctExceptionBuilder::default();
+                output = super::protocol_serde::shape_batch_entry_ids_not_distinct_exception::de_batch_entry_ids_not_distinct_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,16 +50,16 @@
             }
             tmp
         }),
-        "BatchRequestTooLong" => crate::operation::publish_batch::PublishBatchError::BatchRequestTooLongException({
+        "BatchRequestTooLong" => super::operation::publish_batch::PublishBatchError::BatchRequestTooLongException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::BatchRequestTooLongExceptionBuilder::default();
-                output = crate::protocol_serde::shape_batch_request_too_long_exception::de_batch_request_too_long_exception_xml_err(
+                let mut output = super::types::error::builders::BatchRequestTooLongExceptionBuilder::default();
+                output = super::protocol_serde::shape_batch_request_too_long_exception::de_batch_request_too_long_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,13 +68,13 @@
             }
             tmp
         }),
-        "EmptyBatchRequest" => crate::operation::publish_batch::PublishBatchError::EmptyBatchRequestException({
+        "EmptyBatchRequest" => super::operation::publish_batch::PublishBatchError::EmptyBatchRequestException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EmptyBatchRequestExceptionBuilder::default();
-                output = crate::protocol_serde::shape_empty_batch_request_exception::de_empty_batch_request_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::EmptyBatchRequestExceptionBuilder::default();
+                output = super::protocol_serde::shape_empty_batch_request_exception::de_empty_batch_request_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -83,13 +83,13 @@
             }
             tmp
         }),
-        "EndpointDisabled" => crate::operation::publish_batch::PublishBatchError::EndpointDisabledException({
+        "EndpointDisabled" => super::operation::publish_batch::PublishBatchError::EndpointDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EndpointDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_endpoint_disabled_exception::de_endpoint_disabled_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::EndpointDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_endpoint_disabled_exception::de_endpoint_disabled_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -98,13 +98,13 @@
             }
             tmp
         }),
-        "InternalError" => crate::operation::publish_batch::PublishBatchError::InternalErrorException({
+        "InternalError" => super::operation::publish_batch::PublishBatchError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -113,16 +113,16 @@
             }
             tmp
         }),
-        "InvalidBatchEntryId" => crate::operation::publish_batch::PublishBatchError::InvalidBatchEntryIdException({
+        "InvalidBatchEntryId" => super::operation::publish_batch::PublishBatchError::InvalidBatchEntryIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidBatchEntryIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_batch_entry_id_exception::de_invalid_batch_entry_id_exception_xml_err(
+                let mut output = super::types::error::builders::InvalidBatchEntryIdExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_batch_entry_id_exception::de_invalid_batch_entry_id_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -131,13 +131,13 @@
             }
             tmp
         }),
-        "InvalidParameter" => crate::operation::publish_batch::PublishBatchError::InvalidParameterException({
+        "InvalidParameter" => super::operation::publish_batch::PublishBatchError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -146,16 +146,16 @@
             }
             tmp
         }),
-        "ParameterValueInvalid" => crate::operation::publish_batch::PublishBatchError::InvalidParameterValueException({
+        "ParameterValueInvalid" => super::operation::publish_batch::PublishBatchError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_xml_err(
+                let mut output = super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -164,13 +164,13 @@
             }
             tmp
         }),
-        "InvalidSecurity" => crate::operation::publish_batch::PublishBatchError::InvalidSecurityException({
+        "InvalidSecurity" => super::operation::publish_batch::PublishBatchError::InvalidSecurityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSecurityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_security_exception::de_invalid_security_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidSecurityExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_security_exception::de_invalid_security_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -179,13 +179,13 @@
             }
             tmp
         }),
-        "KMSAccessDenied" => crate::operation::publish_batch::PublishBatchError::KmsAccessDeniedException({
+        "KMSAccessDenied" => super::operation::publish_batch::PublishBatchError::KmsAccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -194,13 +194,13 @@
             }
             tmp
         }),
-        "KMSDisabled" => crate::operation::publish_batch::PublishBatchError::KmsDisabledException({
+        "KMSDisabled" => super::operation::publish_batch::PublishBatchError::KmsDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::KmsDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -209,13 +209,13 @@
             }
             tmp
         }),
-        "KMSInvalidState" => crate::operation::publish_batch::PublishBatchError::KmsInvalidStateException({
+        "KMSInvalidState" => super::operation::publish_batch::PublishBatchError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -224,13 +224,13 @@
             }
             tmp
         }),
-        "KMSNotFound" => crate::operation::publish_batch::PublishBatchError::KmsNotFoundException({
+        "KMSNotFound" => super::operation::publish_batch::PublishBatchError::KmsNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::KmsNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -239,13 +239,13 @@
             }
             tmp
         }),
-        "KMSOptInRequired" => crate::operation::publish_batch::PublishBatchError::KmsOptInRequired({
+        "KMSOptInRequired" => super::operation::publish_batch::PublishBatchError::KmsOptInRequired({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsOptInRequiredBuilder::default();
-                output = crate::protocol_serde::shape_kms_opt_in_required::de_kms_opt_in_required_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::KmsOptInRequiredBuilder::default();
+                output = super::protocol_serde::shape_kms_opt_in_required::de_kms_opt_in_required_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -254,13 +254,13 @@
             }
             tmp
         }),
-        "KMSThrottling" => crate::operation::publish_batch::PublishBatchError::KmsThrottlingException({
+        "KMSThrottling" => super::operation::publish_batch::PublishBatchError::KmsThrottlingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsThrottlingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_throttling_exception::de_kms_throttling_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::KmsThrottlingExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_throttling_exception::de_kms_throttling_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -269,13 +269,13 @@
             }
             tmp
         }),
-        "NotFound" => crate::operation::publish_batch::PublishBatchError::NotFoundException({
+        "NotFound" => super::operation::publish_batch::PublishBatchError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -284,16 +284,16 @@
             }
             tmp
         }),
-        "PlatformApplicationDisabled" => crate::operation::publish_batch::PublishBatchError::PlatformApplicationDisabledException({
+        "PlatformApplicationDisabled" => super::operation::publish_batch::PublishBatchError::PlatformApplicationDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PlatformApplicationDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_platform_application_disabled_exception::de_platform_application_disabled_exception_xml_err(
+                let mut output = super::types::error::builders::PlatformApplicationDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_platform_application_disabled_exception::de_platform_application_disabled_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -303,12 +303,12 @@
             tmp
         }),
         "TooManyEntriesInBatchRequest" => {
-            crate::operation::publish_batch::PublishBatchError::TooManyEntriesInBatchRequestException({
+            super::operation::publish_batch::PublishBatchError::TooManyEntriesInBatchRequestException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::TooManyEntriesInBatchRequestExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_too_many_entries_in_batch_request_exception::de_too_many_entries_in_batch_request_exception_xml_err(_response_body, output).map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                    let mut output = super::types::error::builders::TooManyEntriesInBatchRequestExceptionBuilder::default();
+                    output = super::protocol_serde::shape_too_many_entries_in_batch_request_exception::de_too_many_entries_in_batch_request_exception_xml_err(_response_body, output).map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -318,21 +318,21 @@
                 tmp
             })
         }
-        "ValidationException" => crate::operation::publish_batch::PublishBatchError::ValidationException({
+        "ValidationException" => super::operation::publish_batch::PublishBatchError::ValidationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_validation_exception::de_validation_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+                let mut output = super::types::error::builders::ValidationExceptionBuilder::default();
+                output = super::protocol_serde::shape_validation_exception::de_validation_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
                 let output = output.meta(generic);
-                crate::serde_util::validation_exception_correct_errors(output)
+                super::serde_util::validation_exception_correct_errors(output)
                     .build()
-                    .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?
+                    .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?
             };
             tmp
         }),
-        _ => crate::operation::publish_batch::PublishBatchError::generic(generic),
+        _ => super::operation::publish_batch::PublishBatchError::generic(generic),
     })
 }

@@ -341,12 +341,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::publish_batch::PublishBatchOutput, crate::operation::publish_batch::PublishBatchError> {
+) -> std::result::Result<super::operation::publish_batch::PublishBatchOutput, super::operation::publish_batch::PublishBatchError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::publish_batch::builders::PublishBatchOutputBuilder::default();
-        output = crate::protocol_serde::shape_publish_batch::de_publish_batch(_response_body, output)
-            .map_err(crate::operation::publish_batch::PublishBatchError::unhandled)?;
+        let mut output = super::operation::publish_batch::builders::PublishBatchOutputBuilder::default();
+        output = super::protocol_serde::shape_publish_batch::de_publish_batch(_response_body, output)
+            .map_err(super::operation::publish_batch::PublishBatchError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -355,8 +355,8 @@
 #[allow(unused_mut)]
 pub fn de_publish_batch(
     inp: &[u8],
-    mut builder: crate::operation::publish_batch::builders::PublishBatchOutputBuilder,
-) -> std::result::Result<crate::operation::publish_batch::builders::PublishBatchOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::operation::publish_batch::builders::PublishBatchOutputBuilder,
+) -> std::result::Result<super::operation::publish_batch::builders::PublishBatchOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -382,7 +382,7 @@
             s if s.matches("Successful") /* Successful com.amazonaws.sns.synthetic#PublishBatchOutput$Successful */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_publish_batch_result_entry_list::de_publish_batch_result_entry_list(&mut tag, depth + 1)
+                        super::protocol_serde::shape_publish_batch_result_entry_list::de_publish_batch_result_entry_list(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -392,7 +392,7 @@
             s if s.matches("Failed") /* Failed com.amazonaws.sns.synthetic#PublishBatchOutput$Failed */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_batch_result_error_entry_list::de_batch_result_error_entry_list(&mut tag, depth + 1)
+                        super::protocol_serde::shape_batch_result_error_entry_list::de_batch_result_error_entry_list(&mut tag, depth + 1)
                         ?
                     )
                 ;
