--- original
+++ modified
@@ -10,7 +10,7 @@
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("entry") => {
-                crate::protocol_serde::shape_map_string_to_string::de_map_string_to_string_entry(&mut tag, &mut out, depth)?;
+                super::super::protocol_serde::shape_map_string_to_string::de_map_string_to_string_entry(&mut tag, &mut out, depth)?;
             }
             _ => {}
         }
