--- original
+++ modified
@@ -5,30 +5,30 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutOutput,
-    crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError,
+    super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutOutput,
+    super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled(generic)),
+        None => return Err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "AuthorizationError" => {
-            crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::AuthorizationErrorException({
+            super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::AuthorizationErrorException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                            .map_err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
+                        super::super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                            .map_err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -38,13 +38,13 @@
                 tmp
             })
         }
-        "InternalError" => crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::InternalErrorException({
+        "InternalError" => super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -53,13 +53,13 @@
             }
             tmp
         }),
-        "InvalidParameter" => crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::InvalidParameterException({
+        "InvalidParameter" => super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,13 +68,13 @@
             }
             tmp
         }),
-        "Throttled" => crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::ThrottledException({
+        "Throttled" => super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::ThrottledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttled_exception::de_throttled_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
+                let mut output = super::super::types::error::builders::ThrottledExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_throttled_exception::de_throttled_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -83,7 +83,7 @@
             }
             tmp
         }),
-        _ => crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::generic(generic),
+        _ => super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::generic(generic),
     })
 }

@@ -93,14 +93,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutOutput,
-    crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError,
+    super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutOutput,
+    super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutOutputBuilder::default();
-        output = crate::protocol_serde::shape_check_if_phone_number_is_opted_out::de_check_if_phone_number_is_opted_out(_response_body, output)
-            .map_err(crate::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
+        let mut output = super::super::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutOutputBuilder::default();
+        output = super::super::protocol_serde::shape_check_if_phone_number_is_opted_out::de_check_if_phone_number_is_opted_out(_response_body, output)
+            .map_err(super::super::operation::check_if_phone_number_is_opted_out::CheckIfPhoneNumberIsOptedOutError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -109,9 +109,9 @@
 #[allow(unused_mut)]
 pub fn de_check_if_phone_number_is_opted_out(
     inp: &[u8],
-    mut builder: crate::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutOutputBuilder,
+    mut builder: super::super::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutOutputBuilder,
+    super::super::operation::check_if_phone_number_is_opted_out::builders::CheckIfPhoneNumberIsOptedOutOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
