--- original
+++ modified
@@ -4,27 +4,27 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_subscriptions::ListSubscriptionsOutput, crate::operation::list_subscriptions::ListSubscriptionsError>
+) -> std::result::Result<super::operation::list_subscriptions::ListSubscriptionsOutput, super::operation::list_subscriptions::ListSubscriptionsError>
 {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_subscriptions::ListSubscriptionsError::unhandled(generic)),
+        None => return Err(super::operation::list_subscriptions::ListSubscriptionsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AuthorizationError" => crate::operation::list_subscriptions::ListSubscriptionsError::AuthorizationErrorException({
+        "AuthorizationError" => super::operation::list_subscriptions::ListSubscriptionsError::AuthorizationErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
+                let mut output = super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -33,13 +33,13 @@
             }
             tmp
         }),
-        "InternalError" => crate::operation::list_subscriptions::ListSubscriptionsError::InternalErrorException({
+        "InternalError" => super::operation::list_subscriptions::ListSubscriptionsError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -48,13 +48,13 @@
             }
             tmp
         }),
-        "InvalidParameter" => crate::operation::list_subscriptions::ListSubscriptionsError::InvalidParameterException({
+        "InvalidParameter" => super::operation::list_subscriptions::ListSubscriptionsError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -63,7 +63,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_subscriptions::ListSubscriptionsError::generic(generic),
+        _ => super::operation::list_subscriptions::ListSubscriptionsError::generic(generic),
     })
 }

@@ -72,13 +72,13 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_subscriptions::ListSubscriptionsOutput, crate::operation::list_subscriptions::ListSubscriptionsError>
+) -> std::result::Result<super::operation::list_subscriptions::ListSubscriptionsOutput, super::operation::list_subscriptions::ListSubscriptionsError>
 {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_subscriptions::builders::ListSubscriptionsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_subscriptions::de_list_subscriptions(_response_body, output)
-            .map_err(crate::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
+        let mut output = super::operation::list_subscriptions::builders::ListSubscriptionsOutputBuilder::default();
+        output = super::protocol_serde::shape_list_subscriptions::de_list_subscriptions(_response_body, output)
+            .map_err(super::operation::list_subscriptions::ListSubscriptionsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -87,8 +87,8 @@
 #[allow(unused_mut)]
 pub fn de_list_subscriptions(
     inp: &[u8],
-    mut builder: crate::operation::list_subscriptions::builders::ListSubscriptionsOutputBuilder,
-) -> std::result::Result<crate::operation::list_subscriptions::builders::ListSubscriptionsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::operation::list_subscriptions::builders::ListSubscriptionsOutputBuilder,
+) -> std::result::Result<super::operation::list_subscriptions::builders::ListSubscriptionsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -114,7 +114,7 @@
             s if s.matches("Subscriptions") /* Subscriptions com.amazonaws.sns.synthetic#ListSubscriptionsOutput$Subscriptions */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_subscriptions_list::de_subscriptions_list(&mut tag, depth + 1)
+                        super::protocol_serde::shape_subscriptions_list::de_subscriptions_list(&mut tag, depth + 1)
                         ?
                     )
                 ;
