--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::untag_resource::UntagResourceOutput, crate::operation::untag_resource::UntagResourceError> {
+) -> std::result::Result<super::super::operation::untag_resource::UntagResourceOutput, super::super::operation::untag_resource::UntagResourceError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::untag_resource::UntagResourceError::unhandled(generic)),
+        None => return Err(super::super::operation::untag_resource::UntagResourceError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AuthorizationError" => crate::operation::untag_resource::UntagResourceError::AuthorizationErrorException({
+        "AuthorizationError" => super::super::operation::untag_resource::UntagResourceError::AuthorizationErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::AuthorizationErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::AuthorizationErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_authorization_error_exception::de_authorization_error_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "ConcurrentAccess" => crate::operation::untag_resource::UntagResourceError::ConcurrentAccessException({
+        "ConcurrentAccess" => super::super::operation::untag_resource::UntagResourceError::ConcurrentAccessException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConcurrentAccessExceptionBuilder::default();
-                output = crate::protocol_serde::shape_concurrent_access_exception::de_concurrent_access_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::ConcurrentAccessExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_concurrent_access_exception::de_concurrent_access_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "InvalidParameter" => crate::operation::untag_resource::UntagResourceError::InvalidParameterException({
+        "InvalidParameter" => super::super::operation::untag_resource::UntagResourceError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,13 +62,13 @@
             }
             tmp
         }),
-        "ResourceNotFound" => crate::operation::untag_resource::UntagResourceError::ResourceNotFoundException({
+        "ResourceNotFound" => super::super::operation::untag_resource::UntagResourceError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -77,13 +77,13 @@
             }
             tmp
         }),
-        "StaleTag" => crate::operation::untag_resource::UntagResourceError::StaleTagException({
+        "StaleTag" => super::super::operation::untag_resource::UntagResourceError::StaleTagException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::StaleTagExceptionBuilder::default();
-                output = crate::protocol_serde::shape_stale_tag_exception::de_stale_tag_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::StaleTagExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_stale_tag_exception::de_stale_tag_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -92,13 +92,13 @@
             }
             tmp
         }),
-        "TagLimitExceeded" => crate::operation::untag_resource::UntagResourceError::TagLimitExceededException({
+        "TagLimitExceeded" => super::super::operation::untag_resource::UntagResourceError::TagLimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TagLimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_tag_limit_exceeded_exception::de_tag_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::TagLimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_tag_limit_exceeded_exception::de_tag_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -107,13 +107,13 @@
             }
             tmp
         }),
-        "TagPolicy" => crate::operation::untag_resource::UntagResourceError::TagPolicyException({
+        "TagPolicy" => super::super::operation::untag_resource::UntagResourceError::TagPolicyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TagPolicyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_tag_policy_exception::de_tag_policy_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_resource::UntagResourceError::unhandled)?;
+                let mut output = super::super::types::error::builders::TagPolicyExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_tag_policy_exception::de_tag_policy_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_resource::UntagResourceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -122,7 +122,7 @@
             }
             tmp
         }),
-        _ => crate::operation::untag_resource::UntagResourceError::generic(generic),
+        _ => super::super::operation::untag_resource::UntagResourceError::generic(generic),
     })
 }

@@ -131,10 +131,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::untag_resource::UntagResourceOutput, crate::operation::untag_resource::UntagResourceError> {
+) -> std::result::Result<super::super::operation::untag_resource::UntagResourceOutput, super::super::operation::untag_resource::UntagResourceError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::untag_resource::builders::UntagResourceOutputBuilder::default();
+        let mut output = super::super::operation::untag_resource::builders::UntagResourceOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
