--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl RotateKeyOnDemandInput {
     /// Creates a new builder-style object to manufacture [`RotateKeyOnDemandInput`](crate::operation::rotate_key_on_demand::RotateKeyOnDemandInput).
-    pub fn builder() -> crate::operation::rotate_key_on_demand::builders::RotateKeyOnDemandInputBuilder {
-        crate::operation::rotate_key_on_demand::builders::RotateKeyOnDemandInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::rotate_key_on_demand::builders::RotateKeyOnDemandInputBuilder {
+        super::super::super::operation::rotate_key_on_demand::builders::RotateKeyOnDemandInputBuilder::default()
     }
 }

@@ -89,7 +89,7 @@
     /// Consumes the builder and constructs a [`RotateKeyOnDemandInput`](crate::operation::rotate_key_on_demand::RotateKeyOnDemandInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::rotate_key_on_demand::RotateKeyOnDemandInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::rotate_key_on_demand::RotateKeyOnDemandInput { key_id: self.key_id })
+    ) -> ::std::result::Result<super::super::super::operation::rotate_key_on_demand::RotateKeyOnDemandInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::rotate_key_on_demand::RotateKeyOnDemandInput { key_id: self.key_id })
     }
 }
