--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl DeleteUserPoolClientInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserPoolClientInput`](crate::operation::delete_user_pool_client::DeleteUserPoolClientInput).
-    pub fn builder() -> crate::operation::delete_user_pool_client::builders::DeleteUserPoolClientInputBuilder {
-        crate::operation::delete_user_pool_client::builders::DeleteUserPoolClientInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_user_pool_client::builders::DeleteUserPoolClientInputBuilder {
+        super::super::super::operation::delete_user_pool_client::builders::DeleteUserPoolClientInputBuilder::default()
     }
 }

@@ -75,9 +75,9 @@
     /// Consumes the builder and constructs a [`DeleteUserPoolClientInput`](crate::operation::delete_user_pool_client::DeleteUserPoolClientInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_user_pool_client::DeleteUserPoolClientInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_user_pool_client::DeleteUserPoolClientInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_user_pool_client::DeleteUserPoolClientInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_user_pool_client::DeleteUserPoolClientInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
         })
