--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl DetachGroupPolicyInput {
     /// Creates a new builder-style object to manufacture [`DetachGroupPolicyInput`](crate::operation::detach_group_policy::DetachGroupPolicyInput).
-    pub fn builder() -> crate::operation::detach_group_policy::builders::DetachGroupPolicyInputBuilder {
-        crate::operation::detach_group_policy::builders::DetachGroupPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::detach_group_policy::builders::DetachGroupPolicyInputBuilder {
+        super::operation::detach_group_policy::builders::DetachGroupPolicyInputBuilder::default()
     }
 }

@@ -76,8 +76,8 @@
     /// Consumes the builder and constructs a [`DetachGroupPolicyInput`](crate::operation::detach_group_policy::DetachGroupPolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::detach_group_policy::DetachGroupPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::detach_group_policy::DetachGroupPolicyInput {
+    ) -> ::std::result::Result<super::operation::detach_group_policy::DetachGroupPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::detach_group_policy::DetachGroupPolicyInput {
             group_name: self.group_name,
             policy_arn: self.policy_arn,
         })
