--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetFunctionScalingConfigInput {
     /// Creates a new builder-style object to manufacture [`GetFunctionScalingConfigInput`](crate::operation::get_function_scaling_config::GetFunctionScalingConfigInput).
-    pub fn builder() -> crate::operation::get_function_scaling_config::builders::GetFunctionScalingConfigInputBuilder {
-        crate::operation::get_function_scaling_config::builders::GetFunctionScalingConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_function_scaling_config::builders::GetFunctionScalingConfigInputBuilder {
+        super::super::super::operation::get_function_scaling_config::builders::GetFunctionScalingConfigInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_function_scaling_config::GetFunctionScalingConfigInput,
+        super::super::super::operation::get_function_scaling_config::GetFunctionScalingConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_function_scaling_config::GetFunctionScalingConfigInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_function_scaling_config::GetFunctionScalingConfigInput {
             function_name: self.function_name,
             qualifier: self.qualifier,
         })
