--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl ArchivingOptions {
     /// Creates a new builder-style object to manufacture [`ArchivingOptions`](crate::types::ArchivingOptions).
-    pub fn builder() -> crate::types::builders::ArchivingOptionsBuilder {
-        crate::types::builders::ArchivingOptionsBuilder::default()
+    pub fn builder() -> super::types::builders::ArchivingOptionsBuilder {
+        super::types::builders::ArchivingOptionsBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.archive_arn
     }
     /// Consumes the builder and constructs a [`ArchivingOptions`](crate::types::ArchivingOptions).
-    pub fn build(self) -> crate::types::ArchivingOptions {
-        crate::types::ArchivingOptions {
+    pub fn build(self) -> super::types::ArchivingOptions {
+        super::types::ArchivingOptions {
             archive_arn: self.archive_arn,
         }
     }
