--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The raw data of the attachment. It needs to be base64-encoded if you are accessing Amazon SES directly through the HTTPS interface. If you are accessing Amazon SES using an Amazon Web Services SDK, the SDK takes care of the base 64-encoding for you.</p>
     pub raw_content: ::aws_smithy_types::Blob,
     /// <p>A standard descriptor indicating how the attachment should be rendered in the email. Supported values: <code>ATTACHMENT</code> or <code>INLINE</code>.</p>
-    pub content_disposition: ::std::option::Option<crate::types::AttachmentContentDisposition>,
+    pub content_disposition: ::std::option::Option<super::types::AttachmentContentDisposition>,
     /// <p>The file name for the attachment as it will appear in the email. Amazon SES restricts certain file extensions. To ensure attachments are accepted, check the <a href="https://docs.aws.amazon.com/ses/latest/dg/mime-types.html">Unsupported attachment types</a> in the Amazon SES Developer Guide.</p>
     pub file_name: ::std::string::String,
     /// <p>A brief description of the attachment content.</p>
@@ -15,7 +15,7 @@
     /// <p>Unique identifier for the attachment, used for referencing attachments with INLINE disposition in HTML content.</p>
     pub content_id: ::std::option::Option<::std::string::String>,
     /// <p>Specifies how the attachment is encoded. Supported values: <code>BASE64</code>, <code>QUOTED_PRINTABLE</code>, <code>SEVEN_BIT</code>.</p>
-    pub content_transfer_encoding: ::std::option::Option<crate::types::AttachmentContentTransferEncoding>,
+    pub content_transfer_encoding: ::std::option::Option<super::types::AttachmentContentTransferEncoding>,
     /// <p>The MIME type of the attachment.</p><note>
     /// <p>Example: <code>application/pdf</code>, <code>image/jpeg</code></p>
     /// </note>
@@ -27,7 +27,7 @@
         &self.raw_content
     }
     /// <p>A standard descriptor indicating how the attachment should be rendered in the email. Supported values: <code>ATTACHMENT</code> or <code>INLINE</code>.</p>
-    pub fn content_disposition(&self) -> ::std::option::Option<&crate::types::AttachmentContentDisposition> {
+    pub fn content_disposition(&self) -> ::std::option::Option<&super::types::AttachmentContentDisposition> {
         self.content_disposition.as_ref()
     }
     /// <p>The file name for the attachment as it will appear in the email. Amazon SES restricts certain file extensions. To ensure attachments are accepted, check the <a href="https://docs.aws.amazon.com/ses/latest/dg/mime-types.html">Unsupported attachment types</a> in the Amazon SES Developer Guide.</p>
@@ -44,7 +44,7 @@
         self.content_id.as_deref()
     }
     /// <p>Specifies how the attachment is encoded. Supported values: <code>BASE64</code>, <code>QUOTED_PRINTABLE</code>, <code>SEVEN_BIT</code>.</p>
-    pub fn content_transfer_encoding(&self) -> ::std::option::Option<&crate::types::AttachmentContentTransferEncoding> {
+    pub fn content_transfer_encoding(&self) -> ::std::option::Option<&super::types::AttachmentContentTransferEncoding> {
         self.content_transfer_encoding.as_ref()
     }
     /// <p>The MIME type of the attachment.</p><note>
@@ -56,8 +56,8 @@
 }
 impl Attachment {
     /// Creates a new builder-style object to manufacture [`Attachment`](crate::types::Attachment).
-    pub fn builder() -> crate::types::builders::AttachmentBuilder {
-        crate::types::builders::AttachmentBuilder::default()
+    pub fn builder() -> super::types::builders::AttachmentBuilder {
+        super::types::builders::AttachmentBuilder::default()
     }
 }

@@ -66,11 +66,11 @@
 #[non_exhaustive]
 pub struct AttachmentBuilder {
     pub(crate) raw_content: ::std::option::Option<::aws_smithy_types::Blob>,
-    pub(crate) content_disposition: ::std::option::Option<crate::types::AttachmentContentDisposition>,
+    pub(crate) content_disposition: ::std::option::Option<super::types::AttachmentContentDisposition>,
     pub(crate) file_name: ::std::option::Option<::std::string::String>,
     pub(crate) content_description: ::std::option::Option<::std::string::String>,
     pub(crate) content_id: ::std::option::Option<::std::string::String>,
-    pub(crate) content_transfer_encoding: ::std::option::Option<crate::types::AttachmentContentTransferEncoding>,
+    pub(crate) content_transfer_encoding: ::std::option::Option<super::types::AttachmentContentTransferEncoding>,
     pub(crate) content_type: ::std::option::Option<::std::string::String>,
 }
 impl AttachmentBuilder {
@@ -90,17 +90,17 @@
         &self.raw_content
     }
     /// <p>A standard descriptor indicating how the attachment should be rendered in the email. Supported values: <code>ATTACHMENT</code> or <code>INLINE</code>.</p>
-    pub fn content_disposition(mut self, input: crate::types::AttachmentContentDisposition) -> Self {
+    pub fn content_disposition(mut self, input: super::types::AttachmentContentDisposition) -> Self {
         self.content_disposition = ::std::option::Option::Some(input);
         self
     }
     /// <p>A standard descriptor indicating how the attachment should be rendered in the email. Supported values: <code>ATTACHMENT</code> or <code>INLINE</code>.</p>
-    pub fn set_content_disposition(mut self, input: ::std::option::Option<crate::types::AttachmentContentDisposition>) -> Self {
+    pub fn set_content_disposition(mut self, input: ::std::option::Option<super::types::AttachmentContentDisposition>) -> Self {
         self.content_disposition = input;
         self
     }
     /// <p>A standard descriptor indicating how the attachment should be rendered in the email. Supported values: <code>ATTACHMENT</code> or <code>INLINE</code>.</p>
-    pub fn get_content_disposition(&self) -> &::std::option::Option<crate::types::AttachmentContentDisposition> {
+    pub fn get_content_disposition(&self) -> &::std::option::Option<super::types::AttachmentContentDisposition> {
         &self.content_disposition
     }
     /// <p>The file name for the attachment as it will appear in the email. Amazon SES restricts certain file extensions. To ensure attachments are accepted, check the <a href="https://docs.aws.amazon.com/ses/latest/dg/mime-types.html">Unsupported attachment types</a> in the Amazon SES Developer Guide.</p>
@@ -147,17 +147,17 @@
         &self.content_id
     }
     /// <p>Specifies how the attachment is encoded. Supported values: <code>BASE64</code>, <code>QUOTED_PRINTABLE</code>, <code>SEVEN_BIT</code>.</p>
-    pub fn content_transfer_encoding(mut self, input: crate::types::AttachmentContentTransferEncoding) -> Self {
+    pub fn content_transfer_encoding(mut self, input: super::types::AttachmentContentTransferEncoding) -> Self {
         self.content_transfer_encoding = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies how the attachment is encoded. Supported values: <code>BASE64</code>, <code>QUOTED_PRINTABLE</code>, <code>SEVEN_BIT</code>.</p>
-    pub fn set_content_transfer_encoding(mut self, input: ::std::option::Option<crate::types::AttachmentContentTransferEncoding>) -> Self {
+    pub fn set_content_transfer_encoding(mut self, input: ::std::option::Option<super::types::AttachmentContentTransferEncoding>) -> Self {
         self.content_transfer_encoding = input;
         self
     }
     /// <p>Specifies how the attachment is encoded. Supported values: <code>BASE64</code>, <code>QUOTED_PRINTABLE</code>, <code>SEVEN_BIT</code>.</p>
-    pub fn get_content_transfer_encoding(&self) -> &::std::option::Option<crate::types::AttachmentContentTransferEncoding> {
+    pub fn get_content_transfer_encoding(&self) -> &::std::option::Option<super::types::AttachmentContentTransferEncoding> {
         &self.content_transfer_encoding
     }
     /// <p>The MIME type of the attachment.</p><note>
@@ -184,8 +184,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`raw_content`](crate::types::builders::AttachmentBuilder::raw_content)
     /// - [`file_name`](crate::types::builders::AttachmentBuilder::file_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::Attachment, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Attachment {
+    pub fn build(self) -> ::std::result::Result<super::types::Attachment, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::Attachment {
             raw_content: self.raw_content.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "raw_content",
