--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The query identifier.</p>
     pub id: ::std::string::String,
     /// <p>The query namespace - e.g. <code>VDM</code></p>
-    pub namespace: crate::types::MetricNamespace,
+    pub namespace: super::types::MetricNamespace,
     /// <p>The queried metric. This can be one of the following:</p>
     /// <ul>
     /// <li>
@@ -31,9 +31,9 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> – Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub metric: crate::types::Metric,
+    pub metric: super::types::Metric,
     /// <p>An object that contains mapping between <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by.</p>
-    pub dimensions: ::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::string::String>>,
+    pub dimensions: ::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::string::String>>,
     /// <p>Represents the start date for the query interval.</p>
     pub start_date: ::aws_smithy_types::DateTime,
     /// <p>Represents the end date for the query interval.</p>
@@ -46,7 +46,7 @@
         self.id.deref()
     }
     /// <p>The query namespace - e.g. <code>VDM</code></p>
-    pub fn namespace(&self) -> &crate::types::MetricNamespace {
+    pub fn namespace(&self) -> &super::types::MetricNamespace {
         &self.namespace
     }
     /// <p>The queried metric. This can be one of the following:</p>
@@ -72,11 +72,11 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> – Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn metric(&self) -> &crate::types::Metric {
+    pub fn metric(&self) -> &super::types::Metric {
         &self.metric
     }
     /// <p>An object that contains mapping between <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by.</p>
-    pub fn dimensions(&self) -> ::std::option::Option<&::std::collections::HashMap<crate::types::MetricDimensionName, ::std::string::String>> {
+    pub fn dimensions(&self) -> ::std::option::Option<&::std::collections::HashMap<super::types::MetricDimensionName, ::std::string::String>> {
         self.dimensions.as_ref()
     }
     /// <p>Represents the start date for the query interval.</p>
@@ -90,8 +90,8 @@
 }
 impl BatchGetMetricDataQuery {
     /// Creates a new builder-style object to manufacture [`BatchGetMetricDataQuery`](crate::types::BatchGetMetricDataQuery).
-    pub fn builder() -> crate::types::builders::BatchGetMetricDataQueryBuilder {
-        crate::types::builders::BatchGetMetricDataQueryBuilder::default()
+    pub fn builder() -> super::types::builders::BatchGetMetricDataQueryBuilder {
+        super::types::builders::BatchGetMetricDataQueryBuilder::default()
     }
 }

@@ -100,9 +100,9 @@
 #[non_exhaustive]
 pub struct BatchGetMetricDataQueryBuilder {
     pub(crate) id: ::std::option::Option<::std::string::String>,
-    pub(crate) namespace: ::std::option::Option<crate::types::MetricNamespace>,
-    pub(crate) metric: ::std::option::Option<crate::types::Metric>,
-    pub(crate) dimensions: ::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::string::String>>,
+    pub(crate) namespace: ::std::option::Option<super::types::MetricNamespace>,
+    pub(crate) metric: ::std::option::Option<super::types::Metric>,
+    pub(crate) dimensions: ::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::string::String>>,
     pub(crate) start_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) end_date: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -124,17 +124,17 @@
     }
     /// <p>The query namespace - e.g. <code>VDM</code></p>
     /// This field is required.
-    pub fn namespace(mut self, input: crate::types::MetricNamespace) -> Self {
+    pub fn namespace(mut self, input: super::types::MetricNamespace) -> Self {
         self.namespace = ::std::option::Option::Some(input);
         self
     }
     /// <p>The query namespace - e.g. <code>VDM</code></p>
-    pub fn set_namespace(mut self, input: ::std::option::Option<crate::types::MetricNamespace>) -> Self {
+    pub fn set_namespace(mut self, input: ::std::option::Option<super::types::MetricNamespace>) -> Self {
         self.namespace = input;
         self
     }
     /// <p>The query namespace - e.g. <code>VDM</code></p>
-    pub fn get_namespace(&self) -> &::std::option::Option<crate::types::MetricNamespace> {
+    pub fn get_namespace(&self) -> &::std::option::Option<super::types::MetricNamespace> {
         &self.namespace
     }
     /// <p>The queried metric. This can be one of the following:</p>
@@ -161,7 +161,7 @@
     /// <p><code>DELIVERY_COMPLAINT</code> – Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn metric(mut self, input: crate::types::Metric) -> Self {
+    pub fn metric(mut self, input: super::types::Metric) -> Self {
         self.metric = ::std::option::Option::Some(input);
         self
     }
@@ -188,7 +188,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> – Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn set_metric(mut self, input: ::std::option::Option<crate::types::Metric>) -> Self {
+    pub fn set_metric(mut self, input: ::std::option::Option<super::types::Metric>) -> Self {
         self.metric = input;
         self
     }
@@ -215,7 +215,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> – Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn get_metric(&self) -> &::std::option::Option<crate::types::Metric> {
+    pub fn get_metric(&self) -> &::std::option::Option<super::types::Metric> {
         &self.metric
     }
     /// Adds a key-value pair to `dimensions`.
@@ -223,7 +223,7 @@
     /// To override the contents of this collection use [`set_dimensions`](Self::set_dimensions).
     ///
     /// <p>An object that contains mapping between <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by.</p>
-    pub fn dimensions(mut self, k: crate::types::MetricDimensionName, v: impl ::std::convert::Into<::std::string::String>) -> Self {
+    pub fn dimensions(mut self, k: super::types::MetricDimensionName, v: impl ::std::convert::Into<::std::string::String>) -> Self {
         let mut hash_map = self.dimensions.unwrap_or_default();
         hash_map.insert(k, v.into());
         self.dimensions = ::std::option::Option::Some(hash_map);
@@ -232,13 +232,13 @@
     /// <p>An object that contains mapping between <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by.</p>
     pub fn set_dimensions(
         mut self,
-        input: ::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::string::String>>,
+        input: ::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::string::String>>,
     ) -> Self {
         self.dimensions = input;
         self
     }
     /// <p>An object that contains mapping between <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by.</p>
-    pub fn get_dimensions(&self) -> &::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::string::String>> {
+    pub fn get_dimensions(&self) -> &::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::string::String>> {
         &self.dimensions
     }
     /// <p>Represents the start date for the query interval.</p>
@@ -278,8 +278,8 @@
     /// - [`metric`](crate::types::builders::BatchGetMetricDataQueryBuilder::metric)
     /// - [`start_date`](crate::types::builders::BatchGetMetricDataQueryBuilder::start_date)
     /// - [`end_date`](crate::types::builders::BatchGetMetricDataQueryBuilder::end_date)
-    pub fn build(self) -> ::std::result::Result<crate::types::BatchGetMetricDataQuery, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::BatchGetMetricDataQuery {
+    pub fn build(self) -> ::std::result::Result<super::types::BatchGetMetricDataQuery, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::BatchGetMetricDataQuery {
             id: self.id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "id",
