--- original
+++ modified
@@ -6,11 +6,11 @@
     /// <p>Represents the destination of the message, consisting of To:, CC:, and BCC: fields.</p><note>
     /// <p>Amazon SES does not support the SMTPUTF8 extension, as described in <a href="https://tools.ietf.org/html/rfc6531">RFC6531</a>. For this reason, the local part of a destination email address (the part of the email address that precedes the @ sign) may only contain <a href="https://en.wikipedia.org/wiki/Email_address#Local-part">7-bit ASCII characters</a>. If the domain part of an address (the part after the @ sign) contains non-ASCII characters, they must be encoded using Punycode, as described in <a href="https://tools.ietf.org/html/rfc3492.html">RFC3492</a>.</p>
     /// </note>
-    pub destination: ::std::option::Option<crate::types::Destination>,
+    pub destination: ::std::option::Option<super::super::types::Destination>,
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendBulkTemplatedEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub replacement_tags: ::std::option::Option<::std::vec::Vec<crate::types::MessageTag>>,
+    pub replacement_tags: ::std::option::Option<::std::vec::Vec<super::super::types::MessageTag>>,
     /// <p>The <code>ReplacementEmailContent</code> associated with a <code>BulkEmailEntry</code>.</p>
-    pub replacement_email_content: ::std::option::Option<crate::types::ReplacementEmailContent>,
+    pub replacement_email_content: ::std::option::Option<super::super::types::ReplacementEmailContent>,
     /// <p>The list of message headers associated with the <code>BulkEmailEntry</code> data type.</p>
     /// <ul>
     /// <li>
@@ -24,23 +24,23 @@
     /// <p>If the header is not defined within <code>Template</code>, it will simply be added to the email as specified in <code>BulkEmailEntry</code>.</p></li>
     /// </ul></li>
     /// </ul>
-    pub replacement_headers: ::std::option::Option<::std::vec::Vec<crate::types::MessageHeader>>,
+    pub replacement_headers: ::std::option::Option<::std::vec::Vec<super::super::types::MessageHeader>>,
 }
 impl BulkEmailEntry {
     /// <p>Represents the destination of the message, consisting of To:, CC:, and BCC: fields.</p><note>
     /// <p>Amazon SES does not support the SMTPUTF8 extension, as described in <a href="https://tools.ietf.org/html/rfc6531">RFC6531</a>. For this reason, the local part of a destination email address (the part of the email address that precedes the @ sign) may only contain <a href="https://en.wikipedia.org/wiki/Email_address#Local-part">7-bit ASCII characters</a>. If the domain part of an address (the part after the @ sign) contains non-ASCII characters, they must be encoded using Punycode, as described in <a href="https://tools.ietf.org/html/rfc3492.html">RFC3492</a>.</p>
     /// </note>
-    pub fn destination(&self) -> ::std::option::Option<&crate::types::Destination> {
+    pub fn destination(&self) -> ::std::option::Option<&super::super::types::Destination> {
         self.destination.as_ref()
     }
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendBulkTemplatedEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replacement_tags.is_none()`.
-    pub fn replacement_tags(&self) -> &[crate::types::MessageTag] {
+    pub fn replacement_tags(&self) -> &[super::super::types::MessageTag] {
         self.replacement_tags.as_deref().unwrap_or_default()
     }
     /// <p>The <code>ReplacementEmailContent</code> associated with a <code>BulkEmailEntry</code>.</p>
-    pub fn replacement_email_content(&self) -> ::std::option::Option<&crate::types::ReplacementEmailContent> {
+    pub fn replacement_email_content(&self) -> ::std::option::Option<&super::super::types::ReplacementEmailContent> {
         self.replacement_email_content.as_ref()
     }
     /// <p>The list of message headers associated with the <code>BulkEmailEntry</code> data type.</p>
@@ -58,14 +58,14 @@
     /// </ul>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replacement_headers.is_none()`.
-    pub fn replacement_headers(&self) -> &[crate::types::MessageHeader] {
+    pub fn replacement_headers(&self) -> &[super::super::types::MessageHeader] {
         self.replacement_headers.as_deref().unwrap_or_default()
     }
 }
 impl BulkEmailEntry {
     /// Creates a new builder-style object to manufacture [`BulkEmailEntry`](crate::types::BulkEmailEntry).
-    pub fn builder() -> crate::types::builders::BulkEmailEntryBuilder {
-        crate::types::builders::BulkEmailEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::BulkEmailEntryBuilder {
+        super::super::types::builders::BulkEmailEntryBuilder::default()
     }
 }

@@ -73,10 +73,10 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct BulkEmailEntryBuilder {
-    pub(crate) destination: ::std::option::Option<crate::types::Destination>,
-    pub(crate) replacement_tags: ::std::option::Option<::std::vec::Vec<crate::types::MessageTag>>,
-    pub(crate) replacement_email_content: ::std::option::Option<crate::types::ReplacementEmailContent>,
-    pub(crate) replacement_headers: ::std::option::Option<::std::vec::Vec<crate::types::MessageHeader>>,
+    pub(crate) destination: ::std::option::Option<super::super::types::Destination>,
+    pub(crate) replacement_tags: ::std::option::Option<::std::vec::Vec<super::super::types::MessageTag>>,
+    pub(crate) replacement_email_content: ::std::option::Option<super::super::types::ReplacementEmailContent>,
+    pub(crate) replacement_headers: ::std::option::Option<::std::vec::Vec<super::super::types::MessageHeader>>,
 }
 impl BulkEmailEntryBuilder {
     /// <p>Represents the destination of the message, consisting of To:, CC:, and BCC: fields.</p><note>
@@ -83,7 +83,7 @@
     /// <p>Amazon SES does not support the SMTPUTF8 extension, as described in <a href="https://tools.ietf.org/html/rfc6531">RFC6531</a>. For this reason, the local part of a destination email address (the part of the email address that precedes the @ sign) may only contain <a href="https://en.wikipedia.org/wiki/Email_address#Local-part">7-bit ASCII characters</a>. If the domain part of an address (the part after the @ sign) contains non-ASCII characters, they must be encoded using Punycode, as described in <a href="https://tools.ietf.org/html/rfc3492.html">RFC3492</a>.</p>
     /// </note>
     /// This field is required.
-    pub fn destination(mut self, input: crate::types::Destination) -> Self {
+    pub fn destination(mut self, input: super::super::types::Destination) -> Self {
         self.destination = ::std::option::Option::Some(input);
         self
     }
@@ -90,7 +90,7 @@
     /// <p>Represents the destination of the message, consisting of To:, CC:, and BCC: fields.</p><note>
     /// <p>Amazon SES does not support the SMTPUTF8 extension, as described in <a href="https://tools.ietf.org/html/rfc6531">RFC6531</a>. For this reason, the local part of a destination email address (the part of the email address that precedes the @ sign) may only contain <a href="https://en.wikipedia.org/wiki/Email_address#Local-part">7-bit ASCII characters</a>. If the domain part of an address (the part after the @ sign) contains non-ASCII characters, they must be encoded using Punycode, as described in <a href="https://tools.ietf.org/html/rfc3492.html">RFC3492</a>.</p>
     /// </note>
-    pub fn set_destination(mut self, input: ::std::option::Option<crate::types::Destination>) -> Self {
+    pub fn set_destination(mut self, input: ::std::option::Option<super::super::types::Destination>) -> Self {
         self.destination = input;
         self
     }
@@ -97,7 +97,7 @@
     /// <p>Represents the destination of the message, consisting of To:, CC:, and BCC: fields.</p><note>
     /// <p>Amazon SES does not support the SMTPUTF8 extension, as described in <a href="https://tools.ietf.org/html/rfc6531">RFC6531</a>. For this reason, the local part of a destination email address (the part of the email address that precedes the @ sign) may only contain <a href="https://en.wikipedia.org/wiki/Email_address#Local-part">7-bit ASCII characters</a>. If the domain part of an address (the part after the @ sign) contains non-ASCII characters, they must be encoded using Punycode, as described in <a href="https://tools.ietf.org/html/rfc3492.html">RFC3492</a>.</p>
     /// </note>
-    pub fn get_destination(&self) -> &::std::option::Option<crate::types::Destination> {
+    pub fn get_destination(&self) -> &::std::option::Option<super::super::types::Destination> {
         &self.destination
     }
     /// Appends an item to `replacement_tags`.
@@ -105,7 +105,7 @@
     /// To override the contents of this collection use [`set_replacement_tags`](Self::set_replacement_tags).
     ///
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendBulkTemplatedEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub fn replacement_tags(mut self, input: crate::types::MessageTag) -> Self {
+    pub fn replacement_tags(mut self, input: super::super::types::MessageTag) -> Self {
         let mut v = self.replacement_tags.unwrap_or_default();
         v.push(input);
         self.replacement_tags = ::std::option::Option::Some(v);
@@ -112,26 +112,26 @@
         self
     }
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendBulkTemplatedEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub fn set_replacement_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::MessageTag>>) -> Self {
+    pub fn set_replacement_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::MessageTag>>) -> Self {
         self.replacement_tags = input;
         self
     }
     /// <p>A list of tags, in the form of name/value pairs, to apply to an email that you send using the <code>SendBulkTemplatedEmail</code> operation. Tags correspond to characteristics of the email that you define, so that you can publish email sending events.</p>
-    pub fn get_replacement_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::MessageTag>> {
+    pub fn get_replacement_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::MessageTag>> {
         &self.replacement_tags
     }
     /// <p>The <code>ReplacementEmailContent</code> associated with a <code>BulkEmailEntry</code>.</p>
-    pub fn replacement_email_content(mut self, input: crate::types::ReplacementEmailContent) -> Self {
+    pub fn replacement_email_content(mut self, input: super::super::types::ReplacementEmailContent) -> Self {
         self.replacement_email_content = ::std::option::Option::Some(input);
         self
     }
     /// <p>The <code>ReplacementEmailContent</code> associated with a <code>BulkEmailEntry</code>.</p>
-    pub fn set_replacement_email_content(mut self, input: ::std::option::Option<crate::types::ReplacementEmailContent>) -> Self {
+    pub fn set_replacement_email_content(mut self, input: ::std::option::Option<super::super::types::ReplacementEmailContent>) -> Self {
         self.replacement_email_content = input;
         self
     }
     /// <p>The <code>ReplacementEmailContent</code> associated with a <code>BulkEmailEntry</code>.</p>
-    pub fn get_replacement_email_content(&self) -> &::std::option::Option<crate::types::ReplacementEmailContent> {
+    pub fn get_replacement_email_content(&self) -> &::std::option::Option<super::super::types::ReplacementEmailContent> {
         &self.replacement_email_content
     }
     /// Appends an item to `replacement_headers`.
@@ -151,7 +151,7 @@
     /// <p>If the header is not defined within <code>Template</code>, it will simply be added to the email as specified in <code>BulkEmailEntry</code>.</p></li>
     /// </ul></li>
     /// </ul>
-    pub fn replacement_headers(mut self, input: crate::types::MessageHeader) -> Self {
+    pub fn replacement_headers(mut self, input: super::super::types::MessageHeader) -> Self {
         let mut v = self.replacement_headers.unwrap_or_default();
         v.push(input);
         self.replacement_headers = ::std::option::Option::Some(v);
@@ -170,7 +170,7 @@
     /// <p>If the header is not defined within <code>Template</code>, it will simply be added to the email as specified in <code>BulkEmailEntry</code>.</p></li>
     /// </ul></li>
     /// </ul>
-    pub fn set_replacement_headers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::MessageHeader>>) -> Self {
+    pub fn set_replacement_headers(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::MessageHeader>>) -> Self {
         self.replacement_headers = input;
         self
     }
@@ -187,12 +187,12 @@
     /// <p>If the header is not defined within <code>Template</code>, it will simply be added to the email as specified in <code>BulkEmailEntry</code>.</p></li>
     /// </ul></li>
     /// </ul>
-    pub fn get_replacement_headers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::MessageHeader>> {
+    pub fn get_replacement_headers(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::MessageHeader>> {
         &self.replacement_headers
     }
     /// Consumes the builder and constructs a [`BulkEmailEntry`](crate::types::BulkEmailEntry).
-    pub fn build(self) -> crate::types::BulkEmailEntry {
-        crate::types::BulkEmailEntry {
+    pub fn build(self) -> super::super::types::BulkEmailEntry {
+        super::super::types::BulkEmailEntry {
             destination: self.destination,
             replacement_tags: self.replacement_tags,
             replacement_email_content: self.replacement_email_content,
