--- original
+++ modified
@@ -13,7 +13,7 @@
     /// </ul>
     pub dimension_name: ::std::string::String,
     /// <p>The location where the Amazon SES API v2 finds the value of a dimension to publish to Amazon CloudWatch. To use the message tags that you specify using an <code>X-SES-MESSAGE-TAGS</code> header or a parameter to the <code>SendEmail</code> or <code>SendRawEmail</code> API, choose <code>messageTag</code>. To use your own email headers, choose <code>emailHeader</code>. To use link tags, choose <code>linkTags</code>.</p>
-    pub dimension_value_source: crate::types::DimensionValueSource,
+    pub dimension_value_source: super::super::types::DimensionValueSource,
     /// <p>The default value of the dimension that is published to Amazon CloudWatch if you don't provide the value of the dimension when you send an email. This value has to meet the following criteria:</p>
     /// <ul>
     /// <li>
@@ -36,7 +36,7 @@
         self.dimension_name.deref()
     }
     /// <p>The location where the Amazon SES API v2 finds the value of a dimension to publish to Amazon CloudWatch. To use the message tags that you specify using an <code>X-SES-MESSAGE-TAGS</code> header or a parameter to the <code>SendEmail</code> or <code>SendRawEmail</code> API, choose <code>messageTag</code>. To use your own email headers, choose <code>emailHeader</code>. To use link tags, choose <code>linkTags</code>.</p>
-    pub fn dimension_value_source(&self) -> &crate::types::DimensionValueSource {
+    pub fn dimension_value_source(&self) -> &super::super::types::DimensionValueSource {
         &self.dimension_value_source
     }
     /// <p>The default value of the dimension that is published to Amazon CloudWatch if you don't provide the value of the dimension when you send an email. This value has to meet the following criteria:</p>
@@ -53,8 +53,8 @@
 }
 impl CloudWatchDimensionConfiguration {
     /// Creates a new builder-style object to manufacture [`CloudWatchDimensionConfiguration`](crate::types::CloudWatchDimensionConfiguration).
-    pub fn builder() -> crate::types::builders::CloudWatchDimensionConfigurationBuilder {
-        crate::types::builders::CloudWatchDimensionConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::CloudWatchDimensionConfigurationBuilder {
+        super::super::types::builders::CloudWatchDimensionConfigurationBuilder::default()
     }
 }

@@ -63,7 +63,7 @@
 #[non_exhaustive]
 pub struct CloudWatchDimensionConfigurationBuilder {
     pub(crate) dimension_name: ::std::option::Option<::std::string::String>,
-    pub(crate) dimension_value_source: ::std::option::Option<crate::types::DimensionValueSource>,
+    pub(crate) dimension_value_source: ::std::option::Option<super::super::types::DimensionValueSource>,
     pub(crate) default_dimension_value: ::std::option::Option<::std::string::String>,
 }
 impl CloudWatchDimensionConfigurationBuilder {
@@ -102,17 +102,17 @@
     }
     /// <p>The location where the Amazon SES API v2 finds the value of a dimension to publish to Amazon CloudWatch. To use the message tags that you specify using an <code>X-SES-MESSAGE-TAGS</code> header or a parameter to the <code>SendEmail</code> or <code>SendRawEmail</code> API, choose <code>messageTag</code>. To use your own email headers, choose <code>emailHeader</code>. To use link tags, choose <code>linkTags</code>.</p>
     /// This field is required.
-    pub fn dimension_value_source(mut self, input: crate::types::DimensionValueSource) -> Self {
+    pub fn dimension_value_source(mut self, input: super::super::types::DimensionValueSource) -> Self {
         self.dimension_value_source = ::std::option::Option::Some(input);
         self
     }
     /// <p>The location where the Amazon SES API v2 finds the value of a dimension to publish to Amazon CloudWatch. To use the message tags that you specify using an <code>X-SES-MESSAGE-TAGS</code> header or a parameter to the <code>SendEmail</code> or <code>SendRawEmail</code> API, choose <code>messageTag</code>. To use your own email headers, choose <code>emailHeader</code>. To use link tags, choose <code>linkTags</code>.</p>
-    pub fn set_dimension_value_source(mut self, input: ::std::option::Option<crate::types::DimensionValueSource>) -> Self {
+    pub fn set_dimension_value_source(mut self, input: ::std::option::Option<super::super::types::DimensionValueSource>) -> Self {
         self.dimension_value_source = input;
         self
     }
     /// <p>The location where the Amazon SES API v2 finds the value of a dimension to publish to Amazon CloudWatch. To use the message tags that you specify using an <code>X-SES-MESSAGE-TAGS</code> header or a parameter to the <code>SendEmail</code> or <code>SendRawEmail</code> API, choose <code>messageTag</code>. To use your own email headers, choose <code>emailHeader</code>. To use link tags, choose <code>linkTags</code>.</p>
-    pub fn get_dimension_value_source(&self) -> &::std::option::Option<crate::types::DimensionValueSource> {
+    pub fn get_dimension_value_source(&self) -> &::std::option::Option<super::super::types::DimensionValueSource> {
         &self.dimension_value_source
     }
     /// <p>The default value of the dimension that is published to Amazon CloudWatch if you don't provide the value of the dimension when you send an email. This value has to meet the following criteria:</p>
@@ -153,8 +153,8 @@
     /// - [`dimension_name`](crate::types::builders::CloudWatchDimensionConfigurationBuilder::dimension_name)
     /// - [`dimension_value_source`](crate::types::builders::CloudWatchDimensionConfigurationBuilder::dimension_value_source)
     /// - [`default_dimension_value`](crate::types::builders::CloudWatchDimensionConfigurationBuilder::default_dimension_value)
-    pub fn build(self) -> ::std::result::Result<crate::types::CloudWatchDimensionConfiguration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::CloudWatchDimensionConfiguration {
+    pub fn build(self) -> ::std::result::Result<super::super::types::CloudWatchDimensionConfiguration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::CloudWatchDimensionConfiguration {
             dimension_name: self.dimension_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "dimension_name",
