--- original
+++ modified
@@ -6,18 +6,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ConfigurationOverrides {
     /// <p>An object that overrides the open and click tracking settings that would otherwise apply to the message.</p>
-    pub tracking: ::std::option::Option<crate::types::TrackingConfigurationOverrides>,
+    pub tracking: ::std::option::Option<super::types::TrackingConfigurationOverrides>,
 }
 impl ConfigurationOverrides {
     /// <p>An object that overrides the open and click tracking settings that would otherwise apply to the message.</p>
-    pub fn tracking(&self) -> ::std::option::Option<&crate::types::TrackingConfigurationOverrides> {
+    pub fn tracking(&self) -> ::std::option::Option<&super::types::TrackingConfigurationOverrides> {
         self.tracking.as_ref()
     }
 }
 impl ConfigurationOverrides {
     /// Creates a new builder-style object to manufacture [`ConfigurationOverrides`](crate::types::ConfigurationOverrides).
-    pub fn builder() -> crate::types::builders::ConfigurationOverridesBuilder {
-        crate::types::builders::ConfigurationOverridesBuilder::default()
+    pub fn builder() -> super::types::builders::ConfigurationOverridesBuilder {
+        super::types::builders::ConfigurationOverridesBuilder::default()
     }
 }

@@ -25,25 +25,25 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ConfigurationOverridesBuilder {
-    pub(crate) tracking: ::std::option::Option<crate::types::TrackingConfigurationOverrides>,
+    pub(crate) tracking: ::std::option::Option<super::types::TrackingConfigurationOverrides>,
 }
 impl ConfigurationOverridesBuilder {
     /// <p>An object that overrides the open and click tracking settings that would otherwise apply to the message.</p>
-    pub fn tracking(mut self, input: crate::types::TrackingConfigurationOverrides) -> Self {
+    pub fn tracking(mut self, input: super::types::TrackingConfigurationOverrides) -> Self {
         self.tracking = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that overrides the open and click tracking settings that would otherwise apply to the message.</p>
-    pub fn set_tracking(mut self, input: ::std::option::Option<crate::types::TrackingConfigurationOverrides>) -> Self {
+    pub fn set_tracking(mut self, input: ::std::option::Option<super::types::TrackingConfigurationOverrides>) -> Self {
         self.tracking = input;
         self
     }
     /// <p>An object that overrides the open and click tracking settings that would otherwise apply to the message.</p>
-    pub fn get_tracking(&self) -> &::std::option::Option<crate::types::TrackingConfigurationOverrides> {
+    pub fn get_tracking(&self) -> &::std::option::Option<super::types::TrackingConfigurationOverrides> {
         &self.tracking
     }
     /// Consumes the builder and constructs a [`ConfigurationOverrides`](crate::types::ConfigurationOverrides).
-    pub fn build(self) -> crate::types::ConfigurationOverrides {
-        crate::types::ConfigurationOverrides { tracking: self.tracking }
+    pub fn build(self) -> super::types::ConfigurationOverrides {
+        super::types::ConfigurationOverrides { tracking: self.tracking }
     }
 }
