--- original
+++ modified
@@ -39,8 +39,8 @@
 }
 impl CustomVerificationEmailTemplateMetadata {
     /// Creates a new builder-style object to manufacture [`CustomVerificationEmailTemplateMetadata`](crate::types::CustomVerificationEmailTemplateMetadata).
-    pub fn builder() -> crate::types::builders::CustomVerificationEmailTemplateMetadataBuilder {
-        crate::types::builders::CustomVerificationEmailTemplateMetadataBuilder::default()
+    pub fn builder() -> super::types::builders::CustomVerificationEmailTemplateMetadataBuilder {
+        super::types::builders::CustomVerificationEmailTemplateMetadataBuilder::default()
     }
 }

@@ -126,8 +126,8 @@
         &self.failure_redirection_url
     }
     /// Consumes the builder and constructs a [`CustomVerificationEmailTemplateMetadata`](crate::types::CustomVerificationEmailTemplateMetadata).
-    pub fn build(self) -> crate::types::CustomVerificationEmailTemplateMetadata {
-        crate::types::CustomVerificationEmailTemplateMetadata {
+    pub fn build(self) -> super::types::CustomVerificationEmailTemplateMetadata {
+        super::types::CustomVerificationEmailTemplateMetadata {
             template_name: self.template_name,
             from_email_address: self.from_email_address,
             template_subject: self.template_subject,
