--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <li>
     /// <p><code>DISABLED</code> – Amazon SES disables engagement metrics for the configuration set.</p></li>
     /// </ul>
-    pub engagement_metrics: ::std::option::Option<crate::types::FeatureStatus>,
+    pub engagement_metrics: ::std::option::Option<super::types::FeatureStatus>,
 }
 impl DashboardOptions {
     /// <p>Specifies the status of your VDM engagement metrics collection. Can be one of the following:</p>
@@ -21,14 +21,14 @@
     /// <li>
     /// <p><code>DISABLED</code> – Amazon SES disables engagement metrics for the configuration set.</p></li>
     /// </ul>
-    pub fn engagement_metrics(&self) -> ::std::option::Option<&crate::types::FeatureStatus> {
+    pub fn engagement_metrics(&self) -> ::std::option::Option<&super::types::FeatureStatus> {
         self.engagement_metrics.as_ref()
     }
 }
 impl DashboardOptions {
     /// Creates a new builder-style object to manufacture [`DashboardOptions`](crate::types::DashboardOptions).
-    pub fn builder() -> crate::types::builders::DashboardOptionsBuilder {
-        crate::types::builders::DashboardOptionsBuilder::default()
+    pub fn builder() -> super::types::builders::DashboardOptionsBuilder {
+        super::types::builders::DashboardOptionsBuilder::default()
     }
 }

@@ -36,7 +36,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct DashboardOptionsBuilder {
-    pub(crate) engagement_metrics: ::std::option::Option<crate::types::FeatureStatus>,
+    pub(crate) engagement_metrics: ::std::option::Option<super::types::FeatureStatus>,
 }
 impl DashboardOptionsBuilder {
     /// <p>Specifies the status of your VDM engagement metrics collection. Can be one of the following:</p>
@@ -46,7 +46,7 @@
     /// <li>
     /// <p><code>DISABLED</code> – Amazon SES disables engagement metrics for the configuration set.</p></li>
     /// </ul>
-    pub fn engagement_metrics(mut self, input: crate::types::FeatureStatus) -> Self {
+    pub fn engagement_metrics(mut self, input: super::types::FeatureStatus) -> Self {
         self.engagement_metrics = ::std::option::Option::Some(input);
         self
     }
@@ -57,7 +57,7 @@
     /// <li>
     /// <p><code>DISABLED</code> – Amazon SES disables engagement metrics for the configuration set.</p></li>
     /// </ul>
-    pub fn set_engagement_metrics(mut self, input: ::std::option::Option<crate::types::FeatureStatus>) -> Self {
+    pub fn set_engagement_metrics(mut self, input: ::std::option::Option<super::types::FeatureStatus>) -> Self {
         self.engagement_metrics = input;
         self
     }
@@ -68,12 +68,12 @@
     /// <li>
     /// <p><code>DISABLED</code> – Amazon SES disables engagement metrics for the configuration set.</p></li>
     /// </ul>
-    pub fn get_engagement_metrics(&self) -> &::std::option::Option<crate::types::FeatureStatus> {
+    pub fn get_engagement_metrics(&self) -> &::std::option::Option<super::types::FeatureStatus> {
         &self.engagement_metrics
     }
     /// Consumes the builder and constructs a [`DashboardOptions`](crate::types::DashboardOptions).
-    pub fn build(self) -> crate::types::DashboardOptions {
-        crate::types::DashboardOptions {
+    pub fn build(self) -> super::types::DashboardOptions {
+        super::types::DashboardOptions {
             engagement_metrics: self.engagement_metrics,
         }
     }
