--- original
+++ modified
@@ -16,7 +16,7 @@
     /// <li>
     /// <p><code>NOT_APPLICABLE</code> – The warm-up status doesn't apply to this IP address. This status is used for IP addresses in managed dedicated IP pools, where Amazon SES automatically handles the warm-up process.</p></li>
     /// </ul>
-    pub warmup_status: crate::types::WarmupStatus,
+    pub warmup_status: super::types::WarmupStatus,
     /// <p>Indicates the progress of your dedicated IP warm-up:</p>
     /// <ul>
     /// <li>
@@ -43,7 +43,7 @@
     /// <li>
     /// <p><code>NOT_APPLICABLE</code> – The warm-up status doesn't apply to this IP address. This status is used for IP addresses in managed dedicated IP pools, where Amazon SES automatically handles the warm-up process.</p></li>
     /// </ul>
-    pub fn warmup_status(&self) -> &crate::types::WarmupStatus {
+    pub fn warmup_status(&self) -> &super::types::WarmupStatus {
         &self.warmup_status
     }
     /// <p>Indicates the progress of your dedicated IP warm-up:</p>
@@ -63,8 +63,8 @@
 }
 impl DedicatedIp {
     /// Creates a new builder-style object to manufacture [`DedicatedIp`](crate::types::DedicatedIp).
-    pub fn builder() -> crate::types::builders::DedicatedIpBuilder {
-        crate::types::builders::DedicatedIpBuilder::default()
+    pub fn builder() -> super::types::builders::DedicatedIpBuilder {
+        super::types::builders::DedicatedIpBuilder::default()
     }
 }

@@ -73,7 +73,7 @@
 #[non_exhaustive]
 pub struct DedicatedIpBuilder {
     pub(crate) ip: ::std::option::Option<::std::string::String>,
-    pub(crate) warmup_status: ::std::option::Option<crate::types::WarmupStatus>,
+    pub(crate) warmup_status: ::std::option::Option<super::types::WarmupStatus>,
     pub(crate) warmup_percentage: ::std::option::Option<i32>,
     pub(crate) pool_name: ::std::option::Option<::std::string::String>,
 }
@@ -103,7 +103,7 @@
     /// <p><code>NOT_APPLICABLE</code> – The warm-up status doesn't apply to this IP address. This status is used for IP addresses in managed dedicated IP pools, where Amazon SES automatically handles the warm-up process.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn warmup_status(mut self, input: crate::types::WarmupStatus) -> Self {
+    pub fn warmup_status(mut self, input: super::types::WarmupStatus) -> Self {
         self.warmup_status = ::std::option::Option::Some(input);
         self
     }
@@ -116,7 +116,7 @@
     /// <li>
     /// <p><code>NOT_APPLICABLE</code> – The warm-up status doesn't apply to this IP address. This status is used for IP addresses in managed dedicated IP pools, where Amazon SES automatically handles the warm-up process.</p></li>
     /// </ul>
-    pub fn set_warmup_status(mut self, input: ::std::option::Option<crate::types::WarmupStatus>) -> Self {
+    pub fn set_warmup_status(mut self, input: ::std::option::Option<super::types::WarmupStatus>) -> Self {
         self.warmup_status = input;
         self
     }
@@ -129,7 +129,7 @@
     /// <li>
     /// <p><code>NOT_APPLICABLE</code> – The warm-up status doesn't apply to this IP address. This status is used for IP addresses in managed dedicated IP pools, where Amazon SES automatically handles the warm-up process.</p></li>
     /// </ul>
-    pub fn get_warmup_status(&self) -> &::std::option::Option<crate::types::WarmupStatus> {
+    pub fn get_warmup_status(&self) -> &::std::option::Option<super::types::WarmupStatus> {
         &self.warmup_status
     }
     /// <p>Indicates the progress of your dedicated IP warm-up:</p>
@@ -184,8 +184,8 @@
     /// - [`ip`](crate::types::builders::DedicatedIpBuilder::ip)
     /// - [`warmup_status`](crate::types::builders::DedicatedIpBuilder::warmup_status)
     /// - [`warmup_percentage`](crate::types::builders::DedicatedIpBuilder::warmup_percentage)
-    pub fn build(self) -> ::std::result::Result<crate::types::DedicatedIp, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::DedicatedIp {
+    pub fn build(self) -> ::std::result::Result<super::types::DedicatedIp, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::DedicatedIp {
             ip: self.ip.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "ip",
