--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <li>
     /// <p><code>MANAGED</code> – A dedicated IP pool where the reputation and number of IPs are automatically managed by Amazon SES.</p></li>
     /// </ul>
-    pub scaling_mode: crate::types::ScalingMode,
+    pub scaling_mode: super::types::ScalingMode,
 }
 impl DedicatedIpPool {
     /// <p>The name of the dedicated IP pool.</p>
@@ -28,14 +28,14 @@
     /// <li>
     /// <p><code>MANAGED</code> – A dedicated IP pool where the reputation and number of IPs are automatically managed by Amazon SES.</p></li>
     /// </ul>
-    pub fn scaling_mode(&self) -> &crate::types::ScalingMode {
+    pub fn scaling_mode(&self) -> &super::types::ScalingMode {
         &self.scaling_mode
     }
 }
 impl DedicatedIpPool {
     /// Creates a new builder-style object to manufacture [`DedicatedIpPool`](crate::types::DedicatedIpPool).
-    pub fn builder() -> crate::types::builders::DedicatedIpPoolBuilder {
-        crate::types::builders::DedicatedIpPoolBuilder::default()
+    pub fn builder() -> super::types::builders::DedicatedIpPoolBuilder {
+        super::types::builders::DedicatedIpPoolBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
 #[non_exhaustive]
 pub struct DedicatedIpPoolBuilder {
     pub(crate) pool_name: ::std::option::Option<::std::string::String>,
-    pub(crate) scaling_mode: ::std::option::Option<crate::types::ScalingMode>,
+    pub(crate) scaling_mode: ::std::option::Option<super::types::ScalingMode>,
 }
 impl DedicatedIpPoolBuilder {
     /// <p>The name of the dedicated IP pool.</p>
@@ -70,7 +70,7 @@
     /// <p><code>MANAGED</code> – A dedicated IP pool where the reputation and number of IPs are automatically managed by Amazon SES.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn scaling_mode(mut self, input: crate::types::ScalingMode) -> Self {
+    pub fn scaling_mode(mut self, input: super::types::ScalingMode) -> Self {
         self.scaling_mode = ::std::option::Option::Some(input);
         self
     }
@@ -81,7 +81,7 @@
     /// <li>
     /// <p><code>MANAGED</code> – A dedicated IP pool where the reputation and number of IPs are automatically managed by Amazon SES.</p></li>
     /// </ul>
-    pub fn set_scaling_mode(mut self, input: ::std::option::Option<crate::types::ScalingMode>) -> Self {
+    pub fn set_scaling_mode(mut self, input: ::std::option::Option<super::types::ScalingMode>) -> Self {
         self.scaling_mode = input;
         self
     }
@@ -92,7 +92,7 @@
     /// <li>
     /// <p><code>MANAGED</code> – A dedicated IP pool where the reputation and number of IPs are automatically managed by Amazon SES.</p></li>
     /// </ul>
-    pub fn get_scaling_mode(&self) -> &::std::option::Option<crate::types::ScalingMode> {
+    pub fn get_scaling_mode(&self) -> &::std::option::Option<super::types::ScalingMode> {
         &self.scaling_mode
     }
     /// Consumes the builder and constructs a [`DedicatedIpPool`](crate::types::DedicatedIpPool).
@@ -99,8 +99,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`pool_name`](crate::types::builders::DedicatedIpPoolBuilder::pool_name)
     /// - [`scaling_mode`](crate::types::builders::DedicatedIpPoolBuilder::scaling_mode)
-    pub fn build(self) -> ::std::result::Result<crate::types::DedicatedIpPool, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::DedicatedIpPool {
+    pub fn build(self) -> ::std::result::Result<super::types::DedicatedIpPool, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::DedicatedIpPool {
             pool_name: self.pool_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "pool_name",
