--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct DeliveryOptions {
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub tls_policy: ::std::option::Option<crate::types::TlsPolicy>,
+    pub tls_policy: ::std::option::Option<super::super::types::TlsPolicy>,
     /// <p>The name of the dedicated IP pool to associate with the configuration set.</p>
     pub sending_pool_name: ::std::option::Option<::std::string::String>,
     /// <p>The maximum amount of time, in seconds, that Amazon SES API v2 will attempt delivery of email. If specified, the value must greater than or equal to 300 seconds (5 minutes) and less than or equal to 50400 seconds (840 minutes).</p>
@@ -13,7 +13,7 @@
 }
 impl DeliveryOptions {
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn tls_policy(&self) -> ::std::option::Option<&crate::types::TlsPolicy> {
+    pub fn tls_policy(&self) -> ::std::option::Option<&super::super::types::TlsPolicy> {
         self.tls_policy.as_ref()
     }
     /// <p>The name of the dedicated IP pool to associate with the configuration set.</p>
@@ -27,8 +27,8 @@
 }
 impl DeliveryOptions {
     /// Creates a new builder-style object to manufacture [`DeliveryOptions`](crate::types::DeliveryOptions).
-    pub fn builder() -> crate::types::builders::DeliveryOptionsBuilder {
-        crate::types::builders::DeliveryOptionsBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeliveryOptionsBuilder {
+        super::super::types::builders::DeliveryOptionsBuilder::default()
     }
 }

@@ -36,23 +36,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct DeliveryOptionsBuilder {
-    pub(crate) tls_policy: ::std::option::Option<crate::types::TlsPolicy>,
+    pub(crate) tls_policy: ::std::option::Option<super::super::types::TlsPolicy>,
     pub(crate) sending_pool_name: ::std::option::Option<::std::string::String>,
     pub(crate) max_delivery_seconds: ::std::option::Option<i64>,
 }
 impl DeliveryOptionsBuilder {
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn tls_policy(mut self, input: crate::types::TlsPolicy) -> Self {
+    pub fn tls_policy(mut self, input: super::super::types::TlsPolicy) -> Self {
         self.tls_policy = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn set_tls_policy(mut self, input: ::std::option::Option<crate::types::TlsPolicy>) -> Self {
+    pub fn set_tls_policy(mut self, input: ::std::option::Option<super::super::types::TlsPolicy>) -> Self {
         self.tls_policy = input;
         self
     }
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn get_tls_policy(&self) -> &::std::option::Option<crate::types::TlsPolicy> {
+    pub fn get_tls_policy(&self) -> &::std::option::Option<super::super::types::TlsPolicy> {
         &self.tls_policy
     }
     /// <p>The name of the dedicated IP pool to associate with the configuration set.</p>
@@ -84,8 +84,8 @@
         &self.max_delivery_seconds
     }
     /// Consumes the builder and constructs a [`DeliveryOptions`](crate::types::DeliveryOptions).
-    pub fn build(self) -> crate::types::DeliveryOptions {
-        crate::types::DeliveryOptions {
+    pub fn build(self) -> super::super::types::DeliveryOptions {
+        super::super::types::DeliveryOptions {
             tls_policy: self.tls_policy,
             sending_pool_name: self.sending_pool_name,
             max_delivery_seconds: self.max_delivery_seconds,
