--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>The private key must use 1024 or 2048-bit RSA encryption, and must be encoded using base64 encoding.</p>
     pub domain_signing_private_key: ::std::option::Option<::std::string::String>,
     /// <p>\[Easy DKIM\] The key length of the future DKIM key pair to be generated. This can be changed at most once per day.</p>
-    pub next_signing_key_length: ::std::option::Option<crate::types::DkimSigningKeyLength>,
+    pub next_signing_key_length: ::std::option::Option<super::super::types::DkimSigningKeyLength>,
     /// <p>The attribute to use for configuring DKIM for the identity depends on the operation:</p>
     /// <ol>
     /// <li>
@@ -90,7 +90,7 @@
     /// <li>
     /// <p><code>AWS_SES_US_GOV_WEST_1</code> – Configure DKIM for the identity by replicating from a parent identity in AWS GovCloud (US-West) region using Deterministic Easy-DKIM (DEED).</p></li>
     /// </ul>
-    pub domain_signing_attributes_origin: ::std::option::Option<crate::types::DkimSigningAttributesOrigin>,
+    pub domain_signing_attributes_origin: ::std::option::Option<super::super::types::DkimSigningAttributesOrigin>,
 }
 impl DkimSigningAttributes {
     /// <p>\[Bring Your Own DKIM\] A string that's used to identify a public key in the DNS configuration for a domain.</p>
@@ -103,7 +103,7 @@
         self.domain_signing_private_key.as_deref()
     }
     /// <p>\[Easy DKIM\] The key length of the future DKIM key pair to be generated. This can be changed at most once per day.</p>
-    pub fn next_signing_key_length(&self) -> ::std::option::Option<&crate::types::DkimSigningKeyLength> {
+    pub fn next_signing_key_length(&self) -> ::std::option::Option<&super::super::types::DkimSigningKeyLength> {
         self.next_signing_key_length.as_ref()
     }
     /// <p>The attribute to use for configuring DKIM for the identity depends on the operation:</p>
@@ -185,7 +185,7 @@
     /// <li>
     /// <p><code>AWS_SES_US_GOV_WEST_1</code> – Configure DKIM for the identity by replicating from a parent identity in AWS GovCloud (US-West) region using Deterministic Easy-DKIM (DEED).</p></li>
     /// </ul>
-    pub fn domain_signing_attributes_origin(&self) -> ::std::option::Option<&crate::types::DkimSigningAttributesOrigin> {
+    pub fn domain_signing_attributes_origin(&self) -> ::std::option::Option<&super::super::types::DkimSigningAttributesOrigin> {
         self.domain_signing_attributes_origin.as_ref()
     }
 }
@@ -201,8 +201,8 @@
 }
 impl DkimSigningAttributes {
     /// Creates a new builder-style object to manufacture [`DkimSigningAttributes`](crate::types::DkimSigningAttributes).
-    pub fn builder() -> crate::types::builders::DkimSigningAttributesBuilder {
-        crate::types::builders::DkimSigningAttributesBuilder::default()
+    pub fn builder() -> super::super::types::builders::DkimSigningAttributesBuilder {
+        super::super::types::builders::DkimSigningAttributesBuilder::default()
     }
 }

@@ -212,8 +212,8 @@
 pub struct DkimSigningAttributesBuilder {
     pub(crate) domain_signing_selector: ::std::option::Option<::std::string::String>,
     pub(crate) domain_signing_private_key: ::std::option::Option<::std::string::String>,
-    pub(crate) next_signing_key_length: ::std::option::Option<crate::types::DkimSigningKeyLength>,
-    pub(crate) domain_signing_attributes_origin: ::std::option::Option<crate::types::DkimSigningAttributesOrigin>,
+    pub(crate) next_signing_key_length: ::std::option::Option<super::super::types::DkimSigningKeyLength>,
+    pub(crate) domain_signing_attributes_origin: ::std::option::Option<super::super::types::DkimSigningAttributesOrigin>,
 }
 impl DkimSigningAttributesBuilder {
     /// <p>\[Bring Your Own DKIM\] A string that's used to identify a public key in the DNS configuration for a domain.</p>
@@ -248,17 +248,17 @@
         &self.domain_signing_private_key
     }
     /// <p>\[Easy DKIM\] The key length of the future DKIM key pair to be generated. This can be changed at most once per day.</p>
-    pub fn next_signing_key_length(mut self, input: crate::types::DkimSigningKeyLength) -> Self {
+    pub fn next_signing_key_length(mut self, input: super::super::types::DkimSigningKeyLength) -> Self {
         self.next_signing_key_length = ::std::option::Option::Some(input);
         self
     }
     /// <p>\[Easy DKIM\] The key length of the future DKIM key pair to be generated. This can be changed at most once per day.</p>
-    pub fn set_next_signing_key_length(mut self, input: ::std::option::Option<crate::types::DkimSigningKeyLength>) -> Self {
+    pub fn set_next_signing_key_length(mut self, input: ::std::option::Option<super::super::types::DkimSigningKeyLength>) -> Self {
         self.next_signing_key_length = input;
         self
     }
     /// <p>\[Easy DKIM\] The key length of the future DKIM key pair to be generated. This can be changed at most once per day.</p>
-    pub fn get_next_signing_key_length(&self) -> &::std::option::Option<crate::types::DkimSigningKeyLength> {
+    pub fn get_next_signing_key_length(&self) -> &::std::option::Option<super::super::types::DkimSigningKeyLength> {
         &self.next_signing_key_length
     }
     /// <p>The attribute to use for configuring DKIM for the identity depends on the operation:</p>
@@ -340,7 +340,7 @@
     /// <li>
     /// <p><code>AWS_SES_US_GOV_WEST_1</code> – Configure DKIM for the identity by replicating from a parent identity in AWS GovCloud (US-West) region using Deterministic Easy-DKIM (DEED).</p></li>
     /// </ul>
-    pub fn domain_signing_attributes_origin(mut self, input: crate::types::DkimSigningAttributesOrigin) -> Self {
+    pub fn domain_signing_attributes_origin(mut self, input: super::super::types::DkimSigningAttributesOrigin) -> Self {
         self.domain_signing_attributes_origin = ::std::option::Option::Some(input);
         self
     }
@@ -423,7 +423,7 @@
     /// <li>
     /// <p><code>AWS_SES_US_GOV_WEST_1</code> – Configure DKIM for the identity by replicating from a parent identity in AWS GovCloud (US-West) region using Deterministic Easy-DKIM (DEED).</p></li>
     /// </ul>
-    pub fn set_domain_signing_attributes_origin(mut self, input: ::std::option::Option<crate::types::DkimSigningAttributesOrigin>) -> Self {
+    pub fn set_domain_signing_attributes_origin(mut self, input: ::std::option::Option<super::super::types::DkimSigningAttributesOrigin>) -> Self {
         self.domain_signing_attributes_origin = input;
         self
     }
@@ -506,12 +506,12 @@
     /// <li>
     /// <p><code>AWS_SES_US_GOV_WEST_1</code> – Configure DKIM for the identity by replicating from a parent identity in AWS GovCloud (US-West) region using Deterministic Easy-DKIM (DEED).</p></li>
     /// </ul>
-    pub fn get_domain_signing_attributes_origin(&self) -> &::std::option::Option<crate::types::DkimSigningAttributesOrigin> {
+    pub fn get_domain_signing_attributes_origin(&self) -> &::std::option::Option<super::super::types::DkimSigningAttributesOrigin> {
         &self.domain_signing_attributes_origin
     }
     /// Consumes the builder and constructs a [`DkimSigningAttributes`](crate::types::DkimSigningAttributes).
-    pub fn build(self) -> crate::types::DkimSigningAttributes {
-        crate::types::DkimSigningAttributes {
+    pub fn build(self) -> super::super::types::DkimSigningAttributes {
+        super::super::types::DkimSigningAttributes {
             domain_signing_selector: self.domain_signing_selector,
             domain_signing_private_key: self.domain_signing_private_key,
             next_signing_key_length: self.next_signing_key_length,
