--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>The date when you enabled the Deliverability dashboard for the domain.</p>
     pub subscription_start_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>An object that contains information about the inbox placement data settings for the domain.</p>
-    pub inbox_placement_tracking_option: ::std::option::Option<crate::types::InboxPlacementTrackingOption>,
+    pub inbox_placement_tracking_option: ::std::option::Option<super::types::InboxPlacementTrackingOption>,
 }
 impl DomainDeliverabilityTrackingOption {
     /// <p>A verified domain that’s associated with your Amazon Web Services account and currently has an active Deliverability dashboard subscription.</p>
@@ -21,14 +21,14 @@
         self.subscription_start_date.as_ref()
     }
     /// <p>An object that contains information about the inbox placement data settings for the domain.</p>
-    pub fn inbox_placement_tracking_option(&self) -> ::std::option::Option<&crate::types::InboxPlacementTrackingOption> {
+    pub fn inbox_placement_tracking_option(&self) -> ::std::option::Option<&super::types::InboxPlacementTrackingOption> {
         self.inbox_placement_tracking_option.as_ref()
     }
 }
 impl DomainDeliverabilityTrackingOption {
     /// Creates a new builder-style object to manufacture [`DomainDeliverabilityTrackingOption`](crate::types::DomainDeliverabilityTrackingOption).
-    pub fn builder() -> crate::types::builders::DomainDeliverabilityTrackingOptionBuilder {
-        crate::types::builders::DomainDeliverabilityTrackingOptionBuilder::default()
+    pub fn builder() -> super::types::builders::DomainDeliverabilityTrackingOptionBuilder {
+        super::types::builders::DomainDeliverabilityTrackingOptionBuilder::default()
     }
 }

@@ -38,7 +38,7 @@
 pub struct DomainDeliverabilityTrackingOptionBuilder {
     pub(crate) domain: ::std::option::Option<::std::string::String>,
     pub(crate) subscription_start_date: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) inbox_placement_tracking_option: ::std::option::Option<crate::types::InboxPlacementTrackingOption>,
+    pub(crate) inbox_placement_tracking_option: ::std::option::Option<super::types::InboxPlacementTrackingOption>,
 }
 impl DomainDeliverabilityTrackingOptionBuilder {
     /// <p>A verified domain that’s associated with your Amazon Web Services account and currently has an active Deliverability dashboard subscription.</p>
@@ -70,22 +70,22 @@
         &self.subscription_start_date
     }
     /// <p>An object that contains information about the inbox placement data settings for the domain.</p>
-    pub fn inbox_placement_tracking_option(mut self, input: crate::types::InboxPlacementTrackingOption) -> Self {
+    pub fn inbox_placement_tracking_option(mut self, input: super::types::InboxPlacementTrackingOption) -> Self {
         self.inbox_placement_tracking_option = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains information about the inbox placement data settings for the domain.</p>
-    pub fn set_inbox_placement_tracking_option(mut self, input: ::std::option::Option<crate::types::InboxPlacementTrackingOption>) -> Self {
+    pub fn set_inbox_placement_tracking_option(mut self, input: ::std::option::Option<super::types::InboxPlacementTrackingOption>) -> Self {
         self.inbox_placement_tracking_option = input;
         self
     }
     /// <p>An object that contains information about the inbox placement data settings for the domain.</p>
-    pub fn get_inbox_placement_tracking_option(&self) -> &::std::option::Option<crate::types::InboxPlacementTrackingOption> {
+    pub fn get_inbox_placement_tracking_option(&self) -> &::std::option::Option<super::types::InboxPlacementTrackingOption> {
         &self.inbox_placement_tracking_option
     }
     /// Consumes the builder and constructs a [`DomainDeliverabilityTrackingOption`](crate::types::DomainDeliverabilityTrackingOption).
-    pub fn build(self) -> crate::types::DomainDeliverabilityTrackingOption {
-        crate::types::DomainDeliverabilityTrackingOption {
+    pub fn build(self) -> super::types::DomainDeliverabilityTrackingOption {
+        super::types::DomainDeliverabilityTrackingOption {
             domain: self.domain,
             subscription_start_date: self.subscription_start_date,
             inbox_placement_tracking_option: self.inbox_placement_tracking_option,
