--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct EmailContent {
     /// <p>The simple email message. The message consists of a subject, message body and attachments list.</p>
-    pub simple: ::std::option::Option<crate::types::Message>,
+    pub simple: ::std::option::Option<super::types::Message>,
     /// <p>The raw email message. The message has to meet the following criteria:</p>
     /// <ul>
     /// <li>
@@ -23,13 +23,13 @@
     /// <li>
     /// <p>The length of any single line of text in the message can't exceed 1,000 characters. This restriction is defined in <a href="https://tools.ietf.org/html/rfc5321">RFC 5321</a>.</p></li>
     /// </ul>
-    pub raw: ::std::option::Option<crate::types::RawMessage>,
+    pub raw: ::std::option::Option<super::types::RawMessage>,
     /// <p>The template to use for the email message.</p>
-    pub template: ::std::option::Option<crate::types::Template>,
+    pub template: ::std::option::Option<super::types::Template>,
 }
 impl EmailContent {
     /// <p>The simple email message. The message consists of a subject, message body and attachments list.</p>
-    pub fn simple(&self) -> ::std::option::Option<&crate::types::Message> {
+    pub fn simple(&self) -> ::std::option::Option<&super::types::Message> {
         self.simple.as_ref()
     }
     /// <p>The raw email message. The message has to meet the following criteria:</p>
@@ -49,18 +49,18 @@
     /// <li>
     /// <p>The length of any single line of text in the message can't exceed 1,000 characters. This restriction is defined in <a href="https://tools.ietf.org/html/rfc5321">RFC 5321</a>.</p></li>
     /// </ul>
-    pub fn raw(&self) -> ::std::option::Option<&crate::types::RawMessage> {
+    pub fn raw(&self) -> ::std::option::Option<&super::types::RawMessage> {
         self.raw.as_ref()
     }
     /// <p>The template to use for the email message.</p>
-    pub fn template(&self) -> ::std::option::Option<&crate::types::Template> {
+    pub fn template(&self) -> ::std::option::Option<&super::types::Template> {
         self.template.as_ref()
     }
 }
 impl EmailContent {
     /// Creates a new builder-style object to manufacture [`EmailContent`](crate::types::EmailContent).
-    pub fn builder() -> crate::types::builders::EmailContentBuilder {
-        crate::types::builders::EmailContentBuilder::default()
+    pub fn builder() -> super::types::builders::EmailContentBuilder {
+        super::types::builders::EmailContentBuilder::default()
     }
 }

@@ -68,23 +68,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct EmailContentBuilder {
-    pub(crate) simple: ::std::option::Option<crate::types::Message>,
-    pub(crate) raw: ::std::option::Option<crate::types::RawMessage>,
-    pub(crate) template: ::std::option::Option<crate::types::Template>,
+    pub(crate) simple: ::std::option::Option<super::types::Message>,
+    pub(crate) raw: ::std::option::Option<super::types::RawMessage>,
+    pub(crate) template: ::std::option::Option<super::types::Template>,
 }
 impl EmailContentBuilder {
     /// <p>The simple email message. The message consists of a subject, message body and attachments list.</p>
-    pub fn simple(mut self, input: crate::types::Message) -> Self {
+    pub fn simple(mut self, input: super::types::Message) -> Self {
         self.simple = ::std::option::Option::Some(input);
         self
     }
     /// <p>The simple email message. The message consists of a subject, message body and attachments list.</p>
-    pub fn set_simple(mut self, input: ::std::option::Option<crate::types::Message>) -> Self {
+    pub fn set_simple(mut self, input: ::std::option::Option<super::types::Message>) -> Self {
         self.simple = input;
         self
     }
     /// <p>The simple email message. The message consists of a subject, message body and attachments list.</p>
-    pub fn get_simple(&self) -> &::std::option::Option<crate::types::Message> {
+    pub fn get_simple(&self) -> &::std::option::Option<super::types::Message> {
         &self.simple
     }
     /// <p>The raw email message. The message has to meet the following criteria:</p>
@@ -104,7 +104,7 @@
     /// <li>
     /// <p>The length of any single line of text in the message can't exceed 1,000 characters. This restriction is defined in <a href="https://tools.ietf.org/html/rfc5321">RFC 5321</a>.</p></li>
     /// </ul>
-    pub fn raw(mut self, input: crate::types::RawMessage) -> Self {
+    pub fn raw(mut self, input: super::types::RawMessage) -> Self {
         self.raw = ::std::option::Option::Some(input);
         self
     }
@@ -125,7 +125,7 @@
     /// <li>
     /// <p>The length of any single line of text in the message can't exceed 1,000 characters. This restriction is defined in <a href="https://tools.ietf.org/html/rfc5321">RFC 5321</a>.</p></li>
     /// </ul>
-    pub fn set_raw(mut self, input: ::std::option::Option<crate::types::RawMessage>) -> Self {
+    pub fn set_raw(mut self, input: ::std::option::Option<super::types::RawMessage>) -> Self {
         self.raw = input;
         self
     }
@@ -146,26 +146,26 @@
     /// <li>
     /// <p>The length of any single line of text in the message can't exceed 1,000 characters. This restriction is defined in <a href="https://tools.ietf.org/html/rfc5321">RFC 5321</a>.</p></li>
     /// </ul>
-    pub fn get_raw(&self) -> &::std::option::Option<crate::types::RawMessage> {
+    pub fn get_raw(&self) -> &::std::option::Option<super::types::RawMessage> {
         &self.raw
     }
     /// <p>The template to use for the email message.</p>
-    pub fn template(mut self, input: crate::types::Template) -> Self {
+    pub fn template(mut self, input: super::types::Template) -> Self {
         self.template = ::std::option::Option::Some(input);
         self
     }
     /// <p>The template to use for the email message.</p>
-    pub fn set_template(mut self, input: ::std::option::Option<crate::types::Template>) -> Self {
+    pub fn set_template(mut self, input: ::std::option::Option<super::types::Template>) -> Self {
         self.template = input;
         self
     }
     /// <p>The template to use for the email message.</p>
-    pub fn get_template(&self) -> &::std::option::Option<crate::types::Template> {
+    pub fn get_template(&self) -> &::std::option::Option<super::types::Template> {
         &self.template
     }
     /// Consumes the builder and constructs a [`EmailContent`](crate::types::EmailContent).
-    pub fn build(self) -> crate::types::EmailContent {
-        crate::types::EmailContent {
+    pub fn build(self) -> super::types::EmailContent {
+        super::types::EmailContent {
             simple: self.simple,
             raw: self.raw,
             template: self.template,
