--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl EmailTemplateContent {
     /// Creates a new builder-style object to manufacture [`EmailTemplateContent`](crate::types::EmailTemplateContent).
-    pub fn builder() -> crate::types::builders::EmailTemplateContentBuilder {
-        crate::types::builders::EmailTemplateContentBuilder::default()
+    pub fn builder() -> super::super::types::builders::EmailTemplateContentBuilder {
+        super::super::types::builders::EmailTemplateContentBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.html
     }
     /// Consumes the builder and constructs a [`EmailTemplateContent`](crate::types::EmailTemplateContent).
-    pub fn build(self) -> crate::types::EmailTemplateContent {
-        crate::types::EmailTemplateContent {
+    pub fn build(self) -> super::super::types::EmailTemplateContent {
+        super::super::types::EmailTemplateContent {
             subject: self.subject,
             text: self.text,
             html: self.html,
