--- original
+++ modified
@@ -5,48 +5,48 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct EmailAddressInsightsMailboxEvaluations {
     /// <p>Checks that the email address follows proper RFC standards and contains valid characters in the correct format.</p>
-    pub has_valid_syntax: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub has_valid_syntax: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
     /// <p>Checks that the domain exists, has valid DNS records, and is conﬁgured to receive email.</p>
-    pub has_valid_dns_records: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub has_valid_dns_records: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
     /// <p>Checks that the mailbox exists and can receive messages without actually sending an email.</p>
-    pub mailbox_exists: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub mailbox_exists: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
     /// <p>Identiﬁes role-based addresses (such as admin@, support@, or info@) that may have lower engagement rates.</p>
-    pub is_role_address: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub is_role_address: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
     /// <p>Checks disposable or temporary email addresses that could negatively impact your sender reputation.</p>
-    pub is_disposable: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub is_disposable: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
     /// <p>Checks if the input appears to be random text.</p>
-    pub is_random_input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub is_random_input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
 }
 impl EmailAddressInsightsMailboxEvaluations {
     /// <p>Checks that the email address follows proper RFC standards and contains valid characters in the correct format.</p>
-    pub fn has_valid_syntax(&self) -> ::std::option::Option<&crate::types::EmailAddressInsightsVerdict> {
+    pub fn has_valid_syntax(&self) -> ::std::option::Option<&super::super::types::EmailAddressInsightsVerdict> {
         self.has_valid_syntax.as_ref()
     }
     /// <p>Checks that the domain exists, has valid DNS records, and is conﬁgured to receive email.</p>
-    pub fn has_valid_dns_records(&self) -> ::std::option::Option<&crate::types::EmailAddressInsightsVerdict> {
+    pub fn has_valid_dns_records(&self) -> ::std::option::Option<&super::super::types::EmailAddressInsightsVerdict> {
         self.has_valid_dns_records.as_ref()
     }
     /// <p>Checks that the mailbox exists and can receive messages without actually sending an email.</p>
-    pub fn mailbox_exists(&self) -> ::std::option::Option<&crate::types::EmailAddressInsightsVerdict> {
+    pub fn mailbox_exists(&self) -> ::std::option::Option<&super::super::types::EmailAddressInsightsVerdict> {
         self.mailbox_exists.as_ref()
     }
     /// <p>Identiﬁes role-based addresses (such as admin@, support@, or info@) that may have lower engagement rates.</p>
-    pub fn is_role_address(&self) -> ::std::option::Option<&crate::types::EmailAddressInsightsVerdict> {
+    pub fn is_role_address(&self) -> ::std::option::Option<&super::super::types::EmailAddressInsightsVerdict> {
         self.is_role_address.as_ref()
     }
     /// <p>Checks disposable or temporary email addresses that could negatively impact your sender reputation.</p>
-    pub fn is_disposable(&self) -> ::std::option::Option<&crate::types::EmailAddressInsightsVerdict> {
+    pub fn is_disposable(&self) -> ::std::option::Option<&super::super::types::EmailAddressInsightsVerdict> {
         self.is_disposable.as_ref()
     }
     /// <p>Checks if the input appears to be random text.</p>
-    pub fn is_random_input(&self) -> ::std::option::Option<&crate::types::EmailAddressInsightsVerdict> {
+    pub fn is_random_input(&self) -> ::std::option::Option<&super::super::types::EmailAddressInsightsVerdict> {
         self.is_random_input.as_ref()
     }
 }
 impl EmailAddressInsightsMailboxEvaluations {
     /// Creates a new builder-style object to manufacture [`EmailAddressInsightsMailboxEvaluations`](crate::types::EmailAddressInsightsMailboxEvaluations).
-    pub fn builder() -> crate::types::builders::EmailAddressInsightsMailboxEvaluationsBuilder {
-        crate::types::builders::EmailAddressInsightsMailboxEvaluationsBuilder::default()
+    pub fn builder() -> super::super::types::builders::EmailAddressInsightsMailboxEvaluationsBuilder {
+        super::super::types::builders::EmailAddressInsightsMailboxEvaluationsBuilder::default()
     }
 }

@@ -54,101 +54,101 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct EmailAddressInsightsMailboxEvaluationsBuilder {
-    pub(crate) has_valid_syntax: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
-    pub(crate) has_valid_dns_records: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
-    pub(crate) mailbox_exists: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
-    pub(crate) is_role_address: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
-    pub(crate) is_disposable: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
-    pub(crate) is_random_input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>,
+    pub(crate) has_valid_syntax: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
+    pub(crate) has_valid_dns_records: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
+    pub(crate) mailbox_exists: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
+    pub(crate) is_role_address: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
+    pub(crate) is_disposable: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
+    pub(crate) is_random_input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>,
 }
 impl EmailAddressInsightsMailboxEvaluationsBuilder {
     /// <p>Checks that the email address follows proper RFC standards and contains valid characters in the correct format.</p>
-    pub fn has_valid_syntax(mut self, input: crate::types::EmailAddressInsightsVerdict) -> Self {
+    pub fn has_valid_syntax(mut self, input: super::super::types::EmailAddressInsightsVerdict) -> Self {
         self.has_valid_syntax = ::std::option::Option::Some(input);
         self
     }
     /// <p>Checks that the email address follows proper RFC standards and contains valid characters in the correct format.</p>
-    pub fn set_has_valid_syntax(mut self, input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>) -> Self {
+    pub fn set_has_valid_syntax(mut self, input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>) -> Self {
         self.has_valid_syntax = input;
         self
     }
     /// <p>Checks that the email address follows proper RFC standards and contains valid characters in the correct format.</p>
-    pub fn get_has_valid_syntax(&self) -> &::std::option::Option<crate::types::EmailAddressInsightsVerdict> {
+    pub fn get_has_valid_syntax(&self) -> &::std::option::Option<super::super::types::EmailAddressInsightsVerdict> {
         &self.has_valid_syntax
     }
     /// <p>Checks that the domain exists, has valid DNS records, and is conﬁgured to receive email.</p>
-    pub fn has_valid_dns_records(mut self, input: crate::types::EmailAddressInsightsVerdict) -> Self {
+    pub fn has_valid_dns_records(mut self, input: super::super::types::EmailAddressInsightsVerdict) -> Self {
         self.has_valid_dns_records = ::std::option::Option::Some(input);
         self
     }
     /// <p>Checks that the domain exists, has valid DNS records, and is conﬁgured to receive email.</p>
-    pub fn set_has_valid_dns_records(mut self, input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>) -> Self {
+    pub fn set_has_valid_dns_records(mut self, input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>) -> Self {
         self.has_valid_dns_records = input;
         self
     }
     /// <p>Checks that the domain exists, has valid DNS records, and is conﬁgured to receive email.</p>
-    pub fn get_has_valid_dns_records(&self) -> &::std::option::Option<crate::types::EmailAddressInsightsVerdict> {
+    pub fn get_has_valid_dns_records(&self) -> &::std::option::Option<super::super::types::EmailAddressInsightsVerdict> {
         &self.has_valid_dns_records
     }
     /// <p>Checks that the mailbox exists and can receive messages without actually sending an email.</p>
-    pub fn mailbox_exists(mut self, input: crate::types::EmailAddressInsightsVerdict) -> Self {
+    pub fn mailbox_exists(mut self, input: super::super::types::EmailAddressInsightsVerdict) -> Self {
         self.mailbox_exists = ::std::option::Option::Some(input);
         self
     }
     /// <p>Checks that the mailbox exists and can receive messages without actually sending an email.</p>
-    pub fn set_mailbox_exists(mut self, input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>) -> Self {
+    pub fn set_mailbox_exists(mut self, input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>) -> Self {
         self.mailbox_exists = input;
         self
     }
     /// <p>Checks that the mailbox exists and can receive messages without actually sending an email.</p>
-    pub fn get_mailbox_exists(&self) -> &::std::option::Option<crate::types::EmailAddressInsightsVerdict> {
+    pub fn get_mailbox_exists(&self) -> &::std::option::Option<super::super::types::EmailAddressInsightsVerdict> {
         &self.mailbox_exists
     }
     /// <p>Identiﬁes role-based addresses (such as admin@, support@, or info@) that may have lower engagement rates.</p>
-    pub fn is_role_address(mut self, input: crate::types::EmailAddressInsightsVerdict) -> Self {
+    pub fn is_role_address(mut self, input: super::super::types::EmailAddressInsightsVerdict) -> Self {
         self.is_role_address = ::std::option::Option::Some(input);
         self
     }
     /// <p>Identiﬁes role-based addresses (such as admin@, support@, or info@) that may have lower engagement rates.</p>
-    pub fn set_is_role_address(mut self, input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>) -> Self {
+    pub fn set_is_role_address(mut self, input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>) -> Self {
         self.is_role_address = input;
         self
     }
     /// <p>Identiﬁes role-based addresses (such as admin@, support@, or info@) that may have lower engagement rates.</p>
-    pub fn get_is_role_address(&self) -> &::std::option::Option<crate::types::EmailAddressInsightsVerdict> {
+    pub fn get_is_role_address(&self) -> &::std::option::Option<super::super::types::EmailAddressInsightsVerdict> {
         &self.is_role_address
     }
     /// <p>Checks disposable or temporary email addresses that could negatively impact your sender reputation.</p>
-    pub fn is_disposable(mut self, input: crate::types::EmailAddressInsightsVerdict) -> Self {
+    pub fn is_disposable(mut self, input: super::super::types::EmailAddressInsightsVerdict) -> Self {
         self.is_disposable = ::std::option::Option::Some(input);
         self
     }
     /// <p>Checks disposable or temporary email addresses that could negatively impact your sender reputation.</p>
-    pub fn set_is_disposable(mut self, input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>) -> Self {
+    pub fn set_is_disposable(mut self, input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>) -> Self {
         self.is_disposable = input;
         self
     }
     /// <p>Checks disposable or temporary email addresses that could negatively impact your sender reputation.</p>
-    pub fn get_is_disposable(&self) -> &::std::option::Option<crate::types::EmailAddressInsightsVerdict> {
+    pub fn get_is_disposable(&self) -> &::std::option::Option<super::super::types::EmailAddressInsightsVerdict> {
         &self.is_disposable
     }
     /// <p>Checks if the input appears to be random text.</p>
-    pub fn is_random_input(mut self, input: crate::types::EmailAddressInsightsVerdict) -> Self {
+    pub fn is_random_input(mut self, input: super::super::types::EmailAddressInsightsVerdict) -> Self {
         self.is_random_input = ::std::option::Option::Some(input);
         self
     }
     /// <p>Checks if the input appears to be random text.</p>
-    pub fn set_is_random_input(mut self, input: ::std::option::Option<crate::types::EmailAddressInsightsVerdict>) -> Self {
+    pub fn set_is_random_input(mut self, input: ::std::option::Option<super::super::types::EmailAddressInsightsVerdict>) -> Self {
         self.is_random_input = input;
         self
     }
     /// <p>Checks if the input appears to be random text.</p>
-    pub fn get_is_random_input(&self) -> &::std::option::Option<crate::types::EmailAddressInsightsVerdict> {
+    pub fn get_is_random_input(&self) -> &::std::option::Option<super::super::types::EmailAddressInsightsVerdict> {
         &self.is_random_input
     }
     /// Consumes the builder and constructs a [`EmailAddressInsightsMailboxEvaluations`](crate::types::EmailAddressInsightsMailboxEvaluations).
-    pub fn build(self) -> crate::types::EmailAddressInsightsMailboxEvaluations {
-        crate::types::EmailAddressInsightsMailboxEvaluations {
+    pub fn build(self) -> super::super::types::EmailAddressInsightsMailboxEvaluations {
+        super::super::types::EmailAddressInsightsMailboxEvaluations {
             has_valid_syntax: self.has_valid_syntax,
             has_valid_dns_records: self.has_valid_dns_records,
             mailbox_exists: self.mailbox_exists,
