--- original
+++ modified
@@ -27,7 +27,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> - Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub name: ::std::option::Option<crate::types::Metric>,
+    pub name: ::std::option::Option<super::super::types::Metric>,
     /// <p>The aggregation to apply to a metric, can be one of the following:</p>
     /// <ul>
     /// <li>
@@ -35,7 +35,7 @@
     /// <li>
     /// <p><code>RATE</code> - The rate for this metric relative to the <code>SEND</code> metric volume.</p></li>
     /// </ul>
-    pub aggregation: ::std::option::Option<crate::types::MetricAggregation>,
+    pub aggregation: ::std::option::Option<super::super::types::MetricAggregation>,
 }
 impl ExportMetric {
     /// <p>The metric to export, can be one of the following:</p>
@@ -61,7 +61,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> - Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn name(&self) -> ::std::option::Option<&crate::types::Metric> {
+    pub fn name(&self) -> ::std::option::Option<&super::super::types::Metric> {
         self.name.as_ref()
     }
     /// <p>The aggregation to apply to a metric, can be one of the following:</p>
@@ -71,14 +71,14 @@
     /// <li>
     /// <p><code>RATE</code> - The rate for this metric relative to the <code>SEND</code> metric volume.</p></li>
     /// </ul>
-    pub fn aggregation(&self) -> ::std::option::Option<&crate::types::MetricAggregation> {
+    pub fn aggregation(&self) -> ::std::option::Option<&super::super::types::MetricAggregation> {
         self.aggregation.as_ref()
     }
 }
 impl ExportMetric {
     /// Creates a new builder-style object to manufacture [`ExportMetric`](crate::types::ExportMetric).
-    pub fn builder() -> crate::types::builders::ExportMetricBuilder {
-        crate::types::builders::ExportMetricBuilder::default()
+    pub fn builder() -> super::super::types::builders::ExportMetricBuilder {
+        super::super::types::builders::ExportMetricBuilder::default()
     }
 }

@@ -86,8 +86,8 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ExportMetricBuilder {
-    pub(crate) name: ::std::option::Option<crate::types::Metric>,
-    pub(crate) aggregation: ::std::option::Option<crate::types::MetricAggregation>,
+    pub(crate) name: ::std::option::Option<super::super::types::Metric>,
+    pub(crate) aggregation: ::std::option::Option<super::super::types::MetricAggregation>,
 }
 impl ExportMetricBuilder {
     /// <p>The metric to export, can be one of the following:</p>
@@ -113,7 +113,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> - Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn name(mut self, input: crate::types::Metric) -> Self {
+    pub fn name(mut self, input: super::super::types::Metric) -> Self {
         self.name = ::std::option::Option::Some(input);
         self
     }
@@ -140,7 +140,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> - Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn set_name(mut self, input: ::std::option::Option<crate::types::Metric>) -> Self {
+    pub fn set_name(mut self, input: ::std::option::Option<super::super::types::Metric>) -> Self {
         self.name = input;
         self
     }
@@ -167,7 +167,7 @@
     /// <li>
     /// <p><code>DELIVERY_COMPLAINT</code> - Successful deliveries for email sending attempts. Excludes deliveries to the mailbox simulator, for emails addressed to more than one recipient, and emails addressed to recipients hosted by ISPs with which Amazon SES does not have a feedback loop agreement.</p></li>
     /// </ul>
-    pub fn get_name(&self) -> &::std::option::Option<crate::types::Metric> {
+    pub fn get_name(&self) -> &::std::option::Option<super::super::types::Metric> {
         &self.name
     }
     /// <p>The aggregation to apply to a metric, can be one of the following:</p>
@@ -177,7 +177,7 @@
     /// <li>
     /// <p><code>RATE</code> - The rate for this metric relative to the <code>SEND</code> metric volume.</p></li>
     /// </ul>
-    pub fn aggregation(mut self, input: crate::types::MetricAggregation) -> Self {
+    pub fn aggregation(mut self, input: super::super::types::MetricAggregation) -> Self {
         self.aggregation = ::std::option::Option::Some(input);
         self
     }
@@ -188,7 +188,7 @@
     /// <li>
     /// <p><code>RATE</code> - The rate for this metric relative to the <code>SEND</code> metric volume.</p></li>
     /// </ul>
-    pub fn set_aggregation(mut self, input: ::std::option::Option<crate::types::MetricAggregation>) -> Self {
+    pub fn set_aggregation(mut self, input: ::std::option::Option<super::super::types::MetricAggregation>) -> Self {
         self.aggregation = input;
         self
     }
@@ -199,12 +199,12 @@
     /// <li>
     /// <p><code>RATE</code> - The rate for this metric relative to the <code>SEND</code> metric volume.</p></li>
     /// </ul>
-    pub fn get_aggregation(&self) -> &::std::option::Option<crate::types::MetricAggregation> {
+    pub fn get_aggregation(&self) -> &::std::option::Option<super::super::types::MetricAggregation> {
         &self.aggregation
     }
     /// Consumes the builder and constructs a [`ExportMetric`](crate::types::ExportMetric).
-    pub fn build(self) -> crate::types::ExportMetric {
-        crate::types::ExportMetric {
+    pub fn build(self) -> super::super::types::ExportMetric {
+        super::super::types::ExportMetric {
             name: self.name,
             aggregation: self.aggregation,
         }
