--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>A string that represents a job ID.</p>
     pub job_id: ::std::option::Option<::std::string::String>,
     /// <p>An object that contains details about the resource destination the import job is going to target.</p>
-    pub import_destination: ::std::option::Option<crate::types::ImportDestination>,
+    pub import_destination: ::std::option::Option<super::super::types::ImportDestination>,
     /// <p>The status of a job.</p>
     /// <ul>
     /// <li>
@@ -19,7 +19,7 @@
     /// <li>
     /// <p><code>COMPLETED</code> – Job has completed processing successfully.</p></li>
     /// </ul>
-    pub job_status: ::std::option::Option<crate::types::JobStatus>,
+    pub job_status: ::std::option::Option<super::super::types::JobStatus>,
     /// <p>The date and time when the import job was created.</p>
     pub created_timestamp: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The current number of records processed.</p>
@@ -33,7 +33,7 @@
         self.job_id.as_deref()
     }
     /// <p>An object that contains details about the resource destination the import job is going to target.</p>
-    pub fn import_destination(&self) -> ::std::option::Option<&crate::types::ImportDestination> {
+    pub fn import_destination(&self) -> ::std::option::Option<&super::super::types::ImportDestination> {
         self.import_destination.as_ref()
     }
     /// <p>The status of a job.</p>
@@ -47,7 +47,7 @@
     /// <li>
     /// <p><code>COMPLETED</code> – Job has completed processing successfully.</p></li>
     /// </ul>
-    pub fn job_status(&self) -> ::std::option::Option<&crate::types::JobStatus> {
+    pub fn job_status(&self) -> ::std::option::Option<&super::super::types::JobStatus> {
         self.job_status.as_ref()
     }
     /// <p>The date and time when the import job was created.</p>
@@ -65,8 +65,8 @@
 }
 impl ImportJobSummary {
     /// Creates a new builder-style object to manufacture [`ImportJobSummary`](crate::types::ImportJobSummary).
-    pub fn builder() -> crate::types::builders::ImportJobSummaryBuilder {
-        crate::types::builders::ImportJobSummaryBuilder::default()
+    pub fn builder() -> super::super::types::builders::ImportJobSummaryBuilder {
+        super::super::types::builders::ImportJobSummaryBuilder::default()
     }
 }

@@ -75,8 +75,8 @@
 #[non_exhaustive]
 pub struct ImportJobSummaryBuilder {
     pub(crate) job_id: ::std::option::Option<::std::string::String>,
-    pub(crate) import_destination: ::std::option::Option<crate::types::ImportDestination>,
-    pub(crate) job_status: ::std::option::Option<crate::types::JobStatus>,
+    pub(crate) import_destination: ::std::option::Option<super::super::types::ImportDestination>,
+    pub(crate) job_status: ::std::option::Option<super::super::types::JobStatus>,
     pub(crate) created_timestamp: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) processed_records_count: ::std::option::Option<i32>,
     pub(crate) failed_records_count: ::std::option::Option<i32>,
@@ -97,17 +97,17 @@
         &self.job_id
     }
     /// <p>An object that contains details about the resource destination the import job is going to target.</p>
-    pub fn import_destination(mut self, input: crate::types::ImportDestination) -> Self {
+    pub fn import_destination(mut self, input: super::super::types::ImportDestination) -> Self {
         self.import_destination = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains details about the resource destination the import job is going to target.</p>
-    pub fn set_import_destination(mut self, input: ::std::option::Option<crate::types::ImportDestination>) -> Self {
+    pub fn set_import_destination(mut self, input: ::std::option::Option<super::super::types::ImportDestination>) -> Self {
         self.import_destination = input;
         self
     }
     /// <p>An object that contains details about the resource destination the import job is going to target.</p>
-    pub fn get_import_destination(&self) -> &::std::option::Option<crate::types::ImportDestination> {
+    pub fn get_import_destination(&self) -> &::std::option::Option<super::super::types::ImportDestination> {
         &self.import_destination
     }
     /// <p>The status of a job.</p>
@@ -121,7 +121,7 @@
     /// <li>
     /// <p><code>COMPLETED</code> – Job has completed processing successfully.</p></li>
     /// </ul>
-    pub fn job_status(mut self, input: crate::types::JobStatus) -> Self {
+    pub fn job_status(mut self, input: super::super::types::JobStatus) -> Self {
         self.job_status = ::std::option::Option::Some(input);
         self
     }
@@ -136,7 +136,7 @@
     /// <li>
     /// <p><code>COMPLETED</code> – Job has completed processing successfully.</p></li>
     /// </ul>
-    pub fn set_job_status(mut self, input: ::std::option::Option<crate::types::JobStatus>) -> Self {
+    pub fn set_job_status(mut self, input: ::std::option::Option<super::super::types::JobStatus>) -> Self {
         self.job_status = input;
         self
     }
@@ -151,7 +151,7 @@
     /// <li>
     /// <p><code>COMPLETED</code> – Job has completed processing successfully.</p></li>
     /// </ul>
-    pub fn get_job_status(&self) -> &::std::option::Option<crate::types::JobStatus> {
+    pub fn get_job_status(&self) -> &::std::option::Option<super::super::types::JobStatus> {
         &self.job_status
     }
     /// <p>The date and time when the import job was created.</p>
@@ -197,8 +197,8 @@
         &self.failed_records_count
     }
     /// Consumes the builder and constructs a [`ImportJobSummary`](crate::types::ImportJobSummary).
-    pub fn build(self) -> crate::types::ImportJobSummary {
-        crate::types::ImportJobSummary {
+    pub fn build(self) -> super::super::types::ImportJobSummary {
+        super::super::types::ImportJobSummary {
             job_id: self.job_id,
             import_destination: self.import_destination,
             job_status: self.job_status,
