--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl InboxPlacementTrackingOption {
     /// Creates a new builder-style object to manufacture [`InboxPlacementTrackingOption`](crate::types::InboxPlacementTrackingOption).
-    pub fn builder() -> crate::types::builders::InboxPlacementTrackingOptionBuilder {
-        crate::types::builders::InboxPlacementTrackingOptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::InboxPlacementTrackingOptionBuilder {
+        super::super::types::builders::InboxPlacementTrackingOptionBuilder::default()
     }
 }

@@ -71,8 +71,8 @@
         &self.tracked_isps
     }
     /// Consumes the builder and constructs a [`InboxPlacementTrackingOption`](crate::types::InboxPlacementTrackingOption).
-    pub fn build(self) -> crate::types::InboxPlacementTrackingOption {
-        crate::types::InboxPlacementTrackingOption {
+    pub fn build(self) -> super::super::types::InboxPlacementTrackingOption {
+        super::super::types::InboxPlacementTrackingOption {
             global: self.global.unwrap_or_default(),
             tracked_isps: self.tracked_isps,
         }
