--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl ListManagementOptions {
     /// Creates a new builder-style object to manufacture [`ListManagementOptions`](crate::types::ListManagementOptions).
-    pub fn builder() -> crate::types::builders::ListManagementOptionsBuilder {
-        crate::types::builders::ListManagementOptionsBuilder::default()
+    pub fn builder() -> super::types::builders::ListManagementOptionsBuilder {
+        super::types::builders::ListManagementOptionsBuilder::default()
     }
 }

@@ -67,8 +67,8 @@
     /// Consumes the builder and constructs a [`ListManagementOptions`](crate::types::ListManagementOptions).
     /// This method will fail if any of the following fields are not set:
     /// - [`contact_list_name`](crate::types::builders::ListManagementOptionsBuilder::contact_list_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::ListManagementOptions, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ListManagementOptions {
+    pub fn build(self) -> ::std::result::Result<super::types::ListManagementOptions, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ListManagementOptions {
             contact_list_name: self.contact_list_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "contact_list_name",
