--- original
+++ modified
@@ -47,8 +47,8 @@
 }
 impl MessageTag {
     /// Creates a new builder-style object to manufacture [`MessageTag`](crate::types::MessageTag).
-    pub fn builder() -> crate::types::builders::MessageTagBuilder {
-        crate::types::builders::MessageTagBuilder::default()
+    pub fn builder() -> super::super::types::builders::MessageTagBuilder {
+        super::super::types::builders::MessageTagBuilder::default()
     }
 }

@@ -130,8 +130,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`name`](crate::types::builders::MessageTagBuilder::name)
     /// - [`value`](crate::types::builders::MessageTagBuilder::value)
-    pub fn build(self) -> ::std::result::Result<crate::types::MessageTag, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::MessageTag {
+    pub fn build(self) -> ::std::result::Result<super::super::types::MessageTag, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::MessageTag {
             name: self.name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "name",
