--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <li>
     /// <p><code>ACCESS_DENIED</code> – You have insufficient access to retrieve metrics based on the given query.</p></li>
     /// </ul>
-    pub code: ::std::option::Option<crate::types::QueryErrorCode>,
+    pub code: ::std::option::Option<super::types::QueryErrorCode>,
     /// <p>The error message associated with the current query error.</p>
     pub message: ::std::option::Option<::std::string::String>,
 }
@@ -29,7 +29,7 @@
     /// <li>
     /// <p><code>ACCESS_DENIED</code> – You have insufficient access to retrieve metrics based on the given query.</p></li>
     /// </ul>
-    pub fn code(&self) -> ::std::option::Option<&crate::types::QueryErrorCode> {
+    pub fn code(&self) -> ::std::option::Option<&super::types::QueryErrorCode> {
         self.code.as_ref()
     }
     /// <p>The error message associated with the current query error.</p>
@@ -39,8 +39,8 @@
 }
 impl MetricDataError {
     /// Creates a new builder-style object to manufacture [`MetricDataError`](crate::types::MetricDataError).
-    pub fn builder() -> crate::types::builders::MetricDataErrorBuilder {
-        crate::types::builders::MetricDataErrorBuilder::default()
+    pub fn builder() -> super::types::builders::MetricDataErrorBuilder {
+        super::types::builders::MetricDataErrorBuilder::default()
     }
 }

@@ -49,7 +49,7 @@
 #[non_exhaustive]
 pub struct MetricDataErrorBuilder {
     pub(crate) id: ::std::option::Option<::std::string::String>,
-    pub(crate) code: ::std::option::Option<crate::types::QueryErrorCode>,
+    pub(crate) code: ::std::option::Option<super::types::QueryErrorCode>,
     pub(crate) message: ::std::option::Option<::std::string::String>,
 }
 impl MetricDataErrorBuilder {
@@ -74,7 +74,7 @@
     /// <li>
     /// <p><code>ACCESS_DENIED</code> – You have insufficient access to retrieve metrics based on the given query.</p></li>
     /// </ul>
-    pub fn code(mut self, input: crate::types::QueryErrorCode) -> Self {
+    pub fn code(mut self, input: super::types::QueryErrorCode) -> Self {
         self.code = ::std::option::Option::Some(input);
         self
     }
@@ -85,7 +85,7 @@
     /// <li>
     /// <p><code>ACCESS_DENIED</code> – You have insufficient access to retrieve metrics based on the given query.</p></li>
     /// </ul>
-    pub fn set_code(mut self, input: ::std::option::Option<crate::types::QueryErrorCode>) -> Self {
+    pub fn set_code(mut self, input: ::std::option::Option<super::types::QueryErrorCode>) -> Self {
         self.code = input;
         self
     }
@@ -96,7 +96,7 @@
     /// <li>
     /// <p><code>ACCESS_DENIED</code> – You have insufficient access to retrieve metrics based on the given query.</p></li>
     /// </ul>
-    pub fn get_code(&self) -> &::std::option::Option<crate::types::QueryErrorCode> {
+    pub fn get_code(&self) -> &::std::option::Option<super::types::QueryErrorCode> {
         &self.code
     }
     /// <p>The error message associated with the current query error.</p>
@@ -114,8 +114,8 @@
         &self.message
     }
     /// Consumes the builder and constructs a [`MetricDataError`](crate::types::MetricDataError).
-    pub fn build(self) -> crate::types::MetricDataError {
-        crate::types::MetricDataError {
+    pub fn build(self) -> super::types::MetricDataError {
+        super::types::MetricDataError {
             id: self.id,
             code: self.code,
             message: self.message,
