--- original
+++ modified
@@ -5,11 +5,11 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct MetricsDataSource {
     /// <p>An object that contains a mapping between a <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by. Must contain a least 1 dimension but no more than 3 unique ones.</p>
-    pub dimensions: ::std::collections::HashMap<crate::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>,
+    pub dimensions: ::std::collections::HashMap<super::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>,
     /// <p>The metrics namespace - e.g., <code>VDM</code>.</p>
-    pub namespace: crate::types::MetricNamespace,
+    pub namespace: super::types::MetricNamespace,
     /// <p>A list of <code>ExportMetric</code> objects to export.</p>
-    pub metrics: ::std::vec::Vec<crate::types::ExportMetric>,
+    pub metrics: ::std::vec::Vec<super::types::ExportMetric>,
     /// <p>Represents the start date for the export interval as a timestamp.</p>
     pub start_date: ::aws_smithy_types::DateTime,
     /// <p>Represents the end date for the export interval as a timestamp.</p>
@@ -17,15 +17,15 @@
 }
 impl MetricsDataSource {
     /// <p>An object that contains a mapping between a <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by. Must contain a least 1 dimension but no more than 3 unique ones.</p>
-    pub fn dimensions(&self) -> &::std::collections::HashMap<crate::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>> {
+    pub fn dimensions(&self) -> &::std::collections::HashMap<super::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>> {
         &self.dimensions
     }
     /// <p>The metrics namespace - e.g., <code>VDM</code>.</p>
-    pub fn namespace(&self) -> &crate::types::MetricNamespace {
+    pub fn namespace(&self) -> &super::types::MetricNamespace {
         &self.namespace
     }
     /// <p>A list of <code>ExportMetric</code> objects to export.</p>
-    pub fn metrics(&self) -> &[crate::types::ExportMetric] {
+    pub fn metrics(&self) -> &[super::types::ExportMetric] {
         use std::ops::Deref;
         self.metrics.deref()
     }
@@ -40,8 +40,8 @@
 }
 impl MetricsDataSource {
     /// Creates a new builder-style object to manufacture [`MetricsDataSource`](crate::types::MetricsDataSource).
-    pub fn builder() -> crate::types::builders::MetricsDataSourceBuilder {
-        crate::types::builders::MetricsDataSourceBuilder::default()
+    pub fn builder() -> super::types::builders::MetricsDataSourceBuilder {
+        super::types::builders::MetricsDataSourceBuilder::default()
     }
 }

@@ -50,9 +50,9 @@
 #[non_exhaustive]
 pub struct MetricsDataSourceBuilder {
     pub(crate) dimensions:
-        ::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>>,
-    pub(crate) namespace: ::std::option::Option<crate::types::MetricNamespace>,
-    pub(crate) metrics: ::std::option::Option<::std::vec::Vec<crate::types::ExportMetric>>,
+        ::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>>,
+    pub(crate) namespace: ::std::option::Option<super::types::MetricNamespace>,
+    pub(crate) metrics: ::std::option::Option<::std::vec::Vec<super::types::ExportMetric>>,
     pub(crate) start_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) end_date: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -62,7 +62,7 @@
     /// To override the contents of this collection use [`set_dimensions`](Self::set_dimensions).
     ///
     /// <p>An object that contains a mapping between a <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by. Must contain a least 1 dimension but no more than 3 unique ones.</p>
-    pub fn dimensions(mut self, k: crate::types::MetricDimensionName, v: ::std::vec::Vec<::std::string::String>) -> Self {
+    pub fn dimensions(mut self, k: super::types::MetricDimensionName, v: ::std::vec::Vec<::std::string::String>) -> Self {
         let mut hash_map = self.dimensions.unwrap_or_default();
         hash_map.insert(k, v);
         self.dimensions = ::std::option::Option::Some(hash_map);
@@ -71,7 +71,7 @@
     /// <p>An object that contains a mapping between a <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by. Must contain a least 1 dimension but no more than 3 unique ones.</p>
     pub fn set_dimensions(
         mut self,
-        input: ::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>>,
+        input: ::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>>,
     ) -> Self {
         self.dimensions = input;
         self
@@ -79,22 +79,22 @@
     /// <p>An object that contains a mapping between a <code>MetricDimensionName</code> and <code>MetricDimensionValue</code> to filter metrics by. Must contain a least 1 dimension but no more than 3 unique ones.</p>
     pub fn get_dimensions(
         &self,
-    ) -> &::std::option::Option<::std::collections::HashMap<crate::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>> {
+    ) -> &::std::option::Option<::std::collections::HashMap<super::types::MetricDimensionName, ::std::vec::Vec<::std::string::String>>> {
         &self.dimensions
     }
     /// <p>The metrics namespace - e.g., <code>VDM</code>.</p>
     /// This field is required.
-    pub fn namespace(mut self, input: crate::types::MetricNamespace) -> Self {
+    pub fn namespace(mut self, input: super::types::MetricNamespace) -> Self {
         self.namespace = ::std::option::Option::Some(input);
         self
     }
     /// <p>The metrics namespace - e.g., <code>VDM</code>.</p>
-    pub fn set_namespace(mut self, input: ::std::option::Option<crate::types::MetricNamespace>) -> Self {
+    pub fn set_namespace(mut self, input: ::std::option::Option<super::types::MetricNamespace>) -> Self {
         self.namespace = input;
         self
     }
     /// <p>The metrics namespace - e.g., <code>VDM</code>.</p>
-    pub fn get_namespace(&self) -> &::std::option::Option<crate::types::MetricNamespace> {
+    pub fn get_namespace(&self) -> &::std::option::Option<super::types::MetricNamespace> {
         &self.namespace
     }
     /// Appends an item to `metrics`.
@@ -102,7 +102,7 @@
     /// To override the contents of this collection use [`set_metrics`](Self::set_metrics).
     ///
     /// <p>A list of <code>ExportMetric</code> objects to export.</p>
-    pub fn metrics(mut self, input: crate::types::ExportMetric) -> Self {
+    pub fn metrics(mut self, input: super::types::ExportMetric) -> Self {
         let mut v = self.metrics.unwrap_or_default();
         v.push(input);
         self.metrics = ::std::option::Option::Some(v);
@@ -109,12 +109,12 @@
         self
     }
     /// <p>A list of <code>ExportMetric</code> objects to export.</p>
-    pub fn set_metrics(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ExportMetric>>) -> Self {
+    pub fn set_metrics(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ExportMetric>>) -> Self {
         self.metrics = input;
         self
     }
     /// <p>A list of <code>ExportMetric</code> objects to export.</p>
-    pub fn get_metrics(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ExportMetric>> {
+    pub fn get_metrics(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ExportMetric>> {
         &self.metrics
     }
     /// <p>Represents the start date for the export interval as a timestamp.</p>
@@ -154,8 +154,8 @@
     /// - [`metrics`](crate::types::builders::MetricsDataSourceBuilder::metrics)
     /// - [`start_date`](crate::types::builders::MetricsDataSourceBuilder::start_date)
     /// - [`end_date`](crate::types::builders::MetricsDataSourceBuilder::end_date)
-    pub fn build(self) -> ::std::result::Result<crate::types::MetricsDataSource, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::MetricsDataSource {
+    pub fn build(self) -> ::std::result::Result<super::types::MetricsDataSource, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::MetricsDataSource {
             dimensions: self.dimensions.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "dimensions",
