--- original
+++ modified
@@ -17,7 +17,7 @@
     /// <li>
     /// <p><code>DELETING</code> – The resource is being deleted as requested.</p></li>
     /// </ul>
-    pub status: ::std::option::Option<crate::types::Status>,
+    pub status: ::std::option::Option<super::types::Status>,
     /// <p>The ID of the multi-region endpoint (global-endpoint).</p>
     pub endpoint_id: ::std::option::Option<::std::string::String>,
     /// <p>Primary and secondary regions between which multi-region endpoint splits sending traffic.</p>
@@ -43,7 +43,7 @@
     /// <li>
     /// <p><code>DELETING</code> – The resource is being deleted as requested.</p></li>
     /// </ul>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::Status> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::Status> {
         self.status.as_ref()
     }
     /// <p>The ID of the multi-region endpoint (global-endpoint).</p>
@@ -67,8 +67,8 @@
 }
 impl MultiRegionEndpoint {
     /// Creates a new builder-style object to manufacture [`MultiRegionEndpoint`](crate::types::MultiRegionEndpoint).
-    pub fn builder() -> crate::types::builders::MultiRegionEndpointBuilder {
-        crate::types::builders::MultiRegionEndpointBuilder::default()
+    pub fn builder() -> super::types::builders::MultiRegionEndpointBuilder {
+        super::types::builders::MultiRegionEndpointBuilder::default()
     }
 }

@@ -77,7 +77,7 @@
 #[non_exhaustive]
 pub struct MultiRegionEndpointBuilder {
     pub(crate) endpoint_name: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::Status>,
+    pub(crate) status: ::std::option::Option<super::types::Status>,
     pub(crate) endpoint_id: ::std::option::Option<::std::string::String>,
     pub(crate) regions: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) created_timestamp: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -109,7 +109,7 @@
     /// <li>
     /// <p><code>DELETING</code> – The resource is being deleted as requested.</p></li>
     /// </ul>
-    pub fn status(mut self, input: crate::types::Status) -> Self {
+    pub fn status(mut self, input: super::types::Status) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
@@ -124,7 +124,7 @@
     /// <li>
     /// <p><code>DELETING</code> – The resource is being deleted as requested.</p></li>
     /// </ul>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::Status>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::Status>) -> Self {
         self.status = input;
         self
     }
@@ -139,7 +139,7 @@
     /// <li>
     /// <p><code>DELETING</code> – The resource is being deleted as requested.</p></li>
     /// </ul>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::Status> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::Status> {
         &self.status
     }
     /// <p>The ID of the multi-region endpoint (global-endpoint).</p>
@@ -205,8 +205,8 @@
         &self.last_updated_timestamp
     }
     /// Consumes the builder and constructs a [`MultiRegionEndpoint`](crate::types::MultiRegionEndpoint).
-    pub fn build(self) -> crate::types::MultiRegionEndpoint {
-        crate::types::MultiRegionEndpoint {
+    pub fn build(self) -> super::types::MultiRegionEndpoint {
+        super::types::MultiRegionEndpoint {
             endpoint_name: self.endpoint_name,
             status: self.status,
             endpoint_id: self.endpoint_id,
