--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl PinpointDestination {
     /// Creates a new builder-style object to manufacture [`PinpointDestination`](crate::types::PinpointDestination).
-    pub fn builder() -> crate::types::builders::PinpointDestinationBuilder {
-        crate::types::builders::PinpointDestinationBuilder::default()
+    pub fn builder() -> super::super::types::builders::PinpointDestinationBuilder {
+        super::super::types::builders::PinpointDestinationBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.application_arn
     }
     /// Consumes the builder and constructs a [`PinpointDestination`](crate::types::PinpointDestination).
-    pub fn build(self) -> crate::types::PinpointDestination {
-        crate::types::PinpointDestination {
+    pub fn build(self) -> super::super::types::PinpointDestination {
+        super::super::types::PinpointDestination {
             application_arn: self.application_arn,
         }
     }
