--- original
+++ modified
@@ -39,8 +39,8 @@
 }
 impl PlacementStatistics {
     /// Creates a new builder-style object to manufacture [`PlacementStatistics`](crate::types::PlacementStatistics).
-    pub fn builder() -> crate::types::builders::PlacementStatisticsBuilder {
-        crate::types::builders::PlacementStatisticsBuilder::default()
+    pub fn builder() -> super::types::builders::PlacementStatisticsBuilder {
+        super::types::builders::PlacementStatisticsBuilder::default()
     }
 }

@@ -126,8 +126,8 @@
         &self.dkim_percentage
     }
     /// Consumes the builder and constructs a [`PlacementStatistics`](crate::types::PlacementStatistics).
-    pub fn build(self) -> crate::types::PlacementStatistics {
-        crate::types::PlacementStatistics {
+    pub fn build(self) -> super::types::PlacementStatistics {
+        super::types::PlacementStatistics {
             inbox_percentage: self.inbox_percentage,
             spam_percentage: self.spam_percentage,
             missing_percentage: self.missing_percentage,
