--- original
+++ modified
@@ -47,8 +47,8 @@
 }
 impl RawMessage {
     /// Creates a new builder-style object to manufacture [`RawMessage`](crate::types::RawMessage).
-    pub fn builder() -> crate::types::builders::RawMessageBuilder {
-        crate::types::builders::RawMessageBuilder::default()
+    pub fn builder() -> super::types::builders::RawMessageBuilder {
+        super::types::builders::RawMessageBuilder::default()
     }
 }

@@ -125,8 +125,8 @@
     /// Consumes the builder and constructs a [`RawMessage`](crate::types::RawMessage).
     /// This method will fail if any of the following fields are not set:
     /// - [`data`](crate::types::builders::RawMessageBuilder::data)
-    pub fn build(self) -> ::std::result::Result<crate::types::RawMessage, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::RawMessage {
+    pub fn build(self) -> ::std::result::Result<super::types::RawMessage, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::RawMessage {
             data: self.data.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "data",
