--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ReplacementEmailContent {
     /// <p>The <code>ReplacementTemplate</code> associated with <code>ReplacementEmailContent</code>.</p>
-    pub replacement_template: ::std::option::Option<crate::types::ReplacementTemplate>,
+    pub replacement_template: ::std::option::Option<super::types::ReplacementTemplate>,
 }
 impl ReplacementEmailContent {
     /// <p>The <code>ReplacementTemplate</code> associated with <code>ReplacementEmailContent</code>.</p>
-    pub fn replacement_template(&self) -> ::std::option::Option<&crate::types::ReplacementTemplate> {
+    pub fn replacement_template(&self) -> ::std::option::Option<&super::types::ReplacementTemplate> {
         self.replacement_template.as_ref()
     }
 }
 impl ReplacementEmailContent {
     /// Creates a new builder-style object to manufacture [`ReplacementEmailContent`](crate::types::ReplacementEmailContent).
-    pub fn builder() -> crate::types::builders::ReplacementEmailContentBuilder {
-        crate::types::builders::ReplacementEmailContentBuilder::default()
+    pub fn builder() -> super::types::builders::ReplacementEmailContentBuilder {
+        super::types::builders::ReplacementEmailContentBuilder::default()
     }
 }

@@ -24,26 +24,26 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ReplacementEmailContentBuilder {
-    pub(crate) replacement_template: ::std::option::Option<crate::types::ReplacementTemplate>,
+    pub(crate) replacement_template: ::std::option::Option<super::types::ReplacementTemplate>,
 }
 impl ReplacementEmailContentBuilder {
     /// <p>The <code>ReplacementTemplate</code> associated with <code>ReplacementEmailContent</code>.</p>
-    pub fn replacement_template(mut self, input: crate::types::ReplacementTemplate) -> Self {
+    pub fn replacement_template(mut self, input: super::types::ReplacementTemplate) -> Self {
         self.replacement_template = ::std::option::Option::Some(input);
         self
     }
     /// <p>The <code>ReplacementTemplate</code> associated with <code>ReplacementEmailContent</code>.</p>
-    pub fn set_replacement_template(mut self, input: ::std::option::Option<crate::types::ReplacementTemplate>) -> Self {
+    pub fn set_replacement_template(mut self, input: ::std::option::Option<super::types::ReplacementTemplate>) -> Self {
         self.replacement_template = input;
         self
     }
     /// <p>The <code>ReplacementTemplate</code> associated with <code>ReplacementEmailContent</code>.</p>
-    pub fn get_replacement_template(&self) -> &::std::option::Option<crate::types::ReplacementTemplate> {
+    pub fn get_replacement_template(&self) -> &::std::option::Option<super::types::ReplacementTemplate> {
         &self.replacement_template
     }
     /// Consumes the builder and constructs a [`ReplacementEmailContent`](crate::types::ReplacementEmailContent).
-    pub fn build(self) -> crate::types::ReplacementEmailContent {
-        crate::types::ReplacementEmailContent {
+    pub fn build(self) -> super::types::ReplacementEmailContent {
+        super::types::ReplacementEmailContent {
             replacement_template: self.replacement_template,
         }
     }
