--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl SoaRecord {
     /// Creates a new builder-style object to manufacture [`SoaRecord`](crate::types::SoaRecord).
-    pub fn builder() -> crate::types::builders::SoaRecordBuilder {
-        crate::types::builders::SoaRecordBuilder::default()
+    pub fn builder() -> super::types::builders::SoaRecordBuilder {
+        super::types::builders::SoaRecordBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.serial_number
     }
     /// Consumes the builder and constructs a [`SoaRecord`](crate::types::SoaRecord).
-    pub fn build(self) -> crate::types::SoaRecord {
-        crate::types::SoaRecord {
+    pub fn build(self) -> super::types::SoaRecord {
+        super::types::SoaRecord {
             primary_name_server: self.primary_name_server,
             admin_email: self.admin_email,
             serial_number: self.serial_number.unwrap_or_default(),
