--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <li>
     /// <p>DELETE: remove the addresses from the suppression list.</p></li>
     /// </ul>
-    pub suppression_list_import_action: crate::types::SuppressionListImportAction,
+    pub suppression_list_import_action: super::super::types::SuppressionListImportAction,
 }
 impl SuppressionListDestination {
     /// <p>The type of action to perform on the address. The following are possible values:</p>
@@ -21,14 +21,14 @@
     /// <li>
     /// <p>DELETE: remove the addresses from the suppression list.</p></li>
     /// </ul>
-    pub fn suppression_list_import_action(&self) -> &crate::types::SuppressionListImportAction {
+    pub fn suppression_list_import_action(&self) -> &super::super::types::SuppressionListImportAction {
         &self.suppression_list_import_action
     }
 }
 impl SuppressionListDestination {
     /// Creates a new builder-style object to manufacture [`SuppressionListDestination`](crate::types::SuppressionListDestination).
-    pub fn builder() -> crate::types::builders::SuppressionListDestinationBuilder {
-        crate::types::builders::SuppressionListDestinationBuilder::default()
+    pub fn builder() -> super::super::types::builders::SuppressionListDestinationBuilder {
+        super::super::types::builders::SuppressionListDestinationBuilder::default()
     }
 }

@@ -36,7 +36,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct SuppressionListDestinationBuilder {
-    pub(crate) suppression_list_import_action: ::std::option::Option<crate::types::SuppressionListImportAction>,
+    pub(crate) suppression_list_import_action: ::std::option::Option<super::super::types::SuppressionListImportAction>,
 }
 impl SuppressionListDestinationBuilder {
     /// <p>The type of action to perform on the address. The following are possible values:</p>
@@ -47,7 +47,7 @@
     /// <p>DELETE: remove the addresses from the suppression list.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn suppression_list_import_action(mut self, input: crate::types::SuppressionListImportAction) -> Self {
+    pub fn suppression_list_import_action(mut self, input: super::super::types::SuppressionListImportAction) -> Self {
         self.suppression_list_import_action = ::std::option::Option::Some(input);
         self
     }
@@ -58,7 +58,7 @@
     /// <li>
     /// <p>DELETE: remove the addresses from the suppression list.</p></li>
     /// </ul>
-    pub fn set_suppression_list_import_action(mut self, input: ::std::option::Option<crate::types::SuppressionListImportAction>) -> Self {
+    pub fn set_suppression_list_import_action(mut self, input: ::std::option::Option<super::super::types::SuppressionListImportAction>) -> Self {
         self.suppression_list_import_action = input;
         self
     }
@@ -69,14 +69,14 @@
     /// <li>
     /// <p>DELETE: remove the addresses from the suppression list.</p></li>
     /// </ul>
-    pub fn get_suppression_list_import_action(&self) -> &::std::option::Option<crate::types::SuppressionListImportAction> {
+    pub fn get_suppression_list_import_action(&self) -> &::std::option::Option<super::super::types::SuppressionListImportAction> {
         &self.suppression_list_import_action
     }
     /// Consumes the builder and constructs a [`SuppressionListDestination`](crate::types::SuppressionListDestination).
     /// This method will fail if any of the following fields are not set:
     /// - [`suppression_list_import_action`](crate::types::builders::SuppressionListDestinationBuilder::suppression_list_import_action)
-    pub fn build(self) -> ::std::result::Result<crate::types::SuppressionListDestination, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SuppressionListDestination {
+    pub fn build(self) -> ::std::result::Result<super::super::types::SuppressionListDestination, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::SuppressionListDestination {
             suppression_list_import_action: self.suppression_list_import_action.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "suppression_list_import_action",
