--- original
+++ modified
@@ -7,11 +7,11 @@
     /// <p>The email address that is on the suppression list for your account or for a specific tenant.</p>
     pub email_address: ::std::string::String,
     /// <p>The reason that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub reason: crate::types::SuppressionListReason,
+    pub reason: super::types::SuppressionListReason,
     /// <p>The date and time when the suppressed destination was last updated, shown in Unix time format.</p>
     pub last_update_time: ::aws_smithy_types::DateTime,
     /// <p>An optional value that can contain additional information about the reasons that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub attributes: ::std::option::Option<crate::types::SuppressedDestinationAttributes>,
+    pub attributes: ::std::option::Option<super::types::SuppressedDestinationAttributes>,
     /// <p>The name of the tenant that the suppressed destination belongs to. This field is present only when the suppressed destination is on a tenant's suppression list.</p>
     pub tenant_name: ::std::option::Option<::std::string::String>,
 }
@@ -22,7 +22,7 @@
         self.email_address.deref()
     }
     /// <p>The reason that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn reason(&self) -> &crate::types::SuppressionListReason {
+    pub fn reason(&self) -> &super::types::SuppressionListReason {
         &self.reason
     }
     /// <p>The date and time when the suppressed destination was last updated, shown in Unix time format.</p>
@@ -30,7 +30,7 @@
         &self.last_update_time
     }
     /// <p>An optional value that can contain additional information about the reasons that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn attributes(&self) -> ::std::option::Option<&crate::types::SuppressedDestinationAttributes> {
+    pub fn attributes(&self) -> ::std::option::Option<&super::types::SuppressedDestinationAttributes> {
         self.attributes.as_ref()
     }
     /// <p>The name of the tenant that the suppressed destination belongs to. This field is present only when the suppressed destination is on a tenant's suppression list.</p>
@@ -40,8 +40,8 @@
 }
 impl SuppressedDestination {
     /// Creates a new builder-style object to manufacture [`SuppressedDestination`](crate::types::SuppressedDestination).
-    pub fn builder() -> crate::types::builders::SuppressedDestinationBuilder {
-        crate::types::builders::SuppressedDestinationBuilder::default()
+    pub fn builder() -> super::types::builders::SuppressedDestinationBuilder {
+        super::types::builders::SuppressedDestinationBuilder::default()
     }
 }

@@ -50,9 +50,9 @@
 #[non_exhaustive]
 pub struct SuppressedDestinationBuilder {
     pub(crate) email_address: ::std::option::Option<::std::string::String>,
-    pub(crate) reason: ::std::option::Option<crate::types::SuppressionListReason>,
+    pub(crate) reason: ::std::option::Option<super::types::SuppressionListReason>,
     pub(crate) last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) attributes: ::std::option::Option<crate::types::SuppressedDestinationAttributes>,
+    pub(crate) attributes: ::std::option::Option<super::types::SuppressedDestinationAttributes>,
     pub(crate) tenant_name: ::std::option::Option<::std::string::String>,
 }
 impl SuppressedDestinationBuilder {
@@ -73,17 +73,17 @@
     }
     /// <p>The reason that the address was added to the suppression list for your account or for a specific tenant.</p>
     /// This field is required.
-    pub fn reason(mut self, input: crate::types::SuppressionListReason) -> Self {
+    pub fn reason(mut self, input: super::types::SuppressionListReason) -> Self {
         self.reason = ::std::option::Option::Some(input);
         self
     }
     /// <p>The reason that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn set_reason(mut self, input: ::std::option::Option<crate::types::SuppressionListReason>) -> Self {
+    pub fn set_reason(mut self, input: ::std::option::Option<super::types::SuppressionListReason>) -> Self {
         self.reason = input;
         self
     }
     /// <p>The reason that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn get_reason(&self) -> &::std::option::Option<crate::types::SuppressionListReason> {
+    pub fn get_reason(&self) -> &::std::option::Option<super::types::SuppressionListReason> {
         &self.reason
     }
     /// <p>The date and time when the suppressed destination was last updated, shown in Unix time format.</p>
@@ -102,17 +102,17 @@
         &self.last_update_time
     }
     /// <p>An optional value that can contain additional information about the reasons that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn attributes(mut self, input: crate::types::SuppressedDestinationAttributes) -> Self {
+    pub fn attributes(mut self, input: super::types::SuppressedDestinationAttributes) -> Self {
         self.attributes = ::std::option::Option::Some(input);
         self
     }
     /// <p>An optional value that can contain additional information about the reasons that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn set_attributes(mut self, input: ::std::option::Option<crate::types::SuppressedDestinationAttributes>) -> Self {
+    pub fn set_attributes(mut self, input: ::std::option::Option<super::types::SuppressedDestinationAttributes>) -> Self {
         self.attributes = input;
         self
     }
     /// <p>An optional value that can contain additional information about the reasons that the address was added to the suppression list for your account or for a specific tenant.</p>
-    pub fn get_attributes(&self) -> &::std::option::Option<crate::types::SuppressedDestinationAttributes> {
+    pub fn get_attributes(&self) -> &::std::option::Option<super::types::SuppressedDestinationAttributes> {
         &self.attributes
     }
     /// <p>The name of the tenant that the suppressed destination belongs to. This field is present only when the suppressed destination is on a tenant's suppression list.</p>
@@ -134,8 +134,8 @@
     /// - [`email_address`](crate::types::builders::SuppressedDestinationBuilder::email_address)
     /// - [`reason`](crate::types::builders::SuppressedDestinationBuilder::reason)
     /// - [`last_update_time`](crate::types::builders::SuppressedDestinationBuilder::last_update_time)
-    pub fn build(self) -> ::std::result::Result<crate::types::SuppressedDestination, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SuppressedDestination {
+    pub fn build(self) -> ::std::result::Result<super::types::SuppressedDestination, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SuppressedDestination {
             email_address: self.email_address.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "email_address",
