--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct SuppressionConfidenceThreshold {
     /// <p>The confidence level threshold for suppression decisions.</p>
-    pub confidence_verdict_threshold: crate::types::SuppressionConfidenceVerdictThreshold,
+    pub confidence_verdict_threshold: super::types::SuppressionConfidenceVerdictThreshold,
 }
 impl SuppressionConfidenceThreshold {
     /// <p>The confidence level threshold for suppression decisions.</p>
-    pub fn confidence_verdict_threshold(&self) -> &crate::types::SuppressionConfidenceVerdictThreshold {
+    pub fn confidence_verdict_threshold(&self) -> &super::types::SuppressionConfidenceVerdictThreshold {
         &self.confidence_verdict_threshold
     }
 }
 impl SuppressionConfidenceThreshold {
     /// Creates a new builder-style object to manufacture [`SuppressionConfidenceThreshold`](crate::types::SuppressionConfidenceThreshold).
-    pub fn builder() -> crate::types::builders::SuppressionConfidenceThresholdBuilder {
-        crate::types::builders::SuppressionConfidenceThresholdBuilder::default()
+    pub fn builder() -> super::types::builders::SuppressionConfidenceThresholdBuilder {
+        super::types::builders::SuppressionConfidenceThresholdBuilder::default()
     }
 }

@@ -24,29 +24,29 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct SuppressionConfidenceThresholdBuilder {
-    pub(crate) confidence_verdict_threshold: ::std::option::Option<crate::types::SuppressionConfidenceVerdictThreshold>,
+    pub(crate) confidence_verdict_threshold: ::std::option::Option<super::types::SuppressionConfidenceVerdictThreshold>,
 }
 impl SuppressionConfidenceThresholdBuilder {
     /// <p>The confidence level threshold for suppression decisions.</p>
     /// This field is required.
-    pub fn confidence_verdict_threshold(mut self, input: crate::types::SuppressionConfidenceVerdictThreshold) -> Self {
+    pub fn confidence_verdict_threshold(mut self, input: super::types::SuppressionConfidenceVerdictThreshold) -> Self {
         self.confidence_verdict_threshold = ::std::option::Option::Some(input);
         self
     }
     /// <p>The confidence level threshold for suppression decisions.</p>
-    pub fn set_confidence_verdict_threshold(mut self, input: ::std::option::Option<crate::types::SuppressionConfidenceVerdictThreshold>) -> Self {
+    pub fn set_confidence_verdict_threshold(mut self, input: ::std::option::Option<super::types::SuppressionConfidenceVerdictThreshold>) -> Self {
         self.confidence_verdict_threshold = input;
         self
     }
     /// <p>The confidence level threshold for suppression decisions.</p>
-    pub fn get_confidence_verdict_threshold(&self) -> &::std::option::Option<crate::types::SuppressionConfidenceVerdictThreshold> {
+    pub fn get_confidence_verdict_threshold(&self) -> &::std::option::Option<super::types::SuppressionConfidenceVerdictThreshold> {
         &self.confidence_verdict_threshold
     }
     /// Consumes the builder and constructs a [`SuppressionConfidenceThreshold`](crate::types::SuppressionConfidenceThreshold).
     /// This method will fail if any of the following fields are not set:
     /// - [`confidence_verdict_threshold`](crate::types::builders::SuppressionConfidenceThresholdBuilder::confidence_verdict_threshold)
-    pub fn build(self) -> ::std::result::Result<crate::types::SuppressionConfidenceThreshold, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SuppressionConfidenceThreshold {
+    pub fn build(self) -> ::std::result::Result<super::types::SuppressionConfidenceThreshold, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SuppressionConfidenceThreshold {
             confidence_verdict_threshold: self.confidence_verdict_threshold.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "confidence_verdict_threshold",
