--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl TopicFilter {
     /// Creates a new builder-style object to manufacture [`TopicFilter`](crate::types::TopicFilter).
-    pub fn builder() -> crate::types::builders::TopicFilterBuilder {
-        crate::types::builders::TopicFilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::TopicFilterBuilder {
+        super::super::types::builders::TopicFilterBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.use_default_if_preference_unavailable
     }
     /// Consumes the builder and constructs a [`TopicFilter`](crate::types::TopicFilter).
-    pub fn build(self) -> crate::types::TopicFilter {
-        crate::types::TopicFilter {
+    pub fn build(self) -> super::super::types::TopicFilter {
+        super::super::types::TopicFilter {
             topic_name: self.topic_name,
             use_default_if_preference_unavailable: self.use_default_if_preference_unavailable.unwrap_or_default(),
         }
