--- original
+++ modified
@@ -5,24 +5,24 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct VdmOptions {
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Dashboard.</p>
-    pub dashboard_options: ::std::option::Option<crate::types::DashboardOptions>,
+    pub dashboard_options: ::std::option::Option<super::super::types::DashboardOptions>,
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Guardian.</p>
-    pub guardian_options: ::std::option::Option<crate::types::GuardianOptions>,
+    pub guardian_options: ::std::option::Option<super::super::types::GuardianOptions>,
 }
 impl VdmOptions {
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Dashboard.</p>
-    pub fn dashboard_options(&self) -> ::std::option::Option<&crate::types::DashboardOptions> {
+    pub fn dashboard_options(&self) -> ::std::option::Option<&super::super::types::DashboardOptions> {
         self.dashboard_options.as_ref()
     }
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Guardian.</p>
-    pub fn guardian_options(&self) -> ::std::option::Option<&crate::types::GuardianOptions> {
+    pub fn guardian_options(&self) -> ::std::option::Option<&super::super::types::GuardianOptions> {
         self.guardian_options.as_ref()
     }
 }
 impl VdmOptions {
     /// Creates a new builder-style object to manufacture [`VdmOptions`](crate::types::VdmOptions).
-    pub fn builder() -> crate::types::builders::VdmOptionsBuilder {
-        crate::types::builders::VdmOptionsBuilder::default()
+    pub fn builder() -> super::super::types::builders::VdmOptionsBuilder {
+        super::super::types::builders::VdmOptionsBuilder::default()
     }
 }

@@ -30,41 +30,41 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct VdmOptionsBuilder {
-    pub(crate) dashboard_options: ::std::option::Option<crate::types::DashboardOptions>,
-    pub(crate) guardian_options: ::std::option::Option<crate::types::GuardianOptions>,
+    pub(crate) dashboard_options: ::std::option::Option<super::super::types::DashboardOptions>,
+    pub(crate) guardian_options: ::std::option::Option<super::super::types::GuardianOptions>,
 }
 impl VdmOptionsBuilder {
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Dashboard.</p>
-    pub fn dashboard_options(mut self, input: crate::types::DashboardOptions) -> Self {
+    pub fn dashboard_options(mut self, input: super::super::types::DashboardOptions) -> Self {
         self.dashboard_options = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Dashboard.</p>
-    pub fn set_dashboard_options(mut self, input: ::std::option::Option<crate::types::DashboardOptions>) -> Self {
+    pub fn set_dashboard_options(mut self, input: ::std::option::Option<super::super::types::DashboardOptions>) -> Self {
         self.dashboard_options = input;
         self
     }
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Dashboard.</p>
-    pub fn get_dashboard_options(&self) -> &::std::option::Option<crate::types::DashboardOptions> {
+    pub fn get_dashboard_options(&self) -> &::std::option::Option<super::super::types::DashboardOptions> {
         &self.dashboard_options
     }
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Guardian.</p>
-    pub fn guardian_options(mut self, input: crate::types::GuardianOptions) -> Self {
+    pub fn guardian_options(mut self, input: super::super::types::GuardianOptions) -> Self {
         self.guardian_options = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Guardian.</p>
-    pub fn set_guardian_options(mut self, input: ::std::option::Option<crate::types::GuardianOptions>) -> Self {
+    pub fn set_guardian_options(mut self, input: ::std::option::Option<super::super::types::GuardianOptions>) -> Self {
         self.guardian_options = input;
         self
     }
     /// <p>Specifies additional settings for your VDM configuration as applicable to the Guardian.</p>
-    pub fn get_guardian_options(&self) -> &::std::option::Option<crate::types::GuardianOptions> {
+    pub fn get_guardian_options(&self) -> &::std::option::Option<super::super::types::GuardianOptions> {
         &self.guardian_options
     }
     /// Consumes the builder and constructs a [`VdmOptions`](crate::types::VdmOptions).
-    pub fn build(self) -> crate::types::VdmOptions {
-        crate::types::VdmOptions {
+    pub fn build(self) -> super::super::types::VdmOptions {
+        super::super::types::VdmOptions {
             dashboard_options: self.dashboard_options,
             guardian_options: self.guardian_options,
         }
