--- original
+++ modified
@@ -31,9 +31,9 @@
     /// <li>
     /// <p><code>REPLICATION_PRIMARY_INVALID_REGION</code> – The verification failed due to an invalid primary region specified. Ensure you provide a valid Amazon Web Services region where Amazon SES is available and different from the replica region.</p></li>
     /// </ul>
-    pub error_type: ::std::option::Option<crate::types::VerificationError>,
+    pub error_type: ::std::option::Option<super::types::VerificationError>,
     /// <p>An object that contains information about the start of authority (SOA) record associated with the identity.</p>
-    pub soa_record: ::std::option::Option<crate::types::SoaRecord>,
+    pub soa_record: ::std::option::Option<super::types::SoaRecord>,
 }
 impl VerificationInfo {
     /// <p>The last time a verification attempt was made for this identity.</p>
@@ -67,18 +67,18 @@
     /// <li>
     /// <p><code>REPLICATION_PRIMARY_INVALID_REGION</code> – The verification failed due to an invalid primary region specified. Ensure you provide a valid Amazon Web Services region where Amazon SES is available and different from the replica region.</p></li>
     /// </ul>
-    pub fn error_type(&self) -> ::std::option::Option<&crate::types::VerificationError> {
+    pub fn error_type(&self) -> ::std::option::Option<&super::types::VerificationError> {
         self.error_type.as_ref()
     }
     /// <p>An object that contains information about the start of authority (SOA) record associated with the identity.</p>
-    pub fn soa_record(&self) -> ::std::option::Option<&crate::types::SoaRecord> {
+    pub fn soa_record(&self) -> ::std::option::Option<&super::types::SoaRecord> {
         self.soa_record.as_ref()
     }
 }
 impl VerificationInfo {
     /// Creates a new builder-style object to manufacture [`VerificationInfo`](crate::types::VerificationInfo).
-    pub fn builder() -> crate::types::builders::VerificationInfoBuilder {
-        crate::types::builders::VerificationInfoBuilder::default()
+    pub fn builder() -> super::types::builders::VerificationInfoBuilder {
+        super::types::builders::VerificationInfoBuilder::default()
     }
 }

@@ -88,8 +88,8 @@
 pub struct VerificationInfoBuilder {
     pub(crate) last_checked_timestamp: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) last_success_timestamp: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) error_type: ::std::option::Option<crate::types::VerificationError>,
-    pub(crate) soa_record: ::std::option::Option<crate::types::SoaRecord>,
+    pub(crate) error_type: ::std::option::Option<super::types::VerificationError>,
+    pub(crate) soa_record: ::std::option::Option<super::types::SoaRecord>,
 }
 impl VerificationInfoBuilder {
     /// <p>The last time a verification attempt was made for this identity.</p>
@@ -143,7 +143,7 @@
     /// <li>
     /// <p><code>REPLICATION_PRIMARY_INVALID_REGION</code> – The verification failed due to an invalid primary region specified. Ensure you provide a valid Amazon Web Services region where Amazon SES is available and different from the replica region.</p></li>
     /// </ul>
-    pub fn error_type(mut self, input: crate::types::VerificationError) -> Self {
+    pub fn error_type(mut self, input: super::types::VerificationError) -> Self {
         self.error_type = ::std::option::Option::Some(input);
         self
     }
@@ -170,7 +170,7 @@
     /// <li>
     /// <p><code>REPLICATION_PRIMARY_INVALID_REGION</code> – The verification failed due to an invalid primary region specified. Ensure you provide a valid Amazon Web Services region where Amazon SES is available and different from the replica region.</p></li>
     /// </ul>
-    pub fn set_error_type(mut self, input: ::std::option::Option<crate::types::VerificationError>) -> Self {
+    pub fn set_error_type(mut self, input: ::std::option::Option<super::types::VerificationError>) -> Self {
         self.error_type = input;
         self
     }
@@ -197,26 +197,26 @@
     /// <li>
     /// <p><code>REPLICATION_PRIMARY_INVALID_REGION</code> – The verification failed due to an invalid primary region specified. Ensure you provide a valid Amazon Web Services region where Amazon SES is available and different from the replica region.</p></li>
     /// </ul>
-    pub fn get_error_type(&self) -> &::std::option::Option<crate::types::VerificationError> {
+    pub fn get_error_type(&self) -> &::std::option::Option<super::types::VerificationError> {
         &self.error_type
     }
     /// <p>An object that contains information about the start of authority (SOA) record associated with the identity.</p>
-    pub fn soa_record(mut self, input: crate::types::SoaRecord) -> Self {
+    pub fn soa_record(mut self, input: super::types::SoaRecord) -> Self {
         self.soa_record = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains information about the start of authority (SOA) record associated with the identity.</p>
-    pub fn set_soa_record(mut self, input: ::std::option::Option<crate::types::SoaRecord>) -> Self {
+    pub fn set_soa_record(mut self, input: ::std::option::Option<super::types::SoaRecord>) -> Self {
         self.soa_record = input;
         self
     }
     /// <p>An object that contains information about the start of authority (SOA) record associated with the identity.</p>
-    pub fn get_soa_record(&self) -> &::std::option::Option<crate::types::SoaRecord> {
+    pub fn get_soa_record(&self) -> &::std::option::Option<super::types::SoaRecord> {
         &self.soa_record
     }
     /// Consumes the builder and constructs a [`VerificationInfo`](crate::types::VerificationInfo).
-    pub fn build(self) -> crate::types::VerificationInfo {
-        crate::types::VerificationInfo {
+    pub fn build(self) -> super::types::VerificationInfo {
+        super::types::VerificationInfo {
             last_checked_timestamp: self.last_checked_timestamp,
             last_success_timestamp: self.last_success_timestamp,
             error_type: self.error_type,
