--- original
+++ modified
@@ -33,8 +33,8 @@
 }
 impl VolumeStatistics {
     /// Creates a new builder-style object to manufacture [`VolumeStatistics`](crate::types::VolumeStatistics).
-    pub fn builder() -> crate::types::builders::VolumeStatisticsBuilder {
-        crate::types::builders::VolumeStatisticsBuilder::default()
+    pub fn builder() -> super::types::builders::VolumeStatisticsBuilder {
+        super::types::builders::VolumeStatisticsBuilder::default()
     }
 }

@@ -105,8 +105,8 @@
         &self.projected_spam
     }
     /// Consumes the builder and constructs a [`VolumeStatistics`](crate::types::VolumeStatistics).
-    pub fn build(self) -> crate::types::VolumeStatistics {
-        crate::types::VolumeStatistics {
+    pub fn build(self) -> super::types::VolumeStatistics {
+        super::types::VolumeStatistics {
             inbox_raw_count: self.inbox_raw_count,
             spam_raw_count: self.spam_raw_count,
             projected_inbox: self.projected_inbox,
