--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl DescribeUserPoolClientInput {
     /// Creates a new builder-style object to manufacture [`DescribeUserPoolClientInput`](crate::operation::describe_user_pool_client::DescribeUserPoolClientInput).
-    pub fn builder() -> crate::operation::describe_user_pool_client::builders::DescribeUserPoolClientInputBuilder {
-        crate::operation::describe_user_pool_client::builders::DescribeUserPoolClientInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_user_pool_client::builders::DescribeUserPoolClientInputBuilder {
+        super::super::super::operation::describe_user_pool_client::builders::DescribeUserPoolClientInputBuilder::default()
     }
 }

@@ -76,10 +76,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_user_pool_client::DescribeUserPoolClientInput,
+        super::super::super::operation::describe_user_pool_client::DescribeUserPoolClientInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_user_pool_client::DescribeUserPoolClientInput {
+        ::std::result::Result::Ok(super::super::super::operation::describe_user_pool_client::DescribeUserPoolClientInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
         })
