--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteCustomVerificationEmailTemplateInput {
     /// Creates a new builder-style object to manufacture [`DeleteCustomVerificationEmailTemplateInput`](crate::operation::delete_custom_verification_email_template::DeleteCustomVerificationEmailTemplateInput).
-    pub fn builder() -> crate::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateInputBuilder {
-        crate::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateInputBuilder::default()
+    pub fn builder() -> super::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateInputBuilder {
+        super::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateInputBuilder::default()
     }
 }

@@ -46,11 +46,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_custom_verification_email_template::DeleteCustomVerificationEmailTemplateInput,
+        super::operation::delete_custom_verification_email_template::DeleteCustomVerificationEmailTemplateInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::delete_custom_verification_email_template::DeleteCustomVerificationEmailTemplateInput {
+            super::operation::delete_custom_verification_email_template::DeleteCustomVerificationEmailTemplateInput {
                 template_name: self.template_name,
             },
         )
