--- original
+++ modified
@@ -18,7 +18,7 @@
     ///   - [`authentication_result(Option<AuthenticationResultType>)`](crate::operation::admin_initiate_auth::AdminInitiateAuthOutput::authentication_result): <p>The outcome of successful authentication. This is only returned if the user pool has no additional challenges to return. If Amazon Cognito returns another challenge, the response includes <code>ChallengeName</code>, <code>ChallengeParameters</code>, and <code>Session</code> so that your user can answer the challenge.</p>
     ///   - [`available_challenges(Option<Vec::<ChallengeNameType>>)`](crate::operation::admin_initiate_auth::AdminInitiateAuthOutput::available_challenges): <p>This response parameter lists the available authentication challenges that users can select from in <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/authentication-flows-selection-sdk.html#authentication-flows-selection-choice">choice-based authentication</a>. For example, they might be able to choose between passkey authentication, a one-time password from an SMS message, and a traditional password.</p>
     /// - On failure, responds with [`SdkError<AdminInitiateAuthError>`](crate::operation::admin_initiate_auth::AdminInitiateAuthError)
-    pub fn admin_initiate_auth(&self) -> crate::operation::admin_initiate_auth::builders::AdminInitiateAuthFluentBuilder {
-        crate::operation::admin_initiate_auth::builders::AdminInitiateAuthFluentBuilder::new(self.handle.clone())
+    pub fn admin_initiate_auth(&self) -> super::super::operation::admin_initiate_auth::builders::AdminInitiateAuthFluentBuilder {
+        super::super::operation::admin_initiate_auth::builders::AdminInitiateAuthFluentBuilder::new(self.handle.clone())
     }
 }
