--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`permanent(bool)`](crate::operation::admin_set_user_password::builders::AdminSetUserPasswordFluentBuilder::permanent) / [`set_permanent(Option<bool>)`](crate::operation::admin_set_user_password::builders::AdminSetUserPasswordFluentBuilder::set_permanent):<br>required: **false**<br><p>Set to <code>true</code> to set a password that the user can immediately sign in with. Set to <code>false</code> to set a temporary password that the user must change on their next sign-in.</p><br>
     /// - On success, responds with [`AdminSetUserPasswordOutput`](crate::operation::admin_set_user_password::AdminSetUserPasswordOutput)
     /// - On failure, responds with [`SdkError<AdminSetUserPasswordError>`](crate::operation::admin_set_user_password::AdminSetUserPasswordError)
-    pub fn admin_set_user_password(&self) -> crate::operation::admin_set_user_password::builders::AdminSetUserPasswordFluentBuilder {
-        crate::operation::admin_set_user_password::builders::AdminSetUserPasswordFluentBuilder::new(self.handle.clone())
+    pub fn admin_set_user_password(&self) -> super::operation::admin_set_user_password::builders::AdminSetUserPasswordFluentBuilder {
+        super::operation::admin_set_user_password::builders::AdminSetUserPasswordFluentBuilder::new(self.handle.clone())
     }
 }
