--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`device_remembered_status(DeviceRememberedStatusType)`](crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusFluentBuilder::device_remembered_status) / [`set_device_remembered_status(Option<DeviceRememberedStatusType>)`](crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusFluentBuilder::set_device_remembered_status):<br>required: **false**<br><p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p><br>
     /// - On success, responds with [`AdminUpdateDeviceStatusOutput`](crate::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput)
     /// - On failure, responds with [`SdkError<AdminUpdateDeviceStatusError>`](crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError)
-    pub fn admin_update_device_status(&self) -> crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusFluentBuilder {
-        crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusFluentBuilder::new(self.handle.clone())
+    pub fn admin_update_device_status(&self) -> super::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusFluentBuilder {
+        super::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusFluentBuilder::new(self.handle.clone())
     }
 }
