--- original
+++ modified
@@ -8,7 +8,7 @@
     ///   - [`access_token(impl Into<String>)`](crate::operation::change_password::builders::ChangePasswordFluentBuilder::access_token) / [`set_access_token(Option<String>)`](crate::operation::change_password::builders::ChangePasswordFluentBuilder::set_access_token):<br>required: **true**<br><p>A valid access token that Amazon Cognito issued to the user whose password you want to change.</p><br>
     /// - On success, responds with [`ChangePasswordOutput`](crate::operation::change_password::ChangePasswordOutput)
     /// - On failure, responds with [`SdkError<ChangePasswordError>`](crate::operation::change_password::ChangePasswordError)
-    pub fn change_password(&self) -> crate::operation::change_password::builders::ChangePasswordFluentBuilder {
-        crate::operation::change_password::builders::ChangePasswordFluentBuilder::new(self.handle.clone())
+    pub fn change_password(&self) -> super::super::operation::change_password::builders::ChangePasswordFluentBuilder {
+        super::super::operation::change_password::builders::ChangePasswordFluentBuilder::new(self.handle.clone())
     }
 }
