--- original
+++ modified
@@ -10,7 +10,7 @@
     /// - On success, responds with [`ConfirmDeviceOutput`](crate::operation::confirm_device::ConfirmDeviceOutput) with field(s):
     ///   - [`user_confirmation_necessary(bool)`](crate::operation::confirm_device::ConfirmDeviceOutput::user_confirmation_necessary): <p>When <code>true</code>, your user must confirm that they want to remember the device. Prompt the user for an answer.</p> <p>When <code>false</code>, immediately sets the device as remembered and eligible for device authentication.</p> <p>You can configure your user pool to always remember devices, in which case this response is <code>false</code>, or to allow users to opt in, in which case this response is <code>true</code>. Configure this option under <i>Device tracking</i> in the <i>Sign-in</i> menu of your user pool.</p>
     /// - On failure, responds with [`SdkError<ConfirmDeviceError>`](crate::operation::confirm_device::ConfirmDeviceError)
-    pub fn confirm_device(&self) -> crate::operation::confirm_device::builders::ConfirmDeviceFluentBuilder {
-        crate::operation::confirm_device::builders::ConfirmDeviceFluentBuilder::new(self.handle.clone())
+    pub fn confirm_device(&self) -> super::operation::confirm_device::builders::ConfirmDeviceFluentBuilder {
+        super::operation::confirm_device::builders::ConfirmDeviceFluentBuilder::new(self.handle.clone())
     }
 }
