--- original
+++ modified
@@ -32,7 +32,7 @@
     /// - On success, responds with [`CreateUserPoolOutput`](crate::operation::create_user_pool::CreateUserPoolOutput) with field(s):
     ///   - [`user_pool(Option<UserPoolType>)`](crate::operation::create_user_pool::CreateUserPoolOutput::user_pool): <p>The details of the created user pool.</p>
     /// - On failure, responds with [`SdkError<CreateUserPoolError>`](crate::operation::create_user_pool::CreateUserPoolError)
-    pub fn create_user_pool(&self) -> crate::operation::create_user_pool::builders::CreateUserPoolFluentBuilder {
-        crate::operation::create_user_pool::builders::CreateUserPoolFluentBuilder::new(self.handle.clone())
+    pub fn create_user_pool(&self) -> super::super::operation::create_user_pool::builders::CreateUserPoolFluentBuilder {
+        super::super::operation::create_user_pool::builders::CreateUserPoolFluentBuilder::new(self.handle.clone())
     }
 }
