--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On success, responds with [`CreateUserPoolReplicaOutput`](crate::operation::create_user_pool_replica::CreateUserPoolReplicaOutput) with field(s):
     ///   - [`user_pool_replica(Option<UserPoolReplicaType>)`](crate::operation::create_user_pool_replica::CreateUserPoolReplicaOutput::user_pool_replica): <p>Information about the created user pool replica, including its status and role.</p>
     /// - On failure, responds with [`SdkError<CreateUserPoolReplicaError>`](crate::operation::create_user_pool_replica::CreateUserPoolReplicaError)
-    pub fn create_user_pool_replica(&self) -> crate::operation::create_user_pool_replica::builders::CreateUserPoolReplicaFluentBuilder {
-        crate::operation::create_user_pool_replica::builders::CreateUserPoolReplicaFluentBuilder::new(self.handle.clone())
+    pub fn create_user_pool_replica(&self) -> super::super::operation::create_user_pool_replica::builders::CreateUserPoolReplicaFluentBuilder {
+        super::super::operation::create_user_pool_replica::builders::CreateUserPoolReplicaFluentBuilder::new(self.handle.clone())
     }
 }
