--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`user_pool_id(impl Into<String>)`](crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainFluentBuilder::user_pool_id) / [`set_user_pool_id(Option<String>)`](crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainFluentBuilder::set_user_pool_id):<br>required: **true**<br><p>The ID of the user pool where you want to delete the domain.</p><br>
     /// - On success, responds with [`DeleteUserPoolDomainOutput`](crate::operation::delete_user_pool_domain::DeleteUserPoolDomainOutput)
     /// - On failure, responds with [`SdkError<DeleteUserPoolDomainError>`](crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError)
-    pub fn delete_user_pool_domain(&self) -> crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainFluentBuilder {
-        crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainFluentBuilder::new(self.handle.clone())
+    pub fn delete_user_pool_domain(&self) -> super::super::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainFluentBuilder {
+        super::super::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainFluentBuilder::new(self.handle.clone())
     }
 }
