--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`DescribeUserImportJobOutput`](crate::operation::describe_user_import_job::DescribeUserImportJobOutput) with field(s):
     ///   - [`user_import_job(Option<UserImportJobType>)`](crate::operation::describe_user_import_job::DescribeUserImportJobOutput::user_import_job): <p>The details of the user import job. Includes logging destination, status, and the Amazon S3 pre-signed URL for CSV upload.</p>
     /// - On failure, responds with [`SdkError<DescribeUserImportJobError>`](crate::operation::describe_user_import_job::DescribeUserImportJobError)
-    pub fn describe_user_import_job(&self) -> crate::operation::describe_user_import_job::builders::DescribeUserImportJobFluentBuilder {
-        crate::operation::describe_user_import_job::builders::DescribeUserImportJobFluentBuilder::new(self.handle.clone())
+    pub fn describe_user_import_job(&self) -> super::operation::describe_user_import_job::builders::DescribeUserImportJobFluentBuilder {
+        super::operation::describe_user_import_job::builders::DescribeUserImportJobFluentBuilder::new(self.handle.clone())
     }
 }
