--- original
+++ modified
@@ -11,8 +11,8 @@
     /// - On failure, responds with [`SdkError<DescribeManagedLoginBrandingByClientError>`](crate::operation::describe_managed_login_branding_by_client::DescribeManagedLoginBrandingByClientError)
     pub fn describe_managed_login_branding_by_client(
         &self,
-    ) -> crate::operation::describe_managed_login_branding_by_client::builders::DescribeManagedLoginBrandingByClientFluentBuilder {
-        crate::operation::describe_managed_login_branding_by_client::builders::DescribeManagedLoginBrandingByClientFluentBuilder::new(
+    ) -> super::super::operation::describe_managed_login_branding_by_client::builders::DescribeManagedLoginBrandingByClientFluentBuilder {
+        super::super::operation::describe_managed_login_branding_by_client::builders::DescribeManagedLoginBrandingByClientFluentBuilder::new(
             self.handle.clone(),
         )
     }
