--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On failure, responds with [`SdkError<DescribeManagedLoginBrandingError>`](crate::operation::describe_managed_login_branding::DescribeManagedLoginBrandingError)
     pub fn describe_managed_login_branding(
         &self,
-    ) -> crate::operation::describe_managed_login_branding::builders::DescribeManagedLoginBrandingFluentBuilder {
-        crate::operation::describe_managed_login_branding::builders::DescribeManagedLoginBrandingFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::describe_managed_login_branding::builders::DescribeManagedLoginBrandingFluentBuilder {
+        super::super::operation::describe_managed_login_branding::builders::DescribeManagedLoginBrandingFluentBuilder::new(self.handle.clone())
     }
 }
