--- original
+++ modified
@@ -12,7 +12,7 @@
     /// - On success, responds with [`ForgotPasswordOutput`](crate::operation::forgot_password::ForgotPasswordOutput) with field(s):
     ///   - [`code_delivery_details(Option<CodeDeliveryDetailsType>)`](crate::operation::forgot_password::ForgotPasswordOutput::code_delivery_details): <p>Information about the phone number or email address that Amazon Cognito sent the password-recovery code to.</p>
     /// - On failure, responds with [`SdkError<ForgotPasswordError>`](crate::operation::forgot_password::ForgotPasswordError)
-    pub fn forgot_password(&self) -> crate::operation::forgot_password::builders::ForgotPasswordFluentBuilder {
-        crate::operation::forgot_password::builders::ForgotPasswordFluentBuilder::new(self.handle.clone())
+    pub fn forgot_password(&self) -> super::operation::forgot_password::builders::ForgotPasswordFluentBuilder {
+        super::operation::forgot_password::builders::ForgotPasswordFluentBuilder::new(self.handle.clone())
     }
 }
