--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`device_key(impl Into<String>)`](crate::operation::forget_device::builders::ForgetDeviceFluentBuilder::device_key) / [`set_device_key(Option<String>)`](crate::operation::forget_device::builders::ForgetDeviceFluentBuilder::set_device_key):<br>required: **true**<br><p>The unique identifier, or device key, of the device that the user wants to forget.</p><br>
     /// - On success, responds with [`ForgetDeviceOutput`](crate::operation::forget_device::ForgetDeviceOutput)
     /// - On failure, responds with [`SdkError<ForgetDeviceError>`](crate::operation::forget_device::ForgetDeviceError)
-    pub fn forget_device(&self) -> crate::operation::forget_device::builders::ForgetDeviceFluentBuilder {
-        crate::operation::forget_device::builders::ForgetDeviceFluentBuilder::new(self.handle.clone())
+    pub fn forget_device(&self) -> super::operation::forget_device::builders::ForgetDeviceFluentBuilder {
+        super::operation::forget_device::builders::ForgetDeviceFluentBuilder::new(self.handle.clone())
     }
 }
