--- original
+++ modified
@@ -11,7 +11,7 @@
     ///   - [`preferred_mfa_setting(Option<String>)`](crate::operation::get_user::GetUserOutput::preferred_mfa_setting): <p>The user's preferred MFA. Users can prefer SMS message, email message, or TOTP MFA.</p>
     ///   - [`user_mfa_setting_list(Option<Vec::<String>>)`](crate::operation::get_user::GetUserOutput::user_mfa_setting_list): <p>The MFA options that are activated for the user. The possible values in this list are <code>SMS_MFA</code>, <code>EMAIL_OTP</code>, and <code>SOFTWARE_TOKEN_MFA</code>.</p>
     /// - On failure, responds with [`SdkError<GetUserError>`](crate::operation::get_user::GetUserError)
-    pub fn get_user(&self) -> crate::operation::get_user::builders::GetUserFluentBuilder {
-        crate::operation::get_user::builders::GetUserFluentBuilder::new(self.handle.clone())
+    pub fn get_user(&self) -> super::operation::get_user::builders::GetUserFluentBuilder {
+        super::operation::get_user::builders::GetUserFluentBuilder::new(self.handle.clone())
     }
 }
