--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`GetGroupOutput`](crate::operation::get_group::GetGroupOutput) with field(s):
     ///   - [`group(Option<GroupType>)`](crate::operation::get_group::GetGroupOutput::group): <p>A container for the requested group. Includes description, precedence, and IAM role values.</p>
     /// - On failure, responds with [`SdkError<GetGroupError>`](crate::operation::get_group::GetGroupError)
-    pub fn get_group(&self) -> crate::operation::get_group::builders::GetGroupFluentBuilder {
-        crate::operation::get_group::builders::GetGroupFluentBuilder::new(self.handle.clone())
+    pub fn get_group(&self) -> super::super::operation::get_group::builders::GetGroupFluentBuilder {
+        super::super::operation::get_group::builders::GetGroupFluentBuilder::new(self.handle.clone())
     }
 }
