--- original
+++ modified
@@ -7,7 +7,7 @@
     /// - On success, responds with [`GetSigningCertificateOutput`](crate::operation::get_signing_certificate::GetSigningCertificateOutput) with field(s):
     ///   - [`certificate(Option<String>)`](crate::operation::get_signing_certificate::GetSigningCertificateOutput::certificate): <p>The x.509 certificate that signs SAML 2.0 authentication requests for your user pool.</p>
     /// - On failure, responds with [`SdkError<GetSigningCertificateError>`](crate::operation::get_signing_certificate::GetSigningCertificateError)
-    pub fn get_signing_certificate(&self) -> crate::operation::get_signing_certificate::builders::GetSigningCertificateFluentBuilder {
-        crate::operation::get_signing_certificate::builders::GetSigningCertificateFluentBuilder::new(self.handle.clone())
+    pub fn get_signing_certificate(&self) -> super::super::operation::get_signing_certificate::builders::GetSigningCertificateFluentBuilder {
+        super::super::operation::get_signing_certificate::builders::GetSigningCertificateFluentBuilder::new(self.handle.clone())
     }
 }
