--- original
+++ modified
@@ -17,7 +17,7 @@
     ///   - [`authentication_result(Option<AuthenticationResultType>)`](crate::operation::initiate_auth::InitiateAuthOutput::authentication_result): <p>The result of a successful and complete authentication request. This result is only returned if the user doesn't need to pass another challenge. If they must pass another challenge before they get tokens, Amazon Cognito returns a challenge in <code>ChallengeName</code>, <code>ChallengeParameters</code>, and <code>Session</code> response parameters.</p>
     ///   - [`available_challenges(Option<Vec::<ChallengeNameType>>)`](crate::operation::initiate_auth::InitiateAuthOutput::available_challenges): <p>This response parameter lists the available authentication challenges that users can select from in <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/authentication-flows-selection-sdk.html#authentication-flows-selection-choice">choice-based authentication</a>. For example, they might be able to choose between passkey authentication, a one-time password from an SMS message, and a traditional password.</p>
     /// - On failure, responds with [`SdkError<InitiateAuthError>`](crate::operation::initiate_auth::InitiateAuthError)
-    pub fn initiate_auth(&self) -> crate::operation::initiate_auth::builders::InitiateAuthFluentBuilder {
-        crate::operation::initiate_auth::builders::InitiateAuthFluentBuilder::new(self.handle.clone())
+    pub fn initiate_auth(&self) -> super::super::operation::initiate_auth::builders::InitiateAuthFluentBuilder {
+        super::super::operation::initiate_auth::builders::InitiateAuthFluentBuilder::new(self.handle.clone())
     }
 }
