--- original
+++ modified
@@ -11,7 +11,7 @@
     ///   - [`user_pool_clients(Option<Vec::<UserPoolClientDescription>>)`](crate::operation::list_user_pool_clients::ListUserPoolClientsOutput::user_pool_clients): <p>An array of app clients and their details. Includes app client ID and name.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_user_pool_clients::ListUserPoolClientsOutput::next_token): <p>The identifier that Amazon Cognito returned with the previous request to this operation. When you include a pagination token in your request, Amazon Cognito returns the next set of items in the list. By use of this token, you can paginate through the full list of items.</p>
     /// - On failure, responds with [`SdkError<ListUserPoolClientsError>`](crate::operation::list_user_pool_clients::ListUserPoolClientsError)
-    pub fn list_user_pool_clients(&self) -> crate::operation::list_user_pool_clients::builders::ListUserPoolClientsFluentBuilder {
-        crate::operation::list_user_pool_clients::builders::ListUserPoolClientsFluentBuilder::new(self.handle.clone())
+    pub fn list_user_pool_clients(&self) -> super::operation::list_user_pool_clients::builders::ListUserPoolClientsFluentBuilder {
+        super::operation::list_user_pool_clients::builders::ListUserPoolClientsFluentBuilder::new(self.handle.clone())
     }
 }
