--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`user_import_jobs(Option<Vec::<UserImportJobType>>)`](crate::operation::list_user_import_jobs::ListUserImportJobsOutput::user_import_jobs): <p>An array of user import jobs from the requested user pool. For each, the response includes logging destination, status, and the Amazon S3 pre-signed URL for CSV upload.</p>
     ///   - [`pagination_token(Option<String>)`](crate::operation::list_user_import_jobs::ListUserImportJobsOutput::pagination_token): <p>The identifier that Amazon Cognito returned with the previous request to this operation. When you include a pagination token in your request, Amazon Cognito returns the next set of items in the list. By use of this token, you can paginate through the full list of items.</p>
     /// - On failure, responds with [`SdkError<ListUserImportJobsError>`](crate::operation::list_user_import_jobs::ListUserImportJobsError)
-    pub fn list_user_import_jobs(&self) -> crate::operation::list_user_import_jobs::builders::ListUserImportJobsFluentBuilder {
-        crate::operation::list_user_import_jobs::builders::ListUserImportJobsFluentBuilder::new(self.handle.clone())
+    pub fn list_user_import_jobs(&self) -> super::operation::list_user_import_jobs::builders::ListUserImportJobsFluentBuilder {
+        super::operation::list_user_import_jobs::builders::ListUserImportJobsFluentBuilder::new(self.handle.clone())
     }
 }
