--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`devices(Option<Vec::<DeviceType>>)`](crate::operation::list_devices::ListDevicesOutput::devices): <p>An array of devices and their details. Each entry that's returned includes device information, last-accessed and created dates, and the device key.</p>
     ///   - [`pagination_token(Option<String>)`](crate::operation::list_devices::ListDevicesOutput::pagination_token): <p>The identifier that Amazon Cognito returned with the previous request to this operation. When you include a pagination token in your request, Amazon Cognito returns the next set of items in the list. By use of this token, you can paginate through the full list of items.</p>
     /// - On failure, responds with [`SdkError<ListDevicesError>`](crate::operation::list_devices::ListDevicesError)
-    pub fn list_devices(&self) -> crate::operation::list_devices::builders::ListDevicesFluentBuilder {
-        crate::operation::list_devices::builders::ListDevicesFluentBuilder::new(self.handle.clone())
+    pub fn list_devices(&self) -> super::super::operation::list_devices::builders::ListDevicesFluentBuilder {
+        super::super::operation::list_devices::builders::ListDevicesFluentBuilder::new(self.handle.clone())
     }
 }
