--- original
+++ modified
@@ -13,7 +13,7 @@
     ///   - [`users(Option<Vec::<UserType>>)`](crate::operation::list_users::ListUsersOutput::users): <p>An array of user pool users who match your query, and their attributes. Between different requests, you might observe variations in the sequence that users in this response object are sorted into. The sort order of users isn't guaranteed to follow a single pattern, but the paginated list from a single chain of requests won't return duplicates.</p>
     ///   - [`pagination_token(Option<String>)`](crate::operation::list_users::ListUsersOutput::pagination_token): <p>The identifier that Amazon Cognito returned with the previous request to this operation. When you include a pagination token in your request, Amazon Cognito returns the next set of items in the list. By use of this token, you can paginate through the full list of items.</p>
     /// - On failure, responds with [`SdkError<ListUsersError>`](crate::operation::list_users::ListUsersError)
-    pub fn list_users(&self) -> crate::operation::list_users::builders::ListUsersFluentBuilder {
-        crate::operation::list_users::builders::ListUsersFluentBuilder::new(self.handle.clone())
+    pub fn list_users(&self) -> super::operation::list_users::builders::ListUsersFluentBuilder {
+        super::operation::list_users::builders::ListUsersFluentBuilder::new(self.handle.clone())
     }
 }
