--- original
+++ modified
@@ -12,7 +12,7 @@
     /// - On success, responds with [`ResendConfirmationCodeOutput`](crate::operation::resend_confirmation_code::ResendConfirmationCodeOutput) with field(s):
     ///   - [`code_delivery_details(Option<CodeDeliveryDetailsType>)`](crate::operation::resend_confirmation_code::ResendConfirmationCodeOutput::code_delivery_details): <p>Information about the phone number or email address that Amazon Cognito sent the confirmation code to.</p>
     /// - On failure, responds with [`SdkError<ResendConfirmationCodeError>`](crate::operation::resend_confirmation_code::ResendConfirmationCodeError)
-    pub fn resend_confirmation_code(&self) -> crate::operation::resend_confirmation_code::builders::ResendConfirmationCodeFluentBuilder {
-        crate::operation::resend_confirmation_code::builders::ResendConfirmationCodeFluentBuilder::new(self.handle.clone())
+    pub fn resend_confirmation_code(&self) -> super::super::operation::resend_confirmation_code::builders::ResendConfirmationCodeFluentBuilder {
+        super::super::operation::resend_confirmation_code::builders::ResendConfirmationCodeFluentBuilder::new(self.handle.clone())
     }
 }
