--- original
+++ modified
@@ -10,7 +10,7 @@
     /// - On failure, responds with [`SdkError<SetLogDeliveryConfigurationError>`](crate::operation::set_log_delivery_configuration::SetLogDeliveryConfigurationError)
     pub fn set_log_delivery_configuration(
         &self,
-    ) -> crate::operation::set_log_delivery_configuration::builders::SetLogDeliveryConfigurationFluentBuilder {
-        crate::operation::set_log_delivery_configuration::builders::SetLogDeliveryConfigurationFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::set_log_delivery_configuration::builders::SetLogDeliveryConfigurationFluentBuilder {
+        super::operation::set_log_delivery_configuration::builders::SetLogDeliveryConfigurationFluentBuilder::new(self.handle.clone())
     }
 }
