--- original
+++ modified
@@ -16,7 +16,7 @@
     ///   - [`mfa_configuration(Option<UserPoolMfaType>)`](crate::operation::set_user_pool_mfa_config::SetUserPoolMfaConfigOutput::mfa_configuration): <p>Displays multi-factor authentication (MFA) as on, off, or optional. When <code>ON</code>, all users must set up MFA before they can sign in. When <code>OPTIONAL</code>, your application must make a client-side determination of whether a user wants to register an MFA device. For user pools with adaptive authentication with threat protection, choose <code>OPTIONAL</code>.</p> <p>When <code>MfaConfiguration</code> is <code>OPTIONAL</code>, managed login doesn't automatically prompt users to set up MFA. Amazon Cognito generates MFA prompts in API responses and in managed login for users who have chosen and configured a preferred MFA factor.</p>
     ///   - [`web_authn_configuration(Option<WebAuthnConfigurationType>)`](crate::operation::set_user_pool_mfa_config::SetUserPoolMfaConfigOutput::web_authn_configuration): <p>The configuration of your user pool for passkey, or WebAuthn, sign-in with authenticators such as biometric and security-key devices. Includes relying-party configuration and settings for user-verification requirements.</p>
     /// - On failure, responds with [`SdkError<SetUserPoolMfaConfigError>`](crate::operation::set_user_pool_mfa_config::SetUserPoolMfaConfigError)
-    pub fn set_user_pool_mfa_config(&self) -> crate::operation::set_user_pool_mfa_config::builders::SetUserPoolMfaConfigFluentBuilder {
-        crate::operation::set_user_pool_mfa_config::builders::SetUserPoolMfaConfigFluentBuilder::new(self.handle.clone())
+    pub fn set_user_pool_mfa_config(&self) -> super::super::operation::set_user_pool_mfa_config::builders::SetUserPoolMfaConfigFluentBuilder {
+        super::super::operation::set_user_pool_mfa_config::builders::SetUserPoolMfaConfigFluentBuilder::new(self.handle.clone())
     }
 }
