--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`StartUserImportJobOutput`](crate::operation::start_user_import_job::StartUserImportJobOutput) with field(s):
     ///   - [`user_import_job(Option<UserImportJobType>)`](crate::operation::start_user_import_job::StartUserImportJobOutput::user_import_job): <p>The details of the user import job. Includes logging destination, status, and the Amazon S3 pre-signed URL for CSV upload.</p>
     /// - On failure, responds with [`SdkError<StartUserImportJobError>`](crate::operation::start_user_import_job::StartUserImportJobError)
-    pub fn start_user_import_job(&self) -> crate::operation::start_user_import_job::builders::StartUserImportJobFluentBuilder {
-        crate::operation::start_user_import_job::builders::StartUserImportJobFluentBuilder::new(self.handle.clone())
+    pub fn start_user_import_job(&self) -> super::operation::start_user_import_job::builders::StartUserImportJobFluentBuilder {
+        super::operation::start_user_import_job::builders::StartUserImportJobFluentBuilder::new(self.handle.clone())
     }
 }
