--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On success, responds with [`UpdateGroupOutput`](crate::operation::update_group::UpdateGroupOutput) with field(s):
     ///   - [`group(Option<GroupType>)`](crate::operation::update_group::UpdateGroupOutput::group): <p>Contains the updated details of the group, including precedence, IAM role, and description.</p>
     /// - On failure, responds with [`SdkError<UpdateGroupError>`](crate::operation::update_group::UpdateGroupError)
-    pub fn update_group(&self) -> crate::operation::update_group::builders::UpdateGroupFluentBuilder {
-        crate::operation::update_group::builders::UpdateGroupFluentBuilder::new(self.handle.clone())
+    pub fn update_group(&self) -> super::super::operation::update_group::builders::UpdateGroupFluentBuilder {
+        super::super::operation::update_group::builders::UpdateGroupFluentBuilder::new(self.handle.clone())
     }
 }
