--- original
+++ modified
@@ -8,7 +8,7 @@
     ///   - [`device_remembered_status(DeviceRememberedStatusType)`](crate::operation::update_device_status::builders::UpdateDeviceStatusFluentBuilder::device_remembered_status) / [`set_device_remembered_status(Option<DeviceRememberedStatusType>)`](crate::operation::update_device_status::builders::UpdateDeviceStatusFluentBuilder::set_device_remembered_status):<br>required: **false**<br><p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p><br>
     /// - On success, responds with [`UpdateDeviceStatusOutput`](crate::operation::update_device_status::UpdateDeviceStatusOutput)
     /// - On failure, responds with [`SdkError<UpdateDeviceStatusError>`](crate::operation::update_device_status::UpdateDeviceStatusError)
-    pub fn update_device_status(&self) -> crate::operation::update_device_status::builders::UpdateDeviceStatusFluentBuilder {
-        crate::operation::update_device_status::builders::UpdateDeviceStatusFluentBuilder::new(self.handle.clone())
+    pub fn update_device_status(&self) -> super::super::operation::update_device_status::builders::UpdateDeviceStatusFluentBuilder {
+        super::super::operation::update_device_status::builders::UpdateDeviceStatusFluentBuilder::new(self.handle.clone())
     }
 }
