--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On success, responds with [`UpdateUserAttributesOutput`](crate::operation::update_user_attributes::UpdateUserAttributesOutput) with field(s):
     ///   - [`code_delivery_details_list(Option<Vec::<CodeDeliveryDetailsType>>)`](crate::operation::update_user_attributes::UpdateUserAttributesOutput::code_delivery_details_list): <p>When the attribute-update request includes an email address or phone number attribute, Amazon Cognito sends a message to users with a code that confirms ownership of the new value that they entered. The <code>CodeDeliveryDetails</code> object is information about the delivery destination for that link or code. This behavior happens in user pools configured to automatically verify changes to those attributes. For more information, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/signing-up-users-in-your-app.html#verifying-when-users-change-their-email-or-phone-number">Verifying when users change their email or phone number</a>.</p>
     /// - On failure, responds with [`SdkError<UpdateUserAttributesError>`](crate::operation::update_user_attributes::UpdateUserAttributesError)
-    pub fn update_user_attributes(&self) -> crate::operation::update_user_attributes::builders::UpdateUserAttributesFluentBuilder {
-        crate::operation::update_user_attributes::builders::UpdateUserAttributesFluentBuilder::new(self.handle.clone())
+    pub fn update_user_attributes(&self) -> super::super::operation::update_user_attributes::builders::UpdateUserAttributesFluentBuilder {
+        super::super::operation::update_user_attributes::builders::UpdateUserAttributesFluentBuilder::new(self.handle.clone())
     }
 }
