--- original
+++ modified
@@ -8,7 +8,7 @@
     ///   - [`code(impl Into<String>)`](crate::operation::verify_user_attribute::builders::VerifyUserAttributeFluentBuilder::code) / [`set_code(Option<String>)`](crate::operation::verify_user_attribute::builders::VerifyUserAttributeFluentBuilder::set_code):<br>required: **true**<br><p>The verification code that your user pool sent to the added or changed attribute, for example the user's email address.</p><br>
     /// - On success, responds with [`VerifyUserAttributeOutput`](crate::operation::verify_user_attribute::VerifyUserAttributeOutput)
     /// - On failure, responds with [`SdkError<VerifyUserAttributeError>`](crate::operation::verify_user_attribute::VerifyUserAttributeError)
-    pub fn verify_user_attribute(&self) -> crate::operation::verify_user_attribute::builders::VerifyUserAttributeFluentBuilder {
-        crate::operation::verify_user_attribute::builders::VerifyUserAttributeFluentBuilder::new(self.handle.clone())
+    pub fn verify_user_attribute(&self) -> super::operation::verify_user_attribute::builders::VerifyUserAttributeFluentBuilder {
+        super::operation::verify_user_attribute::builders::VerifyUserAttributeFluentBuilder::new(self.handle.clone())
     }
 }
