--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl DeleteMessageBatchRequestEntry {
     /// Creates a new builder-style object to manufacture [`DeleteMessageBatchRequestEntry`](crate::types::DeleteMessageBatchRequestEntry).
-    pub fn builder() -> crate::types::builders::DeleteMessageBatchRequestEntryBuilder {
-        crate::types::builders::DeleteMessageBatchRequestEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeleteMessageBatchRequestEntryBuilder {
+        super::super::types::builders::DeleteMessageBatchRequestEntryBuilder::default()
     }
 }

@@ -85,8 +85,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`id`](crate::types::builders::DeleteMessageBatchRequestEntryBuilder::id)
     /// - [`receipt_handle`](crate::types::builders::DeleteMessageBatchRequestEntryBuilder::receipt_handle)
-    pub fn build(self) -> ::std::result::Result<crate::types::DeleteMessageBatchRequestEntry, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::DeleteMessageBatchRequestEntry {
+    pub fn build(self) -> ::std::result::Result<super::super::types::DeleteMessageBatchRequestEntry, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::DeleteMessageBatchRequestEntry {
             id: self.id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "id",
