--- original
+++ modified
@@ -47,8 +47,8 @@
 }
 impl MessageAttributeValue {
     /// Creates a new builder-style object to manufacture [`MessageAttributeValue`](crate::types::MessageAttributeValue).
-    pub fn builder() -> crate::types::builders::MessageAttributeValueBuilder {
-        crate::types::builders::MessageAttributeValueBuilder::default()
+    pub fn builder() -> super::super::types::builders::MessageAttributeValueBuilder {
+        super::super::types::builders::MessageAttributeValueBuilder::default()
     }
 }

@@ -152,8 +152,8 @@
     /// Consumes the builder and constructs a [`MessageAttributeValue`](crate::types::MessageAttributeValue).
     /// This method will fail if any of the following fields are not set:
     /// - [`data_type`](crate::types::builders::MessageAttributeValueBuilder::data_type)
-    pub fn build(self) -> ::std::result::Result<crate::types::MessageAttributeValue, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::MessageAttributeValue {
+    pub fn build(self) -> ::std::result::Result<super::super::types::MessageAttributeValue, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::MessageAttributeValue {
             string_value: self.string_value,
             binary_value: self.binary_value,
             string_list_values: self.string_list_values,
