--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetMessageInsightsInput {
     /// Creates a new builder-style object to manufacture [`GetMessageInsightsInput`](crate::operation::get_message_insights::GetMessageInsightsInput).
-    pub fn builder() -> crate::operation::get_message_insights::builders::GetMessageInsightsInputBuilder {
-        crate::operation::get_message_insights::builders::GetMessageInsightsInputBuilder::default()
+    pub fn builder() -> super::operation::get_message_insights::builders::GetMessageInsightsInputBuilder {
+        super::operation::get_message_insights::builders::GetMessageInsightsInputBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`GetMessageInsightsInput`](crate::operation::get_message_insights::GetMessageInsightsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_message_insights::GetMessageInsightsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_message_insights::GetMessageInsightsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_message_insights::GetMessageInsightsInput { message_id: self.message_id })
+        ::std::result::Result::Ok(super::operation::get_message_insights::GetMessageInsightsInput { message_id: self.message_id })
     }
 }
