--- original
+++ modified
@@ -105,7 +105,7 @@
     /// </ul>
     /// <p>If you set these attributes to anything other than the values shown for enabling high throughput, normal throughput is in effect and deduplication occurs as specified.</p>
     /// <p>For information on throughput quotas, see <a href="https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html">Quotas related to messages</a> in the <i>Amazon SQS Developer Guide</i>.</p>
-    pub attribute_names: ::std::option::Option<::std::vec::Vec<crate::types::QueueAttributeName>>,
+    pub attribute_names: ::std::option::Option<::std::vec::Vec<super::super::super::types::QueueAttributeName>>,
 }
 impl GetQueueAttributesInput {
     /// <p>The URL of the Amazon SQS queue whose attribute information is retrieved.</p>
@@ -213,14 +213,14 @@
     /// <p>For information on throughput quotas, see <a href="https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html">Quotas related to messages</a> in the <i>Amazon SQS Developer Guide</i>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.attribute_names.is_none()`.
-    pub fn attribute_names(&self) -> &[crate::types::QueueAttributeName] {
+    pub fn attribute_names(&self) -> &[super::super::super::types::QueueAttributeName] {
         self.attribute_names.as_deref().unwrap_or_default()
     }
 }
 impl GetQueueAttributesInput {
     /// Creates a new builder-style object to manufacture [`GetQueueAttributesInput`](crate::operation::get_queue_attributes::GetQueueAttributesInput).
-    pub fn builder() -> crate::operation::get_queue_attributes::builders::GetQueueAttributesInputBuilder {
-        crate::operation::get_queue_attributes::builders::GetQueueAttributesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_queue_attributes::builders::GetQueueAttributesInputBuilder {
+        super::super::super::operation::get_queue_attributes::builders::GetQueueAttributesInputBuilder::default()
     }
 }

@@ -229,7 +229,7 @@
 #[non_exhaustive]
 pub struct GetQueueAttributesInputBuilder {
     pub(crate) queue_url: ::std::option::Option<::std::string::String>,
-    pub(crate) attribute_names: ::std::option::Option<::std::vec::Vec<crate::types::QueueAttributeName>>,
+    pub(crate) attribute_names: ::std::option::Option<::std::vec::Vec<super::super::super::types::QueueAttributeName>>,
 }
 impl GetQueueAttributesInputBuilder {
     /// <p>The URL of the Amazon SQS queue whose attribute information is retrieved.</p>
@@ -352,7 +352,7 @@
     /// </ul>
     /// <p>If you set these attributes to anything other than the values shown for enabling high throughput, normal throughput is in effect and deduplication occurs as specified.</p>
     /// <p>For information on throughput quotas, see <a href="https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html">Quotas related to messages</a> in the <i>Amazon SQS Developer Guide</i>.</p>
-    pub fn attribute_names(mut self, input: crate::types::QueueAttributeName) -> Self {
+    pub fn attribute_names(mut self, input: super::super::super::types::QueueAttributeName) -> Self {
         let mut v = self.attribute_names.unwrap_or_default();
         v.push(input);
         self.attribute_names = ::std::option::Option::Some(v);
@@ -456,7 +456,7 @@
     /// </ul>
     /// <p>If you set these attributes to anything other than the values shown for enabling high throughput, normal throughput is in effect and deduplication occurs as specified.</p>
     /// <p>For information on throughput quotas, see <a href="https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html">Quotas related to messages</a> in the <i>Amazon SQS Developer Guide</i>.</p>
-    pub fn set_attribute_names(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::QueueAttributeName>>) -> Self {
+    pub fn set_attribute_names(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::QueueAttributeName>>) -> Self {
         self.attribute_names = input;
         self
     }
@@ -558,15 +558,15 @@
     /// </ul>
     /// <p>If you set these attributes to anything other than the values shown for enabling high throughput, normal throughput is in effect and deduplication occurs as specified.</p>
     /// <p>For information on throughput quotas, see <a href="https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html">Quotas related to messages</a> in the <i>Amazon SQS Developer Guide</i>.</p>
-    pub fn get_attribute_names(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::QueueAttributeName>> {
+    pub fn get_attribute_names(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::QueueAttributeName>> {
         &self.attribute_names
     }
     /// Consumes the builder and constructs a [`GetQueueAttributesInput`](crate::operation::get_queue_attributes::GetQueueAttributesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_queue_attributes::GetQueueAttributesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_queue_attributes::GetQueueAttributesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_queue_attributes::GetQueueAttributesInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_queue_attributes::GetQueueAttributesInput {
             queue_url: self.queue_url,
             attribute_names: self.attribute_names,
         })
