--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl DescribeOrganizationConfigRuleStatusesInput {
     /// Creates a new builder-style object to manufacture [`DescribeOrganizationConfigRuleStatusesInput`](crate::operation::describe_organization_config_rule_statuses::DescribeOrganizationConfigRuleStatusesInput).
-    pub fn builder() -> crate::operation::describe_organization_config_rule_statuses::builders::DescribeOrganizationConfigRuleStatusesInputBuilder {
-        crate::operation::describe_organization_config_rule_statuses::builders::DescribeOrganizationConfigRuleStatusesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_organization_config_rule_statuses::builders::DescribeOrganizationConfigRuleStatusesInputBuilder {
+        super::super::super::operation::describe_organization_config_rule_statuses::builders::DescribeOrganizationConfigRuleStatusesInputBuilder::default()
     }
 }

@@ -94,11 +94,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_organization_config_rule_statuses::DescribeOrganizationConfigRuleStatusesInput,
+        super::super::super::operation::describe_organization_config_rule_statuses::DescribeOrganizationConfigRuleStatusesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::describe_organization_config_rule_statuses::DescribeOrganizationConfigRuleStatusesInput {
+            super::super::super::operation::describe_organization_config_rule_statuses::DescribeOrganizationConfigRuleStatusesInput {
                 organization_config_rule_names: self.organization_config_rule_names,
                 limit: self.limit,
                 next_token: self.next_token,
