--- original
+++ modified
@@ -13,12 +13,12 @@
 ///
 #[derive(::std::clone::Clone, ::std::fmt::Debug)]
 pub struct FunctionActiveFluentBuilder {
-    handle: ::std::sync::Arc<crate::client::Handle>,
-    inner: crate::operation::get_function_configuration::builders::GetFunctionConfigurationInputBuilder,
+    handle: ::std::sync::Arc<super::super::client::Handle>,
+    inner: super::super::operation::get_function_configuration::builders::GetFunctionConfigurationInputBuilder,
 }
 impl FunctionActiveFluentBuilder {
     /// Creates a new `FunctionActiveFluentBuilder`.
-    pub(crate) fn new(handle: ::std::sync::Arc<crate::client::Handle>) -> Self {
+    pub(crate) fn new(handle: ::std::sync::Arc<super::super::client::Handle>) -> Self {
         Self {
             handle,
             inner: ::std::default::Default::default(),
@@ -25,7 +25,7 @@
         }
     }
     /// Access the GetFunctionConfiguration as a reference.
-    pub fn as_input(&self) -> &crate::operation::get_function_configuration::builders::GetFunctionConfigurationInputBuilder {
+    pub fn as_input(&self) -> &super::super::operation::get_function_configuration::builders::GetFunctionConfigurationInputBuilder {
         &self.inner
     }
     /// Waits for the function's State to be Active. This waiter uses GetFunctionConfiguration API. This should be used after new function creation.
@@ -32,18 +32,18 @@
     pub async fn wait(
         self,
         max_wait: ::std::time::Duration,
-    ) -> ::std::result::Result<crate::waiters::function_active::FunctionActiveFinalPoll, crate::waiters::function_active::WaitUntilFunctionActiveError>
+    ) -> ::std::result::Result<super::super::waiters::function_active::FunctionActiveFinalPoll, super::super::waiters::function_active::WaitUntilFunctionActiveError>
     {
         let input = self
             .inner
             .build()
             .map_err(::aws_smithy_runtime_api::client::waiters::error::WaiterError::construction_failure)?;
-        let runtime_plugins = crate::operation::get_function_configuration::GetFunctionConfiguration::operation_runtime_plugins(
+        let runtime_plugins = super::super::operation::get_function_configuration::GetFunctionConfiguration::operation_runtime_plugins(
             self.handle.runtime_plugins.clone(),
             &self.handle.conf,
             ::std::option::Option::None,
         )
-        .with_operation_plugin(crate::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
+        .with_operation_plugin(super::super::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
         let mut cfg = ::aws_smithy_types::config_bag::ConfigBag::base();
         let runtime_components_builder = runtime_plugins
             .apply_client_configuration(&mut cfg)
@@ -53,19 +53,19 @@
         let time_source = time_components.time_source().expect("a time source is required by waiters");

         let acceptor = move |result: ::std::result::Result<
-            &crate::operation::get_function_configuration::GetFunctionConfigurationOutput,
-            &crate::operation::get_function_configuration::GetFunctionConfigurationError,
+            &super::super::operation::get_function_configuration::GetFunctionConfigurationOutput,
+            &super::super::operation::get_function_configuration::GetFunctionConfigurationError,
         >| {
             // Matches: {"output":{"path":"State","expected":"Active","comparator":"stringEquals"}}
-            if crate::waiters::matchers::match_get_function_configuration_9fb739f92fd5b515a(result) {
+            if super::super::waiters::matchers::match_get_function_configuration_9fb739f92fd5b515a(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Success;
             }
             // Matches: {"output":{"path":"State","expected":"Failed","comparator":"stringEquals"}}
-            if crate::waiters::matchers::match_get_function_configuration_87a1a2d4cc2e2d1e1(result) {
+            if super::super::waiters::matchers::match_get_function_configuration_87a1a2d4cc2e2d1e1(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Failure;
             }
             // Matches: {"output":{"path":"State","expected":"Pending","comparator":"stringEquals"}}
-            if crate::waiters::matchers::match_get_function_configuration_ff08d54fec9077aeb(result) {
+            if super::super::waiters::matchers::match_get_function_configuration_ff08d54fec9077aeb(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Retry;
             }
             ::aws_smithy_runtime::client::waiters::AcceptorState::NoAcceptorsMatched
@@ -73,7 +73,7 @@
         let operation = move || {
             let input = input.clone();
             let runtime_plugins = runtime_plugins.clone();
-            async move { crate::operation::get_function_configuration::GetFunctionConfiguration::orchestrate(&runtime_plugins, input).await }
+            async move { super::super::operation::get_function_configuration::GetFunctionConfiguration::orchestrate(&runtime_plugins, input).await }
         };
         let orchestrator = ::aws_smithy_runtime::client::waiters::WaiterOrchestrator::builder()
             .min_delay(::std::time::Duration::from_secs(5))
@@ -148,9 +148,9 @@

 /// Successful return type for the `function_active` waiter.
 pub type FunctionActiveFinalPoll = ::aws_smithy_runtime_api::client::waiters::FinalPoll<
-    crate::operation::get_function_configuration::GetFunctionConfigurationOutput,
+    super::super::operation::get_function_configuration::GetFunctionConfigurationOutput,
     ::aws_smithy_runtime_api::client::result::SdkError<
-        crate::operation::get_function_configuration::GetFunctionConfigurationError,
+        super::super::operation::get_function_configuration::GetFunctionConfigurationError,
         ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
     >,
 >;
@@ -157,6 +157,6 @@

 /// Error type for the `function_active` waiter.
 pub type WaitUntilFunctionActiveError = ::aws_smithy_runtime_api::client::waiters::error::WaiterError<
-    crate::operation::get_function_configuration::GetFunctionConfigurationOutput,
-    crate::operation::get_function_configuration::GetFunctionConfigurationError,
+    super::super::operation::get_function_configuration::GetFunctionConfigurationOutput,
+    super::super::operation::get_function_configuration::GetFunctionConfigurationError,
 >;
