--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetBucketWebsiteInput {
     /// Creates a new builder-style object to manufacture [`GetBucketWebsiteInput`](crate::operation::get_bucket_website::GetBucketWebsiteInput).
-    pub fn builder() -> crate::operation::get_bucket_website::builders::GetBucketWebsiteInputBuilder {
-        crate::operation::get_bucket_website::builders::GetBucketWebsiteInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_bucket_website::builders::GetBucketWebsiteInputBuilder {
+        super::super::super::operation::get_bucket_website::builders::GetBucketWebsiteInputBuilder::default()
     }
 }

@@ -65,8 +65,8 @@
     /// Consumes the builder and constructs a [`GetBucketWebsiteInput`](crate::operation::get_bucket_website::GetBucketWebsiteInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_bucket_website::GetBucketWebsiteInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_bucket_website::GetBucketWebsiteInput {
+    ) -> ::std::result::Result<super::super::super::operation::get_bucket_website::GetBucketWebsiteInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_bucket_website::GetBucketWebsiteInput {
             bucket: self.bucket,
             expected_bucket_owner: self.expected_bucket_owner,
         })
