--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetDeliverabilityTestReportInput {
     /// Creates a new builder-style object to manufacture [`GetDeliverabilityTestReportInput`](crate::operation::get_deliverability_test_report::GetDeliverabilityTestReportInput).
-    pub fn builder() -> crate::operation::get_deliverability_test_report::builders::GetDeliverabilityTestReportInputBuilder {
-        crate::operation::get_deliverability_test_report::builders::GetDeliverabilityTestReportInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_deliverability_test_report::builders::GetDeliverabilityTestReportInputBuilder {
+        super::super::super::operation::get_deliverability_test_report::builders::GetDeliverabilityTestReportInputBuilder::default()
     }
 }

@@ -46,9 +46,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_deliverability_test_report::GetDeliverabilityTestReportInput,
+        super::super::super::operation::get_deliverability_test_report::GetDeliverabilityTestReportInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_deliverability_test_report::GetDeliverabilityTestReportInput { report_id: self.report_id })
+        ::std::result::Result::Ok(super::super::super::operation::get_deliverability_test_report::GetDeliverabilityTestReportInput { report_id: self.report_id })
     }
 }
