--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl PutResourcePolicyInput {
     /// Creates a new builder-style object to manufacture [`PutResourcePolicyInput`](crate::operation::put_resource_policy::PutResourcePolicyInput).
-    pub fn builder() -> crate::operation::put_resource_policy::builders::PutResourcePolicyInputBuilder {
-        crate::operation::put_resource_policy::builders::PutResourcePolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_resource_policy::builders::PutResourcePolicyInputBuilder {
+        super::super::super::operation::put_resource_policy::builders::PutResourcePolicyInputBuilder::default()
     }
 }

@@ -92,8 +92,8 @@
     /// Consumes the builder and constructs a [`PutResourcePolicyInput`](crate::operation::put_resource_policy::PutResourcePolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_resource_policy::PutResourcePolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::put_resource_policy::PutResourcePolicyInput {
+    ) -> ::std::result::Result<super::super::super::operation::put_resource_policy::PutResourcePolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::put_resource_policy::PutResourcePolicyInput {
             resource_arn: self.resource_arn,
             policy: self.policy,
             revision_id: self.revision_id,
