--- original
+++ modified
@@ -52,8 +52,8 @@
 }
 impl DescribeKeyInput {
     /// Creates a new builder-style object to manufacture [`DescribeKeyInput`](crate::operation::describe_key::DescribeKeyInput).
-    pub fn builder() -> crate::operation::describe_key::builders::DescribeKeyInputBuilder {
-        crate::operation::describe_key::builders::DescribeKeyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_key::builders::DescribeKeyInputBuilder {
+        super::super::super::operation::describe_key::builders::DescribeKeyInputBuilder::default()
     }
 }

@@ -146,8 +146,8 @@
         &self.grant_tokens
     }
     /// Consumes the builder and constructs a [`DescribeKeyInput`](crate::operation::describe_key::DescribeKeyInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::describe_key::DescribeKeyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::describe_key::DescribeKeyInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::describe_key::DescribeKeyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::describe_key::DescribeKeyInput {
             key_id: self.key_id,
             grant_tokens: self.grant_tokens,
         })
