--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl CancelJobInput {
     /// Creates a new builder-style object to manufacture [`CancelJobInput`](crate::operation::cancel_job::CancelJobInput).
-    pub fn builder() -> crate::operation::cancel_job::builders::CancelJobInputBuilder {
-        crate::operation::cancel_job::builders::CancelJobInputBuilder::default()
+    pub fn builder() -> super::operation::cancel_job::builders::CancelJobInputBuilder {
+        super::operation::cancel_job::builders::CancelJobInputBuilder::default()
     }
 }

@@ -70,8 +70,8 @@
         &self.reason
     }
     /// Consumes the builder and constructs a [`CancelJobInput`](crate::operation::cancel_job::CancelJobInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::cancel_job::CancelJobInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::cancel_job::CancelJobInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::cancel_job::CancelJobInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::cancel_job::CancelJobInput {
             job_id: self.job_id,
             reason: self.reason,
         })
