--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteTenantInput {
     /// Creates a new builder-style object to manufacture [`DeleteTenantInput`](crate::operation::delete_tenant::DeleteTenantInput).
-    pub fn builder() -> crate::operation::delete_tenant::builders::DeleteTenantInputBuilder {
-        crate::operation::delete_tenant::builders::DeleteTenantInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_tenant::builders::DeleteTenantInputBuilder {
+        super::super::super::operation::delete_tenant::builders::DeleteTenantInputBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`DeleteTenantInput`](crate::operation::delete_tenant::DeleteTenantInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_tenant::DeleteTenantInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::delete_tenant::DeleteTenantInput {
+    ) -> ::std::result::Result<super::super::super::operation::delete_tenant::DeleteTenantInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::delete_tenant::DeleteTenantInput {
             tenant_name: self.tenant_name,
         })
     }
