--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the configuration set to associate with a dedicated IP pool.</p>
     pub configuration_set_name: ::std::option::Option<::std::string::String>,
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub tls_policy: ::std::option::Option<crate::types::TlsPolicy>,
+    pub tls_policy: ::std::option::Option<super::super::super::types::TlsPolicy>,
     /// <p>The name of the dedicated IP pool to associate with the configuration set.</p>
     pub sending_pool_name: ::std::option::Option<::std::string::String>,
     /// <p>The maximum amount of time, in seconds, that Amazon SES API v2 will attempt delivery of email. If specified, the value must greater than or equal to 300 seconds (5 minutes) and less than or equal to 50400 seconds (840 minutes).</p>
@@ -19,7 +19,7 @@
         self.configuration_set_name.as_deref()
     }
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn tls_policy(&self) -> ::std::option::Option<&crate::types::TlsPolicy> {
+    pub fn tls_policy(&self) -> ::std::option::Option<&super::super::super::types::TlsPolicy> {
         self.tls_policy.as_ref()
     }
     /// <p>The name of the dedicated IP pool to associate with the configuration set.</p>
@@ -33,8 +33,8 @@
 }
 impl PutConfigurationSetDeliveryOptionsInput {
     /// Creates a new builder-style object to manufacture [`PutConfigurationSetDeliveryOptionsInput`](crate::operation::put_configuration_set_delivery_options::PutConfigurationSetDeliveryOptionsInput).
-    pub fn builder() -> crate::operation::put_configuration_set_delivery_options::builders::PutConfigurationSetDeliveryOptionsInputBuilder {
-        crate::operation::put_configuration_set_delivery_options::builders::PutConfigurationSetDeliveryOptionsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_configuration_set_delivery_options::builders::PutConfigurationSetDeliveryOptionsInputBuilder {
+        super::super::super::operation::put_configuration_set_delivery_options::builders::PutConfigurationSetDeliveryOptionsInputBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
 #[non_exhaustive]
 pub struct PutConfigurationSetDeliveryOptionsInputBuilder {
     pub(crate) configuration_set_name: ::std::option::Option<::std::string::String>,
-    pub(crate) tls_policy: ::std::option::Option<crate::types::TlsPolicy>,
+    pub(crate) tls_policy: ::std::option::Option<super::super::super::types::TlsPolicy>,
     pub(crate) sending_pool_name: ::std::option::Option<::std::string::String>,
     pub(crate) max_delivery_seconds: ::std::option::Option<i64>,
 }
@@ -64,17 +64,17 @@
         &self.configuration_set_name
     }
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn tls_policy(mut self, input: crate::types::TlsPolicy) -> Self {
+    pub fn tls_policy(mut self, input: super::super::super::types::TlsPolicy) -> Self {
         self.tls_policy = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn set_tls_policy(mut self, input: ::std::option::Option<crate::types::TlsPolicy>) -> Self {
+    pub fn set_tls_policy(mut self, input: ::std::option::Option<super::super::super::types::TlsPolicy>) -> Self {
         self.tls_policy = input;
         self
     }
     /// <p>Specifies whether messages that use the configuration set are required to use Transport Layer Security (TLS). If the value is <code>Require</code>, messages are only delivered if a TLS connection can be established. If the value is <code>Optional</code>, messages can be delivered in plain text if a TLS connection can't be established.</p>
-    pub fn get_tls_policy(&self) -> &::std::option::Option<crate::types::TlsPolicy> {
+    pub fn get_tls_policy(&self) -> &::std::option::Option<super::super::super::types::TlsPolicy> {
         &self.tls_policy
     }
     /// <p>The name of the dedicated IP pool to associate with the configuration set.</p>
@@ -109,11 +109,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_configuration_set_delivery_options::PutConfigurationSetDeliveryOptionsInput,
+        super::super::super::operation::put_configuration_set_delivery_options::PutConfigurationSetDeliveryOptionsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_configuration_set_delivery_options::PutConfigurationSetDeliveryOptionsInput {
+            super::super::super::operation::put_configuration_set_delivery_options::PutConfigurationSetDeliveryOptionsInput {
                 configuration_set_name: self.configuration_set_name,
                 tls_policy: self.tls_policy,
                 sending_pool_name: self.sending_pool_name,
