--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl UntagSamlProviderInput {
     /// Creates a new builder-style object to manufacture [`UntagSamlProviderInput`](crate::operation::untag_saml_provider::UntagSamlProviderInput).
-    pub fn builder() -> crate::operation::untag_saml_provider::builders::UntagSamlProviderInputBuilder {
-        crate::operation::untag_saml_provider::builders::UntagSamlProviderInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::untag_saml_provider::builders::UntagSamlProviderInputBuilder {
+        super::super::super::operation::untag_saml_provider::builders::UntagSamlProviderInputBuilder::default()
     }
 }

@@ -78,8 +78,8 @@
     /// Consumes the builder and constructs a [`UntagSamlProviderInput`](crate::operation::untag_saml_provider::UntagSamlProviderInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::untag_saml_provider::UntagSamlProviderInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::untag_saml_provider::UntagSamlProviderInput {
+    ) -> ::std::result::Result<super::super::super::operation::untag_saml_provider::UntagSamlProviderInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::untag_saml_provider::UntagSamlProviderInput {
             saml_provider_arn: self.saml_provider_arn,
             tag_keys: self.tag_keys,
         })
