--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl CancelKeyDeletionInput {
     /// Creates a new builder-style object to manufacture [`CancelKeyDeletionInput`](crate::operation::cancel_key_deletion::CancelKeyDeletionInput).
-    pub fn builder() -> crate::operation::cancel_key_deletion::builders::CancelKeyDeletionInputBuilder {
-        crate::operation::cancel_key_deletion::builders::CancelKeyDeletionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::cancel_key_deletion::builders::CancelKeyDeletionInputBuilder {
+        super::super::super::operation::cancel_key_deletion::builders::CancelKeyDeletionInputBuilder::default()
     }
 }

@@ -89,7 +89,7 @@
     /// Consumes the builder and constructs a [`CancelKeyDeletionInput`](crate::operation::cancel_key_deletion::CancelKeyDeletionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::cancel_key_deletion::CancelKeyDeletionInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::cancel_key_deletion::CancelKeyDeletionInput { key_id: self.key_id })
+    ) -> ::std::result::Result<super::super::super::operation::cancel_key_deletion::CancelKeyDeletionInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::cancel_key_deletion::CancelKeyDeletionInput { key_id: self.key_id })
     }
 }
