--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl AdminSetUserPasswordInput {
     /// Creates a new builder-style object to manufacture [`AdminSetUserPasswordInput`](crate::operation::admin_set_user_password::AdminSetUserPasswordInput).
-    pub fn builder() -> crate::operation::admin_set_user_password::builders::AdminSetUserPasswordInputBuilder {
-        crate::operation::admin_set_user_password::builders::AdminSetUserPasswordInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::admin_set_user_password::builders::AdminSetUserPasswordInputBuilder {
+        super::super::super::operation::admin_set_user_password::builders::AdminSetUserPasswordInputBuilder::default()
     }
 }

@@ -119,9 +119,9 @@
     /// Consumes the builder and constructs a [`AdminSetUserPasswordInput`](crate::operation::admin_set_user_password::AdminSetUserPasswordInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::admin_set_user_password::AdminSetUserPasswordInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::admin_set_user_password::AdminSetUserPasswordInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::admin_set_user_password::AdminSetUserPasswordInput {
+        ::std::result::Result::Ok(super::super::super::operation::admin_set_user_password::AdminSetUserPasswordInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
             password: self.password,
