--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteUserPoolDomainInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserPoolDomainInput`](crate::operation::delete_user_pool_domain::DeleteUserPoolDomainInput).
-    pub fn builder() -> crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainInputBuilder {
-        crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainInputBuilder {
+        super::super::super::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainInputBuilder::default()
     }
 }

@@ -66,9 +66,9 @@
     /// Consumes the builder and constructs a [`DeleteUserPoolDomainInput`](crate::operation::delete_user_pool_domain::DeleteUserPoolDomainInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_user_pool_domain::DeleteUserPoolDomainInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_user_pool_domain::DeleteUserPoolDomainInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_user_pool_domain::DeleteUserPoolDomainInput {
             domain: self.domain,
             user_pool_id: self.user_pool_id,
         })
