--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl PutDedicatedIpInPoolInput {
     /// Creates a new builder-style object to manufacture [`PutDedicatedIpInPoolInput`](crate::operation::put_dedicated_ip_in_pool::PutDedicatedIpInPoolInput).
-    pub fn builder() -> crate::operation::put_dedicated_ip_in_pool::builders::PutDedicatedIpInPoolInputBuilder {
-        crate::operation::put_dedicated_ip_in_pool::builders::PutDedicatedIpInPoolInputBuilder::default()
+    pub fn builder() -> super::operation::put_dedicated_ip_in_pool::builders::PutDedicatedIpInPoolInputBuilder {
+        super::operation::put_dedicated_ip_in_pool::builders::PutDedicatedIpInPoolInputBuilder::default()
     }
 }

@@ -67,9 +67,9 @@
     /// Consumes the builder and constructs a [`PutDedicatedIpInPoolInput`](crate::operation::put_dedicated_ip_in_pool::PutDedicatedIpInPoolInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_dedicated_ip_in_pool::PutDedicatedIpInPoolInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::put_dedicated_ip_in_pool::PutDedicatedIpInPoolInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::put_dedicated_ip_in_pool::PutDedicatedIpInPoolInput {
+        ::std::result::Result::Ok(super::operation::put_dedicated_ip_in_pool::PutDedicatedIpInPoolInput {
             ip: self.ip,
             destination_pool_name: self.destination_pool_name,
         })
