--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetCodeSigningConfigInput {
     /// Creates a new builder-style object to manufacture [`GetCodeSigningConfigInput`](crate::operation::get_code_signing_config::GetCodeSigningConfigInput).
-    pub fn builder() -> crate::operation::get_code_signing_config::builders::GetCodeSigningConfigInputBuilder {
-        crate::operation::get_code_signing_config::builders::GetCodeSigningConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_code_signing_config::builders::GetCodeSigningConfigInputBuilder {
+        super::super::super::operation::get_code_signing_config::builders::GetCodeSigningConfigInputBuilder::default()
     }
 }

@@ -44,9 +44,9 @@
     /// Consumes the builder and constructs a [`GetCodeSigningConfigInput`](crate::operation::get_code_signing_config::GetCodeSigningConfigInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_code_signing_config::GetCodeSigningConfigInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_code_signing_config::GetCodeSigningConfigInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_code_signing_config::GetCodeSigningConfigInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_code_signing_config::GetCodeSigningConfigInput {
             code_signing_config_arn: self.code_signing_config_arn,
         })
     }
