--- original
+++ modified
@@ -12,9 +12,9 @@
     /// <p>A list of tags that you want to attach to the new IAM SAML provider. Each tag consists of a key name and an associated value. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p><note>
     /// <p>If any one of the tags is invalid or if you exceed the allowed maximum number of tags, then the entire request fails and the resource is not created.</p>
     /// </note>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::types::Tag>>,
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub assertion_encryption_mode: ::std::option::Option<crate::types::AssertionEncryptionModeType>,
+    pub assertion_encryption_mode: ::std::option::Option<super::types::AssertionEncryptionModeType>,
     /// <p>The private key generated from your external identity provider. The private key must be a .pem file that uses AES-GCM or AES-CBC encryption algorithm to decrypt SAML assertions.</p>
     pub add_private_key: ::std::option::Option<::std::string::String>,
 }
@@ -34,11 +34,11 @@
     /// </note>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn assertion_encryption_mode(&self) -> ::std::option::Option<&crate::types::AssertionEncryptionModeType> {
+    pub fn assertion_encryption_mode(&self) -> ::std::option::Option<&super::types::AssertionEncryptionModeType> {
         self.assertion_encryption_mode.as_ref()
     }
     /// <p>The private key generated from your external identity provider. The private key must be a .pem file that uses AES-GCM or AES-CBC encryption algorithm to decrypt SAML assertions.</p>
@@ -59,8 +59,8 @@
 }
 impl CreateSamlProviderInput {
     /// Creates a new builder-style object to manufacture [`CreateSamlProviderInput`](crate::operation::create_saml_provider::CreateSamlProviderInput).
-    pub fn builder() -> crate::operation::create_saml_provider::builders::CreateSamlProviderInputBuilder {
-        crate::operation::create_saml_provider::builders::CreateSamlProviderInputBuilder::default()
+    pub fn builder() -> super::operation::create_saml_provider::builders::CreateSamlProviderInputBuilder {
+        super::operation::create_saml_provider::builders::CreateSamlProviderInputBuilder::default()
     }
 }

@@ -70,8 +70,8 @@
 pub struct CreateSamlProviderInputBuilder {
     pub(crate) saml_metadata_document: ::std::option::Option<::std::string::String>,
     pub(crate) name: ::std::option::Option<::std::string::String>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
-    pub(crate) assertion_encryption_mode: ::std::option::Option<crate::types::AssertionEncryptionModeType>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::types::Tag>>,
+    pub(crate) assertion_encryption_mode: ::std::option::Option<super::types::AssertionEncryptionModeType>,
     pub(crate) add_private_key: ::std::option::Option<::std::string::String>,
 }
 impl CreateSamlProviderInputBuilder {
@@ -118,7 +118,7 @@
     /// <p>A list of tags that you want to attach to the new IAM SAML provider. Each tag consists of a key name and an associated value. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p><note>
     /// <p>If any one of the tags is invalid or if you exceed the allowed maximum number of tags, then the entire request fails and the resource is not created.</p>
     /// </note>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -127,7 +127,7 @@
     /// <p>A list of tags that you want to attach to the new IAM SAML provider. Each tag consists of a key name and an associated value. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p><note>
     /// <p>If any one of the tags is invalid or if you exceed the allowed maximum number of tags, then the entire request fails and the resource is not created.</p>
     /// </note>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
@@ -134,21 +134,21 @@
     /// <p>A list of tags that you want to attach to the new IAM SAML provider. Each tag consists of a key name and an associated value. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p><note>
     /// <p>If any one of the tags is invalid or if you exceed the allowed maximum number of tags, then the entire request fails and the resource is not created.</p>
     /// </note>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::types::Tag>> {
         &self.tags
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn assertion_encryption_mode(mut self, input: crate::types::AssertionEncryptionModeType) -> Self {
+    pub fn assertion_encryption_mode(mut self, input: super::types::AssertionEncryptionModeType) -> Self {
         self.assertion_encryption_mode = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn set_assertion_encryption_mode(mut self, input: ::std::option::Option<crate::types::AssertionEncryptionModeType>) -> Self {
+    pub fn set_assertion_encryption_mode(mut self, input: ::std::option::Option<super::types::AssertionEncryptionModeType>) -> Self {
         self.assertion_encryption_mode = input;
         self
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn get_assertion_encryption_mode(&self) -> &::std::option::Option<crate::types::AssertionEncryptionModeType> {
+    pub fn get_assertion_encryption_mode(&self) -> &::std::option::Option<super::types::AssertionEncryptionModeType> {
         &self.assertion_encryption_mode
     }
     /// <p>The private key generated from your external identity provider. The private key must be a .pem file that uses AES-GCM or AES-CBC encryption algorithm to decrypt SAML assertions.</p>
@@ -168,9 +168,9 @@
     /// Consumes the builder and constructs a [`CreateSamlProviderInput`](crate::operation::create_saml_provider::CreateSamlProviderInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_saml_provider::CreateSamlProviderInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::create_saml_provider::CreateSamlProviderInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::create_saml_provider::CreateSamlProviderInput {
+        ::std::result::Result::Ok(super::operation::create_saml_provider::CreateSamlProviderInput {
             saml_metadata_document: self.saml_metadata_document,
             name: self.name,
             tags: self.tags,
