--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetCustomRulePolicyInput {
     /// Creates a new builder-style object to manufacture [`GetCustomRulePolicyInput`](crate::operation::get_custom_rule_policy::GetCustomRulePolicyInput).
-    pub fn builder() -> crate::operation::get_custom_rule_policy::builders::GetCustomRulePolicyInputBuilder {
-        crate::operation::get_custom_rule_policy::builders::GetCustomRulePolicyInputBuilder::default()
+    pub fn builder() -> super::operation::get_custom_rule_policy::builders::GetCustomRulePolicyInputBuilder {
+        super::operation::get_custom_rule_policy::builders::GetCustomRulePolicyInputBuilder::default()
     }
 }

@@ -43,9 +43,9 @@
     /// Consumes the builder and constructs a [`GetCustomRulePolicyInput`](crate::operation::get_custom_rule_policy::GetCustomRulePolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_custom_rule_policy::GetCustomRulePolicyInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_custom_rule_policy::GetCustomRulePolicyInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_custom_rule_policy::GetCustomRulePolicyInput {
+        ::std::result::Result::Ok(super::operation::get_custom_rule_policy::GetCustomRulePolicyInput {
             config_rule_name: self.config_rule_name,
         })
     }
