--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>The ARN of the CloudWatch Logs resource for which to query the resource policy.</p>
     pub resource_arn: ::std::option::Option<::std::string::String>,
     /// <p>Specifies the scope of the resource policy. Valid values are <code>ACCOUNT</code> or <code>RESOURCE</code>. When not specified, defaults to <code>ACCOUNT</code>.</p>
-    pub policy_scope: ::std::option::Option<crate::types::PolicyScope>,
+    pub policy_scope: ::std::option::Option<super::types::PolicyScope>,
 }
 impl DescribeResourcePoliciesInput {
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
@@ -26,14 +26,14 @@
         self.resource_arn.as_deref()
     }
     /// <p>Specifies the scope of the resource policy. Valid values are <code>ACCOUNT</code> or <code>RESOURCE</code>. When not specified, defaults to <code>ACCOUNT</code>.</p>
-    pub fn policy_scope(&self) -> ::std::option::Option<&crate::types::PolicyScope> {
+    pub fn policy_scope(&self) -> ::std::option::Option<&super::types::PolicyScope> {
         self.policy_scope.as_ref()
     }
 }
 impl DescribeResourcePoliciesInput {
     /// Creates a new builder-style object to manufacture [`DescribeResourcePoliciesInput`](crate::operation::describe_resource_policies::DescribeResourcePoliciesInput).
-    pub fn builder() -> crate::operation::describe_resource_policies::builders::DescribeResourcePoliciesInputBuilder {
-        crate::operation::describe_resource_policies::builders::DescribeResourcePoliciesInputBuilder::default()
+    pub fn builder() -> super::operation::describe_resource_policies::builders::DescribeResourcePoliciesInputBuilder {
+        super::operation::describe_resource_policies::builders::DescribeResourcePoliciesInputBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
     pub(crate) limit: ::std::option::Option<i32>,
     pub(crate) resource_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) policy_scope: ::std::option::Option<crate::types::PolicyScope>,
+    pub(crate) policy_scope: ::std::option::Option<super::types::PolicyScope>,
 }
 impl DescribeResourcePoliciesInputBuilder {
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
@@ -90,17 +90,17 @@
         &self.resource_arn
     }
     /// <p>Specifies the scope of the resource policy. Valid values are <code>ACCOUNT</code> or <code>RESOURCE</code>. When not specified, defaults to <code>ACCOUNT</code>.</p>
-    pub fn policy_scope(mut self, input: crate::types::PolicyScope) -> Self {
+    pub fn policy_scope(mut self, input: super::types::PolicyScope) -> Self {
         self.policy_scope = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the scope of the resource policy. Valid values are <code>ACCOUNT</code> or <code>RESOURCE</code>. When not specified, defaults to <code>ACCOUNT</code>.</p>
-    pub fn set_policy_scope(mut self, input: ::std::option::Option<crate::types::PolicyScope>) -> Self {
+    pub fn set_policy_scope(mut self, input: ::std::option::Option<super::types::PolicyScope>) -> Self {
         self.policy_scope = input;
         self
     }
     /// <p>Specifies the scope of the resource policy. Valid values are <code>ACCOUNT</code> or <code>RESOURCE</code>. When not specified, defaults to <code>ACCOUNT</code>.</p>
-    pub fn get_policy_scope(&self) -> &::std::option::Option<crate::types::PolicyScope> {
+    pub fn get_policy_scope(&self) -> &::std::option::Option<super::types::PolicyScope> {
         &self.policy_scope
     }
     /// Consumes the builder and constructs a [`DescribeResourcePoliciesInput`](crate::operation::describe_resource_policies::DescribeResourcePoliciesInput).
@@ -107,10 +107,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_resource_policies::DescribeResourcePoliciesInput,
+        super::operation::describe_resource_policies::DescribeResourcePoliciesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_resource_policies::DescribeResourcePoliciesInput {
+        ::std::result::Result::Ok(super::operation::describe_resource_policies::DescribeResourcePoliciesInput {
             next_token: self.next_token,
             limit: self.limit,
             resource_arn: self.resource_arn,
