--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
+        input: super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
     ) -> ::std::result::Result<
-        crate::operation::get_delegated_access_token::GetDelegatedAccessTokenOutput,
+        super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
+            super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::get_delegated_access_token::GetDelegatedAccessTokenError>()
+                err.downcast::<super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::get_delegated_access_token::GetDelegatedAccessTokenOutput>()
+                .downcast::<super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
+        input: super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("GetDelegatedAccessToken")
                 .build()
                 .expect("required fields set"),
@@ -134,14 +134,14 @@
                 GetDelegatedAccessTokenEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
+                super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
+                super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
             >::new())
             .with_retry_classifier(
                 ::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                    crate::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
+                    super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenError,
                 >::builder()
                 .transient_errors({
                     let mut transient_errors: Vec<&'static str> = ::aws_runtime::retries::classifiers::TRANSIENT_ERRORS.into();
@@ -170,11 +170,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_get_delegated_access_token::de_get_delegated_access_token_http_error(status, headers, body)
+            super::super::protocol_serde::shape_get_delegated_access_token::de_get_delegated_access_token_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_get_delegated_access_token::de_get_delegated_access_token_http_response(status, headers, body)
+            super::super::protocol_serde::shape_get_delegated_access_token::de_get_delegated_access_token_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -187,16 +187,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::get_delegated_access_token::GetDelegatedAccessTokenInput>()
+            .downcast::<super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
+                _input: &super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -205,7 +205,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
+                input: &super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -218,7 +218,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_get_delegated_access_token_input::ser_get_delegated_access_token_input_input_input(&input)?,
+            super::super::protocol_serde::shape_get_delegated_access_token_input::ser_get_delegated_access_token_input_input_input(&input)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -251,7 +251,7 @@
             .downcast_ref::<GetDelegatedAccessTokenInput>()
             .ok_or("failed to downcast to GetDelegatedAccessTokenInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -274,12 +274,12 @@
 #[derive(::std::fmt::Debug)]
 pub enum GetDelegatedAccessTokenError {
     /// <p>The trade-in token provided in the request has expired and can no longer be exchanged for credentials. Request a new token and retry the operation.</p>
-    ExpiredTradeInTokenException(crate::types::error::ExpiredTradeInTokenException),
+    ExpiredTradeInTokenException(super::super::types::error::ExpiredTradeInTokenException),
     /// <p>The request was rejected because the total packed size of the session policies and session tags combined was too large. An Amazon Web Services conversion compresses the session policy document, session policy ARNs, and session tags into a packed binary format that has a separate limit. The error message indicates by percentage how close the policies and tags are to the upper size limit. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_session-tags.html">Passing Session Tags in STS</a> in the <i>IAM User Guide</i>.</p>
     /// <p>You could receive this error even though you meet other defined session policy and session tag limits. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html#reference_iam-limits-entity-length">IAM and STS Entity Character Limits</a> in the <i>IAM User Guide</i>.</p>
-    PackedPolicyTooLargeException(crate::types::error::PackedPolicyTooLargeException),
+    PackedPolicyTooLargeException(super::super::types::error::PackedPolicyTooLargeException),
     /// <p>STS is not activated in the requested region for the account that is being asked to generate credentials. The account administrator must use the IAM console to activate STS in that region. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html#sts-regions-activate-deactivate">Activating and Deactivating STS in an Amazon Web Services Region</a> in the <i>IAM User Guide</i>.</p>
-    RegionDisabledException(crate::types::error::RegionDisabledException),
+    RegionDisabledException(super::super::types::error::RegionDisabledException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -287,7 +287,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-GetDelegatedAccessTokenError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl GetDelegatedAccessTokenError {
     /// Creates the `GetDelegatedAccessTokenError::Unhandled` variant from any error type.
@@ -294,7 +294,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -302,7 +302,7 @@

     /// Creates the `GetDelegatedAccessTokenError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -381,21 +381,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::get_delegated_access_token::GetDelegatedAccessTokenError {
+impl ::aws_types::request_id::RequestId for super::super::operation::get_delegated_access_token::GetDelegatedAccessTokenError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::get_delegated_access_token::_get_delegated_access_token_input::GetDelegatedAccessTokenInput;
+pub use super::super::operation::get_delegated_access_token::_get_delegated_access_token_input::GetDelegatedAccessTokenInput;

-pub use crate::operation::get_delegated_access_token::_get_delegated_access_token_output::GetDelegatedAccessTokenOutput;
+pub use super::super::operation::get_delegated_access_token::_get_delegated_access_token_output::GetDelegatedAccessTokenOutput;

 mod _get_delegated_access_token_input;

