--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl GetAuthorizationTokenInput {
     /// Creates a new builder-style object to manufacture [`GetAuthorizationTokenInput`](crate::operation::get_authorization_token::GetAuthorizationTokenInput).
-    pub fn builder() -> crate::operation::get_authorization_token::builders::GetAuthorizationTokenInputBuilder {
-        crate::operation::get_authorization_token::builders::GetAuthorizationTokenInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_authorization_token::builders::GetAuthorizationTokenInputBuilder {
+        super::super::super::operation::get_authorization_token::builders::GetAuthorizationTokenInputBuilder::default()
     }
 }

@@ -86,9 +86,9 @@
     /// Consumes the builder and constructs a [`GetAuthorizationTokenInput`](crate::operation::get_authorization_token::GetAuthorizationTokenInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_authorization_token::GetAuthorizationTokenInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_authorization_token::GetAuthorizationTokenInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_authorization_token::GetAuthorizationTokenInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_authorization_token::GetAuthorizationTokenInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             duration_seconds: self.duration_seconds,
