--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The 12-digit account number of the Amazon Web Services account that owns the domain. It does not include dashes or spaces.</p>
     pub domain_owner: ::std::option::Option<::std::string::String>,
     /// <p>The format of the package from which to get the associated package group.</p>
-    pub format: ::std::option::Option<crate::types::PackageFormat>,
+    pub format: ::std::option::Option<super::super::super::types::PackageFormat>,
     /// <p>The namespace of the package from which to get the associated package group. The package component that specifies its namespace depends on its type. For example:</p><note>
     /// <p>The namespace is required when getting associated package groups from packages of the following formats:</p>
     /// <ul>
@@ -44,7 +44,7 @@
         self.domain_owner.as_deref()
     }
     /// <p>The format of the package from which to get the associated package group.</p>
-    pub fn format(&self) -> ::std::option::Option<&crate::types::PackageFormat> {
+    pub fn format(&self) -> ::std::option::Option<&super::super::super::types::PackageFormat> {
         self.format.as_ref()
     }
     /// <p>The namespace of the package from which to get the associated package group. The package component that specifies its namespace depends on its type. For example:</p><note>
@@ -78,8 +78,8 @@
 }
 impl GetAssociatedPackageGroupInput {
     /// Creates a new builder-style object to manufacture [`GetAssociatedPackageGroupInput`](crate::operation::get_associated_package_group::GetAssociatedPackageGroupInput).
-    pub fn builder() -> crate::operation::get_associated_package_group::builders::GetAssociatedPackageGroupInputBuilder {
-        crate::operation::get_associated_package_group::builders::GetAssociatedPackageGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_associated_package_group::builders::GetAssociatedPackageGroupInputBuilder {
+        super::super::super::operation::get_associated_package_group::builders::GetAssociatedPackageGroupInputBuilder::default()
     }
 }

@@ -89,7 +89,7 @@
 pub struct GetAssociatedPackageGroupInputBuilder {
     pub(crate) domain: ::std::option::Option<::std::string::String>,
     pub(crate) domain_owner: ::std::option::Option<::std::string::String>,
-    pub(crate) format: ::std::option::Option<crate::types::PackageFormat>,
+    pub(crate) format: ::std::option::Option<super::super::super::types::PackageFormat>,
     pub(crate) namespace: ::std::option::Option<::std::string::String>,
     pub(crate) package: ::std::option::Option<::std::string::String>,
 }
@@ -125,17 +125,17 @@
     }
     /// <p>The format of the package from which to get the associated package group.</p>
     /// This field is required.
-    pub fn format(mut self, input: crate::types::PackageFormat) -> Self {
+    pub fn format(mut self, input: super::super::super::types::PackageFormat) -> Self {
         self.format = ::std::option::Option::Some(input);
         self
     }
     /// <p>The format of the package from which to get the associated package group.</p>
-    pub fn set_format(mut self, input: ::std::option::Option<crate::types::PackageFormat>) -> Self {
+    pub fn set_format(mut self, input: ::std::option::Option<super::super::super::types::PackageFormat>) -> Self {
         self.format = input;
         self
     }
     /// <p>The format of the package from which to get the associated package group.</p>
-    pub fn get_format(&self) -> &::std::option::Option<crate::types::PackageFormat> {
+    pub fn get_format(&self) -> &::std::option::Option<super::super::super::types::PackageFormat> {
         &self.format
     }
     /// <p>The namespace of the package from which to get the associated package group. The package component that specifies its namespace depends on its type. For example:</p><note>
@@ -231,10 +231,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_associated_package_group::GetAssociatedPackageGroupInput,
+        super::super::super::operation::get_associated_package_group::GetAssociatedPackageGroupInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_associated_package_group::GetAssociatedPackageGroupInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_associated_package_group::GetAssociatedPackageGroupInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             format: self.format,
