--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl ListTopicsInput {
     /// Creates a new builder-style object to manufacture [`ListTopicsInput`](crate::operation::list_topics::ListTopicsInput).
-    pub fn builder() -> crate::operation::list_topics::builders::ListTopicsInputBuilder {
-        crate::operation::list_topics::builders::ListTopicsInputBuilder::default()
+    pub fn builder() -> super::operation::list_topics::builders::ListTopicsInputBuilder {
+        super::operation::list_topics::builders::ListTopicsInputBuilder::default()
     }
 }

@@ -41,7 +41,7 @@
         &self.next_token
     }
     /// Consumes the builder and constructs a [`ListTopicsInput`](crate::operation::list_topics::ListTopicsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_topics::ListTopicsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_topics::ListTopicsInput { next_token: self.next_token })
+    pub fn build(self) -> ::std::result::Result<super::operation::list_topics::ListTopicsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_topics::ListTopicsInput { next_token: self.next_token })
     }
 }
