--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DescribeDomainInput {
     /// Creates a new builder-style object to manufacture [`DescribeDomainInput`](crate::operation::describe_domain::DescribeDomainInput).
-    pub fn builder() -> crate::operation::describe_domain::builders::DescribeDomainInputBuilder {
-        crate::operation::describe_domain::builders::DescribeDomainInputBuilder::default()
+    pub fn builder() -> super::operation::describe_domain::builders::DescribeDomainInputBuilder {
+        super::operation::describe_domain::builders::DescribeDomainInputBuilder::default()
     }
 }

@@ -65,8 +65,8 @@
     /// Consumes the builder and constructs a [`DescribeDomainInput`](crate::operation::describe_domain::DescribeDomainInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_domain::DescribeDomainInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::describe_domain::DescribeDomainInput {
+    ) -> ::std::result::Result<super::operation::describe_domain::DescribeDomainInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::describe_domain::DescribeDomainInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
         })
