--- original
+++ modified
@@ -35,8 +35,8 @@
 }
 impl ListWebAuthnCredentialsInput {
     /// Creates a new builder-style object to manufacture [`ListWebAuthnCredentialsInput`](crate::operation::list_web_authn_credentials::ListWebAuthnCredentialsInput).
-    pub fn builder() -> crate::operation::list_web_authn_credentials::builders::ListWebAuthnCredentialsInputBuilder {
-        crate::operation::list_web_authn_credentials::builders::ListWebAuthnCredentialsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_web_authn_credentials::builders::ListWebAuthnCredentialsInputBuilder {
+        super::super::super::operation::list_web_authn_credentials::builders::ListWebAuthnCredentialsInputBuilder::default()
     }
 }

@@ -96,10 +96,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_web_authn_credentials::ListWebAuthnCredentialsInput,
+        super::super::super::operation::list_web_authn_credentials::ListWebAuthnCredentialsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_web_authn_credentials::ListWebAuthnCredentialsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_web_authn_credentials::ListWebAuthnCredentialsInput {
             access_token: self.access_token,
             next_token: self.next_token,
             max_results: self.max_results,
