--- original
+++ modified
@@ -60,7 +60,7 @@
     ///   - [`bucket_key_enabled(Option<bool>)`](crate::operation::copy_object::CopyObjectOutput::bucket_key_enabled): <p>Indicates whether the copied object uses an S3 Bucket Key for server-side encryption with Key Management Service (KMS) keys (SSE-KMS).</p>
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::copy_object::CopyObjectOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<CopyObjectError>`](crate::operation::copy_object::CopyObjectError)
-    pub fn copy_object(&self) -> crate::operation::copy_object::builders::CopyObjectFluentBuilder {
-        crate::operation::copy_object::builders::CopyObjectFluentBuilder::new(self.handle.clone())
+    pub fn copy_object(&self) -> super::operation::copy_object::builders::CopyObjectFluentBuilder {
+        super::operation::copy_object::builders::CopyObjectFluentBuilder::new(self.handle.clone())
     }
 }
