--- original
+++ modified
@@ -12,8 +12,8 @@
     /// - On failure, responds with [`SdkError<CreateBucketMetadataTableConfigurationError>`](crate::operation::create_bucket_metadata_table_configuration::CreateBucketMetadataTableConfigurationError)
     pub fn create_bucket_metadata_table_configuration(
         &self,
-    ) -> crate::operation::create_bucket_metadata_table_configuration::builders::CreateBucketMetadataTableConfigurationFluentBuilder {
-        crate::operation::create_bucket_metadata_table_configuration::builders::CreateBucketMetadataTableConfigurationFluentBuilder::new(
+    ) -> super::operation::create_bucket_metadata_table_configuration::builders::CreateBucketMetadataTableConfigurationFluentBuilder {
+        super::operation::create_bucket_metadata_table_configuration::builders::CreateBucketMetadataTableConfigurationFluentBuilder::new(
             self.handle.clone(),
         )
     }
