--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`object_version_id(Option<String>)`](crate::operation::delete_object_annotation::DeleteObjectAnnotationOutput::object_version_id): <p>The version ID of the object that the annotation was deleted from.</p>
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::delete_object_annotation::DeleteObjectAnnotationOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<DeleteObjectAnnotationError>`](crate::operation::delete_object_annotation::DeleteObjectAnnotationError)
-    pub fn delete_object_annotation(&self) -> crate::operation::delete_object_annotation::builders::DeleteObjectAnnotationFluentBuilder {
-        crate::operation::delete_object_annotation::builders::DeleteObjectAnnotationFluentBuilder::new(self.handle.clone())
+    pub fn delete_object_annotation(&self) -> super::super::operation::delete_object_annotation::builders::DeleteObjectAnnotationFluentBuilder {
+        super::super::operation::delete_object_annotation::builders::DeleteObjectAnnotationFluentBuilder::new(self.handle.clone())
     }
 }
