--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`expected_bucket_owner(impl Into<String>)`](crate::operation::delete_public_access_block::builders::DeletePublicAccessBlockFluentBuilder::expected_bucket_owner) / [`set_expected_bucket_owner(Option<String>)`](crate::operation::delete_public_access_block::builders::DeletePublicAccessBlockFluentBuilder::set_expected_bucket_owner):<br>required: **false**<br><p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p><br>
     /// - On success, responds with [`DeletePublicAccessBlockOutput`](crate::operation::delete_public_access_block::DeletePublicAccessBlockOutput)
     /// - On failure, responds with [`SdkError<DeletePublicAccessBlockError>`](crate::operation::delete_public_access_block::DeletePublicAccessBlockError)
-    pub fn delete_public_access_block(&self) -> crate::operation::delete_public_access_block::builders::DeletePublicAccessBlockFluentBuilder {
-        crate::operation::delete_public_access_block::builders::DeletePublicAccessBlockFluentBuilder::new(self.handle.clone())
+    pub fn delete_public_access_block(&self) -> super::operation::delete_public_access_block::builders::DeletePublicAccessBlockFluentBuilder {
+        super::operation::delete_public_access_block::builders::DeletePublicAccessBlockFluentBuilder::new(self.handle.clone())
     }
 }
