--- original
+++ modified
@@ -25,7 +25,7 @@
     ///   - [`storage_class(Option<StorageClass>)`](crate::operation::get_object_attributes::GetObjectAttributesOutput::storage_class): <p>Provides the storage class information of the object. Amazon S3 returns this header for all objects except for S3 Standard storage class objects.</p> <p>For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html">Storage Classes</a>.</p><note>  <p><b>Directory buckets</b> - Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p> </note>
     ///   - [`object_size(Option<i64>)`](crate::operation::get_object_attributes::GetObjectAttributesOutput::object_size): <p>The size of the object in bytes.</p>
     /// - On failure, responds with [`SdkError<GetObjectAttributesError>`](crate::operation::get_object_attributes::GetObjectAttributesError)
-    pub fn get_object_attributes(&self) -> crate::operation::get_object_attributes::builders::GetObjectAttributesFluentBuilder {
-        crate::operation::get_object_attributes::builders::GetObjectAttributesFluentBuilder::new(self.handle.clone())
+    pub fn get_object_attributes(&self) -> super::operation::get_object_attributes::builders::GetObjectAttributesFluentBuilder {
+        super::operation::get_object_attributes::builders::GetObjectAttributesFluentBuilder::new(self.handle.clone())
     }
 }
