--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`GetPublicAccessBlockOutput`](crate::operation::get_public_access_block::GetPublicAccessBlockOutput) with field(s):
     ///   - [`public_access_block_configuration(Option<PublicAccessBlockConfiguration>)`](crate::operation::get_public_access_block::GetPublicAccessBlockOutput::public_access_block_configuration): <p>The <code>PublicAccessBlock</code> configuration currently in effect for this Amazon S3 bucket.</p>
     /// - On failure, responds with [`SdkError<GetPublicAccessBlockError>`](crate::operation::get_public_access_block::GetPublicAccessBlockError)
-    pub fn get_public_access_block(&self) -> crate::operation::get_public_access_block::builders::GetPublicAccessBlockFluentBuilder {
-        crate::operation::get_public_access_block::builders::GetPublicAccessBlockFluentBuilder::new(self.handle.clone())
+    pub fn get_public_access_block(&self) -> super::operation::get_public_access_block::builders::GetPublicAccessBlockFluentBuilder {
+        super::operation::get_public_access_block::builders::GetPublicAccessBlockFluentBuilder::new(self.handle.clone())
     }
 }
