--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`GetBucketPolicyOutput`](crate::operation::get_bucket_policy::GetBucketPolicyOutput) with field(s):
     ///   - [`policy(Option<String>)`](crate::operation::get_bucket_policy::GetBucketPolicyOutput::policy): <p>The bucket policy as a JSON document.</p>
     /// - On failure, responds with [`SdkError<GetBucketPolicyError>`](crate::operation::get_bucket_policy::GetBucketPolicyError)
-    pub fn get_bucket_policy(&self) -> crate::operation::get_bucket_policy::builders::GetBucketPolicyFluentBuilder {
-        crate::operation::get_bucket_policy::builders::GetBucketPolicyFluentBuilder::new(self.handle.clone())
+    pub fn get_bucket_policy(&self) -> super::operation::get_bucket_policy::builders::GetBucketPolicyFluentBuilder {
+        super::operation::get_bucket_policy::builders::GetBucketPolicyFluentBuilder::new(self.handle.clone())
     }
 }
