--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`GetBucketPolicyStatusOutput`](crate::operation::get_bucket_policy_status::GetBucketPolicyStatusOutput) with field(s):
     ///   - [`policy_status(Option<PolicyStatus>)`](crate::operation::get_bucket_policy_status::GetBucketPolicyStatusOutput::policy_status): <p>The policy status for the specified bucket.</p>
     /// - On failure, responds with [`SdkError<GetBucketPolicyStatusError>`](crate::operation::get_bucket_policy_status::GetBucketPolicyStatusError)
-    pub fn get_bucket_policy_status(&self) -> crate::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusFluentBuilder {
-        crate::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusFluentBuilder::new(self.handle.clone())
+    pub fn get_bucket_policy_status(&self) -> super::super::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusFluentBuilder {
+        super::super::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusFluentBuilder::new(self.handle.clone())
     }
 }
