--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On failure, responds with [`SdkError<GetBucketInventoryConfigurationError>`](crate::operation::get_bucket_inventory_configuration::GetBucketInventoryConfigurationError)
     pub fn get_bucket_inventory_configuration(
         &self,
-    ) -> crate::operation::get_bucket_inventory_configuration::builders::GetBucketInventoryConfigurationFluentBuilder {
-        crate::operation::get_bucket_inventory_configuration::builders::GetBucketInventoryConfigurationFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::get_bucket_inventory_configuration::builders::GetBucketInventoryConfigurationFluentBuilder {
+        super::super::operation::get_bucket_inventory_configuration::builders::GetBucketInventoryConfigurationFluentBuilder::new(self.handle.clone())
     }
 }
