--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`owner(Option<Owner>)`](crate::operation::get_bucket_acl::GetBucketAclOutput::owner): <p>Container for the bucket owner's ID.</p>
     ///   - [`grants(Option<Vec::<Grant>>)`](crate::operation::get_bucket_acl::GetBucketAclOutput::grants): <p>A list of grants.</p>
     /// - On failure, responds with [`SdkError<GetBucketAclError>`](crate::operation::get_bucket_acl::GetBucketAclError)
-    pub fn get_bucket_acl(&self) -> crate::operation::get_bucket_acl::builders::GetBucketAclFluentBuilder {
-        crate::operation::get_bucket_acl::builders::GetBucketAclFluentBuilder::new(self.handle.clone())
+    pub fn get_bucket_acl(&self) -> super::super::operation::get_bucket_acl::builders::GetBucketAclFluentBuilder {
+        super::super::operation::get_bucket_acl::builders::GetBucketAclFluentBuilder::new(self.handle.clone())
     }
 }
