--- original
+++ modified
@@ -10,8 +10,8 @@
     /// - On failure, responds with [`SdkError<GetBucketMetadataTableConfigurationError>`](crate::operation::get_bucket_metadata_table_configuration::GetBucketMetadataTableConfigurationError)
     pub fn get_bucket_metadata_table_configuration(
         &self,
-    ) -> crate::operation::get_bucket_metadata_table_configuration::builders::GetBucketMetadataTableConfigurationFluentBuilder {
-        crate::operation::get_bucket_metadata_table_configuration::builders::GetBucketMetadataTableConfigurationFluentBuilder::new(
+    ) -> super::super::operation::get_bucket_metadata_table_configuration::builders::GetBucketMetadataTableConfigurationFluentBuilder {
+        super::super::operation::get_bucket_metadata_table_configuration::builders::GetBucketMetadataTableConfigurationFluentBuilder::new(
             self.handle.clone(),
         )
     }
