--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`GetBucketReplicationOutput`](crate::operation::get_bucket_replication::GetBucketReplicationOutput) with field(s):
     ///   - [`replication_configuration(Option<ReplicationConfiguration>)`](crate::operation::get_bucket_replication::GetBucketReplicationOutput::replication_configuration): <p>A container for replication rules. You can add up to 1,000 rules. The maximum size of a replication configuration is 2 MB.</p>
     /// - On failure, responds with [`SdkError<GetBucketReplicationError>`](crate::operation::get_bucket_replication::GetBucketReplicationError)
-    pub fn get_bucket_replication(&self) -> crate::operation::get_bucket_replication::builders::GetBucketReplicationFluentBuilder {
-        crate::operation::get_bucket_replication::builders::GetBucketReplicationFluentBuilder::new(self.handle.clone())
+    pub fn get_bucket_replication(&self) -> super::super::operation::get_bucket_replication::builders::GetBucketReplicationFluentBuilder {
+        super::super::operation::get_bucket_replication::builders::GetBucketReplicationFluentBuilder::new(self.handle.clone())
     }
 }
