--- original
+++ modified
@@ -12,7 +12,7 @@
     ///   - [`bucket_region(Option<String>)`](crate::operation::head_bucket::HeadBucketOutput::bucket_region): <p>The Region that the bucket is located.</p>
     ///   - [`access_point_alias(Option<bool>)`](crate::operation::head_bucket::HeadBucketOutput::access_point_alias): <p>Indicates whether the bucket name used in the request is an access point alias.</p><note>  <p>For directory buckets, the value of this field is <code>false</code>.</p> </note>
     /// - On failure, responds with [`SdkError<HeadBucketError>`](crate::operation::head_bucket::HeadBucketError)
-    pub fn head_bucket(&self) -> crate::operation::head_bucket::builders::HeadBucketFluentBuilder {
-        crate::operation::head_bucket::builders::HeadBucketFluentBuilder::new(self.handle.clone())
+    pub fn head_bucket(&self) -> super::operation::head_bucket::builders::HeadBucketFluentBuilder {
+        super::operation::head_bucket::builders::HeadBucketFluentBuilder::new(self.handle.clone())
     }
 }
