--- original
+++ modified
@@ -32,7 +32,7 @@
     ///   - [`checksum_algorithm(Option<ChecksumAlgorithm>)`](crate::operation::list_parts::ListPartsOutput::checksum_algorithm): <p>The algorithm that was used to create a checksum of the object.</p>
     ///   - [`checksum_type(Option<ChecksumType>)`](crate::operation::list_parts::ListPartsOutput::checksum_type): <p>The checksum type, which determines how part-level checksums are combined to create an object-level checksum for multipart objects. You can use this header response to verify that the checksum type that is received is the same checksum type that was specified in <code>CreateMultipartUpload</code> request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity in the Amazon S3 User Guide</a>.</p>
     /// - On failure, responds with [`SdkError<ListPartsError>`](crate::operation::list_parts::ListPartsError)
-    pub fn list_parts(&self) -> crate::operation::list_parts::builders::ListPartsFluentBuilder {
-        crate::operation::list_parts::builders::ListPartsFluentBuilder::new(self.handle.clone())
+    pub fn list_parts(&self) -> super::operation::list_parts::builders::ListPartsFluentBuilder {
+        super::operation::list_parts::builders::ListPartsFluentBuilder::new(self.handle.clone())
     }
 }
