--- original
+++ modified
@@ -24,7 +24,7 @@
     ///   - [`next_continuation_token(Option<String>)`](crate::operation::list_object_annotations::ListObjectAnnotationsOutput::next_continuation_token): <p>The continuation token to use to retrieve the next page of results.</p>
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::list_object_annotations::ListObjectAnnotationsOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<ListObjectAnnotationsError>`](crate::operation::list_object_annotations::ListObjectAnnotationsError)
-    pub fn list_object_annotations(&self) -> crate::operation::list_object_annotations::builders::ListObjectAnnotationsFluentBuilder {
-        crate::operation::list_object_annotations::builders::ListObjectAnnotationsFluentBuilder::new(self.handle.clone())
+    pub fn list_object_annotations(&self) -> super::super::operation::list_object_annotations::builders::ListObjectAnnotationsFluentBuilder {
+        super::super::operation::list_object_annotations::builders::ListObjectAnnotationsFluentBuilder::new(self.handle.clone())
     }
 }
