--- original
+++ modified
@@ -30,7 +30,7 @@
     ///   - [`start_after(Option<String>)`](crate::operation::list_objects_v2::ListObjectsV2Output::start_after): <p>If StartAfter was sent with the request, it is included in the response.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::list_objects_v2::ListObjectsV2Output::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<ListObjectsV2Error>`](crate::operation::list_objects_v2::ListObjectsV2Error)
-    pub fn list_objects_v2(&self) -> crate::operation::list_objects_v2::builders::ListObjectsV2FluentBuilder {
-        crate::operation::list_objects_v2::builders::ListObjectsV2FluentBuilder::new(self.handle.clone())
+    pub fn list_objects_v2(&self) -> super::operation::list_objects_v2::builders::ListObjectsV2FluentBuilder {
+        super::operation::list_objects_v2::builders::ListObjectsV2FluentBuilder::new(self.handle.clone())
     }
 }
