--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`expected_bucket_owner(impl Into<String>)`](crate::operation::put_bucket_website::builders::PutBucketWebsiteFluentBuilder::expected_bucket_owner) / [`set_expected_bucket_owner(Option<String>)`](crate::operation::put_bucket_website::builders::PutBucketWebsiteFluentBuilder::set_expected_bucket_owner):<br>required: **false**<br><p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p><br>
     /// - On success, responds with [`PutBucketWebsiteOutput`](crate::operation::put_bucket_website::PutBucketWebsiteOutput)
     /// - On failure, responds with [`SdkError<PutBucketWebsiteError>`](crate::operation::put_bucket_website::PutBucketWebsiteError)
-    pub fn put_bucket_website(&self) -> crate::operation::put_bucket_website::builders::PutBucketWebsiteFluentBuilder {
-        crate::operation::put_bucket_website::builders::PutBucketWebsiteFluentBuilder::new(self.handle.clone())
+    pub fn put_bucket_website(&self) -> super::super::operation::put_bucket_website::builders::PutBucketWebsiteFluentBuilder {
+        super::super::operation::put_bucket_website::builders::PutBucketWebsiteFluentBuilder::new(self.handle.clone())
     }
 }
