--- original
+++ modified
@@ -14,7 +14,7 @@
     /// - On success, responds with [`PutObjectLegalHoldOutput`](crate::operation::put_object_legal_hold::PutObjectLegalHoldOutput) with field(s):
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::put_object_legal_hold::PutObjectLegalHoldOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<PutObjectLegalHoldError>`](crate::operation::put_object_legal_hold::PutObjectLegalHoldError)
-    pub fn put_object_legal_hold(&self) -> crate::operation::put_object_legal_hold::builders::PutObjectLegalHoldFluentBuilder {
-        crate::operation::put_object_legal_hold::builders::PutObjectLegalHoldFluentBuilder::new(self.handle.clone())
+    pub fn put_object_legal_hold(&self) -> super::super::operation::put_object_legal_hold::builders::PutObjectLegalHoldFluentBuilder {
+        super::super::operation::put_object_legal_hold::builders::PutObjectLegalHoldFluentBuilder::new(self.handle.clone())
     }
 }
