--- original
+++ modified
@@ -15,7 +15,7 @@
     /// - On success, responds with [`PutObjectRetentionOutput`](crate::operation::put_object_retention::PutObjectRetentionOutput) with field(s):
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::put_object_retention::PutObjectRetentionOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<PutObjectRetentionError>`](crate::operation::put_object_retention::PutObjectRetentionError)
-    pub fn put_object_retention(&self) -> crate::operation::put_object_retention::builders::PutObjectRetentionFluentBuilder {
-        crate::operation::put_object_retention::builders::PutObjectRetentionFluentBuilder::new(self.handle.clone())
+    pub fn put_object_retention(&self) -> super::super::operation::put_object_retention::builders::PutObjectRetentionFluentBuilder {
+        super::super::operation::put_object_retention::builders::PutObjectRetentionFluentBuilder::new(self.handle.clone())
     }
 }
