--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::restore_object::RestoreObjectOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     ///   - [`restore_output_path(Option<String>)`](crate::operation::restore_object::RestoreObjectOutput::restore_output_path): <p>Indicates the path in the provided S3 output location where Select results will be restored to.</p>
     /// - On failure, responds with [`SdkError<RestoreObjectError>`](crate::operation::restore_object::RestoreObjectError)
-    pub fn restore_object(&self) -> crate::operation::restore_object::builders::RestoreObjectFluentBuilder {
-        crate::operation::restore_object::builders::RestoreObjectFluentBuilder::new(self.handle.clone())
+    pub fn restore_object(&self) -> super::super::operation::restore_object::builders::RestoreObjectFluentBuilder {
+        super::super::operation::restore_object::builders::RestoreObjectFluentBuilder::new(self.handle.clone())
     }
 }
