--- original
+++ modified
@@ -45,7 +45,7 @@
     ///   - [`bucket_key_enabled(Option<bool>)`](crate::operation::upload_part::UploadPartOutput::bucket_key_enabled): <p>Indicates whether the multipart upload uses an S3 Bucket Key for server-side encryption with Key Management Service (KMS) keys (SSE-KMS).</p>
     ///   - [`request_charged(Option<RequestCharged>)`](crate::operation::upload_part::UploadPartOutput::request_charged): <p>If present, indicates that the requester was successfully charged for the request. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html">Using Requester Pays buckets for storage transfers and usage</a> in the <i>Amazon Simple Storage Service user guide</i>.</p><note>  <p>This functionality is not supported for directory buckets.</p> </note>
     /// - On failure, responds with [`SdkError<UploadPartError>`](crate::operation::upload_part::UploadPartError)
-    pub fn upload_part(&self) -> crate::operation::upload_part::builders::UploadPartFluentBuilder {
-        crate::operation::upload_part::builders::UploadPartFluentBuilder::new(self.handle.clone())
+    pub fn upload_part(&self) -> super::super::operation::upload_part::builders::UploadPartFluentBuilder {
+        super::super::operation::upload_part::builders::UploadPartFluentBuilder::new(self.handle.clone())
     }
 }
