--- original
+++ modified
@@ -51,7 +51,7 @@
     ///   - [`bucket_key_enabled(bool)`](crate::operation::write_get_object_response::builders::WriteGetObjectResponseFluentBuilder::bucket_key_enabled) / [`set_bucket_key_enabled(Option<bool>)`](crate::operation::write_get_object_response::builders::WriteGetObjectResponseFluentBuilder::set_bucket_key_enabled):<br>required: **false**<br><p>Indicates whether the object stored in Amazon S3 uses an S3 bucket key for server-side encryption with Amazon Web Services KMS (SSE-KMS).</p><br>
     /// - On success, responds with [`WriteGetObjectResponseOutput`](crate::operation::write_get_object_response::WriteGetObjectResponseOutput)
     /// - On failure, responds with [`SdkError<WriteGetObjectResponseError>`](crate::operation::write_get_object_response::WriteGetObjectResponseError)
-    pub fn write_get_object_response(&self) -> crate::operation::write_get_object_response::builders::WriteGetObjectResponseFluentBuilder {
-        crate::operation::write_get_object_response::builders::WriteGetObjectResponseFluentBuilder::new(self.handle.clone())
+    pub fn write_get_object_response(&self) -> super::operation::write_get_object_response::builders::WriteGetObjectResponseFluentBuilder {
+        super::operation::write_get_object_response::builders::WriteGetObjectResponseFluentBuilder::new(self.handle.clone())
     }
 }
