--- original
+++ modified
@@ -6,9 +6,9 @@
 impl DisableOrganizationsRootCredentialsManagementInput {
     /// Creates a new builder-style object to manufacture [`DisableOrganizationsRootCredentialsManagementInput`](crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementInput).
     pub fn builder(
-    ) -> crate::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementInputBuilder
+    ) -> super::super::super::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementInputBuilder
     {
-        crate::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementInputBuilder::default()
+        super::super::super::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementInputBuilder::default()
     }
 }

@@ -21,11 +21,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementInput,
+        super::super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementInput {},
+            super::super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementInput {},
         )
     }
 }
