--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl DeleteSuppressedDestinationInput {
     /// Creates a new builder-style object to manufacture [`DeleteSuppressedDestinationInput`](crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationInput).
-    pub fn builder() -> crate::operation::delete_suppressed_destination::builders::DeleteSuppressedDestinationInputBuilder {
-        crate::operation::delete_suppressed_destination::builders::DeleteSuppressedDestinationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_suppressed_destination::builders::DeleteSuppressedDestinationInputBuilder {
+        super::super::super::operation::delete_suppressed_destination::builders::DeleteSuppressedDestinationInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationInput,
+        super::super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_suppressed_destination::DeleteSuppressedDestinationInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_suppressed_destination::DeleteSuppressedDestinationInput {
             email_address: self.email_address,
             tenant_name: self.tenant_name,
         })
