--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ListContributorInsightsInput {
     /// Creates a new builder-style object to manufacture [`ListContributorInsightsInput`](crate::operation::list_contributor_insights::ListContributorInsightsInput).
-    pub fn builder() -> crate::operation::list_contributor_insights::builders::ListContributorInsightsInputBuilder {
-        crate::operation::list_contributor_insights::builders::ListContributorInsightsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_contributor_insights::builders::ListContributorInsightsInputBuilder {
+        super::super::super::operation::list_contributor_insights::builders::ListContributorInsightsInputBuilder::default()
     }
 }

@@ -86,10 +86,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_contributor_insights::ListContributorInsightsInput,
+        super::super::super::operation::list_contributor_insights::ListContributorInsightsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_contributor_insights::ListContributorInsightsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_contributor_insights::ListContributorInsightsInput {
             table_name: self.table_name,
             next_token: self.next_token,
             max_results: self.max_results,
