--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The job status used to filter service jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exceptions are the <code>SHARE_IDENTIFIER</code> filter and <code>QUOTA_SHARE_NAME</code> filter, which can be used with <code>jobStatus</code>. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>The <code>SHARE_IDENTIFIER</code> filter or <code>QUOTA_SHARE_NAME</code> filter can be used with the <code>jobStatus</code> field to filter results.</p>
     /// </note>
-    pub job_status: ::std::option::Option<crate::types::ServiceJobStatus>,
+    pub job_status: ::std::option::Option<super::super::super::types::ServiceJobStatus>,
     /// <p>The maximum number of results returned by <code>ListServiceJobs</code> in paginated output. When this parameter is used, <code>ListServiceJobs</code> only returns <code>maxResults</code> results in a single page and a <code>nextToken</code> response element. The remaining results of the initial request can be seen by sending another <code>ListServiceJobs</code> request with the returned <code>nextToken</code> value. This value can be between 1 and 100. If this parameter isn't used, then <code>ListServiceJobs</code> returns up to 100 results and a <code>nextToken</code> value if applicable.</p>
     pub max_results: ::std::option::Option<i32>,
     /// <p>The <code>nextToken</code> value returned from a previous paginated <code>ListServiceJobs</code> request where <code>maxResults</code> was used and the results exceeded the value of that parameter. Pagination continues from the end of the previous results that returned the <code>nextToken</code> value. This value is <code>null</code> when there are no more results to return.</p><note>
@@ -50,7 +50,7 @@
     /// <p>The value for the filter is the quota management share name.</p>
     /// </dd>
     /// </dl>
-    pub filters: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>>,
+    pub filters: ::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>>,
 }
 impl ListServiceJobsInput {
     /// <p>The name or ARN of the job queue with which to list service jobs.</p>
@@ -60,7 +60,7 @@
     /// <p>The job status used to filter service jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exceptions are the <code>SHARE_IDENTIFIER</code> filter and <code>QUOTA_SHARE_NAME</code> filter, which can be used with <code>jobStatus</code>. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>The <code>SHARE_IDENTIFIER</code> filter or <code>QUOTA_SHARE_NAME</code> filter can be used with the <code>jobStatus</code> field to filter results.</p>
     /// </note>
-    pub fn job_status(&self) -> ::std::option::Option<&crate::types::ServiceJobStatus> {
+    pub fn job_status(&self) -> ::std::option::Option<&super::super::super::types::ServiceJobStatus> {
         self.job_status.as_ref()
     }
     /// <p>The maximum number of results returned by <code>ListServiceJobs</code> in paginated output. When this parameter is used, <code>ListServiceJobs</code> only returns <code>maxResults</code> results in a single page and a <code>nextToken</code> response element. The remaining results of the initial request can be seen by sending another <code>ListServiceJobs</code> request with the returned <code>nextToken</code> value. This value can be between 1 and 100. If this parameter isn't used, then <code>ListServiceJobs</code> returns up to 100 results and a <code>nextToken</code> value if applicable.</p>
@@ -110,14 +110,14 @@
     /// </dl>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.filters.is_none()`.
-    pub fn filters(&self) -> &[crate::types::KeyValuesPair] {
+    pub fn filters(&self) -> &[super::super::super::types::KeyValuesPair] {
         self.filters.as_deref().unwrap_or_default()
     }
 }
 impl ListServiceJobsInput {
     /// Creates a new builder-style object to manufacture [`ListServiceJobsInput`](crate::operation::list_service_jobs::ListServiceJobsInput).
-    pub fn builder() -> crate::operation::list_service_jobs::builders::ListServiceJobsInputBuilder {
-        crate::operation::list_service_jobs::builders::ListServiceJobsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_service_jobs::builders::ListServiceJobsInputBuilder {
+        super::super::super::operation::list_service_jobs::builders::ListServiceJobsInputBuilder::default()
     }
 }

@@ -126,10 +126,10 @@
 #[non_exhaustive]
 pub struct ListServiceJobsInputBuilder {
     pub(crate) job_queue: ::std::option::Option<::std::string::String>,
-    pub(crate) job_status: ::std::option::Option<crate::types::ServiceJobStatus>,
+    pub(crate) job_status: ::std::option::Option<super::super::super::types::ServiceJobStatus>,
     pub(crate) max_results: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
-    pub(crate) filters: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>>,
+    pub(crate) filters: ::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>>,
 }
 impl ListServiceJobsInputBuilder {
     /// <p>The name or ARN of the job queue with which to list service jobs.</p>
@@ -149,7 +149,7 @@
     /// <p>The job status used to filter service jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exceptions are the <code>SHARE_IDENTIFIER</code> filter and <code>QUOTA_SHARE_NAME</code> filter, which can be used with <code>jobStatus</code>. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>The <code>SHARE_IDENTIFIER</code> filter or <code>QUOTA_SHARE_NAME</code> filter can be used with the <code>jobStatus</code> field to filter results.</p>
     /// </note>
-    pub fn job_status(mut self, input: crate::types::ServiceJobStatus) -> Self {
+    pub fn job_status(mut self, input: super::super::super::types::ServiceJobStatus) -> Self {
         self.job_status = ::std::option::Option::Some(input);
         self
     }
@@ -156,7 +156,7 @@
     /// <p>The job status used to filter service jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exceptions are the <code>SHARE_IDENTIFIER</code> filter and <code>QUOTA_SHARE_NAME</code> filter, which can be used with <code>jobStatus</code>. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>The <code>SHARE_IDENTIFIER</code> filter or <code>QUOTA_SHARE_NAME</code> filter can be used with the <code>jobStatus</code> field to filter results.</p>
     /// </note>
-    pub fn set_job_status(mut self, input: ::std::option::Option<crate::types::ServiceJobStatus>) -> Self {
+    pub fn set_job_status(mut self, input: ::std::option::Option<super::super::super::types::ServiceJobStatus>) -> Self {
         self.job_status = input;
         self
     }
@@ -163,7 +163,7 @@
     /// <p>The job status used to filter service jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exceptions are the <code>SHARE_IDENTIFIER</code> filter and <code>QUOTA_SHARE_NAME</code> filter, which can be used with <code>jobStatus</code>. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>The <code>SHARE_IDENTIFIER</code> filter or <code>QUOTA_SHARE_NAME</code> filter can be used with the <code>jobStatus</code> field to filter results.</p>
     /// </note>
-    pub fn get_job_status(&self) -> &::std::option::Option<crate::types::ServiceJobStatus> {
+    pub fn get_job_status(&self) -> &::std::option::Option<super::super::super::types::ServiceJobStatus> {
         &self.job_status
     }
     /// <p>The maximum number of results returned by <code>ListServiceJobs</code> in paginated output. When this parameter is used, <code>ListServiceJobs</code> only returns <code>maxResults</code> results in a single page and a <code>nextToken</code> response element. The remaining results of the initial request can be seen by sending another <code>ListServiceJobs</code> request with the returned <code>nextToken</code> value. This value can be between 1 and 100. If this parameter isn't used, then <code>ListServiceJobs</code> returns up to 100 results and a <code>nextToken</code> value if applicable.</p>
@@ -239,7 +239,7 @@
     /// <p>The value for the filter is the quota management share name.</p>
     /// </dd>
     /// </dl>
-    pub fn filters(mut self, input: crate::types::KeyValuesPair) -> Self {
+    pub fn filters(mut self, input: super::super::super::types::KeyValuesPair) -> Self {
         let mut v = self.filters.unwrap_or_default();
         v.push(input);
         self.filters = ::std::option::Option::Some(v);
@@ -280,7 +280,7 @@
     /// <p>The value for the filter is the quota management share name.</p>
     /// </dd>
     /// </dl>
-    pub fn set_filters(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>>) -> Self {
+    pub fn set_filters(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>>) -> Self {
         self.filters = input;
         self
     }
@@ -319,14 +319,14 @@
     /// <p>The value for the filter is the quota management share name.</p>
     /// </dd>
     /// </dl>
-    pub fn get_filters(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>> {
+    pub fn get_filters(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>> {
         &self.filters
     }
     /// Consumes the builder and constructs a [`ListServiceJobsInput`](crate::operation::list_service_jobs::ListServiceJobsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_service_jobs::ListServiceJobsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_service_jobs::ListServiceJobsInput {
+    ) -> ::std::result::Result<super::super::super::operation::list_service_jobs::ListServiceJobsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_service_jobs::ListServiceJobsInput {
             job_queue: self.job_queue,
             job_status: self.job_status,
             max_results: self.max_results,
