--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetDeliveryDestinationInput {
     /// Creates a new builder-style object to manufacture [`GetDeliveryDestinationInput`](crate::operation::get_delivery_destination::GetDeliveryDestinationInput).
-    pub fn builder() -> crate::operation::get_delivery_destination::builders::GetDeliveryDestinationInputBuilder {
-        crate::operation::get_delivery_destination::builders::GetDeliveryDestinationInputBuilder::default()
+    pub fn builder() -> super::operation::get_delivery_destination::builders::GetDeliveryDestinationInputBuilder {
+        super::operation::get_delivery_destination::builders::GetDeliveryDestinationInputBuilder::default()
     }
 }

@@ -45,9 +45,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_delivery_destination::GetDeliveryDestinationInput,
+        super::operation::get_delivery_destination::GetDeliveryDestinationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_delivery_destination::GetDeliveryDestinationInput { name: self.name })
+        ::std::result::Result::Ok(super::operation::get_delivery_destination::GetDeliveryDestinationInput { name: self.name })
     }
 }
