--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ListFunctionsByCodeSigningConfigInput {
     /// Creates a new builder-style object to manufacture [`ListFunctionsByCodeSigningConfigInput`](crate::operation::list_functions_by_code_signing_config::ListFunctionsByCodeSigningConfigInput).
-    pub fn builder() -> crate::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigInputBuilder {
-        crate::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigInputBuilder {
+        super::super::super::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigInputBuilder::default()
     }
 }

@@ -87,11 +87,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_functions_by_code_signing_config::ListFunctionsByCodeSigningConfigInput,
+        super::super::super::operation::list_functions_by_code_signing_config::ListFunctionsByCodeSigningConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::list_functions_by_code_signing_config::ListFunctionsByCodeSigningConfigInput {
+            super::super::super::operation::list_functions_by_code_signing_config::ListFunctionsByCodeSigningConfigInput {
                 code_signing_config_arn: self.code_signing_config_arn,
                 marker: self.marker,
                 max_items: self.max_items,
