--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct ListSamlProvidersInput {}
 impl ListSamlProvidersInput {
     /// Creates a new builder-style object to manufacture [`ListSamlProvidersInput`](crate::operation::list_saml_providers::ListSamlProvidersInput).
-    pub fn builder() -> crate::operation::list_saml_providers::builders::ListSamlProvidersInputBuilder {
-        crate::operation::list_saml_providers::builders::ListSamlProvidersInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_saml_providers::builders::ListSamlProvidersInputBuilder {
+        super::super::super::operation::list_saml_providers::builders::ListSamlProvidersInputBuilder::default()
     }
 }

@@ -18,7 +18,7 @@
     /// Consumes the builder and constructs a [`ListSamlProvidersInput`](crate::operation::list_saml_providers::ListSamlProvidersInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_saml_providers::ListSamlProvidersInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_saml_providers::ListSamlProvidersInput {})
+    ) -> ::std::result::Result<super::super::super::operation::list_saml_providers::ListSamlProvidersInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_saml_providers::ListSamlProvidersInput {})
     }
 }
