--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl GetContextKeysForPrincipalPolicyInput {
     /// Creates a new builder-style object to manufacture [`GetContextKeysForPrincipalPolicyInput`](crate::operation::get_context_keys_for_principal_policy::GetContextKeysForPrincipalPolicyInput).
-    pub fn builder() -> crate::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyInputBuilder {
-        crate::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyInputBuilder {
+        super::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyInputBuilder::default()
     }
 }

@@ -124,11 +124,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_context_keys_for_principal_policy::GetContextKeysForPrincipalPolicyInput,
+        super::operation::get_context_keys_for_principal_policy::GetContextKeysForPrincipalPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_context_keys_for_principal_policy::GetContextKeysForPrincipalPolicyInput {
+            super::operation::get_context_keys_for_principal_policy::GetContextKeysForPrincipalPolicyInput {
                 policy_source_arn: self.policy_source_arn,
                 policy_input_list: self.policy_input_list,
             },
