--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteDataProtectionPolicyInput {
     /// Creates a new builder-style object to manufacture [`DeleteDataProtectionPolicyInput`](crate::operation::delete_data_protection_policy::DeleteDataProtectionPolicyInput).
-    pub fn builder() -> crate::operation::delete_data_protection_policy::builders::DeleteDataProtectionPolicyInputBuilder {
-        crate::operation::delete_data_protection_policy::builders::DeleteDataProtectionPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::delete_data_protection_policy::builders::DeleteDataProtectionPolicyInputBuilder {
+        super::operation::delete_data_protection_policy::builders::DeleteDataProtectionPolicyInputBuilder::default()
     }
 }

@@ -45,10 +45,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_data_protection_policy::DeleteDataProtectionPolicyInput,
+        super::operation::delete_data_protection_policy::DeleteDataProtectionPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_data_protection_policy::DeleteDataProtectionPolicyInput {
+        ::std::result::Result::Ok(super::operation::delete_data_protection_policy::DeleteDataProtectionPolicyInput {
             log_group_identifier: self.log_group_identifier,
         })
     }
