--- original
+++ modified
@@ -17,10 +17,10 @@
     Bs(::std::vec::Vec<::aws_smithy_types::Blob>),
     /// <p>An attribute of type List. For example:</p>
     /// <p><code>"L": \[ {"S": "Cookies"} , {"S": "Coffee"}, {"N": "3.14159"}\]</code></p>
-    L(::std::vec::Vec<crate::types::AttributeValue>),
+    L(::std::vec::Vec<super::types::AttributeValue>),
     /// <p>An attribute of type Map. For example:</p>
     /// <p><code>"M": {"Name": {"S": "Joe"}, "Age": {"N": "35"}}</code></p>
-    M(::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>),
+    M(::std::collections::HashMap<::std::string::String, super::types::AttributeValue>),
     /// <p>An attribute of type Number. For example:</p>
     /// <p><code>"N": "123.45"</code></p>
     /// <p>Numbers are sent across the network to DynamoDB as strings, to maximize compatibility across languages and libraries. However, DynamoDB treats them as number type attributes for mathematical operations.</p>
@@ -90,7 +90,7 @@
     }
     /// Tries to convert the enum instance into [`L`](crate::types::AttributeValue::L), extracting the inner [`Vec`](::std::vec::Vec).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_l(&self) -> ::std::result::Result<&::std::vec::Vec<crate::types::AttributeValue>, &Self> {
+    pub fn as_l(&self) -> ::std::result::Result<&::std::vec::Vec<super::types::AttributeValue>, &Self> {
         if let AttributeValue::L(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -103,7 +103,7 @@
     }
     /// Tries to convert the enum instance into [`M`](crate::types::AttributeValue::M), extracting the inner [`HashMap`](::std::collections::HashMap).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_m(&self) -> ::std::result::Result<&::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>, &Self> {
+    pub fn as_m(&self) -> ::std::result::Result<&::std::collections::HashMap<::std::string::String, super::types::AttributeValue>, &Self> {
         if let AttributeValue::M(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
