--- original
+++ modified
@@ -8,7 +8,7 @@
     pub policy_name: ::std::option::Option<::std::string::String>,
     /// <p>Represents a target tracking scaling policy configuration.</p>
     pub target_tracking_scaling_policy_configuration:
-        ::std::option::Option<crate::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription>,
+        ::std::option::Option<super::super::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription>,
 }
 impl AutoScalingPolicyDescription {
     /// <p>The name of the scaling policy.</p>
@@ -18,14 +18,14 @@
     /// <p>Represents a target tracking scaling policy configuration.</p>
     pub fn target_tracking_scaling_policy_configuration(
         &self,
-    ) -> ::std::option::Option<&crate::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription> {
+    ) -> ::std::option::Option<&super::super::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription> {
         self.target_tracking_scaling_policy_configuration.as_ref()
     }
 }
 impl AutoScalingPolicyDescription {
     /// Creates a new builder-style object to manufacture [`AutoScalingPolicyDescription`](crate::types::AutoScalingPolicyDescription).
-    pub fn builder() -> crate::types::builders::AutoScalingPolicyDescriptionBuilder {
-        crate::types::builders::AutoScalingPolicyDescriptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::AutoScalingPolicyDescriptionBuilder {
+        super::super::types::builders::AutoScalingPolicyDescriptionBuilder::default()
     }
 }

@@ -35,7 +35,7 @@
 pub struct AutoScalingPolicyDescriptionBuilder {
     pub(crate) policy_name: ::std::option::Option<::std::string::String>,
     pub(crate) target_tracking_scaling_policy_configuration:
-        ::std::option::Option<crate::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription>,
+        ::std::option::Option<super::super::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription>,
 }
 impl AutoScalingPolicyDescriptionBuilder {
     /// <p>The name of the scaling policy.</p>
@@ -55,7 +55,7 @@
     /// <p>Represents a target tracking scaling policy configuration.</p>
     pub fn target_tracking_scaling_policy_configuration(
         mut self,
-        input: crate::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription,
+        input: super::super::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription,
     ) -> Self {
         self.target_tracking_scaling_policy_configuration = ::std::option::Option::Some(input);
         self
@@ -63,7 +63,7 @@
     /// <p>Represents a target tracking scaling policy configuration.</p>
     pub fn set_target_tracking_scaling_policy_configuration(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription>,
+        input: ::std::option::Option<super::super::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription>,
     ) -> Self {
         self.target_tracking_scaling_policy_configuration = input;
         self
@@ -71,12 +71,12 @@
     /// <p>Represents a target tracking scaling policy configuration.</p>
     pub fn get_target_tracking_scaling_policy_configuration(
         &self,
-    ) -> &::std::option::Option<crate::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription> {
+    ) -> &::std::option::Option<super::super::types::AutoScalingTargetTrackingScalingPolicyConfigurationDescription> {
         &self.target_tracking_scaling_policy_configuration
     }
     /// Consumes the builder and constructs a [`AutoScalingPolicyDescription`](crate::types::AutoScalingPolicyDescription).
-    pub fn build(self) -> crate::types::AutoScalingPolicyDescription {
-        crate::types::AutoScalingPolicyDescription {
+    pub fn build(self) -> super::super::types::AutoScalingPolicyDescription {
+        super::super::types::AutoScalingPolicyDescription {
             policy_name: self.policy_name,
             target_tracking_scaling_policy_configuration: self.target_tracking_scaling_policy_configuration,
         }
