--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <p>Role ARN used for configuring auto scaling policy.</p>
     pub auto_scaling_role_arn: ::std::option::Option<::std::string::String>,
     /// <p>The scaling policy to apply for scaling target global table or global secondary index capacity units.</p>
-    pub scaling_policy_update: ::std::option::Option<crate::types::AutoScalingPolicyUpdate>,
+    pub scaling_policy_update: ::std::option::Option<super::super::types::AutoScalingPolicyUpdate>,
 }
 impl AutoScalingSettingsUpdate {
     /// <p>The minimum capacity units that a global table or global secondary index should be scaled down to.</p>
@@ -33,14 +33,14 @@
         self.auto_scaling_role_arn.as_deref()
     }
     /// <p>The scaling policy to apply for scaling target global table or global secondary index capacity units.</p>
-    pub fn scaling_policy_update(&self) -> ::std::option::Option<&crate::types::AutoScalingPolicyUpdate> {
+    pub fn scaling_policy_update(&self) -> ::std::option::Option<&super::super::types::AutoScalingPolicyUpdate> {
         self.scaling_policy_update.as_ref()
     }
 }
 impl AutoScalingSettingsUpdate {
     /// Creates a new builder-style object to manufacture [`AutoScalingSettingsUpdate`](crate::types::AutoScalingSettingsUpdate).
-    pub fn builder() -> crate::types::builders::AutoScalingSettingsUpdateBuilder {
-        crate::types::builders::AutoScalingSettingsUpdateBuilder::default()
+    pub fn builder() -> super::super::types::builders::AutoScalingSettingsUpdateBuilder {
+        super::super::types::builders::AutoScalingSettingsUpdateBuilder::default()
     }
 }

@@ -52,7 +52,7 @@
     pub(crate) maximum_units: ::std::option::Option<i64>,
     pub(crate) auto_scaling_disabled: ::std::option::Option<bool>,
     pub(crate) auto_scaling_role_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) scaling_policy_update: ::std::option::Option<crate::types::AutoScalingPolicyUpdate>,
+    pub(crate) scaling_policy_update: ::std::option::Option<super::super::types::AutoScalingPolicyUpdate>,
 }
 impl AutoScalingSettingsUpdateBuilder {
     /// <p>The minimum capacity units that a global table or global secondary index should be scaled down to.</p>
@@ -112,22 +112,22 @@
         &self.auto_scaling_role_arn
     }
     /// <p>The scaling policy to apply for scaling target global table or global secondary index capacity units.</p>
-    pub fn scaling_policy_update(mut self, input: crate::types::AutoScalingPolicyUpdate) -> Self {
+    pub fn scaling_policy_update(mut self, input: super::super::types::AutoScalingPolicyUpdate) -> Self {
         self.scaling_policy_update = ::std::option::Option::Some(input);
         self
     }
     /// <p>The scaling policy to apply for scaling target global table or global secondary index capacity units.</p>
-    pub fn set_scaling_policy_update(mut self, input: ::std::option::Option<crate::types::AutoScalingPolicyUpdate>) -> Self {
+    pub fn set_scaling_policy_update(mut self, input: ::std::option::Option<super::super::types::AutoScalingPolicyUpdate>) -> Self {
         self.scaling_policy_update = input;
         self
     }
     /// <p>The scaling policy to apply for scaling target global table or global secondary index capacity units.</p>
-    pub fn get_scaling_policy_update(&self) -> &::std::option::Option<crate::types::AutoScalingPolicyUpdate> {
+    pub fn get_scaling_policy_update(&self) -> &::std::option::Option<super::super::types::AutoScalingPolicyUpdate> {
         &self.scaling_policy_update
     }
     /// Consumes the builder and constructs a [`AutoScalingSettingsUpdate`](crate::types::AutoScalingSettingsUpdate).
-    pub fn build(self) -> crate::types::AutoScalingSettingsUpdate {
-        crate::types::AutoScalingSettingsUpdate {
+    pub fn build(self) -> super::super::types::AutoScalingSettingsUpdate {
+        super::super::types::AutoScalingSettingsUpdate {
             minimum_units: self.minimum_units,
             maximum_units: self.maximum_units,
             auto_scaling_disabled: self.auto_scaling_disabled,
