--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - Sets the read/write capacity mode to <code>PAY_PER_REQUEST</code>. We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads.</p></li>
     /// </ul>
-    pub billing_mode: ::std::option::Option<crate::types::BillingMode>,
+    pub billing_mode: ::std::option::Option<super::types::BillingMode>,
     /// <p>Represents the time when <code>PAY_PER_REQUEST</code> was last set as the read/write capacity mode.</p>
     pub last_update_to_pay_per_request_date_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -25,7 +25,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - Sets the read/write capacity mode to <code>PAY_PER_REQUEST</code>. We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads.</p></li>
     /// </ul>
-    pub fn billing_mode(&self) -> ::std::option::Option<&crate::types::BillingMode> {
+    pub fn billing_mode(&self) -> ::std::option::Option<&super::types::BillingMode> {
         self.billing_mode.as_ref()
     }
     /// <p>Represents the time when <code>PAY_PER_REQUEST</code> was last set as the read/write capacity mode.</p>
@@ -35,8 +35,8 @@
 }
 impl BillingModeSummary {
     /// Creates a new builder-style object to manufacture [`BillingModeSummary`](crate::types::BillingModeSummary).
-    pub fn builder() -> crate::types::builders::BillingModeSummaryBuilder {
-        crate::types::builders::BillingModeSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::BillingModeSummaryBuilder {
+        super::types::builders::BillingModeSummaryBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct BillingModeSummaryBuilder {
-    pub(crate) billing_mode: ::std::option::Option<crate::types::BillingMode>,
+    pub(crate) billing_mode: ::std::option::Option<super::types::BillingMode>,
     pub(crate) last_update_to_pay_per_request_date_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl BillingModeSummaryBuilder {
@@ -55,7 +55,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - Sets the read/write capacity mode to <code>PAY_PER_REQUEST</code>. We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads.</p></li>
     /// </ul>
-    pub fn billing_mode(mut self, input: crate::types::BillingMode) -> Self {
+    pub fn billing_mode(mut self, input: super::types::BillingMode) -> Self {
         self.billing_mode = ::std::option::Option::Some(input);
         self
     }
@@ -66,7 +66,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - Sets the read/write capacity mode to <code>PAY_PER_REQUEST</code>. We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads.</p></li>
     /// </ul>
-    pub fn set_billing_mode(mut self, input: ::std::option::Option<crate::types::BillingMode>) -> Self {
+    pub fn set_billing_mode(mut self, input: ::std::option::Option<super::types::BillingMode>) -> Self {
         self.billing_mode = input;
         self
     }
@@ -77,7 +77,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - Sets the read/write capacity mode to <code>PAY_PER_REQUEST</code>. We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads.</p></li>
     /// </ul>
-    pub fn get_billing_mode(&self) -> &::std::option::Option<crate::types::BillingMode> {
+    pub fn get_billing_mode(&self) -> &::std::option::Option<super::types::BillingMode> {
         &self.billing_mode
     }
     /// <p>Represents the time when <code>PAY_PER_REQUEST</code> was last set as the read/write capacity mode.</p>
@@ -95,8 +95,8 @@
         &self.last_update_to_pay_per_request_date_time
     }
     /// Consumes the builder and constructs a [`BillingModeSummary`](crate::types::BillingModeSummary).
-    pub fn build(self) -> crate::types::BillingModeSummary {
-        crate::types::BillingModeSummary {
+    pub fn build(self) -> super::types::BillingModeSummary {
+        super::types::BillingModeSummary {
             billing_mode: self.billing_mode,
             last_update_to_pay_per_request_date_time: self.last_update_to_pay_per_request_date_time,
         }
