--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct CancellationReason {
     /// <p>Item in the request which caused the transaction to get cancelled.</p>
-    pub item: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+    pub item: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
     /// <p>Status code for the result of the cancelled transaction.</p>
     pub code: ::std::option::Option<::std::string::String>,
     /// <p>Cancellation reason message description.</p>
@@ -13,7 +13,7 @@
 }
 impl CancellationReason {
     /// <p>Item in the request which caused the transaction to get cancelled.</p>
-    pub fn item(&self) -> ::std::option::Option<&::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>> {
+    pub fn item(&self) -> ::std::option::Option<&::std::collections::HashMap<::std::string::String, super::types::AttributeValue>> {
         self.item.as_ref()
     }
     /// <p>Status code for the result of the cancelled transaction.</p>
@@ -27,8 +27,8 @@
 }
 impl CancellationReason {
     /// Creates a new builder-style object to manufacture [`CancellationReason`](crate::types::CancellationReason).
-    pub fn builder() -> crate::types::builders::CancellationReasonBuilder {
-        crate::types::builders::CancellationReasonBuilder::default()
+    pub fn builder() -> super::types::builders::CancellationReasonBuilder {
+        super::types::builders::CancellationReasonBuilder::default()
     }
 }

@@ -36,7 +36,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct CancellationReasonBuilder {
-    pub(crate) item: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+    pub(crate) item: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
     pub(crate) code: ::std::option::Option<::std::string::String>,
     pub(crate) message: ::std::option::Option<::std::string::String>,
 }
@@ -46,7 +46,7 @@
     /// To override the contents of this collection use [`set_item`](Self::set_item).
     ///
     /// <p>Item in the request which caused the transaction to get cancelled.</p>
-    pub fn item(mut self, k: impl ::std::convert::Into<::std::string::String>, v: crate::types::AttributeValue) -> Self {
+    pub fn item(mut self, k: impl ::std::convert::Into<::std::string::String>, v: super::types::AttributeValue) -> Self {
         let mut hash_map = self.item.unwrap_or_default();
         hash_map.insert(k.into(), v);
         self.item = ::std::option::Option::Some(hash_map);
@@ -55,13 +55,13 @@
     /// <p>Item in the request which caused the transaction to get cancelled.</p>
     pub fn set_item(
         mut self,
-        input: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+        input: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
     ) -> Self {
         self.item = input;
         self
     }
     /// <p>Item in the request which caused the transaction to get cancelled.</p>
-    pub fn get_item(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>> {
+    pub fn get_item(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>> {
         &self.item
     }
     /// <p>Status code for the result of the cancelled transaction.</p>
@@ -93,8 +93,8 @@
         &self.message
     }
     /// Consumes the builder and constructs a [`CancellationReason`](crate::types::CancellationReason).
-    pub fn build(self) -> crate::types::CancellationReason {
-        crate::types::CancellationReason {
+    pub fn build(self) -> super::types::CancellationReason {
+        super::types::CancellationReason {
             item: self.item,
             code: self.code,
             message: self.message,
