--- original
+++ modified
@@ -5,24 +5,24 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ContinuousBackupsDescription {
     /// <p><code>ContinuousBackupsStatus</code> can be one of the following states: ENABLED, DISABLED</p>
-    pub continuous_backups_status: crate::types::ContinuousBackupsStatus,
+    pub continuous_backups_status: super::types::ContinuousBackupsStatus,
     /// <p>The description of the point in time recovery settings applied to the table.</p>
-    pub point_in_time_recovery_description: ::std::option::Option<crate::types::PointInTimeRecoveryDescription>,
+    pub point_in_time_recovery_description: ::std::option::Option<super::types::PointInTimeRecoveryDescription>,
 }
 impl ContinuousBackupsDescription {
     /// <p><code>ContinuousBackupsStatus</code> can be one of the following states: ENABLED, DISABLED</p>
-    pub fn continuous_backups_status(&self) -> &crate::types::ContinuousBackupsStatus {
+    pub fn continuous_backups_status(&self) -> &super::types::ContinuousBackupsStatus {
         &self.continuous_backups_status
     }
     /// <p>The description of the point in time recovery settings applied to the table.</p>
-    pub fn point_in_time_recovery_description(&self) -> ::std::option::Option<&crate::types::PointInTimeRecoveryDescription> {
+    pub fn point_in_time_recovery_description(&self) -> ::std::option::Option<&super::types::PointInTimeRecoveryDescription> {
         self.point_in_time_recovery_description.as_ref()
     }
 }
 impl ContinuousBackupsDescription {
     /// Creates a new builder-style object to manufacture [`ContinuousBackupsDescription`](crate::types::ContinuousBackupsDescription).
-    pub fn builder() -> crate::types::builders::ContinuousBackupsDescriptionBuilder {
-        crate::types::builders::ContinuousBackupsDescriptionBuilder::default()
+    pub fn builder() -> super::types::builders::ContinuousBackupsDescriptionBuilder {
+        super::types::builders::ContinuousBackupsDescriptionBuilder::default()
     }
 }

@@ -30,44 +30,44 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ContinuousBackupsDescriptionBuilder {
-    pub(crate) continuous_backups_status: ::std::option::Option<crate::types::ContinuousBackupsStatus>,
-    pub(crate) point_in_time_recovery_description: ::std::option::Option<crate::types::PointInTimeRecoveryDescription>,
+    pub(crate) continuous_backups_status: ::std::option::Option<super::types::ContinuousBackupsStatus>,
+    pub(crate) point_in_time_recovery_description: ::std::option::Option<super::types::PointInTimeRecoveryDescription>,
 }
 impl ContinuousBackupsDescriptionBuilder {
     /// <p><code>ContinuousBackupsStatus</code> can be one of the following states: ENABLED, DISABLED</p>
     /// This field is required.
-    pub fn continuous_backups_status(mut self, input: crate::types::ContinuousBackupsStatus) -> Self {
+    pub fn continuous_backups_status(mut self, input: super::types::ContinuousBackupsStatus) -> Self {
         self.continuous_backups_status = ::std::option::Option::Some(input);
         self
     }
     /// <p><code>ContinuousBackupsStatus</code> can be one of the following states: ENABLED, DISABLED</p>
-    pub fn set_continuous_backups_status(mut self, input: ::std::option::Option<crate::types::ContinuousBackupsStatus>) -> Self {
+    pub fn set_continuous_backups_status(mut self, input: ::std::option::Option<super::types::ContinuousBackupsStatus>) -> Self {
         self.continuous_backups_status = input;
         self
     }
     /// <p><code>ContinuousBackupsStatus</code> can be one of the following states: ENABLED, DISABLED</p>
-    pub fn get_continuous_backups_status(&self) -> &::std::option::Option<crate::types::ContinuousBackupsStatus> {
+    pub fn get_continuous_backups_status(&self) -> &::std::option::Option<super::types::ContinuousBackupsStatus> {
         &self.continuous_backups_status
     }
     /// <p>The description of the point in time recovery settings applied to the table.</p>
-    pub fn point_in_time_recovery_description(mut self, input: crate::types::PointInTimeRecoveryDescription) -> Self {
+    pub fn point_in_time_recovery_description(mut self, input: super::types::PointInTimeRecoveryDescription) -> Self {
         self.point_in_time_recovery_description = ::std::option::Option::Some(input);
         self
     }
     /// <p>The description of the point in time recovery settings applied to the table.</p>
-    pub fn set_point_in_time_recovery_description(mut self, input: ::std::option::Option<crate::types::PointInTimeRecoveryDescription>) -> Self {
+    pub fn set_point_in_time_recovery_description(mut self, input: ::std::option::Option<super::types::PointInTimeRecoveryDescription>) -> Self {
         self.point_in_time_recovery_description = input;
         self
     }
     /// <p>The description of the point in time recovery settings applied to the table.</p>
-    pub fn get_point_in_time_recovery_description(&self) -> &::std::option::Option<crate::types::PointInTimeRecoveryDescription> {
+    pub fn get_point_in_time_recovery_description(&self) -> &::std::option::Option<super::types::PointInTimeRecoveryDescription> {
         &self.point_in_time_recovery_description
     }
     /// Consumes the builder and constructs a [`ContinuousBackupsDescription`](crate::types::ContinuousBackupsDescription).
     /// This method will fail if any of the following fields are not set:
     /// - [`continuous_backups_status`](crate::types::builders::ContinuousBackupsDescriptionBuilder::continuous_backups_status)
-    pub fn build(self) -> ::std::result::Result<crate::types::ContinuousBackupsDescription, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ContinuousBackupsDescription {
+    pub fn build(self) -> ::std::result::Result<super::types::ContinuousBackupsDescription, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ContinuousBackupsDescription {
             continuous_backups_status: self.continuous_backups_status.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "continuous_backups_status",
