--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl CreateReplicaAction {
     /// Creates a new builder-style object to manufacture [`CreateReplicaAction`](crate::types::CreateReplicaAction).
-    pub fn builder() -> crate::types::builders::CreateReplicaActionBuilder {
-        crate::types::builders::CreateReplicaActionBuilder::default()
+    pub fn builder() -> super::super::types::builders::CreateReplicaActionBuilder {
+        super::super::types::builders::CreateReplicaActionBuilder::default()
     }
 }

@@ -46,8 +46,8 @@
     /// Consumes the builder and constructs a [`CreateReplicaAction`](crate::types::CreateReplicaAction).
     /// This method will fail if any of the following fields are not set:
     /// - [`region_name`](crate::types::builders::CreateReplicaActionBuilder::region_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::CreateReplicaAction, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::CreateReplicaAction {
+    pub fn build(self) -> ::std::result::Result<super::super::types::CreateReplicaAction, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::CreateReplicaAction {
             region_name: self.region_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "region_name",
