--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct DeleteRequest {
     /// <p>A map of attribute name to attribute values, representing the primary key of the item to delete. All of the table's primary key attributes must be specified, and their data types must match those of the table's key schema.</p>
-    pub key: ::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>,
+    pub key: ::std::collections::HashMap<::std::string::String, super::super::types::AttributeValue>,
 }
 impl DeleteRequest {
     /// <p>A map of attribute name to attribute values, representing the primary key of the item to delete. All of the table's primary key attributes must be specified, and their data types must match those of the table's key schema.</p>
-    pub fn key(&self) -> &::std::collections::HashMap<::std::string::String, crate::types::AttributeValue> {
+    pub fn key(&self) -> &::std::collections::HashMap<::std::string::String, super::super::types::AttributeValue> {
         &self.key
     }
 }
 impl DeleteRequest {
     /// Creates a new builder-style object to manufacture [`DeleteRequest`](crate::types::DeleteRequest).
-    pub fn builder() -> crate::types::builders::DeleteRequestBuilder {
-        crate::types::builders::DeleteRequestBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeleteRequestBuilder {
+        super::super::types::builders::DeleteRequestBuilder::default()
     }
 }

@@ -24,7 +24,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct DeleteRequestBuilder {
-    pub(crate) key: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+    pub(crate) key: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::super::types::AttributeValue>>,
 }
 impl DeleteRequestBuilder {
     /// Adds a key-value pair to `key`.
@@ -32,7 +32,7 @@
     /// To override the contents of this collection use [`set_key`](Self::set_key).
     ///
     /// <p>A map of attribute name to attribute values, representing the primary key of the item to delete. All of the table's primary key attributes must be specified, and their data types must match those of the table's key schema.</p>
-    pub fn key(mut self, k: impl ::std::convert::Into<::std::string::String>, v: crate::types::AttributeValue) -> Self {
+    pub fn key(mut self, k: impl ::std::convert::Into<::std::string::String>, v: super::super::types::AttributeValue) -> Self {
         let mut hash_map = self.key.unwrap_or_default();
         hash_map.insert(k.into(), v);
         self.key = ::std::option::Option::Some(hash_map);
@@ -39,19 +39,19 @@
         self
     }
     /// <p>A map of attribute name to attribute values, representing the primary key of the item to delete. All of the table's primary key attributes must be specified, and their data types must match those of the table's key schema.</p>
-    pub fn set_key(mut self, input: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>) -> Self {
+    pub fn set_key(mut self, input: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::super::types::AttributeValue>>) -> Self {
         self.key = input;
         self
     }
     /// <p>A map of attribute name to attribute values, representing the primary key of the item to delete. All of the table's primary key attributes must be specified, and their data types must match those of the table's key schema.</p>
-    pub fn get_key(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>> {
+    pub fn get_key(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, super::super::types::AttributeValue>> {
         &self.key
     }
     /// Consumes the builder and constructs a [`DeleteRequest`](crate::types::DeleteRequest).
     /// This method will fail if any of the following fields are not set:
     /// - [`key`](crate::types::builders::DeleteRequestBuilder::key)
-    pub fn build(self) -> ::std::result::Result<crate::types::DeleteRequest, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::DeleteRequest {
+    pub fn build(self) -> ::std::result::Result<super::super::types::DeleteRequest, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::DeleteRequest {
             key: self.key.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "key",
