--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct Get {
     /// <p>A map of attribute names to <code>AttributeValue</code> objects that specifies the primary key of the item to retrieve.</p>
-    pub key: ::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>,
+    pub key: ::std::collections::HashMap<::std::string::String, super::types::AttributeValue>,
     /// <p>The name of the table from which to retrieve the specified item. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
     pub table_name: ::std::string::String,
     /// <p>A string that identifies one or more attributes of the specified item to retrieve from the table. The attributes in the expression must be separated by commas. If no attribute names are specified, then all attributes of the specified item are returned. If any of the requested attributes are not found, they do not appear in the result.</p>
@@ -15,7 +15,7 @@
 }
 impl Get {
     /// <p>A map of attribute names to <code>AttributeValue</code> objects that specifies the primary key of the item to retrieve.</p>
-    pub fn key(&self) -> &::std::collections::HashMap<::std::string::String, crate::types::AttributeValue> {
+    pub fn key(&self) -> &::std::collections::HashMap<::std::string::String, super::types::AttributeValue> {
         &self.key
     }
     /// <p>The name of the table from which to retrieve the specified item. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
@@ -34,8 +34,8 @@
 }
 impl Get {
     /// Creates a new builder-style object to manufacture [`Get`](crate::types::Get).
-    pub fn builder() -> crate::types::builders::GetBuilder {
-        crate::types::builders::GetBuilder::default()
+    pub fn builder() -> super::types::builders::GetBuilder {
+        super::types::builders::GetBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct GetBuilder {
-    pub(crate) key: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+    pub(crate) key: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
     pub(crate) table_name: ::std::option::Option<::std::string::String>,
     pub(crate) projection_expression: ::std::option::Option<::std::string::String>,
     pub(crate) expression_attribute_names: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
@@ -54,7 +54,7 @@
     /// To override the contents of this collection use [`set_key`](Self::set_key).
     ///
     /// <p>A map of attribute names to <code>AttributeValue</code> objects that specifies the primary key of the item to retrieve.</p>
-    pub fn key(mut self, k: impl ::std::convert::Into<::std::string::String>, v: crate::types::AttributeValue) -> Self {
+    pub fn key(mut self, k: impl ::std::convert::Into<::std::string::String>, v: super::types::AttributeValue) -> Self {
         let mut hash_map = self.key.unwrap_or_default();
         hash_map.insert(k.into(), v);
         self.key = ::std::option::Option::Some(hash_map);
@@ -61,12 +61,12 @@
         self
     }
     /// <p>A map of attribute names to <code>AttributeValue</code> objects that specifies the primary key of the item to retrieve.</p>
-    pub fn set_key(mut self, input: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>) -> Self {
+    pub fn set_key(mut self, input: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>) -> Self {
         self.key = input;
         self
     }
     /// <p>A map of attribute names to <code>AttributeValue</code> objects that specifies the primary key of the item to retrieve.</p>
-    pub fn get_key(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>> {
+    pub fn get_key(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>> {
         &self.key
     }
     /// <p>The name of the table from which to retrieve the specified item. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
@@ -131,8 +131,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`key`](crate::types::builders::GetBuilder::key)
     /// - [`table_name`](crate::types::builders::GetBuilder::table_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::Get, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Get {
+    pub fn build(self) -> ::std::result::Result<super::types::Get, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::Get {
             key: self.key.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field("key", "key was not specified but it is required when building Get")
             })?,
