--- original
+++ modified
@@ -13,7 +13,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct GlobalSecondaryIndexUpdate {
     /// <p>The name of an existing global secondary index, along with new provisioned throughput settings to be applied to that index.</p>
-    pub update: ::std::option::Option<crate::types::UpdateGlobalSecondaryIndexAction>,
+    pub update: ::std::option::Option<super::super::types::UpdateGlobalSecondaryIndexAction>,
     /// <p>The parameters required for creating a global secondary index on an existing table:</p>
     /// <ul>
     /// <li>
@@ -27,13 +27,13 @@
     /// <li>
     /// <p><code>ProvisionedThroughput </code></p></li>
     /// </ul>
-    pub create: ::std::option::Option<crate::types::CreateGlobalSecondaryIndexAction>,
+    pub create: ::std::option::Option<super::super::types::CreateGlobalSecondaryIndexAction>,
     /// <p>The name of an existing global secondary index to be removed.</p>
-    pub delete: ::std::option::Option<crate::types::DeleteGlobalSecondaryIndexAction>,
+    pub delete: ::std::option::Option<super::super::types::DeleteGlobalSecondaryIndexAction>,
 }
 impl GlobalSecondaryIndexUpdate {
     /// <p>The name of an existing global secondary index, along with new provisioned throughput settings to be applied to that index.</p>
-    pub fn update(&self) -> ::std::option::Option<&crate::types::UpdateGlobalSecondaryIndexAction> {
+    pub fn update(&self) -> ::std::option::Option<&super::super::types::UpdateGlobalSecondaryIndexAction> {
         self.update.as_ref()
     }
     /// <p>The parameters required for creating a global secondary index on an existing table:</p>
@@ -49,18 +49,18 @@
     /// <li>
     /// <p><code>ProvisionedThroughput </code></p></li>
     /// </ul>
-    pub fn create(&self) -> ::std::option::Option<&crate::types::CreateGlobalSecondaryIndexAction> {
+    pub fn create(&self) -> ::std::option::Option<&super::super::types::CreateGlobalSecondaryIndexAction> {
         self.create.as_ref()
     }
     /// <p>The name of an existing global secondary index to be removed.</p>
-    pub fn delete(&self) -> ::std::option::Option<&crate::types::DeleteGlobalSecondaryIndexAction> {
+    pub fn delete(&self) -> ::std::option::Option<&super::super::types::DeleteGlobalSecondaryIndexAction> {
         self.delete.as_ref()
     }
 }
 impl GlobalSecondaryIndexUpdate {
     /// Creates a new builder-style object to manufacture [`GlobalSecondaryIndexUpdate`](crate::types::GlobalSecondaryIndexUpdate).
-    pub fn builder() -> crate::types::builders::GlobalSecondaryIndexUpdateBuilder {
-        crate::types::builders::GlobalSecondaryIndexUpdateBuilder::default()
+    pub fn builder() -> super::super::types::builders::GlobalSecondaryIndexUpdateBuilder {
+        super::super::types::builders::GlobalSecondaryIndexUpdateBuilder::default()
     }
 }

@@ -68,23 +68,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct GlobalSecondaryIndexUpdateBuilder {
-    pub(crate) update: ::std::option::Option<crate::types::UpdateGlobalSecondaryIndexAction>,
-    pub(crate) create: ::std::option::Option<crate::types::CreateGlobalSecondaryIndexAction>,
-    pub(crate) delete: ::std::option::Option<crate::types::DeleteGlobalSecondaryIndexAction>,
+    pub(crate) update: ::std::option::Option<super::super::types::UpdateGlobalSecondaryIndexAction>,
+    pub(crate) create: ::std::option::Option<super::super::types::CreateGlobalSecondaryIndexAction>,
+    pub(crate) delete: ::std::option::Option<super::super::types::DeleteGlobalSecondaryIndexAction>,
 }
 impl GlobalSecondaryIndexUpdateBuilder {
     /// <p>The name of an existing global secondary index, along with new provisioned throughput settings to be applied to that index.</p>
-    pub fn update(mut self, input: crate::types::UpdateGlobalSecondaryIndexAction) -> Self {
+    pub fn update(mut self, input: super::super::types::UpdateGlobalSecondaryIndexAction) -> Self {
         self.update = ::std::option::Option::Some(input);
         self
     }
     /// <p>The name of an existing global secondary index, along with new provisioned throughput settings to be applied to that index.</p>
-    pub fn set_update(mut self, input: ::std::option::Option<crate::types::UpdateGlobalSecondaryIndexAction>) -> Self {
+    pub fn set_update(mut self, input: ::std::option::Option<super::super::types::UpdateGlobalSecondaryIndexAction>) -> Self {
         self.update = input;
         self
     }
     /// <p>The name of an existing global secondary index, along with new provisioned throughput settings to be applied to that index.</p>
-    pub fn get_update(&self) -> &::std::option::Option<crate::types::UpdateGlobalSecondaryIndexAction> {
+    pub fn get_update(&self) -> &::std::option::Option<super::super::types::UpdateGlobalSecondaryIndexAction> {
         &self.update
     }
     /// <p>The parameters required for creating a global secondary index on an existing table:</p>
@@ -100,7 +100,7 @@
     /// <li>
     /// <p><code>ProvisionedThroughput </code></p></li>
     /// </ul>
-    pub fn create(mut self, input: crate::types::CreateGlobalSecondaryIndexAction) -> Self {
+    pub fn create(mut self, input: super::super::types::CreateGlobalSecondaryIndexAction) -> Self {
         self.create = ::std::option::Option::Some(input);
         self
     }
@@ -117,7 +117,7 @@
     /// <li>
     /// <p><code>ProvisionedThroughput </code></p></li>
     /// </ul>
-    pub fn set_create(mut self, input: ::std::option::Option<crate::types::CreateGlobalSecondaryIndexAction>) -> Self {
+    pub fn set_create(mut self, input: ::std::option::Option<super::super::types::CreateGlobalSecondaryIndexAction>) -> Self {
         self.create = input;
         self
     }
@@ -134,26 +134,26 @@
     /// <li>
     /// <p><code>ProvisionedThroughput </code></p></li>
     /// </ul>
-    pub fn get_create(&self) -> &::std::option::Option<crate::types::CreateGlobalSecondaryIndexAction> {
+    pub fn get_create(&self) -> &::std::option::Option<super::super::types::CreateGlobalSecondaryIndexAction> {
         &self.create
     }
     /// <p>The name of an existing global secondary index to be removed.</p>
-    pub fn delete(mut self, input: crate::types::DeleteGlobalSecondaryIndexAction) -> Self {
+    pub fn delete(mut self, input: super::super::types::DeleteGlobalSecondaryIndexAction) -> Self {
         self.delete = ::std::option::Option::Some(input);
         self
     }
     /// <p>The name of an existing global secondary index to be removed.</p>
-    pub fn set_delete(mut self, input: ::std::option::Option<crate::types::DeleteGlobalSecondaryIndexAction>) -> Self {
+    pub fn set_delete(mut self, input: ::std::option::Option<super::super::types::DeleteGlobalSecondaryIndexAction>) -> Self {
         self.delete = input;
         self
     }
     /// <p>The name of an existing global secondary index to be removed.</p>
-    pub fn get_delete(&self) -> &::std::option::Option<crate::types::DeleteGlobalSecondaryIndexAction> {
+    pub fn get_delete(&self) -> &::std::option::Option<super::super::types::DeleteGlobalSecondaryIndexAction> {
         &self.delete
     }
     /// Consumes the builder and constructs a [`GlobalSecondaryIndexUpdate`](crate::types::GlobalSecondaryIndexUpdate).
-    pub fn build(self) -> crate::types::GlobalSecondaryIndexUpdate {
-        crate::types::GlobalSecondaryIndexUpdate {
+    pub fn build(self) -> super::super::types::GlobalSecondaryIndexUpdate {
+        super::super::types::GlobalSecondaryIndexUpdate {
             update: self.update,
             create: self.create,
             delete: self.delete,
