--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ItemResponse {
     /// <p>Map of attribute data consisting of the data type and attribute value.</p>
-    pub item: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+    pub item: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
 }
 impl ItemResponse {
     /// <p>Map of attribute data consisting of the data type and attribute value.</p>
-    pub fn item(&self) -> ::std::option::Option<&::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>> {
+    pub fn item(&self) -> ::std::option::Option<&::std::collections::HashMap<::std::string::String, super::types::AttributeValue>> {
         self.item.as_ref()
     }
 }
 impl ItemResponse {
     /// Creates a new builder-style object to manufacture [`ItemResponse`](crate::types::ItemResponse).
-    pub fn builder() -> crate::types::builders::ItemResponseBuilder {
-        crate::types::builders::ItemResponseBuilder::default()
+    pub fn builder() -> super::types::builders::ItemResponseBuilder {
+        super::types::builders::ItemResponseBuilder::default()
     }
 }

@@ -24,7 +24,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ItemResponseBuilder {
-    pub(crate) item: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+    pub(crate) item: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
 }
 impl ItemResponseBuilder {
     /// Adds a key-value pair to `item`.
@@ -32,7 +32,7 @@
     /// To override the contents of this collection use [`set_item`](Self::set_item).
     ///
     /// <p>Map of attribute data consisting of the data type and attribute value.</p>
-    pub fn item(mut self, k: impl ::std::convert::Into<::std::string::String>, v: crate::types::AttributeValue) -> Self {
+    pub fn item(mut self, k: impl ::std::convert::Into<::std::string::String>, v: super::types::AttributeValue) -> Self {
         let mut hash_map = self.item.unwrap_or_default();
         hash_map.insert(k.into(), v);
         self.item = ::std::option::Option::Some(hash_map);
@@ -41,17 +41,17 @@
     /// <p>Map of attribute data consisting of the data type and attribute value.</p>
     pub fn set_item(
         mut self,
-        input: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+        input: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
     ) -> Self {
         self.item = input;
         self
     }
     /// <p>Map of attribute data consisting of the data type and attribute value.</p>
-    pub fn get_item(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>> {
+    pub fn get_item(&self) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>> {
         &self.item
     }
     /// Consumes the builder and constructs a [`ItemResponse`](crate::types::ItemResponse).
-    pub fn build(self) -> crate::types::ItemResponse {
-        crate::types::ItemResponse { item: self.item }
+    pub fn build(self) -> super::types::ItemResponse {
+        super::types::ItemResponse { item: self.item }
     }
 }
