--- original
+++ modified
@@ -7,11 +7,11 @@
     /// <p>The ARN for a specific Kinesis data stream.</p>
     pub stream_arn: ::std::option::Option<::std::string::String>,
     /// <p>The current status of replication.</p>
-    pub destination_status: ::std::option::Option<crate::types::DestinationStatus>,
+    pub destination_status: ::std::option::Option<super::super::types::DestinationStatus>,
     /// <p>The human-readable string that corresponds to the replica status.</p>
     pub destination_status_description: ::std::option::Option<::std::string::String>,
     /// <p>The precision of the Kinesis data stream timestamp. The values are either <code>MILLISECOND</code> or <code>MICROSECOND</code>.</p>
-    pub approximate_creation_date_time_precision: ::std::option::Option<crate::types::ApproximateCreationDateTimePrecision>,
+    pub approximate_creation_date_time_precision: ::std::option::Option<super::super::types::ApproximateCreationDateTimePrecision>,
 }
 impl KinesisDataStreamDestination {
     /// <p>The ARN for a specific Kinesis data stream.</p>
@@ -19,7 +19,7 @@
         self.stream_arn.as_deref()
     }
     /// <p>The current status of replication.</p>
-    pub fn destination_status(&self) -> ::std::option::Option<&crate::types::DestinationStatus> {
+    pub fn destination_status(&self) -> ::std::option::Option<&super::super::types::DestinationStatus> {
         self.destination_status.as_ref()
     }
     /// <p>The human-readable string that corresponds to the replica status.</p>
@@ -27,14 +27,14 @@
         self.destination_status_description.as_deref()
     }
     /// <p>The precision of the Kinesis data stream timestamp. The values are either <code>MILLISECOND</code> or <code>MICROSECOND</code>.</p>
-    pub fn approximate_creation_date_time_precision(&self) -> ::std::option::Option<&crate::types::ApproximateCreationDateTimePrecision> {
+    pub fn approximate_creation_date_time_precision(&self) -> ::std::option::Option<&super::super::types::ApproximateCreationDateTimePrecision> {
         self.approximate_creation_date_time_precision.as_ref()
     }
 }
 impl KinesisDataStreamDestination {
     /// Creates a new builder-style object to manufacture [`KinesisDataStreamDestination`](crate::types::KinesisDataStreamDestination).
-    pub fn builder() -> crate::types::builders::KinesisDataStreamDestinationBuilder {
-        crate::types::builders::KinesisDataStreamDestinationBuilder::default()
+    pub fn builder() -> super::super::types::builders::KinesisDataStreamDestinationBuilder {
+        super::super::types::builders::KinesisDataStreamDestinationBuilder::default()
     }
 }

@@ -43,9 +43,9 @@
 #[non_exhaustive]
 pub struct KinesisDataStreamDestinationBuilder {
     pub(crate) stream_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) destination_status: ::std::option::Option<crate::types::DestinationStatus>,
+    pub(crate) destination_status: ::std::option::Option<super::super::types::DestinationStatus>,
     pub(crate) destination_status_description: ::std::option::Option<::std::string::String>,
-    pub(crate) approximate_creation_date_time_precision: ::std::option::Option<crate::types::ApproximateCreationDateTimePrecision>,
+    pub(crate) approximate_creation_date_time_precision: ::std::option::Option<super::super::types::ApproximateCreationDateTimePrecision>,
 }
 impl KinesisDataStreamDestinationBuilder {
     /// <p>The ARN for a specific Kinesis data stream.</p>
@@ -63,17 +63,17 @@
         &self.stream_arn
     }
     /// <p>The current status of replication.</p>
-    pub fn destination_status(mut self, input: crate::types::DestinationStatus) -> Self {
+    pub fn destination_status(mut self, input: super::super::types::DestinationStatus) -> Self {
         self.destination_status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The current status of replication.</p>
-    pub fn set_destination_status(mut self, input: ::std::option::Option<crate::types::DestinationStatus>) -> Self {
+    pub fn set_destination_status(mut self, input: ::std::option::Option<super::super::types::DestinationStatus>) -> Self {
         self.destination_status = input;
         self
     }
     /// <p>The current status of replication.</p>
-    pub fn get_destination_status(&self) -> &::std::option::Option<crate::types::DestinationStatus> {
+    pub fn get_destination_status(&self) -> &::std::option::Option<super::super::types::DestinationStatus> {
         &self.destination_status
     }
     /// <p>The human-readable string that corresponds to the replica status.</p>
@@ -91,7 +91,7 @@
         &self.destination_status_description
     }
     /// <p>The precision of the Kinesis data stream timestamp. The values are either <code>MILLISECOND</code> or <code>MICROSECOND</code>.</p>
-    pub fn approximate_creation_date_time_precision(mut self, input: crate::types::ApproximateCreationDateTimePrecision) -> Self {
+    pub fn approximate_creation_date_time_precision(mut self, input: super::super::types::ApproximateCreationDateTimePrecision) -> Self {
         self.approximate_creation_date_time_precision = ::std::option::Option::Some(input);
         self
     }
@@ -98,18 +98,18 @@
     /// <p>The precision of the Kinesis data stream timestamp. The values are either <code>MILLISECOND</code> or <code>MICROSECOND</code>.</p>
     pub fn set_approximate_creation_date_time_precision(
         mut self,
-        input: ::std::option::Option<crate::types::ApproximateCreationDateTimePrecision>,
+        input: ::std::option::Option<super::super::types::ApproximateCreationDateTimePrecision>,
     ) -> Self {
         self.approximate_creation_date_time_precision = input;
         self
     }
     /// <p>The precision of the Kinesis data stream timestamp. The values are either <code>MILLISECOND</code> or <code>MICROSECOND</code>.</p>
-    pub fn get_approximate_creation_date_time_precision(&self) -> &::std::option::Option<crate::types::ApproximateCreationDateTimePrecision> {
+    pub fn get_approximate_creation_date_time_precision(&self) -> &::std::option::Option<super::super::types::ApproximateCreationDateTimePrecision> {
         &self.approximate_creation_date_time_precision
     }
     /// Consumes the builder and constructs a [`KinesisDataStreamDestination`](crate::types::KinesisDataStreamDestination).
-    pub fn build(self) -> crate::types::KinesisDataStreamDestination {
-        crate::types::KinesisDataStreamDestination {
+    pub fn build(self) -> super::super::types::KinesisDataStreamDestination {
+        super::super::types::KinesisDataStreamDestination {
             stream_arn: self.stream_arn,
             destination_status: self.destination_status,
             destination_status_description: self.destination_status_description,
