--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl PointInTimeRecoverySpecification {
     /// Creates a new builder-style object to manufacture [`PointInTimeRecoverySpecification`](crate::types::PointInTimeRecoverySpecification).
-    pub fn builder() -> crate::types::builders::PointInTimeRecoverySpecificationBuilder {
-        crate::types::builders::PointInTimeRecoverySpecificationBuilder::default()
+    pub fn builder() -> super::super::types::builders::PointInTimeRecoverySpecificationBuilder {
+        super::super::types::builders::PointInTimeRecoverySpecificationBuilder::default()
     }
 }

@@ -66,8 +66,8 @@
     /// Consumes the builder and constructs a [`PointInTimeRecoverySpecification`](crate::types::PointInTimeRecoverySpecification).
     /// This method will fail if any of the following fields are not set:
     /// - [`point_in_time_recovery_enabled`](crate::types::builders::PointInTimeRecoverySpecificationBuilder::point_in_time_recovery_enabled)
-    pub fn build(self) -> ::std::result::Result<crate::types::PointInTimeRecoverySpecification, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::PointInTimeRecoverySpecification {
+    pub fn build(self) -> ::std::result::Result<super::super::types::PointInTimeRecoverySpecification, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::PointInTimeRecoverySpecification {
             point_in_time_recovery_enabled: self.point_in_time_recovery_enabled.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "point_in_time_recovery_enabled",
