--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ProvisionedThroughput {
     /// Creates a new builder-style object to manufacture [`ProvisionedThroughput`](crate::types::ProvisionedThroughput).
-    pub fn builder() -> crate::types::builders::ProvisionedThroughputBuilder {
-        crate::types::builders::ProvisionedThroughputBuilder::default()
+    pub fn builder() -> super::types::builders::ProvisionedThroughputBuilder {
+        super::types::builders::ProvisionedThroughputBuilder::default()
     }
 }

@@ -79,8 +79,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`read_capacity_units`](crate::types::builders::ProvisionedThroughputBuilder::read_capacity_units)
     /// - [`write_capacity_units`](crate::types::builders::ProvisionedThroughputBuilder::write_capacity_units)
-    pub fn build(self) -> ::std::result::Result<crate::types::ProvisionedThroughput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ProvisionedThroughput {
+    pub fn build(self) -> ::std::result::Result<super::types::ProvisionedThroughput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ProvisionedThroughput {
             read_capacity_units: self.read_capacity_units.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "read_capacity_units",
