--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
     pub provisioned_read_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for managing a global secondary index replica's read capacity units.</p>
-    pub provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+    pub provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<super::types::AutoScalingSettingsUpdate>,
 }
 impl ReplicaGlobalSecondaryIndexSettingsUpdate {
     /// <p>The name of the global secondary index. The name must be unique among all other indexes on this table.</p>
@@ -22,14 +22,14 @@
         self.provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for managing a global secondary index replica's read capacity units.</p>
-    pub fn provisioned_read_capacity_auto_scaling_settings_update(&self) -> ::std::option::Option<&crate::types::AutoScalingSettingsUpdate> {
+    pub fn provisioned_read_capacity_auto_scaling_settings_update(&self) -> ::std::option::Option<&super::types::AutoScalingSettingsUpdate> {
         self.provisioned_read_capacity_auto_scaling_settings_update.as_ref()
     }
 }
 impl ReplicaGlobalSecondaryIndexSettingsUpdate {
     /// Creates a new builder-style object to manufacture [`ReplicaGlobalSecondaryIndexSettingsUpdate`](crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate).
-    pub fn builder() -> crate::types::builders::ReplicaGlobalSecondaryIndexSettingsUpdateBuilder {
-        crate::types::builders::ReplicaGlobalSecondaryIndexSettingsUpdateBuilder::default()
+    pub fn builder() -> super::types::builders::ReplicaGlobalSecondaryIndexSettingsUpdateBuilder {
+        super::types::builders::ReplicaGlobalSecondaryIndexSettingsUpdateBuilder::default()
     }
 }

@@ -39,7 +39,7 @@
 pub struct ReplicaGlobalSecondaryIndexSettingsUpdateBuilder {
     pub(crate) index_name: ::std::option::Option<::std::string::String>,
     pub(crate) provisioned_read_capacity_units: ::std::option::Option<i64>,
-    pub(crate) provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+    pub(crate) provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<super::types::AutoScalingSettingsUpdate>,
 }
 impl ReplicaGlobalSecondaryIndexSettingsUpdateBuilder {
     /// <p>The name of the global secondary index. The name must be unique among all other indexes on this table.</p>
@@ -72,7 +72,7 @@
         &self.provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for managing a global secondary index replica's read capacity units.</p>
-    pub fn provisioned_read_capacity_auto_scaling_settings_update(mut self, input: crate::types::AutoScalingSettingsUpdate) -> Self {
+    pub fn provisioned_read_capacity_auto_scaling_settings_update(mut self, input: super::types::AutoScalingSettingsUpdate) -> Self {
         self.provisioned_read_capacity_auto_scaling_settings_update = ::std::option::Option::Some(input);
         self
     }
@@ -79,13 +79,13 @@
     /// <p>Auto scaling settings for managing a global secondary index replica's read capacity units.</p>
     pub fn set_provisioned_read_capacity_auto_scaling_settings_update(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+        input: ::std::option::Option<super::types::AutoScalingSettingsUpdate>,
     ) -> Self {
         self.provisioned_read_capacity_auto_scaling_settings_update = input;
         self
     }
     /// <p>Auto scaling settings for managing a global secondary index replica's read capacity units.</p>
-    pub fn get_provisioned_read_capacity_auto_scaling_settings_update(&self) -> &::std::option::Option<crate::types::AutoScalingSettingsUpdate> {
+    pub fn get_provisioned_read_capacity_auto_scaling_settings_update(&self) -> &::std::option::Option<super::types::AutoScalingSettingsUpdate> {
         &self.provisioned_read_capacity_auto_scaling_settings_update
     }
     /// Consumes the builder and constructs a [`ReplicaGlobalSecondaryIndexSettingsUpdate`](crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate).
@@ -93,8 +93,8 @@
     /// - [`index_name`](crate::types::builders::ReplicaGlobalSecondaryIndexSettingsUpdateBuilder::index_name)
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate {
+    ) -> ::std::result::Result<super::types::ReplicaGlobalSecondaryIndexSettingsUpdate, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ReplicaGlobalSecondaryIndexSettingsUpdate {
             index_name: self.index_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "index_name",
