--- original
+++ modified
@@ -17,15 +17,15 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The global secondary index is ready for use.</p></li>
     /// </ul>
-    pub index_status: ::std::option::Option<crate::types::IndexStatus>,
+    pub index_status: ::std::option::Option<super::types::IndexStatus>,
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
     pub provisioned_read_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for a global secondary index replica's read capacity units.</p>
-    pub provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
     pub provisioned_write_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for a global secondary index replica's write capacity units.</p>
-    pub provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
 }
 impl ReplicaGlobalSecondaryIndexSettingsDescription {
     /// <p>The name of the global secondary index. The name must be unique among all other indexes on this table.</p>
@@ -44,7 +44,7 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The global secondary index is ready for use.</p></li>
     /// </ul>
-    pub fn index_status(&self) -> ::std::option::Option<&crate::types::IndexStatus> {
+    pub fn index_status(&self) -> ::std::option::Option<&super::types::IndexStatus> {
         self.index_status.as_ref()
     }
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
@@ -52,7 +52,7 @@
         self.provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for a global secondary index replica's read capacity units.</p>
-    pub fn provisioned_read_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&crate::types::AutoScalingSettingsDescription> {
+    pub fn provisioned_read_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&super::types::AutoScalingSettingsDescription> {
         self.provisioned_read_capacity_auto_scaling_settings.as_ref()
     }
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
@@ -60,14 +60,14 @@
         self.provisioned_write_capacity_units
     }
     /// <p>Auto scaling settings for a global secondary index replica's write capacity units.</p>
-    pub fn provisioned_write_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&crate::types::AutoScalingSettingsDescription> {
+    pub fn provisioned_write_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&super::types::AutoScalingSettingsDescription> {
         self.provisioned_write_capacity_auto_scaling_settings.as_ref()
     }
 }
 impl ReplicaGlobalSecondaryIndexSettingsDescription {
     /// Creates a new builder-style object to manufacture [`ReplicaGlobalSecondaryIndexSettingsDescription`](crate::types::ReplicaGlobalSecondaryIndexSettingsDescription).
-    pub fn builder() -> crate::types::builders::ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder {
-        crate::types::builders::ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder::default()
+    pub fn builder() -> super::types::builders::ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder {
+        super::types::builders::ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder::default()
     }
 }

@@ -76,11 +76,11 @@
 #[non_exhaustive]
 pub struct ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder {
     pub(crate) index_name: ::std::option::Option<::std::string::String>,
-    pub(crate) index_status: ::std::option::Option<crate::types::IndexStatus>,
+    pub(crate) index_status: ::std::option::Option<super::types::IndexStatus>,
     pub(crate) provisioned_read_capacity_units: ::std::option::Option<i64>,
-    pub(crate) provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub(crate) provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     pub(crate) provisioned_write_capacity_units: ::std::option::Option<i64>,
-    pub(crate) provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub(crate) provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
 }
 impl ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder {
     /// <p>The name of the global secondary index. The name must be unique among all other indexes on this table.</p>
@@ -109,7 +109,7 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The global secondary index is ready for use.</p></li>
     /// </ul>
-    pub fn index_status(mut self, input: crate::types::IndexStatus) -> Self {
+    pub fn index_status(mut self, input: super::types::IndexStatus) -> Self {
         self.index_status = ::std::option::Option::Some(input);
         self
     }
@@ -124,7 +124,7 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The global secondary index is ready for use.</p></li>
     /// </ul>
-    pub fn set_index_status(mut self, input: ::std::option::Option<crate::types::IndexStatus>) -> Self {
+    pub fn set_index_status(mut self, input: ::std::option::Option<super::types::IndexStatus>) -> Self {
         self.index_status = input;
         self
     }
@@ -139,7 +139,7 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The global secondary index is ready for use.</p></li>
     /// </ul>
-    pub fn get_index_status(&self) -> &::std::option::Option<crate::types::IndexStatus> {
+    pub fn get_index_status(&self) -> &::std::option::Option<super::types::IndexStatus> {
         &self.index_status
     }
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
@@ -157,7 +157,7 @@
         &self.provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for a global secondary index replica's read capacity units.</p>
-    pub fn provisioned_read_capacity_auto_scaling_settings(mut self, input: crate::types::AutoScalingSettingsDescription) -> Self {
+    pub fn provisioned_read_capacity_auto_scaling_settings(mut self, input: super::types::AutoScalingSettingsDescription) -> Self {
         self.provisioned_read_capacity_auto_scaling_settings = ::std::option::Option::Some(input);
         self
     }
@@ -164,13 +164,13 @@
     /// <p>Auto scaling settings for a global secondary index replica's read capacity units.</p>
     pub fn set_provisioned_read_capacity_auto_scaling_settings(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+        input: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     ) -> Self {
         self.provisioned_read_capacity_auto_scaling_settings = input;
         self
     }
     /// <p>Auto scaling settings for a global secondary index replica's read capacity units.</p>
-    pub fn get_provisioned_read_capacity_auto_scaling_settings(&self) -> &::std::option::Option<crate::types::AutoScalingSettingsDescription> {
+    pub fn get_provisioned_read_capacity_auto_scaling_settings(&self) -> &::std::option::Option<super::types::AutoScalingSettingsDescription> {
         &self.provisioned_read_capacity_auto_scaling_settings
     }
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException</code>.</p>
@@ -188,7 +188,7 @@
         &self.provisioned_write_capacity_units
     }
     /// <p>Auto scaling settings for a global secondary index replica's write capacity units.</p>
-    pub fn provisioned_write_capacity_auto_scaling_settings(mut self, input: crate::types::AutoScalingSettingsDescription) -> Self {
+    pub fn provisioned_write_capacity_auto_scaling_settings(mut self, input: super::types::AutoScalingSettingsDescription) -> Self {
         self.provisioned_write_capacity_auto_scaling_settings = ::std::option::Option::Some(input);
         self
     }
@@ -195,13 +195,13 @@
     /// <p>Auto scaling settings for a global secondary index replica's write capacity units.</p>
     pub fn set_provisioned_write_capacity_auto_scaling_settings(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+        input: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     ) -> Self {
         self.provisioned_write_capacity_auto_scaling_settings = input;
         self
     }
     /// <p>Auto scaling settings for a global secondary index replica's write capacity units.</p>
-    pub fn get_provisioned_write_capacity_auto_scaling_settings(&self) -> &::std::option::Option<crate::types::AutoScalingSettingsDescription> {
+    pub fn get_provisioned_write_capacity_auto_scaling_settings(&self) -> &::std::option::Option<super::types::AutoScalingSettingsDescription> {
         &self.provisioned_write_capacity_auto_scaling_settings
     }
     /// Consumes the builder and constructs a [`ReplicaGlobalSecondaryIndexSettingsDescription`](crate::types::ReplicaGlobalSecondaryIndexSettingsDescription).
@@ -209,8 +209,8 @@
     /// - [`index_name`](crate::types::builders::ReplicaGlobalSecondaryIndexSettingsDescriptionBuilder::index_name)
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::types::ReplicaGlobalSecondaryIndexSettingsDescription, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ReplicaGlobalSecondaryIndexSettingsDescription {
+    ) -> ::std::result::Result<super::types::ReplicaGlobalSecondaryIndexSettingsDescription, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ReplicaGlobalSecondaryIndexSettingsDescription {
             index_name: self.index_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "index_name",
