--- original
+++ modified
@@ -9,12 +9,12 @@
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
     pub replica_provisioned_read_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for managing a global table replica's read capacity units.</p>
-    pub replica_provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+    pub replica_provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<super::super::types::AutoScalingSettingsUpdate>,
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     pub replica_global_secondary_index_settings_update:
-        ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate>>,
+        ::std::option::Option<::std::vec::Vec<super::super::types::ReplicaGlobalSecondaryIndexSettingsUpdate>>,
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub replica_table_class: ::std::option::Option<crate::types::TableClass>,
+    pub replica_table_class: ::std::option::Option<super::super::types::TableClass>,
 }
 impl ReplicaSettingsUpdate {
     /// <p>The Region of the replica to be added.</p>
@@ -27,24 +27,24 @@
         self.replica_provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for managing a global table replica's read capacity units.</p>
-    pub fn replica_provisioned_read_capacity_auto_scaling_settings_update(&self) -> ::std::option::Option<&crate::types::AutoScalingSettingsUpdate> {
+    pub fn replica_provisioned_read_capacity_auto_scaling_settings_update(&self) -> ::std::option::Option<&super::super::types::AutoScalingSettingsUpdate> {
         self.replica_provisioned_read_capacity_auto_scaling_settings_update.as_ref()
     }
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replica_global_secondary_index_settings_update.is_none()`.
-    pub fn replica_global_secondary_index_settings_update(&self) -> &[crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate] {
+    pub fn replica_global_secondary_index_settings_update(&self) -> &[super::super::types::ReplicaGlobalSecondaryIndexSettingsUpdate] {
         self.replica_global_secondary_index_settings_update.as_deref().unwrap_or_default()
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn replica_table_class(&self) -> ::std::option::Option<&crate::types::TableClass> {
+    pub fn replica_table_class(&self) -> ::std::option::Option<&super::super::types::TableClass> {
         self.replica_table_class.as_ref()
     }
 }
 impl ReplicaSettingsUpdate {
     /// Creates a new builder-style object to manufacture [`ReplicaSettingsUpdate`](crate::types::ReplicaSettingsUpdate).
-    pub fn builder() -> crate::types::builders::ReplicaSettingsUpdateBuilder {
-        crate::types::builders::ReplicaSettingsUpdateBuilder::default()
+    pub fn builder() -> super::super::types::builders::ReplicaSettingsUpdateBuilder {
+        super::super::types::builders::ReplicaSettingsUpdateBuilder::default()
     }
 }

@@ -54,10 +54,10 @@
 pub struct ReplicaSettingsUpdateBuilder {
     pub(crate) region_name: ::std::option::Option<::std::string::String>,
     pub(crate) replica_provisioned_read_capacity_units: ::std::option::Option<i64>,
-    pub(crate) replica_provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+    pub(crate) replica_provisioned_read_capacity_auto_scaling_settings_update: ::std::option::Option<super::super::types::AutoScalingSettingsUpdate>,
     pub(crate) replica_global_secondary_index_settings_update:
-        ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate>>,
-    pub(crate) replica_table_class: ::std::option::Option<crate::types::TableClass>,
+        ::std::option::Option<::std::vec::Vec<super::super::types::ReplicaGlobalSecondaryIndexSettingsUpdate>>,
+    pub(crate) replica_table_class: ::std::option::Option<super::super::types::TableClass>,
 }
 impl ReplicaSettingsUpdateBuilder {
     /// <p>The Region of the replica to be added.</p>
@@ -90,7 +90,7 @@
         &self.replica_provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for managing a global table replica's read capacity units.</p>
-    pub fn replica_provisioned_read_capacity_auto_scaling_settings_update(mut self, input: crate::types::AutoScalingSettingsUpdate) -> Self {
+    pub fn replica_provisioned_read_capacity_auto_scaling_settings_update(mut self, input: super::super::types::AutoScalingSettingsUpdate) -> Self {
         self.replica_provisioned_read_capacity_auto_scaling_settings_update = ::std::option::Option::Some(input);
         self
     }
@@ -97,7 +97,7 @@
     /// <p>Auto scaling settings for managing a global table replica's read capacity units.</p>
     pub fn set_replica_provisioned_read_capacity_auto_scaling_settings_update(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+        input: ::std::option::Option<super::super::types::AutoScalingSettingsUpdate>,
     ) -> Self {
         self.replica_provisioned_read_capacity_auto_scaling_settings_update = input;
         self
@@ -105,7 +105,7 @@
     /// <p>Auto scaling settings for managing a global table replica's read capacity units.</p>
     pub fn get_replica_provisioned_read_capacity_auto_scaling_settings_update(
         &self,
-    ) -> &::std::option::Option<crate::types::AutoScalingSettingsUpdate> {
+    ) -> &::std::option::Option<super::super::types::AutoScalingSettingsUpdate> {
         &self.replica_provisioned_read_capacity_auto_scaling_settings_update
     }
     /// Appends an item to `replica_global_secondary_index_settings_update`.
@@ -113,7 +113,7 @@
     /// To override the contents of this collection use [`set_replica_global_secondary_index_settings_update`](Self::set_replica_global_secondary_index_settings_update).
     ///
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
-    pub fn replica_global_secondary_index_settings_update(mut self, input: crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate) -> Self {
+    pub fn replica_global_secondary_index_settings_update(mut self, input: super::super::types::ReplicaGlobalSecondaryIndexSettingsUpdate) -> Self {
         let mut v = self.replica_global_secondary_index_settings_update.unwrap_or_default();
         v.push(input);
         self.replica_global_secondary_index_settings_update = ::std::option::Option::Some(v);
@@ -122,7 +122,7 @@
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     pub fn set_replica_global_secondary_index_settings_update(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::types::ReplicaGlobalSecondaryIndexSettingsUpdate>>,
     ) -> Self {
         self.replica_global_secondary_index_settings_update = input;
         self
@@ -130,28 +130,28 @@
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     pub fn get_replica_global_secondary_index_settings_update(
         &self,
-    ) -> &::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsUpdate>> {
+    ) -> &::std::option::Option<::std::vec::Vec<super::super::types::ReplicaGlobalSecondaryIndexSettingsUpdate>> {
         &self.replica_global_secondary_index_settings_update
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn replica_table_class(mut self, input: crate::types::TableClass) -> Self {
+    pub fn replica_table_class(mut self, input: super::super::types::TableClass) -> Self {
         self.replica_table_class = ::std::option::Option::Some(input);
         self
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn set_replica_table_class(mut self, input: ::std::option::Option<crate::types::TableClass>) -> Self {
+    pub fn set_replica_table_class(mut self, input: ::std::option::Option<super::super::types::TableClass>) -> Self {
         self.replica_table_class = input;
         self
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn get_replica_table_class(&self) -> &::std::option::Option<crate::types::TableClass> {
+    pub fn get_replica_table_class(&self) -> &::std::option::Option<super::super::types::TableClass> {
         &self.replica_table_class
     }
     /// Consumes the builder and constructs a [`ReplicaSettingsUpdate`](crate::types::ReplicaSettingsUpdate).
     /// This method will fail if any of the following fields are not set:
     /// - [`region_name`](crate::types::builders::ReplicaSettingsUpdateBuilder::region_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::ReplicaSettingsUpdate, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ReplicaSettingsUpdate {
+    pub fn build(self) -> ::std::result::Result<super::super::types::ReplicaSettingsUpdate, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::ReplicaSettingsUpdate {
             region_name: self.region_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "region_name",
