--- original
+++ modified
@@ -17,21 +17,21 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The Region is ready for use.</p></li>
     /// </ul>
-    pub replica_status: ::std::option::Option<crate::types::ReplicaStatus>,
+    pub replica_status: ::std::option::Option<super::types::ReplicaStatus>,
     /// <p>The read/write capacity mode of the replica.</p>
-    pub replica_billing_mode_summary: ::std::option::Option<crate::types::BillingModeSummary>,
+    pub replica_billing_mode_summary: ::std::option::Option<super::types::BillingModeSummary>,
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
     pub replica_provisioned_read_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for a global table replica's read capacity units.</p>
-    pub replica_provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub replica_provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
     pub replica_provisioned_write_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for a global table replica's write capacity units.</p>
-    pub replica_provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub replica_provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     /// <p>Replica global secondary index settings for the global table.</p>
-    pub replica_global_secondary_index_settings: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsDescription>>,
+    pub replica_global_secondary_index_settings: ::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndexSettingsDescription>>,
     /// <p>Contains details of the table class.</p>
-    pub replica_table_class_summary: ::std::option::Option<crate::types::TableClassSummary>,
+    pub replica_table_class_summary: ::std::option::Option<super::types::TableClassSummary>,
 }
 impl ReplicaSettingsDescription {
     /// <p>The Region name of the replica.</p>
@@ -50,11 +50,11 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The Region is ready for use.</p></li>
     /// </ul>
-    pub fn replica_status(&self) -> ::std::option::Option<&crate::types::ReplicaStatus> {
+    pub fn replica_status(&self) -> ::std::option::Option<&super::types::ReplicaStatus> {
         self.replica_status.as_ref()
     }
     /// <p>The read/write capacity mode of the replica.</p>
-    pub fn replica_billing_mode_summary(&self) -> ::std::option::Option<&crate::types::BillingModeSummary> {
+    pub fn replica_billing_mode_summary(&self) -> ::std::option::Option<&super::types::BillingModeSummary> {
         self.replica_billing_mode_summary.as_ref()
     }
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
@@ -62,7 +62,7 @@
         self.replica_provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for a global table replica's read capacity units.</p>
-    pub fn replica_provisioned_read_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&crate::types::AutoScalingSettingsDescription> {
+    pub fn replica_provisioned_read_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&super::types::AutoScalingSettingsDescription> {
         self.replica_provisioned_read_capacity_auto_scaling_settings.as_ref()
     }
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
@@ -70,24 +70,24 @@
         self.replica_provisioned_write_capacity_units
     }
     /// <p>Auto scaling settings for a global table replica's write capacity units.</p>
-    pub fn replica_provisioned_write_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&crate::types::AutoScalingSettingsDescription> {
+    pub fn replica_provisioned_write_capacity_auto_scaling_settings(&self) -> ::std::option::Option<&super::types::AutoScalingSettingsDescription> {
         self.replica_provisioned_write_capacity_auto_scaling_settings.as_ref()
     }
     /// <p>Replica global secondary index settings for the global table.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replica_global_secondary_index_settings.is_none()`.
-    pub fn replica_global_secondary_index_settings(&self) -> &[crate::types::ReplicaGlobalSecondaryIndexSettingsDescription] {
+    pub fn replica_global_secondary_index_settings(&self) -> &[super::types::ReplicaGlobalSecondaryIndexSettingsDescription] {
         self.replica_global_secondary_index_settings.as_deref().unwrap_or_default()
     }
     /// <p>Contains details of the table class.</p>
-    pub fn replica_table_class_summary(&self) -> ::std::option::Option<&crate::types::TableClassSummary> {
+    pub fn replica_table_class_summary(&self) -> ::std::option::Option<&super::types::TableClassSummary> {
         self.replica_table_class_summary.as_ref()
     }
 }
 impl ReplicaSettingsDescription {
     /// Creates a new builder-style object to manufacture [`ReplicaSettingsDescription`](crate::types::ReplicaSettingsDescription).
-    pub fn builder() -> crate::types::builders::ReplicaSettingsDescriptionBuilder {
-        crate::types::builders::ReplicaSettingsDescriptionBuilder::default()
+    pub fn builder() -> super::types::builders::ReplicaSettingsDescriptionBuilder {
+        super::types::builders::ReplicaSettingsDescriptionBuilder::default()
     }
 }

@@ -96,15 +96,15 @@
 #[non_exhaustive]
 pub struct ReplicaSettingsDescriptionBuilder {
     pub(crate) region_name: ::std::option::Option<::std::string::String>,
-    pub(crate) replica_status: ::std::option::Option<crate::types::ReplicaStatus>,
-    pub(crate) replica_billing_mode_summary: ::std::option::Option<crate::types::BillingModeSummary>,
+    pub(crate) replica_status: ::std::option::Option<super::types::ReplicaStatus>,
+    pub(crate) replica_billing_mode_summary: ::std::option::Option<super::types::BillingModeSummary>,
     pub(crate) replica_provisioned_read_capacity_units: ::std::option::Option<i64>,
-    pub(crate) replica_provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub(crate) replica_provisioned_read_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     pub(crate) replica_provisioned_write_capacity_units: ::std::option::Option<i64>,
-    pub(crate) replica_provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+    pub(crate) replica_provisioned_write_capacity_auto_scaling_settings: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     pub(crate) replica_global_secondary_index_settings:
-        ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsDescription>>,
-    pub(crate) replica_table_class_summary: ::std::option::Option<crate::types::TableClassSummary>,
+        ::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndexSettingsDescription>>,
+    pub(crate) replica_table_class_summary: ::std::option::Option<super::types::TableClassSummary>,
 }
 impl ReplicaSettingsDescriptionBuilder {
     /// <p>The Region name of the replica.</p>
@@ -133,7 +133,7 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The Region is ready for use.</p></li>
     /// </ul>
-    pub fn replica_status(mut self, input: crate::types::ReplicaStatus) -> Self {
+    pub fn replica_status(mut self, input: super::types::ReplicaStatus) -> Self {
         self.replica_status = ::std::option::Option::Some(input);
         self
     }
@@ -148,7 +148,7 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The Region is ready for use.</p></li>
     /// </ul>
-    pub fn set_replica_status(mut self, input: ::std::option::Option<crate::types::ReplicaStatus>) -> Self {
+    pub fn set_replica_status(mut self, input: ::std::option::Option<super::types::ReplicaStatus>) -> Self {
         self.replica_status = input;
         self
     }
@@ -163,21 +163,21 @@
     /// <li>
     /// <p><code>ACTIVE</code> - The Region is ready for use.</p></li>
     /// </ul>
-    pub fn get_replica_status(&self) -> &::std::option::Option<crate::types::ReplicaStatus> {
+    pub fn get_replica_status(&self) -> &::std::option::Option<super::types::ReplicaStatus> {
         &self.replica_status
     }
     /// <p>The read/write capacity mode of the replica.</p>
-    pub fn replica_billing_mode_summary(mut self, input: crate::types::BillingModeSummary) -> Self {
+    pub fn replica_billing_mode_summary(mut self, input: super::types::BillingModeSummary) -> Self {
         self.replica_billing_mode_summary = ::std::option::Option::Some(input);
         self
     }
     /// <p>The read/write capacity mode of the replica.</p>
-    pub fn set_replica_billing_mode_summary(mut self, input: ::std::option::Option<crate::types::BillingModeSummary>) -> Self {
+    pub fn set_replica_billing_mode_summary(mut self, input: ::std::option::Option<super::types::BillingModeSummary>) -> Self {
         self.replica_billing_mode_summary = input;
         self
     }
     /// <p>The read/write capacity mode of the replica.</p>
-    pub fn get_replica_billing_mode_summary(&self) -> &::std::option::Option<crate::types::BillingModeSummary> {
+    pub fn get_replica_billing_mode_summary(&self) -> &::std::option::Option<super::types::BillingModeSummary> {
         &self.replica_billing_mode_summary
     }
     /// <p>The maximum number of strongly consistent reads consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
@@ -195,7 +195,7 @@
         &self.replica_provisioned_read_capacity_units
     }
     /// <p>Auto scaling settings for a global table replica's read capacity units.</p>
-    pub fn replica_provisioned_read_capacity_auto_scaling_settings(mut self, input: crate::types::AutoScalingSettingsDescription) -> Self {
+    pub fn replica_provisioned_read_capacity_auto_scaling_settings(mut self, input: super::types::AutoScalingSettingsDescription) -> Self {
         self.replica_provisioned_read_capacity_auto_scaling_settings = ::std::option::Option::Some(input);
         self
     }
@@ -202,7 +202,7 @@
     /// <p>Auto scaling settings for a global table replica's read capacity units.</p>
     pub fn set_replica_provisioned_read_capacity_auto_scaling_settings(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+        input: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     ) -> Self {
         self.replica_provisioned_read_capacity_auto_scaling_settings = input;
         self
@@ -210,7 +210,7 @@
     /// <p>Auto scaling settings for a global table replica's read capacity units.</p>
     pub fn get_replica_provisioned_read_capacity_auto_scaling_settings(
         &self,
-    ) -> &::std::option::Option<crate::types::AutoScalingSettingsDescription> {
+    ) -> &::std::option::Option<super::types::AutoScalingSettingsDescription> {
         &self.replica_provisioned_read_capacity_auto_scaling_settings
     }
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException</code>. For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithTables.html#ProvisionedThroughput">Specifying Read and Write Requirements</a> in the <i>Amazon DynamoDB Developer Guide</i>.</p>
@@ -228,7 +228,7 @@
         &self.replica_provisioned_write_capacity_units
     }
     /// <p>Auto scaling settings for a global table replica's write capacity units.</p>
-    pub fn replica_provisioned_write_capacity_auto_scaling_settings(mut self, input: crate::types::AutoScalingSettingsDescription) -> Self {
+    pub fn replica_provisioned_write_capacity_auto_scaling_settings(mut self, input: super::types::AutoScalingSettingsDescription) -> Self {
         self.replica_provisioned_write_capacity_auto_scaling_settings = ::std::option::Option::Some(input);
         self
     }
@@ -235,7 +235,7 @@
     /// <p>Auto scaling settings for a global table replica's write capacity units.</p>
     pub fn set_replica_provisioned_write_capacity_auto_scaling_settings(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsDescription>,
+        input: ::std::option::Option<super::types::AutoScalingSettingsDescription>,
     ) -> Self {
         self.replica_provisioned_write_capacity_auto_scaling_settings = input;
         self
@@ -243,7 +243,7 @@
     /// <p>Auto scaling settings for a global table replica's write capacity units.</p>
     pub fn get_replica_provisioned_write_capacity_auto_scaling_settings(
         &self,
-    ) -> &::std::option::Option<crate::types::AutoScalingSettingsDescription> {
+    ) -> &::std::option::Option<super::types::AutoScalingSettingsDescription> {
         &self.replica_provisioned_write_capacity_auto_scaling_settings
     }
     /// Appends an item to `replica_global_secondary_index_settings`.
@@ -251,7 +251,7 @@
     /// To override the contents of this collection use [`set_replica_global_secondary_index_settings`](Self::set_replica_global_secondary_index_settings).
     ///
     /// <p>Replica global secondary index settings for the global table.</p>
-    pub fn replica_global_secondary_index_settings(mut self, input: crate::types::ReplicaGlobalSecondaryIndexSettingsDescription) -> Self {
+    pub fn replica_global_secondary_index_settings(mut self, input: super::types::ReplicaGlobalSecondaryIndexSettingsDescription) -> Self {
         let mut v = self.replica_global_secondary_index_settings.unwrap_or_default();
         v.push(input);
         self.replica_global_secondary_index_settings = ::std::option::Option::Some(v);
@@ -260,7 +260,7 @@
     /// <p>Replica global secondary index settings for the global table.</p>
     pub fn set_replica_global_secondary_index_settings(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsDescription>>,
+        input: ::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndexSettingsDescription>>,
     ) -> Self {
         self.replica_global_secondary_index_settings = input;
         self
@@ -268,28 +268,28 @@
     /// <p>Replica global secondary index settings for the global table.</p>
     pub fn get_replica_global_secondary_index_settings(
         &self,
-    ) -> &::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndexSettingsDescription>> {
+    ) -> &::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndexSettingsDescription>> {
         &self.replica_global_secondary_index_settings
     }
     /// <p>Contains details of the table class.</p>
-    pub fn replica_table_class_summary(mut self, input: crate::types::TableClassSummary) -> Self {
+    pub fn replica_table_class_summary(mut self, input: super::types::TableClassSummary) -> Self {
         self.replica_table_class_summary = ::std::option::Option::Some(input);
         self
     }
     /// <p>Contains details of the table class.</p>
-    pub fn set_replica_table_class_summary(mut self, input: ::std::option::Option<crate::types::TableClassSummary>) -> Self {
+    pub fn set_replica_table_class_summary(mut self, input: ::std::option::Option<super::types::TableClassSummary>) -> Self {
         self.replica_table_class_summary = input;
         self
     }
     /// <p>Contains details of the table class.</p>
-    pub fn get_replica_table_class_summary(&self) -> &::std::option::Option<crate::types::TableClassSummary> {
+    pub fn get_replica_table_class_summary(&self) -> &::std::option::Option<super::types::TableClassSummary> {
         &self.replica_table_class_summary
     }
     /// Consumes the builder and constructs a [`ReplicaSettingsDescription`](crate::types::ReplicaSettingsDescription).
     /// This method will fail if any of the following fields are not set:
     /// - [`region_name`](crate::types::builders::ReplicaSettingsDescriptionBuilder::region_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::ReplicaSettingsDescription, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ReplicaSettingsDescription {
+    pub fn build(self) -> ::std::result::Result<super::types::ReplicaSettingsDescription, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ReplicaSettingsDescription {
             region_name: self.region_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "region_name",
