--- original
+++ modified
@@ -33,8 +33,8 @@
 }
 impl RestoreSummary {
     /// Creates a new builder-style object to manufacture [`RestoreSummary`](crate::types::RestoreSummary).
-    pub fn builder() -> crate::types::builders::RestoreSummaryBuilder {
-        crate::types::builders::RestoreSummaryBuilder::default()
+    pub fn builder() -> super::super::types::builders::RestoreSummaryBuilder {
+        super::super::types::builders::RestoreSummaryBuilder::default()
     }
 }

@@ -110,8 +110,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`restore_date_time`](crate::types::builders::RestoreSummaryBuilder::restore_date_time)
     /// - [`restore_in_progress`](crate::types::builders::RestoreSummaryBuilder::restore_in_progress)
-    pub fn build(self) -> ::std::result::Result<crate::types::RestoreSummary, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::RestoreSummary {
+    pub fn build(self) -> ::std::result::Result<super::super::types::RestoreSummary, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::RestoreSummary {
             source_backup_arn: self.source_backup_arn,
             source_table_arn: self.source_table_arn,
             restore_date_time: self.restore_date_time.ok_or_else(|| {
