--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl S3BucketSource {
     /// Creates a new builder-style object to manufacture [`S3BucketSource`](crate::types::S3BucketSource).
-    pub fn builder() -> crate::types::builders::S3BucketSourceBuilder {
-        crate::types::builders::S3BucketSourceBuilder::default()
+    pub fn builder() -> super::types::builders::S3BucketSourceBuilder {
+        super::types::builders::S3BucketSourceBuilder::default()
     }
 }

@@ -88,8 +88,8 @@
     /// Consumes the builder and constructs a [`S3BucketSource`](crate::types::S3BucketSource).
     /// This method will fail if any of the following fields are not set:
     /// - [`s3_bucket`](crate::types::builders::S3BucketSourceBuilder::s3_bucket)
-    pub fn build(self) -> ::std::result::Result<crate::types::S3BucketSource, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::S3BucketSource {
+    pub fn build(self) -> ::std::result::Result<super::types::S3BucketSource, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::S3BucketSource {
             s3_bucket_owner: self.s3_bucket_owner,
             s3_bucket: self.s3_bucket.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
