--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct TableClassSummary {
     /// <p>The table class of the specified table. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub table_class: ::std::option::Option<crate::types::TableClass>,
+    pub table_class: ::std::option::Option<super::super::types::TableClass>,
     /// <p>The date and time at which the table class was last updated.</p>
     pub last_update_date_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl TableClassSummary {
     /// <p>The table class of the specified table. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn table_class(&self) -> ::std::option::Option<&crate::types::TableClass> {
+    pub fn table_class(&self) -> ::std::option::Option<&super::super::types::TableClass> {
         self.table_class.as_ref()
     }
     /// <p>The date and time at which the table class was last updated.</p>
@@ -21,8 +21,8 @@
 }
 impl TableClassSummary {
     /// Creates a new builder-style object to manufacture [`TableClassSummary`](crate::types::TableClassSummary).
-    pub fn builder() -> crate::types::builders::TableClassSummaryBuilder {
-        crate::types::builders::TableClassSummaryBuilder::default()
+    pub fn builder() -> super::super::types::builders::TableClassSummaryBuilder {
+        super::super::types::builders::TableClassSummaryBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct TableClassSummaryBuilder {
-    pub(crate) table_class: ::std::option::Option<crate::types::TableClass>,
+    pub(crate) table_class: ::std::option::Option<super::super::types::TableClass>,
     pub(crate) last_update_date_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl TableClassSummaryBuilder {
     /// <p>The table class of the specified table. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn table_class(mut self, input: crate::types::TableClass) -> Self {
+    pub fn table_class(mut self, input: super::super::types::TableClass) -> Self {
         self.table_class = ::std::option::Option::Some(input);
         self
     }
     /// <p>The table class of the specified table. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn set_table_class(mut self, input: ::std::option::Option<crate::types::TableClass>) -> Self {
+    pub fn set_table_class(mut self, input: ::std::option::Option<super::super::types::TableClass>) -> Self {
         self.table_class = input;
         self
     }
     /// <p>The table class of the specified table. Valid values are <code>STANDARD</code> and <code>STANDARD_INFREQUENT_ACCESS</code>.</p>
-    pub fn get_table_class(&self) -> &::std::option::Option<crate::types::TableClass> {
+    pub fn get_table_class(&self) -> &::std::option::Option<super::super::types::TableClass> {
         &self.table_class
     }
     /// <p>The date and time at which the table class was last updated.</p>
@@ -63,8 +63,8 @@
         &self.last_update_date_time
     }
     /// Consumes the builder and constructs a [`TableClassSummary`](crate::types::TableClassSummary).
-    pub fn build(self) -> crate::types::TableClassSummary {
-        crate::types::TableClassSummary {
+    pub fn build(self) -> super::super::types::TableClassSummary {
+        super::super::types::TableClassSummary {
             table_class: self.table_class,
             last_update_date_time: self.last_update_date_time,
         }
