--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>Represents the base table's warm throughput value in write units per second.</p>
     pub write_units_per_second: ::std::option::Option<i64>,
     /// <p>Represents warm throughput value of the base table.</p>
-    pub status: ::std::option::Option<crate::types::TableStatus>,
+    pub status: ::std::option::Option<super::super::types::TableStatus>,
 }
 impl TableWarmThroughputDescription {
     /// <p>Represents the base table's warm throughput value in read units per second.</p>
@@ -21,14 +21,14 @@
         self.write_units_per_second
     }
     /// <p>Represents warm throughput value of the base table.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::TableStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::TableStatus> {
         self.status.as_ref()
     }
 }
 impl TableWarmThroughputDescription {
     /// Creates a new builder-style object to manufacture [`TableWarmThroughputDescription`](crate::types::TableWarmThroughputDescription).
-    pub fn builder() -> crate::types::builders::TableWarmThroughputDescriptionBuilder {
-        crate::types::builders::TableWarmThroughputDescriptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::TableWarmThroughputDescriptionBuilder {
+        super::super::types::builders::TableWarmThroughputDescriptionBuilder::default()
     }
 }

@@ -38,7 +38,7 @@
 pub struct TableWarmThroughputDescriptionBuilder {
     pub(crate) read_units_per_second: ::std::option::Option<i64>,
     pub(crate) write_units_per_second: ::std::option::Option<i64>,
-    pub(crate) status: ::std::option::Option<crate::types::TableStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::TableStatus>,
 }
 impl TableWarmThroughputDescriptionBuilder {
     /// <p>Represents the base table's warm throughput value in read units per second.</p>
@@ -70,22 +70,22 @@
         &self.write_units_per_second
     }
     /// <p>Represents warm throughput value of the base table.</p>
-    pub fn status(mut self, input: crate::types::TableStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::TableStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Represents warm throughput value of the base table.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::TableStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::TableStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>Represents warm throughput value of the base table.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::TableStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::TableStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`TableWarmThroughputDescription`](crate::types::TableWarmThroughputDescription).
-    pub fn build(self) -> crate::types::TableWarmThroughputDescription {
-        crate::types::TableWarmThroughputDescription {
+    pub fn build(self) -> super::super::types::TableWarmThroughputDescription {
+        super::super::types::TableWarmThroughputDescription {
             read_units_per_second: self.read_units_per_second,
             write_units_per_second: self.write_units_per_second,
             status: self.status,
