--- original
+++ modified
@@ -73,8 +73,8 @@
 }
 impl ThrottlingReason {
     /// Creates a new builder-style object to manufacture [`ThrottlingReason`](crate::types::ThrottlingReason).
-    pub fn builder() -> crate::types::builders::ThrottlingReasonBuilder {
-        crate::types::builders::ThrottlingReasonBuilder::default()
+    pub fn builder() -> super::types::builders::ThrottlingReasonBuilder {
+        super::types::builders::ThrottlingReasonBuilder::default()
     }
 }

@@ -193,8 +193,8 @@
         &self.resource
     }
     /// Consumes the builder and constructs a [`ThrottlingReason`](crate::types::ThrottlingReason).
-    pub fn build(self) -> crate::types::ThrottlingReason {
-        crate::types::ThrottlingReason {
+    pub fn build(self) -> super::types::ThrottlingReason {
+        super::types::ThrottlingReason {
             reason: self.reason,
             resource: self.resource,
         }
