--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl TimeToLiveSpecification {
     /// Creates a new builder-style object to manufacture [`TimeToLiveSpecification`](crate::types::TimeToLiveSpecification).
-    pub fn builder() -> crate::types::builders::TimeToLiveSpecificationBuilder {
-        crate::types::builders::TimeToLiveSpecificationBuilder::default()
+    pub fn builder() -> super::types::builders::TimeToLiveSpecificationBuilder {
+        super::types::builders::TimeToLiveSpecificationBuilder::default()
     }
 }

@@ -69,8 +69,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`enabled`](crate::types::builders::TimeToLiveSpecificationBuilder::enabled)
     /// - [`attribute_name`](crate::types::builders::TimeToLiveSpecificationBuilder::attribute_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::TimeToLiveSpecification, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::TimeToLiveSpecification {
+    pub fn build(self) -> ::std::result::Result<super::types::TimeToLiveSpecification, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::TimeToLiveSpecification {
             enabled: self.enabled.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "enabled",
