--- original
+++ modified
@@ -9,13 +9,13 @@
     /// <p>The KMS key of the replica that should be used for KMS encryption. To specify a key, use its key ID, Amazon Resource Name (ARN), alias name, or alias ARN. Note that you should only provide this parameter if the key is different from the default DynamoDB KMS key <code>alias/aws/dynamodb</code>.</p>
     pub kms_master_key_id: ::std::option::Option<::std::string::String>,
     /// <p>Replica-specific provisioned throughput. If not specified, uses the source table's provisioned throughput settings.</p>
-    pub provisioned_throughput_override: ::std::option::Option<crate::types::ProvisionedThroughputOverride>,
+    pub provisioned_throughput_override: ::std::option::Option<super::types::ProvisionedThroughputOverride>,
     /// <p>Overrides the maximum on-demand throughput for the replica table.</p>
-    pub on_demand_throughput_override: ::std::option::Option<crate::types::OnDemandThroughputOverride>,
+    pub on_demand_throughput_override: ::std::option::Option<super::types::OnDemandThroughputOverride>,
     /// <p>Replica-specific global secondary index settings.</p>
-    pub global_secondary_indexes: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndex>>,
+    pub global_secondary_indexes: ::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndex>>,
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub table_class_override: ::std::option::Option<crate::types::TableClass>,
+    pub table_class_override: ::std::option::Option<super::types::TableClass>,
 }
 impl UpdateReplicationGroupMemberAction {
     /// <p>The Region where the replica exists.</p>
@@ -28,28 +28,28 @@
         self.kms_master_key_id.as_deref()
     }
     /// <p>Replica-specific provisioned throughput. If not specified, uses the source table's provisioned throughput settings.</p>
-    pub fn provisioned_throughput_override(&self) -> ::std::option::Option<&crate::types::ProvisionedThroughputOverride> {
+    pub fn provisioned_throughput_override(&self) -> ::std::option::Option<&super::types::ProvisionedThroughputOverride> {
         self.provisioned_throughput_override.as_ref()
     }
     /// <p>Overrides the maximum on-demand throughput for the replica table.</p>
-    pub fn on_demand_throughput_override(&self) -> ::std::option::Option<&crate::types::OnDemandThroughputOverride> {
+    pub fn on_demand_throughput_override(&self) -> ::std::option::Option<&super::types::OnDemandThroughputOverride> {
         self.on_demand_throughput_override.as_ref()
     }
     /// <p>Replica-specific global secondary index settings.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.global_secondary_indexes.is_none()`.
-    pub fn global_secondary_indexes(&self) -> &[crate::types::ReplicaGlobalSecondaryIndex] {
+    pub fn global_secondary_indexes(&self) -> &[super::types::ReplicaGlobalSecondaryIndex] {
         self.global_secondary_indexes.as_deref().unwrap_or_default()
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn table_class_override(&self) -> ::std::option::Option<&crate::types::TableClass> {
+    pub fn table_class_override(&self) -> ::std::option::Option<&super::types::TableClass> {
         self.table_class_override.as_ref()
     }
 }
 impl UpdateReplicationGroupMemberAction {
     /// Creates a new builder-style object to manufacture [`UpdateReplicationGroupMemberAction`](crate::types::UpdateReplicationGroupMemberAction).
-    pub fn builder() -> crate::types::builders::UpdateReplicationGroupMemberActionBuilder {
-        crate::types::builders::UpdateReplicationGroupMemberActionBuilder::default()
+    pub fn builder() -> super::types::builders::UpdateReplicationGroupMemberActionBuilder {
+        super::types::builders::UpdateReplicationGroupMemberActionBuilder::default()
     }
 }

@@ -59,10 +59,10 @@
 pub struct UpdateReplicationGroupMemberActionBuilder {
     pub(crate) region_name: ::std::option::Option<::std::string::String>,
     pub(crate) kms_master_key_id: ::std::option::Option<::std::string::String>,
-    pub(crate) provisioned_throughput_override: ::std::option::Option<crate::types::ProvisionedThroughputOverride>,
-    pub(crate) on_demand_throughput_override: ::std::option::Option<crate::types::OnDemandThroughputOverride>,
-    pub(crate) global_secondary_indexes: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndex>>,
-    pub(crate) table_class_override: ::std::option::Option<crate::types::TableClass>,
+    pub(crate) provisioned_throughput_override: ::std::option::Option<super::types::ProvisionedThroughputOverride>,
+    pub(crate) on_demand_throughput_override: ::std::option::Option<super::types::OnDemandThroughputOverride>,
+    pub(crate) global_secondary_indexes: ::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndex>>,
+    pub(crate) table_class_override: ::std::option::Option<super::types::TableClass>,
 }
 impl UpdateReplicationGroupMemberActionBuilder {
     /// <p>The Region where the replica exists.</p>
@@ -95,31 +95,31 @@
         &self.kms_master_key_id
     }
     /// <p>Replica-specific provisioned throughput. If not specified, uses the source table's provisioned throughput settings.</p>
-    pub fn provisioned_throughput_override(mut self, input: crate::types::ProvisionedThroughputOverride) -> Self {
+    pub fn provisioned_throughput_override(mut self, input: super::types::ProvisionedThroughputOverride) -> Self {
         self.provisioned_throughput_override = ::std::option::Option::Some(input);
         self
     }
     /// <p>Replica-specific provisioned throughput. If not specified, uses the source table's provisioned throughput settings.</p>
-    pub fn set_provisioned_throughput_override(mut self, input: ::std::option::Option<crate::types::ProvisionedThroughputOverride>) -> Self {
+    pub fn set_provisioned_throughput_override(mut self, input: ::std::option::Option<super::types::ProvisionedThroughputOverride>) -> Self {
         self.provisioned_throughput_override = input;
         self
     }
     /// <p>Replica-specific provisioned throughput. If not specified, uses the source table's provisioned throughput settings.</p>
-    pub fn get_provisioned_throughput_override(&self) -> &::std::option::Option<crate::types::ProvisionedThroughputOverride> {
+    pub fn get_provisioned_throughput_override(&self) -> &::std::option::Option<super::types::ProvisionedThroughputOverride> {
         &self.provisioned_throughput_override
     }
     /// <p>Overrides the maximum on-demand throughput for the replica table.</p>
-    pub fn on_demand_throughput_override(mut self, input: crate::types::OnDemandThroughputOverride) -> Self {
+    pub fn on_demand_throughput_override(mut self, input: super::types::OnDemandThroughputOverride) -> Self {
         self.on_demand_throughput_override = ::std::option::Option::Some(input);
         self
     }
     /// <p>Overrides the maximum on-demand throughput for the replica table.</p>
-    pub fn set_on_demand_throughput_override(mut self, input: ::std::option::Option<crate::types::OnDemandThroughputOverride>) -> Self {
+    pub fn set_on_demand_throughput_override(mut self, input: ::std::option::Option<super::types::OnDemandThroughputOverride>) -> Self {
         self.on_demand_throughput_override = input;
         self
     }
     /// <p>Overrides the maximum on-demand throughput for the replica table.</p>
-    pub fn get_on_demand_throughput_override(&self) -> &::std::option::Option<crate::types::OnDemandThroughputOverride> {
+    pub fn get_on_demand_throughput_override(&self) -> &::std::option::Option<super::types::OnDemandThroughputOverride> {
         &self.on_demand_throughput_override
     }
     /// Appends an item to `global_secondary_indexes`.
@@ -127,7 +127,7 @@
     /// To override the contents of this collection use [`set_global_secondary_indexes`](Self::set_global_secondary_indexes).
     ///
     /// <p>Replica-specific global secondary index settings.</p>
-    pub fn global_secondary_indexes(mut self, input: crate::types::ReplicaGlobalSecondaryIndex) -> Self {
+    pub fn global_secondary_indexes(mut self, input: super::types::ReplicaGlobalSecondaryIndex) -> Self {
         let mut v = self.global_secondary_indexes.unwrap_or_default();
         v.push(input);
         self.global_secondary_indexes = ::std::option::Option::Some(v);
@@ -134,33 +134,33 @@
         self
     }
     /// <p>Replica-specific global secondary index settings.</p>
-    pub fn set_global_secondary_indexes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndex>>) -> Self {
+    pub fn set_global_secondary_indexes(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndex>>) -> Self {
         self.global_secondary_indexes = input;
         self
     }
     /// <p>Replica-specific global secondary index settings.</p>
-    pub fn get_global_secondary_indexes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ReplicaGlobalSecondaryIndex>> {
+    pub fn get_global_secondary_indexes(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ReplicaGlobalSecondaryIndex>> {
         &self.global_secondary_indexes
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn table_class_override(mut self, input: crate::types::TableClass) -> Self {
+    pub fn table_class_override(mut self, input: super::types::TableClass) -> Self {
         self.table_class_override = ::std::option::Option::Some(input);
         self
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn set_table_class_override(mut self, input: ::std::option::Option<crate::types::TableClass>) -> Self {
+    pub fn set_table_class_override(mut self, input: ::std::option::Option<super::types::TableClass>) -> Self {
         self.table_class_override = input;
         self
     }
     /// <p>Replica-specific table class. If not specified, uses the source table's table class.</p>
-    pub fn get_table_class_override(&self) -> &::std::option::Option<crate::types::TableClass> {
+    pub fn get_table_class_override(&self) -> &::std::option::Option<super::types::TableClass> {
         &self.table_class_override
     }
     /// Consumes the builder and constructs a [`UpdateReplicationGroupMemberAction`](crate::types::UpdateReplicationGroupMemberAction).
     /// This method will fail if any of the following fields are not set:
     /// - [`region_name`](crate::types::builders::UpdateReplicationGroupMemberActionBuilder::region_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::UpdateReplicationGroupMemberAction, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::UpdateReplicationGroupMemberAction {
+    pub fn build(self) -> ::std::result::Result<super::types::UpdateReplicationGroupMemberAction, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::UpdateReplicationGroupMemberAction {
             region_name: self.region_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "region_name",
