--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl VectorCapacity {
     /// Creates a new builder-style object to manufacture [`VectorCapacity`](crate::types::VectorCapacity).
-    pub fn builder() -> crate::types::builders::VectorCapacityBuilder {
-        crate::types::builders::VectorCapacityBuilder::default()
+    pub fn builder() -> super::super::types::builders::VectorCapacityBuilder {
+        super::super::types::builders::VectorCapacityBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.vector_write_request_bytes
     }
     /// Consumes the builder and constructs a [`VectorCapacity`](crate::types::VectorCapacity).
-    pub fn build(self) -> crate::types::VectorCapacity {
-        crate::types::VectorCapacity {
+    pub fn build(self) -> super::super::types::VectorCapacity {
+        super::super::types::VectorCapacity {
             vector_search_request_bytes: self.vector_search_request_bytes,
             vector_write_request_bytes: self.vector_write_request_bytes,
         }
