--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl UpdateGroupInput {
     /// Creates a new builder-style object to manufacture [`UpdateGroupInput`](crate::operation::update_group::UpdateGroupInput).
-    pub fn builder() -> crate::operation::update_group::builders::UpdateGroupInputBuilder {
-        crate::operation::update_group::builders::UpdateGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_group::builders::UpdateGroupInputBuilder {
+        super::super::super::operation::update_group::builders::UpdateGroupInputBuilder::default()
     }
 }

@@ -137,8 +137,8 @@
         &self.precedence
     }
     /// Consumes the builder and constructs a [`UpdateGroupInput`](crate::operation::update_group::UpdateGroupInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::update_group::UpdateGroupInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_group::UpdateGroupInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::update_group::UpdateGroupInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_group::UpdateGroupInput {
             group_name: self.group_name,
             user_pool_id: self.user_pool_id,
             description: self.description,
