--- original
+++ modified
@@ -37,8 +37,8 @@
 }
 impl CapacityReservationRequest {
     /// Creates a new builder-style object to manufacture [`CapacityReservationRequest`](crate::types::CapacityReservationRequest).
-    pub fn builder() -> crate::types::builders::CapacityReservationRequestBuilder {
-        crate::types::builders::CapacityReservationRequestBuilder::default()
+    pub fn builder() -> super::types::builders::CapacityReservationRequestBuilder {
+        super::types::builders::CapacityReservationRequestBuilder::default()
     }
 }

@@ -103,8 +103,8 @@
         &self.reservation_preference
     }
     /// Consumes the builder and constructs a [`CapacityReservationRequest`](crate::types::CapacityReservationRequest).
-    pub fn build(self) -> crate::types::CapacityReservationRequest {
-        crate::types::CapacityReservationRequest {
+    pub fn build(self) -> super::types::CapacityReservationRequest {
+        super::types::CapacityReservationRequest {
             reservation_group_arn: self.reservation_group_arn,
             reservation_preference: self.reservation_preference,
         }
