--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl ComputeEnvironmentOrder {
     /// Creates a new builder-style object to manufacture [`ComputeEnvironmentOrder`](crate::types::ComputeEnvironmentOrder).
-    pub fn builder() -> crate::types::builders::ComputeEnvironmentOrderBuilder {
-        crate::types::builders::ComputeEnvironmentOrderBuilder::default()
+    pub fn builder() -> super::types::builders::ComputeEnvironmentOrderBuilder {
+        super::types::builders::ComputeEnvironmentOrderBuilder::default()
     }
 }

@@ -67,8 +67,8 @@
         &self.compute_environment
     }
     /// Consumes the builder and constructs a [`ComputeEnvironmentOrder`](crate::types::ComputeEnvironmentOrder).
-    pub fn build(self) -> crate::types::ComputeEnvironmentOrder {
-        crate::types::ComputeEnvironmentOrder {
+    pub fn build(self) -> super::types::ComputeEnvironmentOrder {
+        super::types::ComputeEnvironmentOrder {
             order: self.order,
             compute_environment: self.compute_environment,
         }
