--- original
+++ modified
@@ -22,9 +22,9 @@
     /// <p>The environment variables to send to the container. You can add new environment variables, which are added to the container at launch, or you can override the existing environment variables from the Docker image or the job definition.</p><note>
     /// <p>Environment variables cannot start with "<code>AWS_BATCH</code>". This naming convention is reserved for variables that Batch sets.</p>
     /// </note>
-    pub environment: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuePair>>,
+    pub environment: ::std::option::Option<::std::vec::Vec<super::types::KeyValuePair>>,
     /// <p>The type and amount of resources to assign to a container. This overrides the settings in the job definition. The supported resources include <code>GPU</code>, <code>MEMORY</code>, and <code>VCPU</code>.</p>
-    pub resource_requirements: ::std::option::Option<::std::vec::Vec<crate::types::ResourceRequirement>>,
+    pub resource_requirements: ::std::option::Option<::std::vec::Vec<super::types::ResourceRequirement>>,
 }
 impl ContainerOverrides {
     /// <p>This parameter is deprecated, use <code>resourceRequirements</code> to override the <code>vcpus</code> parameter that's set in the job definition. It's not supported for jobs running on Fargate resources. For jobs that run on Amazon EC2 resources, it overrides the <code>vcpus</code> parameter set in the job definition, but doesn't override any vCPU requirement specified in the <code>resourceRequirements</code> structure in the job definition. To override vCPU requirements that are specified in the <code>resourceRequirements</code> structure in the job definition, <code>resourceRequirements</code> must be specified in the <code>SubmitJob</code> request, with <code>type</code> set to <code>VCPU</code> and <code>value</code> set to the new value. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/troubleshooting.html#override-resource-requirements">Can't override job definition resource requirements</a> in the <i>Batch User Guide</i>.</p>
@@ -56,20 +56,20 @@
     /// </note>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.environment.is_none()`.
-    pub fn environment(&self) -> &[crate::types::KeyValuePair] {
+    pub fn environment(&self) -> &[super::types::KeyValuePair] {
         self.environment.as_deref().unwrap_or_default()
     }
     /// <p>The type and amount of resources to assign to a container. This overrides the settings in the job definition. The supported resources include <code>GPU</code>, <code>MEMORY</code>, and <code>VCPU</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.resource_requirements.is_none()`.
-    pub fn resource_requirements(&self) -> &[crate::types::ResourceRequirement] {
+    pub fn resource_requirements(&self) -> &[super::types::ResourceRequirement] {
         self.resource_requirements.as_deref().unwrap_or_default()
     }
 }
 impl ContainerOverrides {
     /// Creates a new builder-style object to manufacture [`ContainerOverrides`](crate::types::ContainerOverrides).
-    pub fn builder() -> crate::types::builders::ContainerOverridesBuilder {
-        crate::types::builders::ContainerOverridesBuilder::default()
+    pub fn builder() -> super::types::builders::ContainerOverridesBuilder {
+        super::types::builders::ContainerOverridesBuilder::default()
     }
 }

@@ -81,8 +81,8 @@
     pub(crate) memory: ::std::option::Option<i32>,
     pub(crate) command: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) instance_type: ::std::option::Option<::std::string::String>,
-    pub(crate) environment: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuePair>>,
-    pub(crate) resource_requirements: ::std::option::Option<::std::vec::Vec<crate::types::ResourceRequirement>>,
+    pub(crate) environment: ::std::option::Option<::std::vec::Vec<super::types::KeyValuePair>>,
+    pub(crate) resource_requirements: ::std::option::Option<::std::vec::Vec<super::types::ResourceRequirement>>,
 }
 impl ContainerOverridesBuilder {
     /// <p>This parameter is deprecated, use <code>resourceRequirements</code> to override the <code>vcpus</code> parameter that's set in the job definition. It's not supported for jobs running on Fargate resources. For jobs that run on Amazon EC2 resources, it overrides the <code>vcpus</code> parameter set in the job definition, but doesn't override any vCPU requirement specified in the <code>resourceRequirements</code> structure in the job definition. To override vCPU requirements that are specified in the <code>resourceRequirements</code> structure in the job definition, <code>resourceRequirements</code> must be specified in the <code>SubmitJob</code> request, with <code>type</code> set to <code>VCPU</code> and <code>value</code> set to the new value. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/troubleshooting.html#override-resource-requirements">Can't override job definition resource requirements</a> in the <i>Batch User Guide</i>.</p>
@@ -172,7 +172,7 @@
     /// <p>The environment variables to send to the container. You can add new environment variables, which are added to the container at launch, or you can override the existing environment variables from the Docker image or the job definition.</p><note>
     /// <p>Environment variables cannot start with "<code>AWS_BATCH</code>". This naming convention is reserved for variables that Batch sets.</p>
     /// </note>
-    pub fn environment(mut self, input: crate::types::KeyValuePair) -> Self {
+    pub fn environment(mut self, input: super::types::KeyValuePair) -> Self {
         let mut v = self.environment.unwrap_or_default();
         v.push(input);
         self.environment = ::std::option::Option::Some(v);
@@ -181,7 +181,7 @@
     /// <p>The environment variables to send to the container. You can add new environment variables, which are added to the container at launch, or you can override the existing environment variables from the Docker image or the job definition.</p><note>
     /// <p>Environment variables cannot start with "<code>AWS_BATCH</code>". This naming convention is reserved for variables that Batch sets.</p>
     /// </note>
-    pub fn set_environment(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuePair>>) -> Self {
+    pub fn set_environment(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::KeyValuePair>>) -> Self {
         self.environment = input;
         self
     }
@@ -188,7 +188,7 @@
     /// <p>The environment variables to send to the container. You can add new environment variables, which are added to the container at launch, or you can override the existing environment variables from the Docker image or the job definition.</p><note>
     /// <p>Environment variables cannot start with "<code>AWS_BATCH</code>". This naming convention is reserved for variables that Batch sets.</p>
     /// </note>
-    pub fn get_environment(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::KeyValuePair>> {
+    pub fn get_environment(&self) -> &::std::option::Option<::std::vec::Vec<super::types::KeyValuePair>> {
         &self.environment
     }
     /// Appends an item to `resource_requirements`.
@@ -196,7 +196,7 @@
     /// To override the contents of this collection use [`set_resource_requirements`](Self::set_resource_requirements).
     ///
     /// <p>The type and amount of resources to assign to a container. This overrides the settings in the job definition. The supported resources include <code>GPU</code>, <code>MEMORY</code>, and <code>VCPU</code>.</p>
-    pub fn resource_requirements(mut self, input: crate::types::ResourceRequirement) -> Self {
+    pub fn resource_requirements(mut self, input: super::types::ResourceRequirement) -> Self {
         let mut v = self.resource_requirements.unwrap_or_default();
         v.push(input);
         self.resource_requirements = ::std::option::Option::Some(v);
@@ -203,17 +203,17 @@
         self
     }
     /// <p>The type and amount of resources to assign to a container. This overrides the settings in the job definition. The supported resources include <code>GPU</code>, <code>MEMORY</code>, and <code>VCPU</code>.</p>
-    pub fn set_resource_requirements(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ResourceRequirement>>) -> Self {
+    pub fn set_resource_requirements(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ResourceRequirement>>) -> Self {
         self.resource_requirements = input;
         self
     }
     /// <p>The type and amount of resources to assign to a container. This overrides the settings in the job definition. The supported resources include <code>GPU</code>, <code>MEMORY</code>, and <code>VCPU</code>.</p>
-    pub fn get_resource_requirements(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ResourceRequirement>> {
+    pub fn get_resource_requirements(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ResourceRequirement>> {
         &self.resource_requirements
     }
     /// Consumes the builder and constructs a [`ContainerOverrides`](crate::types::ContainerOverrides).
-    pub fn build(self) -> crate::types::ContainerOverrides {
-        crate::types::ContainerOverrides {
+    pub fn build(self) -> super::types::ContainerOverrides {
+        super::types::ContainerOverrides {
             vcpus: self.vcpus,
             memory: self.memory,
             command: self.command,
