--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The overrides for the Amazon ECS task definition of a job.</p><note>
     /// <p>This object is currently limited to one element.</p>
     /// </note>
-    pub task_properties: ::std::option::Option<::std::vec::Vec<crate::types::TaskPropertiesOverride>>,
+    pub task_properties: ::std::option::Option<::std::vec::Vec<super::types::TaskPropertiesOverride>>,
 }
 impl EcsPropertiesOverride {
     /// <p>The overrides for the Amazon ECS task definition of a job.</p><note>
@@ -15,14 +15,14 @@
     /// </note>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.task_properties.is_none()`.
-    pub fn task_properties(&self) -> &[crate::types::TaskPropertiesOverride] {
+    pub fn task_properties(&self) -> &[super::types::TaskPropertiesOverride] {
         self.task_properties.as_deref().unwrap_or_default()
     }
 }
 impl EcsPropertiesOverride {
     /// Creates a new builder-style object to manufacture [`EcsPropertiesOverride`](crate::types::EcsPropertiesOverride).
-    pub fn builder() -> crate::types::builders::EcsPropertiesOverrideBuilder {
-        crate::types::builders::EcsPropertiesOverrideBuilder::default()
+    pub fn builder() -> super::types::builders::EcsPropertiesOverrideBuilder {
+        super::types::builders::EcsPropertiesOverrideBuilder::default()
     }
 }

@@ -30,7 +30,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct EcsPropertiesOverrideBuilder {
-    pub(crate) task_properties: ::std::option::Option<::std::vec::Vec<crate::types::TaskPropertiesOverride>>,
+    pub(crate) task_properties: ::std::option::Option<::std::vec::Vec<super::types::TaskPropertiesOverride>>,
 }
 impl EcsPropertiesOverrideBuilder {
     /// Appends an item to `task_properties`.
@@ -40,7 +40,7 @@
     /// <p>The overrides for the Amazon ECS task definition of a job.</p><note>
     /// <p>This object is currently limited to one element.</p>
     /// </note>
-    pub fn task_properties(mut self, input: crate::types::TaskPropertiesOverride) -> Self {
+    pub fn task_properties(mut self, input: super::types::TaskPropertiesOverride) -> Self {
         let mut v = self.task_properties.unwrap_or_default();
         v.push(input);
         self.task_properties = ::std::option::Option::Some(v);
@@ -49,7 +49,7 @@
     /// <p>The overrides for the Amazon ECS task definition of a job.</p><note>
     /// <p>This object is currently limited to one element.</p>
     /// </note>
-    pub fn set_task_properties(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::TaskPropertiesOverride>>) -> Self {
+    pub fn set_task_properties(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::TaskPropertiesOverride>>) -> Self {
         self.task_properties = input;
         self
     }
@@ -56,12 +56,12 @@
     /// <p>The overrides for the Amazon ECS task definition of a job.</p><note>
     /// <p>This object is currently limited to one element.</p>
     /// </note>
-    pub fn get_task_properties(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::TaskPropertiesOverride>> {
+    pub fn get_task_properties(&self) -> &::std::option::Option<::std::vec::Vec<super::types::TaskPropertiesOverride>> {
         &self.task_properties
     }
     /// Consumes the builder and constructs a [`EcsPropertiesOverride`](crate::types::EcsPropertiesOverride).
-    pub fn build(self) -> crate::types::EcsPropertiesOverride {
-        crate::types::EcsPropertiesOverride {
+    pub fn build(self) -> super::types::EcsPropertiesOverride {
+        super::types::EcsPropertiesOverride {
             task_properties: self.task_properties,
         }
     }
