--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl EksSecret {
     /// Creates a new builder-style object to manufacture [`EksSecret`](crate::types::EksSecret).
-    pub fn builder() -> crate::types::builders::EksSecretBuilder {
-        crate::types::builders::EksSecretBuilder::default()
+    pub fn builder() -> super::types::builders::EksSecretBuilder {
+        super::types::builders::EksSecretBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
         &self.optional
     }
     /// Consumes the builder and constructs a [`EksSecret`](crate::types::EksSecret).
-    pub fn build(self) -> crate::types::EksSecret {
-        crate::types::EksSecret {
+    pub fn build(self) -> super::types::EksSecret {
+        super::types::EksSecret {
             secret_name: self.secret_name,
             optional: self.optional,
         }
