--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl EksContainerEnvironmentVariable {
     /// Creates a new builder-style object to manufacture [`EksContainerEnvironmentVariable`](crate::types::EksContainerEnvironmentVariable).
-    pub fn builder() -> crate::types::builders::EksContainerEnvironmentVariableBuilder {
-        crate::types::builders::EksContainerEnvironmentVariableBuilder::default()
+    pub fn builder() -> super::super::types::builders::EksContainerEnvironmentVariableBuilder {
+        super::super::types::builders::EksContainerEnvironmentVariableBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
         &self.value
     }
     /// Consumes the builder and constructs a [`EksContainerEnvironmentVariable`](crate::types::EksContainerEnvironmentVariable).
-    pub fn build(self) -> crate::types::EksContainerEnvironmentVariable {
-        crate::types::EksContainerEnvironmentVariable {
+    pub fn build(self) -> super::super::types::EksContainerEnvironmentVariable {
+        super::super::types::EksContainerEnvironmentVariable {
             name: self.name,
             value: self.value,
         }
