--- original
+++ modified
@@ -109,8 +109,8 @@
 }
 impl EksContainerResourceRequirements {
     /// Creates a new builder-style object to manufacture [`EksContainerResourceRequirements`](crate::types::EksContainerResourceRequirements).
-    pub fn builder() -> crate::types::builders::EksContainerResourceRequirementsBuilder {
-        crate::types::builders::EksContainerResourceRequirementsBuilder::default()
+    pub fn builder() -> super::types::builders::EksContainerResourceRequirementsBuilder {
+        super::types::builders::EksContainerResourceRequirementsBuilder::default()
     }
 }

@@ -295,8 +295,8 @@
         &self.requests
     }
     /// Consumes the builder and constructs a [`EksContainerResourceRequirements`](crate::types::EksContainerResourceRequirements).
-    pub fn build(self) -> crate::types::EksContainerResourceRequirements {
-        crate::types::EksContainerResourceRequirements {
+    pub fn build(self) -> super::types::EksContainerResourceRequirements {
+        super::types::EksContainerResourceRequirements {
             limits: self.limits,
             requests: self.requests,
         }
