--- original
+++ modified
@@ -13,19 +13,19 @@
     pub dns_policy: ::std::option::Option<::std::string::String>,
     /// <p>References a Kubernetes secret resource. It holds a list of secrets. These secrets help to gain access to pull an images from a private registry.</p>
     /// <p><code>ImagePullSecret$name</code> is required when this object is used.</p>
-    pub image_pull_secrets: ::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>>,
+    pub image_pull_secrets: ::std::option::Option<::std::vec::Vec<super::super::types::ImagePullSecret>>,
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainer>>,
+    pub containers: ::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>>,
     /// <p>These containers run before application containers, always runs to completion, and must complete successfully before the next container starts. These containers are registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store. For more information, see <a href="https://kubernetes.io/docs/concepts/workloads/pods/init-containers/">Init Containers</a> in the <i>Kubernetes documentation</i>.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub init_containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainer>>,
+    pub init_containers: ::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>>,
     /// <p>Specifies the volumes for a job definition that uses Amazon EKS resources.</p>
-    pub volumes: ::std::option::Option<::std::vec::Vec<crate::types::EksVolume>>,
+    pub volumes: ::std::option::Option<::std::vec::Vec<super::super::types::EksVolume>>,
     /// <p>Metadata about the Kubernetes pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub metadata: ::std::option::Option<crate::types::EksMetadata>,
+    pub metadata: ::std::option::Option<super::super::types::EksMetadata>,
     /// <p>Indicates if the processes in a container are shared, or visible, to other containers in the same pod. For more information, see <a href="https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/">Share Process Namespace between Containers in a Pod</a>.</p>
     pub share_process_namespace: ::std::option::Option<bool>,
 }
@@ -47,7 +47,7 @@
     /// <p><code>ImagePullSecret$name</code> is required when this object is used.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.image_pull_secrets.is_none()`.
-    pub fn image_pull_secrets(&self) -> &[crate::types::ImagePullSecret] {
+    pub fn image_pull_secrets(&self) -> &[super::super::types::ImagePullSecret] {
         self.image_pull_secrets.as_deref().unwrap_or_default()
     }
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p><note>
@@ -55,7 +55,7 @@
     /// </note>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.containers.is_none()`.
-    pub fn containers(&self) -> &[crate::types::EksContainer] {
+    pub fn containers(&self) -> &[super::super::types::EksContainer] {
         self.containers.as_deref().unwrap_or_default()
     }
     /// <p>These containers run before application containers, always runs to completion, and must complete successfully before the next container starts. These containers are registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store. For more information, see <a href="https://kubernetes.io/docs/concepts/workloads/pods/init-containers/">Init Containers</a> in the <i>Kubernetes documentation</i>.</p><note>
@@ -63,17 +63,17 @@
     /// </note>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.init_containers.is_none()`.
-    pub fn init_containers(&self) -> &[crate::types::EksContainer] {
+    pub fn init_containers(&self) -> &[super::super::types::EksContainer] {
         self.init_containers.as_deref().unwrap_or_default()
     }
     /// <p>Specifies the volumes for a job definition that uses Amazon EKS resources.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.volumes.is_none()`.
-    pub fn volumes(&self) -> &[crate::types::EksVolume] {
+    pub fn volumes(&self) -> &[super::super::types::EksVolume] {
         self.volumes.as_deref().unwrap_or_default()
     }
     /// <p>Metadata about the Kubernetes pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn metadata(&self) -> ::std::option::Option<&crate::types::EksMetadata> {
+    pub fn metadata(&self) -> ::std::option::Option<&super::super::types::EksMetadata> {
         self.metadata.as_ref()
     }
     /// <p>Indicates if the processes in a container are shared, or visible, to other containers in the same pod. For more information, see <a href="https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/">Share Process Namespace between Containers in a Pod</a>.</p>
@@ -83,8 +83,8 @@
 }
 impl EksPodProperties {
     /// Creates a new builder-style object to manufacture [`EksPodProperties`](crate::types::EksPodProperties).
-    pub fn builder() -> crate::types::builders::EksPodPropertiesBuilder {
-        crate::types::builders::EksPodPropertiesBuilder::default()
+    pub fn builder() -> super::super::types::builders::EksPodPropertiesBuilder {
+        super::super::types::builders::EksPodPropertiesBuilder::default()
     }
 }

@@ -95,11 +95,11 @@
     pub(crate) service_account_name: ::std::option::Option<::std::string::String>,
     pub(crate) host_network: ::std::option::Option<bool>,
     pub(crate) dns_policy: ::std::option::Option<::std::string::String>,
-    pub(crate) image_pull_secrets: ::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>>,
-    pub(crate) containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainer>>,
-    pub(crate) init_containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainer>>,
-    pub(crate) volumes: ::std::option::Option<::std::vec::Vec<crate::types::EksVolume>>,
-    pub(crate) metadata: ::std::option::Option<crate::types::EksMetadata>,
+    pub(crate) image_pull_secrets: ::std::option::Option<::std::vec::Vec<super::super::types::ImagePullSecret>>,
+    pub(crate) containers: ::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>>,
+    pub(crate) init_containers: ::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>>,
+    pub(crate) volumes: ::std::option::Option<::std::vec::Vec<super::super::types::EksVolume>>,
+    pub(crate) metadata: ::std::option::Option<super::super::types::EksMetadata>,
     pub(crate) share_process_namespace: ::std::option::Option<bool>,
 }
 impl EksPodPropertiesBuilder {
@@ -154,7 +154,7 @@
     ///
     /// <p>References a Kubernetes secret resource. It holds a list of secrets. These secrets help to gain access to pull an images from a private registry.</p>
     /// <p><code>ImagePullSecret$name</code> is required when this object is used.</p>
-    pub fn image_pull_secrets(mut self, input: crate::types::ImagePullSecret) -> Self {
+    pub fn image_pull_secrets(mut self, input: super::super::types::ImagePullSecret) -> Self {
         let mut v = self.image_pull_secrets.unwrap_or_default();
         v.push(input);
         self.image_pull_secrets = ::std::option::Option::Some(v);
@@ -162,13 +162,13 @@
     }
     /// <p>References a Kubernetes secret resource. It holds a list of secrets. These secrets help to gain access to pull an images from a private registry.</p>
     /// <p><code>ImagePullSecret$name</code> is required when this object is used.</p>
-    pub fn set_image_pull_secrets(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>>) -> Self {
+    pub fn set_image_pull_secrets(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::ImagePullSecret>>) -> Self {
         self.image_pull_secrets = input;
         self
     }
     /// <p>References a Kubernetes secret resource. It holds a list of secrets. These secrets help to gain access to pull an images from a private registry.</p>
     /// <p><code>ImagePullSecret$name</code> is required when this object is used.</p>
-    pub fn get_image_pull_secrets(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>> {
+    pub fn get_image_pull_secrets(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::ImagePullSecret>> {
         &self.image_pull_secrets
     }
     /// Appends an item to `containers`.
@@ -178,7 +178,7 @@
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub fn containers(mut self, input: crate::types::EksContainer) -> Self {
+    pub fn containers(mut self, input: super::super::types::EksContainer) -> Self {
         let mut v = self.containers.unwrap_or_default();
         v.push(input);
         self.containers = ::std::option::Option::Some(v);
@@ -187,7 +187,7 @@
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub fn set_containers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EksContainer>>) -> Self {
+    pub fn set_containers(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>>) -> Self {
         self.containers = input;
         self
     }
@@ -194,7 +194,7 @@
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub fn get_containers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EksContainer>> {
+    pub fn get_containers(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>> {
         &self.containers
     }
     /// Appends an item to `init_containers`.
@@ -204,7 +204,7 @@
     /// <p>These containers run before application containers, always runs to completion, and must complete successfully before the next container starts. These containers are registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store. For more information, see <a href="https://kubernetes.io/docs/concepts/workloads/pods/init-containers/">Init Containers</a> in the <i>Kubernetes documentation</i>.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub fn init_containers(mut self, input: crate::types::EksContainer) -> Self {
+    pub fn init_containers(mut self, input: super::super::types::EksContainer) -> Self {
         let mut v = self.init_containers.unwrap_or_default();
         v.push(input);
         self.init_containers = ::std::option::Option::Some(v);
@@ -213,7 +213,7 @@
     /// <p>These containers run before application containers, always runs to completion, and must complete successfully before the next container starts. These containers are registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store. For more information, see <a href="https://kubernetes.io/docs/concepts/workloads/pods/init-containers/">Init Containers</a> in the <i>Kubernetes documentation</i>.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub fn set_init_containers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EksContainer>>) -> Self {
+    pub fn set_init_containers(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>>) -> Self {
         self.init_containers = input;
         self
     }
@@ -220,7 +220,7 @@
     /// <p>These containers run before application containers, always runs to completion, and must complete successfully before the next container starts. These containers are registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store. For more information, see <a href="https://kubernetes.io/docs/concepts/workloads/pods/init-containers/">Init Containers</a> in the <i>Kubernetes documentation</i>.</p><note>
     /// <p>This object is limited to 10 elements.</p>
     /// </note>
-    pub fn get_init_containers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EksContainer>> {
+    pub fn get_init_containers(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::EksContainer>> {
         &self.init_containers
     }
     /// Appends an item to `volumes`.
@@ -228,7 +228,7 @@
     /// To override the contents of this collection use [`set_volumes`](Self::set_volumes).
     ///
     /// <p>Specifies the volumes for a job definition that uses Amazon EKS resources.</p>
-    pub fn volumes(mut self, input: crate::types::EksVolume) -> Self {
+    pub fn volumes(mut self, input: super::super::types::EksVolume) -> Self {
         let mut v = self.volumes.unwrap_or_default();
         v.push(input);
         self.volumes = ::std::option::Option::Some(v);
@@ -235,26 +235,26 @@
         self
     }
     /// <p>Specifies the volumes for a job definition that uses Amazon EKS resources.</p>
-    pub fn set_volumes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EksVolume>>) -> Self {
+    pub fn set_volumes(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::EksVolume>>) -> Self {
         self.volumes = input;
         self
     }
     /// <p>Specifies the volumes for a job definition that uses Amazon EKS resources.</p>
-    pub fn get_volumes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EksVolume>> {
+    pub fn get_volumes(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::EksVolume>> {
         &self.volumes
     }
     /// <p>Metadata about the Kubernetes pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn metadata(mut self, input: crate::types::EksMetadata) -> Self {
+    pub fn metadata(mut self, input: super::super::types::EksMetadata) -> Self {
         self.metadata = ::std::option::Option::Some(input);
         self
     }
     /// <p>Metadata about the Kubernetes pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn set_metadata(mut self, input: ::std::option::Option<crate::types::EksMetadata>) -> Self {
+    pub fn set_metadata(mut self, input: ::std::option::Option<super::super::types::EksMetadata>) -> Self {
         self.metadata = input;
         self
     }
     /// <p>Metadata about the Kubernetes pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn get_metadata(&self) -> &::std::option::Option<crate::types::EksMetadata> {
+    pub fn get_metadata(&self) -> &::std::option::Option<super::super::types::EksMetadata> {
         &self.metadata
     }
     /// <p>Indicates if the processes in a container are shared, or visible, to other containers in the same pod. For more information, see <a href="https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/">Share Process Namespace between Containers in a Pod</a>.</p>
@@ -272,8 +272,8 @@
         &self.share_process_namespace
     }
     /// Consumes the builder and constructs a [`EksPodProperties`](crate::types::EksPodProperties).
-    pub fn build(self) -> crate::types::EksPodProperties {
-        crate::types::EksPodProperties {
+    pub fn build(self) -> super::super::types::EksPodProperties {
+        super::super::types::EksPodProperties {
             service_account_name: self.service_account_name,
             host_network: self.host_network,
             dns_policy: self.dns_policy,
