--- original
+++ modified
@@ -12,19 +12,19 @@
     /// <p>Valid values: <code>Default</code> | <code>ClusterFirst</code> | <code>ClusterFirstWithHostNet</code></p>
     pub dns_policy: ::std::option::Option<::std::string::String>,
     /// <p>Displays the reference pointer to the Kubernetes secret resource. These secrets help to gain access to pull an images from a private registry.</p>
-    pub image_pull_secrets: ::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>>,
+    pub image_pull_secrets: ::std::option::Option<::std::vec::Vec<super::types::ImagePullSecret>>,
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p>
-    pub containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>>,
+    pub containers: ::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>>,
     /// <p>The container registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store.</p>
-    pub init_containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>>,
+    pub init_containers: ::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>>,
     /// <p>Specifies the volumes for a job definition using Amazon EKS resources.</p>
-    pub volumes: ::std::option::Option<::std::vec::Vec<crate::types::EksVolume>>,
+    pub volumes: ::std::option::Option<::std::vec::Vec<super::types::EksVolume>>,
     /// <p>The name of the pod for this job.</p>
     pub pod_name: ::std::option::Option<::std::string::String>,
     /// <p>The name of the node for this job.</p>
     pub node_name: ::std::option::Option<::std::string::String>,
     /// <p>Describes and uniquely identifies Kubernetes resources. For example, the compute environment that a pod runs in or the <code>jobID</code> for a job running in the pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub metadata: ::std::option::Option<crate::types::EksMetadata>,
+    pub metadata: ::std::option::Option<super::types::EksMetadata>,
     /// <p>Indicates if the processes in a container are shared, or visible, to other containers in the same pod. For more information, see <a href="https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/">Share Process Namespace between Containers in a Pod</a>.</p>
     pub share_process_namespace: ::std::option::Option<bool>,
 }
@@ -45,25 +45,25 @@
     /// <p>Displays the reference pointer to the Kubernetes secret resource. These secrets help to gain access to pull an images from a private registry.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.image_pull_secrets.is_none()`.
-    pub fn image_pull_secrets(&self) -> &[crate::types::ImagePullSecret] {
+    pub fn image_pull_secrets(&self) -> &[super::types::ImagePullSecret] {
         self.image_pull_secrets.as_deref().unwrap_or_default()
     }
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.containers.is_none()`.
-    pub fn containers(&self) -> &[crate::types::EksContainerDetail] {
+    pub fn containers(&self) -> &[super::types::EksContainerDetail] {
         self.containers.as_deref().unwrap_or_default()
     }
     /// <p>The container registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.init_containers.is_none()`.
-    pub fn init_containers(&self) -> &[crate::types::EksContainerDetail] {
+    pub fn init_containers(&self) -> &[super::types::EksContainerDetail] {
         self.init_containers.as_deref().unwrap_or_default()
     }
     /// <p>Specifies the volumes for a job definition using Amazon EKS resources.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.volumes.is_none()`.
-    pub fn volumes(&self) -> &[crate::types::EksVolume] {
+    pub fn volumes(&self) -> &[super::types::EksVolume] {
         self.volumes.as_deref().unwrap_or_default()
     }
     /// <p>The name of the pod for this job.</p>
@@ -75,7 +75,7 @@
         self.node_name.as_deref()
     }
     /// <p>Describes and uniquely identifies Kubernetes resources. For example, the compute environment that a pod runs in or the <code>jobID</code> for a job running in the pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn metadata(&self) -> ::std::option::Option<&crate::types::EksMetadata> {
+    pub fn metadata(&self) -> ::std::option::Option<&super::types::EksMetadata> {
         self.metadata.as_ref()
     }
     /// <p>Indicates if the processes in a container are shared, or visible, to other containers in the same pod. For more information, see <a href="https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/">Share Process Namespace between Containers in a Pod</a>.</p>
@@ -85,8 +85,8 @@
 }
 impl EksPodPropertiesDetail {
     /// Creates a new builder-style object to manufacture [`EksPodPropertiesDetail`](crate::types::EksPodPropertiesDetail).
-    pub fn builder() -> crate::types::builders::EksPodPropertiesDetailBuilder {
-        crate::types::builders::EksPodPropertiesDetailBuilder::default()
+    pub fn builder() -> super::types::builders::EksPodPropertiesDetailBuilder {
+        super::types::builders::EksPodPropertiesDetailBuilder::default()
     }
 }

@@ -97,13 +97,13 @@
     pub(crate) service_account_name: ::std::option::Option<::std::string::String>,
     pub(crate) host_network: ::std::option::Option<bool>,
     pub(crate) dns_policy: ::std::option::Option<::std::string::String>,
-    pub(crate) image_pull_secrets: ::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>>,
-    pub(crate) containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>>,
-    pub(crate) init_containers: ::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>>,
-    pub(crate) volumes: ::std::option::Option<::std::vec::Vec<crate::types::EksVolume>>,
+    pub(crate) image_pull_secrets: ::std::option::Option<::std::vec::Vec<super::types::ImagePullSecret>>,
+    pub(crate) containers: ::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>>,
+    pub(crate) init_containers: ::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>>,
+    pub(crate) volumes: ::std::option::Option<::std::vec::Vec<super::types::EksVolume>>,
     pub(crate) pod_name: ::std::option::Option<::std::string::String>,
     pub(crate) node_name: ::std::option::Option<::std::string::String>,
-    pub(crate) metadata: ::std::option::Option<crate::types::EksMetadata>,
+    pub(crate) metadata: ::std::option::Option<super::types::EksMetadata>,
     pub(crate) share_process_namespace: ::std::option::Option<bool>,
 }
 impl EksPodPropertiesDetailBuilder {
@@ -157,7 +157,7 @@
     /// To override the contents of this collection use [`set_image_pull_secrets`](Self::set_image_pull_secrets).
     ///
     /// <p>Displays the reference pointer to the Kubernetes secret resource. These secrets help to gain access to pull an images from a private registry.</p>
-    pub fn image_pull_secrets(mut self, input: crate::types::ImagePullSecret) -> Self {
+    pub fn image_pull_secrets(mut self, input: super::types::ImagePullSecret) -> Self {
         let mut v = self.image_pull_secrets.unwrap_or_default();
         v.push(input);
         self.image_pull_secrets = ::std::option::Option::Some(v);
@@ -164,12 +164,12 @@
         self
     }
     /// <p>Displays the reference pointer to the Kubernetes secret resource. These secrets help to gain access to pull an images from a private registry.</p>
-    pub fn set_image_pull_secrets(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>>) -> Self {
+    pub fn set_image_pull_secrets(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ImagePullSecret>>) -> Self {
         self.image_pull_secrets = input;
         self
     }
     /// <p>Displays the reference pointer to the Kubernetes secret resource. These secrets help to gain access to pull an images from a private registry.</p>
-    pub fn get_image_pull_secrets(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ImagePullSecret>> {
+    pub fn get_image_pull_secrets(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ImagePullSecret>> {
         &self.image_pull_secrets
     }
     /// Appends an item to `containers`.
@@ -177,7 +177,7 @@
     /// To override the contents of this collection use [`set_containers`](Self::set_containers).
     ///
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p>
-    pub fn containers(mut self, input: crate::types::EksContainerDetail) -> Self {
+    pub fn containers(mut self, input: super::types::EksContainerDetail) -> Self {
         let mut v = self.containers.unwrap_or_default();
         v.push(input);
         self.containers = ::std::option::Option::Some(v);
@@ -184,12 +184,12 @@
         self
     }
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p>
-    pub fn set_containers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>>) -> Self {
+    pub fn set_containers(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>>) -> Self {
         self.containers = input;
         self
     }
     /// <p>The properties of the container that's used on the Amazon EKS pod.</p>
-    pub fn get_containers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>> {
+    pub fn get_containers(&self) -> &::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>> {
         &self.containers
     }
     /// Appends an item to `init_containers`.
@@ -197,7 +197,7 @@
     /// To override the contents of this collection use [`set_init_containers`](Self::set_init_containers).
     ///
     /// <p>The container registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store.</p>
-    pub fn init_containers(mut self, input: crate::types::EksContainerDetail) -> Self {
+    pub fn init_containers(mut self, input: super::types::EksContainerDetail) -> Self {
         let mut v = self.init_containers.unwrap_or_default();
         v.push(input);
         self.init_containers = ::std::option::Option::Some(v);
@@ -204,12 +204,12 @@
         self
     }
     /// <p>The container registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store.</p>
-    pub fn set_init_containers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>>) -> Self {
+    pub fn set_init_containers(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>>) -> Self {
         self.init_containers = input;
         self
     }
     /// <p>The container registered with the Amazon EKS Connector agent and persists the registration information in the Kubernetes backend data store.</p>
-    pub fn get_init_containers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EksContainerDetail>> {
+    pub fn get_init_containers(&self) -> &::std::option::Option<::std::vec::Vec<super::types::EksContainerDetail>> {
         &self.init_containers
     }
     /// Appends an item to `volumes`.
@@ -217,7 +217,7 @@
     /// To override the contents of this collection use [`set_volumes`](Self::set_volumes).
     ///
     /// <p>Specifies the volumes for a job definition using Amazon EKS resources.</p>
-    pub fn volumes(mut self, input: crate::types::EksVolume) -> Self {
+    pub fn volumes(mut self, input: super::types::EksVolume) -> Self {
         let mut v = self.volumes.unwrap_or_default();
         v.push(input);
         self.volumes = ::std::option::Option::Some(v);
@@ -224,12 +224,12 @@
         self
     }
     /// <p>Specifies the volumes for a job definition using Amazon EKS resources.</p>
-    pub fn set_volumes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EksVolume>>) -> Self {
+    pub fn set_volumes(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::EksVolume>>) -> Self {
         self.volumes = input;
         self
     }
     /// <p>Specifies the volumes for a job definition using Amazon EKS resources.</p>
-    pub fn get_volumes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EksVolume>> {
+    pub fn get_volumes(&self) -> &::std::option::Option<::std::vec::Vec<super::types::EksVolume>> {
         &self.volumes
     }
     /// <p>The name of the pod for this job.</p>
@@ -261,17 +261,17 @@
         &self.node_name
     }
     /// <p>Describes and uniquely identifies Kubernetes resources. For example, the compute environment that a pod runs in or the <code>jobID</code> for a job running in the pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn metadata(mut self, input: crate::types::EksMetadata) -> Self {
+    pub fn metadata(mut self, input: super::types::EksMetadata) -> Self {
         self.metadata = ::std::option::Option::Some(input);
         self
     }
     /// <p>Describes and uniquely identifies Kubernetes resources. For example, the compute environment that a pod runs in or the <code>jobID</code> for a job running in the pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn set_metadata(mut self, input: ::std::option::Option<crate::types::EksMetadata>) -> Self {
+    pub fn set_metadata(mut self, input: ::std::option::Option<super::types::EksMetadata>) -> Self {
         self.metadata = input;
         self
     }
     /// <p>Describes and uniquely identifies Kubernetes resources. For example, the compute environment that a pod runs in or the <code>jobID</code> for a job running in the pod. For more information, see <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/">Understanding Kubernetes Objects</a> in the <i>Kubernetes documentation</i>.</p>
-    pub fn get_metadata(&self) -> &::std::option::Option<crate::types::EksMetadata> {
+    pub fn get_metadata(&self) -> &::std::option::Option<super::types::EksMetadata> {
         &self.metadata
     }
     /// <p>Indicates if the processes in a container are shared, or visible, to other containers in the same pod. For more information, see <a href="https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/">Share Process Namespace between Containers in a Pod</a>.</p>
@@ -289,8 +289,8 @@
         &self.share_process_namespace
     }
     /// Consumes the builder and constructs a [`EksPodPropertiesDetail`](crate::types::EksPodPropertiesDetail).
-    pub fn build(self) -> crate::types::EksPodPropertiesDetail {
-        crate::types::EksPodPropertiesDetail {
+    pub fn build(self) -> super::types::EksPodPropertiesDetail {
+        super::types::EksPodPropertiesDetail {
             service_account_name: self.service_account_name,
             host_network: self.host_network,
             dns_policy: self.dns_policy,
