--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl EphemeralStorage {
     /// Creates a new builder-style object to manufacture [`EphemeralStorage`](crate::types::EphemeralStorage).
-    pub fn builder() -> crate::types::builders::EphemeralStorageBuilder {
-        crate::types::builders::EphemeralStorageBuilder::default()
+    pub fn builder() -> super::types::builders::EphemeralStorageBuilder {
+        super::types::builders::EphemeralStorageBuilder::default()
     }
 }

@@ -43,8 +43,8 @@
         &self.size_in_gib
     }
     /// Consumes the builder and constructs a [`EphemeralStorage`](crate::types::EphemeralStorage).
-    pub fn build(self) -> crate::types::EphemeralStorage {
-        crate::types::EphemeralStorage {
+    pub fn build(self) -> super::types::EphemeralStorage {
+        super::types::EphemeralStorage {
             size_in_gib: self.size_in_gib,
         }
     }
