--- original
+++ modified
@@ -12,7 +12,7 @@
     /// <p>The string can contain up to 512 characters.</p>
     pub on_exit_code: ::std::option::Option<::std::string::String>,
     /// <p>Specifies the action to take if all of the specified conditions (<code>onStatusReason</code>, <code>onReason</code>, and <code>onExitCode</code>) are met. The values aren't case sensitive.</p>
-    pub action: ::std::option::Option<crate::types::RetryAction>,
+    pub action: ::std::option::Option<super::types::RetryAction>,
 }
 impl EvaluateOnExit {
     /// <p>Contains a glob pattern to match against the <code>StatusReason</code> returned for a job. The pattern can contain up to 512 characters. It can contain letters, numbers, periods (.), colons (:), and white spaces (including spaces or tabs). It can optionally end with an asterisk (*) so that only the start of the string needs to be an exact match.</p>
@@ -29,14 +29,14 @@
         self.on_exit_code.as_deref()
     }
     /// <p>Specifies the action to take if all of the specified conditions (<code>onStatusReason</code>, <code>onReason</code>, and <code>onExitCode</code>) are met. The values aren't case sensitive.</p>
-    pub fn action(&self) -> ::std::option::Option<&crate::types::RetryAction> {
+    pub fn action(&self) -> ::std::option::Option<&super::types::RetryAction> {
         self.action.as_ref()
     }
 }
 impl EvaluateOnExit {
     /// Creates a new builder-style object to manufacture [`EvaluateOnExit`](crate::types::EvaluateOnExit).
-    pub fn builder() -> crate::types::builders::EvaluateOnExitBuilder {
-        crate::types::builders::EvaluateOnExitBuilder::default()
+    pub fn builder() -> super::types::builders::EvaluateOnExitBuilder {
+        super::types::builders::EvaluateOnExitBuilder::default()
     }
 }

@@ -47,7 +47,7 @@
     pub(crate) on_status_reason: ::std::option::Option<::std::string::String>,
     pub(crate) on_reason: ::std::option::Option<::std::string::String>,
     pub(crate) on_exit_code: ::std::option::Option<::std::string::String>,
-    pub(crate) action: ::std::option::Option<crate::types::RetryAction>,
+    pub(crate) action: ::std::option::Option<super::types::RetryAction>,
 }
 impl EvaluateOnExitBuilder {
     /// <p>Contains a glob pattern to match against the <code>StatusReason</code> returned for a job. The pattern can contain up to 512 characters. It can contain letters, numbers, periods (.), colons (:), and white spaces (including spaces or tabs). It can optionally end with an asterisk (*) so that only the start of the string needs to be an exact match.</p>
@@ -97,22 +97,22 @@
     }
     /// <p>Specifies the action to take if all of the specified conditions (<code>onStatusReason</code>, <code>onReason</code>, and <code>onExitCode</code>) are met. The values aren't case sensitive.</p>
     /// This field is required.
-    pub fn action(mut self, input: crate::types::RetryAction) -> Self {
+    pub fn action(mut self, input: super::types::RetryAction) -> Self {
         self.action = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the action to take if all of the specified conditions (<code>onStatusReason</code>, <code>onReason</code>, and <code>onExitCode</code>) are met. The values aren't case sensitive.</p>
-    pub fn set_action(mut self, input: ::std::option::Option<crate::types::RetryAction>) -> Self {
+    pub fn set_action(mut self, input: ::std::option::Option<super::types::RetryAction>) -> Self {
         self.action = input;
         self
     }
     /// <p>Specifies the action to take if all of the specified conditions (<code>onStatusReason</code>, <code>onReason</code>, and <code>onExitCode</code>) are met. The values aren't case sensitive.</p>
-    pub fn get_action(&self) -> &::std::option::Option<crate::types::RetryAction> {
+    pub fn get_action(&self) -> &::std::option::Option<super::types::RetryAction> {
         &self.action
     }
     /// Consumes the builder and constructs a [`EvaluateOnExit`](crate::types::EvaluateOnExit).
-    pub fn build(self) -> crate::types::EvaluateOnExit {
-        crate::types::EvaluateOnExit {
+    pub fn build(self) -> super::types::EvaluateOnExit {
+        super::types::EvaluateOnExit {
             on_status_reason: self.on_status_reason,
             on_reason: self.on_reason,
             on_exit_code: self.on_exit_code,
