--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl FargatePlatformConfiguration {
     /// Creates a new builder-style object to manufacture [`FargatePlatformConfiguration`](crate::types::FargatePlatformConfiguration).
-    pub fn builder() -> crate::types::builders::FargatePlatformConfigurationBuilder {
-        crate::types::builders::FargatePlatformConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::FargatePlatformConfigurationBuilder {
+        super::super::types::builders::FargatePlatformConfigurationBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.platform_version
     }
     /// Consumes the builder and constructs a [`FargatePlatformConfiguration`](crate::types::FargatePlatformConfiguration).
-    pub fn build(self) -> crate::types::FargatePlatformConfiguration {
-        crate::types::FargatePlatformConfiguration {
+    pub fn build(self) -> super::super::types::FargatePlatformConfiguration {
+        super::super::types::FargatePlatformConfiguration {
             platform_version: self.platform_version,
         }
     }
