--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct FirelensConfiguration {
     /// <p>The log router to use. The valid values are <code>fluentd</code> or <code>fluentbit</code>.</p>
-    pub r#type: ::std::option::Option<crate::types::FirelensConfigurationType>,
+    pub r#type: ::std::option::Option<super::types::FirelensConfigurationType>,
     /// <p>The options to use when configuring the log router. This field is optional and can be used to specify a custom configuration file or to add additional metadata, such as the task, task definition, cluster, and container instance details to the log event. If specified, the syntax to use is <code>"options":{"enable-ecs-log-metadata":"true|false","config-file-type:"s3|file","config-file-value":"arn:aws:s3:::mybucket/fluent.conf|filepath"}</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html#firelens-taskdef">Creating a task definition that uses a FireLens configuration</a> in the <i>Amazon Elastic Container Service Developer Guide</i>.</p>
     pub options: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
 impl FirelensConfiguration {
     /// <p>The log router to use. The valid values are <code>fluentd</code> or <code>fluentbit</code>.</p>
-    pub fn r#type(&self) -> ::std::option::Option<&crate::types::FirelensConfigurationType> {
+    pub fn r#type(&self) -> ::std::option::Option<&super::types::FirelensConfigurationType> {
         self.r#type.as_ref()
     }
     /// <p>The options to use when configuring the log router. This field is optional and can be used to specify a custom configuration file or to add additional metadata, such as the task, task definition, cluster, and container instance details to the log event. If specified, the syntax to use is <code>"options":{"enable-ecs-log-metadata":"true|false","config-file-type:"s3|file","config-file-value":"arn:aws:s3:::mybucket/fluent.conf|filepath"}</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html#firelens-taskdef">Creating a task definition that uses a FireLens configuration</a> in the <i>Amazon Elastic Container Service Developer Guide</i>.</p>
@@ -21,8 +21,8 @@
 }
 impl FirelensConfiguration {
     /// Creates a new builder-style object to manufacture [`FirelensConfiguration`](crate::types::FirelensConfiguration).
-    pub fn builder() -> crate::types::builders::FirelensConfigurationBuilder {
-        crate::types::builders::FirelensConfigurationBuilder::default()
+    pub fn builder() -> super::types::builders::FirelensConfigurationBuilder {
+        super::types::builders::FirelensConfigurationBuilder::default()
     }
 }

@@ -30,23 +30,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct FirelensConfigurationBuilder {
-    pub(crate) r#type: ::std::option::Option<crate::types::FirelensConfigurationType>,
+    pub(crate) r#type: ::std::option::Option<super::types::FirelensConfigurationType>,
     pub(crate) options: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
 impl FirelensConfigurationBuilder {
     /// <p>The log router to use. The valid values are <code>fluentd</code> or <code>fluentbit</code>.</p>
     /// This field is required.
-    pub fn r#type(mut self, input: crate::types::FirelensConfigurationType) -> Self {
+    pub fn r#type(mut self, input: super::types::FirelensConfigurationType) -> Self {
         self.r#type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The log router to use. The valid values are <code>fluentd</code> or <code>fluentbit</code>.</p>
-    pub fn set_type(mut self, input: ::std::option::Option<crate::types::FirelensConfigurationType>) -> Self {
+    pub fn set_type(mut self, input: ::std::option::Option<super::types::FirelensConfigurationType>) -> Self {
         self.r#type = input;
         self
     }
     /// <p>The log router to use. The valid values are <code>fluentd</code> or <code>fluentbit</code>.</p>
-    pub fn get_type(&self) -> &::std::option::Option<crate::types::FirelensConfigurationType> {
+    pub fn get_type(&self) -> &::std::option::Option<super::types::FirelensConfigurationType> {
         &self.r#type
     }
     /// Adds a key-value pair to `options`.
@@ -70,8 +70,8 @@
         &self.options
     }
     /// Consumes the builder and constructs a [`FirelensConfiguration`](crate::types::FirelensConfiguration).
-    pub fn build(self) -> crate::types::FirelensConfiguration {
-        crate::types::FirelensConfiguration {
+    pub fn build(self) -> super::types::FirelensConfiguration {
+        super::types::FirelensConfiguration {
             r#type: self.r#type,
             options: self.options,
         }
