--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct FrontOfQueueDetail {
     /// <p>The Amazon Resource Names (ARNs) of the first 100 <code>RUNNABLE</code> jobs in a named job queue. For first-in-first-out (FIFO) job queues, jobs are ordered based on their submission time. For fair-share scheduling (FSS) job queues, jobs are ordered based on their job priority and share usage.</p>
-    pub jobs: ::std::option::Option<::std::vec::Vec<crate::types::FrontOfQueueJobSummary>>,
+    pub jobs: ::std::option::Option<::std::vec::Vec<super::super::types::FrontOfQueueJobSummary>>,
     /// <p>The Unix timestamp (in milliseconds) for when each of the first 100 <code>RUNNABLE</code> jobs were last updated.</p>
     pub last_updated_at: ::std::option::Option<i64>,
 }
@@ -13,7 +13,7 @@
     /// <p>The Amazon Resource Names (ARNs) of the first 100 <code>RUNNABLE</code> jobs in a named job queue. For first-in-first-out (FIFO) job queues, jobs are ordered based on their submission time. For fair-share scheduling (FSS) job queues, jobs are ordered based on their job priority and share usage.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.jobs.is_none()`.
-    pub fn jobs(&self) -> &[crate::types::FrontOfQueueJobSummary] {
+    pub fn jobs(&self) -> &[super::super::types::FrontOfQueueJobSummary] {
         self.jobs.as_deref().unwrap_or_default()
     }
     /// <p>The Unix timestamp (in milliseconds) for when each of the first 100 <code>RUNNABLE</code> jobs were last updated.</p>
@@ -23,8 +23,8 @@
 }
 impl FrontOfQueueDetail {
     /// Creates a new builder-style object to manufacture [`FrontOfQueueDetail`](crate::types::FrontOfQueueDetail).
-    pub fn builder() -> crate::types::builders::FrontOfQueueDetailBuilder {
-        crate::types::builders::FrontOfQueueDetailBuilder::default()
+    pub fn builder() -> super::super::types::builders::FrontOfQueueDetailBuilder {
+        super::super::types::builders::FrontOfQueueDetailBuilder::default()
     }
 }

@@ -32,7 +32,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct FrontOfQueueDetailBuilder {
-    pub(crate) jobs: ::std::option::Option<::std::vec::Vec<crate::types::FrontOfQueueJobSummary>>,
+    pub(crate) jobs: ::std::option::Option<::std::vec::Vec<super::super::types::FrontOfQueueJobSummary>>,
     pub(crate) last_updated_at: ::std::option::Option<i64>,
 }
 impl FrontOfQueueDetailBuilder {
@@ -41,7 +41,7 @@
     /// To override the contents of this collection use [`set_jobs`](Self::set_jobs).
     ///
     /// <p>The Amazon Resource Names (ARNs) of the first 100 <code>RUNNABLE</code> jobs in a named job queue. For first-in-first-out (FIFO) job queues, jobs are ordered based on their submission time. For fair-share scheduling (FSS) job queues, jobs are ordered based on their job priority and share usage.</p>
-    pub fn jobs(mut self, input: crate::types::FrontOfQueueJobSummary) -> Self {
+    pub fn jobs(mut self, input: super::super::types::FrontOfQueueJobSummary) -> Self {
         let mut v = self.jobs.unwrap_or_default();
         v.push(input);
         self.jobs = ::std::option::Option::Some(v);
@@ -48,12 +48,12 @@
         self
     }
     /// <p>The Amazon Resource Names (ARNs) of the first 100 <code>RUNNABLE</code> jobs in a named job queue. For first-in-first-out (FIFO) job queues, jobs are ordered based on their submission time. For fair-share scheduling (FSS) job queues, jobs are ordered based on their job priority and share usage.</p>
-    pub fn set_jobs(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::FrontOfQueueJobSummary>>) -> Self {
+    pub fn set_jobs(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::FrontOfQueueJobSummary>>) -> Self {
         self.jobs = input;
         self
     }
     /// <p>The Amazon Resource Names (ARNs) of the first 100 <code>RUNNABLE</code> jobs in a named job queue. For first-in-first-out (FIFO) job queues, jobs are ordered based on their submission time. For fair-share scheduling (FSS) job queues, jobs are ordered based on their job priority and share usage.</p>
-    pub fn get_jobs(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::FrontOfQueueJobSummary>> {
+    pub fn get_jobs(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::FrontOfQueueJobSummary>> {
         &self.jobs
     }
     /// <p>The Unix timestamp (in milliseconds) for when each of the first 100 <code>RUNNABLE</code> jobs were last updated.</p>
@@ -71,8 +71,8 @@
         &self.last_updated_at
     }
     /// Consumes the builder and constructs a [`FrontOfQueueDetail`](crate::types::FrontOfQueueDetail).
-    pub fn build(self) -> crate::types::FrontOfQueueDetail {
-        crate::types::FrontOfQueueDetail {
+    pub fn build(self) -> super::super::types::FrontOfQueueDetail {
+        super::super::types::FrontOfQueueDetail {
             jobs: self.jobs,
             last_updated_at: self.last_updated_at,
         }
