--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl ImagePullSecret {
     /// Creates a new builder-style object to manufacture [`ImagePullSecret`](crate::types::ImagePullSecret).
-    pub fn builder() -> crate::types::builders::ImagePullSecretBuilder {
-        crate::types::builders::ImagePullSecretBuilder::default()
+    pub fn builder() -> super::super::types::builders::ImagePullSecretBuilder {
+        super::super::types::builders::ImagePullSecretBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
         &self.name
     }
     /// Consumes the builder and constructs a [`ImagePullSecret`](crate::types::ImagePullSecret).
-    pub fn build(self) -> crate::types::ImagePullSecret {
-        crate::types::ImagePullSecret { name: self.name }
+    pub fn build(self) -> super::super::types::ImagePullSecret {
+        super::super::types::ImagePullSecret { name: self.name }
     }
 }
