--- original
+++ modified
@@ -19,8 +19,8 @@
 }
 impl JobTimeout {
     /// Creates a new builder-style object to manufacture [`JobTimeout`](crate::types::JobTimeout).
-    pub fn builder() -> crate::types::builders::JobTimeoutBuilder {
-        crate::types::builders::JobTimeoutBuilder::default()
+    pub fn builder() -> super::types::builders::JobTimeoutBuilder {
+        super::types::builders::JobTimeoutBuilder::default()
     }
 }

@@ -52,8 +52,8 @@
         &self.attempt_duration_seconds
     }
     /// Consumes the builder and constructs a [`JobTimeout`](crate::types::JobTimeout).
-    pub fn build(self) -> crate::types::JobTimeout {
-        crate::types::JobTimeout {
+    pub fn build(self) -> super::types::JobTimeout {
+        super::types::JobTimeout {
             attempt_duration_seconds: self.attempt_duration_seconds,
         }
     }
