--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl JobCapacityUsageSummary {
     /// Creates a new builder-style object to manufacture [`JobCapacityUsageSummary`](crate::types::JobCapacityUsageSummary).
-    pub fn builder() -> crate::types::builders::JobCapacityUsageSummaryBuilder {
-        crate::types::builders::JobCapacityUsageSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::JobCapacityUsageSummaryBuilder {
+        super::types::builders::JobCapacityUsageSummaryBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.quantity
     }
     /// Consumes the builder and constructs a [`JobCapacityUsageSummary`](crate::types::JobCapacityUsageSummary).
-    pub fn build(self) -> crate::types::JobCapacityUsageSummary {
-        crate::types::JobCapacityUsageSummary {
+    pub fn build(self) -> super::types::JobCapacityUsageSummary {
+        super::types::JobCapacityUsageSummary {
             capacity_unit: self.capacity_unit,
             quantity: self.quantity,
         }
