--- original
+++ modified
@@ -38,7 +38,7 @@
     /// </ul>
     pub target_instance_types: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>The EKS node initialization process to use. You only need to specify this value if you are using a custom AMI. The default value is <code>EKS_BOOTSTRAP_SH</code>. If <i>imageType</i> is a custom AMI based on EKS_AL2023 or EKS_AL2023_NVIDIA then you must choose <code>EKS_NODEADM</code>.</p>
-    pub userdata_type: ::std::option::Option<crate::types::UserdataType>,
+    pub userdata_type: ::std::option::Option<super::super::types::UserdataType>,
 }
 impl LaunchTemplateSpecificationOverride {
     /// <p>The ID of the launch template.</p>
@@ -79,14 +79,14 @@
         self.target_instance_types.as_deref().unwrap_or_default()
     }
     /// <p>The EKS node initialization process to use. You only need to specify this value if you are using a custom AMI. The default value is <code>EKS_BOOTSTRAP_SH</code>. If <i>imageType</i> is a custom AMI based on EKS_AL2023 or EKS_AL2023_NVIDIA then you must choose <code>EKS_NODEADM</code>.</p>
-    pub fn userdata_type(&self) -> ::std::option::Option<&crate::types::UserdataType> {
+    pub fn userdata_type(&self) -> ::std::option::Option<&super::super::types::UserdataType> {
         self.userdata_type.as_ref()
     }
 }
 impl LaunchTemplateSpecificationOverride {
     /// Creates a new builder-style object to manufacture [`LaunchTemplateSpecificationOverride`](crate::types::LaunchTemplateSpecificationOverride).
-    pub fn builder() -> crate::types::builders::LaunchTemplateSpecificationOverrideBuilder {
-        crate::types::builders::LaunchTemplateSpecificationOverrideBuilder::default()
+    pub fn builder() -> super::super::types::builders::LaunchTemplateSpecificationOverrideBuilder {
+        super::super::types::builders::LaunchTemplateSpecificationOverrideBuilder::default()
     }
 }

@@ -98,7 +98,7 @@
     pub(crate) launch_template_name: ::std::option::Option<::std::string::String>,
     pub(crate) version: ::std::option::Option<::std::string::String>,
     pub(crate) target_instance_types: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
-    pub(crate) userdata_type: ::std::option::Option<crate::types::UserdataType>,
+    pub(crate) userdata_type: ::std::option::Option<super::super::types::UserdataType>,
 }
 impl LaunchTemplateSpecificationOverrideBuilder {
     /// <p>The ID of the launch template.</p>
@@ -221,22 +221,22 @@
         &self.target_instance_types
     }
     /// <p>The EKS node initialization process to use. You only need to specify this value if you are using a custom AMI. The default value is <code>EKS_BOOTSTRAP_SH</code>. If <i>imageType</i> is a custom AMI based on EKS_AL2023 or EKS_AL2023_NVIDIA then you must choose <code>EKS_NODEADM</code>.</p>
-    pub fn userdata_type(mut self, input: crate::types::UserdataType) -> Self {
+    pub fn userdata_type(mut self, input: super::super::types::UserdataType) -> Self {
         self.userdata_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The EKS node initialization process to use. You only need to specify this value if you are using a custom AMI. The default value is <code>EKS_BOOTSTRAP_SH</code>. If <i>imageType</i> is a custom AMI based on EKS_AL2023 or EKS_AL2023_NVIDIA then you must choose <code>EKS_NODEADM</code>.</p>
-    pub fn set_userdata_type(mut self, input: ::std::option::Option<crate::types::UserdataType>) -> Self {
+    pub fn set_userdata_type(mut self, input: ::std::option::Option<super::super::types::UserdataType>) -> Self {
         self.userdata_type = input;
         self
     }
     /// <p>The EKS node initialization process to use. You only need to specify this value if you are using a custom AMI. The default value is <code>EKS_BOOTSTRAP_SH</code>. If <i>imageType</i> is a custom AMI based on EKS_AL2023 or EKS_AL2023_NVIDIA then you must choose <code>EKS_NODEADM</code>.</p>
-    pub fn get_userdata_type(&self) -> &::std::option::Option<crate::types::UserdataType> {
+    pub fn get_userdata_type(&self) -> &::std::option::Option<super::super::types::UserdataType> {
         &self.userdata_type
     }
     /// Consumes the builder and constructs a [`LaunchTemplateSpecificationOverride`](crate::types::LaunchTemplateSpecificationOverride).
-    pub fn build(self) -> crate::types::LaunchTemplateSpecificationOverride {
-        crate::types::LaunchTemplateSpecificationOverride {
+    pub fn build(self) -> super::super::types::LaunchTemplateSpecificationOverride {
+        super::super::types::LaunchTemplateSpecificationOverride {
             launch_template_id: self.launch_template_id,
             launch_template_name: self.launch_template_name,
             version: self.version,
