--- original
+++ modified
@@ -61,11 +61,11 @@
     /// <p>If you have a custom driver that's not listed earlier that you want to work with the Amazon ECS container agent, you can fork the Amazon ECS container agent project that's <a href="https://github.com/aws/amazon-ecs-agent">available on GitHub</a> and customize it to work with that driver. We encourage you to submit pull requests for changes that you want to have included. However, Amazon Web Services doesn't currently support running modified copies of this software.</p>
     /// </note>
     /// <p>This parameter requires version 1.18 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
-    pub log_driver: ::std::option::Option<crate::types::LogDriver>,
+    pub log_driver: ::std::option::Option<super::super::types::LogDriver>,
     /// <p>The configuration options to send to the log driver. This parameter requires version 1.19 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
     pub options: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     /// <p>The secrets to pass to the log configuration. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/specifying-sensitive-data.html">Specifying sensitive data</a> in the <i>Batch User Guide</i>.</p>
-    pub secret_options: ::std::option::Option<::std::vec::Vec<crate::types::Secret>>,
+    pub secret_options: ::std::option::Option<::std::vec::Vec<super::super::types::Secret>>,
 }
 impl LogConfiguration {
     /// <p>The log driver to use for the container. The valid values that are listed for this parameter are log drivers that the Amazon ECS container agent can communicate with by default.</p>
@@ -125,7 +125,7 @@
     /// <p>If you have a custom driver that's not listed earlier that you want to work with the Amazon ECS container agent, you can fork the Amazon ECS container agent project that's <a href="https://github.com/aws/amazon-ecs-agent">available on GitHub</a> and customize it to work with that driver. We encourage you to submit pull requests for changes that you want to have included. However, Amazon Web Services doesn't currently support running modified copies of this software.</p>
     /// </note>
     /// <p>This parameter requires version 1.18 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
-    pub fn log_driver(&self) -> ::std::option::Option<&crate::types::LogDriver> {
+    pub fn log_driver(&self) -> ::std::option::Option<&super::super::types::LogDriver> {
         self.log_driver.as_ref()
     }
     /// <p>The configuration options to send to the log driver. This parameter requires version 1.19 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
@@ -135,14 +135,14 @@
     /// <p>The secrets to pass to the log configuration. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/specifying-sensitive-data.html">Specifying sensitive data</a> in the <i>Batch User Guide</i>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.secret_options.is_none()`.
-    pub fn secret_options(&self) -> &[crate::types::Secret] {
+    pub fn secret_options(&self) -> &[super::super::types::Secret] {
         self.secret_options.as_deref().unwrap_or_default()
     }
 }
 impl LogConfiguration {
     /// Creates a new builder-style object to manufacture [`LogConfiguration`](crate::types::LogConfiguration).
-    pub fn builder() -> crate::types::builders::LogConfigurationBuilder {
-        crate::types::builders::LogConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::LogConfigurationBuilder {
+        super::super::types::builders::LogConfigurationBuilder::default()
     }
 }

@@ -150,9 +150,9 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct LogConfigurationBuilder {
-    pub(crate) log_driver: ::std::option::Option<crate::types::LogDriver>,
+    pub(crate) log_driver: ::std::option::Option<super::super::types::LogDriver>,
     pub(crate) options: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
-    pub(crate) secret_options: ::std::option::Option<::std::vec::Vec<crate::types::Secret>>,
+    pub(crate) secret_options: ::std::option::Option<::std::vec::Vec<super::super::types::Secret>>,
 }
 impl LogConfigurationBuilder {
     /// <p>The log driver to use for the container. The valid values that are listed for this parameter are log drivers that the Amazon ECS container agent can communicate with by default.</p>
@@ -213,7 +213,7 @@
     /// </note>
     /// <p>This parameter requires version 1.18 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
     /// This field is required.
-    pub fn log_driver(mut self, input: crate::types::LogDriver) -> Self {
+    pub fn log_driver(mut self, input: super::super::types::LogDriver) -> Self {
         self.log_driver = ::std::option::Option::Some(input);
         self
     }
@@ -274,7 +274,7 @@
     /// <p>If you have a custom driver that's not listed earlier that you want to work with the Amazon ECS container agent, you can fork the Amazon ECS container agent project that's <a href="https://github.com/aws/amazon-ecs-agent">available on GitHub</a> and customize it to work with that driver. We encourage you to submit pull requests for changes that you want to have included. However, Amazon Web Services doesn't currently support running modified copies of this software.</p>
     /// </note>
     /// <p>This parameter requires version 1.18 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
-    pub fn set_log_driver(mut self, input: ::std::option::Option<crate::types::LogDriver>) -> Self {
+    pub fn set_log_driver(mut self, input: ::std::option::Option<super::super::types::LogDriver>) -> Self {
         self.log_driver = input;
         self
     }
@@ -335,7 +335,7 @@
     /// <p>If you have a custom driver that's not listed earlier that you want to work with the Amazon ECS container agent, you can fork the Amazon ECS container agent project that's <a href="https://github.com/aws/amazon-ecs-agent">available on GitHub</a> and customize it to work with that driver. We encourage you to submit pull requests for changes that you want to have included. However, Amazon Web Services doesn't currently support running modified copies of this software.</p>
     /// </note>
     /// <p>This parameter requires version 1.18 of the Docker Remote API or greater on your container instance. To check the Docker Remote API version on your container instance, log in to your container instance and run the following command: <code>sudo docker version | grep "Server API version"</code></p>
-    pub fn get_log_driver(&self) -> &::std::option::Option<crate::types::LogDriver> {
+    pub fn get_log_driver(&self) -> &::std::option::Option<super::super::types::LogDriver> {
         &self.log_driver
     }
     /// Adds a key-value pair to `options`.
@@ -363,7 +363,7 @@
     /// To override the contents of this collection use [`set_secret_options`](Self::set_secret_options).
     ///
     /// <p>The secrets to pass to the log configuration. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/specifying-sensitive-data.html">Specifying sensitive data</a> in the <i>Batch User Guide</i>.</p>
-    pub fn secret_options(mut self, input: crate::types::Secret) -> Self {
+    pub fn secret_options(mut self, input: super::super::types::Secret) -> Self {
         let mut v = self.secret_options.unwrap_or_default();
         v.push(input);
         self.secret_options = ::std::option::Option::Some(v);
@@ -370,17 +370,17 @@
         self
     }
     /// <p>The secrets to pass to the log configuration. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/specifying-sensitive-data.html">Specifying sensitive data</a> in the <i>Batch User Guide</i>.</p>
-    pub fn set_secret_options(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Secret>>) -> Self {
+    pub fn set_secret_options(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::Secret>>) -> Self {
         self.secret_options = input;
         self
     }
     /// <p>The secrets to pass to the log configuration. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/specifying-sensitive-data.html">Specifying sensitive data</a> in the <i>Batch User Guide</i>.</p>
-    pub fn get_secret_options(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Secret>> {
+    pub fn get_secret_options(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::Secret>> {
         &self.secret_options
     }
     /// Consumes the builder and constructs a [`LogConfiguration`](crate::types::LogConfiguration).
-    pub fn build(self) -> crate::types::LogConfiguration {
-        crate::types::LogConfiguration {
+    pub fn build(self) -> super::super::types::LogConfiguration {
+        super::super::types::LogConfiguration {
             log_driver: self.log_driver,
             options: self.options,
             secret_options: self.secret_options,
