--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl MountPoint {
     /// Creates a new builder-style object to manufacture [`MountPoint`](crate::types::MountPoint).
-    pub fn builder() -> crate::types::builders::MountPointBuilder {
-        crate::types::builders::MountPointBuilder::default()
+    pub fn builder() -> super::super::types::builders::MountPointBuilder {
+        super::super::types::builders::MountPointBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.source_volume
     }
     /// Consumes the builder and constructs a [`MountPoint`](crate::types::MountPoint).
-    pub fn build(self) -> crate::types::MountPoint {
-        crate::types::MountPoint {
+    pub fn build(self) -> super::super::types::MountPoint {
+        super::super::types::MountPoint {
             container_path: self.container_path,
             read_only: self.read_only,
             source_volume: self.source_volume,
