--- original
+++ modified
@@ -17,7 +17,7 @@
     /// </ul>
     pub num_nodes: ::std::option::Option<i32>,
     /// <p>The node property overrides for the job.</p>
-    pub node_property_overrides: ::std::option::Option<::std::vec::Vec<crate::types::NodePropertyOverride>>,
+    pub node_property_overrides: ::std::option::Option<::std::vec::Vec<super::super::types::NodePropertyOverride>>,
 }
 impl NodeOverrides {
     /// <p>The number of nodes to use with a multi-node parallel job. This value overrides the number of nodes that are specified in the job definition. To use this override, you must meet the following conditions:</p>
@@ -35,14 +35,14 @@
     /// <p>The node property overrides for the job.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.node_property_overrides.is_none()`.
-    pub fn node_property_overrides(&self) -> &[crate::types::NodePropertyOverride] {
+    pub fn node_property_overrides(&self) -> &[super::super::types::NodePropertyOverride] {
         self.node_property_overrides.as_deref().unwrap_or_default()
     }
 }
 impl NodeOverrides {
     /// Creates a new builder-style object to manufacture [`NodeOverrides`](crate::types::NodeOverrides).
-    pub fn builder() -> crate::types::builders::NodeOverridesBuilder {
-        crate::types::builders::NodeOverridesBuilder::default()
+    pub fn builder() -> super::super::types::builders::NodeOverridesBuilder {
+        super::super::types::builders::NodeOverridesBuilder::default()
     }
 }

@@ -51,7 +51,7 @@
 #[non_exhaustive]
 pub struct NodeOverridesBuilder {
     pub(crate) num_nodes: ::std::option::Option<i32>,
-    pub(crate) node_property_overrides: ::std::option::Option<::std::vec::Vec<crate::types::NodePropertyOverride>>,
+    pub(crate) node_property_overrides: ::std::option::Option<::std::vec::Vec<super::super::types::NodePropertyOverride>>,
 }
 impl NodeOverridesBuilder {
     /// <p>The number of nodes to use with a multi-node parallel job. This value overrides the number of nodes that are specified in the job definition. To use this override, you must meet the following conditions:</p>
@@ -97,7 +97,7 @@
     /// To override the contents of this collection use [`set_node_property_overrides`](Self::set_node_property_overrides).
     ///
     /// <p>The node property overrides for the job.</p>
-    pub fn node_property_overrides(mut self, input: crate::types::NodePropertyOverride) -> Self {
+    pub fn node_property_overrides(mut self, input: super::super::types::NodePropertyOverride) -> Self {
         let mut v = self.node_property_overrides.unwrap_or_default();
         v.push(input);
         self.node_property_overrides = ::std::option::Option::Some(v);
@@ -104,17 +104,17 @@
         self
     }
     /// <p>The node property overrides for the job.</p>
-    pub fn set_node_property_overrides(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::NodePropertyOverride>>) -> Self {
+    pub fn set_node_property_overrides(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::NodePropertyOverride>>) -> Self {
         self.node_property_overrides = input;
         self
     }
     /// <p>The node property overrides for the job.</p>
-    pub fn get_node_property_overrides(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::NodePropertyOverride>> {
+    pub fn get_node_property_overrides(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::NodePropertyOverride>> {
         &self.node_property_overrides
     }
     /// Consumes the builder and constructs a [`NodeOverrides`](crate::types::NodeOverrides).
-    pub fn build(self) -> crate::types::NodeOverrides {
-        crate::types::NodeOverrides {
+    pub fn build(self) -> super::super::types::NodeOverrides {
+        super::super::types::NodeOverrides {
             num_nodes: self.num_nodes,
             node_property_overrides: self.node_property_overrides,
         }
