--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl NodePropertiesSummary {
     /// Creates a new builder-style object to manufacture [`NodePropertiesSummary`](crate::types::NodePropertiesSummary).
-    pub fn builder() -> crate::types::builders::NodePropertiesSummaryBuilder {
-        crate::types::builders::NodePropertiesSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::NodePropertiesSummaryBuilder {
+        super::types::builders::NodePropertiesSummaryBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.node_index
     }
     /// Consumes the builder and constructs a [`NodePropertiesSummary`](crate::types::NodePropertiesSummary).
-    pub fn build(self) -> crate::types::NodePropertiesSummary {
-        crate::types::NodePropertiesSummary {
+    pub fn build(self) -> super::types::NodePropertiesSummary {
+        super::types::NodePropertiesSummary {
             is_main_node: self.is_main_node,
             num_nodes: self.num_nodes,
             node_index: self.node_index,
