--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The range of nodes, using node index values. A range of <code>0:3</code> indicates nodes with index values of <code>0</code> through <code>3</code>. If the starting range value is omitted (<code>:n</code>), then <code>0</code> is used to start the range. If the ending range value is omitted (<code>n:</code>), then the highest possible node index is used to end the range. Your accumulative node ranges must account for all nodes (<code>0:n</code>). You can nest node ranges (for example, <code>0:10</code> and <code>4:5</code>). In this case, the <code>4:5</code> range properties override the <code>0:10</code> properties.</p>
     pub target_nodes: ::std::option::Option<::std::string::String>,
     /// <p>The container details for the node range.</p>
-    pub container: ::std::option::Option<crate::types::ContainerProperties>,
+    pub container: ::std::option::Option<super::super::types::ContainerProperties>,
     /// <p>The instance types of the underlying host infrastructure of a multi-node parallel job.</p><note>
     /// <p>This parameter isn't applicable to jobs that are running on Fargate resources.</p>
     /// <p>In addition, this list object is currently limited to one element.</p>
@@ -14,11 +14,11 @@
     /// </note>
     pub instance_types: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub ecs_properties: ::std::option::Option<crate::types::EcsProperties>,
+    pub ecs_properties: ::std::option::Option<super::super::types::EcsProperties>,
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub eks_properties: ::std::option::Option<crate::types::EksProperties>,
+    pub eks_properties: ::std::option::Option<super::super::types::EksProperties>,
     /// <p>Contains a list of consumable resources required by a job.</p>
-    pub consumable_resource_properties: ::std::option::Option<crate::types::ConsumableResourceProperties>,
+    pub consumable_resource_properties: ::std::option::Option<super::super::types::ConsumableResourceProperties>,
 }
 impl NodeRangeProperty {
     /// <p>The range of nodes, using node index values. A range of <code>0:3</code> indicates nodes with index values of <code>0</code> through <code>3</code>. If the starting range value is omitted (<code>:n</code>), then <code>0</code> is used to start the range. If the ending range value is omitted (<code>n:</code>), then the highest possible node index is used to end the range. Your accumulative node ranges must account for all nodes (<code>0:n</code>). You can nest node ranges (for example, <code>0:10</code> and <code>4:5</code>). In this case, the <code>4:5</code> range properties override the <code>0:10</code> properties.</p>
@@ -26,7 +26,7 @@
         self.target_nodes.as_deref()
     }
     /// <p>The container details for the node range.</p>
-    pub fn container(&self) -> ::std::option::Option<&crate::types::ContainerProperties> {
+    pub fn container(&self) -> ::std::option::Option<&super::super::types::ContainerProperties> {
         self.container.as_ref()
     }
     /// <p>The instance types of the underlying host infrastructure of a multi-node parallel job.</p><note>
@@ -39,22 +39,22 @@
         self.instance_types.as_deref().unwrap_or_default()
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn ecs_properties(&self) -> ::std::option::Option<&crate::types::EcsProperties> {
+    pub fn ecs_properties(&self) -> ::std::option::Option<&super::super::types::EcsProperties> {
         self.ecs_properties.as_ref()
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn eks_properties(&self) -> ::std::option::Option<&crate::types::EksProperties> {
+    pub fn eks_properties(&self) -> ::std::option::Option<&super::super::types::EksProperties> {
         self.eks_properties.as_ref()
     }
     /// <p>Contains a list of consumable resources required by a job.</p>
-    pub fn consumable_resource_properties(&self) -> ::std::option::Option<&crate::types::ConsumableResourceProperties> {
+    pub fn consumable_resource_properties(&self) -> ::std::option::Option<&super::super::types::ConsumableResourceProperties> {
         self.consumable_resource_properties.as_ref()
     }
 }
 impl NodeRangeProperty {
     /// Creates a new builder-style object to manufacture [`NodeRangeProperty`](crate::types::NodeRangeProperty).
-    pub fn builder() -> crate::types::builders::NodeRangePropertyBuilder {
-        crate::types::builders::NodeRangePropertyBuilder::default()
+    pub fn builder() -> super::super::types::builders::NodeRangePropertyBuilder {
+        super::super::types::builders::NodeRangePropertyBuilder::default()
     }
 }

@@ -63,11 +63,11 @@
 #[non_exhaustive]
 pub struct NodeRangePropertyBuilder {
     pub(crate) target_nodes: ::std::option::Option<::std::string::String>,
-    pub(crate) container: ::std::option::Option<crate::types::ContainerProperties>,
+    pub(crate) container: ::std::option::Option<super::super::types::ContainerProperties>,
     pub(crate) instance_types: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
-    pub(crate) ecs_properties: ::std::option::Option<crate::types::EcsProperties>,
-    pub(crate) eks_properties: ::std::option::Option<crate::types::EksProperties>,
-    pub(crate) consumable_resource_properties: ::std::option::Option<crate::types::ConsumableResourceProperties>,
+    pub(crate) ecs_properties: ::std::option::Option<super::super::types::EcsProperties>,
+    pub(crate) eks_properties: ::std::option::Option<super::super::types::EksProperties>,
+    pub(crate) consumable_resource_properties: ::std::option::Option<super::super::types::ConsumableResourceProperties>,
 }
 impl NodeRangePropertyBuilder {
     /// <p>The range of nodes, using node index values. A range of <code>0:3</code> indicates nodes with index values of <code>0</code> through <code>3</code>. If the starting range value is omitted (<code>:n</code>), then <code>0</code> is used to start the range. If the ending range value is omitted (<code>n:</code>), then the highest possible node index is used to end the range. Your accumulative node ranges must account for all nodes (<code>0:n</code>). You can nest node ranges (for example, <code>0:10</code> and <code>4:5</code>). In this case, the <code>4:5</code> range properties override the <code>0:10</code> properties.</p>
@@ -86,17 +86,17 @@
         &self.target_nodes
     }
     /// <p>The container details for the node range.</p>
-    pub fn container(mut self, input: crate::types::ContainerProperties) -> Self {
+    pub fn container(mut self, input: super::super::types::ContainerProperties) -> Self {
         self.container = ::std::option::Option::Some(input);
         self
     }
     /// <p>The container details for the node range.</p>
-    pub fn set_container(mut self, input: ::std::option::Option<crate::types::ContainerProperties>) -> Self {
+    pub fn set_container(mut self, input: ::std::option::Option<super::super::types::ContainerProperties>) -> Self {
         self.container = input;
         self
     }
     /// <p>The container details for the node range.</p>
-    pub fn get_container(&self) -> &::std::option::Option<crate::types::ContainerProperties> {
+    pub fn get_container(&self) -> &::std::option::Option<super::super::types::ContainerProperties> {
         &self.container
     }
     /// Appends an item to `instance_types`.
@@ -129,50 +129,50 @@
         &self.instance_types
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn ecs_properties(mut self, input: crate::types::EcsProperties) -> Self {
+    pub fn ecs_properties(mut self, input: super::super::types::EcsProperties) -> Self {
         self.ecs_properties = ::std::option::Option::Some(input);
         self
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn set_ecs_properties(mut self, input: ::std::option::Option<crate::types::EcsProperties>) -> Self {
+    pub fn set_ecs_properties(mut self, input: ::std::option::Option<super::super::types::EcsProperties>) -> Self {
         self.ecs_properties = input;
         self
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn get_ecs_properties(&self) -> &::std::option::Option<crate::types::EcsProperties> {
+    pub fn get_ecs_properties(&self) -> &::std::option::Option<super::super::types::EcsProperties> {
         &self.ecs_properties
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn eks_properties(mut self, input: crate::types::EksProperties) -> Self {
+    pub fn eks_properties(mut self, input: super::super::types::EksProperties) -> Self {
         self.eks_properties = ::std::option::Option::Some(input);
         self
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn set_eks_properties(mut self, input: ::std::option::Option<crate::types::EksProperties>) -> Self {
+    pub fn set_eks_properties(mut self, input: ::std::option::Option<super::super::types::EksProperties>) -> Self {
         self.eks_properties = input;
         self
     }
     /// <p>This is an object that represents the properties of the node range for a multi-node parallel job.</p>
-    pub fn get_eks_properties(&self) -> &::std::option::Option<crate::types::EksProperties> {
+    pub fn get_eks_properties(&self) -> &::std::option::Option<super::super::types::EksProperties> {
         &self.eks_properties
     }
     /// <p>Contains a list of consumable resources required by a job.</p>
-    pub fn consumable_resource_properties(mut self, input: crate::types::ConsumableResourceProperties) -> Self {
+    pub fn consumable_resource_properties(mut self, input: super::super::types::ConsumableResourceProperties) -> Self {
         self.consumable_resource_properties = ::std::option::Option::Some(input);
         self
     }
     /// <p>Contains a list of consumable resources required by a job.</p>
-    pub fn set_consumable_resource_properties(mut self, input: ::std::option::Option<crate::types::ConsumableResourceProperties>) -> Self {
+    pub fn set_consumable_resource_properties(mut self, input: ::std::option::Option<super::super::types::ConsumableResourceProperties>) -> Self {
         self.consumable_resource_properties = input;
         self
     }
     /// <p>Contains a list of consumable resources required by a job.</p>
-    pub fn get_consumable_resource_properties(&self) -> &::std::option::Option<crate::types::ConsumableResourceProperties> {
+    pub fn get_consumable_resource_properties(&self) -> &::std::option::Option<super::super::types::ConsumableResourceProperties> {
         &self.consumable_resource_properties
     }
     /// Consumes the builder and constructs a [`NodeRangeProperty`](crate::types::NodeRangeProperty).
-    pub fn build(self) -> crate::types::NodeRangeProperty {
-        crate::types::NodeRangeProperty {
+    pub fn build(self) -> super::super::types::NodeRangeProperty {
+        super::super::types::NodeRangeProperty {
             target_nodes: self.target_nodes,
             container: self.container,
             instance_types: self.instance_types,
