--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl QueueSnapshotCapacityUsage {
     /// Creates a new builder-style object to manufacture [`QueueSnapshotCapacityUsage`](crate::types::QueueSnapshotCapacityUsage).
-    pub fn builder() -> crate::types::builders::QueueSnapshotCapacityUsageBuilder {
-        crate::types::builders::QueueSnapshotCapacityUsageBuilder::default()
+    pub fn builder() -> super::types::builders::QueueSnapshotCapacityUsageBuilder {
+        super::types::builders::QueueSnapshotCapacityUsageBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.quantity
     }
     /// Consumes the builder and constructs a [`QueueSnapshotCapacityUsage`](crate::types::QueueSnapshotCapacityUsage).
-    pub fn build(self) -> crate::types::QueueSnapshotCapacityUsage {
-        crate::types::QueueSnapshotCapacityUsage {
+    pub fn build(self) -> super::types::QueueSnapshotCapacityUsage {
+        super::types::QueueSnapshotCapacityUsage {
             capacity_unit: self.capacity_unit,
             quantity: self.quantity,
         }
