--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct QuotaSharePreemptionConfiguration {
     /// <p>Specifies whether jobs within a quota share can be preempted by another, higher priority job in the same quota share.</p>
-    pub in_share_preemption: ::std::option::Option<crate::types::QuotaShareInSharePreemptionState>,
+    pub in_share_preemption: ::std::option::Option<super::types::QuotaShareInSharePreemptionState>,
 }
 impl QuotaSharePreemptionConfiguration {
     /// <p>Specifies whether jobs within a quota share can be preempted by another, higher priority job in the same quota share.</p>
-    pub fn in_share_preemption(&self) -> ::std::option::Option<&crate::types::QuotaShareInSharePreemptionState> {
+    pub fn in_share_preemption(&self) -> ::std::option::Option<&super::types::QuotaShareInSharePreemptionState> {
         self.in_share_preemption.as_ref()
     }
 }
 impl QuotaSharePreemptionConfiguration {
     /// Creates a new builder-style object to manufacture [`QuotaSharePreemptionConfiguration`](crate::types::QuotaSharePreemptionConfiguration).
-    pub fn builder() -> crate::types::builders::QuotaSharePreemptionConfigurationBuilder {
-        crate::types::builders::QuotaSharePreemptionConfigurationBuilder::default()
+    pub fn builder() -> super::types::builders::QuotaSharePreemptionConfigurationBuilder {
+        super::types::builders::QuotaSharePreemptionConfigurationBuilder::default()
     }
 }

@@ -24,27 +24,27 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct QuotaSharePreemptionConfigurationBuilder {
-    pub(crate) in_share_preemption: ::std::option::Option<crate::types::QuotaShareInSharePreemptionState>,
+    pub(crate) in_share_preemption: ::std::option::Option<super::types::QuotaShareInSharePreemptionState>,
 }
 impl QuotaSharePreemptionConfigurationBuilder {
     /// <p>Specifies whether jobs within a quota share can be preempted by another, higher priority job in the same quota share.</p>
     /// This field is required.
-    pub fn in_share_preemption(mut self, input: crate::types::QuotaShareInSharePreemptionState) -> Self {
+    pub fn in_share_preemption(mut self, input: super::types::QuotaShareInSharePreemptionState) -> Self {
         self.in_share_preemption = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether jobs within a quota share can be preempted by another, higher priority job in the same quota share.</p>
-    pub fn set_in_share_preemption(mut self, input: ::std::option::Option<crate::types::QuotaShareInSharePreemptionState>) -> Self {
+    pub fn set_in_share_preemption(mut self, input: ::std::option::Option<super::types::QuotaShareInSharePreemptionState>) -> Self {
         self.in_share_preemption = input;
         self
     }
     /// <p>Specifies whether jobs within a quota share can be preempted by another, higher priority job in the same quota share.</p>
-    pub fn get_in_share_preemption(&self) -> &::std::option::Option<crate::types::QuotaShareInSharePreemptionState> {
+    pub fn get_in_share_preemption(&self) -> &::std::option::Option<super::types::QuotaShareInSharePreemptionState> {
         &self.in_share_preemption
     }
     /// Consumes the builder and constructs a [`QuotaSharePreemptionConfiguration`](crate::types::QuotaSharePreemptionConfiguration).
-    pub fn build(self) -> crate::types::QuotaSharePreemptionConfiguration {
-        crate::types::QuotaSharePreemptionConfiguration {
+    pub fn build(self) -> super::types::QuotaSharePreemptionConfiguration {
+        super::types::QuotaSharePreemptionConfiguration {
             in_share_preemption: self.in_share_preemption,
         }
     }
