--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl RepositoryCredentials {
     /// Creates a new builder-style object to manufacture [`RepositoryCredentials`](crate::types::RepositoryCredentials).
-    pub fn builder() -> crate::types::builders::RepositoryCredentialsBuilder {
-        crate::types::builders::RepositoryCredentialsBuilder::default()
+    pub fn builder() -> super::super::types::builders::RepositoryCredentialsBuilder {
+        super::super::types::builders::RepositoryCredentialsBuilder::default()
     }
 }

@@ -43,8 +43,8 @@
         &self.credentials_parameter
     }
     /// Consumes the builder and constructs a [`RepositoryCredentials`](crate::types::RepositoryCredentials).
-    pub fn build(self) -> crate::types::RepositoryCredentials {
-        crate::types::RepositoryCredentials {
+    pub fn build(self) -> super::super::types::RepositoryCredentials {
+        super::super::types::RepositoryCredentials {
             credentials_parameter: self.credentials_parameter,
         }
     }
