--- original
+++ modified
@@ -9,9 +9,9 @@
     /// <p>The Amazon Resource Name (ARN) of the scheduling policy. An example is <code>arn:<i>aws</i>:batch:<i>us-east-1</i>:<i>123456789012</i>:scheduling-policy/<i>HighPriority</i> </code>.</p>
     pub arn: ::std::option::Option<::std::string::String>,
     /// <p>The quota share scheduling policy details.</p>
-    pub quota_share_policy: ::std::option::Option<crate::types::QuotaSharePolicy>,
+    pub quota_share_policy: ::std::option::Option<super::types::QuotaSharePolicy>,
     /// <p>The fair-share scheduling policy details.</p>
-    pub fairshare_policy: ::std::option::Option<crate::types::FairsharePolicy>,
+    pub fairshare_policy: ::std::option::Option<super::types::FairsharePolicy>,
     /// <p>The tags that you apply to the fair-share scheduling policy to categorize and organize your resources. Each tag consists of a key and an optional value. For more information, see <a href="https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html">Tagging Amazon Web Services resources</a> in <i>Amazon Web Services General Reference</i>.</p>
     pub tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
@@ -25,11 +25,11 @@
         self.arn.as_deref()
     }
     /// <p>The quota share scheduling policy details.</p>
-    pub fn quota_share_policy(&self) -> ::std::option::Option<&crate::types::QuotaSharePolicy> {
+    pub fn quota_share_policy(&self) -> ::std::option::Option<&super::types::QuotaSharePolicy> {
         self.quota_share_policy.as_ref()
     }
     /// <p>The fair-share scheduling policy details.</p>
-    pub fn fairshare_policy(&self) -> ::std::option::Option<&crate::types::FairsharePolicy> {
+    pub fn fairshare_policy(&self) -> ::std::option::Option<&super::types::FairsharePolicy> {
         self.fairshare_policy.as_ref()
     }
     /// <p>The tags that you apply to the fair-share scheduling policy to categorize and organize your resources. Each tag consists of a key and an optional value. For more information, see <a href="https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html">Tagging Amazon Web Services resources</a> in <i>Amazon Web Services General Reference</i>.</p>
@@ -39,8 +39,8 @@
 }
 impl SchedulingPolicyDetail {
     /// Creates a new builder-style object to manufacture [`SchedulingPolicyDetail`](crate::types::SchedulingPolicyDetail).
-    pub fn builder() -> crate::types::builders::SchedulingPolicyDetailBuilder {
-        crate::types::builders::SchedulingPolicyDetailBuilder::default()
+    pub fn builder() -> super::types::builders::SchedulingPolicyDetailBuilder {
+        super::types::builders::SchedulingPolicyDetailBuilder::default()
     }
 }

@@ -50,8 +50,8 @@
 pub struct SchedulingPolicyDetailBuilder {
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) arn: ::std::option::Option<::std::string::String>,
-    pub(crate) quota_share_policy: ::std::option::Option<crate::types::QuotaSharePolicy>,
-    pub(crate) fairshare_policy: ::std::option::Option<crate::types::FairsharePolicy>,
+    pub(crate) quota_share_policy: ::std::option::Option<super::types::QuotaSharePolicy>,
+    pub(crate) fairshare_policy: ::std::option::Option<super::types::FairsharePolicy>,
     pub(crate) tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
 impl SchedulingPolicyDetailBuilder {
@@ -86,31 +86,31 @@
         &self.arn
     }
     /// <p>The quota share scheduling policy details.</p>
-    pub fn quota_share_policy(mut self, input: crate::types::QuotaSharePolicy) -> Self {
+    pub fn quota_share_policy(mut self, input: super::types::QuotaSharePolicy) -> Self {
         self.quota_share_policy = ::std::option::Option::Some(input);
         self
     }
     /// <p>The quota share scheduling policy details.</p>
-    pub fn set_quota_share_policy(mut self, input: ::std::option::Option<crate::types::QuotaSharePolicy>) -> Self {
+    pub fn set_quota_share_policy(mut self, input: ::std::option::Option<super::types::QuotaSharePolicy>) -> Self {
         self.quota_share_policy = input;
         self
     }
     /// <p>The quota share scheduling policy details.</p>
-    pub fn get_quota_share_policy(&self) -> &::std::option::Option<crate::types::QuotaSharePolicy> {
+    pub fn get_quota_share_policy(&self) -> &::std::option::Option<super::types::QuotaSharePolicy> {
         &self.quota_share_policy
     }
     /// <p>The fair-share scheduling policy details.</p>
-    pub fn fairshare_policy(mut self, input: crate::types::FairsharePolicy) -> Self {
+    pub fn fairshare_policy(mut self, input: super::types::FairsharePolicy) -> Self {
         self.fairshare_policy = ::std::option::Option::Some(input);
         self
     }
     /// <p>The fair-share scheduling policy details.</p>
-    pub fn set_fairshare_policy(mut self, input: ::std::option::Option<crate::types::FairsharePolicy>) -> Self {
+    pub fn set_fairshare_policy(mut self, input: ::std::option::Option<super::types::FairsharePolicy>) -> Self {
         self.fairshare_policy = input;
         self
     }
     /// <p>The fair-share scheduling policy details.</p>
-    pub fn get_fairshare_policy(&self) -> &::std::option::Option<crate::types::FairsharePolicy> {
+    pub fn get_fairshare_policy(&self) -> &::std::option::Option<super::types::FairsharePolicy> {
         &self.fairshare_policy
     }
     /// Adds a key-value pair to `tags`.
@@ -134,8 +134,8 @@
         &self.tags
     }
     /// Consumes the builder and constructs a [`SchedulingPolicyDetail`](crate::types::SchedulingPolicyDetail).
-    pub fn build(self) -> crate::types::SchedulingPolicyDetail {
-        crate::types::SchedulingPolicyDetail {
+    pub fn build(self) -> super::types::SchedulingPolicyDetail {
+        super::types::SchedulingPolicyDetail {
             name: self.name,
             arn: self.arn,
             quota_share_policy: self.quota_share_policy,
