--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl Secret {
     /// Creates a new builder-style object to manufacture [`Secret`](crate::types::Secret).
-    pub fn builder() -> crate::types::builders::SecretBuilder {
-        crate::types::builders::SecretBuilder::default()
+    pub fn builder() -> super::types::builders::SecretBuilder {
+        super::types::builders::SecretBuilder::default()
     }
 }

@@ -82,8 +82,8 @@
         &self.value_from
     }
     /// Consumes the builder and constructs a [`Secret`](crate::types::Secret).
-    pub fn build(self) -> crate::types::Secret {
-        crate::types::Secret {
+    pub fn build(self) -> super::types::Secret {
+        super::types::Secret {
             name: self.name,
             value_from: self.value_from,
         }
