--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ServiceResourceId {
     /// <p>The name of the resource identifier.</p>
-    pub name: ::std::option::Option<crate::types::ServiceResourceIdName>,
+    pub name: ::std::option::Option<super::super::types::ServiceResourceIdName>,
     /// <p>The value of the resource identifier.</p>
     pub value: ::std::option::Option<::std::string::String>,
 }
 impl ServiceResourceId {
     /// <p>The name of the resource identifier.</p>
-    pub fn name(&self) -> ::std::option::Option<&crate::types::ServiceResourceIdName> {
+    pub fn name(&self) -> ::std::option::Option<&super::super::types::ServiceResourceIdName> {
         self.name.as_ref()
     }
     /// <p>The value of the resource identifier.</p>
@@ -21,8 +21,8 @@
 }
 impl ServiceResourceId {
     /// Creates a new builder-style object to manufacture [`ServiceResourceId`](crate::types::ServiceResourceId).
-    pub fn builder() -> crate::types::builders::ServiceResourceIdBuilder {
-        crate::types::builders::ServiceResourceIdBuilder::default()
+    pub fn builder() -> super::super::types::builders::ServiceResourceIdBuilder {
+        super::super::types::builders::ServiceResourceIdBuilder::default()
     }
 }

@@ -30,23 +30,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ServiceResourceIdBuilder {
-    pub(crate) name: ::std::option::Option<crate::types::ServiceResourceIdName>,
+    pub(crate) name: ::std::option::Option<super::super::types::ServiceResourceIdName>,
     pub(crate) value: ::std::option::Option<::std::string::String>,
 }
 impl ServiceResourceIdBuilder {
     /// <p>The name of the resource identifier.</p>
     /// This field is required.
-    pub fn name(mut self, input: crate::types::ServiceResourceIdName) -> Self {
+    pub fn name(mut self, input: super::super::types::ServiceResourceIdName) -> Self {
         self.name = ::std::option::Option::Some(input);
         self
     }
     /// <p>The name of the resource identifier.</p>
-    pub fn set_name(mut self, input: ::std::option::Option<crate::types::ServiceResourceIdName>) -> Self {
+    pub fn set_name(mut self, input: ::std::option::Option<super::super::types::ServiceResourceIdName>) -> Self {
         self.name = input;
         self
     }
     /// <p>The name of the resource identifier.</p>
-    pub fn get_name(&self) -> &::std::option::Option<crate::types::ServiceResourceIdName> {
+    pub fn get_name(&self) -> &::std::option::Option<super::super::types::ServiceResourceIdName> {
         &self.name
     }
     /// <p>The value of the resource identifier.</p>
@@ -65,8 +65,8 @@
         &self.value
     }
     /// Consumes the builder and constructs a [`ServiceResourceId`](crate::types::ServiceResourceId).
-    pub fn build(self) -> crate::types::ServiceResourceId {
-        crate::types::ServiceResourceId {
+    pub fn build(self) -> super::super::types::ServiceResourceId {
+        super::super::types::ServiceResourceId {
             name: self.name,
             value: self.value,
         }
