--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl ServiceJobCapacityUsageSummary {
     /// Creates a new builder-style object to manufacture [`ServiceJobCapacityUsageSummary`](crate::types::ServiceJobCapacityUsageSummary).
-    pub fn builder() -> crate::types::builders::ServiceJobCapacityUsageSummaryBuilder {
-        crate::types::builders::ServiceJobCapacityUsageSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::ServiceJobCapacityUsageSummaryBuilder {
+        super::types::builders::ServiceJobCapacityUsageSummaryBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.quantity
     }
     /// Consumes the builder and constructs a [`ServiceJobCapacityUsageSummary`](crate::types::ServiceJobCapacityUsageSummary).
-    pub fn build(self) -> crate::types::ServiceJobCapacityUsageSummary {
-        crate::types::ServiceJobCapacityUsageSummary {
+    pub fn build(self) -> super::types::ServiceJobCapacityUsageSummary {
+        super::types::ServiceJobCapacityUsageSummary {
             capacity_unit: self.capacity_unit,
             quantity: self.quantity,
         }
