--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The number of times to move a service job to <code>RUNNABLE</code> status. You can specify between 1 and 10 attempts.</p>
     pub attempts: ::std::option::Option<i32>,
     /// <p>Array of <code>ServiceJobEvaluateOnExit</code> objects that specify conditions under which the service job should be retried or failed.</p>
-    pub evaluate_on_exit: ::std::option::Option<::std::vec::Vec<crate::types::ServiceJobEvaluateOnExit>>,
+    pub evaluate_on_exit: ::std::option::Option<::std::vec::Vec<super::super::types::ServiceJobEvaluateOnExit>>,
 }
 impl ServiceJobRetryStrategy {
     /// <p>The number of times to move a service job to <code>RUNNABLE</code> status. You can specify between 1 and 10 attempts.</p>
@@ -17,14 +17,14 @@
     /// <p>Array of <code>ServiceJobEvaluateOnExit</code> objects that specify conditions under which the service job should be retried or failed.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.evaluate_on_exit.is_none()`.
-    pub fn evaluate_on_exit(&self) -> &[crate::types::ServiceJobEvaluateOnExit] {
+    pub fn evaluate_on_exit(&self) -> &[super::super::types::ServiceJobEvaluateOnExit] {
         self.evaluate_on_exit.as_deref().unwrap_or_default()
     }
 }
 impl ServiceJobRetryStrategy {
     /// Creates a new builder-style object to manufacture [`ServiceJobRetryStrategy`](crate::types::ServiceJobRetryStrategy).
-    pub fn builder() -> crate::types::builders::ServiceJobRetryStrategyBuilder {
-        crate::types::builders::ServiceJobRetryStrategyBuilder::default()
+    pub fn builder() -> super::super::types::builders::ServiceJobRetryStrategyBuilder {
+        super::super::types::builders::ServiceJobRetryStrategyBuilder::default()
     }
 }

@@ -33,7 +33,7 @@
 #[non_exhaustive]
 pub struct ServiceJobRetryStrategyBuilder {
     pub(crate) attempts: ::std::option::Option<i32>,
-    pub(crate) evaluate_on_exit: ::std::option::Option<::std::vec::Vec<crate::types::ServiceJobEvaluateOnExit>>,
+    pub(crate) evaluate_on_exit: ::std::option::Option<::std::vec::Vec<super::super::types::ServiceJobEvaluateOnExit>>,
 }
 impl ServiceJobRetryStrategyBuilder {
     /// <p>The number of times to move a service job to <code>RUNNABLE</code> status. You can specify between 1 and 10 attempts.</p>
@@ -56,7 +56,7 @@
     /// To override the contents of this collection use [`set_evaluate_on_exit`](Self::set_evaluate_on_exit).
     ///
     /// <p>Array of <code>ServiceJobEvaluateOnExit</code> objects that specify conditions under which the service job should be retried or failed.</p>
-    pub fn evaluate_on_exit(mut self, input: crate::types::ServiceJobEvaluateOnExit) -> Self {
+    pub fn evaluate_on_exit(mut self, input: super::super::types::ServiceJobEvaluateOnExit) -> Self {
         let mut v = self.evaluate_on_exit.unwrap_or_default();
         v.push(input);
         self.evaluate_on_exit = ::std::option::Option::Some(v);
@@ -63,17 +63,17 @@
         self
     }
     /// <p>Array of <code>ServiceJobEvaluateOnExit</code> objects that specify conditions under which the service job should be retried or failed.</p>
-    pub fn set_evaluate_on_exit(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ServiceJobEvaluateOnExit>>) -> Self {
+    pub fn set_evaluate_on_exit(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::ServiceJobEvaluateOnExit>>) -> Self {
         self.evaluate_on_exit = input;
         self
     }
     /// <p>Array of <code>ServiceJobEvaluateOnExit</code> objects that specify conditions under which the service job should be retried or failed.</p>
-    pub fn get_evaluate_on_exit(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ServiceJobEvaluateOnExit>> {
+    pub fn get_evaluate_on_exit(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::ServiceJobEvaluateOnExit>> {
         &self.evaluate_on_exit
     }
     /// Consumes the builder and constructs a [`ServiceJobRetryStrategy`](crate::types::ServiceJobRetryStrategy).
-    pub fn build(self) -> crate::types::ServiceJobRetryStrategy {
-        crate::types::ServiceJobRetryStrategy {
+    pub fn build(self) -> super::super::types::ServiceJobRetryStrategy {
+        super::super::types::ServiceJobRetryStrategy {
             attempts: self.attempts,
             evaluate_on_exit: self.evaluate_on_exit,
         }
