--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl ShareAttributes {
     /// Creates a new builder-style object to manufacture [`ShareAttributes`](crate::types::ShareAttributes).
-    pub fn builder() -> crate::types::builders::ShareAttributesBuilder {
-        crate::types::builders::ShareAttributesBuilder::default()
+    pub fn builder() -> super::types::builders::ShareAttributesBuilder {
+        super::types::builders::ShareAttributesBuilder::default()
     }
 }

@@ -79,8 +79,8 @@
         &self.weight_factor
     }
     /// Consumes the builder and constructs a [`ShareAttributes`](crate::types::ShareAttributes).
-    pub fn build(self) -> crate::types::ShareAttributes {
-        crate::types::ShareAttributes {
+    pub fn build(self) -> super::types::ShareAttributes {
+        super::types::ShareAttributes {
             share_identifier: self.share_identifier,
             weight_factor: self.weight_factor,
         }
