--- original
+++ modified
@@ -37,8 +37,8 @@
 }
 impl TaskContainerDependency {
     /// Creates a new builder-style object to manufacture [`TaskContainerDependency`](crate::types::TaskContainerDependency).
-    pub fn builder() -> crate::types::builders::TaskContainerDependencyBuilder {
-        crate::types::builders::TaskContainerDependencyBuilder::default()
+    pub fn builder() -> super::super::types::builders::TaskContainerDependencyBuilder {
+        super::super::types::builders::TaskContainerDependencyBuilder::default()
     }
 }

@@ -103,8 +103,8 @@
         &self.condition
     }
     /// Consumes the builder and constructs a [`TaskContainerDependency`](crate::types::TaskContainerDependency).
-    pub fn build(self) -> crate::types::TaskContainerDependency {
-        crate::types::TaskContainerDependency {
+    pub fn build(self) -> super::super::types::TaskContainerDependency {
+        super::super::types::TaskContainerDependency {
             container_name: self.container_name,
             condition: self.condition,
         }
