--- original
+++ modified
@@ -35,7 +35,7 @@
     }
 }
 impl ::std::error::Error for ResourceInUseException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::ResourceInUseException {
+impl ::aws_types::request_id::RequestId for super::super::super::types::error::ResourceInUseException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -48,8 +48,8 @@
 }
 impl ResourceInUseException {
     /// Creates a new builder-style object to manufacture [`ResourceInUseException`](crate::types::error::ResourceInUseException).
-    pub fn builder() -> crate::types::error::builders::ResourceInUseExceptionBuilder {
-        crate::types::error::builders::ResourceInUseExceptionBuilder::default()
+    pub fn builder() -> super::super::super::types::error::builders::ResourceInUseExceptionBuilder {
+        super::super::super::types::error::builders::ResourceInUseExceptionBuilder::default()
     }
 }

@@ -87,8 +87,8 @@
         self
     }
     /// Consumes the builder and constructs a [`ResourceInUseException`](crate::types::error::ResourceInUseException).
-    pub fn build(self) -> crate::types::error::ResourceInUseException {
-        crate::types::error::ResourceInUseException {
+    pub fn build(self) -> super::super::super::types::error::ResourceInUseException {
+        super::super::super::types::error::ResourceInUseException {
             message: self.message,
             meta: self.meta.unwrap_or_default(),
         }
