--- original
+++ modified
@@ -34,8 +34,8 @@
 }
 impl GetDiscoveredResourceCountsInput {
     /// Creates a new builder-style object to manufacture [`GetDiscoveredResourceCountsInput`](crate::operation::get_discovered_resource_counts::GetDiscoveredResourceCountsInput).
-    pub fn builder() -> crate::operation::get_discovered_resource_counts::builders::GetDiscoveredResourceCountsInputBuilder {
-        crate::operation::get_discovered_resource_counts::builders::GetDiscoveredResourceCountsInputBuilder::default()
+    pub fn builder() -> super::operation::get_discovered_resource_counts::builders::GetDiscoveredResourceCountsInputBuilder {
+        super::operation::get_discovered_resource_counts::builders::GetDiscoveredResourceCountsInputBuilder::default()
     }
 }

@@ -109,10 +109,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_discovered_resource_counts::GetDiscoveredResourceCountsInput,
+        super::operation::get_discovered_resource_counts::GetDiscoveredResourceCountsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_discovered_resource_counts::GetDiscoveredResourceCountsInput {
+        ::std::result::Result::Ok(super::operation::get_discovered_resource_counts::GetDiscoveredResourceCountsInput {
             resource_types: self.resource_types,
             limit: self.limit,
             next_token: self.next_token,
