--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl SelectResourceConfigInput {
     /// Creates a new builder-style object to manufacture [`SelectResourceConfigInput`](crate::operation::select_resource_config::SelectResourceConfigInput).
-    pub fn builder() -> crate::operation::select_resource_config::builders::SelectResourceConfigInputBuilder {
-        crate::operation::select_resource_config::builders::SelectResourceConfigInputBuilder::default()
+    pub fn builder() -> super::operation::select_resource_config::builders::SelectResourceConfigInputBuilder {
+        super::operation::select_resource_config::builders::SelectResourceConfigInputBuilder::default()
     }
 }

@@ -86,9 +86,9 @@
     /// Consumes the builder and constructs a [`SelectResourceConfigInput`](crate::operation::select_resource_config::SelectResourceConfigInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::select_resource_config::SelectResourceConfigInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::select_resource_config::SelectResourceConfigInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::select_resource_config::SelectResourceConfigInput {
+        ::std::result::Result::Ok(super::operation::select_resource_config::SelectResourceConfigInput {
             expression: self.expression,
             limit: self.limit,
             next_token: self.next_token,
