--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl ListSubscriptionsInput {
     /// Creates a new builder-style object to manufacture [`ListSubscriptionsInput`](crate::operation::list_subscriptions::ListSubscriptionsInput).
-    pub fn builder() -> crate::operation::list_subscriptions::builders::ListSubscriptionsInputBuilder {
-        crate::operation::list_subscriptions::builders::ListSubscriptionsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_subscriptions::builders::ListSubscriptionsInputBuilder {
+        super::super::super::operation::list_subscriptions::builders::ListSubscriptionsInputBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
     /// Consumes the builder and constructs a [`ListSubscriptionsInput`](crate::operation::list_subscriptions::ListSubscriptionsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_subscriptions::ListSubscriptionsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_subscriptions::ListSubscriptionsInput { next_token: self.next_token })
+    ) -> ::std::result::Result<super::super::super::operation::list_subscriptions::ListSubscriptionsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_subscriptions::ListSubscriptionsInput { next_token: self.next_token })
     }
 }
