--- original
+++ modified
@@ -18,7 +18,7 @@
     /// </ul>
     /// <p>If you omit this parameter, the response includes the <code>DEFAULT</code>, <code>CUSTOM</code>, and <code>INACTIVE</code> categories.</p>
     /// <p>For more information about automatically indexed fields and using the <code>AUTO</code> category, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub index_categories: ::std::option::Option<::std::vec::Vec<crate::types::IndexCategory>>,
+    pub index_categories: ::std::option::Option<::std::vec::Vec<super::types::IndexCategory>>,
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
     pub next_token: ::std::option::Option<::std::string::String>,
 }
@@ -44,7 +44,7 @@
     /// <p>For more information about automatically indexed fields and using the <code>AUTO</code> category, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.index_categories.is_none()`.
-    pub fn index_categories(&self) -> &[crate::types::IndexCategory] {
+    pub fn index_categories(&self) -> &[super::types::IndexCategory] {
         self.index_categories.as_deref().unwrap_or_default()
     }
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
@@ -54,8 +54,8 @@
 }
 impl DescribeFieldIndexesInput {
     /// Creates a new builder-style object to manufacture [`DescribeFieldIndexesInput`](crate::operation::describe_field_indexes::DescribeFieldIndexesInput).
-    pub fn builder() -> crate::operation::describe_field_indexes::builders::DescribeFieldIndexesInputBuilder {
-        crate::operation::describe_field_indexes::builders::DescribeFieldIndexesInputBuilder::default()
+    pub fn builder() -> super::operation::describe_field_indexes::builders::DescribeFieldIndexesInputBuilder {
+        super::operation::describe_field_indexes::builders::DescribeFieldIndexesInputBuilder::default()
     }
 }

@@ -64,7 +64,7 @@
 #[non_exhaustive]
 pub struct DescribeFieldIndexesInputBuilder {
     pub(crate) log_group_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
-    pub(crate) index_categories: ::std::option::Option<::std::vec::Vec<crate::types::IndexCategory>>,
+    pub(crate) index_categories: ::std::option::Option<::std::vec::Vec<super::types::IndexCategory>>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
 }
 impl DescribeFieldIndexesInputBuilder {
@@ -105,7 +105,7 @@
     /// </ul>
     /// <p>If you omit this parameter, the response includes the <code>DEFAULT</code>, <code>CUSTOM</code>, and <code>INACTIVE</code> categories.</p>
     /// <p>For more information about automatically indexed fields and using the <code>AUTO</code> category, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn index_categories(mut self, input: crate::types::IndexCategory) -> Self {
+    pub fn index_categories(mut self, input: super::types::IndexCategory) -> Self {
         let mut v = self.index_categories.unwrap_or_default();
         v.push(input);
         self.index_categories = ::std::option::Option::Some(v);
@@ -124,7 +124,7 @@
     /// </ul>
     /// <p>If you omit this parameter, the response includes the <code>DEFAULT</code>, <code>CUSTOM</code>, and <code>INACTIVE</code> categories.</p>
     /// <p>For more information about automatically indexed fields and using the <code>AUTO</code> category, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn set_index_categories(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::IndexCategory>>) -> Self {
+    pub fn set_index_categories(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::IndexCategory>>) -> Self {
         self.index_categories = input;
         self
     }
@@ -141,7 +141,7 @@
     /// </ul>
     /// <p>If you omit this parameter, the response includes the <code>DEFAULT</code>, <code>CUSTOM</code>, and <code>INACTIVE</code> categories.</p>
     /// <p>For more information about automatically indexed fields and using the <code>AUTO</code> category, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn get_index_categories(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::IndexCategory>> {
+    pub fn get_index_categories(&self) -> &::std::option::Option<::std::vec::Vec<super::types::IndexCategory>> {
         &self.index_categories
     }
     /// <p>The token for the next set of items to return. The token expires after 24 hours.</p>
@@ -161,9 +161,9 @@
     /// Consumes the builder and constructs a [`DescribeFieldIndexesInput`](crate::operation::describe_field_indexes::DescribeFieldIndexesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_field_indexes::DescribeFieldIndexesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::describe_field_indexes::DescribeFieldIndexesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::describe_field_indexes::DescribeFieldIndexesInput {
+        ::std::result::Result::Ok(super::operation::describe_field_indexes::DescribeFieldIndexesInput {
             log_group_identifiers: self.log_group_identifiers,
             index_categories: self.index_categories,
             next_token: self.next_token,
