--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl TagLogGroupInput {
     /// Creates a new builder-style object to manufacture [`TagLogGroupInput`](crate::operation::tag_log_group::TagLogGroupInput).
-    pub fn builder() -> crate::operation::tag_log_group::builders::TagLogGroupInputBuilder {
-        crate::operation::tag_log_group::builders::TagLogGroupInputBuilder::default()
+    pub fn builder() -> super::operation::tag_log_group::builders::TagLogGroupInputBuilder {
+        super::operation::tag_log_group::builders::TagLogGroupInputBuilder::default()
     }
 }

@@ -70,8 +70,8 @@
         &self.tags
     }
     /// Consumes the builder and constructs a [`TagLogGroupInput`](crate::operation::tag_log_group::TagLogGroupInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::tag_log_group::TagLogGroupInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::tag_log_group::TagLogGroupInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::tag_log_group::TagLogGroupInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::tag_log_group::TagLogGroupInput {
             log_group_name: self.log_group_name,
             tags: self.tags,
         })
