--- original
+++ modified
@@ -10,9 +10,9 @@
     pub content_md5: ::std::option::Option<::std::string::String>,
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub checksum_algorithm: ::std::option::Option<crate::types::ChecksumAlgorithm>,
+    pub checksum_algorithm: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>,
     /// <p>Container for Payer.</p>
-    pub request_payment_configuration: ::std::option::Option<crate::types::RequestPaymentConfiguration>,
+    pub request_payment_configuration: ::std::option::Option<super::super::super::types::RequestPaymentConfiguration>,
     /// <p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p>
     pub expected_bucket_owner: ::std::option::Option<::std::string::String>,
 }
@@ -28,11 +28,11 @@
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn checksum_algorithm(&self) -> ::std::option::Option<&crate::types::ChecksumAlgorithm> {
+    pub fn checksum_algorithm(&self) -> ::std::option::Option<&super::super::super::types::ChecksumAlgorithm> {
         self.checksum_algorithm.as_ref()
     }
     /// <p>Container for Payer.</p>
-    pub fn request_payment_configuration(&self) -> ::std::option::Option<&crate::types::RequestPaymentConfiguration> {
+    pub fn request_payment_configuration(&self) -> ::std::option::Option<&super::super::super::types::RequestPaymentConfiguration> {
         self.request_payment_configuration.as_ref()
     }
     /// <p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p>
@@ -42,8 +42,8 @@
 }
 impl PutBucketRequestPaymentInput {
     /// Creates a new builder-style object to manufacture [`PutBucketRequestPaymentInput`](crate::operation::put_bucket_request_payment::PutBucketRequestPaymentInput).
-    pub fn builder() -> crate::operation::put_bucket_request_payment::builders::PutBucketRequestPaymentInputBuilder {
-        crate::operation::put_bucket_request_payment::builders::PutBucketRequestPaymentInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_bucket_request_payment::builders::PutBucketRequestPaymentInputBuilder {
+        super::super::super::operation::put_bucket_request_payment::builders::PutBucketRequestPaymentInputBuilder::default()
     }
 }

@@ -53,8 +53,8 @@
 pub struct PutBucketRequestPaymentInputBuilder {
     pub(crate) bucket: ::std::option::Option<::std::string::String>,
     pub(crate) content_md5: ::std::option::Option<::std::string::String>,
-    pub(crate) checksum_algorithm: ::std::option::Option<crate::types::ChecksumAlgorithm>,
-    pub(crate) request_payment_configuration: ::std::option::Option<crate::types::RequestPaymentConfiguration>,
+    pub(crate) checksum_algorithm: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>,
+    pub(crate) request_payment_configuration: ::std::option::Option<super::super::super::types::RequestPaymentConfiguration>,
     pub(crate) expected_bucket_owner: ::std::option::Option<::std::string::String>,
 }
 impl PutBucketRequestPaymentInputBuilder {
@@ -92,34 +92,34 @@
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn checksum_algorithm(mut self, input: crate::types::ChecksumAlgorithm) -> Self {
+    pub fn checksum_algorithm(mut self, input: super::super::super::types::ChecksumAlgorithm) -> Self {
         self.checksum_algorithm = ::std::option::Option::Some(input);
         self
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<crate::types::ChecksumAlgorithm>) -> Self {
+    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>) -> Self {
         self.checksum_algorithm = input;
         self
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<crate::types::ChecksumAlgorithm> {
+    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<super::super::super::types::ChecksumAlgorithm> {
         &self.checksum_algorithm
     }
     /// <p>Container for Payer.</p>
     /// This field is required.
-    pub fn request_payment_configuration(mut self, input: crate::types::RequestPaymentConfiguration) -> Self {
+    pub fn request_payment_configuration(mut self, input: super::super::super::types::RequestPaymentConfiguration) -> Self {
         self.request_payment_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Container for Payer.</p>
-    pub fn set_request_payment_configuration(mut self, input: ::std::option::Option<crate::types::RequestPaymentConfiguration>) -> Self {
+    pub fn set_request_payment_configuration(mut self, input: ::std::option::Option<super::super::super::types::RequestPaymentConfiguration>) -> Self {
         self.request_payment_configuration = input;
         self
     }
     /// <p>Container for Payer.</p>
-    pub fn get_request_payment_configuration(&self) -> &::std::option::Option<crate::types::RequestPaymentConfiguration> {
+    pub fn get_request_payment_configuration(&self) -> &::std::option::Option<super::super::super::types::RequestPaymentConfiguration> {
         &self.request_payment_configuration
     }
     /// <p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p>
@@ -140,10 +140,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_bucket_request_payment::PutBucketRequestPaymentInput,
+        super::super::super::operation::put_bucket_request_payment::PutBucketRequestPaymentInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_bucket_request_payment::PutBucketRequestPaymentInput {
+        ::std::result::Result::Ok(super::super::super::operation::put_bucket_request_payment::PutBucketRequestPaymentInput {
             bucket: self.bucket,
             content_md5: self.content_md5,
             checksum_algorithm: self.checksum_algorithm,
