--- original
+++ modified
@@ -6,13 +6,13 @@
     /// <p>The canned ACL to apply to the bucket.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub acl: ::std::option::Option<crate::types::BucketCannedAcl>,
+    pub acl: ::std::option::Option<super::types::BucketCannedAcl>,
     /// <p>The name of the bucket to create.</p>
     /// <p><b>General purpose buckets</b> - For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html">Bucket naming rules</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p><b>Directory buckets </b> - When you use this operation with a directory bucket, you must use path-style requests in the format <code>https://s3express-control.<i>region-code</i>.amazonaws.com/<i>bucket-name</i> </code>. Virtual-hosted-style requests aren't supported. Directory bucket names must be unique in the chosen Zone (Availability Zone or Local Zone). Bucket names must also follow the format <code> <i>bucket-base-name</i>--<i>zone-id</i>--x-s3</code> (for example, <code> <i>DOC-EXAMPLE-BUCKET</i>--<i>usw2-az1</i>--x-s3</code>). For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/directory-bucket-naming-rules.html">Directory bucket naming rules</a> in the <i>Amazon S3 User Guide</i></p>
     pub bucket: ::std::option::Option<::std::string::String>,
     /// <p>The configuration information for the bucket.</p>
-    pub create_bucket_configuration: ::std::option::Option<crate::types::CreateBucketConfiguration>,
+    pub create_bucket_configuration: ::std::option::Option<super::types::CreateBucketConfiguration>,
     /// <p>Allows grantee the read, write, read ACP, and write ACP permissions on the bucket.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
@@ -45,18 +45,18 @@
     /// <p>By default, <code>ObjectOwnership</code> is set to <code>BucketOwnerEnforced</code> and ACLs are disabled. We recommend keeping ACLs disabled, except in uncommon use cases where you must control access for each object individually. For more information about S3 Object Ownership, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html">Controlling ownership of objects and disabling ACLs for your bucket</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets use the bucket owner enforced setting for S3 Object Ownership.</p>
     /// </note>
-    pub object_ownership: ::std::option::Option<crate::types::ObjectOwnership>,
+    pub object_ownership: ::std::option::Option<super::types::ObjectOwnership>,
     /// <p>Specifies the namespace where you want to create your general purpose bucket. When you create a general purpose bucket, you can choose to create a bucket in the shared global namespace or you can choose to create a bucket in your account regional namespace. Your account regional namespace is a subdivision of the global namespace that only your account can create buckets in. For more information on bucket namespaces, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/gpbucketnamespaces.html">Namespaces for general purpose buckets</a>.</p>
     /// <p>General purpose buckets in your account regional namespace must follow a specific naming convention. These buckets consist of a bucket name prefix that you create, and a suffix that contains your 12-digit Amazon Web Services Account ID, the Amazon Web Services Region code, and ends with <code>-an</code>. Bucket names must follow the format <code>bucket-name-prefix-accountId-region-an</code> (for example, <code>amzn-s3-demo-bucket-111122223333-us-west-2-an</code>). For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#account-regional-naming-rules">Account regional namespace naming rules</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub bucket_namespace: ::std::option::Option<crate::types::BucketNamespace>,
+    pub bucket_namespace: ::std::option::Option<super::types::BucketNamespace>,
 }
 impl CreateBucketInput {
     /// <p>The canned ACL to apply to the bucket.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn acl(&self) -> ::std::option::Option<&crate::types::BucketCannedAcl> {
+    pub fn acl(&self) -> ::std::option::Option<&super::types::BucketCannedAcl> {
         self.acl.as_ref()
     }
     /// <p>The name of the bucket to create.</p>
@@ -66,7 +66,7 @@
         self.bucket.as_deref()
     }
     /// <p>The configuration information for the bucket.</p>
-    pub fn create_bucket_configuration(&self) -> ::std::option::Option<&crate::types::CreateBucketConfiguration> {
+    pub fn create_bucket_configuration(&self) -> ::std::option::Option<&super::types::CreateBucketConfiguration> {
         self.create_bucket_configuration.as_ref()
     }
     /// <p>Allows grantee the read, write, read ACP, and write ACP permissions on the bucket.</p><note>
@@ -113,7 +113,7 @@
     /// <p>By default, <code>ObjectOwnership</code> is set to <code>BucketOwnerEnforced</code> and ACLs are disabled. We recommend keeping ACLs disabled, except in uncommon use cases where you must control access for each object individually. For more information about S3 Object Ownership, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html">Controlling ownership of objects and disabling ACLs for your bucket</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets use the bucket owner enforced setting for S3 Object Ownership.</p>
     /// </note>
-    pub fn object_ownership(&self) -> ::std::option::Option<&crate::types::ObjectOwnership> {
+    pub fn object_ownership(&self) -> ::std::option::Option<&super::types::ObjectOwnership> {
         self.object_ownership.as_ref()
     }
     /// <p>Specifies the namespace where you want to create your general purpose bucket. When you create a general purpose bucket, you can choose to create a bucket in the shared global namespace or you can choose to create a bucket in your account regional namespace. Your account regional namespace is a subdivision of the global namespace that only your account can create buckets in. For more information on bucket namespaces, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/gpbucketnamespaces.html">Namespaces for general purpose buckets</a>.</p>
@@ -120,14 +120,14 @@
     /// <p>General purpose buckets in your account regional namespace must follow a specific naming convention. These buckets consist of a bucket name prefix that you create, and a suffix that contains your 12-digit Amazon Web Services Account ID, the Amazon Web Services Region code, and ends with <code>-an</code>. Bucket names must follow the format <code>bucket-name-prefix-accountId-region-an</code> (for example, <code>amzn-s3-demo-bucket-111122223333-us-west-2-an</code>). For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#account-regional-naming-rules">Account regional namespace naming rules</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn bucket_namespace(&self) -> ::std::option::Option<&crate::types::BucketNamespace> {
+    pub fn bucket_namespace(&self) -> ::std::option::Option<&super::types::BucketNamespace> {
         self.bucket_namespace.as_ref()
     }
 }
 impl CreateBucketInput {
     /// Creates a new builder-style object to manufacture [`CreateBucketInput`](crate::operation::create_bucket::CreateBucketInput).
-    pub fn builder() -> crate::operation::create_bucket::builders::CreateBucketInputBuilder {
-        crate::operation::create_bucket::builders::CreateBucketInputBuilder::default()
+    pub fn builder() -> super::operation::create_bucket::builders::CreateBucketInputBuilder {
+        super::operation::create_bucket::builders::CreateBucketInputBuilder::default()
     }
 }

@@ -135,9 +135,9 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct CreateBucketInputBuilder {
-    pub(crate) acl: ::std::option::Option<crate::types::BucketCannedAcl>,
+    pub(crate) acl: ::std::option::Option<super::types::BucketCannedAcl>,
     pub(crate) bucket: ::std::option::Option<::std::string::String>,
-    pub(crate) create_bucket_configuration: ::std::option::Option<crate::types::CreateBucketConfiguration>,
+    pub(crate) create_bucket_configuration: ::std::option::Option<super::types::CreateBucketConfiguration>,
     pub(crate) grant_full_control: ::std::option::Option<::std::string::String>,
     pub(crate) grant_read: ::std::option::Option<::std::string::String>,
     pub(crate) grant_read_acp: ::std::option::Option<::std::string::String>,
@@ -144,14 +144,14 @@
     pub(crate) grant_write: ::std::option::Option<::std::string::String>,
     pub(crate) grant_write_acp: ::std::option::Option<::std::string::String>,
     pub(crate) object_lock_enabled_for_bucket: ::std::option::Option<bool>,
-    pub(crate) object_ownership: ::std::option::Option<crate::types::ObjectOwnership>,
-    pub(crate) bucket_namespace: ::std::option::Option<crate::types::BucketNamespace>,
+    pub(crate) object_ownership: ::std::option::Option<super::types::ObjectOwnership>,
+    pub(crate) bucket_namespace: ::std::option::Option<super::types::BucketNamespace>,
 }
 impl CreateBucketInputBuilder {
     /// <p>The canned ACL to apply to the bucket.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn acl(mut self, input: crate::types::BucketCannedAcl) -> Self {
+    pub fn acl(mut self, input: super::types::BucketCannedAcl) -> Self {
         self.acl = ::std::option::Option::Some(input);
         self
     }
@@ -158,7 +158,7 @@
     /// <p>The canned ACL to apply to the bucket.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn set_acl(mut self, input: ::std::option::Option<crate::types::BucketCannedAcl>) -> Self {
+    pub fn set_acl(mut self, input: ::std::option::Option<super::types::BucketCannedAcl>) -> Self {
         self.acl = input;
         self
     }
@@ -165,7 +165,7 @@
     /// <p>The canned ACL to apply to the bucket.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn get_acl(&self) -> &::std::option::Option<crate::types::BucketCannedAcl> {
+    pub fn get_acl(&self) -> &::std::option::Option<super::types::BucketCannedAcl> {
         &self.acl
     }
     /// <p>The name of the bucket to create.</p>
@@ -190,17 +190,17 @@
         &self.bucket
     }
     /// <p>The configuration information for the bucket.</p>
-    pub fn create_bucket_configuration(mut self, input: crate::types::CreateBucketConfiguration) -> Self {
+    pub fn create_bucket_configuration(mut self, input: super::types::CreateBucketConfiguration) -> Self {
         self.create_bucket_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>The configuration information for the bucket.</p>
-    pub fn set_create_bucket_configuration(mut self, input: ::std::option::Option<crate::types::CreateBucketConfiguration>) -> Self {
+    pub fn set_create_bucket_configuration(mut self, input: ::std::option::Option<super::types::CreateBucketConfiguration>) -> Self {
         self.create_bucket_configuration = input;
         self
     }
     /// <p>The configuration information for the bucket.</p>
-    pub fn get_create_bucket_configuration(&self) -> &::std::option::Option<crate::types::CreateBucketConfiguration> {
+    pub fn get_create_bucket_configuration(&self) -> &::std::option::Option<super::types::CreateBucketConfiguration> {
         &self.create_bucket_configuration
     }
     /// <p>Allows grantee the read, write, read ACP, and write ACP permissions on the bucket.</p><note>
@@ -333,7 +333,7 @@
     /// <p>By default, <code>ObjectOwnership</code> is set to <code>BucketOwnerEnforced</code> and ACLs are disabled. We recommend keeping ACLs disabled, except in uncommon use cases where you must control access for each object individually. For more information about S3 Object Ownership, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html">Controlling ownership of objects and disabling ACLs for your bucket</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets use the bucket owner enforced setting for S3 Object Ownership.</p>
     /// </note>
-    pub fn object_ownership(mut self, input: crate::types::ObjectOwnership) -> Self {
+    pub fn object_ownership(mut self, input: super::types::ObjectOwnership) -> Self {
         self.object_ownership = ::std::option::Option::Some(input);
         self
     }
@@ -344,7 +344,7 @@
     /// <p>By default, <code>ObjectOwnership</code> is set to <code>BucketOwnerEnforced</code> and ACLs are disabled. We recommend keeping ACLs disabled, except in uncommon use cases where you must control access for each object individually. For more information about S3 Object Ownership, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html">Controlling ownership of objects and disabling ACLs for your bucket</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets use the bucket owner enforced setting for S3 Object Ownership.</p>
     /// </note>
-    pub fn set_object_ownership(mut self, input: ::std::option::Option<crate::types::ObjectOwnership>) -> Self {
+    pub fn set_object_ownership(mut self, input: ::std::option::Option<super::types::ObjectOwnership>) -> Self {
         self.object_ownership = input;
         self
     }
@@ -355,7 +355,7 @@
     /// <p>By default, <code>ObjectOwnership</code> is set to <code>BucketOwnerEnforced</code> and ACLs are disabled. We recommend keeping ACLs disabled, except in uncommon use cases where you must control access for each object individually. For more information about S3 Object Ownership, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html">Controlling ownership of objects and disabling ACLs for your bucket</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets use the bucket owner enforced setting for S3 Object Ownership.</p>
     /// </note>
-    pub fn get_object_ownership(&self) -> &::std::option::Option<crate::types::ObjectOwnership> {
+    pub fn get_object_ownership(&self) -> &::std::option::Option<super::types::ObjectOwnership> {
         &self.object_ownership
     }
     /// <p>Specifies the namespace where you want to create your general purpose bucket. When you create a general purpose bucket, you can choose to create a bucket in the shared global namespace or you can choose to create a bucket in your account regional namespace. Your account regional namespace is a subdivision of the global namespace that only your account can create buckets in. For more information on bucket namespaces, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/gpbucketnamespaces.html">Namespaces for general purpose buckets</a>.</p>
@@ -362,7 +362,7 @@
     /// <p>General purpose buckets in your account regional namespace must follow a specific naming convention. These buckets consist of a bucket name prefix that you create, and a suffix that contains your 12-digit Amazon Web Services Account ID, the Amazon Web Services Region code, and ends with <code>-an</code>. Bucket names must follow the format <code>bucket-name-prefix-accountId-region-an</code> (for example, <code>amzn-s3-demo-bucket-111122223333-us-west-2-an</code>). For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#account-regional-naming-rules">Account regional namespace naming rules</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn bucket_namespace(mut self, input: crate::types::BucketNamespace) -> Self {
+    pub fn bucket_namespace(mut self, input: super::types::BucketNamespace) -> Self {
         self.bucket_namespace = ::std::option::Option::Some(input);
         self
     }
@@ -370,7 +370,7 @@
     /// <p>General purpose buckets in your account regional namespace must follow a specific naming convention. These buckets consist of a bucket name prefix that you create, and a suffix that contains your 12-digit Amazon Web Services Account ID, the Amazon Web Services Region code, and ends with <code>-an</code>. Bucket names must follow the format <code>bucket-name-prefix-accountId-region-an</code> (for example, <code>amzn-s3-demo-bucket-111122223333-us-west-2-an</code>). For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#account-regional-naming-rules">Account regional namespace naming rules</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn set_bucket_namespace(mut self, input: ::std::option::Option<crate::types::BucketNamespace>) -> Self {
+    pub fn set_bucket_namespace(mut self, input: ::std::option::Option<super::types::BucketNamespace>) -> Self {
         self.bucket_namespace = input;
         self
     }
@@ -378,14 +378,14 @@
     /// <p>General purpose buckets in your account regional namespace must follow a specific naming convention. These buckets consist of a bucket name prefix that you create, and a suffix that contains your 12-digit Amazon Web Services Account ID, the Amazon Web Services Region code, and ends with <code>-an</code>. Bucket names must follow the format <code>bucket-name-prefix-accountId-region-an</code> (for example, <code>amzn-s3-demo-bucket-111122223333-us-west-2-an</code>). For information about bucket naming restrictions, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#account-regional-naming-rules">Account regional namespace naming rules</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn get_bucket_namespace(&self) -> &::std::option::Option<crate::types::BucketNamespace> {
+    pub fn get_bucket_namespace(&self) -> &::std::option::Option<super::types::BucketNamespace> {
         &self.bucket_namespace
     }
     /// Consumes the builder and constructs a [`CreateBucketInput`](crate::operation::create_bucket::CreateBucketInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_bucket::CreateBucketInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_bucket::CreateBucketInput {
+    ) -> ::std::result::Result<super::operation::create_bucket::CreateBucketInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::create_bucket::CreateBucketInput {
             acl: self.acl,
             bucket: self.bucket,
             create_bucket_configuration: self.create_bucket_configuration,
