--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl ListCustomVerificationEmailTemplatesInput {
     /// Creates a new builder-style object to manufacture [`ListCustomVerificationEmailTemplatesInput`](crate::operation::list_custom_verification_email_templates::ListCustomVerificationEmailTemplatesInput).
-    pub fn builder() -> crate::operation::list_custom_verification_email_templates::builders::ListCustomVerificationEmailTemplatesInputBuilder {
-        crate::operation::list_custom_verification_email_templates::builders::ListCustomVerificationEmailTemplatesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_custom_verification_email_templates::builders::ListCustomVerificationEmailTemplatesInputBuilder {
+        super::super::super::operation::list_custom_verification_email_templates::builders::ListCustomVerificationEmailTemplatesInputBuilder::default()
     }
 }

@@ -71,11 +71,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_custom_verification_email_templates::ListCustomVerificationEmailTemplatesInput,
+        super::super::super::operation::list_custom_verification_email_templates::ListCustomVerificationEmailTemplatesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::list_custom_verification_email_templates::ListCustomVerificationEmailTemplatesInput {
+            super::super::super::operation::list_custom_verification_email_templates::ListCustomVerificationEmailTemplatesInput {
                 next_token: self.next_token,
                 page_size: self.page_size,
             },
