--- original
+++ modified
@@ -102,8 +102,8 @@
 }
 impl PutKeyPolicyInput {
     /// Creates a new builder-style object to manufacture [`PutKeyPolicyInput`](crate::operation::put_key_policy::PutKeyPolicyInput).
-    pub fn builder() -> crate::operation::put_key_policy::builders::PutKeyPolicyInputBuilder {
-        crate::operation::put_key_policy::builders::PutKeyPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_key_policy::builders::PutKeyPolicyInputBuilder {
+        super::super::super::operation::put_key_policy::builders::PutKeyPolicyInputBuilder::default()
     }
 }

@@ -283,8 +283,8 @@
     /// Consumes the builder and constructs a [`PutKeyPolicyInput`](crate::operation::put_key_policy::PutKeyPolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_key_policy::PutKeyPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::put_key_policy::PutKeyPolicyInput {
+    ) -> ::std::result::Result<super::super::super::operation::put_key_policy::PutKeyPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::put_key_policy::PutKeyPolicyInput {
             key_id: self.key_id,
             policy_name: self.policy_name,
             policy: self.policy,
