--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteSubscriptionFilterInput {
     /// Creates a new builder-style object to manufacture [`DeleteSubscriptionFilterInput`](crate::operation::delete_subscription_filter::DeleteSubscriptionFilterInput).
-    pub fn builder() -> crate::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterInputBuilder {
-        crate::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterInputBuilder {
+        super::super::super::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_subscription_filter::DeleteSubscriptionFilterInput,
+        super::super::super::operation::delete_subscription_filter::DeleteSubscriptionFilterInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_subscription_filter::DeleteSubscriptionFilterInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_subscription_filter::DeleteSubscriptionFilterInput {
             log_group_name: self.log_group_name,
             filter_name: self.filter_name,
         })
