--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl AdminGetUserAuthFactorsInput {
     /// Creates a new builder-style object to manufacture [`AdminGetUserAuthFactorsInput`](crate::operation::admin_get_user_auth_factors::AdminGetUserAuthFactorsInput).
-    pub fn builder() -> crate::operation::admin_get_user_auth_factors::builders::AdminGetUserAuthFactorsInputBuilder {
-        crate::operation::admin_get_user_auth_factors::builders::AdminGetUserAuthFactorsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::admin_get_user_auth_factors::builders::AdminGetUserAuthFactorsInputBuilder {
+        super::super::super::operation::admin_get_user_auth_factors::builders::AdminGetUserAuthFactorsInputBuilder::default()
     }
 }

@@ -75,10 +75,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::admin_get_user_auth_factors::AdminGetUserAuthFactorsInput,
+        super::super::super::operation::admin_get_user_auth_factors::AdminGetUserAuthFactorsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::admin_get_user_auth_factors::AdminGetUserAuthFactorsInput {
+        ::std::result::Result::Ok(super::super::super::operation::admin_get_user_auth_factors::AdminGetUserAuthFactorsInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
         })
