--- original
+++ modified
@@ -26,7 +26,7 @@
     /// <p>If you don't specify otherwise in the configuration of your app client, your ID tokens are valid for one hour.</p>
     pub id_token_validity: ::std::option::Option<i32>,
     /// <p>The units that validity times are represented in. The default unit for refresh tokens is days, and the default for ID and access tokens are hours.</p>
-    pub token_validity_units: ::std::option::Option<crate::types::TokenValidityUnitsType>,
+    pub token_validity_units: ::std::option::Option<super::super::super::types::TokenValidityUnitsType>,
     /// <p>The list of user attributes that you want your app client to have read access to. After your user authenticates in your app, their access token authorizes them to read their own attribute value for any attribute in this list.</p>
     /// <p>When you don't specify the <code>ReadAttributes</code> for your app client, your app can read the values of <code>email_verified</code>, <code>phone_number_verified</code>, and the standard attributes of your user pool. When your user pool app client has read access to these default attributes, <code>ReadAttributes</code> doesn't return any information. Amazon Cognito only populates <code>ReadAttributes</code> in the API response if you have specified your own custom set of read attributes.</p>
     pub read_attributes: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -54,7 +54,7 @@
     /// <p><code>ALLOW_REFRESH_TOKEN_AUTH</code>: Enable authflow to refresh tokens.</p></li>
     /// </ul>
     /// <p>In some environments, you will see the values <code>ADMIN_NO_SRP_AUTH</code>, <code>CUSTOM_AUTH_FLOW_ONLY</code>, or <code>USER_PASSWORD_AUTH</code>. You can't assign these legacy <code>ExplicitAuthFlows</code> values to user pool clients at the same time as values that begin with <code>ALLOW_</code>, like <code>ALLOW_USER_SRP_AUTH</code>.</p>
-    pub explicit_auth_flows: ::std::option::Option<::std::vec::Vec<crate::types::ExplicitAuthFlowsType>>,
+    pub explicit_auth_flows: ::std::option::Option<::std::vec::Vec<super::super::super::types::ExplicitAuthFlowsType>>,
     /// <p>A list of provider names for the identity providers (IdPs) that are supported on this client. The following are supported: <code>COGNITO</code>, <code>Facebook</code>, <code>Google</code>, <code>SignInWithApple</code>, and <code>LoginWithAmazon</code>. You can also specify the names that you configured for the SAML and OIDC IdPs in your user pool, for example <code>MySAMLIdP</code> or <code>MyOIDCIdP</code>.</p>
     /// <p>This parameter sets the IdPs that <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-managed-login.html">managed login</a> will display on the login page for your app client. The removal of <code>COGNITO</code> from this list doesn't prevent authentication operations for local users with the user pools API in an Amazon Web Services SDK. The only way to prevent SDK-based authentication is to block access with a <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-waf.html">WAF rule</a>.</p>
     pub supported_identity_providers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -97,7 +97,7 @@
     /// <p>Issue the access token from the <code>/oauth2/token</code> endpoint directly to a non-person user using a combination of the client ID and client secret.</p>
     /// </dd>
     /// </dl>
-    pub allowed_o_auth_flows: ::std::option::Option<::std::vec::Vec<crate::types::OAuthFlowType>>,
+    pub allowed_o_auth_flows: ::std::option::Option<::std::vec::Vec<super::super::super::types::OAuthFlowType>>,
     /// <p>The OAuth, OpenID Connect (OIDC), and custom scopes that you want to permit your app client to authorize access with. Scopes govern access control to user pool self-service API operations, user data from the <code>userInfo</code> endpoint, and third-party APIs. Scope values include <code>phone</code>, <code>email</code>, <code>openid</code>, and <code>profile</code>. The <code>aws.cognito.signin.user.admin</code> scope authorizes user self-service operations. Custom scopes with resource servers authorize access to external APIs.</p>
     pub allowed_o_auth_scopes: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>Set to <code>true</code> to use OAuth 2.0 authorization server features in your app client.</p>
@@ -116,10 +116,10 @@
     pub allowed_o_auth_flows_user_pool_client: ::std::option::Option<bool>,
     /// <p>The user pool analytics configuration for collecting metrics and sending them to your Amazon Pinpoint campaign.</p>
     /// <p>In Amazon Web Services Regions where Amazon Pinpoint isn't available, user pools might not have access to analytics or might be configurable with campaigns in the US East (N. Virginia) Region. For more information, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html">Using Amazon Pinpoint analytics</a>.</p>
-    pub analytics_configuration: ::std::option::Option<crate::types::AnalyticsConfigurationType>,
+    pub analytics_configuration: ::std::option::Option<super::super::super::types::AnalyticsConfigurationType>,
     /// <p>When <code>ENABLED</code>, suppresses messages that might indicate a valid user exists when someone attempts sign-in. This parameters sets your preference for the errors and responses that you want Amazon Cognito APIs to return during authentication, account confirmation, and password recovery when the user doesn't exist in the user pool. When set to <code>ENABLED</code> and the user doesn't exist, authentication returns an error indicating either the username or password was incorrect. Account confirmation and password recovery return a response indicating a code was sent to a simulated destination. When set to <code>LEGACY</code>, those APIs return a <code>UserNotFoundException</code> exception if the user doesn't exist in the user pool.</p>
     /// <p>Defaults to <code>LEGACY</code>.</p>
-    pub prevent_user_existence_errors: ::std::option::Option<crate::types::PreventUserExistenceErrorTypes>,
+    pub prevent_user_existence_errors: ::std::option::Option<super::super::super::types::PreventUserExistenceErrorTypes>,
     /// <p>Activates or deactivates <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html">token revocation</a> in the target app client.</p>
     pub enable_token_revocation: ::std::option::Option<bool>,
     /// <p>When <code>true</code>, your application can include additional <code>UserContextData</code> in authentication requests. This data includes the IP address, and contributes to analysis by threat protection features. For more information about propagation of user context data, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-adaptive-authentication.html#user-pool-settings-adaptive-authentication-device-fingerprint">Adding session data to API requests</a>. If you don’t include this parameter, you can't send the source IP address to Amazon Cognito threat protection features. You can only activate <code>EnablePropagateAdditionalUserContextData</code> in an app client that has a client secret.</p>
@@ -127,7 +127,7 @@
     /// <p>Amazon Cognito creates a session token for each API request in an authentication flow. <code>AuthSessionValidity</code> is the duration, in minutes, of that session token. Your user pool native user must respond to each authentication challenge before the session expires.</p>
     pub auth_session_validity: ::std::option::Option<i32>,
     /// <p>The configuration of your app client for refresh token rotation. When enabled, your app client issues new ID, access, and refresh tokens when users renew their sessions with refresh tokens. When disabled, token refresh issues only ID and access tokens.</p>
-    pub refresh_token_rotation: ::std::option::Option<crate::types::RefreshTokenRotationType>,
+    pub refresh_token_rotation: ::std::option::Option<super::super::super::types::RefreshTokenRotationType>,
 }
 impl UpdateUserPoolClientInput {
     /// <p>The ID of the user pool where you want to update the app client.</p>
@@ -164,7 +164,7 @@
         self.id_token_validity
     }
     /// <p>The units that validity times are represented in. The default unit for refresh tokens is days, and the default for ID and access tokens are hours.</p>
-    pub fn token_validity_units(&self) -> ::std::option::Option<&crate::types::TokenValidityUnitsType> {
+    pub fn token_validity_units(&self) -> ::std::option::Option<&super::super::super::types::TokenValidityUnitsType> {
         self.token_validity_units.as_ref()
     }
     /// <p>The list of user attributes that you want your app client to have read access to. After your user authenticates in your app, their access token authorizes them to read their own attribute value for any attribute in this list.</p>
@@ -204,7 +204,7 @@
     /// <p>In some environments, you will see the values <code>ADMIN_NO_SRP_AUTH</code>, <code>CUSTOM_AUTH_FLOW_ONLY</code>, or <code>USER_PASSWORD_AUTH</code>. You can't assign these legacy <code>ExplicitAuthFlows</code> values to user pool clients at the same time as values that begin with <code>ALLOW_</code>, like <code>ALLOW_USER_SRP_AUTH</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.explicit_auth_flows.is_none()`.
-    pub fn explicit_auth_flows(&self) -> &[crate::types::ExplicitAuthFlowsType] {
+    pub fn explicit_auth_flows(&self) -> &[super::super::super::types::ExplicitAuthFlowsType] {
         self.explicit_auth_flows.as_deref().unwrap_or_default()
     }
     /// <p>A list of provider names for the identity providers (IdPs) that are supported on this client. The following are supported: <code>COGNITO</code>, <code>Facebook</code>, <code>Google</code>, <code>SignInWithApple</code>, and <code>LoginWithAmazon</code>. You can also specify the names that you configured for the SAML and OIDC IdPs in your user pool, for example <code>MySAMLIdP</code> or <code>MyOIDCIdP</code>.</p>
@@ -265,7 +265,7 @@
     /// </dl>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.allowed_o_auth_flows.is_none()`.
-    pub fn allowed_o_auth_flows(&self) -> &[crate::types::OAuthFlowType] {
+    pub fn allowed_o_auth_flows(&self) -> &[super::super::super::types::OAuthFlowType] {
         self.allowed_o_auth_flows.as_deref().unwrap_or_default()
     }
     /// <p>The OAuth, OpenID Connect (OIDC), and custom scopes that you want to permit your app client to authorize access with. Scopes govern access control to user pool self-service API operations, user data from the <code>userInfo</code> endpoint, and third-party APIs. Scope values include <code>phone</code>, <code>email</code>, <code>openid</code>, and <code>profile</code>. The <code>aws.cognito.signin.user.admin</code> scope authorizes user self-service operations. Custom scopes with resource servers authorize access to external APIs.</p>
@@ -292,12 +292,12 @@
     }
     /// <p>The user pool analytics configuration for collecting metrics and sending them to your Amazon Pinpoint campaign.</p>
     /// <p>In Amazon Web Services Regions where Amazon Pinpoint isn't available, user pools might not have access to analytics or might be configurable with campaigns in the US East (N. Virginia) Region. For more information, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html">Using Amazon Pinpoint analytics</a>.</p>
-    pub fn analytics_configuration(&self) -> ::std::option::Option<&crate::types::AnalyticsConfigurationType> {
+    pub fn analytics_configuration(&self) -> ::std::option::Option<&super::super::super::types::AnalyticsConfigurationType> {
         self.analytics_configuration.as_ref()
     }
     /// <p>When <code>ENABLED</code>, suppresses messages that might indicate a valid user exists when someone attempts sign-in. This parameters sets your preference for the errors and responses that you want Amazon Cognito APIs to return during authentication, account confirmation, and password recovery when the user doesn't exist in the user pool. When set to <code>ENABLED</code> and the user doesn't exist, authentication returns an error indicating either the username or password was incorrect. Account confirmation and password recovery return a response indicating a code was sent to a simulated destination. When set to <code>LEGACY</code>, those APIs return a <code>UserNotFoundException</code> exception if the user doesn't exist in the user pool.</p>
     /// <p>Defaults to <code>LEGACY</code>.</p>
-    pub fn prevent_user_existence_errors(&self) -> ::std::option::Option<&crate::types::PreventUserExistenceErrorTypes> {
+    pub fn prevent_user_existence_errors(&self) -> ::std::option::Option<&super::super::super::types::PreventUserExistenceErrorTypes> {
         self.prevent_user_existence_errors.as_ref()
     }
     /// <p>Activates or deactivates <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html">token revocation</a> in the target app client.</p>
@@ -313,7 +313,7 @@
         self.auth_session_validity
     }
     /// <p>The configuration of your app client for refresh token rotation. When enabled, your app client issues new ID, access, and refresh tokens when users renew their sessions with refresh tokens. When disabled, token refresh issues only ID and access tokens.</p>
-    pub fn refresh_token_rotation(&self) -> ::std::option::Option<&crate::types::RefreshTokenRotationType> {
+    pub fn refresh_token_rotation(&self) -> ::std::option::Option<&super::super::super::types::RefreshTokenRotationType> {
         self.refresh_token_rotation.as_ref()
     }
 }
@@ -351,8 +351,8 @@
 }
 impl UpdateUserPoolClientInput {
     /// Creates a new builder-style object to manufacture [`UpdateUserPoolClientInput`](crate::operation::update_user_pool_client::UpdateUserPoolClientInput).
-    pub fn builder() -> crate::operation::update_user_pool_client::builders::UpdateUserPoolClientInputBuilder {
-        crate::operation::update_user_pool_client::builders::UpdateUserPoolClientInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_user_pool_client::builders::UpdateUserPoolClientInputBuilder {
+        super::super::super::operation::update_user_pool_client::builders::UpdateUserPoolClientInputBuilder::default()
     }
 }

@@ -366,23 +366,23 @@
     pub(crate) refresh_token_validity: ::std::option::Option<i32>,
     pub(crate) access_token_validity: ::std::option::Option<i32>,
     pub(crate) id_token_validity: ::std::option::Option<i32>,
-    pub(crate) token_validity_units: ::std::option::Option<crate::types::TokenValidityUnitsType>,
+    pub(crate) token_validity_units: ::std::option::Option<super::super::super::types::TokenValidityUnitsType>,
     pub(crate) read_attributes: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) write_attributes: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
-    pub(crate) explicit_auth_flows: ::std::option::Option<::std::vec::Vec<crate::types::ExplicitAuthFlowsType>>,
+    pub(crate) explicit_auth_flows: ::std::option::Option<::std::vec::Vec<super::super::super::types::ExplicitAuthFlowsType>>,
     pub(crate) supported_identity_providers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) callback_urls: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) logout_urls: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) default_redirect_uri: ::std::option::Option<::std::string::String>,
-    pub(crate) allowed_o_auth_flows: ::std::option::Option<::std::vec::Vec<crate::types::OAuthFlowType>>,
+    pub(crate) allowed_o_auth_flows: ::std::option::Option<::std::vec::Vec<super::super::super::types::OAuthFlowType>>,
     pub(crate) allowed_o_auth_scopes: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) allowed_o_auth_flows_user_pool_client: ::std::option::Option<bool>,
-    pub(crate) analytics_configuration: ::std::option::Option<crate::types::AnalyticsConfigurationType>,
-    pub(crate) prevent_user_existence_errors: ::std::option::Option<crate::types::PreventUserExistenceErrorTypes>,
+    pub(crate) analytics_configuration: ::std::option::Option<super::super::super::types::AnalyticsConfigurationType>,
+    pub(crate) prevent_user_existence_errors: ::std::option::Option<super::super::super::types::PreventUserExistenceErrorTypes>,
     pub(crate) enable_token_revocation: ::std::option::Option<bool>,
     pub(crate) enable_propagate_additional_user_context_data: ::std::option::Option<bool>,
     pub(crate) auth_session_validity: ::std::option::Option<i32>,
-    pub(crate) refresh_token_rotation: ::std::option::Option<crate::types::RefreshTokenRotationType>,
+    pub(crate) refresh_token_rotation: ::std::option::Option<super::super::super::types::RefreshTokenRotationType>,
 }
 impl UpdateUserPoolClientInputBuilder {
     /// <p>The ID of the user pool where you want to update the app client.</p>
@@ -499,17 +499,17 @@
         &self.id_token_validity
     }
     /// <p>The units that validity times are represented in. The default unit for refresh tokens is days, and the default for ID and access tokens are hours.</p>
-    pub fn token_validity_units(mut self, input: crate::types::TokenValidityUnitsType) -> Self {
+    pub fn token_validity_units(mut self, input: super::super::super::types::TokenValidityUnitsType) -> Self {
         self.token_validity_units = ::std::option::Option::Some(input);
         self
     }
     /// <p>The units that validity times are represented in. The default unit for refresh tokens is days, and the default for ID and access tokens are hours.</p>
-    pub fn set_token_validity_units(mut self, input: ::std::option::Option<crate::types::TokenValidityUnitsType>) -> Self {
+    pub fn set_token_validity_units(mut self, input: ::std::option::Option<super::super::super::types::TokenValidityUnitsType>) -> Self {
         self.token_validity_units = input;
         self
     }
     /// <p>The units that validity times are represented in. The default unit for refresh tokens is days, and the default for ID and access tokens are hours.</p>
-    pub fn get_token_validity_units(&self) -> &::std::option::Option<crate::types::TokenValidityUnitsType> {
+    pub fn get_token_validity_units(&self) -> &::std::option::Option<super::super::super::types::TokenValidityUnitsType> {
         &self.token_validity_units
     }
     /// Appends an item to `read_attributes`.
@@ -585,7 +585,7 @@
     /// <p><code>ALLOW_REFRESH_TOKEN_AUTH</code>: Enable authflow to refresh tokens.</p></li>
     /// </ul>
     /// <p>In some environments, you will see the values <code>ADMIN_NO_SRP_AUTH</code>, <code>CUSTOM_AUTH_FLOW_ONLY</code>, or <code>USER_PASSWORD_AUTH</code>. You can't assign these legacy <code>ExplicitAuthFlows</code> values to user pool clients at the same time as values that begin with <code>ALLOW_</code>, like <code>ALLOW_USER_SRP_AUTH</code>.</p>
-    pub fn explicit_auth_flows(mut self, input: crate::types::ExplicitAuthFlowsType) -> Self {
+    pub fn explicit_auth_flows(mut self, input: super::super::super::types::ExplicitAuthFlowsType) -> Self {
         let mut v = self.explicit_auth_flows.unwrap_or_default();
         v.push(input);
         self.explicit_auth_flows = ::std::option::Option::Some(v);
@@ -611,7 +611,7 @@
     /// <p><code>ALLOW_REFRESH_TOKEN_AUTH</code>: Enable authflow to refresh tokens.</p></li>
     /// </ul>
     /// <p>In some environments, you will see the values <code>ADMIN_NO_SRP_AUTH</code>, <code>CUSTOM_AUTH_FLOW_ONLY</code>, or <code>USER_PASSWORD_AUTH</code>. You can't assign these legacy <code>ExplicitAuthFlows</code> values to user pool clients at the same time as values that begin with <code>ALLOW_</code>, like <code>ALLOW_USER_SRP_AUTH</code>.</p>
-    pub fn set_explicit_auth_flows(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ExplicitAuthFlowsType>>) -> Self {
+    pub fn set_explicit_auth_flows(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::ExplicitAuthFlowsType>>) -> Self {
         self.explicit_auth_flows = input;
         self
     }
@@ -635,7 +635,7 @@
     /// <p><code>ALLOW_REFRESH_TOKEN_AUTH</code>: Enable authflow to refresh tokens.</p></li>
     /// </ul>
     /// <p>In some environments, you will see the values <code>ADMIN_NO_SRP_AUTH</code>, <code>CUSTOM_AUTH_FLOW_ONLY</code>, or <code>USER_PASSWORD_AUTH</code>. You can't assign these legacy <code>ExplicitAuthFlows</code> values to user pool clients at the same time as values that begin with <code>ALLOW_</code>, like <code>ALLOW_USER_SRP_AUTH</code>.</p>
-    pub fn get_explicit_auth_flows(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ExplicitAuthFlowsType>> {
+    pub fn get_explicit_auth_flows(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::ExplicitAuthFlowsType>> {
         &self.explicit_auth_flows
     }
     /// Appends an item to `supported_identity_providers`.
@@ -776,7 +776,7 @@
     /// <p>Issue the access token from the <code>/oauth2/token</code> endpoint directly to a non-person user using a combination of the client ID and client secret.</p>
     /// </dd>
     /// </dl>
-    pub fn allowed_o_auth_flows(mut self, input: crate::types::OAuthFlowType) -> Self {
+    pub fn allowed_o_auth_flows(mut self, input: super::super::super::types::OAuthFlowType) -> Self {
         let mut v = self.allowed_o_auth_flows.unwrap_or_default();
         v.push(input);
         self.allowed_o_auth_flows = ::std::option::Option::Some(v);
@@ -803,7 +803,7 @@
     /// <p>Issue the access token from the <code>/oauth2/token</code> endpoint directly to a non-person user using a combination of the client ID and client secret.</p>
     /// </dd>
     /// </dl>
-    pub fn set_allowed_o_auth_flows(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::OAuthFlowType>>) -> Self {
+    pub fn set_allowed_o_auth_flows(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::OAuthFlowType>>) -> Self {
         self.allowed_o_auth_flows = input;
         self
     }
@@ -828,7 +828,7 @@
     /// <p>Issue the access token from the <code>/oauth2/token</code> endpoint directly to a non-person user using a combination of the client ID and client secret.</p>
     /// </dd>
     /// </dl>
-    pub fn get_allowed_o_auth_flows(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::OAuthFlowType>> {
+    pub fn get_allowed_o_auth_flows(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::OAuthFlowType>> {
         &self.allowed_o_auth_flows
     }
     /// Appends an item to `allowed_o_auth_scopes`.
@@ -903,36 +903,36 @@
     }
     /// <p>The user pool analytics configuration for collecting metrics and sending them to your Amazon Pinpoint campaign.</p>
     /// <p>In Amazon Web Services Regions where Amazon Pinpoint isn't available, user pools might not have access to analytics or might be configurable with campaigns in the US East (N. Virginia) Region. For more information, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html">Using Amazon Pinpoint analytics</a>.</p>
-    pub fn analytics_configuration(mut self, input: crate::types::AnalyticsConfigurationType) -> Self {
+    pub fn analytics_configuration(mut self, input: super::super::super::types::AnalyticsConfigurationType) -> Self {
         self.analytics_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>The user pool analytics configuration for collecting metrics and sending them to your Amazon Pinpoint campaign.</p>
     /// <p>In Amazon Web Services Regions where Amazon Pinpoint isn't available, user pools might not have access to analytics or might be configurable with campaigns in the US East (N. Virginia) Region. For more information, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html">Using Amazon Pinpoint analytics</a>.</p>
-    pub fn set_analytics_configuration(mut self, input: ::std::option::Option<crate::types::AnalyticsConfigurationType>) -> Self {
+    pub fn set_analytics_configuration(mut self, input: ::std::option::Option<super::super::super::types::AnalyticsConfigurationType>) -> Self {
         self.analytics_configuration = input;
         self
     }
     /// <p>The user pool analytics configuration for collecting metrics and sending them to your Amazon Pinpoint campaign.</p>
     /// <p>In Amazon Web Services Regions where Amazon Pinpoint isn't available, user pools might not have access to analytics or might be configurable with campaigns in the US East (N. Virginia) Region. For more information, see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html">Using Amazon Pinpoint analytics</a>.</p>
-    pub fn get_analytics_configuration(&self) -> &::std::option::Option<crate::types::AnalyticsConfigurationType> {
+    pub fn get_analytics_configuration(&self) -> &::std::option::Option<super::super::super::types::AnalyticsConfigurationType> {
         &self.analytics_configuration
     }
     /// <p>When <code>ENABLED</code>, suppresses messages that might indicate a valid user exists when someone attempts sign-in. This parameters sets your preference for the errors and responses that you want Amazon Cognito APIs to return during authentication, account confirmation, and password recovery when the user doesn't exist in the user pool. When set to <code>ENABLED</code> and the user doesn't exist, authentication returns an error indicating either the username or password was incorrect. Account confirmation and password recovery return a response indicating a code was sent to a simulated destination. When set to <code>LEGACY</code>, those APIs return a <code>UserNotFoundException</code> exception if the user doesn't exist in the user pool.</p>
     /// <p>Defaults to <code>LEGACY</code>.</p>
-    pub fn prevent_user_existence_errors(mut self, input: crate::types::PreventUserExistenceErrorTypes) -> Self {
+    pub fn prevent_user_existence_errors(mut self, input: super::super::super::types::PreventUserExistenceErrorTypes) -> Self {
         self.prevent_user_existence_errors = ::std::option::Option::Some(input);
         self
     }
     /// <p>When <code>ENABLED</code>, suppresses messages that might indicate a valid user exists when someone attempts sign-in. This parameters sets your preference for the errors and responses that you want Amazon Cognito APIs to return during authentication, account confirmation, and password recovery when the user doesn't exist in the user pool. When set to <code>ENABLED</code> and the user doesn't exist, authentication returns an error indicating either the username or password was incorrect. Account confirmation and password recovery return a response indicating a code was sent to a simulated destination. When set to <code>LEGACY</code>, those APIs return a <code>UserNotFoundException</code> exception if the user doesn't exist in the user pool.</p>
     /// <p>Defaults to <code>LEGACY</code>.</p>
-    pub fn set_prevent_user_existence_errors(mut self, input: ::std::option::Option<crate::types::PreventUserExistenceErrorTypes>) -> Self {
+    pub fn set_prevent_user_existence_errors(mut self, input: ::std::option::Option<super::super::super::types::PreventUserExistenceErrorTypes>) -> Self {
         self.prevent_user_existence_errors = input;
         self
     }
     /// <p>When <code>ENABLED</code>, suppresses messages that might indicate a valid user exists when someone attempts sign-in. This parameters sets your preference for the errors and responses that you want Amazon Cognito APIs to return during authentication, account confirmation, and password recovery when the user doesn't exist in the user pool. When set to <code>ENABLED</code> and the user doesn't exist, authentication returns an error indicating either the username or password was incorrect. Account confirmation and password recovery return a response indicating a code was sent to a simulated destination. When set to <code>LEGACY</code>, those APIs return a <code>UserNotFoundException</code> exception if the user doesn't exist in the user pool.</p>
     /// <p>Defaults to <code>LEGACY</code>.</p>
-    pub fn get_prevent_user_existence_errors(&self) -> &::std::option::Option<crate::types::PreventUserExistenceErrorTypes> {
+    pub fn get_prevent_user_existence_errors(&self) -> &::std::option::Option<super::super::super::types::PreventUserExistenceErrorTypes> {
         &self.prevent_user_existence_errors
     }
     /// <p>Activates or deactivates <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html">token revocation</a> in the target app client.</p>
@@ -978,25 +978,25 @@
         &self.auth_session_validity
     }
     /// <p>The configuration of your app client for refresh token rotation. When enabled, your app client issues new ID, access, and refresh tokens when users renew their sessions with refresh tokens. When disabled, token refresh issues only ID and access tokens.</p>
-    pub fn refresh_token_rotation(mut self, input: crate::types::RefreshTokenRotationType) -> Self {
+    pub fn refresh_token_rotation(mut self, input: super::super::super::types::RefreshTokenRotationType) -> Self {
         self.refresh_token_rotation = ::std::option::Option::Some(input);
         self
     }
     /// <p>The configuration of your app client for refresh token rotation. When enabled, your app client issues new ID, access, and refresh tokens when users renew their sessions with refresh tokens. When disabled, token refresh issues only ID and access tokens.</p>
-    pub fn set_refresh_token_rotation(mut self, input: ::std::option::Option<crate::types::RefreshTokenRotationType>) -> Self {
+    pub fn set_refresh_token_rotation(mut self, input: ::std::option::Option<super::super::super::types::RefreshTokenRotationType>) -> Self {
         self.refresh_token_rotation = input;
         self
     }
     /// <p>The configuration of your app client for refresh token rotation. When enabled, your app client issues new ID, access, and refresh tokens when users renew their sessions with refresh tokens. When disabled, token refresh issues only ID and access tokens.</p>
-    pub fn get_refresh_token_rotation(&self) -> &::std::option::Option<crate::types::RefreshTokenRotationType> {
+    pub fn get_refresh_token_rotation(&self) -> &::std::option::Option<super::super::super::types::RefreshTokenRotationType> {
         &self.refresh_token_rotation
     }
     /// Consumes the builder and constructs a [`UpdateUserPoolClientInput`](crate::operation::update_user_pool_client::UpdateUserPoolClientInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_user_pool_client::UpdateUserPoolClientInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::update_user_pool_client::UpdateUserPoolClientInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_user_pool_client::UpdateUserPoolClientInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_user_pool_client::UpdateUserPoolClientInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
             client_name: self.client_name,
