--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl GetBucketLifecycleConfigurationInput {
     /// Creates a new builder-style object to manufacture [`GetBucketLifecycleConfigurationInput`](crate::operation::get_bucket_lifecycle_configuration::GetBucketLifecycleConfigurationInput).
-    pub fn builder() -> crate::operation::get_bucket_lifecycle_configuration::builders::GetBucketLifecycleConfigurationInputBuilder {
-        crate::operation::get_bucket_lifecycle_configuration::builders::GetBucketLifecycleConfigurationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_bucket_lifecycle_configuration::builders::GetBucketLifecycleConfigurationInputBuilder {
+        super::super::super::operation::get_bucket_lifecycle_configuration::builders::GetBucketLifecycleConfigurationInputBuilder::default()
     }
 }

@@ -76,11 +76,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_bucket_lifecycle_configuration::GetBucketLifecycleConfigurationInput,
+        super::super::super::operation::get_bucket_lifecycle_configuration::GetBucketLifecycleConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_bucket_lifecycle_configuration::GetBucketLifecycleConfigurationInput {
+            super::super::super::operation::get_bucket_lifecycle_configuration::GetBucketLifecycleConfigurationInput {
                 bucket: self.bucket,
                 expected_bucket_owner: self.expected_bucket_owner,
             },
