--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetFunctionRecursionConfigInput {
     /// Creates a new builder-style object to manufacture [`GetFunctionRecursionConfigInput`](crate::operation::get_function_recursion_config::GetFunctionRecursionConfigInput).
-    pub fn builder() -> crate::operation::get_function_recursion_config::builders::GetFunctionRecursionConfigInputBuilder {
-        crate::operation::get_function_recursion_config::builders::GetFunctionRecursionConfigInputBuilder::default()
+    pub fn builder() -> super::operation::get_function_recursion_config::builders::GetFunctionRecursionConfigInputBuilder {
+        super::operation::get_function_recursion_config::builders::GetFunctionRecursionConfigInputBuilder::default()
     }
 }

@@ -45,10 +45,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_function_recursion_config::GetFunctionRecursionConfigInput,
+        super::operation::get_function_recursion_config::GetFunctionRecursionConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_function_recursion_config::GetFunctionRecursionConfigInput {
+        ::std::result::Result::Ok(super::operation::get_function_recursion_config::GetFunctionRecursionConfigInput {
             function_name: self.function_name,
         })
     }
