--- original
+++ modified
@@ -7,9 +7,9 @@
     /// <p>The Amazon Resource Name (ARN) of the scheduling policy to update.</p>
     pub arn: ::std::option::Option<::std::string::String>,
     /// <p>The quota share scheduling policy details. Once set during creation, a quotaSharePolicy cannot be removed or changed to a fairsharePolicy.</p>
-    pub quota_share_policy: ::std::option::Option<crate::types::QuotaSharePolicy>,
+    pub quota_share_policy: ::std::option::Option<super::super::super::types::QuotaSharePolicy>,
     /// <p>The fair-share policy scheduling details. Once set during creation, a fairsharePolicy cannot be removed or changed to a quotaSharePolicy.</p>
-    pub fairshare_policy: ::std::option::Option<crate::types::FairsharePolicy>,
+    pub fairshare_policy: ::std::option::Option<super::super::super::types::FairsharePolicy>,
 }
 impl UpdateSchedulingPolicyInput {
     /// <p>The Amazon Resource Name (ARN) of the scheduling policy to update.</p>
@@ -17,18 +17,18 @@
         self.arn.as_deref()
     }
     /// <p>The quota share scheduling policy details. Once set during creation, a quotaSharePolicy cannot be removed or changed to a fairsharePolicy.</p>
-    pub fn quota_share_policy(&self) -> ::std::option::Option<&crate::types::QuotaSharePolicy> {
+    pub fn quota_share_policy(&self) -> ::std::option::Option<&super::super::super::types::QuotaSharePolicy> {
         self.quota_share_policy.as_ref()
     }
     /// <p>The fair-share policy scheduling details. Once set during creation, a fairsharePolicy cannot be removed or changed to a quotaSharePolicy.</p>
-    pub fn fairshare_policy(&self) -> ::std::option::Option<&crate::types::FairsharePolicy> {
+    pub fn fairshare_policy(&self) -> ::std::option::Option<&super::super::super::types::FairsharePolicy> {
         self.fairshare_policy.as_ref()
     }
 }
 impl UpdateSchedulingPolicyInput {
     /// Creates a new builder-style object to manufacture [`UpdateSchedulingPolicyInput`](crate::operation::update_scheduling_policy::UpdateSchedulingPolicyInput).
-    pub fn builder() -> crate::operation::update_scheduling_policy::builders::UpdateSchedulingPolicyInputBuilder {
-        crate::operation::update_scheduling_policy::builders::UpdateSchedulingPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_scheduling_policy::builders::UpdateSchedulingPolicyInputBuilder {
+        super::super::super::operation::update_scheduling_policy::builders::UpdateSchedulingPolicyInputBuilder::default()
     }
 }

@@ -37,8 +37,8 @@
 #[non_exhaustive]
 pub struct UpdateSchedulingPolicyInputBuilder {
     pub(crate) arn: ::std::option::Option<::std::string::String>,
-    pub(crate) quota_share_policy: ::std::option::Option<crate::types::QuotaSharePolicy>,
-    pub(crate) fairshare_policy: ::std::option::Option<crate::types::FairsharePolicy>,
+    pub(crate) quota_share_policy: ::std::option::Option<super::super::super::types::QuotaSharePolicy>,
+    pub(crate) fairshare_policy: ::std::option::Option<super::super::super::types::FairsharePolicy>,
 }
 impl UpdateSchedulingPolicyInputBuilder {
     /// <p>The Amazon Resource Name (ARN) of the scheduling policy to update.</p>
@@ -57,31 +57,31 @@
         &self.arn
     }
     /// <p>The quota share scheduling policy details. Once set during creation, a quotaSharePolicy cannot be removed or changed to a fairsharePolicy.</p>
-    pub fn quota_share_policy(mut self, input: crate::types::QuotaSharePolicy) -> Self {
+    pub fn quota_share_policy(mut self, input: super::super::super::types::QuotaSharePolicy) -> Self {
         self.quota_share_policy = ::std::option::Option::Some(input);
         self
     }
     /// <p>The quota share scheduling policy details. Once set during creation, a quotaSharePolicy cannot be removed or changed to a fairsharePolicy.</p>
-    pub fn set_quota_share_policy(mut self, input: ::std::option::Option<crate::types::QuotaSharePolicy>) -> Self {
+    pub fn set_quota_share_policy(mut self, input: ::std::option::Option<super::super::super::types::QuotaSharePolicy>) -> Self {
         self.quota_share_policy = input;
         self
     }
     /// <p>The quota share scheduling policy details. Once set during creation, a quotaSharePolicy cannot be removed or changed to a fairsharePolicy.</p>
-    pub fn get_quota_share_policy(&self) -> &::std::option::Option<crate::types::QuotaSharePolicy> {
+    pub fn get_quota_share_policy(&self) -> &::std::option::Option<super::super::super::types::QuotaSharePolicy> {
         &self.quota_share_policy
     }
     /// <p>The fair-share policy scheduling details. Once set during creation, a fairsharePolicy cannot be removed or changed to a quotaSharePolicy.</p>
-    pub fn fairshare_policy(mut self, input: crate::types::FairsharePolicy) -> Self {
+    pub fn fairshare_policy(mut self, input: super::super::super::types::FairsharePolicy) -> Self {
         self.fairshare_policy = ::std::option::Option::Some(input);
         self
     }
     /// <p>The fair-share policy scheduling details. Once set during creation, a fairsharePolicy cannot be removed or changed to a quotaSharePolicy.</p>
-    pub fn set_fairshare_policy(mut self, input: ::std::option::Option<crate::types::FairsharePolicy>) -> Self {
+    pub fn set_fairshare_policy(mut self, input: ::std::option::Option<super::super::super::types::FairsharePolicy>) -> Self {
         self.fairshare_policy = input;
         self
     }
     /// <p>The fair-share policy scheduling details. Once set during creation, a fairsharePolicy cannot be removed or changed to a quotaSharePolicy.</p>
-    pub fn get_fairshare_policy(&self) -> &::std::option::Option<crate::types::FairsharePolicy> {
+    pub fn get_fairshare_policy(&self) -> &::std::option::Option<super::super::super::types::FairsharePolicy> {
         &self.fairshare_policy
     }
     /// Consumes the builder and constructs a [`UpdateSchedulingPolicyInput`](crate::operation::update_scheduling_policy::UpdateSchedulingPolicyInput).
@@ -88,10 +88,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_scheduling_policy::UpdateSchedulingPolicyInput,
+        super::super::super::operation::update_scheduling_policy::UpdateSchedulingPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_scheduling_policy::UpdateSchedulingPolicyInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_scheduling_policy::UpdateSchedulingPolicyInput {
             arn: self.arn,
             quota_share_policy: self.quota_share_policy,
             fairshare_policy: self.fairshare_policy,
