--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl DescribeOrganizationConformancePacksInput {
     /// Creates a new builder-style object to manufacture [`DescribeOrganizationConformancePacksInput`](crate::operation::describe_organization_conformance_packs::DescribeOrganizationConformancePacksInput).
-    pub fn builder() -> crate::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksInputBuilder {
-        crate::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksInputBuilder {
+        super::super::super::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksInputBuilder::default()
     }
 }

@@ -94,11 +94,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_organization_conformance_packs::DescribeOrganizationConformancePacksInput,
+        super::super::super::operation::describe_organization_conformance_packs::DescribeOrganizationConformancePacksInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::describe_organization_conformance_packs::DescribeOrganizationConformancePacksInput {
+            super::super::super::operation::describe_organization_conformance_packs::DescribeOrganizationConformancePacksInput {
                 organization_conformance_pack_names: self.organization_conformance_pack_names,
                 limit: self.limit,
                 next_token: self.next_token,
