--- original
+++ modified
@@ -18,8 +18,8 @@
 }
 impl PutRetentionConfigurationInput {
     /// Creates a new builder-style object to manufacture [`PutRetentionConfigurationInput`](crate::operation::put_retention_configuration::PutRetentionConfigurationInput).
-    pub fn builder() -> crate::operation::put_retention_configuration::builders::PutRetentionConfigurationInputBuilder {
-        crate::operation::put_retention_configuration::builders::PutRetentionConfigurationInputBuilder::default()
+    pub fn builder() -> super::operation::put_retention_configuration::builders::PutRetentionConfigurationInputBuilder {
+        super::operation::put_retention_configuration::builders::PutRetentionConfigurationInputBuilder::default()
     }
 }

@@ -55,10 +55,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_retention_configuration::PutRetentionConfigurationInput,
+        super::operation::put_retention_configuration::PutRetentionConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_retention_configuration::PutRetentionConfigurationInput {
+        ::std::result::Result::Ok(super::operation::put_retention_configuration::PutRetentionConfigurationInput {
             retention_period_in_days: self.retention_period_in_days,
         })
     }
