--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On success, responds with [`CreateDomainOutput`](crate::operation::create_domain::CreateDomainOutput) with field(s):
     ///   - [`domain(Option<DomainDescription>)`](crate::operation::create_domain::CreateDomainOutput::domain): <p>Contains information about the created domain after processing the request.</p>
     /// - On failure, responds with [`SdkError<CreateDomainError>`](crate::operation::create_domain::CreateDomainError)
-    pub fn create_domain(&self) -> crate::operation::create_domain::builders::CreateDomainFluentBuilder {
-        crate::operation::create_domain::builders::CreateDomainFluentBuilder::new(self.handle.clone())
+    pub fn create_domain(&self) -> super::operation::create_domain::builders::CreateDomainFluentBuilder {
+        super::operation::create_domain::builders::CreateDomainFluentBuilder::new(self.handle.clone())
     }
 }
