--- original
+++ modified
@@ -16,7 +16,7 @@
     ///   - [`successful_versions(Option<HashMap::<String, SuccessfulPackageVersionInfo>>)`](crate::operation::dispose_package_versions::DisposePackageVersionsOutput::successful_versions): <p>A list of the package versions that were successfully disposed.</p>
     ///   - [`failed_versions(Option<HashMap::<String, PackageVersionError>>)`](crate::operation::dispose_package_versions::DisposePackageVersionsOutput::failed_versions): <p>A <code>PackageVersionError</code> object that contains a map of errors codes for the disposed package versions that failed. The possible error codes are:</p> <ul>  <li>   <p><code>ALREADY_EXISTS</code></p></li>  <li>   <p><code>MISMATCHED_REVISION</code></p></li>  <li>   <p><code>MISMATCHED_STATUS</code></p></li>  <li>   <p><code>NOT_ALLOWED</code></p></li>  <li>   <p><code>NOT_FOUND</code></p></li>  <li>   <p><code>SKIPPED</code></p></li> </ul>
     /// - On failure, responds with [`SdkError<DisposePackageVersionsError>`](crate::operation::dispose_package_versions::DisposePackageVersionsError)
-    pub fn dispose_package_versions(&self) -> crate::operation::dispose_package_versions::builders::DisposePackageVersionsFluentBuilder {
-        crate::operation::dispose_package_versions::builders::DisposePackageVersionsFluentBuilder::new(self.handle.clone())
+    pub fn dispose_package_versions(&self) -> super::operation::dispose_package_versions::builders::DisposePackageVersionsFluentBuilder {
+        super::operation::dispose_package_versions::builders::DisposePackageVersionsFluentBuilder::new(self.handle.clone())
     }
 }
