--- original
+++ modified
@@ -22,7 +22,7 @@
     /// - On failure, responds with [`SdkError<ListPackageVersionDependenciesError>`](crate::operation::list_package_version_dependencies::ListPackageVersionDependenciesError)
     pub fn list_package_version_dependencies(
         &self,
-    ) -> crate::operation::list_package_version_dependencies::builders::ListPackageVersionDependenciesFluentBuilder {
-        crate::operation::list_package_version_dependencies::builders::ListPackageVersionDependenciesFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::list_package_version_dependencies::builders::ListPackageVersionDependenciesFluentBuilder {
+        super::operation::list_package_version_dependencies::builders::ListPackageVersionDependenciesFluentBuilder::new(self.handle.clone())
     }
 }
