--- original
+++ modified
@@ -13,7 +13,7 @@
     ///   - [`package_groups(Option<Vec::<PackageGroupSummary>>)`](crate::operation::list_sub_package_groups::ListSubPackageGroupsOutput::package_groups): <p>A list of sub package groups for the requested package group.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_sub_package_groups::ListSubPackageGroupsOutput::next_token): <p>If there are additional results, this is the token for the next set of results.</p>
     /// - On failure, responds with [`SdkError<ListSubPackageGroupsError>`](crate::operation::list_sub_package_groups::ListSubPackageGroupsError)
-    pub fn list_sub_package_groups(&self) -> crate::operation::list_sub_package_groups::builders::ListSubPackageGroupsFluentBuilder {
-        crate::operation::list_sub_package_groups::builders::ListSubPackageGroupsFluentBuilder::new(self.handle.clone())
+    pub fn list_sub_package_groups(&self) -> super::super::operation::list_sub_package_groups::builders::ListSubPackageGroupsFluentBuilder {
+        super::super::operation::list_sub_package_groups::builders::ListSubPackageGroupsFluentBuilder::new(self.handle.clone())
     }
 }
