--- original
+++ modified
@@ -22,7 +22,7 @@
     ///   - [`next_token(Option<String>)`](crate::operation::list_package_version_assets::ListPackageVersionAssetsOutput::next_token): <p>If there are additional results, this is the token for the next set of results.</p>
     ///   - [`assets(Option<Vec::<AssetSummary>>)`](crate::operation::list_package_version_assets::ListPackageVersionAssetsOutput::assets): <p>The returned list of <a href="https://docs.aws.amazon.com/codeartifact/latest/APIReference/API_AssetSummary.html">AssetSummary</a> objects.</p>
     /// - On failure, responds with [`SdkError<ListPackageVersionAssetsError>`](crate::operation::list_package_version_assets::ListPackageVersionAssetsError)
-    pub fn list_package_version_assets(&self) -> crate::operation::list_package_version_assets::builders::ListPackageVersionAssetsFluentBuilder {
-        crate::operation::list_package_version_assets::builders::ListPackageVersionAssetsFluentBuilder::new(self.handle.clone())
+    pub fn list_package_version_assets(&self) -> super::operation::list_package_version_assets::builders::ListPackageVersionAssetsFluentBuilder {
+        super::operation::list_package_version_assets::builders::ListPackageVersionAssetsFluentBuilder::new(self.handle.clone())
     }
 }
