--- original
+++ modified
@@ -15,7 +15,7 @@
     /// - On failure, responds with [`SdkError<PutPackageOriginConfigurationError>`](crate::operation::put_package_origin_configuration::PutPackageOriginConfigurationError)
     pub fn put_package_origin_configuration(
         &self,
-    ) -> crate::operation::put_package_origin_configuration::builders::PutPackageOriginConfigurationFluentBuilder {
-        crate::operation::put_package_origin_configuration::builders::PutPackageOriginConfigurationFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::put_package_origin_configuration::builders::PutPackageOriginConfigurationFluentBuilder {
+        super::super::operation::put_package_origin_configuration::builders::PutPackageOriginConfigurationFluentBuilder::new(self.handle.clone())
     }
 }
