--- original
+++ modified
@@ -12,7 +12,7 @@
     /// - On failure, responds with [`SdkError<PutDomainPermissionsPolicyError>`](crate::operation::put_domain_permissions_policy::PutDomainPermissionsPolicyError)
     pub fn put_domain_permissions_policy(
         &self,
-    ) -> crate::operation::put_domain_permissions_policy::builders::PutDomainPermissionsPolicyFluentBuilder {
-        crate::operation::put_domain_permissions_policy::builders::PutDomainPermissionsPolicyFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::put_domain_permissions_policy::builders::PutDomainPermissionsPolicyFluentBuilder {
+        super::super::operation::put_domain_permissions_policy::builders::PutDomainPermissionsPolicyFluentBuilder::new(self.handle.clone())
     }
 }
