--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DescribeDeliveryDestinationsInput {
     /// Creates a new builder-style object to manufacture [`DescribeDeliveryDestinationsInput`](crate::operation::describe_delivery_destinations::DescribeDeliveryDestinationsInput).
-    pub fn builder() -> crate::operation::describe_delivery_destinations::builders::DescribeDeliveryDestinationsInputBuilder {
-        crate::operation::describe_delivery_destinations::builders::DescribeDeliveryDestinationsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_delivery_destinations::builders::DescribeDeliveryDestinationsInputBuilder {
+        super::super::super::operation::describe_delivery_destinations::builders::DescribeDeliveryDestinationsInputBuilder::default()
     }
 }

@@ -65,10 +65,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_delivery_destinations::DescribeDeliveryDestinationsInput,
+        super::super::super::operation::describe_delivery_destinations::DescribeDeliveryDestinationsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_delivery_destinations::DescribeDeliveryDestinationsInput {
+        ::std::result::Result::Ok(super::super::super::operation::describe_delivery_destinations::DescribeDeliveryDestinationsInput {
             next_token: self.next_token,
             limit: self.limit,
         })
