--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl CreateGroupInput {
     /// Creates a new builder-style object to manufacture [`CreateGroupInput`](crate::operation::create_group::CreateGroupInput).
-    pub fn builder() -> crate::operation::create_group::builders::CreateGroupInputBuilder {
-        crate::operation::create_group::builders::CreateGroupInputBuilder::default()
+    pub fn builder() -> super::operation::create_group::builders::CreateGroupInputBuilder {
+        super::operation::create_group::builders::CreateGroupInputBuilder::default()
     }
 }

@@ -137,8 +137,8 @@
         &self.precedence
     }
     /// Consumes the builder and constructs a [`CreateGroupInput`](crate::operation::create_group::CreateGroupInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::create_group::CreateGroupInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_group::CreateGroupInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::create_group::CreateGroupInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::create_group::CreateGroupInput {
             group_name: self.group_name,
             user_pool_id: self.user_pool_id,
             description: self.description,
