--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl StartMessageMoveTaskInput {
     /// Creates a new builder-style object to manufacture [`StartMessageMoveTaskInput`](crate::operation::start_message_move_task::StartMessageMoveTaskInput).
-    pub fn builder() -> crate::operation::start_message_move_task::builders::StartMessageMoveTaskInputBuilder {
-        crate::operation::start_message_move_task::builders::StartMessageMoveTaskInputBuilder::default()
+    pub fn builder() -> super::operation::start_message_move_task::builders::StartMessageMoveTaskInputBuilder {
+        super::operation::start_message_move_task::builders::StartMessageMoveTaskInputBuilder::default()
     }
 }

@@ -86,9 +86,9 @@
     /// Consumes the builder and constructs a [`StartMessageMoveTaskInput`](crate::operation::start_message_move_task::StartMessageMoveTaskInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::start_message_move_task::StartMessageMoveTaskInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::start_message_move_task::StartMessageMoveTaskInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::start_message_move_task::StartMessageMoveTaskInput {
+        ::std::result::Result::Ok(super::operation::start_message_move_task::StartMessageMoveTaskInput {
             source_arn: self.source_arn,
             destination_arn: self.destination_arn,
             max_number_of_messages_per_second: self.max_number_of_messages_per_second,
