--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ListExportsInput {
     /// Creates a new builder-style object to manufacture [`ListExportsInput`](crate::operation::list_exports::ListExportsInput).
-    pub fn builder() -> crate::operation::list_exports::builders::ListExportsInputBuilder {
-        crate::operation::list_exports::builders::ListExportsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_exports::builders::ListExportsInputBuilder {
+        super::super::super::operation::list_exports::builders::ListExportsInputBuilder::default()
     }
 }

@@ -83,8 +83,8 @@
         &self.next_token
     }
     /// Consumes the builder and constructs a [`ListExportsInput`](crate::operation::list_exports::ListExportsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_exports::ListExportsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_exports::ListExportsInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::list_exports::ListExportsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_exports::ListExportsInput {
             table_arn: self.table_arn,
             max_results: self.max_results,
             next_token: self.next_token,
