--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetBucketLoggingInput {
     /// Creates a new builder-style object to manufacture [`GetBucketLoggingInput`](crate::operation::get_bucket_logging::GetBucketLoggingInput).
-    pub fn builder() -> crate::operation::get_bucket_logging::builders::GetBucketLoggingInputBuilder {
-        crate::operation::get_bucket_logging::builders::GetBucketLoggingInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_bucket_logging::builders::GetBucketLoggingInputBuilder {
+        super::super::super::operation::get_bucket_logging::builders::GetBucketLoggingInputBuilder::default()
     }
 }

@@ -65,8 +65,8 @@
     /// Consumes the builder and constructs a [`GetBucketLoggingInput`](crate::operation::get_bucket_logging::GetBucketLoggingInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_bucket_logging::GetBucketLoggingInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_bucket_logging::GetBucketLoggingInput {
+    ) -> ::std::result::Result<super::super::super::operation::get_bucket_logging::GetBucketLoggingInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_bucket_logging::GetBucketLoggingInput {
             bucket: self.bucket,
             expected_bucket_owner: self.expected_bucket_owner,
         })
