--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>An updated description for the scheduled query.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>The updated query language for the scheduled query.</p>
-    pub query_language: ::std::option::Option<crate::types::QueryLanguage>,
+    pub query_language: ::std::option::Option<super::super::super::types::QueryLanguage>,
     /// <p>The updated query string to execute.</p>
     pub query_string: ::std::option::Option<::std::string::String>,
     /// <p>The updated array of log group names or ARNs to query.</p>
@@ -22,7 +22,7 @@
     /// <p>The updated time offset in seconds that defines the end of the lookback period for the query.</p>
     pub end_time_offset: ::std::option::Option<i64>,
     /// <p>The updated configuration for where to deliver query results.</p>
-    pub destination_configuration: ::std::option::Option<crate::types::DestinationConfiguration>,
+    pub destination_configuration: ::std::option::Option<super::super::super::types::DestinationConfiguration>,
     /// <p>The updated start time for the scheduled query in Unix epoch format.</p>
     pub schedule_start_time: ::std::option::Option<i64>,
     /// <p>The updated end time for the scheduled query in Unix epoch format.</p>
@@ -30,7 +30,7 @@
     /// <p>The updated ARN of the IAM role that grants permissions to execute the query and deliver results.</p>
     pub execution_role_arn: ::std::option::Option<::std::string::String>,
     /// <p>The updated state of the scheduled query.</p>
-    pub state: ::std::option::Option<crate::types::ScheduledQueryState>,
+    pub state: ::std::option::Option<super::super::super::types::ScheduledQueryState>,
 }
 impl UpdateScheduledQueryInput {
     /// <p>The ARN or name of the scheduled query to update.</p>
@@ -42,7 +42,7 @@
         self.description.as_deref()
     }
     /// <p>The updated query language for the scheduled query.</p>
-    pub fn query_language(&self) -> ::std::option::Option<&crate::types::QueryLanguage> {
+    pub fn query_language(&self) -> ::std::option::Option<&super::super::super::types::QueryLanguage> {
         self.query_language.as_ref()
     }
     /// <p>The updated query string to execute.</p>
@@ -72,7 +72,7 @@
         self.end_time_offset
     }
     /// <p>The updated configuration for where to deliver query results.</p>
-    pub fn destination_configuration(&self) -> ::std::option::Option<&crate::types::DestinationConfiguration> {
+    pub fn destination_configuration(&self) -> ::std::option::Option<&super::super::super::types::DestinationConfiguration> {
         self.destination_configuration.as_ref()
     }
     /// <p>The updated start time for the scheduled query in Unix epoch format.</p>
@@ -88,14 +88,14 @@
         self.execution_role_arn.as_deref()
     }
     /// <p>The updated state of the scheduled query.</p>
-    pub fn state(&self) -> ::std::option::Option<&crate::types::ScheduledQueryState> {
+    pub fn state(&self) -> ::std::option::Option<&super::super::super::types::ScheduledQueryState> {
         self.state.as_ref()
     }
 }
 impl UpdateScheduledQueryInput {
     /// Creates a new builder-style object to manufacture [`UpdateScheduledQueryInput`](crate::operation::update_scheduled_query::UpdateScheduledQueryInput).
-    pub fn builder() -> crate::operation::update_scheduled_query::builders::UpdateScheduledQueryInputBuilder {
-        crate::operation::update_scheduled_query::builders::UpdateScheduledQueryInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_scheduled_query::builders::UpdateScheduledQueryInputBuilder {
+        super::super::super::operation::update_scheduled_query::builders::UpdateScheduledQueryInputBuilder::default()
     }
 }

@@ -105,7 +105,7 @@
 pub struct UpdateScheduledQueryInputBuilder {
     pub(crate) identifier: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) query_language: ::std::option::Option<crate::types::QueryLanguage>,
+    pub(crate) query_language: ::std::option::Option<super::super::super::types::QueryLanguage>,
     pub(crate) query_string: ::std::option::Option<::std::string::String>,
     pub(crate) log_group_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) schedule_expression: ::std::option::Option<::std::string::String>,
@@ -112,11 +112,11 @@
     pub(crate) timezone: ::std::option::Option<::std::string::String>,
     pub(crate) start_time_offset: ::std::option::Option<i64>,
     pub(crate) end_time_offset: ::std::option::Option<i64>,
-    pub(crate) destination_configuration: ::std::option::Option<crate::types::DestinationConfiguration>,
+    pub(crate) destination_configuration: ::std::option::Option<super::super::super::types::DestinationConfiguration>,
     pub(crate) schedule_start_time: ::std::option::Option<i64>,
     pub(crate) schedule_end_time: ::std::option::Option<i64>,
     pub(crate) execution_role_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) state: ::std::option::Option<crate::types::ScheduledQueryState>,
+    pub(crate) state: ::std::option::Option<super::super::super::types::ScheduledQueryState>,
 }
 impl UpdateScheduledQueryInputBuilder {
     /// <p>The ARN or name of the scheduled query to update.</p>
@@ -150,17 +150,17 @@
     }
     /// <p>The updated query language for the scheduled query.</p>
     /// This field is required.
-    pub fn query_language(mut self, input: crate::types::QueryLanguage) -> Self {
+    pub fn query_language(mut self, input: super::super::super::types::QueryLanguage) -> Self {
         self.query_language = ::std::option::Option::Some(input);
         self
     }
     /// <p>The updated query language for the scheduled query.</p>
-    pub fn set_query_language(mut self, input: ::std::option::Option<crate::types::QueryLanguage>) -> Self {
+    pub fn set_query_language(mut self, input: ::std::option::Option<super::super::super::types::QueryLanguage>) -> Self {
         self.query_language = input;
         self
     }
     /// <p>The updated query language for the scheduled query.</p>
-    pub fn get_query_language(&self) -> &::std::option::Option<crate::types::QueryLanguage> {
+    pub fn get_query_language(&self) -> &::std::option::Option<super::super::super::types::QueryLanguage> {
         &self.query_language
     }
     /// <p>The updated query string to execute.</p>
@@ -256,17 +256,17 @@
         &self.end_time_offset
     }
     /// <p>The updated configuration for where to deliver query results.</p>
-    pub fn destination_configuration(mut self, input: crate::types::DestinationConfiguration) -> Self {
+    pub fn destination_configuration(mut self, input: super::super::super::types::DestinationConfiguration) -> Self {
         self.destination_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>The updated configuration for where to deliver query results.</p>
-    pub fn set_destination_configuration(mut self, input: ::std::option::Option<crate::types::DestinationConfiguration>) -> Self {
+    pub fn set_destination_configuration(mut self, input: ::std::option::Option<super::super::super::types::DestinationConfiguration>) -> Self {
         self.destination_configuration = input;
         self
     }
     /// <p>The updated configuration for where to deliver query results.</p>
-    pub fn get_destination_configuration(&self) -> &::std::option::Option<crate::types::DestinationConfiguration> {
+    pub fn get_destination_configuration(&self) -> &::std::option::Option<super::super::super::types::DestinationConfiguration> {
         &self.destination_configuration
     }
     /// <p>The updated start time for the scheduled query in Unix epoch format.</p>
@@ -313,25 +313,25 @@
         &self.execution_role_arn
     }
     /// <p>The updated state of the scheduled query.</p>
-    pub fn state(mut self, input: crate::types::ScheduledQueryState) -> Self {
+    pub fn state(mut self, input: super::super::super::types::ScheduledQueryState) -> Self {
         self.state = ::std::option::Option::Some(input);
         self
     }
     /// <p>The updated state of the scheduled query.</p>
-    pub fn set_state(mut self, input: ::std::option::Option<crate::types::ScheduledQueryState>) -> Self {
+    pub fn set_state(mut self, input: ::std::option::Option<super::super::super::types::ScheduledQueryState>) -> Self {
         self.state = input;
         self
     }
     /// <p>The updated state of the scheduled query.</p>
-    pub fn get_state(&self) -> &::std::option::Option<crate::types::ScheduledQueryState> {
+    pub fn get_state(&self) -> &::std::option::Option<super::super::super::types::ScheduledQueryState> {
         &self.state
     }
     /// Consumes the builder and constructs a [`UpdateScheduledQueryInput`](crate::operation::update_scheduled_query::UpdateScheduledQueryInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_scheduled_query::UpdateScheduledQueryInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::update_scheduled_query::UpdateScheduledQueryInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_scheduled_query::UpdateScheduledQueryInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_scheduled_query::UpdateScheduledQueryInput {
             identifier: self.identifier,
             description: self.description,
             query_language: self.query_language,
