--- original
+++ modified
@@ -29,8 +29,8 @@
 impl DescribeOrganizationConformancePackStatusesInput {
     /// Creates a new builder-style object to manufacture [`DescribeOrganizationConformancePackStatusesInput`](crate::operation::describe_organization_conformance_pack_statuses::DescribeOrganizationConformancePackStatusesInput).
     pub fn builder(
-    ) -> crate::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesInputBuilder {
-        crate::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesInputBuilder::default(
+    ) -> super::super::super::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesInputBuilder {
+        super::super::super::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesInputBuilder::default(
         )
     }
 }
@@ -96,11 +96,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_organization_conformance_pack_statuses::DescribeOrganizationConformancePackStatusesInput,
+        super::super::super::operation::describe_organization_conformance_pack_statuses::DescribeOrganizationConformancePackStatusesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::describe_organization_conformance_pack_statuses::DescribeOrganizationConformancePackStatusesInput {
+            super::super::super::operation::describe_organization_conformance_pack_statuses::DescribeOrganizationConformancePackStatusesInput {
                 organization_conformance_pack_names: self.organization_conformance_pack_names,
                 limit: self.limit,
                 next_token: self.next_token,
