--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AggregateConformancePackComplianceSummary {
     /// <p>Returns an <code>AggregateConformancePackComplianceCount</code> object.</p>
-    pub compliance_summary: ::std::option::Option<crate::types::AggregateConformancePackComplianceCount>,
+    pub compliance_summary: ::std::option::Option<super::types::AggregateConformancePackComplianceCount>,
     /// <p>Groups the result based on Amazon Web Services account ID or Amazon Web Services Region.</p>
     pub group_name: ::std::option::Option<::std::string::String>,
 }
 impl AggregateConformancePackComplianceSummary {
     /// <p>Returns an <code>AggregateConformancePackComplianceCount</code> object.</p>
-    pub fn compliance_summary(&self) -> ::std::option::Option<&crate::types::AggregateConformancePackComplianceCount> {
+    pub fn compliance_summary(&self) -> ::std::option::Option<&super::types::AggregateConformancePackComplianceCount> {
         self.compliance_summary.as_ref()
     }
     /// <p>Groups the result based on Amazon Web Services account ID or Amazon Web Services Region.</p>
@@ -21,8 +21,8 @@
 }
 impl AggregateConformancePackComplianceSummary {
     /// Creates a new builder-style object to manufacture [`AggregateConformancePackComplianceSummary`](crate::types::AggregateConformancePackComplianceSummary).
-    pub fn builder() -> crate::types::builders::AggregateConformancePackComplianceSummaryBuilder {
-        crate::types::builders::AggregateConformancePackComplianceSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::AggregateConformancePackComplianceSummaryBuilder {
+        super::types::builders::AggregateConformancePackComplianceSummaryBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AggregateConformancePackComplianceSummaryBuilder {
-    pub(crate) compliance_summary: ::std::option::Option<crate::types::AggregateConformancePackComplianceCount>,
+    pub(crate) compliance_summary: ::std::option::Option<super::types::AggregateConformancePackComplianceCount>,
     pub(crate) group_name: ::std::option::Option<::std::string::String>,
 }
 impl AggregateConformancePackComplianceSummaryBuilder {
     /// <p>Returns an <code>AggregateConformancePackComplianceCount</code> object.</p>
-    pub fn compliance_summary(mut self, input: crate::types::AggregateConformancePackComplianceCount) -> Self {
+    pub fn compliance_summary(mut self, input: super::types::AggregateConformancePackComplianceCount) -> Self {
         self.compliance_summary = ::std::option::Option::Some(input);
         self
     }
     /// <p>Returns an <code>AggregateConformancePackComplianceCount</code> object.</p>
-    pub fn set_compliance_summary(mut self, input: ::std::option::Option<crate::types::AggregateConformancePackComplianceCount>) -> Self {
+    pub fn set_compliance_summary(mut self, input: ::std::option::Option<super::types::AggregateConformancePackComplianceCount>) -> Self {
         self.compliance_summary = input;
         self
     }
     /// <p>Returns an <code>AggregateConformancePackComplianceCount</code> object.</p>
-    pub fn get_compliance_summary(&self) -> &::std::option::Option<crate::types::AggregateConformancePackComplianceCount> {
+    pub fn get_compliance_summary(&self) -> &::std::option::Option<super::types::AggregateConformancePackComplianceCount> {
         &self.compliance_summary
     }
     /// <p>Groups the result based on Amazon Web Services account ID or Amazon Web Services Region.</p>
@@ -63,8 +63,8 @@
         &self.group_name
     }
     /// Consumes the builder and constructs a [`AggregateConformancePackComplianceSummary`](crate::types::AggregateConformancePackComplianceSummary).
-    pub fn build(self) -> crate::types::AggregateConformancePackComplianceSummary {
-        crate::types::AggregateConformancePackComplianceSummary {
+    pub fn build(self) -> super::types::AggregateConformancePackComplianceSummary {
+        super::types::AggregateConformancePackComplianceSummary {
             compliance_summary: self.compliance_summary,
             group_name: self.group_name,
         }
