--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The source account ID or an organization.</p>
     pub source_id: ::std::option::Option<::std::string::String>,
     /// <p>The source account or an organization.</p>
-    pub source_type: ::std::option::Option<crate::types::AggregatedSourceType>,
+    pub source_type: ::std::option::Option<super::super::types::AggregatedSourceType>,
     /// <p>The region authorized to collect aggregated data.</p>
     pub aws_region: ::std::option::Option<::std::string::String>,
     /// <p>Filters the last updated status type.</p>
@@ -19,7 +19,7 @@
     /// <li>
     /// <p>Valid value OUTDATED indicates the data is not the most recent.</p></li>
     /// </ul>
-    pub last_update_status: ::std::option::Option<crate::types::AggregatedSourceStatusType>,
+    pub last_update_status: ::std::option::Option<super::super::types::AggregatedSourceStatusType>,
     /// <p>The time of the last update.</p>
     pub last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The error code that Config returned when the source account aggregation last failed.</p>
@@ -33,7 +33,7 @@
         self.source_id.as_deref()
     }
     /// <p>The source account or an organization.</p>
-    pub fn source_type(&self) -> ::std::option::Option<&crate::types::AggregatedSourceType> {
+    pub fn source_type(&self) -> ::std::option::Option<&super::super::types::AggregatedSourceType> {
         self.source_type.as_ref()
     }
     /// <p>The region authorized to collect aggregated data.</p>
@@ -49,7 +49,7 @@
     /// <li>
     /// <p>Valid value OUTDATED indicates the data is not the most recent.</p></li>
     /// </ul>
-    pub fn last_update_status(&self) -> ::std::option::Option<&crate::types::AggregatedSourceStatusType> {
+    pub fn last_update_status(&self) -> ::std::option::Option<&super::super::types::AggregatedSourceStatusType> {
         self.last_update_status.as_ref()
     }
     /// <p>The time of the last update.</p>
@@ -67,8 +67,8 @@
 }
 impl AggregatedSourceStatus {
     /// Creates a new builder-style object to manufacture [`AggregatedSourceStatus`](crate::types::AggregatedSourceStatus).
-    pub fn builder() -> crate::types::builders::AggregatedSourceStatusBuilder {
-        crate::types::builders::AggregatedSourceStatusBuilder::default()
+    pub fn builder() -> super::super::types::builders::AggregatedSourceStatusBuilder {
+        super::super::types::builders::AggregatedSourceStatusBuilder::default()
     }
 }

@@ -77,9 +77,9 @@
 #[non_exhaustive]
 pub struct AggregatedSourceStatusBuilder {
     pub(crate) source_id: ::std::option::Option<::std::string::String>,
-    pub(crate) source_type: ::std::option::Option<crate::types::AggregatedSourceType>,
+    pub(crate) source_type: ::std::option::Option<super::super::types::AggregatedSourceType>,
     pub(crate) aws_region: ::std::option::Option<::std::string::String>,
-    pub(crate) last_update_status: ::std::option::Option<crate::types::AggregatedSourceStatusType>,
+    pub(crate) last_update_status: ::std::option::Option<super::super::types::AggregatedSourceStatusType>,
     pub(crate) last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) last_error_code: ::std::option::Option<::std::string::String>,
     pub(crate) last_error_message: ::std::option::Option<::std::string::String>,
@@ -100,17 +100,17 @@
         &self.source_id
     }
     /// <p>The source account or an organization.</p>
-    pub fn source_type(mut self, input: crate::types::AggregatedSourceType) -> Self {
+    pub fn source_type(mut self, input: super::super::types::AggregatedSourceType) -> Self {
         self.source_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The source account or an organization.</p>
-    pub fn set_source_type(mut self, input: ::std::option::Option<crate::types::AggregatedSourceType>) -> Self {
+    pub fn set_source_type(mut self, input: ::std::option::Option<super::super::types::AggregatedSourceType>) -> Self {
         self.source_type = input;
         self
     }
     /// <p>The source account or an organization.</p>
-    pub fn get_source_type(&self) -> &::std::option::Option<crate::types::AggregatedSourceType> {
+    pub fn get_source_type(&self) -> &::std::option::Option<super::super::types::AggregatedSourceType> {
         &self.source_type
     }
     /// <p>The region authorized to collect aggregated data.</p>
@@ -136,7 +136,7 @@
     /// <li>
     /// <p>Valid value OUTDATED indicates the data is not the most recent.</p></li>
     /// </ul>
-    pub fn last_update_status(mut self, input: crate::types::AggregatedSourceStatusType) -> Self {
+    pub fn last_update_status(mut self, input: super::super::types::AggregatedSourceStatusType) -> Self {
         self.last_update_status = ::std::option::Option::Some(input);
         self
     }
@@ -149,7 +149,7 @@
     /// <li>
     /// <p>Valid value OUTDATED indicates the data is not the most recent.</p></li>
     /// </ul>
-    pub fn set_last_update_status(mut self, input: ::std::option::Option<crate::types::AggregatedSourceStatusType>) -> Self {
+    pub fn set_last_update_status(mut self, input: ::std::option::Option<super::super::types::AggregatedSourceStatusType>) -> Self {
         self.last_update_status = input;
         self
     }
@@ -162,7 +162,7 @@
     /// <li>
     /// <p>Valid value OUTDATED indicates the data is not the most recent.</p></li>
     /// </ul>
-    pub fn get_last_update_status(&self) -> &::std::option::Option<crate::types::AggregatedSourceStatusType> {
+    pub fn get_last_update_status(&self) -> &::std::option::Option<super::super::types::AggregatedSourceStatusType> {
         &self.last_update_status
     }
     /// <p>The time of the last update.</p>
@@ -208,8 +208,8 @@
         &self.last_error_message
     }
     /// Consumes the builder and constructs a [`AggregatedSourceStatus`](crate::types::AggregatedSourceStatus).
-    pub fn build(self) -> crate::types::AggregatedSourceStatus {
-        crate::types::AggregatedSourceStatus {
+    pub fn build(self) -> super::super::types::AggregatedSourceStatus {
+        super::super::types::AggregatedSourceStatus {
             source_id: self.source_id,
             source_type: self.source_type,
             aws_region: self.aws_region,
