--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The name of the conformance pack.</p>
     pub conformance_pack_name: ::std::option::Option<::std::string::String>,
     /// <p>The compliance status of the conformance pack.</p>
-    pub compliance: ::std::option::Option<crate::types::AggregateConformancePackCompliance>,
+    pub compliance: ::std::option::Option<super::types::AggregateConformancePackCompliance>,
     /// <p>The 12-digit Amazon Web Services account ID of the source account.</p>
     pub account_id: ::std::option::Option<::std::string::String>,
     /// <p>The source Amazon Web Services Region from where the data is aggregated.</p>
@@ -20,7 +20,7 @@
         self.conformance_pack_name.as_deref()
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn compliance(&self) -> ::std::option::Option<&crate::types::AggregateConformancePackCompliance> {
+    pub fn compliance(&self) -> ::std::option::Option<&super::types::AggregateConformancePackCompliance> {
         self.compliance.as_ref()
     }
     /// <p>The 12-digit Amazon Web Services account ID of the source account.</p>
@@ -34,8 +34,8 @@
 }
 impl AggregateComplianceByConformancePack {
     /// Creates a new builder-style object to manufacture [`AggregateComplianceByConformancePack`](crate::types::AggregateComplianceByConformancePack).
-    pub fn builder() -> crate::types::builders::AggregateComplianceByConformancePackBuilder {
-        crate::types::builders::AggregateComplianceByConformancePackBuilder::default()
+    pub fn builder() -> super::types::builders::AggregateComplianceByConformancePackBuilder {
+        super::types::builders::AggregateComplianceByConformancePackBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
 #[non_exhaustive]
 pub struct AggregateComplianceByConformancePackBuilder {
     pub(crate) conformance_pack_name: ::std::option::Option<::std::string::String>,
-    pub(crate) compliance: ::std::option::Option<crate::types::AggregateConformancePackCompliance>,
+    pub(crate) compliance: ::std::option::Option<super::types::AggregateConformancePackCompliance>,
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
     pub(crate) aws_region: ::std::option::Option<::std::string::String>,
 }
@@ -64,17 +64,17 @@
         &self.conformance_pack_name
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn compliance(mut self, input: crate::types::AggregateConformancePackCompliance) -> Self {
+    pub fn compliance(mut self, input: super::types::AggregateConformancePackCompliance) -> Self {
         self.compliance = ::std::option::Option::Some(input);
         self
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn set_compliance(mut self, input: ::std::option::Option<crate::types::AggregateConformancePackCompliance>) -> Self {
+    pub fn set_compliance(mut self, input: ::std::option::Option<super::types::AggregateConformancePackCompliance>) -> Self {
         self.compliance = input;
         self
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn get_compliance(&self) -> &::std::option::Option<crate::types::AggregateConformancePackCompliance> {
+    pub fn get_compliance(&self) -> &::std::option::Option<super::types::AggregateConformancePackCompliance> {
         &self.compliance
     }
     /// <p>The 12-digit Amazon Web Services account ID of the source account.</p>
@@ -106,8 +106,8 @@
         &self.aws_region
     }
     /// Consumes the builder and constructs a [`AggregateComplianceByConformancePack`](crate::types::AggregateComplianceByConformancePack).
-    pub fn build(self) -> crate::types::AggregateComplianceByConformancePack {
-        crate::types::AggregateComplianceByConformancePack {
+    pub fn build(self) -> super::types::AggregateComplianceByConformancePack {
+        super::types::AggregateComplianceByConformancePack {
             conformance_pack_name: self.conformance_pack_name,
             compliance: self.compliance,
             account_id: self.account_id,
