--- original
+++ modified
@@ -6,7 +6,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AggregateConformancePackCompliance {
     /// <p>The compliance status of the conformance pack.</p>
-    pub compliance_type: ::std::option::Option<crate::types::ConformancePackComplianceType>,
+    pub compliance_type: ::std::option::Option<super::super::types::ConformancePackComplianceType>,
     /// <p>The number of compliant Config Rules.</p>
     pub compliant_rule_count: i32,
     /// <p>The number of noncompliant Config Rules.</p>
@@ -16,7 +16,7 @@
 }
 impl AggregateConformancePackCompliance {
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn compliance_type(&self) -> ::std::option::Option<&crate::types::ConformancePackComplianceType> {
+    pub fn compliance_type(&self) -> ::std::option::Option<&super::super::types::ConformancePackComplianceType> {
         self.compliance_type.as_ref()
     }
     /// <p>The number of compliant Config Rules.</p>
@@ -34,8 +34,8 @@
 }
 impl AggregateConformancePackCompliance {
     /// Creates a new builder-style object to manufacture [`AggregateConformancePackCompliance`](crate::types::AggregateConformancePackCompliance).
-    pub fn builder() -> crate::types::builders::AggregateConformancePackComplianceBuilder {
-        crate::types::builders::AggregateConformancePackComplianceBuilder::default()
+    pub fn builder() -> super::super::types::builders::AggregateConformancePackComplianceBuilder {
+        super::super::types::builders::AggregateConformancePackComplianceBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AggregateConformancePackComplianceBuilder {
-    pub(crate) compliance_type: ::std::option::Option<crate::types::ConformancePackComplianceType>,
+    pub(crate) compliance_type: ::std::option::Option<super::super::types::ConformancePackComplianceType>,
     pub(crate) compliant_rule_count: ::std::option::Option<i32>,
     pub(crate) non_compliant_rule_count: ::std::option::Option<i32>,
     pub(crate) total_rule_count: ::std::option::Option<i32>,
@@ -50,17 +50,17 @@
 }
 impl AggregateConformancePackComplianceBuilder {
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn compliance_type(mut self, input: crate::types::ConformancePackComplianceType) -> Self {
+    pub fn compliance_type(mut self, input: super::super::types::ConformancePackComplianceType) -> Self {
         self.compliance_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn set_compliance_type(mut self, input: ::std::option::Option<crate::types::ConformancePackComplianceType>) -> Self {
+    pub fn set_compliance_type(mut self, input: ::std::option::Option<super::super::types::ConformancePackComplianceType>) -> Self {
         self.compliance_type = input;
         self
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn get_compliance_type(&self) -> &::std::option::Option<crate::types::ConformancePackComplianceType> {
+    pub fn get_compliance_type(&self) -> &::std::option::Option<super::super::types::ConformancePackComplianceType> {
         &self.compliance_type
     }
     /// <p>The number of compliant Config Rules.</p>
@@ -106,8 +106,8 @@
         &self.total_rule_count
     }
     /// Consumes the builder and constructs a [`AggregateConformancePackCompliance`](crate::types::AggregateConformancePackCompliance).
-    pub fn build(self) -> crate::types::AggregateConformancePackCompliance {
-        crate::types::AggregateConformancePackCompliance {
+    pub fn build(self) -> super::super::types::AggregateConformancePackCompliance {
+        super::super::types::AggregateConformancePackCompliance {
             compliance_type: self.compliance_type,
             compliant_rule_count: self.compliant_rule_count.unwrap_or_default(),
             non_compliant_rule_count: self.non_compliant_rule_count.unwrap_or_default(),
