--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl AggregateConformancePackComplianceSummaryFilters {
     /// Creates a new builder-style object to manufacture [`AggregateConformancePackComplianceSummaryFilters`](crate::types::AggregateConformancePackComplianceSummaryFilters).
-    pub fn builder() -> crate::types::builders::AggregateConformancePackComplianceSummaryFiltersBuilder {
-        crate::types::builders::AggregateConformancePackComplianceSummaryFiltersBuilder::default()
+    pub fn builder() -> super::super::types::builders::AggregateConformancePackComplianceSummaryFiltersBuilder {
+        super::super::types::builders::AggregateConformancePackComplianceSummaryFiltersBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.aws_region
     }
     /// Consumes the builder and constructs a [`AggregateConformancePackComplianceSummaryFilters`](crate::types::AggregateConformancePackComplianceSummaryFilters).
-    pub fn build(self) -> crate::types::AggregateConformancePackComplianceSummaryFilters {
-        crate::types::AggregateConformancePackComplianceSummaryFilters {
+    pub fn build(self) -> super::super::types::AggregateConformancePackComplianceSummaryFilters {
+        super::super::types::AggregateConformancePackComplianceSummaryFilters {
             account_id: self.account_id,
             aws_region: self.aws_region,
         }
