--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl AggregateConformancePackComplianceCount {
     /// Creates a new builder-style object to manufacture [`AggregateConformancePackComplianceCount`](crate::types::AggregateConformancePackComplianceCount).
-    pub fn builder() -> crate::types::builders::AggregateConformancePackComplianceCountBuilder {
-        crate::types::builders::AggregateConformancePackComplianceCountBuilder::default()
+    pub fn builder() -> super::types::builders::AggregateConformancePackComplianceCountBuilder {
+        super::types::builders::AggregateConformancePackComplianceCountBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.non_compliant_conformance_pack_count
     }
     /// Consumes the builder and constructs a [`AggregateConformancePackComplianceCount`](crate::types::AggregateConformancePackComplianceCount).
-    pub fn build(self) -> crate::types::AggregateConformancePackComplianceCount {
-        crate::types::AggregateConformancePackComplianceCount {
+    pub fn build(self) -> super::types::AggregateConformancePackComplianceCount {
+        super::types::AggregateConformancePackComplianceCount {
             compliant_conformance_pack_count: self.compliant_conformance_pack_count.unwrap_or_default(),
             non_compliant_conformance_pack_count: self.non_compliant_conformance_pack_count.unwrap_or_default(),
         }
