--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the conformance pack.</p>
     pub conformance_pack_name: ::std::option::Option<::std::string::String>,
     /// <p>The compliance status of the conformance pack.</p>
-    pub compliance_type: ::std::option::Option<crate::types::ConformancePackComplianceType>,
+    pub compliance_type: ::std::option::Option<super::types::ConformancePackComplianceType>,
     /// <p>The 12-digit Amazon Web Services account ID of the source account.</p>
     pub account_id: ::std::option::Option<::std::string::String>,
     /// <p>The source Amazon Web Services Region from where the data is aggregated.</p>
@@ -19,7 +19,7 @@
         self.conformance_pack_name.as_deref()
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn compliance_type(&self) -> ::std::option::Option<&crate::types::ConformancePackComplianceType> {
+    pub fn compliance_type(&self) -> ::std::option::Option<&super::types::ConformancePackComplianceType> {
         self.compliance_type.as_ref()
     }
     /// <p>The 12-digit Amazon Web Services account ID of the source account.</p>
@@ -33,8 +33,8 @@
 }
 impl AggregateConformancePackComplianceFilters {
     /// Creates a new builder-style object to manufacture [`AggregateConformancePackComplianceFilters`](crate::types::AggregateConformancePackComplianceFilters).
-    pub fn builder() -> crate::types::builders::AggregateConformancePackComplianceFiltersBuilder {
-        crate::types::builders::AggregateConformancePackComplianceFiltersBuilder::default()
+    pub fn builder() -> super::types::builders::AggregateConformancePackComplianceFiltersBuilder {
+        super::types::builders::AggregateConformancePackComplianceFiltersBuilder::default()
     }
 }

@@ -43,7 +43,7 @@
 #[non_exhaustive]
 pub struct AggregateConformancePackComplianceFiltersBuilder {
     pub(crate) conformance_pack_name: ::std::option::Option<::std::string::String>,
-    pub(crate) compliance_type: ::std::option::Option<crate::types::ConformancePackComplianceType>,
+    pub(crate) compliance_type: ::std::option::Option<super::types::ConformancePackComplianceType>,
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
     pub(crate) aws_region: ::std::option::Option<::std::string::String>,
 }
@@ -63,17 +63,17 @@
         &self.conformance_pack_name
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn compliance_type(mut self, input: crate::types::ConformancePackComplianceType) -> Self {
+    pub fn compliance_type(mut self, input: super::types::ConformancePackComplianceType) -> Self {
         self.compliance_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn set_compliance_type(mut self, input: ::std::option::Option<crate::types::ConformancePackComplianceType>) -> Self {
+    pub fn set_compliance_type(mut self, input: ::std::option::Option<super::types::ConformancePackComplianceType>) -> Self {
         self.compliance_type = input;
         self
     }
     /// <p>The compliance status of the conformance pack.</p>
-    pub fn get_compliance_type(&self) -> &::std::option::Option<crate::types::ConformancePackComplianceType> {
+    pub fn get_compliance_type(&self) -> &::std::option::Option<super::types::ConformancePackComplianceType> {
         &self.compliance_type
     }
     /// <p>The 12-digit Amazon Web Services account ID of the source account.</p>
@@ -105,8 +105,8 @@
         &self.aws_region
     }
     /// Consumes the builder and constructs a [`AggregateConformancePackComplianceFilters`](crate::types::AggregateConformancePackComplianceFilters).
-    pub fn build(self) -> crate::types::AggregateConformancePackComplianceFilters {
-        crate::types::AggregateConformancePackComplianceFilters {
+    pub fn build(self) -> super::types::AggregateConformancePackComplianceFilters {
+        super::types::AggregateConformancePackComplianceFilters {
             conformance_pack_name: self.conformance_pack_name,
             compliance_type: self.compliance_type,
             account_id: self.account_id,
