--- original
+++ modified
@@ -23,13 +23,13 @@
     /// <li>
     /// <p>ResourceDeletedNotRecorded – The resource was deleted, but its configuration was not recorded since the recorder doesn't record resources of this type.</p></li>
     /// </ul>
-    pub configuration_item_status: ::std::option::Option<crate::types::ConfigurationItemStatus>,
+    pub configuration_item_status: ::std::option::Option<super::types::ConfigurationItemStatus>,
     /// <p>An identifier that indicates the ordering of the configuration items of a resource.</p>
     pub configuration_state_id: ::std::option::Option<::std::string::String>,
     /// <p>The Amazon Resource Name (ARN) of the resource.</p>
     pub arn: ::std::option::Option<::std::string::String>,
     /// <p>The type of Amazon Web Services resource.</p>
-    pub resource_type: ::std::option::Option<crate::types::ResourceType>,
+    pub resource_type: ::std::option::Option<super::types::ResourceType>,
     /// <p>The ID of the resource (for example., sg-xxxxxx).</p>
     pub resource_id: ::std::option::Option<::std::string::String>,
     /// <p>The custom name of the resource, if available.</p>
@@ -48,7 +48,7 @@
     /// <p>The recording frequency that Config uses to record configuration changes for the resource.</p><note>
     /// <p>This field only appears in the API response when <code>DAILY</code> recording is enabled for a resource type. If this field is not present, <code>CONTINUOUS</code> recording is enabled for that resource type. For more information on daily recording and continuous recording, see <a href="https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html#select-resources-recording-frequency">Recording Frequency</a> in the <i>Config Developer Guide</i>.</p>
     /// </note>
-    pub recording_frequency: ::std::option::Option<crate::types::RecordingFrequency>,
+    pub recording_frequency: ::std::option::Option<super::types::RecordingFrequency>,
     /// <p>The time when configuration changes for the resource were delivered.</p><note>
     /// <p>This field is optional and is not guaranteed to be present in a configuration item (CI). If you are using daily recording, this field will be populated. However, if you are using continuous recording, this field will be omitted since the delivery time is instantaneous as the CI is available right away. For more information on daily recording and continuous recording, see <a href="https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html#select-resources-recording-frequency">Recording Frequency</a> in the <i>Config Developer Guide</i>.</p>
     /// </note>
@@ -80,7 +80,7 @@
     /// <li>
     /// <p>ResourceDeletedNotRecorded – The resource was deleted, but its configuration was not recorded since the recorder doesn't record resources of this type.</p></li>
     /// </ul>
-    pub fn configuration_item_status(&self) -> ::std::option::Option<&crate::types::ConfigurationItemStatus> {
+    pub fn configuration_item_status(&self) -> ::std::option::Option<&super::types::ConfigurationItemStatus> {
         self.configuration_item_status.as_ref()
     }
     /// <p>An identifier that indicates the ordering of the configuration items of a resource.</p>
@@ -92,7 +92,7 @@
         self.arn.as_deref()
     }
     /// <p>The type of Amazon Web Services resource.</p>
-    pub fn resource_type(&self) -> ::std::option::Option<&crate::types::ResourceType> {
+    pub fn resource_type(&self) -> ::std::option::Option<&super::types::ResourceType> {
         self.resource_type.as_ref()
     }
     /// <p>The ID of the resource (for example., sg-xxxxxx).</p>
@@ -127,7 +127,7 @@
     /// <p>The recording frequency that Config uses to record configuration changes for the resource.</p><note>
     /// <p>This field only appears in the API response when <code>DAILY</code> recording is enabled for a resource type. If this field is not present, <code>CONTINUOUS</code> recording is enabled for that resource type. For more information on daily recording and continuous recording, see <a href="https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html#select-resources-recording-frequency">Recording Frequency</a> in the <i>Config Developer Guide</i>.</p>
     /// </note>
-    pub fn recording_frequency(&self) -> ::std::option::Option<&crate::types::RecordingFrequency> {
+    pub fn recording_frequency(&self) -> ::std::option::Option<&super::types::RecordingFrequency> {
         self.recording_frequency.as_ref()
     }
     /// <p>The time when configuration changes for the resource were delivered.</p><note>
@@ -139,8 +139,8 @@
 }
 impl BaseConfigurationItem {
     /// Creates a new builder-style object to manufacture [`BaseConfigurationItem`](crate::types::BaseConfigurationItem).
-    pub fn builder() -> crate::types::builders::BaseConfigurationItemBuilder {
-        crate::types::builders::BaseConfigurationItemBuilder::default()
+    pub fn builder() -> super::types::builders::BaseConfigurationItemBuilder {
+        super::types::builders::BaseConfigurationItemBuilder::default()
     }
 }

@@ -151,10 +151,10 @@
     pub(crate) version: ::std::option::Option<::std::string::String>,
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
     pub(crate) configuration_item_capture_time: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) configuration_item_status: ::std::option::Option<crate::types::ConfigurationItemStatus>,
+    pub(crate) configuration_item_status: ::std::option::Option<super::types::ConfigurationItemStatus>,
     pub(crate) configuration_state_id: ::std::option::Option<::std::string::String>,
     pub(crate) arn: ::std::option::Option<::std::string::String>,
-    pub(crate) resource_type: ::std::option::Option<crate::types::ResourceType>,
+    pub(crate) resource_type: ::std::option::Option<super::types::ResourceType>,
     pub(crate) resource_id: ::std::option::Option<::std::string::String>,
     pub(crate) resource_name: ::std::option::Option<::std::string::String>,
     pub(crate) aws_region: ::std::option::Option<::std::string::String>,
@@ -162,7 +162,7 @@
     pub(crate) resource_creation_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) configuration: ::std::option::Option<::std::string::String>,
     pub(crate) supplementary_configuration: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
-    pub(crate) recording_frequency: ::std::option::Option<crate::types::RecordingFrequency>,
+    pub(crate) recording_frequency: ::std::option::Option<super::types::RecordingFrequency>,
     pub(crate) configuration_item_delivery_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl BaseConfigurationItemBuilder {
@@ -221,7 +221,7 @@
     /// <li>
     /// <p>ResourceDeletedNotRecorded – The resource was deleted, but its configuration was not recorded since the recorder doesn't record resources of this type.</p></li>
     /// </ul>
-    pub fn configuration_item_status(mut self, input: crate::types::ConfigurationItemStatus) -> Self {
+    pub fn configuration_item_status(mut self, input: super::types::ConfigurationItemStatus) -> Self {
         self.configuration_item_status = ::std::option::Option::Some(input);
         self
     }
@@ -238,7 +238,7 @@
     /// <li>
     /// <p>ResourceDeletedNotRecorded – The resource was deleted, but its configuration was not recorded since the recorder doesn't record resources of this type.</p></li>
     /// </ul>
-    pub fn set_configuration_item_status(mut self, input: ::std::option::Option<crate::types::ConfigurationItemStatus>) -> Self {
+    pub fn set_configuration_item_status(mut self, input: ::std::option::Option<super::types::ConfigurationItemStatus>) -> Self {
         self.configuration_item_status = input;
         self
     }
@@ -255,7 +255,7 @@
     /// <li>
     /// <p>ResourceDeletedNotRecorded – The resource was deleted, but its configuration was not recorded since the recorder doesn't record resources of this type.</p></li>
     /// </ul>
-    pub fn get_configuration_item_status(&self) -> &::std::option::Option<crate::types::ConfigurationItemStatus> {
+    pub fn get_configuration_item_status(&self) -> &::std::option::Option<super::types::ConfigurationItemStatus> {
         &self.configuration_item_status
     }
     /// <p>An identifier that indicates the ordering of the configuration items of a resource.</p>
@@ -287,17 +287,17 @@
         &self.arn
     }
     /// <p>The type of Amazon Web Services resource.</p>
-    pub fn resource_type(mut self, input: crate::types::ResourceType) -> Self {
+    pub fn resource_type(mut self, input: super::types::ResourceType) -> Self {
         self.resource_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of Amazon Web Services resource.</p>
-    pub fn set_resource_type(mut self, input: ::std::option::Option<crate::types::ResourceType>) -> Self {
+    pub fn set_resource_type(mut self, input: ::std::option::Option<super::types::ResourceType>) -> Self {
         self.resource_type = input;
         self
     }
     /// <p>The type of Amazon Web Services resource.</p>
-    pub fn get_resource_type(&self) -> &::std::option::Option<crate::types::ResourceType> {
+    pub fn get_resource_type(&self) -> &::std::option::Option<super::types::ResourceType> {
         &self.resource_type
     }
     /// <p>The ID of the resource (for example., sg-xxxxxx).</p>
@@ -419,7 +419,7 @@
     /// <p>The recording frequency that Config uses to record configuration changes for the resource.</p><note>
     /// <p>This field only appears in the API response when <code>DAILY</code> recording is enabled for a resource type. If this field is not present, <code>CONTINUOUS</code> recording is enabled for that resource type. For more information on daily recording and continuous recording, see <a href="https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html#select-resources-recording-frequency">Recording Frequency</a> in the <i>Config Developer Guide</i>.</p>
     /// </note>
-    pub fn recording_frequency(mut self, input: crate::types::RecordingFrequency) -> Self {
+    pub fn recording_frequency(mut self, input: super::types::RecordingFrequency) -> Self {
         self.recording_frequency = ::std::option::Option::Some(input);
         self
     }
@@ -426,7 +426,7 @@
     /// <p>The recording frequency that Config uses to record configuration changes for the resource.</p><note>
     /// <p>This field only appears in the API response when <code>DAILY</code> recording is enabled for a resource type. If this field is not present, <code>CONTINUOUS</code> recording is enabled for that resource type. For more information on daily recording and continuous recording, see <a href="https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html#select-resources-recording-frequency">Recording Frequency</a> in the <i>Config Developer Guide</i>.</p>
     /// </note>
-    pub fn set_recording_frequency(mut self, input: ::std::option::Option<crate::types::RecordingFrequency>) -> Self {
+    pub fn set_recording_frequency(mut self, input: ::std::option::Option<super::types::RecordingFrequency>) -> Self {
         self.recording_frequency = input;
         self
     }
@@ -433,7 +433,7 @@
     /// <p>The recording frequency that Config uses to record configuration changes for the resource.</p><note>
     /// <p>This field only appears in the API response when <code>DAILY</code> recording is enabled for a resource type. If this field is not present, <code>CONTINUOUS</code> recording is enabled for that resource type. For more information on daily recording and continuous recording, see <a href="https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html#select-resources-recording-frequency">Recording Frequency</a> in the <i>Config Developer Guide</i>.</p>
     /// </note>
-    pub fn get_recording_frequency(&self) -> &::std::option::Option<crate::types::RecordingFrequency> {
+    pub fn get_recording_frequency(&self) -> &::std::option::Option<super::types::RecordingFrequency> {
         &self.recording_frequency
     }
     /// <p>The time when configuration changes for the resource were delivered.</p><note>
@@ -457,8 +457,8 @@
         &self.configuration_item_delivery_time
     }
     /// Consumes the builder and constructs a [`BaseConfigurationItem`](crate::types::BaseConfigurationItem).
-    pub fn build(self) -> crate::types::BaseConfigurationItem {
-        crate::types::BaseConfigurationItem {
+    pub fn build(self) -> super::types::BaseConfigurationItem {
+        super::types::BaseConfigurationItem {
             version: self.version,
             account_id: self.account_id,
             configuration_item_capture_time: self.configuration_item_capture_time,
