--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>The ID of the Amazon Web Services resource that was evaluated.</p>
     pub resource_id: ::std::option::Option<::std::string::String>,
     /// <p>Indicates whether the Amazon Web Services resource complies with all of the Config rules that evaluated it.</p>
-    pub compliance: ::std::option::Option<crate::types::Compliance>,
+    pub compliance: ::std::option::Option<super::types::Compliance>,
 }
 impl ComplianceByResource {
     /// <p>The type of the Amazon Web Services resource that was evaluated.</p>
@@ -21,14 +21,14 @@
         self.resource_id.as_deref()
     }
     /// <p>Indicates whether the Amazon Web Services resource complies with all of the Config rules that evaluated it.</p>
-    pub fn compliance(&self) -> ::std::option::Option<&crate::types::Compliance> {
+    pub fn compliance(&self) -> ::std::option::Option<&super::types::Compliance> {
         self.compliance.as_ref()
     }
 }
 impl ComplianceByResource {
     /// Creates a new builder-style object to manufacture [`ComplianceByResource`](crate::types::ComplianceByResource).
-    pub fn builder() -> crate::types::builders::ComplianceByResourceBuilder {
-        crate::types::builders::ComplianceByResourceBuilder::default()
+    pub fn builder() -> super::types::builders::ComplianceByResourceBuilder {
+        super::types::builders::ComplianceByResourceBuilder::default()
     }
 }

@@ -38,7 +38,7 @@
 pub struct ComplianceByResourceBuilder {
     pub(crate) resource_type: ::std::option::Option<::std::string::String>,
     pub(crate) resource_id: ::std::option::Option<::std::string::String>,
-    pub(crate) compliance: ::std::option::Option<crate::types::Compliance>,
+    pub(crate) compliance: ::std::option::Option<super::types::Compliance>,
 }
 impl ComplianceByResourceBuilder {
     /// <p>The type of the Amazon Web Services resource that was evaluated.</p>
@@ -70,22 +70,22 @@
         &self.resource_id
     }
     /// <p>Indicates whether the Amazon Web Services resource complies with all of the Config rules that evaluated it.</p>
-    pub fn compliance(mut self, input: crate::types::Compliance) -> Self {
+    pub fn compliance(mut self, input: super::types::Compliance) -> Self {
         self.compliance = ::std::option::Option::Some(input);
         self
     }
     /// <p>Indicates whether the Amazon Web Services resource complies with all of the Config rules that evaluated it.</p>
-    pub fn set_compliance(mut self, input: ::std::option::Option<crate::types::Compliance>) -> Self {
+    pub fn set_compliance(mut self, input: ::std::option::Option<super::types::Compliance>) -> Self {
         self.compliance = input;
         self
     }
     /// <p>Indicates whether the Amazon Web Services resource complies with all of the Config rules that evaluated it.</p>
-    pub fn get_compliance(&self) -> &::std::option::Option<crate::types::Compliance> {
+    pub fn get_compliance(&self) -> &::std::option::Option<super::types::Compliance> {
         &self.compliance
     }
     /// Consumes the builder and constructs a [`ComplianceByResource`](crate::types::ComplianceByResource).
-    pub fn build(self) -> crate::types::ComplianceByResource {
-        crate::types::ComplianceByResource {
+    pub fn build(self) -> super::types::ComplianceByResource {
+        super::types::ComplianceByResource {
             resource_type: self.resource_type,
             resource_id: self.resource_id,
             compliance: self.compliance,
