--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ConfigurationRecorderFilter {
     /// <p>The name of the type of filter. Currently, only <code>recordingScope</code> is supported.</p>
-    pub filter_name: ::std::option::Option<crate::types::ConfigurationRecorderFilterName>,
+    pub filter_name: ::std::option::Option<super::types::ConfigurationRecorderFilterName>,
     /// <p>The value of the filter. For <code>recordingScope</code>, valid values include: <code>INTERNAL</code> and <code>PAID</code>.</p>
     /// <p><code>INTERNAL</code> indicates that the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the configuration recorder are recorded for free.</p>
     /// <p><code>PAID</code> indicates that the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the configuration recorder impact the costs to your bill.</p>
@@ -13,7 +13,7 @@
 }
 impl ConfigurationRecorderFilter {
     /// <p>The name of the type of filter. Currently, only <code>recordingScope</code> is supported.</p>
-    pub fn filter_name(&self) -> ::std::option::Option<&crate::types::ConfigurationRecorderFilterName> {
+    pub fn filter_name(&self) -> ::std::option::Option<&super::types::ConfigurationRecorderFilterName> {
         self.filter_name.as_ref()
     }
     /// <p>The value of the filter. For <code>recordingScope</code>, valid values include: <code>INTERNAL</code> and <code>PAID</code>.</p>
@@ -27,8 +27,8 @@
 }
 impl ConfigurationRecorderFilter {
     /// Creates a new builder-style object to manufacture [`ConfigurationRecorderFilter`](crate::types::ConfigurationRecorderFilter).
-    pub fn builder() -> crate::types::builders::ConfigurationRecorderFilterBuilder {
-        crate::types::builders::ConfigurationRecorderFilterBuilder::default()
+    pub fn builder() -> super::types::builders::ConfigurationRecorderFilterBuilder {
+        super::types::builders::ConfigurationRecorderFilterBuilder::default()
     }
 }

@@ -36,22 +36,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ConfigurationRecorderFilterBuilder {
-    pub(crate) filter_name: ::std::option::Option<crate::types::ConfigurationRecorderFilterName>,
+    pub(crate) filter_name: ::std::option::Option<super::types::ConfigurationRecorderFilterName>,
     pub(crate) filter_value: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
 }
 impl ConfigurationRecorderFilterBuilder {
     /// <p>The name of the type of filter. Currently, only <code>recordingScope</code> is supported.</p>
-    pub fn filter_name(mut self, input: crate::types::ConfigurationRecorderFilterName) -> Self {
+    pub fn filter_name(mut self, input: super::types::ConfigurationRecorderFilterName) -> Self {
         self.filter_name = ::std::option::Option::Some(input);
         self
     }
     /// <p>The name of the type of filter. Currently, only <code>recordingScope</code> is supported.</p>
-    pub fn set_filter_name(mut self, input: ::std::option::Option<crate::types::ConfigurationRecorderFilterName>) -> Self {
+    pub fn set_filter_name(mut self, input: ::std::option::Option<super::types::ConfigurationRecorderFilterName>) -> Self {
         self.filter_name = input;
         self
     }
     /// <p>The name of the type of filter. Currently, only <code>recordingScope</code> is supported.</p>
-    pub fn get_filter_name(&self) -> &::std::option::Option<crate::types::ConfigurationRecorderFilterName> {
+    pub fn get_filter_name(&self) -> &::std::option::Option<super::types::ConfigurationRecorderFilterName> {
         &self.filter_name
     }
     /// Appends an item to `filter_value`.
@@ -81,8 +81,8 @@
         &self.filter_value
     }
     /// Consumes the builder and constructs a [`ConfigurationRecorderFilter`](crate::types::ConfigurationRecorderFilter).
-    pub fn build(self) -> crate::types::ConfigurationRecorderFilter {
-        crate::types::ConfigurationRecorderFilter {
+    pub fn build(self) -> super::types::ConfigurationRecorderFilter {
+        super::types::ConfigurationRecorderFilter {
             filter_name: self.filter_name,
             filter_value: self.filter_value,
         }
