--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the conformance pack name.</p>
     pub conformance_pack_name: ::std::string::String,
     /// <p>The status of the conformance pack.</p>
-    pub conformance_pack_compliance_status: crate::types::ConformancePackComplianceType,
+    pub conformance_pack_compliance_status: super::types::ConformancePackComplianceType,
 }
 impl ConformancePackComplianceSummary {
     /// <p>The name of the conformance pack name.</p>
@@ -16,14 +16,14 @@
         self.conformance_pack_name.deref()
     }
     /// <p>The status of the conformance pack.</p>
-    pub fn conformance_pack_compliance_status(&self) -> &crate::types::ConformancePackComplianceType {
+    pub fn conformance_pack_compliance_status(&self) -> &super::types::ConformancePackComplianceType {
         &self.conformance_pack_compliance_status
     }
 }
 impl ConformancePackComplianceSummary {
     /// Creates a new builder-style object to manufacture [`ConformancePackComplianceSummary`](crate::types::ConformancePackComplianceSummary).
-    pub fn builder() -> crate::types::builders::ConformancePackComplianceSummaryBuilder {
-        crate::types::builders::ConformancePackComplianceSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::ConformancePackComplianceSummaryBuilder {
+        super::types::builders::ConformancePackComplianceSummaryBuilder::default()
     }
 }

@@ -32,7 +32,7 @@
 #[non_exhaustive]
 pub struct ConformancePackComplianceSummaryBuilder {
     pub(crate) conformance_pack_name: ::std::option::Option<::std::string::String>,
-    pub(crate) conformance_pack_compliance_status: ::std::option::Option<crate::types::ConformancePackComplianceType>,
+    pub(crate) conformance_pack_compliance_status: ::std::option::Option<super::types::ConformancePackComplianceType>,
 }
 impl ConformancePackComplianceSummaryBuilder {
     /// <p>The name of the conformance pack name.</p>
@@ -52,17 +52,17 @@
     }
     /// <p>The status of the conformance pack.</p>
     /// This field is required.
-    pub fn conformance_pack_compliance_status(mut self, input: crate::types::ConformancePackComplianceType) -> Self {
+    pub fn conformance_pack_compliance_status(mut self, input: super::types::ConformancePackComplianceType) -> Self {
         self.conformance_pack_compliance_status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The status of the conformance pack.</p>
-    pub fn set_conformance_pack_compliance_status(mut self, input: ::std::option::Option<crate::types::ConformancePackComplianceType>) -> Self {
+    pub fn set_conformance_pack_compliance_status(mut self, input: ::std::option::Option<super::types::ConformancePackComplianceType>) -> Self {
         self.conformance_pack_compliance_status = input;
         self
     }
     /// <p>The status of the conformance pack.</p>
-    pub fn get_conformance_pack_compliance_status(&self) -> &::std::option::Option<crate::types::ConformancePackComplianceType> {
+    pub fn get_conformance_pack_compliance_status(&self) -> &::std::option::Option<super::types::ConformancePackComplianceType> {
         &self.conformance_pack_compliance_status
     }
     /// Consumes the builder and constructs a [`ConformancePackComplianceSummary`](crate::types::ConformancePackComplianceSummary).
@@ -69,8 +69,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`conformance_pack_name`](crate::types::builders::ConformancePackComplianceSummaryBuilder::conformance_pack_name)
     /// - [`conformance_pack_compliance_status`](crate::types::builders::ConformancePackComplianceSummaryBuilder::conformance_pack_compliance_status)
-    pub fn build(self) -> ::std::result::Result<crate::types::ConformancePackComplianceSummary, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ConformancePackComplianceSummary {
+    pub fn build(self) -> ::std::result::Result<super::types::ConformancePackComplianceSummary, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ConformancePackComplianceSummary {
             conformance_pack_name: self.conformance_pack_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "conformance_pack_name",
