--- original
+++ modified
@@ -9,9 +9,9 @@
     /// <p>The Amazon Resource Name (ARN) of the aggregator.</p>
     pub configuration_aggregator_arn: ::std::option::Option<::std::string::String>,
     /// <p>Provides a list of source accounts and regions to be aggregated.</p>
-    pub account_aggregation_sources: ::std::option::Option<::std::vec::Vec<crate::types::AccountAggregationSource>>,
+    pub account_aggregation_sources: ::std::option::Option<::std::vec::Vec<super::types::AccountAggregationSource>>,
     /// <p>Provides an organization and list of regions to be aggregated.</p>
-    pub organization_aggregation_source: ::std::option::Option<crate::types::OrganizationAggregationSource>,
+    pub organization_aggregation_source: ::std::option::Option<super::types::OrganizationAggregationSource>,
     /// <p>The time stamp when the configuration aggregator was created.</p>
     pub creation_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The time of the last update.</p>
@@ -19,7 +19,7 @@
     /// <p>Amazon Web Services service that created the configuration aggregator.</p>
     pub created_by: ::std::option::Option<::std::string::String>,
     /// <p>An object to filter the data you specify for an aggregator.</p>
-    pub aggregator_filters: ::std::option::Option<crate::types::AggregatorFilters>,
+    pub aggregator_filters: ::std::option::Option<super::types::AggregatorFilters>,
 }
 impl ConfigurationAggregator {
     /// <p>The name of the aggregator.</p>
@@ -33,11 +33,11 @@
     /// <p>Provides a list of source accounts and regions to be aggregated.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.account_aggregation_sources.is_none()`.
-    pub fn account_aggregation_sources(&self) -> &[crate::types::AccountAggregationSource] {
+    pub fn account_aggregation_sources(&self) -> &[super::types::AccountAggregationSource] {
         self.account_aggregation_sources.as_deref().unwrap_or_default()
     }
     /// <p>Provides an organization and list of regions to be aggregated.</p>
-    pub fn organization_aggregation_source(&self) -> ::std::option::Option<&crate::types::OrganizationAggregationSource> {
+    pub fn organization_aggregation_source(&self) -> ::std::option::Option<&super::types::OrganizationAggregationSource> {
         self.organization_aggregation_source.as_ref()
     }
     /// <p>The time stamp when the configuration aggregator was created.</p>
@@ -53,14 +53,14 @@
         self.created_by.as_deref()
     }
     /// <p>An object to filter the data you specify for an aggregator.</p>
-    pub fn aggregator_filters(&self) -> ::std::option::Option<&crate::types::AggregatorFilters> {
+    pub fn aggregator_filters(&self) -> ::std::option::Option<&super::types::AggregatorFilters> {
         self.aggregator_filters.as_ref()
     }
 }
 impl ConfigurationAggregator {
     /// Creates a new builder-style object to manufacture [`ConfigurationAggregator`](crate::types::ConfigurationAggregator).
-    pub fn builder() -> crate::types::builders::ConfigurationAggregatorBuilder {
-        crate::types::builders::ConfigurationAggregatorBuilder::default()
+    pub fn builder() -> super::types::builders::ConfigurationAggregatorBuilder {
+        super::types::builders::ConfigurationAggregatorBuilder::default()
     }
 }

@@ -70,12 +70,12 @@
 pub struct ConfigurationAggregatorBuilder {
     pub(crate) configuration_aggregator_name: ::std::option::Option<::std::string::String>,
     pub(crate) configuration_aggregator_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) account_aggregation_sources: ::std::option::Option<::std::vec::Vec<crate::types::AccountAggregationSource>>,
-    pub(crate) organization_aggregation_source: ::std::option::Option<crate::types::OrganizationAggregationSource>,
+    pub(crate) account_aggregation_sources: ::std::option::Option<::std::vec::Vec<super::types::AccountAggregationSource>>,
+    pub(crate) organization_aggregation_source: ::std::option::Option<super::types::OrganizationAggregationSource>,
     pub(crate) creation_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) last_updated_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) created_by: ::std::option::Option<::std::string::String>,
-    pub(crate) aggregator_filters: ::std::option::Option<crate::types::AggregatorFilters>,
+    pub(crate) aggregator_filters: ::std::option::Option<super::types::AggregatorFilters>,
 }
 impl ConfigurationAggregatorBuilder {
     /// <p>The name of the aggregator.</p>
@@ -111,7 +111,7 @@
     /// To override the contents of this collection use [`set_account_aggregation_sources`](Self::set_account_aggregation_sources).
     ///
     /// <p>Provides a list of source accounts and regions to be aggregated.</p>
-    pub fn account_aggregation_sources(mut self, input: crate::types::AccountAggregationSource) -> Self {
+    pub fn account_aggregation_sources(mut self, input: super::types::AccountAggregationSource) -> Self {
         let mut v = self.account_aggregation_sources.unwrap_or_default();
         v.push(input);
         self.account_aggregation_sources = ::std::option::Option::Some(v);
@@ -118,26 +118,26 @@
         self
     }
     /// <p>Provides a list of source accounts and regions to be aggregated.</p>
-    pub fn set_account_aggregation_sources(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::AccountAggregationSource>>) -> Self {
+    pub fn set_account_aggregation_sources(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::AccountAggregationSource>>) -> Self {
         self.account_aggregation_sources = input;
         self
     }
     /// <p>Provides a list of source accounts and regions to be aggregated.</p>
-    pub fn get_account_aggregation_sources(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::AccountAggregationSource>> {
+    pub fn get_account_aggregation_sources(&self) -> &::std::option::Option<::std::vec::Vec<super::types::AccountAggregationSource>> {
         &self.account_aggregation_sources
     }
     /// <p>Provides an organization and list of regions to be aggregated.</p>
-    pub fn organization_aggregation_source(mut self, input: crate::types::OrganizationAggregationSource) -> Self {
+    pub fn organization_aggregation_source(mut self, input: super::types::OrganizationAggregationSource) -> Self {
         self.organization_aggregation_source = ::std::option::Option::Some(input);
         self
     }
     /// <p>Provides an organization and list of regions to be aggregated.</p>
-    pub fn set_organization_aggregation_source(mut self, input: ::std::option::Option<crate::types::OrganizationAggregationSource>) -> Self {
+    pub fn set_organization_aggregation_source(mut self, input: ::std::option::Option<super::types::OrganizationAggregationSource>) -> Self {
         self.organization_aggregation_source = input;
         self
     }
     /// <p>Provides an organization and list of regions to be aggregated.</p>
-    pub fn get_organization_aggregation_source(&self) -> &::std::option::Option<crate::types::OrganizationAggregationSource> {
+    pub fn get_organization_aggregation_source(&self) -> &::std::option::Option<super::types::OrganizationAggregationSource> {
         &self.organization_aggregation_source
     }
     /// <p>The time stamp when the configuration aggregator was created.</p>
@@ -183,22 +183,22 @@
         &self.created_by
     }
     /// <p>An object to filter the data you specify for an aggregator.</p>
-    pub fn aggregator_filters(mut self, input: crate::types::AggregatorFilters) -> Self {
+    pub fn aggregator_filters(mut self, input: super::types::AggregatorFilters) -> Self {
         self.aggregator_filters = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object to filter the data you specify for an aggregator.</p>
-    pub fn set_aggregator_filters(mut self, input: ::std::option::Option<crate::types::AggregatorFilters>) -> Self {
+    pub fn set_aggregator_filters(mut self, input: ::std::option::Option<super::types::AggregatorFilters>) -> Self {
         self.aggregator_filters = input;
         self
     }
     /// <p>An object to filter the data you specify for an aggregator.</p>
-    pub fn get_aggregator_filters(&self) -> &::std::option::Option<crate::types::AggregatorFilters> {
+    pub fn get_aggregator_filters(&self) -> &::std::option::Option<super::types::AggregatorFilters> {
         &self.aggregator_filters
     }
     /// Consumes the builder and constructs a [`ConfigurationAggregator`](crate::types::ConfigurationAggregator).
-    pub fn build(self) -> crate::types::ConfigurationAggregator {
-        crate::types::ConfigurationAggregator {
+    pub fn build(self) -> super::types::ConfigurationAggregator {
+        super::types::ConfigurationAggregator {
             configuration_aggregator_name: self.configuration_aggregator_name,
             configuration_aggregator_arn: self.configuration_aggregator_arn,
             account_aggregation_sources: self.account_aggregation_sources,
