--- original
+++ modified
@@ -33,7 +33,7 @@
     /// <p>If you are running ephemeral workloads, you may see increased activity from Config as it records configuration changes associated with creating and deleting these temporary resources. An <i>ephemeral workload</i> is a temporary use of computing resources that are loaded and run when needed. Examples include Amazon Elastic Compute Cloud (Amazon EC2) Spot Instances, Amazon EMR jobs, and Auto Scaling.</p>
     /// <p>If you want to avoid the increased activity from running ephemeral workloads, you can set up the configuration recorder to exclude these resource types from being recorded, or run these types of workloads in a separate account with Config turned off to avoid increased configuration recording and rule evaluations.</p>
     /// </note>
-    pub recording_group: ::std::option::Option<crate::types::RecordingGroup>,
+    pub recording_group: ::std::option::Option<super::super::types::RecordingGroup>,
     /// <p>Specifies the default recording frequency for the configuration recorder. Config supports <i>Continuous recording</i> and <i>Daily recording</i>.</p>
     /// <ul>
     /// <li>
@@ -45,15 +45,15 @@
     /// <p>Firewall Manager depends on continuous recording to monitor your resources. If you are using Firewall Manager, it is recommended that you set the recording frequency to Continuous.</p>
     /// </note>
     /// <p>You can also override the recording frequency for specific resource types.</p>
-    pub recording_mode: ::std::option::Option<crate::types::RecordingMode>,
+    pub recording_mode: ::std::option::Option<super::super::types::RecordingMode>,
     /// <p>Specifies whether the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the specified configuration recorder are recorded for free (<code>INTERNAL</code>) or if it impacts the costs to your bill (<code>PAID</code>).</p>
-    pub recording_scope: ::std::option::Option<crate::types::RecordingScope>,
+    pub recording_scope: ::std::option::Option<super::super::types::RecordingScope>,
     /// <p>For service-linked configuration recorders, specifies the linked Amazon Web Services service for the configuration recorder.</p>
     pub service_principal: ::std::option::Option<::std::string::String>,
     /// <p>The Amazon Resource Name (ARN) of the connector that specifies the connection between a third-party cloud service provider and Config.</p>
     pub connector_arn: ::std::option::Option<::std::string::String>,
     /// <p>Specifies the scope of resources to record from the third-party cloud service provider connected through the connector.</p>
-    pub scope_configuration: ::std::option::Option<crate::types::ScopeConfiguration>,
+    pub scope_configuration: ::std::option::Option<super::super::types::ScopeConfiguration>,
 }
 impl ConfigurationRecorder {
     /// <p>The Amazon Resource Name (ARN) of the specified configuration recorder.</p>
@@ -90,7 +90,7 @@
     /// <p>If you are running ephemeral workloads, you may see increased activity from Config as it records configuration changes associated with creating and deleting these temporary resources. An <i>ephemeral workload</i> is a temporary use of computing resources that are loaded and run when needed. Examples include Amazon Elastic Compute Cloud (Amazon EC2) Spot Instances, Amazon EMR jobs, and Auto Scaling.</p>
     /// <p>If you want to avoid the increased activity from running ephemeral workloads, you can set up the configuration recorder to exclude these resource types from being recorded, or run these types of workloads in a separate account with Config turned off to avoid increased configuration recording and rule evaluations.</p>
     /// </note>
-    pub fn recording_group(&self) -> ::std::option::Option<&crate::types::RecordingGroup> {
+    pub fn recording_group(&self) -> ::std::option::Option<&super::super::types::RecordingGroup> {
         self.recording_group.as_ref()
     }
     /// <p>Specifies the default recording frequency for the configuration recorder. Config supports <i>Continuous recording</i> and <i>Daily recording</i>.</p>
@@ -104,11 +104,11 @@
     /// <p>Firewall Manager depends on continuous recording to monitor your resources. If you are using Firewall Manager, it is recommended that you set the recording frequency to Continuous.</p>
     /// </note>
     /// <p>You can also override the recording frequency for specific resource types.</p>
-    pub fn recording_mode(&self) -> ::std::option::Option<&crate::types::RecordingMode> {
+    pub fn recording_mode(&self) -> ::std::option::Option<&super::super::types::RecordingMode> {
         self.recording_mode.as_ref()
     }
     /// <p>Specifies whether the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the specified configuration recorder are recorded for free (<code>INTERNAL</code>) or if it impacts the costs to your bill (<code>PAID</code>).</p>
-    pub fn recording_scope(&self) -> ::std::option::Option<&crate::types::RecordingScope> {
+    pub fn recording_scope(&self) -> ::std::option::Option<&super::super::types::RecordingScope> {
         self.recording_scope.as_ref()
     }
     /// <p>For service-linked configuration recorders, specifies the linked Amazon Web Services service for the configuration recorder.</p>
@@ -120,14 +120,14 @@
         self.connector_arn.as_deref()
     }
     /// <p>Specifies the scope of resources to record from the third-party cloud service provider connected through the connector.</p>
-    pub fn scope_configuration(&self) -> ::std::option::Option<&crate::types::ScopeConfiguration> {
+    pub fn scope_configuration(&self) -> ::std::option::Option<&super::super::types::ScopeConfiguration> {
         self.scope_configuration.as_ref()
     }
 }
 impl ConfigurationRecorder {
     /// Creates a new builder-style object to manufacture [`ConfigurationRecorder`](crate::types::ConfigurationRecorder).
-    pub fn builder() -> crate::types::builders::ConfigurationRecorderBuilder {
-        crate::types::builders::ConfigurationRecorderBuilder::default()
+    pub fn builder() -> super::super::types::builders::ConfigurationRecorderBuilder {
+        super::super::types::builders::ConfigurationRecorderBuilder::default()
     }
 }

@@ -138,12 +138,12 @@
     pub(crate) arn: ::std::option::Option<::std::string::String>,
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) role_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) recording_group: ::std::option::Option<crate::types::RecordingGroup>,
-    pub(crate) recording_mode: ::std::option::Option<crate::types::RecordingMode>,
-    pub(crate) recording_scope: ::std::option::Option<crate::types::RecordingScope>,
+    pub(crate) recording_group: ::std::option::Option<super::super::types::RecordingGroup>,
+    pub(crate) recording_mode: ::std::option::Option<super::super::types::RecordingMode>,
+    pub(crate) recording_scope: ::std::option::Option<super::super::types::RecordingScope>,
     pub(crate) service_principal: ::std::option::Option<::std::string::String>,
     pub(crate) connector_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) scope_configuration: ::std::option::Option<crate::types::ScopeConfiguration>,
+    pub(crate) scope_configuration: ::std::option::Option<super::super::types::ScopeConfiguration>,
 }
 impl ConfigurationRecorderBuilder {
     /// <p>The Amazon Resource Name (ARN) of the specified configuration recorder.</p>
@@ -242,7 +242,7 @@
     /// <p>If you are running ephemeral workloads, you may see increased activity from Config as it records configuration changes associated with creating and deleting these temporary resources. An <i>ephemeral workload</i> is a temporary use of computing resources that are loaded and run when needed. Examples include Amazon Elastic Compute Cloud (Amazon EC2) Spot Instances, Amazon EMR jobs, and Auto Scaling.</p>
     /// <p>If you want to avoid the increased activity from running ephemeral workloads, you can set up the configuration recorder to exclude these resource types from being recorded, or run these types of workloads in a separate account with Config turned off to avoid increased configuration recording and rule evaluations.</p>
     /// </note>
-    pub fn recording_group(mut self, input: crate::types::RecordingGroup) -> Self {
+    pub fn recording_group(mut self, input: super::super::types::RecordingGroup) -> Self {
         self.recording_group = ::std::option::Option::Some(input);
         self
     }
@@ -252,7 +252,7 @@
     /// <p>If you are running ephemeral workloads, you may see increased activity from Config as it records configuration changes associated with creating and deleting these temporary resources. An <i>ephemeral workload</i> is a temporary use of computing resources that are loaded and run when needed. Examples include Amazon Elastic Compute Cloud (Amazon EC2) Spot Instances, Amazon EMR jobs, and Auto Scaling.</p>
     /// <p>If you want to avoid the increased activity from running ephemeral workloads, you can set up the configuration recorder to exclude these resource types from being recorded, or run these types of workloads in a separate account with Config turned off to avoid increased configuration recording and rule evaluations.</p>
     /// </note>
-    pub fn set_recording_group(mut self, input: ::std::option::Option<crate::types::RecordingGroup>) -> Self {
+    pub fn set_recording_group(mut self, input: ::std::option::Option<super::super::types::RecordingGroup>) -> Self {
         self.recording_group = input;
         self
     }
@@ -262,7 +262,7 @@
     /// <p>If you are running ephemeral workloads, you may see increased activity from Config as it records configuration changes associated with creating and deleting these temporary resources. An <i>ephemeral workload</i> is a temporary use of computing resources that are loaded and run when needed. Examples include Amazon Elastic Compute Cloud (Amazon EC2) Spot Instances, Amazon EMR jobs, and Auto Scaling.</p>
     /// <p>If you want to avoid the increased activity from running ephemeral workloads, you can set up the configuration recorder to exclude these resource types from being recorded, or run these types of workloads in a separate account with Config turned off to avoid increased configuration recording and rule evaluations.</p>
     /// </note>
-    pub fn get_recording_group(&self) -> &::std::option::Option<crate::types::RecordingGroup> {
+    pub fn get_recording_group(&self) -> &::std::option::Option<super::super::types::RecordingGroup> {
         &self.recording_group
     }
     /// <p>Specifies the default recording frequency for the configuration recorder. Config supports <i>Continuous recording</i> and <i>Daily recording</i>.</p>
@@ -276,7 +276,7 @@
     /// <p>Firewall Manager depends on continuous recording to monitor your resources. If you are using Firewall Manager, it is recommended that you set the recording frequency to Continuous.</p>
     /// </note>
     /// <p>You can also override the recording frequency for specific resource types.</p>
-    pub fn recording_mode(mut self, input: crate::types::RecordingMode) -> Self {
+    pub fn recording_mode(mut self, input: super::super::types::RecordingMode) -> Self {
         self.recording_mode = ::std::option::Option::Some(input);
         self
     }
@@ -291,7 +291,7 @@
     /// <p>Firewall Manager depends on continuous recording to monitor your resources. If you are using Firewall Manager, it is recommended that you set the recording frequency to Continuous.</p>
     /// </note>
     /// <p>You can also override the recording frequency for specific resource types.</p>
-    pub fn set_recording_mode(mut self, input: ::std::option::Option<crate::types::RecordingMode>) -> Self {
+    pub fn set_recording_mode(mut self, input: ::std::option::Option<super::super::types::RecordingMode>) -> Self {
         self.recording_mode = input;
         self
     }
@@ -306,21 +306,21 @@
     /// <p>Firewall Manager depends on continuous recording to monitor your resources. If you are using Firewall Manager, it is recommended that you set the recording frequency to Continuous.</p>
     /// </note>
     /// <p>You can also override the recording frequency for specific resource types.</p>
-    pub fn get_recording_mode(&self) -> &::std::option::Option<crate::types::RecordingMode> {
+    pub fn get_recording_mode(&self) -> &::std::option::Option<super::super::types::RecordingMode> {
         &self.recording_mode
     }
     /// <p>Specifies whether the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the specified configuration recorder are recorded for free (<code>INTERNAL</code>) or if it impacts the costs to your bill (<code>PAID</code>).</p>
-    pub fn recording_scope(mut self, input: crate::types::RecordingScope) -> Self {
+    pub fn recording_scope(mut self, input: super::super::types::RecordingScope) -> Self {
         self.recording_scope = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the specified configuration recorder are recorded for free (<code>INTERNAL</code>) or if it impacts the costs to your bill (<code>PAID</code>).</p>
-    pub fn set_recording_scope(mut self, input: ::std::option::Option<crate::types::RecordingScope>) -> Self {
+    pub fn set_recording_scope(mut self, input: ::std::option::Option<super::super::types::RecordingScope>) -> Self {
         self.recording_scope = input;
         self
     }
     /// <p>Specifies whether the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_ConfigurationItem.html">ConfigurationItems</a> in scope for the specified configuration recorder are recorded for free (<code>INTERNAL</code>) or if it impacts the costs to your bill (<code>PAID</code>).</p>
-    pub fn get_recording_scope(&self) -> &::std::option::Option<crate::types::RecordingScope> {
+    pub fn get_recording_scope(&self) -> &::std::option::Option<super::super::types::RecordingScope> {
         &self.recording_scope
     }
     /// <p>For service-linked configuration recorders, specifies the linked Amazon Web Services service for the configuration recorder.</p>
@@ -352,22 +352,22 @@
         &self.connector_arn
     }
     /// <p>Specifies the scope of resources to record from the third-party cloud service provider connected through the connector.</p>
-    pub fn scope_configuration(mut self, input: crate::types::ScopeConfiguration) -> Self {
+    pub fn scope_configuration(mut self, input: super::super::types::ScopeConfiguration) -> Self {
         self.scope_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the scope of resources to record from the third-party cloud service provider connected through the connector.</p>
-    pub fn set_scope_configuration(mut self, input: ::std::option::Option<crate::types::ScopeConfiguration>) -> Self {
+    pub fn set_scope_configuration(mut self, input: ::std::option::Option<super::super::types::ScopeConfiguration>) -> Self {
         self.scope_configuration = input;
         self
     }
     /// <p>Specifies the scope of resources to record from the third-party cloud service provider connected through the connector.</p>
-    pub fn get_scope_configuration(&self) -> &::std::option::Option<crate::types::ScopeConfiguration> {
+    pub fn get_scope_configuration(&self) -> &::std::option::Option<super::super::types::ScopeConfiguration> {
         &self.scope_configuration
     }
     /// Consumes the builder and constructs a [`ConfigurationRecorder`](crate::types::ConfigurationRecorder).
-    pub fn build(self) -> crate::types::ConfigurationRecorder {
-        crate::types::ConfigurationRecorder {
+    pub fn build(self) -> super::super::types::ConfigurationRecorder {
+        super::super::types::ConfigurationRecorder {
             arn: self.arn,
             name: self.name,
             role_arn: self.role_arn,
