--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>The name of the connector.</p>
     pub name: ::std::string::String,
     /// <p>The third-party cloud service provider. Currently, <code>AZURE</code> is supported.</p>
-    pub provider: crate::types::Provider,
+    pub provider: super::types::Provider,
     /// <p>The Azure tenant identifier for the connector.</p>
     pub tenant_identifier: ::std::string::String,
     /// <p>The date and time that the connector was created.</p>
@@ -27,7 +27,7 @@
         self.name.deref()
     }
     /// <p>The third-party cloud service provider. Currently, <code>AZURE</code> is supported.</p>
-    pub fn provider(&self) -> &crate::types::Provider {
+    pub fn provider(&self) -> &super::types::Provider {
         &self.provider
     }
     /// <p>The Azure tenant identifier for the connector.</p>
@@ -42,8 +42,8 @@
 }
 impl ConnectorSummary {
     /// Creates a new builder-style object to manufacture [`ConnectorSummary`](crate::types::ConnectorSummary).
-    pub fn builder() -> crate::types::builders::ConnectorSummaryBuilder {
-        crate::types::builders::ConnectorSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::ConnectorSummaryBuilder {
+        super::types::builders::ConnectorSummaryBuilder::default()
     }
 }

@@ -53,7 +53,7 @@
 pub struct ConnectorSummaryBuilder {
     pub(crate) arn: ::std::option::Option<::std::string::String>,
     pub(crate) name: ::std::option::Option<::std::string::String>,
-    pub(crate) provider: ::std::option::Option<crate::types::Provider>,
+    pub(crate) provider: ::std::option::Option<super::types::Provider>,
     pub(crate) tenant_identifier: ::std::option::Option<::std::string::String>,
     pub(crate) created_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -90,17 +90,17 @@
     }
     /// <p>The third-party cloud service provider. Currently, <code>AZURE</code> is supported.</p>
     /// This field is required.
-    pub fn provider(mut self, input: crate::types::Provider) -> Self {
+    pub fn provider(mut self, input: super::types::Provider) -> Self {
         self.provider = ::std::option::Option::Some(input);
         self
     }
     /// <p>The third-party cloud service provider. Currently, <code>AZURE</code> is supported.</p>
-    pub fn set_provider(mut self, input: ::std::option::Option<crate::types::Provider>) -> Self {
+    pub fn set_provider(mut self, input: ::std::option::Option<super::types::Provider>) -> Self {
         self.provider = input;
         self
     }
     /// <p>The third-party cloud service provider. Currently, <code>AZURE</code> is supported.</p>
-    pub fn get_provider(&self) -> &::std::option::Option<crate::types::Provider> {
+    pub fn get_provider(&self) -> &::std::option::Option<super::types::Provider> {
         &self.provider
     }
     /// <p>The Azure tenant identifier for the connector.</p>
@@ -140,8 +140,8 @@
     /// - [`provider`](crate::types::builders::ConnectorSummaryBuilder::provider)
     /// - [`tenant_identifier`](crate::types::builders::ConnectorSummaryBuilder::tenant_identifier)
     /// - [`created_time`](crate::types::builders::ConnectorSummaryBuilder::created_time)
-    pub fn build(self) -> ::std::result::Result<crate::types::ConnectorSummary, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ConnectorSummary {
+    pub fn build(self) -> ::std::result::Result<super::types::ConnectorSummary, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ConnectorSummary {
             arn: self.arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "arn",
