--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ConnectorConfiguration {
     /// <p>The configuration for an Azure connector.</p>
-    pub azure: ::std::option::Option<crate::types::AzureConnectorConfiguration>,
+    pub azure: ::std::option::Option<super::super::types::AzureConnectorConfiguration>,
 }
 impl ConnectorConfiguration {
     /// <p>The configuration for an Azure connector.</p>
-    pub fn azure(&self) -> ::std::option::Option<&crate::types::AzureConnectorConfiguration> {
+    pub fn azure(&self) -> ::std::option::Option<&super::super::types::AzureConnectorConfiguration> {
         self.azure.as_ref()
     }
 }
 impl ConnectorConfiguration {
     /// Creates a new builder-style object to manufacture [`ConnectorConfiguration`](crate::types::ConnectorConfiguration).
-    pub fn builder() -> crate::types::builders::ConnectorConfigurationBuilder {
-        crate::types::builders::ConnectorConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::ConnectorConfigurationBuilder {
+        super::super::types::builders::ConnectorConfigurationBuilder::default()
     }
 }

@@ -24,25 +24,25 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ConnectorConfigurationBuilder {
-    pub(crate) azure: ::std::option::Option<crate::types::AzureConnectorConfiguration>,
+    pub(crate) azure: ::std::option::Option<super::super::types::AzureConnectorConfiguration>,
 }
 impl ConnectorConfigurationBuilder {
     /// <p>The configuration for an Azure connector.</p>
-    pub fn azure(mut self, input: crate::types::AzureConnectorConfiguration) -> Self {
+    pub fn azure(mut self, input: super::super::types::AzureConnectorConfiguration) -> Self {
         self.azure = ::std::option::Option::Some(input);
         self
     }
     /// <p>The configuration for an Azure connector.</p>
-    pub fn set_azure(mut self, input: ::std::option::Option<crate::types::AzureConnectorConfiguration>) -> Self {
+    pub fn set_azure(mut self, input: ::std::option::Option<super::super::types::AzureConnectorConfiguration>) -> Self {
         self.azure = input;
         self
     }
     /// <p>The configuration for an Azure connector.</p>
-    pub fn get_azure(&self) -> &::std::option::Option<crate::types::AzureConnectorConfiguration> {
+    pub fn get_azure(&self) -> &::std::option::Option<super::super::types::AzureConnectorConfiguration> {
         &self.azure
     }
     /// Consumes the builder and constructs a [`ConnectorConfiguration`](crate::types::ConnectorConfiguration).
-    pub fn build(self) -> crate::types::ConnectorConfiguration {
-        crate::types::ConnectorConfiguration { azure: self.azure }
+    pub fn build(self) -> super::super::types::ConnectorConfiguration {
+        super::super::types::ConnectorConfiguration { azure: self.azure }
     }
 }
