--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl ConformancePackComplianceScore {
     /// Creates a new builder-style object to manufacture [`ConformancePackComplianceScore`](crate::types::ConformancePackComplianceScore).
-    pub fn builder() -> crate::types::builders::ConformancePackComplianceScoreBuilder {
-        crate::types::builders::ConformancePackComplianceScoreBuilder::default()
+    pub fn builder() -> super::super::types::builders::ConformancePackComplianceScoreBuilder {
+        super::super::types::builders::ConformancePackComplianceScoreBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.last_updated_time
     }
     /// Consumes the builder and constructs a [`ConformancePackComplianceScore`](crate::types::ConformancePackComplianceScore).
-    pub fn build(self) -> crate::types::ConformancePackComplianceScore {
-        crate::types::ConformancePackComplianceScore {
+    pub fn build(self) -> super::super::types::ConformancePackComplianceScore {
+        super::super::types::ConformancePackComplianceScore {
             score: self.score,
             conformance_pack_name: self.conformance_pack_name,
             last_updated_time: self.last_updated_time,
