--- original
+++ modified
@@ -6,7 +6,7 @@
 pub struct ConfigStreamDeliveryInfo {
     /// <p>Status of the last attempted delivery.</p>
     /// <p><b>Note</b> Providing an SNS topic on a <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeliveryChannel.html">DeliveryChannel</a> for Config is optional. If the SNS delivery is turned off, the last status will be <b>Not_Applicable</b>.</p>
-    pub last_status: ::std::option::Option<crate::types::DeliveryStatus>,
+    pub last_status: ::std::option::Option<super::types::DeliveryStatus>,
     /// <p>The error code from the last attempted delivery.</p>
     pub last_error_code: ::std::option::Option<::std::string::String>,
     /// <p>The error message from the last attempted delivery.</p>
@@ -17,7 +17,7 @@
 impl ConfigStreamDeliveryInfo {
     /// <p>Status of the last attempted delivery.</p>
     /// <p><b>Note</b> Providing an SNS topic on a <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeliveryChannel.html">DeliveryChannel</a> for Config is optional. If the SNS delivery is turned off, the last status will be <b>Not_Applicable</b>.</p>
-    pub fn last_status(&self) -> ::std::option::Option<&crate::types::DeliveryStatus> {
+    pub fn last_status(&self) -> ::std::option::Option<&super::types::DeliveryStatus> {
         self.last_status.as_ref()
     }
     /// <p>The error code from the last attempted delivery.</p>
@@ -35,8 +35,8 @@
 }
 impl ConfigStreamDeliveryInfo {
     /// Creates a new builder-style object to manufacture [`ConfigStreamDeliveryInfo`](crate::types::ConfigStreamDeliveryInfo).
-    pub fn builder() -> crate::types::builders::ConfigStreamDeliveryInfoBuilder {
-        crate::types::builders::ConfigStreamDeliveryInfoBuilder::default()
+    pub fn builder() -> super::types::builders::ConfigStreamDeliveryInfoBuilder {
+        super::types::builders::ConfigStreamDeliveryInfoBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ConfigStreamDeliveryInfoBuilder {
-    pub(crate) last_status: ::std::option::Option<crate::types::DeliveryStatus>,
+    pub(crate) last_status: ::std::option::Option<super::types::DeliveryStatus>,
     pub(crate) last_error_code: ::std::option::Option<::std::string::String>,
     pub(crate) last_error_message: ::std::option::Option<::std::string::String>,
     pub(crate) last_status_change_time: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -52,19 +52,19 @@
 impl ConfigStreamDeliveryInfoBuilder {
     /// <p>Status of the last attempted delivery.</p>
     /// <p><b>Note</b> Providing an SNS topic on a <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeliveryChannel.html">DeliveryChannel</a> for Config is optional. If the SNS delivery is turned off, the last status will be <b>Not_Applicable</b>.</p>
-    pub fn last_status(mut self, input: crate::types::DeliveryStatus) -> Self {
+    pub fn last_status(mut self, input: super::types::DeliveryStatus) -> Self {
         self.last_status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Status of the last attempted delivery.</p>
     /// <p><b>Note</b> Providing an SNS topic on a <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeliveryChannel.html">DeliveryChannel</a> for Config is optional. If the SNS delivery is turned off, the last status will be <b>Not_Applicable</b>.</p>
-    pub fn set_last_status(mut self, input: ::std::option::Option<crate::types::DeliveryStatus>) -> Self {
+    pub fn set_last_status(mut self, input: ::std::option::Option<super::types::DeliveryStatus>) -> Self {
         self.last_status = input;
         self
     }
     /// <p>Status of the last attempted delivery.</p>
     /// <p><b>Note</b> Providing an SNS topic on a <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeliveryChannel.html">DeliveryChannel</a> for Config is optional. If the SNS delivery is turned off, the last status will be <b>Not_Applicable</b>.</p>
-    pub fn get_last_status(&self) -> &::std::option::Option<crate::types::DeliveryStatus> {
+    pub fn get_last_status(&self) -> &::std::option::Option<super::types::DeliveryStatus> {
         &self.last_status
     }
     /// <p>The error code from the last attempted delivery.</p>
@@ -110,8 +110,8 @@
         &self.last_status_change_time
     }
     /// Consumes the builder and constructs a [`ConfigStreamDeliveryInfo`](crate::types::ConfigStreamDeliveryInfo).
-    pub fn build(self) -> crate::types::ConfigStreamDeliveryInfo {
-        crate::types::ConfigStreamDeliveryInfo {
+    pub fn build(self) -> super::types::ConfigStreamDeliveryInfo {
+        super::types::ConfigStreamDeliveryInfo {
             last_status: self.last_status,
             last_error_code: self.last_error_code,
             last_error_message: self.last_error_message,
