--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>The Amazon Resource Name (ARN) of the connector.</p>
     pub arn: ::std::string::String,
     /// <p>The provider-specific configuration for connecting to the third-party cloud service provider.</p>
-    pub connector_configuration: ::std::option::Option<crate::types::ConnectorConfiguration>,
+    pub connector_configuration: ::std::option::Option<super::types::ConnectorConfiguration>,
     /// <p>The date and time that the connector was created.</p>
     pub created_time: ::aws_smithy_types::DateTime,
 }
@@ -25,7 +25,7 @@
         self.arn.deref()
     }
     /// <p>The provider-specific configuration for connecting to the third-party cloud service provider.</p>
-    pub fn connector_configuration(&self) -> ::std::option::Option<&crate::types::ConnectorConfiguration> {
+    pub fn connector_configuration(&self) -> ::std::option::Option<&super::types::ConnectorConfiguration> {
         self.connector_configuration.as_ref()
     }
     /// <p>The date and time that the connector was created.</p>
@@ -35,8 +35,8 @@
 }
 impl Connector {
     /// Creates a new builder-style object to manufacture [`Connector`](crate::types::Connector).
-    pub fn builder() -> crate::types::builders::ConnectorBuilder {
-        crate::types::builders::ConnectorBuilder::default()
+    pub fn builder() -> super::types::builders::ConnectorBuilder {
+        super::types::builders::ConnectorBuilder::default()
     }
 }

@@ -46,7 +46,7 @@
 pub struct ConnectorBuilder {
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) arn: ::std::option::Option<::std::string::String>,
-    pub(crate) connector_configuration: ::std::option::Option<crate::types::ConnectorConfiguration>,
+    pub(crate) connector_configuration: ::std::option::Option<super::types::ConnectorConfiguration>,
     pub(crate) created_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl ConnectorBuilder {
@@ -82,17 +82,17 @@
     }
     /// <p>The provider-specific configuration for connecting to the third-party cloud service provider.</p>
     /// This field is required.
-    pub fn connector_configuration(mut self, input: crate::types::ConnectorConfiguration) -> Self {
+    pub fn connector_configuration(mut self, input: super::types::ConnectorConfiguration) -> Self {
         self.connector_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>The provider-specific configuration for connecting to the third-party cloud service provider.</p>
-    pub fn set_connector_configuration(mut self, input: ::std::option::Option<crate::types::ConnectorConfiguration>) -> Self {
+    pub fn set_connector_configuration(mut self, input: ::std::option::Option<super::types::ConnectorConfiguration>) -> Self {
         self.connector_configuration = input;
         self
     }
     /// <p>The provider-specific configuration for connecting to the third-party cloud service provider.</p>
-    pub fn get_connector_configuration(&self) -> &::std::option::Option<crate::types::ConnectorConfiguration> {
+    pub fn get_connector_configuration(&self) -> &::std::option::Option<super::types::ConnectorConfiguration> {
         &self.connector_configuration
     }
     /// <p>The date and time that the connector was created.</p>
@@ -115,8 +115,8 @@
     /// - [`name`](crate::types::builders::ConnectorBuilder::name)
     /// - [`arn`](crate::types::builders::ConnectorBuilder::arn)
     /// - [`created_time`](crate::types::builders::ConnectorBuilder::created_time)
-    pub fn build(self) -> ::std::result::Result<crate::types::Connector, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Connector {
+    pub fn build(self) -> ::std::result::Result<super::types::Connector, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::Connector {
             name: self.name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "name",
