--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl CustomPolicyDetails {
     /// Creates a new builder-style object to manufacture [`CustomPolicyDetails`](crate::types::CustomPolicyDetails).
-    pub fn builder() -> crate::types::builders::CustomPolicyDetailsBuilder {
-        crate::types::builders::CustomPolicyDetailsBuilder::default()
+    pub fn builder() -> super::super::types::builders::CustomPolicyDetailsBuilder {
+        super::super::types::builders::CustomPolicyDetailsBuilder::default()
     }
 }

@@ -91,8 +91,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`policy_runtime`](crate::types::builders::CustomPolicyDetailsBuilder::policy_runtime)
     /// - [`policy_text`](crate::types::builders::CustomPolicyDetailsBuilder::policy_text)
-    pub fn build(self) -> ::std::result::Result<crate::types::CustomPolicyDetails, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::CustomPolicyDetails {
+    pub fn build(self) -> ::std::result::Result<super::super::types::CustomPolicyDetails, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::CustomPolicyDetails {
             policy_runtime: self.policy_runtime.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "policy_runtime",
