--- original
+++ modified
@@ -8,11 +8,11 @@
     /// <p>The name of the delivery channel.</p>
     pub name: ::std::option::Option<::std::string::String>,
     /// <p>A list containing the status of the delivery of the snapshot to the specified Amazon S3 bucket.</p>
-    pub config_snapshot_delivery_info: ::std::option::Option<crate::types::ConfigExportDeliveryInfo>,
+    pub config_snapshot_delivery_info: ::std::option::Option<super::super::types::ConfigExportDeliveryInfo>,
     /// <p>A list that contains the status of the delivery of the configuration history to the specified Amazon S3 bucket.</p>
-    pub config_history_delivery_info: ::std::option::Option<crate::types::ConfigExportDeliveryInfo>,
+    pub config_history_delivery_info: ::std::option::Option<super::super::types::ConfigExportDeliveryInfo>,
     /// <p>A list containing the status of the delivery of the configuration stream notification to the specified Amazon SNS topic.</p>
-    pub config_stream_delivery_info: ::std::option::Option<crate::types::ConfigStreamDeliveryInfo>,
+    pub config_stream_delivery_info: ::std::option::Option<super::super::types::ConfigStreamDeliveryInfo>,
 }
 impl DeliveryChannelStatus {
     /// <p>The name of the delivery channel.</p>
@@ -20,22 +20,22 @@
         self.name.as_deref()
     }
     /// <p>A list containing the status of the delivery of the snapshot to the specified Amazon S3 bucket.</p>
-    pub fn config_snapshot_delivery_info(&self) -> ::std::option::Option<&crate::types::ConfigExportDeliveryInfo> {
+    pub fn config_snapshot_delivery_info(&self) -> ::std::option::Option<&super::super::types::ConfigExportDeliveryInfo> {
         self.config_snapshot_delivery_info.as_ref()
     }
     /// <p>A list that contains the status of the delivery of the configuration history to the specified Amazon S3 bucket.</p>
-    pub fn config_history_delivery_info(&self) -> ::std::option::Option<&crate::types::ConfigExportDeliveryInfo> {
+    pub fn config_history_delivery_info(&self) -> ::std::option::Option<&super::super::types::ConfigExportDeliveryInfo> {
         self.config_history_delivery_info.as_ref()
     }
     /// <p>A list containing the status of the delivery of the configuration stream notification to the specified Amazon SNS topic.</p>
-    pub fn config_stream_delivery_info(&self) -> ::std::option::Option<&crate::types::ConfigStreamDeliveryInfo> {
+    pub fn config_stream_delivery_info(&self) -> ::std::option::Option<&super::super::types::ConfigStreamDeliveryInfo> {
         self.config_stream_delivery_info.as_ref()
     }
 }
 impl DeliveryChannelStatus {
     /// Creates a new builder-style object to manufacture [`DeliveryChannelStatus`](crate::types::DeliveryChannelStatus).
-    pub fn builder() -> crate::types::builders::DeliveryChannelStatusBuilder {
-        crate::types::builders::DeliveryChannelStatusBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeliveryChannelStatusBuilder {
+        super::super::types::builders::DeliveryChannelStatusBuilder::default()
     }
 }

@@ -44,9 +44,9 @@
 #[non_exhaustive]
 pub struct DeliveryChannelStatusBuilder {
     pub(crate) name: ::std::option::Option<::std::string::String>,
-    pub(crate) config_snapshot_delivery_info: ::std::option::Option<crate::types::ConfigExportDeliveryInfo>,
-    pub(crate) config_history_delivery_info: ::std::option::Option<crate::types::ConfigExportDeliveryInfo>,
-    pub(crate) config_stream_delivery_info: ::std::option::Option<crate::types::ConfigStreamDeliveryInfo>,
+    pub(crate) config_snapshot_delivery_info: ::std::option::Option<super::super::types::ConfigExportDeliveryInfo>,
+    pub(crate) config_history_delivery_info: ::std::option::Option<super::super::types::ConfigExportDeliveryInfo>,
+    pub(crate) config_stream_delivery_info: ::std::option::Option<super::super::types::ConfigStreamDeliveryInfo>,
 }
 impl DeliveryChannelStatusBuilder {
     /// <p>The name of the delivery channel.</p>
@@ -64,50 +64,50 @@
         &self.name
     }
     /// <p>A list containing the status of the delivery of the snapshot to the specified Amazon S3 bucket.</p>
-    pub fn config_snapshot_delivery_info(mut self, input: crate::types::ConfigExportDeliveryInfo) -> Self {
+    pub fn config_snapshot_delivery_info(mut self, input: super::super::types::ConfigExportDeliveryInfo) -> Self {
         self.config_snapshot_delivery_info = ::std::option::Option::Some(input);
         self
     }
     /// <p>A list containing the status of the delivery of the snapshot to the specified Amazon S3 bucket.</p>
-    pub fn set_config_snapshot_delivery_info(mut self, input: ::std::option::Option<crate::types::ConfigExportDeliveryInfo>) -> Self {
+    pub fn set_config_snapshot_delivery_info(mut self, input: ::std::option::Option<super::super::types::ConfigExportDeliveryInfo>) -> Self {
         self.config_snapshot_delivery_info = input;
         self
     }
     /// <p>A list containing the status of the delivery of the snapshot to the specified Amazon S3 bucket.</p>
-    pub fn get_config_snapshot_delivery_info(&self) -> &::std::option::Option<crate::types::ConfigExportDeliveryInfo> {
+    pub fn get_config_snapshot_delivery_info(&self) -> &::std::option::Option<super::super::types::ConfigExportDeliveryInfo> {
         &self.config_snapshot_delivery_info
     }
     /// <p>A list that contains the status of the delivery of the configuration history to the specified Amazon S3 bucket.</p>
-    pub fn config_history_delivery_info(mut self, input: crate::types::ConfigExportDeliveryInfo) -> Self {
+    pub fn config_history_delivery_info(mut self, input: super::super::types::ConfigExportDeliveryInfo) -> Self {
         self.config_history_delivery_info = ::std::option::Option::Some(input);
         self
     }
     /// <p>A list that contains the status of the delivery of the configuration history to the specified Amazon S3 bucket.</p>
-    pub fn set_config_history_delivery_info(mut self, input: ::std::option::Option<crate::types::ConfigExportDeliveryInfo>) -> Self {
+    pub fn set_config_history_delivery_info(mut self, input: ::std::option::Option<super::super::types::ConfigExportDeliveryInfo>) -> Self {
         self.config_history_delivery_info = input;
         self
     }
     /// <p>A list that contains the status of the delivery of the configuration history to the specified Amazon S3 bucket.</p>
-    pub fn get_config_history_delivery_info(&self) -> &::std::option::Option<crate::types::ConfigExportDeliveryInfo> {
+    pub fn get_config_history_delivery_info(&self) -> &::std::option::Option<super::super::types::ConfigExportDeliveryInfo> {
         &self.config_history_delivery_info
     }
     /// <p>A list containing the status of the delivery of the configuration stream notification to the specified Amazon SNS topic.</p>
-    pub fn config_stream_delivery_info(mut self, input: crate::types::ConfigStreamDeliveryInfo) -> Self {
+    pub fn config_stream_delivery_info(mut self, input: super::super::types::ConfigStreamDeliveryInfo) -> Self {
         self.config_stream_delivery_info = ::std::option::Option::Some(input);
         self
     }
     /// <p>A list containing the status of the delivery of the configuration stream notification to the specified Amazon SNS topic.</p>
-    pub fn set_config_stream_delivery_info(mut self, input: ::std::option::Option<crate::types::ConfigStreamDeliveryInfo>) -> Self {
+    pub fn set_config_stream_delivery_info(mut self, input: ::std::option::Option<super::super::types::ConfigStreamDeliveryInfo>) -> Self {
         self.config_stream_delivery_info = input;
         self
     }
     /// <p>A list containing the status of the delivery of the configuration stream notification to the specified Amazon SNS topic.</p>
-    pub fn get_config_stream_delivery_info(&self) -> &::std::option::Option<crate::types::ConfigStreamDeliveryInfo> {
+    pub fn get_config_stream_delivery_info(&self) -> &::std::option::Option<super::super::types::ConfigStreamDeliveryInfo> {
         &self.config_stream_delivery_info
     }
     /// Consumes the builder and constructs a [`DeliveryChannelStatus`](crate::types::DeliveryChannelStatus).
-    pub fn build(self) -> crate::types::DeliveryChannelStatus {
-        crate::types::DeliveryChannelStatus {
+    pub fn build(self) -> super::super::types::DeliveryChannelStatus {
+        super::super::types::DeliveryChannelStatus {
             name: self.name,
             config_snapshot_delivery_info: self.config_snapshot_delivery_info,
             config_history_delivery_info: self.config_history_delivery_info,
