--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl GroupedResourceCount {
     /// Creates a new builder-style object to manufacture [`GroupedResourceCount`](crate::types::GroupedResourceCount).
-    pub fn builder() -> crate::types::builders::GroupedResourceCountBuilder {
-        crate::types::builders::GroupedResourceCountBuilder::default()
+    pub fn builder() -> super::types::builders::GroupedResourceCountBuilder {
+        super::types::builders::GroupedResourceCountBuilder::default()
     }
 }

@@ -68,8 +68,8 @@
     /// Consumes the builder and constructs a [`GroupedResourceCount`](crate::types::GroupedResourceCount).
     /// This method will fail if any of the following fields are not set:
     /// - [`group_name`](crate::types::builders::GroupedResourceCountBuilder::group_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::GroupedResourceCount, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::GroupedResourceCount {
+    pub fn build(self) -> ::std::result::Result<super::types::GroupedResourceCount, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::GroupedResourceCount {
             group_name: self.group_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "group_name",
