--- original
+++ modified
@@ -17,7 +17,7 @@
     /// </note>
     pub delivery_s3_key_prefix: ::std::option::Option<::std::string::String>,
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+    pub conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<super::super::types::ConformancePackInputParameter>>,
     /// <p>A comma-separated list of accounts excluded from organization conformance pack.</p>
     pub excluded_accounts: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>Last time when organization conformation pack was updated.</p>
@@ -49,7 +49,7 @@
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.conformance_pack_input_parameters.is_none()`.
-    pub fn conformance_pack_input_parameters(&self) -> &[crate::types::ConformancePackInputParameter] {
+    pub fn conformance_pack_input_parameters(&self) -> &[super::super::types::ConformancePackInputParameter] {
         self.conformance_pack_input_parameters.as_deref().unwrap_or_default()
     }
     /// <p>A comma-separated list of accounts excluded from organization conformance pack.</p>
@@ -65,8 +65,8 @@
 }
 impl OrganizationConformancePack {
     /// Creates a new builder-style object to manufacture [`OrganizationConformancePack`](crate::types::OrganizationConformancePack).
-    pub fn builder() -> crate::types::builders::OrganizationConformancePackBuilder {
-        crate::types::builders::OrganizationConformancePackBuilder::default()
+    pub fn builder() -> super::super::types::builders::OrganizationConformancePackBuilder {
+        super::super::types::builders::OrganizationConformancePackBuilder::default()
     }
 }

@@ -78,7 +78,7 @@
     pub(crate) organization_conformance_pack_arn: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_s3_bucket: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_s3_key_prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+    pub(crate) conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<super::super::types::ConformancePackInputParameter>>,
     pub(crate) excluded_accounts: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -158,7 +158,7 @@
     /// To override the contents of this collection use [`set_conformance_pack_input_parameters`](Self::set_conformance_pack_input_parameters).
     ///
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub fn conformance_pack_input_parameters(mut self, input: crate::types::ConformancePackInputParameter) -> Self {
+    pub fn conformance_pack_input_parameters(mut self, input: super::super::types::ConformancePackInputParameter) -> Self {
         let mut v = self.conformance_pack_input_parameters.unwrap_or_default();
         v.push(input);
         self.conformance_pack_input_parameters = ::std::option::Option::Some(v);
@@ -167,13 +167,13 @@
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
     pub fn set_conformance_pack_input_parameters(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::types::ConformancePackInputParameter>>,
     ) -> Self {
         self.conformance_pack_input_parameters = input;
         self
     }
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub fn get_conformance_pack_input_parameters(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>> {
+    pub fn get_conformance_pack_input_parameters(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::ConformancePackInputParameter>> {
         &self.conformance_pack_input_parameters
     }
     /// Appends an item to `excluded_accounts`.
@@ -216,8 +216,8 @@
     /// - [`organization_conformance_pack_name`](crate::types::builders::OrganizationConformancePackBuilder::organization_conformance_pack_name)
     /// - [`organization_conformance_pack_arn`](crate::types::builders::OrganizationConformancePackBuilder::organization_conformance_pack_arn)
     /// - [`last_update_time`](crate::types::builders::OrganizationConformancePackBuilder::last_update_time)
-    pub fn build(self) -> ::std::result::Result<crate::types::OrganizationConformancePack, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::OrganizationConformancePack {
+    pub fn build(self) -> ::std::result::Result<super::super::types::OrganizationConformancePack, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::OrganizationConformancePack {
             organization_conformance_pack_name: self.organization_conformance_pack_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "organization_conformance_pack_name",
