--- original
+++ modified
@@ -28,7 +28,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when an organization Config rule update failed in one or more member accounts within that organization.</p></li>
     /// </ul>
-    pub organization_rule_status: crate::types::OrganizationRuleStatus,
+    pub organization_rule_status: super::super::types::OrganizationRuleStatus,
     /// <p>An error code that is returned when organization Config rule creation or deletion has failed.</p>
     pub error_code: ::std::option::Option<::std::string::String>,
     /// <p>An error message indicating that organization Config rule creation or deletion failed due to an error.</p>
@@ -64,7 +64,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when an organization Config rule update failed in one or more member accounts within that organization.</p></li>
     /// </ul>
-    pub fn organization_rule_status(&self) -> &crate::types::OrganizationRuleStatus {
+    pub fn organization_rule_status(&self) -> &super::super::types::OrganizationRuleStatus {
         &self.organization_rule_status
     }
     /// <p>An error code that is returned when organization Config rule creation or deletion has failed.</p>
@@ -82,8 +82,8 @@
 }
 impl OrganizationConfigRuleStatus {
     /// Creates a new builder-style object to manufacture [`OrganizationConfigRuleStatus`](crate::types::OrganizationConfigRuleStatus).
-    pub fn builder() -> crate::types::builders::OrganizationConfigRuleStatusBuilder {
-        crate::types::builders::OrganizationConfigRuleStatusBuilder::default()
+    pub fn builder() -> super::super::types::builders::OrganizationConfigRuleStatusBuilder {
+        super::super::types::builders::OrganizationConfigRuleStatusBuilder::default()
     }
 }

@@ -92,7 +92,7 @@
 #[non_exhaustive]
 pub struct OrganizationConfigRuleStatusBuilder {
     pub(crate) organization_config_rule_name: ::std::option::Option<::std::string::String>,
-    pub(crate) organization_rule_status: ::std::option::Option<crate::types::OrganizationRuleStatus>,
+    pub(crate) organization_rule_status: ::std::option::Option<super::super::types::OrganizationRuleStatus>,
     pub(crate) error_code: ::std::option::Option<::std::string::String>,
     pub(crate) error_message: ::std::option::Option<::std::string::String>,
     pub(crate) last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -136,7 +136,7 @@
     /// <p><code>UPDATE_FAILED</code> when an organization Config rule update failed in one or more member accounts within that organization.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn organization_rule_status(mut self, input: crate::types::OrganizationRuleStatus) -> Self {
+    pub fn organization_rule_status(mut self, input: super::super::types::OrganizationRuleStatus) -> Self {
         self.organization_rule_status = ::std::option::Option::Some(input);
         self
     }
@@ -162,7 +162,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when an organization Config rule update failed in one or more member accounts within that organization.</p></li>
     /// </ul>
-    pub fn set_organization_rule_status(mut self, input: ::std::option::Option<crate::types::OrganizationRuleStatus>) -> Self {
+    pub fn set_organization_rule_status(mut self, input: ::std::option::Option<super::super::types::OrganizationRuleStatus>) -> Self {
         self.organization_rule_status = input;
         self
     }
@@ -188,7 +188,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when an organization Config rule update failed in one or more member accounts within that organization.</p></li>
     /// </ul>
-    pub fn get_organization_rule_status(&self) -> &::std::option::Option<crate::types::OrganizationRuleStatus> {
+    pub fn get_organization_rule_status(&self) -> &::std::option::Option<super::super::types::OrganizationRuleStatus> {
         &self.organization_rule_status
     }
     /// <p>An error code that is returned when organization Config rule creation or deletion has failed.</p>
@@ -237,8 +237,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`organization_config_rule_name`](crate::types::builders::OrganizationConfigRuleStatusBuilder::organization_config_rule_name)
     /// - [`organization_rule_status`](crate::types::builders::OrganizationConfigRuleStatusBuilder::organization_rule_status)
-    pub fn build(self) -> ::std::result::Result<crate::types::OrganizationConfigRuleStatus, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::OrganizationConfigRuleStatus {
+    pub fn build(self) -> ::std::result::Result<super::super::types::OrganizationConfigRuleStatus, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::OrganizationConfigRuleStatus {
             organization_config_rule_name: self.organization_config_rule_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "organization_config_rule_name",
