--- original
+++ modified
@@ -28,7 +28,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub status: ::std::option::Option<crate::types::OrganizationResourceDetailedStatus>,
+    pub status: ::std::option::Option<super::types::OrganizationResourceDetailedStatus>,
 }
 impl OrganizationResourceDetailedStatusFilters {
     /// <p>The 12-digit account ID of the member account within an organization.</p>
@@ -57,14 +57,14 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::OrganizationResourceDetailedStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::OrganizationResourceDetailedStatus> {
         self.status.as_ref()
     }
 }
 impl OrganizationResourceDetailedStatusFilters {
     /// Creates a new builder-style object to manufacture [`OrganizationResourceDetailedStatusFilters`](crate::types::OrganizationResourceDetailedStatusFilters).
-    pub fn builder() -> crate::types::builders::OrganizationResourceDetailedStatusFiltersBuilder {
-        crate::types::builders::OrganizationResourceDetailedStatusFiltersBuilder::default()
+    pub fn builder() -> super::types::builders::OrganizationResourceDetailedStatusFiltersBuilder {
+        super::types::builders::OrganizationResourceDetailedStatusFiltersBuilder::default()
     }
 }

@@ -73,7 +73,7 @@
 #[non_exhaustive]
 pub struct OrganizationResourceDetailedStatusFiltersBuilder {
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::OrganizationResourceDetailedStatus>,
+    pub(crate) status: ::std::option::Option<super::types::OrganizationResourceDetailedStatus>,
 }
 impl OrganizationResourceDetailedStatusFiltersBuilder {
     /// <p>The 12-digit account ID of the member account within an organization.</p>
@@ -112,7 +112,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn status(mut self, input: crate::types::OrganizationResourceDetailedStatus) -> Self {
+    pub fn status(mut self, input: super::types::OrganizationResourceDetailedStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
@@ -138,7 +138,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::OrganizationResourceDetailedStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::OrganizationResourceDetailedStatus>) -> Self {
         self.status = input;
         self
     }
@@ -164,12 +164,12 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::OrganizationResourceDetailedStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::OrganizationResourceDetailedStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`OrganizationResourceDetailedStatusFilters`](crate::types::OrganizationResourceDetailedStatusFilters).
-    pub fn build(self) -> crate::types::OrganizationResourceDetailedStatusFilters {
-        crate::types::OrganizationResourceDetailedStatusFilters {
+    pub fn build(self) -> super::types::OrganizationResourceDetailedStatusFilters {
+        super::types::OrganizationResourceDetailedStatusFilters {
             account_id: self.account_id,
             status: self.status,
         }
