--- original
+++ modified
@@ -9,15 +9,15 @@
     /// <p>Amazon Resource Name (ARN) of organization Config rule.</p>
     pub organization_config_rule_arn: ::std::string::String,
     /// <p>An <code>OrganizationManagedRuleMetadata</code> object.</p>
-    pub organization_managed_rule_metadata: ::std::option::Option<crate::types::OrganizationManagedRuleMetadata>,
+    pub organization_managed_rule_metadata: ::std::option::Option<super::super::types::OrganizationManagedRuleMetadata>,
     /// <p>An <code>OrganizationCustomRuleMetadata</code> object.</p>
-    pub organization_custom_rule_metadata: ::std::option::Option<crate::types::OrganizationCustomRuleMetadata>,
+    pub organization_custom_rule_metadata: ::std::option::Option<super::super::types::OrganizationCustomRuleMetadata>,
     /// <p>A comma-separated list of accounts excluded from organization Config rule.</p>
     pub excluded_accounts: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>The timestamp of the last update.</p>
     pub last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>An object that specifies metadata for your organization's Config Custom Policy rule. The metadata includes the runtime system in use, which accounts have debug logging enabled, and other custom rule metadata, such as resource type, resource ID of Amazon Web Services resource, and organization trigger types that initiate Config to evaluate Amazon Web Services resources against a rule.</p>
-    pub organization_custom_policy_rule_metadata: ::std::option::Option<crate::types::OrganizationCustomPolicyRuleMetadataNoPolicy>,
+    pub organization_custom_policy_rule_metadata: ::std::option::Option<super::super::types::OrganizationCustomPolicyRuleMetadataNoPolicy>,
 }
 impl OrganizationConfigRule {
     /// <p>The name that you assign to organization Config rule.</p>
@@ -31,11 +31,11 @@
         self.organization_config_rule_arn.deref()
     }
     /// <p>An <code>OrganizationManagedRuleMetadata</code> object.</p>
-    pub fn organization_managed_rule_metadata(&self) -> ::std::option::Option<&crate::types::OrganizationManagedRuleMetadata> {
+    pub fn organization_managed_rule_metadata(&self) -> ::std::option::Option<&super::super::types::OrganizationManagedRuleMetadata> {
         self.organization_managed_rule_metadata.as_ref()
     }
     /// <p>An <code>OrganizationCustomRuleMetadata</code> object.</p>
-    pub fn organization_custom_rule_metadata(&self) -> ::std::option::Option<&crate::types::OrganizationCustomRuleMetadata> {
+    pub fn organization_custom_rule_metadata(&self) -> ::std::option::Option<&super::super::types::OrganizationCustomRuleMetadata> {
         self.organization_custom_rule_metadata.as_ref()
     }
     /// <p>A comma-separated list of accounts excluded from organization Config rule.</p>
@@ -49,14 +49,14 @@
         self.last_update_time.as_ref()
     }
     /// <p>An object that specifies metadata for your organization's Config Custom Policy rule. The metadata includes the runtime system in use, which accounts have debug logging enabled, and other custom rule metadata, such as resource type, resource ID of Amazon Web Services resource, and organization trigger types that initiate Config to evaluate Amazon Web Services resources against a rule.</p>
-    pub fn organization_custom_policy_rule_metadata(&self) -> ::std::option::Option<&crate::types::OrganizationCustomPolicyRuleMetadataNoPolicy> {
+    pub fn organization_custom_policy_rule_metadata(&self) -> ::std::option::Option<&super::super::types::OrganizationCustomPolicyRuleMetadataNoPolicy> {
         self.organization_custom_policy_rule_metadata.as_ref()
     }
 }
 impl OrganizationConfigRule {
     /// Creates a new builder-style object to manufacture [`OrganizationConfigRule`](crate::types::OrganizationConfigRule).
-    pub fn builder() -> crate::types::builders::OrganizationConfigRuleBuilder {
-        crate::types::builders::OrganizationConfigRuleBuilder::default()
+    pub fn builder() -> super::super::types::builders::OrganizationConfigRuleBuilder {
+        super::super::types::builders::OrganizationConfigRuleBuilder::default()
     }
 }

@@ -66,11 +66,11 @@
 pub struct OrganizationConfigRuleBuilder {
     pub(crate) organization_config_rule_name: ::std::option::Option<::std::string::String>,
     pub(crate) organization_config_rule_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) organization_managed_rule_metadata: ::std::option::Option<crate::types::OrganizationManagedRuleMetadata>,
-    pub(crate) organization_custom_rule_metadata: ::std::option::Option<crate::types::OrganizationCustomRuleMetadata>,
+    pub(crate) organization_managed_rule_metadata: ::std::option::Option<super::super::types::OrganizationManagedRuleMetadata>,
+    pub(crate) organization_custom_rule_metadata: ::std::option::Option<super::super::types::OrganizationCustomRuleMetadata>,
     pub(crate) excluded_accounts: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) organization_custom_policy_rule_metadata: ::std::option::Option<crate::types::OrganizationCustomPolicyRuleMetadataNoPolicy>,
+    pub(crate) organization_custom_policy_rule_metadata: ::std::option::Option<super::super::types::OrganizationCustomPolicyRuleMetadataNoPolicy>,
 }
 impl OrganizationConfigRuleBuilder {
     /// <p>The name that you assign to organization Config rule.</p>
@@ -104,31 +104,31 @@
         &self.organization_config_rule_arn
     }
     /// <p>An <code>OrganizationManagedRuleMetadata</code> object.</p>
-    pub fn organization_managed_rule_metadata(mut self, input: crate::types::OrganizationManagedRuleMetadata) -> Self {
+    pub fn organization_managed_rule_metadata(mut self, input: super::super::types::OrganizationManagedRuleMetadata) -> Self {
         self.organization_managed_rule_metadata = ::std::option::Option::Some(input);
         self
     }
     /// <p>An <code>OrganizationManagedRuleMetadata</code> object.</p>
-    pub fn set_organization_managed_rule_metadata(mut self, input: ::std::option::Option<crate::types::OrganizationManagedRuleMetadata>) -> Self {
+    pub fn set_organization_managed_rule_metadata(mut self, input: ::std::option::Option<super::super::types::OrganizationManagedRuleMetadata>) -> Self {
         self.organization_managed_rule_metadata = input;
         self
     }
     /// <p>An <code>OrganizationManagedRuleMetadata</code> object.</p>
-    pub fn get_organization_managed_rule_metadata(&self) -> &::std::option::Option<crate::types::OrganizationManagedRuleMetadata> {
+    pub fn get_organization_managed_rule_metadata(&self) -> &::std::option::Option<super::super::types::OrganizationManagedRuleMetadata> {
         &self.organization_managed_rule_metadata
     }
     /// <p>An <code>OrganizationCustomRuleMetadata</code> object.</p>
-    pub fn organization_custom_rule_metadata(mut self, input: crate::types::OrganizationCustomRuleMetadata) -> Self {
+    pub fn organization_custom_rule_metadata(mut self, input: super::super::types::OrganizationCustomRuleMetadata) -> Self {
         self.organization_custom_rule_metadata = ::std::option::Option::Some(input);
         self
     }
     /// <p>An <code>OrganizationCustomRuleMetadata</code> object.</p>
-    pub fn set_organization_custom_rule_metadata(mut self, input: ::std::option::Option<crate::types::OrganizationCustomRuleMetadata>) -> Self {
+    pub fn set_organization_custom_rule_metadata(mut self, input: ::std::option::Option<super::super::types::OrganizationCustomRuleMetadata>) -> Self {
         self.organization_custom_rule_metadata = input;
         self
     }
     /// <p>An <code>OrganizationCustomRuleMetadata</code> object.</p>
-    pub fn get_organization_custom_rule_metadata(&self) -> &::std::option::Option<crate::types::OrganizationCustomRuleMetadata> {
+    pub fn get_organization_custom_rule_metadata(&self) -> &::std::option::Option<super::super::types::OrganizationCustomRuleMetadata> {
         &self.organization_custom_rule_metadata
     }
     /// Appends an item to `excluded_accounts`.
@@ -166,7 +166,7 @@
         &self.last_update_time
     }
     /// <p>An object that specifies metadata for your organization's Config Custom Policy rule. The metadata includes the runtime system in use, which accounts have debug logging enabled, and other custom rule metadata, such as resource type, resource ID of Amazon Web Services resource, and organization trigger types that initiate Config to evaluate Amazon Web Services resources against a rule.</p>
-    pub fn organization_custom_policy_rule_metadata(mut self, input: crate::types::OrganizationCustomPolicyRuleMetadataNoPolicy) -> Self {
+    pub fn organization_custom_policy_rule_metadata(mut self, input: super::super::types::OrganizationCustomPolicyRuleMetadataNoPolicy) -> Self {
         self.organization_custom_policy_rule_metadata = ::std::option::Option::Some(input);
         self
     }
@@ -173,13 +173,13 @@
     /// <p>An object that specifies metadata for your organization's Config Custom Policy rule. The metadata includes the runtime system in use, which accounts have debug logging enabled, and other custom rule metadata, such as resource type, resource ID of Amazon Web Services resource, and organization trigger types that initiate Config to evaluate Amazon Web Services resources against a rule.</p>
     pub fn set_organization_custom_policy_rule_metadata(
         mut self,
-        input: ::std::option::Option<crate::types::OrganizationCustomPolicyRuleMetadataNoPolicy>,
+        input: ::std::option::Option<super::super::types::OrganizationCustomPolicyRuleMetadataNoPolicy>,
     ) -> Self {
         self.organization_custom_policy_rule_metadata = input;
         self
     }
     /// <p>An object that specifies metadata for your organization's Config Custom Policy rule. The metadata includes the runtime system in use, which accounts have debug logging enabled, and other custom rule metadata, such as resource type, resource ID of Amazon Web Services resource, and organization trigger types that initiate Config to evaluate Amazon Web Services resources against a rule.</p>
-    pub fn get_organization_custom_policy_rule_metadata(&self) -> &::std::option::Option<crate::types::OrganizationCustomPolicyRuleMetadataNoPolicy> {
+    pub fn get_organization_custom_policy_rule_metadata(&self) -> &::std::option::Option<super::super::types::OrganizationCustomPolicyRuleMetadataNoPolicy> {
         &self.organization_custom_policy_rule_metadata
     }
     /// Consumes the builder and constructs a [`OrganizationConfigRule`](crate::types::OrganizationConfigRule).
@@ -186,8 +186,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`organization_config_rule_name`](crate::types::builders::OrganizationConfigRuleBuilder::organization_config_rule_name)
     /// - [`organization_config_rule_arn`](crate::types::builders::OrganizationConfigRuleBuilder::organization_config_rule_arn)
-    pub fn build(self) -> ::std::result::Result<crate::types::OrganizationConfigRule, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::OrganizationConfigRule {
+    pub fn build(self) -> ::std::result::Result<super::super::types::OrganizationConfigRule, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::OrganizationConfigRule {
             organization_config_rule_name: self.organization_config_rule_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "organization_config_rule_name",
