--- original
+++ modified
@@ -30,7 +30,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub status: crate::types::OrganizationResourceDetailedStatus,
+    pub status: super::super::types::OrganizationResourceDetailedStatus,
     /// <p>An error code that is returned when conformance pack creation or deletion failed in the member account.</p>
     pub error_code: ::std::option::Option<::std::string::String>,
     /// <p>An error message indicating that conformance pack account creation or deletion has failed due to an error in the member account.</p>
@@ -71,7 +71,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn status(&self) -> &crate::types::OrganizationResourceDetailedStatus {
+    pub fn status(&self) -> &super::super::types::OrganizationResourceDetailedStatus {
         &self.status
     }
     /// <p>An error code that is returned when conformance pack creation or deletion failed in the member account.</p>
@@ -89,8 +89,8 @@
 }
 impl OrganizationConformancePackDetailedStatus {
     /// Creates a new builder-style object to manufacture [`OrganizationConformancePackDetailedStatus`](crate::types::OrganizationConformancePackDetailedStatus).
-    pub fn builder() -> crate::types::builders::OrganizationConformancePackDetailedStatusBuilder {
-        crate::types::builders::OrganizationConformancePackDetailedStatusBuilder::default()
+    pub fn builder() -> super::super::types::builders::OrganizationConformancePackDetailedStatusBuilder {
+        super::super::types::builders::OrganizationConformancePackDetailedStatusBuilder::default()
     }
 }

@@ -100,7 +100,7 @@
 pub struct OrganizationConformancePackDetailedStatusBuilder {
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
     pub(crate) conformance_pack_name: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::OrganizationResourceDetailedStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::OrganizationResourceDetailedStatus>,
     pub(crate) error_code: ::std::option::Option<::std::string::String>,
     pub(crate) error_message: ::std::option::Option<::std::string::String>,
     pub(crate) last_update_time: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -159,7 +159,7 @@
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn status(mut self, input: crate::types::OrganizationResourceDetailedStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::OrganizationResourceDetailedStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
@@ -185,7 +185,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::OrganizationResourceDetailedStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::OrganizationResourceDetailedStatus>) -> Self {
         self.status = input;
         self
     }
@@ -211,7 +211,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when conformance pack deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::OrganizationResourceDetailedStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::OrganizationResourceDetailedStatus> {
         &self.status
     }
     /// <p>An error code that is returned when conformance pack creation or deletion failed in the member account.</p>
@@ -263,8 +263,8 @@
     /// - [`status`](crate::types::builders::OrganizationConformancePackDetailedStatusBuilder::status)
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::types::OrganizationConformancePackDetailedStatus, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::OrganizationConformancePackDetailedStatus {
+    ) -> ::std::result::Result<super::super::types::OrganizationConformancePackDetailedStatus, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::OrganizationConformancePackDetailedStatus {
             account_id: self.account_id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "account_id",
