--- original
+++ modified
@@ -17,13 +17,13 @@
     /// <li>
     /// <p><code>ScheduledNotification</code> - Triggers a periodic evaluation at the frequency specified for <code>MaximumExecutionFrequency</code>.</p></li>
     /// </ul>
-    pub organization_config_rule_trigger_types: ::std::vec::Vec<crate::types::OrganizationConfigRuleTriggerType>,
+    pub organization_config_rule_trigger_types: ::std::vec::Vec<super::types::OrganizationConfigRuleTriggerType>,
     /// <p>A string, in JSON format, that is passed to your organization Config rule Lambda function.</p>
     pub input_parameters: ::std::option::Option<::std::string::String>,
     /// <p>The maximum frequency with which Config runs evaluations for a rule. Your custom rule is triggered when Config delivers the configuration snapshot. For more information, see <code>ConfigSnapshotDeliveryProperties</code>.</p><note>
     /// <p>By default, rules with a periodic trigger are evaluated every 24 hours. To change the frequency, specify a valid value for the <code>MaximumExecutionFrequency</code> parameter.</p>
     /// </note>
-    pub maximum_execution_frequency: ::std::option::Option<crate::types::MaximumExecutionFrequency>,
+    pub maximum_execution_frequency: ::std::option::Option<super::types::MaximumExecutionFrequency>,
     /// <p>The type of the Amazon Web Services resource that was evaluated.</p>
     pub resource_types_scope: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>The ID of the Amazon Web Services resource that was evaluated.</p>
@@ -52,7 +52,7 @@
     /// <li>
     /// <p><code>ScheduledNotification</code> - Triggers a periodic evaluation at the frequency specified for <code>MaximumExecutionFrequency</code>.</p></li>
     /// </ul>
-    pub fn organization_config_rule_trigger_types(&self) -> &[crate::types::OrganizationConfigRuleTriggerType] {
+    pub fn organization_config_rule_trigger_types(&self) -> &[super::types::OrganizationConfigRuleTriggerType] {
         use std::ops::Deref;
         self.organization_config_rule_trigger_types.deref()
     }
@@ -63,7 +63,7 @@
     /// <p>The maximum frequency with which Config runs evaluations for a rule. Your custom rule is triggered when Config delivers the configuration snapshot. For more information, see <code>ConfigSnapshotDeliveryProperties</code>.</p><note>
     /// <p>By default, rules with a periodic trigger are evaluated every 24 hours. To change the frequency, specify a valid value for the <code>MaximumExecutionFrequency</code> parameter.</p>
     /// </note>
-    pub fn maximum_execution_frequency(&self) -> ::std::option::Option<&crate::types::MaximumExecutionFrequency> {
+    pub fn maximum_execution_frequency(&self) -> ::std::option::Option<&super::types::MaximumExecutionFrequency> {
         self.maximum_execution_frequency.as_ref()
     }
     /// <p>The type of the Amazon Web Services resource that was evaluated.</p>
@@ -87,8 +87,8 @@
 }
 impl OrganizationCustomRuleMetadata {
     /// Creates a new builder-style object to manufacture [`OrganizationCustomRuleMetadata`](crate::types::OrganizationCustomRuleMetadata).
-    pub fn builder() -> crate::types::builders::OrganizationCustomRuleMetadataBuilder {
-        crate::types::builders::OrganizationCustomRuleMetadataBuilder::default()
+    pub fn builder() -> super::types::builders::OrganizationCustomRuleMetadataBuilder {
+        super::types::builders::OrganizationCustomRuleMetadataBuilder::default()
     }
 }

@@ -98,9 +98,9 @@
 pub struct OrganizationCustomRuleMetadataBuilder {
     pub(crate) description: ::std::option::Option<::std::string::String>,
     pub(crate) lambda_function_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) organization_config_rule_trigger_types: ::std::option::Option<::std::vec::Vec<crate::types::OrganizationConfigRuleTriggerType>>,
+    pub(crate) organization_config_rule_trigger_types: ::std::option::Option<::std::vec::Vec<super::types::OrganizationConfigRuleTriggerType>>,
     pub(crate) input_parameters: ::std::option::Option<::std::string::String>,
-    pub(crate) maximum_execution_frequency: ::std::option::Option<crate::types::MaximumExecutionFrequency>,
+    pub(crate) maximum_execution_frequency: ::std::option::Option<super::types::MaximumExecutionFrequency>,
     pub(crate) resource_types_scope: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) resource_id_scope: ::std::option::Option<::std::string::String>,
     pub(crate) tag_key_scope: ::std::option::Option<::std::string::String>,
@@ -149,7 +149,7 @@
     /// <li>
     /// <p><code>ScheduledNotification</code> - Triggers a periodic evaluation at the frequency specified for <code>MaximumExecutionFrequency</code>.</p></li>
     /// </ul>
-    pub fn organization_config_rule_trigger_types(mut self, input: crate::types::OrganizationConfigRuleTriggerType) -> Self {
+    pub fn organization_config_rule_trigger_types(mut self, input: super::types::OrganizationConfigRuleTriggerType) -> Self {
         let mut v = self.organization_config_rule_trigger_types.unwrap_or_default();
         v.push(input);
         self.organization_config_rule_trigger_types = ::std::option::Option::Some(v);
@@ -166,7 +166,7 @@
     /// </ul>
     pub fn set_organization_config_rule_trigger_types(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::OrganizationConfigRuleTriggerType>>,
+        input: ::std::option::Option<::std::vec::Vec<super::types::OrganizationConfigRuleTriggerType>>,
     ) -> Self {
         self.organization_config_rule_trigger_types = input;
         self
@@ -182,7 +182,7 @@
     /// </ul>
     pub fn get_organization_config_rule_trigger_types(
         &self,
-    ) -> &::std::option::Option<::std::vec::Vec<crate::types::OrganizationConfigRuleTriggerType>> {
+    ) -> &::std::option::Option<::std::vec::Vec<super::types::OrganizationConfigRuleTriggerType>> {
         &self.organization_config_rule_trigger_types
     }
     /// <p>A string, in JSON format, that is passed to your organization Config rule Lambda function.</p>
@@ -202,7 +202,7 @@
     /// <p>The maximum frequency with which Config runs evaluations for a rule. Your custom rule is triggered when Config delivers the configuration snapshot. For more information, see <code>ConfigSnapshotDeliveryProperties</code>.</p><note>
     /// <p>By default, rules with a periodic trigger are evaluated every 24 hours. To change the frequency, specify a valid value for the <code>MaximumExecutionFrequency</code> parameter.</p>
     /// </note>
-    pub fn maximum_execution_frequency(mut self, input: crate::types::MaximumExecutionFrequency) -> Self {
+    pub fn maximum_execution_frequency(mut self, input: super::types::MaximumExecutionFrequency) -> Self {
         self.maximum_execution_frequency = ::std::option::Option::Some(input);
         self
     }
@@ -209,7 +209,7 @@
     /// <p>The maximum frequency with which Config runs evaluations for a rule. Your custom rule is triggered when Config delivers the configuration snapshot. For more information, see <code>ConfigSnapshotDeliveryProperties</code>.</p><note>
     /// <p>By default, rules with a periodic trigger are evaluated every 24 hours. To change the frequency, specify a valid value for the <code>MaximumExecutionFrequency</code> parameter.</p>
     /// </note>
-    pub fn set_maximum_execution_frequency(mut self, input: ::std::option::Option<crate::types::MaximumExecutionFrequency>) -> Self {
+    pub fn set_maximum_execution_frequency(mut self, input: ::std::option::Option<super::types::MaximumExecutionFrequency>) -> Self {
         self.maximum_execution_frequency = input;
         self
     }
@@ -216,7 +216,7 @@
     /// <p>The maximum frequency with which Config runs evaluations for a rule. Your custom rule is triggered when Config delivers the configuration snapshot. For more information, see <code>ConfigSnapshotDeliveryProperties</code>.</p><note>
     /// <p>By default, rules with a periodic trigger are evaluated every 24 hours. To change the frequency, specify a valid value for the <code>MaximumExecutionFrequency</code> parameter.</p>
     /// </note>
-    pub fn get_maximum_execution_frequency(&self) -> &::std::option::Option<crate::types::MaximumExecutionFrequency> {
+    pub fn get_maximum_execution_frequency(&self) -> &::std::option::Option<super::types::MaximumExecutionFrequency> {
         &self.maximum_execution_frequency
     }
     /// Appends an item to `resource_types_scope`.
@@ -285,8 +285,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`lambda_function_arn`](crate::types::builders::OrganizationCustomRuleMetadataBuilder::lambda_function_arn)
     /// - [`organization_config_rule_trigger_types`](crate::types::builders::OrganizationCustomRuleMetadataBuilder::organization_config_rule_trigger_types)
-    pub fn build(self) -> ::std::result::Result<crate::types::OrganizationCustomRuleMetadata, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::OrganizationCustomRuleMetadata {
+    pub fn build(self) -> ::std::result::Result<super::types::OrganizationCustomRuleMetadata, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::OrganizationCustomRuleMetadata {
             description: self.description,
             lambda_function_arn: self.lambda_function_arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
