--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl PendingAggregationRequest {
     /// Creates a new builder-style object to manufacture [`PendingAggregationRequest`](crate::types::PendingAggregationRequest).
-    pub fn builder() -> crate::types::builders::PendingAggregationRequestBuilder {
-        crate::types::builders::PendingAggregationRequestBuilder::default()
+    pub fn builder() -> super::super::types::builders::PendingAggregationRequestBuilder {
+        super::super::types::builders::PendingAggregationRequestBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.requester_aws_region
     }
     /// Consumes the builder and constructs a [`PendingAggregationRequest`](crate::types::PendingAggregationRequest).
-    pub fn build(self) -> crate::types::PendingAggregationRequest {
-        crate::types::PendingAggregationRequest {
+    pub fn build(self) -> super::super::types::PendingAggregationRequest {
+        super::super::types::PendingAggregationRequest {
             requester_account_id: self.requester_account_id,
             requester_aws_region: self.requester_aws_region,
         }
