--- original
+++ modified
@@ -43,7 +43,7 @@
     /// <p>Middle East (UAE)</p></li>
     /// </ul>
     /// </note>
-    pub use_only: ::std::option::Option<crate::types::RecordingStrategyType>,
+    pub use_only: ::std::option::Option<super::types::RecordingStrategyType>,
 }
 impl RecordingStrategy {
     /// <p>The recording strategy for the configuration recorder.</p>
@@ -85,14 +85,14 @@
     /// <p>Middle East (UAE)</p></li>
     /// </ul>
     /// </note>
-    pub fn use_only(&self) -> ::std::option::Option<&crate::types::RecordingStrategyType> {
+    pub fn use_only(&self) -> ::std::option::Option<&super::types::RecordingStrategyType> {
         self.use_only.as_ref()
     }
 }
 impl RecordingStrategy {
     /// Creates a new builder-style object to manufacture [`RecordingStrategy`](crate::types::RecordingStrategy).
-    pub fn builder() -> crate::types::builders::RecordingStrategyBuilder {
-        crate::types::builders::RecordingStrategyBuilder::default()
+    pub fn builder() -> super::types::builders::RecordingStrategyBuilder {
+        super::types::builders::RecordingStrategyBuilder::default()
     }
 }

@@ -100,7 +100,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct RecordingStrategyBuilder {
-    pub(crate) use_only: ::std::option::Option<crate::types::RecordingStrategyType>,
+    pub(crate) use_only: ::std::option::Option<super::types::RecordingStrategyType>,
 }
 impl RecordingStrategyBuilder {
     /// <p>The recording strategy for the configuration recorder.</p>
@@ -142,7 +142,7 @@
     /// <p>Middle East (UAE)</p></li>
     /// </ul>
     /// </note>
-    pub fn use_only(mut self, input: crate::types::RecordingStrategyType) -> Self {
+    pub fn use_only(mut self, input: super::types::RecordingStrategyType) -> Self {
         self.use_only = ::std::option::Option::Some(input);
         self
     }
@@ -185,7 +185,7 @@
     /// <p>Middle East (UAE)</p></li>
     /// </ul>
     /// </note>
-    pub fn set_use_only(mut self, input: ::std::option::Option<crate::types::RecordingStrategyType>) -> Self {
+    pub fn set_use_only(mut self, input: ::std::option::Option<super::types::RecordingStrategyType>) -> Self {
         self.use_only = input;
         self
     }
@@ -228,11 +228,11 @@
     /// <p>Middle East (UAE)</p></li>
     /// </ul>
     /// </note>
-    pub fn get_use_only(&self) -> &::std::option::Option<crate::types::RecordingStrategyType> {
+    pub fn get_use_only(&self) -> &::std::option::Option<super::types::RecordingStrategyType> {
         &self.use_only
     }
     /// Consumes the builder and constructs a [`RecordingStrategy`](crate::types::RecordingStrategy).
-    pub fn build(self) -> crate::types::RecordingStrategy {
-        crate::types::RecordingStrategy { use_only: self.use_only }
+    pub fn build(self) -> super::types::RecordingStrategy {
+        super::types::RecordingStrategy { use_only: self.use_only }
     }
 }
