--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The details of the step.</p>
     pub name: ::std::option::Option<::std::string::String>,
     /// <p>The valid status of the step.</p>
-    pub state: ::std::option::Option<crate::types::RemediationExecutionStepState>,
+    pub state: ::std::option::Option<super::types::RemediationExecutionStepState>,
     /// <p>An error message if the step was interrupted during execution.</p>
     pub error_message: ::std::option::Option<::std::string::String>,
     /// <p>The time when the step started.</p>
@@ -21,7 +21,7 @@
         self.name.as_deref()
     }
     /// <p>The valid status of the step.</p>
-    pub fn state(&self) -> ::std::option::Option<&crate::types::RemediationExecutionStepState> {
+    pub fn state(&self) -> ::std::option::Option<&super::types::RemediationExecutionStepState> {
         self.state.as_ref()
     }
     /// <p>An error message if the step was interrupted during execution.</p>
@@ -39,8 +39,8 @@
 }
 impl RemediationExecutionStep {
     /// Creates a new builder-style object to manufacture [`RemediationExecutionStep`](crate::types::RemediationExecutionStep).
-    pub fn builder() -> crate::types::builders::RemediationExecutionStepBuilder {
-        crate::types::builders::RemediationExecutionStepBuilder::default()
+    pub fn builder() -> super::types::builders::RemediationExecutionStepBuilder {
+        super::types::builders::RemediationExecutionStepBuilder::default()
     }
 }

@@ -49,7 +49,7 @@
 #[non_exhaustive]
 pub struct RemediationExecutionStepBuilder {
     pub(crate) name: ::std::option::Option<::std::string::String>,
-    pub(crate) state: ::std::option::Option<crate::types::RemediationExecutionStepState>,
+    pub(crate) state: ::std::option::Option<super::types::RemediationExecutionStepState>,
     pub(crate) error_message: ::std::option::Option<::std::string::String>,
     pub(crate) start_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) stop_time: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -70,17 +70,17 @@
         &self.name
     }
     /// <p>The valid status of the step.</p>
-    pub fn state(mut self, input: crate::types::RemediationExecutionStepState) -> Self {
+    pub fn state(mut self, input: super::types::RemediationExecutionStepState) -> Self {
         self.state = ::std::option::Option::Some(input);
         self
     }
     /// <p>The valid status of the step.</p>
-    pub fn set_state(mut self, input: ::std::option::Option<crate::types::RemediationExecutionStepState>) -> Self {
+    pub fn set_state(mut self, input: ::std::option::Option<super::types::RemediationExecutionStepState>) -> Self {
         self.state = input;
         self
     }
     /// <p>The valid status of the step.</p>
-    pub fn get_state(&self) -> &::std::option::Option<crate::types::RemediationExecutionStepState> {
+    pub fn get_state(&self) -> &::std::option::Option<super::types::RemediationExecutionStepState> {
         &self.state
     }
     /// <p>An error message if the step was interrupted during execution.</p>
@@ -126,8 +126,8 @@
         &self.stop_time
     }
     /// Consumes the builder and constructs a [`RemediationExecutionStep`](crate::types::RemediationExecutionStep).
-    pub fn build(self) -> crate::types::RemediationExecutionStep {
-        crate::types::RemediationExecutionStep {
+    pub fn build(self) -> super::types::RemediationExecutionStep {
+        super::types::RemediationExecutionStep {
             name: self.name,
             state: self.state,
             error_message: self.error_message,
