--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the Config rule.</p>
     pub config_rule_name: ::std::string::String,
     /// <p>The type of the target. Target executes remediation. For example, SSM document.</p>
-    pub target_type: crate::types::RemediationTargetType,
+    pub target_type: super::types::RemediationTargetType,
     /// <p>Target ID is the name of the SSM document.</p>
     pub target_id: ::std::string::String,
     /// <p>Version of the target. For example, version of the SSM document.</p><note>
@@ -15,13 +15,13 @@
     /// </note>
     pub target_version: ::std::option::Option<::std::string::String>,
     /// <p>An object of the RemediationParameterValue.</p>
-    pub parameters: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::RemediationParameterValue>>,
+    pub parameters: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::RemediationParameterValue>>,
     /// <p>The type of a resource.</p>
     pub resource_type: ::std::option::Option<::std::string::String>,
     /// <p>The remediation is triggered automatically.</p>
     pub automatic: bool,
     /// <p>An ExecutionControls object.</p>
-    pub execution_controls: ::std::option::Option<crate::types::ExecutionControls>,
+    pub execution_controls: ::std::option::Option<super::types::ExecutionControls>,
     /// <p>The maximum number of failed attempts for auto-remediation. If you do not select a number, the default is 5.</p>
     /// <p>For example, if you specify MaximumAutomaticAttempts as 5 with RetryAttemptSeconds as 50 seconds, Config will put a RemediationException on your behalf for the failing resource after the 5th failed attempt within 50 seconds.</p>
     pub maximum_automatic_attempts: ::std::option::Option<i32>,
@@ -40,7 +40,7 @@
         self.config_rule_name.deref()
     }
     /// <p>The type of the target. Target executes remediation. For example, SSM document.</p>
-    pub fn target_type(&self) -> &crate::types::RemediationTargetType {
+    pub fn target_type(&self) -> &super::types::RemediationTargetType {
         &self.target_type
     }
     /// <p>Target ID is the name of the SSM document.</p>
@@ -55,7 +55,7 @@
         self.target_version.as_deref()
     }
     /// <p>An object of the RemediationParameterValue.</p>
-    pub fn parameters(&self) -> ::std::option::Option<&::std::collections::HashMap<::std::string::String, crate::types::RemediationParameterValue>> {
+    pub fn parameters(&self) -> ::std::option::Option<&::std::collections::HashMap<::std::string::String, super::types::RemediationParameterValue>> {
         self.parameters.as_ref()
     }
     /// <p>The type of a resource.</p>
@@ -67,7 +67,7 @@
         self.automatic
     }
     /// <p>An ExecutionControls object.</p>
-    pub fn execution_controls(&self) -> ::std::option::Option<&crate::types::ExecutionControls> {
+    pub fn execution_controls(&self) -> ::std::option::Option<&super::types::ExecutionControls> {
         self.execution_controls.as_ref()
     }
     /// <p>The maximum number of failed attempts for auto-remediation. If you do not select a number, the default is 5.</p>
@@ -91,8 +91,8 @@
 }
 impl RemediationConfiguration {
     /// Creates a new builder-style object to manufacture [`RemediationConfiguration`](crate::types::RemediationConfiguration).
-    pub fn builder() -> crate::types::builders::RemediationConfigurationBuilder {
-        crate::types::builders::RemediationConfigurationBuilder::default()
+    pub fn builder() -> super::types::builders::RemediationConfigurationBuilder {
+        super::types::builders::RemediationConfigurationBuilder::default()
     }
 }

@@ -101,13 +101,13 @@
 #[non_exhaustive]
 pub struct RemediationConfigurationBuilder {
     pub(crate) config_rule_name: ::std::option::Option<::std::string::String>,
-    pub(crate) target_type: ::std::option::Option<crate::types::RemediationTargetType>,
+    pub(crate) target_type: ::std::option::Option<super::types::RemediationTargetType>,
     pub(crate) target_id: ::std::option::Option<::std::string::String>,
     pub(crate) target_version: ::std::option::Option<::std::string::String>,
-    pub(crate) parameters: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::RemediationParameterValue>>,
+    pub(crate) parameters: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::RemediationParameterValue>>,
     pub(crate) resource_type: ::std::option::Option<::std::string::String>,
     pub(crate) automatic: ::std::option::Option<bool>,
-    pub(crate) execution_controls: ::std::option::Option<crate::types::ExecutionControls>,
+    pub(crate) execution_controls: ::std::option::Option<super::types::ExecutionControls>,
     pub(crate) maximum_automatic_attempts: ::std::option::Option<i32>,
     pub(crate) retry_attempt_seconds: ::std::option::Option<i64>,
     pub(crate) arn: ::std::option::Option<::std::string::String>,
@@ -131,17 +131,17 @@
     }
     /// <p>The type of the target. Target executes remediation. For example, SSM document.</p>
     /// This field is required.
-    pub fn target_type(mut self, input: crate::types::RemediationTargetType) -> Self {
+    pub fn target_type(mut self, input: super::types::RemediationTargetType) -> Self {
         self.target_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of the target. Target executes remediation. For example, SSM document.</p>
-    pub fn set_target_type(mut self, input: ::std::option::Option<crate::types::RemediationTargetType>) -> Self {
+    pub fn set_target_type(mut self, input: ::std::option::Option<super::types::RemediationTargetType>) -> Self {
         self.target_type = input;
         self
     }
     /// <p>The type of the target. Target executes remediation. For example, SSM document.</p>
-    pub fn get_target_type(&self) -> &::std::option::Option<crate::types::RemediationTargetType> {
+    pub fn get_target_type(&self) -> &::std::option::Option<super::types::RemediationTargetType> {
         &self.target_type
     }
     /// <p>Target ID is the name of the SSM document.</p>
@@ -184,7 +184,7 @@
     /// To override the contents of this collection use [`set_parameters`](Self::set_parameters).
     ///
     /// <p>An object of the RemediationParameterValue.</p>
-    pub fn parameters(mut self, k: impl ::std::convert::Into<::std::string::String>, v: crate::types::RemediationParameterValue) -> Self {
+    pub fn parameters(mut self, k: impl ::std::convert::Into<::std::string::String>, v: super::types::RemediationParameterValue) -> Self {
         let mut hash_map = self.parameters.unwrap_or_default();
         hash_map.insert(k.into(), v);
         self.parameters = ::std::option::Option::Some(hash_map);
@@ -193,7 +193,7 @@
     /// <p>An object of the RemediationParameterValue.</p>
     pub fn set_parameters(
         mut self,
-        input: ::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::RemediationParameterValue>>,
+        input: ::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::RemediationParameterValue>>,
     ) -> Self {
         self.parameters = input;
         self
@@ -201,7 +201,7 @@
     /// <p>An object of the RemediationParameterValue.</p>
     pub fn get_parameters(
         &self,
-    ) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, crate::types::RemediationParameterValue>> {
+    ) -> &::std::option::Option<::std::collections::HashMap<::std::string::String, super::types::RemediationParameterValue>> {
         &self.parameters
     }
     /// <p>The type of a resource.</p>
@@ -233,17 +233,17 @@
         &self.automatic
     }
     /// <p>An ExecutionControls object.</p>
-    pub fn execution_controls(mut self, input: crate::types::ExecutionControls) -> Self {
+    pub fn execution_controls(mut self, input: super::types::ExecutionControls) -> Self {
         self.execution_controls = ::std::option::Option::Some(input);
         self
     }
     /// <p>An ExecutionControls object.</p>
-    pub fn set_execution_controls(mut self, input: ::std::option::Option<crate::types::ExecutionControls>) -> Self {
+    pub fn set_execution_controls(mut self, input: ::std::option::Option<super::types::ExecutionControls>) -> Self {
         self.execution_controls = input;
         self
     }
     /// <p>An ExecutionControls object.</p>
-    pub fn get_execution_controls(&self) -> &::std::option::Option<crate::types::ExecutionControls> {
+    pub fn get_execution_controls(&self) -> &::std::option::Option<super::types::ExecutionControls> {
         &self.execution_controls
     }
     /// <p>The maximum number of failed attempts for auto-remediation. If you do not select a number, the default is 5.</p>
@@ -313,8 +313,8 @@
     /// - [`config_rule_name`](crate::types::builders::RemediationConfigurationBuilder::config_rule_name)
     /// - [`target_type`](crate::types::builders::RemediationConfigurationBuilder::target_type)
     /// - [`target_id`](crate::types::builders::RemediationConfigurationBuilder::target_id)
-    pub fn build(self) -> ::std::result::Result<crate::types::RemediationConfiguration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::RemediationConfiguration {
+    pub fn build(self) -> ::std::result::Result<super::types::RemediationConfiguration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::RemediationConfiguration {
             config_rule_name: self.config_rule_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "config_rule_name",
