--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl RemediationException {
     /// Creates a new builder-style object to manufacture [`RemediationException`](crate::types::RemediationException).
-    pub fn builder() -> crate::types::builders::RemediationExceptionBuilder {
-        crate::types::builders::RemediationExceptionBuilder::default()
+    pub fn builder() -> super::types::builders::RemediationExceptionBuilder {
+        super::types::builders::RemediationExceptionBuilder::default()
     }
 }

@@ -136,8 +136,8 @@
     /// - [`config_rule_name`](crate::types::builders::RemediationExceptionBuilder::config_rule_name)
     /// - [`resource_type`](crate::types::builders::RemediationExceptionBuilder::resource_type)
     /// - [`resource_id`](crate::types::builders::RemediationExceptionBuilder::resource_id)
-    pub fn build(self) -> ::std::result::Result<crate::types::RemediationException, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::RemediationException {
+    pub fn build(self) -> ::std::result::Result<super::types::RemediationException, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::RemediationException {
             config_rule_name: self.config_rule_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "config_rule_name",
