--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ResourceCount {
     /// <p>The resource type (for example, <code>"AWS::EC2::Instance"</code>).</p>
-    pub resource_type: ::std::option::Option<crate::types::ResourceType>,
+    pub resource_type: ::std::option::Option<super::types::ResourceType>,
     /// <p>The number of resources.</p>
     pub count: i64,
 }
 impl ResourceCount {
     /// <p>The resource type (for example, <code>"AWS::EC2::Instance"</code>).</p>
-    pub fn resource_type(&self) -> ::std::option::Option<&crate::types::ResourceType> {
+    pub fn resource_type(&self) -> ::std::option::Option<&super::types::ResourceType> {
         self.resource_type.as_ref()
     }
     /// <p>The number of resources.</p>
@@ -21,8 +21,8 @@
 }
 impl ResourceCount {
     /// Creates a new builder-style object to manufacture [`ResourceCount`](crate::types::ResourceCount).
-    pub fn builder() -> crate::types::builders::ResourceCountBuilder {
-        crate::types::builders::ResourceCountBuilder::default()
+    pub fn builder() -> super::types::builders::ResourceCountBuilder {
+        super::types::builders::ResourceCountBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ResourceCountBuilder {
-    pub(crate) resource_type: ::std::option::Option<crate::types::ResourceType>,
+    pub(crate) resource_type: ::std::option::Option<super::types::ResourceType>,
     pub(crate) count: ::std::option::Option<i64>,
 }
 impl ResourceCountBuilder {
     /// <p>The resource type (for example, <code>"AWS::EC2::Instance"</code>).</p>
-    pub fn resource_type(mut self, input: crate::types::ResourceType) -> Self {
+    pub fn resource_type(mut self, input: super::types::ResourceType) -> Self {
         self.resource_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The resource type (for example, <code>"AWS::EC2::Instance"</code>).</p>
-    pub fn set_resource_type(mut self, input: ::std::option::Option<crate::types::ResourceType>) -> Self {
+    pub fn set_resource_type(mut self, input: ::std::option::Option<super::types::ResourceType>) -> Self {
         self.resource_type = input;
         self
     }
     /// <p>The resource type (for example, <code>"AWS::EC2::Instance"</code>).</p>
-    pub fn get_resource_type(&self) -> &::std::option::Option<crate::types::ResourceType> {
+    pub fn get_resource_type(&self) -> &::std::option::Option<super::types::ResourceType> {
         &self.resource_type
     }
     /// <p>The number of resources.</p>
@@ -63,8 +63,8 @@
         &self.count
     }
     /// Consumes the builder and constructs a [`ResourceCount`](crate::types::ResourceCount).
-    pub fn build(self) -> crate::types::ResourceCount {
-        crate::types::ResourceCount {
+    pub fn build(self) -> super::types::ResourceCount {
+        super::types::ResourceCount {
             resource_type: self.resource_type,
             count: self.count.unwrap_or_default(),
         }
