--- original
+++ modified
@@ -33,8 +33,8 @@
 }
 impl ResourceFilters {
     /// Creates a new builder-style object to manufacture [`ResourceFilters`](crate::types::ResourceFilters).
-    pub fn builder() -> crate::types::builders::ResourceFiltersBuilder {
-        crate::types::builders::ResourceFiltersBuilder::default()
+    pub fn builder() -> super::super::types::builders::ResourceFiltersBuilder {
+        super::super::types::builders::ResourceFiltersBuilder::default()
     }
 }

@@ -105,8 +105,8 @@
         &self.region
     }
     /// Consumes the builder and constructs a [`ResourceFilters`](crate::types::ResourceFilters).
-    pub fn build(self) -> crate::types::ResourceFilters {
-        crate::types::ResourceFilters {
+    pub fn build(self) -> super::super::types::ResourceFilters {
+        super::super::types::ResourceFilters {
             account_id: self.account_id,
             resource_id: self.resource_id,
             resource_name: self.resource_name,
