--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ResourceCountFilters {
     /// <p>The type of the Amazon Web Services resource.</p>
-    pub resource_type: ::std::option::Option<crate::types::ResourceType>,
+    pub resource_type: ::std::option::Option<super::super::types::ResourceType>,
     /// <p>The 12-digit ID of the account.</p>
     pub account_id: ::std::option::Option<::std::string::String>,
     /// <p>The region where the account is located.</p>
@@ -13,7 +13,7 @@
 }
 impl ResourceCountFilters {
     /// <p>The type of the Amazon Web Services resource.</p>
-    pub fn resource_type(&self) -> ::std::option::Option<&crate::types::ResourceType> {
+    pub fn resource_type(&self) -> ::std::option::Option<&super::super::types::ResourceType> {
         self.resource_type.as_ref()
     }
     /// <p>The 12-digit ID of the account.</p>
@@ -27,8 +27,8 @@
 }
 impl ResourceCountFilters {
     /// Creates a new builder-style object to manufacture [`ResourceCountFilters`](crate::types::ResourceCountFilters).
-    pub fn builder() -> crate::types::builders::ResourceCountFiltersBuilder {
-        crate::types::builders::ResourceCountFiltersBuilder::default()
+    pub fn builder() -> super::super::types::builders::ResourceCountFiltersBuilder {
+        super::super::types::builders::ResourceCountFiltersBuilder::default()
     }
 }

@@ -36,23 +36,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ResourceCountFiltersBuilder {
-    pub(crate) resource_type: ::std::option::Option<crate::types::ResourceType>,
+    pub(crate) resource_type: ::std::option::Option<super::super::types::ResourceType>,
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
     pub(crate) region: ::std::option::Option<::std::string::String>,
 }
 impl ResourceCountFiltersBuilder {
     /// <p>The type of the Amazon Web Services resource.</p>
-    pub fn resource_type(mut self, input: crate::types::ResourceType) -> Self {
+    pub fn resource_type(mut self, input: super::super::types::ResourceType) -> Self {
         self.resource_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of the Amazon Web Services resource.</p>
-    pub fn set_resource_type(mut self, input: ::std::option::Option<crate::types::ResourceType>) -> Self {
+    pub fn set_resource_type(mut self, input: ::std::option::Option<super::super::types::ResourceType>) -> Self {
         self.resource_type = input;
         self
     }
     /// <p>The type of the Amazon Web Services resource.</p>
-    pub fn get_resource_type(&self) -> &::std::option::Option<crate::types::ResourceType> {
+    pub fn get_resource_type(&self) -> &::std::option::Option<super::super::types::ResourceType> {
         &self.resource_type
     }
     /// <p>The 12-digit ID of the account.</p>
@@ -84,8 +84,8 @@
         &self.region
     }
     /// Consumes the builder and constructs a [`ResourceCountFilters`](crate::types::ResourceCountFilters).
-    pub fn build(self) -> crate::types::ResourceCountFilters {
-        crate::types::ResourceCountFilters {
+    pub fn build(self) -> super::super::types::ResourceCountFilters {
+        super::super::types::ResourceCountFilters {
             resource_type: self.resource_type,
             account_id: self.account_id,
             region: self.region,
