--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl SsmControls {
     /// Creates a new builder-style object to manufacture [`SsmControls`](crate::types::SsmControls).
-    pub fn builder() -> crate::types::builders::SsmControlsBuilder {
-        crate::types::builders::SsmControlsBuilder::default()
+    pub fn builder() -> super::types::builders::SsmControlsBuilder {
+        super::types::builders::SsmControlsBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.error_percentage
     }
     /// Consumes the builder and constructs a [`SsmControls`](crate::types::SsmControls).
-    pub fn build(self) -> crate::types::SsmControls {
-        crate::types::SsmControls {
+    pub fn build(self) -> super::types::SsmControls {
+        super::types::SsmControls {
             concurrent_execution_rate_percentage: self.concurrent_execution_rate_percentage,
             error_percentage: self.error_percentage,
         }
