--- original
+++ modified
@@ -28,7 +28,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when Config rule deletion has failed in the member account.</p></li>
     /// </ul>
-    pub member_account_rule_status: ::std::option::Option<crate::types::MemberAccountRuleStatus>,
+    pub member_account_rule_status: ::std::option::Option<super::types::MemberAccountRuleStatus>,
 }
 impl StatusDetailFilters {
     /// <p>The 12-digit account ID of the member account within an organization.</p>
@@ -57,14 +57,14 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when Config rule deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn member_account_rule_status(&self) -> ::std::option::Option<&crate::types::MemberAccountRuleStatus> {
+    pub fn member_account_rule_status(&self) -> ::std::option::Option<&super::types::MemberAccountRuleStatus> {
         self.member_account_rule_status.as_ref()
     }
 }
 impl StatusDetailFilters {
     /// Creates a new builder-style object to manufacture [`StatusDetailFilters`](crate::types::StatusDetailFilters).
-    pub fn builder() -> crate::types::builders::StatusDetailFiltersBuilder {
-        crate::types::builders::StatusDetailFiltersBuilder::default()
+    pub fn builder() -> super::types::builders::StatusDetailFiltersBuilder {
+        super::types::builders::StatusDetailFiltersBuilder::default()
     }
 }

@@ -73,7 +73,7 @@
 #[non_exhaustive]
 pub struct StatusDetailFiltersBuilder {
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
-    pub(crate) member_account_rule_status: ::std::option::Option<crate::types::MemberAccountRuleStatus>,
+    pub(crate) member_account_rule_status: ::std::option::Option<super::types::MemberAccountRuleStatus>,
 }
 impl StatusDetailFiltersBuilder {
     /// <p>The 12-digit account ID of the member account within an organization.</p>
@@ -112,7 +112,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when Config rule deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn member_account_rule_status(mut self, input: crate::types::MemberAccountRuleStatus) -> Self {
+    pub fn member_account_rule_status(mut self, input: super::types::MemberAccountRuleStatus) -> Self {
         self.member_account_rule_status = ::std::option::Option::Some(input);
         self
     }
@@ -138,7 +138,7 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when Config rule deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn set_member_account_rule_status(mut self, input: ::std::option::Option<crate::types::MemberAccountRuleStatus>) -> Self {
+    pub fn set_member_account_rule_status(mut self, input: ::std::option::Option<super::types::MemberAccountRuleStatus>) -> Self {
         self.member_account_rule_status = input;
         self
     }
@@ -164,12 +164,12 @@
     /// <li>
     /// <p><code>UPDATE_FAILED</code> when Config rule deletion has failed in the member account.</p></li>
     /// </ul>
-    pub fn get_member_account_rule_status(&self) -> &::std::option::Option<crate::types::MemberAccountRuleStatus> {
+    pub fn get_member_account_rule_status(&self) -> &::std::option::Option<super::types::MemberAccountRuleStatus> {
         &self.member_account_rule_status
     }
     /// Consumes the builder and constructs a [`StatusDetailFilters`](crate::types::StatusDetailFilters).
-    pub fn build(self) -> crate::types::StatusDetailFilters {
-        crate::types::StatusDetailFilters {
+    pub fn build(self) -> super::types::StatusDetailFilters {
+        super::types::StatusDetailFilters {
             account_id: self.account_id,
             member_account_rule_status: self.member_account_rule_status,
         }
