--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl TemplateSsmDocumentDetails {
     /// Creates a new builder-style object to manufacture [`TemplateSsmDocumentDetails`](crate::types::TemplateSsmDocumentDetails).
-    pub fn builder() -> crate::types::builders::TemplateSsmDocumentDetailsBuilder {
-        crate::types::builders::TemplateSsmDocumentDetailsBuilder::default()
+    pub fn builder() -> super::super::types::builders::TemplateSsmDocumentDetailsBuilder {
+        super::super::types::builders::TemplateSsmDocumentDetailsBuilder::default()
     }
 }

@@ -78,8 +78,8 @@
     /// Consumes the builder and constructs a [`TemplateSsmDocumentDetails`](crate::types::TemplateSsmDocumentDetails).
     /// This method will fail if any of the following fields are not set:
     /// - [`document_name`](crate::types::builders::TemplateSsmDocumentDetailsBuilder::document_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::TemplateSsmDocumentDetails, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::TemplateSsmDocumentDetails {
+    pub fn build(self) -> ::std::result::Result<super::super::types::TemplateSsmDocumentDetails, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::TemplateSsmDocumentDetails {
             document_name: self.document_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "document_name",
