--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListRolesInput {
     /// Creates a new builder-style object to manufacture [`ListRolesInput`](crate::operation::list_roles::ListRolesInput).
-    pub fn builder() -> crate::operation::list_roles::builders::ListRolesInputBuilder {
-        crate::operation::list_roles::builders::ListRolesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_roles::builders::ListRolesInputBuilder {
+        super::super::super::operation::list_roles::builders::ListRolesInputBuilder::default()
     }
 }

@@ -93,8 +93,8 @@
         &self.max_items
     }
     /// Consumes the builder and constructs a [`ListRolesInput`](crate::operation::list_roles::ListRolesInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_roles::ListRolesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_roles::ListRolesInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::list_roles::ListRolesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_roles::ListRolesInput {
             path_prefix: self.path_prefix,
             marker: self.marker,
             max_items: self.max_items,
