--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>The pattern of the package group from which to list allowed repositories.</p>
     pub package_group: ::std::option::Option<::std::string::String>,
     /// <p>The origin configuration restriction type of which to list allowed repositories.</p>
-    pub origin_restriction_type: ::std::option::Option<crate::types::PackageGroupOriginRestrictionType>,
+    pub origin_restriction_type: ::std::option::Option<super::super::super::types::PackageGroupOriginRestrictionType>,
     /// <p>The maximum number of results to return per page.</p>
     pub max_results: ::std::option::Option<i32>,
     /// <p>The token for the next set of results. Use the value returned in the previous response in the next request to retrieve the next set of results.</p>
@@ -30,7 +30,7 @@
         self.package_group.as_deref()
     }
     /// <p>The origin configuration restriction type of which to list allowed repositories.</p>
-    pub fn origin_restriction_type(&self) -> ::std::option::Option<&crate::types::PackageGroupOriginRestrictionType> {
+    pub fn origin_restriction_type(&self) -> ::std::option::Option<&super::super::super::types::PackageGroupOriginRestrictionType> {
         self.origin_restriction_type.as_ref()
     }
     /// <p>The maximum number of results to return per page.</p>
@@ -44,8 +44,8 @@
 }
 impl ListAllowedRepositoriesForGroupInput {
     /// Creates a new builder-style object to manufacture [`ListAllowedRepositoriesForGroupInput`](crate::operation::list_allowed_repositories_for_group::ListAllowedRepositoriesForGroupInput).
-    pub fn builder() -> crate::operation::list_allowed_repositories_for_group::builders::ListAllowedRepositoriesForGroupInputBuilder {
-        crate::operation::list_allowed_repositories_for_group::builders::ListAllowedRepositoriesForGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_allowed_repositories_for_group::builders::ListAllowedRepositoriesForGroupInputBuilder {
+        super::super::super::operation::list_allowed_repositories_for_group::builders::ListAllowedRepositoriesForGroupInputBuilder::default()
     }
 }

@@ -56,7 +56,7 @@
     pub(crate) domain: ::std::option::Option<::std::string::String>,
     pub(crate) domain_owner: ::std::option::Option<::std::string::String>,
     pub(crate) package_group: ::std::option::Option<::std::string::String>,
-    pub(crate) origin_restriction_type: ::std::option::Option<crate::types::PackageGroupOriginRestrictionType>,
+    pub(crate) origin_restriction_type: ::std::option::Option<super::super::super::types::PackageGroupOriginRestrictionType>,
     pub(crate) max_results: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
 }
@@ -107,17 +107,17 @@
     }
     /// <p>The origin configuration restriction type of which to list allowed repositories.</p>
     /// This field is required.
-    pub fn origin_restriction_type(mut self, input: crate::types::PackageGroupOriginRestrictionType) -> Self {
+    pub fn origin_restriction_type(mut self, input: super::super::super::types::PackageGroupOriginRestrictionType) -> Self {
         self.origin_restriction_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The origin configuration restriction type of which to list allowed repositories.</p>
-    pub fn set_origin_restriction_type(mut self, input: ::std::option::Option<crate::types::PackageGroupOriginRestrictionType>) -> Self {
+    pub fn set_origin_restriction_type(mut self, input: ::std::option::Option<super::super::super::types::PackageGroupOriginRestrictionType>) -> Self {
         self.origin_restriction_type = input;
         self
     }
     /// <p>The origin configuration restriction type of which to list allowed repositories.</p>
-    pub fn get_origin_restriction_type(&self) -> &::std::option::Option<crate::types::PackageGroupOriginRestrictionType> {
+    pub fn get_origin_restriction_type(&self) -> &::std::option::Option<super::super::super::types::PackageGroupOriginRestrictionType> {
         &self.origin_restriction_type
     }
     /// <p>The maximum number of results to return per page.</p>
@@ -152,11 +152,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_allowed_repositories_for_group::ListAllowedRepositoriesForGroupInput,
+        super::super::super::operation::list_allowed_repositories_for_group::ListAllowedRepositoriesForGroupInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::list_allowed_repositories_for_group::ListAllowedRepositoriesForGroupInput {
+            super::super::super::operation::list_allowed_repositories_for_group::ListAllowedRepositoriesForGroupInput {
                 domain: self.domain,
                 domain_owner: self.domain_owner,
                 package_group: self.package_group,
