--- original
+++ modified
@@ -20,11 +20,11 @@
     /// </note>
     pub delivery_s3_key_prefix: ::std::option::Option<::std::string::String>,
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+    pub conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<super::types::ConformancePackInputParameter>>,
     /// <p>A list of Amazon Web Services accounts to be excluded from an organization conformance pack while deploying a conformance pack.</p>
     pub excluded_accounts: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>The tags for the organization conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::types::Tag>>,
 }
 impl PutOrganizationConformancePackInput {
     /// <p>Name of the organization conformance pack you want to create.</p>
@@ -56,7 +56,7 @@
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.conformance_pack_input_parameters.is_none()`.
-    pub fn conformance_pack_input_parameters(&self) -> &[crate::types::ConformancePackInputParameter] {
+    pub fn conformance_pack_input_parameters(&self) -> &[super::types::ConformancePackInputParameter] {
         self.conformance_pack_input_parameters.as_deref().unwrap_or_default()
     }
     /// <p>A list of Amazon Web Services accounts to be excluded from an organization conformance pack while deploying a conformance pack.</p>
@@ -68,14 +68,14 @@
     /// <p>The tags for the organization conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
 }
 impl PutOrganizationConformancePackInput {
     /// Creates a new builder-style object to manufacture [`PutOrganizationConformancePackInput`](crate::operation::put_organization_conformance_pack::PutOrganizationConformancePackInput).
-    pub fn builder() -> crate::operation::put_organization_conformance_pack::builders::PutOrganizationConformancePackInputBuilder {
-        crate::operation::put_organization_conformance_pack::builders::PutOrganizationConformancePackInputBuilder::default()
+    pub fn builder() -> super::operation::put_organization_conformance_pack::builders::PutOrganizationConformancePackInputBuilder {
+        super::operation::put_organization_conformance_pack::builders::PutOrganizationConformancePackInputBuilder::default()
     }
 }

@@ -88,9 +88,9 @@
     pub(crate) template_body: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_s3_bucket: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_s3_key_prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+    pub(crate) conformance_pack_input_parameters: ::std::option::Option<::std::vec::Vec<super::types::ConformancePackInputParameter>>,
     pub(crate) excluded_accounts: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::types::Tag>>,
 }
 impl PutOrganizationConformancePackInputBuilder {
     /// <p>Name of the organization conformance pack you want to create.</p>
@@ -187,7 +187,7 @@
     /// To override the contents of this collection use [`set_conformance_pack_input_parameters`](Self::set_conformance_pack_input_parameters).
     ///
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub fn conformance_pack_input_parameters(mut self, input: crate::types::ConformancePackInputParameter) -> Self {
+    pub fn conformance_pack_input_parameters(mut self, input: super::types::ConformancePackInputParameter) -> Self {
         let mut v = self.conformance_pack_input_parameters.unwrap_or_default();
         v.push(input);
         self.conformance_pack_input_parameters = ::std::option::Option::Some(v);
@@ -196,13 +196,13 @@
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
     pub fn set_conformance_pack_input_parameters(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>>,
+        input: ::std::option::Option<::std::vec::Vec<super::types::ConformancePackInputParameter>>,
     ) -> Self {
         self.conformance_pack_input_parameters = input;
         self
     }
     /// <p>A list of <code>ConformancePackInputParameter</code> objects.</p>
-    pub fn get_conformance_pack_input_parameters(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ConformancePackInputParameter>> {
+    pub fn get_conformance_pack_input_parameters(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ConformancePackInputParameter>> {
         &self.conformance_pack_input_parameters
     }
     /// Appends an item to `excluded_accounts`.
@@ -230,7 +230,7 @@
     /// To override the contents of this collection use [`set_tags`](Self::set_tags).
     ///
     /// <p>The tags for the organization conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -237,12 +237,12 @@
         self
     }
     /// <p>The tags for the organization conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
     /// <p>The tags for the organization conformance pack. Each tag consists of a key and an optional value, both of which you define.</p>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::types::Tag>> {
         &self.tags
     }
     /// Consumes the builder and constructs a [`PutOrganizationConformancePackInput`](crate::operation::put_organization_conformance_pack::PutOrganizationConformancePackInput).
@@ -249,10 +249,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_organization_conformance_pack::PutOrganizationConformancePackInput,
+        super::operation::put_organization_conformance_pack::PutOrganizationConformancePackInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_organization_conformance_pack::PutOrganizationConformancePackInput {
+        ::std::result::Result::Ok(super::operation::put_organization_conformance_pack::PutOrganizationConformancePackInput {
             organization_conformance_pack_name: self.organization_conformance_pack_name,
             template_s3_uri: self.template_s3_uri,
             template_body: self.template_body,
