--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl ListAttachedUserPoliciesInput {
     /// Creates a new builder-style object to manufacture [`ListAttachedUserPoliciesInput`](crate::operation::list_attached_user_policies::ListAttachedUserPoliciesInput).
-    pub fn builder() -> crate::operation::list_attached_user_policies::builders::ListAttachedUserPoliciesInputBuilder {
-        crate::operation::list_attached_user_policies::builders::ListAttachedUserPoliciesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_attached_user_policies::builders::ListAttachedUserPoliciesInputBuilder {
+        super::super::super::operation::list_attached_user_policies::builders::ListAttachedUserPoliciesInputBuilder::default()
     }
 }

@@ -123,10 +123,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_attached_user_policies::ListAttachedUserPoliciesInput,
+        super::super::super::operation::list_attached_user_policies::ListAttachedUserPoliciesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_attached_user_policies::ListAttachedUserPoliciesInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_attached_user_policies::ListAttachedUserPoliciesInput {
             user_name: self.user_name,
             path_prefix: self.path_prefix,
             marker: self.marker,
