--- original
+++ modified
@@ -13,9 +13,9 @@
     /// <p>The two-character code for the country or region, in ISO 3166-1 alpha-2 format.</p>
     pub iso2_country_code: ::std::option::Option<::std::string::String>,
     /// <p>The list of supported routes.</p>
-    pub route_type: ::std::option::Option<crate::types::RouteType>,
+    pub route_type: ::std::option::Option<super::types::RouteType>,
     /// <p>The capabilities of each phone number.</p>
-    pub number_capabilities: ::std::option::Option<::std::vec::Vec<crate::types::NumberCapability>>,
+    pub number_capabilities: ::std::option::Option<::std::vec::Vec<super::types::NumberCapability>>,
 }
 impl PhoneNumberInformation {
     /// <p>The date and time when the phone number was created.</p>
@@ -35,13 +35,13 @@
         self.iso2_country_code.as_deref()
     }
     /// <p>The list of supported routes.</p>
-    pub fn route_type(&self) -> ::std::option::Option<&crate::types::RouteType> {
+    pub fn route_type(&self) -> ::std::option::Option<&super::types::RouteType> {
         self.route_type.as_ref()
     }
     /// <p>The capabilities of each phone number.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.number_capabilities.is_none()`.
-    pub fn number_capabilities(&self) -> &[crate::types::NumberCapability] {
+    pub fn number_capabilities(&self) -> &[super::types::NumberCapability] {
         self.number_capabilities.as_deref().unwrap_or_default()
     }
 }
@@ -59,8 +59,8 @@
 }
 impl PhoneNumberInformation {
     /// Creates a new builder-style object to manufacture [`PhoneNumberInformation`](crate::types::PhoneNumberInformation).
-    pub fn builder() -> crate::types::builders::PhoneNumberInformationBuilder {
-        crate::types::builders::PhoneNumberInformationBuilder::default()
+    pub fn builder() -> super::types::builders::PhoneNumberInformationBuilder {
+        super::types::builders::PhoneNumberInformationBuilder::default()
     }
 }

@@ -72,8 +72,8 @@
     pub(crate) phone_number: ::std::option::Option<::std::string::String>,
     pub(crate) status: ::std::option::Option<::std::string::String>,
     pub(crate) iso2_country_code: ::std::option::Option<::std::string::String>,
-    pub(crate) route_type: ::std::option::Option<crate::types::RouteType>,
-    pub(crate) number_capabilities: ::std::option::Option<::std::vec::Vec<crate::types::NumberCapability>>,
+    pub(crate) route_type: ::std::option::Option<super::types::RouteType>,
+    pub(crate) number_capabilities: ::std::option::Option<::std::vec::Vec<super::types::NumberCapability>>,
 }
 impl PhoneNumberInformationBuilder {
     /// <p>The date and time when the phone number was created.</p>
@@ -133,17 +133,17 @@
         &self.iso2_country_code
     }
     /// <p>The list of supported routes.</p>
-    pub fn route_type(mut self, input: crate::types::RouteType) -> Self {
+    pub fn route_type(mut self, input: super::types::RouteType) -> Self {
         self.route_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The list of supported routes.</p>
-    pub fn set_route_type(mut self, input: ::std::option::Option<crate::types::RouteType>) -> Self {
+    pub fn set_route_type(mut self, input: ::std::option::Option<super::types::RouteType>) -> Self {
         self.route_type = input;
         self
     }
     /// <p>The list of supported routes.</p>
-    pub fn get_route_type(&self) -> &::std::option::Option<crate::types::RouteType> {
+    pub fn get_route_type(&self) -> &::std::option::Option<super::types::RouteType> {
         &self.route_type
     }
     /// Appends an item to `number_capabilities`.
@@ -151,7 +151,7 @@
     /// To override the contents of this collection use [`set_number_capabilities`](Self::set_number_capabilities).
     ///
     /// <p>The capabilities of each phone number.</p>
-    pub fn number_capabilities(mut self, input: crate::types::NumberCapability) -> Self {
+    pub fn number_capabilities(mut self, input: super::types::NumberCapability) -> Self {
         let mut v = self.number_capabilities.unwrap_or_default();
         v.push(input);
         self.number_capabilities = ::std::option::Option::Some(v);
@@ -158,17 +158,17 @@
         self
     }
     /// <p>The capabilities of each phone number.</p>
-    pub fn set_number_capabilities(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::NumberCapability>>) -> Self {
+    pub fn set_number_capabilities(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::NumberCapability>>) -> Self {
         self.number_capabilities = input;
         self
     }
     /// <p>The capabilities of each phone number.</p>
-    pub fn get_number_capabilities(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::NumberCapability>> {
+    pub fn get_number_capabilities(&self) -> &::std::option::Option<::std::vec::Vec<super::types::NumberCapability>> {
         &self.number_capabilities
     }
     /// Consumes the builder and constructs a [`PhoneNumberInformation`](crate::types::PhoneNumberInformation).
-    pub fn build(self) -> crate::types::PhoneNumberInformation {
-        crate::types::PhoneNumberInformation {
+    pub fn build(self) -> super::types::PhoneNumberInformation {
+        super::types::PhoneNumberInformation {
             created_at: self.created_at,
             phone_number: self.phone_number,
             status: self.status,
