--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The destination phone number.</p>
     pub phone_number: ::std::option::Option<::std::string::String>,
     /// <p>The destination phone number's verification status.</p>
-    pub status: ::std::option::Option<crate::types::SmsSandboxPhoneNumberVerificationStatus>,
+    pub status: ::std::option::Option<super::super::types::SmsSandboxPhoneNumberVerificationStatus>,
 }
 impl SmsSandboxPhoneNumber {
     /// <p>The destination phone number.</p>
@@ -16,7 +16,7 @@
         self.phone_number.as_deref()
     }
     /// <p>The destination phone number's verification status.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::SmsSandboxPhoneNumberVerificationStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::SmsSandboxPhoneNumberVerificationStatus> {
         self.status.as_ref()
     }
 }
@@ -30,8 +30,8 @@
 }
 impl SmsSandboxPhoneNumber {
     /// Creates a new builder-style object to manufacture [`SmsSandboxPhoneNumber`](crate::types::SmsSandboxPhoneNumber).
-    pub fn builder() -> crate::types::builders::SmsSandboxPhoneNumberBuilder {
-        crate::types::builders::SmsSandboxPhoneNumberBuilder::default()
+    pub fn builder() -> super::super::types::builders::SmsSandboxPhoneNumberBuilder {
+        super::super::types::builders::SmsSandboxPhoneNumberBuilder::default()
     }
 }

@@ -40,7 +40,7 @@
 #[non_exhaustive]
 pub struct SmsSandboxPhoneNumberBuilder {
     pub(crate) phone_number: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::SmsSandboxPhoneNumberVerificationStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::SmsSandboxPhoneNumberVerificationStatus>,
 }
 impl SmsSandboxPhoneNumberBuilder {
     /// <p>The destination phone number.</p>
@@ -58,22 +58,22 @@
         &self.phone_number
     }
     /// <p>The destination phone number's verification status.</p>
-    pub fn status(mut self, input: crate::types::SmsSandboxPhoneNumberVerificationStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::SmsSandboxPhoneNumberVerificationStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The destination phone number's verification status.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::SmsSandboxPhoneNumberVerificationStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::SmsSandboxPhoneNumberVerificationStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>The destination phone number's verification status.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::SmsSandboxPhoneNumberVerificationStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::SmsSandboxPhoneNumberVerificationStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`SmsSandboxPhoneNumber`](crate::types::SmsSandboxPhoneNumber).
-    pub fn build(self) -> crate::types::SmsSandboxPhoneNumber {
-        crate::types::SmsSandboxPhoneNumber {
+    pub fn build(self) -> super::super::types::SmsSandboxPhoneNumber {
+        super::super::types::SmsSandboxPhoneNumber {
             phone_number: self.phone_number,
             status: self.status,
         }
