--- original
+++ modified
@@ -11,7 +11,7 @@
     ///   - [`name(Option<String>)`](crate::operation::create_scheduling_policy::CreateSchedulingPolicyOutput::name): <p>The name of the scheduling policy.</p>
     ///   - [`arn(Option<String>)`](crate::operation::create_scheduling_policy::CreateSchedulingPolicyOutput::arn): <p>The Amazon Resource Name (ARN) of the scheduling policy. The format is <code>aws:<i>Partition</i>:batch:<i>Region</i>:<i>Account</i>:scheduling-policy/<i>Name</i> </code>. For example, <code>aws:aws:batch:us-west-2:123456789012:scheduling-policy/MySchedulingPolicy</code>.</p>
     /// - On failure, responds with [`SdkError<CreateSchedulingPolicyError>`](crate::operation::create_scheduling_policy::CreateSchedulingPolicyError)
-    pub fn create_scheduling_policy(&self) -> crate::operation::create_scheduling_policy::builders::CreateSchedulingPolicyFluentBuilder {
-        crate::operation::create_scheduling_policy::builders::CreateSchedulingPolicyFluentBuilder::new(self.handle.clone())
+    pub fn create_scheduling_policy(&self) -> super::super::operation::create_scheduling_policy::builders::CreateSchedulingPolicyFluentBuilder {
+        super::super::operation::create_scheduling_policy::builders::CreateSchedulingPolicyFluentBuilder::new(self.handle.clone())
     }
 }
