--- original
+++ modified
@@ -13,7 +13,7 @@
     /// - On failure, responds with [`SdkError<DescribeComputeEnvironmentsError>`](crate::operation::describe_compute_environments::DescribeComputeEnvironmentsError)
     pub fn describe_compute_environments(
         &self,
-    ) -> crate::operation::describe_compute_environments::builders::DescribeComputeEnvironmentsFluentBuilder {
-        crate::operation::describe_compute_environments::builders::DescribeComputeEnvironmentsFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::describe_compute_environments::builders::DescribeComputeEnvironmentsFluentBuilder {
+        super::operation::describe_compute_environments::builders::DescribeComputeEnvironmentsFluentBuilder::new(self.handle.clone())
     }
 }
