--- original
+++ modified
@@ -15,7 +15,7 @@
     ///   - [`job_summary_list(Option<Vec::<JobSummary>>)`](crate::operation::list_jobs::ListJobsOutput::job_summary_list): <p>A list of job summaries that match the request.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_jobs::ListJobsOutput::next_token): <p>The <code>nextToken</code> value to include in a future <code>ListJobs</code> request. When the results of a <code>ListJobs</code> request exceed <code>maxResults</code>, this value can be used to retrieve the next page of results. This value is <code>null</code> when there are no more results to return.</p>
     /// - On failure, responds with [`SdkError<ListJobsError>`](crate::operation::list_jobs::ListJobsError)
-    pub fn list_jobs(&self) -> crate::operation::list_jobs::builders::ListJobsFluentBuilder {
-        crate::operation::list_jobs::builders::ListJobsFluentBuilder::new(self.handle.clone())
+    pub fn list_jobs(&self) -> super::operation::list_jobs::builders::ListJobsFluentBuilder {
+        super::operation::list_jobs::builders::ListJobsFluentBuilder::new(self.handle.clone())
     }
 }
