--- original
+++ modified
@@ -25,7 +25,7 @@
     ///   - [`job_name(Option<String>)`](crate::operation::submit_job::SubmitJobOutput::job_name): <p>The name of the job.</p>
     ///   - [`job_id(Option<String>)`](crate::operation::submit_job::SubmitJobOutput::job_id): <p>The unique identifier for the job.</p>
     /// - On failure, responds with [`SdkError<SubmitJobError>`](crate::operation::submit_job::SubmitJobError)
-    pub fn submit_job(&self) -> crate::operation::submit_job::builders::SubmitJobFluentBuilder {
-        crate::operation::submit_job::builders::SubmitJobFluentBuilder::new(self.handle.clone())
+    pub fn submit_job(&self) -> super::operation::submit_job::builders::SubmitJobFluentBuilder {
+        super::operation::submit_job::builders::SubmitJobFluentBuilder::new(self.handle.clone())
     }
 }
