--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <li>
     /// <p><code>ACCOUNT</code> – Use the account-level suppression list.</p></li>
     /// </ul>
-    pub suppression_scope: ::std::option::Option<crate::types::SuppressionListScope>,
+    pub suppression_scope: ::std::option::Option<super::types::SuppressionListScope>,
     /// <p>A list that contains the reasons that email addresses are automatically added to the suppression list for your account or for a specific tenant. This list can contain any or all of the following:</p>
     /// <ul>
     /// <li>
@@ -21,9 +21,9 @@
     /// <li>
     /// <p><code>BOUNCE</code> – Amazon SES adds an email address to the suppression list for your account or for a specific tenant when a message sent to that address results in a hard bounce.</p></li>
     /// </ul>
-    pub suppressed_reasons: ::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>>,
+    pub suppressed_reasons: ::std::option::Option<::std::vec::Vec<super::types::SuppressionListReason>>,
     /// <p>An object that contains information about the email address suppression preferences for the configuration set in the current Amazon Web Services Region.</p>
-    pub validation_options: ::std::option::Option<crate::types::SuppressionValidationOptions>,
+    pub validation_options: ::std::option::Option<super::types::SuppressionValidationOptions>,
 }
 impl PutConfigurationSetSuppressionOptionsInput {
     /// <p>The name of the configuration set to change the suppression list preferences for.</p>
@@ -37,7 +37,7 @@
     /// <li>
     /// <p><code>ACCOUNT</code> – Use the account-level suppression list.</p></li>
     /// </ul>
-    pub fn suppression_scope(&self) -> ::std::option::Option<&crate::types::SuppressionListScope> {
+    pub fn suppression_scope(&self) -> ::std::option::Option<&super::types::SuppressionListScope> {
         self.suppression_scope.as_ref()
     }
     /// <p>A list that contains the reasons that email addresses are automatically added to the suppression list for your account or for a specific tenant. This list can contain any or all of the following:</p>
@@ -49,18 +49,18 @@
     /// </ul>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.suppressed_reasons.is_none()`.
-    pub fn suppressed_reasons(&self) -> &[crate::types::SuppressionListReason] {
+    pub fn suppressed_reasons(&self) -> &[super::types::SuppressionListReason] {
         self.suppressed_reasons.as_deref().unwrap_or_default()
     }
     /// <p>An object that contains information about the email address suppression preferences for the configuration set in the current Amazon Web Services Region.</p>
-    pub fn validation_options(&self) -> ::std::option::Option<&crate::types::SuppressionValidationOptions> {
+    pub fn validation_options(&self) -> ::std::option::Option<&super::types::SuppressionValidationOptions> {
         self.validation_options.as_ref()
     }
 }
 impl PutConfigurationSetSuppressionOptionsInput {
     /// Creates a new builder-style object to manufacture [`PutConfigurationSetSuppressionOptionsInput`](crate::operation::put_configuration_set_suppression_options::PutConfigurationSetSuppressionOptionsInput).
-    pub fn builder() -> crate::operation::put_configuration_set_suppression_options::builders::PutConfigurationSetSuppressionOptionsInputBuilder {
-        crate::operation::put_configuration_set_suppression_options::builders::PutConfigurationSetSuppressionOptionsInputBuilder::default()
+    pub fn builder() -> super::operation::put_configuration_set_suppression_options::builders::PutConfigurationSetSuppressionOptionsInputBuilder {
+        super::operation::put_configuration_set_suppression_options::builders::PutConfigurationSetSuppressionOptionsInputBuilder::default()
     }
 }

@@ -69,9 +69,9 @@
 #[non_exhaustive]
 pub struct PutConfigurationSetSuppressionOptionsInputBuilder {
     pub(crate) configuration_set_name: ::std::option::Option<::std::string::String>,
-    pub(crate) suppression_scope: ::std::option::Option<crate::types::SuppressionListScope>,
-    pub(crate) suppressed_reasons: ::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>>,
-    pub(crate) validation_options: ::std::option::Option<crate::types::SuppressionValidationOptions>,
+    pub(crate) suppression_scope: ::std::option::Option<super::types::SuppressionListScope>,
+    pub(crate) suppressed_reasons: ::std::option::Option<::std::vec::Vec<super::types::SuppressionListReason>>,
+    pub(crate) validation_options: ::std::option::Option<super::types::SuppressionValidationOptions>,
 }
 impl PutConfigurationSetSuppressionOptionsInputBuilder {
     /// <p>The name of the configuration set to change the suppression list preferences for.</p>
@@ -96,7 +96,7 @@
     /// <li>
     /// <p><code>ACCOUNT</code> – Use the account-level suppression list.</p></li>
     /// </ul>
-    pub fn suppression_scope(mut self, input: crate::types::SuppressionListScope) -> Self {
+    pub fn suppression_scope(mut self, input: super::types::SuppressionListScope) -> Self {
         self.suppression_scope = ::std::option::Option::Some(input);
         self
     }
@@ -107,7 +107,7 @@
     /// <li>
     /// <p><code>ACCOUNT</code> – Use the account-level suppression list.</p></li>
     /// </ul>
-    pub fn set_suppression_scope(mut self, input: ::std::option::Option<crate::types::SuppressionListScope>) -> Self {
+    pub fn set_suppression_scope(mut self, input: ::std::option::Option<super::types::SuppressionListScope>) -> Self {
         self.suppression_scope = input;
         self
     }
@@ -118,7 +118,7 @@
     /// <li>
     /// <p><code>ACCOUNT</code> – Use the account-level suppression list.</p></li>
     /// </ul>
-    pub fn get_suppression_scope(&self) -> &::std::option::Option<crate::types::SuppressionListScope> {
+    pub fn get_suppression_scope(&self) -> &::std::option::Option<super::types::SuppressionListScope> {
         &self.suppression_scope
     }
     /// Appends an item to `suppressed_reasons`.
@@ -132,7 +132,7 @@
     /// <li>
     /// <p><code>BOUNCE</code> – Amazon SES adds an email address to the suppression list for your account or for a specific tenant when a message sent to that address results in a hard bounce.</p></li>
     /// </ul>
-    pub fn suppressed_reasons(mut self, input: crate::types::SuppressionListReason) -> Self {
+    pub fn suppressed_reasons(mut self, input: super::types::SuppressionListReason) -> Self {
         let mut v = self.suppressed_reasons.unwrap_or_default();
         v.push(input);
         self.suppressed_reasons = ::std::option::Option::Some(v);
@@ -145,7 +145,7 @@
     /// <li>
     /// <p><code>BOUNCE</code> – Amazon SES adds an email address to the suppression list for your account or for a specific tenant when a message sent to that address results in a hard bounce.</p></li>
     /// </ul>
-    pub fn set_suppressed_reasons(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>>) -> Self {
+    pub fn set_suppressed_reasons(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::SuppressionListReason>>) -> Self {
         self.suppressed_reasons = input;
         self
     }
@@ -156,21 +156,21 @@
     /// <li>
     /// <p><code>BOUNCE</code> – Amazon SES adds an email address to the suppression list for your account or for a specific tenant when a message sent to that address results in a hard bounce.</p></li>
     /// </ul>
-    pub fn get_suppressed_reasons(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>> {
+    pub fn get_suppressed_reasons(&self) -> &::std::option::Option<::std::vec::Vec<super::types::SuppressionListReason>> {
         &self.suppressed_reasons
     }
     /// <p>An object that contains information about the email address suppression preferences for the configuration set in the current Amazon Web Services Region.</p>
-    pub fn validation_options(mut self, input: crate::types::SuppressionValidationOptions) -> Self {
+    pub fn validation_options(mut self, input: super::types::SuppressionValidationOptions) -> Self {
         self.validation_options = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains information about the email address suppression preferences for the configuration set in the current Amazon Web Services Region.</p>
-    pub fn set_validation_options(mut self, input: ::std::option::Option<crate::types::SuppressionValidationOptions>) -> Self {
+    pub fn set_validation_options(mut self, input: ::std::option::Option<super::types::SuppressionValidationOptions>) -> Self {
         self.validation_options = input;
         self
     }
     /// <p>An object that contains information about the email address suppression preferences for the configuration set in the current Amazon Web Services Region.</p>
-    pub fn get_validation_options(&self) -> &::std::option::Option<crate::types::SuppressionValidationOptions> {
+    pub fn get_validation_options(&self) -> &::std::option::Option<super::types::SuppressionValidationOptions> {
         &self.validation_options
     }
     /// Consumes the builder and constructs a [`PutConfigurationSetSuppressionOptionsInput`](crate::operation::put_configuration_set_suppression_options::PutConfigurationSetSuppressionOptionsInput).
@@ -177,11 +177,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_configuration_set_suppression_options::PutConfigurationSetSuppressionOptionsInput,
+        super::operation::put_configuration_set_suppression_options::PutConfigurationSetSuppressionOptionsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_configuration_set_suppression_options::PutConfigurationSetSuppressionOptionsInput {
+            super::operation::put_configuration_set_suppression_options::PutConfigurationSetSuppressionOptionsInput {
                 configuration_set_name: self.configuration_set_name,
                 suppression_scope: self.suppression_scope,
                 suppressed_reasons: self.suppressed_reasons,
