--- original
+++ modified
@@ -116,8 +116,8 @@
 }
 impl FilterLogEventsInput {
     /// Creates a new builder-style object to manufacture [`FilterLogEventsInput`](crate::operation::filter_log_events::FilterLogEventsInput).
-    pub fn builder() -> crate::operation::filter_log_events::builders::FilterLogEventsInputBuilder {
-        crate::operation::filter_log_events::builders::FilterLogEventsInputBuilder::default()
+    pub fn builder() -> super::operation::filter_log_events::builders::FilterLogEventsInputBuilder {
+        super::operation::filter_log_events::builders::FilterLogEventsInputBuilder::default()
     }
 }

@@ -367,8 +367,8 @@
     /// Consumes the builder and constructs a [`FilterLogEventsInput`](crate::operation::filter_log_events::FilterLogEventsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::filter_log_events::FilterLogEventsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::filter_log_events::FilterLogEventsInput {
+    ) -> ::std::result::Result<super::operation::filter_log_events::FilterLogEventsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::filter_log_events::FilterLogEventsInput {
             log_group_name: self.log_group_name,
             log_group_identifier: self.log_group_identifier,
             log_stream_names: self.log_stream_names,
