--- original
+++ modified
@@ -57,8 +57,8 @@
 }
 impl UpdateLoginProfileInput {
     /// Creates a new builder-style object to manufacture [`UpdateLoginProfileInput`](crate::operation::update_login_profile::UpdateLoginProfileInput).
-    pub fn builder() -> crate::operation::update_login_profile::builders::UpdateLoginProfileInputBuilder {
-        crate::operation::update_login_profile::builders::UpdateLoginProfileInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_login_profile::builders::UpdateLoginProfileInputBuilder {
+        super::super::super::operation::update_login_profile::builders::UpdateLoginProfileInputBuilder::default()
     }
 }

@@ -150,9 +150,9 @@
     /// Consumes the builder and constructs a [`UpdateLoginProfileInput`](crate::operation::update_login_profile::UpdateLoginProfileInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_login_profile::UpdateLoginProfileInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::update_login_profile::UpdateLoginProfileInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_login_profile::UpdateLoginProfileInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_login_profile::UpdateLoginProfileInput {
             user_name: self.user_name,
             password: self.password,
             password_reset_required: self.password_reset_required,
