--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The <code>ProviderName</code> should be set to <code>Cognito</code> for users in Cognito user pools.</p><important>
     /// <p>All attributes in the DestinationUser profile must be mutable. If you have assigned the user any immutable custom attributes, the operation won't succeed.</p>
     /// </important>
-    pub destination_user: ::std::option::Option<crate::types::ProviderUserIdentifierType>,
+    pub destination_user: ::std::option::Option<super::types::ProviderUserIdentifierType>,
     /// <p>An external IdP account for a user who doesn't exist yet in the user pool. This user must be a federated user (for example, a SAML or Facebook user), not another native user.</p>
     /// <p>If the <code>SourceUser</code> is using a federated social IdP, such as Facebook, Google, or Login with Amazon, you must set the <code>ProviderAttributeName</code> to <code>Cognito_Subject</code>. For social IdPs, the <code>ProviderName</code> will be <code>Facebook</code>, <code>Google</code>, or <code>LoginWithAmazon</code>, and Amazon Cognito will automatically parse the Facebook, Google, and Login with Amazon tokens for <code>id</code>, <code>sub</code>, and <code>user_id</code>, respectively. The <code>ProviderAttributeValue</code> for the user must be the same value as the <code>id</code>, <code>sub</code>, or <code>user_id</code> value found in the social IdP token.</p>
     /// <p>For OIDC, the <code>ProviderAttributeName</code> can be any mapped value from a claim in the ID token, or that your app retrieves from the <code>userInfo</code> endpoint. For SAML, the <code>ProviderAttributeName</code> can be any mapped value from a claim in the SAML assertion.</p>
@@ -22,7 +22,7 @@
     /// <li>
     /// <p>When you set <code>ProviderAttributeName</code> to <code>Cognito_Subject</code>, Amazon Cognito will automatically parse the default unique identifier found in the subject from the IdP token.</p></li>
     /// </ul>
-    pub source_user: ::std::option::Option<crate::types::ProviderUserIdentifierType>,
+    pub source_user: ::std::option::Option<super::types::ProviderUserIdentifierType>,
 }
 impl AdminLinkProviderForUserInput {
     /// <p>The ID of the user pool where you want to link a federated identity.</p>
@@ -35,7 +35,7 @@
     /// <p>The <code>ProviderName</code> should be set to <code>Cognito</code> for users in Cognito user pools.</p><important>
     /// <p>All attributes in the DestinationUser profile must be mutable. If you have assigned the user any immutable custom attributes, the operation won't succeed.</p>
     /// </important>
-    pub fn destination_user(&self) -> ::std::option::Option<&crate::types::ProviderUserIdentifierType> {
+    pub fn destination_user(&self) -> ::std::option::Option<&super::types::ProviderUserIdentifierType> {
         self.destination_user.as_ref()
     }
     /// <p>An external IdP account for a user who doesn't exist yet in the user pool. This user must be a federated user (for example, a SAML or Facebook user), not another native user.</p>
@@ -48,14 +48,14 @@
     /// <li>
     /// <p>When you set <code>ProviderAttributeName</code> to <code>Cognito_Subject</code>, Amazon Cognito will automatically parse the default unique identifier found in the subject from the IdP token.</p></li>
     /// </ul>
-    pub fn source_user(&self) -> ::std::option::Option<&crate::types::ProviderUserIdentifierType> {
+    pub fn source_user(&self) -> ::std::option::Option<&super::types::ProviderUserIdentifierType> {
         self.source_user.as_ref()
     }
 }
 impl AdminLinkProviderForUserInput {
     /// Creates a new builder-style object to manufacture [`AdminLinkProviderForUserInput`](crate::operation::admin_link_provider_for_user::AdminLinkProviderForUserInput).
-    pub fn builder() -> crate::operation::admin_link_provider_for_user::builders::AdminLinkProviderForUserInputBuilder {
-        crate::operation::admin_link_provider_for_user::builders::AdminLinkProviderForUserInputBuilder::default()
+    pub fn builder() -> super::operation::admin_link_provider_for_user::builders::AdminLinkProviderForUserInputBuilder {
+        super::operation::admin_link_provider_for_user::builders::AdminLinkProviderForUserInputBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
 #[non_exhaustive]
 pub struct AdminLinkProviderForUserInputBuilder {
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
-    pub(crate) destination_user: ::std::option::Option<crate::types::ProviderUserIdentifierType>,
-    pub(crate) source_user: ::std::option::Option<crate::types::ProviderUserIdentifierType>,
+    pub(crate) destination_user: ::std::option::Option<super::types::ProviderUserIdentifierType>,
+    pub(crate) source_user: ::std::option::Option<super::types::ProviderUserIdentifierType>,
 }
 impl AdminLinkProviderForUserInputBuilder {
     /// <p>The ID of the user pool where you want to link a federated identity.</p>
@@ -90,7 +90,7 @@
     /// <p>All attributes in the DestinationUser profile must be mutable. If you have assigned the user any immutable custom attributes, the operation won't succeed.</p>
     /// </important>
     /// This field is required.
-    pub fn destination_user(mut self, input: crate::types::ProviderUserIdentifierType) -> Self {
+    pub fn destination_user(mut self, input: super::types::ProviderUserIdentifierType) -> Self {
         self.destination_user = ::std::option::Option::Some(input);
         self
     }
@@ -100,7 +100,7 @@
     /// <p>The <code>ProviderName</code> should be set to <code>Cognito</code> for users in Cognito user pools.</p><important>
     /// <p>All attributes in the DestinationUser profile must be mutable. If you have assigned the user any immutable custom attributes, the operation won't succeed.</p>
     /// </important>
-    pub fn set_destination_user(mut self, input: ::std::option::Option<crate::types::ProviderUserIdentifierType>) -> Self {
+    pub fn set_destination_user(mut self, input: ::std::option::Option<super::types::ProviderUserIdentifierType>) -> Self {
         self.destination_user = input;
         self
     }
@@ -110,7 +110,7 @@
     /// <p>The <code>ProviderName</code> should be set to <code>Cognito</code> for users in Cognito user pools.</p><important>
     /// <p>All attributes in the DestinationUser profile must be mutable. If you have assigned the user any immutable custom attributes, the operation won't succeed.</p>
     /// </important>
-    pub fn get_destination_user(&self) -> &::std::option::Option<crate::types::ProviderUserIdentifierType> {
+    pub fn get_destination_user(&self) -> &::std::option::Option<super::types::ProviderUserIdentifierType> {
         &self.destination_user
     }
     /// <p>An external IdP account for a user who doesn't exist yet in the user pool. This user must be a federated user (for example, a SAML or Facebook user), not another native user.</p>
@@ -124,7 +124,7 @@
     /// <p>When you set <code>ProviderAttributeName</code> to <code>Cognito_Subject</code>, Amazon Cognito will automatically parse the default unique identifier found in the subject from the IdP token.</p></li>
     /// </ul>
     /// This field is required.
-    pub fn source_user(mut self, input: crate::types::ProviderUserIdentifierType) -> Self {
+    pub fn source_user(mut self, input: super::types::ProviderUserIdentifierType) -> Self {
         self.source_user = ::std::option::Option::Some(input);
         self
     }
@@ -138,7 +138,7 @@
     /// <li>
     /// <p>When you set <code>ProviderAttributeName</code> to <code>Cognito_Subject</code>, Amazon Cognito will automatically parse the default unique identifier found in the subject from the IdP token.</p></li>
     /// </ul>
-    pub fn set_source_user(mut self, input: ::std::option::Option<crate::types::ProviderUserIdentifierType>) -> Self {
+    pub fn set_source_user(mut self, input: ::std::option::Option<super::types::ProviderUserIdentifierType>) -> Self {
         self.source_user = input;
         self
     }
@@ -152,7 +152,7 @@
     /// <li>
     /// <p>When you set <code>ProviderAttributeName</code> to <code>Cognito_Subject</code>, Amazon Cognito will automatically parse the default unique identifier found in the subject from the IdP token.</p></li>
     /// </ul>
-    pub fn get_source_user(&self) -> &::std::option::Option<crate::types::ProviderUserIdentifierType> {
+    pub fn get_source_user(&self) -> &::std::option::Option<super::types::ProviderUserIdentifierType> {
         &self.source_user
     }
     /// Consumes the builder and constructs a [`AdminLinkProviderForUserInput`](crate::operation::admin_link_provider_for_user::AdminLinkProviderForUserInput).
@@ -159,10 +159,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::admin_link_provider_for_user::AdminLinkProviderForUserInput,
+        super::operation::admin_link_provider_for_user::AdminLinkProviderForUserInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::admin_link_provider_for_user::AdminLinkProviderForUserInput {
+        ::std::result::Result::Ok(super::operation::admin_link_provider_for_user::AdminLinkProviderForUserInput {
             user_pool_id: self.user_pool_id,
             destination_user: self.destination_user,
             source_user: self.source_user,
