--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct PutAccountVdmAttributesInput {
     /// <p>The VDM attributes that you wish to apply to your Amazon SES account.</p>
-    pub vdm_attributes: ::std::option::Option<crate::types::VdmAttributes>,
+    pub vdm_attributes: ::std::option::Option<super::types::VdmAttributes>,
 }
 impl PutAccountVdmAttributesInput {
     /// <p>The VDM attributes that you wish to apply to your Amazon SES account.</p>
-    pub fn vdm_attributes(&self) -> ::std::option::Option<&crate::types::VdmAttributes> {
+    pub fn vdm_attributes(&self) -> ::std::option::Option<&super::types::VdmAttributes> {
         self.vdm_attributes.as_ref()
     }
 }
 impl PutAccountVdmAttributesInput {
     /// Creates a new builder-style object to manufacture [`PutAccountVdmAttributesInput`](crate::operation::put_account_vdm_attributes::PutAccountVdmAttributesInput).
-    pub fn builder() -> crate::operation::put_account_vdm_attributes::builders::PutAccountVdmAttributesInputBuilder {
-        crate::operation::put_account_vdm_attributes::builders::PutAccountVdmAttributesInputBuilder::default()
+    pub fn builder() -> super::operation::put_account_vdm_attributes::builders::PutAccountVdmAttributesInputBuilder {
+        super::operation::put_account_vdm_attributes::builders::PutAccountVdmAttributesInputBuilder::default()
     }
 }

@@ -24,22 +24,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct PutAccountVdmAttributesInputBuilder {
-    pub(crate) vdm_attributes: ::std::option::Option<crate::types::VdmAttributes>,
+    pub(crate) vdm_attributes: ::std::option::Option<super::types::VdmAttributes>,
 }
 impl PutAccountVdmAttributesInputBuilder {
     /// <p>The VDM attributes that you wish to apply to your Amazon SES account.</p>
     /// This field is required.
-    pub fn vdm_attributes(mut self, input: crate::types::VdmAttributes) -> Self {
+    pub fn vdm_attributes(mut self, input: super::types::VdmAttributes) -> Self {
         self.vdm_attributes = ::std::option::Option::Some(input);
         self
     }
     /// <p>The VDM attributes that you wish to apply to your Amazon SES account.</p>
-    pub fn set_vdm_attributes(mut self, input: ::std::option::Option<crate::types::VdmAttributes>) -> Self {
+    pub fn set_vdm_attributes(mut self, input: ::std::option::Option<super::types::VdmAttributes>) -> Self {
         self.vdm_attributes = input;
         self
     }
     /// <p>The VDM attributes that you wish to apply to your Amazon SES account.</p>
-    pub fn get_vdm_attributes(&self) -> &::std::option::Option<crate::types::VdmAttributes> {
+    pub fn get_vdm_attributes(&self) -> &::std::option::Option<super::types::VdmAttributes> {
         &self.vdm_attributes
     }
     /// Consumes the builder and constructs a [`PutAccountVdmAttributesInput`](crate::operation::put_account_vdm_attributes::PutAccountVdmAttributesInput).
@@ -46,10 +46,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_account_vdm_attributes::PutAccountVdmAttributesInput,
+        super::operation::put_account_vdm_attributes::PutAccountVdmAttributesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_account_vdm_attributes::PutAccountVdmAttributesInput {
+        ::std::result::Result::Ok(super::operation::put_account_vdm_attributes::PutAccountVdmAttributesInput {
             vdm_attributes: self.vdm_attributes,
         })
     }
