--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl DescribeRiskConfigurationInput {
     /// Creates a new builder-style object to manufacture [`DescribeRiskConfigurationInput`](crate::operation::describe_risk_configuration::DescribeRiskConfigurationInput).
-    pub fn builder() -> crate::operation::describe_risk_configuration::builders::DescribeRiskConfigurationInputBuilder {
-        crate::operation::describe_risk_configuration::builders::DescribeRiskConfigurationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_risk_configuration::builders::DescribeRiskConfigurationInputBuilder {
+        super::super::super::operation::describe_risk_configuration::builders::DescribeRiskConfigurationInputBuilder::default()
     }
 }

@@ -74,10 +74,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_risk_configuration::DescribeRiskConfigurationInput,
+        super::super::super::operation::describe_risk_configuration::DescribeRiskConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_risk_configuration::DescribeRiskConfigurationInput {
+        ::std::result::Result::Ok(super::super::super::operation::describe_risk_configuration::DescribeRiskConfigurationInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
         })
