--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetPlatformApplicationAttributesInput {
     /// Creates a new builder-style object to manufacture [`GetPlatformApplicationAttributesInput`](crate::operation::get_platform_application_attributes::GetPlatformApplicationAttributesInput).
-    pub fn builder() -> crate::operation::get_platform_application_attributes::builders::GetPlatformApplicationAttributesInputBuilder {
-        crate::operation::get_platform_application_attributes::builders::GetPlatformApplicationAttributesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_platform_application_attributes::builders::GetPlatformApplicationAttributesInputBuilder {
+        super::super::super::operation::get_platform_application_attributes::builders::GetPlatformApplicationAttributesInputBuilder::default()
     }
 }

@@ -46,11 +46,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_platform_application_attributes::GetPlatformApplicationAttributesInput,
+        super::super::super::operation::get_platform_application_attributes::GetPlatformApplicationAttributesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_platform_application_attributes::GetPlatformApplicationAttributesInput {
+            super::super::super::operation::get_platform_application_attributes::GetPlatformApplicationAttributesInput {
                 platform_application_arn: self.platform_application_arn,
             },
         )
