--- original
+++ modified
@@ -221,8 +221,8 @@
 }
 impl SubscribeInput {
     /// Creates a new builder-style object to manufacture [`SubscribeInput`](crate::operation::subscribe::SubscribeInput).
-    pub fn builder() -> crate::operation::subscribe::builders::SubscribeInputBuilder {
-        crate::operation::subscribe::builders::SubscribeInputBuilder::default()
+    pub fn builder() -> super::operation::subscribe::builders::SubscribeInputBuilder {
+        super::operation::subscribe::builders::SubscribeInputBuilder::default()
     }
 }

@@ -589,8 +589,8 @@
         &self.return_subscription_arn
     }
     /// Consumes the builder and constructs a [`SubscribeInput`](crate::operation::subscribe::SubscribeInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::subscribe::SubscribeInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::subscribe::SubscribeInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::subscribe::SubscribeInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::subscribe::SubscribeInput {
             topic_arn: self.topic_arn,
             protocol: self.protocol,
             endpoint: self.endpoint,
