--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl UntagRoleInput {
     /// Creates a new builder-style object to manufacture [`UntagRoleInput`](crate::operation::untag_role::UntagRoleInput).
-    pub fn builder() -> crate::operation::untag_role::builders::UntagRoleInputBuilder {
-        crate::operation::untag_role::builders::UntagRoleInputBuilder::default()
+    pub fn builder() -> super::operation::untag_role::builders::UntagRoleInputBuilder {
+        super::operation::untag_role::builders::UntagRoleInputBuilder::default()
     }
 }

@@ -76,8 +76,8 @@
         &self.tag_keys
     }
     /// Consumes the builder and constructs a [`UntagRoleInput`](crate::operation::untag_role::UntagRoleInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::untag_role::UntagRoleInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::untag_role::UntagRoleInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::untag_role::UntagRoleInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::untag_role::UntagRoleInput {
             role_name: self.role_name,
             tag_keys: self.tag_keys,
         })
