--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl GetUiCustomizationInput {
     /// Creates a new builder-style object to manufacture [`GetUiCustomizationInput`](crate::operation::get_ui_customization::GetUiCustomizationInput).
-    pub fn builder() -> crate::operation::get_ui_customization::builders::GetUiCustomizationInputBuilder {
-        crate::operation::get_ui_customization::builders::GetUiCustomizationInputBuilder::default()
+    pub fn builder() -> super::operation::get_ui_customization::builders::GetUiCustomizationInputBuilder {
+        super::operation::get_ui_customization::builders::GetUiCustomizationInputBuilder::default()
     }
 }

@@ -73,9 +73,9 @@
     /// Consumes the builder and constructs a [`GetUiCustomizationInput`](crate::operation::get_ui_customization::GetUiCustomizationInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_ui_customization::GetUiCustomizationInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_ui_customization::GetUiCustomizationInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_ui_customization::GetUiCustomizationInput {
+        ::std::result::Result::Ok(super::operation::get_ui_customization::GetUiCustomizationInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
         })
