--- original
+++ modified
@@ -35,8 +35,8 @@
 }
 impl RevokeTokenInput {
     /// Creates a new builder-style object to manufacture [`RevokeTokenInput`](crate::operation::revoke_token::RevokeTokenInput).
-    pub fn builder() -> crate::operation::revoke_token::builders::RevokeTokenInputBuilder {
-        crate::operation::revoke_token::builders::RevokeTokenInputBuilder::default()
+    pub fn builder() -> super::operation::revoke_token::builders::RevokeTokenInputBuilder {
+        super::operation::revoke_token::builders::RevokeTokenInputBuilder::default()
     }
 }

@@ -94,8 +94,8 @@
         &self.client_secret
     }
     /// Consumes the builder and constructs a [`RevokeTokenInput`](crate::operation::revoke_token::RevokeTokenInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::revoke_token::RevokeTokenInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::revoke_token::RevokeTokenInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::revoke_token::RevokeTokenInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::revoke_token::RevokeTokenInput {
             token: self.token,
             client_id: self.client_id,
             client_secret: self.client_secret,
