--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
+        input: super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
     ) -> ::std::result::Result<
-        crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsOutput,
+        super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
+            super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError>()
+                err.downcast::<super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsOutput>()
+                .downcast::<super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
+        input: super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("DescribePendingAggregationRequests")
                 .build()
                 .expect("required fields set"),
@@ -142,13 +142,13 @@
                 DescribePendingAggregationRequestsEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
+                super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
+                super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
+                super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -213,15 +213,15 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_describe_pending_aggregation_requests::de_describe_pending_aggregation_requests_http_error(
+            super::protocol_serde::shape_describe_pending_aggregation_requests::de_describe_pending_aggregation_requests_http_error(
                 status, headers, body,
             )
         } else {
-            crate::protocol_serde::shape_describe_pending_aggregation_requests::de_describe_pending_aggregation_requests_http_response(
+            super::protocol_serde::shape_describe_pending_aggregation_requests::de_describe_pending_aggregation_requests_http_response(
                 status, headers, body,
             )
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -234,16 +234,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput>()
+            .downcast::<super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
+                _input: &super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -252,7 +252,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
+                input: &super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -269,7 +269,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_describe_pending_aggregation_requests::ser_describe_pending_aggregation_requests_input(&input)?,
+            super::protocol_serde::shape_describe_pending_aggregation_requests::ser_describe_pending_aggregation_requests_input(&input)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -302,7 +302,7 @@
             .downcast_ref::<DescribePendingAggregationRequestsInput>()
             .ok_or("failed to downcast to DescribePendingAggregationRequestsInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -325,11 +325,11 @@
 #[derive(::std::fmt::Debug)]
 pub enum DescribePendingAggregationRequestsError {
     /// <p>The specified limit is outside the allowable range.</p>
-    InvalidLimitException(crate::types::error::InvalidLimitException),
+    InvalidLimitException(super::types::error::InvalidLimitException),
     /// <p>The specified next token is not valid. Specify the <code>nextToken</code> string that was returned in the previous response to get the next page of results.</p>
-    InvalidNextTokenException(crate::types::error::InvalidNextTokenException),
+    InvalidNextTokenException(super::types::error::InvalidNextTokenException),
     /// <p>One or more of the specified parameters are not valid. Verify that your parameters are valid and try again.</p>
-    InvalidParameterValueException(crate::types::error::InvalidParameterValueException),
+    InvalidParameterValueException(super::types::error::InvalidParameterValueException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -337,7 +337,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-DescribePendingAggregationRequestsError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl DescribePendingAggregationRequestsError {
     /// Creates the `DescribePendingAggregationRequestsError::Unhandled` variant from any error type.
@@ -344,7 +344,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -352,7 +352,7 @@

     /// Creates the `DescribePendingAggregationRequestsError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -431,21 +431,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError {
+impl ::aws_types::request_id::RequestId for super::operation::describe_pending_aggregation_requests::DescribePendingAggregationRequestsError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::describe_pending_aggregation_requests::_describe_pending_aggregation_requests_input::DescribePendingAggregationRequestsInput;
+pub use super::operation::describe_pending_aggregation_requests::_describe_pending_aggregation_requests_input::DescribePendingAggregationRequestsInput;

-pub use crate::operation::describe_pending_aggregation_requests::_describe_pending_aggregation_requests_output::DescribePendingAggregationRequestsOutput;
+pub use super::operation::describe_pending_aggregation_requests::_describe_pending_aggregation_requests_output::DescribePendingAggregationRequestsOutput;

 mod _describe_pending_aggregation_requests_input;

