--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
+        input: super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
     ) -> ::std::result::Result<
-        crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryOutput,
+        super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
+            super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError>()
+                err.downcast::<super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryOutput>()
+                .downcast::<super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
+        input: super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("GetAggregateConfigRuleComplianceSummary")
                 .build()
                 .expect("required fields set"),
@@ -142,13 +142,13 @@
                 GetAggregateConfigRuleComplianceSummaryEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
+                super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
+                super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
+                super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -218,15 +218,15 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_get_aggregate_config_rule_compliance_summary::de_get_aggregate_config_rule_compliance_summary_http_error(
+            super::protocol_serde::shape_get_aggregate_config_rule_compliance_summary::de_get_aggregate_config_rule_compliance_summary_http_error(
                 status, headers, body,
             )
         } else {
-            crate::protocol_serde::shape_get_aggregate_config_rule_compliance_summary::de_get_aggregate_config_rule_compliance_summary_http_response(
+            super::protocol_serde::shape_get_aggregate_config_rule_compliance_summary::de_get_aggregate_config_rule_compliance_summary_http_response(
                 status, headers, body,
             )
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -239,16 +239,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput>()
+            .downcast::<super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
+                _input: &super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -257,7 +257,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
+                input: &super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -274,7 +274,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_get_aggregate_config_rule_compliance_summary::ser_get_aggregate_config_rule_compliance_summary_input(
+            super::protocol_serde::shape_get_aggregate_config_rule_compliance_summary::ser_get_aggregate_config_rule_compliance_summary_input(
                 &input,
             )?,
         );
@@ -309,7 +309,7 @@
             .downcast_ref::<GetAggregateConfigRuleComplianceSummaryInput>()
             .ok_or("failed to downcast to GetAggregateConfigRuleComplianceSummaryInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -332,11 +332,11 @@
 #[derive(::std::fmt::Debug)]
 pub enum GetAggregateConfigRuleComplianceSummaryError {
     /// <p>The specified limit is outside the allowable range.</p>
-    InvalidLimitException(crate::types::error::InvalidLimitException),
+    InvalidLimitException(super::types::error::InvalidLimitException),
     /// <p>The specified next token is not valid. Specify the <code>nextToken</code> string that was returned in the previous response to get the next page of results.</p>
-    InvalidNextTokenException(crate::types::error::InvalidNextTokenException),
+    InvalidNextTokenException(super::types::error::InvalidNextTokenException),
     /// <p>You have specified a configuration aggregator that does not exist.</p>
-    NoSuchConfigurationAggregatorException(crate::types::error::NoSuchConfigurationAggregatorException),
+    NoSuchConfigurationAggregatorException(super::types::error::NoSuchConfigurationAggregatorException),
     /// <p>The requested operation is not valid. You will see this exception if there are missing required fields or if the input value fails the validation.</p>
     /// <p>For <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_PutStoredQuery.html">PutStoredQuery</a>, one of the following errors:</p>
     /// <ul>
@@ -370,7 +370,7 @@
     /// <li>
     /// <p>You have provided a service principal for service-linked configuration recorder that is not valid.</p></li>
     /// </ul>
-    ValidationException(crate::types::error::ValidationException),
+    ValidationException(super::types::error::ValidationException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -378,7 +378,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-GetAggregateConfigRuleComplianceSummaryError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl GetAggregateConfigRuleComplianceSummaryError {
     /// Creates the `GetAggregateConfigRuleComplianceSummaryError::Unhandled` variant from any error type.
@@ -385,7 +385,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -393,7 +393,7 @@

     /// Creates the `GetAggregateConfigRuleComplianceSummaryError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -480,7 +480,7 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
@@ -487,7 +487,7 @@
     }
 }
 impl ::aws_types::request_id::RequestId
-    for crate::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError
+    for super::operation::get_aggregate_config_rule_compliance_summary::GetAggregateConfigRuleComplianceSummaryError
 {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
@@ -494,9 +494,9 @@
     }
 }

-pub use crate::operation::get_aggregate_config_rule_compliance_summary::_get_aggregate_config_rule_compliance_summary_input::GetAggregateConfigRuleComplianceSummaryInput;
+pub use super::operation::get_aggregate_config_rule_compliance_summary::_get_aggregate_config_rule_compliance_summary_input::GetAggregateConfigRuleComplianceSummaryInput;

-pub use crate::operation::get_aggregate_config_rule_compliance_summary::_get_aggregate_config_rule_compliance_summary_output::GetAggregateConfigRuleComplianceSummaryOutput;
+pub use super::operation::get_aggregate_config_rule_compliance_summary::_get_aggregate_config_rule_compliance_summary_output::GetAggregateConfigRuleComplianceSummaryOutput;

 mod _get_aggregate_config_rule_compliance_summary_input;

