--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
+        input: super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
     ) -> ::std::result::Result<
-        crate::operation::put_configuration_aggregator::PutConfigurationAggregatorOutput,
+        super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
+            super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::put_configuration_aggregator::PutConfigurationAggregatorError>()
+                err.downcast::<super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::put_configuration_aggregator::PutConfigurationAggregatorOutput>()
+                .downcast::<super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
+        input: super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("PutConfigurationAggregator")
                 .build()
                 .expect("required fields set"),
@@ -142,13 +142,13 @@
                 PutConfigurationAggregatorEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
+                super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
+                super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
+                super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -213,11 +213,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_put_configuration_aggregator::de_put_configuration_aggregator_http_error(status, headers, body)
+            super::super::protocol_serde::shape_put_configuration_aggregator::de_put_configuration_aggregator_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_put_configuration_aggregator::de_put_configuration_aggregator_http_response(status, headers, body)
+            super::super::protocol_serde::shape_put_configuration_aggregator::de_put_configuration_aggregator_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -230,16 +230,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::put_configuration_aggregator::PutConfigurationAggregatorInput>()
+            .downcast::<super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
+                _input: &super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -248,7 +248,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
+                input: &super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -265,7 +265,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_put_configuration_aggregator::ser_put_configuration_aggregator_input(&input)?,
+            super::super::protocol_serde::shape_put_configuration_aggregator::ser_put_configuration_aggregator_input(&input)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -298,7 +298,7 @@
             .downcast_ref::<PutConfigurationAggregatorInput>()
             .ok_or("failed to downcast to PutConfigurationAggregatorInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -321,15 +321,15 @@
 #[derive(::std::fmt::Debug)]
 pub enum PutConfigurationAggregatorError {
     /// <p>One or more of the specified parameters are not valid. Verify that your parameters are valid and try again.</p>
-    InvalidParameterValueException(crate::types::error::InvalidParameterValueException),
+    InvalidParameterValueException(super::super::types::error::InvalidParameterValueException),
     /// <p>You have provided a null or empty Amazon Resource Name (ARN) for the IAM role assumed by Config and used by the customer managed configuration recorder.</p>
-    InvalidRoleException(crate::types::error::InvalidRoleException),
+    InvalidRoleException(super::super::types::error::InvalidRoleException),
     /// <p>For <code>PutServiceLinkedConfigurationRecorder</code> API, this exception is thrown if the number of service-linked roles in the account exceeds the limit.</p>
     /// <p>For <code>StartConfigRulesEvaluation</code> API, this exception is thrown if an evaluation is in progress or if you call the <code>StartConfigRulesEvaluation</code> API more than once per minute.</p>
     /// <p>For <code>PutConfigurationAggregator</code> API, this exception is thrown if the number of accounts and aggregators exceeds the limit.</p>
-    LimitExceededException(crate::types::error::LimitExceededException),
+    LimitExceededException(super::super::types::error::LimitExceededException),
     /// <p>Organization is no longer available.</p>
-    NoAvailableOrganizationException(crate::types::error::NoAvailableOrganizationException),
+    NoAvailableOrganizationException(super::super::types::error::NoAvailableOrganizationException),
     /// <p>For <code>PutConfigurationAggregator</code> API, you can see this exception for the following reasons:</p>
     /// <ul>
     /// <li>
@@ -342,9 +342,9 @@
     /// <p>You are not a registered delegated administrator for Config with permissions to call <code>ListDelegatedAdministrators</code> API. Ensure that the management account registers delagated administrator for Config service principal name before the delegated administrator creates an aggregator.</p></li>
     /// </ul>
     /// <p>For all <code>OrganizationConfigRule</code> and <code>OrganizationConformancePack</code> APIs, Config throws an exception if APIs are called from member accounts. All APIs must be called from organization management account.</p>
-    OrganizationAccessDeniedException(crate::types::error::OrganizationAccessDeniedException),
+    OrganizationAccessDeniedException(super::super::types::error::OrganizationAccessDeniedException),
     /// <p>Config resource cannot be created because your organization does not have all features enabled.</p>
-    OrganizationAllFeaturesNotEnabledException(crate::types::error::OrganizationAllFeaturesNotEnabledException),
+    OrganizationAllFeaturesNotEnabledException(super::super::types::error::OrganizationAllFeaturesNotEnabledException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -352,7 +352,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-PutConfigurationAggregatorError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl PutConfigurationAggregatorError {
     /// Creates the `PutConfigurationAggregatorError::Unhandled` variant from any error type.
@@ -359,7 +359,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -367,7 +367,7 @@

     /// Creates the `PutConfigurationAggregatorError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -470,21 +470,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::put_configuration_aggregator::PutConfigurationAggregatorError {
+impl ::aws_types::request_id::RequestId for super::super::operation::put_configuration_aggregator::PutConfigurationAggregatorError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::put_configuration_aggregator::_put_configuration_aggregator_input::PutConfigurationAggregatorInput;
+pub use super::super::operation::put_configuration_aggregator::_put_configuration_aggregator_input::PutConfigurationAggregatorInput;

-pub use crate::operation::put_configuration_aggregator::_put_configuration_aggregator_output::PutConfigurationAggregatorOutput;
+pub use super::super::operation::put_configuration_aggregator::_put_configuration_aggregator_output::PutConfigurationAggregatorOutput;

 mod _put_configuration_aggregator_input;

