--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::start_configuration_recorder::StartConfigurationRecorderInput,
+        input: super::operation::start_configuration_recorder::StartConfigurationRecorderInput,
     ) -> ::std::result::Result<
-        crate::operation::start_configuration_recorder::StartConfigurationRecorderOutput,
+        super::operation::start_configuration_recorder::StartConfigurationRecorderOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::start_configuration_recorder::StartConfigurationRecorderError,
+            super::operation::start_configuration_recorder::StartConfigurationRecorderError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::start_configuration_recorder::StartConfigurationRecorderError>()
+                err.downcast::<super::operation::start_configuration_recorder::StartConfigurationRecorderError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::start_configuration_recorder::StartConfigurationRecorderOutput>()
+                .downcast::<super::operation::start_configuration_recorder::StartConfigurationRecorderOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::start_configuration_recorder::StartConfigurationRecorderInput,
+        input: super::operation::start_configuration_recorder::StartConfigurationRecorderInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("StartConfigurationRecorder")
                 .build()
                 .expect("required fields set"),
@@ -142,13 +142,13 @@
                 StartConfigurationRecorderEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::start_configuration_recorder::StartConfigurationRecorderError,
+                super::operation::start_configuration_recorder::StartConfigurationRecorderError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::start_configuration_recorder::StartConfigurationRecorderError,
+                super::operation::start_configuration_recorder::StartConfigurationRecorderError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::start_configuration_recorder::StartConfigurationRecorderError,
+                super::operation::start_configuration_recorder::StartConfigurationRecorderError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -213,11 +213,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_start_configuration_recorder::de_start_configuration_recorder_http_error(status, headers, body)
+            super::protocol_serde::shape_start_configuration_recorder::de_start_configuration_recorder_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_start_configuration_recorder::de_start_configuration_recorder_http_response(status, headers, body)
+            super::protocol_serde::shape_start_configuration_recorder::de_start_configuration_recorder_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -230,16 +230,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::start_configuration_recorder::StartConfigurationRecorderInput>()
+            .downcast::<super::operation::start_configuration_recorder::StartConfigurationRecorderInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::start_configuration_recorder::StartConfigurationRecorderInput,
+                _input: &super::operation::start_configuration_recorder::StartConfigurationRecorderInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -248,7 +248,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::start_configuration_recorder::StartConfigurationRecorderInput,
+                input: &super::operation::start_configuration_recorder::StartConfigurationRecorderInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -265,7 +265,7 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from(
-            crate::protocol_serde::shape_start_configuration_recorder::ser_start_configuration_recorder_input(&input)?,
+            super::protocol_serde::shape_start_configuration_recorder::ser_start_configuration_recorder_input(&input)?,
         );
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
@@ -298,7 +298,7 @@
             .downcast_ref::<StartConfigurationRecorderInput>()
             .ok_or("failed to downcast to StartConfigurationRecorderInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -321,14 +321,14 @@
 #[derive(::std::fmt::Debug)]
 pub enum StartConfigurationRecorderError {
     /// <p>There is no delivery channel available to record configurations.</p>
-    NoAvailableDeliveryChannelException(crate::types::error::NoAvailableDeliveryChannelException),
+    NoAvailableDeliveryChannelException(super::types::error::NoAvailableDeliveryChannelException),
     /// <p>You have specified a configuration recorder that does not exist.</p>
-    NoSuchConfigurationRecorderException(crate::types::error::NoSuchConfigurationRecorderException),
+    NoSuchConfigurationRecorderException(super::types::error::NoSuchConfigurationRecorderException),
     /// <p>The requested operation is not valid.</p>
     /// <p>For <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_PutConfigurationRecorder.html">PutConfigurationRecorder</a>, you will see this exception because you cannot use this operation to create a service-linked configuration recorder. Use the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_PutServiceLinkedConfigurationRecorder.html">PutServiceLinkedConfigurationRecorder</a> operation to create a service-linked configuration recorder.</p>
     /// <p>For <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeleteConfigurationRecorder.html">DeleteConfigurationRecorder</a>, you will see this exception because you cannot use this operation to delete a service-linked configuration recorder. Use the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeleteServiceLinkedConfigurationRecorder.html">DeleteServiceLinkedConfigurationRecorder</a> operation to delete a service-linked configuration recorder.</p>
     /// <p>For <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_StartConfigurationRecorder.html">StartConfigurationRecorder</a> and <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_StopConfigurationRecorder.html">StopConfigurationRecorder</a>, you will see this exception because these operations do not affect service-linked configuration recorders. Service-linked configuration recorders are always recording. To stop recording, you must delete the service-linked configuration recorder. Use the <a href="https://docs.aws.amazon.com/config/latest/APIReference/API_DeleteServiceLinkedConfigurationRecorder.html">DeleteServiceLinkedConfigurationRecorder</a> operation to delete a service-linked configuration recorder.</p>
-    UnmodifiableEntityException(crate::types::error::UnmodifiableEntityException),
+    UnmodifiableEntityException(super::types::error::UnmodifiableEntityException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -336,7 +336,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-StartConfigurationRecorderError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl StartConfigurationRecorderError {
     /// Creates the `StartConfigurationRecorderError::Unhandled` variant from any error type.
@@ -343,7 +343,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -351,7 +351,7 @@

     /// Creates the `StartConfigurationRecorderError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -430,21 +430,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::start_configuration_recorder::StartConfigurationRecorderError {
+impl ::aws_types::request_id::RequestId for super::operation::start_configuration_recorder::StartConfigurationRecorderError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::start_configuration_recorder::_start_configuration_recorder_input::StartConfigurationRecorderInput;
+pub use super::operation::start_configuration_recorder::_start_configuration_recorder_input::StartConfigurationRecorderInput;

-pub use crate::operation::start_configuration_recorder::_start_configuration_recorder_output::StartConfigurationRecorderOutput;
+pub use super::operation::start_configuration_recorder::_start_configuration_recorder_output::StartConfigurationRecorderOutput;

 mod _start_configuration_recorder_input;

