--- original
+++ modified
@@ -33,8 +33,8 @@
 }
 impl DescribeConfigurationRecordersInput {
     /// Creates a new builder-style object to manufacture [`DescribeConfigurationRecordersInput`](crate::operation::describe_configuration_recorders::DescribeConfigurationRecordersInput).
-    pub fn builder() -> crate::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersInputBuilder {
-        crate::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersInputBuilder {
+        super::super::super::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersInputBuilder::default()
     }
 }

@@ -105,10 +105,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_configuration_recorders::DescribeConfigurationRecordersInput,
+        super::super::super::operation::describe_configuration_recorders::DescribeConfigurationRecordersInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_configuration_recorders::DescribeConfigurationRecordersInput {
+        ::std::result::Result::Ok(super::super::super::operation::describe_configuration_recorders::DescribeConfigurationRecordersInput {
             configuration_recorder_names: self.configuration_recorder_names,
             service_principal: self.service_principal,
             arn: self.arn,
