--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct GenerateCredentialReportInput {}
 impl GenerateCredentialReportInput {
     /// Creates a new builder-style object to manufacture [`GenerateCredentialReportInput`](crate::operation::generate_credential_report::GenerateCredentialReportInput).
-    pub fn builder() -> crate::operation::generate_credential_report::builders::GenerateCredentialReportInputBuilder {
-        crate::operation::generate_credential_report::builders::GenerateCredentialReportInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::generate_credential_report::builders::GenerateCredentialReportInputBuilder {
+        super::super::super::operation::generate_credential_report::builders::GenerateCredentialReportInputBuilder::default()
     }
 }

@@ -19,9 +19,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::generate_credential_report::GenerateCredentialReportInput,
+        super::super::super::operation::generate_credential_report::GenerateCredentialReportInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::generate_credential_report::GenerateCredentialReportInput {})
+        ::std::result::Result::Ok(super::super::super::operation::generate_credential_report::GenerateCredentialReportInput {})
     }
 }
