--- original
+++ modified
@@ -20,11 +20,11 @@
     // Both `config` and `cloneable` are the same config, but the cloneable one
     // is kept around so that it is possible to convert back into a builder. This can be
     // optimized in the future.
-    pub(crate) config: crate::config::FrozenLayer,
+    pub(crate) config: super::config::FrozenLayer,
     cloneable: ::aws_smithy_types::config_bag::CloneableLayer,
-    pub(crate) runtime_components: crate::config::RuntimeComponentsBuilder,
-    pub(crate) runtime_plugins: ::std::vec::Vec<crate::config::SharedRuntimePlugin>,
-    pub(crate) behavior_version: ::std::option::Option<crate::config::BehaviorVersion>,
+    pub(crate) runtime_components: super::config::RuntimeComponentsBuilder,
+    pub(crate) runtime_plugins: ::std::vec::Vec<super::config::SharedRuntimePlugin>,
+    pub(crate) behavior_version: ::std::option::Option<super::config::BehaviorVersion>,
 }
 impl Config {
     ///
@@ -47,11 +47,11 @@
         }
     }
     /// Return a reference to the stalled stream protection configuration contained in this config, if any.
-    pub fn stalled_stream_protection(&self) -> ::std::option::Option<&crate::config::StalledStreamProtectionConfig> {
-        self.config.load::<crate::config::StalledStreamProtectionConfig>()
+    pub fn stalled_stream_protection(&self) -> ::std::option::Option<&super::config::StalledStreamProtectionConfig> {
+        self.config.load::<super::config::StalledStreamProtectionConfig>()
     }
     /// Return the [`SharedHttpClient`](crate::config::SharedHttpClient) to use when making requests, if any.
-    pub fn http_client(&self) -> Option<crate::config::SharedHttpClient> {
+    pub fn http_client(&self) -> Option<super::config::SharedHttpClient> {
         self.runtime_components.http_client()
     }
     /// Return the auth schemes configured on this service config
@@ -77,7 +77,7 @@
     }

     /// Return a cloned shared async sleep implementation from this config, if any.
-    pub fn sleep_impl(&self) -> ::std::option::Option<crate::config::SharedAsyncSleep> {
+    pub fn sleep_impl(&self) -> ::std::option::Option<super::config::SharedAsyncSleep> {
         self.runtime_components.sleep_impl()
     }

@@ -94,11 +94,11 @@
         self.config.load::<::aws_smithy_runtime::client::retries::RetryPartition>()
     }
     /// Returns the configured identity cache for auth.
-    pub fn identity_cache(&self) -> ::std::option::Option<crate::config::SharedIdentityCache> {
+    pub fn identity_cache(&self) -> ::std::option::Option<super::config::SharedIdentityCache> {
         self.runtime_components.identity_cache()
     }
     /// Returns interceptors currently registered by the user.
-    pub fn interceptors(&self) -> impl Iterator<Item = crate::config::SharedInterceptor> + '_ {
+    pub fn interceptors(&self) -> impl Iterator<Item = super::config::SharedInterceptor> + '_ {
         self.runtime_components.interceptors()
     }
     /// Return time source used for this service.
@@ -148,14 +148,14 @@
         "logs"
     }
     /// Returns the AWS region, if it was provided.
-    pub fn region(&self) -> ::std::option::Option<&crate::config::Region> {
-        self.config.load::<crate::config::Region>()
+    pub fn region(&self) -> ::std::option::Option<&super::config::Region> {
+        self.config.load::<super::config::Region>()
     }
     /// This function was intended to be removed, and has been broken since release-2023-11-15 as it always returns a `None`. Do not use.
     #[deprecated(
         note = "This function was intended to be removed, and has been broken since release-2023-11-15 as it always returns a `None`. Do not use."
     )]
-    pub fn credentials_provider(&self) -> Option<crate::config::SharedCredentialsProvider> {
+    pub fn credentials_provider(&self) -> Option<super::config::SharedCredentialsProvider> {
         ::std::option::Option::None
     }
 }
@@ -163,15 +163,15 @@
 #[derive(::std::clone::Clone, ::std::fmt::Debug)]
 pub struct Builder {
     pub(crate) config: ::aws_smithy_types::config_bag::CloneableLayer,
-    pub(crate) runtime_components: crate::config::RuntimeComponentsBuilder,
-    pub(crate) runtime_plugins: ::std::vec::Vec<crate::config::SharedRuntimePlugin>,
-    pub(crate) behavior_version: ::std::option::Option<crate::config::BehaviorVersion>,
+    pub(crate) runtime_components: super::config::RuntimeComponentsBuilder,
+    pub(crate) runtime_plugins: ::std::vec::Vec<super::config::SharedRuntimePlugin>,
+    pub(crate) behavior_version: ::std::option::Option<super::config::BehaviorVersion>,
 }
 impl ::std::default::Default for Builder {
     fn default() -> Self {
         Self {
             config: ::std::default::Default::default(),
-            runtime_components: crate::config::RuntimeComponentsBuilder::new("service config"),
+            runtime_components: super::config::RuntimeComponentsBuilder::new("service config"),
             runtime_plugins: ::std::default::Default::default(),
             behavior_version: ::std::default::Default::default(),
         }
@@ -193,7 +193,7 @@
     #[allow(unused)]
     pub(crate) fn from_config_bag(config_bag: &::aws_smithy_types::config_bag::ConfigBag) -> Self {
         let mut builder = Self::new();
-        builder.set_stalled_stream_protection(config_bag.load::<crate::config::StalledStreamProtectionConfig>().cloned());
+        builder.set_stalled_stream_protection(config_bag.load::<super::config::StalledStreamProtectionConfig>().cloned());
         builder.set_auth_scheme_preference(config_bag.load::<::aws_smithy_runtime_api::client::auth::AuthSchemePreference>().cloned());
         builder.set_retry_config(config_bag.load::<::aws_smithy_types::retry::RetryConfig>().cloned());
         builder.set_timeout_config(config_bag.load::<::aws_smithy_types::timeout::TimeoutConfig>().cloned());
@@ -205,7 +205,7 @@
         builder.set_endpoint_url(config_bag.load::<::aws_types::endpoint_config::EndpointUrl>().map(|ty| ty.0.clone()));
         builder.set_use_dual_stack(config_bag.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0));
         builder.set_use_fips(config_bag.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0));
-        builder.set_region(config_bag.load::<crate::config::Region>().cloned());
+        builder.set_region(config_bag.load::<super::config::Region>().cloned());
         builder
     }
     /// Names operation-input members whose values are captured *and* emitted as
@@ -253,7 +253,7 @@
     }
     /// Set the [`StalledStreamProtectionConfig`](crate::config::StalledStreamProtectionConfig)
     /// to configure protection for stalled streams.
-    pub fn stalled_stream_protection(mut self, stalled_stream_protection_config: crate::config::StalledStreamProtectionConfig) -> Self {
+    pub fn stalled_stream_protection(mut self, stalled_stream_protection_config: super::config::StalledStreamProtectionConfig) -> Self {
         self.set_stalled_stream_protection(::std::option::Option::Some(stalled_stream_protection_config));
         self
     }
@@ -261,7 +261,7 @@
     /// to configure protection for stalled streams.
     pub fn set_stalled_stream_protection(
         &mut self,
-        stalled_stream_protection_config: ::std::option::Option<crate::config::StalledStreamProtectionConfig>,
+        stalled_stream_protection_config: ::std::option::Option<super::config::StalledStreamProtectionConfig>,
     ) -> &mut Self {
         self.config.store_or_unset(stalled_stream_protection_config);
         self
@@ -269,7 +269,7 @@
     /// Sets the idempotency token provider to use for service calls that require tokens.
     pub fn idempotency_token_provider(
         mut self,
-        idempotency_token_provider: impl ::std::convert::Into<crate::idempotency_token::IdempotencyTokenProvider>,
+        idempotency_token_provider: impl ::std::convert::Into<super::idempotency_token::IdempotencyTokenProvider>,
     ) -> Self {
         self.set_idempotency_token_provider(::std::option::Option::Some(idempotency_token_provider.into()));
         self
@@ -277,7 +277,7 @@
     /// Sets the idempotency token provider to use for service calls that require tokens.
     pub fn set_idempotency_token_provider(
         &mut self,
-        idempotency_token_provider: ::std::option::Option<crate::idempotency_token::IdempotencyTokenProvider>,
+        idempotency_token_provider: ::std::option::Option<super::idempotency_token::IdempotencyTokenProvider>,
     ) -> &mut Self {
         self.config.store_or_unset(idempotency_token_provider);
         self
@@ -311,8 +311,8 @@
     /// # }
     /// # }
     /// ```
-    pub fn http_client(mut self, http_client: impl crate::config::HttpClient + 'static) -> Self {
-        self.set_http_client(::std::option::Option::Some(crate::config::IntoShared::into_shared(http_client)));
+    pub fn http_client(mut self, http_client: impl super::config::HttpClient + 'static) -> Self {
+        self.set_http_client(::std::option::Option::Some(super::config::IntoShared::into_shared(http_client)));
         self
     }

@@ -345,7 +345,7 @@
     /// # }
     /// # }
     /// ```
-    pub fn set_http_client(&mut self, http_client: Option<crate::config::SharedHttpClient>) -> &mut Self {
+    pub fn set_http_client(&mut self, http_client: Option<super::config::SharedHttpClient>) -> &mut Self {
         self.runtime_components.set_http_client(http_client);
         self
     }
@@ -492,7 +492,7 @@
     ///     // other configurations
     ///     .build();
     /// ```
-    pub fn auth_scheme_resolver(mut self, auth_scheme_resolver: impl crate::config::auth::ResolveAuthScheme + 'static) -> Self {
+    pub fn auth_scheme_resolver(mut self, auth_scheme_resolver: impl super::config::auth::ResolveAuthScheme + 'static) -> Self {
         self.set_auth_scheme_resolver(auth_scheme_resolver);
         self
     }
@@ -501,7 +501,7 @@
     ///
     /// # Examples
     /// See an example for [`Self::auth_scheme_resolver`].
-    pub fn set_auth_scheme_resolver(&mut self, auth_scheme_resolver: impl crate::config::auth::ResolveAuthScheme + 'static) -> &mut Self {
+    pub fn set_auth_scheme_resolver(&mut self, auth_scheme_resolver: impl super::config::auth::ResolveAuthScheme + 'static) -> &mut Self {
         self.runtime_components
             .set_auth_scheme_option_resolver(::std::option::Option::Some(auth_scheme_resolver.into_shared_resolver()));
         self
@@ -608,7 +608,7 @@
     /// let config = aws_sdk_cloudwatchlogs::Config::builder().endpoint_resolver(resolver).build();
     /// let client = aws_sdk_cloudwatchlogs::Client::from_conf(config);
     /// ```
-    pub fn endpoint_resolver(mut self, endpoint_resolver: impl crate::config::endpoint::ResolveEndpoint + 'static) -> Self {
+    pub fn endpoint_resolver(mut self, endpoint_resolver: impl super::config::endpoint::ResolveEndpoint + 'static) -> Self {
         self.set_endpoint_resolver(::std::option::Option::Some(endpoint_resolver.into_shared_resolver()));
         self
     }
@@ -689,7 +689,7 @@
     /// let sleep_impl = SharedAsyncSleep::new(ForeverSleep);
     /// let config = Config::builder().sleep_impl(sleep_impl).build();
     /// ```
-    pub fn sleep_impl(mut self, sleep_impl: impl crate::config::AsyncSleep + 'static) -> Self {
+    pub fn sleep_impl(mut self, sleep_impl: impl super::config::AsyncSleep + 'static) -> Self {
         self.set_sleep_impl(Some(::aws_smithy_runtime_api::shared::IntoShared::into_shared(sleep_impl)));
         self
     }
@@ -719,7 +719,7 @@
     /// set_never_ending_sleep_impl(&mut builder);
     /// let config = builder.build();
     /// ```
-    pub fn set_sleep_impl(&mut self, sleep_impl: ::std::option::Option<crate::config::SharedAsyncSleep>) -> &mut Self {
+    pub fn set_sleep_impl(&mut self, sleep_impl: ::std::option::Option<super::config::SharedAsyncSleep>) -> &mut Self {
         self.runtime_components.set_sleep_impl(sleep_impl);
         self
     }
@@ -891,7 +891,7 @@
     /// let client = aws_sdk_cloudwatchlogs::Client::from_conf(config);
     /// ```
     ///
-    pub fn identity_cache(mut self, identity_cache: impl crate::config::ResolveCachedIdentity + 'static) -> Self {
+    pub fn identity_cache(mut self, identity_cache: impl super::config::ResolveCachedIdentity + 'static) -> Self {
         self.set_identity_cache(identity_cache);
         self
     }
@@ -937,7 +937,7 @@
     /// let client = aws_sdk_cloudwatchlogs::Client::from_conf(config);
     /// ```
     ///
-    pub fn set_identity_cache(&mut self, identity_cache: impl crate::config::ResolveCachedIdentity + 'static) -> &mut Self {
+    pub fn set_identity_cache(&mut self, identity_cache: impl super::config::ResolveCachedIdentity + 'static) -> &mut Self {
         self.runtime_components.set_identity_cache(::std::option::Option::Some(identity_cache));
         self
     }
@@ -988,19 +988,19 @@
     ///     .build();
     /// # }
     /// ```
-    pub fn interceptor(mut self, interceptor: impl crate::config::Intercept + 'static) -> Self {
-        self.push_interceptor(crate::config::SharedInterceptor::new(interceptor));
+    pub fn interceptor(mut self, interceptor: impl super::config::Intercept + 'static) -> Self {
+        self.push_interceptor(super::config::SharedInterceptor::new(interceptor));
         self
     }

     /// Like [`Self::interceptor`], but takes a [`SharedInterceptor`](crate::config::SharedInterceptor).
-    pub fn push_interceptor(&mut self, interceptor: crate::config::SharedInterceptor) -> &mut Self {
+    pub fn push_interceptor(&mut self, interceptor: super::config::SharedInterceptor) -> &mut Self {
         self.runtime_components.push_interceptor(interceptor);
         self
     }

     /// Set [`SharedInterceptor`](crate::config::SharedInterceptor)s for the builder.
-    pub fn set_interceptors(&mut self, interceptors: impl IntoIterator<Item = crate::config::SharedInterceptor>) -> &mut Self {
+    pub fn set_interceptors(&mut self, interceptors: impl IntoIterator<Item = super::config::SharedInterceptor>) -> &mut Self {
         self.runtime_components.set_interceptors(interceptors.into_iter());
         self
     }
@@ -1241,24 +1241,24 @@
     ///     .region(Region::new("us-east-1"))
     ///     .build();
     /// ```
-    pub fn region(mut self, region: impl ::std::convert::Into<::std::option::Option<crate::config::Region>>) -> Self {
+    pub fn region(mut self, region: impl ::std::convert::Into<::std::option::Option<super::config::Region>>) -> Self {
         self.set_region(region.into());
         self
     }
     /// Sets the AWS region to use when making requests.
-    pub fn set_region(&mut self, region: ::std::option::Option<crate::config::Region>) -> &mut Self {
+    pub fn set_region(&mut self, region: ::std::option::Option<super::config::Region>) -> &mut Self {
         self.config.store_or_unset(region);
         self
     }
     /// Sets the credentials provider for this service
-    pub fn credentials_provider(mut self, credentials_provider: impl crate::config::ProvideCredentials + 'static) -> Self {
-        self.set_credentials_provider(::std::option::Option::Some(crate::config::SharedCredentialsProvider::new(
+    pub fn credentials_provider(mut self, credentials_provider: impl super::config::ProvideCredentials + 'static) -> Self {
+        self.set_credentials_provider(::std::option::Option::Some(super::config::SharedCredentialsProvider::new(
             credentials_provider,
         )));
         self
     }
     /// Sets the credentials provider for this service
-    pub fn set_credentials_provider(&mut self, credentials_provider: ::std::option::Option<crate::config::SharedCredentialsProvider>) -> &mut Self {
+    pub fn set_credentials_provider(&mut self, credentials_provider: ::std::option::Option<super::config::SharedCredentialsProvider>) -> &mut Self {
         if let Some(credentials_provider) = credentials_provider {
             self.runtime_components
                 .set_identity_resolver(::aws_runtime::auth::sigv4::SCHEME_ID, credentials_provider);
@@ -1295,7 +1295,7 @@
     /// let client = aws_sdk_cloudwatchlogs::Client::from_conf(config);
     /// ```
     ///
-    pub fn behavior_version(mut self, behavior_version: crate::config::BehaviorVersion) -> Self {
+    pub fn behavior_version(mut self, behavior_version: super::config::BehaviorVersion) -> Self {
         self.set_behavior_version(Some(behavior_version));
         self
     }
@@ -1330,7 +1330,7 @@
     /// let client = aws_sdk_cloudwatchlogs::Client::from_conf(config);
     /// ```
     ///
-    pub fn set_behavior_version(&mut self, behavior_version: Option<crate::config::BehaviorVersion>) -> &mut Self {
+    pub fn set_behavior_version(&mut self, behavior_version: Option<super::config::BehaviorVersion>) -> &mut Self {
         self.behavior_version = behavior_version;
         self
     }
@@ -1339,18 +1339,18 @@
     ///
     /// This is equivalent to enabling the `behavior-version-latest` Cargo feature
     pub fn behavior_version_latest(mut self) -> Self {
-        self.set_behavior_version(Some(crate::config::BehaviorVersion::latest()));
+        self.set_behavior_version(Some(super::config::BehaviorVersion::latest()));
         self
     }
     /// Adds a runtime plugin to the config.
     #[allow(unused)]
-    pub(crate) fn runtime_plugin(mut self, plugin: impl crate::config::RuntimePlugin + 'static) -> Self {
-        self.push_runtime_plugin(crate::config::SharedRuntimePlugin::new(plugin));
+    pub(crate) fn runtime_plugin(mut self, plugin: impl super::config::RuntimePlugin + 'static) -> Self {
+        self.push_runtime_plugin(super::config::SharedRuntimePlugin::new(plugin));
         self
     }
     /// Adds a runtime plugin to the config.
     #[allow(unused)]
-    pub(crate) fn push_runtime_plugin(&mut self, plugin: crate::config::SharedRuntimePlugin) -> &mut Self {
+    pub(crate) fn push_runtime_plugin(&mut self, plugin: super::config::SharedRuntimePlugin) -> &mut Self {
         self.runtime_plugins.push(plugin);
         self
     }
@@ -1363,10 +1363,10 @@
             ::aws_smithy_async::time::StaticTimeSource::new(::std::time::UNIX_EPOCH + ::std::time::Duration::from_secs(1234567890)),
         )));
         self.config.store_put(::aws_runtime::user_agent::AwsUserAgent::for_tests());
-        self.set_credentials_provider(Some(crate::config::SharedCredentialsProvider::new(
+        self.set_credentials_provider(Some(super::config::SharedCredentialsProvider::new(
             ::aws_credential_types::Credentials::for_tests(),
         )));
-        self.behavior_version = ::std::option::Option::Some(crate::config::BehaviorVersion::latest());
+        self.behavior_version = ::std::option::Option::Some(super::config::BehaviorVersion::latest());
         self
     }
     #[cfg(any(feature = "test-util", test))]
@@ -1381,8 +1381,8 @@
     /// Apply test defaults to the builder. V2 of this function sets additional test defaults such as region configuration (if applicable).
     pub fn apply_test_defaults_v2(&mut self) -> &mut Self {
         self.apply_test_defaults();
-        if self.config.load::<crate::config::Region>().is_none() {
-            self.set_region(::std::option::Option::Some(crate::config::Region::new("us-east-1")));
+        if self.config.load::<super::config::Region>().is_none() {
+            self.set_region(::std::option::Option::Some(super::config::Region::new("us-east-1")));
         }
         self
     }
@@ -1401,7 +1401,7 @@
             self.runtime_components
                 .set_time_source(::std::option::Option::Some(::std::default::Default::default()));
         }
-        layer.store_put(crate::meta::API_METADATA.clone());
+        layer.store_put(super::meta::API_METADATA.clone());
         layer.store_put(::aws_types::SigningName::from_static("logs"));
         layer
             .load::<::aws_types::region::Region>()
@@ -1408,7 +1408,7 @@
             .cloned()
             .map(|r| layer.store_put(::aws_types::region::SigningRegion::from(r)));
         Config {
-            config: crate::config::Layer::from(layer.clone())
+            config: super::config::Layer::from(layer.clone())
                 .with_name("aws_sdk_cloudwatchlogs::config::Config")
                 .freeze(),
             cloneable: layer,
@@ -1425,21 +1425,21 @@
 }

 impl ServiceRuntimePlugin {
-    pub fn new(_service_config: crate::config::Config) -> Self {
+    pub fn new(_service_config: super::config::Config) -> Self {
         let config = {
             let mut cfg = ::aws_smithy_types::config_bag::Layer::new("Logs_20140328");
-            cfg.store_put(crate::idempotency_token::default_provider());
+            cfg.store_put(super::idempotency_token::default_provider());
             cfg.store_put(::aws_smithy_runtime::client::orchestrator::AuthSchemeAndEndpointOrchestrationV2);
             ::std::option::Option::Some(cfg.freeze())
         };
         let mut runtime_components = ::aws_smithy_runtime_api::client::runtime_components::RuntimeComponentsBuilder::new("ServiceRuntimePlugin");
         runtime_components.set_auth_scheme_option_resolver(::std::option::Option::Some({
-            use crate::config::auth::ResolveAuthScheme;
-            crate::config::auth::DefaultAuthSchemeResolver::default().into_shared_resolver()
+            use super::config::auth::ResolveAuthScheme;
+            super::config::auth::DefaultAuthSchemeResolver::default().into_shared_resolver()
         }));
         runtime_components.set_endpoint_resolver(::std::option::Option::Some({
-            use crate::config::endpoint::ResolveEndpoint;
-            crate::config::endpoint::DefaultResolver::new().into_shared_resolver()
+            use super::config::endpoint::ResolveEndpoint;
+            super::config::endpoint::DefaultResolver::new().into_shared_resolver()
         }));
         runtime_components.push_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(
             ::aws_smithy_runtime::client::http::connection_poisoning::ConnectionPoisoningInterceptor::new(),
@@ -1446,7 +1446,7 @@
         ));
         runtime_components.push_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::HttpStatusCodeClassifier::default());
         runtime_components.push_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(
-            crate::sdk_feature_tracker::retry_mode::RetryModeFeatureTrackerInterceptor::new(),
+            super::sdk_feature_tracker::retry_mode::RetryModeFeatureTrackerInterceptor::new(),
         ));
         runtime_components.push_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(
             ::aws_runtime::service_clock_skew::ServiceClockSkewInterceptor::new(),
@@ -1461,10 +1461,10 @@
             ::aws_runtime::auth::sigv4::SigV4AuthScheme::new(),
         ));
         runtime_components.push_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(
-            crate::config::endpoint::EndpointOverrideFeatureTrackerInterceptor,
+            super::config::endpoint::EndpointOverrideFeatureTrackerInterceptor,
         ));
         runtime_components.push_interceptor(::aws_smithy_runtime_api::client::interceptors::SharedInterceptor::permanent(
-            crate::observability_feature::ObservabilityFeatureTrackerInterceptor,
+            super::observability_feature::ObservabilityFeatureTrackerInterceptor,
         ));
         Self { config, runtime_components }
     }
@@ -1628,7 +1628,7 @@

 pub use ::aws_smithy_async::rt::sleep::Sleep;

-pub(crate) fn base_client_runtime_plugins(mut config: crate::Config) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
+pub(crate) fn base_client_runtime_plugins(mut config: super::Config) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
     let mut configured_plugins = ::std::vec::Vec::new();
     ::std::mem::swap(&mut config.runtime_plugins, &mut configured_plugins);
     #[cfg(feature = "behavior-version-latest")]
@@ -1662,7 +1662,7 @@
                                 .with_runtime_components(config.runtime_components.clone())
                         )
                         // codegen config
-                        .with_client_plugin(crate::config::ServiceRuntimePlugin::new(config.clone()))
+                        .with_client_plugin(super::config::ServiceRuntimePlugin::new(config.clone()))
                         .with_client_plugin(::aws_smithy_runtime::client::auth::no_auth::NoAuthRuntimePlugin::new())
                         .with_client_plugin(
                             ::aws_smithy_runtime::client::metrics::MetricsRuntimePlugin::builder()
