--- original
+++ modified
@@ -3,8 +3,8 @@
 /// Matcher union: {"success":true}
 pub(crate) fn match_get_instance_profile_c955e57777ec0d736(
     _result: ::std::result::Result<
-        &crate::operation::get_instance_profile::GetInstanceProfileOutput,
-        &crate::operation::get_instance_profile::GetInstanceProfileError,
+        &super::operation::get_instance_profile::GetInstanceProfileOutput,
+        &super::operation::get_instance_profile::GetInstanceProfileError,
     >,
 ) -> bool {
     _result.is_ok()
@@ -13,8 +13,8 @@
 /// Matcher union: {"errorType":"NoSuchEntityException"}
 pub(crate) fn match_get_instance_profile_73f4bff11904ca908(
     _result: ::std::result::Result<
-        &crate::operation::get_instance_profile::GetInstanceProfileOutput,
-        &crate::operation::get_instance_profile::GetInstanceProfileError,
+        &super::operation::get_instance_profile::GetInstanceProfileOutput,
+        &super::operation::get_instance_profile::GetInstanceProfileError,
     >,
 ) -> bool {
     if let ::std::result::Result::Err(err) = _result {
@@ -27,7 +27,7 @@

 /// Matcher union: {"success":true}
 pub(crate) fn match_get_policy_c955e57777ec0d736(
-    _result: ::std::result::Result<&crate::operation::get_policy::GetPolicyOutput, &crate::operation::get_policy::GetPolicyError>,
+    _result: ::std::result::Result<&super::operation::get_policy::GetPolicyOutput, &super::operation::get_policy::GetPolicyError>,
 ) -> bool {
     _result.is_ok()
 }
@@ -34,7 +34,7 @@

 /// Matcher union: {"errorType":"NoSuchEntity"}
 pub(crate) fn match_get_policy_606386b4be9df73c9(
-    _result: ::std::result::Result<&crate::operation::get_policy::GetPolicyOutput, &crate::operation::get_policy::GetPolicyError>,
+    _result: ::std::result::Result<&super::operation::get_policy::GetPolicyOutput, &super::operation::get_policy::GetPolicyError>,
 ) -> bool {
     if let ::std::result::Result::Err(err) = _result {
         if let ::std::option::Option::Some(code) = ::aws_smithy_types::error::metadata::ProvideErrorMetadata::code(err) {
@@ -46,7 +46,7 @@

 /// Matcher union: {"success":true}
 pub(crate) fn match_get_role_c955e57777ec0d736(
-    _result: ::std::result::Result<&crate::operation::get_role::GetRoleOutput, &crate::operation::get_role::GetRoleError>,
+    _result: ::std::result::Result<&super::operation::get_role::GetRoleOutput, &super::operation::get_role::GetRoleError>,
 ) -> bool {
     _result.is_ok()
 }
@@ -53,7 +53,7 @@

 /// Matcher union: {"errorType":"NoSuchEntity"}
 pub(crate) fn match_get_role_606386b4be9df73c9(
-    _result: ::std::result::Result<&crate::operation::get_role::GetRoleOutput, &crate::operation::get_role::GetRoleError>,
+    _result: ::std::result::Result<&super::operation::get_role::GetRoleOutput, &super::operation::get_role::GetRoleError>,
 ) -> bool {
     if let ::std::result::Result::Err(err) = _result {
         if let ::std::option::Option::Some(code) = ::aws_smithy_types::error::metadata::ProvideErrorMetadata::code(err) {
@@ -65,7 +65,7 @@

 /// Matcher union: {"success":true}
 pub(crate) fn match_get_user_c955e57777ec0d736(
-    _result: ::std::result::Result<&crate::operation::get_user::GetUserOutput, &crate::operation::get_user::GetUserError>,
+    _result: ::std::result::Result<&super::operation::get_user::GetUserOutput, &super::operation::get_user::GetUserError>,
 ) -> bool {
     _result.is_ok()
 }
@@ -72,7 +72,7 @@

 /// Matcher union: {"errorType":"NoSuchEntity"}
 pub(crate) fn match_get_user_606386b4be9df73c9(
-    _result: ::std::result::Result<&crate::operation::get_user::GetUserOutput, &crate::operation::get_user::GetUserError>,
+    _result: ::std::result::Result<&super::operation::get_user::GetUserOutput, &super::operation::get_user::GetUserError>,
 ) -> bool {
     if let ::std::result::Result::Err(err) = _result {
         if let ::std::option::Option::Some(code) = ::aws_smithy_types::error::metadata::ProvideErrorMetadata::code(err) {
