--- original
+++ modified
@@ -48,8 +48,8 @@
 }
 impl ListKeyPoliciesInput {
     /// Creates a new builder-style object to manufacture [`ListKeyPoliciesInput`](crate::operation::list_key_policies::ListKeyPoliciesInput).
-    pub fn builder() -> crate::operation::list_key_policies::builders::ListKeyPoliciesInputBuilder {
-        crate::operation::list_key_policies::builders::ListKeyPoliciesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_key_policies::builders::ListKeyPoliciesInputBuilder {
+        super::super::super::operation::list_key_policies::builders::ListKeyPoliciesInputBuilder::default()
     }
 }

@@ -141,8 +141,8 @@
     /// Consumes the builder and constructs a [`ListKeyPoliciesInput`](crate::operation::list_key_policies::ListKeyPoliciesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_key_policies::ListKeyPoliciesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_key_policies::ListKeyPoliciesInput {
+    ) -> ::std::result::Result<super::super::super::operation::list_key_policies::ListKeyPoliciesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_key_policies::ListKeyPoliciesInput {
             key_id: self.key_id,
             limit: self.limit,
             marker: self.marker,
