--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<DeleteRemediationConfigurationError>`](crate::operation::delete_remediation_configuration::DeleteRemediationConfigurationError)
     pub fn delete_remediation_configuration(
         &self,
-    ) -> crate::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationFluentBuilder {
-        crate::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationFluentBuilder {
+        super::super::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationFluentBuilder::new(self.handle.clone())
     }
 }
