--- original
+++ modified
@@ -11,8 +11,8 @@
     /// - On failure, responds with [`SdkError<DeleteServiceLinkedConfigurationRecorderError>`](crate::operation::delete_service_linked_configuration_recorder::DeleteServiceLinkedConfigurationRecorderError)
     pub fn delete_service_linked_configuration_recorder(
         &self,
-    ) -> crate::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderFluentBuilder {
-        crate::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderFluentBuilder::new(
+    ) -> super::super::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderFluentBuilder {
+        super::super::operation::delete_service_linked_configuration_recorder::builders::DeleteServiceLinkedConfigurationRecorderFluentBuilder::new(
             self.handle.clone(),
         )
     }
