--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On failure, responds with [`SdkError<DeleteOrganizationConfigRuleError>`](crate::operation::delete_organization_config_rule::DeleteOrganizationConfigRuleError)
     pub fn delete_organization_config_rule(
         &self,
-    ) -> crate::operation::delete_organization_config_rule::builders::DeleteOrganizationConfigRuleFluentBuilder {
-        crate::operation::delete_organization_config_rule::builders::DeleteOrganizationConfigRuleFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::delete_organization_config_rule::builders::DeleteOrganizationConfigRuleFluentBuilder {
+        super::super::operation::delete_organization_config_rule::builders::DeleteOrganizationConfigRuleFluentBuilder::new(self.handle.clone())
     }
 }
