--- original
+++ modified
@@ -14,8 +14,8 @@
     /// - On failure, responds with [`SdkError<DescribeAggregateComplianceByConformancePacksError>`](crate::operation::describe_aggregate_compliance_by_conformance_packs::DescribeAggregateComplianceByConformancePacksError)
     pub fn describe_aggregate_compliance_by_conformance_packs(
         &self,
-    ) -> crate::operation::describe_aggregate_compliance_by_conformance_packs::builders::DescribeAggregateComplianceByConformancePacksFluentBuilder
+    ) -> super::operation::describe_aggregate_compliance_by_conformance_packs::builders::DescribeAggregateComplianceByConformancePacksFluentBuilder
     {
-        crate::operation::describe_aggregate_compliance_by_conformance_packs::builders::DescribeAggregateComplianceByConformancePacksFluentBuilder::new(self.handle.clone())
+        super::operation::describe_aggregate_compliance_by_conformance_packs::builders::DescribeAggregateComplianceByConformancePacksFluentBuilder::new(self.handle.clone())
     }
 }
