--- original
+++ modified
@@ -13,8 +13,8 @@
     /// - On failure, responds with [`SdkError<DescribeOrganizationConformancePackStatusesError>`](crate::operation::describe_organization_conformance_pack_statuses::DescribeOrganizationConformancePackStatusesError)
     pub fn describe_organization_conformance_pack_statuses(
         &self,
-    ) -> crate::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesFluentBuilder {
-        crate::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesFluentBuilder::new(
+    ) -> super::super::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesFluentBuilder {
+        super::super::operation::describe_organization_conformance_pack_statuses::builders::DescribeOrganizationConformancePackStatusesFluentBuilder::new(
             self.handle.clone(),
         )
     }
