--- original
+++ modified
@@ -13,8 +13,8 @@
     /// - On failure, responds with [`SdkError<DescribeOrganizationConformancePacksError>`](crate::operation::describe_organization_conformance_packs::DescribeOrganizationConformancePacksError)
     pub fn describe_organization_conformance_packs(
         &self,
-    ) -> crate::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksFluentBuilder {
-        crate::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksFluentBuilder::new(
+    ) -> super::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksFluentBuilder {
+        super::operation::describe_organization_conformance_packs::builders::DescribeOrganizationConformancePacksFluentBuilder::new(
             self.handle.clone(),
         )
     }
