--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On failure, responds with [`SdkError<DescribeConfigurationRecordersError>`](crate::operation::describe_configuration_recorders::DescribeConfigurationRecordersError)
     pub fn describe_configuration_recorders(
         &self,
-    ) -> crate::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersFluentBuilder {
-        crate::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersFluentBuilder {
+        super::super::operation::describe_configuration_recorders::builders::DescribeConfigurationRecordersFluentBuilder::new(self.handle.clone())
     }
 }
