--- original
+++ modified
@@ -16,9 +16,9 @@
     /// - On failure, responds with [`SdkError<GetAggregateConformancePackComplianceSummaryError>`](crate::operation::get_aggregate_conformance_pack_compliance_summary::GetAggregateConformancePackComplianceSummaryError)
     pub fn get_aggregate_conformance_pack_compliance_summary(
         &self,
-    ) -> crate::operation::get_aggregate_conformance_pack_compliance_summary::builders::GetAggregateConformancePackComplianceSummaryFluentBuilder
+    ) -> super::operation::get_aggregate_conformance_pack_compliance_summary::builders::GetAggregateConformancePackComplianceSummaryFluentBuilder
     {
-        crate::operation::get_aggregate_conformance_pack_compliance_summary::builders::GetAggregateConformancePackComplianceSummaryFluentBuilder::new(
+        super::operation::get_aggregate_conformance_pack_compliance_summary::builders::GetAggregateConformancePackComplianceSummaryFluentBuilder::new(
             self.handle.clone(),
         )
     }
