--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<GetOrganizationCustomRulePolicyError>`](crate::operation::get_organization_custom_rule_policy::GetOrganizationCustomRulePolicyError)
     pub fn get_organization_custom_rule_policy(
         &self,
-    ) -> crate::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyFluentBuilder {
-        crate::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyFluentBuilder {
+        super::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyFluentBuilder::new(self.handle.clone())
     }
 }
