--- original
+++ modified
@@ -15,8 +15,8 @@
     /// - On failure, responds with [`SdkError<ListConformancePackComplianceScoresError>`](crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError)
     pub fn list_conformance_pack_compliance_scores(
         &self,
-    ) -> crate::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresFluentBuilder {
-        crate::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresFluentBuilder::new(
+    ) -> super::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresFluentBuilder {
+        super::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresFluentBuilder::new(
             self.handle.clone(),
         )
     }
