--- original
+++ modified
@@ -15,7 +15,7 @@
     /// - On failure, responds with [`SdkError<ListAggregateDiscoveredResourcesError>`](crate::operation::list_aggregate_discovered_resources::ListAggregateDiscoveredResourcesError)
     pub fn list_aggregate_discovered_resources(
         &self,
-    ) -> crate::operation::list_aggregate_discovered_resources::builders::ListAggregateDiscoveredResourcesFluentBuilder {
-        crate::operation::list_aggregate_discovered_resources::builders::ListAggregateDiscoveredResourcesFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::list_aggregate_discovered_resources::builders::ListAggregateDiscoveredResourcesFluentBuilder {
+        super::operation::list_aggregate_discovered_resources::builders::ListAggregateDiscoveredResourcesFluentBuilder::new(self.handle.clone())
     }
 }
