--- original
+++ modified
@@ -12,7 +12,7 @@
     /// - On success, responds with [`PutOrganizationConfigRuleOutput`](crate::operation::put_organization_config_rule::PutOrganizationConfigRuleOutput) with field(s):
     ///   - [`organization_config_rule_arn(Option<String>)`](crate::operation::put_organization_config_rule::PutOrganizationConfigRuleOutput::organization_config_rule_arn): <p>The Amazon Resource Name (ARN) of an organization Config rule.</p>
     /// - On failure, responds with [`SdkError<PutOrganizationConfigRuleError>`](crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError)
-    pub fn put_organization_config_rule(&self) -> crate::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleFluentBuilder {
-        crate::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleFluentBuilder::new(self.handle.clone())
+    pub fn put_organization_config_rule(&self) -> super::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleFluentBuilder {
+        super::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleFluentBuilder::new(self.handle.clone())
     }
 }
