--- original
+++ modified
@@ -13,8 +13,8 @@
     /// - On failure, responds with [`SdkError<PutThirdPartyServiceLinkedConfigurationRecorderError>`](crate::operation::put_third_party_service_linked_configuration_recorder::PutThirdPartyServiceLinkedConfigurationRecorderError)
     pub fn put_third_party_service_linked_configuration_recorder(
         &self,
-    ) -> crate::operation::put_third_party_service_linked_configuration_recorder::builders::PutThirdPartyServiceLinkedConfigurationRecorderFluentBuilder
+    ) -> super::super::operation::put_third_party_service_linked_configuration_recorder::builders::PutThirdPartyServiceLinkedConfigurationRecorderFluentBuilder
     {
-        crate::operation::put_third_party_service_linked_configuration_recorder::builders::PutThirdPartyServiceLinkedConfigurationRecorderFluentBuilder::new(self.handle.clone())
+        super::super::operation::put_third_party_service_linked_configuration_recorder::builders::PutThirdPartyServiceLinkedConfigurationRecorderFluentBuilder::new(self.handle.clone())
     }
 }
