--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On failure, responds with [`SdkError<PutAggregationAuthorizationError>`](crate::operation::put_aggregation_authorization::PutAggregationAuthorizationError)
     pub fn put_aggregation_authorization(
         &self,
-    ) -> crate::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationFluentBuilder {
-        crate::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationFluentBuilder {
+        super::super::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationFluentBuilder::new(self.handle.clone())
     }
 }
