--- original
+++ modified
@@ -10,7 +10,7 @@
     /// - On success, responds with [`PutRemediationExceptionsOutput`](crate::operation::put_remediation_exceptions::PutRemediationExceptionsOutput) with field(s):
     ///   - [`failed_batches(Option<Vec::<FailedRemediationExceptionBatch>>)`](crate::operation::put_remediation_exceptions::PutRemediationExceptionsOutput::failed_batches): <p>Returns a list of failed remediation exceptions batch objects. Each object in the batch consists of a list of failed items and failure messages.</p>
     /// - On failure, responds with [`SdkError<PutRemediationExceptionsError>`](crate::operation::put_remediation_exceptions::PutRemediationExceptionsError)
-    pub fn put_remediation_exceptions(&self) -> crate::operation::put_remediation_exceptions::builders::PutRemediationExceptionsFluentBuilder {
-        crate::operation::put_remediation_exceptions::builders::PutRemediationExceptionsFluentBuilder::new(self.handle.clone())
+    pub fn put_remediation_exceptions(&self) -> super::operation::put_remediation_exceptions::builders::PutRemediationExceptionsFluentBuilder {
+        super::operation::put_remediation_exceptions::builders::PutRemediationExceptionsFluentBuilder::new(self.handle.clone())
     }
 }
