--- original
+++ modified
@@ -16,7 +16,7 @@
     /// - On failure, responds with [`SdkError<SelectAggregateResourceConfigError>`](crate::operation::select_aggregate_resource_config::SelectAggregateResourceConfigError)
     pub fn select_aggregate_resource_config(
         &self,
-    ) -> crate::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigFluentBuilder {
-        crate::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigFluentBuilder {
+        super::super::operation::select_aggregate_resource_config::builders::SelectAggregateResourceConfigFluentBuilder::new(self.handle.clone())
     }
 }
