--- original
+++ modified
@@ -35,8 +35,8 @@
 }
 impl AdminRemoveUserFromGroupInput {
     /// Creates a new builder-style object to manufacture [`AdminRemoveUserFromGroupInput`](crate::operation::admin_remove_user_from_group::AdminRemoveUserFromGroupInput).
-    pub fn builder() -> crate::operation::admin_remove_user_from_group::builders::AdminRemoveUserFromGroupInputBuilder {
-        crate::operation::admin_remove_user_from_group::builders::AdminRemoveUserFromGroupInputBuilder::default()
+    pub fn builder() -> super::operation::admin_remove_user_from_group::builders::AdminRemoveUserFromGroupInputBuilder {
+        super::operation::admin_remove_user_from_group::builders::AdminRemoveUserFromGroupInputBuilder::default()
     }
 }

@@ -98,10 +98,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::admin_remove_user_from_group::AdminRemoveUserFromGroupInput,
+        super::operation::admin_remove_user_from_group::AdminRemoveUserFromGroupInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::admin_remove_user_from_group::AdminRemoveUserFromGroupInput {
+        ::std::result::Result::Ok(super::operation::admin_remove_user_from_group::AdminRemoveUserFromGroupInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
             group_name: self.group_name,
