--- original
+++ modified
@@ -48,8 +48,8 @@
 }
 impl CreateAliasInput {
     /// Creates a new builder-style object to manufacture [`CreateAliasInput`](crate::operation::create_alias::CreateAliasInput).
-    pub fn builder() -> crate::operation::create_alias::builders::CreateAliasInputBuilder {
-        crate::operation::create_alias::builders::CreateAliasInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::create_alias::builders::CreateAliasInputBuilder {
+        super::super::super::operation::create_alias::builders::CreateAliasInputBuilder::default()
     }
 }

@@ -134,8 +134,8 @@
         &self.target_key_id
     }
     /// Consumes the builder and constructs a [`CreateAliasInput`](crate::operation::create_alias::CreateAliasInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::create_alias::CreateAliasInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_alias::CreateAliasInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::create_alias::CreateAliasInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::create_alias::CreateAliasInput {
             alias_name: self.alias_name,
             target_key_id: self.target_key_id,
         })
