--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetBucketPolicyStatusInput {
     /// Creates a new builder-style object to manufacture [`GetBucketPolicyStatusInput`](crate::operation::get_bucket_policy_status::GetBucketPolicyStatusInput).
-    pub fn builder() -> crate::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusInputBuilder {
-        crate::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusInputBuilder::default()
+    pub fn builder() -> super::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusInputBuilder {
+        super::operation::get_bucket_policy_status::builders::GetBucketPolicyStatusInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`GetBucketPolicyStatusInput`](crate::operation::get_bucket_policy_status::GetBucketPolicyStatusInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_bucket_policy_status::GetBucketPolicyStatusInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_bucket_policy_status::GetBucketPolicyStatusInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_bucket_policy_status::GetBucketPolicyStatusInput {
+        ::std::result::Result::Ok(super::operation::get_bucket_policy_status::GetBucketPolicyStatusInput {
             bucket: self.bucket,
             expected_bucket_owner: self.expected_bucket_owner,
         })
