--- original
+++ modified
@@ -75,8 +75,8 @@
 }
 impl GetTokensFromRefreshTokenInput {
     /// Creates a new builder-style object to manufacture [`GetTokensFromRefreshTokenInput`](crate::operation::get_tokens_from_refresh_token::GetTokensFromRefreshTokenInput).
-    pub fn builder() -> crate::operation::get_tokens_from_refresh_token::builders::GetTokensFromRefreshTokenInputBuilder {
-        crate::operation::get_tokens_from_refresh_token::builders::GetTokensFromRefreshTokenInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_tokens_from_refresh_token::builders::GetTokensFromRefreshTokenInputBuilder {
+        super::super::super::operation::get_tokens_from_refresh_token::builders::GetTokensFromRefreshTokenInputBuilder::default()
     }
 }

@@ -219,10 +219,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_tokens_from_refresh_token::GetTokensFromRefreshTokenInput,
+        super::super::super::operation::get_tokens_from_refresh_token::GetTokensFromRefreshTokenInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_tokens_from_refresh_token::GetTokensFromRefreshTokenInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_tokens_from_refresh_token::GetTokensFromRefreshTokenInput {
             refresh_token: self.refresh_token,
             client_id: self.client_id,
             client_secret: self.client_secret,
