--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl StartConfigurationRecorderInput {
     /// Creates a new builder-style object to manufacture [`StartConfigurationRecorderInput`](crate::operation::start_configuration_recorder::StartConfigurationRecorderInput).
-    pub fn builder() -> crate::operation::start_configuration_recorder::builders::StartConfigurationRecorderInputBuilder {
-        crate::operation::start_configuration_recorder::builders::StartConfigurationRecorderInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::start_configuration_recorder::builders::StartConfigurationRecorderInputBuilder {
+        super::super::super::operation::start_configuration_recorder::builders::StartConfigurationRecorderInputBuilder::default()
     }
 }

@@ -46,10 +46,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::start_configuration_recorder::StartConfigurationRecorderInput,
+        super::super::super::operation::start_configuration_recorder::StartConfigurationRecorderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::start_configuration_recorder::StartConfigurationRecorderInput {
+        ::std::result::Result::Ok(super::super::super::operation::start_configuration_recorder::StartConfigurationRecorderInput {
             configuration_recorder_name: self.configuration_recorder_name,
         })
     }
