--- original
+++ modified
@@ -48,8 +48,8 @@
 }
 impl EnableMfaDeviceInput {
     /// Creates a new builder-style object to manufacture [`EnableMfaDeviceInput`](crate::operation::enable_mfa_device::EnableMfaDeviceInput).
-    pub fn builder() -> crate::operation::enable_mfa_device::builders::EnableMfaDeviceInputBuilder {
-        crate::operation::enable_mfa_device::builders::EnableMfaDeviceInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::enable_mfa_device::builders::EnableMfaDeviceInputBuilder {
+        super::super::super::operation::enable_mfa_device::builders::EnableMfaDeviceInputBuilder::default()
     }
 }

@@ -150,8 +150,8 @@
     /// Consumes the builder and constructs a [`EnableMfaDeviceInput`](crate::operation::enable_mfa_device::EnableMfaDeviceInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::enable_mfa_device::EnableMfaDeviceInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::enable_mfa_device::EnableMfaDeviceInput {
+    ) -> ::std::result::Result<super::super::super::operation::enable_mfa_device::EnableMfaDeviceInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::enable_mfa_device::EnableMfaDeviceInput {
             user_name: self.user_name,
             serial_number: self.serial_number,
             authentication_code1: self.authentication_code1,
