--- original
+++ modified
@@ -18,8 +18,8 @@
 }
 impl DeleteLoginProfileInput {
     /// Creates a new builder-style object to manufacture [`DeleteLoginProfileInput`](crate::operation::delete_login_profile::DeleteLoginProfileInput).
-    pub fn builder() -> crate::operation::delete_login_profile::builders::DeleteLoginProfileInputBuilder {
-        crate::operation::delete_login_profile::builders::DeleteLoginProfileInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_login_profile::builders::DeleteLoginProfileInputBuilder {
+        super::super::super::operation::delete_login_profile::builders::DeleteLoginProfileInputBuilder::default()
     }
 }

@@ -53,8 +53,8 @@
     /// Consumes the builder and constructs a [`DeleteLoginProfileInput`](crate::operation::delete_login_profile::DeleteLoginProfileInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_login_profile::DeleteLoginProfileInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_login_profile::DeleteLoginProfileInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_login_profile::DeleteLoginProfileInput { user_name: self.user_name })
+        ::std::result::Result::Ok(super::super::super::operation::delete_login_profile::DeleteLoginProfileInput { user_name: self.user_name })
     }
 }
