--- original
+++ modified
@@ -6,12 +6,12 @@
     /// <p>The name of the bucket for which the accelerate configuration is set.</p>
     pub bucket: ::std::option::Option<::std::string::String>,
     /// <p>Container for setting the transfer acceleration state.</p>
-    pub accelerate_configuration: ::std::option::Option<crate::types::AccelerateConfiguration>,
+    pub accelerate_configuration: ::std::option::Option<super::super::super::types::AccelerateConfiguration>,
     /// <p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p>
     pub expected_bucket_owner: ::std::option::Option<::std::string::String>,
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub checksum_algorithm: ::std::option::Option<crate::types::ChecksumAlgorithm>,
+    pub checksum_algorithm: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>,
 }
 impl PutBucketAccelerateConfigurationInput {
     /// <p>The name of the bucket for which the accelerate configuration is set.</p>
@@ -19,7 +19,7 @@
         self.bucket.as_deref()
     }
     /// <p>Container for setting the transfer acceleration state.</p>
-    pub fn accelerate_configuration(&self) -> ::std::option::Option<&crate::types::AccelerateConfiguration> {
+    pub fn accelerate_configuration(&self) -> ::std::option::Option<&super::super::super::types::AccelerateConfiguration> {
         self.accelerate_configuration.as_ref()
     }
     /// <p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p>
@@ -28,14 +28,14 @@
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn checksum_algorithm(&self) -> ::std::option::Option<&crate::types::ChecksumAlgorithm> {
+    pub fn checksum_algorithm(&self) -> ::std::option::Option<&super::super::super::types::ChecksumAlgorithm> {
         self.checksum_algorithm.as_ref()
     }
 }
 impl PutBucketAccelerateConfigurationInput {
     /// Creates a new builder-style object to manufacture [`PutBucketAccelerateConfigurationInput`](crate::operation::put_bucket_accelerate_configuration::PutBucketAccelerateConfigurationInput).
-    pub fn builder() -> crate::operation::put_bucket_accelerate_configuration::builders::PutBucketAccelerateConfigurationInputBuilder {
-        crate::operation::put_bucket_accelerate_configuration::builders::PutBucketAccelerateConfigurationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_bucket_accelerate_configuration::builders::PutBucketAccelerateConfigurationInputBuilder {
+        super::super::super::operation::put_bucket_accelerate_configuration::builders::PutBucketAccelerateConfigurationInputBuilder::default()
     }
 }

@@ -44,9 +44,9 @@
 #[non_exhaustive]
 pub struct PutBucketAccelerateConfigurationInputBuilder {
     pub(crate) bucket: ::std::option::Option<::std::string::String>,
-    pub(crate) accelerate_configuration: ::std::option::Option<crate::types::AccelerateConfiguration>,
+    pub(crate) accelerate_configuration: ::std::option::Option<super::super::super::types::AccelerateConfiguration>,
     pub(crate) expected_bucket_owner: ::std::option::Option<::std::string::String>,
-    pub(crate) checksum_algorithm: ::std::option::Option<crate::types::ChecksumAlgorithm>,
+    pub(crate) checksum_algorithm: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>,
 }
 impl PutBucketAccelerateConfigurationInputBuilder {
     /// <p>The name of the bucket for which the accelerate configuration is set.</p>
@@ -66,17 +66,17 @@
     }
     /// <p>Container for setting the transfer acceleration state.</p>
     /// This field is required.
-    pub fn accelerate_configuration(mut self, input: crate::types::AccelerateConfiguration) -> Self {
+    pub fn accelerate_configuration(mut self, input: super::super::super::types::AccelerateConfiguration) -> Self {
         self.accelerate_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Container for setting the transfer acceleration state.</p>
-    pub fn set_accelerate_configuration(mut self, input: ::std::option::Option<crate::types::AccelerateConfiguration>) -> Self {
+    pub fn set_accelerate_configuration(mut self, input: ::std::option::Option<super::super::super::types::AccelerateConfiguration>) -> Self {
         self.accelerate_configuration = input;
         self
     }
     /// <p>Container for setting the transfer acceleration state.</p>
-    pub fn get_accelerate_configuration(&self) -> &::std::option::Option<crate::types::AccelerateConfiguration> {
+    pub fn get_accelerate_configuration(&self) -> &::std::option::Option<super::super::super::types::AccelerateConfiguration> {
         &self.accelerate_configuration
     }
     /// <p>The account ID of the expected bucket owner. If the account ID that you provide does not match the actual owner of the bucket, the request fails with the HTTP status code <code>403 Forbidden</code> (access denied).</p>
@@ -95,19 +95,19 @@
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn checksum_algorithm(mut self, input: crate::types::ChecksumAlgorithm) -> Self {
+    pub fn checksum_algorithm(mut self, input: super::super::super::types::ChecksumAlgorithm) -> Self {
         self.checksum_algorithm = ::std::option::Option::Some(input);
         self
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<crate::types::ChecksumAlgorithm>) -> Self {
+    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>) -> Self {
         self.checksum_algorithm = input;
         self
     }
     /// <p>Indicates the algorithm used to create the checksum for the request when you use the SDK. This header will not provide any additional functionality if you don't use the SDK. When you send this header, there must be a corresponding <code>x-amz-checksum</code> or <code>x-amz-trailer</code> header sent. Otherwise, Amazon S3 fails the request with the HTTP status code <code>400 Bad Request</code>. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// <p>If you provide an individual checksum, Amazon S3 ignores any provided <code>ChecksumAlgorithm</code> parameter.</p>
-    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<crate::types::ChecksumAlgorithm> {
+    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<super::super::super::types::ChecksumAlgorithm> {
         &self.checksum_algorithm
     }
     /// Consumes the builder and constructs a [`PutBucketAccelerateConfigurationInput`](crate::operation::put_bucket_accelerate_configuration::PutBucketAccelerateConfigurationInput).
@@ -114,11 +114,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_bucket_accelerate_configuration::PutBucketAccelerateConfigurationInput,
+        super::super::super::operation::put_bucket_accelerate_configuration::PutBucketAccelerateConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_bucket_accelerate_configuration::PutBucketAccelerateConfigurationInput {
+            super::super::super::operation::put_bucket_accelerate_configuration::PutBucketAccelerateConfigurationInput {
                 bucket: self.bucket,
                 accelerate_configuration: self.accelerate_configuration,
                 expected_bucket_owner: self.expected_bucket_owner,
