--- original
+++ modified
@@ -2,14 +2,14 @@
 #[allow(unused_mut)]
 pub fn de_idp_rejected_claim_exception_xml_err(
     inp: &[u8],
-    mut builder: crate::types::error::builders::IdpRejectedClaimExceptionBuilder,
-) -> std::result::Result<crate::types::error::builders::IdpRejectedClaimExceptionBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::super::types::error::builders::IdpRejectedClaimExceptionBuilder,
+) -> std::result::Result<super::super::types::error::builders::IdpRejectedClaimExceptionBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     if inp.is_empty() {
         return Ok(builder);
     }
     let mut document = ::aws_smithy_xml::decode::Document::try_from(inp)?;
     #[allow(unused_mut)]
-    let mut error_decoder = crate::rest_xml_wrapped_errors::error_scope(&mut document)?;
+    let mut error_decoder = super::super::rest_xml_wrapped_errors::error_scope(&mut document)?;
     #[allow(unused_variables)]
     let depth = 0u32;
     while let Some(mut tag) = error_decoder.next_tag() {
