--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::assume_role_with_saml::AssumeRoleWithSamlOutput,
-    crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError,
+    super::operation::assume_role_with_saml::AssumeRoleWithSamlOutput,
+    super::operation::assume_role_with_saml::AssumeRoleWithSAMLError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled(generic)),
+        None => return Err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ExpiredTokenException" => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::ExpiredTokenException({
+        "ExpiredTokenException" => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::ExpiredTokenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ExpiredTokenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_expired_token_exception::de_expired_token_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+                let mut output = super::types::error::builders::ExpiredTokenExceptionBuilder::default();
+                output = super::protocol_serde::shape_expired_token_exception::de_expired_token_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "IDPRejectedClaim" => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::IdpRejectedClaimException({
+        "IDPRejectedClaim" => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::IdpRejectedClaimException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::IdpRejectedClaimExceptionBuilder::default();
-                output = crate::protocol_serde::shape_idp_rejected_claim_exception::de_idp_rejected_claim_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+                let mut output = super::types::error::builders::IdpRejectedClaimExceptionBuilder::default();
+                output = super::protocol_serde::shape_idp_rejected_claim_exception::de_idp_rejected_claim_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,16 +50,16 @@
             }
             tmp
         }),
-        "InvalidIdentityToken" => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::InvalidIdentityTokenException({
+        "InvalidIdentityToken" => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::InvalidIdentityTokenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidIdentityTokenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_identity_token_exception::de_invalid_identity_token_exception_xml_err(
+                let mut output = super::types::error::builders::InvalidIdentityTokenExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_identity_token_exception::de_invalid_identity_token_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+                .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,16 +68,16 @@
             }
             tmp
         }),
-        "MalformedPolicyDocument" => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::MalformedPolicyDocumentException({
+        "MalformedPolicyDocument" => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::MalformedPolicyDocumentException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::MalformedPolicyDocumentExceptionBuilder::default();
-                output = crate::protocol_serde::shape_malformed_policy_document_exception::de_malformed_policy_document_exception_xml_err(
+                let mut output = super::types::error::builders::MalformedPolicyDocumentExceptionBuilder::default();
+                output = super::protocol_serde::shape_malformed_policy_document_exception::de_malformed_policy_document_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+                .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -86,16 +86,16 @@
             }
             tmp
         }),
-        "PackedPolicyTooLarge" => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::PackedPolicyTooLargeException({
+        "PackedPolicyTooLarge" => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::PackedPolicyTooLargeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PackedPolicyTooLargeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_packed_policy_too_large_exception::de_packed_policy_too_large_exception_xml_err(
+                let mut output = super::types::error::builders::PackedPolicyTooLargeExceptionBuilder::default();
+                output = super::protocol_serde::shape_packed_policy_too_large_exception::de_packed_policy_too_large_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+                .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -104,13 +104,13 @@
             }
             tmp
         }),
-        "RegionDisabledException" => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::RegionDisabledException({
+        "RegionDisabledException" => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::RegionDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RegionDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_region_disabled_exception::de_region_disabled_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+                let mut output = super::types::error::builders::RegionDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_region_disabled_exception::de_region_disabled_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -119,7 +119,7 @@
             }
             tmp
         }),
-        _ => crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::generic(generic),
+        _ => super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::generic(generic),
     })
 }

@@ -129,14 +129,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::assume_role_with_saml::AssumeRoleWithSamlOutput,
-    crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError,
+    super::operation::assume_role_with_saml::AssumeRoleWithSamlOutput,
+    super::operation::assume_role_with_saml::AssumeRoleWithSAMLError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::assume_role_with_saml::builders::AssumeRoleWithSamlOutputBuilder::default();
-        output = crate::protocol_serde::shape_assume_role_with_saml::de_assume_role_with_saml(_response_body, output)
-            .map_err(crate::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
+        let mut output = super::operation::assume_role_with_saml::builders::AssumeRoleWithSamlOutputBuilder::default();
+        output = super::protocol_serde::shape_assume_role_with_saml::de_assume_role_with_saml(_response_body, output)
+            .map_err(super::operation::assume_role_with_saml::AssumeRoleWithSAMLError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -145,8 +145,8 @@
 #[allow(unused_mut)]
 pub fn de_assume_role_with_saml(
     inp: &[u8],
-    mut builder: crate::operation::assume_role_with_saml::builders::AssumeRoleWithSamlOutputBuilder,
-) -> std::result::Result<crate::operation::assume_role_with_saml::builders::AssumeRoleWithSamlOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
+    mut builder: super::operation::assume_role_with_saml::builders::AssumeRoleWithSamlOutputBuilder,
+) -> std::result::Result<super::operation::assume_role_with_saml::builders::AssumeRoleWithSamlOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
 {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

@@ -173,7 +173,7 @@
             s if s.matches("Credentials") /* Credentials com.amazonaws.sts.synthetic#AssumeRoleWithSAMLOutput$Credentials */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_credentials::de_credentials(&mut tag, depth + 1)
+                        super::protocol_serde::shape_credentials::de_credentials(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -183,7 +183,7 @@
             s if s.matches("AssumedRoleUser") /* AssumedRoleUser com.amazonaws.sts.synthetic#AssumeRoleWithSAMLOutput$AssumedRoleUser */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_assumed_role_user::de_assumed_role_user(&mut tag, depth + 1)
+                        super::protocol_serde::shape_assumed_role_user::de_assumed_role_user(&mut tag, depth + 1)
                         ?
                     )
                 ;
