--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListStoredQueriesInput {
     /// Creates a new builder-style object to manufacture [`ListStoredQueriesInput`](crate::operation::list_stored_queries::ListStoredQueriesInput).
-    pub fn builder() -> crate::operation::list_stored_queries::builders::ListStoredQueriesInputBuilder {
-        crate::operation::list_stored_queries::builders::ListStoredQueriesInputBuilder::default()
+    pub fn builder() -> super::operation::list_stored_queries::builders::ListStoredQueriesInputBuilder {
+        super::operation::list_stored_queries::builders::ListStoredQueriesInputBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
     /// Consumes the builder and constructs a [`ListStoredQueriesInput`](crate::operation::list_stored_queries::ListStoredQueriesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_stored_queries::ListStoredQueriesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_stored_queries::ListStoredQueriesInput {
+    ) -> ::std::result::Result<super::operation::list_stored_queries::ListStoredQueriesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_stored_queries::ListStoredQueriesInput {
             next_token: self.next_token,
             max_results: self.max_results,
         })
