--- original
+++ modified
@@ -40,8 +40,8 @@
 }
 impl ListServiceSpecificCredentialsInput {
     /// Creates a new builder-style object to manufacture [`ListServiceSpecificCredentialsInput`](crate::operation::list_service_specific_credentials::ListServiceSpecificCredentialsInput).
-    pub fn builder() -> crate::operation::list_service_specific_credentials::builders::ListServiceSpecificCredentialsInputBuilder {
-        crate::operation::list_service_specific_credentials::builders::ListServiceSpecificCredentialsInputBuilder::default()
+    pub fn builder() -> super::operation::list_service_specific_credentials::builders::ListServiceSpecificCredentialsInputBuilder {
+        super::operation::list_service_specific_credentials::builders::ListServiceSpecificCredentialsInputBuilder::default()
     }
 }

@@ -133,10 +133,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_service_specific_credentials::ListServiceSpecificCredentialsInput,
+        super::operation::list_service_specific_credentials::ListServiceSpecificCredentialsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_service_specific_credentials::ListServiceSpecificCredentialsInput {
+        ::std::result::Result::Ok(super::operation::list_service_specific_credentials::ListServiceSpecificCredentialsInput {
             user_name: self.user_name,
             service_name: self.service_name,
             all_users: self.all_users,
