--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl ListResourceTenantsInput {
     /// Creates a new builder-style object to manufacture [`ListResourceTenantsInput`](crate::operation::list_resource_tenants::ListResourceTenantsInput).
-    pub fn builder() -> crate::operation::list_resource_tenants::builders::ListResourceTenantsInputBuilder {
-        crate::operation::list_resource_tenants::builders::ListResourceTenantsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_resource_tenants::builders::ListResourceTenantsInputBuilder {
+        super::super::super::operation::list_resource_tenants::builders::ListResourceTenantsInputBuilder::default()
     }
 }

@@ -87,9 +87,9 @@
     /// Consumes the builder and constructs a [`ListResourceTenantsInput`](crate::operation::list_resource_tenants::ListResourceTenantsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_resource_tenants::ListResourceTenantsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_resource_tenants::ListResourceTenantsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_resource_tenants::ListResourceTenantsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_resource_tenants::ListResourceTenantsInput {
             resource_arn: self.resource_arn,
             page_size: self.page_size,
             next_token: self.next_token,
