--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>The field delimiter to use between record fields when the final output format of a delivery is in <code>Plain</code>, <code>W3C</code>, or <code>Raw</code> format.</p>
     pub field_delimiter: ::std::option::Option<::std::string::String>,
     /// <p>This structure contains parameters that are valid only when the delivery's delivery destination is an S3 bucket.</p>
-    pub s3_delivery_configuration: ::std::option::Option<crate::types::S3DeliveryConfiguration>,
+    pub s3_delivery_configuration: ::std::option::Option<super::super::super::types::S3DeliveryConfiguration>,
 }
 impl UpdateDeliveryConfigurationInput {
     /// <p>The ID of the delivery to be updated by this request.</p>
@@ -28,14 +28,14 @@
         self.field_delimiter.as_deref()
     }
     /// <p>This structure contains parameters that are valid only when the delivery's delivery destination is an S3 bucket.</p>
-    pub fn s3_delivery_configuration(&self) -> ::std::option::Option<&crate::types::S3DeliveryConfiguration> {
+    pub fn s3_delivery_configuration(&self) -> ::std::option::Option<&super::super::super::types::S3DeliveryConfiguration> {
         self.s3_delivery_configuration.as_ref()
     }
 }
 impl UpdateDeliveryConfigurationInput {
     /// Creates a new builder-style object to manufacture [`UpdateDeliveryConfigurationInput`](crate::operation::update_delivery_configuration::UpdateDeliveryConfigurationInput).
-    pub fn builder() -> crate::operation::update_delivery_configuration::builders::UpdateDeliveryConfigurationInputBuilder {
-        crate::operation::update_delivery_configuration::builders::UpdateDeliveryConfigurationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_delivery_configuration::builders::UpdateDeliveryConfigurationInputBuilder {
+        super::super::super::operation::update_delivery_configuration::builders::UpdateDeliveryConfigurationInputBuilder::default()
     }
 }

@@ -46,7 +46,7 @@
     pub(crate) id: ::std::option::Option<::std::string::String>,
     pub(crate) record_fields: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) field_delimiter: ::std::option::Option<::std::string::String>,
-    pub(crate) s3_delivery_configuration: ::std::option::Option<crate::types::S3DeliveryConfiguration>,
+    pub(crate) s3_delivery_configuration: ::std::option::Option<super::super::super::types::S3DeliveryConfiguration>,
 }
 impl UpdateDeliveryConfigurationInputBuilder {
     /// <p>The ID of the delivery to be updated by this request.</p>
@@ -99,17 +99,17 @@
         &self.field_delimiter
     }
     /// <p>This structure contains parameters that are valid only when the delivery's delivery destination is an S3 bucket.</p>
-    pub fn s3_delivery_configuration(mut self, input: crate::types::S3DeliveryConfiguration) -> Self {
+    pub fn s3_delivery_configuration(mut self, input: super::super::super::types::S3DeliveryConfiguration) -> Self {
         self.s3_delivery_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>This structure contains parameters that are valid only when the delivery's delivery destination is an S3 bucket.</p>
-    pub fn set_s3_delivery_configuration(mut self, input: ::std::option::Option<crate::types::S3DeliveryConfiguration>) -> Self {
+    pub fn set_s3_delivery_configuration(mut self, input: ::std::option::Option<super::super::super::types::S3DeliveryConfiguration>) -> Self {
         self.s3_delivery_configuration = input;
         self
     }
     /// <p>This structure contains parameters that are valid only when the delivery's delivery destination is an S3 bucket.</p>
-    pub fn get_s3_delivery_configuration(&self) -> &::std::option::Option<crate::types::S3DeliveryConfiguration> {
+    pub fn get_s3_delivery_configuration(&self) -> &::std::option::Option<super::super::super::types::S3DeliveryConfiguration> {
         &self.s3_delivery_configuration
     }
     /// Consumes the builder and constructs a [`UpdateDeliveryConfigurationInput`](crate::operation::update_delivery_configuration::UpdateDeliveryConfigurationInput).
@@ -116,10 +116,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_delivery_configuration::UpdateDeliveryConfigurationInput,
+        super::super::super::operation::update_delivery_configuration::UpdateDeliveryConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_delivery_configuration::UpdateDeliveryConfigurationInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_delivery_configuration::UpdateDeliveryConfigurationInput {
             id: self.id,
             record_fields: self.record_fields,
             field_delimiter: self.field_delimiter,
