--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl GetServiceLinkedRoleDeletionStatusInput {
     /// Creates a new builder-style object to manufacture [`GetServiceLinkedRoleDeletionStatusInput`](crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusInput).
-    pub fn builder() -> crate::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusInputBuilder {
-        crate::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusInputBuilder::default()
+    pub fn builder() -> super::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusInputBuilder {
+        super::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusInputBuilder::default()
     }
 }

@@ -75,11 +75,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusInput,
+        super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusInput {
+            super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusInput {
                 deletion_task_id: self.deletion_task_id,
             },
         )
