--- original
+++ modified
@@ -4,7 +4,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct DescribeAccountPoliciesInput {
     /// <p>Use this parameter to limit the returned policies to only the policies that match the policy type that you specify.</p>
-    pub policy_type: ::std::option::Option<crate::types::PolicyType>,
+    pub policy_type: ::std::option::Option<super::super::super::types::PolicyType>,
     /// <p>Use this parameter to limit the returned policies to only the policy with the name that you specify.</p>
     pub policy_name: ::std::option::Option<::std::string::String>,
     /// <p>If you are using an account that is set up as a monitoring account for CloudWatch unified cross-account observability, you can use this to specify the account ID of a source account. If you do, the operation returns the account policy for the specified account. Currently, you can specify only one account ID in this parameter.</p>
@@ -15,7 +15,7 @@
 }
 impl DescribeAccountPoliciesInput {
     /// <p>Use this parameter to limit the returned policies to only the policies that match the policy type that you specify.</p>
-    pub fn policy_type(&self) -> ::std::option::Option<&crate::types::PolicyType> {
+    pub fn policy_type(&self) -> ::std::option::Option<&super::super::super::types::PolicyType> {
         self.policy_type.as_ref()
     }
     /// <p>Use this parameter to limit the returned policies to only the policy with the name that you specify.</p>
@@ -36,8 +36,8 @@
 }
 impl DescribeAccountPoliciesInput {
     /// Creates a new builder-style object to manufacture [`DescribeAccountPoliciesInput`](crate::operation::describe_account_policies::DescribeAccountPoliciesInput).
-    pub fn builder() -> crate::operation::describe_account_policies::builders::DescribeAccountPoliciesInputBuilder {
-        crate::operation::describe_account_policies::builders::DescribeAccountPoliciesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_account_policies::builders::DescribeAccountPoliciesInputBuilder {
+        super::super::super::operation::describe_account_policies::builders::DescribeAccountPoliciesInputBuilder::default()
     }
 }

@@ -45,7 +45,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct DescribeAccountPoliciesInputBuilder {
-    pub(crate) policy_type: ::std::option::Option<crate::types::PolicyType>,
+    pub(crate) policy_type: ::std::option::Option<super::super::super::types::PolicyType>,
     pub(crate) policy_name: ::std::option::Option<::std::string::String>,
     pub(crate) account_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
@@ -53,17 +53,17 @@
 impl DescribeAccountPoliciesInputBuilder {
     /// <p>Use this parameter to limit the returned policies to only the policies that match the policy type that you specify.</p>
     /// This field is required.
-    pub fn policy_type(mut self, input: crate::types::PolicyType) -> Self {
+    pub fn policy_type(mut self, input: super::super::super::types::PolicyType) -> Self {
         self.policy_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>Use this parameter to limit the returned policies to only the policies that match the policy type that you specify.</p>
-    pub fn set_policy_type(mut self, input: ::std::option::Option<crate::types::PolicyType>) -> Self {
+    pub fn set_policy_type(mut self, input: ::std::option::Option<super::super::super::types::PolicyType>) -> Self {
         self.policy_type = input;
         self
     }
     /// <p>Use this parameter to limit the returned policies to only the policies that match the policy type that you specify.</p>
-    pub fn get_policy_type(&self) -> &::std::option::Option<crate::types::PolicyType> {
+    pub fn get_policy_type(&self) -> &::std::option::Option<super::super::super::types::PolicyType> {
         &self.policy_type
     }
     /// <p>Use this parameter to limit the returned policies to only the policy with the name that you specify.</p>
@@ -121,10 +121,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_account_policies::DescribeAccountPoliciesInput,
+        super::super::super::operation::describe_account_policies::DescribeAccountPoliciesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_account_policies::DescribeAccountPoliciesInput {
+        ::std::result::Result::Ok(super::super::super::operation::describe_account_policies::DescribeAccountPoliciesInput {
             policy_type: self.policy_type,
             policy_name: self.policy_name,
             account_identifiers: self.account_identifiers,
