--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl PutLogGroupDeletionProtectionInput {
     /// Creates a new builder-style object to manufacture [`PutLogGroupDeletionProtectionInput`](crate::operation::put_log_group_deletion_protection::PutLogGroupDeletionProtectionInput).
-    pub fn builder() -> crate::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionInputBuilder {
-        crate::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionInputBuilder::default()
+    pub fn builder() -> super::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionInputBuilder {
+        super::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionInputBuilder::default()
     }
 }

@@ -97,10 +97,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_log_group_deletion_protection::PutLogGroupDeletionProtectionInput,
+        super::operation::put_log_group_deletion_protection::PutLogGroupDeletionProtectionInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_log_group_deletion_protection::PutLogGroupDeletionProtectionInput {
+        ::std::result::Result::Ok(super::operation::put_log_group_deletion_protection::PutLogGroupDeletionProtectionInput {
             log_group_identifier: self.log_group_identifier,
             deletion_protection_enabled: self.deletion_protection_enabled,
         })
