--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::vec::Vec<crate::types::AggregateComplianceByConformancePack>>,
+    Option<::std::vec::Vec<super::super::types::AggregateComplianceByConformancePack>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -27,7 +27,7 @@
                     }
                     _ => {
                         let value =
-                            crate::protocol_serde::shape_aggregate_compliance_by_conformance_pack::de_aggregate_compliance_by_conformance_pack(
+                            super::super::protocol_serde::shape_aggregate_compliance_by_conformance_pack::de_aggregate_compliance_by_conformance_pack(
                                 tokens,
                                 _value,
                                 depth + 1,
