--- original
+++ modified
@@ -5,32 +5,32 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_delivery_channel::PutDeliveryChannelOutput,
-    crate::operation::put_delivery_channel::PutDeliveryChannelError,
+    super::super::operation::put_delivery_channel::PutDeliveryChannelOutput,
+    super::super::operation::put_delivery_channel::PutDeliveryChannelError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled(generic)),
+        None => return Err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InsufficientDeliveryPolicyException" => {
-            crate::operation::put_delivery_channel::PutDeliveryChannelError::InsufficientDeliveryPolicyException({
+            super::super::operation::put_delivery_channel::PutDeliveryChannelError::InsufficientDeliveryPolicyException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InsufficientDeliveryPolicyExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_insufficient_delivery_policy_exception::de_insufficient_delivery_policy_exception_json_err(
+                    let mut output = super::super::types::error::builders::InsufficientDeliveryPolicyExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_insufficient_delivery_policy_exception::de_insufficient_delivery_policy_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                    .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -41,17 +41,17 @@
             })
         }
         "InvalidDeliveryChannelNameException" => {
-            crate::operation::put_delivery_channel::PutDeliveryChannelError::InvalidDeliveryChannelNameException({
+            super::super::operation::put_delivery_channel::PutDeliveryChannelError::InvalidDeliveryChannelNameException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidDeliveryChannelNameExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::InvalidDeliveryChannelNameExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_invalid_delivery_channel_name_exception::de_invalid_delivery_channel_name_exception_json_err(
+                        super::super::protocol_serde::shape_invalid_delivery_channel_name_exception::de_invalid_delivery_channel_name_exception_json_err(
                             _response_body,
                             output,
                         )
-                        .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                        .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -61,14 +61,14 @@
                 tmp
             })
         }
-        "InvalidS3KeyPrefixException" => crate::operation::put_delivery_channel::PutDeliveryChannelError::InvalidS3KeyPrefixException({
+        "InvalidS3KeyPrefixException" => super::super::operation::put_delivery_channel::PutDeliveryChannelError::InvalidS3KeyPrefixException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidS3KeyPrefixExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::InvalidS3KeyPrefixExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_invalid_s3_key_prefix_exception::de_invalid_s3_key_prefix_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                    super::super::protocol_serde::shape_invalid_s3_key_prefix_exception::de_invalid_s3_key_prefix_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -77,16 +77,16 @@
             }
             tmp
         }),
-        "InvalidS3KmsKeyArnException" => crate::operation::put_delivery_channel::PutDeliveryChannelError::InvalidS3KmsKeyArnException({
+        "InvalidS3KmsKeyArnException" => super::super::operation::put_delivery_channel::PutDeliveryChannelError::InvalidS3KmsKeyArnException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidS3KmsKeyArnExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_s3_kms_key_arn_exception::de_invalid_s3_kms_key_arn_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidS3KmsKeyArnExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_s3_kms_key_arn_exception::de_invalid_s3_kms_key_arn_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -95,14 +95,14 @@
             }
             tmp
         }),
-        "InvalidSNSTopicARNException" => crate::operation::put_delivery_channel::PutDeliveryChannelError::InvalidSnsTopicArnException({
+        "InvalidSNSTopicARNException" => super::super::operation::put_delivery_channel::PutDeliveryChannelError::InvalidSnsTopicArnException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSnsTopicArnExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::InvalidSnsTopicArnExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_invalid_sns_topic_arn_exception::de_invalid_sns_topic_arn_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                    super::super::protocol_serde::shape_invalid_sns_topic_arn_exception::de_invalid_sns_topic_arn_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -112,12 +112,12 @@
             tmp
         }),
         "MaxNumberOfDeliveryChannelsExceededException" => {
-            crate::operation::put_delivery_channel::PutDeliveryChannelError::MaxNumberOfDeliveryChannelsExceededException({
+            super::super::operation::put_delivery_channel::PutDeliveryChannelError::MaxNumberOfDeliveryChannelsExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::MaxNumberOfDeliveryChannelsExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_max_number_of_delivery_channels_exceeded_exception::de_max_number_of_delivery_channels_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                    let mut output = super::super::types::error::builders::MaxNumberOfDeliveryChannelsExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_max_number_of_delivery_channels_exceeded_exception::de_max_number_of_delivery_channels_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -128,12 +128,12 @@
             })
         }
         "NoAvailableConfigurationRecorderException" => {
-            crate::operation::put_delivery_channel::PutDeliveryChannelError::NoAvailableConfigurationRecorderException({
+            super::super::operation::put_delivery_channel::PutDeliveryChannelError::NoAvailableConfigurationRecorderException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoAvailableConfigurationRecorderExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_available_configuration_recorder_exception::de_no_available_configuration_recorder_exception_json_err(_response_body, output).map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                    let mut output = super::super::types::error::builders::NoAvailableConfigurationRecorderExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_no_available_configuration_recorder_exception::de_no_available_configuration_recorder_exception_json_err(_response_body, output).map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -143,13 +143,13 @@
                 tmp
             })
         }
-        "NoSuchBucketException" => crate::operation::put_delivery_channel::PutDeliveryChannelError::NoSuchBucketException({
+        "NoSuchBucketException" => super::super::operation::put_delivery_channel::PutDeliveryChannelError::NoSuchBucketException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchBucketExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_bucket_exception::de_no_such_bucket_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchBucketExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_bucket_exception::de_no_such_bucket_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::put_delivery_channel::PutDeliveryChannelError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -158,7 +158,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_delivery_channel::PutDeliveryChannelError::generic(generic),
+        _ => super::super::operation::put_delivery_channel::PutDeliveryChannelError::generic(generic),
     })
 }

@@ -168,12 +168,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_delivery_channel::PutDeliveryChannelOutput,
-    crate::operation::put_delivery_channel::PutDeliveryChannelError,
+    super::super::operation::put_delivery_channel::PutDeliveryChannelOutput,
+    super::super::operation::put_delivery_channel::PutDeliveryChannelError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_delivery_channel::builders::PutDeliveryChannelOutputBuilder::default();
+        let mut output = super::super::operation::put_delivery_channel::builders::PutDeliveryChannelOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -180,11 +180,11 @@
 }

 pub fn ser_put_delivery_channel_input(
-    input: &crate::operation::put_delivery_channel::PutDeliveryChannelInput,
+    input: &super::super::operation::put_delivery_channel::PutDeliveryChannelInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_delivery_channel_input::ser_put_delivery_channel_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_put_delivery_channel_input::ser_put_delivery_channel_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
