--- original
+++ modified
@@ -5,12 +5,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusOutput,
-    crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError,
+    super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusOutput,
+    super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
@@ -17,7 +17,7 @@
         Some(code) => code,
         None => {
             return Err(
-                crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled(
+                super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled(
                     generic,
                 ),
             )
@@ -26,13 +26,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidLimitException" => crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::InvalidLimitException({
+        "InvalidLimitException" => super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::InvalidLimitException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidLimitExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_limit_exception::de_invalid_limit_exception_json_err(_response_body, output).map_err(crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidLimitExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_limit_exception::de_invalid_limit_exception_json_err(_response_body, output).map_err(super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -42,13 +42,13 @@
                                                         }
             tmp
         }),
-        "InvalidNextTokenException" => crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::InvalidNextTokenException({
+        "InvalidNextTokenException" => super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::InvalidNextTokenException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidNextTokenExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_next_token_exception::de_invalid_next_token_exception_json_err(_response_body, output).map_err(crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidNextTokenExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_next_token_exception::de_invalid_next_token_exception_json_err(_response_body, output).map_err(super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -58,13 +58,13 @@
                                                         }
             tmp
         }),
-        "NoSuchOrganizationConformancePackException" => crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::NoSuchOrganizationConformancePackException({
+        "NoSuchOrganizationConformancePackException" => super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::NoSuchOrganizationConformancePackException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoSuchOrganizationConformancePackExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_such_organization_conformance_pack_exception::de_no_such_organization_conformance_pack_exception_json_err(_response_body, output).map_err(crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::NoSuchOrganizationConformancePackExceptionBuilder::default();
+                    output = super::protocol_serde::shape_no_such_organization_conformance_pack_exception::de_no_such_organization_conformance_pack_exception_json_err(_response_body, output).map_err(super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -74,13 +74,13 @@
                                                         }
             tmp
         }),
-        "OrganizationAccessDeniedException" => crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::OrganizationAccessDeniedException({
+        "OrganizationAccessDeniedException" => super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::OrganizationAccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationAccessDeniedExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_access_denied_exception::de_organization_access_denied_exception_json_err(_response_body, output).map_err(crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::OrganizationAccessDeniedExceptionBuilder::default();
+                    output = super::protocol_serde::shape_organization_access_denied_exception::de_organization_access_denied_exception_json_err(_response_body, output).map_err(super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -90,7 +90,7 @@
                                                         }
             tmp
         }),
-        _ => crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::generic(generic)
+        _ => super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::generic(generic)
     })
 }

@@ -100,19 +100,19 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusOutput,
-    crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError,
+    super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusOutput,
+    super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusOutputBuilder::default();
+        let mut output = super::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_get_organization_conformance_pack_detailed_status::de_get_organization_conformance_pack_detailed_status(
+            super::protocol_serde::shape_get_organization_conformance_pack_detailed_status::de_get_organization_conformance_pack_detailed_status(
                 _response_body,
                 output,
             )
             .map_err(
-                crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled,
+                super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusError::unhandled,
             )?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
@@ -120,11 +120,11 @@
 }

 pub fn ser_get_organization_conformance_pack_detailed_status_input(
-    input: &crate::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput,
+    input: &super::operation::get_organization_conformance_pack_detailed_status::GetOrganizationConformancePackDetailedStatusInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_get_organization_conformance_pack_detailed_status_input::ser_get_organization_conformance_pack_detailed_status_input_input(&mut object, input)?;
+    super::protocol_serde::shape_get_organization_conformance_pack_detailed_status_input::ser_get_organization_conformance_pack_detailed_status_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -131,12 +131,12 @@

 pub(crate) fn de_get_organization_conformance_pack_detailed_status(
     _value: &[u8],
-    mut builder: crate::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusOutputBuilder,
+    mut builder: super::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusOutputBuilder,
+    super::operation::get_organization_conformance_pack_detailed_status::builders::GetOrganizationConformancePackDetailedStatusOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -147,7 +147,7 @@
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "OrganizationConformancePackDetailedStatuses" => {
                     builder = builder.set_organization_conformance_pack_detailed_statuses(
-                            crate::protocol_serde::shape_organization_conformance_pack_detailed_statuses::de_organization_conformance_pack_detailed_statuses(tokens, _value, depth + 1)?
+                            super::protocol_serde::shape_organization_conformance_pack_detailed_statuses::de_organization_conformance_pack_detailed_statuses(tokens, _value, depth + 1)?
                         );
                 }
                 "NextToken" => {
