--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::put_config_rule::PutConfigRuleOutput, crate::operation::put_config_rule::PutConfigRuleError> {
+) -> std::result::Result<super::super::operation::put_config_rule::PutConfigRuleOutput, super::super::operation::put_config_rule::PutConfigRuleError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_config_rule::PutConfigRuleError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::put_config_rule::PutConfigRuleError::unhandled(generic)),
+        None => return Err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InsufficientPermissionsException" => crate::operation::put_config_rule::PutConfigRuleError::InsufficientPermissionsException({
+        "InsufficientPermissionsException" => super::super::operation::put_config_rule::PutConfigRuleError::InsufficientPermissionsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InsufficientPermissionsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_insufficient_permissions_exception::de_insufficient_permissions_exception_json_err(
+                let mut output = super::super::types::error::builders::InsufficientPermissionsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_insufficient_permissions_exception::de_insufficient_permissions_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::put_config_rule::PutConfigRuleError::unhandled)?;
+                .map_err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,16 +35,16 @@
             }
             tmp
         }),
-        "InvalidParameterValueException" => crate::operation::put_config_rule::PutConfigRuleError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::super::operation::put_config_rule::PutConfigRuleError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::put_config_rule::PutConfigRuleError::unhandled)?;
+                .map_err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,12 +54,12 @@
             tmp
         }),
         "MaxNumberOfConfigRulesExceededException" => {
-            crate::operation::put_config_rule::PutConfigRuleError::MaxNumberOfConfigRulesExceededException({
+            super::super::operation::put_config_rule::PutConfigRuleError::MaxNumberOfConfigRulesExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::MaxNumberOfConfigRulesExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_max_number_of_config_rules_exceeded_exception::de_max_number_of_config_rules_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::put_config_rule::PutConfigRuleError::unhandled)?;
+                    let mut output = super::super::types::error::builders::MaxNumberOfConfigRulesExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_max_number_of_config_rules_exceeded_exception::de_max_number_of_config_rules_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -70,12 +70,12 @@
             })
         }
         "NoAvailableConfigurationRecorderException" => {
-            crate::operation::put_config_rule::PutConfigRuleError::NoAvailableConfigurationRecorderException({
+            super::super::operation::put_config_rule::PutConfigRuleError::NoAvailableConfigurationRecorderException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoAvailableConfigurationRecorderExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_available_configuration_recorder_exception::de_no_available_configuration_recorder_exception_json_err(_response_body, output).map_err(crate::operation::put_config_rule::PutConfigRuleError::unhandled)?;
+                    let mut output = super::super::types::error::builders::NoAvailableConfigurationRecorderExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_no_available_configuration_recorder_exception::de_no_available_configuration_recorder_exception_json_err(_response_body, output).map_err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -85,13 +85,13 @@
                 tmp
             })
         }
-        "ResourceInUseException" => crate::operation::put_config_rule::PutConfigRuleError::ResourceInUseException({
+        "ResourceInUseException" => super::super::operation::put_config_rule::PutConfigRuleError::ResourceInUseException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceInUseExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_in_use_exception::de_resource_in_use_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_config_rule::PutConfigRuleError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceInUseExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_in_use_exception::de_resource_in_use_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::put_config_rule::PutConfigRuleError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -100,7 +100,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_config_rule::PutConfigRuleError::generic(generic),
+        _ => super::super::operation::put_config_rule::PutConfigRuleError::generic(generic),
     })
 }

@@ -109,10 +109,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::put_config_rule::PutConfigRuleOutput, crate::operation::put_config_rule::PutConfigRuleError> {
+) -> std::result::Result<super::super::operation::put_config_rule::PutConfigRuleOutput, super::super::operation::put_config_rule::PutConfigRuleError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_config_rule::builders::PutConfigRuleOutputBuilder::default();
+        let mut output = super::super::operation::put_config_rule::builders::PutConfigRuleOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -119,11 +119,11 @@
 }

 pub fn ser_put_config_rule_input(
-    input: &crate::operation::put_config_rule::PutConfigRuleInput,
+    input: &super::super::operation::put_config_rule::PutConfigRuleInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_config_rule_input::ser_put_config_rule_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_put_config_rule_input::ser_put_config_rule_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
