--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::StoredQuery>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::StoredQuery>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::StoredQueryBuilder::default();
+            let mut builder = super::types::builders::StoredQueryBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -65,7 +65,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::stored_query_correct_errors(builder).build().map_err(|err| {
+            Ok(Some(super::serde_util::stored_query_correct_errors(builder).build().map_err(|err| {
                 ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err)
             })?))
         }
@@ -77,7 +77,7 @@

 pub fn ser_stored_query(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::StoredQuery,
+    input: &super::types::StoredQuery,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     if let Some(var_1) = &input.query_id {
         object.key("QueryId").string(var_1.as_str());
