--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ConformancePackComplianceSummary>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ConformancePackComplianceSummary>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ConformancePackComplianceSummaryBuilder::default();
+            let mut builder = super::types::builders::ConformancePackComplianceSummaryBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,7 +31,7 @@
                         "ConformancePackComplianceStatus" => {
                             builder = builder.set_conformance_pack_compliance_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ConformancePackComplianceType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::ConformancePackComplianceType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -45,7 +45,7 @@
                 }
             }
             Ok(Some(
-                crate::serde_util::conformance_pack_compliance_summary_correct_errors(builder)
+                super::serde_util::conformance_pack_compliance_summary_correct_errors(builder)
                     .build()
                     .map_err(|err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err))?,
             ))
