--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderOutput,
-    crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError,
+    super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderOutput,
+    super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled(generic))
+            return Err(super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled(generic))
         }
     };

@@ -23,13 +23,13 @@
     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "ConflictException" => {
-            crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::ConflictException({
+            super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::ConflictException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ConflictExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output).map_err(
-                        crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
+                    let mut output = super::super::types::error::builders::ConflictExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_conflict_exception::de_conflict_exception_json_err(_response_body, output).map_err(
+                        super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
                     )?;
                     let output = output.meta(generic);
                     output.build()
@@ -41,18 +41,18 @@
             })
         }
         "InsufficientPermissionsException" => {
-            crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::InsufficientPermissionsException(
+            super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::InsufficientPermissionsException(
                 {
                     #[allow(unused_mut)]
                     let mut tmp = {
                         #[allow(unused_mut)]
-                        let mut output = crate::types::error::builders::InsufficientPermissionsExceptionBuilder::default();
-                        output = crate::protocol_serde::shape_insufficient_permissions_exception::de_insufficient_permissions_exception_json_err(
+                        let mut output = super::super::types::error::builders::InsufficientPermissionsExceptionBuilder::default();
+                        output = super::super::protocol_serde::shape_insufficient_permissions_exception::de_insufficient_permissions_exception_json_err(
                             _response_body,
                             output,
                         )
                         .map_err(
-                            crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
+                            super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
                         )?;
                         let output = output.meta(generic);
                         output.build()
@@ -65,14 +65,14 @@
             )
         }
         "LimitExceededException" => {
-            crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::LimitExceededException({
+            super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::LimitExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
+                    let mut output = super::super::types::error::builders::LimitExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
                         .map_err(
-                            crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
+                            super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
                         )?;
                     let output = output.meta(generic);
                     output.build()
@@ -84,13 +84,13 @@
             })
         }
         "ValidationException" => {
-            crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::ValidationException({
+            super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::ValidationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output).map_err(
-                        crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
+                    let mut output = super::super::types::error::builders::ValidationExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output).map_err(
+                        super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled,
                     )?;
                     let output = output.meta(generic);
                     output.build()
@@ -101,7 +101,7 @@
                 tmp
             })
         }
-        _ => crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::generic(generic),
+        _ => super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::generic(generic),
     })
 }

@@ -111,18 +111,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderOutput,
-    crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError,
+    super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderOutput,
+    super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::put_service_linked_configuration_recorder::builders::PutServiceLinkedConfigurationRecorderOutputBuilder::default();
-        output = crate::protocol_serde::shape_put_service_linked_configuration_recorder::de_put_service_linked_configuration_recorder(
+            super::super::operation::put_service_linked_configuration_recorder::builders::PutServiceLinkedConfigurationRecorderOutputBuilder::default();
+        output = super::super::protocol_serde::shape_put_service_linked_configuration_recorder::de_put_service_linked_configuration_recorder(
             _response_body,
             output,
         )
-        .map_err(crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled)?;
+        .map_err(super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -129,11 +129,11 @@
 }

 pub fn ser_put_service_linked_configuration_recorder_input(
-    input: &crate::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderInput,
+    input: &super::super::operation::put_service_linked_configuration_recorder::PutServiceLinkedConfigurationRecorderInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_service_linked_configuration_recorder_input::ser_put_service_linked_configuration_recorder_input_input(
+    super::super::protocol_serde::shape_put_service_linked_configuration_recorder_input::ser_put_service_linked_configuration_recorder_input_input(
         &mut object,
         input,
     )?;
@@ -143,12 +143,12 @@

 pub(crate) fn de_put_service_linked_configuration_recorder(
     _value: &[u8],
-    mut builder: crate::operation::put_service_linked_configuration_recorder::builders::PutServiceLinkedConfigurationRecorderOutputBuilder,
+    mut builder: super::super::operation::put_service_linked_configuration_recorder::builders::PutServiceLinkedConfigurationRecorderOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::put_service_linked_configuration_recorder::builders::PutServiceLinkedConfigurationRecorderOutputBuilder,
+    super::super::operation::put_service_linked_configuration_recorder::builders::PutServiceLinkedConfigurationRecorderOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
