--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::deliver_config_snapshot::DeliverConfigSnapshotOutput,
-    crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError,
+    super::operation::deliver_config_snapshot::DeliverConfigSnapshotOutput,
+    super::operation::deliver_config_snapshot::DeliverConfigSnapshotError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled(generic)),
+        None => return Err(super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "NoAvailableConfigurationRecorderException" => {
-            crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::NoAvailableConfigurationRecorderException({
+            super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::NoAvailableConfigurationRecorderException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoAvailableConfigurationRecorderExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_available_configuration_recorder_exception::de_no_available_configuration_recorder_exception_json_err(_response_body, output).map_err(crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
+                    let mut output = super::types::error::builders::NoAvailableConfigurationRecorderExceptionBuilder::default();
+                    output = super::protocol_serde::shape_no_available_configuration_recorder_exception::de_no_available_configuration_recorder_exception_json_err(_response_body, output).map_err(super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -37,12 +37,12 @@
             })
         }
         "NoRunningConfigurationRecorderException" => {
-            crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::NoRunningConfigurationRecorderException({
+            super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::NoRunningConfigurationRecorderException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoRunningConfigurationRecorderExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_running_configuration_recorder_exception::de_no_running_configuration_recorder_exception_json_err(_response_body, output).map_err(crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
+                    let mut output = super::types::error::builders::NoRunningConfigurationRecorderExceptionBuilder::default();
+                    output = super::protocol_serde::shape_no_running_configuration_recorder_exception::de_no_running_configuration_recorder_exception_json_err(_response_body, output).map_err(super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -52,16 +52,16 @@
                 tmp
             })
         }
-        "NoSuchDeliveryChannelException" => crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::NoSuchDeliveryChannelException({
+        "NoSuchDeliveryChannelException" => super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::NoSuchDeliveryChannelException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchDeliveryChannelExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_delivery_channel_exception::de_no_such_delivery_channel_exception_json_err(
+                let mut output = super::types::error::builders::NoSuchDeliveryChannelExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_delivery_channel_exception::de_no_such_delivery_channel_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
+                .map_err(super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,7 +70,7 @@
             }
             tmp
         }),
-        _ => crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::generic(generic),
+        _ => super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::generic(generic),
     })
 }

@@ -80,14 +80,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::deliver_config_snapshot::DeliverConfigSnapshotOutput,
-    crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError,
+    super::operation::deliver_config_snapshot::DeliverConfigSnapshotOutput,
+    super::operation::deliver_config_snapshot::DeliverConfigSnapshotError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::deliver_config_snapshot::builders::DeliverConfigSnapshotOutputBuilder::default();
-        output = crate::protocol_serde::shape_deliver_config_snapshot::de_deliver_config_snapshot(_response_body, output)
-            .map_err(crate::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
+        let mut output = super::operation::deliver_config_snapshot::builders::DeliverConfigSnapshotOutputBuilder::default();
+        output = super::protocol_serde::shape_deliver_config_snapshot::de_deliver_config_snapshot(_response_body, output)
+            .map_err(super::operation::deliver_config_snapshot::DeliverConfigSnapshotError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -94,11 +94,11 @@
 }

 pub fn ser_deliver_config_snapshot_input(
-    input: &crate::operation::deliver_config_snapshot::DeliverConfigSnapshotInput,
+    input: &super::operation::deliver_config_snapshot::DeliverConfigSnapshotInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_deliver_config_snapshot_input::ser_deliver_config_snapshot_input_input(&mut object, input)?;
+    super::protocol_serde::shape_deliver_config_snapshot_input::ser_deliver_config_snapshot_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -105,12 +105,12 @@

 pub(crate) fn de_deliver_config_snapshot(
     _value: &[u8],
-    mut builder: crate::operation::deliver_config_snapshot::builders::DeliverConfigSnapshotOutputBuilder,
+    mut builder: super::operation::deliver_config_snapshot::builders::DeliverConfigSnapshotOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::deliver_config_snapshot::builders::DeliverConfigSnapshotOutputBuilder,
+    super::operation::deliver_config_snapshot::builders::DeliverConfigSnapshotOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
