--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ComplianceSummaryByResourceType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ComplianceSummaryByResourceType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ComplianceSummaryByResourceTypeBuilder::default();
+            let mut builder = super::types::builders::ComplianceSummaryByResourceTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -29,7 +29,7 @@
                             );
                         }
                         "ComplianceSummary" => {
-                            builder = builder.set_compliance_summary(crate::protocol_serde::shape_compliance_summary::de_compliance_summary(
+                            builder = builder.set_compliance_summary(super::protocol_serde::shape_compliance_summary::de_compliance_summary(
                                 tokens,
                                 _value,
                                 depth + 1,
