--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::FailedDeleteRemediationExceptionsBatch>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::FailedDeleteRemediationExceptionsBatch>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::FailedDeleteRemediationExceptionsBatchBuilder::default();
+            let mut builder = super::super::types::builders::FailedDeleteRemediationExceptionsBatchBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -30,7 +30,7 @@
                         }
                         "FailedItems" => {
                             builder = builder.set_failed_items(
-                                crate::protocol_serde::shape_remediation_exception_resource_keys::de_remediation_exception_resource_keys(
+                                super::super::protocol_serde::shape_remediation_exception_resource_keys::de_remediation_exception_resource_keys(
                                     tokens,
                                     _value,
                                     depth + 1,
