--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::collections::HashMap<::std::string::String, crate::types::RemediationParameterValue>>,
+    Option<::std::collections::HashMap<::std::string::String, super::types::RemediationParameterValue>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -25,7 +25,7 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => {
                         let key = key.to_unescaped().map(|u| u.into_owned())?;
                         let value =
-                            crate::protocol_serde::shape_remediation_parameter_value::de_remediation_parameter_value(tokens, _value, depth + 1)?;
+                            super::protocol_serde::shape_remediation_parameter_value::de_remediation_parameter_value(tokens, _value, depth + 1)?;
                         match value {
                             Some(value) => {
                                 map.insert(key, value);
