--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::RecordingMode>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::RecordingMode>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::RecordingModeBuilder::default();
+            let mut builder = super::super::types::builders::RecordingModeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -24,13 +24,13 @@
                         "recordingFrequency" => {
                             builder = builder.set_recording_frequency(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::RecordingFrequency::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::RecordingFrequency::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "recordingModeOverrides" => {
                             builder = builder.set_recording_mode_overrides(
-                                crate::protocol_serde::shape_recording_mode_overrides::de_recording_mode_overrides(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_recording_mode_overrides::de_recording_mode_overrides(tokens, _value, depth + 1)?,
                             );
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
@@ -42,7 +42,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::recording_mode_correct_errors(builder).build().map_err(
+            Ok(Some(super::super::serde_util::recording_mode_correct_errors(builder).build().map_err(
                 |err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err),
             )?))
         }
@@ -54,7 +54,7 @@

 pub fn ser_recording_mode(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::RecordingMode,
+    input: &super::super::types::RecordingMode,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     {
         object.key("recordingFrequency").string(input.recording_frequency.as_str());
@@ -65,7 +65,7 @@
             {
                 #[allow(unused_mut)]
                 let mut object_4 = array_2.value().start_object();
-                crate::protocol_serde::shape_recording_mode_override::ser_recording_mode_override(&mut object_4, item_3)?;
+                super::super::protocol_serde::shape_recording_mode_override::ser_recording_mode_override(&mut object_4, item_3)?;
                 object_4.finish();
             }
         }
