--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<::std::vec::Vec<crate::types::AggregateResourceIdentifier>>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<::std::vec::Vec<super::super::types::AggregateResourceIdentifier>>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -24,7 +24,7 @@
                     }
                     _ => {
                         let value =
-                            crate::protocol_serde::shape_aggregate_resource_identifier::de_aggregate_resource_identifier(tokens, _value, depth + 1)?;
+                            super::super::protocol_serde::shape_aggregate_resource_identifier::de_aggregate_resource_identifier(tokens, _value, depth + 1)?;
                         if let Some(value) = value {
                             items.push(value);
                         } else {
