--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_configuration_recorder::PutConfigurationRecorderOutput,
-    crate::operation::put_configuration_recorder::PutConfigurationRecorderError,
+    super::operation::put_configuration_recorder::PutConfigurationRecorderOutput,
+    super::operation::put_configuration_recorder::PutConfigurationRecorderError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled(
+            return Err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled(
                 generic,
             ))
         }
@@ -25,12 +25,12 @@
     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InvalidConfigurationRecorderNameException" => {
-            crate::operation::put_configuration_recorder::PutConfigurationRecorderError::InvalidConfigurationRecorderNameException({
+            super::operation::put_configuration_recorder::PutConfigurationRecorderError::InvalidConfigurationRecorderNameException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidConfigurationRecorderNameExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_configuration_recorder_name_exception::de_invalid_configuration_recorder_name_exception_json_err(_response_body, output).map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidConfigurationRecorderNameExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_configuration_recorder_name_exception::de_invalid_configuration_recorder_name_exception_json_err(_response_body, output).map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -41,16 +41,16 @@
             })
         }
         "InvalidRecordingGroupException" => {
-            crate::operation::put_configuration_recorder::PutConfigurationRecorderError::InvalidRecordingGroupException({
+            super::operation::put_configuration_recorder::PutConfigurationRecorderError::InvalidRecordingGroupException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidRecordingGroupExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_recording_group_exception::de_invalid_recording_group_exception_json_err(
+                    let mut output = super::types::error::builders::InvalidRecordingGroupExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_recording_group_exception::de_invalid_recording_group_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+                    .map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -60,13 +60,13 @@
                 tmp
             })
         }
-        "InvalidRoleException" => crate::operation::put_configuration_recorder::PutConfigurationRecorderError::InvalidRoleException({
+        "InvalidRoleException" => super::operation::put_configuration_recorder::PutConfigurationRecorderError::InvalidRoleException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRoleExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_role_exception::de_invalid_role_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidRoleExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_role_exception::de_invalid_role_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -76,12 +76,12 @@
             tmp
         }),
         "MaxNumberOfConfigurationRecordersExceededException" => {
-            crate::operation::put_configuration_recorder::PutConfigurationRecorderError::MaxNumberOfConfigurationRecordersExceededException({
+            super::operation::put_configuration_recorder::PutConfigurationRecorderError::MaxNumberOfConfigurationRecordersExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::MaxNumberOfConfigurationRecordersExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_max_number_of_configuration_recorders_exceeded_exception::de_max_number_of_configuration_recorders_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+                    let mut output = super::types::error::builders::MaxNumberOfConfigurationRecordersExceededExceptionBuilder::default();
+                    output = super::protocol_serde::shape_max_number_of_configuration_recorders_exceeded_exception::de_max_number_of_configuration_recorders_exceeded_exception_json_err(_response_body, output).map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -91,14 +91,14 @@
                 tmp
             })
         }
-        "UnmodifiableEntityException" => crate::operation::put_configuration_recorder::PutConfigurationRecorderError::UnmodifiableEntityException({
+        "UnmodifiableEntityException" => super::operation::put_configuration_recorder::PutConfigurationRecorderError::UnmodifiableEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnmodifiableEntityExceptionBuilder::default();
+                let mut output = super::types::error::builders::UnmodifiableEntityExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_unmodifiable_entity_exception::de_unmodifiable_entity_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+                    super::protocol_serde::shape_unmodifiable_entity_exception::de_unmodifiable_entity_exception_json_err(_response_body, output)
+                        .map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -107,13 +107,13 @@
             }
             tmp
         }),
-        "ValidationException" => crate::operation::put_configuration_recorder::PutConfigurationRecorderError::ValidationException({
+        "ValidationException" => super::operation::put_configuration_recorder::PutConfigurationRecorderError::ValidationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
+                let mut output = super::types::error::builders::ValidationExceptionBuilder::default();
+                output = super::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
+                    .map_err(super::operation::put_configuration_recorder::PutConfigurationRecorderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -122,7 +122,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_configuration_recorder::PutConfigurationRecorderError::generic(generic),
+        _ => super::operation::put_configuration_recorder::PutConfigurationRecorderError::generic(generic),
     })
 }

@@ -132,12 +132,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_configuration_recorder::PutConfigurationRecorderOutput,
-    crate::operation::put_configuration_recorder::PutConfigurationRecorderError,
+    super::operation::put_configuration_recorder::PutConfigurationRecorderOutput,
+    super::operation::put_configuration_recorder::PutConfigurationRecorderError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_configuration_recorder::builders::PutConfigurationRecorderOutputBuilder::default();
+        let mut output = super::operation::put_configuration_recorder::builders::PutConfigurationRecorderOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -144,11 +144,11 @@
 }

 pub fn ser_put_configuration_recorder_input(
-    input: &crate::operation::put_configuration_recorder::PutConfigurationRecorderInput,
+    input: &super::operation::put_configuration_recorder::PutConfigurationRecorderInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_configuration_recorder_input::ser_put_configuration_recorder_input_input(&mut object, input)?;
+    super::protocol_serde::shape_put_configuration_recorder_input::ser_put_configuration_recorder_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
