--- original
+++ modified
@@ -5,29 +5,29 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresOutput,
-    crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError,
+    super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresOutput,
+    super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled(generic)),
+        None => return Err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InvalidLimitException" => {
-            crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::InvalidLimitException({
+            super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::InvalidLimitException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidLimitExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_limit_exception::de_invalid_limit_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
+                    let mut output = super::super::types::error::builders::InvalidLimitExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_limit_exception::de_invalid_limit_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -38,14 +38,14 @@
             })
         }
         "InvalidNextTokenException" => {
-            crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::InvalidNextTokenException({
+            super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::InvalidNextTokenException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidNextTokenExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::InvalidNextTokenExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_invalid_next_token_exception::de_invalid_next_token_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
+                        super::super::protocol_serde::shape_invalid_next_token_exception::de_invalid_next_token_exception_json_err(_response_body, output)
+                            .map_err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -56,16 +56,16 @@
             })
         }
         "InvalidParameterValueException" => {
-            crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::InvalidParameterValueException({
+            super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
+                    .map_err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -75,7 +75,7 @@
                 tmp
             })
         }
-        _ => crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::generic(generic),
+        _ => super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::generic(generic),
     })
 }

@@ -85,29 +85,29 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresOutput,
-    crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError,
+    super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresOutput,
+    super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresOutputBuilder::default();
+            super::super::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_list_conformance_pack_compliance_scores::de_list_conformance_pack_compliance_scores(_response_body, output)
-                .map_err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
+            super::super::protocol_serde::shape_list_conformance_pack_compliance_scores::de_list_conformance_pack_compliance_scores(_response_body, output)
+                .map_err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_conformance_pack_compliance_scores_output_output_correct_errors(output)
+        super::super::serde_util::list_conformance_pack_compliance_scores_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?
+            .map_err(super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresError::unhandled)?
     })
 }

 pub fn ser_list_conformance_pack_compliance_scores_input(
-    input: &crate::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresInput,
+    input: &super::super::operation::list_conformance_pack_compliance_scores::ListConformancePackComplianceScoresInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_list_conformance_pack_compliance_scores_input::ser_list_conformance_pack_compliance_scores_input_input(
+    super::super::protocol_serde::shape_list_conformance_pack_compliance_scores_input::ser_list_conformance_pack_compliance_scores_input_input(
         &mut object,
         input,
     )?;
@@ -117,12 +117,12 @@

 pub(crate) fn de_list_conformance_pack_compliance_scores(
     _value: &[u8],
-    mut builder: crate::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresOutputBuilder,
+    mut builder: super::super::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresOutputBuilder,
+    super::super::operation::list_conformance_pack_compliance_scores::builders::ListConformancePackComplianceScoresOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -140,7 +140,7 @@
                 }
                 "ConformancePackComplianceScores" => {
                     builder = builder.set_conformance_pack_compliance_scores(
-                        crate::protocol_serde::shape_conformance_pack_compliance_scores::de_conformance_pack_compliance_scores(
+                        super::super::protocol_serde::shape_conformance_pack_compliance_scores::de_conformance_pack_compliance_scores(
                             tokens,
                             _value,
                             depth + 1,
