--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_organization_config_rule::PutOrganizationConfigRuleOutput,
-    crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError,
+    super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleOutput,
+    super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled(
+            return Err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled(
                 generic,
             ))
         }
@@ -25,16 +25,16 @@
     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InsufficientPermissionsException" => {
-            crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::InsufficientPermissionsException({
+            super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::InsufficientPermissionsException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InsufficientPermissionsExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_insufficient_permissions_exception::de_insufficient_permissions_exception_json_err(
+                    let mut output = super::super::types::error::builders::InsufficientPermissionsExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_insufficient_permissions_exception::de_insufficient_permissions_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                    .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -45,16 +45,16 @@
             })
         }
         "InvalidParameterValueException" => {
-            crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::InvalidParameterValueException({
+            super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                    .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -65,12 +65,12 @@
             })
         }
         "MaxNumberOfOrganizationConfigRulesExceededException" => {
-            crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::MaxNumberOfOrganizationConfigRulesExceededException({
+            super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::MaxNumberOfOrganizationConfigRulesExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::MaxNumberOfOrganizationConfigRulesExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_max_number_of_organization_config_rules_exceeded_exception::de_max_number_of_organization_config_rules_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                    let mut output = super::super::types::error::builders::MaxNumberOfOrganizationConfigRulesExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_max_number_of_organization_config_rules_exceeded_exception::de_max_number_of_organization_config_rules_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -81,16 +81,16 @@
             })
         }
         "NoAvailableOrganizationException" => {
-            crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::NoAvailableOrganizationException({
+            super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::NoAvailableOrganizationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoAvailableOrganizationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_available_organization_exception::de_no_available_organization_exception_json_err(
+                    let mut output = super::super::types::error::builders::NoAvailableOrganizationExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_no_available_organization_exception::de_no_available_organization_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                    .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -101,16 +101,16 @@
             })
         }
         "OrganizationAccessDeniedException" => {
-            crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::OrganizationAccessDeniedException({
+            super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::OrganizationAccessDeniedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationAccessDeniedExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_access_denied_exception::de_organization_access_denied_exception_json_err(
+                    let mut output = super::super::types::error::builders::OrganizationAccessDeniedExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_organization_access_denied_exception::de_organization_access_denied_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                    .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -121,12 +121,12 @@
             })
         }
         "OrganizationAllFeaturesNotEnabledException" => {
-            crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::OrganizationAllFeaturesNotEnabledException({
+            super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::OrganizationAllFeaturesNotEnabledException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationAllFeaturesNotEnabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_all_features_not_enabled_exception::de_organization_all_features_not_enabled_exception_json_err(_response_body, output).map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                    let mut output = super::super::types::error::builders::OrganizationAllFeaturesNotEnabledExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_organization_all_features_not_enabled_exception::de_organization_all_features_not_enabled_exception_json_err(_response_body, output).map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -136,13 +136,13 @@
                 tmp
             })
         }
-        "ResourceInUseException" => crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::ResourceInUseException({
+        "ResourceInUseException" => super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::ResourceInUseException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceInUseExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_in_use_exception::de_resource_in_use_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceInUseExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_in_use_exception::de_resource_in_use_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -151,13 +151,13 @@
             }
             tmp
         }),
-        "ValidationException" => crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::ValidationException({
+        "ValidationException" => super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::ValidationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ValidationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+                let mut output = super::super::types::error::builders::ValidationExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_validation_exception::de_validation_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -166,7 +166,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::generic(generic),
+        _ => super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::generic(generic),
     })
 }

@@ -176,14 +176,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_organization_config_rule::PutOrganizationConfigRuleOutput,
-    crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError,
+    super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleOutput,
+    super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleOutputBuilder::default();
-        output = crate::protocol_serde::shape_put_organization_config_rule::de_put_organization_config_rule(_response_body, output)
-            .map_err(crate::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
+        let mut output = super::super::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleOutputBuilder::default();
+        output = super::super::protocol_serde::shape_put_organization_config_rule::de_put_organization_config_rule(_response_body, output)
+            .map_err(super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -190,11 +190,11 @@
 }

 pub fn ser_put_organization_config_rule_input(
-    input: &crate::operation::put_organization_config_rule::PutOrganizationConfigRuleInput,
+    input: &super::super::operation::put_organization_config_rule::PutOrganizationConfigRuleInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_put_organization_config_rule_input::ser_put_organization_config_rule_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_put_organization_config_rule_input::ser_put_organization_config_rule_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -201,12 +201,12 @@

 pub(crate) fn de_put_organization_config_rule(
     _value: &[u8],
-    mut builder: crate::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleOutputBuilder,
+    mut builder: super::super::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleOutputBuilder,
+    super::super::operation::put_organization_config_rule::builders::PutOrganizationConfigRuleOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
