--- original
+++ modified
@@ -5,33 +5,33 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsOutput,
-    crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError,
+    super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsOutput,
+    super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled(generic)),
+        None => return Err(super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "NoSuchRemediationExceptionException" => {
-            crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::NoSuchRemediationExceptionException({
+            super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::NoSuchRemediationExceptionException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoSuchRemediationExceptionExceptionBuilder::default();
+                    let mut output = super::types::error::builders::NoSuchRemediationExceptionExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_no_such_remediation_exception_exception::de_no_such_remediation_exception_exception_json_err(
+                        super::protocol_serde::shape_no_such_remediation_exception_exception::de_no_such_remediation_exception_exception_json_err(
                             _response_body,
                             output,
                         )
-                        .map_err(crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled)?;
+                        .map_err(super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -41,7 +41,7 @@
                 tmp
             })
         }
-        _ => crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::generic(generic),
+        _ => super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::generic(generic),
     })
 }

@@ -51,14 +51,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsOutput,
-    crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError,
+    super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsOutput,
+    super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_remediation_exceptions::builders::DeleteRemediationExceptionsOutputBuilder::default();
-        output = crate::protocol_serde::shape_delete_remediation_exceptions::de_delete_remediation_exceptions(_response_body, output)
-            .map_err(crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled)?;
+        let mut output = super::operation::delete_remediation_exceptions::builders::DeleteRemediationExceptionsOutputBuilder::default();
+        output = super::protocol_serde::shape_delete_remediation_exceptions::de_delete_remediation_exceptions(_response_body, output)
+            .map_err(super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -65,11 +65,11 @@
 }

 pub fn ser_delete_remediation_exceptions_input(
-    input: &crate::operation::delete_remediation_exceptions::DeleteRemediationExceptionsInput,
+    input: &super::operation::delete_remediation_exceptions::DeleteRemediationExceptionsInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_delete_remediation_exceptions_input::ser_delete_remediation_exceptions_input_input(&mut object, input)?;
+    super::protocol_serde::shape_delete_remediation_exceptions_input::ser_delete_remediation_exceptions_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -76,12 +76,12 @@

 pub(crate) fn de_delete_remediation_exceptions(
     _value: &[u8],
-    mut builder: crate::operation::delete_remediation_exceptions::builders::DeleteRemediationExceptionsOutputBuilder,
+    mut builder: super::operation::delete_remediation_exceptions::builders::DeleteRemediationExceptionsOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::delete_remediation_exceptions::builders::DeleteRemediationExceptionsOutputBuilder,
+    super::operation::delete_remediation_exceptions::builders::DeleteRemediationExceptionsOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -93,7 +93,7 @@
                 match key.to_unescaped()?.as_ref() {
                     "FailedBatches" => {
                         builder = builder.set_failed_batches(
-                            crate::protocol_serde::shape_failed_delete_remediation_exceptions_batches::de_failed_delete_remediation_exceptions_batches(tokens, _value, depth + 1)?
+                            super::protocol_serde::shape_failed_delete_remediation_exceptions_batches::de_failed_delete_remediation_exceptions_batches(tokens, _value, depth + 1)?
                         );
                     }
                     _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
