--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteAggregationAuthorizationInput {
     /// Creates a new builder-style object to manufacture [`DeleteAggregationAuthorizationInput`](crate::operation::delete_aggregation_authorization::DeleteAggregationAuthorizationInput).
-    pub fn builder() -> crate::operation::delete_aggregation_authorization::builders::DeleteAggregationAuthorizationInputBuilder {
-        crate::operation::delete_aggregation_authorization::builders::DeleteAggregationAuthorizationInputBuilder::default()
+    pub fn builder() -> super::operation::delete_aggregation_authorization::builders::DeleteAggregationAuthorizationInputBuilder {
+        super::operation::delete_aggregation_authorization::builders::DeleteAggregationAuthorizationInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_aggregation_authorization::DeleteAggregationAuthorizationInput,
+        super::operation::delete_aggregation_authorization::DeleteAggregationAuthorizationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_aggregation_authorization::DeleteAggregationAuthorizationInput {
+        ::std::result::Result::Ok(super::operation::delete_aggregation_authorization::DeleteAggregationAuthorizationInput {
             authorized_account_id: self.authorized_account_id,
             authorized_aws_region: self.authorized_aws_region,
         })
