--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl DescribeConfigurationAggregatorsInput {
     /// Creates a new builder-style object to manufacture [`DescribeConfigurationAggregatorsInput`](crate::operation::describe_configuration_aggregators::DescribeConfigurationAggregatorsInput).
-    pub fn builder() -> crate::operation::describe_configuration_aggregators::builders::DescribeConfigurationAggregatorsInputBuilder {
-        crate::operation::describe_configuration_aggregators::builders::DescribeConfigurationAggregatorsInputBuilder::default()
+    pub fn builder() -> super::operation::describe_configuration_aggregators::builders::DescribeConfigurationAggregatorsInputBuilder {
+        super::operation::describe_configuration_aggregators::builders::DescribeConfigurationAggregatorsInputBuilder::default()
     }
 }

@@ -94,11 +94,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_configuration_aggregators::DescribeConfigurationAggregatorsInput,
+        super::operation::describe_configuration_aggregators::DescribeConfigurationAggregatorsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::describe_configuration_aggregators::DescribeConfigurationAggregatorsInput {
+            super::operation::describe_configuration_aggregators::DescribeConfigurationAggregatorsInput {
                 configuration_aggregator_names: self.configuration_aggregator_names,
                 next_token: self.next_token,
                 limit: self.limit,
