--- original
+++ modified
@@ -46,8 +46,8 @@
 }
 impl AssociateKmsKeyInput {
     /// Creates a new builder-style object to manufacture [`AssociateKmsKeyInput`](crate::operation::associate_kms_key::AssociateKmsKeyInput).
-    pub fn builder() -> crate::operation::associate_kms_key::builders::AssociateKmsKeyInputBuilder {
-        crate::operation::associate_kms_key::builders::AssociateKmsKeyInputBuilder::default()
+    pub fn builder() -> super::operation::associate_kms_key::builders::AssociateKmsKeyInputBuilder {
+        super::operation::associate_kms_key::builders::AssociateKmsKeyInputBuilder::default()
     }
 }

@@ -136,8 +136,8 @@
     /// Consumes the builder and constructs a [`AssociateKmsKeyInput`](crate::operation::associate_kms_key::AssociateKmsKeyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::associate_kms_key::AssociateKmsKeyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::associate_kms_key::AssociateKmsKeyInput {
+    ) -> ::std::result::Result<super::operation::associate_kms_key::AssociateKmsKeyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::associate_kms_key::AssociateKmsKeyInput {
             log_group_name: self.log_group_name,
             kms_key_id: self.kms_key_id,
             resource_identifier: self.resource_identifier,
