--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl VerifySoftwareTokenInput {
     /// Creates a new builder-style object to manufacture [`VerifySoftwareTokenInput`](crate::operation::verify_software_token::VerifySoftwareTokenInput).
-    pub fn builder() -> crate::operation::verify_software_token::builders::VerifySoftwareTokenInputBuilder {
-        crate::operation::verify_software_token::builders::VerifySoftwareTokenInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::verify_software_token::builders::VerifySoftwareTokenInputBuilder {
+        super::super::super::operation::verify_software_token::builders::VerifySoftwareTokenInputBuilder::default()
     }
 }

@@ -117,9 +117,9 @@
     /// Consumes the builder and constructs a [`VerifySoftwareTokenInput`](crate::operation::verify_software_token::VerifySoftwareTokenInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::verify_software_token::VerifySoftwareTokenInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::verify_software_token::VerifySoftwareTokenInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::verify_software_token::VerifySoftwareTokenInput {
+        ::std::result::Result::Ok(super::super::super::operation::verify_software_token::VerifySoftwareTokenInput {
             access_token: self.access_token,
             session: self.session,
             user_code: self.user_code,
