--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetPublicAccessBlockInput {
     /// Creates a new builder-style object to manufacture [`GetPublicAccessBlockInput`](crate::operation::get_public_access_block::GetPublicAccessBlockInput).
-    pub fn builder() -> crate::operation::get_public_access_block::builders::GetPublicAccessBlockInputBuilder {
-        crate::operation::get_public_access_block::builders::GetPublicAccessBlockInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_public_access_block::builders::GetPublicAccessBlockInputBuilder {
+        super::super::super::operation::get_public_access_block::builders::GetPublicAccessBlockInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`GetPublicAccessBlockInput`](crate::operation::get_public_access_block::GetPublicAccessBlockInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_public_access_block::GetPublicAccessBlockInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_public_access_block::GetPublicAccessBlockInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_public_access_block::GetPublicAccessBlockInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_public_access_block::GetPublicAccessBlockInput {
             bucket: self.bucket,
             expected_bucket_owner: self.expected_bucket_owner,
         })
