--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl ListTenantsInput {
     /// Creates a new builder-style object to manufacture [`ListTenantsInput`](crate::operation::list_tenants::ListTenantsInput).
-    pub fn builder() -> crate::operation::list_tenants::builders::ListTenantsInputBuilder {
-        crate::operation::list_tenants::builders::ListTenantsInputBuilder::default()
+    pub fn builder() -> super::operation::list_tenants::builders::ListTenantsInputBuilder {
+        super::operation::list_tenants::builders::ListTenantsInputBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.page_size
     }
     /// Consumes the builder and constructs a [`ListTenantsInput`](crate::operation::list_tenants::ListTenantsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_tenants::ListTenantsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_tenants::ListTenantsInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_tenants::ListTenantsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_tenants::ListTenantsInput {
             next_token: self.next_token,
             page_size: self.page_size,
         })
