--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl PutDataProtectionPolicyInput {
     /// Creates a new builder-style object to manufacture [`PutDataProtectionPolicyInput`](crate::operation::put_data_protection_policy::PutDataProtectionPolicyInput).
-    pub fn builder() -> crate::operation::put_data_protection_policy::builders::PutDataProtectionPolicyInputBuilder {
-        crate::operation::put_data_protection_policy::builders::PutDataProtectionPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_data_protection_policy::builders::PutDataProtectionPolicyInputBuilder {
+        super::super::super::operation::put_data_protection_policy::builders::PutDataProtectionPolicyInputBuilder::default()
     }
 }

@@ -82,10 +82,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_data_protection_policy::PutDataProtectionPolicyInput,
+        super::super::super::operation::put_data_protection_policy::PutDataProtectionPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_data_protection_policy::PutDataProtectionPolicyInput {
+        ::std::result::Result::Ok(super::super::super::operation::put_data_protection_policy::PutDataProtectionPolicyInput {
             resource_arn: self.resource_arn,
             data_protection_policy: self.data_protection_policy,
         })
