--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl ConfirmSubscriptionInput {
     /// Creates a new builder-style object to manufacture [`ConfirmSubscriptionInput`](crate::operation::confirm_subscription::ConfirmSubscriptionInput).
-    pub fn builder() -> crate::operation::confirm_subscription::builders::ConfirmSubscriptionInputBuilder {
-        crate::operation::confirm_subscription::builders::ConfirmSubscriptionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::confirm_subscription::builders::ConfirmSubscriptionInputBuilder {
+        super::super::super::operation::confirm_subscription::builders::ConfirmSubscriptionInputBuilder::default()
     }
 }

@@ -88,9 +88,9 @@
     /// Consumes the builder and constructs a [`ConfirmSubscriptionInput`](crate::operation::confirm_subscription::ConfirmSubscriptionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::confirm_subscription::ConfirmSubscriptionInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::confirm_subscription::ConfirmSubscriptionInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::confirm_subscription::ConfirmSubscriptionInput {
+        ::std::result::Result::Ok(super::super::super::operation::confirm_subscription::ConfirmSubscriptionInput {
             topic_arn: self.topic_arn,
             token: self.token,
             authenticate_on_unsubscribe: self.authenticate_on_unsubscribe,
