--- original
+++ modified
@@ -6,11 +6,11 @@
     /// <p>The name of the capacity provider to update.</p>
     pub capacity_provider_name: ::std::option::Option<::std::string::String>,
     /// <p>The updated scaling configuration for the capacity provider.</p>
-    pub capacity_provider_scaling_config: ::std::option::Option<crate::types::CapacityProviderScalingConfig>,
+    pub capacity_provider_scaling_config: ::std::option::Option<super::super::super::types::CapacityProviderScalingConfig>,
     /// <p>Configuration for tag propagation to managed resources launched by the capacity provider.</p>
-    pub propagate_tags: ::std::option::Option<crate::types::PropagateTags>,
+    pub propagate_tags: ::std::option::Option<super::super::super::types::PropagateTags>,
     /// <p>The updated telemetry configuration for the capacity provider.</p>
-    pub telemetry_config: ::std::option::Option<crate::types::CapacityProviderTelemetryConfig>,
+    pub telemetry_config: ::std::option::Option<super::super::super::types::CapacityProviderTelemetryConfig>,
 }
 impl UpdateCapacityProviderInput {
     /// <p>The name of the capacity provider to update.</p>
@@ -18,22 +18,22 @@
         self.capacity_provider_name.as_deref()
     }
     /// <p>The updated scaling configuration for the capacity provider.</p>
-    pub fn capacity_provider_scaling_config(&self) -> ::std::option::Option<&crate::types::CapacityProviderScalingConfig> {
+    pub fn capacity_provider_scaling_config(&self) -> ::std::option::Option<&super::super::super::types::CapacityProviderScalingConfig> {
         self.capacity_provider_scaling_config.as_ref()
     }
     /// <p>Configuration for tag propagation to managed resources launched by the capacity provider.</p>
-    pub fn propagate_tags(&self) -> ::std::option::Option<&crate::types::PropagateTags> {
+    pub fn propagate_tags(&self) -> ::std::option::Option<&super::super::super::types::PropagateTags> {
         self.propagate_tags.as_ref()
     }
     /// <p>The updated telemetry configuration for the capacity provider.</p>
-    pub fn telemetry_config(&self) -> ::std::option::Option<&crate::types::CapacityProviderTelemetryConfig> {
+    pub fn telemetry_config(&self) -> ::std::option::Option<&super::super::super::types::CapacityProviderTelemetryConfig> {
         self.telemetry_config.as_ref()
     }
 }
 impl UpdateCapacityProviderInput {
     /// Creates a new builder-style object to manufacture [`UpdateCapacityProviderInput`](crate::operation::update_capacity_provider::UpdateCapacityProviderInput).
-    pub fn builder() -> crate::operation::update_capacity_provider::builders::UpdateCapacityProviderInputBuilder {
-        crate::operation::update_capacity_provider::builders::UpdateCapacityProviderInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_capacity_provider::builders::UpdateCapacityProviderInputBuilder {
+        super::super::super::operation::update_capacity_provider::builders::UpdateCapacityProviderInputBuilder::default()
     }
 }

@@ -42,9 +42,9 @@
 #[non_exhaustive]
 pub struct UpdateCapacityProviderInputBuilder {
     pub(crate) capacity_provider_name: ::std::option::Option<::std::string::String>,
-    pub(crate) capacity_provider_scaling_config: ::std::option::Option<crate::types::CapacityProviderScalingConfig>,
-    pub(crate) propagate_tags: ::std::option::Option<crate::types::PropagateTags>,
-    pub(crate) telemetry_config: ::std::option::Option<crate::types::CapacityProviderTelemetryConfig>,
+    pub(crate) capacity_provider_scaling_config: ::std::option::Option<super::super::super::types::CapacityProviderScalingConfig>,
+    pub(crate) propagate_tags: ::std::option::Option<super::super::super::types::PropagateTags>,
+    pub(crate) telemetry_config: ::std::option::Option<super::super::super::types::CapacityProviderTelemetryConfig>,
 }
 impl UpdateCapacityProviderInputBuilder {
     /// <p>The name of the capacity provider to update.</p>
@@ -63,45 +63,45 @@
         &self.capacity_provider_name
     }
     /// <p>The updated scaling configuration for the capacity provider.</p>
-    pub fn capacity_provider_scaling_config(mut self, input: crate::types::CapacityProviderScalingConfig) -> Self {
+    pub fn capacity_provider_scaling_config(mut self, input: super::super::super::types::CapacityProviderScalingConfig) -> Self {
         self.capacity_provider_scaling_config = ::std::option::Option::Some(input);
         self
     }
     /// <p>The updated scaling configuration for the capacity provider.</p>
-    pub fn set_capacity_provider_scaling_config(mut self, input: ::std::option::Option<crate::types::CapacityProviderScalingConfig>) -> Self {
+    pub fn set_capacity_provider_scaling_config(mut self, input: ::std::option::Option<super::super::super::types::CapacityProviderScalingConfig>) -> Self {
         self.capacity_provider_scaling_config = input;
         self
     }
     /// <p>The updated scaling configuration for the capacity provider.</p>
-    pub fn get_capacity_provider_scaling_config(&self) -> &::std::option::Option<crate::types::CapacityProviderScalingConfig> {
+    pub fn get_capacity_provider_scaling_config(&self) -> &::std::option::Option<super::super::super::types::CapacityProviderScalingConfig> {
         &self.capacity_provider_scaling_config
     }
     /// <p>Configuration for tag propagation to managed resources launched by the capacity provider.</p>
-    pub fn propagate_tags(mut self, input: crate::types::PropagateTags) -> Self {
+    pub fn propagate_tags(mut self, input: super::super::super::types::PropagateTags) -> Self {
         self.propagate_tags = ::std::option::Option::Some(input);
         self
     }
     /// <p>Configuration for tag propagation to managed resources launched by the capacity provider.</p>
-    pub fn set_propagate_tags(mut self, input: ::std::option::Option<crate::types::PropagateTags>) -> Self {
+    pub fn set_propagate_tags(mut self, input: ::std::option::Option<super::super::super::types::PropagateTags>) -> Self {
         self.propagate_tags = input;
         self
     }
     /// <p>Configuration for tag propagation to managed resources launched by the capacity provider.</p>
-    pub fn get_propagate_tags(&self) -> &::std::option::Option<crate::types::PropagateTags> {
+    pub fn get_propagate_tags(&self) -> &::std::option::Option<super::super::super::types::PropagateTags> {
         &self.propagate_tags
     }
     /// <p>The updated telemetry configuration for the capacity provider.</p>
-    pub fn telemetry_config(mut self, input: crate::types::CapacityProviderTelemetryConfig) -> Self {
+    pub fn telemetry_config(mut self, input: super::super::super::types::CapacityProviderTelemetryConfig) -> Self {
         self.telemetry_config = ::std::option::Option::Some(input);
         self
     }
     /// <p>The updated telemetry configuration for the capacity provider.</p>
-    pub fn set_telemetry_config(mut self, input: ::std::option::Option<crate::types::CapacityProviderTelemetryConfig>) -> Self {
+    pub fn set_telemetry_config(mut self, input: ::std::option::Option<super::super::super::types::CapacityProviderTelemetryConfig>) -> Self {
         self.telemetry_config = input;
         self
     }
     /// <p>The updated telemetry configuration for the capacity provider.</p>
-    pub fn get_telemetry_config(&self) -> &::std::option::Option<crate::types::CapacityProviderTelemetryConfig> {
+    pub fn get_telemetry_config(&self) -> &::std::option::Option<super::super::super::types::CapacityProviderTelemetryConfig> {
         &self.telemetry_config
     }
     /// Consumes the builder and constructs a [`UpdateCapacityProviderInput`](crate::operation::update_capacity_provider::UpdateCapacityProviderInput).
@@ -108,10 +108,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_capacity_provider::UpdateCapacityProviderInput,
+        super::super::super::operation::update_capacity_provider::UpdateCapacityProviderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_capacity_provider::UpdateCapacityProviderInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_capacity_provider::UpdateCapacityProviderInput {
             capacity_provider_name: self.capacity_provider_name,
             capacity_provider_scaling_config: self.capacity_provider_scaling_config,
             propagate_tags: self.propagate_tags,
