--- original
+++ modified
@@ -24,7 +24,7 @@
     }
 }
 impl ::std::error::Error for InternalServerException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::InternalServerException {
+impl ::aws_types::request_id::RequestId for super::types::error::InternalServerException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -37,8 +37,8 @@
 }
 impl InternalServerException {
     /// Creates a new builder-style object to manufacture [`InternalServerException`](crate::types::error::InternalServerException).
-    pub fn builder() -> crate::types::error::builders::InternalServerExceptionBuilder {
-        crate::types::error::builders::InternalServerExceptionBuilder::default()
+    pub fn builder() -> super::types::error::builders::InternalServerExceptionBuilder {
+        super::types::error::builders::InternalServerExceptionBuilder::default()
     }
 }

@@ -79,8 +79,8 @@
     /// Consumes the builder and constructs a [`InternalServerException`](crate::types::error::InternalServerException).
     /// This method will fail if any of the following fields are not set:
     /// - [`message`](crate::types::error::builders::InternalServerExceptionBuilder::message)
-    pub fn build(self) -> ::std::result::Result<crate::types::error::InternalServerException, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::error::InternalServerException {
+    pub fn build(self) -> ::std::result::Result<super::types::error::InternalServerException, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::error::InternalServerException {
             message: self.message.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "message",
