--- original
+++ modified
@@ -32,7 +32,7 @@
     }
 }
 impl ::std::error::Error for ThrottlingException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::ThrottlingException {
+impl ::aws_types::request_id::RequestId for super::types::error::ThrottlingException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -45,8 +45,8 @@
 }
 impl ThrottlingException {
     /// Creates a new builder-style object to manufacture [`ThrottlingException`](crate::types::error::ThrottlingException).
-    pub fn builder() -> crate::types::error::builders::ThrottlingExceptionBuilder {
-        crate::types::error::builders::ThrottlingExceptionBuilder::default()
+    pub fn builder() -> super::types::error::builders::ThrottlingExceptionBuilder {
+        super::types::error::builders::ThrottlingExceptionBuilder::default()
     }
 }

@@ -102,8 +102,8 @@
     /// Consumes the builder and constructs a [`ThrottlingException`](crate::types::error::ThrottlingException).
     /// This method will fail if any of the following fields are not set:
     /// - [`message`](crate::types::error::builders::ThrottlingExceptionBuilder::message)
-    pub fn build(self) -> ::std::result::Result<crate::types::error::ThrottlingException, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::error::ThrottlingException {
+    pub fn build(self) -> ::std::result::Result<super::types::error::ThrottlingException, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::error::ThrottlingException {
             message: self.message.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "message",
