--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On success, responds with [`CreateLoginProfileOutput`](crate::operation::create_login_profile::CreateLoginProfileOutput) with field(s):
     ///   - [`login_profile(Option<LoginProfile>)`](crate::operation::create_login_profile::CreateLoginProfileOutput::login_profile): <p>A structure containing the user name and password create date.</p>
     /// - On failure, responds with [`SdkError<CreateLoginProfileError>`](crate::operation::create_login_profile::CreateLoginProfileError)
-    pub fn create_login_profile(&self) -> crate::operation::create_login_profile::builders::CreateLoginProfileFluentBuilder {
-        crate::operation::create_login_profile::builders::CreateLoginProfileFluentBuilder::new(self.handle.clone())
+    pub fn create_login_profile(&self) -> super::operation::create_login_profile::builders::CreateLoginProfileFluentBuilder {
+        super::operation::create_login_profile::builders::CreateLoginProfileFluentBuilder::new(self.handle.clone())
     }
 }
