--- original
+++ modified
@@ -10,7 +10,7 @@
     /// - On success, responds with [`CreateUserOutput`](crate::operation::create_user::CreateUserOutput) with field(s):
     ///   - [`user(Option<User>)`](crate::operation::create_user::CreateUserOutput::user): <p>A structure with details about the new IAM user.</p>
     /// - On failure, responds with [`SdkError<CreateUserError>`](crate::operation::create_user::CreateUserError)
-    pub fn create_user(&self) -> crate::operation::create_user::builders::CreateUserFluentBuilder {
-        crate::operation::create_user::builders::CreateUserFluentBuilder::new(self.handle.clone())
+    pub fn create_user(&self) -> super::operation::create_user::builders::CreateUserFluentBuilder {
+        super::operation::create_user::builders::CreateUserFluentBuilder::new(self.handle.clone())
     }
 }
