--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On success, responds with [`CreatePolicyOutput`](crate::operation::create_policy::CreatePolicyOutput) with field(s):
     ///   - [`policy(Option<Policy>)`](crate::operation::create_policy::CreatePolicyOutput::policy): <p>A structure containing details about the new policy.</p>
     /// - On failure, responds with [`SdkError<CreatePolicyError>`](crate::operation::create_policy::CreatePolicyError)
-    pub fn create_policy(&self) -> crate::operation::create_policy::builders::CreatePolicyFluentBuilder {
-        crate::operation::create_policy::builders::CreatePolicyFluentBuilder::new(self.handle.clone())
+    pub fn create_policy(&self) -> super::super::operation::create_policy::builders::CreatePolicyFluentBuilder {
+        super::super::operation::create_policy::builders::CreatePolicyFluentBuilder::new(self.handle.clone())
     }
 }
