--- original
+++ modified
@@ -6,7 +6,7 @@
     ///   - [`saml_provider_arn(impl Into<String>)`](crate::operation::delete_saml_provider::builders::DeleteSAMLProviderFluentBuilder::saml_provider_arn) / [`set_saml_provider_arn(Option<String>)`](crate::operation::delete_saml_provider::builders::DeleteSAMLProviderFluentBuilder::set_saml_provider_arn):<br>required: **true**<br><p>The Amazon Resource Name (ARN) of the SAML provider to delete.</p><br>
     /// - On success, responds with [`DeleteSamlProviderOutput`](crate::operation::delete_saml_provider::DeleteSamlProviderOutput)
     /// - On failure, responds with [`SdkError<DeleteSAMLProviderError>`](crate::operation::delete_saml_provider::DeleteSAMLProviderError)
-    pub fn delete_saml_provider(&self) -> crate::operation::delete_saml_provider::builders::DeleteSAMLProviderFluentBuilder {
-        crate::operation::delete_saml_provider::builders::DeleteSAMLProviderFluentBuilder::new(self.handle.clone())
+    pub fn delete_saml_provider(&self) -> super::operation::delete_saml_provider::builders::DeleteSAMLProviderFluentBuilder {
+        super::operation::delete_saml_provider::builders::DeleteSAMLProviderFluentBuilder::new(self.handle.clone())
     }
 }
