--- original
+++ modified
@@ -10,7 +10,7 @@
     /// - On failure, responds with [`SdkError<GetContextKeysForPrincipalPolicyError>`](crate::operation::get_context_keys_for_principal_policy::GetContextKeysForPrincipalPolicyError)
     pub fn get_context_keys_for_principal_policy(
         &self,
-    ) -> crate::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyFluentBuilder {
-        crate::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyFluentBuilder {
+        super::operation::get_context_keys_for_principal_policy::builders::GetContextKeysForPrincipalPolicyFluentBuilder::new(self.handle.clone())
     }
 }
