--- original
+++ modified
@@ -7,7 +7,7 @@
     /// - On success, responds with [`GetRoleOutput`](crate::operation::get_role::GetRoleOutput) with field(s):
     ///   - [`role(Option<Role>)`](crate::operation::get_role::GetRoleOutput::role): <p>A structure containing details about the IAM role.</p>
     /// - On failure, responds with [`SdkError<GetRoleError>`](crate::operation::get_role::GetRoleError)
-    pub fn get_role(&self) -> crate::operation::get_role::builders::GetRoleFluentBuilder {
-        crate::operation::get_role::builders::GetRoleFluentBuilder::new(self.handle.clone())
+    pub fn get_role(&self) -> super::operation::get_role::builders::GetRoleFluentBuilder {
+        super::operation::get_role::builders::GetRoleFluentBuilder::new(self.handle.clone())
     }
 }
