--- original
+++ modified
@@ -8,7 +8,7 @@
     ///   - [`report_format(Option<ReportFormatType>)`](crate::operation::get_credential_report::GetCredentialReportOutput::report_format): <p>The format (MIME type) of the credential report.</p>
     ///   - [`generated_time(Option<DateTime>)`](crate::operation::get_credential_report::GetCredentialReportOutput::generated_time): <p>The date and time when the credential report was created, in <a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>.</p>
     /// - On failure, responds with [`SdkError<GetCredentialReportError>`](crate::operation::get_credential_report::GetCredentialReportError)
-    pub fn get_credential_report(&self) -> crate::operation::get_credential_report::builders::GetCredentialReportFluentBuilder {
-        crate::operation::get_credential_report::builders::GetCredentialReportFluentBuilder::new(self.handle.clone())
+    pub fn get_credential_report(&self) -> super::operation::get_credential_report::builders::GetCredentialReportFluentBuilder {
+        super::operation::get_credential_report::builders::GetCredentialReportFluentBuilder::new(self.handle.clone())
     }
 }
