--- original
+++ modified
@@ -12,7 +12,7 @@
     ///   - [`is_truncated(bool)`](crate::operation::list_virtual_mfa_devices::ListVirtualMfaDevicesOutput::is_truncated): <p>A flag that indicates whether there are more items to return. If your results were truncated, you can make a subsequent pagination request using the <code>Marker</code> request parameter to retrieve more items. Note that IAM might return fewer than the <code>MaxItems</code> number of results even when there are more results available. We recommend that you check <code>IsTruncated</code> after every call to ensure that you receive all your results.</p>
     ///   - [`marker(Option<String>)`](crate::operation::list_virtual_mfa_devices::ListVirtualMfaDevicesOutput::marker): <p>When <code>IsTruncated</code> is <code>true</code>, this element is present and contains the value to use for the <code>Marker</code> parameter in a subsequent pagination request.</p>
     /// - On failure, responds with [`SdkError<ListVirtualMFADevicesError>`](crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError)
-    pub fn list_virtual_mfa_devices(&self) -> crate::operation::list_virtual_mfa_devices::builders::ListVirtualMFADevicesFluentBuilder {
-        crate::operation::list_virtual_mfa_devices::builders::ListVirtualMFADevicesFluentBuilder::new(self.handle.clone())
+    pub fn list_virtual_mfa_devices(&self) -> super::super::operation::list_virtual_mfa_devices::builders::ListVirtualMFADevicesFluentBuilder {
+        super::super::operation::list_virtual_mfa_devices::builders::ListVirtualMFADevicesFluentBuilder::new(self.handle.clone())
     }
 }
