--- original
+++ modified
@@ -13,7 +13,7 @@
     /// - On failure, responds with [`SdkError<ListPoliciesGrantingServiceAccessError>`](crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError)
     pub fn list_policies_granting_service_access(
         &self,
-    ) -> crate::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessFluentBuilder {
-        crate::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessFluentBuilder {
+        super::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessFluentBuilder::new(self.handle.clone())
     }
 }
