--- original
+++ modified
@@ -11,7 +11,7 @@
     ///   - [`marker(Option<String>)`](crate::operation::list_delegation_requests::ListDelegationRequestsOutput::marker): <p>When <code>isTruncated</code> is <code>true</code>, this element is present and contains the value to use for the <code>Marker</code> parameter in a subsequent pagination request.</p>
     ///   - [`is_truncated(bool)`](crate::operation::list_delegation_requests::ListDelegationRequestsOutput::is_truncated): <p>A flag that indicates whether there are more items to return. If your results were truncated, you can make a subsequent pagination request using the <code>Marker</code> request parameter to retrieve more items.</p>
     /// - On failure, responds with [`SdkError<ListDelegationRequestsError>`](crate::operation::list_delegation_requests::ListDelegationRequestsError)
-    pub fn list_delegation_requests(&self) -> crate::operation::list_delegation_requests::builders::ListDelegationRequestsFluentBuilder {
-        crate::operation::list_delegation_requests::builders::ListDelegationRequestsFluentBuilder::new(self.handle.clone())
+    pub fn list_delegation_requests(&self) -> super::operation::list_delegation_requests::builders::ListDelegationRequestsFluentBuilder {
+        super::operation::list_delegation_requests::builders::ListDelegationRequestsFluentBuilder::new(self.handle.clone())
     }
 }
