--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<PutUserPermissionsBoundaryError>`](crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError)
     pub fn put_user_permissions_boundary(
         &self,
-    ) -> crate::operation::put_user_permissions_boundary::builders::PutUserPermissionsBoundaryFluentBuilder {
-        crate::operation::put_user_permissions_boundary::builders::PutUserPermissionsBoundaryFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::put_user_permissions_boundary::builders::PutUserPermissionsBoundaryFluentBuilder {
+        super::super::operation::put_user_permissions_boundary::builders::PutUserPermissionsBoundaryFluentBuilder::new(self.handle.clone())
     }
 }
