--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`tags(Tag)`](crate::operation::tag_server_certificate::builders::TagServerCertificateFluentBuilder::tags) / [`set_tags(Option<Vec::<Tag>>)`](crate::operation::tag_server_certificate::builders::TagServerCertificateFluentBuilder::set_tags):<br>required: **true**<br><p>The list of tags that you want to attach to the IAM server certificate. Each tag consists of a key name and an associated value.</p><br>
     /// - On success, responds with [`TagServerCertificateOutput`](crate::operation::tag_server_certificate::TagServerCertificateOutput)
     /// - On failure, responds with [`SdkError<TagServerCertificateError>`](crate::operation::tag_server_certificate::TagServerCertificateError)
-    pub fn tag_server_certificate(&self) -> crate::operation::tag_server_certificate::builders::TagServerCertificateFluentBuilder {
-        crate::operation::tag_server_certificate::builders::TagServerCertificateFluentBuilder::new(self.handle.clone())
+    pub fn tag_server_certificate(&self) -> super::operation::tag_server_certificate::builders::TagServerCertificateFluentBuilder {
+        super::operation::tag_server_certificate::builders::TagServerCertificateFluentBuilder::new(self.handle.clone())
     }
 }
