--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`tag_keys(impl Into<String>)`](crate::operation::untag_server_certificate::builders::UntagServerCertificateFluentBuilder::tag_keys) / [`set_tag_keys(Option<Vec::<String>>)`](crate::operation::untag_server_certificate::builders::UntagServerCertificateFluentBuilder::set_tag_keys):<br>required: **true**<br><p>A list of key names as a simple array of strings. The tags with matching keys are removed from the specified IAM server certificate.</p><br>
     /// - On success, responds with [`UntagServerCertificateOutput`](crate::operation::untag_server_certificate::UntagServerCertificateOutput)
     /// - On failure, responds with [`SdkError<UntagServerCertificateError>`](crate::operation::untag_server_certificate::UntagServerCertificateError)
-    pub fn untag_server_certificate(&self) -> crate::operation::untag_server_certificate::builders::UntagServerCertificateFluentBuilder {
-        crate::operation::untag_server_certificate::builders::UntagServerCertificateFluentBuilder::new(self.handle.clone())
+    pub fn untag_server_certificate(&self) -> super::super::operation::untag_server_certificate::builders::UntagServerCertificateFluentBuilder {
+        super::super::operation::untag_server_certificate::builders::UntagServerCertificateFluentBuilder::new(self.handle.clone())
     }
 }
