--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`tag_keys(impl Into<String>)`](crate::operation::untag_policy::builders::UntagPolicyFluentBuilder::tag_keys) / [`set_tag_keys(Option<Vec::<String>>)`](crate::operation::untag_policy::builders::UntagPolicyFluentBuilder::set_tag_keys):<br>required: **true**<br><p>A list of key names as a simple array of strings. The tags with matching keys are removed from the specified policy.</p><br>
     /// - On success, responds with [`UntagPolicyOutput`](crate::operation::untag_policy::UntagPolicyOutput)
     /// - On failure, responds with [`SdkError<UntagPolicyError>`](crate::operation::untag_policy::UntagPolicyError)
-    pub fn untag_policy(&self) -> crate::operation::untag_policy::builders::UntagPolicyFluentBuilder {
-        crate::operation::untag_policy::builders::UntagPolicyFluentBuilder::new(self.handle.clone())
+    pub fn untag_policy(&self) -> super::super::operation::untag_policy::builders::UntagPolicyFluentBuilder {
+        super::super::operation::untag_policy::builders::UntagPolicyFluentBuilder::new(self.handle.clone())
     }
 }
