--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`tag_keys(impl Into<String>)`](crate::operation::untag_user::builders::UntagUserFluentBuilder::tag_keys) / [`set_tag_keys(Option<Vec::<String>>)`](crate::operation::untag_user::builders::UntagUserFluentBuilder::set_tag_keys):<br>required: **true**<br><p>A list of key names as a simple array of strings. The tags with matching keys are removed from the specified user.</p><br>
     /// - On success, responds with [`UntagUserOutput`](crate::operation::untag_user::UntagUserOutput)
     /// - On failure, responds with [`SdkError<UntagUserError>`](crate::operation::untag_user::UntagUserError)
-    pub fn untag_user(&self) -> crate::operation::untag_user::builders::UntagUserFluentBuilder {
-        crate::operation::untag_user::builders::UntagUserFluentBuilder::new(self.handle.clone())
+    pub fn untag_user(&self) -> super::operation::untag_user::builders::UntagUserFluentBuilder {
+        super::operation::untag_user::builders::UntagUserFluentBuilder::new(self.handle.clone())
     }
 }
