--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`tag_keys(impl Into<String>)`](crate::operation::untag_role::builders::UntagRoleFluentBuilder::tag_keys) / [`set_tag_keys(Option<Vec::<String>>)`](crate::operation::untag_role::builders::UntagRoleFluentBuilder::set_tag_keys):<br>required: **true**<br><p>A list of key names as a simple array of strings. The tags with matching keys are removed from the specified role.</p><br>
     /// - On success, responds with [`UntagRoleOutput`](crate::operation::untag_role::UntagRoleOutput)
     /// - On failure, responds with [`SdkError<UntagRoleError>`](crate::operation::untag_role::UntagRoleError)
-    pub fn untag_role(&self) -> crate::operation::untag_role::builders::UntagRoleFluentBuilder {
-        crate::operation::untag_role::builders::UntagRoleFluentBuilder::new(self.handle.clone())
+    pub fn untag_role(&self) -> super::super::operation::untag_role::builders::UntagRoleFluentBuilder {
+        super::super::operation::untag_role::builders::UntagRoleFluentBuilder::new(self.handle.clone())
     }
 }
