--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`UploadSigningCertificateOutput`](crate::operation::upload_signing_certificate::UploadSigningCertificateOutput) with field(s):
     ///   - [`certificate(Option<SigningCertificate>)`](crate::operation::upload_signing_certificate::UploadSigningCertificateOutput::certificate): <p>Information about the certificate.</p>
     /// - On failure, responds with [`SdkError<UploadSigningCertificateError>`](crate::operation::upload_signing_certificate::UploadSigningCertificateError)
-    pub fn upload_signing_certificate(&self) -> crate::operation::upload_signing_certificate::builders::UploadSigningCertificateFluentBuilder {
-        crate::operation::upload_signing_certificate::builders::UploadSigningCertificateFluentBuilder::new(self.handle.clone())
+    pub fn upload_signing_certificate(&self) -> super::operation::upload_signing_certificate::builders::UploadSigningCertificateFluentBuilder {
+        super::operation::upload_signing_certificate::builders::UploadSigningCertificateFluentBuilder::new(self.handle.clone())
     }
 }
