--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct EnableOrganizationsRootSessionsInput {}
 impl EnableOrganizationsRootSessionsInput {
     /// Creates a new builder-style object to manufacture [`EnableOrganizationsRootSessionsInput`](crate::operation::enable_organizations_root_sessions::EnableOrganizationsRootSessionsInput).
-    pub fn builder() -> crate::operation::enable_organizations_root_sessions::builders::EnableOrganizationsRootSessionsInputBuilder {
-        crate::operation::enable_organizations_root_sessions::builders::EnableOrganizationsRootSessionsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::enable_organizations_root_sessions::builders::EnableOrganizationsRootSessionsInputBuilder {
+        super::super::super::operation::enable_organizations_root_sessions::builders::EnableOrganizationsRootSessionsInputBuilder::default()
     }
 }

@@ -19,9 +19,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::enable_organizations_root_sessions::EnableOrganizationsRootSessionsInput,
+        super::super::super::operation::enable_organizations_root_sessions::EnableOrganizationsRootSessionsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::enable_organizations_root_sessions::EnableOrganizationsRootSessionsInput {})
+        ::std::result::Result::Ok(super::super::super::operation::enable_organizations_root_sessions::EnableOrganizationsRootSessionsInput {})
     }
 }
