--- original
+++ modified
@@ -39,12 +39,12 @@
     /// <p>You must specify both the <code>AccessKeyId</code> and <code>SecretAccessKey</code> value in the authentication credential, even if you are only updating one value.</p>
     /// <p>This parameter doesn't establish or change your authentication credentials on the proxy. It just tells KMS the credential that you established with your external key store proxy. For example, if you rotate the credential on your external key store proxy, you can use this parameter to update the credential in KMS.</p>
     /// <p>You can change this value when the external key store is connected or disconnected.</p>
-    pub xks_proxy_authentication_credential: ::std::option::Option<crate::types::XksProxyAuthenticationCredentialType>,
+    pub xks_proxy_authentication_credential: ::std::option::Option<super::super::super::types::XksProxyAuthenticationCredentialType>,
     /// <p>Changes the connectivity setting for the external key store. To indicate that the external key store proxy uses a Amazon VPC endpoint service to communicate with KMS, specify <code>VPC_ENDPOINT_SERVICE</code>. Otherwise, specify <code>PUBLIC_ENDPOINT</code>.</p>
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>VPC_ENDPOINT_SERVICE</code>, you must also change the <code>XksProxyUriEndpoint</code> and add an <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>PUBLIC_ENDPOINT</code>, you must also change the <code>XksProxyUriEndpoint</code> and specify a null or empty string for the <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>To change this value, the external key store must be disconnected.</p>
-    pub xks_proxy_connectivity: ::std::option::Option<crate::types::XksProxyConnectivityType>,
+    pub xks_proxy_connectivity: ::std::option::Option<super::super::super::types::XksProxyConnectivityType>,
 }
 impl UpdateCustomKeyStoreInput {
     /// <p>Identifies the custom key store that you want to update. Enter the ID of the custom key store. To find the ID of a custom key store, use the <code>DescribeCustomKeyStores</code> operation.</p>
@@ -99,7 +99,7 @@
     /// <p>You must specify both the <code>AccessKeyId</code> and <code>SecretAccessKey</code> value in the authentication credential, even if you are only updating one value.</p>
     /// <p>This parameter doesn't establish or change your authentication credentials on the proxy. It just tells KMS the credential that you established with your external key store proxy. For example, if you rotate the credential on your external key store proxy, you can use this parameter to update the credential in KMS.</p>
     /// <p>You can change this value when the external key store is connected or disconnected.</p>
-    pub fn xks_proxy_authentication_credential(&self) -> ::std::option::Option<&crate::types::XksProxyAuthenticationCredentialType> {
+    pub fn xks_proxy_authentication_credential(&self) -> ::std::option::Option<&super::super::super::types::XksProxyAuthenticationCredentialType> {
         self.xks_proxy_authentication_credential.as_ref()
     }
     /// <p>Changes the connectivity setting for the external key store. To indicate that the external key store proxy uses a Amazon VPC endpoint service to communicate with KMS, specify <code>VPC_ENDPOINT_SERVICE</code>. Otherwise, specify <code>PUBLIC_ENDPOINT</code>.</p>
@@ -106,7 +106,7 @@
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>VPC_ENDPOINT_SERVICE</code>, you must also change the <code>XksProxyUriEndpoint</code> and add an <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>PUBLIC_ENDPOINT</code>, you must also change the <code>XksProxyUriEndpoint</code> and specify a null or empty string for the <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>To change this value, the external key store must be disconnected.</p>
-    pub fn xks_proxy_connectivity(&self) -> ::std::option::Option<&crate::types::XksProxyConnectivityType> {
+    pub fn xks_proxy_connectivity(&self) -> ::std::option::Option<&super::super::super::types::XksProxyConnectivityType> {
         self.xks_proxy_connectivity.as_ref()
     }
 }
@@ -128,8 +128,8 @@
 }
 impl UpdateCustomKeyStoreInput {
     /// Creates a new builder-style object to manufacture [`UpdateCustomKeyStoreInput`](crate::operation::update_custom_key_store::UpdateCustomKeyStoreInput).
-    pub fn builder() -> crate::operation::update_custom_key_store::builders::UpdateCustomKeyStoreInputBuilder {
-        crate::operation::update_custom_key_store::builders::UpdateCustomKeyStoreInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_custom_key_store::builders::UpdateCustomKeyStoreInputBuilder {
+        super::super::super::operation::update_custom_key_store::builders::UpdateCustomKeyStoreInputBuilder::default()
     }
 }

@@ -145,8 +145,8 @@
     pub(crate) xks_proxy_uri_path: ::std::option::Option<::std::string::String>,
     pub(crate) xks_proxy_vpc_endpoint_service_name: ::std::option::Option<::std::string::String>,
     pub(crate) xks_proxy_vpc_endpoint_service_owner: ::std::option::Option<::std::string::String>,
-    pub(crate) xks_proxy_authentication_credential: ::std::option::Option<crate::types::XksProxyAuthenticationCredentialType>,
-    pub(crate) xks_proxy_connectivity: ::std::option::Option<crate::types::XksProxyConnectivityType>,
+    pub(crate) xks_proxy_authentication_credential: ::std::option::Option<super::super::super::types::XksProxyAuthenticationCredentialType>,
+    pub(crate) xks_proxy_connectivity: ::std::option::Option<super::super::super::types::XksProxyConnectivityType>,
 }
 impl UpdateCustomKeyStoreInputBuilder {
     /// <p>Identifies the custom key store that you want to update. Enter the ID of the custom key store. To find the ID of a custom key store, use the <code>DescribeCustomKeyStores</code> operation.</p>
@@ -314,7 +314,7 @@
     /// <p>You must specify both the <code>AccessKeyId</code> and <code>SecretAccessKey</code> value in the authentication credential, even if you are only updating one value.</p>
     /// <p>This parameter doesn't establish or change your authentication credentials on the proxy. It just tells KMS the credential that you established with your external key store proxy. For example, if you rotate the credential on your external key store proxy, you can use this parameter to update the credential in KMS.</p>
     /// <p>You can change this value when the external key store is connected or disconnected.</p>
-    pub fn xks_proxy_authentication_credential(mut self, input: crate::types::XksProxyAuthenticationCredentialType) -> Self {
+    pub fn xks_proxy_authentication_credential(mut self, input: super::super::super::types::XksProxyAuthenticationCredentialType) -> Self {
         self.xks_proxy_authentication_credential = ::std::option::Option::Some(input);
         self
     }
@@ -324,7 +324,7 @@
     /// <p>You can change this value when the external key store is connected or disconnected.</p>
     pub fn set_xks_proxy_authentication_credential(
         mut self,
-        input: ::std::option::Option<crate::types::XksProxyAuthenticationCredentialType>,
+        input: ::std::option::Option<super::super::super::types::XksProxyAuthenticationCredentialType>,
     ) -> Self {
         self.xks_proxy_authentication_credential = input;
         self
@@ -333,7 +333,7 @@
     /// <p>You must specify both the <code>AccessKeyId</code> and <code>SecretAccessKey</code> value in the authentication credential, even if you are only updating one value.</p>
     /// <p>This parameter doesn't establish or change your authentication credentials on the proxy. It just tells KMS the credential that you established with your external key store proxy. For example, if you rotate the credential on your external key store proxy, you can use this parameter to update the credential in KMS.</p>
     /// <p>You can change this value when the external key store is connected or disconnected.</p>
-    pub fn get_xks_proxy_authentication_credential(&self) -> &::std::option::Option<crate::types::XksProxyAuthenticationCredentialType> {
+    pub fn get_xks_proxy_authentication_credential(&self) -> &::std::option::Option<super::super::super::types::XksProxyAuthenticationCredentialType> {
         &self.xks_proxy_authentication_credential
     }
     /// <p>Changes the connectivity setting for the external key store. To indicate that the external key store proxy uses a Amazon VPC endpoint service to communicate with KMS, specify <code>VPC_ENDPOINT_SERVICE</code>. Otherwise, specify <code>PUBLIC_ENDPOINT</code>.</p>
@@ -340,7 +340,7 @@
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>VPC_ENDPOINT_SERVICE</code>, you must also change the <code>XksProxyUriEndpoint</code> and add an <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>PUBLIC_ENDPOINT</code>, you must also change the <code>XksProxyUriEndpoint</code> and specify a null or empty string for the <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>To change this value, the external key store must be disconnected.</p>
-    pub fn xks_proxy_connectivity(mut self, input: crate::types::XksProxyConnectivityType) -> Self {
+    pub fn xks_proxy_connectivity(mut self, input: super::super::super::types::XksProxyConnectivityType) -> Self {
         self.xks_proxy_connectivity = ::std::option::Option::Some(input);
         self
     }
@@ -348,7 +348,7 @@
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>VPC_ENDPOINT_SERVICE</code>, you must also change the <code>XksProxyUriEndpoint</code> and add an <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>PUBLIC_ENDPOINT</code>, you must also change the <code>XksProxyUriEndpoint</code> and specify a null or empty string for the <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>To change this value, the external key store must be disconnected.</p>
-    pub fn set_xks_proxy_connectivity(mut self, input: ::std::option::Option<crate::types::XksProxyConnectivityType>) -> Self {
+    pub fn set_xks_proxy_connectivity(mut self, input: ::std::option::Option<super::super::super::types::XksProxyConnectivityType>) -> Self {
         self.xks_proxy_connectivity = input;
         self
     }
@@ -356,15 +356,15 @@
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>VPC_ENDPOINT_SERVICE</code>, you must also change the <code>XksProxyUriEndpoint</code> and add an <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>If you change the <code>XksProxyConnectivity</code> to <code>PUBLIC_ENDPOINT</code>, you must also change the <code>XksProxyUriEndpoint</code> and specify a null or empty string for the <code>XksProxyVpcEndpointServiceName</code> value.</p>
     /// <p>To change this value, the external key store must be disconnected.</p>
-    pub fn get_xks_proxy_connectivity(&self) -> &::std::option::Option<crate::types::XksProxyConnectivityType> {
+    pub fn get_xks_proxy_connectivity(&self) -> &::std::option::Option<super::super::super::types::XksProxyConnectivityType> {
         &self.xks_proxy_connectivity
     }
     /// Consumes the builder and constructs a [`UpdateCustomKeyStoreInput`](crate::operation::update_custom_key_store::UpdateCustomKeyStoreInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_custom_key_store::UpdateCustomKeyStoreInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::update_custom_key_store::UpdateCustomKeyStoreInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_custom_key_store::UpdateCustomKeyStoreInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_custom_key_store::UpdateCustomKeyStoreInput {
             custom_key_store_id: self.custom_key_store_id,
             new_custom_key_store_name: self.new_custom_key_store_name,
             key_store_password: self.key_store_password,
