--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl DeleteRepositoryPermissionsPolicyInput {
     /// Creates a new builder-style object to manufacture [`DeleteRepositoryPermissionsPolicyInput`](crate::operation::delete_repository_permissions_policy::DeleteRepositoryPermissionsPolicyInput).
-    pub fn builder() -> crate::operation::delete_repository_permissions_policy::builders::DeleteRepositoryPermissionsPolicyInputBuilder {
-        crate::operation::delete_repository_permissions_policy::builders::DeleteRepositoryPermissionsPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_repository_permissions_policy::builders::DeleteRepositoryPermissionsPolicyInputBuilder {
+        super::super::super::operation::delete_repository_permissions_policy::builders::DeleteRepositoryPermissionsPolicyInputBuilder::default()
     }
 }

@@ -109,11 +109,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_repository_permissions_policy::DeleteRepositoryPermissionsPolicyInput,
+        super::super::super::operation::delete_repository_permissions_policy::DeleteRepositoryPermissionsPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::delete_repository_permissions_policy::DeleteRepositoryPermissionsPolicyInput {
+            super::super::super::operation::delete_repository_permissions_policy::DeleteRepositoryPermissionsPolicyInput {
                 domain: self.domain,
                 domain_owner: self.domain_owner,
                 repository: self.repository,
