--- original
+++ modified
@@ -60,8 +60,8 @@
 }
 impl AdminResetUserPasswordInput {
     /// Creates a new builder-style object to manufacture [`AdminResetUserPasswordInput`](crate::operation::admin_reset_user_password::AdminResetUserPasswordInput).
-    pub fn builder() -> crate::operation::admin_reset_user_password::builders::AdminResetUserPasswordInputBuilder {
-        crate::operation::admin_reset_user_password::builders::AdminResetUserPasswordInputBuilder::default()
+    pub fn builder() -> super::operation::admin_reset_user_password::builders::AdminResetUserPasswordInputBuilder {
+        super::operation::admin_reset_user_password::builders::AdminResetUserPasswordInputBuilder::default()
     }
 }

@@ -171,10 +171,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::admin_reset_user_password::AdminResetUserPasswordInput,
+        super::operation::admin_reset_user_password::AdminResetUserPasswordInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::admin_reset_user_password::AdminResetUserPasswordInput {
+        ::std::result::Result::Ok(super::operation::admin_reset_user_password::AdminResetUserPasswordInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
             client_metadata: self.client_metadata,
