--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl AdminEnableUserInput {
     /// Creates a new builder-style object to manufacture [`AdminEnableUserInput`](crate::operation::admin_enable_user::AdminEnableUserInput).
-    pub fn builder() -> crate::operation::admin_enable_user::builders::AdminEnableUserInputBuilder {
-        crate::operation::admin_enable_user::builders::AdminEnableUserInputBuilder::default()
+    pub fn builder() -> super::operation::admin_enable_user::builders::AdminEnableUserInputBuilder {
+        super::operation::admin_enable_user::builders::AdminEnableUserInputBuilder::default()
     }
 }

@@ -75,8 +75,8 @@
     /// Consumes the builder and constructs a [`AdminEnableUserInput`](crate::operation::admin_enable_user::AdminEnableUserInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::admin_enable_user::AdminEnableUserInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::admin_enable_user::AdminEnableUserInput {
+    ) -> ::std::result::Result<super::operation::admin_enable_user::AdminEnableUserInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::admin_enable_user::AdminEnableUserInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
         })
