--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteIdentityProviderInput {
     /// Creates a new builder-style object to manufacture [`DeleteIdentityProviderInput`](crate::operation::delete_identity_provider::DeleteIdentityProviderInput).
-    pub fn builder() -> crate::operation::delete_identity_provider::builders::DeleteIdentityProviderInputBuilder {
-        crate::operation::delete_identity_provider::builders::DeleteIdentityProviderInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_identity_provider::builders::DeleteIdentityProviderInputBuilder {
+        super::super::super::operation::delete_identity_provider::builders::DeleteIdentityProviderInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_identity_provider::DeleteIdentityProviderInput,
+        super::super::super::operation::delete_identity_provider::DeleteIdentityProviderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_identity_provider::DeleteIdentityProviderInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_identity_provider::DeleteIdentityProviderInput {
             user_pool_id: self.user_pool_id,
             provider_name: self.provider_name,
         })
