--- original
+++ modified
@@ -8,9 +8,9 @@
     /// <p>The <code>Content-MD5</code> header for the inventory table configuration.</p>
     pub content_md5: ::std::option::Option<::std::string::String>,
     /// <p>The checksum algorithm to use with your inventory table configuration.</p>
-    pub checksum_algorithm: ::std::option::Option<crate::types::ChecksumAlgorithm>,
+    pub checksum_algorithm: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>,
     /// <p>The contents of your inventory table configuration.</p>
-    pub inventory_table_configuration: ::std::option::Option<crate::types::InventoryTableConfigurationUpdates>,
+    pub inventory_table_configuration: ::std::option::Option<super::super::super::types::InventoryTableConfigurationUpdates>,
     /// <p>The expected owner of the general purpose bucket that corresponds to the metadata table configuration that you want to enable or disable an inventory table for.</p>
     pub expected_bucket_owner: ::std::option::Option<::std::string::String>,
 }
@@ -24,11 +24,11 @@
         self.content_md5.as_deref()
     }
     /// <p>The checksum algorithm to use with your inventory table configuration.</p>
-    pub fn checksum_algorithm(&self) -> ::std::option::Option<&crate::types::ChecksumAlgorithm> {
+    pub fn checksum_algorithm(&self) -> ::std::option::Option<&super::super::super::types::ChecksumAlgorithm> {
         self.checksum_algorithm.as_ref()
     }
     /// <p>The contents of your inventory table configuration.</p>
-    pub fn inventory_table_configuration(&self) -> ::std::option::Option<&crate::types::InventoryTableConfigurationUpdates> {
+    pub fn inventory_table_configuration(&self) -> ::std::option::Option<&super::super::super::types::InventoryTableConfigurationUpdates> {
         self.inventory_table_configuration.as_ref()
     }
     /// <p>The expected owner of the general purpose bucket that corresponds to the metadata table configuration that you want to enable or disable an inventory table for.</p>
@@ -39,9 +39,9 @@
 impl UpdateBucketMetadataInventoryTableConfigurationInput {
     /// Creates a new builder-style object to manufacture [`UpdateBucketMetadataInventoryTableConfigurationInput`](crate::operation::update_bucket_metadata_inventory_table_configuration::UpdateBucketMetadataInventoryTableConfigurationInput).
     pub fn builder(
-    ) -> crate::operation::update_bucket_metadata_inventory_table_configuration::builders::UpdateBucketMetadataInventoryTableConfigurationInputBuilder
+    ) -> super::super::super::operation::update_bucket_metadata_inventory_table_configuration::builders::UpdateBucketMetadataInventoryTableConfigurationInputBuilder
     {
-        crate::operation::update_bucket_metadata_inventory_table_configuration::builders::UpdateBucketMetadataInventoryTableConfigurationInputBuilder::default()
+        super::super::super::operation::update_bucket_metadata_inventory_table_configuration::builders::UpdateBucketMetadataInventoryTableConfigurationInputBuilder::default()
     }
 }

@@ -51,8 +51,8 @@
 pub struct UpdateBucketMetadataInventoryTableConfigurationInputBuilder {
     pub(crate) bucket: ::std::option::Option<::std::string::String>,
     pub(crate) content_md5: ::std::option::Option<::std::string::String>,
-    pub(crate) checksum_algorithm: ::std::option::Option<crate::types::ChecksumAlgorithm>,
-    pub(crate) inventory_table_configuration: ::std::option::Option<crate::types::InventoryTableConfigurationUpdates>,
+    pub(crate) checksum_algorithm: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>,
+    pub(crate) inventory_table_configuration: ::std::option::Option<super::super::super::types::InventoryTableConfigurationUpdates>,
     pub(crate) expected_bucket_owner: ::std::option::Option<::std::string::String>,
 }
 impl UpdateBucketMetadataInventoryTableConfigurationInputBuilder {
@@ -86,32 +86,32 @@
         &self.content_md5
     }
     /// <p>The checksum algorithm to use with your inventory table configuration.</p>
-    pub fn checksum_algorithm(mut self, input: crate::types::ChecksumAlgorithm) -> Self {
+    pub fn checksum_algorithm(mut self, input: super::super::super::types::ChecksumAlgorithm) -> Self {
         self.checksum_algorithm = ::std::option::Option::Some(input);
         self
     }
     /// <p>The checksum algorithm to use with your inventory table configuration.</p>
-    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<crate::types::ChecksumAlgorithm>) -> Self {
+    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<super::super::super::types::ChecksumAlgorithm>) -> Self {
         self.checksum_algorithm = input;
         self
     }
     /// <p>The checksum algorithm to use with your inventory table configuration.</p>
-    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<crate::types::ChecksumAlgorithm> {
+    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<super::super::super::types::ChecksumAlgorithm> {
         &self.checksum_algorithm
     }
     /// <p>The contents of your inventory table configuration.</p>
     /// This field is required.
-    pub fn inventory_table_configuration(mut self, input: crate::types::InventoryTableConfigurationUpdates) -> Self {
+    pub fn inventory_table_configuration(mut self, input: super::super::super::types::InventoryTableConfigurationUpdates) -> Self {
         self.inventory_table_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>The contents of your inventory table configuration.</p>
-    pub fn set_inventory_table_configuration(mut self, input: ::std::option::Option<crate::types::InventoryTableConfigurationUpdates>) -> Self {
+    pub fn set_inventory_table_configuration(mut self, input: ::std::option::Option<super::super::super::types::InventoryTableConfigurationUpdates>) -> Self {
         self.inventory_table_configuration = input;
         self
     }
     /// <p>The contents of your inventory table configuration.</p>
-    pub fn get_inventory_table_configuration(&self) -> &::std::option::Option<crate::types::InventoryTableConfigurationUpdates> {
+    pub fn get_inventory_table_configuration(&self) -> &::std::option::Option<super::super::super::types::InventoryTableConfigurationUpdates> {
         &self.inventory_table_configuration
     }
     /// <p>The expected owner of the general purpose bucket that corresponds to the metadata table configuration that you want to enable or disable an inventory table for.</p>
@@ -132,11 +132,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_bucket_metadata_inventory_table_configuration::UpdateBucketMetadataInventoryTableConfigurationInput,
+        super::super::super::operation::update_bucket_metadata_inventory_table_configuration::UpdateBucketMetadataInventoryTableConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::update_bucket_metadata_inventory_table_configuration::UpdateBucketMetadataInventoryTableConfigurationInput {
+            super::super::super::operation::update_bucket_metadata_inventory_table_configuration::UpdateBucketMetadataInventoryTableConfigurationInput {
                 bucket: self.bucket,
                 content_md5: self.content_md5,
                 checksum_algorithm: self.checksum_algorithm,
