--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl CreateLogStreamInput {
     /// Creates a new builder-style object to manufacture [`CreateLogStreamInput`](crate::operation::create_log_stream::CreateLogStreamInput).
-    pub fn builder() -> crate::operation::create_log_stream::builders::CreateLogStreamInputBuilder {
-        crate::operation::create_log_stream::builders::CreateLogStreamInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::create_log_stream::builders::CreateLogStreamInputBuilder {
+        super::super::super::operation::create_log_stream::builders::CreateLogStreamInputBuilder::default()
     }
 }

@@ -66,8 +66,8 @@
     /// Consumes the builder and constructs a [`CreateLogStreamInput`](crate::operation::create_log_stream::CreateLogStreamInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_log_stream::CreateLogStreamInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_log_stream::CreateLogStreamInput {
+    ) -> ::std::result::Result<super::super::super::operation::create_log_stream::CreateLogStreamInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::create_log_stream::CreateLogStreamInput {
             log_group_name: self.log_group_name,
             log_stream_name: self.log_stream_name,
         })
