--- original
+++ modified
@@ -16,7 +16,7 @@
     /// <p>The length constraint applies only to the full ARN. If you specify only the function name, it is limited to 64 characters in length.</p>
     pub function_name: ::std::option::Option<::std::string::String>,
     /// <p>Set to <code>Tail</code> to include the execution log in the response. Applies to synchronously invoked functions only.</p>
-    pub log_type: ::std::option::Option<crate::types::LogType>,
+    pub log_type: ::std::option::Option<super::types::LogType>,
     /// <p>Up to 3,583 bytes of base64-encoded data about the invoking client to pass to the function in the context object.</p>
     pub client_context: ::std::option::Option<::std::string::String>,
     /// <p>The alias name.</p>
@@ -33,7 +33,7 @@
     /// <li>
     /// <p><code>DryRun</code> – Validate parameter values and verify that the IAM user or role has permission to invoke the function.</p></li>
     /// </ul>
-    pub invocation_type: ::std::option::Option<crate::types::ResponseStreamingInvocationType>,
+    pub invocation_type: ::std::option::Option<super::types::ResponseStreamingInvocationType>,
 }
 impl InvokeWithResponseStreamInput {
     /// <p>The name or ARN of the Lambda function.</p>
@@ -51,7 +51,7 @@
         self.function_name.as_deref()
     }
     /// <p>Set to <code>Tail</code> to include the execution log in the response. Applies to synchronously invoked functions only.</p>
-    pub fn log_type(&self) -> ::std::option::Option<&crate::types::LogType> {
+    pub fn log_type(&self) -> ::std::option::Option<&super::types::LogType> {
         self.log_type.as_ref()
     }
     /// <p>Up to 3,583 bytes of base64-encoded data about the invoking client to pass to the function in the context object.</p>
@@ -78,7 +78,7 @@
     /// <li>
     /// <p><code>DryRun</code> – Validate parameter values and verify that the IAM user or role has permission to invoke the function.</p></li>
     /// </ul>
-    pub fn invocation_type(&self) -> ::std::option::Option<&crate::types::ResponseStreamingInvocationType> {
+    pub fn invocation_type(&self) -> ::std::option::Option<&super::types::ResponseStreamingInvocationType> {
         self.invocation_type.as_ref()
     }
 }
@@ -97,8 +97,8 @@
 }
 impl InvokeWithResponseStreamInput {
     /// Creates a new builder-style object to manufacture [`InvokeWithResponseStreamInput`](crate::operation::invoke_with_response_stream::InvokeWithResponseStreamInput).
-    pub fn builder() -> crate::operation::invoke_with_response_stream::builders::InvokeWithResponseStreamInputBuilder {
-        crate::operation::invoke_with_response_stream::builders::InvokeWithResponseStreamInputBuilder::default()
+    pub fn builder() -> super::operation::invoke_with_response_stream::builders::InvokeWithResponseStreamInputBuilder {
+        super::operation::invoke_with_response_stream::builders::InvokeWithResponseStreamInputBuilder::default()
     }
 }

@@ -107,12 +107,12 @@
 #[non_exhaustive]
 pub struct InvokeWithResponseStreamInputBuilder {
     pub(crate) function_name: ::std::option::Option<::std::string::String>,
-    pub(crate) log_type: ::std::option::Option<crate::types::LogType>,
+    pub(crate) log_type: ::std::option::Option<super::types::LogType>,
     pub(crate) client_context: ::std::option::Option<::std::string::String>,
     pub(crate) qualifier: ::std::option::Option<::std::string::String>,
     pub(crate) payload: ::std::option::Option<::aws_smithy_types::Blob>,
     pub(crate) tenant_id: ::std::option::Option<::std::string::String>,
-    pub(crate) invocation_type: ::std::option::Option<crate::types::ResponseStreamingInvocationType>,
+    pub(crate) invocation_type: ::std::option::Option<super::types::ResponseStreamingInvocationType>,
 }
 impl InvokeWithResponseStreamInputBuilder {
     /// <p>The name or ARN of the Lambda function.</p>
@@ -161,17 +161,17 @@
         &self.function_name
     }
     /// <p>Set to <code>Tail</code> to include the execution log in the response. Applies to synchronously invoked functions only.</p>
-    pub fn log_type(mut self, input: crate::types::LogType) -> Self {
+    pub fn log_type(mut self, input: super::types::LogType) -> Self {
         self.log_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>Set to <code>Tail</code> to include the execution log in the response. Applies to synchronously invoked functions only.</p>
-    pub fn set_log_type(mut self, input: ::std::option::Option<crate::types::LogType>) -> Self {
+    pub fn set_log_type(mut self, input: ::std::option::Option<super::types::LogType>) -> Self {
         self.log_type = input;
         self
     }
     /// <p>Set to <code>Tail</code> to include the execution log in the response. Applies to synchronously invoked functions only.</p>
-    pub fn get_log_type(&self) -> &::std::option::Option<crate::types::LogType> {
+    pub fn get_log_type(&self) -> &::std::option::Option<super::types::LogType> {
         &self.log_type
     }
     /// <p>Up to 3,583 bytes of base64-encoded data about the invoking client to pass to the function in the context object.</p>
@@ -240,7 +240,7 @@
     /// <li>
     /// <p><code>DryRun</code> – Validate parameter values and verify that the IAM user or role has permission to invoke the function.</p></li>
     /// </ul>
-    pub fn invocation_type(mut self, input: crate::types::ResponseStreamingInvocationType) -> Self {
+    pub fn invocation_type(mut self, input: super::types::ResponseStreamingInvocationType) -> Self {
         self.invocation_type = ::std::option::Option::Some(input);
         self
     }
@@ -251,7 +251,7 @@
     /// <li>
     /// <p><code>DryRun</code> – Validate parameter values and verify that the IAM user or role has permission to invoke the function.</p></li>
     /// </ul>
-    pub fn set_invocation_type(mut self, input: ::std::option::Option<crate::types::ResponseStreamingInvocationType>) -> Self {
+    pub fn set_invocation_type(mut self, input: ::std::option::Option<super::types::ResponseStreamingInvocationType>) -> Self {
         self.invocation_type = input;
         self
     }
@@ -262,7 +262,7 @@
     /// <li>
     /// <p><code>DryRun</code> – Validate parameter values and verify that the IAM user or role has permission to invoke the function.</p></li>
     /// </ul>
-    pub fn get_invocation_type(&self) -> &::std::option::Option<crate::types::ResponseStreamingInvocationType> {
+    pub fn get_invocation_type(&self) -> &::std::option::Option<super::types::ResponseStreamingInvocationType> {
         &self.invocation_type
     }
     /// Consumes the builder and constructs a [`InvokeWithResponseStreamInput`](crate::operation::invoke_with_response_stream::InvokeWithResponseStreamInput).
@@ -269,10 +269,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::invoke_with_response_stream::InvokeWithResponseStreamInput,
+        super::operation::invoke_with_response_stream::InvokeWithResponseStreamInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamInput {
+        ::std::result::Result::Ok(super::operation::invoke_with_response_stream::InvokeWithResponseStreamInput {
             function_name: self.function_name,
             log_type: self.log_type,
             client_context: self.client_context,
