--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl DescribePackageGroupInput {
     /// Creates a new builder-style object to manufacture [`DescribePackageGroupInput`](crate::operation::describe_package_group::DescribePackageGroupInput).
-    pub fn builder() -> crate::operation::describe_package_group::builders::DescribePackageGroupInputBuilder {
-        crate::operation::describe_package_group::builders::DescribePackageGroupInputBuilder::default()
+    pub fn builder() -> super::operation::describe_package_group::builders::DescribePackageGroupInputBuilder {
+        super::operation::describe_package_group::builders::DescribePackageGroupInputBuilder::default()
     }
 }

@@ -87,9 +87,9 @@
     /// Consumes the builder and constructs a [`DescribePackageGroupInput`](crate::operation::describe_package_group::DescribePackageGroupInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_package_group::DescribePackageGroupInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::describe_package_group::DescribePackageGroupInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::describe_package_group::DescribePackageGroupInput {
+        ::std::result::Result::Ok(super::operation::describe_package_group::DescribePackageGroupInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             package_group: self.package_group,
