--- original
+++ modified
@@ -9,8 +9,8 @@
     }
 }
 impl ::aws_smithy_eventstream::frame::UnmarshallMessage for InvokeWithResponseStreamResponseEventUnmarshaller {
-    type Output = crate::types::InvokeWithResponseStreamResponseEvent;
-    type Error = crate::types::error::InvokeWithResponseStreamResponseEventError;
+    type Output = super::types::InvokeWithResponseStreamResponseEvent;
+    type Error = super::types::error::InvokeWithResponseStreamResponseEventError;
     fn unmarshall(
         &self,
         message: &::aws_smithy_types::event_stream::Message,
@@ -20,7 +20,7 @@
         match response_headers.message_type.as_str() {
             "event" => match response_headers.smithy_type.as_str() {
                 "PayloadChunk" => {
-                    let mut builder = crate::types::builders::InvokeResponseStreamUpdateBuilder::default();
+                    let mut builder = super::types::builders::InvokeResponseStreamUpdateBuilder::default();
                     let content_type = response_headers.content_type().unwrap_or_default();
                     if content_type != "application/octet-stream" {
                         return Err(::aws_smithy_eventstream::error::Error::unmarshalling(format!(
@@ -29,35 +29,35 @@
                     }
                     builder = builder.set_payload(Some(::aws_smithy_types::Blob::from_maybe_shared(message.payload().clone())));
                     Ok(::aws_smithy_eventstream::frame::UnmarshalledMessage::Event(
-                        crate::types::InvokeWithResponseStreamResponseEvent::PayloadChunk(builder.build()),
+                        super::types::InvokeWithResponseStreamResponseEvent::PayloadChunk(builder.build()),
                     ))
                 }
                 "InvokeComplete" => {
                     let parsed =
-                            crate::protocol_serde::shape_invoke_with_response_stream_complete_event::de_invoke_with_response_stream_complete_event_payload(&message.payload()[..])
+                            super::protocol_serde::shape_invoke_with_response_stream_complete_event::de_invoke_with_response_stream_complete_event_payload(&message.payload()[..])
                                                 .map_err(|err| {
                                                     ::aws_smithy_eventstream::error::Error::unmarshalling(format!("failed to unmarshall InvokeComplete: {err}"))
                                                 })?
                         ;
                     Ok(::aws_smithy_eventstream::frame::UnmarshalledMessage::Event(
-                        crate::types::InvokeWithResponseStreamResponseEvent::InvokeComplete(parsed),
+                        super::types::InvokeWithResponseStreamResponseEvent::InvokeComplete(parsed),
                     ))
                 }
                 _unknown_variant => Ok(::aws_smithy_eventstream::frame::UnmarshalledMessage::Event(
-                    crate::types::InvokeWithResponseStreamResponseEvent::Unknown,
+                    super::types::InvokeWithResponseStreamResponseEvent::Unknown,
                 )),
             },
             "exception" => {
-                let generic = match crate::protocol_serde::parse_event_stream_error_metadata(message.payload()) {
+                let generic = match super::protocol_serde::parse_event_stream_error_metadata(message.payload()) {
                     Ok(builder) => builder.build(),
                     Err(err) => {
                         return Ok(::aws_smithy_eventstream::frame::UnmarshalledMessage::Error(
-                            crate::types::error::InvokeWithResponseStreamResponseEventError::unhandled(err),
+                            super::types::error::InvokeWithResponseStreamResponseEventError::unhandled(err),
                         ))
                     }
                 };
                 Ok(::aws_smithy_eventstream::frame::UnmarshalledMessage::Error(
-                    crate::types::error::InvokeWithResponseStreamResponseEventError::generic(generic),
+                    super::types::error::InvokeWithResponseStreamResponseEventError::generic(generic),
                 ))
             }
             value => {
