--- original
+++ modified
@@ -14,7 +14,7 @@
     /// <p>The resource compliance status.</p><note>
     /// <p>For the <code>GetAggregateComplianceDetailsByConfigRuleRequest</code> data type, Config supports only the <code>COMPLIANT</code> and <code>NON_COMPLIANT</code>. Config does not support the <code>NOT_APPLICABLE</code> and <code>INSUFFICIENT_DATA</code> values.</p>
     /// </note>
-    pub compliance_type: ::std::option::Option<crate::types::ComplianceType>,
+    pub compliance_type: ::std::option::Option<super::types::ComplianceType>,
     /// <p>The maximum number of evaluation results returned on each page. The default is 50. You cannot specify a number greater than 100. If you specify 0, Config uses the default.</p>
     pub limit: ::std::option::Option<i32>,
     /// <p>The <code>nextToken</code> string returned on a previous page that you use to get the next page of results in a paginated response.</p>
@@ -40,7 +40,7 @@
     /// <p>The resource compliance status.</p><note>
     /// <p>For the <code>GetAggregateComplianceDetailsByConfigRuleRequest</code> data type, Config supports only the <code>COMPLIANT</code> and <code>NON_COMPLIANT</code>. Config does not support the <code>NOT_APPLICABLE</code> and <code>INSUFFICIENT_DATA</code> values.</p>
     /// </note>
-    pub fn compliance_type(&self) -> ::std::option::Option<&crate::types::ComplianceType> {
+    pub fn compliance_type(&self) -> ::std::option::Option<&super::types::ComplianceType> {
         self.compliance_type.as_ref()
     }
     /// <p>The maximum number of evaluation results returned on each page. The default is 50. You cannot specify a number greater than 100. If you specify 0, Config uses the default.</p>
@@ -55,8 +55,8 @@
 impl GetAggregateComplianceDetailsByConfigRuleInput {
     /// Creates a new builder-style object to manufacture [`GetAggregateComplianceDetailsByConfigRuleInput`](crate::operation::get_aggregate_compliance_details_by_config_rule::GetAggregateComplianceDetailsByConfigRuleInput).
     pub fn builder(
-    ) -> crate::operation::get_aggregate_compliance_details_by_config_rule::builders::GetAggregateComplianceDetailsByConfigRuleInputBuilder {
-        crate::operation::get_aggregate_compliance_details_by_config_rule::builders::GetAggregateComplianceDetailsByConfigRuleInputBuilder::default()
+    ) -> super::operation::get_aggregate_compliance_details_by_config_rule::builders::GetAggregateComplianceDetailsByConfigRuleInputBuilder {
+        super::operation::get_aggregate_compliance_details_by_config_rule::builders::GetAggregateComplianceDetailsByConfigRuleInputBuilder::default()
     }
 }

@@ -68,7 +68,7 @@
     pub(crate) config_rule_name: ::std::option::Option<::std::string::String>,
     pub(crate) account_id: ::std::option::Option<::std::string::String>,
     pub(crate) aws_region: ::std::option::Option<::std::string::String>,
-    pub(crate) compliance_type: ::std::option::Option<crate::types::ComplianceType>,
+    pub(crate) compliance_type: ::std::option::Option<super::types::ComplianceType>,
     pub(crate) limit: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
 }
@@ -136,7 +136,7 @@
     /// <p>The resource compliance status.</p><note>
     /// <p>For the <code>GetAggregateComplianceDetailsByConfigRuleRequest</code> data type, Config supports only the <code>COMPLIANT</code> and <code>NON_COMPLIANT</code>. Config does not support the <code>NOT_APPLICABLE</code> and <code>INSUFFICIENT_DATA</code> values.</p>
     /// </note>
-    pub fn compliance_type(mut self, input: crate::types::ComplianceType) -> Self {
+    pub fn compliance_type(mut self, input: super::types::ComplianceType) -> Self {
         self.compliance_type = ::std::option::Option::Some(input);
         self
     }
@@ -143,7 +143,7 @@
     /// <p>The resource compliance status.</p><note>
     /// <p>For the <code>GetAggregateComplianceDetailsByConfigRuleRequest</code> data type, Config supports only the <code>COMPLIANT</code> and <code>NON_COMPLIANT</code>. Config does not support the <code>NOT_APPLICABLE</code> and <code>INSUFFICIENT_DATA</code> values.</p>
     /// </note>
-    pub fn set_compliance_type(mut self, input: ::std::option::Option<crate::types::ComplianceType>) -> Self {
+    pub fn set_compliance_type(mut self, input: ::std::option::Option<super::types::ComplianceType>) -> Self {
         self.compliance_type = input;
         self
     }
@@ -150,7 +150,7 @@
     /// <p>The resource compliance status.</p><note>
     /// <p>For the <code>GetAggregateComplianceDetailsByConfigRuleRequest</code> data type, Config supports only the <code>COMPLIANT</code> and <code>NON_COMPLIANT</code>. Config does not support the <code>NOT_APPLICABLE</code> and <code>INSUFFICIENT_DATA</code> values.</p>
     /// </note>
-    pub fn get_compliance_type(&self) -> &::std::option::Option<crate::types::ComplianceType> {
+    pub fn get_compliance_type(&self) -> &::std::option::Option<super::types::ComplianceType> {
         &self.compliance_type
     }
     /// <p>The maximum number of evaluation results returned on each page. The default is 50. You cannot specify a number greater than 100. If you specify 0, Config uses the default.</p>
@@ -185,11 +185,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_aggregate_compliance_details_by_config_rule::GetAggregateComplianceDetailsByConfigRuleInput,
+        super::operation::get_aggregate_compliance_details_by_config_rule::GetAggregateComplianceDetailsByConfigRuleInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_aggregate_compliance_details_by_config_rule::GetAggregateComplianceDetailsByConfigRuleInput {
+            super::operation::get_aggregate_compliance_details_by_config_rule::GetAggregateComplianceDetailsByConfigRuleInput {
                 configuration_aggregator_name: self.configuration_aggregator_name,
                 config_rule_name: self.config_rule_name,
                 account_id: self.account_id,
