--- original
+++ modified
@@ -50,11 +50,11 @@
     /// <li>
     /// <p>SM2DSA uses the SM3 hashing algorithm. For details, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/offline-operations.html#key-spec-sm-offline-verification">Offline verification with SM2 key pairs</a>.</p></li>
     /// </ul>
-    pub message_type: ::std::option::Option<crate::types::MessageType>,
+    pub message_type: ::std::option::Option<super::types::MessageType>,
     /// <p>The signature that the <code>Sign</code> operation generated.</p>
     pub signature: ::std::option::Option<::aws_smithy_types::Blob>,
     /// <p>The signing algorithm that was used to sign the message. If you submit a different algorithm, the signature verification fails.</p>
-    pub signing_algorithm: ::std::option::Option<crate::types::SigningAlgorithmSpec>,
+    pub signing_algorithm: ::std::option::Option<super::types::SigningAlgorithmSpec>,
     /// <p>A list of grant tokens.</p>
     /// <p>Use a grant token when your permission to call this operation comes from a new grant that has not yet achieved <i>eventual consistency</i>. For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/grants.html#grant_token">Grant token</a> and <a href="https://docs.aws.amazon.com/kms/latest/developerguide/using-grant-token.html">Using a grant token</a> in the <i>Key Management Service Developer Guide</i>.</p>
     pub grant_tokens: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -114,7 +114,7 @@
     /// <li>
     /// <p>SM2DSA uses the SM3 hashing algorithm. For details, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/offline-operations.html#key-spec-sm-offline-verification">Offline verification with SM2 key pairs</a>.</p></li>
     /// </ul>
-    pub fn message_type(&self) -> ::std::option::Option<&crate::types::MessageType> {
+    pub fn message_type(&self) -> ::std::option::Option<&super::types::MessageType> {
         self.message_type.as_ref()
     }
     /// <p>The signature that the <code>Sign</code> operation generated.</p>
@@ -122,7 +122,7 @@
         self.signature.as_ref()
     }
     /// <p>The signing algorithm that was used to sign the message. If you submit a different algorithm, the signature verification fails.</p>
-    pub fn signing_algorithm(&self) -> ::std::option::Option<&crate::types::SigningAlgorithmSpec> {
+    pub fn signing_algorithm(&self) -> ::std::option::Option<&super::types::SigningAlgorithmSpec> {
         self.signing_algorithm.as_ref()
     }
     /// <p>A list of grant tokens.</p>
@@ -153,8 +153,8 @@
 }
 impl VerifyInput {
     /// Creates a new builder-style object to manufacture [`VerifyInput`](crate::operation::verify::VerifyInput).
-    pub fn builder() -> crate::operation::verify::builders::VerifyInputBuilder {
-        crate::operation::verify::builders::VerifyInputBuilder::default()
+    pub fn builder() -> super::operation::verify::builders::VerifyInputBuilder {
+        super::operation::verify::builders::VerifyInputBuilder::default()
     }
 }

@@ -164,9 +164,9 @@
 pub struct VerifyInputBuilder {
     pub(crate) key_id: ::std::option::Option<::std::string::String>,
     pub(crate) message: ::std::option::Option<::aws_smithy_types::Blob>,
-    pub(crate) message_type: ::std::option::Option<crate::types::MessageType>,
+    pub(crate) message_type: ::std::option::Option<super::types::MessageType>,
     pub(crate) signature: ::std::option::Option<::aws_smithy_types::Blob>,
-    pub(crate) signing_algorithm: ::std::option::Option<crate::types::SigningAlgorithmSpec>,
+    pub(crate) signing_algorithm: ::std::option::Option<super::types::SigningAlgorithmSpec>,
     pub(crate) grant_tokens: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) dry_run: ::std::option::Option<bool>,
 }
@@ -272,7 +272,7 @@
     /// <li>
     /// <p>SM2DSA uses the SM3 hashing algorithm. For details, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/offline-operations.html#key-spec-sm-offline-verification">Offline verification with SM2 key pairs</a>.</p></li>
     /// </ul>
-    pub fn message_type(mut self, input: crate::types::MessageType) -> Self {
+    pub fn message_type(mut self, input: super::types::MessageType) -> Self {
         self.message_type = ::std::option::Option::Some(input);
         self
     }
@@ -305,7 +305,7 @@
     /// <li>
     /// <p>SM2DSA uses the SM3 hashing algorithm. For details, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/offline-operations.html#key-spec-sm-offline-verification">Offline verification with SM2 key pairs</a>.</p></li>
     /// </ul>
-    pub fn set_message_type(mut self, input: ::std::option::Option<crate::types::MessageType>) -> Self {
+    pub fn set_message_type(mut self, input: ::std::option::Option<super::types::MessageType>) -> Self {
         self.message_type = input;
         self
     }
@@ -338,7 +338,7 @@
     /// <li>
     /// <p>SM2DSA uses the SM3 hashing algorithm. For details, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/offline-operations.html#key-spec-sm-offline-verification">Offline verification with SM2 key pairs</a>.</p></li>
     /// </ul>
-    pub fn get_message_type(&self) -> &::std::option::Option<crate::types::MessageType> {
+    pub fn get_message_type(&self) -> &::std::option::Option<super::types::MessageType> {
         &self.message_type
     }
     /// <p>The signature that the <code>Sign</code> operation generated.</p>
@@ -358,17 +358,17 @@
     }
     /// <p>The signing algorithm that was used to sign the message. If you submit a different algorithm, the signature verification fails.</p>
     /// This field is required.
-    pub fn signing_algorithm(mut self, input: crate::types::SigningAlgorithmSpec) -> Self {
+    pub fn signing_algorithm(mut self, input: super::types::SigningAlgorithmSpec) -> Self {
         self.signing_algorithm = ::std::option::Option::Some(input);
         self
     }
     /// <p>The signing algorithm that was used to sign the message. If you submit a different algorithm, the signature verification fails.</p>
-    pub fn set_signing_algorithm(mut self, input: ::std::option::Option<crate::types::SigningAlgorithmSpec>) -> Self {
+    pub fn set_signing_algorithm(mut self, input: ::std::option::Option<super::types::SigningAlgorithmSpec>) -> Self {
         self.signing_algorithm = input;
         self
     }
     /// <p>The signing algorithm that was used to sign the message. If you submit a different algorithm, the signature verification fails.</p>
-    pub fn get_signing_algorithm(&self) -> &::std::option::Option<crate::types::SigningAlgorithmSpec> {
+    pub fn get_signing_algorithm(&self) -> &::std::option::Option<super::types::SigningAlgorithmSpec> {
         &self.signing_algorithm
     }
     /// Appends an item to `grant_tokens`.
@@ -412,8 +412,8 @@
         &self.dry_run
     }
     /// Consumes the builder and constructs a [`VerifyInput`](crate::operation::verify::VerifyInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::verify::VerifyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::verify::VerifyInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::verify::VerifyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::verify::VerifyInput {
             key_id: self.key_id,
             message: self.message,
             message_type: self.message_type,
