--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListDirectoryBucketsInput {
     /// Creates a new builder-style object to manufacture [`ListDirectoryBucketsInput`](crate::operation::list_directory_buckets::ListDirectoryBucketsInput).
-    pub fn builder() -> crate::operation::list_directory_buckets::builders::ListDirectoryBucketsInputBuilder {
-        crate::operation::list_directory_buckets::builders::ListDirectoryBucketsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_directory_buckets::builders::ListDirectoryBucketsInputBuilder {
+        super::super::super::operation::list_directory_buckets::builders::ListDirectoryBucketsInputBuilder::default()
     }
 }

@@ -64,9 +64,9 @@
     /// Consumes the builder and constructs a [`ListDirectoryBucketsInput`](crate::operation::list_directory_buckets::ListDirectoryBucketsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_directory_buckets::ListDirectoryBucketsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_directory_buckets::ListDirectoryBucketsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_directory_buckets::ListDirectoryBucketsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_directory_buckets::ListDirectoryBucketsInput {
             continuation_token: self.continuation_token,
             max_directory_buckets: self.max_directory_buckets,
         })
