--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl AdminListGroupsForUserInput {
     /// Creates a new builder-style object to manufacture [`AdminListGroupsForUserInput`](crate::operation::admin_list_groups_for_user::AdminListGroupsForUserInput).
-    pub fn builder() -> crate::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserInputBuilder {
-        crate::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserInputBuilder {
+        super::super::super::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserInputBuilder::default()
     }
 }

@@ -119,10 +119,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::admin_list_groups_for_user::AdminListGroupsForUserInput,
+        super::super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserInput {
+        ::std::result::Result::Ok(super::super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserInput {
             username: self.username,
             user_pool_id: self.user_pool_id,
             limit: self.limit,
