--- original
+++ modified
@@ -2,7 +2,7 @@

 /// Matcher union: {"success":true}
 pub(crate) fn match_head_bucket_c955e57777ec0d736(
-    _result: ::std::result::Result<&crate::operation::head_bucket::HeadBucketOutput, &crate::operation::head_bucket::HeadBucketError>,
+    _result: ::std::result::Result<&super::super::operation::head_bucket::HeadBucketOutput, &super::super::operation::head_bucket::HeadBucketError>,
 ) -> bool {
     _result.is_ok()
 }
@@ -9,7 +9,7 @@

 /// Matcher union: {"errorType":"NotFound"}
 pub(crate) fn match_head_bucket_01223db2f9ae15f05(
-    _result: ::std::result::Result<&crate::operation::head_bucket::HeadBucketOutput, &crate::operation::head_bucket::HeadBucketError>,
+    _result: ::std::result::Result<&super::super::operation::head_bucket::HeadBucketOutput, &super::super::operation::head_bucket::HeadBucketError>,
 ) -> bool {
     if let ::std::result::Result::Err(err) = _result {
         if let ::std::option::Option::Some(code) = ::aws_smithy_types::error::metadata::ProvideErrorMetadata::code(err) {
@@ -21,7 +21,7 @@

 /// Matcher union: {"success":true}
 pub(crate) fn match_head_object_c955e57777ec0d736(
-    _result: ::std::result::Result<&crate::operation::head_object::HeadObjectOutput, &crate::operation::head_object::HeadObjectError>,
+    _result: ::std::result::Result<&super::super::operation::head_object::HeadObjectOutput, &super::super::operation::head_object::HeadObjectError>,
 ) -> bool {
     _result.is_ok()
 }
@@ -28,7 +28,7 @@

 /// Matcher union: {"errorType":"NotFound"}
 pub(crate) fn match_head_object_01223db2f9ae15f05(
-    _result: ::std::result::Result<&crate::operation::head_object::HeadObjectOutput, &crate::operation::head_object::HeadObjectError>,
+    _result: ::std::result::Result<&super::super::operation::head_object::HeadObjectOutput, &super::super::operation::head_object::HeadObjectError>,
 ) -> bool {
     if let ::std::result::Result::Err(err) = _result {
         if let ::std::option::Option::Some(code) = ::aws_smithy_types::error::metadata::ProvideErrorMetadata::code(err) {
