--- original
+++ modified
@@ -13,12 +13,12 @@
 ///
 #[derive(::std::clone::Clone, ::std::fmt::Debug)]
 pub struct ObjectExistsFluentBuilder {
-    handle: ::std::sync::Arc<crate::client::Handle>,
-    inner: crate::operation::head_object::builders::HeadObjectInputBuilder,
+    handle: ::std::sync::Arc<super::client::Handle>,
+    inner: super::operation::head_object::builders::HeadObjectInputBuilder,
 }
 impl ObjectExistsFluentBuilder {
     /// Creates a new `ObjectExistsFluentBuilder`.
-    pub(crate) fn new(handle: ::std::sync::Arc<crate::client::Handle>) -> Self {
+    pub(crate) fn new(handle: ::std::sync::Arc<super::client::Handle>) -> Self {
         Self {
             handle,
             inner: ::std::default::Default::default(),
@@ -25,7 +25,7 @@
         }
     }
     /// Access the HeadObject as a reference.
-    pub fn as_input(&self) -> &crate::operation::head_object::builders::HeadObjectInputBuilder {
+    pub fn as_input(&self) -> &super::operation::head_object::builders::HeadObjectInputBuilder {
         &self.inner
     }
     /// Wait for `object_exists`
@@ -32,17 +32,17 @@
     pub async fn wait(
         self,
         max_wait: ::std::time::Duration,
-    ) -> ::std::result::Result<crate::waiters::object_exists::ObjectExistsFinalPoll, crate::waiters::object_exists::WaitUntilObjectExistsError> {
+    ) -> ::std::result::Result<super::waiters::object_exists::ObjectExistsFinalPoll, super::waiters::object_exists::WaitUntilObjectExistsError> {
         let input = self
             .inner
             .build()
             .map_err(::aws_smithy_runtime_api::client::waiters::error::WaiterError::construction_failure)?;
-        let runtime_plugins = crate::operation::head_object::HeadObject::operation_runtime_plugins(
+        let runtime_plugins = super::operation::head_object::HeadObject::operation_runtime_plugins(
             self.handle.runtime_plugins.clone(),
             &self.handle.conf,
             ::std::option::Option::None,
         )
-        .with_operation_plugin(crate::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
+        .with_operation_plugin(super::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
         let mut cfg = ::aws_smithy_types::config_bag::ConfigBag::base();
         let runtime_components_builder = runtime_plugins
             .apply_client_configuration(&mut cfg)
@@ -52,15 +52,15 @@
         let time_source = time_components.time_source().expect("a time source is required by waiters");

         let acceptor = move |result: ::std::result::Result<
-            &crate::operation::head_object::HeadObjectOutput,
-            &crate::operation::head_object::HeadObjectError,
+            &super::operation::head_object::HeadObjectOutput,
+            &super::operation::head_object::HeadObjectError,
         >| {
             // Matches: {"success":true}
-            if crate::waiters::matchers::match_head_object_c955e57777ec0d736(result) {
+            if super::waiters::matchers::match_head_object_c955e57777ec0d736(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Success;
             }
             // Matches: {"errorType":"NotFound"}
-            if crate::waiters::matchers::match_head_object_01223db2f9ae15f05(result) {
+            if super::waiters::matchers::match_head_object_01223db2f9ae15f05(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Retry;
             }
             ::aws_smithy_runtime::client::waiters::AcceptorState::NoAcceptorsMatched
@@ -68,7 +68,7 @@
         let operation = move || {
             let input = input.clone();
             let runtime_plugins = runtime_plugins.clone();
-            async move { crate::operation::head_object::HeadObject::orchestrate(&runtime_plugins, input).await }
+            async move { super::operation::head_object::HeadObject::orchestrate(&runtime_plugins, input).await }
         };
         let orchestrator = ::aws_smithy_runtime::client::waiters::WaiterOrchestrator::builder()
             .min_delay(::std::time::Duration::from_secs(5))
@@ -469,7 +469,7 @@
     /// <p>Confirms that the requester knows that they will be charged for the request. Bucket owners need not specify this parameter in their requests. If either the source or destination S3 bucket has Requester Pays enabled, the requester will pay for the corresponding charges. For information about downloading objects from Requester Pays buckets, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/ObjectsinRequesterPaysBuckets.html">Downloading Objects in Requester Pays Buckets</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn request_payer(mut self, input: crate::types::RequestPayer) -> Self {
+    pub fn request_payer(mut self, input: super::types::RequestPayer) -> Self {
         self.inner = self.inner.request_payer(input);
         self
     }
@@ -476,7 +476,7 @@
     /// <p>Confirms that the requester knows that they will be charged for the request. Bucket owners need not specify this parameter in their requests. If either the source or destination S3 bucket has Requester Pays enabled, the requester will pay for the corresponding charges. For information about downloading objects from Requester Pays buckets, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/ObjectsinRequesterPaysBuckets.html">Downloading Objects in Requester Pays Buckets</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn set_request_payer(mut self, input: ::std::option::Option<crate::types::RequestPayer>) -> Self {
+    pub fn set_request_payer(mut self, input: ::std::option::Option<super::types::RequestPayer>) -> Self {
         self.inner = self.inner.set_request_payer(input);
         self
     }
@@ -483,7 +483,7 @@
     /// <p>Confirms that the requester knows that they will be charged for the request. Bucket owners need not specify this parameter in their requests. If either the source or destination S3 bucket has Requester Pays enabled, the requester will pay for the corresponding charges. For information about downloading objects from Requester Pays buckets, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/ObjectsinRequesterPaysBuckets.html">Downloading Objects in Requester Pays Buckets</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets.</p>
     /// </note>
-    pub fn get_request_payer(&self) -> &::std::option::Option<crate::types::RequestPayer> {
+    pub fn get_request_payer(&self) -> &::std::option::Option<super::types::RequestPayer> {
         self.inner.get_request_payer()
     }
     /// <p>Part number of the object being read. This is a positive integer between 1 and 10,000. Effectively performs a 'ranged' HEAD request for the part specified. Useful querying about the size of the part and the number of parts in this object.</p>
@@ -517,7 +517,7 @@
     /// <p>To retrieve the checksum, this parameter must be enabled.</p>
     /// <p><b>General purpose buckets</b> - If you enable checksum mode and the object is uploaded with a <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_Checksum.html">checksum</a> and encrypted with an Key Management Service (KMS) key, you must have permission to use the <code>kms:Decrypt</code> action to retrieve the checksum.</p>
     /// <p><b>Directory buckets</b> - If you enable <code>ChecksumMode</code> and the object is encrypted with Amazon Web Services Key Management Service (Amazon Web Services KMS), you must also have the <code>kms:GenerateDataKey</code> and <code>kms:Decrypt</code> permissions in IAM identity-based policies and KMS key policies for the KMS key to retrieve the checksum of the object.</p>
-    pub fn checksum_mode(mut self, input: crate::types::ChecksumMode) -> Self {
+    pub fn checksum_mode(mut self, input: super::types::ChecksumMode) -> Self {
         self.inner = self.inner.checksum_mode(input);
         self
     }
@@ -524,7 +524,7 @@
     /// <p>To retrieve the checksum, this parameter must be enabled.</p>
     /// <p><b>General purpose buckets</b> - If you enable checksum mode and the object is uploaded with a <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_Checksum.html">checksum</a> and encrypted with an Key Management Service (KMS) key, you must have permission to use the <code>kms:Decrypt</code> action to retrieve the checksum.</p>
     /// <p><b>Directory buckets</b> - If you enable <code>ChecksumMode</code> and the object is encrypted with Amazon Web Services Key Management Service (Amazon Web Services KMS), you must also have the <code>kms:GenerateDataKey</code> and <code>kms:Decrypt</code> permissions in IAM identity-based policies and KMS key policies for the KMS key to retrieve the checksum of the object.</p>
-    pub fn set_checksum_mode(mut self, input: ::std::option::Option<crate::types::ChecksumMode>) -> Self {
+    pub fn set_checksum_mode(mut self, input: ::std::option::Option<super::types::ChecksumMode>) -> Self {
         self.inner = self.inner.set_checksum_mode(input);
         self
     }
@@ -531,7 +531,7 @@
     /// <p>To retrieve the checksum, this parameter must be enabled.</p>
     /// <p><b>General purpose buckets</b> - If you enable checksum mode and the object is uploaded with a <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_Checksum.html">checksum</a> and encrypted with an Key Management Service (KMS) key, you must have permission to use the <code>kms:Decrypt</code> action to retrieve the checksum.</p>
     /// <p><b>Directory buckets</b> - If you enable <code>ChecksumMode</code> and the object is encrypted with Amazon Web Services Key Management Service (Amazon Web Services KMS), you must also have the <code>kms:GenerateDataKey</code> and <code>kms:Decrypt</code> permissions in IAM identity-based policies and KMS key policies for the KMS key to retrieve the checksum of the object.</p>
-    pub fn get_checksum_mode(&self) -> &::std::option::Option<crate::types::ChecksumMode> {
+    pub fn get_checksum_mode(&self) -> &::std::option::Option<super::types::ChecksumMode> {
         self.inner.get_checksum_mode()
     }
 }
@@ -538,9 +538,9 @@

 /// Successful return type for the `object_exists` waiter.
 pub type ObjectExistsFinalPoll = ::aws_smithy_runtime_api::client::waiters::FinalPoll<
-    crate::operation::head_object::HeadObjectOutput,
+    super::operation::head_object::HeadObjectOutput,
     ::aws_smithy_runtime_api::client::result::SdkError<
-        crate::operation::head_object::HeadObjectError,
+        super::operation::head_object::HeadObjectError,
         ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
     >,
 >;
@@ -547,6 +547,6 @@

 /// Error type for the `object_exists` waiter.
 pub type WaitUntilObjectExistsError = ::aws_smithy_runtime_api::client::waiters::error::WaiterError<
-    crate::operation::head_object::HeadObjectOutput,
-    crate::operation::head_object::HeadObjectError,
+    super::operation::head_object::HeadObjectOutput,
+    super::operation::head_object::HeadObjectError,
 >;
