--- original
+++ modified
@@ -18,7 +18,7 @@
     ///   - [`ciphertext_for_recipient(Option<Blob>)`](crate::operation::decrypt::DecryptOutput::ciphertext_for_recipient): <p>The plaintext data encrypted with the public key from the attestation document. This ciphertext can be decrypted only by using a private key from the attested environment.</p> <p>This field is included in the response only when the <code>Recipient</code> parameter in the request includes a valid attestation document from an Amazon Web Services Nitro enclave or NitroTPM. For information about the interaction between KMS and Amazon Web Services Nitro Enclaves or Amazon Web Services NitroTPM, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/cryptographic-attestation.html">Cryptographic attestation support in KMS</a> in the <i>Key Management Service Developer Guide</i>.</p>
     ///   - [`key_material_id(Option<String>)`](crate::operation::decrypt::DecryptOutput::key_material_id): <p>The identifier of the key material used to decrypt the ciphertext. This field is present only when the operation uses a symmetric encryption KMS key. This field is omitted if the request includes the <code>Recipient</code> parameter.</p>
     /// - On failure, responds with [`SdkError<DecryptError>`](crate::operation::decrypt::DecryptError)
-    pub fn decrypt(&self) -> crate::operation::decrypt::builders::DecryptFluentBuilder {
-        crate::operation::decrypt::builders::DecryptFluentBuilder::new(self.handle.clone())
+    pub fn decrypt(&self) -> super::super::operation::decrypt::builders::DecryptFluentBuilder {
+        super::super::operation::decrypt::builders::DecryptFluentBuilder::new(self.handle.clone())
     }
 }
