--- original
+++ modified
@@ -6,7 +6,7 @@
     ///   - [`custom_key_store_id(impl Into<String>)`](crate::operation::delete_custom_key_store::builders::DeleteCustomKeyStoreFluentBuilder::custom_key_store_id) / [`set_custom_key_store_id(Option<String>)`](crate::operation::delete_custom_key_store::builders::DeleteCustomKeyStoreFluentBuilder::set_custom_key_store_id):<br>required: **true**<br><p>Enter the ID of the custom key store you want to delete. To find the ID of a custom key store, use the <code>DescribeCustomKeyStores</code> operation.</p><br>
     /// - On success, responds with [`DeleteCustomKeyStoreOutput`](crate::operation::delete_custom_key_store::DeleteCustomKeyStoreOutput)
     /// - On failure, responds with [`SdkError<DeleteCustomKeyStoreError>`](crate::operation::delete_custom_key_store::DeleteCustomKeyStoreError)
-    pub fn delete_custom_key_store(&self) -> crate::operation::delete_custom_key_store::builders::DeleteCustomKeyStoreFluentBuilder {
-        crate::operation::delete_custom_key_store::builders::DeleteCustomKeyStoreFluentBuilder::new(self.handle.clone())
+    pub fn delete_custom_key_store(&self) -> super::operation::delete_custom_key_store::builders::DeleteCustomKeyStoreFluentBuilder {
+        super::operation::delete_custom_key_store::builders::DeleteCustomKeyStoreFluentBuilder::new(self.handle.clone())
     }
 }
