--- original
+++ modified
@@ -16,7 +16,7 @@
     ///   - [`key_agreement_algorithm(Option<KeyAgreementAlgorithmSpec>)`](crate::operation::derive_shared_secret::DeriveSharedSecretOutput::key_agreement_algorithm): <p>Identifies the key agreement algorithm used to derive the shared secret.</p>
     ///   - [`key_origin(Option<OriginType>)`](crate::operation::derive_shared_secret::DeriveSharedSecretOutput::key_origin): <p>The source of the key material for the specified KMS key.</p> <p>When this value is <code>AWS_KMS</code>, KMS created the key material. When this value is <code>EXTERNAL</code>, the key material was imported or the KMS key doesn't have any key material.</p> <p>The only valid values for DeriveSharedSecret are <code>AWS_KMS</code> and <code>EXTERNAL</code>. DeriveSharedSecret does not support KMS keys with a <code>KeyOrigin</code> value of <code>AWS_CLOUDHSM</code> or <code>EXTERNAL_KEY_STORE</code>.</p>
     /// - On failure, responds with [`SdkError<DeriveSharedSecretError>`](crate::operation::derive_shared_secret::DeriveSharedSecretError)
-    pub fn derive_shared_secret(&self) -> crate::operation::derive_shared_secret::builders::DeriveSharedSecretFluentBuilder {
-        crate::operation::derive_shared_secret::builders::DeriveSharedSecretFluentBuilder::new(self.handle.clone())
+    pub fn derive_shared_secret(&self) -> super::super::operation::derive_shared_secret::builders::DeriveSharedSecretFluentBuilder {
+        super::super::operation::derive_shared_secret::builders::DeriveSharedSecretFluentBuilder::new(self.handle.clone())
     }
 }
