--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`DescribeKeyOutput`](crate::operation::describe_key::DescribeKeyOutput) with field(s):
     ///   - [`key_metadata(Option<KeyMetadata>)`](crate::operation::describe_key::DescribeKeyOutput::key_metadata): <p>Metadata associated with the key.</p>
     /// - On failure, responds with [`SdkError<DescribeKeyError>`](crate::operation::describe_key::DescribeKeyError)
-    pub fn describe_key(&self) -> crate::operation::describe_key::builders::DescribeKeyFluentBuilder {
-        crate::operation::describe_key::builders::DescribeKeyFluentBuilder::new(self.handle.clone())
+    pub fn describe_key(&self) -> super::super::operation::describe_key::builders::DescribeKeyFluentBuilder {
+        super::super::operation::describe_key::builders::DescribeKeyFluentBuilder::new(self.handle.clone())
     }
 }
