--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`policy(Option<String>)`](crate::operation::get_key_policy::GetKeyPolicyOutput::policy): <p>A key policy document in JSON format.</p>
     ///   - [`policy_name(Option<String>)`](crate::operation::get_key_policy::GetKeyPolicyOutput::policy_name): <p>The name of the key policy. The only valid value is <code>default</code>.</p>
     /// - On failure, responds with [`SdkError<GetKeyPolicyError>`](crate::operation::get_key_policy::GetKeyPolicyError)
-    pub fn get_key_policy(&self) -> crate::operation::get_key_policy::builders::GetKeyPolicyFluentBuilder {
-        crate::operation::get_key_policy::builders::GetKeyPolicyFluentBuilder::new(self.handle.clone())
+    pub fn get_key_policy(&self) -> super::operation::get_key_policy::builders::GetKeyPolicyFluentBuilder {
+        super::operation::get_key_policy::builders::GetKeyPolicyFluentBuilder::new(self.handle.clone())
     }
 }
