--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`tracking_start_date(Option<DateTime>)`](crate::operation::get_key_last_usage::GetKeyLastUsageOutput::tracking_start_date): <p>The date from which KMS began recording cryptographic activity for this key, or the date the KMS key was created, whichever is later.</p>
     ///   - [`key_creation_date(Option<DateTime>)`](crate::operation::get_key_last_usage::GetKeyLastUsageOutput::key_creation_date): <p>The date and time when the KMS key was created.</p>
     /// - On failure, responds with [`SdkError<GetKeyLastUsageError>`](crate::operation::get_key_last_usage::GetKeyLastUsageError)
-    pub fn get_key_last_usage(&self) -> crate::operation::get_key_last_usage::builders::GetKeyLastUsageFluentBuilder {
-        crate::operation::get_key_last_usage::builders::GetKeyLastUsageFluentBuilder::new(self.handle.clone())
+    pub fn get_key_last_usage(&self) -> super::super::operation::get_key_last_usage::builders::GetKeyLastUsageFluentBuilder {
+        super::super::operation::get_key_last_usage::builders::GetKeyLastUsageFluentBuilder::new(self.handle.clone())
     }
 }
