--- original
+++ modified
@@ -12,7 +12,7 @@
     ///   - [`next_marker(Option<String>)`](crate::operation::list_resource_tags::ListResourceTagsOutput::next_marker): <p>When <code>Truncated</code> is true, this element is present and contains the value to use for the <code>Marker</code> parameter in a subsequent request.</p> <p>Do not assume or infer any information from this value.</p>
     ///   - [`truncated(bool)`](crate::operation::list_resource_tags::ListResourceTagsOutput::truncated): <p>A flag that indicates whether there are more items in the list. When this value is true, the list in this response is truncated. To get more items, pass the value of the <code>NextMarker</code> element in this response to the <code>Marker</code> parameter in a subsequent request.</p>
     /// - On failure, responds with [`SdkError<ListResourceTagsError>`](crate::operation::list_resource_tags::ListResourceTagsError)
-    pub fn list_resource_tags(&self) -> crate::operation::list_resource_tags::builders::ListResourceTagsFluentBuilder {
-        crate::operation::list_resource_tags::builders::ListResourceTagsFluentBuilder::new(self.handle.clone())
+    pub fn list_resource_tags(&self) -> super::super::operation::list_resource_tags::builders::ListResourceTagsFluentBuilder {
+        super::super::operation::list_resource_tags::builders::ListResourceTagsFluentBuilder::new(self.handle.clone())
     }
 }
