--- original
+++ modified
@@ -13,7 +13,7 @@
     ///   - [`next_marker(Option<String>)`](crate::operation::list_retirable_grants::ListRetirableGrantsOutput::next_marker): <p>When <code>Truncated</code> is true, this element is present and contains the value to use for the <code>Marker</code> parameter in a subsequent request.</p>
     ///   - [`truncated(bool)`](crate::operation::list_retirable_grants::ListRetirableGrantsOutput::truncated): <p>A flag that indicates whether there are more items in the list. When this value is true, the list in this response is truncated. To get more items, pass the value of the <code>NextMarker</code> element in this response to the <code>Marker</code> parameter in a subsequent request.</p>
     /// - On failure, responds with [`SdkError<ListRetirableGrantsError>`](crate::operation::list_retirable_grants::ListRetirableGrantsError)
-    pub fn list_retirable_grants(&self) -> crate::operation::list_retirable_grants::builders::ListRetirableGrantsFluentBuilder {
-        crate::operation::list_retirable_grants::builders::ListRetirableGrantsFluentBuilder::new(self.handle.clone())
+    pub fn list_retirable_grants(&self) -> super::operation::list_retirable_grants::builders::ListRetirableGrantsFluentBuilder {
+        super::operation::list_retirable_grants::builders::ListRetirableGrantsFluentBuilder::new(self.handle.clone())
     }
 }
