--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`dry_run(bool)`](crate::operation::retire_grant::builders::RetireGrantFluentBuilder::dry_run) / [`set_dry_run(Option<bool>)`](crate::operation::retire_grant::builders::RetireGrantFluentBuilder::set_dry_run):<br>required: **false**<br><p>Checks if your request will succeed. <code>DryRun</code> is an optional parameter.</p> <p>To learn more about how to use this parameter, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/testing-permissions.html">Testing your permissions</a> in the <i>Key Management Service Developer Guide</i>.</p><br>
     /// - On success, responds with [`RetireGrantOutput`](crate::operation::retire_grant::RetireGrantOutput)
     /// - On failure, responds with [`SdkError<RetireGrantError>`](crate::operation::retire_grant::RetireGrantError)
-    pub fn retire_grant(&self) -> crate::operation::retire_grant::builders::RetireGrantFluentBuilder {
-        crate::operation::retire_grant::builders::RetireGrantFluentBuilder::new(self.handle.clone())
+    pub fn retire_grant(&self) -> super::super::operation::retire_grant::builders::RetireGrantFluentBuilder {
+        super::super::operation::retire_grant::builders::RetireGrantFluentBuilder::new(self.handle.clone())
     }
 }
