--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`signature(Option<Blob>)`](crate::operation::sign::SignOutput::signature): <p>The cryptographic signature that was generated for the message.</p> <ul>  <li>   <p>When used with the supported RSA signing algorithms, the encoding of this value is defined by <a href="https://tools.ietf.org/html/rfc8017">PKCS #1 in RFC 8017</a>.</p></li>  <li>   <p>When used with the <code>ECDSA_SHA_256</code>, <code>ECDSA_SHA_384</code>, or <code>ECDSA_SHA_512</code> signing algorithms, this value is a DER-encoded object as defined by ANSI X9.62–2005 and <a href="https://tools.ietf.org/html/rfc3279#section-2.2.3">RFC 3279 Section 2.2.3</a>. This is the most commonly used signature format and is appropriate for most uses.</p></li> </ul> <p>When you use the HTTP API or the Amazon Web Services CLI, the value is Base64-encoded. Otherwise, it is not Base64-encoded.</p>
     ///   - [`signing_algorithm(Option<SigningAlgorithmSpec>)`](crate::operation::sign::SignOutput::signing_algorithm): <p>The signing algorithm that was used to sign the message.</p>
     /// - On failure, responds with [`SdkError<SignError>`](crate::operation::sign::SignError)
-    pub fn sign(&self) -> crate::operation::sign::builders::SignFluentBuilder {
-        crate::operation::sign::builders::SignFluentBuilder::new(self.handle.clone())
+    pub fn sign(&self) -> super::super::operation::sign::builders::SignFluentBuilder {
+        super::super::operation::sign::builders::SignFluentBuilder::new(self.handle.clone())
     }
 }
