--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`mac_valid(bool)`](crate::operation::verify_mac::VerifyMacOutput::mac_valid): <p>A Boolean value that indicates whether the HMAC was verified. A value of <code>True</code> indicates that the HMAC (<code>Mac</code>) was generated with the specified <code>Message</code>, HMAC KMS key (<code>KeyID</code>) and <code>MacAlgorithm.</code>.</p> <p>If the HMAC is not verified, the <code>VerifyMac</code> operation fails with a <code>KMSInvalidMacException</code> exception. This exception indicates that one or more of the inputs changed since the HMAC was computed.</p>
     ///   - [`mac_algorithm(Option<MacAlgorithmSpec>)`](crate::operation::verify_mac::VerifyMacOutput::mac_algorithm): <p>The MAC algorithm used in the verification.</p>
     /// - On failure, responds with [`SdkError<VerifyMacError>`](crate::operation::verify_mac::VerifyMacError)
-    pub fn verify_mac(&self) -> crate::operation::verify_mac::builders::VerifyMacFluentBuilder {
-        crate::operation::verify_mac::builders::VerifyMacFluentBuilder::new(self.handle.clone())
+    pub fn verify_mac(&self) -> super::super::operation::verify_mac::builders::VerifyMacFluentBuilder {
+        super::super::operation::verify_mac::builders::VerifyMacFluentBuilder::new(self.handle.clone())
     }
 }
