--- original
+++ modified
@@ -15,7 +15,7 @@
     ///   - [`signature_valid(bool)`](crate::operation::verify::VerifyOutput::signature_valid): <p>A Boolean value that indicates whether the signature was verified. A value of <code>True</code> indicates that the <code>Signature</code> was produced by signing the <code>Message</code> with the specified <code>KeyID</code> and <code>SigningAlgorithm.</code> If the signature is not verified, the <code>Verify</code> operation fails with a <code>KMSInvalidSignatureException</code> exception.</p>
     ///   - [`signing_algorithm(Option<SigningAlgorithmSpec>)`](crate::operation::verify::VerifyOutput::signing_algorithm): <p>The signing algorithm that was used to verify the signature.</p>
     /// - On failure, responds with [`SdkError<VerifyError>`](crate::operation::verify::VerifyError)
-    pub fn verify(&self) -> crate::operation::verify::builders::VerifyFluentBuilder {
-        crate::operation::verify::builders::VerifyFluentBuilder::new(self.handle.clone())
+    pub fn verify(&self) -> super::super::operation::verify::builders::VerifyFluentBuilder {
+        super::super::operation::verify::builders::VerifyFluentBuilder::new(self.handle.clone())
     }
 }
