--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DescribeConsumableResourceInput {
     /// Creates a new builder-style object to manufacture [`DescribeConsumableResourceInput`](crate::operation::describe_consumable_resource::DescribeConsumableResourceInput).
-    pub fn builder() -> crate::operation::describe_consumable_resource::builders::DescribeConsumableResourceInputBuilder {
-        crate::operation::describe_consumable_resource::builders::DescribeConsumableResourceInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::describe_consumable_resource::builders::DescribeConsumableResourceInputBuilder {
+        super::super::super::operation::describe_consumable_resource::builders::DescribeConsumableResourceInputBuilder::default()
     }
 }

@@ -45,10 +45,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_consumable_resource::DescribeConsumableResourceInput,
+        super::super::super::operation::describe_consumable_resource::DescribeConsumableResourceInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_consumable_resource::DescribeConsumableResourceInput {
+        ::std::result::Result::Ok(super::super::super::operation::describe_consumable_resource::DescribeConsumableResourceInput {
             consumable_resource: self.consumable_resource,
         })
     }
