--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetOrganizationCustomRulePolicyInput {
     /// Creates a new builder-style object to manufacture [`GetOrganizationCustomRulePolicyInput`](crate::operation::get_organization_custom_rule_policy::GetOrganizationCustomRulePolicyInput).
-    pub fn builder() -> crate::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyInputBuilder {
-        crate::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyInputBuilder {
+        super::super::super::operation::get_organization_custom_rule_policy::builders::GetOrganizationCustomRulePolicyInputBuilder::default()
     }
 }

@@ -45,11 +45,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_organization_custom_rule_policy::GetOrganizationCustomRulePolicyInput,
+        super::super::super::operation::get_organization_custom_rule_policy::GetOrganizationCustomRulePolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_organization_custom_rule_policy::GetOrganizationCustomRulePolicyInput {
+            super::super::super::operation::get_organization_custom_rule_policy::GetOrganizationCustomRulePolicyInput {
                 organization_config_rule_name: self.organization_config_rule_name,
             },
         )
