--- original
+++ modified
@@ -50,8 +50,8 @@
 }
 impl PutUserPolicyInput {
     /// Creates a new builder-style object to manufacture [`PutUserPolicyInput`](crate::operation::put_user_policy::PutUserPolicyInput).
-    pub fn builder() -> crate::operation::put_user_policy::builders::PutUserPolicyInputBuilder {
-        crate::operation::put_user_policy::builders::PutUserPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::put_user_policy::builders::PutUserPolicyInputBuilder {
+        super::operation::put_user_policy::builders::PutUserPolicyInputBuilder::default()
     }
 }

@@ -148,8 +148,8 @@
     /// Consumes the builder and constructs a [`PutUserPolicyInput`](crate::operation::put_user_policy::PutUserPolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_user_policy::PutUserPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::put_user_policy::PutUserPolicyInput {
+    ) -> ::std::result::Result<super::operation::put_user_policy::PutUserPolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::put_user_policy::PutUserPolicyInput {
             user_name: self.user_name,
             policy_name: self.policy_name,
             policy_document: self.policy_document,
