--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl ListUsersInGroupInput {
     /// Creates a new builder-style object to manufacture [`ListUsersInGroupInput`](crate::operation::list_users_in_group::ListUsersInGroupInput).
-    pub fn builder() -> crate::operation::list_users_in_group::builders::ListUsersInGroupInputBuilder {
-        crate::operation::list_users_in_group::builders::ListUsersInGroupInputBuilder::default()
+    pub fn builder() -> super::operation::list_users_in_group::builders::ListUsersInGroupInputBuilder {
+        super::operation::list_users_in_group::builders::ListUsersInGroupInputBuilder::default()
     }
 }

@@ -108,8 +108,8 @@
     /// Consumes the builder and constructs a [`ListUsersInGroupInput`](crate::operation::list_users_in_group::ListUsersInGroupInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_users_in_group::ListUsersInGroupInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_users_in_group::ListUsersInGroupInput {
+    ) -> ::std::result::Result<super::operation::list_users_in_group::ListUsersInGroupInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_users_in_group::ListUsersInGroupInput {
             user_pool_id: self.user_pool_id,
             group_name: self.group_name,
             limit: self.limit,
