--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl UntagUserInput {
     /// Creates a new builder-style object to manufacture [`UntagUserInput`](crate::operation::untag_user::UntagUserInput).
-    pub fn builder() -> crate::operation::untag_user::builders::UntagUserInputBuilder {
-        crate::operation::untag_user::builders::UntagUserInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::untag_user::builders::UntagUserInputBuilder {
+        super::super::super::operation::untag_user::builders::UntagUserInputBuilder::default()
     }
 }

@@ -76,8 +76,8 @@
         &self.tag_keys
     }
     /// Consumes the builder and constructs a [`UntagUserInput`](crate::operation::untag_user::UntagUserInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::untag_user::UntagUserInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::untag_user::UntagUserInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::untag_user::UntagUserInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::untag_user::UntagUserInput {
             user_name: self.user_name,
             tag_keys: self.tag_keys,
         })
