--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl DeleteUserInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserInput`](crate::operation::delete_user::DeleteUserInput).
-    pub fn builder() -> crate::operation::delete_user::builders::DeleteUserInputBuilder {
-        crate::operation::delete_user::builders::DeleteUserInputBuilder::default()
+    pub fn builder() -> super::operation::delete_user::builders::DeleteUserInputBuilder {
+        super::operation::delete_user::builders::DeleteUserInputBuilder::default()
     }
 }

@@ -47,7 +47,7 @@
         &self.user_name
     }
     /// Consumes the builder and constructs a [`DeleteUserInput`](crate::operation::delete_user::DeleteUserInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::delete_user::DeleteUserInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::delete_user::DeleteUserInput { user_name: self.user_name })
+    pub fn build(self) -> ::std::result::Result<super::operation::delete_user::DeleteUserInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::delete_user::DeleteUserInput { user_name: self.user_name })
     }
 }
