--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ListSourcesForS3TableIntegrationInput {
     /// Creates a new builder-style object to manufacture [`ListSourcesForS3TableIntegrationInput`](crate::operation::list_sources_for_s3_table_integration::ListSourcesForS3TableIntegrationInput).
-    pub fn builder() -> crate::operation::list_sources_for_s3_table_integration::builders::ListSourcesForS3TableIntegrationInputBuilder {
-        crate::operation::list_sources_for_s3_table_integration::builders::ListSourcesForS3TableIntegrationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_sources_for_s3_table_integration::builders::ListSourcesForS3TableIntegrationInputBuilder {
+        super::super::super::operation::list_sources_for_s3_table_integration::builders::ListSourcesForS3TableIntegrationInputBuilder::default()
     }
 }

@@ -87,11 +87,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_sources_for_s3_table_integration::ListSourcesForS3TableIntegrationInput,
+        super::super::super::operation::list_sources_for_s3_table_integration::ListSourcesForS3TableIntegrationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::list_sources_for_s3_table_integration::ListSourcesForS3TableIntegrationInput {
+            super::super::super::operation::list_sources_for_s3_table_integration::ListSourcesForS3TableIntegrationInput {
                 integration_arn: self.integration_arn,
                 max_results: self.max_results,
                 next_token: self.next_token,
