--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl UpdatePackageGroupInput {
     /// Creates a new builder-style object to manufacture [`UpdatePackageGroupInput`](crate::operation::update_package_group::UpdatePackageGroupInput).
-    pub fn builder() -> crate::operation::update_package_group::builders::UpdatePackageGroupInputBuilder {
-        crate::operation::update_package_group::builders::UpdatePackageGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_package_group::builders::UpdatePackageGroupInputBuilder {
+        super::super::super::operation::update_package_group::builders::UpdatePackageGroupInputBuilder::default()
     }
 }

@@ -129,9 +129,9 @@
     /// Consumes the builder and constructs a [`UpdatePackageGroupInput`](crate::operation::update_package_group::UpdatePackageGroupInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_package_group::UpdatePackageGroupInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::update_package_group::UpdatePackageGroupInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_package_group::UpdatePackageGroupInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_package_group::UpdatePackageGroupInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             package_group: self.package_group,
