--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl DescribeRepositoryInput {
     /// Creates a new builder-style object to manufacture [`DescribeRepositoryInput`](crate::operation::describe_repository::DescribeRepositoryInput).
-    pub fn builder() -> crate::operation::describe_repository::builders::DescribeRepositoryInputBuilder {
-        crate::operation::describe_repository::builders::DescribeRepositoryInputBuilder::default()
+    pub fn builder() -> super::operation::describe_repository::builders::DescribeRepositoryInputBuilder {
+        super::operation::describe_repository::builders::DescribeRepositoryInputBuilder::default()
     }
 }

@@ -87,8 +87,8 @@
     /// Consumes the builder and constructs a [`DescribeRepositoryInput`](crate::operation::describe_repository::DescribeRepositoryInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_repository::DescribeRepositoryInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::describe_repository::DescribeRepositoryInput {
+    ) -> ::std::result::Result<super::operation::describe_repository::DescribeRepositoryInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::describe_repository::DescribeRepositoryInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             repository: self.repository,
