--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl UpdateKeyDescriptionInput {
     /// Creates a new builder-style object to manufacture [`UpdateKeyDescriptionInput`](crate::operation::update_key_description::UpdateKeyDescriptionInput).
-    pub fn builder() -> crate::operation::update_key_description::builders::UpdateKeyDescriptionInputBuilder {
-        crate::operation::update_key_description::builders::UpdateKeyDescriptionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_key_description::builders::UpdateKeyDescriptionInputBuilder {
+        super::super::super::operation::update_key_description::builders::UpdateKeyDescriptionInputBuilder::default()
     }
 }

@@ -121,9 +121,9 @@
     /// Consumes the builder and constructs a [`UpdateKeyDescriptionInput`](crate::operation::update_key_description::UpdateKeyDescriptionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_key_description::UpdateKeyDescriptionInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::update_key_description::UpdateKeyDescriptionInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_key_description::UpdateKeyDescriptionInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_key_description::UpdateKeyDescriptionInput {
             key_id: self.key_id,
             description: self.description,
         })
