--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The unique identifier, or device key, of the device that you want to update the status for.</p>
     pub device_key: ::std::option::Option<::std::string::String>,
     /// <p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p>
-    pub device_remembered_status: ::std::option::Option<crate::types::DeviceRememberedStatusType>,
+    pub device_remembered_status: ::std::option::Option<super::types::DeviceRememberedStatusType>,
 }
 impl AdminUpdateDeviceStatusInput {
     /// <p>The ID of the user pool where you want to change a user's device status.</p>
@@ -27,7 +27,7 @@
         self.device_key.as_deref()
     }
     /// <p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p>
-    pub fn device_remembered_status(&self) -> ::std::option::Option<&crate::types::DeviceRememberedStatusType> {
+    pub fn device_remembered_status(&self) -> ::std::option::Option<&super::types::DeviceRememberedStatusType> {
         self.device_remembered_status.as_ref()
     }
 }
@@ -43,8 +43,8 @@
 }
 impl AdminUpdateDeviceStatusInput {
     /// Creates a new builder-style object to manufacture [`AdminUpdateDeviceStatusInput`](crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput).
-    pub fn builder() -> crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusInputBuilder {
-        crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusInputBuilder::default()
+    pub fn builder() -> super::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusInputBuilder {
+        super::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusInputBuilder::default()
     }
 }

@@ -55,7 +55,7 @@
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) username: ::std::option::Option<::std::string::String>,
     pub(crate) device_key: ::std::option::Option<::std::string::String>,
-    pub(crate) device_remembered_status: ::std::option::Option<crate::types::DeviceRememberedStatusType>,
+    pub(crate) device_remembered_status: ::std::option::Option<super::types::DeviceRememberedStatusType>,
 }
 impl AdminUpdateDeviceStatusInputBuilder {
     /// <p>The ID of the user pool where you want to change a user's device status.</p>
@@ -104,17 +104,17 @@
         &self.device_key
     }
     /// <p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p>
-    pub fn device_remembered_status(mut self, input: crate::types::DeviceRememberedStatusType) -> Self {
+    pub fn device_remembered_status(mut self, input: super::types::DeviceRememberedStatusType) -> Self {
         self.device_remembered_status = ::std::option::Option::Some(input);
         self
     }
     /// <p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p>
-    pub fn set_device_remembered_status(mut self, input: ::std::option::Option<crate::types::DeviceRememberedStatusType>) -> Self {
+    pub fn set_device_remembered_status(mut self, input: ::std::option::Option<super::types::DeviceRememberedStatusType>) -> Self {
         self.device_remembered_status = input;
         self
     }
     /// <p>To enable device authentication with the specified device, set to <code>remembered</code>.To disable, set to <code>not_remembered</code>.</p>
-    pub fn get_device_remembered_status(&self) -> &::std::option::Option<crate::types::DeviceRememberedStatusType> {
+    pub fn get_device_remembered_status(&self) -> &::std::option::Option<super::types::DeviceRememberedStatusType> {
         &self.device_remembered_status
     }
     /// Consumes the builder and constructs a [`AdminUpdateDeviceStatusInput`](crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput).
@@ -121,10 +121,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
+        super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput {
+        ::std::result::Result::Ok(super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
             device_key: self.device_key,
