--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ChainedInvokeSucceededDetails {
     /// <p>The response payload from the successful operation.</p>
-    pub result: ::std::option::Option<crate::types::EventResult>,
+    pub result: ::std::option::Option<super::types::EventResult>,
 }
 impl ChainedInvokeSucceededDetails {
     /// <p>The response payload from the successful operation.</p>
-    pub fn result(&self) -> ::std::option::Option<&crate::types::EventResult> {
+    pub fn result(&self) -> ::std::option::Option<&super::types::EventResult> {
         self.result.as_ref()
     }
 }
 impl ChainedInvokeSucceededDetails {
     /// Creates a new builder-style object to manufacture [`ChainedInvokeSucceededDetails`](crate::types::ChainedInvokeSucceededDetails).
-    pub fn builder() -> crate::types::builders::ChainedInvokeSucceededDetailsBuilder {
-        crate::types::builders::ChainedInvokeSucceededDetailsBuilder::default()
+    pub fn builder() -> super::types::builders::ChainedInvokeSucceededDetailsBuilder {
+        super::types::builders::ChainedInvokeSucceededDetailsBuilder::default()
     }
 }

@@ -24,26 +24,26 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ChainedInvokeSucceededDetailsBuilder {
-    pub(crate) result: ::std::option::Option<crate::types::EventResult>,
+    pub(crate) result: ::std::option::Option<super::types::EventResult>,
 }
 impl ChainedInvokeSucceededDetailsBuilder {
     /// <p>The response payload from the successful operation.</p>
     /// This field is required.
-    pub fn result(mut self, input: crate::types::EventResult) -> Self {
+    pub fn result(mut self, input: super::types::EventResult) -> Self {
         self.result = ::std::option::Option::Some(input);
         self
     }
     /// <p>The response payload from the successful operation.</p>
-    pub fn set_result(mut self, input: ::std::option::Option<crate::types::EventResult>) -> Self {
+    pub fn set_result(mut self, input: ::std::option::Option<super::types::EventResult>) -> Self {
         self.result = input;
         self
     }
     /// <p>The response payload from the successful operation.</p>
-    pub fn get_result(&self) -> &::std::option::Option<crate::types::EventResult> {
+    pub fn get_result(&self) -> &::std::option::Option<super::types::EventResult> {
         &self.result
     }
     /// Consumes the builder and constructs a [`ChainedInvokeSucceededDetails`](crate::types::ChainedInvokeSucceededDetails).
-    pub fn build(self) -> crate::types::ChainedInvokeSucceededDetails {
-        crate::types::ChainedInvokeSucceededDetails { result: self.result }
+    pub fn build(self) -> super::types::ChainedInvokeSucceededDetails {
+        super::types::ChainedInvokeSucceededDetails { result: self.result }
     }
 }
