--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl ChainedInvokeOptions {
     /// Creates a new builder-style object to manufacture [`ChainedInvokeOptions`](crate::types::ChainedInvokeOptions).
-    pub fn builder() -> crate::types::builders::ChainedInvokeOptionsBuilder {
-        crate::types::builders::ChainedInvokeOptionsBuilder::default()
+    pub fn builder() -> super::types::builders::ChainedInvokeOptionsBuilder {
+        super::types::builders::ChainedInvokeOptionsBuilder::default()
     }
 }

@@ -67,8 +67,8 @@
     /// Consumes the builder and constructs a [`ChainedInvokeOptions`](crate::types::ChainedInvokeOptions).
     /// This method will fail if any of the following fields are not set:
     /// - [`function_name`](crate::types::builders::ChainedInvokeOptionsBuilder::function_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::ChainedInvokeOptions, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ChainedInvokeOptions {
+    pub fn build(self) -> ::std::result::Result<super::types::ChainedInvokeOptions, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ChainedInvokeOptions {
             function_name: self.function_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "function_name",
