--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl Concurrency {
     /// Creates a new builder-style object to manufacture [`Concurrency`](crate::types::Concurrency).
-    pub fn builder() -> crate::types::builders::ConcurrencyBuilder {
-        crate::types::builders::ConcurrencyBuilder::default()
+    pub fn builder() -> super::super::types::builders::ConcurrencyBuilder {
+        super::super::types::builders::ConcurrencyBuilder::default()
     }
 }

@@ -41,8 +41,8 @@
         &self.reserved_concurrent_executions
     }
     /// Consumes the builder and constructs a [`Concurrency`](crate::types::Concurrency).
-    pub fn build(self) -> crate::types::Concurrency {
-        crate::types::Concurrency {
+    pub fn build(self) -> super::super::types::Concurrency {
+        super::super::types::Concurrency {
             reserved_concurrent_executions: self.reserved_concurrent_executions,
         }
     }
