--- original
+++ modified
@@ -45,8 +45,8 @@
 }
 impl ErrorObject {
     /// Creates a new builder-style object to manufacture [`ErrorObject`](crate::types::ErrorObject).
-    pub fn builder() -> crate::types::builders::ErrorObjectBuilder {
-        crate::types::builders::ErrorObjectBuilder::default()
+    pub fn builder() -> super::types::builders::ErrorObjectBuilder {
+        super::types::builders::ErrorObjectBuilder::default()
     }
 }

@@ -123,8 +123,8 @@
         &self.stack_trace
     }
     /// Consumes the builder and constructs a [`ErrorObject`](crate::types::ErrorObject).
-    pub fn build(self) -> crate::types::ErrorObject {
-        crate::types::ErrorObject {
+    pub fn build(self) -> super::types::ErrorObject {
+        super::types::ErrorObject {
             error_message: self.error_message,
             error_type: self.error_type,
             error_data: self.error_data,
