--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl EventInput {
     /// Creates a new builder-style object to manufacture [`EventInput`](crate::types::EventInput).
-    pub fn builder() -> crate::types::builders::EventInputBuilder {
-        crate::types::builders::EventInputBuilder::default()
+    pub fn builder() -> super::types::builders::EventInputBuilder {
+        super::types::builders::EventInputBuilder::default()
     }
 }

@@ -71,8 +71,8 @@
         &self.truncated
     }
     /// Consumes the builder and constructs a [`EventInput`](crate::types::EventInput).
-    pub fn build(self) -> crate::types::EventInput {
-        crate::types::EventInput {
+    pub fn build(self) -> super::types::EventInput {
+        super::types::EventInput {
             payload: self.payload,
             truncated: self.truncated,
         }
