--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl ExecutionDetails {
     /// Creates a new builder-style object to manufacture [`ExecutionDetails`](crate::types::ExecutionDetails).
-    pub fn builder() -> crate::types::builders::ExecutionDetailsBuilder {
-        crate::types::builders::ExecutionDetailsBuilder::default()
+    pub fn builder() -> super::types::builders::ExecutionDetailsBuilder {
+        super::types::builders::ExecutionDetailsBuilder::default()
     }
 }

@@ -49,8 +49,8 @@
         &self.input_payload
     }
     /// Consumes the builder and constructs a [`ExecutionDetails`](crate::types::ExecutionDetails).
-    pub fn build(self) -> crate::types::ExecutionDetails {
-        crate::types::ExecutionDetails {
+    pub fn build(self) -> super::types::ExecutionDetails {
+        super::types::ExecutionDetails {
             input_payload: self.input_payload,
         }
     }
