--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl Filter {
     /// Creates a new builder-style object to manufacture [`Filter`](crate::types::Filter).
-    pub fn builder() -> crate::types::builders::FilterBuilder {
-        crate::types::builders::FilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::FilterBuilder {
+        super::super::types::builders::FilterBuilder::default()
     }
 }

@@ -42,7 +42,7 @@
         &self.pattern
     }
     /// Consumes the builder and constructs a [`Filter`](crate::types::Filter).
-    pub fn build(self) -> crate::types::Filter {
-        crate::types::Filter { pattern: self.pattern }
+    pub fn build(self) -> super::super::types::Filter {
+        super::super::types::Filter { pattern: self.pattern }
     }
 }
