--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl FunctionScalingConfig {
     /// Creates a new builder-style object to manufacture [`FunctionScalingConfig`](crate::types::FunctionScalingConfig).
-    pub fn builder() -> crate::types::builders::FunctionScalingConfigBuilder {
-        crate::types::builders::FunctionScalingConfigBuilder::default()
+    pub fn builder() -> super::super::types::builders::FunctionScalingConfigBuilder {
+        super::super::types::builders::FunctionScalingConfigBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.max_execution_environments
     }
     /// Consumes the builder and constructs a [`FunctionScalingConfig`](crate::types::FunctionScalingConfig).
-    pub fn build(self) -> crate::types::FunctionScalingConfig {
-        crate::types::FunctionScalingConfig {
+    pub fn build(self) -> super::super::types::FunctionScalingConfig {
+        super::super::types::FunctionScalingConfig {
             min_execution_environments: self.min_execution_environments,
             max_execution_environments: self.max_execution_environments,
         }
