--- original
+++ modified
@@ -13,11 +13,11 @@
     /// <p>The resolved URI for the image.</p>
     pub resolved_image_uri: ::std::option::Option<::std::string::String>,
     /// <p>The resolved Amazon S3 object that contains the deployment package.</p>
-    pub resolved_s3_object: ::std::option::Option<crate::types::ResolvedS3Object>,
+    pub resolved_s3_object: ::std::option::Option<super::types::ResolvedS3Object>,
     /// <p>The ARN of the Key Management Service (KMS) customer managed key that's used to encrypt your function's .zip deployment package. If you don't provide a customer managed key, Lambda uses an <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#aws-owned-cmk">Amazon Web Services owned key</a>.</p>
     pub source_kms_key_arn: ::std::option::Option<::std::string::String>,
     /// <p>An object that contains details about an error related to function deployment package retrieval.</p>
-    pub error: ::std::option::Option<crate::types::FunctionCodeLocationError>,
+    pub error: ::std::option::Option<super::types::FunctionCodeLocationError>,
 }
 impl FunctionCodeLocation {
     /// <p>The service that's hosting the file.</p>
@@ -37,7 +37,7 @@
         self.resolved_image_uri.as_deref()
     }
     /// <p>The resolved Amazon S3 object that contains the deployment package.</p>
-    pub fn resolved_s3_object(&self) -> ::std::option::Option<&crate::types::ResolvedS3Object> {
+    pub fn resolved_s3_object(&self) -> ::std::option::Option<&super::types::ResolvedS3Object> {
         self.resolved_s3_object.as_ref()
     }
     /// <p>The ARN of the Key Management Service (KMS) customer managed key that's used to encrypt your function's .zip deployment package. If you don't provide a customer managed key, Lambda uses an <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#aws-owned-cmk">Amazon Web Services owned key</a>.</p>
@@ -45,14 +45,14 @@
         self.source_kms_key_arn.as_deref()
     }
     /// <p>An object that contains details about an error related to function deployment package retrieval.</p>
-    pub fn error(&self) -> ::std::option::Option<&crate::types::FunctionCodeLocationError> {
+    pub fn error(&self) -> ::std::option::Option<&super::types::FunctionCodeLocationError> {
         self.error.as_ref()
     }
 }
 impl FunctionCodeLocation {
     /// Creates a new builder-style object to manufacture [`FunctionCodeLocation`](crate::types::FunctionCodeLocation).
-    pub fn builder() -> crate::types::builders::FunctionCodeLocationBuilder {
-        crate::types::builders::FunctionCodeLocationBuilder::default()
+    pub fn builder() -> super::types::builders::FunctionCodeLocationBuilder {
+        super::types::builders::FunctionCodeLocationBuilder::default()
     }
 }

@@ -64,9 +64,9 @@
     pub(crate) location: ::std::option::Option<::std::string::String>,
     pub(crate) image_uri: ::std::option::Option<::std::string::String>,
     pub(crate) resolved_image_uri: ::std::option::Option<::std::string::String>,
-    pub(crate) resolved_s3_object: ::std::option::Option<crate::types::ResolvedS3Object>,
+    pub(crate) resolved_s3_object: ::std::option::Option<super::types::ResolvedS3Object>,
     pub(crate) source_kms_key_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) error: ::std::option::Option<crate::types::FunctionCodeLocationError>,
+    pub(crate) error: ::std::option::Option<super::types::FunctionCodeLocationError>,
 }
 impl FunctionCodeLocationBuilder {
     /// <p>The service that's hosting the file.</p>
@@ -126,17 +126,17 @@
         &self.resolved_image_uri
     }
     /// <p>The resolved Amazon S3 object that contains the deployment package.</p>
-    pub fn resolved_s3_object(mut self, input: crate::types::ResolvedS3Object) -> Self {
+    pub fn resolved_s3_object(mut self, input: super::types::ResolvedS3Object) -> Self {
         self.resolved_s3_object = ::std::option::Option::Some(input);
         self
     }
     /// <p>The resolved Amazon S3 object that contains the deployment package.</p>
-    pub fn set_resolved_s3_object(mut self, input: ::std::option::Option<crate::types::ResolvedS3Object>) -> Self {
+    pub fn set_resolved_s3_object(mut self, input: ::std::option::Option<super::types::ResolvedS3Object>) -> Self {
         self.resolved_s3_object = input;
         self
     }
     /// <p>The resolved Amazon S3 object that contains the deployment package.</p>
-    pub fn get_resolved_s3_object(&self) -> &::std::option::Option<crate::types::ResolvedS3Object> {
+    pub fn get_resolved_s3_object(&self) -> &::std::option::Option<super::types::ResolvedS3Object> {
         &self.resolved_s3_object
     }
     /// <p>The ARN of the Key Management Service (KMS) customer managed key that's used to encrypt your function's .zip deployment package. If you don't provide a customer managed key, Lambda uses an <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#aws-owned-cmk">Amazon Web Services owned key</a>.</p>
@@ -154,22 +154,22 @@
         &self.source_kms_key_arn
     }
     /// <p>An object that contains details about an error related to function deployment package retrieval.</p>
-    pub fn error(mut self, input: crate::types::FunctionCodeLocationError) -> Self {
+    pub fn error(mut self, input: super::types::FunctionCodeLocationError) -> Self {
         self.error = ::std::option::Option::Some(input);
         self
     }
     /// <p>An object that contains details about an error related to function deployment package retrieval.</p>
-    pub fn set_error(mut self, input: ::std::option::Option<crate::types::FunctionCodeLocationError>) -> Self {
+    pub fn set_error(mut self, input: ::std::option::Option<super::types::FunctionCodeLocationError>) -> Self {
         self.error = input;
         self
     }
     /// <p>An object that contains details about an error related to function deployment package retrieval.</p>
-    pub fn get_error(&self) -> &::std::option::Option<crate::types::FunctionCodeLocationError> {
+    pub fn get_error(&self) -> &::std::option::Option<super::types::FunctionCodeLocationError> {
         &self.error
     }
     /// Consumes the builder and constructs a [`FunctionCodeLocation`](crate::types::FunctionCodeLocation).
-    pub fn build(self) -> crate::types::FunctionCodeLocation {
-        crate::types::FunctionCodeLocation {
+    pub fn build(self) -> super::types::FunctionCodeLocation {
+        super::types::FunctionCodeLocation {
             repository_type: self.repository_type,
             location: self.location,
             image_uri: self.image_uri,
