--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl ImageConfigError {
     /// Creates a new builder-style object to manufacture [`ImageConfigError`](crate::types::ImageConfigError).
-    pub fn builder() -> crate::types::builders::ImageConfigErrorBuilder {
-        crate::types::builders::ImageConfigErrorBuilder::default()
+    pub fn builder() -> super::types::builders::ImageConfigErrorBuilder {
+        super::types::builders::ImageConfigErrorBuilder::default()
     }
 }

@@ -71,8 +71,8 @@
         &self.message
     }
     /// Consumes the builder and constructs a [`ImageConfigError`](crate::types::ImageConfigError).
-    pub fn build(self) -> crate::types::ImageConfigError {
-        crate::types::ImageConfigError {
+    pub fn build(self) -> super::types::ImageConfigError {
+        super::types::ImageConfigError {
             error_code: self.error_code,
             message: self.message,
         }
