--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl InvokeResponseStreamUpdate {
     /// Creates a new builder-style object to manufacture [`InvokeResponseStreamUpdate`](crate::types::InvokeResponseStreamUpdate).
-    pub fn builder() -> crate::types::builders::InvokeResponseStreamUpdateBuilder {
-        crate::types::builders::InvokeResponseStreamUpdateBuilder::default()
+    pub fn builder() -> super::super::types::builders::InvokeResponseStreamUpdateBuilder {
+        super::super::types::builders::InvokeResponseStreamUpdateBuilder::default()
     }
 }

@@ -49,8 +49,8 @@
         &self.payload
     }
     /// Consumes the builder and constructs a [`InvokeResponseStreamUpdate`](crate::types::InvokeResponseStreamUpdate).
-    pub fn build(self) -> crate::types::InvokeResponseStreamUpdate {
-        crate::types::InvokeResponseStreamUpdate { payload: self.payload }
+    pub fn build(self) -> super::super::types::InvokeResponseStreamUpdate {
+        super::super::types::InvokeResponseStreamUpdate { payload: self.payload }
     }
 }
 impl ::std::fmt::Debug for InvokeResponseStreamUpdateBuilder {
