--- original
+++ modified
@@ -5,9 +5,9 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub enum InvokeWithResponseStreamResponseEvent {
     /// <p>An object that's returned when the stream has ended and all the payload chunks have been returned.</p>
-    InvokeComplete(crate::types::InvokeWithResponseStreamCompleteEvent),
+    InvokeComplete(super::super::types::InvokeWithResponseStreamCompleteEvent),
     /// <p>A chunk of the streamed response payload.</p>
-    PayloadChunk(crate::types::InvokeResponseStreamUpdate),
+    PayloadChunk(super::super::types::InvokeResponseStreamUpdate),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -21,7 +21,7 @@
 impl InvokeWithResponseStreamResponseEvent {
     /// Tries to convert the enum instance into [`InvokeComplete`](crate::types::InvokeWithResponseStreamResponseEvent::InvokeComplete), extracting the inner [`InvokeWithResponseStreamCompleteEvent`](crate::types::InvokeWithResponseStreamCompleteEvent).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_invoke_complete(&self) -> ::std::result::Result<&crate::types::InvokeWithResponseStreamCompleteEvent, &Self> {
+    pub fn as_invoke_complete(&self) -> ::std::result::Result<&super::super::types::InvokeWithResponseStreamCompleteEvent, &Self> {
         if let InvokeWithResponseStreamResponseEvent::InvokeComplete(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -34,7 +34,7 @@
     }
     /// Tries to convert the enum instance into [`PayloadChunk`](crate::types::InvokeWithResponseStreamResponseEvent::PayloadChunk), extracting the inner [`InvokeResponseStreamUpdate`](crate::types::InvokeResponseStreamUpdate).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_payload_chunk(&self) -> ::std::result::Result<&crate::types::InvokeResponseStreamUpdate, &Self> {
+    pub fn as_payload_chunk(&self) -> ::std::result::Result<&super::super::types::InvokeResponseStreamUpdate, &Self> {
         if let InvokeWithResponseStreamResponseEvent::PayloadChunk(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
