--- original
+++ modified
@@ -17,7 +17,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the layer archive from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub s3_object_storage_mode: ::std::option::Option<crate::types::S3ObjectStorageMode>,
+    pub s3_object_storage_mode: ::std::option::Option<super::types::S3ObjectStorageMode>,
     /// <p>The base64-encoded contents of the layer archive. Amazon Web Services SDK and Amazon Web Services CLI clients handle the encoding for you.</p>
     pub zip_file: ::std::option::Option<::aws_smithy_types::Blob>,
 }
@@ -41,7 +41,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the layer archive from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn s3_object_storage_mode(&self) -> ::std::option::Option<&crate::types::S3ObjectStorageMode> {
+    pub fn s3_object_storage_mode(&self) -> ::std::option::Option<&super::types::S3ObjectStorageMode> {
         self.s3_object_storage_mode.as_ref()
     }
     /// <p>The base64-encoded contents of the layer archive. Amazon Web Services SDK and Amazon Web Services CLI clients handle the encoding for you.</p>
@@ -62,8 +62,8 @@
 }
 impl LayerVersionContentInput {
     /// Creates a new builder-style object to manufacture [`LayerVersionContentInput`](crate::types::LayerVersionContentInput).
-    pub fn builder() -> crate::types::builders::LayerVersionContentInputBuilder {
-        crate::types::builders::LayerVersionContentInputBuilder::default()
+    pub fn builder() -> super::types::builders::LayerVersionContentInputBuilder {
+        super::types::builders::LayerVersionContentInputBuilder::default()
     }
 }

@@ -74,7 +74,7 @@
     pub(crate) s3_bucket: ::std::option::Option<::std::string::String>,
     pub(crate) s3_key: ::std::option::Option<::std::string::String>,
     pub(crate) s3_object_version: ::std::option::Option<::std::string::String>,
-    pub(crate) s3_object_storage_mode: ::std::option::Option<crate::types::S3ObjectStorageMode>,
+    pub(crate) s3_object_storage_mode: ::std::option::Option<super::types::S3ObjectStorageMode>,
     pub(crate) zip_file: ::std::option::Option<::aws_smithy_types::Blob>,
 }
 impl LayerVersionContentInputBuilder {
@@ -127,7 +127,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the layer archive from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn s3_object_storage_mode(mut self, input: crate::types::S3ObjectStorageMode) -> Self {
+    pub fn s3_object_storage_mode(mut self, input: super::types::S3ObjectStorageMode) -> Self {
         self.s3_object_storage_mode = ::std::option::Option::Some(input);
         self
     }
@@ -138,7 +138,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the layer archive from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn set_s3_object_storage_mode(mut self, input: ::std::option::Option<crate::types::S3ObjectStorageMode>) -> Self {
+    pub fn set_s3_object_storage_mode(mut self, input: ::std::option::Option<super::types::S3ObjectStorageMode>) -> Self {
         self.s3_object_storage_mode = input;
         self
     }
@@ -149,7 +149,7 @@
     /// <li>
     /// <p><code>REFERENCE</code> – Lambda references the layer archive from the specified Amazon S3 bucket.</p></li>
     /// </ul>
-    pub fn get_s3_object_storage_mode(&self) -> &::std::option::Option<crate::types::S3ObjectStorageMode> {
+    pub fn get_s3_object_storage_mode(&self) -> &::std::option::Option<super::types::S3ObjectStorageMode> {
         &self.s3_object_storage_mode
     }
     /// <p>The base64-encoded contents of the layer archive. Amazon Web Services SDK and Amazon Web Services CLI clients handle the encoding for you.</p>
@@ -167,8 +167,8 @@
         &self.zip_file
     }
     /// Consumes the builder and constructs a [`LayerVersionContentInput`](crate::types::LayerVersionContentInput).
-    pub fn build(self) -> crate::types::LayerVersionContentInput {
-        crate::types::LayerVersionContentInput {
+    pub fn build(self) -> super::types::LayerVersionContentInput {
+        super::types::LayerVersionContentInput {
             s3_bucket: self.s3_bucket,
             s3_key: self.s3_key,
             s3_object_version: self.s3_object_version,
