--- original
+++ modified
@@ -66,7 +66,7 @@
     VpcSubnet,
     /// `Unknown` contains new variants that have been added since this code was generated.
     #[deprecated(note = "Don't directly match on `Unknown`. See the docs on this enum for the correct way to handle unknown variants.")]
-    Unknown(crate::primitives::sealed_enum_unknown::UnknownVariantValue),
+    Unknown(super::super::primitives::sealed_enum_unknown::UnknownVariantValue),
 }
 impl ::std::convert::From<&str> for SourceAccessType {
     fn from(s: &str) -> Self {
@@ -79,7 +79,7 @@
             "VIRTUAL_HOST" => SourceAccessType::VirtualHost,
             "VPC_SECURITY_GROUP" => SourceAccessType::VpcSecurityGroup,
             "VPC_SUBNET" => SourceAccessType::VpcSubnet,
-            other => SourceAccessType::Unknown(crate::primitives::sealed_enum_unknown::UnknownVariantValue(other.to_owned())),
+            other => SourceAccessType::Unknown(super::super::primitives::sealed_enum_unknown::UnknownVariantValue(other.to_owned())),
         }
     }
 }
@@ -128,10 +128,10 @@
     /// Parses the enum value while disallowing unknown variants.
     ///
     /// Unknown variants will result in an error.
-    pub fn try_parse(value: &str) -> ::std::result::Result<Self, crate::error::UnknownVariantError> {
+    pub fn try_parse(value: &str) -> ::std::result::Result<Self, super::super::error::UnknownVariantError> {
         match Self::from(value) {
             #[allow(deprecated)]
-            Self::Unknown(_) => ::std::result::Result::Err(crate::error::UnknownVariantError::new(value)),
+            Self::Unknown(_) => ::std::result::Result::Err(super::super::error::UnknownVariantError::new(value)),
             known => Ok(known),
         }
     }
