--- original
+++ modified
@@ -5,24 +5,24 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct StepFailedDetails {
     /// <p>Details about the step failure.</p>
-    pub error: ::std::option::Option<crate::types::EventError>,
+    pub error: ::std::option::Option<super::super::types::EventError>,
     /// <p>Information about retry attempts for this step operation.</p>
-    pub retry_details: ::std::option::Option<crate::types::RetryDetails>,
+    pub retry_details: ::std::option::Option<super::super::types::RetryDetails>,
 }
 impl StepFailedDetails {
     /// <p>Details about the step failure.</p>
-    pub fn error(&self) -> ::std::option::Option<&crate::types::EventError> {
+    pub fn error(&self) -> ::std::option::Option<&super::super::types::EventError> {
         self.error.as_ref()
     }
     /// <p>Information about retry attempts for this step operation.</p>
-    pub fn retry_details(&self) -> ::std::option::Option<&crate::types::RetryDetails> {
+    pub fn retry_details(&self) -> ::std::option::Option<&super::super::types::RetryDetails> {
         self.retry_details.as_ref()
     }
 }
 impl StepFailedDetails {
     /// Creates a new builder-style object to manufacture [`StepFailedDetails`](crate::types::StepFailedDetails).
-    pub fn builder() -> crate::types::builders::StepFailedDetailsBuilder {
-        crate::types::builders::StepFailedDetailsBuilder::default()
+    pub fn builder() -> super::super::types::builders::StepFailedDetailsBuilder {
+        super::super::types::builders::StepFailedDetailsBuilder::default()
     }
 }

@@ -30,43 +30,43 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct StepFailedDetailsBuilder {
-    pub(crate) error: ::std::option::Option<crate::types::EventError>,
-    pub(crate) retry_details: ::std::option::Option<crate::types::RetryDetails>,
+    pub(crate) error: ::std::option::Option<super::super::types::EventError>,
+    pub(crate) retry_details: ::std::option::Option<super::super::types::RetryDetails>,
 }
 impl StepFailedDetailsBuilder {
     /// <p>Details about the step failure.</p>
     /// This field is required.
-    pub fn error(mut self, input: crate::types::EventError) -> Self {
+    pub fn error(mut self, input: super::super::types::EventError) -> Self {
         self.error = ::std::option::Option::Some(input);
         self
     }
     /// <p>Details about the step failure.</p>
-    pub fn set_error(mut self, input: ::std::option::Option<crate::types::EventError>) -> Self {
+    pub fn set_error(mut self, input: ::std::option::Option<super::super::types::EventError>) -> Self {
         self.error = input;
         self
     }
     /// <p>Details about the step failure.</p>
-    pub fn get_error(&self) -> &::std::option::Option<crate::types::EventError> {
+    pub fn get_error(&self) -> &::std::option::Option<super::super::types::EventError> {
         &self.error
     }
     /// <p>Information about retry attempts for this step operation.</p>
     /// This field is required.
-    pub fn retry_details(mut self, input: crate::types::RetryDetails) -> Self {
+    pub fn retry_details(mut self, input: super::super::types::RetryDetails) -> Self {
         self.retry_details = ::std::option::Option::Some(input);
         self
     }
     /// <p>Information about retry attempts for this step operation.</p>
-    pub fn set_retry_details(mut self, input: ::std::option::Option<crate::types::RetryDetails>) -> Self {
+    pub fn set_retry_details(mut self, input: ::std::option::Option<super::super::types::RetryDetails>) -> Self {
         self.retry_details = input;
         self
     }
     /// <p>Information about retry attempts for this step operation.</p>
-    pub fn get_retry_details(&self) -> &::std::option::Option<crate::types::RetryDetails> {
+    pub fn get_retry_details(&self) -> &::std::option::Option<super::super::types::RetryDetails> {
         &self.retry_details
     }
     /// Consumes the builder and constructs a [`StepFailedDetails`](crate::types::StepFailedDetails).
-    pub fn build(self) -> crate::types::StepFailedDetails {
-        crate::types::StepFailedDetails {
+    pub fn build(self) -> super::super::types::StepFailedDetails {
+        super::super::types::StepFailedDetails {
             error: self.error,
             retry_details: self.retry_details,
         }
