--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl TraceHeader {
     /// Creates a new builder-style object to manufacture [`TraceHeader`](crate::types::TraceHeader).
-    pub fn builder() -> crate::types::builders::TraceHeaderBuilder {
-        crate::types::builders::TraceHeaderBuilder::default()
+    pub fn builder() -> super::types::builders::TraceHeaderBuilder {
+        super::types::builders::TraceHeaderBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.x_amzn_trace_id
     }
     /// Consumes the builder and constructs a [`TraceHeader`](crate::types::TraceHeader).
-    pub fn build(self) -> crate::types::TraceHeader {
-        crate::types::TraceHeader {
+    pub fn build(self) -> super::types::TraceHeader {
+        super::types::TraceHeader {
             x_amzn_trace_id: self.x_amzn_trace_id,
         }
     }
