--- original
+++ modified
@@ -31,8 +31,8 @@
 }
 impl VpcConfig {
     /// Creates a new builder-style object to manufacture [`VpcConfig`](crate::types::VpcConfig).
-    pub fn builder() -> crate::types::builders::VpcConfigBuilder {
-        crate::types::builders::VpcConfigBuilder::default()
+    pub fn builder() -> super::types::builders::VpcConfigBuilder {
+        super::types::builders::VpcConfigBuilder::default()
     }
 }

@@ -100,8 +100,8 @@
         &self.ipv6_allowed_for_dual_stack
     }
     /// Consumes the builder and constructs a [`VpcConfig`](crate::types::VpcConfig).
-    pub fn build(self) -> crate::types::VpcConfig {
-        crate::types::VpcConfig {
+    pub fn build(self) -> super::types::VpcConfig {
+        super::types::VpcConfig {
             subnet_ids: self.subnet_ids,
             security_group_ids: self.security_group_ids,
             ipv6_allowed_for_dual_stack: self.ipv6_allowed_for_dual_stack,
