--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl WaitSucceededDetails {
     /// Creates a new builder-style object to manufacture [`WaitSucceededDetails`](crate::types::WaitSucceededDetails).
-    pub fn builder() -> crate::types::builders::WaitSucceededDetailsBuilder {
-        crate::types::builders::WaitSucceededDetailsBuilder::default()
+    pub fn builder() -> super::super::types::builders::WaitSucceededDetailsBuilder {
+        super::super::types::builders::WaitSucceededDetailsBuilder::default()
     }
 }

@@ -42,7 +42,7 @@
         &self.duration
     }
     /// Consumes the builder and constructs a [`WaitSucceededDetails`](crate::types::WaitSucceededDetails).
-    pub fn build(self) -> crate::types::WaitSucceededDetails {
-        crate::types::WaitSucceededDetails { duration: self.duration }
+    pub fn build(self) -> super::super::types::WaitSucceededDetails {
+        super::super::types::WaitSucceededDetails { duration: self.duration }
     }
 }
