--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl WaitDetails {
     /// Creates a new builder-style object to manufacture [`WaitDetails`](crate::types::WaitDetails).
-    pub fn builder() -> crate::types::builders::WaitDetailsBuilder {
-        crate::types::builders::WaitDetailsBuilder::default()
+    pub fn builder() -> super::super::types::builders::WaitDetailsBuilder {
+        super::super::types::builders::WaitDetailsBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.scheduled_end_timestamp
     }
     /// Consumes the builder and constructs a [`WaitDetails`](crate::types::WaitDetails).
-    pub fn build(self) -> crate::types::WaitDetails {
-        crate::types::WaitDetails {
+    pub fn build(self) -> super::super::types::WaitDetails {
+        super::super::types::WaitDetails {
             scheduled_end_timestamp: self.scheduled_end_timestamp,
         }
     }
