--- original
+++ modified
@@ -14,7 +14,7 @@
     /// <p>A description of the package group.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>One or more tag key-value pairs for the package group.</p>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>,
 }
 impl CreatePackageGroupInput {
     /// <p>The name of the domain in which you want to create a package group.</p>
@@ -40,14 +40,14 @@
     /// <p>One or more tag key-value pairs for the package group.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::super::super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
 }
 impl CreatePackageGroupInput {
     /// Creates a new builder-style object to manufacture [`CreatePackageGroupInput`](crate::operation::create_package_group::CreatePackageGroupInput).
-    pub fn builder() -> crate::operation::create_package_group::builders::CreatePackageGroupInputBuilder {
-        crate::operation::create_package_group::builders::CreatePackageGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::create_package_group::builders::CreatePackageGroupInputBuilder {
+        super::super::super::operation::create_package_group::builders::CreatePackageGroupInputBuilder::default()
     }
 }

@@ -60,7 +60,7 @@
     pub(crate) package_group: ::std::option::Option<::std::string::String>,
     pub(crate) contact_info: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>,
 }
 impl CreatePackageGroupInputBuilder {
     /// <p>The name of the domain in which you want to create a package group.</p>
@@ -140,7 +140,7 @@
     /// To override the contents of this collection use [`set_tags`](Self::set_tags).
     ///
     /// <p>One or more tag key-value pairs for the package group.</p>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::super::super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -147,20 +147,20 @@
         self
     }
     /// <p>One or more tag key-value pairs for the package group.</p>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
     /// <p>One or more tag key-value pairs for the package group.</p>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>> {
         &self.tags
     }
     /// Consumes the builder and constructs a [`CreatePackageGroupInput`](crate::operation::create_package_group::CreatePackageGroupInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_package_group::CreatePackageGroupInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::create_package_group::CreatePackageGroupInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::create_package_group::CreatePackageGroupInput {
+        ::std::result::Result::Ok(super::super::super::operation::create_package_group::CreatePackageGroupInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             package_group: self.package_group,
