--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl AddPermissionInput {
     /// Creates a new builder-style object to manufacture [`AddPermissionInput`](crate::operation::add_permission::AddPermissionInput).
-    pub fn builder() -> crate::operation::add_permission::builders::AddPermissionInputBuilder {
-        crate::operation::add_permission::builders::AddPermissionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::add_permission::builders::AddPermissionInputBuilder {
+        super::super::super::operation::add_permission::builders::AddPermissionInputBuilder::default()
     }
 }

@@ -129,8 +129,8 @@
     /// Consumes the builder and constructs a [`AddPermissionInput`](crate::operation::add_permission::AddPermissionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::add_permission::AddPermissionInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::add_permission::AddPermissionInput {
+    ) -> ::std::result::Result<super::super::super::operation::add_permission::AddPermissionInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::add_permission::AddPermissionInput {
             topic_arn: self.topic_arn,
             label: self.label,
             aws_account_id: self.aws_account_id,
