--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetMultiRegionEndpointInput {
     /// Creates a new builder-style object to manufacture [`GetMultiRegionEndpointInput`](crate::operation::get_multi_region_endpoint::GetMultiRegionEndpointInput).
-    pub fn builder() -> crate::operation::get_multi_region_endpoint::builders::GetMultiRegionEndpointInputBuilder {
-        crate::operation::get_multi_region_endpoint::builders::GetMultiRegionEndpointInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_multi_region_endpoint::builders::GetMultiRegionEndpointInputBuilder {
+        super::super::super::operation::get_multi_region_endpoint::builders::GetMultiRegionEndpointInputBuilder::default()
     }
 }

@@ -46,10 +46,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_multi_region_endpoint::GetMultiRegionEndpointInput,
+        super::super::super::operation::get_multi_region_endpoint::GetMultiRegionEndpointInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_multi_region_endpoint::GetMultiRegionEndpointInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_multi_region_endpoint::GetMultiRegionEndpointInput {
             endpoint_name: self.endpoint_name,
         })
     }
