--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The invocation's idempotency token.</p>
     pub client_request_token: ::std::option::Option<::std::string::String>,
     /// <p>The invocation's status.</p>
-    pub status: ::std::option::Option<crate::types::AsyncInvokeStatus>,
+    pub status: ::std::option::Option<super::super::types::AsyncInvokeStatus>,
     /// <p>An error message.</p>
     pub failure_message: ::std::option::Option<::std::string::String>,
     /// <p>When the invocation was submitted.</p>
@@ -21,7 +21,7 @@
     /// <p>When the invocation ended.</p>
     pub end_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The invocation's output data settings.</p>
-    pub output_data_config: ::std::option::Option<crate::types::AsyncInvokeOutputDataConfig>,
+    pub output_data_config: ::std::option::Option<super::super::types::AsyncInvokeOutputDataConfig>,
 }
 impl AsyncInvokeSummary {
     /// <p>The invocation's ARN.</p>
@@ -39,7 +39,7 @@
         self.client_request_token.as_deref()
     }
     /// <p>The invocation's status.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::AsyncInvokeStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::AsyncInvokeStatus> {
         self.status.as_ref()
     }
     /// <p>An error message.</p>
@@ -59,7 +59,7 @@
         self.end_time.as_ref()
     }
     /// <p>The invocation's output data settings.</p>
-    pub fn output_data_config(&self) -> ::std::option::Option<&crate::types::AsyncInvokeOutputDataConfig> {
+    pub fn output_data_config(&self) -> ::std::option::Option<&super::super::types::AsyncInvokeOutputDataConfig> {
         self.output_data_config.as_ref()
     }
 }
@@ -80,8 +80,8 @@
 }
 impl AsyncInvokeSummary {
     /// Creates a new builder-style object to manufacture [`AsyncInvokeSummary`](crate::types::AsyncInvokeSummary).
-    pub fn builder() -> crate::types::builders::AsyncInvokeSummaryBuilder {
-        crate::types::builders::AsyncInvokeSummaryBuilder::default()
+    pub fn builder() -> super::super::types::builders::AsyncInvokeSummaryBuilder {
+        super::super::types::builders::AsyncInvokeSummaryBuilder::default()
     }
 }

@@ -92,12 +92,12 @@
     pub(crate) invocation_arn: ::std::option::Option<::std::string::String>,
     pub(crate) model_arn: ::std::option::Option<::std::string::String>,
     pub(crate) client_request_token: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::AsyncInvokeStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::AsyncInvokeStatus>,
     pub(crate) failure_message: ::std::option::Option<::std::string::String>,
     pub(crate) submit_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) last_modified_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) end_time: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) output_data_config: ::std::option::Option<crate::types::AsyncInvokeOutputDataConfig>,
+    pub(crate) output_data_config: ::std::option::Option<super::super::types::AsyncInvokeOutputDataConfig>,
 }
 impl AsyncInvokeSummaryBuilder {
     /// <p>The invocation's ARN.</p>
@@ -145,17 +145,17 @@
         &self.client_request_token
     }
     /// <p>The invocation's status.</p>
-    pub fn status(mut self, input: crate::types::AsyncInvokeStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::AsyncInvokeStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The invocation's status.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::AsyncInvokeStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::AsyncInvokeStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>The invocation's status.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::AsyncInvokeStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::AsyncInvokeStatus> {
         &self.status
     }
     /// <p>An error message.</p>
@@ -217,17 +217,17 @@
     }
     /// <p>The invocation's output data settings.</p>
     /// This field is required.
-    pub fn output_data_config(mut self, input: crate::types::AsyncInvokeOutputDataConfig) -> Self {
+    pub fn output_data_config(mut self, input: super::super::types::AsyncInvokeOutputDataConfig) -> Self {
         self.output_data_config = ::std::option::Option::Some(input);
         self
     }
     /// <p>The invocation's output data settings.</p>
-    pub fn set_output_data_config(mut self, input: ::std::option::Option<crate::types::AsyncInvokeOutputDataConfig>) -> Self {
+    pub fn set_output_data_config(mut self, input: ::std::option::Option<super::super::types::AsyncInvokeOutputDataConfig>) -> Self {
         self.output_data_config = input;
         self
     }
     /// <p>The invocation's output data settings.</p>
-    pub fn get_output_data_config(&self) -> &::std::option::Option<crate::types::AsyncInvokeOutputDataConfig> {
+    pub fn get_output_data_config(&self) -> &::std::option::Option<super::super::types::AsyncInvokeOutputDataConfig> {
         &self.output_data_config
     }
     /// Consumes the builder and constructs a [`AsyncInvokeSummary`](crate::types::AsyncInvokeSummary).
@@ -235,8 +235,8 @@
     /// - [`invocation_arn`](crate::types::builders::AsyncInvokeSummaryBuilder::invocation_arn)
     /// - [`model_arn`](crate::types::builders::AsyncInvokeSummaryBuilder::model_arn)
     /// - [`submit_time`](crate::types::builders::AsyncInvokeSummaryBuilder::submit_time)
-    pub fn build(self) -> ::std::result::Result<crate::types::AsyncInvokeSummary, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::AsyncInvokeSummary {
+    pub fn build(self) -> ::std::result::Result<super::super::types::AsyncInvokeSummary, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::AsyncInvokeSummary {
             invocation_arn: self.invocation_arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "invocation_arn",
