--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl CitationsConfig {
     /// Creates a new builder-style object to manufacture [`CitationsConfig`](crate::types::CitationsConfig).
-    pub fn builder() -> crate::types::builders::CitationsConfigBuilder {
-        crate::types::builders::CitationsConfigBuilder::default()
+    pub fn builder() -> super::types::builders::CitationsConfigBuilder {
+        super::types::builders::CitationsConfigBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`CitationsConfig`](crate::types::CitationsConfig).
     /// This method will fail if any of the following fields are not set:
     /// - [`enabled`](crate::types::builders::CitationsConfigBuilder::enabled)
-    pub fn build(self) -> ::std::result::Result<crate::types::CitationsConfig, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::CitationsConfig {
+    pub fn build(self) -> ::std::result::Result<super::types::CitationsConfig, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::CitationsConfig {
             enabled: self.enabled.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "enabled",
