--- original
+++ modified
@@ -5,17 +5,17 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq)]
 pub enum ContentBlockDelta {
     /// <p>Incremental citation information that is streamed as part of the response generation process.</p>
-    Citation(crate::types::CitationsDelta),
+    Citation(super::types::CitationsDelta),
     /// <p>A streaming delta event containing incremental image data.</p>
-    Image(crate::types::ImageBlockDelta),
+    Image(super::types::ImageBlockDelta),
     /// <p>Contains content regarding the reasoning that is carried out by the model. Reasoning refers to a Chain of Thought (CoT) that the model generates to enhance the accuracy of its final response.</p>
-    ReasoningContent(crate::types::ReasoningContentBlockDelta),
+    ReasoningContent(super::types::ReasoningContentBlockDelta),
     /// <p>The content text.</p>
     Text(::std::string::String),
     /// <p>An incremental update that contains the results from a tool call.</p>
-    ToolResult(::std::vec::Vec<crate::types::ToolResultBlockDelta>),
+    ToolResult(::std::vec::Vec<super::types::ToolResultBlockDelta>),
     /// <p>Information about a tool that the model is requesting to use.</p>
-    ToolUse(crate::types::ToolUseBlockDelta),
+    ToolUse(super::types::ToolUseBlockDelta),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -29,7 +29,7 @@
 impl ContentBlockDelta {
     /// Tries to convert the enum instance into [`Citation`](crate::types::ContentBlockDelta::Citation), extracting the inner [`CitationsDelta`](crate::types::CitationsDelta).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_citation(&self) -> ::std::result::Result<&crate::types::CitationsDelta, &Self> {
+    pub fn as_citation(&self) -> ::std::result::Result<&super::types::CitationsDelta, &Self> {
         if let ContentBlockDelta::Citation(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -42,7 +42,7 @@
     }
     /// Tries to convert the enum instance into [`Image`](crate::types::ContentBlockDelta::Image), extracting the inner [`ImageBlockDelta`](crate::types::ImageBlockDelta).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_image(&self) -> ::std::result::Result<&crate::types::ImageBlockDelta, &Self> {
+    pub fn as_image(&self) -> ::std::result::Result<&super::types::ImageBlockDelta, &Self> {
         if let ContentBlockDelta::Image(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -55,7 +55,7 @@
     }
     /// Tries to convert the enum instance into [`ReasoningContent`](crate::types::ContentBlockDelta::ReasoningContent), extracting the inner [`ReasoningContentBlockDelta`](crate::types::ReasoningContentBlockDelta).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_reasoning_content(&self) -> ::std::result::Result<&crate::types::ReasoningContentBlockDelta, &Self> {
+    pub fn as_reasoning_content(&self) -> ::std::result::Result<&super::types::ReasoningContentBlockDelta, &Self> {
         if let ContentBlockDelta::ReasoningContent(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -81,7 +81,7 @@
     }
     /// Tries to convert the enum instance into [`ToolResult`](crate::types::ContentBlockDelta::ToolResult), extracting the inner [`Vec`](::std::vec::Vec).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_tool_result(&self) -> ::std::result::Result<&::std::vec::Vec<crate::types::ToolResultBlockDelta>, &Self> {
+    pub fn as_tool_result(&self) -> ::std::result::Result<&::std::vec::Vec<super::types::ToolResultBlockDelta>, &Self> {
         if let ContentBlockDelta::ToolResult(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -94,7 +94,7 @@
     }
     /// Tries to convert the enum instance into [`ToolUse`](crate::types::ContentBlockDelta::ToolUse), extracting the inner [`ToolUseBlockDelta`](crate::types::ToolUseBlockDelta).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_tool_use(&self) -> ::std::result::Result<&crate::types::ToolUseBlockDelta, &Self> {
+    pub fn as_tool_use(&self) -> ::std::result::Result<&super::types::ToolUseBlockDelta, &Self> {
         if let ContentBlockDelta::ToolUse(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
