--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl ContentBlockStopEvent {
     /// Creates a new builder-style object to manufacture [`ContentBlockStopEvent`](crate::types::ContentBlockStopEvent).
-    pub fn builder() -> crate::types::builders::ContentBlockStopEventBuilder {
-        crate::types::builders::ContentBlockStopEventBuilder::default()
+    pub fn builder() -> super::types::builders::ContentBlockStopEventBuilder {
+        super::types::builders::ContentBlockStopEventBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`ContentBlockStopEvent`](crate::types::ContentBlockStopEvent).
     /// This method will fail if any of the following fields are not set:
     /// - [`content_block_index`](crate::types::builders::ContentBlockStopEventBuilder::content_block_index)
-    pub fn build(self) -> ::std::result::Result<crate::types::ContentBlockStopEvent, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ContentBlockStopEvent {
+    pub fn build(self) -> ::std::result::Result<super::types::ContentBlockStopEvent, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ContentBlockStopEvent {
             content_block_index: self.content_block_index.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "content_block_index",
