--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl ConverseMetrics {
     /// Creates a new builder-style object to manufacture [`ConverseMetrics`](crate::types::ConverseMetrics).
-    pub fn builder() -> crate::types::builders::ConverseMetricsBuilder {
-        crate::types::builders::ConverseMetricsBuilder::default()
+    pub fn builder() -> super::types::builders::ConverseMetricsBuilder {
+        super::types::builders::ConverseMetricsBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`ConverseMetrics`](crate::types::ConverseMetrics).
     /// This method will fail if any of the following fields are not set:
     /// - [`latency_ms`](crate::types::builders::ConverseMetricsBuilder::latency_ms)
-    pub fn build(self) -> ::std::result::Result<crate::types::ConverseMetrics, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ConverseMetrics {
+    pub fn build(self) -> ::std::result::Result<super::types::ConverseMetrics, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ConverseMetrics {
             latency_ms: self.latency_ms.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "latency_ms",
