--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl DocumentCharLocation {
     /// Creates a new builder-style object to manufacture [`DocumentCharLocation`](crate::types::DocumentCharLocation).
-    pub fn builder() -> crate::types::builders::DocumentCharLocationBuilder {
-        crate::types::builders::DocumentCharLocationBuilder::default()
+    pub fn builder() -> super::super::types::builders::DocumentCharLocationBuilder {
+        super::super::types::builders::DocumentCharLocationBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.end
     }
     /// Consumes the builder and constructs a [`DocumentCharLocation`](crate::types::DocumentCharLocation).
-    pub fn build(self) -> crate::types::DocumentCharLocation {
-        crate::types::DocumentCharLocation {
+    pub fn build(self) -> super::super::types::DocumentCharLocation {
+        super::super::types::DocumentCharLocation {
             document_index: self.document_index,
             start: self.start,
             end: self.end,
