--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl DocumentChunkLocation {
     /// Creates a new builder-style object to manufacture [`DocumentChunkLocation`](crate::types::DocumentChunkLocation).
-    pub fn builder() -> crate::types::builders::DocumentChunkLocationBuilder {
-        crate::types::builders::DocumentChunkLocationBuilder::default()
+    pub fn builder() -> super::super::types::builders::DocumentChunkLocationBuilder {
+        super::super::types::builders::DocumentChunkLocationBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.end
     }
     /// Consumes the builder and constructs a [`DocumentChunkLocation`](crate::types::DocumentChunkLocation).
-    pub fn build(self) -> crate::types::DocumentChunkLocation {
-        crate::types::DocumentChunkLocation {
+    pub fn build(self) -> super::super::types::DocumentChunkLocation {
+        super::super::types::DocumentChunkLocation {
             document_index: self.document_index,
             start: self.start,
             end: self.end,
