--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl DocumentPageLocation {
     /// Creates a new builder-style object to manufacture [`DocumentPageLocation`](crate::types::DocumentPageLocation).
-    pub fn builder() -> crate::types::builders::DocumentPageLocationBuilder {
-        crate::types::builders::DocumentPageLocationBuilder::default()
+    pub fn builder() -> super::types::builders::DocumentPageLocationBuilder {
+        super::types::builders::DocumentPageLocationBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.end
     }
     /// Consumes the builder and constructs a [`DocumentPageLocation`](crate::types::DocumentPageLocation).
-    pub fn build(self) -> crate::types::DocumentPageLocation {
-        crate::types::DocumentPageLocation {
+    pub fn build(self) -> super::types::DocumentPageLocation {
+        super::types::DocumentPageLocation {
             document_index: self.document_index,
             start: self.start,
             end: self.end,
