--- original
+++ modified
@@ -5,17 +5,17 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq)]
 pub struct GuardrailImageBlock {
     /// <p>The format details for the file type of the image blocked by the guardrail.</p>
-    pub format: crate::types::GuardrailImageFormat,
+    pub format: super::super::types::GuardrailImageFormat,
     /// <p>The image source (image bytes) details of the image blocked by the guardrail.</p>
-    pub source: ::std::option::Option<crate::types::GuardrailImageSource>,
+    pub source: ::std::option::Option<super::super::types::GuardrailImageSource>,
 }
 impl GuardrailImageBlock {
     /// <p>The format details for the file type of the image blocked by the guardrail.</p>
-    pub fn format(&self) -> &crate::types::GuardrailImageFormat {
+    pub fn format(&self) -> &super::super::types::GuardrailImageFormat {
         &self.format
     }
     /// <p>The image source (image bytes) details of the image blocked by the guardrail.</p>
-    pub fn source(&self) -> ::std::option::Option<&crate::types::GuardrailImageSource> {
+    pub fn source(&self) -> ::std::option::Option<&super::super::types::GuardrailImageSource> {
         self.source.as_ref()
     }
 }
@@ -29,8 +29,8 @@
 }
 impl GuardrailImageBlock {
     /// Creates a new builder-style object to manufacture [`GuardrailImageBlock`](crate::types::GuardrailImageBlock).
-    pub fn builder() -> crate::types::builders::GuardrailImageBlockBuilder {
-        crate::types::builders::GuardrailImageBlockBuilder::default()
+    pub fn builder() -> super::super::types::builders::GuardrailImageBlockBuilder {
+        super::super::types::builders::GuardrailImageBlockBuilder::default()
     }
 }

@@ -38,45 +38,45 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default)]
 #[non_exhaustive]
 pub struct GuardrailImageBlockBuilder {
-    pub(crate) format: ::std::option::Option<crate::types::GuardrailImageFormat>,
-    pub(crate) source: ::std::option::Option<crate::types::GuardrailImageSource>,
+    pub(crate) format: ::std::option::Option<super::super::types::GuardrailImageFormat>,
+    pub(crate) source: ::std::option::Option<super::super::types::GuardrailImageSource>,
 }
 impl GuardrailImageBlockBuilder {
     /// <p>The format details for the file type of the image blocked by the guardrail.</p>
     /// This field is required.
-    pub fn format(mut self, input: crate::types::GuardrailImageFormat) -> Self {
+    pub fn format(mut self, input: super::super::types::GuardrailImageFormat) -> Self {
         self.format = ::std::option::Option::Some(input);
         self
     }
     /// <p>The format details for the file type of the image blocked by the guardrail.</p>
-    pub fn set_format(mut self, input: ::std::option::Option<crate::types::GuardrailImageFormat>) -> Self {
+    pub fn set_format(mut self, input: ::std::option::Option<super::super::types::GuardrailImageFormat>) -> Self {
         self.format = input;
         self
     }
     /// <p>The format details for the file type of the image blocked by the guardrail.</p>
-    pub fn get_format(&self) -> &::std::option::Option<crate::types::GuardrailImageFormat> {
+    pub fn get_format(&self) -> &::std::option::Option<super::super::types::GuardrailImageFormat> {
         &self.format
     }
     /// <p>The image source (image bytes) details of the image blocked by the guardrail.</p>
     /// This field is required.
-    pub fn source(mut self, input: crate::types::GuardrailImageSource) -> Self {
+    pub fn source(mut self, input: super::super::types::GuardrailImageSource) -> Self {
         self.source = ::std::option::Option::Some(input);
         self
     }
     /// <p>The image source (image bytes) details of the image blocked by the guardrail.</p>
-    pub fn set_source(mut self, input: ::std::option::Option<crate::types::GuardrailImageSource>) -> Self {
+    pub fn set_source(mut self, input: ::std::option::Option<super::super::types::GuardrailImageSource>) -> Self {
         self.source = input;
         self
     }
     /// <p>The image source (image bytes) details of the image blocked by the guardrail.</p>
-    pub fn get_source(&self) -> &::std::option::Option<crate::types::GuardrailImageSource> {
+    pub fn get_source(&self) -> &::std::option::Option<super::super::types::GuardrailImageSource> {
         &self.source
     }
     /// Consumes the builder and constructs a [`GuardrailImageBlock`](crate::types::GuardrailImageBlock).
     /// This method will fail if any of the following fields are not set:
     /// - [`format`](crate::types::builders::GuardrailImageBlockBuilder::format)
-    pub fn build(self) -> ::std::result::Result<crate::types::GuardrailImageBlock, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::GuardrailImageBlock {
+    pub fn build(self) -> ::std::result::Result<super::super::types::GuardrailImageBlock, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::GuardrailImageBlock {
             format: self.format.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "format",
