--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct GuardrailWordPolicyAssessment {
     /// <p>Custom words in the assessment.</p>
-    pub custom_words: ::std::vec::Vec<crate::types::GuardrailCustomWord>,
+    pub custom_words: ::std::vec::Vec<super::super::types::GuardrailCustomWord>,
     /// <p>Managed word lists in the assessment.</p>
-    pub managed_word_lists: ::std::vec::Vec<crate::types::GuardrailManagedWord>,
+    pub managed_word_lists: ::std::vec::Vec<super::super::types::GuardrailManagedWord>,
 }
 impl GuardrailWordPolicyAssessment {
     /// <p>Custom words in the assessment.</p>
-    pub fn custom_words(&self) -> &[crate::types::GuardrailCustomWord] {
+    pub fn custom_words(&self) -> &[super::super::types::GuardrailCustomWord] {
         use std::ops::Deref;
         self.custom_words.deref()
     }
     /// <p>Managed word lists in the assessment.</p>
-    pub fn managed_word_lists(&self) -> &[crate::types::GuardrailManagedWord] {
+    pub fn managed_word_lists(&self) -> &[super::super::types::GuardrailManagedWord] {
         use std::ops::Deref;
         self.managed_word_lists.deref()
     }
@@ -23,8 +23,8 @@
 }
 impl GuardrailWordPolicyAssessment {
     /// Creates a new builder-style object to manufacture [`GuardrailWordPolicyAssessment`](crate::types::GuardrailWordPolicyAssessment).
-    pub fn builder() -> crate::types::builders::GuardrailWordPolicyAssessmentBuilder {
-        crate::types::builders::GuardrailWordPolicyAssessmentBuilder::default()
+    pub fn builder() -> super::super::types::builders::GuardrailWordPolicyAssessmentBuilder {
+        super::super::types::builders::GuardrailWordPolicyAssessmentBuilder::default()
     }
 }

@@ -32,8 +32,8 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct GuardrailWordPolicyAssessmentBuilder {
-    pub(crate) custom_words: ::std::option::Option<::std::vec::Vec<crate::types::GuardrailCustomWord>>,
-    pub(crate) managed_word_lists: ::std::option::Option<::std::vec::Vec<crate::types::GuardrailManagedWord>>,
+    pub(crate) custom_words: ::std::option::Option<::std::vec::Vec<super::super::types::GuardrailCustomWord>>,
+    pub(crate) managed_word_lists: ::std::option::Option<::std::vec::Vec<super::super::types::GuardrailManagedWord>>,
 }
 impl GuardrailWordPolicyAssessmentBuilder {
     /// Appends an item to `custom_words`.
@@ -41,7 +41,7 @@
     /// To override the contents of this collection use [`set_custom_words`](Self::set_custom_words).
     ///
     /// <p>Custom words in the assessment.</p>
-    pub fn custom_words(mut self, input: crate::types::GuardrailCustomWord) -> Self {
+    pub fn custom_words(mut self, input: super::super::types::GuardrailCustomWord) -> Self {
         let mut v = self.custom_words.unwrap_or_default();
         v.push(input);
         self.custom_words = ::std::option::Option::Some(v);
@@ -48,12 +48,12 @@
         self
     }
     /// <p>Custom words in the assessment.</p>
-    pub fn set_custom_words(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::GuardrailCustomWord>>) -> Self {
+    pub fn set_custom_words(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::GuardrailCustomWord>>) -> Self {
         self.custom_words = input;
         self
     }
     /// <p>Custom words in the assessment.</p>
-    pub fn get_custom_words(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::GuardrailCustomWord>> {
+    pub fn get_custom_words(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::GuardrailCustomWord>> {
         &self.custom_words
     }
     /// Appends an item to `managed_word_lists`.
@@ -61,7 +61,7 @@
     /// To override the contents of this collection use [`set_managed_word_lists`](Self::set_managed_word_lists).
     ///
     /// <p>Managed word lists in the assessment.</p>
-    pub fn managed_word_lists(mut self, input: crate::types::GuardrailManagedWord) -> Self {
+    pub fn managed_word_lists(mut self, input: super::super::types::GuardrailManagedWord) -> Self {
         let mut v = self.managed_word_lists.unwrap_or_default();
         v.push(input);
         self.managed_word_lists = ::std::option::Option::Some(v);
@@ -68,12 +68,12 @@
         self
     }
     /// <p>Managed word lists in the assessment.</p>
-    pub fn set_managed_word_lists(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::GuardrailManagedWord>>) -> Self {
+    pub fn set_managed_word_lists(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::GuardrailManagedWord>>) -> Self {
         self.managed_word_lists = input;
         self
     }
     /// <p>Managed word lists in the assessment.</p>
-    pub fn get_managed_word_lists(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::GuardrailManagedWord>> {
+    pub fn get_managed_word_lists(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::GuardrailManagedWord>> {
         &self.managed_word_lists
     }
     /// Consumes the builder and constructs a [`GuardrailWordPolicyAssessment`](crate::types::GuardrailWordPolicyAssessment).
@@ -80,8 +80,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`custom_words`](crate::types::builders::GuardrailWordPolicyAssessmentBuilder::custom_words)
     /// - [`managed_word_lists`](crate::types::builders::GuardrailWordPolicyAssessmentBuilder::managed_word_lists)
-    pub fn build(self) -> ::std::result::Result<crate::types::GuardrailWordPolicyAssessment, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::GuardrailWordPolicyAssessment {
+    pub fn build(self) -> ::std::result::Result<super::super::types::GuardrailWordPolicyAssessment, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::GuardrailWordPolicyAssessment {
             custom_words: self.custom_words.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "custom_words",
