--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The regex query.</p>
     pub regex: ::std::option::Option<::std::string::String>,
     /// <p>The region filter action.</p>
-    pub action: crate::types::GuardrailSensitiveInformationPolicyAction,
+    pub action: super::super::types::GuardrailSensitiveInformationPolicyAction,
     /// <p>Indicates whether custom regex entities that breach the guardrail configuration are detected.</p>
     pub detected: ::std::option::Option<bool>,
 }
@@ -29,7 +29,7 @@
         self.regex.as_deref()
     }
     /// <p>The region filter action.</p>
-    pub fn action(&self) -> &crate::types::GuardrailSensitiveInformationPolicyAction {
+    pub fn action(&self) -> &super::super::types::GuardrailSensitiveInformationPolicyAction {
         &self.action
     }
     /// <p>Indicates whether custom regex entities that breach the guardrail configuration are detected.</p>
@@ -39,8 +39,8 @@
 }
 impl GuardrailRegexFilter {
     /// Creates a new builder-style object to manufacture [`GuardrailRegexFilter`](crate::types::GuardrailRegexFilter).
-    pub fn builder() -> crate::types::builders::GuardrailRegexFilterBuilder {
-        crate::types::builders::GuardrailRegexFilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::GuardrailRegexFilterBuilder {
+        super::super::types::builders::GuardrailRegexFilterBuilder::default()
     }
 }

@@ -51,7 +51,7 @@
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) r#match: ::std::option::Option<::std::string::String>,
     pub(crate) regex: ::std::option::Option<::std::string::String>,
-    pub(crate) action: ::std::option::Option<crate::types::GuardrailSensitiveInformationPolicyAction>,
+    pub(crate) action: ::std::option::Option<super::super::types::GuardrailSensitiveInformationPolicyAction>,
     pub(crate) detected: ::std::option::Option<bool>,
 }
 impl GuardrailRegexFilterBuilder {
@@ -99,17 +99,17 @@
     }
     /// <p>The region filter action.</p>
     /// This field is required.
-    pub fn action(mut self, input: crate::types::GuardrailSensitiveInformationPolicyAction) -> Self {
+    pub fn action(mut self, input: super::super::types::GuardrailSensitiveInformationPolicyAction) -> Self {
         self.action = ::std::option::Option::Some(input);
         self
     }
     /// <p>The region filter action.</p>
-    pub fn set_action(mut self, input: ::std::option::Option<crate::types::GuardrailSensitiveInformationPolicyAction>) -> Self {
+    pub fn set_action(mut self, input: ::std::option::Option<super::super::types::GuardrailSensitiveInformationPolicyAction>) -> Self {
         self.action = input;
         self
     }
     /// <p>The region filter action.</p>
-    pub fn get_action(&self) -> &::std::option::Option<crate::types::GuardrailSensitiveInformationPolicyAction> {
+    pub fn get_action(&self) -> &::std::option::Option<super::super::types::GuardrailSensitiveInformationPolicyAction> {
         &self.action
     }
     /// <p>Indicates whether custom regex entities that breach the guardrail configuration are detected.</p>
@@ -129,8 +129,8 @@
     /// Consumes the builder and constructs a [`GuardrailRegexFilter`](crate::types::GuardrailRegexFilter).
     /// This method will fail if any of the following fields are not set:
     /// - [`action`](crate::types::builders::GuardrailRegexFilterBuilder::action)
-    pub fn build(self) -> ::std::result::Result<crate::types::GuardrailRegexFilter, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::GuardrailRegexFilter {
+    pub fn build(self) -> ::std::result::Result<super::super::types::GuardrailRegexFilter, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::GuardrailRegexFilter {
             name: self.name,
             r#match: self.r#match,
             regex: self.regex,
