--- original
+++ modified
@@ -5,31 +5,31 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct GuardrailContentFilter {
     /// <p>The guardrail type.</p>
-    pub r#type: crate::types::GuardrailContentFilterType,
+    pub r#type: super::super::types::GuardrailContentFilterType,
     /// <p>The guardrail confidence.</p>
-    pub confidence: crate::types::GuardrailContentFilterConfidence,
+    pub confidence: super::super::types::GuardrailContentFilterConfidence,
     /// <p>The filter strength setting for the guardrail content filter.</p>
-    pub filter_strength: ::std::option::Option<crate::types::GuardrailContentFilterStrength>,
+    pub filter_strength: ::std::option::Option<super::super::types::GuardrailContentFilterStrength>,
     /// <p>The guardrail action.</p>
-    pub action: crate::types::GuardrailContentPolicyAction,
+    pub action: super::super::types::GuardrailContentPolicyAction,
     /// <p>Indicates whether content that breaches the guardrail configuration is detected.</p>
     pub detected: ::std::option::Option<bool>,
 }
 impl GuardrailContentFilter {
     /// <p>The guardrail type.</p>
-    pub fn r#type(&self) -> &crate::types::GuardrailContentFilterType {
+    pub fn r#type(&self) -> &super::super::types::GuardrailContentFilterType {
         &self.r#type
     }
     /// <p>The guardrail confidence.</p>
-    pub fn confidence(&self) -> &crate::types::GuardrailContentFilterConfidence {
+    pub fn confidence(&self) -> &super::super::types::GuardrailContentFilterConfidence {
         &self.confidence
     }
     /// <p>The filter strength setting for the guardrail content filter.</p>
-    pub fn filter_strength(&self) -> ::std::option::Option<&crate::types::GuardrailContentFilterStrength> {
+    pub fn filter_strength(&self) -> ::std::option::Option<&super::super::types::GuardrailContentFilterStrength> {
         self.filter_strength.as_ref()
     }
     /// <p>The guardrail action.</p>
-    pub fn action(&self) -> &crate::types::GuardrailContentPolicyAction {
+    pub fn action(&self) -> &super::super::types::GuardrailContentPolicyAction {
         &self.action
     }
     /// <p>Indicates whether content that breaches the guardrail configuration is detected.</p>
@@ -39,8 +39,8 @@
 }
 impl GuardrailContentFilter {
     /// Creates a new builder-style object to manufacture [`GuardrailContentFilter`](crate::types::GuardrailContentFilter).
-    pub fn builder() -> crate::types::builders::GuardrailContentFilterBuilder {
-        crate::types::builders::GuardrailContentFilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::GuardrailContentFilterBuilder {
+        super::super::types::builders::GuardrailContentFilterBuilder::default()
     }
 }

@@ -48,70 +48,70 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct GuardrailContentFilterBuilder {
-    pub(crate) r#type: ::std::option::Option<crate::types::GuardrailContentFilterType>,
-    pub(crate) confidence: ::std::option::Option<crate::types::GuardrailContentFilterConfidence>,
-    pub(crate) filter_strength: ::std::option::Option<crate::types::GuardrailContentFilterStrength>,
-    pub(crate) action: ::std::option::Option<crate::types::GuardrailContentPolicyAction>,
+    pub(crate) r#type: ::std::option::Option<super::super::types::GuardrailContentFilterType>,
+    pub(crate) confidence: ::std::option::Option<super::super::types::GuardrailContentFilterConfidence>,
+    pub(crate) filter_strength: ::std::option::Option<super::super::types::GuardrailContentFilterStrength>,
+    pub(crate) action: ::std::option::Option<super::super::types::GuardrailContentPolicyAction>,
     pub(crate) detected: ::std::option::Option<bool>,
 }
 impl GuardrailContentFilterBuilder {
     /// <p>The guardrail type.</p>
     /// This field is required.
-    pub fn r#type(mut self, input: crate::types::GuardrailContentFilterType) -> Self {
+    pub fn r#type(mut self, input: super::super::types::GuardrailContentFilterType) -> Self {
         self.r#type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The guardrail type.</p>
-    pub fn set_type(mut self, input: ::std::option::Option<crate::types::GuardrailContentFilterType>) -> Self {
+    pub fn set_type(mut self, input: ::std::option::Option<super::super::types::GuardrailContentFilterType>) -> Self {
         self.r#type = input;
         self
     }
     /// <p>The guardrail type.</p>
-    pub fn get_type(&self) -> &::std::option::Option<crate::types::GuardrailContentFilterType> {
+    pub fn get_type(&self) -> &::std::option::Option<super::super::types::GuardrailContentFilterType> {
         &self.r#type
     }
     /// <p>The guardrail confidence.</p>
     /// This field is required.
-    pub fn confidence(mut self, input: crate::types::GuardrailContentFilterConfidence) -> Self {
+    pub fn confidence(mut self, input: super::super::types::GuardrailContentFilterConfidence) -> Self {
         self.confidence = ::std::option::Option::Some(input);
         self
     }
     /// <p>The guardrail confidence.</p>
-    pub fn set_confidence(mut self, input: ::std::option::Option<crate::types::GuardrailContentFilterConfidence>) -> Self {
+    pub fn set_confidence(mut self, input: ::std::option::Option<super::super::types::GuardrailContentFilterConfidence>) -> Self {
         self.confidence = input;
         self
     }
     /// <p>The guardrail confidence.</p>
-    pub fn get_confidence(&self) -> &::std::option::Option<crate::types::GuardrailContentFilterConfidence> {
+    pub fn get_confidence(&self) -> &::std::option::Option<super::super::types::GuardrailContentFilterConfidence> {
         &self.confidence
     }
     /// <p>The filter strength setting for the guardrail content filter.</p>
-    pub fn filter_strength(mut self, input: crate::types::GuardrailContentFilterStrength) -> Self {
+    pub fn filter_strength(mut self, input: super::super::types::GuardrailContentFilterStrength) -> Self {
         self.filter_strength = ::std::option::Option::Some(input);
         self
     }
     /// <p>The filter strength setting for the guardrail content filter.</p>
-    pub fn set_filter_strength(mut self, input: ::std::option::Option<crate::types::GuardrailContentFilterStrength>) -> Self {
+    pub fn set_filter_strength(mut self, input: ::std::option::Option<super::super::types::GuardrailContentFilterStrength>) -> Self {
         self.filter_strength = input;
         self
     }
     /// <p>The filter strength setting for the guardrail content filter.</p>
-    pub fn get_filter_strength(&self) -> &::std::option::Option<crate::types::GuardrailContentFilterStrength> {
+    pub fn get_filter_strength(&self) -> &::std::option::Option<super::super::types::GuardrailContentFilterStrength> {
         &self.filter_strength
     }
     /// <p>The guardrail action.</p>
     /// This field is required.
-    pub fn action(mut self, input: crate::types::GuardrailContentPolicyAction) -> Self {
+    pub fn action(mut self, input: super::super::types::GuardrailContentPolicyAction) -> Self {
         self.action = ::std::option::Option::Some(input);
         self
     }
     /// <p>The guardrail action.</p>
-    pub fn set_action(mut self, input: ::std::option::Option<crate::types::GuardrailContentPolicyAction>) -> Self {
+    pub fn set_action(mut self, input: ::std::option::Option<super::super::types::GuardrailContentPolicyAction>) -> Self {
         self.action = input;
         self
     }
     /// <p>The guardrail action.</p>
-    pub fn get_action(&self) -> &::std::option::Option<crate::types::GuardrailContentPolicyAction> {
+    pub fn get_action(&self) -> &::std::option::Option<super::super::types::GuardrailContentPolicyAction> {
         &self.action
     }
     /// <p>Indicates whether content that breaches the guardrail configuration is detected.</p>
@@ -133,8 +133,8 @@
     /// - [`r#type`](crate::types::builders::GuardrailContentFilterBuilder::type)
     /// - [`confidence`](crate::types::builders::GuardrailContentFilterBuilder::confidence)
     /// - [`action`](crate::types::builders::GuardrailContentFilterBuilder::action)
-    pub fn build(self) -> ::std::result::Result<crate::types::GuardrailContentFilter, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::GuardrailContentFilter {
+    pub fn build(self) -> ::std::result::Result<super::super::types::GuardrailContentFilter, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::GuardrailContentFilter {
             r#type: self.r#type.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "r#type",
