--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl GuardrailAutomatedReasoningInputTextReference {
     /// Creates a new builder-style object to manufacture [`GuardrailAutomatedReasoningInputTextReference`](crate::types::GuardrailAutomatedReasoningInputTextReference).
-    pub fn builder() -> crate::types::builders::GuardrailAutomatedReasoningInputTextReferenceBuilder {
-        crate::types::builders::GuardrailAutomatedReasoningInputTextReferenceBuilder::default()
+    pub fn builder() -> super::super::types::builders::GuardrailAutomatedReasoningInputTextReferenceBuilder {
+        super::super::types::builders::GuardrailAutomatedReasoningInputTextReferenceBuilder::default()
     }
 }

@@ -49,8 +49,8 @@
         &self.text
     }
     /// Consumes the builder and constructs a [`GuardrailAutomatedReasoningInputTextReference`](crate::types::GuardrailAutomatedReasoningInputTextReference).
-    pub fn build(self) -> crate::types::GuardrailAutomatedReasoningInputTextReference {
-        crate::types::GuardrailAutomatedReasoningInputTextReference { text: self.text }
+    pub fn build(self) -> super::super::types::GuardrailAutomatedReasoningInputTextReference {
+        super::super::types::GuardrailAutomatedReasoningInputTextReference { text: self.text }
     }
 }
 impl ::std::fmt::Debug for GuardrailAutomatedReasoningInputTextReferenceBuilder {
