--- original
+++ modified
@@ -5,9 +5,9 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq)]
 pub enum GuardrailContentBlock {
     /// <p>Image within guardrail content block to be evaluated by the guardrail.</p>
-    Image(crate::types::GuardrailImageBlock),
+    Image(super::types::GuardrailImageBlock),
     /// <p>Text within content block to be evaluated by the guardrail.</p>
-    Text(crate::types::GuardrailTextBlock),
+    Text(super::types::GuardrailTextBlock),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -21,7 +21,7 @@
 impl GuardrailContentBlock {
     /// Tries to convert the enum instance into [`Image`](crate::types::GuardrailContentBlock::Image), extracting the inner [`GuardrailImageBlock`](crate::types::GuardrailImageBlock).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_image(&self) -> ::std::result::Result<&crate::types::GuardrailImageBlock, &Self> {
+    pub fn as_image(&self) -> ::std::result::Result<&super::types::GuardrailImageBlock, &Self> {
         if let GuardrailContentBlock::Image(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -34,7 +34,7 @@
     }
     /// Tries to convert the enum instance into [`Text`](crate::types::GuardrailContentBlock::Text), extracting the inner [`GuardrailTextBlock`](crate::types::GuardrailTextBlock).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_text(&self) -> ::std::result::Result<&crate::types::GuardrailTextBlock, &Self> {
+    pub fn as_text(&self) -> ::std::result::Result<&super::types::GuardrailTextBlock, &Self> {
         if let GuardrailContentBlock::Text(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
