--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The match for the custom word.</p>
     pub r#match: ::std::string::String,
     /// <p>The action for the custom word.</p>
-    pub action: crate::types::GuardrailWordPolicyAction,
+    pub action: super::types::GuardrailWordPolicyAction,
     /// <p>Indicates whether custom word content that breaches the guardrail configuration is detected.</p>
     pub detected: ::std::option::Option<bool>,
 }
@@ -18,7 +18,7 @@
         self.r#match.deref()
     }
     /// <p>The action for the custom word.</p>
-    pub fn action(&self) -> &crate::types::GuardrailWordPolicyAction {
+    pub fn action(&self) -> &super::types::GuardrailWordPolicyAction {
         &self.action
     }
     /// <p>Indicates whether custom word content that breaches the guardrail configuration is detected.</p>
@@ -28,8 +28,8 @@
 }
 impl GuardrailCustomWord {
     /// Creates a new builder-style object to manufacture [`GuardrailCustomWord`](crate::types::GuardrailCustomWord).
-    pub fn builder() -> crate::types::builders::GuardrailCustomWordBuilder {
-        crate::types::builders::GuardrailCustomWordBuilder::default()
+    pub fn builder() -> super::types::builders::GuardrailCustomWordBuilder {
+        super::types::builders::GuardrailCustomWordBuilder::default()
     }
 }

@@ -38,7 +38,7 @@
 #[non_exhaustive]
 pub struct GuardrailCustomWordBuilder {
     pub(crate) r#match: ::std::option::Option<::std::string::String>,
-    pub(crate) action: ::std::option::Option<crate::types::GuardrailWordPolicyAction>,
+    pub(crate) action: ::std::option::Option<super::types::GuardrailWordPolicyAction>,
     pub(crate) detected: ::std::option::Option<bool>,
 }
 impl GuardrailCustomWordBuilder {
@@ -59,17 +59,17 @@
     }
     /// <p>The action for the custom word.</p>
     /// This field is required.
-    pub fn action(mut self, input: crate::types::GuardrailWordPolicyAction) -> Self {
+    pub fn action(mut self, input: super::types::GuardrailWordPolicyAction) -> Self {
         self.action = ::std::option::Option::Some(input);
         self
     }
     /// <p>The action for the custom word.</p>
-    pub fn set_action(mut self, input: ::std::option::Option<crate::types::GuardrailWordPolicyAction>) -> Self {
+    pub fn set_action(mut self, input: ::std::option::Option<super::types::GuardrailWordPolicyAction>) -> Self {
         self.action = input;
         self
     }
     /// <p>The action for the custom word.</p>
-    pub fn get_action(&self) -> &::std::option::Option<crate::types::GuardrailWordPolicyAction> {
+    pub fn get_action(&self) -> &::std::option::Option<super::types::GuardrailWordPolicyAction> {
         &self.action
     }
     /// <p>Indicates whether custom word content that breaches the guardrail configuration is detected.</p>
@@ -90,8 +90,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`r#match`](crate::types::builders::GuardrailCustomWordBuilder::match)
     /// - [`action`](crate::types::builders::GuardrailCustomWordBuilder::action)
-    pub fn build(self) -> ::std::result::Result<crate::types::GuardrailCustomWord, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::GuardrailCustomWord {
+    pub fn build(self) -> ::std::result::Result<super::types::GuardrailCustomWord, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::GuardrailCustomWord {
             r#match: self.r#match.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "r#match",
