--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl GuardrailTextCharactersCoverage {
     /// Creates a new builder-style object to manufacture [`GuardrailTextCharactersCoverage`](crate::types::GuardrailTextCharactersCoverage).
-    pub fn builder() -> crate::types::builders::GuardrailTextCharactersCoverageBuilder {
-        crate::types::builders::GuardrailTextCharactersCoverageBuilder::default()
+    pub fn builder() -> super::types::builders::GuardrailTextCharactersCoverageBuilder {
+        super::types::builders::GuardrailTextCharactersCoverageBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.total
     }
     /// Consumes the builder and constructs a [`GuardrailTextCharactersCoverage`](crate::types::GuardrailTextCharactersCoverage).
-    pub fn build(self) -> crate::types::GuardrailTextCharactersCoverage {
-        crate::types::GuardrailTextCharactersCoverage {
+    pub fn build(self) -> super::types::GuardrailTextCharactersCoverage {
+        super::types::GuardrailTextCharactersCoverage {
             guarded: self.guarded,
             total: self.total,
         }
