--- original
+++ modified
@@ -40,8 +40,8 @@
 }
 impl InferenceConfiguration {
     /// Creates a new builder-style object to manufacture [`InferenceConfiguration`](crate::types::InferenceConfiguration).
-    pub fn builder() -> crate::types::builders::InferenceConfigurationBuilder {
-        crate::types::builders::InferenceConfigurationBuilder::default()
+    pub fn builder() -> super::types::builders::InferenceConfigurationBuilder {
+        super::types::builders::InferenceConfigurationBuilder::default()
     }
 }

@@ -124,8 +124,8 @@
         &self.stop_sequences
     }
     /// Consumes the builder and constructs a [`InferenceConfiguration`](crate::types::InferenceConfiguration).
-    pub fn build(self) -> crate::types::InferenceConfiguration {
-        crate::types::InferenceConfiguration {
+    pub fn build(self) -> super::types::InferenceConfiguration {
+        super::types::InferenceConfiguration {
             max_tokens: self.max_tokens,
             temperature: self.temperature,
             top_p: self.top_p,
