--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq)]
 pub enum InvokeModelWithBidirectionalStreamInput {
     /// <p>The audio chunk that is used as input for the invocation step.</p>
-    Chunk(crate::types::BidirectionalInputPayloadPart),
+    Chunk(super::super::types::BidirectionalInputPayloadPart),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -20,7 +20,7 @@
     #[allow(irrefutable_let_patterns)]
     /// Tries to convert the enum instance into [`Chunk`](crate::types::InvokeModelWithBidirectionalStreamInput::Chunk), extracting the inner [`BidirectionalInputPayloadPart`](crate::types::BidirectionalInputPayloadPart).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_chunk(&self) -> ::std::result::Result<&crate::types::BidirectionalInputPayloadPart, &Self> {
+    pub fn as_chunk(&self) -> ::std::result::Result<&super::super::types::BidirectionalInputPayloadPart, &Self> {
         if let InvokeModelWithBidirectionalStreamInput::Chunk(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
