--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl InvokeModelTokensRequest {
     /// Creates a new builder-style object to manufacture [`InvokeModelTokensRequest`](crate::types::InvokeModelTokensRequest).
-    pub fn builder() -> crate::types::builders::InvokeModelTokensRequestBuilder {
-        crate::types::builders::InvokeModelTokensRequestBuilder::default()
+    pub fn builder() -> super::types::builders::InvokeModelTokensRequestBuilder {
+        super::types::builders::InvokeModelTokensRequestBuilder::default()
     }
 }

@@ -52,8 +52,8 @@
     /// Consumes the builder and constructs a [`InvokeModelTokensRequest`](crate::types::InvokeModelTokensRequest).
     /// This method will fail if any of the following fields are not set:
     /// - [`body`](crate::types::builders::InvokeModelTokensRequestBuilder::body)
-    pub fn build(self) -> ::std::result::Result<crate::types::InvokeModelTokensRequest, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::InvokeModelTokensRequest {
+    pub fn build(self) -> ::std::result::Result<super::types::InvokeModelTokensRequest, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::InvokeModelTokensRequest {
             body: self.body.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "body",
