--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl PromptRouterTrace {
     /// Creates a new builder-style object to manufacture [`PromptRouterTrace`](crate::types::PromptRouterTrace).
-    pub fn builder() -> crate::types::builders::PromptRouterTraceBuilder {
-        crate::types::builders::PromptRouterTraceBuilder::default()
+    pub fn builder() -> super::super::types::builders::PromptRouterTraceBuilder {
+        super::super::types::builders::PromptRouterTraceBuilder::default()
     }
 }

@@ -42,8 +42,8 @@
         &self.invoked_model_id
     }
     /// Consumes the builder and constructs a [`PromptRouterTrace`](crate::types::PromptRouterTrace).
-    pub fn build(self) -> crate::types::PromptRouterTrace {
-        crate::types::PromptRouterTrace {
+    pub fn build(self) -> super::super::types::PromptRouterTrace {
+        super::super::types::PromptRouterTrace {
             invoked_model_id: self.invoked_model_id,
         }
     }
