--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl S3Location {
     /// Creates a new builder-style object to manufacture [`S3Location`](crate::types::S3Location).
-    pub fn builder() -> crate::types::builders::S3LocationBuilder {
-        crate::types::builders::S3LocationBuilder::default()
+    pub fn builder() -> super::super::types::builders::S3LocationBuilder {
+        super::super::types::builders::S3LocationBuilder::default()
     }
 }

@@ -67,8 +67,8 @@
     /// Consumes the builder and constructs a [`S3Location`](crate::types::S3Location).
     /// This method will fail if any of the following fields are not set:
     /// - [`uri`](crate::types::builders::S3LocationBuilder::uri)
-    pub fn build(self) -> ::std::result::Result<crate::types::S3Location, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::S3Location {
+    pub fn build(self) -> ::std::result::Result<super::super::types::S3Location, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::S3Location {
             uri: self.uri.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "uri",
