--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl SystemTool {
     /// Creates a new builder-style object to manufacture [`SystemTool`](crate::types::SystemTool).
-    pub fn builder() -> crate::types::builders::SystemToolBuilder {
-        crate::types::builders::SystemToolBuilder::default()
+    pub fn builder() -> super::types::builders::SystemToolBuilder {
+        super::types::builders::SystemToolBuilder::default()
     }
 }

@@ -46,8 +46,8 @@
     /// Consumes the builder and constructs a [`SystemTool`](crate::types::SystemTool).
     /// This method will fail if any of the following fields are not set:
     /// - [`name`](crate::types::builders::SystemToolBuilder::name)
-    pub fn build(self) -> ::std::result::Result<crate::types::SystemTool, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SystemTool {
+    pub fn build(self) -> ::std::result::Result<super::types::SystemTool, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SystemTool {
             name: self.name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "name",
