--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ToolRemovalBlock {
     /// <p>A reference to the tool to remove from the available tool set.</p>
-    pub tool: ::std::option::Option<crate::types::ToolReference>,
+    pub tool: ::std::option::Option<super::types::ToolReference>,
 }
 impl ToolRemovalBlock {
     /// <p>A reference to the tool to remove from the available tool set.</p>
-    pub fn tool(&self) -> ::std::option::Option<&crate::types::ToolReference> {
+    pub fn tool(&self) -> ::std::option::Option<&super::types::ToolReference> {
         self.tool.as_ref()
     }
 }
 impl ToolRemovalBlock {
     /// Creates a new builder-style object to manufacture [`ToolRemovalBlock`](crate::types::ToolRemovalBlock).
-    pub fn builder() -> crate::types::builders::ToolRemovalBlockBuilder {
-        crate::types::builders::ToolRemovalBlockBuilder::default()
+    pub fn builder() -> super::types::builders::ToolRemovalBlockBuilder {
+        super::types::builders::ToolRemovalBlockBuilder::default()
     }
 }

@@ -24,26 +24,26 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ToolRemovalBlockBuilder {
-    pub(crate) tool: ::std::option::Option<crate::types::ToolReference>,
+    pub(crate) tool: ::std::option::Option<super::types::ToolReference>,
 }
 impl ToolRemovalBlockBuilder {
     /// <p>A reference to the tool to remove from the available tool set.</p>
     /// This field is required.
-    pub fn tool(mut self, input: crate::types::ToolReference) -> Self {
+    pub fn tool(mut self, input: super::types::ToolReference) -> Self {
         self.tool = ::std::option::Option::Some(input);
         self
     }
     /// <p>A reference to the tool to remove from the available tool set.</p>
-    pub fn set_tool(mut self, input: ::std::option::Option<crate::types::ToolReference>) -> Self {
+    pub fn set_tool(mut self, input: ::std::option::Option<super::types::ToolReference>) -> Self {
         self.tool = input;
         self
     }
     /// <p>A reference to the tool to remove from the available tool set.</p>
-    pub fn get_tool(&self) -> &::std::option::Option<crate::types::ToolReference> {
+    pub fn get_tool(&self) -> &::std::option::Option<super::types::ToolReference> {
         &self.tool
     }
     /// Consumes the builder and constructs a [`ToolRemovalBlock`](crate::types::ToolRemovalBlock).
-    pub fn build(self) -> crate::types::ToolRemovalBlock {
-        crate::types::ToolRemovalBlock { tool: self.tool }
+    pub fn build(self) -> super::types::ToolRemovalBlock {
+        super::types::ToolRemovalBlock { tool: self.tool }
     }
 }
