--- original
+++ modified
@@ -5,11 +5,11 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub enum Tool {
     /// <p>CachePoint to include in the tool configuration.</p>
-    CachePoint(crate::types::CachePointBlock),
+    CachePoint(super::super::types::CachePointBlock),
     /// <p>Specifies the system-defined tool that you want use.</p>
-    SystemTool(crate::types::SystemTool),
+    SystemTool(super::super::types::SystemTool),
     /// <p>The specfication for the tool.</p>
-    ToolSpec(crate::types::ToolSpecification),
+    ToolSpec(super::super::types::ToolSpecification),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -23,7 +23,7 @@
 impl Tool {
     /// Tries to convert the enum instance into [`CachePoint`](crate::types::Tool::CachePoint), extracting the inner [`CachePointBlock`](crate::types::CachePointBlock).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_cache_point(&self) -> ::std::result::Result<&crate::types::CachePointBlock, &Self> {
+    pub fn as_cache_point(&self) -> ::std::result::Result<&super::super::types::CachePointBlock, &Self> {
         if let Tool::CachePoint(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -36,7 +36,7 @@
     }
     /// Tries to convert the enum instance into [`SystemTool`](crate::types::Tool::SystemTool), extracting the inner [`SystemTool`](crate::types::SystemTool).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_system_tool(&self) -> ::std::result::Result<&crate::types::SystemTool, &Self> {
+    pub fn as_system_tool(&self) -> ::std::result::Result<&super::super::types::SystemTool, &Self> {
         if let Tool::SystemTool(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -49,7 +49,7 @@
     }
     /// Tries to convert the enum instance into [`ToolSpec`](crate::types::Tool::ToolSpec), extracting the inner [`ToolSpecification`](crate::types::ToolSpecification).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_tool_spec(&self) -> ::std::result::Result<&crate::types::ToolSpecification, &Self> {
+    pub fn as_tool_spec(&self) -> ::std::result::Result<&super::super::types::ToolSpecification, &Self> {
         if let Tool::ToolSpec(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
