--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl WebLocation {
     /// Creates a new builder-style object to manufacture [`WebLocation`](crate::types::WebLocation).
-    pub fn builder() -> crate::types::builders::WebLocationBuilder {
-        crate::types::builders::WebLocationBuilder::default()
+    pub fn builder() -> super::super::types::builders::WebLocationBuilder {
+        super::super::types::builders::WebLocationBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.domain
     }
     /// Consumes the builder and constructs a [`WebLocation`](crate::types::WebLocation).
-    pub fn build(self) -> crate::types::WebLocation {
-        crate::types::WebLocation {
+    pub fn build(self) -> super::super::types::WebLocation {
+        super::super::types::WebLocation {
             url: self.url,
             domain: self.domain,
         }
