--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl TerminateServiceJobInput {
     /// Creates a new builder-style object to manufacture [`TerminateServiceJobInput`](crate::operation::terminate_service_job::TerminateServiceJobInput).
-    pub fn builder() -> crate::operation::terminate_service_job::builders::TerminateServiceJobInputBuilder {
-        crate::operation::terminate_service_job::builders::TerminateServiceJobInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::terminate_service_job::builders::TerminateServiceJobInputBuilder {
+        super::super::super::operation::terminate_service_job::builders::TerminateServiceJobInputBuilder::default()
     }
 }

@@ -66,9 +66,9 @@
     /// Consumes the builder and constructs a [`TerminateServiceJobInput`](crate::operation::terminate_service_job::TerminateServiceJobInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::terminate_service_job::TerminateServiceJobInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::terminate_service_job::TerminateServiceJobInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::terminate_service_job::TerminateServiceJobInput {
+        ::std::result::Result::Ok(super::super::super::operation::terminate_service_job::TerminateServiceJobInput {
             job_id: self.job_id,
             reason: self.reason,
         })
