--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteLogAnomalyDetectorInput {
     /// Creates a new builder-style object to manufacture [`DeleteLogAnomalyDetectorInput`](crate::operation::delete_log_anomaly_detector::DeleteLogAnomalyDetectorInput).
-    pub fn builder() -> crate::operation::delete_log_anomaly_detector::builders::DeleteLogAnomalyDetectorInputBuilder {
-        crate::operation::delete_log_anomaly_detector::builders::DeleteLogAnomalyDetectorInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_log_anomaly_detector::builders::DeleteLogAnomalyDetectorInputBuilder {
+        super::super::super::operation::delete_log_anomaly_detector::builders::DeleteLogAnomalyDetectorInputBuilder::default()
     }
 }

@@ -45,10 +45,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_log_anomaly_detector::DeleteLogAnomalyDetectorInput,
+        super::super::super::operation::delete_log_anomaly_detector::DeleteLogAnomalyDetectorInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_log_anomaly_detector::DeleteLogAnomalyDetectorInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_log_anomaly_detector::DeleteLogAnomalyDetectorInput {
             anomaly_detector_arn: self.anomaly_detector_arn,
         })
     }
