--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl StartUserImportJobInput {
     /// Creates a new builder-style object to manufacture [`StartUserImportJobInput`](crate::operation::start_user_import_job::StartUserImportJobInput).
-    pub fn builder() -> crate::operation::start_user_import_job::builders::StartUserImportJobInputBuilder {
-        crate::operation::start_user_import_job::builders::StartUserImportJobInputBuilder::default()
+    pub fn builder() -> super::operation::start_user_import_job::builders::StartUserImportJobInputBuilder {
+        super::operation::start_user_import_job::builders::StartUserImportJobInputBuilder::default()
     }
 }

@@ -67,9 +67,9 @@
     /// Consumes the builder and constructs a [`StartUserImportJobInput`](crate::operation::start_user_import_job::StartUserImportJobInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::start_user_import_job::StartUserImportJobInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::start_user_import_job::StartUserImportJobInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::start_user_import_job::StartUserImportJobInput {
+        ::std::result::Result::Ok(super::operation::start_user_import_job::StartUserImportJobInput {
             user_pool_id: self.user_pool_id,
             job_id: self.job_id,
         })
