--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetGroupInput {
     /// Creates a new builder-style object to manufacture [`GetGroupInput`](crate::operation::get_group::GetGroupInput).
-    pub fn builder() -> crate::operation::get_group::builders::GetGroupInputBuilder {
-        crate::operation::get_group::builders::GetGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_group::builders::GetGroupInputBuilder {
+        super::super::super::operation::get_group::builders::GetGroupInputBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
         &self.user_pool_id
     }
     /// Consumes the builder and constructs a [`GetGroupInput`](crate::operation::get_group::GetGroupInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::get_group::GetGroupInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_group::GetGroupInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::get_group::GetGroupInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_group::GetGroupInput {
             group_name: self.group_name,
             user_pool_id: self.user_pool_id,
         })
