--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteConfigurationSetInput {
     /// Creates a new builder-style object to manufacture [`DeleteConfigurationSetInput`](crate::operation::delete_configuration_set::DeleteConfigurationSetInput).
-    pub fn builder() -> crate::operation::delete_configuration_set::builders::DeleteConfigurationSetInputBuilder {
-        crate::operation::delete_configuration_set::builders::DeleteConfigurationSetInputBuilder::default()
+    pub fn builder() -> super::operation::delete_configuration_set::builders::DeleteConfigurationSetInputBuilder {
+        super::operation::delete_configuration_set::builders::DeleteConfigurationSetInputBuilder::default()
     }
 }

@@ -46,10 +46,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_configuration_set::DeleteConfigurationSetInput,
+        super::operation::delete_configuration_set::DeleteConfigurationSetInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_configuration_set::DeleteConfigurationSetInput {
+        ::std::result::Result::Ok(super::operation::delete_configuration_set::DeleteConfigurationSetInput {
             configuration_set_name: self.configuration_set_name,
         })
     }
