--- original
+++ modified
@@ -33,8 +33,8 @@
 }
 impl CreatePlatformEndpointInput {
     /// Creates a new builder-style object to manufacture [`CreatePlatformEndpointInput`](crate::operation::create_platform_endpoint::CreatePlatformEndpointInput).
-    pub fn builder() -> crate::operation::create_platform_endpoint::builders::CreatePlatformEndpointInputBuilder {
-        crate::operation::create_platform_endpoint::builders::CreatePlatformEndpointInputBuilder::default()
+    pub fn builder() -> super::operation::create_platform_endpoint::builders::CreatePlatformEndpointInputBuilder {
+        super::operation::create_platform_endpoint::builders::CreatePlatformEndpointInputBuilder::default()
     }
 }

@@ -116,10 +116,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::create_platform_endpoint::CreatePlatformEndpointInput,
+        super::operation::create_platform_endpoint::CreatePlatformEndpointInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::create_platform_endpoint::CreatePlatformEndpointInput {
+        ::std::result::Result::Ok(super::operation::create_platform_endpoint::CreatePlatformEndpointInput {
             platform_application_arn: self.platform_application_arn,
             token: self.token,
             custom_user_data: self.custom_user_data,
