--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl GetUserAuthFactorsInput {
     /// Creates a new builder-style object to manufacture [`GetUserAuthFactorsInput`](crate::operation::get_user_auth_factors::GetUserAuthFactorsInput).
-    pub fn builder() -> crate::operation::get_user_auth_factors::builders::GetUserAuthFactorsInputBuilder {
-        crate::operation::get_user_auth_factors::builders::GetUserAuthFactorsInputBuilder::default()
+    pub fn builder() -> super::operation::get_user_auth_factors::builders::GetUserAuthFactorsInputBuilder {
+        super::operation::get_user_auth_factors::builders::GetUserAuthFactorsInputBuilder::default()
     }
 }

@@ -51,9 +51,9 @@
     /// Consumes the builder and constructs a [`GetUserAuthFactorsInput`](crate::operation::get_user_auth_factors::GetUserAuthFactorsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_user_auth_factors::GetUserAuthFactorsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_user_auth_factors::GetUserAuthFactorsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_user_auth_factors::GetUserAuthFactorsInput {
+        ::std::result::Result::Ok(super::operation::get_user_auth_factors::GetUserAuthFactorsInput {
             access_token: self.access_token,
         })
     }
