--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteResourceConfigInput {
     /// Creates a new builder-style object to manufacture [`DeleteResourceConfigInput`](crate::operation::delete_resource_config::DeleteResourceConfigInput).
-    pub fn builder() -> crate::operation::delete_resource_config::builders::DeleteResourceConfigInputBuilder {
-        crate::operation::delete_resource_config::builders::DeleteResourceConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_resource_config::builders::DeleteResourceConfigInputBuilder {
+        super::super::super::operation::delete_resource_config::builders::DeleteResourceConfigInputBuilder::default()
     }
 }

@@ -66,9 +66,9 @@
     /// Consumes the builder and constructs a [`DeleteResourceConfigInput`](crate::operation::delete_resource_config::DeleteResourceConfigInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_resource_config::DeleteResourceConfigInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_resource_config::DeleteResourceConfigInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_resource_config::DeleteResourceConfigInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_resource_config::DeleteResourceConfigInput {
             resource_type: self.resource_type,
             resource_id: self.resource_id,
         })
