--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListCodeSigningConfigsInput {
     /// Creates a new builder-style object to manufacture [`ListCodeSigningConfigsInput`](crate::operation::list_code_signing_configs::ListCodeSigningConfigsInput).
-    pub fn builder() -> crate::operation::list_code_signing_configs::builders::ListCodeSigningConfigsInputBuilder {
-        crate::operation::list_code_signing_configs::builders::ListCodeSigningConfigsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_code_signing_configs::builders::ListCodeSigningConfigsInputBuilder {
+        super::super::super::operation::list_code_signing_configs::builders::ListCodeSigningConfigsInputBuilder::default()
     }
 }

@@ -65,10 +65,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_code_signing_configs::ListCodeSigningConfigsInput,
+        super::super::super::operation::list_code_signing_configs::ListCodeSigningConfigsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_code_signing_configs::ListCodeSigningConfigsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_code_signing_configs::ListCodeSigningConfigsInput {
             marker: self.marker,
             max_items: self.max_items,
         })
