--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl DeactivateMfaDeviceInput {
     /// Creates a new builder-style object to manufacture [`DeactivateMfaDeviceInput`](crate::operation::deactivate_mfa_device::DeactivateMfaDeviceInput).
-    pub fn builder() -> crate::operation::deactivate_mfa_device::builders::DeactivateMfaDeviceInputBuilder {
-        crate::operation::deactivate_mfa_device::builders::DeactivateMfaDeviceInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::deactivate_mfa_device::builders::DeactivateMfaDeviceInputBuilder {
+        super::super::super::operation::deactivate_mfa_device::builders::DeactivateMfaDeviceInputBuilder::default()
     }
 }

@@ -80,9 +80,9 @@
     /// Consumes the builder and constructs a [`DeactivateMfaDeviceInput`](crate::operation::deactivate_mfa_device::DeactivateMfaDeviceInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::deactivate_mfa_device::DeactivateMfaDeviceInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::deactivate_mfa_device::DeactivateMfaDeviceInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::deactivate_mfa_device::DeactivateMfaDeviceInput {
+        ::std::result::Result::Ok(super::super::super::operation::deactivate_mfa_device::DeactivateMfaDeviceInput {
             user_name: self.user_name,
             serial_number: self.serial_number,
         })
