--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ListGroupsInput {
     /// Creates a new builder-style object to manufacture [`ListGroupsInput`](crate::operation::list_groups::ListGroupsInput).
-    pub fn builder() -> crate::operation::list_groups::builders::ListGroupsInputBuilder {
-        crate::operation::list_groups::builders::ListGroupsInputBuilder::default()
+    pub fn builder() -> super::operation::list_groups::builders::ListGroupsInputBuilder {
+        super::operation::list_groups::builders::ListGroupsInputBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.next_token
     }
     /// Consumes the builder and constructs a [`ListGroupsInput`](crate::operation::list_groups::ListGroupsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_groups::ListGroupsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_groups::ListGroupsInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_groups::ListGroupsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_groups::ListGroupsInput {
             user_pool_id: self.user_pool_id,
             limit: self.limit,
             next_token: self.next_token,
