--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct GetComplianceSummaryByConfigRuleInput {}
 impl GetComplianceSummaryByConfigRuleInput {
     /// Creates a new builder-style object to manufacture [`GetComplianceSummaryByConfigRuleInput`](crate::operation::get_compliance_summary_by_config_rule::GetComplianceSummaryByConfigRuleInput).
-    pub fn builder() -> crate::operation::get_compliance_summary_by_config_rule::builders::GetComplianceSummaryByConfigRuleInputBuilder {
-        crate::operation::get_compliance_summary_by_config_rule::builders::GetComplianceSummaryByConfigRuleInputBuilder::default()
+    pub fn builder() -> super::operation::get_compliance_summary_by_config_rule::builders::GetComplianceSummaryByConfigRuleInputBuilder {
+        super::operation::get_compliance_summary_by_config_rule::builders::GetComplianceSummaryByConfigRuleInputBuilder::default()
     }
 }

@@ -19,9 +19,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_compliance_summary_by_config_rule::GetComplianceSummaryByConfigRuleInput,
+        super::operation::get_compliance_summary_by_config_rule::GetComplianceSummaryByConfigRuleInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_compliance_summary_by_config_rule::GetComplianceSummaryByConfigRuleInput {})
+        ::std::result::Result::Ok(super::operation::get_compliance_summary_by_config_rule::GetComplianceSummaryByConfigRuleInput {})
     }
 }
