--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>For more information about ARNs, see <a href="https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html">Amazon Resource Names (ARNs)</a> in the <i>Amazon Web Services General Reference</i>.</p>
     pub saml_provider_arn: ::std::option::Option<::std::string::String>,
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub assertion_encryption_mode: ::std::option::Option<crate::types::AssertionEncryptionModeType>,
+    pub assertion_encryption_mode: ::std::option::Option<super::types::AssertionEncryptionModeType>,
     /// <p>Specifies the new private key from your external identity provider. The private key must be a .pem file that uses AES-GCM or AES-CBC encryption algorithm to decrypt SAML assertions.</p>
     pub add_private_key: ::std::option::Option<::std::string::String>,
     /// <p>The Key ID of the private key to remove.</p>
@@ -26,7 +26,7 @@
         self.saml_provider_arn.as_deref()
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn assertion_encryption_mode(&self) -> ::std::option::Option<&crate::types::AssertionEncryptionModeType> {
+    pub fn assertion_encryption_mode(&self) -> ::std::option::Option<&super::types::AssertionEncryptionModeType> {
         self.assertion_encryption_mode.as_ref()
     }
     /// <p>Specifies the new private key from your external identity provider. The private key must be a .pem file that uses AES-GCM or AES-CBC encryption algorithm to decrypt SAML assertions.</p>
@@ -51,8 +51,8 @@
 }
 impl UpdateSamlProviderInput {
     /// Creates a new builder-style object to manufacture [`UpdateSamlProviderInput`](crate::operation::update_saml_provider::UpdateSamlProviderInput).
-    pub fn builder() -> crate::operation::update_saml_provider::builders::UpdateSamlProviderInputBuilder {
-        crate::operation::update_saml_provider::builders::UpdateSamlProviderInputBuilder::default()
+    pub fn builder() -> super::operation::update_saml_provider::builders::UpdateSamlProviderInputBuilder {
+        super::operation::update_saml_provider::builders::UpdateSamlProviderInputBuilder::default()
     }
 }

@@ -62,7 +62,7 @@
 pub struct UpdateSamlProviderInputBuilder {
     pub(crate) saml_metadata_document: ::std::option::Option<::std::string::String>,
     pub(crate) saml_provider_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) assertion_encryption_mode: ::std::option::Option<crate::types::AssertionEncryptionModeType>,
+    pub(crate) assertion_encryption_mode: ::std::option::Option<super::types::AssertionEncryptionModeType>,
     pub(crate) add_private_key: ::std::option::Option<::std::string::String>,
     pub(crate) remove_private_key: ::std::option::Option<::std::string::String>,
 }
@@ -100,17 +100,17 @@
         &self.saml_provider_arn
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn assertion_encryption_mode(mut self, input: crate::types::AssertionEncryptionModeType) -> Self {
+    pub fn assertion_encryption_mode(mut self, input: super::types::AssertionEncryptionModeType) -> Self {
         self.assertion_encryption_mode = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn set_assertion_encryption_mode(mut self, input: ::std::option::Option<crate::types::AssertionEncryptionModeType>) -> Self {
+    pub fn set_assertion_encryption_mode(mut self, input: ::std::option::Option<super::types::AssertionEncryptionModeType>) -> Self {
         self.assertion_encryption_mode = input;
         self
     }
     /// <p>Specifies the encryption setting for the SAML provider.</p>
-    pub fn get_assertion_encryption_mode(&self) -> &::std::option::Option<crate::types::AssertionEncryptionModeType> {
+    pub fn get_assertion_encryption_mode(&self) -> &::std::option::Option<super::types::AssertionEncryptionModeType> {
         &self.assertion_encryption_mode
     }
     /// <p>Specifies the new private key from your external identity provider. The private key must be a .pem file that uses AES-GCM or AES-CBC encryption algorithm to decrypt SAML assertions.</p>
@@ -144,9 +144,9 @@
     /// Consumes the builder and constructs a [`UpdateSamlProviderInput`](crate::operation::update_saml_provider::UpdateSamlProviderInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_saml_provider::UpdateSamlProviderInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::update_saml_provider::UpdateSamlProviderInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_saml_provider::UpdateSamlProviderInput {
+        ::std::result::Result::Ok(super::operation::update_saml_provider::UpdateSamlProviderInput {
             saml_metadata_document: self.saml_metadata_document,
             saml_provider_arn: self.saml_provider_arn,
             assertion_encryption_mode: self.assertion_encryption_mode,
