--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl ListResourceServersInput {
     /// Creates a new builder-style object to manufacture [`ListResourceServersInput`](crate::operation::list_resource_servers::ListResourceServersInput).
-    pub fn builder() -> crate::operation::list_resource_servers::builders::ListResourceServersInputBuilder {
-        crate::operation::list_resource_servers::builders::ListResourceServersInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_resource_servers::builders::ListResourceServersInputBuilder {
+        super::super::super::operation::list_resource_servers::builders::ListResourceServersInputBuilder::default()
     }
 }

@@ -86,9 +86,9 @@
     /// Consumes the builder and constructs a [`ListResourceServersInput`](crate::operation::list_resource_servers::ListResourceServersInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_resource_servers::ListResourceServersInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_resource_servers::ListResourceServersInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_resource_servers::ListResourceServersInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_resource_servers::ListResourceServersInput {
             user_pool_id: self.user_pool_id,
             max_results: self.max_results,
             next_token: self.next_token,
