--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl GetLogRecordInput {
     /// Creates a new builder-style object to manufacture [`GetLogRecordInput`](crate::operation::get_log_record::GetLogRecordInput).
-    pub fn builder() -> crate::operation::get_log_record::builders::GetLogRecordInputBuilder {
-        crate::operation::get_log_record::builders::GetLogRecordInputBuilder::default()
+    pub fn builder() -> super::operation::get_log_record::builders::GetLogRecordInputBuilder {
+        super::operation::get_log_record::builders::GetLogRecordInputBuilder::default()
     }
 }

@@ -70,8 +70,8 @@
     /// Consumes the builder and constructs a [`GetLogRecordInput`](crate::operation::get_log_record::GetLogRecordInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_log_record::GetLogRecordInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_log_record::GetLogRecordInput {
+    ) -> ::std::result::Result<super::operation::get_log_record::GetLogRecordInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::get_log_record::GetLogRecordInput {
             log_record_pointer: self.log_record_pointer,
             unmask: self.unmask,
         })
