--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListTagsOfResourceInput {
     /// Creates a new builder-style object to manufacture [`ListTagsOfResourceInput`](crate::operation::list_tags_of_resource::ListTagsOfResourceInput).
-    pub fn builder() -> crate::operation::list_tags_of_resource::builders::ListTagsOfResourceInputBuilder {
-        crate::operation::list_tags_of_resource::builders::ListTagsOfResourceInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_tags_of_resource::builders::ListTagsOfResourceInputBuilder {
+        super::super::super::operation::list_tags_of_resource::builders::ListTagsOfResourceInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`ListTagsOfResourceInput`](crate::operation::list_tags_of_resource::ListTagsOfResourceInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_tags_of_resource::ListTagsOfResourceInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_tags_of_resource::ListTagsOfResourceInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_tags_of_resource::ListTagsOfResourceInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_tags_of_resource::ListTagsOfResourceInput {
             resource_arn: self.resource_arn,
             next_token: self.next_token,
         })
