--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl ListDeliverabilityTestReportsInput {
     /// Creates a new builder-style object to manufacture [`ListDeliverabilityTestReportsInput`](crate::operation::list_deliverability_test_reports::ListDeliverabilityTestReportsInput).
-    pub fn builder() -> crate::operation::list_deliverability_test_reports::builders::ListDeliverabilityTestReportsInputBuilder {
-        crate::operation::list_deliverability_test_reports::builders::ListDeliverabilityTestReportsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_deliverability_test_reports::builders::ListDeliverabilityTestReportsInputBuilder {
+        super::super::super::operation::list_deliverability_test_reports::builders::ListDeliverabilityTestReportsInputBuilder::default()
     }
 }

@@ -71,10 +71,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_deliverability_test_reports::ListDeliverabilityTestReportsInput,
+        super::super::super::operation::list_deliverability_test_reports::ListDeliverabilityTestReportsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_deliverability_test_reports::ListDeliverabilityTestReportsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_deliverability_test_reports::ListDeliverabilityTestReportsInput {
             next_token: self.next_token,
             page_size: self.page_size,
         })
