--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl GetIdentityProviderByIdentifierInput {
     /// Creates a new builder-style object to manufacture [`GetIdentityProviderByIdentifierInput`](crate::operation::get_identity_provider_by_identifier::GetIdentityProviderByIdentifierInput).
-    pub fn builder() -> crate::operation::get_identity_provider_by_identifier::builders::GetIdentityProviderByIdentifierInputBuilder {
-        crate::operation::get_identity_provider_by_identifier::builders::GetIdentityProviderByIdentifierInputBuilder::default()
+    pub fn builder() -> super::operation::get_identity_provider_by_identifier::builders::GetIdentityProviderByIdentifierInputBuilder {
+        super::operation::get_identity_provider_by_identifier::builders::GetIdentityProviderByIdentifierInputBuilder::default()
     }
 }

@@ -67,11 +67,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_identity_provider_by_identifier::GetIdentityProviderByIdentifierInput,
+        super::operation::get_identity_provider_by_identifier::GetIdentityProviderByIdentifierInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_identity_provider_by_identifier::GetIdentityProviderByIdentifierInput {
+            super::operation::get_identity_provider_by_identifier::GetIdentityProviderByIdentifierInput {
                 user_pool_id: self.user_pool_id,
                 idp_identifier: self.idp_identifier,
             },
