--- original
+++ modified
@@ -36,8 +36,8 @@
 }
 impl VerifyUserAttributeInput {
     /// Creates a new builder-style object to manufacture [`VerifyUserAttributeInput`](crate::operation::verify_user_attribute::VerifyUserAttributeInput).
-    pub fn builder() -> crate::operation::verify_user_attribute::builders::VerifyUserAttributeInputBuilder {
-        crate::operation::verify_user_attribute::builders::VerifyUserAttributeInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::verify_user_attribute::builders::VerifyUserAttributeInputBuilder {
+        super::super::super::operation::verify_user_attribute::builders::VerifyUserAttributeInputBuilder::default()
     }
 }

@@ -98,9 +98,9 @@
     /// Consumes the builder and constructs a [`VerifyUserAttributeInput`](crate::operation::verify_user_attribute::VerifyUserAttributeInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::verify_user_attribute::VerifyUserAttributeInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::verify_user_attribute::VerifyUserAttributeInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::verify_user_attribute::VerifyUserAttributeInput {
+        ::std::result::Result::Ok(super::super::super::operation::verify_user_attribute::VerifyUserAttributeInput {
             access_token: self.access_token,
             attribute_name: self.attribute_name,
             code: self.code,
