--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListMessageMoveTasksInput {
     /// Creates a new builder-style object to manufacture [`ListMessageMoveTasksInput`](crate::operation::list_message_move_tasks::ListMessageMoveTasksInput).
-    pub fn builder() -> crate::operation::list_message_move_tasks::builders::ListMessageMoveTasksInputBuilder {
-        crate::operation::list_message_move_tasks::builders::ListMessageMoveTasksInputBuilder::default()
+    pub fn builder() -> super::operation::list_message_move_tasks::builders::ListMessageMoveTasksInputBuilder {
+        super::operation::list_message_move_tasks::builders::ListMessageMoveTasksInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`ListMessageMoveTasksInput`](crate::operation::list_message_move_tasks::ListMessageMoveTasksInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_message_move_tasks::ListMessageMoveTasksInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_message_move_tasks::ListMessageMoveTasksInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_message_move_tasks::ListMessageMoveTasksInput {
+        ::std::result::Result::Ok(super::operation::list_message_move_tasks::ListMessageMoveTasksInput {
             source_arn: self.source_arn,
             max_results: self.max_results,
         })
