--- original
+++ modified
@@ -7,8 +7,8 @@
 pub struct GetDeliverabilityDashboardOptionsInput {}
 impl GetDeliverabilityDashboardOptionsInput {
     /// Creates a new builder-style object to manufacture [`GetDeliverabilityDashboardOptionsInput`](crate::operation::get_deliverability_dashboard_options::GetDeliverabilityDashboardOptionsInput).
-    pub fn builder() -> crate::operation::get_deliverability_dashboard_options::builders::GetDeliverabilityDashboardOptionsInputBuilder {
-        crate::operation::get_deliverability_dashboard_options::builders::GetDeliverabilityDashboardOptionsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_deliverability_dashboard_options::builders::GetDeliverabilityDashboardOptionsInputBuilder {
+        super::super::super::operation::get_deliverability_dashboard_options::builders::GetDeliverabilityDashboardOptionsInputBuilder::default()
     }
 }

@@ -21,9 +21,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_deliverability_dashboard_options::GetDeliverabilityDashboardOptionsInput,
+        super::super::super::operation::get_deliverability_dashboard_options::GetDeliverabilityDashboardOptionsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_deliverability_dashboard_options::GetDeliverabilityDashboardOptionsInput {})
+        ::std::result::Result::Ok(super::super::super::operation::get_deliverability_dashboard_options::GetDeliverabilityDashboardOptionsInput {})
     }
 }
