--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteBucketTaggingInput {
     /// Creates a new builder-style object to manufacture [`DeleteBucketTaggingInput`](crate::operation::delete_bucket_tagging::DeleteBucketTaggingInput).
-    pub fn builder() -> crate::operation::delete_bucket_tagging::builders::DeleteBucketTaggingInputBuilder {
-        crate::operation::delete_bucket_tagging::builders::DeleteBucketTaggingInputBuilder::default()
+    pub fn builder() -> super::operation::delete_bucket_tagging::builders::DeleteBucketTaggingInputBuilder {
+        super::operation::delete_bucket_tagging::builders::DeleteBucketTaggingInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`DeleteBucketTaggingInput`](crate::operation::delete_bucket_tagging::DeleteBucketTaggingInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_bucket_tagging::DeleteBucketTaggingInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::delete_bucket_tagging::DeleteBucketTaggingInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_bucket_tagging::DeleteBucketTaggingInput {
+        ::std::result::Result::Ok(super::operation::delete_bucket_tagging::DeleteBucketTaggingInput {
             bucket: self.bucket,
             expected_bucket_owner: self.expected_bucket_owner,
         })
