--- original
+++ modified
@@ -10,7 +10,7 @@
     /// <p>A filter pattern for extracting metric data out of ingested log events.</p>
     pub filter_pattern: ::std::option::Option<::std::string::String>,
     /// <p>A collection of information that defines how metric data gets emitted.</p>
-    pub metric_transformations: ::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>>,
+    pub metric_transformations: ::std::option::Option<::std::vec::Vec<super::types::MetricTransformation>>,
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
     /// <p>If the log group uses either a log-group level or account-level transformer, and you specify <code>true</code>, the metric filter will be applied on the transformed version of the log events instead of the original ingested log events.</p>
     pub apply_on_transformed_logs: ::std::option::Option<bool>,
@@ -35,7 +35,7 @@
     /// <p>A collection of information that defines how metric data gets emitted.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.metric_transformations.is_none()`.
-    pub fn metric_transformations(&self) -> &[crate::types::MetricTransformation] {
+    pub fn metric_transformations(&self) -> &[super::types::MetricTransformation] {
         self.metric_transformations.as_deref().unwrap_or_default()
     }
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
@@ -56,8 +56,8 @@
 }
 impl PutMetricFilterInput {
     /// Creates a new builder-style object to manufacture [`PutMetricFilterInput`](crate::operation::put_metric_filter::PutMetricFilterInput).
-    pub fn builder() -> crate::operation::put_metric_filter::builders::PutMetricFilterInputBuilder {
-        crate::operation::put_metric_filter::builders::PutMetricFilterInputBuilder::default()
+    pub fn builder() -> super::operation::put_metric_filter::builders::PutMetricFilterInputBuilder {
+        super::operation::put_metric_filter::builders::PutMetricFilterInputBuilder::default()
     }
 }

@@ -68,7 +68,7 @@
     pub(crate) log_group_name: ::std::option::Option<::std::string::String>,
     pub(crate) filter_name: ::std::option::Option<::std::string::String>,
     pub(crate) filter_pattern: ::std::option::Option<::std::string::String>,
-    pub(crate) metric_transformations: ::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>>,
+    pub(crate) metric_transformations: ::std::option::Option<::std::vec::Vec<super::types::MetricTransformation>>,
     pub(crate) apply_on_transformed_logs: ::std::option::Option<bool>,
     pub(crate) field_selection_criteria: ::std::option::Option<::std::string::String>,
     pub(crate) emit_system_field_dimensions: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -124,7 +124,7 @@
     /// To override the contents of this collection use [`set_metric_transformations`](Self::set_metric_transformations).
     ///
     /// <p>A collection of information that defines how metric data gets emitted.</p>
-    pub fn metric_transformations(mut self, input: crate::types::MetricTransformation) -> Self {
+    pub fn metric_transformations(mut self, input: super::types::MetricTransformation) -> Self {
         let mut v = self.metric_transformations.unwrap_or_default();
         v.push(input);
         self.metric_transformations = ::std::option::Option::Some(v);
@@ -131,12 +131,12 @@
         self
     }
     /// <p>A collection of information that defines how metric data gets emitted.</p>
-    pub fn set_metric_transformations(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>>) -> Self {
+    pub fn set_metric_transformations(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::MetricTransformation>>) -> Self {
         self.metric_transformations = input;
         self
     }
     /// <p>A collection of information that defines how metric data gets emitted.</p>
-    pub fn get_metric_transformations(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>> {
+    pub fn get_metric_transformations(&self) -> &::std::option::Option<::std::vec::Vec<super::types::MetricTransformation>> {
         &self.metric_transformations
     }
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
@@ -193,8 +193,8 @@
     /// Consumes the builder and constructs a [`PutMetricFilterInput`](crate::operation::put_metric_filter::PutMetricFilterInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::put_metric_filter::PutMetricFilterInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::put_metric_filter::PutMetricFilterInput {
+    ) -> ::std::result::Result<super::operation::put_metric_filter::PutMetricFilterInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::put_metric_filter::PutMetricFilterInput {
             log_group_name: self.log_group_name,
             filter_name: self.filter_name,
             filter_pattern: self.filter_pattern,
