--- original
+++ modified
@@ -46,8 +46,8 @@
 }
 impl Credentials {
     /// Creates a new builder-style object to manufacture [`Credentials`](crate::types::Credentials).
-    pub fn builder() -> crate::types::builders::CredentialsBuilder {
-        crate::types::builders::CredentialsBuilder::default()
+    pub fn builder() -> super::super::types::builders::CredentialsBuilder {
+        super::super::types::builders::CredentialsBuilder::default()
     }
 }

@@ -127,8 +127,8 @@
     /// - [`secret_access_key`](crate::types::builders::CredentialsBuilder::secret_access_key)
     /// - [`session_token`](crate::types::builders::CredentialsBuilder::session_token)
     /// - [`expiration`](crate::types::builders::CredentialsBuilder::expiration)
-    pub fn build(self) -> ::std::result::Result<crate::types::Credentials, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Credentials {
+    pub fn build(self) -> ::std::result::Result<super::super::types::Credentials, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::Credentials {
             access_key_id: self.access_key_id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "access_key_id",
