--- original
+++ modified
@@ -6,7 +6,7 @@
     /// <p>The ARN of the anomaly detector that you want to update.</p>
     pub anomaly_detector_arn: ::std::option::Option<::std::string::String>,
     /// <p>Specifies how often the anomaly detector runs and look for anomalies. Set this value according to the frequency that the log group receives new logs. For example, if the log group receives new log events every 10 minutes, then setting <code>evaluationFrequency</code> to <code>FIFTEEN_MIN</code> might be appropriate.</p>
-    pub evaluation_frequency: ::std::option::Option<crate::types::EvaluationFrequency>,
+    pub evaluation_frequency: ::std::option::Option<super::super::super::types::EvaluationFrequency>,
     /// <p>A symbolic description of how CloudWatch Logs should interpret the data in each log event. For example, a log event can contain timestamps, IP addresses, strings, and so on. You use the filter pattern to specify what to look for in the log event message.</p>
     pub filter_pattern: ::std::option::Option<::std::string::String>,
     /// <p>The number of days to use as the life cycle of anomalies. After this time, anomalies are automatically baselined and the anomaly detector model will treat new occurrences of similar event as normal. Therefore, if you do not correct the cause of an anomaly during this time, it will be considered normal going forward and will not be detected.</p>
@@ -20,7 +20,7 @@
         self.anomaly_detector_arn.as_deref()
     }
     /// <p>Specifies how often the anomaly detector runs and look for anomalies. Set this value according to the frequency that the log group receives new logs. For example, if the log group receives new log events every 10 minutes, then setting <code>evaluationFrequency</code> to <code>FIFTEEN_MIN</code> might be appropriate.</p>
-    pub fn evaluation_frequency(&self) -> ::std::option::Option<&crate::types::EvaluationFrequency> {
+    pub fn evaluation_frequency(&self) -> ::std::option::Option<&super::super::super::types::EvaluationFrequency> {
         self.evaluation_frequency.as_ref()
     }
     /// <p>A symbolic description of how CloudWatch Logs should interpret the data in each log event. For example, a log event can contain timestamps, IP addresses, strings, and so on. You use the filter pattern to specify what to look for in the log event message.</p>
@@ -38,8 +38,8 @@
 }
 impl UpdateLogAnomalyDetectorInput {
     /// Creates a new builder-style object to manufacture [`UpdateLogAnomalyDetectorInput`](crate::operation::update_log_anomaly_detector::UpdateLogAnomalyDetectorInput).
-    pub fn builder() -> crate::operation::update_log_anomaly_detector::builders::UpdateLogAnomalyDetectorInputBuilder {
-        crate::operation::update_log_anomaly_detector::builders::UpdateLogAnomalyDetectorInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_log_anomaly_detector::builders::UpdateLogAnomalyDetectorInputBuilder {
+        super::super::super::operation::update_log_anomaly_detector::builders::UpdateLogAnomalyDetectorInputBuilder::default()
     }
 }

@@ -48,7 +48,7 @@
 #[non_exhaustive]
 pub struct UpdateLogAnomalyDetectorInputBuilder {
     pub(crate) anomaly_detector_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) evaluation_frequency: ::std::option::Option<crate::types::EvaluationFrequency>,
+    pub(crate) evaluation_frequency: ::std::option::Option<super::super::super::types::EvaluationFrequency>,
     pub(crate) filter_pattern: ::std::option::Option<::std::string::String>,
     pub(crate) anomaly_visibility_time: ::std::option::Option<i64>,
     pub(crate) enabled: ::std::option::Option<bool>,
@@ -70,17 +70,17 @@
         &self.anomaly_detector_arn
     }
     /// <p>Specifies how often the anomaly detector runs and look for anomalies. Set this value according to the frequency that the log group receives new logs. For example, if the log group receives new log events every 10 minutes, then setting <code>evaluationFrequency</code> to <code>FIFTEEN_MIN</code> might be appropriate.</p>
-    pub fn evaluation_frequency(mut self, input: crate::types::EvaluationFrequency) -> Self {
+    pub fn evaluation_frequency(mut self, input: super::super::super::types::EvaluationFrequency) -> Self {
         self.evaluation_frequency = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies how often the anomaly detector runs and look for anomalies. Set this value according to the frequency that the log group receives new logs. For example, if the log group receives new log events every 10 minutes, then setting <code>evaluationFrequency</code> to <code>FIFTEEN_MIN</code> might be appropriate.</p>
-    pub fn set_evaluation_frequency(mut self, input: ::std::option::Option<crate::types::EvaluationFrequency>) -> Self {
+    pub fn set_evaluation_frequency(mut self, input: ::std::option::Option<super::super::super::types::EvaluationFrequency>) -> Self {
         self.evaluation_frequency = input;
         self
     }
     /// <p>Specifies how often the anomaly detector runs and look for anomalies. Set this value according to the frequency that the log group receives new logs. For example, if the log group receives new log events every 10 minutes, then setting <code>evaluationFrequency</code> to <code>FIFTEEN_MIN</code> might be appropriate.</p>
-    pub fn get_evaluation_frequency(&self) -> &::std::option::Option<crate::types::EvaluationFrequency> {
+    pub fn get_evaluation_frequency(&self) -> &::std::option::Option<super::super::super::types::EvaluationFrequency> {
         &self.evaluation_frequency
     }
     /// <p>A symbolic description of how CloudWatch Logs should interpret the data in each log event. For example, a log event can contain timestamps, IP addresses, strings, and so on. You use the filter pattern to specify what to look for in the log event message.</p>
@@ -130,10 +130,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_log_anomaly_detector::UpdateLogAnomalyDetectorInput,
+        super::super::super::operation::update_log_anomaly_detector::UpdateLogAnomalyDetectorInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_log_anomaly_detector::UpdateLogAnomalyDetectorInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_log_anomaly_detector::UpdateLogAnomalyDetectorInput {
             anomaly_detector_arn: self.anomaly_detector_arn,
             evaluation_frequency: self.evaluation_frequency,
             filter_pattern: self.filter_pattern,
