--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl PutDedicatedIpWarmupAttributesInput {
     /// Creates a new builder-style object to manufacture [`PutDedicatedIpWarmupAttributesInput`](crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput).
-    pub fn builder() -> crate::operation::put_dedicated_ip_warmup_attributes::builders::PutDedicatedIpWarmupAttributesInputBuilder {
-        crate::operation::put_dedicated_ip_warmup_attributes::builders::PutDedicatedIpWarmupAttributesInputBuilder::default()
+    pub fn builder() -> super::operation::put_dedicated_ip_warmup_attributes::builders::PutDedicatedIpWarmupAttributesInputBuilder {
+        super::operation::put_dedicated_ip_warmup_attributes::builders::PutDedicatedIpWarmupAttributesInputBuilder::default()
     }
 }

@@ -68,11 +68,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput,
+        super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput {
+            super::operation::put_dedicated_ip_warmup_attributes::PutDedicatedIpWarmupAttributesInput {
                 ip: self.ip,
                 warmup_percentage: self.warmup_percentage,
             },
