--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The updated name of the resource server.</p>
     pub name: ::std::option::Option<::std::string::String>,
     /// <p>An array of updated custom scope names and descriptions that you want to associate with your resource server.</p>
-    pub scopes: ::std::option::Option<::std::vec::Vec<crate::types::ResourceServerScopeType>>,
+    pub scopes: ::std::option::Option<::std::vec::Vec<super::types::ResourceServerScopeType>>,
 }
 impl UpdateResourceServerInput {
     /// <p>The ID of the user pool that contains the resource server that you want to update.</p>
@@ -30,14 +30,14 @@
     /// <p>An array of updated custom scope names and descriptions that you want to associate with your resource server.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.scopes.is_none()`.
-    pub fn scopes(&self) -> &[crate::types::ResourceServerScopeType] {
+    pub fn scopes(&self) -> &[super::types::ResourceServerScopeType] {
         self.scopes.as_deref().unwrap_or_default()
     }
 }
 impl UpdateResourceServerInput {
     /// Creates a new builder-style object to manufacture [`UpdateResourceServerInput`](crate::operation::update_resource_server::UpdateResourceServerInput).
-    pub fn builder() -> crate::operation::update_resource_server::builders::UpdateResourceServerInputBuilder {
-        crate::operation::update_resource_server::builders::UpdateResourceServerInputBuilder::default()
+    pub fn builder() -> super::operation::update_resource_server::builders::UpdateResourceServerInputBuilder {
+        super::operation::update_resource_server::builders::UpdateResourceServerInputBuilder::default()
     }
 }

@@ -48,7 +48,7 @@
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) identifier: ::std::option::Option<::std::string::String>,
     pub(crate) name: ::std::option::Option<::std::string::String>,
-    pub(crate) scopes: ::std::option::Option<::std::vec::Vec<crate::types::ResourceServerScopeType>>,
+    pub(crate) scopes: ::std::option::Option<::std::vec::Vec<super::types::ResourceServerScopeType>>,
 }
 impl UpdateResourceServerInputBuilder {
     /// <p>The ID of the user pool that contains the resource server that you want to update.</p>
@@ -104,7 +104,7 @@
     /// To override the contents of this collection use [`set_scopes`](Self::set_scopes).
     ///
     /// <p>An array of updated custom scope names and descriptions that you want to associate with your resource server.</p>
-    pub fn scopes(mut self, input: crate::types::ResourceServerScopeType) -> Self {
+    pub fn scopes(mut self, input: super::types::ResourceServerScopeType) -> Self {
         let mut v = self.scopes.unwrap_or_default();
         v.push(input);
         self.scopes = ::std::option::Option::Some(v);
@@ -111,20 +111,20 @@
         self
     }
     /// <p>An array of updated custom scope names and descriptions that you want to associate with your resource server.</p>
-    pub fn set_scopes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ResourceServerScopeType>>) -> Self {
+    pub fn set_scopes(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ResourceServerScopeType>>) -> Self {
         self.scopes = input;
         self
     }
     /// <p>An array of updated custom scope names and descriptions that you want to associate with your resource server.</p>
-    pub fn get_scopes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ResourceServerScopeType>> {
+    pub fn get_scopes(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ResourceServerScopeType>> {
         &self.scopes
     }
     /// Consumes the builder and constructs a [`UpdateResourceServerInput`](crate::operation::update_resource_server::UpdateResourceServerInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_resource_server::UpdateResourceServerInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::update_resource_server::UpdateResourceServerInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_resource_server::UpdateResourceServerInput {
+        ::std::result::Result::Ok(super::operation::update_resource_server::UpdateResourceServerInput {
             user_pool_id: self.user_pool_id,
             identifier: self.identifier,
             name: self.name,
