--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <p>The job status used to filter jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exception is the <code>SHARE_IDENTIFIER</code> filter and <code>jobStatus</code> can be used together. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>Array job parents are updated to <code>PENDING</code> when any child job is updated to <code>RUNNABLE</code> and remain in <code>PENDING</code> status while child jobs are running. To view these jobs, filter by <code>PENDING</code> status until all child jobs reach a terminal state.</p>
     /// </note>
-    pub job_status: ::std::option::Option<crate::types::JobStatus>,
+    pub job_status: ::std::option::Option<super::super::super::types::JobStatus>,
     /// <p>The maximum number of results returned by <code>ListJobs</code> in a paginated output. When this parameter is used, <code>ListJobs</code> returns up to <code>maxResults</code> results in a single page and a <code>nextToken</code> response element, if applicable. The remaining results of the initial request can be seen by sending another <code>ListJobs</code> request with the returned <code>nextToken</code> value.</p>
     /// <p>The following outlines key parameters and limitations:</p>
     /// <ul>
@@ -66,7 +66,7 @@
     /// <p>The value for the filter is the fairshare scheduling share identifier.</p>
     /// </dd>
     /// </dl>
-    pub filters: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>>,
+    pub filters: ::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>>,
 }
 impl ListJobsInput {
     /// <p>The name or full Amazon Resource Name (ARN) of the job queue used to list jobs.</p>
@@ -84,7 +84,7 @@
     /// <p>The job status used to filter jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exception is the <code>SHARE_IDENTIFIER</code> filter and <code>jobStatus</code> can be used together. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>Array job parents are updated to <code>PENDING</code> when any child job is updated to <code>RUNNABLE</code> and remain in <code>PENDING</code> status while child jobs are running. To view these jobs, filter by <code>PENDING</code> status until all child jobs reach a terminal state.</p>
     /// </note>
-    pub fn job_status(&self) -> ::std::option::Option<&crate::types::JobStatus> {
+    pub fn job_status(&self) -> ::std::option::Option<&super::super::super::types::JobStatus> {
         self.job_status.as_ref()
     }
     /// <p>The maximum number of results returned by <code>ListJobs</code> in a paginated output. When this parameter is used, <code>ListJobs</code> returns up to <code>maxResults</code> results in a single page and a <code>nextToken</code> response element, if applicable. The remaining results of the initial request can be seen by sending another <code>ListJobs</code> request with the returned <code>nextToken</code> value.</p>
@@ -145,14 +145,14 @@
     /// </dl>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.filters.is_none()`.
-    pub fn filters(&self) -> &[crate::types::KeyValuesPair] {
+    pub fn filters(&self) -> &[super::super::super::types::KeyValuesPair] {
         self.filters.as_deref().unwrap_or_default()
     }
 }
 impl ListJobsInput {
     /// Creates a new builder-style object to manufacture [`ListJobsInput`](crate::operation::list_jobs::ListJobsInput).
-    pub fn builder() -> crate::operation::list_jobs::builders::ListJobsInputBuilder {
-        crate::operation::list_jobs::builders::ListJobsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_jobs::builders::ListJobsInputBuilder {
+        super::super::super::operation::list_jobs::builders::ListJobsInputBuilder::default()
     }
 }

@@ -163,10 +163,10 @@
     pub(crate) job_queue: ::std::option::Option<::std::string::String>,
     pub(crate) array_job_id: ::std::option::Option<::std::string::String>,
     pub(crate) multi_node_job_id: ::std::option::Option<::std::string::String>,
-    pub(crate) job_status: ::std::option::Option<crate::types::JobStatus>,
+    pub(crate) job_status: ::std::option::Option<super::super::super::types::JobStatus>,
     pub(crate) max_results: ::std::option::Option<i32>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
-    pub(crate) filters: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>>,
+    pub(crate) filters: ::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>>,
 }
 impl ListJobsInputBuilder {
     /// <p>The name or full Amazon Resource Name (ARN) of the job queue used to list jobs.</p>
@@ -214,7 +214,7 @@
     /// <p>The job status used to filter jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exception is the <code>SHARE_IDENTIFIER</code> filter and <code>jobStatus</code> can be used together. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>Array job parents are updated to <code>PENDING</code> when any child job is updated to <code>RUNNABLE</code> and remain in <code>PENDING</code> status while child jobs are running. To view these jobs, filter by <code>PENDING</code> status until all child jobs reach a terminal state.</p>
     /// </note>
-    pub fn job_status(mut self, input: crate::types::JobStatus) -> Self {
+    pub fn job_status(mut self, input: super::super::super::types::JobStatus) -> Self {
         self.job_status = ::std::option::Option::Some(input);
         self
     }
@@ -221,7 +221,7 @@
     /// <p>The job status used to filter jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exception is the <code>SHARE_IDENTIFIER</code> filter and <code>jobStatus</code> can be used together. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>Array job parents are updated to <code>PENDING</code> when any child job is updated to <code>RUNNABLE</code> and remain in <code>PENDING</code> status while child jobs are running. To view these jobs, filter by <code>PENDING</code> status until all child jobs reach a terminal state.</p>
     /// </note>
-    pub fn set_job_status(mut self, input: ::std::option::Option<crate::types::JobStatus>) -> Self {
+    pub fn set_job_status(mut self, input: ::std::option::Option<super::super::super::types::JobStatus>) -> Self {
         self.job_status = input;
         self
     }
@@ -228,7 +228,7 @@
     /// <p>The job status used to filter jobs in the specified queue. If the <code>filters</code> parameter is specified, the <code>jobStatus</code> parameter is ignored and jobs with any status are returned. The exception is the <code>SHARE_IDENTIFIER</code> filter and <code>jobStatus</code> can be used together. If you don't specify a status, only <code>RUNNING</code> jobs are returned.</p><note>
     /// <p>Array job parents are updated to <code>PENDING</code> when any child job is updated to <code>RUNNABLE</code> and remain in <code>PENDING</code> status while child jobs are running. To view these jobs, filter by <code>PENDING</code> status until all child jobs reach a terminal state.</p>
     /// </note>
-    pub fn get_job_status(&self) -> &::std::option::Option<crate::types::JobStatus> {
+    pub fn get_job_status(&self) -> &::std::option::Option<super::super::super::types::JobStatus> {
         &self.job_status
     }
     /// <p>The maximum number of results returned by <code>ListJobs</code> in a paginated output. When this parameter is used, <code>ListJobs</code> returns up to <code>maxResults</code> results in a single page and a <code>nextToken</code> response element, if applicable. The remaining results of the initial request can be seen by sending another <code>ListJobs</code> request with the returned <code>nextToken</code> value.</p>
@@ -337,7 +337,7 @@
     /// <p>The value for the filter is the fairshare scheduling share identifier.</p>
     /// </dd>
     /// </dl>
-    pub fn filters(mut self, input: crate::types::KeyValuesPair) -> Self {
+    pub fn filters(mut self, input: super::super::super::types::KeyValuesPair) -> Self {
         let mut v = self.filters.unwrap_or_default();
         v.push(input);
         self.filters = ::std::option::Option::Some(v);
@@ -378,7 +378,7 @@
     /// <p>The value for the filter is the fairshare scheduling share identifier.</p>
     /// </dd>
     /// </dl>
-    pub fn set_filters(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>>) -> Self {
+    pub fn set_filters(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>>) -> Self {
         self.filters = input;
         self
     }
@@ -417,12 +417,12 @@
     /// <p>The value for the filter is the fairshare scheduling share identifier.</p>
     /// </dd>
     /// </dl>
-    pub fn get_filters(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::KeyValuesPair>> {
+    pub fn get_filters(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::KeyValuesPair>> {
         &self.filters
     }
     /// Consumes the builder and constructs a [`ListJobsInput`](crate::operation::list_jobs::ListJobsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_jobs::ListJobsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_jobs::ListJobsInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::list_jobs::ListJobsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_jobs::ListJobsInput {
             job_queue: self.job_queue,
             array_job_id: self.array_job_id,
             multi_node_job_id: self.multi_node_job_id,
