--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl GetConformancePackComplianceSummaryInput {
     /// Creates a new builder-style object to manufacture [`GetConformancePackComplianceSummaryInput`](crate::operation::get_conformance_pack_compliance_summary::GetConformancePackComplianceSummaryInput).
-    pub fn builder() -> crate::operation::get_conformance_pack_compliance_summary::builders::GetConformancePackComplianceSummaryInputBuilder {
-        crate::operation::get_conformance_pack_compliance_summary::builders::GetConformancePackComplianceSummaryInputBuilder::default()
+    pub fn builder() -> super::operation::get_conformance_pack_compliance_summary::builders::GetConformancePackComplianceSummaryInputBuilder {
+        super::operation::get_conformance_pack_compliance_summary::builders::GetConformancePackComplianceSummaryInputBuilder::default()
     }
 }

@@ -94,11 +94,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_conformance_pack_compliance_summary::GetConformancePackComplianceSummaryInput,
+        super::operation::get_conformance_pack_compliance_summary::GetConformancePackComplianceSummaryInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_conformance_pack_compliance_summary::GetConformancePackComplianceSummaryInput {
+            super::operation::get_conformance_pack_compliance_summary::GetConformancePackComplianceSummaryInput {
                 conformance_pack_names: self.conformance_pack_names,
                 limit: self.limit,
                 next_token: self.next_token,
