--- original
+++ modified
@@ -11,7 +11,7 @@
     #[allow(missing_docs)] // documentation missing in model
     pub message: ::std::option::Option<::std::string::String>,
     #[allow(missing_docs)] // documentation missing in model
-    pub reason: ::std::option::Option<crate::types::ThrottleReason>,
+    pub reason: ::std::option::Option<super::super::super::types::ThrottleReason>,
     pub(crate) meta: ::aws_smithy_types::error::ErrorMetadata,
 }
 impl TooManyRequestsException {
@@ -24,7 +24,7 @@
         self.r#type.as_deref()
     }
     #[allow(missing_docs)] // documentation missing in model
-    pub fn reason(&self) -> ::std::option::Option<&crate::types::ThrottleReason> {
+    pub fn reason(&self) -> ::std::option::Option<&super::super::super::types::ThrottleReason> {
         self.reason.as_ref()
     }
 }
@@ -46,7 +46,7 @@
     }
 }
 impl ::std::error::Error for TooManyRequestsException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::TooManyRequestsException {
+impl ::aws_types::request_id::RequestId for super::super::super::types::error::TooManyRequestsException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -59,8 +59,8 @@
 }
 impl TooManyRequestsException {
     /// Creates a new builder-style object to manufacture [`TooManyRequestsException`](crate::types::error::TooManyRequestsException).
-    pub fn builder() -> crate::types::error::builders::TooManyRequestsExceptionBuilder {
-        crate::types::error::builders::TooManyRequestsExceptionBuilder::default()
+    pub fn builder() -> super::super::super::types::error::builders::TooManyRequestsExceptionBuilder {
+        super::super::super::types::error::builders::TooManyRequestsExceptionBuilder::default()
     }
 }

@@ -71,7 +71,7 @@
     pub(crate) retry_after_seconds: ::std::option::Option<::std::string::String>,
     pub(crate) r#type: ::std::option::Option<::std::string::String>,
     pub(crate) message: ::std::option::Option<::std::string::String>,
-    pub(crate) reason: ::std::option::Option<crate::types::ThrottleReason>,
+    pub(crate) reason: ::std::option::Option<super::super::super::types::ThrottleReason>,
     meta: std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
 }
 impl TooManyRequestsExceptionBuilder {
@@ -118,17 +118,17 @@
         &self.message
     }
     #[allow(missing_docs)] // documentation missing in model
-    pub fn reason(mut self, input: crate::types::ThrottleReason) -> Self {
+    pub fn reason(mut self, input: super::super::super::types::ThrottleReason) -> Self {
         self.reason = ::std::option::Option::Some(input);
         self
     }
     #[allow(missing_docs)] // documentation missing in model
-    pub fn set_reason(mut self, input: ::std::option::Option<crate::types::ThrottleReason>) -> Self {
+    pub fn set_reason(mut self, input: ::std::option::Option<super::super::super::types::ThrottleReason>) -> Self {
         self.reason = input;
         self
     }
     #[allow(missing_docs)] // documentation missing in model
-    pub fn get_reason(&self) -> &::std::option::Option<crate::types::ThrottleReason> {
+    pub fn get_reason(&self) -> &::std::option::Option<super::super::super::types::ThrottleReason> {
         &self.reason
     }
     /// Sets error metadata
@@ -143,8 +143,8 @@
         self
     }
     /// Consumes the builder and constructs a [`TooManyRequestsException`](crate::types::error::TooManyRequestsException).
-    pub fn build(self) -> crate::types::error::TooManyRequestsException {
-        crate::types::error::TooManyRequestsException {
+    pub fn build(self) -> super::super::super::types::error::TooManyRequestsException {
+        super::super::super::types::error::TooManyRequestsException {
             retry_after_seconds: self.retry_after_seconds,
             r#type: self.r#type,
             message: self.message,
