--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetJobQueueSnapshotInput {
     /// Creates a new builder-style object to manufacture [`GetJobQueueSnapshotInput`](crate::operation::get_job_queue_snapshot::GetJobQueueSnapshotInput).
-    pub fn builder() -> crate::operation::get_job_queue_snapshot::builders::GetJobQueueSnapshotInputBuilder {
-        crate::operation::get_job_queue_snapshot::builders::GetJobQueueSnapshotInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_job_queue_snapshot::builders::GetJobQueueSnapshotInputBuilder {
+        super::super::super::operation::get_job_queue_snapshot::builders::GetJobQueueSnapshotInputBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
     /// Consumes the builder and constructs a [`GetJobQueueSnapshotInput`](crate::operation::get_job_queue_snapshot::GetJobQueueSnapshotInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_job_queue_snapshot::GetJobQueueSnapshotInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_job_queue_snapshot::GetJobQueueSnapshotInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_job_queue_snapshot::GetJobQueueSnapshotInput { job_queue: self.job_queue })
+        ::std::result::Result::Ok(super::super::super::operation::get_job_queue_snapshot::GetJobQueueSnapshotInput { job_queue: self.job_queue })
     }
 }
