--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DescribeIdentityProviderInput {
     /// Creates a new builder-style object to manufacture [`DescribeIdentityProviderInput`](crate::operation::describe_identity_provider::DescribeIdentityProviderInput).
-    pub fn builder() -> crate::operation::describe_identity_provider::builders::DescribeIdentityProviderInputBuilder {
-        crate::operation::describe_identity_provider::builders::DescribeIdentityProviderInputBuilder::default()
+    pub fn builder() -> super::operation::describe_identity_provider::builders::DescribeIdentityProviderInputBuilder {
+        super::operation::describe_identity_provider::builders::DescribeIdentityProviderInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_identity_provider::DescribeIdentityProviderInput,
+        super::operation::describe_identity_provider::DescribeIdentityProviderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_identity_provider::DescribeIdentityProviderInput {
+        ::std::result::Result::Ok(super::operation::describe_identity_provider::DescribeIdentityProviderInput {
             user_pool_id: self.user_pool_id,
             provider_name: self.provider_name,
         })
