--- original
+++ modified
@@ -8,7 +8,7 @@
     pub policy_arn: ::std::option::Option<::std::string::String>,
     /// <p>The entity type to use for filtering the results.</p>
     /// <p>For example, when <code>EntityFilter</code> is <code>Role</code>, only the roles that are attached to the specified policy are returned. This parameter is optional. If it is not included, all attached entities (users, groups, and roles) are returned. The argument for this parameter must be one of the valid values listed below.</p>
-    pub entity_filter: ::std::option::Option<crate::types::EntityType>,
+    pub entity_filter: ::std::option::Option<super::types::EntityType>,
     /// <p>The path prefix for filtering the results. This parameter is optional. If it is not included, it defaults to a slash (/), listing all entities.</p>
     /// <p>This parameter allows (through its <a href="http://wikipedia.org/wiki/regex">regex pattern</a>) a string of characters consisting of either a forward slash (/) by itself or a string that must begin and end with forward slashes. In addition, it can contain any ASCII character from the ! (<code>\u0021</code>) through the DEL character (<code>\u007F</code>), including most punctuation characters, digits, and upper and lowercased letters.</p>
     pub path_prefix: ::std::option::Option<::std::string::String>,
@@ -15,7 +15,7 @@
     /// <p>The policy usage method to use for filtering the results.</p>
     /// <p>To list only permissions policies, set <code>PolicyUsageFilter</code> to <code>PermissionsPolicy</code>. To list only the policies used to set permissions boundaries, set the value to <code>PermissionsBoundary</code>.</p>
     /// <p>This parameter is optional. If it is not included, all policies are returned.</p>
-    pub policy_usage_filter: ::std::option::Option<crate::types::PolicyUsageType>,
+    pub policy_usage_filter: ::std::option::Option<super::types::PolicyUsageType>,
     /// <p>Use this parameter only when paginating results and only after you receive a response indicating that the results are truncated. Set it to the value of the <code>Marker</code> element in the response that you received to indicate where the next call should start.</p>
     pub marker: ::std::option::Option<::std::string::String>,
     /// <p>Use this only when paginating results to indicate the maximum number of items you want in the response. If additional items exist beyond the maximum you specify, the <code>IsTruncated</code> response element is <code>true</code>.</p>
@@ -30,7 +30,7 @@
     }
     /// <p>The entity type to use for filtering the results.</p>
     /// <p>For example, when <code>EntityFilter</code> is <code>Role</code>, only the roles that are attached to the specified policy are returned. This parameter is optional. If it is not included, all attached entities (users, groups, and roles) are returned. The argument for this parameter must be one of the valid values listed below.</p>
-    pub fn entity_filter(&self) -> ::std::option::Option<&crate::types::EntityType> {
+    pub fn entity_filter(&self) -> ::std::option::Option<&super::types::EntityType> {
         self.entity_filter.as_ref()
     }
     /// <p>The path prefix for filtering the results. This parameter is optional. If it is not included, it defaults to a slash (/), listing all entities.</p>
@@ -41,7 +41,7 @@
     /// <p>The policy usage method to use for filtering the results.</p>
     /// <p>To list only permissions policies, set <code>PolicyUsageFilter</code> to <code>PermissionsPolicy</code>. To list only the policies used to set permissions boundaries, set the value to <code>PermissionsBoundary</code>.</p>
     /// <p>This parameter is optional. If it is not included, all policies are returned.</p>
-    pub fn policy_usage_filter(&self) -> ::std::option::Option<&crate::types::PolicyUsageType> {
+    pub fn policy_usage_filter(&self) -> ::std::option::Option<&super::types::PolicyUsageType> {
         self.policy_usage_filter.as_ref()
     }
     /// <p>Use this parameter only when paginating results and only after you receive a response indicating that the results are truncated. Set it to the value of the <code>Marker</code> element in the response that you received to indicate where the next call should start.</p>
@@ -56,8 +56,8 @@
 }
 impl ListEntitiesForPolicyInput {
     /// Creates a new builder-style object to manufacture [`ListEntitiesForPolicyInput`](crate::operation::list_entities_for_policy::ListEntitiesForPolicyInput).
-    pub fn builder() -> crate::operation::list_entities_for_policy::builders::ListEntitiesForPolicyInputBuilder {
-        crate::operation::list_entities_for_policy::builders::ListEntitiesForPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::list_entities_for_policy::builders::ListEntitiesForPolicyInputBuilder {
+        super::operation::list_entities_for_policy::builders::ListEntitiesForPolicyInputBuilder::default()
     }
 }

@@ -66,9 +66,9 @@
 #[non_exhaustive]
 pub struct ListEntitiesForPolicyInputBuilder {
     pub(crate) policy_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) entity_filter: ::std::option::Option<crate::types::EntityType>,
+    pub(crate) entity_filter: ::std::option::Option<super::types::EntityType>,
     pub(crate) path_prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) policy_usage_filter: ::std::option::Option<crate::types::PolicyUsageType>,
+    pub(crate) policy_usage_filter: ::std::option::Option<super::types::PolicyUsageType>,
     pub(crate) marker: ::std::option::Option<::std::string::String>,
     pub(crate) max_items: ::std::option::Option<i32>,
 }
@@ -93,19 +93,19 @@
     }
     /// <p>The entity type to use for filtering the results.</p>
     /// <p>For example, when <code>EntityFilter</code> is <code>Role</code>, only the roles that are attached to the specified policy are returned. This parameter is optional. If it is not included, all attached entities (users, groups, and roles) are returned. The argument for this parameter must be one of the valid values listed below.</p>
-    pub fn entity_filter(mut self, input: crate::types::EntityType) -> Self {
+    pub fn entity_filter(mut self, input: super::types::EntityType) -> Self {
         self.entity_filter = ::std::option::Option::Some(input);
         self
     }
     /// <p>The entity type to use for filtering the results.</p>
     /// <p>For example, when <code>EntityFilter</code> is <code>Role</code>, only the roles that are attached to the specified policy are returned. This parameter is optional. If it is not included, all attached entities (users, groups, and roles) are returned. The argument for this parameter must be one of the valid values listed below.</p>
-    pub fn set_entity_filter(mut self, input: ::std::option::Option<crate::types::EntityType>) -> Self {
+    pub fn set_entity_filter(mut self, input: ::std::option::Option<super::types::EntityType>) -> Self {
         self.entity_filter = input;
         self
     }
     /// <p>The entity type to use for filtering the results.</p>
     /// <p>For example, when <code>EntityFilter</code> is <code>Role</code>, only the roles that are attached to the specified policy are returned. This parameter is optional. If it is not included, all attached entities (users, groups, and roles) are returned. The argument for this parameter must be one of the valid values listed below.</p>
-    pub fn get_entity_filter(&self) -> &::std::option::Option<crate::types::EntityType> {
+    pub fn get_entity_filter(&self) -> &::std::option::Option<super::types::EntityType> {
         &self.entity_filter
     }
     /// <p>The path prefix for filtering the results. This parameter is optional. If it is not included, it defaults to a slash (/), listing all entities.</p>
@@ -128,7 +128,7 @@
     /// <p>The policy usage method to use for filtering the results.</p>
     /// <p>To list only permissions policies, set <code>PolicyUsageFilter</code> to <code>PermissionsPolicy</code>. To list only the policies used to set permissions boundaries, set the value to <code>PermissionsBoundary</code>.</p>
     /// <p>This parameter is optional. If it is not included, all policies are returned.</p>
-    pub fn policy_usage_filter(mut self, input: crate::types::PolicyUsageType) -> Self {
+    pub fn policy_usage_filter(mut self, input: super::types::PolicyUsageType) -> Self {
         self.policy_usage_filter = ::std::option::Option::Some(input);
         self
     }
@@ -135,7 +135,7 @@
     /// <p>The policy usage method to use for filtering the results.</p>
     /// <p>To list only permissions policies, set <code>PolicyUsageFilter</code> to <code>PermissionsPolicy</code>. To list only the policies used to set permissions boundaries, set the value to <code>PermissionsBoundary</code>.</p>
     /// <p>This parameter is optional. If it is not included, all policies are returned.</p>
-    pub fn set_policy_usage_filter(mut self, input: ::std::option::Option<crate::types::PolicyUsageType>) -> Self {
+    pub fn set_policy_usage_filter(mut self, input: ::std::option::Option<super::types::PolicyUsageType>) -> Self {
         self.policy_usage_filter = input;
         self
     }
@@ -142,7 +142,7 @@
     /// <p>The policy usage method to use for filtering the results.</p>
     /// <p>To list only permissions policies, set <code>PolicyUsageFilter</code> to <code>PermissionsPolicy</code>. To list only the policies used to set permissions boundaries, set the value to <code>PermissionsBoundary</code>.</p>
     /// <p>This parameter is optional. If it is not included, all policies are returned.</p>
-    pub fn get_policy_usage_filter(&self) -> &::std::option::Option<crate::types::PolicyUsageType> {
+    pub fn get_policy_usage_filter(&self) -> &::std::option::Option<super::types::PolicyUsageType> {
         &self.policy_usage_filter
     }
     /// <p>Use this parameter only when paginating results and only after you receive a response indicating that the results are truncated. Set it to the value of the <code>Marker</code> element in the response that you received to indicate where the next call should start.</p>
@@ -179,9 +179,9 @@
     /// Consumes the builder and constructs a [`ListEntitiesForPolicyInput`](crate::operation::list_entities_for_policy::ListEntitiesForPolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_entities_for_policy::ListEntitiesForPolicyInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_entities_for_policy::ListEntitiesForPolicyInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_entities_for_policy::ListEntitiesForPolicyInput {
+        ::std::result::Result::Ok(super::operation::list_entities_for_policy::ListEntitiesForPolicyInput {
             policy_arn: self.policy_arn,
             entity_filter: self.entity_filter,
             path_prefix: self.path_prefix,
