--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl ListEmailIdentitiesInput {
     /// Creates a new builder-style object to manufacture [`ListEmailIdentitiesInput`](crate::operation::list_email_identities::ListEmailIdentitiesInput).
-    pub fn builder() -> crate::operation::list_email_identities::builders::ListEmailIdentitiesInputBuilder {
-        crate::operation::list_email_identities::builders::ListEmailIdentitiesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_email_identities::builders::ListEmailIdentitiesInputBuilder {
+        super::super::super::operation::list_email_identities::builders::ListEmailIdentitiesInputBuilder::default()
     }
 }

@@ -70,9 +70,9 @@
     /// Consumes the builder and constructs a [`ListEmailIdentitiesInput`](crate::operation::list_email_identities::ListEmailIdentitiesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_email_identities::ListEmailIdentitiesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_email_identities::ListEmailIdentitiesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_email_identities::ListEmailIdentitiesInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_email_identities::ListEmailIdentitiesInput {
             next_token: self.next_token,
             page_size: self.page_size,
         })
