--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct DeleteAccountPasswordPolicyInput {}
 impl DeleteAccountPasswordPolicyInput {
     /// Creates a new builder-style object to manufacture [`DeleteAccountPasswordPolicyInput`](crate::operation::delete_account_password_policy::DeleteAccountPasswordPolicyInput).
-    pub fn builder() -> crate::operation::delete_account_password_policy::builders::DeleteAccountPasswordPolicyInputBuilder {
-        crate::operation::delete_account_password_policy::builders::DeleteAccountPasswordPolicyInputBuilder::default()
+    pub fn builder() -> super::operation::delete_account_password_policy::builders::DeleteAccountPasswordPolicyInputBuilder {
+        super::operation::delete_account_password_policy::builders::DeleteAccountPasswordPolicyInputBuilder::default()
     }
 }

@@ -19,9 +19,9 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_account_password_policy::DeleteAccountPasswordPolicyInput,
+        super::operation::delete_account_password_policy::DeleteAccountPasswordPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_account_password_policy::DeleteAccountPasswordPolicyInput {})
+        ::std::result::Result::Ok(super::operation::delete_account_password_policy::DeleteAccountPasswordPolicyInput {})
     }
 }
