--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl VerifySmsSandboxPhoneNumberInput {
     /// Creates a new builder-style object to manufacture [`VerifySmsSandboxPhoneNumberInput`](crate::operation::verify_sms_sandbox_phone_number::VerifySmsSandboxPhoneNumberInput).
-    pub fn builder() -> crate::operation::verify_sms_sandbox_phone_number::builders::VerifySmsSandboxPhoneNumberInputBuilder {
-        crate::operation::verify_sms_sandbox_phone_number::builders::VerifySmsSandboxPhoneNumberInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::verify_sms_sandbox_phone_number::builders::VerifySmsSandboxPhoneNumberInputBuilder {
+        super::super::super::operation::verify_sms_sandbox_phone_number::builders::VerifySmsSandboxPhoneNumberInputBuilder::default()
     }
 }

@@ -75,10 +75,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::verify_sms_sandbox_phone_number::VerifySmsSandboxPhoneNumberInput,
+        super::super::super::operation::verify_sms_sandbox_phone_number::VerifySmsSandboxPhoneNumberInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::verify_sms_sandbox_phone_number::VerifySmsSandboxPhoneNumberInput {
+        ::std::result::Result::Ok(super::super::super::operation::verify_sms_sandbox_phone_number::VerifySmsSandboxPhoneNumberInput {
             phone_number: self.phone_number,
             one_time_password: self.one_time_password,
         })
