--- original
+++ modified
@@ -7,7 +7,7 @@
     /// - On success, responds with [`DescribeTableOutput`](crate::operation::describe_table::DescribeTableOutput) with field(s):
     ///   - [`table(Option<TableDescription>)`](crate::operation::describe_table::DescribeTableOutput::table): <p>The properties of the table.</p>
     /// - On failure, responds with [`SdkError<DescribeTableError>`](crate::operation::describe_table::DescribeTableError)
-    pub fn describe_table(&self) -> crate::operation::describe_table::builders::DescribeTableFluentBuilder {
-        crate::operation::describe_table::builders::DescribeTableFluentBuilder::new(self.handle.clone())
+    pub fn describe_table(&self) -> super::super::operation::describe_table::builders::DescribeTableFluentBuilder {
+        super::super::operation::describe_table::builders::DescribeTableFluentBuilder::new(self.handle.clone())
     }
 }
