--- original
+++ modified
@@ -6,7 +6,7 @@
     /// - On success, responds with [`DescribeEndpointsOutput`](crate::operation::describe_endpoints::DescribeEndpointsOutput) with field(s):
     ///   - [`endpoints(Vec::<Endpoint>)`](crate::operation::describe_endpoints::DescribeEndpointsOutput::endpoints): <p>List of endpoints.</p>
     /// - On failure, responds with [`SdkError<DescribeEndpointsError>`](crate::operation::describe_endpoints::DescribeEndpointsError)
-    pub fn describe_endpoints(&self) -> crate::operation::describe_endpoints::builders::DescribeEndpointsFluentBuilder {
-        crate::operation::describe_endpoints::builders::DescribeEndpointsFluentBuilder::new(self.handle.clone())
+    pub fn describe_endpoints(&self) -> super::operation::describe_endpoints::builders::DescribeEndpointsFluentBuilder {
+        super::operation::describe_endpoints::builders::DescribeEndpointsFluentBuilder::new(self.handle.clone())
     }
 }
