--- original
+++ modified
@@ -12,7 +12,7 @@
     /// - On success, responds with [`ImportTableOutput`](crate::operation::import_table::ImportTableOutput) with field(s):
     ///   - [`import_table_description(Option<ImportTableDescription>)`](crate::operation::import_table::ImportTableOutput::import_table_description): <p>Represents the properties of the table created for the import, and parameters of the import. The import parameters include import status, how many items were processed, and how many errors were encountered.</p>
     /// - On failure, responds with [`SdkError<ImportTableError>`](crate::operation::import_table::ImportTableError)
-    pub fn import_table(&self) -> crate::operation::import_table::builders::ImportTableFluentBuilder {
-        crate::operation::import_table::builders::ImportTableFluentBuilder::new(self.handle.clone())
+    pub fn import_table(&self) -> super::super::operation::import_table::builders::ImportTableFluentBuilder {
+        super::super::operation::import_table::builders::ImportTableFluentBuilder::new(self.handle.clone())
     }
 }
