--- original
+++ modified
@@ -13,7 +13,7 @@
     ///   - [`backup_summaries(Option<Vec::<BackupSummary>>)`](crate::operation::list_backups::ListBackupsOutput::backup_summaries): <p>List of <code>BackupSummary</code> objects.</p>
     ///   - [`last_evaluated_backup_arn(Option<String>)`](crate::operation::list_backups::ListBackupsOutput::last_evaluated_backup_arn): <p>The ARN of the backup last evaluated when the current page of results was returned, inclusive of the current page of results. This value may be specified as the <code>ExclusiveStartBackupArn</code> of a new <code>ListBackups</code> operation in order to fetch the next page of results.</p> <p>If <code>LastEvaluatedBackupArn</code> is empty, then the last page of results has been processed and there are no more results to be retrieved.</p> <p>If <code>LastEvaluatedBackupArn</code> is not empty, this may or may not indicate that there is more data to be returned. All results are guaranteed to have been returned if and only if no value for <code>LastEvaluatedBackupArn</code> is returned.</p>
     /// - On failure, responds with [`SdkError<ListBackupsError>`](crate::operation::list_backups::ListBackupsError)
-    pub fn list_backups(&self) -> crate::operation::list_backups::builders::ListBackupsFluentBuilder {
-        crate::operation::list_backups::builders::ListBackupsFluentBuilder::new(self.handle.clone())
+    pub fn list_backups(&self) -> super::super::operation::list_backups::builders::ListBackupsFluentBuilder {
+        super::super::operation::list_backups::builders::ListBackupsFluentBuilder::new(self.handle.clone())
     }
 }
