--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`consumed_capacity(Option<Vec::<ConsumedCapacity>>)`](crate::operation::transact_get_items::TransactGetItemsOutput::consumed_capacity): <p>If the <i>ReturnConsumedCapacity</i> value was <code>TOTAL</code>, this is an array of <code>ConsumedCapacity</code> objects, one for each table addressed by <code>TransactGetItem</code> objects in the <i>TransactItems</i> parameter. These <code>ConsumedCapacity</code> objects report the read-capacity units consumed by the <code>TransactGetItems</code> call in that table.</p>
     ///   - [`responses(Option<Vec::<ItemResponse>>)`](crate::operation::transact_get_items::TransactGetItemsOutput::responses): <p>An ordered array of up to 100 <code>ItemResponse</code> objects, each of which corresponds to the <code>TransactGetItem</code> object in the same position in the <i>TransactItems</i> array. Each <code>ItemResponse</code> object contains a Map of the name-value pairs that are the projected attributes of the requested item.</p> <p>If a requested item could not be retrieved, the corresponding <code>ItemResponse</code> object is Null, or if the requested item has no projected attributes, the corresponding <code>ItemResponse</code> object is an empty Map.</p>
     /// - On failure, responds with [`SdkError<TransactGetItemsError>`](crate::operation::transact_get_items::TransactGetItemsError)
-    pub fn transact_get_items(&self) -> crate::operation::transact_get_items::builders::TransactGetItemsFluentBuilder {
-        crate::operation::transact_get_items::builders::TransactGetItemsFluentBuilder::new(self.handle.clone())
+    pub fn transact_get_items(&self) -> super::operation::transact_get_items::builders::TransactGetItemsFluentBuilder {
+        super::operation::transact_get_items::builders::TransactGetItemsFluentBuilder::new(self.handle.clone())
     }
 }
