--- original
+++ modified
@@ -13,7 +13,7 @@
     ///   - [`contributor_insights_status(Option<ContributorInsightsStatus>)`](crate::operation::update_contributor_insights::UpdateContributorInsightsOutput::contributor_insights_status): <p>The status of contributor insights</p>
     ///   - [`contributor_insights_mode(Option<ContributorInsightsMode>)`](crate::operation::update_contributor_insights::UpdateContributorInsightsOutput::contributor_insights_mode): <p>The updated mode of CloudWatch Contributor Insights that determines whether to monitor all access and throttled events or to track throttled events exclusively.</p>
     /// - On failure, responds with [`SdkError<UpdateContributorInsightsError>`](crate::operation::update_contributor_insights::UpdateContributorInsightsError)
-    pub fn update_contributor_insights(&self) -> crate::operation::update_contributor_insights::builders::UpdateContributorInsightsFluentBuilder {
-        crate::operation::update_contributor_insights::builders::UpdateContributorInsightsFluentBuilder::new(self.handle.clone())
+    pub fn update_contributor_insights(&self) -> super::operation::update_contributor_insights::builders::UpdateContributorInsightsFluentBuilder {
+        super::operation::update_contributor_insights::builders::UpdateContributorInsightsFluentBuilder::new(self.handle.clone())
     }
 }
