--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetEmailAddressInsightsInput {
     /// Creates a new builder-style object to manufacture [`GetEmailAddressInsightsInput`](crate::operation::get_email_address_insights::GetEmailAddressInsightsInput).
-    pub fn builder() -> crate::operation::get_email_address_insights::builders::GetEmailAddressInsightsInputBuilder {
-        crate::operation::get_email_address_insights::builders::GetEmailAddressInsightsInputBuilder::default()
+    pub fn builder() -> super::operation::get_email_address_insights::builders::GetEmailAddressInsightsInputBuilder {
+        super::operation::get_email_address_insights::builders::GetEmailAddressInsightsInputBuilder::default()
     }
 }

@@ -46,10 +46,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_email_address_insights::GetEmailAddressInsightsInput,
+        super::operation::get_email_address_insights::GetEmailAddressInsightsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_email_address_insights::GetEmailAddressInsightsInput {
+        ::std::result::Result::Ok(super::operation::get_email_address_insights::GetEmailAddressInsightsInput {
             email_address: self.email_address,
         })
     }
