--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetSigningCertificateInput {
     /// Creates a new builder-style object to manufacture [`GetSigningCertificateInput`](crate::operation::get_signing_certificate::GetSigningCertificateInput).
-    pub fn builder() -> crate::operation::get_signing_certificate::builders::GetSigningCertificateInputBuilder {
-        crate::operation::get_signing_certificate::builders::GetSigningCertificateInputBuilder::default()
+    pub fn builder() -> super::operation::get_signing_certificate::builders::GetSigningCertificateInputBuilder {
+        super::operation::get_signing_certificate::builders::GetSigningCertificateInputBuilder::default()
     }
 }

@@ -45,9 +45,9 @@
     /// Consumes the builder and constructs a [`GetSigningCertificateInput`](crate::operation::get_signing_certificate::GetSigningCertificateInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_signing_certificate::GetSigningCertificateInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_signing_certificate::GetSigningCertificateInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_signing_certificate::GetSigningCertificateInput {
+        ::std::result::Result::Ok(super::operation::get_signing_certificate::GetSigningCertificateInput {
             user_pool_id: self.user_pool_id,
         })
     }
