--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DecodeAuthorizationMessageInput {
     /// Creates a new builder-style object to manufacture [`DecodeAuthorizationMessageInput`](crate::operation::decode_authorization_message::DecodeAuthorizationMessageInput).
-    pub fn builder() -> crate::operation::decode_authorization_message::builders::DecodeAuthorizationMessageInputBuilder {
-        crate::operation::decode_authorization_message::builders::DecodeAuthorizationMessageInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::decode_authorization_message::builders::DecodeAuthorizationMessageInputBuilder {
+        super::super::super::operation::decode_authorization_message::builders::DecodeAuthorizationMessageInputBuilder::default()
     }
 }

@@ -45,10 +45,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::decode_authorization_message::DecodeAuthorizationMessageInput,
+        super::super::super::operation::decode_authorization_message::DecodeAuthorizationMessageInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::decode_authorization_message::DecodeAuthorizationMessageInput {
+        ::std::result::Result::Ok(super::super::super::operation::decode_authorization_message::DecodeAuthorizationMessageInput {
             encoded_message: self.encoded_message,
         })
     }
