--- original
+++ modified
@@ -18,8 +18,8 @@
 }
 impl GetLoginProfileInput {
     /// Creates a new builder-style object to manufacture [`GetLoginProfileInput`](crate::operation::get_login_profile::GetLoginProfileInput).
-    pub fn builder() -> crate::operation::get_login_profile::builders::GetLoginProfileInputBuilder {
-        crate::operation::get_login_profile::builders::GetLoginProfileInputBuilder::default()
+    pub fn builder() -> super::operation::get_login_profile::builders::GetLoginProfileInputBuilder {
+        super::operation::get_login_profile::builders::GetLoginProfileInputBuilder::default()
     }
 }

@@ -53,7 +53,7 @@
     /// Consumes the builder and constructs a [`GetLoginProfileInput`](crate::operation::get_login_profile::GetLoginProfileInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_login_profile::GetLoginProfileInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_login_profile::GetLoginProfileInput { user_name: self.user_name })
+    ) -> ::std::result::Result<super::operation::get_login_profile::GetLoginProfileInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::get_login_profile::GetLoginProfileInput { user_name: self.user_name })
     }
 }
