--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl ListGlobalTablesInput {
     /// Creates a new builder-style object to manufacture [`ListGlobalTablesInput`](crate::operation::list_global_tables::ListGlobalTablesInput).
-    pub fn builder() -> crate::operation::list_global_tables::builders::ListGlobalTablesInputBuilder {
-        crate::operation::list_global_tables::builders::ListGlobalTablesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_global_tables::builders::ListGlobalTablesInputBuilder {
+        super::super::super::operation::list_global_tables::builders::ListGlobalTablesInputBuilder::default()
     }
 }

@@ -90,8 +90,8 @@
     /// Consumes the builder and constructs a [`ListGlobalTablesInput`](crate::operation::list_global_tables::ListGlobalTablesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_global_tables::ListGlobalTablesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_global_tables::ListGlobalTablesInput {
+    ) -> ::std::result::Result<super::super::super::operation::list_global_tables::ListGlobalTablesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_global_tables::ListGlobalTablesInput {
             exclusive_start_global_table_name: self.exclusive_start_global_table_name,
             limit: self.limit,
             region_name: self.region_name,
