--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::publish_package_version::PublishPackageVersionInput,
+        input: super::operation::publish_package_version::PublishPackageVersionInput,
     ) -> ::std::result::Result<
-        crate::operation::publish_package_version::PublishPackageVersionOutput,
+        super::operation::publish_package_version::PublishPackageVersionOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::publish_package_version::PublishPackageVersionError,
+            super::operation::publish_package_version::PublishPackageVersionError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::publish_package_version::PublishPackageVersionError>()
+                err.downcast::<super::operation::publish_package_version::PublishPackageVersionError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::publish_package_version::PublishPackageVersionOutput>()
+                .downcast::<super::operation::publish_package_version::PublishPackageVersionOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::publish_package_version::PublishPackageVersionInput,
+        input: super::operation::publish_package_version::PublishPackageVersionInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("PublishPackageVersion")
                 .build()
                 .expect("required fields set"),
@@ -142,13 +142,13 @@
                 PublishPackageVersionEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::publish_package_version::PublishPackageVersionError,
+                super::operation::publish_package_version::PublishPackageVersionError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::publish_package_version::PublishPackageVersionError,
+                super::operation::publish_package_version::PublishPackageVersionError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::publish_package_version::PublishPackageVersionError,
+                super::operation::publish_package_version::PublishPackageVersionError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -248,11 +248,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_publish_package_version::de_publish_package_version_http_error(status, headers, body)
+            super::protocol_serde::shape_publish_package_version::de_publish_package_version_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_publish_package_version::de_publish_package_version_http_response(status, headers, body)
+            super::protocol_serde::shape_publish_package_version::de_publish_package_version_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -265,16 +265,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::publish_package_version::PublishPackageVersionInput>()
+            .downcast::<super::operation::publish_package_version::PublishPackageVersionInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::publish_package_version::PublishPackageVersionInput,
+                _input: &super::operation::publish_package_version::PublishPackageVersionInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -282,7 +282,7 @@
                 ::std::result::Result::Ok(())
             }
             fn uri_query(
-                _input: &crate::operation::publish_package_version::PublishPackageVersionInput,
+                _input: &super::operation::publish_package_version::PublishPackageVersionInput,
                 mut output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 let mut query = ::aws_smithy_http::query::Writer::new(output);
@@ -365,13 +365,13 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::publish_package_version::PublishPackageVersionInput,
+                input: &super::operation::publish_package_version::PublishPackageVersionInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
                 uri_base(input, &mut uri)?;
                 uri_query(input, &mut uri)?;
-                let builder = crate::protocol_serde::shape_publish_package_version::ser_publish_package_version_headers(input, builder)?;
+                let builder = super::protocol_serde::shape_publish_package_version::ser_publish_package_version_headers(input, builder)?;
                 ::std::result::Result::Ok(builder.method("POST").uri(uri))
             }
             let mut builder = update_http_builder(&input, ::http_1x::request::Builder::new())?;
@@ -378,7 +378,7 @@
             builder = _header_serialization_settings.set_default_header(builder, ::http_1x::header::CONTENT_TYPE, "application/octet-stream");
             builder
         };
-        let body = crate::protocol_serde::shape_publish_package_version_input::ser_asset_content_http_payload(input.asset_content)?.into_inner();
+        let body = super::protocol_serde::shape_publish_package_version_input::ser_asset_content_http_payload(input.asset_content)?.into_inner();
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
             request_builder = _header_serialization_settings.set_default_header(request_builder, ::http_1x::header::CONTENT_LENGTH, &content_length);
@@ -410,7 +410,7 @@
             .downcast_ref::<PublishPackageVersionInput>()
             .ok_or("failed to downcast to PublishPackageVersionInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -433,19 +433,19 @@
 #[derive(::std::fmt::Debug)]
 pub enum PublishPackageVersionError {
     /// <p>The operation did not succeed because of an unauthorized access attempt.</p>
-    AccessDeniedException(crate::types::error::AccessDeniedException),
+    AccessDeniedException(super::types::error::AccessDeniedException),
     /// <p>The operation did not succeed because prerequisites are not met.</p>
-    ConflictException(crate::types::error::ConflictException),
+    ConflictException(super::types::error::ConflictException),
     /// <p>The operation did not succeed because of an error that occurred inside CodeArtifact.</p>
-    InternalServerException(crate::types::error::InternalServerException),
+    InternalServerException(super::types::error::InternalServerException),
     /// <p>The operation did not succeed because the resource requested is not found in the service.</p>
-    ResourceNotFoundException(crate::types::error::ResourceNotFoundException),
+    ResourceNotFoundException(super::types::error::ResourceNotFoundException),
     /// <p>The operation did not succeed because it would have exceeded a service limit for your account.</p>
-    ServiceQuotaExceededException(crate::types::error::ServiceQuotaExceededException),
+    ServiceQuotaExceededException(super::types::error::ServiceQuotaExceededException),
     /// <p>The operation did not succeed because too many requests are sent to the service.</p>
-    ThrottlingException(crate::types::error::ThrottlingException),
+    ThrottlingException(super::types::error::ThrottlingException),
     /// <p>The operation did not succeed because a parameter in the request was sent with an invalid value.</p>
-    ValidationException(crate::types::error::ValidationException),
+    ValidationException(super::types::error::ValidationException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -453,7 +453,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-PublishPackageVersionError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl PublishPackageVersionError {
     /// Creates the `PublishPackageVersionError::Unhandled` variant from any error type.
@@ -460,7 +460,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -468,7 +468,7 @@

     /// Creates the `PublishPackageVersionError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -579,21 +579,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::publish_package_version::PublishPackageVersionError {
+impl ::aws_types::request_id::RequestId for super::operation::publish_package_version::PublishPackageVersionError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::publish_package_version::_publish_package_version_input::PublishPackageVersionInput;
+pub use super::operation::publish_package_version::_publish_package_version_input::PublishPackageVersionInput;

-pub use crate::operation::publish_package_version::_publish_package_version_output::PublishPackageVersionOutput;
+pub use super::operation::publish_package_version::_publish_package_version_output::PublishPackageVersionOutput;

 mod _publish_package_version_input;

