--- original
+++ modified
@@ -13,12 +13,12 @@
 ///
 #[derive(::std::clone::Clone, ::std::fmt::Debug)]
 pub struct ExportCompletedFluentBuilder {
-    handle: ::std::sync::Arc<crate::client::Handle>,
-    inner: crate::operation::describe_export::builders::DescribeExportInputBuilder,
+    handle: ::std::sync::Arc<super::super::client::Handle>,
+    inner: super::super::operation::describe_export::builders::DescribeExportInputBuilder,
 }
 impl ExportCompletedFluentBuilder {
     /// Creates a new `ExportCompletedFluentBuilder`.
-    pub(crate) fn new(handle: ::std::sync::Arc<crate::client::Handle>) -> Self {
+    pub(crate) fn new(handle: ::std::sync::Arc<super::super::client::Handle>) -> Self {
         Self {
             handle,
             inner: ::std::default::Default::default(),
@@ -25,7 +25,7 @@
         }
     }
     /// Access the DescribeExport as a reference.
-    pub fn as_input(&self) -> &crate::operation::describe_export::builders::DescribeExportInputBuilder {
+    pub fn as_input(&self) -> &super::super::operation::describe_export::builders::DescribeExportInputBuilder {
         &self.inner
     }
     /// Wait for `export_completed`
@@ -33,19 +33,19 @@
         self,
         max_wait: ::std::time::Duration,
     ) -> ::std::result::Result<
-        crate::waiters::export_completed::ExportCompletedFinalPoll,
-        crate::waiters::export_completed::WaitUntilExportCompletedError,
+        super::super::waiters::export_completed::ExportCompletedFinalPoll,
+        super::super::waiters::export_completed::WaitUntilExportCompletedError,
     > {
         let input = self
             .inner
             .build()
             .map_err(::aws_smithy_runtime_api::client::waiters::error::WaiterError::construction_failure)?;
-        let runtime_plugins = crate::operation::describe_export::DescribeExport::operation_runtime_plugins(
+        let runtime_plugins = super::super::operation::describe_export::DescribeExport::operation_runtime_plugins(
             self.handle.runtime_plugins.clone(),
             &self.handle.conf,
             ::std::option::Option::None,
         )
-        .with_operation_plugin(crate::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
+        .with_operation_plugin(super::super::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
         let mut cfg = ::aws_smithy_types::config_bag::ConfigBag::base();
         let runtime_components_builder = runtime_plugins
             .apply_client_configuration(&mut cfg)
@@ -55,15 +55,15 @@
         let time_source = time_components.time_source().expect("a time source is required by waiters");

         let acceptor = move |result: ::std::result::Result<
-            &crate::operation::describe_export::DescribeExportOutput,
-            &crate::operation::describe_export::DescribeExportError,
+            &super::super::operation::describe_export::DescribeExportOutput,
+            &super::super::operation::describe_export::DescribeExportError,
         >| {
             // Matches: {"output":{"path":"ExportDescription.ExportStatus","expected":"COMPLETED","comparator":"stringEquals"}}
-            if crate::waiters::matchers::match_describe_export_a4219f96bc64309c4(result) {
+            if super::super::waiters::matchers::match_describe_export_a4219f96bc64309c4(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Success;
             }
             // Matches: {"output":{"path":"ExportDescription.ExportStatus","expected":"FAILED","comparator":"stringEquals"}}
-            if crate::waiters::matchers::match_describe_export_8d899fd0681b891e9(result) {
+            if super::super::waiters::matchers::match_describe_export_8d899fd0681b891e9(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Failure;
             }
             ::aws_smithy_runtime::client::waiters::AcceptorState::NoAcceptorsMatched
@@ -71,7 +71,7 @@
         let operation = move || {
             let input = input.clone();
             let runtime_plugins = runtime_plugins.clone();
-            async move { crate::operation::describe_export::DescribeExport::orchestrate(&runtime_plugins, input).await }
+            async move { super::super::operation::describe_export::DescribeExport::orchestrate(&runtime_plugins, input).await }
         };
         let orchestrator = ::aws_smithy_runtime::client::waiters::WaiterOrchestrator::builder()
             .min_delay(::std::time::Duration::from_secs(20))
@@ -102,9 +102,9 @@

 /// Successful return type for the `export_completed` waiter.
 pub type ExportCompletedFinalPoll = ::aws_smithy_runtime_api::client::waiters::FinalPoll<
-    crate::operation::describe_export::DescribeExportOutput,
+    super::super::operation::describe_export::DescribeExportOutput,
     ::aws_smithy_runtime_api::client::result::SdkError<
-        crate::operation::describe_export::DescribeExportError,
+        super::super::operation::describe_export::DescribeExportError,
         ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
     >,
 >;
@@ -111,6 +111,6 @@

 /// Error type for the `export_completed` waiter.
 pub type WaitUntilExportCompletedError = ::aws_smithy_runtime_api::client::waiters::error::WaiterError<
-    crate::operation::describe_export::DescribeExportOutput,
-    crate::operation::describe_export::DescribeExportError,
+    super::super::operation::describe_export::DescribeExportOutput,
+    super::super::operation::describe_export::DescribeExportError,
 >;
