--- original
+++ modified
@@ -13,12 +13,12 @@
 ///
 #[derive(::std::clone::Clone, ::std::fmt::Debug)]
 pub struct TableExistsFluentBuilder {
-    handle: ::std::sync::Arc<crate::client::Handle>,
-    inner: crate::operation::describe_table::builders::DescribeTableInputBuilder,
+    handle: ::std::sync::Arc<super::client::Handle>,
+    inner: super::operation::describe_table::builders::DescribeTableInputBuilder,
 }
 impl TableExistsFluentBuilder {
     /// Creates a new `TableExistsFluentBuilder`.
-    pub(crate) fn new(handle: ::std::sync::Arc<crate::client::Handle>) -> Self {
+    pub(crate) fn new(handle: ::std::sync::Arc<super::client::Handle>) -> Self {
         Self {
             handle,
             inner: ::std::default::Default::default(),
@@ -25,7 +25,7 @@
         }
     }
     /// Access the DescribeTable as a reference.
-    pub fn as_input(&self) -> &crate::operation::describe_table::builders::DescribeTableInputBuilder {
+    pub fn as_input(&self) -> &super::operation::describe_table::builders::DescribeTableInputBuilder {
         &self.inner
     }
     /// Wait for `table_exists`
@@ -32,17 +32,17 @@
     pub async fn wait(
         self,
         max_wait: ::std::time::Duration,
-    ) -> ::std::result::Result<crate::waiters::table_exists::TableExistsFinalPoll, crate::waiters::table_exists::WaitUntilTableExistsError> {
+    ) -> ::std::result::Result<super::waiters::table_exists::TableExistsFinalPoll, super::waiters::table_exists::WaitUntilTableExistsError> {
         let input = self
             .inner
             .build()
             .map_err(::aws_smithy_runtime_api::client::waiters::error::WaiterError::construction_failure)?;
-        let runtime_plugins = crate::operation::describe_table::DescribeTable::operation_runtime_plugins(
+        let runtime_plugins = super::operation::describe_table::DescribeTable::operation_runtime_plugins(
             self.handle.runtime_plugins.clone(),
             &self.handle.conf,
             ::std::option::Option::None,
         )
-        .with_operation_plugin(crate::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
+        .with_operation_plugin(super::sdk_feature_tracker::waiter::WaiterFeatureTrackerRuntimePlugin::new());
         let mut cfg = ::aws_smithy_types::config_bag::ConfigBag::base();
         let runtime_components_builder = runtime_plugins
             .apply_client_configuration(&mut cfg)
@@ -52,15 +52,15 @@
         let time_source = time_components.time_source().expect("a time source is required by waiters");

         let acceptor = move |result: ::std::result::Result<
-            &crate::operation::describe_table::DescribeTableOutput,
-            &crate::operation::describe_table::DescribeTableError,
+            &super::operation::describe_table::DescribeTableOutput,
+            &super::operation::describe_table::DescribeTableError,
         >| {
             // Matches: {"output":{"path":"Table.TableStatus","expected":"ACTIVE","comparator":"stringEquals"}}
-            if crate::waiters::matchers::match_describe_table_0429b99996ae6dab6(result) {
+            if super::waiters::matchers::match_describe_table_0429b99996ae6dab6(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Success;
             }
             // Matches: {"errorType":"ResourceNotFoundException"}
-            if crate::waiters::matchers::match_describe_table_1cce2c05524fb92d4(result) {
+            if super::waiters::matchers::match_describe_table_1cce2c05524fb92d4(result) {
                 return ::aws_smithy_runtime::client::waiters::AcceptorState::Retry;
             }
             ::aws_smithy_runtime::client::waiters::AcceptorState::NoAcceptorsMatched
@@ -68,7 +68,7 @@
         let operation = move || {
             let input = input.clone();
             let runtime_plugins = runtime_plugins.clone();
-            async move { crate::operation::describe_table::DescribeTable::orchestrate(&runtime_plugins, input).await }
+            async move { super::operation::describe_table::DescribeTable::orchestrate(&runtime_plugins, input).await }
         };
         let orchestrator = ::aws_smithy_runtime::client::waiters::WaiterOrchestrator::builder()
             .min_delay(::std::time::Duration::from_secs(20))
@@ -99,9 +99,9 @@

 /// Successful return type for the `table_exists` waiter.
 pub type TableExistsFinalPoll = ::aws_smithy_runtime_api::client::waiters::FinalPoll<
-    crate::operation::describe_table::DescribeTableOutput,
+    super::operation::describe_table::DescribeTableOutput,
     ::aws_smithy_runtime_api::client::result::SdkError<
-        crate::operation::describe_table::DescribeTableError,
+        super::operation::describe_table::DescribeTableError,
         ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
     >,
 >;
@@ -108,6 +108,6 @@

 /// Error type for the `table_exists` waiter.
 pub type WaitUntilTableExistsError = ::aws_smithy_runtime_api::client::waiters::error::WaiterError<
-    crate::operation::describe_table::DescribeTableOutput,
-    crate::operation::describe_table::DescribeTableError,
+    super::operation::describe_table::DescribeTableOutput,
+    super::operation::describe_table::DescribeTableError,
 >;
