--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListOpenIdConnectProviderTagsInput {
     /// Creates a new builder-style object to manufacture [`ListOpenIdConnectProviderTagsInput`](crate::operation::list_open_id_connect_provider_tags::ListOpenIdConnectProviderTagsInput).
-    pub fn builder() -> crate::operation::list_open_id_connect_provider_tags::builders::ListOpenIdConnectProviderTagsInputBuilder {
-        crate::operation::list_open_id_connect_provider_tags::builders::ListOpenIdConnectProviderTagsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_open_id_connect_provider_tags::builders::ListOpenIdConnectProviderTagsInputBuilder {
+        super::super::super::operation::list_open_id_connect_provider_tags::builders::ListOpenIdConnectProviderTagsInputBuilder::default()
     }
 }

@@ -97,10 +97,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_open_id_connect_provider_tags::ListOpenIdConnectProviderTagsInput,
+        super::super::super::operation::list_open_id_connect_provider_tags::ListOpenIdConnectProviderTagsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_open_id_connect_provider_tags::ListOpenIdConnectProviderTagsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_open_id_connect_provider_tags::ListOpenIdConnectProviderTagsInput {
             open_id_connect_provider_arn: self.open_id_connect_provider_arn,
             marker: self.marker,
             max_items: self.max_items,
