--- original
+++ modified
@@ -25,8 +25,8 @@
 }
 impl PutEmailIdentityFeedbackAttributesInput {
     /// Creates a new builder-style object to manufacture [`PutEmailIdentityFeedbackAttributesInput`](crate::operation::put_email_identity_feedback_attributes::PutEmailIdentityFeedbackAttributesInput).
-    pub fn builder() -> crate::operation::put_email_identity_feedback_attributes::builders::PutEmailIdentityFeedbackAttributesInputBuilder {
-        crate::operation::put_email_identity_feedback_attributes::builders::PutEmailIdentityFeedbackAttributesInputBuilder::default()
+    pub fn builder() -> super::operation::put_email_identity_feedback_attributes::builders::PutEmailIdentityFeedbackAttributesInputBuilder {
+        super::operation::put_email_identity_feedback_attributes::builders::PutEmailIdentityFeedbackAttributesInputBuilder::default()
     }
 }

@@ -77,11 +77,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_email_identity_feedback_attributes::PutEmailIdentityFeedbackAttributesInput,
+        super::operation::put_email_identity_feedback_attributes::PutEmailIdentityFeedbackAttributesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_email_identity_feedback_attributes::PutEmailIdentityFeedbackAttributesInput {
+            super::operation::put_email_identity_feedback_attributes::PutEmailIdentityFeedbackAttributesInput {
                 email_identity: self.email_identity,
                 email_forwarding_enabled: self.email_forwarding_enabled,
             },
