--- original
+++ modified
@@ -22,7 +22,7 @@
     /// <p>A description of the alias.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>The <a href="https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html#configuring-alias-routing">routing configuration</a> of the alias.</p>
-    pub routing_config: ::std::option::Option<crate::types::AliasRoutingConfiguration>,
+    pub routing_config: ::std::option::Option<super::super::super::types::AliasRoutingConfiguration>,
     /// <p>Only update the alias if the revision ID matches the ID that's specified. Use this option to avoid modifying an alias that has changed since you last read it.</p>
     pub revision_id: ::std::option::Option<::std::string::String>,
 }
@@ -54,7 +54,7 @@
         self.description.as_deref()
     }
     /// <p>The <a href="https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html#configuring-alias-routing">routing configuration</a> of the alias.</p>
-    pub fn routing_config(&self) -> ::std::option::Option<&crate::types::AliasRoutingConfiguration> {
+    pub fn routing_config(&self) -> ::std::option::Option<&super::super::super::types::AliasRoutingConfiguration> {
         self.routing_config.as_ref()
     }
     /// <p>Only update the alias if the revision ID matches the ID that's specified. Use this option to avoid modifying an alias that has changed since you last read it.</p>
@@ -64,8 +64,8 @@
 }
 impl UpdateAliasInput {
     /// Creates a new builder-style object to manufacture [`UpdateAliasInput`](crate::operation::update_alias::UpdateAliasInput).
-    pub fn builder() -> crate::operation::update_alias::builders::UpdateAliasInputBuilder {
-        crate::operation::update_alias::builders::UpdateAliasInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_alias::builders::UpdateAliasInputBuilder {
+        super::super::super::operation::update_alias::builders::UpdateAliasInputBuilder::default()
     }
 }

@@ -77,7 +77,7 @@
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) function_version: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) routing_config: ::std::option::Option<crate::types::AliasRoutingConfiguration>,
+    pub(crate) routing_config: ::std::option::Option<super::super::super::types::AliasRoutingConfiguration>,
     pub(crate) revision_id: ::std::option::Option<::std::string::String>,
 }
 impl UpdateAliasInputBuilder {
@@ -170,17 +170,17 @@
         &self.description
     }
     /// <p>The <a href="https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html#configuring-alias-routing">routing configuration</a> of the alias.</p>
-    pub fn routing_config(mut self, input: crate::types::AliasRoutingConfiguration) -> Self {
+    pub fn routing_config(mut self, input: super::super::super::types::AliasRoutingConfiguration) -> Self {
         self.routing_config = ::std::option::Option::Some(input);
         self
     }
     /// <p>The <a href="https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html#configuring-alias-routing">routing configuration</a> of the alias.</p>
-    pub fn set_routing_config(mut self, input: ::std::option::Option<crate::types::AliasRoutingConfiguration>) -> Self {
+    pub fn set_routing_config(mut self, input: ::std::option::Option<super::super::super::types::AliasRoutingConfiguration>) -> Self {
         self.routing_config = input;
         self
     }
     /// <p>The <a href="https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html#configuring-alias-routing">routing configuration</a> of the alias.</p>
-    pub fn get_routing_config(&self) -> &::std::option::Option<crate::types::AliasRoutingConfiguration> {
+    pub fn get_routing_config(&self) -> &::std::option::Option<super::super::super::types::AliasRoutingConfiguration> {
         &self.routing_config
     }
     /// <p>Only update the alias if the revision ID matches the ID that's specified. Use this option to avoid modifying an alias that has changed since you last read it.</p>
@@ -198,8 +198,8 @@
         &self.revision_id
     }
     /// Consumes the builder and constructs a [`UpdateAliasInput`](crate::operation::update_alias::UpdateAliasInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::update_alias::UpdateAliasInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_alias::UpdateAliasInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::update_alias::UpdateAliasInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_alias::UpdateAliasInput {
             function_name: self.function_name,
             name: self.name,
             function_version: self.function_version,
