--- original
+++ modified
@@ -4,7 +4,7 @@
  *  SPDX-License-Identifier: Apache-2.0
  */

-use crate::endpoint_lib::diagnostic::DiagnosticCollector;
+use super::super::endpoint_lib::diagnostic::DiagnosticCollector;
 use std::borrow::Cow;
 use std::error::Error;
 use std::fmt::{Display, Formatter};
@@ -88,61 +88,3 @@
 pub(crate) fn parse_arn<'a>(input: &'a str, e: &mut DiagnosticCollector) -> Option<Arn<'a>> {
     e.capture(Arn::parse(input))
 }
-
-#[cfg(test)]
-mod test {
-    use super::Arn;
-
-    #[test]
-    fn arn_parser() {
-        let arn = "arn:aws:s3:us-east-2:012345678:outpost:op-1234";
-        let parsed = Arn::parse(arn).expect("valid ARN");
-        assert_eq!(
-            parsed,
-            Arn {
-                partition: "aws",
-                service: "s3",
-                region: "us-east-2",
-                account_id: "012345678",
-                resource_id: vec!["outpost", "op-1234"]
-            }
-        );
-    }
-
-    #[test]
-    fn allow_slash_arns() {
-        let arn = "arn:aws:s3:us-east-2:012345678:outpost/op-1234";
-        let parsed = Arn::parse(arn).expect("valid ARN");
-        assert_eq!(
-            parsed,
-            Arn {
-                partition: "aws",
-                service: "s3",
-                region: "us-east-2",
-                account_id: "012345678",
-                resource_id: vec!["outpost", "op-1234"]
-            }
-        );
-    }
-
-    #[test]
-    fn resource_id_must_be_nonempty() {
-        let arn = "arn:aws:s3:us-east-2:012345678:";
-        Arn::parse(arn).expect_err("empty resource");
-    }
-
-    #[test]
-    fn arns_with_empty_parts() {
-        let arn = "arn:aws:s3:::my_corporate_bucket/Development/*";
-        assert_eq!(
-            Arn::parse(arn).expect("valid arn"),
-            Arn {
-                partition: "aws",
-                service: "s3",
-                region: "",
-                account_id: "",
-                resource_id: vec!["my_corporate_bucket", "Development", "*"]
-            }
-        );
-    }
-}
