--- original
+++ modified
@@ -86,7 +86,7 @@
 macro_rules! coalesce {
     ($a:expr) => {$a};
     ($a:expr, $b:expr) => {{
-        use crate::endpoint_lib::coalesce::Coalesce;
+        use super::super::endpoint_lib::coalesce::Coalesce;
         let a = $a;
         let b = $b;
         (&&&(&a, &b)).coalesce()(a, b)
@@ -97,114 +97,3 @@
 }

 pub(crate) use coalesce;
-
-#[cfg(test)]
-mod test {
-
-    #[test]
-    fn base_cases() {
-        // All types optional
-        let a = Some("a");
-        let b = Some("b");
-
-        assert_eq!(coalesce!(a, b), Some("a"));
-
-        let a = None;
-        let b = Some("b");
-
-        assert_eq!(coalesce!(a, b), Some("b"));
-
-        let a = Some("a");
-        let b = None;
-
-        assert_eq!(coalesce!(a, b), Some("a"));
-
-        let a: Option<&str> = None;
-        let b: Option<&str> = None;
-
-        assert_eq!(coalesce!(a, b), None);
-
-        // Some non-optional types
-        let a = "a";
-        let b = Some("b");
-
-        assert_eq!(coalesce!(a, b), "a");
-
-        let a = Some("a");
-        let b = "b";
-
-        assert_eq!(coalesce!(a, b), "a");
-
-        let a = None;
-        let b = "b";
-
-        assert_eq!(coalesce!(a, b), "b");
-
-        let a = "a";
-        let b: Option<&str> = None;
-
-        assert_eq!(coalesce!(a, b), "a");
-
-        // All types non-optional
-        let a = "a";
-        let b = "b";
-
-        assert_eq!(coalesce!(a, b), "a");
-
-        // Works with trailing comma (makes codegen easier)
-        let a = "a";
-        let b = "b";
-
-        assert_eq!(coalesce!(a, b,), "a");
-    }
-
-    #[test]
-    fn longer_cases() {
-        assert_eq!(coalesce!(None, None, None, None, None, None, None, None, None, None, "a"), "a");
-
-        // In the generated code all of the inputs are typed variables, so the turbofish isn't needed in practice
-        assert_eq!(
-            coalesce!(
-                None,
-                None,
-                None,
-                None,
-                "a",
-                None::<&str>,
-                None::<&str>,
-                None::<&str>,
-                None::<&str>,
-                None::<&str>,
-                None::<&str>
-            ),
-            "a"
-        );
-
-        assert_eq!(
-            coalesce!(
-                None,
-                None,
-                None,
-                None,
-                "a",
-                None::<&str>,
-                Some("b"),
-                None::<&str>,
-                None::<&str>,
-                None::<&str>,
-                "c"
-            ),
-            "a"
-        );
-
-        assert_eq!(coalesce!(None, None, None, Some("a"), None, None, None, None, None, None, "b"), "a");
-        assert_eq!(
-            coalesce!(Some("a"), None, None, Some("b"), None, None, None, None, None, None,),
-            Some("a")
-        );
-
-        assert_eq!(coalesce!("a", "b", "c", "d", "e", "f", "g",), "a");
-
-        assert_eq!(coalesce!(None, None, None, None, None, None, None, None, None, None::<&str>,), None);
-    }
-}
