--- original
+++ modified
@@ -4,7 +4,7 @@
  *  SPDX-License-Identifier: Apache-2.0
  */

-use crate::endpoint_lib::diagnostic::DiagnosticCollector;
+use super::endpoint_lib::diagnostic::DiagnosticCollector;
 use http_1x::Uri;
 use url::{Host, Url as ParsedUrl};

@@ -60,46 +60,3 @@
     }
     Some(Url { url, uri, raw })
 }
-
-#[cfg(test)]
-mod test {
-    use super::*;
-    use crate::endpoint_lib::diagnostic::DiagnosticCollector;
-
-    #[allow(clippy::bool_assert_comparison)]
-    #[test]
-    fn parse_simple_url() {
-        let url = "https://control.vpce-1a2b3c4d-5e6f.s3.us-west-2.vpce.amazonaws.com";
-        let url = parse_url(url, &mut DiagnosticCollector::new()).expect("valid url");
-        assert_eq!(url.path(), "");
-        assert_eq!(url.normalized_path(), "/");
-        assert_eq!(url.is_ip(), false);
-        assert_eq!(url.scheme(), "https");
-        assert_eq!(url.authority(), "control.vpce-1a2b3c4d-5e6f.s3.us-west-2.vpce.amazonaws.com");
-    }
-
-    #[test]
-    fn schemes_are_normalized() {
-        let url = "HTTPS://control.vpce-1a2b3c4d-5e6f.s3.us-west-2.vpce.amazonaws.com";
-        let url = parse_url(url, &mut DiagnosticCollector::new()).expect("valid url");
-        assert_eq!(url.scheme(), "https");
-    }
-
-    #[allow(clippy::bool_assert_comparison)]
-    #[test]
-    fn parse_url_with_port() {
-        let url = "http://localhost:8000/path";
-        let url = parse_url(url, &mut DiagnosticCollector::new()).expect("valid url");
-        assert_eq!(url.path(), "/path");
-        assert_eq!(url.normalized_path(), "/path");
-        assert_eq!(url.is_ip(), false);
-        assert_eq!(url.scheme(), "http");
-        assert_eq!(url.authority(), "localhost:8000");
-    }
-
-    #[test]
-    fn only_http_https_supported() {
-        let url = "wss://localhost:8443/path";
-        assert_eq!(parse_url(url, &mut DiagnosticCollector::new()), None);
-    }
-}
