--- original
+++ modified
@@ -8,9 +8,9 @@
     /// <p>Descriptive name for this code signing configuration.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>Signing profiles for this code signing configuration.</p>
-    pub allowed_publishers: ::std::option::Option<crate::types::AllowedPublishers>,
+    pub allowed_publishers: ::std::option::Option<super::super::super::types::AllowedPublishers>,
     /// <p>The code signing policy.</p>
-    pub code_signing_policies: ::std::option::Option<crate::types::CodeSigningPolicies>,
+    pub code_signing_policies: ::std::option::Option<super::super::super::types::CodeSigningPolicies>,
 }
 impl UpdateCodeSigningConfigInput {
     /// <p>The The Amazon Resource Name (ARN) of the code signing configuration.</p>
@@ -22,18 +22,18 @@
         self.description.as_deref()
     }
     /// <p>Signing profiles for this code signing configuration.</p>
-    pub fn allowed_publishers(&self) -> ::std::option::Option<&crate::types::AllowedPublishers> {
+    pub fn allowed_publishers(&self) -> ::std::option::Option<&super::super::super::types::AllowedPublishers> {
         self.allowed_publishers.as_ref()
     }
     /// <p>The code signing policy.</p>
-    pub fn code_signing_policies(&self) -> ::std::option::Option<&crate::types::CodeSigningPolicies> {
+    pub fn code_signing_policies(&self) -> ::std::option::Option<&super::super::super::types::CodeSigningPolicies> {
         self.code_signing_policies.as_ref()
     }
 }
 impl UpdateCodeSigningConfigInput {
     /// Creates a new builder-style object to manufacture [`UpdateCodeSigningConfigInput`](crate::operation::update_code_signing_config::UpdateCodeSigningConfigInput).
-    pub fn builder() -> crate::operation::update_code_signing_config::builders::UpdateCodeSigningConfigInputBuilder {
-        crate::operation::update_code_signing_config::builders::UpdateCodeSigningConfigInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_code_signing_config::builders::UpdateCodeSigningConfigInputBuilder {
+        super::super::super::operation::update_code_signing_config::builders::UpdateCodeSigningConfigInputBuilder::default()
     }
 }

@@ -43,8 +43,8 @@
 pub struct UpdateCodeSigningConfigInputBuilder {
     pub(crate) code_signing_config_arn: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) allowed_publishers: ::std::option::Option<crate::types::AllowedPublishers>,
-    pub(crate) code_signing_policies: ::std::option::Option<crate::types::CodeSigningPolicies>,
+    pub(crate) allowed_publishers: ::std::option::Option<super::super::super::types::AllowedPublishers>,
+    pub(crate) code_signing_policies: ::std::option::Option<super::super::super::types::CodeSigningPolicies>,
 }
 impl UpdateCodeSigningConfigInputBuilder {
     /// <p>The The Amazon Resource Name (ARN) of the code signing configuration.</p>
@@ -77,31 +77,31 @@
         &self.description
     }
     /// <p>Signing profiles for this code signing configuration.</p>
-    pub fn allowed_publishers(mut self, input: crate::types::AllowedPublishers) -> Self {
+    pub fn allowed_publishers(mut self, input: super::super::super::types::AllowedPublishers) -> Self {
         self.allowed_publishers = ::std::option::Option::Some(input);
         self
     }
     /// <p>Signing profiles for this code signing configuration.</p>
-    pub fn set_allowed_publishers(mut self, input: ::std::option::Option<crate::types::AllowedPublishers>) -> Self {
+    pub fn set_allowed_publishers(mut self, input: ::std::option::Option<super::super::super::types::AllowedPublishers>) -> Self {
         self.allowed_publishers = input;
         self
     }
     /// <p>Signing profiles for this code signing configuration.</p>
-    pub fn get_allowed_publishers(&self) -> &::std::option::Option<crate::types::AllowedPublishers> {
+    pub fn get_allowed_publishers(&self) -> &::std::option::Option<super::super::super::types::AllowedPublishers> {
         &self.allowed_publishers
     }
     /// <p>The code signing policy.</p>
-    pub fn code_signing_policies(mut self, input: crate::types::CodeSigningPolicies) -> Self {
+    pub fn code_signing_policies(mut self, input: super::super::super::types::CodeSigningPolicies) -> Self {
         self.code_signing_policies = ::std::option::Option::Some(input);
         self
     }
     /// <p>The code signing policy.</p>
-    pub fn set_code_signing_policies(mut self, input: ::std::option::Option<crate::types::CodeSigningPolicies>) -> Self {
+    pub fn set_code_signing_policies(mut self, input: ::std::option::Option<super::super::super::types::CodeSigningPolicies>) -> Self {
         self.code_signing_policies = input;
         self
     }
     /// <p>The code signing policy.</p>
-    pub fn get_code_signing_policies(&self) -> &::std::option::Option<crate::types::CodeSigningPolicies> {
+    pub fn get_code_signing_policies(&self) -> &::std::option::Option<super::super::super::types::CodeSigningPolicies> {
         &self.code_signing_policies
     }
     /// Consumes the builder and constructs a [`UpdateCodeSigningConfigInput`](crate::operation::update_code_signing_config::UpdateCodeSigningConfigInput).
@@ -108,10 +108,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_code_signing_config::UpdateCodeSigningConfigInput,
+        super::super::super::operation::update_code_signing_config::UpdateCodeSigningConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_code_signing_config::UpdateCodeSigningConfigInput {
+        ::std::result::Result::Ok(super::super::super::operation::update_code_signing_config::UpdateCodeSigningConfigInput {
             code_signing_config_arn: self.code_signing_config_arn,
             description: self.description,
             allowed_publishers: self.allowed_publishers,
