--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DescribeDeliverySourcesInput {
     /// Creates a new builder-style object to manufacture [`DescribeDeliverySourcesInput`](crate::operation::describe_delivery_sources::DescribeDeliverySourcesInput).
-    pub fn builder() -> crate::operation::describe_delivery_sources::builders::DescribeDeliverySourcesInputBuilder {
-        crate::operation::describe_delivery_sources::builders::DescribeDeliverySourcesInputBuilder::default()
+    pub fn builder() -> super::operation::describe_delivery_sources::builders::DescribeDeliverySourcesInputBuilder {
+        super::operation::describe_delivery_sources::builders::DescribeDeliverySourcesInputBuilder::default()
     }
 }

@@ -65,10 +65,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_delivery_sources::DescribeDeliverySourcesInput,
+        super::operation::describe_delivery_sources::DescribeDeliverySourcesInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::describe_delivery_sources::DescribeDeliverySourcesInput {
+        ::std::result::Result::Ok(super::operation::describe_delivery_sources::DescribeDeliverySourcesInput {
             next_token: self.next_token,
             limit: self.limit,
         })
