--- original
+++ modified
@@ -10,8 +10,8 @@
     /// - On failure, responds with [`SdkError<AssociateSourceToS3TableIntegrationError>`](crate::operation::associate_source_to_s3_table_integration::AssociateSourceToS3TableIntegrationError)
     pub fn associate_source_to_s3_table_integration(
         &self,
-    ) -> crate::operation::associate_source_to_s3_table_integration::builders::AssociateSourceToS3TableIntegrationFluentBuilder {
-        crate::operation::associate_source_to_s3_table_integration::builders::AssociateSourceToS3TableIntegrationFluentBuilder::new(
+    ) -> super::super::operation::associate_source_to_s3_table_integration::builders::AssociateSourceToS3TableIntegrationFluentBuilder {
+        super::super::operation::associate_source_to_s3_table_integration::builders::AssociateSourceToS3TableIntegrationFluentBuilder::new(
             self.handle.clone(),
         )
     }
