--- original
+++ modified
@@ -12,7 +12,7 @@
     /// - On success, responds with [`CreateDeliveryOutput`](crate::operation::create_delivery::CreateDeliveryOutput) with field(s):
     ///   - [`delivery(Option<Delivery>)`](crate::operation::create_delivery::CreateDeliveryOutput::delivery): <p>A structure that contains information about the delivery that you just created.</p>
     /// - On failure, responds with [`SdkError<CreateDeliveryError>`](crate::operation::create_delivery::CreateDeliveryError)
-    pub fn create_delivery(&self) -> crate::operation::create_delivery::builders::CreateDeliveryFluentBuilder {
-        crate::operation::create_delivery::builders::CreateDeliveryFluentBuilder::new(self.handle.clone())
+    pub fn create_delivery(&self) -> super::operation::create_delivery::builders::CreateDeliveryFluentBuilder {
+        super::operation::create_delivery::builders::CreateDeliveryFluentBuilder::new(self.handle.clone())
     }
 }
