--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`log_stream_name(impl Into<String>)`](crate::operation::create_log_stream::builders::CreateLogStreamFluentBuilder::log_stream_name) / [`set_log_stream_name(Option<String>)`](crate::operation::create_log_stream::builders::CreateLogStreamFluentBuilder::set_log_stream_name):<br>required: **true**<br><p>The name of the log stream.</p><br>
     /// - On success, responds with [`CreateLogStreamOutput`](crate::operation::create_log_stream::CreateLogStreamOutput)
     /// - On failure, responds with [`SdkError<CreateLogStreamError>`](crate::operation::create_log_stream::CreateLogStreamError)
-    pub fn create_log_stream(&self) -> crate::operation::create_log_stream::builders::CreateLogStreamFluentBuilder {
-        crate::operation::create_log_stream::builders::CreateLogStreamFluentBuilder::new(self.handle.clone())
+    pub fn create_log_stream(&self) -> super::super::operation::create_log_stream::builders::CreateLogStreamFluentBuilder {
+        super::super::operation::create_log_stream::builders::CreateLogStreamFluentBuilder::new(self.handle.clone())
     }
 }
