--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`filter_name(impl Into<String>)`](crate::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterFluentBuilder::filter_name) / [`set_filter_name(Option<String>)`](crate::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterFluentBuilder::set_filter_name):<br>required: **true**<br><p>The name of the subscription filter.</p><br>
     /// - On success, responds with [`DeleteSubscriptionFilterOutput`](crate::operation::delete_subscription_filter::DeleteSubscriptionFilterOutput)
     /// - On failure, responds with [`SdkError<DeleteSubscriptionFilterError>`](crate::operation::delete_subscription_filter::DeleteSubscriptionFilterError)
-    pub fn delete_subscription_filter(&self) -> crate::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterFluentBuilder {
-        crate::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterFluentBuilder::new(self.handle.clone())
+    pub fn delete_subscription_filter(&self) -> super::super::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterFluentBuilder {
+        super::super::operation::delete_subscription_filter::builders::DeleteSubscriptionFilterFluentBuilder::new(self.handle.clone())
     }
 }
