--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`policy_type(PolicyType)`](crate::operation::delete_account_policy::builders::DeleteAccountPolicyFluentBuilder::policy_type) / [`set_policy_type(Option<PolicyType>)`](crate::operation::delete_account_policy::builders::DeleteAccountPolicyFluentBuilder::set_policy_type):<br>required: **true**<br><p>The type of policy to delete.</p><br>
     /// - On success, responds with [`DeleteAccountPolicyOutput`](crate::operation::delete_account_policy::DeleteAccountPolicyOutput)
     /// - On failure, responds with [`SdkError<DeleteAccountPolicyError>`](crate::operation::delete_account_policy::DeleteAccountPolicyError)
-    pub fn delete_account_policy(&self) -> crate::operation::delete_account_policy::builders::DeleteAccountPolicyFluentBuilder {
-        crate::operation::delete_account_policy::builders::DeleteAccountPolicyFluentBuilder::new(self.handle.clone())
+    pub fn delete_account_policy(&self) -> super::super::operation::delete_account_policy::builders::DeleteAccountPolicyFluentBuilder {
+        super::super::operation::delete_account_policy::builders::DeleteAccountPolicyFluentBuilder::new(self.handle.clone())
     }
 }
