--- original
+++ modified
@@ -6,7 +6,7 @@
     ///   - [`destination_name(impl Into<String>)`](crate::operation::delete_destination::builders::DeleteDestinationFluentBuilder::destination_name) / [`set_destination_name(Option<String>)`](crate::operation::delete_destination::builders::DeleteDestinationFluentBuilder::set_destination_name):<br>required: **true**<br><p>The name of the destination.</p><br>
     /// - On success, responds with [`DeleteDestinationOutput`](crate::operation::delete_destination::DeleteDestinationOutput)
     /// - On failure, responds with [`SdkError<DeleteDestinationError>`](crate::operation::delete_destination::DeleteDestinationError)
-    pub fn delete_destination(&self) -> crate::operation::delete_destination::builders::DeleteDestinationFluentBuilder {
-        crate::operation::delete_destination::builders::DeleteDestinationFluentBuilder::new(self.handle.clone())
+    pub fn delete_destination(&self) -> super::super::operation::delete_destination::builders::DeleteDestinationFluentBuilder {
+        super::super::operation::delete_destination::builders::DeleteDestinationFluentBuilder::new(self.handle.clone())
     }
 }
