--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`deliveries(Option<Vec::<Delivery>>)`](crate::operation::describe_deliveries::DescribeDeliveriesOutput::deliveries): <p>An array of structures. Each structure contains information about one delivery in the account.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::describe_deliveries::DescribeDeliveriesOutput::next_token): <p>The token for the next set of items to return. The token expires after 24 hours.</p>
     /// - On failure, responds with [`SdkError<DescribeDeliveriesError>`](crate::operation::describe_deliveries::DescribeDeliveriesError)
-    pub fn describe_deliveries(&self) -> crate::operation::describe_deliveries::builders::DescribeDeliveriesFluentBuilder {
-        crate::operation::describe_deliveries::builders::DescribeDeliveriesFluentBuilder::new(self.handle.clone())
+    pub fn describe_deliveries(&self) -> super::operation::describe_deliveries::builders::DescribeDeliveriesFluentBuilder {
+        super::operation::describe_deliveries::builders::DescribeDeliveriesFluentBuilder::new(self.handle.clone())
     }
 }
