--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`delivery_sources(Option<Vec::<DeliverySource>>)`](crate::operation::describe_delivery_sources::DescribeDeliverySourcesOutput::delivery_sources): <p>An array of structures. Each structure contains information about one delivery source in the account.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::describe_delivery_sources::DescribeDeliverySourcesOutput::next_token): <p>The token for the next set of items to return. The token expires after 24 hours.</p>
     /// - On failure, responds with [`SdkError<DescribeDeliverySourcesError>`](crate::operation::describe_delivery_sources::DescribeDeliverySourcesError)
-    pub fn describe_delivery_sources(&self) -> crate::operation::describe_delivery_sources::builders::DescribeDeliverySourcesFluentBuilder {
-        crate::operation::describe_delivery_sources::builders::DescribeDeliverySourcesFluentBuilder::new(self.handle.clone())
+    pub fn describe_delivery_sources(&self) -> super::operation::describe_delivery_sources::builders::DescribeDeliverySourcesFluentBuilder {
+        super::operation::describe_delivery_sources::builders::DescribeDeliverySourcesFluentBuilder::new(self.handle.clone())
     }
 }
