--- original
+++ modified
@@ -21,7 +21,7 @@
     ///   - [`searched_log_streams(Option<Vec::<SearchedLogStream>>)`](crate::operation::filter_log_events::FilterLogEventsOutput::searched_log_streams): <p><b>Important</b> As of May 15, 2020, this parameter is no longer supported. This parameter returns an empty list.</p> <p>Indicates which log streams have been searched and whether each has been searched completely.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::filter_log_events::FilterLogEventsOutput::next_token): <p>The token for the next set of items in the sorting direction specified by the <code>startFromHead</code> parameter in the first request. The token expires after 24 hours.</p> <p>If the results don't include a <code>nextToken</code>, then pagination is finished.</p>
     /// - On failure, responds with [`SdkError<FilterLogEventsError>`](crate::operation::filter_log_events::FilterLogEventsError)
-    pub fn filter_log_events(&self) -> crate::operation::filter_log_events::builders::FilterLogEventsFluentBuilder {
-        crate::operation::filter_log_events::builders::FilterLogEventsFluentBuilder::new(self.handle.clone())
+    pub fn filter_log_events(&self) -> super::operation::filter_log_events::builders::FilterLogEventsFluentBuilder {
+        super::operation::filter_log_events::builders::FilterLogEventsFluentBuilder::new(self.handle.clone())
     }
 }
