--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`storage_tier(Option<StorageTier>)`](crate::operation::get_storage_tier_policy::GetStorageTierPolicyOutput::storage_tier): <p>The current storage tier for the account.</p>
     ///   - [`last_updated_time(Option<i64>)`](crate::operation::get_storage_tier_policy::GetStorageTierPolicyOutput::last_updated_time): <p>The time when the storage tier policy was last updated, expressed as the number of milliseconds after <code>January 1, 1970 00:00:00 UTC</code>.</p>
     /// - On failure, responds with [`SdkError<GetStorageTierPolicyError>`](crate::operation::get_storage_tier_policy::GetStorageTierPolicyError)
-    pub fn get_storage_tier_policy(&self) -> crate::operation::get_storage_tier_policy::builders::GetStorageTierPolicyFluentBuilder {
-        crate::operation::get_storage_tier_policy::builders::GetStorageTierPolicyFluentBuilder::new(self.handle.clone())
+    pub fn get_storage_tier_policy(&self) -> super::operation::get_storage_tier_policy::builders::GetStorageTierPolicyFluentBuilder {
+        super::operation::get_storage_tier_policy::builders::GetStorageTierPolicyFluentBuilder::new(self.handle.clone())
     }
 }
