--- original
+++ modified
@@ -16,7 +16,7 @@
     ///   - [`log_groups(Option<Vec::<LogGroupSummary>>)`](crate::operation::list_log_groups::ListLogGroupsOutput::log_groups): <p>An array of structures, where each structure contains the information about one log group.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_log_groups::ListLogGroupsOutput::next_token): <p>The token for the next set of items to return. The token expires after 24 hours.</p>
     /// - On failure, responds with [`SdkError<ListLogGroupsError>`](crate::operation::list_log_groups::ListLogGroupsError)
-    pub fn list_log_groups(&self) -> crate::operation::list_log_groups::builders::ListLogGroupsFluentBuilder {
-        crate::operation::list_log_groups::builders::ListLogGroupsFluentBuilder::new(self.handle.clone())
+    pub fn list_log_groups(&self) -> super::operation::list_log_groups::builders::ListLogGroupsFluentBuilder {
+        super::operation::list_log_groups::builders::ListLogGroupsFluentBuilder::new(self.handle.clone())
     }
 }
