--- original
+++ modified
@@ -11,7 +11,7 @@
     ///   - [`syslog_configurations(Option<Vec::<SyslogConfiguration>>)`](crate::operation::list_syslog_configurations::ListSyslogConfigurationsOutput::syslog_configurations): <p>The list of syslog configurations.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_syslog_configurations::ListSyslogConfigurationsOutput::next_token): <p>The token for the next set of items to return. The token expires after 24 hours.</p>
     /// - On failure, responds with [`SdkError<ListSyslogConfigurationsError>`](crate::operation::list_syslog_configurations::ListSyslogConfigurationsError)
-    pub fn list_syslog_configurations(&self) -> crate::operation::list_syslog_configurations::builders::ListSyslogConfigurationsFluentBuilder {
-        crate::operation::list_syslog_configurations::builders::ListSyslogConfigurationsFluentBuilder::new(self.handle.clone())
+    pub fn list_syslog_configurations(&self) -> super::operation::list_syslog_configurations::builders::ListSyslogConfigurationsFluentBuilder {
+        super::operation::list_syslog_configurations::builders::ListSyslogConfigurationsFluentBuilder::new(self.handle.clone())
     }
 }
