--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`policy_document(Option<String>)`](crate::operation::put_data_protection_policy::PutDataProtectionPolicyOutput::policy_document): <p>The data protection policy used for this log group.</p>
     ///   - [`last_updated_time(Option<i64>)`](crate::operation::put_data_protection_policy::PutDataProtectionPolicyOutput::last_updated_time): <p>The date and time that this policy was most recently updated.</p>
     /// - On failure, responds with [`SdkError<PutDataProtectionPolicyError>`](crate::operation::put_data_protection_policy::PutDataProtectionPolicyError)
-    pub fn put_data_protection_policy(&self) -> crate::operation::put_data_protection_policy::builders::PutDataProtectionPolicyFluentBuilder {
-        crate::operation::put_data_protection_policy::builders::PutDataProtectionPolicyFluentBuilder::new(self.handle.clone())
+    pub fn put_data_protection_policy(&self) -> super::super::operation::put_data_protection_policy::builders::PutDataProtectionPolicyFluentBuilder {
+        super::super::operation::put_data_protection_policy::builders::PutDataProtectionPolicyFluentBuilder::new(self.handle.clone())
     }
 }
