--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<PutLogGroupDeletionProtectionError>`](crate::operation::put_log_group_deletion_protection::PutLogGroupDeletionProtectionError)
     pub fn put_log_group_deletion_protection(
         &self,
-    ) -> crate::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionFluentBuilder {
-        crate::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionFluentBuilder {
+        super::operation::put_log_group_deletion_protection::builders::PutLogGroupDeletionProtectionFluentBuilder::new(self.handle.clone())
     }
 }
