--- original
+++ modified
@@ -12,7 +12,7 @@
     ///   - [`emit_system_field_dimensions(impl Into<String>)`](crate::operation::put_metric_filter::builders::PutMetricFilterFluentBuilder::emit_system_field_dimensions) / [`set_emit_system_field_dimensions(Option<Vec::<String>>)`](crate::operation::put_metric_filter::builders::PutMetricFilterFluentBuilder::set_emit_system_field_dimensions):<br>required: **false**<br><p>A list of system fields to emit as additional dimensions in the generated metrics. Valid values are <code>@aws.account</code> and <code>@aws.region</code>. These dimensions help identify the source of centralized log data and count toward the total dimension limit for metric filters.</p><br>
     /// - On success, responds with [`PutMetricFilterOutput`](crate::operation::put_metric_filter::PutMetricFilterOutput)
     /// - On failure, responds with [`SdkError<PutMetricFilterError>`](crate::operation::put_metric_filter::PutMetricFilterError)
-    pub fn put_metric_filter(&self) -> crate::operation::put_metric_filter::builders::PutMetricFilterFluentBuilder {
-        crate::operation::put_metric_filter::builders::PutMetricFilterFluentBuilder::new(self.handle.clone())
+    pub fn put_metric_filter(&self) -> super::super::operation::put_metric_filter::builders::PutMetricFilterFluentBuilder {
+        super::super::operation::put_metric_filter::builders::PutMetricFilterFluentBuilder::new(self.handle.clone())
     }
 }
