--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`transformer_config(Processor)`](crate::operation::put_transformer::builders::PutTransformerFluentBuilder::transformer_config) / [`set_transformer_config(Option<Vec::<Processor>>)`](crate::operation::put_transformer::builders::PutTransformerFluentBuilder::set_transformer_config):<br>required: **true**<br><p>This structure contains the configuration of this log transformer. A log transformer is an array of processors, where each processor applies one type of transformation to the log events that are ingested.</p><br>
     /// - On success, responds with [`PutTransformerOutput`](crate::operation::put_transformer::PutTransformerOutput)
     /// - On failure, responds with [`SdkError<PutTransformerError>`](crate::operation::put_transformer::PutTransformerError)
-    pub fn put_transformer(&self) -> crate::operation::put_transformer::builders::PutTransformerFluentBuilder {
-        crate::operation::put_transformer::builders::PutTransformerFluentBuilder::new(self.handle.clone())
+    pub fn put_transformer(&self) -> super::operation::put_transformer::builders::PutTransformerFluentBuilder {
+        super::operation::put_transformer::builders::PutTransformerFluentBuilder::new(self.handle.clone())
     }
 }
