--- original
+++ modified
@@ -7,7 +7,7 @@
     /// - On success, responds with [`StopQueryOutput`](crate::operation::stop_query::StopQueryOutput) with field(s):
     ///   - [`success(bool)`](crate::operation::stop_query::StopQueryOutput::success): <p>This is true if the query was stopped by the <code>StopQuery</code> operation.</p>
     /// - On failure, responds with [`SdkError<StopQueryError>`](crate::operation::stop_query::StopQueryError)
-    pub fn stop_query(&self) -> crate::operation::stop_query::builders::StopQueryFluentBuilder {
-        crate::operation::stop_query::builders::StopQueryFluentBuilder::new(self.handle.clone())
+    pub fn stop_query(&self) -> super::super::operation::stop_query::builders::StopQueryFluentBuilder {
+        super::super::operation::stop_query::builders::StopQueryFluentBuilder::new(self.handle.clone())
     }
 }
