--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`TestTransformerOutput`](crate::operation::test_transformer::TestTransformerOutput) with field(s):
     ///   - [`transformed_logs(Option<Vec::<TransformedLogRecord>>)`](crate::operation::test_transformer::TestTransformerOutput::transformed_logs): <p>An array where each member of the array includes both the original version and the transformed version of one of the log events that you input.</p>
     /// - On failure, responds with [`SdkError<TestTransformerError>`](crate::operation::test_transformer::TestTransformerError)
-    pub fn test_transformer(&self) -> crate::operation::test_transformer::builders::TestTransformerFluentBuilder {
-        crate::operation::test_transformer::builders::TestTransformerFluentBuilder::new(self.handle.clone())
+    pub fn test_transformer(&self) -> super::operation::test_transformer::builders::TestTransformerFluentBuilder {
+        super::operation::test_transformer::builders::TestTransformerFluentBuilder::new(self.handle.clone())
     }
 }
