--- original
+++ modified
@@ -39,7 +39,7 @@
     ///   - [`creation_time(Option<i64>)`](crate::operation::update_scheduled_query::UpdateScheduledQueryOutput::creation_time): <p>The timestamp when the scheduled query was originally created.</p>
     ///   - [`last_updated_time(Option<i64>)`](crate::operation::update_scheduled_query::UpdateScheduledQueryOutput::last_updated_time): <p>The timestamp when the scheduled query was last updated.</p>
     /// - On failure, responds with [`SdkError<UpdateScheduledQueryError>`](crate::operation::update_scheduled_query::UpdateScheduledQueryError)
-    pub fn update_scheduled_query(&self) -> crate::operation::update_scheduled_query::builders::UpdateScheduledQueryFluentBuilder {
-        crate::operation::update_scheduled_query::builders::UpdateScheduledQueryFluentBuilder::new(self.handle.clone())
+    pub fn update_scheduled_query(&self) -> super::operation::update_scheduled_query::builders::UpdateScheduledQueryFluentBuilder {
+        super::operation::update_scheduled_query::builders::UpdateScheduledQueryFluentBuilder::new(self.handle.clone())
     }
 }
