--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteUserPoolReplicaInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserPoolReplicaInput`](crate::operation::delete_user_pool_replica::DeleteUserPoolReplicaInput).
-    pub fn builder() -> crate::operation::delete_user_pool_replica::builders::DeleteUserPoolReplicaInputBuilder {
-        crate::operation::delete_user_pool_replica::builders::DeleteUserPoolReplicaInputBuilder::default()
+    pub fn builder() -> super::operation::delete_user_pool_replica::builders::DeleteUserPoolReplicaInputBuilder {
+        super::operation::delete_user_pool_replica::builders::DeleteUserPoolReplicaInputBuilder::default()
     }
 }

@@ -66,9 +66,9 @@
     /// Consumes the builder and constructs a [`DeleteUserPoolReplicaInput`](crate::operation::delete_user_pool_replica::DeleteUserPoolReplicaInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_user_pool_replica::DeleteUserPoolReplicaInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::delete_user_pool_replica::DeleteUserPoolReplicaInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_user_pool_replica::DeleteUserPoolReplicaInput {
+        ::std::result::Result::Ok(super::operation::delete_user_pool_replica::DeleteUserPoolReplicaInput {
             user_pool_id: self.user_pool_id,
             region_name: self.region_name,
         })
