--- original
+++ modified
@@ -12,7 +12,7 @@
     pub cloud_watch_logs_role_arn: ::std::option::Option<::std::string::String>,
     /// <p>The password hashing algorithm used to generate the hashes in the CSV file for this import job.</p>
     /// <p>Valid values: <code>BCRYPT</code> | <code>SCRYPT</code> | <code>ARGON2ID</code> | <code>PBKDF2_SHA256</code></p>
-    pub password_hashing_algorithm: ::std::option::Option<crate::types::PasswordHashingAlgorithmType>,
+    pub password_hashing_algorithm: ::std::option::Option<super::types::PasswordHashingAlgorithmType>,
 }
 impl CreateUserImportJobInput {
     /// <p>A friendly name for the user import job.</p>
@@ -29,14 +29,14 @@
     }
     /// <p>The password hashing algorithm used to generate the hashes in the CSV file for this import job.</p>
     /// <p>Valid values: <code>BCRYPT</code> | <code>SCRYPT</code> | <code>ARGON2ID</code> | <code>PBKDF2_SHA256</code></p>
-    pub fn password_hashing_algorithm(&self) -> ::std::option::Option<&crate::types::PasswordHashingAlgorithmType> {
+    pub fn password_hashing_algorithm(&self) -> ::std::option::Option<&super::types::PasswordHashingAlgorithmType> {
         self.password_hashing_algorithm.as_ref()
     }
 }
 impl CreateUserImportJobInput {
     /// Creates a new builder-style object to manufacture [`CreateUserImportJobInput`](crate::operation::create_user_import_job::CreateUserImportJobInput).
-    pub fn builder() -> crate::operation::create_user_import_job::builders::CreateUserImportJobInputBuilder {
-        crate::operation::create_user_import_job::builders::CreateUserImportJobInputBuilder::default()
+    pub fn builder() -> super::operation::create_user_import_job::builders::CreateUserImportJobInputBuilder {
+        super::operation::create_user_import_job::builders::CreateUserImportJobInputBuilder::default()
     }
 }

@@ -47,7 +47,7 @@
     pub(crate) job_name: ::std::option::Option<::std::string::String>,
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) cloud_watch_logs_role_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) password_hashing_algorithm: ::std::option::Option<crate::types::PasswordHashingAlgorithmType>,
+    pub(crate) password_hashing_algorithm: ::std::option::Option<super::types::PasswordHashingAlgorithmType>,
 }
 impl CreateUserImportJobInputBuilder {
     /// <p>A friendly name for the user import job.</p>
@@ -97,27 +97,27 @@
     }
     /// <p>The password hashing algorithm used to generate the hashes in the CSV file for this import job.</p>
     /// <p>Valid values: <code>BCRYPT</code> | <code>SCRYPT</code> | <code>ARGON2ID</code> | <code>PBKDF2_SHA256</code></p>
-    pub fn password_hashing_algorithm(mut self, input: crate::types::PasswordHashingAlgorithmType) -> Self {
+    pub fn password_hashing_algorithm(mut self, input: super::types::PasswordHashingAlgorithmType) -> Self {
         self.password_hashing_algorithm = ::std::option::Option::Some(input);
         self
     }
     /// <p>The password hashing algorithm used to generate the hashes in the CSV file for this import job.</p>
     /// <p>Valid values: <code>BCRYPT</code> | <code>SCRYPT</code> | <code>ARGON2ID</code> | <code>PBKDF2_SHA256</code></p>
-    pub fn set_password_hashing_algorithm(mut self, input: ::std::option::Option<crate::types::PasswordHashingAlgorithmType>) -> Self {
+    pub fn set_password_hashing_algorithm(mut self, input: ::std::option::Option<super::types::PasswordHashingAlgorithmType>) -> Self {
         self.password_hashing_algorithm = input;
         self
     }
     /// <p>The password hashing algorithm used to generate the hashes in the CSV file for this import job.</p>
     /// <p>Valid values: <code>BCRYPT</code> | <code>SCRYPT</code> | <code>ARGON2ID</code> | <code>PBKDF2_SHA256</code></p>
-    pub fn get_password_hashing_algorithm(&self) -> &::std::option::Option<crate::types::PasswordHashingAlgorithmType> {
+    pub fn get_password_hashing_algorithm(&self) -> &::std::option::Option<super::types::PasswordHashingAlgorithmType> {
         &self.password_hashing_algorithm
     }
     /// Consumes the builder and constructs a [`CreateUserImportJobInput`](crate::operation::create_user_import_job::CreateUserImportJobInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_user_import_job::CreateUserImportJobInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::create_user_import_job::CreateUserImportJobInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::create_user_import_job::CreateUserImportJobInput {
+        ::std::result::Result::Ok(super::operation::create_user_import_job::CreateUserImportJobInput {
             job_name: self.job_name,
             user_pool_id: self.user_pool_id,
             cloud_watch_logs_role_arn: self.cloud_watch_logs_role_arn,
