--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl DeleteTenantResourceAssociationInput {
     /// Creates a new builder-style object to manufacture [`DeleteTenantResourceAssociationInput`](crate::operation::delete_tenant_resource_association::DeleteTenantResourceAssociationInput).
-    pub fn builder() -> crate::operation::delete_tenant_resource_association::builders::DeleteTenantResourceAssociationInputBuilder {
-        crate::operation::delete_tenant_resource_association::builders::DeleteTenantResourceAssociationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_tenant_resource_association::builders::DeleteTenantResourceAssociationInputBuilder {
+        super::super::super::operation::delete_tenant_resource_association::builders::DeleteTenantResourceAssociationInputBuilder::default()
     }
 }

@@ -68,11 +68,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_tenant_resource_association::DeleteTenantResourceAssociationInput,
+        super::super::super::operation::delete_tenant_resource_association::DeleteTenantResourceAssociationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::delete_tenant_resource_association::DeleteTenantResourceAssociationInput {
+            super::super::super::operation::delete_tenant_resource_association::DeleteTenantResourceAssociationInput {
                 tenant_name: self.tenant_name,
                 resource_arn: self.resource_arn,
             },
