--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListUserPoolReplicasInput {
     /// Creates a new builder-style object to manufacture [`ListUserPoolReplicasInput`](crate::operation::list_user_pool_replicas::ListUserPoolReplicasInput).
-    pub fn builder() -> crate::operation::list_user_pool_replicas::builders::ListUserPoolReplicasInputBuilder {
-        crate::operation::list_user_pool_replicas::builders::ListUserPoolReplicasInputBuilder::default()
+    pub fn builder() -> super::operation::list_user_pool_replicas::builders::ListUserPoolReplicasInputBuilder {
+        super::operation::list_user_pool_replicas::builders::ListUserPoolReplicasInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`ListUserPoolReplicasInput`](crate::operation::list_user_pool_replicas::ListUserPoolReplicasInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_user_pool_replicas::ListUserPoolReplicasInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_user_pool_replicas::ListUserPoolReplicasInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_user_pool_replicas::ListUserPoolReplicasInput {
+        ::std::result::Result::Ok(super::operation::list_user_pool_replicas::ListUserPoolReplicasInput {
             user_pool_id: self.user_pool_id,
             next_token: self.next_token,
         })
