--- original
+++ modified
@@ -6,8 +6,8 @@
 pub struct GetAccountInput {}
 impl GetAccountInput {
     /// Creates a new builder-style object to manufacture [`GetAccountInput`](crate::operation::get_account::GetAccountInput).
-    pub fn builder() -> crate::operation::get_account::builders::GetAccountInputBuilder {
-        crate::operation::get_account::builders::GetAccountInputBuilder::default()
+    pub fn builder() -> super::operation::get_account::builders::GetAccountInputBuilder {
+        super::operation::get_account::builders::GetAccountInputBuilder::default()
     }
 }

@@ -17,7 +17,7 @@
 pub struct GetAccountInputBuilder {}
 impl GetAccountInputBuilder {
     /// Consumes the builder and constructs a [`GetAccountInput`](crate::operation::get_account::GetAccountInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::get_account::GetAccountInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_account::GetAccountInput {})
+    pub fn build(self) -> ::std::result::Result<super::operation::get_account::GetAccountInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::get_account::GetAccountInput {})
     }
 }
