--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl UpdateRoleInput {
     /// Creates a new builder-style object to manufacture [`UpdateRoleInput`](crate::operation::update_role::UpdateRoleInput).
-    pub fn builder() -> crate::operation::update_role::builders::UpdateRoleInputBuilder {
-        crate::operation::update_role::builders::UpdateRoleInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_role::builders::UpdateRoleInputBuilder {
+        super::super::super::operation::update_role::builders::UpdateRoleInputBuilder::default()
     }
 }

@@ -99,8 +99,8 @@
         &self.max_session_duration
     }
     /// Consumes the builder and constructs a [`UpdateRoleInput`](crate::operation::update_role::UpdateRoleInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::update_role::UpdateRoleInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_role::UpdateRoleInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::update_role::UpdateRoleInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_role::UpdateRoleInput {
             role_name: self.role_name,
             description: self.description,
             max_session_duration: self.max_session_duration,
