--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`statement(Option<String>)`](crate::operation::add_layer_version_permission::AddLayerVersionPermissionOutput::statement): <p>The permission statement.</p>
     ///   - [`revision_id(Option<String>)`](crate::operation::add_layer_version_permission::AddLayerVersionPermissionOutput::revision_id): <p>A unique identifier for the current revision of the policy.</p>
     /// - On failure, responds with [`SdkError<AddLayerVersionPermissionError>`](crate::operation::add_layer_version_permission::AddLayerVersionPermissionError)
-    pub fn add_layer_version_permission(&self) -> crate::operation::add_layer_version_permission::builders::AddLayerVersionPermissionFluentBuilder {
-        crate::operation::add_layer_version_permission::builders::AddLayerVersionPermissionFluentBuilder::new(self.handle.clone())
+    pub fn add_layer_version_permission(&self) -> super::super::operation::add_layer_version_permission::builders::AddLayerVersionPermissionFluentBuilder {
+        super::super::operation::add_layer_version_permission::builders::AddLayerVersionPermissionFluentBuilder::new(self.handle.clone())
     }
 }
