--- original
+++ modified
@@ -16,7 +16,7 @@
     ///   - [`creation_time(String)`](crate::operation::create_function_url_config::CreateFunctionUrlConfigOutput::creation_time): <p>When the function URL was created, in <a href="https://www.w3.org/TR/NOTE-datetime">ISO-8601 format</a> (YYYY-MM-DDThh:mm:ss.sTZD).</p>
     ///   - [`invoke_mode(Option<InvokeMode>)`](crate::operation::create_function_url_config::CreateFunctionUrlConfigOutput::invoke_mode): <p>Use one of the following options:</p> <ul>  <li>   <p><code>BUFFERED</code> – This is the default option. Lambda invokes your function using the <code>Invoke</code> API operation. Invocation results are available when the payload is complete. The maximum payload size is 6 MB.</p></li>  <li>   <p><code>RESPONSE_STREAM</code> – Your function streams payload results as they become available. Lambda invokes your function using the <code>InvokeWithResponseStream</code> API operation. The maximum response payload size is 200 MB.</p></li> </ul>
     /// - On failure, responds with [`SdkError<CreateFunctionUrlConfigError>`](crate::operation::create_function_url_config::CreateFunctionUrlConfigError)
-    pub fn create_function_url_config(&self) -> crate::operation::create_function_url_config::builders::CreateFunctionUrlConfigFluentBuilder {
-        crate::operation::create_function_url_config::builders::CreateFunctionUrlConfigFluentBuilder::new(self.handle.clone())
+    pub fn create_function_url_config(&self) -> super::super::operation::create_function_url_config::builders::CreateFunctionUrlConfigFluentBuilder {
+        super::super::operation::create_function_url_config::builders::CreateFunctionUrlConfigFluentBuilder::new(self.handle.clone())
     }
 }
