--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<DeleteProvisionedConcurrencyConfigError>`](crate::operation::delete_provisioned_concurrency_config::DeleteProvisionedConcurrencyConfigError)
     pub fn delete_provisioned_concurrency_config(
         &self,
-    ) -> crate::operation::delete_provisioned_concurrency_config::builders::DeleteProvisionedConcurrencyConfigFluentBuilder {
-        crate::operation::delete_provisioned_concurrency_config::builders::DeleteProvisionedConcurrencyConfigFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::delete_provisioned_concurrency_config::builders::DeleteProvisionedConcurrencyConfigFluentBuilder {
+        super::operation::delete_provisioned_concurrency_config::builders::DeleteProvisionedConcurrencyConfigFluentBuilder::new(self.handle.clone())
     }
 }
