--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`qualifier(impl Into<String>)`](crate::operation::delete_function_url_config::builders::DeleteFunctionUrlConfigFluentBuilder::qualifier) / [`set_qualifier(Option<String>)`](crate::operation::delete_function_url_config::builders::DeleteFunctionUrlConfigFluentBuilder::set_qualifier):<br>required: **false**<br><p>The alias name.</p><br>
     /// - On success, responds with [`DeleteFunctionUrlConfigOutput`](crate::operation::delete_function_url_config::DeleteFunctionUrlConfigOutput)
     /// - On failure, responds with [`SdkError<DeleteFunctionUrlConfigError>`](crate::operation::delete_function_url_config::DeleteFunctionUrlConfigError)
-    pub fn delete_function_url_config(&self) -> crate::operation::delete_function_url_config::builders::DeleteFunctionUrlConfigFluentBuilder {
-        crate::operation::delete_function_url_config::builders::DeleteFunctionUrlConfigFluentBuilder::new(self.handle.clone())
+    pub fn delete_function_url_config(&self) -> super::operation::delete_function_url_config::builders::DeleteFunctionUrlConfigFluentBuilder {
+        super::operation::delete_function_url_config::builders::DeleteFunctionUrlConfigFluentBuilder::new(self.handle.clone())
     }
 }
