--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`name(impl Into<String>)`](crate::operation::delete_alias::builders::DeleteAliasFluentBuilder::name) / [`set_name(Option<String>)`](crate::operation::delete_alias::builders::DeleteAliasFluentBuilder::set_name):<br>required: **true**<br><p>The name of the alias.</p><br>
     /// - On success, responds with [`DeleteAliasOutput`](crate::operation::delete_alias::DeleteAliasOutput)
     /// - On failure, responds with [`SdkError<DeleteAliasError>`](crate::operation::delete_alias::DeleteAliasError)
-    pub fn delete_alias(&self) -> crate::operation::delete_alias::builders::DeleteAliasFluentBuilder {
-        crate::operation::delete_alias::builders::DeleteAliasFluentBuilder::new(self.handle.clone())
+    pub fn delete_alias(&self) -> super::operation::delete_alias::builders::DeleteAliasFluentBuilder {
+        super::operation::delete_alias::builders::DeleteAliasFluentBuilder::new(self.handle.clone())
     }
 }
