--- original
+++ modified
@@ -15,7 +15,7 @@
     /// - On failure, responds with [`SdkError<GetDurableExecutionHistoryError>`](crate::operation::get_durable_execution_history::GetDurableExecutionHistoryError)
     pub fn get_durable_execution_history(
         &self,
-    ) -> crate::operation::get_durable_execution_history::builders::GetDurableExecutionHistoryFluentBuilder {
-        crate::operation::get_durable_execution_history::builders::GetDurableExecutionHistoryFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::get_durable_execution_history::builders::GetDurableExecutionHistoryFluentBuilder {
+        super::super::operation::get_durable_execution_history::builders::GetDurableExecutionHistoryFluentBuilder::new(self.handle.clone())
     }
 }
