--- original
+++ modified
@@ -7,7 +7,7 @@
     /// - On success, responds with [`GetCodeSigningConfigOutput`](crate::operation::get_code_signing_config::GetCodeSigningConfigOutput) with field(s):
     ///   - [`code_signing_config(Option<CodeSigningConfig>)`](crate::operation::get_code_signing_config::GetCodeSigningConfigOutput::code_signing_config): <p>The code signing configuration</p>
     /// - On failure, responds with [`SdkError<GetCodeSigningConfigError>`](crate::operation::get_code_signing_config::GetCodeSigningConfigError)
-    pub fn get_code_signing_config(&self) -> crate::operation::get_code_signing_config::builders::GetCodeSigningConfigFluentBuilder {
-        crate::operation::get_code_signing_config::builders::GetCodeSigningConfigFluentBuilder::new(self.handle.clone())
+    pub fn get_code_signing_config(&self) -> super::operation::get_code_signing_config::builders::GetCodeSigningConfigFluentBuilder {
+        super::operation::get_code_signing_config::builders::GetCodeSigningConfigFluentBuilder::new(self.handle.clone())
     }
 }
