--- original
+++ modified
@@ -12,7 +12,7 @@
     ///   - [`tags_error(Option<TagsError>)`](crate::operation::get_function::GetFunctionOutput::tags_error): <p>An object that contains details about an error related to retrieving tags.</p>
     ///   - [`concurrency(Option<Concurrency>)`](crate::operation::get_function::GetFunctionOutput::concurrency): <p>The function's <a href="https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html">reserved concurrency</a>.</p>
     /// - On failure, responds with [`SdkError<GetFunctionError>`](crate::operation::get_function::GetFunctionError)
-    pub fn get_function(&self) -> crate::operation::get_function::builders::GetFunctionFluentBuilder {
-        crate::operation::get_function::builders::GetFunctionFluentBuilder::new(self.handle.clone())
+    pub fn get_function(&self) -> super::operation::get_function::builders::GetFunctionFluentBuilder {
+        super::operation::get_function::builders::GetFunctionFluentBuilder::new(self.handle.clone())
     }
 }
