--- original
+++ modified
@@ -47,7 +47,7 @@
     ///   - [`config_sha256(Option<String>)`](crate::operation::get_function_configuration::GetFunctionConfigurationOutput::config_sha256): <p>The SHA256 hash of the function configuration.</p>
     ///   - [`durable_config(Option<DurableConfig>)`](crate::operation::get_function_configuration::GetFunctionConfigurationOutput::durable_config): <p>The function's durable execution configuration settings, if the function is configured for durability.</p>
     /// - On failure, responds with [`SdkError<GetFunctionConfigurationError>`](crate::operation::get_function_configuration::GetFunctionConfigurationError)
-    pub fn get_function_configuration(&self) -> crate::operation::get_function_configuration::builders::GetFunctionConfigurationFluentBuilder {
-        crate::operation::get_function_configuration::builders::GetFunctionConfigurationFluentBuilder::new(self.handle.clone())
+    pub fn get_function_configuration(&self) -> super::operation::get_function_configuration::builders::GetFunctionConfigurationFluentBuilder {
+        super::operation::get_function_configuration::builders::GetFunctionConfigurationFluentBuilder::new(self.handle.clone())
     }
 }
