--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`status(i32)`](crate::operation::invoke_async::InvokeAsyncOutput::status): <p>The status code.</p>
     /// - On failure, responds with [`SdkError<InvokeAsyncError>`](crate::operation::invoke_async::InvokeAsyncError)
     #[deprecated]
-    pub fn invoke_async(&self) -> crate::operation::invoke_async::builders::InvokeAsyncFluentBuilder {
-        crate::operation::invoke_async::builders::InvokeAsyncFluentBuilder::new(self.handle.clone())
+    pub fn invoke_async(&self) -> super::super::operation::invoke_async::builders::InvokeAsyncFluentBuilder {
+        super::super::operation::invoke_async::builders::InvokeAsyncFluentBuilder::new(self.handle.clone())
     }
 }
