--- original
+++ modified
@@ -13,7 +13,7 @@
     /// - On failure, responds with [`SdkError<ListProvisionedConcurrencyConfigsError>`](crate::operation::list_provisioned_concurrency_configs::ListProvisionedConcurrencyConfigsError)
     pub fn list_provisioned_concurrency_configs(
         &self,
-    ) -> crate::operation::list_provisioned_concurrency_configs::builders::ListProvisionedConcurrencyConfigsFluentBuilder {
-        crate::operation::list_provisioned_concurrency_configs::builders::ListProvisionedConcurrencyConfigsFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::list_provisioned_concurrency_configs::builders::ListProvisionedConcurrencyConfigsFluentBuilder {
+        super::operation::list_provisioned_concurrency_configs::builders::ListProvisionedConcurrencyConfigsFluentBuilder::new(self.handle.clone())
     }
 }
