--- original
+++ modified
@@ -12,7 +12,7 @@
     ///   - [`next_marker(Option<String>)`](crate::operation::list_aliases::ListAliasesOutput::next_marker): <p>The pagination token that's included if more results are available.</p>
     ///   - [`aliases(Option<Vec::<AliasConfiguration>>)`](crate::operation::list_aliases::ListAliasesOutput::aliases): <p>A list of aliases.</p>
     /// - On failure, responds with [`SdkError<ListAliasesError>`](crate::operation::list_aliases::ListAliasesError)
-    pub fn list_aliases(&self) -> crate::operation::list_aliases::builders::ListAliasesFluentBuilder {
-        crate::operation::list_aliases::builders::ListAliasesFluentBuilder::new(self.handle.clone())
+    pub fn list_aliases(&self) -> super::super::operation::list_aliases::builders::ListAliasesFluentBuilder {
+        super::super::operation::list_aliases::builders::ListAliasesFluentBuilder::new(self.handle.clone())
     }
 }
