--- original
+++ modified
@@ -13,7 +13,7 @@
     /// - On failure, responds with [`SdkError<ListFunctionsByCodeSigningConfigError>`](crate::operation::list_functions_by_code_signing_config::ListFunctionsByCodeSigningConfigError)
     pub fn list_functions_by_code_signing_config(
         &self,
-    ) -> crate::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigFluentBuilder {
-        crate::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigFluentBuilder {
+        super::operation::list_functions_by_code_signing_config::builders::ListFunctionsByCodeSigningConfigFluentBuilder::new(self.handle.clone())
     }
 }
