--- original
+++ modified
@@ -68,7 +68,7 @@
     /// - On failure, responds with [`SdkError<UpdateFunctionConfigurationError>`](crate::operation::update_function_configuration::UpdateFunctionConfigurationError)
     pub fn update_function_configuration(
         &self,
-    ) -> crate::operation::update_function_configuration::builders::UpdateFunctionConfigurationFluentBuilder {
-        crate::operation::update_function_configuration::builders::UpdateFunctionConfigurationFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::update_function_configuration::builders::UpdateFunctionConfigurationFluentBuilder {
+        super::super::operation::update_function_configuration::builders::UpdateFunctionConfigurationFluentBuilder::new(self.handle.clone())
     }
 }
