--- original
+++ modified
@@ -10,7 +10,7 @@
     /// - On success, responds with [`UpdateCapacityProviderOutput`](crate::operation::update_capacity_provider::UpdateCapacityProviderOutput) with field(s):
     ///   - [`capacity_provider(Option<CapacityProvider>)`](crate::operation::update_capacity_provider::UpdateCapacityProviderOutput::capacity_provider): <p>Information about the updated capacity provider.</p>
     /// - On failure, responds with [`SdkError<UpdateCapacityProviderError>`](crate::operation::update_capacity_provider::UpdateCapacityProviderError)
-    pub fn update_capacity_provider(&self) -> crate::operation::update_capacity_provider::builders::UpdateCapacityProviderFluentBuilder {
-        crate::operation::update_capacity_provider::builders::UpdateCapacityProviderFluentBuilder::new(self.handle.clone())
+    pub fn update_capacity_provider(&self) -> super::operation::update_capacity_provider::builders::UpdateCapacityProviderFluentBuilder {
+        super::operation::update_capacity_provider::builders::UpdateCapacityProviderFluentBuilder::new(self.handle.clone())
     }
 }
