--- original
+++ modified
@@ -60,7 +60,7 @@
     ///   - [`event_source_mapping_arn(Option<String>)`](crate::operation::update_event_source_mapping::UpdateEventSourceMappingOutput::event_source_mapping_arn): <p>The Amazon Resource Name (ARN) of the event source mapping.</p>
     ///   - [`provisioned_poller_config(Option<ProvisionedPollerConfig>)`](crate::operation::update_event_source_mapping::UpdateEventSourceMappingOutput::provisioned_poller_config): <p>(Amazon SQS, Amazon MSK, and self-managed Apache Kafka only) The provisioned mode configuration for the event source. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/invocation-eventsourcemapping.html#invocation-eventsourcemapping-provisioned-mode">provisioned mode</a>.</p>
     /// - On failure, responds with [`SdkError<UpdateEventSourceMappingError>`](crate::operation::update_event_source_mapping::UpdateEventSourceMappingError)
-    pub fn update_event_source_mapping(&self) -> crate::operation::update_event_source_mapping::builders::UpdateEventSourceMappingFluentBuilder {
-        crate::operation::update_event_source_mapping::builders::UpdateEventSourceMappingFluentBuilder::new(self.handle.clone())
+    pub fn update_event_source_mapping(&self) -> super::operation::update_event_source_mapping::builders::UpdateEventSourceMappingFluentBuilder {
+        super::operation::update_event_source_mapping::builders::UpdateEventSourceMappingFluentBuilder::new(self.handle.clone())
     }
 }
