--- original
+++ modified
@@ -17,7 +17,7 @@
     /// - On failure, responds with [`SdkError<UpdateFunctionEventInvokeConfigError>`](crate::operation::update_function_event_invoke_config::UpdateFunctionEventInvokeConfigError)
     pub fn update_function_event_invoke_config(
         &self,
-    ) -> crate::operation::update_function_event_invoke_config::builders::UpdateFunctionEventInvokeConfigFluentBuilder {
-        crate::operation::update_function_event_invoke_config::builders::UpdateFunctionEventInvokeConfigFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::update_function_event_invoke_config::builders::UpdateFunctionEventInvokeConfigFluentBuilder {
+        super::operation::update_function_event_invoke_config::builders::UpdateFunctionEventInvokeConfigFluentBuilder::new(self.handle.clone())
     }
 }
