--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteAliasInput {
     /// Creates a new builder-style object to manufacture [`DeleteAliasInput`](crate::operation::delete_alias::DeleteAliasInput).
-    pub fn builder() -> crate::operation::delete_alias::builders::DeleteAliasInputBuilder {
-        crate::operation::delete_alias::builders::DeleteAliasInputBuilder::default()
+    pub fn builder() -> super::operation::delete_alias::builders::DeleteAliasInputBuilder {
+        super::operation::delete_alias::builders::DeleteAliasInputBuilder::default()
     }
 }

@@ -42,7 +42,7 @@
         &self.alias_name
     }
     /// Consumes the builder and constructs a [`DeleteAliasInput`](crate::operation::delete_alias::DeleteAliasInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::delete_alias::DeleteAliasInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::delete_alias::DeleteAliasInput { alias_name: self.alias_name })
+    pub fn build(self) -> ::std::result::Result<super::operation::delete_alias::DeleteAliasInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::delete_alias::DeleteAliasInput { alias_name: self.alias_name })
     }
 }
