--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl SetDefaultPolicyVersionInput {
     /// Creates a new builder-style object to manufacture [`SetDefaultPolicyVersionInput`](crate::operation::set_default_policy_version::SetDefaultPolicyVersionInput).
-    pub fn builder() -> crate::operation::set_default_policy_version::builders::SetDefaultPolicyVersionInputBuilder {
-        crate::operation::set_default_policy_version::builders::SetDefaultPolicyVersionInputBuilder::default()
+    pub fn builder() -> super::operation::set_default_policy_version::builders::SetDefaultPolicyVersionInputBuilder {
+        super::operation::set_default_policy_version::builders::SetDefaultPolicyVersionInputBuilder::default()
     }
 }

@@ -77,10 +77,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::set_default_policy_version::SetDefaultPolicyVersionInput,
+        super::operation::set_default_policy_version::SetDefaultPolicyVersionInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::set_default_policy_version::SetDefaultPolicyVersionInput {
+        ::std::result::Result::Ok(super::operation::set_default_policy_version::SetDefaultPolicyVersionInput {
             policy_arn: self.policy_arn,
             version_id: self.version_id,
         })
