--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetAsyncInvokeInput {
     /// Creates a new builder-style object to manufacture [`GetAsyncInvokeInput`](crate::operation::get_async_invoke::GetAsyncInvokeInput).
-    pub fn builder() -> crate::operation::get_async_invoke::builders::GetAsyncInvokeInputBuilder {
-        crate::operation::get_async_invoke::builders::GetAsyncInvokeInputBuilder::default()
+    pub fn builder() -> super::operation::get_async_invoke::builders::GetAsyncInvokeInputBuilder {
+        super::operation::get_async_invoke::builders::GetAsyncInvokeInputBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
     /// Consumes the builder and constructs a [`GetAsyncInvokeInput`](crate::operation::get_async_invoke::GetAsyncInvokeInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_async_invoke::GetAsyncInvokeInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_async_invoke::GetAsyncInvokeInput {
+    ) -> ::std::result::Result<super::operation::get_async_invoke::GetAsyncInvokeInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::get_async_invoke::GetAsyncInvokeInput {
             invocation_arn: self.invocation_arn,
         })
     }
