--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the tenant whose suppression list you want to retrieve. If you omit this parameter, the operation targets the account-level suppression list.</p>
     pub tenant_name: ::std::option::Option<::std::string::String>,
     /// <p>The factors that caused the email address to be added to the suppression list for your account or for a specific tenant.</p>
-    pub reasons: ::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>>,
+    pub reasons: ::std::option::Option<::std::vec::Vec<super::super::super::types::SuppressionListReason>>,
     /// <p>Used to filter the list of suppressed email destinations so that it only includes addresses that were added to the list after a specific date.</p>
     pub start_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>Used to filter the list of suppressed email destinations so that it only includes addresses that were added to the list before a specific date.</p>
@@ -25,7 +25,7 @@
     /// <p>The factors that caused the email address to be added to the suppression list for your account or for a specific tenant.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.reasons.is_none()`.
-    pub fn reasons(&self) -> &[crate::types::SuppressionListReason] {
+    pub fn reasons(&self) -> &[super::super::super::types::SuppressionListReason] {
         self.reasons.as_deref().unwrap_or_default()
     }
     /// <p>Used to filter the list of suppressed email destinations so that it only includes addresses that were added to the list after a specific date.</p>
@@ -47,8 +47,8 @@
 }
 impl ListSuppressedDestinationsInput {
     /// Creates a new builder-style object to manufacture [`ListSuppressedDestinationsInput`](crate::operation::list_suppressed_destinations::ListSuppressedDestinationsInput).
-    pub fn builder() -> crate::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsInputBuilder {
-        crate::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsInputBuilder {
+        super::super::super::operation::list_suppressed_destinations::builders::ListSuppressedDestinationsInputBuilder::default()
     }
 }

@@ -57,7 +57,7 @@
 #[non_exhaustive]
 pub struct ListSuppressedDestinationsInputBuilder {
     pub(crate) tenant_name: ::std::option::Option<::std::string::String>,
-    pub(crate) reasons: ::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>>,
+    pub(crate) reasons: ::std::option::Option<::std::vec::Vec<super::super::super::types::SuppressionListReason>>,
     pub(crate) start_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) end_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) next_token: ::std::option::Option<::std::string::String>,
@@ -83,7 +83,7 @@
     /// To override the contents of this collection use [`set_reasons`](Self::set_reasons).
     ///
     /// <p>The factors that caused the email address to be added to the suppression list for your account or for a specific tenant.</p>
-    pub fn reasons(mut self, input: crate::types::SuppressionListReason) -> Self {
+    pub fn reasons(mut self, input: super::super::super::types::SuppressionListReason) -> Self {
         let mut v = self.reasons.unwrap_or_default();
         v.push(input);
         self.reasons = ::std::option::Option::Some(v);
@@ -90,12 +90,12 @@
         self
     }
     /// <p>The factors that caused the email address to be added to the suppression list for your account or for a specific tenant.</p>
-    pub fn set_reasons(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>>) -> Self {
+    pub fn set_reasons(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::SuppressionListReason>>) -> Self {
         self.reasons = input;
         self
     }
     /// <p>The factors that caused the email address to be added to the suppression list for your account or for a specific tenant.</p>
-    pub fn get_reasons(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::SuppressionListReason>> {
+    pub fn get_reasons(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::SuppressionListReason>> {
         &self.reasons
     }
     /// <p>Used to filter the list of suppressed email destinations so that it only includes addresses that were added to the list after a specific date.</p>
@@ -158,10 +158,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_suppressed_destinations::ListSuppressedDestinationsInput,
+        super::super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_suppressed_destinations::ListSuppressedDestinationsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_suppressed_destinations::ListSuppressedDestinationsInput {
             tenant_name: self.tenant_name,
             reasons: self.reasons,
             start_date: self.start_date,
