--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<CreateTenantResourceAssociationError>`](crate::operation::create_tenant_resource_association::CreateTenantResourceAssociationError)
     pub fn create_tenant_resource_association(
         &self,
-    ) -> crate::operation::create_tenant_resource_association::builders::CreateTenantResourceAssociationFluentBuilder {
-        crate::operation::create_tenant_resource_association::builders::CreateTenantResourceAssociationFluentBuilder::new(self.handle.clone())
+    ) -> super::super::operation::create_tenant_resource_association::builders::CreateTenantResourceAssociationFluentBuilder {
+        super::super::operation::create_tenant_resource_association::builders::CreateTenantResourceAssociationFluentBuilder::new(self.handle.clone())
     }
 }
