--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`status(Option<Status>)`](crate::operation::create_multi_region_endpoint::CreateMultiRegionEndpointOutput::status): <p>A status of the multi-region endpoint (global-endpoint) right after the create request.</p> <ul>  <li>   <p><code>CREATING</code> – The resource is being provisioned.</p></li>  <li>   <p><code>READY</code> – The resource is ready to use.</p></li>  <li>   <p><code>FAILED</code> – The resource failed to be provisioned.</p></li>  <li>   <p><code>DELETING</code> – The resource is being deleted as requested.</p></li> </ul>
     ///   - [`endpoint_id(Option<String>)`](crate::operation::create_multi_region_endpoint::CreateMultiRegionEndpointOutput::endpoint_id): <p>The ID of the multi-region endpoint (global-endpoint).</p>
     /// - On failure, responds with [`SdkError<CreateMultiRegionEndpointError>`](crate::operation::create_multi_region_endpoint::CreateMultiRegionEndpointError)
-    pub fn create_multi_region_endpoint(&self) -> crate::operation::create_multi_region_endpoint::builders::CreateMultiRegionEndpointFluentBuilder {
-        crate::operation::create_multi_region_endpoint::builders::CreateMultiRegionEndpointFluentBuilder::new(self.handle.clone())
+    pub fn create_multi_region_endpoint(&self) -> super::operation::create_multi_region_endpoint::builders::CreateMultiRegionEndpointFluentBuilder {
+        super::operation::create_multi_region_endpoint::builders::CreateMultiRegionEndpointFluentBuilder::new(self.handle.clone())
     }
 }
