--- original
+++ modified
@@ -10,7 +10,7 @@
     ///   - [`attributes_data(impl Into<String>)`](crate::operation::create_contact::builders::CreateContactFluentBuilder::attributes_data) / [`set_attributes_data(Option<String>)`](crate::operation::create_contact::builders::CreateContactFluentBuilder::set_attributes_data):<br>required: **false**<br><p>The attribute data attached to a contact.</p><br>
     /// - On success, responds with [`CreateContactOutput`](crate::operation::create_contact::CreateContactOutput)
     /// - On failure, responds with [`SdkError<CreateContactError>`](crate::operation::create_contact::CreateContactError)
-    pub fn create_contact(&self) -> crate::operation::create_contact::builders::CreateContactFluentBuilder {
-        crate::operation::create_contact::builders::CreateContactFluentBuilder::new(self.handle.clone())
+    pub fn create_contact(&self) -> super::super::operation::create_contact::builders::CreateContactFluentBuilder {
+        super::super::operation::create_contact::builders::CreateContactFluentBuilder::new(self.handle.clone())
     }
 }
