--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`archiving_options(ArchivingOptions)`](crate::operation::create_configuration_set::builders::CreateConfigurationSetFluentBuilder::archiving_options) / [`set_archiving_options(Option<ArchivingOptions>)`](crate::operation::create_configuration_set::builders::CreateConfigurationSetFluentBuilder::set_archiving_options):<br>required: **false**<br><p>An object that defines the MailManager archiving options for emails that you send using the configuration set.</p><br>
     /// - On success, responds with [`CreateConfigurationSetOutput`](crate::operation::create_configuration_set::CreateConfigurationSetOutput)
     /// - On failure, responds with [`SdkError<CreateConfigurationSetError>`](crate::operation::create_configuration_set::CreateConfigurationSetError)
-    pub fn create_configuration_set(&self) -> crate::operation::create_configuration_set::builders::CreateConfigurationSetFluentBuilder {
-        crate::operation::create_configuration_set::builders::CreateConfigurationSetFluentBuilder::new(self.handle.clone())
+    pub fn create_configuration_set(&self) -> super::super::operation::create_configuration_set::builders::CreateConfigurationSetFluentBuilder {
+        super::super::operation::create_configuration_set::builders::CreateConfigurationSetFluentBuilder::new(self.handle.clone())
     }
 }
