--- original
+++ modified
@@ -8,7 +8,7 @@
     ///   - [`scaling_mode(ScalingMode)`](crate::operation::create_dedicated_ip_pool::builders::CreateDedicatedIpPoolFluentBuilder::scaling_mode) / [`set_scaling_mode(Option<ScalingMode>)`](crate::operation::create_dedicated_ip_pool::builders::CreateDedicatedIpPoolFluentBuilder::set_scaling_mode):<br>required: **false**<br><p>The type of scaling mode.</p><br>
     /// - On success, responds with [`CreateDedicatedIpPoolOutput`](crate::operation::create_dedicated_ip_pool::CreateDedicatedIpPoolOutput)
     /// - On failure, responds with [`SdkError<CreateDedicatedIpPoolError>`](crate::operation::create_dedicated_ip_pool::CreateDedicatedIpPoolError)
-    pub fn create_dedicated_ip_pool(&self) -> crate::operation::create_dedicated_ip_pool::builders::CreateDedicatedIpPoolFluentBuilder {
-        crate::operation::create_dedicated_ip_pool::builders::CreateDedicatedIpPoolFluentBuilder::new(self.handle.clone())
+    pub fn create_dedicated_ip_pool(&self) -> super::super::operation::create_dedicated_ip_pool::builders::CreateDedicatedIpPoolFluentBuilder {
+        super::super::operation::create_dedicated_ip_pool::builders::CreateDedicatedIpPoolFluentBuilder::new(self.handle.clone())
     }
 }
