--- original
+++ modified
@@ -9,8 +9,8 @@
     /// - On failure, responds with [`SdkError<DeleteConfigurationSetEventDestinationError>`](crate::operation::delete_configuration_set_event_destination::DeleteConfigurationSetEventDestinationError)
     pub fn delete_configuration_set_event_destination(
         &self,
-    ) -> crate::operation::delete_configuration_set_event_destination::builders::DeleteConfigurationSetEventDestinationFluentBuilder {
-        crate::operation::delete_configuration_set_event_destination::builders::DeleteConfigurationSetEventDestinationFluentBuilder::new(
+    ) -> super::operation::delete_configuration_set_event_destination::builders::DeleteConfigurationSetEventDestinationFluentBuilder {
+        super::operation::delete_configuration_set_event_destination::builders::DeleteConfigurationSetEventDestinationFluentBuilder::new(
             self.handle.clone(),
         )
     }
