--- original
+++ modified
@@ -8,8 +8,8 @@
     /// - On failure, responds with [`SdkError<DeleteCustomVerificationEmailTemplateError>`](crate::operation::delete_custom_verification_email_template::DeleteCustomVerificationEmailTemplateError)
     pub fn delete_custom_verification_email_template(
         &self,
-    ) -> crate::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateFluentBuilder {
-        crate::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateFluentBuilder::new(
+    ) -> super::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateFluentBuilder {
+        super::operation::delete_custom_verification_email_template::builders::DeleteCustomVerificationEmailTemplateFluentBuilder::new(
             self.handle.clone(),
         )
     }
