--- original
+++ modified
@@ -15,7 +15,7 @@
     ///   - [`created_timestamp(Option<DateTime>)`](crate::operation::get_contact::GetContactOutput::created_timestamp): <p>A timestamp noting when the contact was created.</p>
     ///   - [`last_updated_timestamp(Option<DateTime>)`](crate::operation::get_contact::GetContactOutput::last_updated_timestamp): <p>A timestamp noting the last time the contact's information was updated.</p>
     /// - On failure, responds with [`SdkError<GetContactError>`](crate::operation::get_contact::GetContactError)
-    pub fn get_contact(&self) -> crate::operation::get_contact::builders::GetContactFluentBuilder {
-        crate::operation::get_contact::builders::GetContactFluentBuilder::new(self.handle.clone())
+    pub fn get_contact(&self) -> super::operation::get_contact::builders::GetContactFluentBuilder {
+        super::operation::get_contact::builders::GetContactFluentBuilder::new(self.handle.clone())
     }
 }
