--- original
+++ modified
@@ -15,7 +15,7 @@
     ///   - [`vdm_options(Option<VdmOptions>)`](crate::operation::get_configuration_set::GetConfigurationSetOutput::vdm_options): <p>An object that contains information about the VDM preferences for your configuration set.</p>
     ///   - [`archiving_options(Option<ArchivingOptions>)`](crate::operation::get_configuration_set::GetConfigurationSetOutput::archiving_options): <p>An object that defines the MailManager archive where sent emails are archived that you send using the configuration set.</p>
     /// - On failure, responds with [`SdkError<GetConfigurationSetError>`](crate::operation::get_configuration_set::GetConfigurationSetError)
-    pub fn get_configuration_set(&self) -> crate::operation::get_configuration_set::builders::GetConfigurationSetFluentBuilder {
-        crate::operation::get_configuration_set::builders::GetConfigurationSetFluentBuilder::new(self.handle.clone())
+    pub fn get_configuration_set(&self) -> super::operation::get_configuration_set::builders::GetConfigurationSetFluentBuilder {
+        super::operation::get_configuration_set::builders::GetConfigurationSetFluentBuilder::new(self.handle.clone())
     }
 }
