--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`GetSuppressedDestinationOutput`](crate::operation::get_suppressed_destination::GetSuppressedDestinationOutput) with field(s):
     ///   - [`suppressed_destination(Option<SuppressedDestination>)`](crate::operation::get_suppressed_destination::GetSuppressedDestinationOutput::suppressed_destination): <p>An object containing information about the suppressed email address.</p>
     /// - On failure, responds with [`SdkError<GetSuppressedDestinationError>`](crate::operation::get_suppressed_destination::GetSuppressedDestinationError)
-    pub fn get_suppressed_destination(&self) -> crate::operation::get_suppressed_destination::builders::GetSuppressedDestinationFluentBuilder {
-        crate::operation::get_suppressed_destination::builders::GetSuppressedDestinationFluentBuilder::new(self.handle.clone())
+    pub fn get_suppressed_destination(&self) -> super::super::operation::get_suppressed_destination::builders::GetSuppressedDestinationFluentBuilder {
+        super::super::operation::get_suppressed_destination::builders::GetSuppressedDestinationFluentBuilder::new(self.handle.clone())
     }
 }
