--- original
+++ modified
@@ -15,7 +15,7 @@
     /// - On failure, responds with [`SdkError<GetCustomVerificationEmailTemplateError>`](crate::operation::get_custom_verification_email_template::GetCustomVerificationEmailTemplateError)
     pub fn get_custom_verification_email_template(
         &self,
-    ) -> crate::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateFluentBuilder {
-        crate::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateFluentBuilder {
+        super::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateFluentBuilder::new(self.handle.clone())
     }
 }
