--- original
+++ modified
@@ -9,8 +9,8 @@
     /// - On failure, responds with [`SdkError<GetConfigurationSetEventDestinationsError>`](crate::operation::get_configuration_set_event_destinations::GetConfigurationSetEventDestinationsError)
     pub fn get_configuration_set_event_destinations(
         &self,
-    ) -> crate::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsFluentBuilder {
-        crate::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsFluentBuilder::new(
+    ) -> super::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsFluentBuilder {
+        super::operation::get_configuration_set_event_destinations::builders::GetConfigurationSetEventDestinationsFluentBuilder::new(
             self.handle.clone(),
         )
     }
