--- original
+++ modified
@@ -14,7 +14,7 @@
     ///   - [`vdm_attributes(Option<VdmAttributes>)`](crate::operation::get_account::GetAccountOutput::vdm_attributes): <p>The VDM attributes that apply to your Amazon SES account.</p>
     ///   - [`pricing_attributes(Option<PricingAttributes>)`](crate::operation::get_account::GetAccountOutput::pricing_attributes): <p>The pricing attributes that apply to your Amazon SES account, including the currently active pricing plan and any scheduled change.</p>
     /// - On failure, responds with [`SdkError<GetAccountError>`](crate::operation::get_account::GetAccountError)
-    pub fn get_account(&self) -> crate::operation::get_account::builders::GetAccountFluentBuilder {
-        crate::operation::get_account::builders::GetAccountFluentBuilder::new(self.handle.clone())
+    pub fn get_account(&self) -> super::operation::get_account::builders::GetAccountFluentBuilder {
+        super::operation::get_account::builders::GetAccountFluentBuilder::new(self.handle.clone())
     }
 }
