--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On failure, responds with [`SdkError<PutAccountSendingAttributesError>`](crate::operation::put_account_sending_attributes::PutAccountSendingAttributesError)
     pub fn put_account_sending_attributes(
         &self,
-    ) -> crate::operation::put_account_sending_attributes::builders::PutAccountSendingAttributesFluentBuilder {
-        crate::operation::put_account_sending_attributes::builders::PutAccountSendingAttributesFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::put_account_sending_attributes::builders::PutAccountSendingAttributesFluentBuilder {
+        super::operation::put_account_sending_attributes::builders::PutAccountSendingAttributesFluentBuilder::new(self.handle.clone())
     }
 }
