--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<PutConfigurationSetSendingOptionsError>`](crate::operation::put_configuration_set_sending_options::PutConfigurationSetSendingOptionsError)
     pub fn put_configuration_set_sending_options(
         &self,
-    ) -> crate::operation::put_configuration_set_sending_options::builders::PutConfigurationSetSendingOptionsFluentBuilder {
-        crate::operation::put_configuration_set_sending_options::builders::PutConfigurationSetSendingOptionsFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::put_configuration_set_sending_options::builders::PutConfigurationSetSendingOptionsFluentBuilder {
+        super::operation::put_configuration_set_sending_options::builders::PutConfigurationSetSendingOptionsFluentBuilder::new(self.handle.clone())
     }
 }
