--- original
+++ modified
@@ -8,8 +8,8 @@
     /// - On failure, responds with [`SdkError<PutAccountDedicatedIpWarmupAttributesError>`](crate::operation::put_account_dedicated_ip_warmup_attributes::PutAccountDedicatedIpWarmupAttributesError)
     pub fn put_account_dedicated_ip_warmup_attributes(
         &self,
-    ) -> crate::operation::put_account_dedicated_ip_warmup_attributes::builders::PutAccountDedicatedIpWarmupAttributesFluentBuilder {
-        crate::operation::put_account_dedicated_ip_warmup_attributes::builders::PutAccountDedicatedIpWarmupAttributesFluentBuilder::new(
+    ) -> super::operation::put_account_dedicated_ip_warmup_attributes::builders::PutAccountDedicatedIpWarmupAttributesFluentBuilder {
+        super::operation::put_account_dedicated_ip_warmup_attributes::builders::PutAccountDedicatedIpWarmupAttributesFluentBuilder::new(
             self.handle.clone(),
         )
     }
