--- original
+++ modified
@@ -10,8 +10,8 @@
     /// - On failure, responds with [`SdkError<UpdateConfigurationSetEventDestinationError>`](crate::operation::update_configuration_set_event_destination::UpdateConfigurationSetEventDestinationError)
     pub fn update_configuration_set_event_destination(
         &self,
-    ) -> crate::operation::update_configuration_set_event_destination::builders::UpdateConfigurationSetEventDestinationFluentBuilder {
-        crate::operation::update_configuration_set_event_destination::builders::UpdateConfigurationSetEventDestinationFluentBuilder::new(
+    ) -> super::operation::update_configuration_set_event_destination::builders::UpdateConfigurationSetEventDestinationFluentBuilder {
+        super::operation::update_configuration_set_event_destination::builders::UpdateConfigurationSetEventDestinationFluentBuilder::new(
             self.handle.clone(),
         )
     }
