--- original
+++ modified
@@ -13,8 +13,8 @@
     /// - On failure, responds with [`SdkError<UpdateCustomVerificationEmailTemplateError>`](crate::operation::update_custom_verification_email_template::UpdateCustomVerificationEmailTemplateError)
     pub fn update_custom_verification_email_template(
         &self,
-    ) -> crate::operation::update_custom_verification_email_template::builders::UpdateCustomVerificationEmailTemplateFluentBuilder {
-        crate::operation::update_custom_verification_email_template::builders::UpdateCustomVerificationEmailTemplateFluentBuilder::new(
+    ) -> super::operation::update_custom_verification_email_template::builders::UpdateCustomVerificationEmailTemplateFluentBuilder {
+        super::operation::update_custom_verification_email_template::builders::UpdateCustomVerificationEmailTemplateFluentBuilder::new(
             self.handle.clone(),
         )
     }
