--- original
+++ modified
@@ -10,8 +10,8 @@
     /// - On failure, responds with [`SdkError<UpdateReputationEntityCustomerManagedStatusError>`](crate::operation::update_reputation_entity_customer_managed_status::UpdateReputationEntityCustomerManagedStatusError)
     pub fn update_reputation_entity_customer_managed_status(
         &self,
-    ) -> crate::operation::update_reputation_entity_customer_managed_status::builders::UpdateReputationEntityCustomerManagedStatusFluentBuilder {
-        crate::operation::update_reputation_entity_customer_managed_status::builders::UpdateReputationEntityCustomerManagedStatusFluentBuilder::new(
+    ) -> super::super::operation::update_reputation_entity_customer_managed_status::builders::UpdateReputationEntityCustomerManagedStatusFluentBuilder {
+        super::super::operation::update_reputation_entity_customer_managed_status::builders::UpdateReputationEntityCustomerManagedStatusFluentBuilder::new(
             self.handle.clone(),
         )
     }
