--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl ListUserPoolClientsInput {
     /// Creates a new builder-style object to manufacture [`ListUserPoolClientsInput`](crate::operation::list_user_pool_clients::ListUserPoolClientsInput).
-    pub fn builder() -> crate::operation::list_user_pool_clients::builders::ListUserPoolClientsInputBuilder {
-        crate::operation::list_user_pool_clients::builders::ListUserPoolClientsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_user_pool_clients::builders::ListUserPoolClientsInputBuilder {
+        super::super::super::operation::list_user_pool_clients::builders::ListUserPoolClientsInputBuilder::default()
     }
 }

@@ -87,9 +87,9 @@
     /// Consumes the builder and constructs a [`ListUserPoolClientsInput`](crate::operation::list_user_pool_clients::ListUserPoolClientsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_user_pool_clients::ListUserPoolClientsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::list_user_pool_clients::ListUserPoolClientsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_user_pool_clients::ListUserPoolClientsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_user_pool_clients::ListUserPoolClientsInput {
             user_pool_id: self.user_pool_id,
             max_results: self.max_results,
             next_token: self.next_token,
