--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl AddKeyEntry {
     /// Creates a new builder-style object to manufacture [`AddKeyEntry`](crate::types::AddKeyEntry).
-    pub fn builder() -> crate::types::builders::AddKeyEntryBuilder {
-        crate::types::builders::AddKeyEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::AddKeyEntryBuilder {
+        super::super::types::builders::AddKeyEntryBuilder::default()
     }
 }

@@ -91,8 +91,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`key`](crate::types::builders::AddKeyEntryBuilder::key)
     /// - [`value`](crate::types::builders::AddKeyEntryBuilder::value)
-    pub fn build(self) -> ::std::result::Result<crate::types::AddKeyEntry, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::AddKeyEntry {
+    pub fn build(self) -> ::std::result::Result<super::super::types::AddKeyEntry, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::AddKeyEntry {
             key: self.key.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "key",
