--- original
+++ modified
@@ -8,7 +8,7 @@
     pub log_group_count: ::std::option::Option<i32>,
     /// <p>An array of key-value pairs that identify the data source characteristics used to group the log groups.</p>
     /// <p>The size and content of this array depends on the <code>groupBy</code> parameter specified in the request.</p>
-    pub grouping_identifiers: ::std::option::Option<::std::vec::Vec<crate::types::GroupingIdentifier>>,
+    pub grouping_identifiers: ::std::option::Option<::std::vec::Vec<super::types::GroupingIdentifier>>,
 }
 impl AggregateLogGroupSummary {
     /// <p>The number of log groups in this aggregate summary group.</p>
@@ -19,14 +19,14 @@
     /// <p>The size and content of this array depends on the <code>groupBy</code> parameter specified in the request.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.grouping_identifiers.is_none()`.
-    pub fn grouping_identifiers(&self) -> &[crate::types::GroupingIdentifier] {
+    pub fn grouping_identifiers(&self) -> &[super::types::GroupingIdentifier] {
         self.grouping_identifiers.as_deref().unwrap_or_default()
     }
 }
 impl AggregateLogGroupSummary {
     /// Creates a new builder-style object to manufacture [`AggregateLogGroupSummary`](crate::types::AggregateLogGroupSummary).
-    pub fn builder() -> crate::types::builders::AggregateLogGroupSummaryBuilder {
-        crate::types::builders::AggregateLogGroupSummaryBuilder::default()
+    pub fn builder() -> super::types::builders::AggregateLogGroupSummaryBuilder {
+        super::types::builders::AggregateLogGroupSummaryBuilder::default()
     }
 }

@@ -35,7 +35,7 @@
 #[non_exhaustive]
 pub struct AggregateLogGroupSummaryBuilder {
     pub(crate) log_group_count: ::std::option::Option<i32>,
-    pub(crate) grouping_identifiers: ::std::option::Option<::std::vec::Vec<crate::types::GroupingIdentifier>>,
+    pub(crate) grouping_identifiers: ::std::option::Option<::std::vec::Vec<super::types::GroupingIdentifier>>,
 }
 impl AggregateLogGroupSummaryBuilder {
     /// <p>The number of log groups in this aggregate summary group.</p>
@@ -58,7 +58,7 @@
     ///
     /// <p>An array of key-value pairs that identify the data source characteristics used to group the log groups.</p>
     /// <p>The size and content of this array depends on the <code>groupBy</code> parameter specified in the request.</p>
-    pub fn grouping_identifiers(mut self, input: crate::types::GroupingIdentifier) -> Self {
+    pub fn grouping_identifiers(mut self, input: super::types::GroupingIdentifier) -> Self {
         let mut v = self.grouping_identifiers.unwrap_or_default();
         v.push(input);
         self.grouping_identifiers = ::std::option::Option::Some(v);
@@ -66,18 +66,18 @@
     }
     /// <p>An array of key-value pairs that identify the data source characteristics used to group the log groups.</p>
     /// <p>The size and content of this array depends on the <code>groupBy</code> parameter specified in the request.</p>
-    pub fn set_grouping_identifiers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::GroupingIdentifier>>) -> Self {
+    pub fn set_grouping_identifiers(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::GroupingIdentifier>>) -> Self {
         self.grouping_identifiers = input;
         self
     }
     /// <p>An array of key-value pairs that identify the data source characteristics used to group the log groups.</p>
     /// <p>The size and content of this array depends on the <code>groupBy</code> parameter specified in the request.</p>
-    pub fn get_grouping_identifiers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::GroupingIdentifier>> {
+    pub fn get_grouping_identifiers(&self) -> &::std::option::Option<::std::vec::Vec<super::types::GroupingIdentifier>> {
         &self.grouping_identifiers
     }
     /// Consumes the builder and constructs a [`AggregateLogGroupSummary`](crate::types::AggregateLogGroupSummary).
-    pub fn build(self) -> crate::types::AggregateLogGroupSummary {
-        crate::types::AggregateLogGroupSummary {
+    pub fn build(self) -> super::types::AggregateLogGroupSummary {
+        super::types::AggregateLogGroupSummary {
             log_group_count: self.log_group_count,
             grouping_identifiers: self.grouping_identifiers,
         }
