--- original
+++ modified
@@ -57,8 +57,8 @@
 }
 impl DateTimeConverter {
     /// Creates a new builder-style object to manufacture [`DateTimeConverter`](crate::types::DateTimeConverter).
-    pub fn builder() -> crate::types::builders::DateTimeConverterBuilder {
-        crate::types::builders::DateTimeConverterBuilder::default()
+    pub fn builder() -> super::types::builders::DateTimeConverterBuilder {
+        super::types::builders::DateTimeConverterBuilder::default()
     }
 }

@@ -189,8 +189,8 @@
     /// - [`source`](crate::types::builders::DateTimeConverterBuilder::source)
     /// - [`target`](crate::types::builders::DateTimeConverterBuilder::target)
     /// - [`match_patterns`](crate::types::builders::DateTimeConverterBuilder::match_patterns)
-    pub fn build(self) -> ::std::result::Result<crate::types::DateTimeConverter, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::DateTimeConverter {
+    pub fn build(self) -> ::std::result::Result<super::types::DateTimeConverter, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::DateTimeConverter {
             source: self.source.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "source",
