--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl DeliveryDestinationConfiguration {
     /// Creates a new builder-style object to manufacture [`DeliveryDestinationConfiguration`](crate::types::DeliveryDestinationConfiguration).
-    pub fn builder() -> crate::types::builders::DeliveryDestinationConfigurationBuilder {
-        crate::types::builders::DeliveryDestinationConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeliveryDestinationConfigurationBuilder {
+        super::super::types::builders::DeliveryDestinationConfigurationBuilder::default()
     }
 }

@@ -46,8 +46,8 @@
     /// Consumes the builder and constructs a [`DeliveryDestinationConfiguration`](crate::types::DeliveryDestinationConfiguration).
     /// This method will fail if any of the following fields are not set:
     /// - [`destination_resource_arn`](crate::types::builders::DeliveryDestinationConfigurationBuilder::destination_resource_arn)
-    pub fn build(self) -> ::std::result::Result<crate::types::DeliveryDestinationConfiguration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::DeliveryDestinationConfiguration {
+    pub fn build(self) -> ::std::result::Result<super::super::types::DeliveryDestinationConfiguration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::DeliveryDestinationConfiguration {
             destination_resource_arn: self.destination_resource_arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "destination_resource_arn",
