--- original
+++ modified
@@ -16,13 +16,13 @@
     /// <p>The ARN of the delivery destination that is associated with this delivery.</p>
     pub delivery_destination_arn: ::std::option::Option<::std::string::String>,
     /// <p>Displays whether the delivery destination associated with this delivery is CloudWatch Logs, Amazon S3, Firehose, or X-Ray.</p>
-    pub delivery_destination_type: ::std::option::Option<crate::types::DeliveryDestinationType>,
+    pub delivery_destination_type: ::std::option::Option<super::types::DeliveryDestinationType>,
     /// <p>The record fields used in this delivery.</p>
     pub record_fields: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>The field delimiter that is used between record fields when the final output format of a delivery is in <code>Plain</code>, <code>W3C</code>, or <code>Raw</code> format.</p>
     pub field_delimiter: ::std::option::Option<::std::string::String>,
     /// <p>This structure contains delivery configurations that apply only when the delivery destination resource is an S3 bucket.</p>
-    pub s3_delivery_configuration: ::std::option::Option<crate::types::S3DeliveryConfiguration>,
+    pub s3_delivery_configuration: ::std::option::Option<super::types::S3DeliveryConfiguration>,
     /// <p>The tags that have been assigned to this delivery.</p>
     pub tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
@@ -44,7 +44,7 @@
         self.delivery_destination_arn.as_deref()
     }
     /// <p>Displays whether the delivery destination associated with this delivery is CloudWatch Logs, Amazon S3, Firehose, or X-Ray.</p>
-    pub fn delivery_destination_type(&self) -> ::std::option::Option<&crate::types::DeliveryDestinationType> {
+    pub fn delivery_destination_type(&self) -> ::std::option::Option<&super::types::DeliveryDestinationType> {
         self.delivery_destination_type.as_ref()
     }
     /// <p>The record fields used in this delivery.</p>
@@ -58,7 +58,7 @@
         self.field_delimiter.as_deref()
     }
     /// <p>This structure contains delivery configurations that apply only when the delivery destination resource is an S3 bucket.</p>
-    pub fn s3_delivery_configuration(&self) -> ::std::option::Option<&crate::types::S3DeliveryConfiguration> {
+    pub fn s3_delivery_configuration(&self) -> ::std::option::Option<&super::types::S3DeliveryConfiguration> {
         self.s3_delivery_configuration.as_ref()
     }
     /// <p>The tags that have been assigned to this delivery.</p>
@@ -68,8 +68,8 @@
 }
 impl Delivery {
     /// Creates a new builder-style object to manufacture [`Delivery`](crate::types::Delivery).
-    pub fn builder() -> crate::types::builders::DeliveryBuilder {
-        crate::types::builders::DeliveryBuilder::default()
+    pub fn builder() -> super::types::builders::DeliveryBuilder {
+        super::types::builders::DeliveryBuilder::default()
     }
 }

@@ -81,10 +81,10 @@
     pub(crate) arn: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_source_name: ::std::option::Option<::std::string::String>,
     pub(crate) delivery_destination_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) delivery_destination_type: ::std::option::Option<crate::types::DeliveryDestinationType>,
+    pub(crate) delivery_destination_type: ::std::option::Option<super::types::DeliveryDestinationType>,
     pub(crate) record_fields: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) field_delimiter: ::std::option::Option<::std::string::String>,
-    pub(crate) s3_delivery_configuration: ::std::option::Option<crate::types::S3DeliveryConfiguration>,
+    pub(crate) s3_delivery_configuration: ::std::option::Option<super::types::S3DeliveryConfiguration>,
     pub(crate) tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
 impl DeliveryBuilder {
@@ -145,17 +145,17 @@
         &self.delivery_destination_arn
     }
     /// <p>Displays whether the delivery destination associated with this delivery is CloudWatch Logs, Amazon S3, Firehose, or X-Ray.</p>
-    pub fn delivery_destination_type(mut self, input: crate::types::DeliveryDestinationType) -> Self {
+    pub fn delivery_destination_type(mut self, input: super::types::DeliveryDestinationType) -> Self {
         self.delivery_destination_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>Displays whether the delivery destination associated with this delivery is CloudWatch Logs, Amazon S3, Firehose, or X-Ray.</p>
-    pub fn set_delivery_destination_type(mut self, input: ::std::option::Option<crate::types::DeliveryDestinationType>) -> Self {
+    pub fn set_delivery_destination_type(mut self, input: ::std::option::Option<super::types::DeliveryDestinationType>) -> Self {
         self.delivery_destination_type = input;
         self
     }
     /// <p>Displays whether the delivery destination associated with this delivery is CloudWatch Logs, Amazon S3, Firehose, or X-Ray.</p>
-    pub fn get_delivery_destination_type(&self) -> &::std::option::Option<crate::types::DeliveryDestinationType> {
+    pub fn get_delivery_destination_type(&self) -> &::std::option::Option<super::types::DeliveryDestinationType> {
         &self.delivery_destination_type
     }
     /// Appends an item to `record_fields`.
@@ -193,17 +193,17 @@
         &self.field_delimiter
     }
     /// <p>This structure contains delivery configurations that apply only when the delivery destination resource is an S3 bucket.</p>
-    pub fn s3_delivery_configuration(mut self, input: crate::types::S3DeliveryConfiguration) -> Self {
+    pub fn s3_delivery_configuration(mut self, input: super::types::S3DeliveryConfiguration) -> Self {
         self.s3_delivery_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>This structure contains delivery configurations that apply only when the delivery destination resource is an S3 bucket.</p>
-    pub fn set_s3_delivery_configuration(mut self, input: ::std::option::Option<crate::types::S3DeliveryConfiguration>) -> Self {
+    pub fn set_s3_delivery_configuration(mut self, input: ::std::option::Option<super::types::S3DeliveryConfiguration>) -> Self {
         self.s3_delivery_configuration = input;
         self
     }
     /// <p>This structure contains delivery configurations that apply only when the delivery destination resource is an S3 bucket.</p>
-    pub fn get_s3_delivery_configuration(&self) -> &::std::option::Option<crate::types::S3DeliveryConfiguration> {
+    pub fn get_s3_delivery_configuration(&self) -> &::std::option::Option<super::types::S3DeliveryConfiguration> {
         &self.s3_delivery_configuration
     }
     /// Adds a key-value pair to `tags`.
@@ -227,8 +227,8 @@
         &self.tags
     }
     /// Consumes the builder and constructs a [`Delivery`](crate::types::Delivery).
-    pub fn build(self) -> crate::types::Delivery {
-        crate::types::Delivery {
+    pub fn build(self) -> super::types::Delivery {
+        super::types::Delivery {
             id: self.id,
             arn: self.arn,
             delivery_source_name: self.delivery_source_name,
