--- original
+++ modified
@@ -32,9 +32,9 @@
     /// <p>The map of key-value pairs that configure the delivery source.</p>
     pub delivery_source_configuration: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     /// <p>The status of the delivery source. A delivery source can have the status <code>ACTIVE</code> or <code>INACTIVE</code>. Note: This value is defined for selective log types.</p>
-    pub status: ::std::option::Option<crate::types::DeliverySourceStatus>,
+    pub status: ::std::option::Option<super::types::DeliverySourceStatus>,
     /// <p>The reason for the status of the delivery source. A status reason of <code>RESOURCE_DELETED</code> indicates that the resource associated with the delivery source has been deleted. Note: This value is defined for selective log types.</p>
-    pub status_reason: ::std::option::Option<crate::types::DeliverySourceStatusReason>,
+    pub status_reason: ::std::option::Option<super::types::DeliverySourceStatusReason>,
 }
 impl DeliverySource {
     /// <p>The unique name of the delivery source.</p>
@@ -68,18 +68,18 @@
         self.delivery_source_configuration.as_ref()
     }
     /// <p>The status of the delivery source. A delivery source can have the status <code>ACTIVE</code> or <code>INACTIVE</code>. Note: This value is defined for selective log types.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::DeliverySourceStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::DeliverySourceStatus> {
         self.status.as_ref()
     }
     /// <p>The reason for the status of the delivery source. A status reason of <code>RESOURCE_DELETED</code> indicates that the resource associated with the delivery source has been deleted. Note: This value is defined for selective log types.</p>
-    pub fn status_reason(&self) -> ::std::option::Option<&crate::types::DeliverySourceStatusReason> {
+    pub fn status_reason(&self) -> ::std::option::Option<&super::types::DeliverySourceStatusReason> {
         self.status_reason.as_ref()
     }
 }
 impl DeliverySource {
     /// Creates a new builder-style object to manufacture [`DeliverySource`](crate::types::DeliverySource).
-    pub fn builder() -> crate::types::builders::DeliverySourceBuilder {
-        crate::types::builders::DeliverySourceBuilder::default()
+    pub fn builder() -> super::types::builders::DeliverySourceBuilder {
+        super::types::builders::DeliverySourceBuilder::default()
     }
 }

@@ -94,8 +94,8 @@
     pub(crate) log_type: ::std::option::Option<::std::string::String>,
     pub(crate) tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     pub(crate) delivery_source_configuration: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
-    pub(crate) status: ::std::option::Option<crate::types::DeliverySourceStatus>,
-    pub(crate) status_reason: ::std::option::Option<crate::types::DeliverySourceStatusReason>,
+    pub(crate) status: ::std::option::Option<super::types::DeliverySourceStatus>,
+    pub(crate) status_reason: ::std::option::Option<super::types::DeliverySourceStatusReason>,
 }
 impl DeliverySourceBuilder {
     /// <p>The unique name of the delivery source.</p>
@@ -224,36 +224,36 @@
         &self.delivery_source_configuration
     }
     /// <p>The status of the delivery source. A delivery source can have the status <code>ACTIVE</code> or <code>INACTIVE</code>. Note: This value is defined for selective log types.</p>
-    pub fn status(mut self, input: crate::types::DeliverySourceStatus) -> Self {
+    pub fn status(mut self, input: super::types::DeliverySourceStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The status of the delivery source. A delivery source can have the status <code>ACTIVE</code> or <code>INACTIVE</code>. Note: This value is defined for selective log types.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::DeliverySourceStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::DeliverySourceStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>The status of the delivery source. A delivery source can have the status <code>ACTIVE</code> or <code>INACTIVE</code>. Note: This value is defined for selective log types.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::DeliverySourceStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::DeliverySourceStatus> {
         &self.status
     }
     /// <p>The reason for the status of the delivery source. A status reason of <code>RESOURCE_DELETED</code> indicates that the resource associated with the delivery source has been deleted. Note: This value is defined for selective log types.</p>
-    pub fn status_reason(mut self, input: crate::types::DeliverySourceStatusReason) -> Self {
+    pub fn status_reason(mut self, input: super::types::DeliverySourceStatusReason) -> Self {
         self.status_reason = ::std::option::Option::Some(input);
         self
     }
     /// <p>The reason for the status of the delivery source. A status reason of <code>RESOURCE_DELETED</code> indicates that the resource associated with the delivery source has been deleted. Note: This value is defined for selective log types.</p>
-    pub fn set_status_reason(mut self, input: ::std::option::Option<crate::types::DeliverySourceStatusReason>) -> Self {
+    pub fn set_status_reason(mut self, input: ::std::option::Option<super::types::DeliverySourceStatusReason>) -> Self {
         self.status_reason = input;
         self
     }
     /// <p>The reason for the status of the delivery source. A status reason of <code>RESOURCE_DELETED</code> indicates that the resource associated with the delivery source has been deleted. Note: This value is defined for selective log types.</p>
-    pub fn get_status_reason(&self) -> &::std::option::Option<crate::types::DeliverySourceStatusReason> {
+    pub fn get_status_reason(&self) -> &::std::option::Option<super::types::DeliverySourceStatusReason> {
         &self.status_reason
     }
     /// Consumes the builder and constructs a [`DeliverySource`](crate::types::DeliverySource).
-    pub fn build(self) -> crate::types::DeliverySource {
-        crate::types::DeliverySource {
+    pub fn build(self) -> super::types::DeliverySource {
+        super::types::DeliverySource {
             name: self.name,
             arn: self.arn,
             resource_arns: self.resource_arns,
