--- original
+++ modified
@@ -45,8 +45,8 @@
 }
 impl Destination {
     /// Creates a new builder-style object to manufacture [`Destination`](crate::types::Destination).
-    pub fn builder() -> crate::types::builders::DestinationBuilder {
-        crate::types::builders::DestinationBuilder::default()
+    pub fn builder() -> super::super::types::builders::DestinationBuilder {
+        super::super::types::builders::DestinationBuilder::default()
     }
 }

@@ -147,8 +147,8 @@
         &self.creation_time
     }
     /// Consumes the builder and constructs a [`Destination`](crate::types::Destination).
-    pub fn build(self) -> crate::types::Destination {
-        crate::types::Destination {
+    pub fn build(self) -> super::super::types::Destination {
+        super::super::types::Destination {
             destination_name: self.destination_name,
             target_arn: self.target_arn,
             role_arn: self.role_arn,
