--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ExportTaskStatus {
     /// <p>The status code of the export task.</p>
-    pub code: ::std::option::Option<crate::types::ExportTaskStatusCode>,
+    pub code: ::std::option::Option<super::super::types::ExportTaskStatusCode>,
     /// <p>The status message related to the status code.</p>
     pub message: ::std::option::Option<::std::string::String>,
 }
 impl ExportTaskStatus {
     /// <p>The status code of the export task.</p>
-    pub fn code(&self) -> ::std::option::Option<&crate::types::ExportTaskStatusCode> {
+    pub fn code(&self) -> ::std::option::Option<&super::super::types::ExportTaskStatusCode> {
         self.code.as_ref()
     }
     /// <p>The status message related to the status code.</p>
@@ -21,8 +21,8 @@
 }
 impl ExportTaskStatus {
     /// Creates a new builder-style object to manufacture [`ExportTaskStatus`](crate::types::ExportTaskStatus).
-    pub fn builder() -> crate::types::builders::ExportTaskStatusBuilder {
-        crate::types::builders::ExportTaskStatusBuilder::default()
+    pub fn builder() -> super::super::types::builders::ExportTaskStatusBuilder {
+        super::super::types::builders::ExportTaskStatusBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ExportTaskStatusBuilder {
-    pub(crate) code: ::std::option::Option<crate::types::ExportTaskStatusCode>,
+    pub(crate) code: ::std::option::Option<super::super::types::ExportTaskStatusCode>,
     pub(crate) message: ::std::option::Option<::std::string::String>,
 }
 impl ExportTaskStatusBuilder {
     /// <p>The status code of the export task.</p>
-    pub fn code(mut self, input: crate::types::ExportTaskStatusCode) -> Self {
+    pub fn code(mut self, input: super::super::types::ExportTaskStatusCode) -> Self {
         self.code = ::std::option::Option::Some(input);
         self
     }
     /// <p>The status code of the export task.</p>
-    pub fn set_code(mut self, input: ::std::option::Option<crate::types::ExportTaskStatusCode>) -> Self {
+    pub fn set_code(mut self, input: ::std::option::Option<super::super::types::ExportTaskStatusCode>) -> Self {
         self.code = input;
         self
     }
     /// <p>The status code of the export task.</p>
-    pub fn get_code(&self) -> &::std::option::Option<crate::types::ExportTaskStatusCode> {
+    pub fn get_code(&self) -> &::std::option::Option<super::super::types::ExportTaskStatusCode> {
         &self.code
     }
     /// <p>The status message related to the status code.</p>
@@ -63,8 +63,8 @@
         &self.message
     }
     /// Consumes the builder and constructs a [`ExportTaskStatus`](crate::types::ExportTaskStatus).
-    pub fn build(self) -> crate::types::ExportTaskStatus {
-        crate::types::ExportTaskStatus {
+    pub fn build(self) -> super::super::types::ExportTaskStatus {
+        super::super::types::ExportTaskStatus {
             code: self.code,
             message: self.message,
         }
