--- original
+++ modified
@@ -15,7 +15,7 @@
     /// <p>The time and date of the most recent log event that matches this field index.</p>
     pub last_event_time: ::std::option::Option<i64>,
     /// <p>The type of index. Specify <code>FACET</code> for facet-based indexing or <code>FIELD_INDEX</code> for field-based indexing. This determines how the field is indexed and can be queried.</p>
-    pub r#type: ::std::option::Option<crate::types::IndexType>,
+    pub r#type: ::std::option::Option<super::types::IndexType>,
     /// <p>The category of the field index:</p>
     /// <ul>
     /// <li>
@@ -28,7 +28,7 @@
     /// <p><code>INACTIVE</code>: Fields that CloudWatch Logs indexed before but does not index now. This happens if you remove a field from the field index policy or if CloudWatch Logs automatically selects a different field based on your queries.</p></li>
     /// </ul>
     /// <p>For more information about automatically indexed fields, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub index_category: ::std::option::Option<crate::types::IndexCategory>,
+    pub index_category: ::std::option::Option<super::types::IndexCategory>,
 }
 impl FieldIndex {
     /// <p>If this field index appears in an index policy that applies only to a single log group, the ARN of that log group is displayed here.</p>
@@ -52,7 +52,7 @@
         self.last_event_time
     }
     /// <p>The type of index. Specify <code>FACET</code> for facet-based indexing or <code>FIELD_INDEX</code> for field-based indexing. This determines how the field is indexed and can be queried.</p>
-    pub fn r#type(&self) -> ::std::option::Option<&crate::types::IndexType> {
+    pub fn r#type(&self) -> ::std::option::Option<&super::types::IndexType> {
         self.r#type.as_ref()
     }
     /// <p>The category of the field index:</p>
@@ -67,14 +67,14 @@
     /// <p><code>INACTIVE</code>: Fields that CloudWatch Logs indexed before but does not index now. This happens if you remove a field from the field index policy or if CloudWatch Logs automatically selects a different field based on your queries.</p></li>
     /// </ul>
     /// <p>For more information about automatically indexed fields, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn index_category(&self) -> ::std::option::Option<&crate::types::IndexCategory> {
+    pub fn index_category(&self) -> ::std::option::Option<&super::types::IndexCategory> {
         self.index_category.as_ref()
     }
 }
 impl FieldIndex {
     /// Creates a new builder-style object to manufacture [`FieldIndex`](crate::types::FieldIndex).
-    pub fn builder() -> crate::types::builders::FieldIndexBuilder {
-        crate::types::builders::FieldIndexBuilder::default()
+    pub fn builder() -> super::types::builders::FieldIndexBuilder {
+        super::types::builders::FieldIndexBuilder::default()
     }
 }

@@ -87,8 +87,8 @@
     pub(crate) last_scan_time: ::std::option::Option<i64>,
     pub(crate) first_event_time: ::std::option::Option<i64>,
     pub(crate) last_event_time: ::std::option::Option<i64>,
-    pub(crate) r#type: ::std::option::Option<crate::types::IndexType>,
-    pub(crate) index_category: ::std::option::Option<crate::types::IndexCategory>,
+    pub(crate) r#type: ::std::option::Option<super::types::IndexType>,
+    pub(crate) index_category: ::std::option::Option<super::types::IndexCategory>,
 }
 impl FieldIndexBuilder {
     /// <p>If this field index appears in an index policy that applies only to a single log group, the ARN of that log group is displayed here.</p>
@@ -162,17 +162,17 @@
         &self.last_event_time
     }
     /// <p>The type of index. Specify <code>FACET</code> for facet-based indexing or <code>FIELD_INDEX</code> for field-based indexing. This determines how the field is indexed and can be queried.</p>
-    pub fn r#type(mut self, input: crate::types::IndexType) -> Self {
+    pub fn r#type(mut self, input: super::types::IndexType) -> Self {
         self.r#type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of index. Specify <code>FACET</code> for facet-based indexing or <code>FIELD_INDEX</code> for field-based indexing. This determines how the field is indexed and can be queried.</p>
-    pub fn set_type(mut self, input: ::std::option::Option<crate::types::IndexType>) -> Self {
+    pub fn set_type(mut self, input: ::std::option::Option<super::types::IndexType>) -> Self {
         self.r#type = input;
         self
     }
     /// <p>The type of index. Specify <code>FACET</code> for facet-based indexing or <code>FIELD_INDEX</code> for field-based indexing. This determines how the field is indexed and can be queried.</p>
-    pub fn get_type(&self) -> &::std::option::Option<crate::types::IndexType> {
+    pub fn get_type(&self) -> &::std::option::Option<super::types::IndexType> {
         &self.r#type
     }
     /// <p>The category of the field index:</p>
@@ -187,7 +187,7 @@
     /// <p><code>INACTIVE</code>: Fields that CloudWatch Logs indexed before but does not index now. This happens if you remove a field from the field index policy or if CloudWatch Logs automatically selects a different field based on your queries.</p></li>
     /// </ul>
     /// <p>For more information about automatically indexed fields, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn index_category(mut self, input: crate::types::IndexCategory) -> Self {
+    pub fn index_category(mut self, input: super::types::IndexCategory) -> Self {
         self.index_category = ::std::option::Option::Some(input);
         self
     }
@@ -203,7 +203,7 @@
     /// <p><code>INACTIVE</code>: Fields that CloudWatch Logs indexed before but does not index now. This happens if you remove a field from the field index policy or if CloudWatch Logs automatically selects a different field based on your queries.</p></li>
     /// </ul>
     /// <p>For more information about automatically indexed fields, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn set_index_category(mut self, input: ::std::option::Option<crate::types::IndexCategory>) -> Self {
+    pub fn set_index_category(mut self, input: ::std::option::Option<super::types::IndexCategory>) -> Self {
         self.index_category = input;
         self
     }
@@ -219,12 +219,12 @@
     /// <p><code>INACTIVE</code>: Fields that CloudWatch Logs indexed before but does not index now. This happens if you remove a field from the field index policy or if CloudWatch Logs automatically selects a different field based on your queries.</p></li>
     /// </ul>
     /// <p>For more information about automatically indexed fields, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatchLogs-Field-Indexing-Automatic.html">Automatically indexed fields</a>.</p>
-    pub fn get_index_category(&self) -> &::std::option::Option<crate::types::IndexCategory> {
+    pub fn get_index_category(&self) -> &::std::option::Option<super::types::IndexCategory> {
         &self.index_category
     }
     /// Consumes the builder and constructs a [`FieldIndex`](crate::types::FieldIndex).
-    pub fn build(self) -> crate::types::FieldIndex {
-        crate::types::FieldIndex {
+    pub fn build(self) -> super::types::FieldIndex {
+        super::types::FieldIndex {
             log_group_identifier: self.log_group_identifier,
             field_index_name: self.field_index_name,
             last_scan_time: self.last_scan_time,
