--- original
+++ modified
@@ -39,8 +39,8 @@
 }
 impl FilteredLogEvent {
     /// Creates a new builder-style object to manufacture [`FilteredLogEvent`](crate::types::FilteredLogEvent).
-    pub fn builder() -> crate::types::builders::FilteredLogEventBuilder {
-        crate::types::builders::FilteredLogEventBuilder::default()
+    pub fn builder() -> super::types::builders::FilteredLogEventBuilder {
+        super::types::builders::FilteredLogEventBuilder::default()
     }
 }

@@ -126,8 +126,8 @@
         &self.event_id
     }
     /// Consumes the builder and constructs a [`FilteredLogEvent`](crate::types::FilteredLogEvent).
-    pub fn build(self) -> crate::types::FilteredLogEvent {
-        crate::types::FilteredLogEvent {
+    pub fn build(self) -> super::types::FilteredLogEvent {
+        super::types::FilteredLogEvent {
             log_stream_name: self.log_stream_name,
             timestamp: self.timestamp,
             message: self.message,
