--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl Grok {
     /// Creates a new builder-style object to manufacture [`Grok`](crate::types::Grok).
-    pub fn builder() -> crate::types::builders::GrokBuilder {
-        crate::types::builders::GrokBuilder::default()
+    pub fn builder() -> super::super::types::builders::GrokBuilder {
+        super::super::types::builders::GrokBuilder::default()
     }
 }

@@ -68,8 +68,8 @@
     /// Consumes the builder and constructs a [`Grok`](crate::types::Grok).
     /// This method will fail if any of the following fields are not set:
     /// - [`r#match`](crate::types::builders::GrokBuilder::match)
-    pub fn build(self) -> ::std::result::Result<crate::types::Grok, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Grok {
+    pub fn build(self) -> ::std::result::Result<super::super::types::Grok, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::Grok {
             source: self.source,
             r#match: self.r#match.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
