--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <p>The name of this policy. Responses about log group-level field index policies don't have this field, because those policies don't have names.</p>
     pub policy_name: ::std::option::Option<::std::string::String>,
     /// <p>This field indicates whether this is an account-level index policy or an index policy that applies only to a single log group.</p>
-    pub source: ::std::option::Option<crate::types::IndexSource>,
+    pub source: ::std::option::Option<super::types::IndexSource>,
 }
 impl IndexPolicy {
     /// <p>The ARN of the log group that this index policy applies to.</p>
@@ -33,14 +33,14 @@
         self.policy_name.as_deref()
     }
     /// <p>This field indicates whether this is an account-level index policy or an index policy that applies only to a single log group.</p>
-    pub fn source(&self) -> ::std::option::Option<&crate::types::IndexSource> {
+    pub fn source(&self) -> ::std::option::Option<&super::types::IndexSource> {
         self.source.as_ref()
     }
 }
 impl IndexPolicy {
     /// Creates a new builder-style object to manufacture [`IndexPolicy`](crate::types::IndexPolicy).
-    pub fn builder() -> crate::types::builders::IndexPolicyBuilder {
-        crate::types::builders::IndexPolicyBuilder::default()
+    pub fn builder() -> super::types::builders::IndexPolicyBuilder {
+        super::types::builders::IndexPolicyBuilder::default()
     }
 }

@@ -52,7 +52,7 @@
     pub(crate) last_update_time: ::std::option::Option<i64>,
     pub(crate) policy_document: ::std::option::Option<::std::string::String>,
     pub(crate) policy_name: ::std::option::Option<::std::string::String>,
-    pub(crate) source: ::std::option::Option<crate::types::IndexSource>,
+    pub(crate) source: ::std::option::Option<super::types::IndexSource>,
 }
 impl IndexPolicyBuilder {
     /// <p>The ARN of the log group that this index policy applies to.</p>
@@ -112,22 +112,22 @@
         &self.policy_name
     }
     /// <p>This field indicates whether this is an account-level index policy or an index policy that applies only to a single log group.</p>
-    pub fn source(mut self, input: crate::types::IndexSource) -> Self {
+    pub fn source(mut self, input: super::types::IndexSource) -> Self {
         self.source = ::std::option::Option::Some(input);
         self
     }
     /// <p>This field indicates whether this is an account-level index policy or an index policy that applies only to a single log group.</p>
-    pub fn set_source(mut self, input: ::std::option::Option<crate::types::IndexSource>) -> Self {
+    pub fn set_source(mut self, input: ::std::option::Option<super::types::IndexSource>) -> Self {
         self.source = input;
         self
     }
     /// <p>This field indicates whether this is an account-level index policy or an index policy that applies only to a single log group.</p>
-    pub fn get_source(&self) -> &::std::option::Option<crate::types::IndexSource> {
+    pub fn get_source(&self) -> &::std::option::Option<super::types::IndexSource> {
         &self.source
     }
     /// Consumes the builder and constructs a [`IndexPolicy`](crate::types::IndexPolicy).
-    pub fn build(self) -> crate::types::IndexPolicy {
-        crate::types::IndexPolicy {
+    pub fn build(self) -> super::types::IndexPolicy {
+        super::types::IndexPolicy {
             log_group_identifier: self.log_group_identifier,
             last_update_time: self.last_update_time,
             policy_document: self.policy_document,
