--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl InputLogEvent {
     /// Creates a new builder-style object to manufacture [`InputLogEvent`](crate::types::InputLogEvent).
-    pub fn builder() -> crate::types::builders::InputLogEventBuilder {
-        crate::types::builders::InputLogEventBuilder::default()
+    pub fn builder() -> super::types::builders::InputLogEventBuilder {
+        super::types::builders::InputLogEventBuilder::default()
     }
 }

@@ -69,8 +69,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`timestamp`](crate::types::builders::InputLogEventBuilder::timestamp)
     /// - [`message`](crate::types::builders::InputLogEventBuilder::message)
-    pub fn build(self) -> ::std::result::Result<crate::types::InputLogEvent, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::InputLogEvent {
+    pub fn build(self) -> ::std::result::Result<super::types::InputLogEvent, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::InputLogEvent {
             timestamp: self.timestamp.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "timestamp",
