--- original
+++ modified
@@ -6,7 +6,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub enum IntegrationDetails {
     /// <p>This structure contains complete information about one integration between CloudWatch Logs and OpenSearch Service.</p>
-    OpenSearchIntegrationDetails(crate::types::OpenSearchIntegrationDetails),
+    OpenSearchIntegrationDetails(super::types::OpenSearchIntegrationDetails),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -21,7 +21,7 @@
     #[allow(irrefutable_let_patterns)]
     /// Tries to convert the enum instance into [`OpenSearchIntegrationDetails`](crate::types::IntegrationDetails::OpenSearchIntegrationDetails), extracting the inner [`OpenSearchIntegrationDetails`](crate::types::OpenSearchIntegrationDetails).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_open_search_integration_details(&self) -> ::std::result::Result<&crate::types::OpenSearchIntegrationDetails, &Self> {
+    pub fn as_open_search_integration_details(&self) -> ::std::result::Result<&super::types::OpenSearchIntegrationDetails, &Self> {
         if let IntegrationDetails::OpenSearchIntegrationDetails(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
