--- original
+++ modified
@@ -17,7 +17,7 @@
     pub flatten: bool,
     /// <p>If you set <code>flatten</code> to <code>true</code>, use <code>flattenedElement</code> to specify which element, <code>first</code> or <code>last</code>, to keep.</p>
     /// <p>You must specify this parameter if <code>flatten</code> is <code>true</code></p>
-    pub flattened_element: ::std::option::Option<crate::types::FlattenedElement>,
+    pub flattened_element: ::std::option::Option<super::types::FlattenedElement>,
 }
 impl ListToMap {
     /// <p>The key in the log event that has a list of objects that will be converted to a map.</p>
@@ -44,14 +44,14 @@
     }
     /// <p>If you set <code>flatten</code> to <code>true</code>, use <code>flattenedElement</code> to specify which element, <code>first</code> or <code>last</code>, to keep.</p>
     /// <p>You must specify this parameter if <code>flatten</code> is <code>true</code></p>
-    pub fn flattened_element(&self) -> ::std::option::Option<&crate::types::FlattenedElement> {
+    pub fn flattened_element(&self) -> ::std::option::Option<&super::types::FlattenedElement> {
         self.flattened_element.as_ref()
     }
 }
 impl ListToMap {
     /// Creates a new builder-style object to manufacture [`ListToMap`](crate::types::ListToMap).
-    pub fn builder() -> crate::types::builders::ListToMapBuilder {
-        crate::types::builders::ListToMapBuilder::default()
+    pub fn builder() -> super::types::builders::ListToMapBuilder {
+        super::types::builders::ListToMapBuilder::default()
     }
 }

@@ -64,7 +64,7 @@
     pub(crate) value_key: ::std::option::Option<::std::string::String>,
     pub(crate) target: ::std::option::Option<::std::string::String>,
     pub(crate) flatten: ::std::option::Option<bool>,
-    pub(crate) flattened_element: ::std::option::Option<crate::types::FlattenedElement>,
+    pub(crate) flattened_element: ::std::option::Option<super::types::FlattenedElement>,
 }
 impl ListToMapBuilder {
     /// <p>The key in the log event that has a list of objects that will be converted to a map.</p>
@@ -141,19 +141,19 @@
     }
     /// <p>If you set <code>flatten</code> to <code>true</code>, use <code>flattenedElement</code> to specify which element, <code>first</code> or <code>last</code>, to keep.</p>
     /// <p>You must specify this parameter if <code>flatten</code> is <code>true</code></p>
-    pub fn flattened_element(mut self, input: crate::types::FlattenedElement) -> Self {
+    pub fn flattened_element(mut self, input: super::types::FlattenedElement) -> Self {
         self.flattened_element = ::std::option::Option::Some(input);
         self
     }
     /// <p>If you set <code>flatten</code> to <code>true</code>, use <code>flattenedElement</code> to specify which element, <code>first</code> or <code>last</code>, to keep.</p>
     /// <p>You must specify this parameter if <code>flatten</code> is <code>true</code></p>
-    pub fn set_flattened_element(mut self, input: ::std::option::Option<crate::types::FlattenedElement>) -> Self {
+    pub fn set_flattened_element(mut self, input: ::std::option::Option<super::types::FlattenedElement>) -> Self {
         self.flattened_element = input;
         self
     }
     /// <p>If you set <code>flatten</code> to <code>true</code>, use <code>flattenedElement</code> to specify which element, <code>first</code> or <code>last</code>, to keep.</p>
     /// <p>You must specify this parameter if <code>flatten</code> is <code>true</code></p>
-    pub fn get_flattened_element(&self) -> &::std::option::Option<crate::types::FlattenedElement> {
+    pub fn get_flattened_element(&self) -> &::std::option::Option<super::types::FlattenedElement> {
         &self.flattened_element
     }
     /// Consumes the builder and constructs a [`ListToMap`](crate::types::ListToMap).
@@ -160,8 +160,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`source`](crate::types::builders::ListToMapBuilder::source)
     /// - [`key`](crate::types::builders::ListToMapBuilder::key)
-    pub fn build(self) -> ::std::result::Result<crate::types::ListToMap, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ListToMap {
+    pub fn build(self) -> ::std::result::Result<super::types::ListToMap, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ListToMap {
             source: self.source.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "source",
