--- original
+++ modified
@@ -17,8 +17,8 @@
 }
 impl LiveTailSessionMetadata {
     /// Creates a new builder-style object to manufacture [`LiveTailSessionMetadata`](crate::types::LiveTailSessionMetadata).
-    pub fn builder() -> crate::types::builders::LiveTailSessionMetadataBuilder {
-        crate::types::builders::LiveTailSessionMetadataBuilder::default()
+    pub fn builder() -> super::super::types::builders::LiveTailSessionMetadataBuilder {
+        super::super::types::builders::LiveTailSessionMetadataBuilder::default()
     }
 }

@@ -47,8 +47,8 @@
         &self.sampled
     }
     /// Consumes the builder and constructs a [`LiveTailSessionMetadata`](crate::types::LiveTailSessionMetadata).
-    pub fn build(self) -> crate::types::LiveTailSessionMetadata {
-        crate::types::LiveTailSessionMetadata {
+    pub fn build(self) -> super::super::types::LiveTailSessionMetadata {
+        super::super::types::LiveTailSessionMetadata {
             sampled: self.sampled.unwrap_or_default(),
         }
     }
