--- original
+++ modified
@@ -39,8 +39,8 @@
 }
 impl LiveTailSessionLogEvent {
     /// Creates a new builder-style object to manufacture [`LiveTailSessionLogEvent`](crate::types::LiveTailSessionLogEvent).
-    pub fn builder() -> crate::types::builders::LiveTailSessionLogEventBuilder {
-        crate::types::builders::LiveTailSessionLogEventBuilder::default()
+    pub fn builder() -> super::types::builders::LiveTailSessionLogEventBuilder {
+        super::types::builders::LiveTailSessionLogEventBuilder::default()
     }
 }

@@ -126,8 +126,8 @@
         &self.ingestion_time
     }
     /// Consumes the builder and constructs a [`LiveTailSessionLogEvent`](crate::types::LiveTailSessionLogEvent).
-    pub fn build(self) -> crate::types::LiveTailSessionLogEvent {
-        crate::types::LiveTailSessionLogEvent {
+    pub fn build(self) -> super::types::LiveTailSessionLogEvent {
+        super::types::LiveTailSessionLogEvent {
             log_stream_name: self.log_stream_name,
             log_group_identifier: self.log_group_identifier,
             message: self.message,
