--- original
+++ modified
@@ -21,9 +21,9 @@
     /// <p>The Amazon Resource Name (ARN) of the KMS key to use when encrypting log data.</p>
     pub kms_key_id: ::std::option::Option<::std::string::String>,
     /// <p>Displays whether this log group has a protection policy, or whether it had one in the past. For more information, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutDataProtectionPolicy.html">PutDataProtectionPolicy</a>.</p>
-    pub data_protection_status: ::std::option::Option<crate::types::DataProtectionStatus>,
+    pub data_protection_status: ::std::option::Option<super::super::types::DataProtectionStatus>,
     /// <p>Displays all the properties that this log group has inherited from account-level settings.</p>
-    pub inherited_properties: ::std::option::Option<::std::vec::Vec<crate::types::InheritedProperty>>,
+    pub inherited_properties: ::std::option::Option<::std::vec::Vec<super::super::types::InheritedProperty>>,
     /// <p>This specifies the log group class for this log group. There are three classes:</p>
     /// <ul>
     /// <li>
@@ -34,7 +34,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by the Standard and Infrequent Access classes, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub log_group_class: ::std::option::Option<crate::types::LogGroupClass>,
+    pub log_group_class: ::std::option::Option<super::super::types::LogGroupClass>,
     /// <p>The Amazon Resource Name (ARN) of the log group. This version of the ARN doesn't include a trailing <code>:*</code> after the log group name.</p>
     /// <p>Use this version to refer to the ARN in the following situations:</p>
     /// <ul>
@@ -83,13 +83,13 @@
         self.kms_key_id.as_deref()
     }
     /// <p>Displays whether this log group has a protection policy, or whether it had one in the past. For more information, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutDataProtectionPolicy.html">PutDataProtectionPolicy</a>.</p>
-    pub fn data_protection_status(&self) -> ::std::option::Option<&crate::types::DataProtectionStatus> {
+    pub fn data_protection_status(&self) -> ::std::option::Option<&super::super::types::DataProtectionStatus> {
         self.data_protection_status.as_ref()
     }
     /// <p>Displays all the properties that this log group has inherited from account-level settings.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.inherited_properties.is_none()`.
-    pub fn inherited_properties(&self) -> &[crate::types::InheritedProperty] {
+    pub fn inherited_properties(&self) -> &[super::super::types::InheritedProperty] {
         self.inherited_properties.as_deref().unwrap_or_default()
     }
     /// <p>This specifies the log group class for this log group. There are three classes:</p>
@@ -102,7 +102,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by the Standard and Infrequent Access classes, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn log_group_class(&self) -> ::std::option::Option<&crate::types::LogGroupClass> {
+    pub fn log_group_class(&self) -> ::std::option::Option<&super::super::types::LogGroupClass> {
         self.log_group_class.as_ref()
     }
     /// <p>The Amazon Resource Name (ARN) of the log group. This version of the ARN doesn't include a trailing <code>:*</code> after the log group name.</p>
@@ -129,8 +129,8 @@
 }
 impl LogGroup {
     /// Creates a new builder-style object to manufacture [`LogGroup`](crate::types::LogGroup).
-    pub fn builder() -> crate::types::builders::LogGroupBuilder {
-        crate::types::builders::LogGroupBuilder::default()
+    pub fn builder() -> super::super::types::builders::LogGroupBuilder {
+        super::super::types::builders::LogGroupBuilder::default()
     }
 }

@@ -145,9 +145,9 @@
     pub(crate) arn: ::std::option::Option<::std::string::String>,
     pub(crate) stored_bytes: ::std::option::Option<i64>,
     pub(crate) kms_key_id: ::std::option::Option<::std::string::String>,
-    pub(crate) data_protection_status: ::std::option::Option<crate::types::DataProtectionStatus>,
-    pub(crate) inherited_properties: ::std::option::Option<::std::vec::Vec<crate::types::InheritedProperty>>,
-    pub(crate) log_group_class: ::std::option::Option<crate::types::LogGroupClass>,
+    pub(crate) data_protection_status: ::std::option::Option<super::super::types::DataProtectionStatus>,
+    pub(crate) inherited_properties: ::std::option::Option<::std::vec::Vec<super::super::types::InheritedProperty>>,
+    pub(crate) log_group_class: ::std::option::Option<super::super::types::LogGroupClass>,
     pub(crate) log_group_arn: ::std::option::Option<::std::string::String>,
     pub(crate) deletion_protection_enabled: ::std::option::Option<bool>,
     pub(crate) bearer_token_authentication_enabled: ::std::option::Option<bool>,
@@ -258,17 +258,17 @@
         &self.kms_key_id
     }
     /// <p>Displays whether this log group has a protection policy, or whether it had one in the past. For more information, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutDataProtectionPolicy.html">PutDataProtectionPolicy</a>.</p>
-    pub fn data_protection_status(mut self, input: crate::types::DataProtectionStatus) -> Self {
+    pub fn data_protection_status(mut self, input: super::super::types::DataProtectionStatus) -> Self {
         self.data_protection_status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Displays whether this log group has a protection policy, or whether it had one in the past. For more information, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutDataProtectionPolicy.html">PutDataProtectionPolicy</a>.</p>
-    pub fn set_data_protection_status(mut self, input: ::std::option::Option<crate::types::DataProtectionStatus>) -> Self {
+    pub fn set_data_protection_status(mut self, input: ::std::option::Option<super::super::types::DataProtectionStatus>) -> Self {
         self.data_protection_status = input;
         self
     }
     /// <p>Displays whether this log group has a protection policy, or whether it had one in the past. For more information, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutDataProtectionPolicy.html">PutDataProtectionPolicy</a>.</p>
-    pub fn get_data_protection_status(&self) -> &::std::option::Option<crate::types::DataProtectionStatus> {
+    pub fn get_data_protection_status(&self) -> &::std::option::Option<super::super::types::DataProtectionStatus> {
         &self.data_protection_status
     }
     /// Appends an item to `inherited_properties`.
@@ -276,7 +276,7 @@
     /// To override the contents of this collection use [`set_inherited_properties`](Self::set_inherited_properties).
     ///
     /// <p>Displays all the properties that this log group has inherited from account-level settings.</p>
-    pub fn inherited_properties(mut self, input: crate::types::InheritedProperty) -> Self {
+    pub fn inherited_properties(mut self, input: super::super::types::InheritedProperty) -> Self {
         let mut v = self.inherited_properties.unwrap_or_default();
         v.push(input);
         self.inherited_properties = ::std::option::Option::Some(v);
@@ -283,12 +283,12 @@
         self
     }
     /// <p>Displays all the properties that this log group has inherited from account-level settings.</p>
-    pub fn set_inherited_properties(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::InheritedProperty>>) -> Self {
+    pub fn set_inherited_properties(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::InheritedProperty>>) -> Self {
         self.inherited_properties = input;
         self
     }
     /// <p>Displays all the properties that this log group has inherited from account-level settings.</p>
-    pub fn get_inherited_properties(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::InheritedProperty>> {
+    pub fn get_inherited_properties(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::InheritedProperty>> {
         &self.inherited_properties
     }
     /// <p>This specifies the log group class for this log group. There are three classes:</p>
@@ -301,7 +301,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by the Standard and Infrequent Access classes, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn log_group_class(mut self, input: crate::types::LogGroupClass) -> Self {
+    pub fn log_group_class(mut self, input: super::super::types::LogGroupClass) -> Self {
         self.log_group_class = ::std::option::Option::Some(input);
         self
     }
@@ -315,7 +315,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by the Standard and Infrequent Access classes, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn set_log_group_class(mut self, input: ::std::option::Option<crate::types::LogGroupClass>) -> Self {
+    pub fn set_log_group_class(mut self, input: ::std::option::Option<super::super::types::LogGroupClass>) -> Self {
         self.log_group_class = input;
         self
     }
@@ -329,7 +329,7 @@
     /// <p>Use the <code>Delivery</code> log class only for delivering Lambda logs to store in Amazon S3 or Amazon Data Firehose. Log events in log groups in the Delivery class are kept in CloudWatch Logs for only one day. This log class doesn't offer rich CloudWatch Logs capabilities such as CloudWatch Logs Insights queries.</p></li>
     /// </ul>
     /// <p>For details about the features supported by the Standard and Infrequent Access classes, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CloudWatch_Logs_Log_Classes.html">Log classes</a></p>
-    pub fn get_log_group_class(&self) -> &::std::option::Option<crate::types::LogGroupClass> {
+    pub fn get_log_group_class(&self) -> &::std::option::Option<super::super::types::LogGroupClass> {
         &self.log_group_class
     }
     /// <p>The Amazon Resource Name (ARN) of the log group. This version of the ARN doesn't include a trailing <code>:*</code> after the log group name.</p>
@@ -402,8 +402,8 @@
         &self.bearer_token_authentication_enabled
     }
     /// Consumes the builder and constructs a [`LogGroup`](crate::types::LogGroup).
-    pub fn build(self) -> crate::types::LogGroup {
-        crate::types::LogGroup {
+    pub fn build(self) -> super::super::types::LogGroup {
+        super::super::types::LogGroup {
             log_group_name: self.log_group_name,
             creation_time: self.creation_time,
             retention_in_days: self.retention_in_days,
