--- original
+++ modified
@@ -41,8 +41,8 @@
 }
 impl LookupTableConfiguration {
     /// Creates a new builder-style object to manufacture [`LookupTableConfiguration`](crate::types::LookupTableConfiguration).
-    pub fn builder() -> crate::types::builders::LookupTableConfigurationBuilder {
-        crate::types::builders::LookupTableConfigurationBuilder::default()
+    pub fn builder() -> super::types::builders::LookupTableConfigurationBuilder {
+        super::types::builders::LookupTableConfigurationBuilder::default()
     }
 }

@@ -139,8 +139,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`table_name`](crate::types::builders::LookupTableConfigurationBuilder::table_name)
     /// - [`role_arn`](crate::types::builders::LookupTableConfigurationBuilder::role_arn)
-    pub fn build(self) -> ::std::result::Result<crate::types::LookupTableConfiguration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::LookupTableConfiguration {
+    pub fn build(self) -> ::std::result::Result<super::types::LookupTableConfiguration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::LookupTableConfiguration {
             table_name: self.table_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "table_name",
