--- original
+++ modified
@@ -9,7 +9,7 @@
     /// <p>A symbolic description of how CloudWatch Logs should interpret the data in each log event. For example, a log event can contain timestamps, IP addresses, strings, and so on. You use the filter pattern to specify what to look for in the log event message.</p>
     pub filter_pattern: ::std::option::Option<::std::string::String>,
     /// <p>The metric transformations.</p>
-    pub metric_transformations: ::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>>,
+    pub metric_transformations: ::std::option::Option<::std::vec::Vec<super::super::types::MetricTransformation>>,
     /// <p>The creation time of the metric filter, expressed as the number of milliseconds after <code>Jan 1, 1970 00:00:00 UTC</code>.</p>
     pub creation_time: ::std::option::Option<i64>,
     /// <p>The name of the log group.</p>
@@ -34,7 +34,7 @@
     /// <p>The metric transformations.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.metric_transformations.is_none()`.
-    pub fn metric_transformations(&self) -> &[crate::types::MetricTransformation] {
+    pub fn metric_transformations(&self) -> &[super::super::types::MetricTransformation] {
         self.metric_transformations.as_deref().unwrap_or_default()
     }
     /// <p>The creation time of the metric filter, expressed as the number of milliseconds after <code>Jan 1, 1970 00:00:00 UTC</code>.</p>
@@ -63,8 +63,8 @@
 }
 impl MetricFilter {
     /// Creates a new builder-style object to manufacture [`MetricFilter`](crate::types::MetricFilter).
-    pub fn builder() -> crate::types::builders::MetricFilterBuilder {
-        crate::types::builders::MetricFilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::MetricFilterBuilder {
+        super::super::types::builders::MetricFilterBuilder::default()
     }
 }

@@ -74,7 +74,7 @@
 pub struct MetricFilterBuilder {
     pub(crate) filter_name: ::std::option::Option<::std::string::String>,
     pub(crate) filter_pattern: ::std::option::Option<::std::string::String>,
-    pub(crate) metric_transformations: ::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>>,
+    pub(crate) metric_transformations: ::std::option::Option<::std::vec::Vec<super::super::types::MetricTransformation>>,
     pub(crate) creation_time: ::std::option::Option<i64>,
     pub(crate) log_group_name: ::std::option::Option<::std::string::String>,
     pub(crate) apply_on_transformed_logs: ::std::option::Option<bool>,
@@ -115,7 +115,7 @@
     /// To override the contents of this collection use [`set_metric_transformations`](Self::set_metric_transformations).
     ///
     /// <p>The metric transformations.</p>
-    pub fn metric_transformations(mut self, input: crate::types::MetricTransformation) -> Self {
+    pub fn metric_transformations(mut self, input: super::super::types::MetricTransformation) -> Self {
         let mut v = self.metric_transformations.unwrap_or_default();
         v.push(input);
         self.metric_transformations = ::std::option::Option::Some(v);
@@ -122,12 +122,12 @@
         self
     }
     /// <p>The metric transformations.</p>
-    pub fn set_metric_transformations(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>>) -> Self {
+    pub fn set_metric_transformations(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::MetricTransformation>>) -> Self {
         self.metric_transformations = input;
         self
     }
     /// <p>The metric transformations.</p>
-    pub fn get_metric_transformations(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::MetricTransformation>> {
+    pub fn get_metric_transformations(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::MetricTransformation>> {
         &self.metric_transformations
     }
     /// <p>The creation time of the metric filter, expressed as the number of milliseconds after <code>Jan 1, 1970 00:00:00 UTC</code>.</p>
@@ -210,8 +210,8 @@
         &self.emit_system_field_dimensions
     }
     /// Consumes the builder and constructs a [`MetricFilter`](crate::types::MetricFilter).
-    pub fn build(self) -> crate::types::MetricFilter {
-        crate::types::MetricFilter {
+    pub fn build(self) -> super::super::types::MetricFilter {
+        super::super::types::MetricFilter {
             filter_name: self.filter_name,
             filter_pattern: self.filter_pattern,
             metric_transformations: self.metric_transformations,
