--- original
+++ modified
@@ -11,7 +11,7 @@
     /// <p>The ID of the application.</p>
     pub application_id: ::std::option::Option<::std::string::String>,
     /// <p>This structure contains information about the status of this OpenSearch Service resource.</p>
-    pub status: ::std::option::Option<crate::types::OpenSearchResourceStatus>,
+    pub status: ::std::option::Option<super::types::OpenSearchResourceStatus>,
 }
 impl OpenSearchApplication {
     /// <p>The endpoint of the application.</p>
@@ -27,14 +27,14 @@
         self.application_id.as_deref()
     }
     /// <p>This structure contains information about the status of this OpenSearch Service resource.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::OpenSearchResourceStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::OpenSearchResourceStatus> {
         self.status.as_ref()
     }
 }
 impl OpenSearchApplication {
     /// Creates a new builder-style object to manufacture [`OpenSearchApplication`](crate::types::OpenSearchApplication).
-    pub fn builder() -> crate::types::builders::OpenSearchApplicationBuilder {
-        crate::types::builders::OpenSearchApplicationBuilder::default()
+    pub fn builder() -> super::types::builders::OpenSearchApplicationBuilder {
+        super::types::builders::OpenSearchApplicationBuilder::default()
     }
 }

@@ -45,7 +45,7 @@
     pub(crate) application_endpoint: ::std::option::Option<::std::string::String>,
     pub(crate) application_arn: ::std::option::Option<::std::string::String>,
     pub(crate) application_id: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::OpenSearchResourceStatus>,
+    pub(crate) status: ::std::option::Option<super::types::OpenSearchResourceStatus>,
 }
 impl OpenSearchApplicationBuilder {
     /// <p>The endpoint of the application.</p>
@@ -91,22 +91,22 @@
         &self.application_id
     }
     /// <p>This structure contains information about the status of this OpenSearch Service resource.</p>
-    pub fn status(mut self, input: crate::types::OpenSearchResourceStatus) -> Self {
+    pub fn status(mut self, input: super::types::OpenSearchResourceStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>This structure contains information about the status of this OpenSearch Service resource.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::OpenSearchResourceStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::OpenSearchResourceStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>This structure contains information about the status of this OpenSearch Service resource.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::OpenSearchResourceStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::OpenSearchResourceStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`OpenSearchApplication`](crate::types::OpenSearchApplication).
-    pub fn build(self) -> crate::types::OpenSearchApplication {
-        crate::types::OpenSearchApplication {
+    pub fn build(self) -> super::types::OpenSearchApplication {
+        super::types::OpenSearchApplication {
             application_endpoint: self.application_endpoint,
             application_arn: self.application_arn,
             application_id: self.application_id,
