--- original
+++ modified
@@ -54,8 +54,8 @@
 }
 impl ParseKeyValue {
     /// Creates a new builder-style object to manufacture [`ParseKeyValue`](crate::types::ParseKeyValue).
-    pub fn builder() -> crate::types::builders::ParseKeyValueBuilder {
-        crate::types::builders::ParseKeyValueBuilder::default()
+    pub fn builder() -> super::types::builders::ParseKeyValueBuilder {
+        super::types::builders::ParseKeyValueBuilder::default()
     }
 }

@@ -174,8 +174,8 @@
         &self.overwrite_if_exists
     }
     /// Consumes the builder and constructs a [`ParseKeyValue`](crate::types::ParseKeyValue).
-    pub fn build(self) -> crate::types::ParseKeyValue {
-        crate::types::ParseKeyValue {
+    pub fn build(self) -> super::types::ParseKeyValue {
+        super::types::ParseKeyValue {
             source: self.source,
             destination: self.destination,
             field_delimiter: self.field_delimiter,
