--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl ParseJson {
     /// Creates a new builder-style object to manufacture [`ParseJson`](crate::types::ParseJson).
-    pub fn builder() -> crate::types::builders::ParseJsonBuilder {
-        crate::types::builders::ParseJsonBuilder::default()
+    pub fn builder() -> super::types::builders::ParseJsonBuilder {
+        super::types::builders::ParseJsonBuilder::default()
     }
 }

@@ -65,8 +65,8 @@
         &self.destination
     }
     /// Consumes the builder and constructs a [`ParseJson`](crate::types::ParseJson).
-    pub fn build(self) -> crate::types::ParseJson {
-        crate::types::ParseJson {
+    pub fn build(self) -> super::types::ParseJson {
+        super::types::ParseJson {
             source: self.source,
             destination: self.destination,
         }
