--- original
+++ modified
@@ -8,9 +8,9 @@
     /// <p>The path to the field in the log event that you want to parse. If you omit this value, the whole log message is parsed.</p>
     pub source: ::std::option::Option<::std::string::String>,
     /// <p>Specify the service or process that produces the log events that will be converted with this processor.</p>
-    pub event_source: crate::types::EventSource,
+    pub event_source: super::types::EventSource,
     /// <p>Specify which version of the OCSF schema to use for the transformed log events.</p>
-    pub ocsf_version: crate::types::OcsfVersion,
+    pub ocsf_version: super::types::OcsfVersion,
     /// <p>The version of the OCSF mapping to use for parsing log data.</p>
     pub mapping_version: ::std::option::Option<::std::string::String>,
 }
@@ -20,11 +20,11 @@
         self.source.as_deref()
     }
     /// <p>Specify the service or process that produces the log events that will be converted with this processor.</p>
-    pub fn event_source(&self) -> &crate::types::EventSource {
+    pub fn event_source(&self) -> &super::types::EventSource {
         &self.event_source
     }
     /// <p>Specify which version of the OCSF schema to use for the transformed log events.</p>
-    pub fn ocsf_version(&self) -> &crate::types::OcsfVersion {
+    pub fn ocsf_version(&self) -> &super::types::OcsfVersion {
         &self.ocsf_version
     }
     /// <p>The version of the OCSF mapping to use for parsing log data.</p>
@@ -34,8 +34,8 @@
 }
 impl ParseToOcsf {
     /// Creates a new builder-style object to manufacture [`ParseToOcsf`](crate::types::ParseToOcsf).
-    pub fn builder() -> crate::types::builders::ParseToOcsfBuilder {
-        crate::types::builders::ParseToOcsfBuilder::default()
+    pub fn builder() -> super::types::builders::ParseToOcsfBuilder {
+        super::types::builders::ParseToOcsfBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
 #[non_exhaustive]
 pub struct ParseToOcsfBuilder {
     pub(crate) source: ::std::option::Option<::std::string::String>,
-    pub(crate) event_source: ::std::option::Option<crate::types::EventSource>,
-    pub(crate) ocsf_version: ::std::option::Option<crate::types::OcsfVersion>,
+    pub(crate) event_source: ::std::option::Option<super::types::EventSource>,
+    pub(crate) ocsf_version: ::std::option::Option<super::types::OcsfVersion>,
     pub(crate) mapping_version: ::std::option::Option<::std::string::String>,
 }
 impl ParseToOcsfBuilder {
@@ -65,32 +65,32 @@
     }
     /// <p>Specify the service or process that produces the log events that will be converted with this processor.</p>
     /// This field is required.
-    pub fn event_source(mut self, input: crate::types::EventSource) -> Self {
+    pub fn event_source(mut self, input: super::types::EventSource) -> Self {
         self.event_source = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specify the service or process that produces the log events that will be converted with this processor.</p>
-    pub fn set_event_source(mut self, input: ::std::option::Option<crate::types::EventSource>) -> Self {
+    pub fn set_event_source(mut self, input: ::std::option::Option<super::types::EventSource>) -> Self {
         self.event_source = input;
         self
     }
     /// <p>Specify the service or process that produces the log events that will be converted with this processor.</p>
-    pub fn get_event_source(&self) -> &::std::option::Option<crate::types::EventSource> {
+    pub fn get_event_source(&self) -> &::std::option::Option<super::types::EventSource> {
         &self.event_source
     }
     /// <p>Specify which version of the OCSF schema to use for the transformed log events.</p>
     /// This field is required.
-    pub fn ocsf_version(mut self, input: crate::types::OcsfVersion) -> Self {
+    pub fn ocsf_version(mut self, input: super::types::OcsfVersion) -> Self {
         self.ocsf_version = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specify which version of the OCSF schema to use for the transformed log events.</p>
-    pub fn set_ocsf_version(mut self, input: ::std::option::Option<crate::types::OcsfVersion>) -> Self {
+    pub fn set_ocsf_version(mut self, input: ::std::option::Option<super::types::OcsfVersion>) -> Self {
         self.ocsf_version = input;
         self
     }
     /// <p>Specify which version of the OCSF schema to use for the transformed log events.</p>
-    pub fn get_ocsf_version(&self) -> &::std::option::Option<crate::types::OcsfVersion> {
+    pub fn get_ocsf_version(&self) -> &::std::option::Option<super::types::OcsfVersion> {
         &self.ocsf_version
     }
     /// <p>The version of the OCSF mapping to use for parsing log data.</p>
@@ -111,8 +111,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`event_source`](crate::types::builders::ParseToOcsfBuilder::event_source)
     /// - [`ocsf_version`](crate::types::builders::ParseToOcsfBuilder::ocsf_version)
-    pub fn build(self) -> ::std::result::Result<crate::types::ParseToOcsf, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ParseToOcsf {
+    pub fn build(self) -> ::std::result::Result<super::types::ParseToOcsf, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ParseToOcsf {
             source: self.source,
             event_source: self.event_source.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
