--- original
+++ modified
@@ -17,8 +17,8 @@
 }
 impl ParseRoute53 {
     /// Creates a new builder-style object to manufacture [`ParseRoute53`](crate::types::ParseRoute53).
-    pub fn builder() -> crate::types::builders::ParseRoute53Builder {
-        crate::types::builders::ParseRoute53Builder::default()
+    pub fn builder() -> super::super::types::builders::ParseRoute53Builder {
+        super::super::types::builders::ParseRoute53Builder::default()
     }
 }

@@ -44,7 +44,7 @@
         &self.source
     }
     /// Consumes the builder and constructs a [`ParseRoute53`](crate::types::ParseRoute53).
-    pub fn build(self) -> crate::types::ParseRoute53 {
-        crate::types::ParseRoute53 { source: self.source }
+    pub fn build(self) -> super::super::types::ParseRoute53 {
+        super::super::types::ParseRoute53 { source: self.source }
     }
 }
