--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl PatternToken {
     /// Creates a new builder-style object to manufacture [`PatternToken`](crate::types::PatternToken).
-    pub fn builder() -> crate::types::builders::PatternTokenBuilder {
-        crate::types::builders::PatternTokenBuilder::default()
+    pub fn builder() -> super::super::types::builders::PatternTokenBuilder {
+        super::super::types::builders::PatternTokenBuilder::default()
     }
 }

@@ -138,8 +138,8 @@
         &self.inferred_token_name
     }
     /// Consumes the builder and constructs a [`PatternToken`](crate::types::PatternToken).
-    pub fn build(self) -> crate::types::PatternToken {
-        crate::types::PatternToken {
+    pub fn build(self) -> super::super::types::PatternToken {
+        super::super::types::PatternToken {
             dynamic_token_position: self.dynamic_token_position.unwrap_or_default(),
             is_dynamic: self.is_dynamic,
             token_string: self.token_string,
