--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct QueryDefinition {
     /// <p>The query language used for this query. For more information about the query languages that CloudWatch Logs supports, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_AnalyzeLogData_Languages.html">Supported query languages</a>.</p>
-    pub query_language: ::std::option::Option<crate::types::QueryLanguage>,
+    pub query_language: ::std::option::Option<super::super::types::QueryLanguage>,
     /// <p>The unique ID of the query definition.</p>
     pub query_definition_id: ::std::option::Option<::std::string::String>,
     /// <p>The name of the query definition.</p>
@@ -17,11 +17,11 @@
     /// <p>If this query definition contains a list of log groups that it is limited to, that list appears here.</p>
     pub log_group_names: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     /// <p>If this query definition contains a list of query parameters that define placeholder variables for the query string, that list appears here.</p>
-    pub parameters: ::std::option::Option<::std::vec::Vec<crate::types::QueryParameter>>,
+    pub parameters: ::std::option::Option<::std::vec::Vec<super::super::types::QueryParameter>>,
 }
 impl QueryDefinition {
     /// <p>The query language used for this query. For more information about the query languages that CloudWatch Logs supports, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_AnalyzeLogData_Languages.html">Supported query languages</a>.</p>
-    pub fn query_language(&self) -> ::std::option::Option<&crate::types::QueryLanguage> {
+    pub fn query_language(&self) -> ::std::option::Option<&super::super::types::QueryLanguage> {
         self.query_language.as_ref()
     }
     /// <p>The unique ID of the query definition.</p>
@@ -49,14 +49,14 @@
     /// <p>If this query definition contains a list of query parameters that define placeholder variables for the query string, that list appears here.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.parameters.is_none()`.
-    pub fn parameters(&self) -> &[crate::types::QueryParameter] {
+    pub fn parameters(&self) -> &[super::super::types::QueryParameter] {
         self.parameters.as_deref().unwrap_or_default()
     }
 }
 impl QueryDefinition {
     /// Creates a new builder-style object to manufacture [`QueryDefinition`](crate::types::QueryDefinition).
-    pub fn builder() -> crate::types::builders::QueryDefinitionBuilder {
-        crate::types::builders::QueryDefinitionBuilder::default()
+    pub fn builder() -> super::super::types::builders::QueryDefinitionBuilder {
+        super::super::types::builders::QueryDefinitionBuilder::default()
     }
 }

@@ -64,27 +64,27 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct QueryDefinitionBuilder {
-    pub(crate) query_language: ::std::option::Option<crate::types::QueryLanguage>,
+    pub(crate) query_language: ::std::option::Option<super::super::types::QueryLanguage>,
     pub(crate) query_definition_id: ::std::option::Option<::std::string::String>,
     pub(crate) name: ::std::option::Option<::std::string::String>,
     pub(crate) query_string: ::std::option::Option<::std::string::String>,
     pub(crate) last_modified: ::std::option::Option<i64>,
     pub(crate) log_group_names: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
-    pub(crate) parameters: ::std::option::Option<::std::vec::Vec<crate::types::QueryParameter>>,
+    pub(crate) parameters: ::std::option::Option<::std::vec::Vec<super::super::types::QueryParameter>>,
 }
 impl QueryDefinitionBuilder {
     /// <p>The query language used for this query. For more information about the query languages that CloudWatch Logs supports, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_AnalyzeLogData_Languages.html">Supported query languages</a>.</p>
-    pub fn query_language(mut self, input: crate::types::QueryLanguage) -> Self {
+    pub fn query_language(mut self, input: super::super::types::QueryLanguage) -> Self {
         self.query_language = ::std::option::Option::Some(input);
         self
     }
     /// <p>The query language used for this query. For more information about the query languages that CloudWatch Logs supports, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_AnalyzeLogData_Languages.html">Supported query languages</a>.</p>
-    pub fn set_query_language(mut self, input: ::std::option::Option<crate::types::QueryLanguage>) -> Self {
+    pub fn set_query_language(mut self, input: ::std::option::Option<super::super::types::QueryLanguage>) -> Self {
         self.query_language = input;
         self
     }
     /// <p>The query language used for this query. For more information about the query languages that CloudWatch Logs supports, see <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_AnalyzeLogData_Languages.html">Supported query languages</a>.</p>
-    pub fn get_query_language(&self) -> &::std::option::Option<crate::types::QueryLanguage> {
+    pub fn get_query_language(&self) -> &::std::option::Option<super::super::types::QueryLanguage> {
         &self.query_language
     }
     /// <p>The unique ID of the query definition.</p>
@@ -168,7 +168,7 @@
     /// To override the contents of this collection use [`set_parameters`](Self::set_parameters).
     ///
     /// <p>If this query definition contains a list of query parameters that define placeholder variables for the query string, that list appears here.</p>
-    pub fn parameters(mut self, input: crate::types::QueryParameter) -> Self {
+    pub fn parameters(mut self, input: super::super::types::QueryParameter) -> Self {
         let mut v = self.parameters.unwrap_or_default();
         v.push(input);
         self.parameters = ::std::option::Option::Some(v);
@@ -175,17 +175,17 @@
         self
     }
     /// <p>If this query definition contains a list of query parameters that define placeholder variables for the query string, that list appears here.</p>
-    pub fn set_parameters(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::QueryParameter>>) -> Self {
+    pub fn set_parameters(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::QueryParameter>>) -> Self {
         self.parameters = input;
         self
     }
     /// <p>If this query definition contains a list of query parameters that define placeholder variables for the query string, that list appears here.</p>
-    pub fn get_parameters(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::QueryParameter>> {
+    pub fn get_parameters(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::QueryParameter>> {
         &self.parameters
     }
     /// Consumes the builder and constructs a [`QueryDefinition`](crate::types::QueryDefinition).
-    pub fn build(self) -> crate::types::QueryDefinition {
-        crate::types::QueryDefinition {
+    pub fn build(self) -> super::super::types::QueryDefinition {
+        super::super::types::QueryDefinition {
             query_language: self.query_language,
             query_definition_id: self.query_definition_id,
             name: self.name,
