--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl QueryParameter {
     /// Creates a new builder-style object to manufacture [`QueryParameter`](crate::types::QueryParameter).
-    pub fn builder() -> crate::types::builders::QueryParameterBuilder {
-        crate::types::builders::QueryParameterBuilder::default()
+    pub fn builder() -> super::super::types::builders::QueryParameterBuilder {
+        super::super::types::builders::QueryParameterBuilder::default()
     }
 }

@@ -88,8 +88,8 @@
     /// Consumes the builder and constructs a [`QueryParameter`](crate::types::QueryParameter).
     /// This method will fail if any of the following fields are not set:
     /// - [`name`](crate::types::builders::QueryParameterBuilder::name)
-    pub fn build(self) -> ::std::result::Result<crate::types::QueryParameter, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::QueryParameter {
+    pub fn build(self) -> ::std::result::Result<super::super::types::QueryParameter, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::QueryParameter {
             name: self.name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "name",
