--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct QueryCompileError {
     /// <p>Reserved.</p>
-    pub location: ::std::option::Option<crate::types::QueryCompileErrorLocation>,
+    pub location: ::std::option::Option<super::super::types::QueryCompileErrorLocation>,
     /// <p>Reserved.</p>
     pub message: ::std::option::Option<::std::string::String>,
 }
 impl QueryCompileError {
     /// <p>Reserved.</p>
-    pub fn location(&self) -> ::std::option::Option<&crate::types::QueryCompileErrorLocation> {
+    pub fn location(&self) -> ::std::option::Option<&super::super::types::QueryCompileErrorLocation> {
         self.location.as_ref()
     }
     /// <p>Reserved.</p>
@@ -21,8 +21,8 @@
 }
 impl QueryCompileError {
     /// Creates a new builder-style object to manufacture [`QueryCompileError`](crate::types::QueryCompileError).
-    pub fn builder() -> crate::types::builders::QueryCompileErrorBuilder {
-        crate::types::builders::QueryCompileErrorBuilder::default()
+    pub fn builder() -> super::super::types::builders::QueryCompileErrorBuilder {
+        super::super::types::builders::QueryCompileErrorBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct QueryCompileErrorBuilder {
-    pub(crate) location: ::std::option::Option<crate::types::QueryCompileErrorLocation>,
+    pub(crate) location: ::std::option::Option<super::super::types::QueryCompileErrorLocation>,
     pub(crate) message: ::std::option::Option<::std::string::String>,
 }
 impl QueryCompileErrorBuilder {
     /// <p>Reserved.</p>
-    pub fn location(mut self, input: crate::types::QueryCompileErrorLocation) -> Self {
+    pub fn location(mut self, input: super::super::types::QueryCompileErrorLocation) -> Self {
         self.location = ::std::option::Option::Some(input);
         self
     }
     /// <p>Reserved.</p>
-    pub fn set_location(mut self, input: ::std::option::Option<crate::types::QueryCompileErrorLocation>) -> Self {
+    pub fn set_location(mut self, input: ::std::option::Option<super::super::types::QueryCompileErrorLocation>) -> Self {
         self.location = input;
         self
     }
     /// <p>Reserved.</p>
-    pub fn get_location(&self) -> &::std::option::Option<crate::types::QueryCompileErrorLocation> {
+    pub fn get_location(&self) -> &::std::option::Option<super::super::types::QueryCompileErrorLocation> {
         &self.location
     }
     /// <p>Reserved.</p>
@@ -63,8 +63,8 @@
         &self.message
     }
     /// Consumes the builder and constructs a [`QueryCompileError`](crate::types::QueryCompileError).
-    pub fn build(self) -> crate::types::QueryCompileError {
-        crate::types::QueryCompileError {
+    pub fn build(self) -> super::super::types::QueryCompileError {
+        super::super::types::QueryCompileError {
             location: self.location,
             message: self.message,
         }
