--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl RecordField {
     /// Creates a new builder-style object to manufacture [`RecordField`](crate::types::RecordField).
-    pub fn builder() -> crate::types::builders::RecordFieldBuilder {
-        crate::types::builders::RecordFieldBuilder::default()
+    pub fn builder() -> super::super::types::builders::RecordFieldBuilder {
+        super::super::types::builders::RecordFieldBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.mandatory
     }
     /// Consumes the builder and constructs a [`RecordField`](crate::types::RecordField).
-    pub fn build(self) -> crate::types::RecordField {
-        crate::types::RecordField {
+    pub fn build(self) -> super::super::types::RecordField {
+        super::super::types::RecordField {
             name: self.name,
             mandatory: self.mandatory,
         }
