--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl RejectedLogEventsInfo {
     /// Creates a new builder-style object to manufacture [`RejectedLogEventsInfo`](crate::types::RejectedLogEventsInfo).
-    pub fn builder() -> crate::types::builders::RejectedLogEventsInfoBuilder {
-        crate::types::builders::RejectedLogEventsInfoBuilder::default()
+    pub fn builder() -> super::types::builders::RejectedLogEventsInfoBuilder {
+        super::types::builders::RejectedLogEventsInfoBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.expired_log_event_end_index
     }
     /// Consumes the builder and constructs a [`RejectedLogEventsInfo`](crate::types::RejectedLogEventsInfo).
-    pub fn build(self) -> crate::types::RejectedLogEventsInfo {
-        crate::types::RejectedLogEventsInfo {
+    pub fn build(self) -> super::types::RejectedLogEventsInfo {
+        super::types::RejectedLogEventsInfo {
             too_new_log_event_start_index: self.too_new_log_event_start_index,
             too_old_log_event_end_index: self.too_old_log_event_end_index,
             expired_log_event_end_index: self.expired_log_event_end_index,
