--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl S3TablesIntegration {
     /// Creates a new builder-style object to manufacture [`S3TablesIntegration`](crate::types::S3TablesIntegration).
-    pub fn builder() -> crate::types::builders::S3TablesIntegrationBuilder {
-        crate::types::builders::S3TablesIntegrationBuilder::default()
+    pub fn builder() -> super::types::builders::S3TablesIntegrationBuilder {
+        super::types::builders::S3TablesIntegrationBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.datasource_type
     }
     /// Consumes the builder and constructs a [`S3TablesIntegration`](crate::types::S3TablesIntegration).
-    pub fn build(self) -> crate::types::S3TablesIntegration {
-        crate::types::S3TablesIntegration {
+    pub fn build(self) -> super::types::S3TablesIntegration {
+        super::types::S3TablesIntegration {
             datasource_name: self.datasource_name,
             datasource_type: self.datasource_type,
         }
