--- original
+++ modified
@@ -35,8 +35,8 @@
 }
 impl S3Configuration {
     /// Creates a new builder-style object to manufacture [`S3Configuration`](crate::types::S3Configuration).
-    pub fn builder() -> crate::types::builders::S3ConfigurationBuilder {
-        crate::types::builders::S3ConfigurationBuilder::default()
+    pub fn builder() -> super::types::builders::S3ConfigurationBuilder {
+        super::types::builders::S3ConfigurationBuilder::default()
     }
 }

@@ -112,8 +112,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`destination_identifier`](crate::types::builders::S3ConfigurationBuilder::destination_identifier)
     /// - [`role_arn`](crate::types::builders::S3ConfigurationBuilder::role_arn)
-    pub fn build(self) -> ::std::result::Result<crate::types::S3Configuration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::S3Configuration {
+    pub fn build(self) -> ::std::result::Result<super::types::S3Configuration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::S3Configuration {
             destination_identifier: self.destination_identifier.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "destination_identifier",
