--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl SplitStringEntry {
     /// Creates a new builder-style object to manufacture [`SplitStringEntry`](crate::types::SplitStringEntry).
-    pub fn builder() -> crate::types::builders::SplitStringEntryBuilder {
-        crate::types::builders::SplitStringEntryBuilder::default()
+    pub fn builder() -> super::types::builders::SplitStringEntryBuilder {
+        super::types::builders::SplitStringEntryBuilder::default()
     }
 }

@@ -70,8 +70,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`source`](crate::types::builders::SplitStringEntryBuilder::source)
     /// - [`delimiter`](crate::types::builders::SplitStringEntryBuilder::delimiter)
-    pub fn build(self) -> ::std::result::Result<crate::types::SplitStringEntry, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SplitStringEntry {
+    pub fn build(self) -> ::std::result::Result<super::types::SplitStringEntry, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SplitStringEntry {
             source: self.source.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "source",
