--- original
+++ modified
@@ -5,9 +5,9 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub enum StartLiveTailResponseStream {
     /// <p>This object contains information about this Live Tail session, including the log groups included and the log stream filters, if any.</p>
-    SessionStart(crate::types::LiveTailSessionStart),
+    SessionStart(super::super::types::LiveTailSessionStart),
     /// <p>This object contains the log events and session metadata.</p>
-    SessionUpdate(crate::types::LiveTailSessionUpdate),
+    SessionUpdate(super::super::types::LiveTailSessionUpdate),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -21,7 +21,7 @@
 impl StartLiveTailResponseStream {
     /// Tries to convert the enum instance into [`SessionStart`](crate::types::StartLiveTailResponseStream::SessionStart), extracting the inner [`LiveTailSessionStart`](crate::types::LiveTailSessionStart).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_session_start(&self) -> ::std::result::Result<&crate::types::LiveTailSessionStart, &Self> {
+    pub fn as_session_start(&self) -> ::std::result::Result<&super::super::types::LiveTailSessionStart, &Self> {
         if let StartLiveTailResponseStream::SessionStart(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -34,7 +34,7 @@
     }
     /// Tries to convert the enum instance into [`SessionUpdate`](crate::types::StartLiveTailResponseStream::SessionUpdate), extracting the inner [`LiveTailSessionUpdate`](crate::types::LiveTailSessionUpdate).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_session_update(&self) -> ::std::result::Result<&crate::types::LiveTailSessionUpdate, &Self> {
+    pub fn as_session_update(&self) -> ::std::result::Result<&super::super::types::LiveTailSessionUpdate, &Self> {
         if let StartLiveTailResponseStream::SessionUpdate(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
