--- original
+++ modified
@@ -15,7 +15,7 @@
     /// <p></p>
     pub role_arn: ::std::option::Option<::std::string::String>,
     /// <p>The method used to distribute log data to the destination, which can be either random or grouped by log stream.</p>
-    pub distribution: ::std::option::Option<crate::types::Distribution>,
+    pub distribution: ::std::option::Option<super::super::types::Distribution>,
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
     /// <p>If this value is <code>true</code>, the subscription filter is applied on the transformed version of the log events instead of the original ingested log events.</p>
     pub apply_on_transformed_logs: bool,
@@ -48,7 +48,7 @@
         self.role_arn.as_deref()
     }
     /// <p>The method used to distribute log data to the destination, which can be either random or grouped by log stream.</p>
-    pub fn distribution(&self) -> ::std::option::Option<&crate::types::Distribution> {
+    pub fn distribution(&self) -> ::std::option::Option<&super::super::types::Distribution> {
         self.distribution.as_ref()
     }
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
@@ -73,8 +73,8 @@
 }
 impl SubscriptionFilter {
     /// Creates a new builder-style object to manufacture [`SubscriptionFilter`](crate::types::SubscriptionFilter).
-    pub fn builder() -> crate::types::builders::SubscriptionFilterBuilder {
-        crate::types::builders::SubscriptionFilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::SubscriptionFilterBuilder {
+        super::super::types::builders::SubscriptionFilterBuilder::default()
     }
 }

@@ -87,7 +87,7 @@
     pub(crate) filter_pattern: ::std::option::Option<::std::string::String>,
     pub(crate) destination_arn: ::std::option::Option<::std::string::String>,
     pub(crate) role_arn: ::std::option::Option<::std::string::String>,
-    pub(crate) distribution: ::std::option::Option<crate::types::Distribution>,
+    pub(crate) distribution: ::std::option::Option<super::super::types::Distribution>,
     pub(crate) apply_on_transformed_logs: ::std::option::Option<bool>,
     pub(crate) creation_time: ::std::option::Option<i64>,
     pub(crate) field_selection_criteria: ::std::option::Option<::std::string::String>,
@@ -165,17 +165,17 @@
         &self.role_arn
     }
     /// <p>The method used to distribute log data to the destination, which can be either random or grouped by log stream.</p>
-    pub fn distribution(mut self, input: crate::types::Distribution) -> Self {
+    pub fn distribution(mut self, input: super::super::types::Distribution) -> Self {
         self.distribution = ::std::option::Option::Some(input);
         self
     }
     /// <p>The method used to distribute log data to the destination, which can be either random or grouped by log stream.</p>
-    pub fn set_distribution(mut self, input: ::std::option::Option<crate::types::Distribution>) -> Self {
+    pub fn set_distribution(mut self, input: ::std::option::Option<super::super::types::Distribution>) -> Self {
         self.distribution = input;
         self
     }
     /// <p>The method used to distribute log data to the destination, which can be either random or grouped by log stream.</p>
-    pub fn get_distribution(&self) -> &::std::option::Option<crate::types::Distribution> {
+    pub fn get_distribution(&self) -> &::std::option::Option<super::super::types::Distribution> {
         &self.distribution
     }
     /// <p>This parameter is valid only for log groups that have an active log transformer. For more information about log transformers, see <a href="https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutTransformer.html">PutTransformer</a>.</p>
@@ -244,8 +244,8 @@
         &self.emit_system_fields
     }
     /// Consumes the builder and constructs a [`SubscriptionFilter`](crate::types::SubscriptionFilter).
-    pub fn build(self) -> crate::types::SubscriptionFilter {
-        crate::types::SubscriptionFilter {
+    pub fn build(self) -> super::super::types::SubscriptionFilter {
+        super::super::types::SubscriptionFilter {
             filter_name: self.filter_name,
             log_group_name: self.log_group_name,
             filter_pattern: self.filter_pattern,
