--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl SubstituteStringEntry {
     /// Creates a new builder-style object to manufacture [`SubstituteStringEntry`](crate::types::SubstituteStringEntry).
-    pub fn builder() -> crate::types::builders::SubstituteStringEntryBuilder {
-        crate::types::builders::SubstituteStringEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::SubstituteStringEntryBuilder {
+        super::super::types::builders::SubstituteStringEntryBuilder::default()
     }
 }

@@ -94,8 +94,8 @@
     /// - [`source`](crate::types::builders::SubstituteStringEntryBuilder::source)
     /// - [`from`](crate::types::builders::SubstituteStringEntryBuilder::from)
     /// - [`to`](crate::types::builders::SubstituteStringEntryBuilder::to)
-    pub fn build(self) -> ::std::result::Result<crate::types::SubstituteStringEntry, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SubstituteStringEntry {
+    pub fn build(self) -> ::std::result::Result<super::super::types::SubstituteStringEntry, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::SubstituteStringEntry {
             source: self.source.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "source",
