--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl TransformedLogRecord {
     /// Creates a new builder-style object to manufacture [`TransformedLogRecord`](crate::types::TransformedLogRecord).
-    pub fn builder() -> crate::types::builders::TransformedLogRecordBuilder {
-        crate::types::builders::TransformedLogRecordBuilder::default()
+    pub fn builder() -> super::super::types::builders::TransformedLogRecordBuilder {
+        super::super::types::builders::TransformedLogRecordBuilder::default()
     }
 }

@@ -84,8 +84,8 @@
         &self.transformed_event_message
     }
     /// Consumes the builder and constructs a [`TransformedLogRecord`](crate::types::TransformedLogRecord).
-    pub fn build(self) -> crate::types::TransformedLogRecord {
-        crate::types::TransformedLogRecord {
+    pub fn build(self) -> super::super::types::TransformedLogRecord {
+        super::super::types::TransformedLogRecord {
             event_number: self.event_number.unwrap_or_default(),
             event_message: self.event_message,
             transformed_event_message: self.transformed_event_message,
