--- original
+++ modified
@@ -7,13 +7,13 @@
     /// <p>The unique identifier for this query execution.</p>
     pub query_id: ::std::option::Option<::std::string::String>,
     /// <p>The execution status of the scheduled query run.</p>
-    pub execution_status: ::std::option::Option<crate::types::ExecutionStatus>,
+    pub execution_status: ::std::option::Option<super::super::types::ExecutionStatus>,
     /// <p>The timestamp when the scheduled query execution was triggered.</p>
     pub triggered_timestamp: ::std::option::Option<i64>,
     /// <p>Error message if the query execution failed.</p>
     pub error_message: ::std::option::Option<::std::string::String>,
     /// <p>Information about destination processing for this query execution.</p>
-    pub destinations: ::std::option::Option<::std::vec::Vec<crate::types::ScheduledQueryDestination>>,
+    pub destinations: ::std::option::Option<::std::vec::Vec<super::super::types::ScheduledQueryDestination>>,
 }
 impl TriggerHistoryRecord {
     /// <p>The unique identifier for this query execution.</p>
@@ -21,7 +21,7 @@
         self.query_id.as_deref()
     }
     /// <p>The execution status of the scheduled query run.</p>
-    pub fn execution_status(&self) -> ::std::option::Option<&crate::types::ExecutionStatus> {
+    pub fn execution_status(&self) -> ::std::option::Option<&super::super::types::ExecutionStatus> {
         self.execution_status.as_ref()
     }
     /// <p>The timestamp when the scheduled query execution was triggered.</p>
@@ -35,14 +35,14 @@
     /// <p>Information about destination processing for this query execution.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.destinations.is_none()`.
-    pub fn destinations(&self) -> &[crate::types::ScheduledQueryDestination] {
+    pub fn destinations(&self) -> &[super::super::types::ScheduledQueryDestination] {
         self.destinations.as_deref().unwrap_or_default()
     }
 }
 impl TriggerHistoryRecord {
     /// Creates a new builder-style object to manufacture [`TriggerHistoryRecord`](crate::types::TriggerHistoryRecord).
-    pub fn builder() -> crate::types::builders::TriggerHistoryRecordBuilder {
-        crate::types::builders::TriggerHistoryRecordBuilder::default()
+    pub fn builder() -> super::super::types::builders::TriggerHistoryRecordBuilder {
+        super::super::types::builders::TriggerHistoryRecordBuilder::default()
     }
 }

@@ -51,10 +51,10 @@
 #[non_exhaustive]
 pub struct TriggerHistoryRecordBuilder {
     pub(crate) query_id: ::std::option::Option<::std::string::String>,
-    pub(crate) execution_status: ::std::option::Option<crate::types::ExecutionStatus>,
+    pub(crate) execution_status: ::std::option::Option<super::super::types::ExecutionStatus>,
     pub(crate) triggered_timestamp: ::std::option::Option<i64>,
     pub(crate) error_message: ::std::option::Option<::std::string::String>,
-    pub(crate) destinations: ::std::option::Option<::std::vec::Vec<crate::types::ScheduledQueryDestination>>,
+    pub(crate) destinations: ::std::option::Option<::std::vec::Vec<super::super::types::ScheduledQueryDestination>>,
 }
 impl TriggerHistoryRecordBuilder {
     /// <p>The unique identifier for this query execution.</p>
@@ -72,17 +72,17 @@
         &self.query_id
     }
     /// <p>The execution status of the scheduled query run.</p>
-    pub fn execution_status(mut self, input: crate::types::ExecutionStatus) -> Self {
+    pub fn execution_status(mut self, input: super::super::types::ExecutionStatus) -> Self {
         self.execution_status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The execution status of the scheduled query run.</p>
-    pub fn set_execution_status(mut self, input: ::std::option::Option<crate::types::ExecutionStatus>) -> Self {
+    pub fn set_execution_status(mut self, input: ::std::option::Option<super::super::types::ExecutionStatus>) -> Self {
         self.execution_status = input;
         self
     }
     /// <p>The execution status of the scheduled query run.</p>
-    pub fn get_execution_status(&self) -> &::std::option::Option<crate::types::ExecutionStatus> {
+    pub fn get_execution_status(&self) -> &::std::option::Option<super::super::types::ExecutionStatus> {
         &self.execution_status
     }
     /// <p>The timestamp when the scheduled query execution was triggered.</p>
@@ -118,7 +118,7 @@
     /// To override the contents of this collection use [`set_destinations`](Self::set_destinations).
     ///
     /// <p>Information about destination processing for this query execution.</p>
-    pub fn destinations(mut self, input: crate::types::ScheduledQueryDestination) -> Self {
+    pub fn destinations(mut self, input: super::super::types::ScheduledQueryDestination) -> Self {
         let mut v = self.destinations.unwrap_or_default();
         v.push(input);
         self.destinations = ::std::option::Option::Some(v);
@@ -125,17 +125,17 @@
         self
     }
     /// <p>Information about destination processing for this query execution.</p>
-    pub fn set_destinations(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ScheduledQueryDestination>>) -> Self {
+    pub fn set_destinations(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::ScheduledQueryDestination>>) -> Self {
         self.destinations = input;
         self
     }
     /// <p>Information about destination processing for this query execution.</p>
-    pub fn get_destinations(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ScheduledQueryDestination>> {
+    pub fn get_destinations(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::ScheduledQueryDestination>> {
         &self.destinations
     }
     /// Consumes the builder and constructs a [`TriggerHistoryRecord`](crate::types::TriggerHistoryRecord).
-    pub fn build(self) -> crate::types::TriggerHistoryRecord {
-        crate::types::TriggerHistoryRecord {
+    pub fn build(self) -> super::super::types::TriggerHistoryRecord {
+        super::super::types::TriggerHistoryRecord {
             query_id: self.query_id,
             execution_status: self.execution_status,
             triggered_timestamp: self.triggered_timestamp,
