--- original
+++ modified
@@ -7,7 +7,7 @@
     Option<
         ::std::collections::HashMap<
             ::std::string::String,
-            ::std::vec::Vec<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>,
+            ::std::vec::Vec<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>,
         >,
     >,
     ::aws_smithy_json::deserialize::error::DeserializeError,
@@ -29,7 +29,7 @@
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => {
                         let key = key.to_unescaped().map(|u| u.into_owned())?;
-                        let value = crate::protocol_serde::shape_item_list::de_item_list(tokens, _value, depth + 1)?;
+                        let value = super::protocol_serde::shape_item_list::de_item_list(tokens, _value, depth + 1)?;
                         match value {
                             Some(value) => {
                                 map.insert(key, value);
