--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ReplicaSettingsDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::ReplicaSettingsDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ReplicaSettingsDescriptionBuilder::default();
+            let mut builder = super::super::types::builders::ReplicaSettingsDescriptionBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,13 +31,13 @@
                         "ReplicaStatus" => {
                             builder = builder.set_replica_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ReplicaStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::ReplicaStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "ReplicaBillingModeSummary" => {
                             builder = builder.set_replica_billing_mode_summary(
-                                crate::protocol_serde::shape_billing_mode_summary::de_billing_mode_summary(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_billing_mode_summary::de_billing_mode_summary(tokens, _value, depth + 1)?,
                             );
                         }
                         "ReplicaProvisionedReadCapacityUnits" => {
@@ -49,7 +49,7 @@
                         }
                         "ReplicaProvisionedReadCapacityAutoScalingSettings" => {
                             builder = builder.set_replica_provisioned_read_capacity_auto_scaling_settings(
-                                crate::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
+                                super::super::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -65,7 +65,7 @@
                         }
                         "ReplicaProvisionedWriteCapacityAutoScalingSettings" => {
                             builder = builder.set_replica_provisioned_write_capacity_auto_scaling_settings(
-                                crate::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
+                                super::super::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -74,12 +74,12 @@
                         }
                         "ReplicaGlobalSecondaryIndexSettings" => {
                             builder = builder.set_replica_global_secondary_index_settings(
-                                    crate::protocol_serde::shape_replica_global_secondary_index_settings_description_list::de_replica_global_secondary_index_settings_description_list(tokens, _value, depth + 1)?
+                                    super::super::protocol_serde::shape_replica_global_secondary_index_settings_description_list::de_replica_global_secondary_index_settings_description_list(tokens, _value, depth + 1)?
                                 );
                         }
                         "ReplicaTableClassSummary" => {
                             builder = builder.set_replica_table_class_summary(
-                                crate::protocol_serde::shape_table_class_summary::de_table_class_summary(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_table_class_summary::de_table_class_summary(tokens, _value, depth + 1)?,
                             );
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
@@ -92,7 +92,7 @@
                 }
             }
             Ok(Some(
-                crate::serde_util::replica_settings_description_correct_errors(builder)
+                super::super::serde_util::replica_settings_description_correct_errors(builder)
                     .build()
                     .map_err(|err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err))?,
             ))
