--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ReplicaAutoScalingDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::ReplicaAutoScalingDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ReplicaAutoScalingDescriptionBuilder::default();
+            let mut builder = super::super::types::builders::ReplicaAutoScalingDescriptionBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -30,12 +30,12 @@
                         }
                         "GlobalSecondaryIndexes" => {
                             builder = builder.set_global_secondary_indexes(
-                                    crate::protocol_serde::shape_replica_global_secondary_index_auto_scaling_description_list::de_replica_global_secondary_index_auto_scaling_description_list(tokens, _value, depth + 1)?
+                                    super::super::protocol_serde::shape_replica_global_secondary_index_auto_scaling_description_list::de_replica_global_secondary_index_auto_scaling_description_list(tokens, _value, depth + 1)?
                                 );
                         }
                         "ReplicaProvisionedReadCapacityAutoScalingSettings" => {
                             builder = builder.set_replica_provisioned_read_capacity_auto_scaling_settings(
-                                crate::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
+                                super::super::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -44,7 +44,7 @@
                         }
                         "ReplicaProvisionedWriteCapacityAutoScalingSettings" => {
                             builder = builder.set_replica_provisioned_write_capacity_auto_scaling_settings(
-                                crate::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
+                                super::super::protocol_serde::shape_auto_scaling_settings_description::de_auto_scaling_settings_description(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -54,7 +54,7 @@
                         "ReplicaStatus" => {
                             builder = builder.set_replica_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ReplicaStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::ReplicaStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
