--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::TableDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::TableDescription>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::TableDescriptionBuilder::default();
+            let mut builder = super::types::builders::TableDescriptionBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -24,7 +24,7 @@
                         match key.to_unescaped()?.as_ref() {
                             "AttributeDefinitions" => {
                                 builder = builder.set_attribute_definitions(
-                                    crate::protocol_serde::shape_attribute_definitions::de_attribute_definitions(tokens, _value, depth + 1)?,
+                                    super::protocol_serde::shape_attribute_definitions::de_attribute_definitions(tokens, _value, depth + 1)?,
                                 );
                             }
                             "TableName" => {
@@ -35,12 +35,12 @@
                                 );
                             }
                             "KeySchema" => {
-                                builder = builder.set_key_schema(crate::protocol_serde::shape_key_schema::de_key_schema(tokens, _value, depth + 1)?);
+                                builder = builder.set_key_schema(super::protocol_serde::shape_key_schema::de_key_schema(tokens, _value, depth + 1)?);
                             }
                             "TableStatus" => {
                                 builder = builder.set_table_status(
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                        .map(|s| s.to_unescaped().map(|u| crate::types::TableStatus::from(u.as_ref())))
+                                        .map(|s| s.to_unescaped().map(|u| super::types::TableStatus::from(u.as_ref())))
                                         .transpose()?,
                                 );
                             }
@@ -52,7 +52,7 @@
                             }
                             "ProvisionedThroughput" => {
                                 builder = builder.set_provisioned_throughput(
-                                    crate::protocol_serde::shape_provisioned_throughput_description::de_provisioned_throughput_description(
+                                    super::protocol_serde::shape_provisioned_throughput_description::de_provisioned_throughput_description(
                                         tokens,
                                         _value,
                                         depth + 1,
@@ -89,12 +89,12 @@
                             }
                             "BillingModeSummary" => {
                                 builder = builder.set_billing_mode_summary(
-                                    crate::protocol_serde::shape_billing_mode_summary::de_billing_mode_summary(tokens, _value, depth + 1)?,
+                                    super::protocol_serde::shape_billing_mode_summary::de_billing_mode_summary(tokens, _value, depth + 1)?,
                                 );
                             }
                             "LocalSecondaryIndexes" => {
                                 builder = builder.set_local_secondary_indexes(
-                                    crate::protocol_serde::shape_local_secondary_index_description_list::de_local_secondary_index_description_list(
+                                    super::protocol_serde::shape_local_secondary_index_description_list::de_local_secondary_index_description_list(
                                         tokens,
                                         _value,
                                         depth + 1,
@@ -103,7 +103,7 @@
                             }
                             "GlobalSecondaryIndexes" => {
                                 builder = builder.set_global_secondary_indexes(
-                                    crate::protocol_serde::shape_global_secondary_index_description_list::de_global_secondary_index_description_list(
+                                    super::protocol_serde::shape_global_secondary_index_description_list::de_global_secondary_index_description_list(
                                         tokens,
                                         _value,
                                         depth + 1,
@@ -112,7 +112,7 @@
                             }
                             "StreamSpecification" => {
                                 builder = builder.set_stream_specification(
-                                    crate::protocol_serde::shape_stream_specification::de_stream_specification(tokens, _value, depth + 1)?,
+                                    super::protocol_serde::shape_stream_specification::de_stream_specification(tokens, _value, depth + 1)?,
                                 );
                             }
                             "LatestStreamLabel" => {
@@ -137,7 +137,7 @@
                                 );
                             }
                             "Replicas" => {
-                                builder = builder.set_replicas(crate::protocol_serde::shape_replica_description_list::de_replica_description_list(
+                                builder = builder.set_replicas(super::protocol_serde::shape_replica_description_list::de_replica_description_list(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -145,7 +145,7 @@
                             }
                             "GlobalTableWitnesses" => {
                                 builder = builder.set_global_table_witnesses(
-                                    crate::protocol_serde::shape_global_table_witness_description_list::de_global_table_witness_description_list(
+                                    super::protocol_serde::shape_global_table_witness_description_list::de_global_table_witness_description_list(
                                         tokens,
                                         _value,
                                         depth + 1,
@@ -157,13 +157,13 @@
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                         .map(|s| {
                                             s.to_unescaped()
-                                                .map(|u| crate::types::GlobalTableSettingsReplicationMode::from(u.as_ref()))
+                                                .map(|u| super::types::GlobalTableSettingsReplicationMode::from(u.as_ref()))
                                         })
                                         .transpose()?,
                                 );
                             }
                             "RestoreSummary" => {
-                                builder = builder.set_restore_summary(crate::protocol_serde::shape_restore_summary::de_restore_summary(
+                                builder = builder.set_restore_summary(super::protocol_serde::shape_restore_summary::de_restore_summary(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -170,7 +170,7 @@
                                 )?);
                             }
                             "SSEDescription" => {
-                                builder = builder.set_sse_description(crate::protocol_serde::shape_sse_description::de_sse_description(
+                                builder = builder.set_sse_description(super::protocol_serde::shape_sse_description::de_sse_description(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -177,7 +177,7 @@
                                 )?);
                             }
                             "ArchivalSummary" => {
-                                builder = builder.set_archival_summary(crate::protocol_serde::shape_archival_summary::de_archival_summary(
+                                builder = builder.set_archival_summary(super::protocol_serde::shape_archival_summary::de_archival_summary(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -184,7 +184,7 @@
                                 )?);
                             }
                             "TableClassSummary" => {
-                                builder = builder.set_table_class_summary(crate::protocol_serde::shape_table_class_summary::de_table_class_summary(
+                                builder = builder.set_table_class_summary(super::protocol_serde::shape_table_class_summary::de_table_class_summary(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -196,12 +196,12 @@
                             }
                             "OnDemandThroughput" => {
                                 builder = builder.set_on_demand_throughput(
-                                    crate::protocol_serde::shape_on_demand_throughput::de_on_demand_throughput(tokens, _value, depth + 1)?,
+                                    super::protocol_serde::shape_on_demand_throughput::de_on_demand_throughput(tokens, _value, depth + 1)?,
                                 );
                             }
                             "WarmThroughput" => {
                                 builder = builder.set_warm_throughput(
-                                    crate::protocol_serde::shape_table_warm_throughput_description::de_table_warm_throughput_description(
+                                    super::protocol_serde::shape_table_warm_throughput_description::de_table_warm_throughput_description(
                                         tokens,
                                         _value,
                                         depth + 1,
@@ -211,13 +211,13 @@
                             "MultiRegionConsistency" => {
                                 builder = builder.set_multi_region_consistency(
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                        .map(|s| s.to_unescaped().map(|u| crate::types::MultiRegionConsistency::from(u.as_ref())))
+                                        .map(|s| s.to_unescaped().map(|u| super::types::MultiRegionConsistency::from(u.as_ref())))
                                         .transpose()?,
                                 );
                             }
                             "VectorIndexes" => {
                                 builder = builder.set_vector_indexes(
-                                    crate::protocol_serde::shape_vector_index_description_list::de_vector_index_description_list(
+                                    super::protocol_serde::shape_vector_index_description_list::de_vector_index_description_list(
                                         tokens,
                                         _value,
                                         depth + 1,
