--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeOutput,
-    crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError,
+    super::operation::export_table_to_point_in_time::ExportTableToPointInTimeOutput,
+    super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled(
+            return Err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ExportConflictException" => crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::ExportConflictException({
+        "ExportConflictException" => super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::ExportConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ExportConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_export_conflict_exception::de_export_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+                let mut output = super::types::error::builders::ExportConflictExceptionBuilder::default();
+                output = super::protocol_serde::shape_export_conflict_exception::de_export_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "InternalServerError" => crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::InternalServerError({
+        "InternalServerError" => super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::InternalServerError({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalServerErrorBuilder::default();
-                output = crate::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
-                    .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+                let mut output = super::types::error::builders::InternalServerErrorBuilder::default();
+                output = super::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
+                    .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,14 +54,14 @@
             }
             tmp
         }),
-        "InvalidExportTimeException" => crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::InvalidExportTimeException({
+        "InvalidExportTimeException" => super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::InvalidExportTimeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidExportTimeExceptionBuilder::default();
+                let mut output = super::types::error::builders::InvalidExportTimeExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_invalid_export_time_exception::de_invalid_export_time_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+                    super::protocol_serde::shape_invalid_export_time_exception::de_invalid_export_time_exception_json_err(_response_body, output)
+                        .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,13 +70,13 @@
             }
             tmp
         }),
-        "LimitExceededException" => crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::LimitExceededException({
+        "LimitExceededException" => super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
+                    .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -86,12 +86,12 @@
             tmp
         }),
         "PointInTimeRecoveryUnavailableException" => {
-            crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::PointInTimeRecoveryUnavailableException({
+            super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::PointInTimeRecoveryUnavailableException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::PointInTimeRecoveryUnavailableExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_point_in_time_recovery_unavailable_exception::de_point_in_time_recovery_unavailable_exception_json_err(_response_body, output).map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+                    let mut output = super::types::error::builders::PointInTimeRecoveryUnavailableExceptionBuilder::default();
+                    output = super::protocol_serde::shape_point_in_time_recovery_unavailable_exception::de_point_in_time_recovery_unavailable_exception_json_err(_response_body, output).map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -101,13 +101,13 @@
                 tmp
             })
         }
-        "TableNotFoundException" => crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::TableNotFoundException({
+        "TableNotFoundException" => super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::TableNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TableNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_table_not_found_exception::de_table_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+                let mut output = super::types::error::builders::TableNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_table_not_found_exception::de_table_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -116,7 +116,7 @@
             }
             tmp
         }),
-        _ => crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::generic(generic),
+        _ => super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::generic(generic),
     })
 }

@@ -126,14 +126,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeOutput,
-    crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError,
+    super::operation::export_table_to_point_in_time::ExportTableToPointInTimeOutput,
+    super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeOutputBuilder::default();
-        output = crate::protocol_serde::shape_export_table_to_point_in_time::de_export_table_to_point_in_time(_response_body, output)
-            .map_err(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
+        let mut output = super::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeOutputBuilder::default();
+        output = super::protocol_serde::shape_export_table_to_point_in_time::de_export_table_to_point_in_time(_response_body, output)
+            .map_err(super::operation::export_table_to_point_in_time::ExportTableToPointInTimeError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -140,11 +140,11 @@
 }

 pub fn ser_export_table_to_point_in_time_input(
-    input: &crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput,
+    input: &super::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_export_table_to_point_in_time_input::ser_export_table_to_point_in_time_input_input(&mut object, input)?;
+    super::protocol_serde::shape_export_table_to_point_in_time_input::ser_export_table_to_point_in_time_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -151,12 +151,12 @@

 pub(crate) fn de_export_table_to_point_in_time(
     _value: &[u8],
-    mut builder: crate::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeOutputBuilder,
+    mut builder: super::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeOutputBuilder,
+    super::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -166,7 +166,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "ExportDescription" => {
-                    builder = builder.set_export_description(crate::protocol_serde::shape_export_description::de_export_description(
+                    builder = builder.set_export_description(super::protocol_serde::shape_export_description::de_export_description(
                         tokens,
                         _value,
                         depth + 1,
