--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<::std::vec::Vec<crate::types::KinesisDataStreamDestination>>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<::std::vec::Vec<super::super::types::KinesisDataStreamDestination>>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -23,7 +23,7 @@
                         break;
                     }
                     _ => {
-                        let value = crate::protocol_serde::shape_kinesis_data_stream_destination::de_kinesis_data_stream_destination(
+                        let value = super::super::protocol_serde::shape_kinesis_data_stream_destination::de_kinesis_data_stream_destination(
                             tokens,
                             _value,
                             depth + 1,
