--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_item::GetItemOutput, crate::operation::get_item::GetItemError> {
+) -> std::result::Result<super::super::operation::get_item::GetItemOutput, super::super::operation::get_item::GetItemError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_item::GetItemError::unhandled(generic)),
+        None => return Err(super::super::operation::get_item::GetItemError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalServerError" => crate::operation::get_item::GetItemError::InternalServerError({
+        "InternalServerError" => super::super::operation::get_item::GetItemError::InternalServerError({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalServerErrorBuilder::default();
-                output = crate::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
-                    .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalServerErrorBuilder::default();
+                output = super::super::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InvalidEndpointException" => crate::operation::get_item::GetItemError::InvalidEndpointException({
+        "InvalidEndpointException" => super::super::operation::get_item::GetItemError::InvalidEndpointException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidEndpointExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_endpoint_exception::de_invalid_endpoint_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidEndpointExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_endpoint_exception::de_invalid_endpoint_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -48,12 +48,12 @@
             tmp
         }),
         "ProvisionedThroughputExceededException" => {
-            crate::operation::get_item::GetItemError::ProvisionedThroughputExceededException({
+            super::super::operation::get_item::GetItemError::ProvisionedThroughputExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ProvisionedThroughputExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_provisioned_throughput_exceeded_exception::de_provisioned_throughput_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::get_item::GetItemError::unhandled)?;
+                    let mut output = super::super::types::error::builders::ProvisionedThroughputExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_provisioned_throughput_exceeded_exception::de_provisioned_throughput_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::get_item::GetItemError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -63,13 +63,13 @@
                 tmp
             })
         }
-        "RequestLimitExceeded" => crate::operation::get_item::GetItemError::RequestLimitExceeded({
+        "RequestLimitExceeded" => super::super::operation::get_item::GetItemError::RequestLimitExceeded({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RequestLimitExceededBuilder::default();
-                output = crate::protocol_serde::shape_request_limit_exceeded::de_request_limit_exceeded_json_err(_response_body, output)
-                    .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::RequestLimitExceededBuilder::default();
+                output = super::super::protocol_serde::shape_request_limit_exceeded::de_request_limit_exceeded_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -78,13 +78,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::get_item::GetItemError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::get_item::GetItemError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,13 +93,13 @@
             }
             tmp
         }),
-        "ThrottlingException" => crate::operation::get_item::GetItemError::ThrottlingException({
+        "ThrottlingException" => super::super::operation::get_item::GetItemError::ThrottlingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottlingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::ThrottlingExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,7 +108,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_item::GetItemError::generic(generic),
+        _ => super::super::operation::get_item::GetItemError::generic(generic),
     })
 }

@@ -117,12 +117,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_item::GetItemOutput, crate::operation::get_item::GetItemError> {
+) -> std::result::Result<super::super::operation::get_item::GetItemOutput, super::super::operation::get_item::GetItemError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_item::builders::GetItemOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_item::de_get_item(_response_body, output)
-            .map_err(crate::operation::get_item::GetItemError::unhandled)?;
+        let mut output = super::super::operation::get_item::builders::GetItemOutputBuilder::default();
+        output = super::super::protocol_serde::shape_get_item::de_get_item(_response_body, output)
+            .map_err(super::super::operation::get_item::GetItemError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -129,11 +129,11 @@
 }

 pub fn ser_get_item_input(
-    input: &crate::operation::get_item::GetItemInput,
+    input: &super::super::operation::get_item::GetItemInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_get_item_input::ser_get_item_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_get_item_input::ser_get_item_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -140,9 +140,9 @@

 pub(crate) fn de_get_item(
     _value: &[u8],
-    mut builder: crate::operation::get_item::builders::GetItemOutputBuilder,
-) -> ::std::result::Result<crate::operation::get_item::builders::GetItemOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    mut builder: super::super::operation::get_item::builders::GetItemOutputBuilder,
+) -> ::std::result::Result<super::super::operation::get_item::builders::GetItemOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -152,10 +152,10 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Item" => {
-                    builder = builder.set_item(crate::protocol_serde::shape_attribute_map::de_attribute_map(tokens, _value, depth + 1)?);
+                    builder = builder.set_item(super::super::protocol_serde::shape_attribute_map::de_attribute_map(tokens, _value, depth + 1)?);
                 }
                 "ConsumedCapacity" => {
-                    builder = builder.set_consumed_capacity(crate::protocol_serde::shape_consumed_capacity::de_consumed_capacity(
+                    builder = builder.set_consumed_capacity(super::super::protocol_serde::shape_consumed_capacity::de_consumed_capacity(
                         tokens,
                         _value,
                         depth + 1,
