--- original
+++ modified
@@ -4,7 +4,7 @@
     _value: &'a [u8],
     depth: u32,
 ) -> ::std::result::Result<
-    Option<::std::vec::Vec<::std::collections::HashMap<::std::string::String, crate::types::AttributeValue>>>,
+    Option<::std::vec::Vec<::std::collections::HashMap<::std::string::String, super::types::AttributeValue>>>,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 >
 where
@@ -26,7 +26,7 @@
                         break;
                     }
                     _ => {
-                        let value = crate::protocol_serde::shape_key::de_key(tokens, _value, depth + 1)?;
+                        let value = super::protocol_serde::shape_key::de_key(tokens, _value, depth + 1)?;
                         if let Some(value) = value {
                             items.push(value);
                         } else {
