--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::SourceTableDetails>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::SourceTableDetails>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::SourceTableDetailsBuilder::default();
+            let mut builder = super::super::types::builders::SourceTableDetailsBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -51,7 +51,7 @@
                                 );
                             }
                             "KeySchema" => {
-                                builder = builder.set_key_schema(crate::protocol_serde::shape_key_schema::de_key_schema(tokens, _value, depth + 1)?);
+                                builder = builder.set_key_schema(super::super::protocol_serde::shape_key_schema::de_key_schema(tokens, _value, depth + 1)?);
                             }
                             "TableCreationDateTime" => {
                                 builder = builder.set_table_creation_date_time(::aws_smithy_json::deserialize::token::expect_timestamp_or_null(
@@ -61,12 +61,12 @@
                             }
                             "ProvisionedThroughput" => {
                                 builder = builder.set_provisioned_throughput(
-                                    crate::protocol_serde::shape_provisioned_throughput::de_provisioned_throughput(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_provisioned_throughput::de_provisioned_throughput(tokens, _value, depth + 1)?,
                                 );
                             }
                             "OnDemandThroughput" => {
                                 builder = builder.set_on_demand_throughput(
-                                    crate::protocol_serde::shape_on_demand_throughput::de_on_demand_throughput(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_on_demand_throughput::de_on_demand_throughput(tokens, _value, depth + 1)?,
                                 );
                             }
                             "ItemCount" => {
@@ -79,7 +79,7 @@
                             "BillingMode" => {
                                 builder = builder.set_billing_mode(
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                        .map(|s| s.to_unescaped().map(|u| crate::types::BillingMode::from(u.as_ref())))
+                                        .map(|s| s.to_unescaped().map(|u| super::super::types::BillingMode::from(u.as_ref())))
                                         .transpose()?,
                                 );
                             }
@@ -93,7 +93,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::source_table_details_correct_errors(builder).build().map_err(
+            Ok(Some(super::super::serde_util::source_table_details_correct_errors(builder).build().map_err(
                 |err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err),
             )?))
         }
