--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::update_item::UpdateItemOutput, crate::operation::update_item::UpdateItemError> {
+) -> std::result::Result<super::super::operation::update_item::UpdateItemOutput, super::super::operation::update_item::UpdateItemError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_item::UpdateItemError::unhandled(generic)),
+        None => return Err(super::super::operation::update_item::UpdateItemError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConditionalCheckFailedException" => crate::operation::update_item::UpdateItemError::ConditionalCheckFailedException({
+        "ConditionalCheckFailedException" => super::super::operation::update_item::UpdateItemError::ConditionalCheckFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConditionalCheckFailedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_conditional_check_failed_exception::de_conditional_check_failed_exception_json_err(
+                let mut output = super::super::types::error::builders::ConditionalCheckFailedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_conditional_check_failed_exception::de_conditional_check_failed_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InternalServerError" => crate::operation::update_item::UpdateItemError::InternalServerError({
+        "InternalServerError" => super::super::operation::update_item::UpdateItemError::InternalServerError({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalServerErrorBuilder::default();
-                output = crate::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
-                    .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalServerErrorBuilder::default();
+                output = super::super::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "InvalidEndpointException" => crate::operation::update_item::UpdateItemError::InvalidEndpointException({
+        "InvalidEndpointException" => super::super::operation::update_item::UpdateItemError::InvalidEndpointException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidEndpointExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_endpoint_exception::de_invalid_endpoint_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidEndpointExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_endpoint_exception::de_invalid_endpoint_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,12 +65,12 @@
             }
             tmp
         }),
-        "ItemCollectionSizeLimitExceededException" => crate::operation::update_item::UpdateItemError::ItemCollectionSizeLimitExceededException({
+        "ItemCollectionSizeLimitExceededException" => super::super::operation::update_item::UpdateItemError::ItemCollectionSizeLimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ItemCollectionSizeLimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_item_collection_size_limit_exceeded_exception::de_item_collection_size_limit_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::ItemCollectionSizeLimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_item_collection_size_limit_exceeded_exception::de_item_collection_size_limit_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -80,12 +80,12 @@
             tmp
         }),
         "ProvisionedThroughputExceededException" => {
-            crate::operation::update_item::UpdateItemError::ProvisionedThroughputExceededException({
+            super::super::operation::update_item::UpdateItemError::ProvisionedThroughputExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ProvisionedThroughputExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_provisioned_throughput_exceeded_exception::de_provisioned_throughput_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                    let mut output = super::super::types::error::builders::ProvisionedThroughputExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_provisioned_throughput_exceeded_exception::de_provisioned_throughput_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -95,16 +95,16 @@
                 tmp
             })
         }
-        "ReplicatedWriteConflictException" => crate::operation::update_item::UpdateItemError::ReplicatedWriteConflictException({
+        "ReplicatedWriteConflictException" => super::super::operation::update_item::UpdateItemError::ReplicatedWriteConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ReplicatedWriteConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_replicated_write_conflict_exception::de_replicated_write_conflict_exception_json_err(
+                let mut output = super::super::types::error::builders::ReplicatedWriteConflictExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_replicated_write_conflict_exception::de_replicated_write_conflict_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -113,13 +113,13 @@
             }
             tmp
         }),
-        "RequestLimitExceeded" => crate::operation::update_item::UpdateItemError::RequestLimitExceeded({
+        "RequestLimitExceeded" => super::super::operation::update_item::UpdateItemError::RequestLimitExceeded({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RequestLimitExceededBuilder::default();
-                output = crate::protocol_serde::shape_request_limit_exceeded::de_request_limit_exceeded_json_err(_response_body, output)
-                    .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::RequestLimitExceededBuilder::default();
+                output = super::super::protocol_serde::shape_request_limit_exceeded::de_request_limit_exceeded_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -128,13 +128,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::update_item::UpdateItemError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::update_item::UpdateItemError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -143,13 +143,13 @@
             }
             tmp
         }),
-        "ThrottlingException" => crate::operation::update_item::UpdateItemError::ThrottlingException({
+        "ThrottlingException" => super::super::operation::update_item::UpdateItemError::ThrottlingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottlingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                let mut output = super::super::types::error::builders::ThrottlingExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -158,14 +158,14 @@
             }
             tmp
         }),
-        "TransactionConflictException" => crate::operation::update_item::UpdateItemError::TransactionConflictException({
+        "TransactionConflictException" => super::super::operation::update_item::UpdateItemError::TransactionConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TransactionConflictExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::TransactionConflictExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_transaction_conflict_exception::de_transaction_conflict_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+                    super::super::protocol_serde::shape_transaction_conflict_exception::de_transaction_conflict_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -174,7 +174,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_item::UpdateItemError::generic(generic),
+        _ => super::super::operation::update_item::UpdateItemError::generic(generic),
     })
 }

@@ -183,12 +183,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::update_item::UpdateItemOutput, crate::operation::update_item::UpdateItemError> {
+) -> std::result::Result<super::super::operation::update_item::UpdateItemOutput, super::super::operation::update_item::UpdateItemError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_item::builders::UpdateItemOutputBuilder::default();
-        output = crate::protocol_serde::shape_update_item::de_update_item(_response_body, output)
-            .map_err(crate::operation::update_item::UpdateItemError::unhandled)?;
+        let mut output = super::super::operation::update_item::builders::UpdateItemOutputBuilder::default();
+        output = super::super::protocol_serde::shape_update_item::de_update_item(_response_body, output)
+            .map_err(super::super::operation::update_item::UpdateItemError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -195,11 +195,11 @@
 }

 pub fn ser_update_item_input(
-    input: &crate::operation::update_item::UpdateItemInput,
+    input: &super::super::operation::update_item::UpdateItemInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_update_item_input::ser_update_item_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_update_item_input::ser_update_item_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -206,10 +206,10 @@

 pub(crate) fn de_update_item(
     _value: &[u8],
-    mut builder: crate::operation::update_item::builders::UpdateItemOutputBuilder,
-) -> ::std::result::Result<crate::operation::update_item::builders::UpdateItemOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError>
+    mut builder: super::super::operation::update_item::builders::UpdateItemOutputBuilder,
+) -> ::std::result::Result<super::super::operation::update_item::builders::UpdateItemOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError>
 {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -220,10 +220,10 @@
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => {
                 match key.to_unescaped()?.as_ref() {
                     "Attributes" => {
-                        builder = builder.set_attributes(crate::protocol_serde::shape_attribute_map::de_attribute_map(tokens, _value, depth + 1)?);
+                        builder = builder.set_attributes(super::super::protocol_serde::shape_attribute_map::de_attribute_map(tokens, _value, depth + 1)?);
                     }
                     "ConsumedCapacity" => {
-                        builder = builder.set_consumed_capacity(crate::protocol_serde::shape_consumed_capacity::de_consumed_capacity(
+                        builder = builder.set_consumed_capacity(super::super::protocol_serde::shape_consumed_capacity::de_consumed_capacity(
                             tokens,
                             _value,
                             depth + 1,
@@ -231,7 +231,7 @@
                     }
                     "ItemCollectionMetrics" => {
                         builder = builder.set_item_collection_metrics(
-                            crate::protocol_serde::shape_item_collection_metrics::de_item_collection_metrics(tokens, _value, depth + 1)?,
+                            super::super::protocol_serde::shape_item_collection_metrics::de_item_collection_metrics(tokens, _value, depth + 1)?,
                         );
                     }
                     _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
