--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::EnableKinesisStreamingConfiguration>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::EnableKinesisStreamingConfiguration>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::EnableKinesisStreamingConfigurationBuilder::default();
+            let mut builder = super::types::builders::EnableKinesisStreamingConfigurationBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -26,7 +26,7 @@
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
                                     .map(|s| {
                                         s.to_unescaped()
-                                            .map(|u| crate::types::ApproximateCreationDateTimePrecision::from(u.as_ref()))
+                                            .map(|u| super::types::ApproximateCreationDateTimePrecision::from(u.as_ref()))
                                     })
                                     .transpose()?,
                             );
@@ -50,7 +50,7 @@

 pub fn ser_enable_kinesis_streaming_configuration(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::EnableKinesisStreamingConfiguration,
+    input: &super::types::EnableKinesisStreamingConfiguration,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     if let Some(var_1) = &input.approximate_creation_date_time_precision {
         object.key("ApproximateCreationDateTimePrecision").string(var_1.as_str());
