--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::execute_statement::ExecuteStatementOutput, crate::operation::execute_statement::ExecuteStatementError> {
+) -> std::result::Result<super::operation::execute_statement::ExecuteStatementOutput, super::operation::execute_statement::ExecuteStatementError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::execute_statement::ExecuteStatementError::unhandled(generic)),
+        None => return Err(super::operation::execute_statement::ExecuteStatementError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConditionalCheckFailedException" => crate::operation::execute_statement::ExecuteStatementError::ConditionalCheckFailedException({
+        "ConditionalCheckFailedException" => super::operation::execute_statement::ExecuteStatementError::ConditionalCheckFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConditionalCheckFailedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_conditional_check_failed_exception::de_conditional_check_failed_exception_json_err(
+                let mut output = super::types::error::builders::ConditionalCheckFailedExceptionBuilder::default();
+                output = super::protocol_serde::shape_conditional_check_failed_exception::de_conditional_check_failed_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "DuplicateItemException" => crate::operation::execute_statement::ExecuteStatementError::DuplicateItemException({
+        "DuplicateItemException" => super::operation::execute_statement::ExecuteStatementError::DuplicateItemException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DuplicateItemExceptionBuilder::default();
-                output = crate::protocol_serde::shape_duplicate_item_exception::de_duplicate_item_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                let mut output = super::types::error::builders::DuplicateItemExceptionBuilder::default();
+                output = super::protocol_serde::shape_duplicate_item_exception::de_duplicate_item_exception_json_err(_response_body, output)
+                    .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "InternalServerError" => crate::operation::execute_statement::ExecuteStatementError::InternalServerError({
+        "InternalServerError" => super::operation::execute_statement::ExecuteStatementError::InternalServerError({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalServerErrorBuilder::default();
-                output = crate::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
-                    .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                let mut output = super::types::error::builders::InternalServerErrorBuilder::default();
+                output = super::protocol_serde::shape_internal_server_error::de_internal_server_error_json_err(_response_body, output)
+                    .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -66,12 +66,12 @@
             tmp
         }),
         "ItemCollectionSizeLimitExceededException" => {
-            crate::operation::execute_statement::ExecuteStatementError::ItemCollectionSizeLimitExceededException({
+            super::operation::execute_statement::ExecuteStatementError::ItemCollectionSizeLimitExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ItemCollectionSizeLimitExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_item_collection_size_limit_exceeded_exception::de_item_collection_size_limit_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                    let mut output = super::types::error::builders::ItemCollectionSizeLimitExceededExceptionBuilder::default();
+                    output = super::protocol_serde::shape_item_collection_size_limit_exceeded_exception::de_item_collection_size_limit_exceeded_exception_json_err(_response_body, output).map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -82,12 +82,12 @@
             })
         }
         "ProvisionedThroughputExceededException" => {
-            crate::operation::execute_statement::ExecuteStatementError::ProvisionedThroughputExceededException({
+            super::operation::execute_statement::ExecuteStatementError::ProvisionedThroughputExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ProvisionedThroughputExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_provisioned_throughput_exceeded_exception::de_provisioned_throughput_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                    let mut output = super::types::error::builders::ProvisionedThroughputExceededExceptionBuilder::default();
+                    output = super::protocol_serde::shape_provisioned_throughput_exceeded_exception::de_provisioned_throughput_exceeded_exception_json_err(_response_body, output).map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -97,13 +97,13 @@
                 tmp
             })
         }
-        "RequestLimitExceeded" => crate::operation::execute_statement::ExecuteStatementError::RequestLimitExceeded({
+        "RequestLimitExceeded" => super::operation::execute_statement::ExecuteStatementError::RequestLimitExceeded({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RequestLimitExceededBuilder::default();
-                output = crate::protocol_serde::shape_request_limit_exceeded::de_request_limit_exceeded_json_err(_response_body, output)
-                    .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                let mut output = super::types::error::builders::RequestLimitExceededBuilder::default();
+                output = super::protocol_serde::shape_request_limit_exceeded::de_request_limit_exceeded_json_err(_response_body, output)
+                    .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -112,13 +112,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::execute_statement::ExecuteStatementError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::execute_statement::ExecuteStatementError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -127,13 +127,13 @@
             }
             tmp
         }),
-        "ThrottlingException" => crate::operation::execute_statement::ExecuteStatementError::ThrottlingException({
+        "ThrottlingException" => super::operation::execute_statement::ExecuteStatementError::ThrottlingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ThrottlingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                let mut output = super::types::error::builders::ThrottlingExceptionBuilder::default();
+                output = super::protocol_serde::shape_throttling_exception::de_throttling_exception_json_err(_response_body, output)
+                    .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -142,14 +142,14 @@
             }
             tmp
         }),
-        "TransactionConflictException" => crate::operation::execute_statement::ExecuteStatementError::TransactionConflictException({
+        "TransactionConflictException" => super::operation::execute_statement::ExecuteStatementError::TransactionConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TransactionConflictExceptionBuilder::default();
+                let mut output = super::types::error::builders::TransactionConflictExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_transaction_conflict_exception::de_transaction_conflict_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+                    super::protocol_serde::shape_transaction_conflict_exception::de_transaction_conflict_exception_json_err(_response_body, output)
+                        .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -158,7 +158,7 @@
             }
             tmp
         }),
-        _ => crate::operation::execute_statement::ExecuteStatementError::generic(generic),
+        _ => super::operation::execute_statement::ExecuteStatementError::generic(generic),
     })
 }

@@ -167,12 +167,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::execute_statement::ExecuteStatementOutput, crate::operation::execute_statement::ExecuteStatementError> {
+) -> std::result::Result<super::operation::execute_statement::ExecuteStatementOutput, super::operation::execute_statement::ExecuteStatementError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::execute_statement::builders::ExecuteStatementOutputBuilder::default();
-        output = crate::protocol_serde::shape_execute_statement::de_execute_statement(_response_body, output)
-            .map_err(crate::operation::execute_statement::ExecuteStatementError::unhandled)?;
+        let mut output = super::operation::execute_statement::builders::ExecuteStatementOutputBuilder::default();
+        output = super::protocol_serde::shape_execute_statement::de_execute_statement(_response_body, output)
+            .map_err(super::operation::execute_statement::ExecuteStatementError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -179,11 +179,11 @@
 }

 pub fn ser_execute_statement_input(
-    input: &crate::operation::execute_statement::ExecuteStatementInput,
+    input: &super::operation::execute_statement::ExecuteStatementInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_execute_statement_input::ser_execute_statement_input_input(&mut object, input)?;
+    super::protocol_serde::shape_execute_statement_input::ser_execute_statement_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -190,12 +190,12 @@

 pub(crate) fn de_execute_statement(
     _value: &[u8],
-    mut builder: crate::operation::execute_statement::builders::ExecuteStatementOutputBuilder,
+    mut builder: super::operation::execute_statement::builders::ExecuteStatementOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::execute_statement::builders::ExecuteStatementOutputBuilder,
+    super::operation::execute_statement::builders::ExecuteStatementOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -205,7 +205,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Items" => {
-                    builder = builder.set_items(crate::protocol_serde::shape_item_list::de_item_list(tokens, _value, depth + 1)?);
+                    builder = builder.set_items(super::protocol_serde::shape_item_list::de_item_list(tokens, _value, depth + 1)?);
                 }
                 "NextToken" => {
                     builder = builder.set_next_token(
@@ -215,7 +215,7 @@
                     );
                 }
                 "ConsumedCapacity" => {
-                    builder = builder.set_consumed_capacity(crate::protocol_serde::shape_consumed_capacity::de_consumed_capacity(
+                    builder = builder.set_consumed_capacity(super::protocol_serde::shape_consumed_capacity::de_consumed_capacity(
                         tokens,
                         _value,
                         depth + 1,
@@ -222,7 +222,7 @@
                     )?);
                 }
                 "LastEvaluatedKey" => {
-                    builder = builder.set_last_evaluated_key(crate::protocol_serde::shape_key::de_key(tokens, _value, depth + 1)?);
+                    builder = builder.set_last_evaluated_key(super::protocol_serde::shape_key::de_key(tokens, _value, depth + 1)?);
                 }
                 _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
             },
