--- original
+++ modified
@@ -14,7 +14,7 @@
     /// <p>This element is different from the <code>Description</code> element such that this is a request specific message injected by the partner. The <code>Description</code> is typically a generic explanation of what the delegation request is targeted to do.</p>
     pub request_message: ::std::option::Option<::std::string::String>,
     /// <p>Contains information about the permissions being delegated in a delegation request.</p>
-    pub permissions: ::std::option::Option<crate::types::DelegationPermission>,
+    pub permissions: ::std::option::Option<super::super::types::DelegationPermission>,
     /// <p>JSON content of the associated permission policy of this delegation request.</p>
     pub permission_policy: ::std::option::Option<::std::string::String>,
     /// <p>If the <code>PermissionPolicy</code> includes role creation permissions, this element will include the list of permissions boundary policies associated with the role creation. See <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html">Permissions boundaries for IAM entities</a> for more details about IAM permission boundaries.</p>
@@ -26,7 +26,7 @@
     pub approver_id: ::std::option::Option<::std::string::String>,
     /// <p>The state of this delegation request.</p>
     /// <p>See the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/temporary-delegation-building-integration.html#temporary-delegation-request-lifecycle">Understanding the Request Lifecycle</a> for an explanation of how these states are transitioned.</p>
-    pub state: ::std::option::Option<crate::types::StateType>,
+    pub state: ::std::option::Option<super::super::types::StateType>,
     /// <p>The expiry time of this delegation request</p>
     /// <p>See the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/temporary-delegation-building-integration.html#temporary-delegation-request-lifecycle">Understanding the Request Lifecycle</a> for details on the life time of a delegation request at each state.</p>
     pub expiration_time: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -68,7 +68,7 @@
         self.request_message.as_deref()
     }
     /// <p>Contains information about the permissions being delegated in a delegation request.</p>
-    pub fn permissions(&self) -> ::std::option::Option<&crate::types::DelegationPermission> {
+    pub fn permissions(&self) -> ::std::option::Option<&super::super::types::DelegationPermission> {
         self.permissions.as_ref()
     }
     /// <p>JSON content of the associated permission policy of this delegation request.</p>
@@ -92,7 +92,7 @@
     }
     /// <p>The state of this delegation request.</p>
     /// <p>See the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/temporary-delegation-building-integration.html#temporary-delegation-request-lifecycle">Understanding the Request Lifecycle</a> for an explanation of how these states are transitioned.</p>
-    pub fn state(&self) -> ::std::option::Option<&crate::types::StateType> {
+    pub fn state(&self) -> ::std::option::Option<&super::super::types::StateType> {
         self.state.as_ref()
     }
     /// <p>The expiry time of this delegation request</p>
@@ -139,8 +139,8 @@
 }
 impl DelegationRequest {
     /// Creates a new builder-style object to manufacture [`DelegationRequest`](crate::types::DelegationRequest).
-    pub fn builder() -> crate::types::builders::DelegationRequestBuilder {
-        crate::types::builders::DelegationRequestBuilder::default()
+    pub fn builder() -> super::super::types::builders::DelegationRequestBuilder {
+        super::super::types::builders::DelegationRequestBuilder::default()
     }
 }

@@ -152,12 +152,12 @@
     pub(crate) owner_account_id: ::std::option::Option<::std::string::String>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
     pub(crate) request_message: ::std::option::Option<::std::string::String>,
-    pub(crate) permissions: ::std::option::Option<crate::types::DelegationPermission>,
+    pub(crate) permissions: ::std::option::Option<super::super::types::DelegationPermission>,
     pub(crate) permission_policy: ::std::option::Option<::std::string::String>,
     pub(crate) role_permission_restriction_arns: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
     pub(crate) owner_id: ::std::option::Option<::std::string::String>,
     pub(crate) approver_id: ::std::option::Option<::std::string::String>,
-    pub(crate) state: ::std::option::Option<crate::types::StateType>,
+    pub(crate) state: ::std::option::Option<super::super::types::StateType>,
     pub(crate) expiration_time: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) requestor_id: ::std::option::Option<::std::string::String>,
     pub(crate) requestor_name: ::std::option::Option<::std::string::String>,
@@ -230,17 +230,17 @@
         &self.request_message
     }
     /// <p>Contains information about the permissions being delegated in a delegation request.</p>
-    pub fn permissions(mut self, input: crate::types::DelegationPermission) -> Self {
+    pub fn permissions(mut self, input: super::super::types::DelegationPermission) -> Self {
         self.permissions = ::std::option::Option::Some(input);
         self
     }
     /// <p>Contains information about the permissions being delegated in a delegation request.</p>
-    pub fn set_permissions(mut self, input: ::std::option::Option<crate::types::DelegationPermission>) -> Self {
+    pub fn set_permissions(mut self, input: ::std::option::Option<super::super::types::DelegationPermission>) -> Self {
         self.permissions = input;
         self
     }
     /// <p>Contains information about the permissions being delegated in a delegation request.</p>
-    pub fn get_permissions(&self) -> &::std::option::Option<crate::types::DelegationPermission> {
+    pub fn get_permissions(&self) -> &::std::option::Option<super::super::types::DelegationPermission> {
         &self.permissions
     }
     /// <p>JSON content of the associated permission policy of this delegation request.</p>
@@ -310,19 +310,19 @@
     }
     /// <p>The state of this delegation request.</p>
     /// <p>See the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/temporary-delegation-building-integration.html#temporary-delegation-request-lifecycle">Understanding the Request Lifecycle</a> for an explanation of how these states are transitioned.</p>
-    pub fn state(mut self, input: crate::types::StateType) -> Self {
+    pub fn state(mut self, input: super::super::types::StateType) -> Self {
         self.state = ::std::option::Option::Some(input);
         self
     }
     /// <p>The state of this delegation request.</p>
     /// <p>See the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/temporary-delegation-building-integration.html#temporary-delegation-request-lifecycle">Understanding the Request Lifecycle</a> for an explanation of how these states are transitioned.</p>
-    pub fn set_state(mut self, input: ::std::option::Option<crate::types::StateType>) -> Self {
+    pub fn set_state(mut self, input: ::std::option::Option<super::super::types::StateType>) -> Self {
         self.state = input;
         self
     }
     /// <p>The state of this delegation request.</p>
     /// <p>See the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/temporary-delegation-building-integration.html#temporary-delegation-request-lifecycle">Understanding the Request Lifecycle</a> for an explanation of how these states are transitioned.</p>
-    pub fn get_state(&self) -> &::std::option::Option<crate::types::StateType> {
+    pub fn get_state(&self) -> &::std::option::Option<super::super::types::StateType> {
         &self.state
     }
     /// <p>The expiry time of this delegation request</p>
@@ -469,8 +469,8 @@
         &self.updated_time
     }
     /// Consumes the builder and constructs a [`DelegationRequest`](crate::types::DelegationRequest).
-    pub fn build(self) -> crate::types::DelegationRequest {
-        crate::types::DelegationRequest {
+    pub fn build(self) -> super::super::types::DelegationRequest {
+        super::super::types::DelegationRequest {
             delegation_request_id: self.delegation_request_id,
             owner_account_id: self.owner_account_id,
             description: self.description,
