--- original
+++ modified
@@ -6,7 +6,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct EntityDetails {
     /// <p>The&nbsp;<code>EntityInfo</code> object that contains details about the entity (user or role).</p>
-    pub entity_info: ::std::option::Option<crate::types::EntityInfo>,
+    pub entity_info: ::std::option::Option<super::types::EntityInfo>,
     /// <p>The date and time, in&nbsp;<a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>, when the authenticated entity last attempted to access Amazon Web Services. Amazon Web Services does not report unauthenticated requests.</p>
     /// <p>This field is null if no IAM entities attempted to access the service within the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#service-last-accessed-reporting-period">tracking period</a>.</p>
     pub last_authenticated: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -13,7 +13,7 @@
 }
 impl EntityDetails {
     /// <p>The&nbsp;<code>EntityInfo</code> object that contains details about the entity (user or role).</p>
-    pub fn entity_info(&self) -> ::std::option::Option<&crate::types::EntityInfo> {
+    pub fn entity_info(&self) -> ::std::option::Option<&super::types::EntityInfo> {
         self.entity_info.as_ref()
     }
     /// <p>The date and time, in&nbsp;<a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>, when the authenticated entity last attempted to access Amazon Web Services. Amazon Web Services does not report unauthenticated requests.</p>
@@ -24,8 +24,8 @@
 }
 impl EntityDetails {
     /// Creates a new builder-style object to manufacture [`EntityDetails`](crate::types::EntityDetails).
-    pub fn builder() -> crate::types::builders::EntityDetailsBuilder {
-        crate::types::builders::EntityDetailsBuilder::default()
+    pub fn builder() -> super::types::builders::EntityDetailsBuilder {
+        super::types::builders::EntityDetailsBuilder::default()
     }
 }

@@ -33,23 +33,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct EntityDetailsBuilder {
-    pub(crate) entity_info: ::std::option::Option<crate::types::EntityInfo>,
+    pub(crate) entity_info: ::std::option::Option<super::types::EntityInfo>,
     pub(crate) last_authenticated: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl EntityDetailsBuilder {
     /// <p>The&nbsp;<code>EntityInfo</code> object that contains details about the entity (user or role).</p>
     /// This field is required.
-    pub fn entity_info(mut self, input: crate::types::EntityInfo) -> Self {
+    pub fn entity_info(mut self, input: super::types::EntityInfo) -> Self {
         self.entity_info = ::std::option::Option::Some(input);
         self
     }
     /// <p>The&nbsp;<code>EntityInfo</code> object that contains details about the entity (user or role).</p>
-    pub fn set_entity_info(mut self, input: ::std::option::Option<crate::types::EntityInfo>) -> Self {
+    pub fn set_entity_info(mut self, input: ::std::option::Option<super::types::EntityInfo>) -> Self {
         self.entity_info = input;
         self
     }
     /// <p>The&nbsp;<code>EntityInfo</code> object that contains details about the entity (user or role).</p>
-    pub fn get_entity_info(&self) -> &::std::option::Option<crate::types::EntityInfo> {
+    pub fn get_entity_info(&self) -> &::std::option::Option<super::types::EntityInfo> {
         &self.entity_info
     }
     /// <p>The date and time, in&nbsp;<a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>, when the authenticated entity last attempted to access Amazon Web Services. Amazon Web Services does not report unauthenticated requests.</p>
@@ -70,8 +70,8 @@
         &self.last_authenticated
     }
     /// Consumes the builder and constructs a [`EntityDetails`](crate::types::EntityDetails).
-    pub fn build(self) -> crate::types::EntityDetails {
-        crate::types::EntityDetails {
+    pub fn build(self) -> super::types::EntityDetails {
+        super::types::EntityDetails {
             entity_info: self.entity_info,
             last_authenticated: self.last_authenticated,
         }
