--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl ErrorDetails {
     /// Creates a new builder-style object to manufacture [`ErrorDetails`](crate::types::ErrorDetails).
-    pub fn builder() -> crate::types::builders::ErrorDetailsBuilder {
-        crate::types::builders::ErrorDetailsBuilder::default()
+    pub fn builder() -> super::super::types::builders::ErrorDetailsBuilder {
+        super::super::types::builders::ErrorDetailsBuilder::default()
     }
 }

@@ -71,8 +71,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`message`](crate::types::builders::ErrorDetailsBuilder::message)
     /// - [`code`](crate::types::builders::ErrorDetailsBuilder::code)
-    pub fn build(self) -> ::std::result::Result<crate::types::ErrorDetails, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ErrorDetails {
+    pub fn build(self) -> ::std::result::Result<super::super::types::ErrorDetails, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::ErrorDetails {
             message: self.message.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "message",
