--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl LoginProfile {
     /// Creates a new builder-style object to manufacture [`LoginProfile`](crate::types::LoginProfile).
-    pub fn builder() -> crate::types::builders::LoginProfileBuilder {
-        crate::types::builders::LoginProfileBuilder::default()
+    pub fn builder() -> super::super::types::builders::LoginProfileBuilder {
+        super::super::types::builders::LoginProfileBuilder::default()
     }
 }

@@ -91,8 +91,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`user_name`](crate::types::builders::LoginProfileBuilder::user_name)
     /// - [`create_date`](crate::types::builders::LoginProfileBuilder::create_date)
-    pub fn build(self) -> ::std::result::Result<crate::types::LoginProfile, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::LoginProfile {
+    pub fn build(self) -> ::std::result::Result<super::super::types::LoginProfile, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::LoginProfile {
             user_name: self.user_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "user_name",
