--- original
+++ modified
@@ -17,8 +17,8 @@
 }
 impl OpenIdConnectProviderListEntry {
     /// Creates a new builder-style object to manufacture [`OpenIdConnectProviderListEntry`](crate::types::OpenIdConnectProviderListEntry).
-    pub fn builder() -> crate::types::builders::OpenIdConnectProviderListEntryBuilder {
-        crate::types::builders::OpenIdConnectProviderListEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::OpenIdConnectProviderListEntryBuilder {
+        super::super::types::builders::OpenIdConnectProviderListEntryBuilder::default()
     }
 }

@@ -47,7 +47,7 @@
         &self.arn
     }
     /// Consumes the builder and constructs a [`OpenIdConnectProviderListEntry`](crate::types::OpenIdConnectProviderListEntry).
-    pub fn build(self) -> crate::types::OpenIdConnectProviderListEntry {
-        crate::types::OpenIdConnectProviderListEntry { arn: self.arn }
+    pub fn build(self) -> super::super::types::OpenIdConnectProviderListEntry {
+        super::super::types::OpenIdConnectProviderListEntry { arn: self.arn }
     }
 }
