--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl PolicyDetail {
     /// Creates a new builder-style object to manufacture [`PolicyDetail`](crate::types::PolicyDetail).
-    pub fn builder() -> crate::types::builders::PolicyDetailBuilder {
-        crate::types::builders::PolicyDetailBuilder::default()
+    pub fn builder() -> super::super::types::builders::PolicyDetailBuilder {
+        super::super::types::builders::PolicyDetailBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
         &self.policy_document
     }
     /// Consumes the builder and constructs a [`PolicyDetail`](crate::types::PolicyDetail).
-    pub fn build(self) -> crate::types::PolicyDetail {
-        crate::types::PolicyDetail {
+    pub fn build(self) -> super::super::types::PolicyDetail {
+        super::super::types::PolicyDetail {
             policy_name: self.policy_name,
             policy_document: self.policy_document,
         }
