--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl PolicyUser {
     /// Creates a new builder-style object to manufacture [`PolicyUser`](crate::types::PolicyUser).
-    pub fn builder() -> crate::types::builders::PolicyUserBuilder {
-        crate::types::builders::PolicyUserBuilder::default()
+    pub fn builder() -> super::super::types::builders::PolicyUserBuilder {
+        super::super::types::builders::PolicyUserBuilder::default()
     }
 }

@@ -65,8 +65,8 @@
         &self.user_id
     }
     /// Consumes the builder and constructs a [`PolicyUser`](crate::types::PolicyUser).
-    pub fn build(self) -> crate::types::PolicyUser {
-        crate::types::PolicyUser {
+    pub fn build(self) -> super::super::types::PolicyUser {
+        super::super::types::PolicyUser {
             user_name: self.user_name,
             user_id: self.user_id,
         }
