--- original
+++ modified
@@ -19,18 +19,18 @@
     /// <p>The trust policy that grants permission to assume the role.</p>
     pub assume_role_policy_document: ::std::option::Option<::std::string::String>,
     /// <p>A list of instance profiles that contain this role.</p>
-    pub instance_profile_list: ::std::option::Option<::std::vec::Vec<crate::types::InstanceProfile>>,
+    pub instance_profile_list: ::std::option::Option<::std::vec::Vec<super::super::types::InstanceProfile>>,
     /// <p>A list of inline policies embedded in the role. These policies are the role's access (permissions) policies.</p>
-    pub role_policy_list: ::std::option::Option<::std::vec::Vec<crate::types::PolicyDetail>>,
+    pub role_policy_list: ::std::option::Option<::std::vec::Vec<super::super::types::PolicyDetail>>,
     /// <p>A list of managed policies attached to the role. These policies are the role's access (permissions) policies.</p>
-    pub attached_managed_policies: ::std::option::Option<::std::vec::Vec<crate::types::AttachedPolicy>>,
+    pub attached_managed_policies: ::std::option::Option<::std::vec::Vec<super::super::types::AttachedPolicy>>,
     /// <p>The ARN of the policy used to set the permissions boundary for the role.</p>
     /// <p>For more information about permissions boundaries, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html">Permissions boundaries for IAM identities </a> in the <i>IAM User Guide</i>.</p>
-    pub permissions_boundary: ::std::option::Option<crate::types::AttachedPermissionsBoundary>,
+    pub permissions_boundary: ::std::option::Option<super::super::types::AttachedPermissionsBoundary>,
     /// <p>A list of tags that are attached to the role. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::super::types::Tag>>,
     /// <p>Contains information about the last time that an IAM role was used. This includes the date and time and the Region in which the role was last used. Activity is only reported for the trailing 400 days. This period can be shorter if your Region began supporting these features within the last year. The role might have been used more than 400 days ago. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#access-advisor_tracking-period">Regions where data is tracked</a> in the <i>IAM User Guide</i>.</p>
-    pub role_last_used: ::std::option::Option<crate::types::RoleLastUsed>,
+    pub role_last_used: ::std::option::Option<super::super::types::RoleLastUsed>,
 }
 impl RoleDetail {
     /// <p>The path to the role. For more information about paths, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/Using_Identifiers.html">IAM identifiers</a> in the <i>IAM User Guide</i>.</p>
@@ -61,41 +61,41 @@
     /// <p>A list of instance profiles that contain this role.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.instance_profile_list.is_none()`.
-    pub fn instance_profile_list(&self) -> &[crate::types::InstanceProfile] {
+    pub fn instance_profile_list(&self) -> &[super::super::types::InstanceProfile] {
         self.instance_profile_list.as_deref().unwrap_or_default()
     }
     /// <p>A list of inline policies embedded in the role. These policies are the role's access (permissions) policies.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.role_policy_list.is_none()`.
-    pub fn role_policy_list(&self) -> &[crate::types::PolicyDetail] {
+    pub fn role_policy_list(&self) -> &[super::super::types::PolicyDetail] {
         self.role_policy_list.as_deref().unwrap_or_default()
     }
     /// <p>A list of managed policies attached to the role. These policies are the role's access (permissions) policies.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.attached_managed_policies.is_none()`.
-    pub fn attached_managed_policies(&self) -> &[crate::types::AttachedPolicy] {
+    pub fn attached_managed_policies(&self) -> &[super::super::types::AttachedPolicy] {
         self.attached_managed_policies.as_deref().unwrap_or_default()
     }
     /// <p>The ARN of the policy used to set the permissions boundary for the role.</p>
     /// <p>For more information about permissions boundaries, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html">Permissions boundaries for IAM identities </a> in the <i>IAM User Guide</i>.</p>
-    pub fn permissions_boundary(&self) -> ::std::option::Option<&crate::types::AttachedPermissionsBoundary> {
+    pub fn permissions_boundary(&self) -> ::std::option::Option<&super::super::types::AttachedPermissionsBoundary> {
         self.permissions_boundary.as_ref()
     }
     /// <p>A list of tags that are attached to the role. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
     /// <p>Contains information about the last time that an IAM role was used. This includes the date and time and the Region in which the role was last used. Activity is only reported for the trailing 400 days. This period can be shorter if your Region began supporting these features within the last year. The role might have been used more than 400 days ago. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#access-advisor_tracking-period">Regions where data is tracked</a> in the <i>IAM User Guide</i>.</p>
-    pub fn role_last_used(&self) -> ::std::option::Option<&crate::types::RoleLastUsed> {
+    pub fn role_last_used(&self) -> ::std::option::Option<&super::super::types::RoleLastUsed> {
         self.role_last_used.as_ref()
     }
 }
 impl RoleDetail {
     /// Creates a new builder-style object to manufacture [`RoleDetail`](crate::types::RoleDetail).
-    pub fn builder() -> crate::types::builders::RoleDetailBuilder {
-        crate::types::builders::RoleDetailBuilder::default()
+    pub fn builder() -> super::super::types::builders::RoleDetailBuilder {
+        super::super::types::builders::RoleDetailBuilder::default()
     }
 }

@@ -109,12 +109,12 @@
     pub(crate) arn: ::std::option::Option<::std::string::String>,
     pub(crate) create_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) assume_role_policy_document: ::std::option::Option<::std::string::String>,
-    pub(crate) instance_profile_list: ::std::option::Option<::std::vec::Vec<crate::types::InstanceProfile>>,
-    pub(crate) role_policy_list: ::std::option::Option<::std::vec::Vec<crate::types::PolicyDetail>>,
-    pub(crate) attached_managed_policies: ::std::option::Option<::std::vec::Vec<crate::types::AttachedPolicy>>,
-    pub(crate) permissions_boundary: ::std::option::Option<crate::types::AttachedPermissionsBoundary>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
-    pub(crate) role_last_used: ::std::option::Option<crate::types::RoleLastUsed>,
+    pub(crate) instance_profile_list: ::std::option::Option<::std::vec::Vec<super::super::types::InstanceProfile>>,
+    pub(crate) role_policy_list: ::std::option::Option<::std::vec::Vec<super::super::types::PolicyDetail>>,
+    pub(crate) attached_managed_policies: ::std::option::Option<::std::vec::Vec<super::super::types::AttachedPolicy>>,
+    pub(crate) permissions_boundary: ::std::option::Option<super::super::types::AttachedPermissionsBoundary>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::super::types::Tag>>,
+    pub(crate) role_last_used: ::std::option::Option<super::super::types::RoleLastUsed>,
 }
 impl RoleDetailBuilder {
     /// <p>The path to the role. For more information about paths, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/Using_Identifiers.html">IAM identifiers</a> in the <i>IAM User Guide</i>.</p>
@@ -209,7 +209,7 @@
     /// To override the contents of this collection use [`set_instance_profile_list`](Self::set_instance_profile_list).
     ///
     /// <p>A list of instance profiles that contain this role.</p>
-    pub fn instance_profile_list(mut self, input: crate::types::InstanceProfile) -> Self {
+    pub fn instance_profile_list(mut self, input: super::super::types::InstanceProfile) -> Self {
         let mut v = self.instance_profile_list.unwrap_or_default();
         v.push(input);
         self.instance_profile_list = ::std::option::Option::Some(v);
@@ -216,12 +216,12 @@
         self
     }
     /// <p>A list of instance profiles that contain this role.</p>
-    pub fn set_instance_profile_list(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::InstanceProfile>>) -> Self {
+    pub fn set_instance_profile_list(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::InstanceProfile>>) -> Self {
         self.instance_profile_list = input;
         self
     }
     /// <p>A list of instance profiles that contain this role.</p>
-    pub fn get_instance_profile_list(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::InstanceProfile>> {
+    pub fn get_instance_profile_list(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::InstanceProfile>> {
         &self.instance_profile_list
     }
     /// Appends an item to `role_policy_list`.
@@ -229,7 +229,7 @@
     /// To override the contents of this collection use [`set_role_policy_list`](Self::set_role_policy_list).
     ///
     /// <p>A list of inline policies embedded in the role. These policies are the role's access (permissions) policies.</p>
-    pub fn role_policy_list(mut self, input: crate::types::PolicyDetail) -> Self {
+    pub fn role_policy_list(mut self, input: super::super::types::PolicyDetail) -> Self {
         let mut v = self.role_policy_list.unwrap_or_default();
         v.push(input);
         self.role_policy_list = ::std::option::Option::Some(v);
@@ -236,12 +236,12 @@
         self
     }
     /// <p>A list of inline policies embedded in the role. These policies are the role's access (permissions) policies.</p>
-    pub fn set_role_policy_list(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::PolicyDetail>>) -> Self {
+    pub fn set_role_policy_list(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::PolicyDetail>>) -> Self {
         self.role_policy_list = input;
         self
     }
     /// <p>A list of inline policies embedded in the role. These policies are the role's access (permissions) policies.</p>
-    pub fn get_role_policy_list(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::PolicyDetail>> {
+    pub fn get_role_policy_list(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::PolicyDetail>> {
         &self.role_policy_list
     }
     /// Appends an item to `attached_managed_policies`.
@@ -249,7 +249,7 @@
     /// To override the contents of this collection use [`set_attached_managed_policies`](Self::set_attached_managed_policies).
     ///
     /// <p>A list of managed policies attached to the role. These policies are the role's access (permissions) policies.</p>
-    pub fn attached_managed_policies(mut self, input: crate::types::AttachedPolicy) -> Self {
+    pub fn attached_managed_policies(mut self, input: super::super::types::AttachedPolicy) -> Self {
         let mut v = self.attached_managed_policies.unwrap_or_default();
         v.push(input);
         self.attached_managed_policies = ::std::option::Option::Some(v);
@@ -256,29 +256,29 @@
         self
     }
     /// <p>A list of managed policies attached to the role. These policies are the role's access (permissions) policies.</p>
-    pub fn set_attached_managed_policies(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::AttachedPolicy>>) -> Self {
+    pub fn set_attached_managed_policies(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::AttachedPolicy>>) -> Self {
         self.attached_managed_policies = input;
         self
     }
     /// <p>A list of managed policies attached to the role. These policies are the role's access (permissions) policies.</p>
-    pub fn get_attached_managed_policies(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::AttachedPolicy>> {
+    pub fn get_attached_managed_policies(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::AttachedPolicy>> {
         &self.attached_managed_policies
     }
     /// <p>The ARN of the policy used to set the permissions boundary for the role.</p>
     /// <p>For more information about permissions boundaries, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html">Permissions boundaries for IAM identities </a> in the <i>IAM User Guide</i>.</p>
-    pub fn permissions_boundary(mut self, input: crate::types::AttachedPermissionsBoundary) -> Self {
+    pub fn permissions_boundary(mut self, input: super::super::types::AttachedPermissionsBoundary) -> Self {
         self.permissions_boundary = ::std::option::Option::Some(input);
         self
     }
     /// <p>The ARN of the policy used to set the permissions boundary for the role.</p>
     /// <p>For more information about permissions boundaries, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html">Permissions boundaries for IAM identities </a> in the <i>IAM User Guide</i>.</p>
-    pub fn set_permissions_boundary(mut self, input: ::std::option::Option<crate::types::AttachedPermissionsBoundary>) -> Self {
+    pub fn set_permissions_boundary(mut self, input: ::std::option::Option<super::super::types::AttachedPermissionsBoundary>) -> Self {
         self.permissions_boundary = input;
         self
     }
     /// <p>The ARN of the policy used to set the permissions boundary for the role.</p>
     /// <p>For more information about permissions boundaries, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html">Permissions boundaries for IAM identities </a> in the <i>IAM User Guide</i>.</p>
-    pub fn get_permissions_boundary(&self) -> &::std::option::Option<crate::types::AttachedPermissionsBoundary> {
+    pub fn get_permissions_boundary(&self) -> &::std::option::Option<super::super::types::AttachedPermissionsBoundary> {
         &self.permissions_boundary
     }
     /// Appends an item to `tags`.
@@ -286,7 +286,7 @@
     /// To override the contents of this collection use [`set_tags`](Self::set_tags).
     ///
     /// <p>A list of tags that are attached to the role. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -293,31 +293,31 @@
         self
     }
     /// <p>A list of tags that are attached to the role. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
     /// <p>A list of tags that are attached to the role. For more information about tagging, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html">Tagging IAM resources</a> in the <i>IAM User Guide</i>.</p>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::Tag>> {
         &self.tags
     }
     /// <p>Contains information about the last time that an IAM role was used. This includes the date and time and the Region in which the role was last used. Activity is only reported for the trailing 400 days. This period can be shorter if your Region began supporting these features within the last year. The role might have been used more than 400 days ago. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#access-advisor_tracking-period">Regions where data is tracked</a> in the <i>IAM User Guide</i>.</p>
-    pub fn role_last_used(mut self, input: crate::types::RoleLastUsed) -> Self {
+    pub fn role_last_used(mut self, input: super::super::types::RoleLastUsed) -> Self {
         self.role_last_used = ::std::option::Option::Some(input);
         self
     }
     /// <p>Contains information about the last time that an IAM role was used. This includes the date and time and the Region in which the role was last used. Activity is only reported for the trailing 400 days. This period can be shorter if your Region began supporting these features within the last year. The role might have been used more than 400 days ago. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#access-advisor_tracking-period">Regions where data is tracked</a> in the <i>IAM User Guide</i>.</p>
-    pub fn set_role_last_used(mut self, input: ::std::option::Option<crate::types::RoleLastUsed>) -> Self {
+    pub fn set_role_last_used(mut self, input: ::std::option::Option<super::super::types::RoleLastUsed>) -> Self {
         self.role_last_used = input;
         self
     }
     /// <p>Contains information about the last time that an IAM role was used. This includes the date and time and the Region in which the role was last used. Activity is only reported for the trailing 400 days. This period can be shorter if your Region began supporting these features within the last year. The role might have been used more than 400 days ago. For more information, see <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#access-advisor_tracking-period">Regions where data is tracked</a> in the <i>IAM User Guide</i>.</p>
-    pub fn get_role_last_used(&self) -> &::std::option::Option<crate::types::RoleLastUsed> {
+    pub fn get_role_last_used(&self) -> &::std::option::Option<super::super::types::RoleLastUsed> {
         &self.role_last_used
     }
     /// Consumes the builder and constructs a [`RoleDetail`](crate::types::RoleDetail).
-    pub fn build(self) -> crate::types::RoleDetail {
-        crate::types::RoleDetail {
+    pub fn build(self) -> super::super::types::RoleDetail {
+        super::super::types::RoleDetail {
             path: self.path,
             role_name: self.role_name,
             role_id: self.role_id,
