--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl SamlPrivateKey {
     /// Creates a new builder-style object to manufacture [`SamlPrivateKey`](crate::types::SamlPrivateKey).
-    pub fn builder() -> crate::types::builders::SamlPrivateKeyBuilder {
-        crate::types::builders::SamlPrivateKeyBuilder::default()
+    pub fn builder() -> super::super::types::builders::SamlPrivateKeyBuilder {
+        super::super::types::builders::SamlPrivateKeyBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
         &self.timestamp
     }
     /// Consumes the builder and constructs a [`SamlPrivateKey`](crate::types::SamlPrivateKey).
-    pub fn build(self) -> crate::types::SamlPrivateKey {
-        crate::types::SamlPrivateKey {
+    pub fn build(self) -> super::super::types::SamlPrivateKey {
+        super::super::types::SamlPrivateKey {
             key_id: self.key_id,
             timestamp: self.timestamp,
         }
