--- original
+++ modified
@@ -50,8 +50,8 @@
 }
 impl ServerCertificateMetadata {
     /// Creates a new builder-style object to manufacture [`ServerCertificateMetadata`](crate::types::ServerCertificateMetadata).
-    pub fn builder() -> crate::types::builders::ServerCertificateMetadataBuilder {
-        crate::types::builders::ServerCertificateMetadataBuilder::default()
+    pub fn builder() -> super::super::types::builders::ServerCertificateMetadataBuilder {
+        super::super::types::builders::ServerCertificateMetadataBuilder::default()
     }
 }

@@ -161,8 +161,8 @@
     /// - [`server_certificate_name`](crate::types::builders::ServerCertificateMetadataBuilder::server_certificate_name)
     /// - [`server_certificate_id`](crate::types::builders::ServerCertificateMetadataBuilder::server_certificate_id)
     /// - [`arn`](crate::types::builders::ServerCertificateMetadataBuilder::arn)
-    pub fn build(self) -> ::std::result::Result<crate::types::ServerCertificateMetadata, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ServerCertificateMetadata {
+    pub fn build(self) -> ::std::result::Result<super::super::types::ServerCertificateMetadata, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::ServerCertificateMetadata {
             path: self.path.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "path",
