--- original
+++ modified
@@ -24,7 +24,7 @@
     pub total_authenticated_entities: ::std::option::Option<i32>,
     /// <p>An object that contains details about the most recent attempt to access a tracked action within the service.</p>
     /// <p>This field is null if there no tracked actions or if the principal did not use the tracked actions within the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#service-last-accessed-reporting-period">tracking period</a>. This field is also null if the report was generated at the service level and not the action level. For more information, see the <code>Granularity</code> field in <a href="https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html">GenerateServiceLastAccessedDetails</a>.</p>
-    pub tracked_actions_last_accessed: ::std::option::Option<::std::vec::Vec<crate::types::TrackedActionLastAccessed>>,
+    pub tracked_actions_last_accessed: ::std::option::Option<::std::vec::Vec<super::super::types::TrackedActionLastAccessed>>,
 }
 impl ServiceLastAccessed {
     /// <p>The name of the service in which access was attempted.</p>
@@ -62,14 +62,14 @@
     /// <p>This field is null if there no tracked actions or if the principal did not use the tracked actions within the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#service-last-accessed-reporting-period">tracking period</a>. This field is also null if the report was generated at the service level and not the action level. For more information, see the <code>Granularity</code> field in <a href="https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html">GenerateServiceLastAccessedDetails</a>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tracked_actions_last_accessed.is_none()`.
-    pub fn tracked_actions_last_accessed(&self) -> &[crate::types::TrackedActionLastAccessed] {
+    pub fn tracked_actions_last_accessed(&self) -> &[super::super::types::TrackedActionLastAccessed] {
         self.tracked_actions_last_accessed.as_deref().unwrap_or_default()
     }
 }
 impl ServiceLastAccessed {
     /// Creates a new builder-style object to manufacture [`ServiceLastAccessed`](crate::types::ServiceLastAccessed).
-    pub fn builder() -> crate::types::builders::ServiceLastAccessedBuilder {
-        crate::types::builders::ServiceLastAccessedBuilder::default()
+    pub fn builder() -> super::super::types::builders::ServiceLastAccessedBuilder {
+        super::super::types::builders::ServiceLastAccessedBuilder::default()
     }
 }

@@ -83,7 +83,7 @@
     pub(crate) last_authenticated_entity: ::std::option::Option<::std::string::String>,
     pub(crate) last_authenticated_region: ::std::option::Option<::std::string::String>,
     pub(crate) total_authenticated_entities: ::std::option::Option<i32>,
-    pub(crate) tracked_actions_last_accessed: ::std::option::Option<::std::vec::Vec<crate::types::TrackedActionLastAccessed>>,
+    pub(crate) tracked_actions_last_accessed: ::std::option::Option<::std::vec::Vec<super::super::types::TrackedActionLastAccessed>>,
 }
 impl ServiceLastAccessedBuilder {
     /// <p>The name of the service in which access was attempted.</p>
@@ -193,7 +193,7 @@
     ///
     /// <p>An object that contains details about the most recent attempt to access a tracked action within the service.</p>
     /// <p>This field is null if there no tracked actions or if the principal did not use the tracked actions within the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#service-last-accessed-reporting-period">tracking period</a>. This field is also null if the report was generated at the service level and not the action level. For more information, see the <code>Granularity</code> field in <a href="https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html">GenerateServiceLastAccessedDetails</a>.</p>
-    pub fn tracked_actions_last_accessed(mut self, input: crate::types::TrackedActionLastAccessed) -> Self {
+    pub fn tracked_actions_last_accessed(mut self, input: super::super::types::TrackedActionLastAccessed) -> Self {
         let mut v = self.tracked_actions_last_accessed.unwrap_or_default();
         v.push(input);
         self.tracked_actions_last_accessed = ::std::option::Option::Some(v);
@@ -203,7 +203,7 @@
     /// <p>This field is null if there no tracked actions or if the principal did not use the tracked actions within the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#service-last-accessed-reporting-period">tracking period</a>. This field is also null if the report was generated at the service level and not the action level. For more information, see the <code>Granularity</code> field in <a href="https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html">GenerateServiceLastAccessedDetails</a>.</p>
     pub fn set_tracked_actions_last_accessed(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::TrackedActionLastAccessed>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::types::TrackedActionLastAccessed>>,
     ) -> Self {
         self.tracked_actions_last_accessed = input;
         self
@@ -210,7 +210,7 @@
     }
     /// <p>An object that contains details about the most recent attempt to access a tracked action within the service.</p>
     /// <p>This field is null if there no tracked actions or if the principal did not use the tracked actions within the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_access-advisor.html#service-last-accessed-reporting-period">tracking period</a>. This field is also null if the report was generated at the service level and not the action level. For more information, see the <code>Granularity</code> field in <a href="https://docs.aws.amazon.com/IAM/latest/APIReference/API_GenerateServiceLastAccessedDetails.html">GenerateServiceLastAccessedDetails</a>.</p>
-    pub fn get_tracked_actions_last_accessed(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::TrackedActionLastAccessed>> {
+    pub fn get_tracked_actions_last_accessed(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::TrackedActionLastAccessed>> {
         &self.tracked_actions_last_accessed
     }
     /// Consumes the builder and constructs a [`ServiceLastAccessed`](crate::types::ServiceLastAccessed).
@@ -217,8 +217,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`service_name`](crate::types::builders::ServiceLastAccessedBuilder::service_name)
     /// - [`service_namespace`](crate::types::builders::ServiceLastAccessedBuilder::service_namespace)
-    pub fn build(self) -> ::std::result::Result<crate::types::ServiceLastAccessed, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ServiceLastAccessed {
+    pub fn build(self) -> ::std::result::Result<super::super::types::ServiceLastAccessed, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::ServiceLastAccessed {
             service_name: self.service_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "service_name",
