--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl SourceRoleTemplate {
     /// Creates a new builder-style object to manufacture [`SourceRoleTemplate`](crate::types::SourceRoleTemplate).
-    pub fn builder() -> crate::types::builders::SourceRoleTemplateBuilder {
-        crate::types::builders::SourceRoleTemplateBuilder::default()
+    pub fn builder() -> super::types::builders::SourceRoleTemplateBuilder {
+        super::types::builders::SourceRoleTemplateBuilder::default()
     }
 }

@@ -69,8 +69,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`template_arn`](crate::types::builders::SourceRoleTemplateBuilder::template_arn)
     /// - [`template_minor_version`](crate::types::builders::SourceRoleTemplateBuilder::template_minor_version)
-    pub fn build(self) -> ::std::result::Result<crate::types::SourceRoleTemplate, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SourceRoleTemplate {
+    pub fn build(self) -> ::std::result::Result<super::types::SourceRoleTemplate, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SourceRoleTemplate {
             template_arn: self.template_arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "template_arn",
