--- original
+++ modified
@@ -14,7 +14,7 @@
     /// <p>The SSH public key.</p>
     pub ssh_public_key_body: ::std::string::String,
     /// <p>The status of the SSH public key. <code>Active</code> means that the key can be used for authentication with an CodeCommit repository. <code>Inactive</code> means that the key cannot be used.</p>
-    pub status: crate::types::StatusType,
+    pub status: super::types::StatusType,
     /// <p>The date and time, in <a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>, when the SSH public key was uploaded.</p>
     pub upload_date: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
@@ -40,7 +40,7 @@
         self.ssh_public_key_body.deref()
     }
     /// <p>The status of the SSH public key. <code>Active</code> means that the key can be used for authentication with an CodeCommit repository. <code>Inactive</code> means that the key cannot be used.</p>
-    pub fn status(&self) -> &crate::types::StatusType {
+    pub fn status(&self) -> &super::types::StatusType {
         &self.status
     }
     /// <p>The date and time, in <a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>, when the SSH public key was uploaded.</p>
@@ -50,8 +50,8 @@
 }
 impl SshPublicKey {
     /// Creates a new builder-style object to manufacture [`SshPublicKey`](crate::types::SshPublicKey).
-    pub fn builder() -> crate::types::builders::SshPublicKeyBuilder {
-        crate::types::builders::SshPublicKeyBuilder::default()
+    pub fn builder() -> super::types::builders::SshPublicKeyBuilder {
+        super::types::builders::SshPublicKeyBuilder::default()
     }
 }

@@ -63,7 +63,7 @@
     pub(crate) ssh_public_key_id: ::std::option::Option<::std::string::String>,
     pub(crate) fingerprint: ::std::option::Option<::std::string::String>,
     pub(crate) ssh_public_key_body: ::std::option::Option<::std::string::String>,
-    pub(crate) status: ::std::option::Option<crate::types::StatusType>,
+    pub(crate) status: ::std::option::Option<super::types::StatusType>,
     pub(crate) upload_date: ::std::option::Option<::aws_smithy_types::DateTime>,
 }
 impl SshPublicKeyBuilder {
@@ -129,17 +129,17 @@
     }
     /// <p>The status of the SSH public key. <code>Active</code> means that the key can be used for authentication with an CodeCommit repository. <code>Inactive</code> means that the key cannot be used.</p>
     /// This field is required.
-    pub fn status(mut self, input: crate::types::StatusType) -> Self {
+    pub fn status(mut self, input: super::types::StatusType) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The status of the SSH public key. <code>Active</code> means that the key can be used for authentication with an CodeCommit repository. <code>Inactive</code> means that the key cannot be used.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::StatusType>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::StatusType>) -> Self {
         self.status = input;
         self
     }
     /// <p>The status of the SSH public key. <code>Active</code> means that the key can be used for authentication with an CodeCommit repository. <code>Inactive</code> means that the key cannot be used.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::StatusType> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::StatusType> {
         &self.status
     }
     /// <p>The date and time, in <a href="http://www.iso.org/iso/iso8601">ISO 8601 date-time format</a>, when the SSH public key was uploaded.</p>
@@ -163,8 +163,8 @@
     /// - [`fingerprint`](crate::types::builders::SshPublicKeyBuilder::fingerprint)
     /// - [`ssh_public_key_body`](crate::types::builders::SshPublicKeyBuilder::ssh_public_key_body)
     /// - [`status`](crate::types::builders::SshPublicKeyBuilder::status)
-    pub fn build(self) -> ::std::result::Result<crate::types::SshPublicKey, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SshPublicKey {
+    pub fn build(self) -> ::std::result::Result<super::types::SshPublicKey, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SshPublicKey {
             user_name: self.user_name.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "user_name",
