--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The region authorized to collect aggregated data.</p>
     pub authorized_aws_region: ::std::option::Option<::std::string::String>,
     /// <p>An array of tag object.</p>
-    pub tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>,
 }
 impl PutAggregationAuthorizationInput {
     /// <p>The 12-digit account ID of the account authorized to aggregate data.</p>
@@ -22,14 +22,14 @@
     /// <p>An array of tag object.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.tags.is_none()`.
-    pub fn tags(&self) -> &[crate::types::Tag] {
+    pub fn tags(&self) -> &[super::super::super::types::Tag] {
         self.tags.as_deref().unwrap_or_default()
     }
 }
 impl PutAggregationAuthorizationInput {
     /// Creates a new builder-style object to manufacture [`PutAggregationAuthorizationInput`](crate::operation::put_aggregation_authorization::PutAggregationAuthorizationInput).
-    pub fn builder() -> crate::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationInputBuilder {
-        crate::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationInputBuilder {
+        super::super::super::operation::put_aggregation_authorization::builders::PutAggregationAuthorizationInputBuilder::default()
     }
 }

@@ -39,7 +39,7 @@
 pub struct PutAggregationAuthorizationInputBuilder {
     pub(crate) authorized_account_id: ::std::option::Option<::std::string::String>,
     pub(crate) authorized_aws_region: ::std::option::Option<::std::string::String>,
-    pub(crate) tags: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>,
+    pub(crate) tags: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>,
 }
 impl PutAggregationAuthorizationInputBuilder {
     /// <p>The 12-digit account ID of the account authorized to aggregate data.</p>
@@ -77,7 +77,7 @@
     /// To override the contents of this collection use [`set_tags`](Self::set_tags).
     ///
     /// <p>An array of tag object.</p>
-    pub fn tags(mut self, input: crate::types::Tag) -> Self {
+    pub fn tags(mut self, input: super::super::super::types::Tag) -> Self {
         let mut v = self.tags.unwrap_or_default();
         v.push(input);
         self.tags = ::std::option::Option::Some(v);
@@ -84,12 +84,12 @@
         self
     }
     /// <p>An array of tag object.</p>
-    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tag>>) -> Self {
+    pub fn set_tags(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>>) -> Self {
         self.tags = input;
         self
     }
     /// <p>An array of tag object.</p>
-    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tag>> {
+    pub fn get_tags(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::Tag>> {
         &self.tags
     }
     /// Consumes the builder and constructs a [`PutAggregationAuthorizationInput`](crate::operation::put_aggregation_authorization::PutAggregationAuthorizationInput).
@@ -96,10 +96,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_aggregation_authorization::PutAggregationAuthorizationInput,
+        super::super::super::operation::put_aggregation_authorization::PutAggregationAuthorizationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_aggregation_authorization::PutAggregationAuthorizationInput {
+        ::std::result::Result::Ok(super::super::super::operation::put_aggregation_authorization::PutAggregationAuthorizationInput {
             authorized_account_id: self.authorized_account_id,
             authorized_aws_region: self.authorized_aws_region,
             tags: self.tags,
