--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteEmailIdentityInput {
     /// Creates a new builder-style object to manufacture [`DeleteEmailIdentityInput`](crate::operation::delete_email_identity::DeleteEmailIdentityInput).
-    pub fn builder() -> crate::operation::delete_email_identity::builders::DeleteEmailIdentityInputBuilder {
-        crate::operation::delete_email_identity::builders::DeleteEmailIdentityInputBuilder::default()
+    pub fn builder() -> super::operation::delete_email_identity::builders::DeleteEmailIdentityInputBuilder {
+        super::operation::delete_email_identity::builders::DeleteEmailIdentityInputBuilder::default()
     }
 }

@@ -45,9 +45,9 @@
     /// Consumes the builder and constructs a [`DeleteEmailIdentityInput`](crate::operation::delete_email_identity::DeleteEmailIdentityInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_email_identity::DeleteEmailIdentityInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::delete_email_identity::DeleteEmailIdentityInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_email_identity::DeleteEmailIdentityInput {
+        ::std::result::Result::Ok(super::operation::delete_email_identity::DeleteEmailIdentityInput {
             email_identity: self.email_identity,
         })
     }
