--- original
+++ modified
@@ -26,8 +26,8 @@
 }
 impl DescribeRetentionConfigurationsInput {
     /// Creates a new builder-style object to manufacture [`DescribeRetentionConfigurationsInput`](crate::operation::describe_retention_configurations::DescribeRetentionConfigurationsInput).
-    pub fn builder() -> crate::operation::describe_retention_configurations::builders::DescribeRetentionConfigurationsInputBuilder {
-        crate::operation::describe_retention_configurations::builders::DescribeRetentionConfigurationsInputBuilder::default()
+    pub fn builder() -> super::operation::describe_retention_configurations::builders::DescribeRetentionConfigurationsInputBuilder {
+        super::operation::describe_retention_configurations::builders::DescribeRetentionConfigurationsInputBuilder::default()
     }
 }

@@ -83,11 +83,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::describe_retention_configurations::DescribeRetentionConfigurationsInput,
+        super::operation::describe_retention_configurations::DescribeRetentionConfigurationsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::describe_retention_configurations::DescribeRetentionConfigurationsInput {
+            super::operation::describe_retention_configurations::DescribeRetentionConfigurationsInput {
                 retention_configuration_names: self.retention_configuration_names,
                 next_token: self.next_token,
             },
