--- original
+++ modified
@@ -29,8 +29,8 @@
 }
 impl AdminDisableUserInput {
     /// Creates a new builder-style object to manufacture [`AdminDisableUserInput`](crate::operation::admin_disable_user::AdminDisableUserInput).
-    pub fn builder() -> crate::operation::admin_disable_user::builders::AdminDisableUserInputBuilder {
-        crate::operation::admin_disable_user::builders::AdminDisableUserInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::admin_disable_user::builders::AdminDisableUserInputBuilder {
+        super::super::super::operation::admin_disable_user::builders::AdminDisableUserInputBuilder::default()
     }
 }

@@ -75,8 +75,8 @@
     /// Consumes the builder and constructs a [`AdminDisableUserInput`](crate::operation::admin_disable_user::AdminDisableUserInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::admin_disable_user::AdminDisableUserInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::admin_disable_user::AdminDisableUserInput {
+    ) -> ::std::result::Result<super::super::super::operation::admin_disable_user::AdminDisableUserInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::admin_disable_user::AdminDisableUserInput {
             user_pool_id: self.user_pool_id,
             username: self.username,
         })
