--- original
+++ modified
@@ -5,8 +5,8 @@
 pub struct GetAccountPropertiesInput {}
 impl GetAccountPropertiesInput {
     /// Creates a new builder-style object to manufacture [`GetAccountPropertiesInput`](crate::operation::get_account_properties::GetAccountPropertiesInput).
-    pub fn builder() -> crate::operation::get_account_properties::builders::GetAccountPropertiesInputBuilder {
-        crate::operation::get_account_properties::builders::GetAccountPropertiesInputBuilder::default()
+    pub fn builder() -> super::operation::get_account_properties::builders::GetAccountPropertiesInputBuilder {
+        super::operation::get_account_properties::builders::GetAccountPropertiesInputBuilder::default()
     }
 }

@@ -18,8 +18,8 @@
     /// Consumes the builder and constructs a [`GetAccountPropertiesInput`](crate::operation::get_account_properties::GetAccountPropertiesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_account_properties::GetAccountPropertiesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::get_account_properties::GetAccountPropertiesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_account_properties::GetAccountPropertiesInput {})
+        ::std::result::Result::Ok(super::operation::get_account_properties::GetAccountPropertiesInput {})
     }
 }
