--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetLogAnomalyDetectorInput {
     /// Creates a new builder-style object to manufacture [`GetLogAnomalyDetectorInput`](crate::operation::get_log_anomaly_detector::GetLogAnomalyDetectorInput).
-    pub fn builder() -> crate::operation::get_log_anomaly_detector::builders::GetLogAnomalyDetectorInputBuilder {
-        crate::operation::get_log_anomaly_detector::builders::GetLogAnomalyDetectorInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_log_anomaly_detector::builders::GetLogAnomalyDetectorInputBuilder {
+        super::super::super::operation::get_log_anomaly_detector::builders::GetLogAnomalyDetectorInputBuilder::default()
     }
 }

@@ -44,9 +44,9 @@
     /// Consumes the builder and constructs a [`GetLogAnomalyDetectorInput`](crate::operation::get_log_anomaly_detector::GetLogAnomalyDetectorInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_log_anomaly_detector::GetLogAnomalyDetectorInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::get_log_anomaly_detector::GetLogAnomalyDetectorInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::get_log_anomaly_detector::GetLogAnomalyDetectorInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_log_anomaly_detector::GetLogAnomalyDetectorInput {
             anomaly_detector_arn: self.anomaly_detector_arn,
         })
     }
