--- original
+++ modified
@@ -4,7 +4,7 @@
     /// Resolve a priority list of auth scheme options with the given parameters
     fn resolve_auth_scheme<'a>(
         &'a self,
-        params: &'a crate::config::auth::Params,
+        params: &'a super::super::config::auth::Params,
         cfg: &'a ::aws_smithy_types::config_bag::ConfigBag,
         runtime_components: &'a ::aws_smithy_runtime_api::client::runtime_components::RuntimeComponents,
     ) -> ::aws_smithy_runtime_api::client::auth::AuthSchemeOptionsFuture<'a>;
@@ -30,7 +30,7 @@
         cfg: &'a ::aws_smithy_types::config_bag::ConfigBag,
         runtime_components: &'a ::aws_smithy_runtime_api::client::runtime_components::RuntimeComponents,
     ) -> ::aws_smithy_runtime_api::client::auth::AuthSchemeOptionsFuture<'a> {
-        match params.get::<crate::config::auth::Params>() {
+        match params.get::<super::super::config::auth::Params>() {
             ::std::option::Option::Some(concrete_params) => self.0.resolve_auth_scheme(concrete_params, cfg, runtime_components),
             ::std::option::Option::None => ::aws_smithy_runtime_api::client::auth::AuthSchemeOptionsFuture::ready(::std::result::Result::Err(
                 "params of expected type was not present".into(),
@@ -64,10 +64,10 @@
     }
 }

-impl crate::config::auth::ResolveAuthScheme for DefaultAuthSchemeResolver {
+impl super::super::config::auth::ResolveAuthScheme for DefaultAuthSchemeResolver {
     fn resolve_auth_scheme<'a>(
         &'a self,
-        params: &'a crate::config::auth::Params,
+        params: &'a super::super::config::auth::Params,
         _cfg: &'a ::aws_smithy_types::config_bag::ConfigBag,
         _runtime_components: &'a ::aws_smithy_runtime_api::client::runtime_components::RuntimeComponents,
     ) -> ::aws_smithy_runtime_api::client::auth::AuthSchemeOptionsFuture<'a> {
@@ -91,8 +91,8 @@
 }
 impl Params {
     /// Create a builder for [`Params`]
-    pub fn builder() -> crate::config::auth::ParamsBuilder {
-        crate::config::auth::ParamsBuilder::default()
+    pub fn builder() -> super::super::config::auth::ParamsBuilder {
+        super::super::config::auth::ParamsBuilder::default()
     }

     /// Return the operation name for [`Params`]
@@ -121,8 +121,8 @@
     ///
     /// Return [`BuildError`] if any of the required fields are unset.
     ///
-    pub fn build(self) -> ::std::result::Result<crate::config::auth::Params, crate::config::auth::BuildError> {
-        ::std::result::Result::Ok(crate::config::auth::Params {
+    pub fn build(self) -> ::std::result::Result<super::super::config::auth::Params, super::super::config::auth::BuildError> {
+        ::std::result::Result::Ok(super::super::config::auth::Params {
             operation_name: self.operation_name.ok_or_else(|| BuildError::missing("operation_name"))?,
         })
     }
