--- original
+++ modified
@@ -11,12 +11,12 @@
     pub package_group: ::std::option::Option<::std::string::String>,
     /// <p>The origin configuration settings that determine how package versions can enter repositories.</p>
     pub restrictions: ::std::option::Option<
-        ::std::collections::HashMap<crate::types::PackageGroupOriginRestrictionType, crate::types::PackageGroupOriginRestrictionMode>,
+        ::std::collections::HashMap<super::super::super::types::PackageGroupOriginRestrictionType, super::super::super::types::PackageGroupOriginRestrictionMode>,
     >,
     /// <p>The repository name and restrictions to add to the allowed repository list of the specified package group.</p>
-    pub add_allowed_repositories: ::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>>,
+    pub add_allowed_repositories: ::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>>,
     /// <p>The repository name and restrictions to remove from the allowed repository list of the specified package group.</p>
-    pub remove_allowed_repositories: ::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>>,
+    pub remove_allowed_repositories: ::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>>,
 }
 impl UpdatePackageGroupOriginConfigurationInput {
     /// <p>The name of the domain which contains the package group for which to update the origin configuration.</p>
@@ -35,7 +35,7 @@
     pub fn restrictions(
         &self,
     ) -> ::std::option::Option<
-        &::std::collections::HashMap<crate::types::PackageGroupOriginRestrictionType, crate::types::PackageGroupOriginRestrictionMode>,
+        &::std::collections::HashMap<super::super::super::types::PackageGroupOriginRestrictionType, super::super::super::types::PackageGroupOriginRestrictionMode>,
     > {
         self.restrictions.as_ref()
     }
@@ -42,20 +42,20 @@
     /// <p>The repository name and restrictions to add to the allowed repository list of the specified package group.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.add_allowed_repositories.is_none()`.
-    pub fn add_allowed_repositories(&self) -> &[crate::types::PackageGroupAllowedRepository] {
+    pub fn add_allowed_repositories(&self) -> &[super::super::super::types::PackageGroupAllowedRepository] {
         self.add_allowed_repositories.as_deref().unwrap_or_default()
     }
     /// <p>The repository name and restrictions to remove from the allowed repository list of the specified package group.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.remove_allowed_repositories.is_none()`.
-    pub fn remove_allowed_repositories(&self) -> &[crate::types::PackageGroupAllowedRepository] {
+    pub fn remove_allowed_repositories(&self) -> &[super::super::super::types::PackageGroupAllowedRepository] {
         self.remove_allowed_repositories.as_deref().unwrap_or_default()
     }
 }
 impl UpdatePackageGroupOriginConfigurationInput {
     /// Creates a new builder-style object to manufacture [`UpdatePackageGroupOriginConfigurationInput`](crate::operation::update_package_group_origin_configuration::UpdatePackageGroupOriginConfigurationInput).
-    pub fn builder() -> crate::operation::update_package_group_origin_configuration::builders::UpdatePackageGroupOriginConfigurationInputBuilder {
-        crate::operation::update_package_group_origin_configuration::builders::UpdatePackageGroupOriginConfigurationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_package_group_origin_configuration::builders::UpdatePackageGroupOriginConfigurationInputBuilder {
+        super::super::super::operation::update_package_group_origin_configuration::builders::UpdatePackageGroupOriginConfigurationInputBuilder::default()
     }
 }

@@ -67,10 +67,10 @@
     pub(crate) domain_owner: ::std::option::Option<::std::string::String>,
     pub(crate) package_group: ::std::option::Option<::std::string::String>,
     pub(crate) restrictions: ::std::option::Option<
-        ::std::collections::HashMap<crate::types::PackageGroupOriginRestrictionType, crate::types::PackageGroupOriginRestrictionMode>,
+        ::std::collections::HashMap<super::super::super::types::PackageGroupOriginRestrictionType, super::super::super::types::PackageGroupOriginRestrictionMode>,
     >,
-    pub(crate) add_allowed_repositories: ::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>>,
-    pub(crate) remove_allowed_repositories: ::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>>,
+    pub(crate) add_allowed_repositories: ::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>>,
+    pub(crate) remove_allowed_repositories: ::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>>,
 }
 impl UpdatePackageGroupOriginConfigurationInputBuilder {
     /// <p>The name of the domain which contains the package group for which to update the origin configuration.</p>
@@ -122,7 +122,7 @@
     /// To override the contents of this collection use [`set_restrictions`](Self::set_restrictions).
     ///
     /// <p>The origin configuration settings that determine how package versions can enter repositories.</p>
-    pub fn restrictions(mut self, k: crate::types::PackageGroupOriginRestrictionType, v: crate::types::PackageGroupOriginRestrictionMode) -> Self {
+    pub fn restrictions(mut self, k: super::super::super::types::PackageGroupOriginRestrictionType, v: super::super::super::types::PackageGroupOriginRestrictionMode) -> Self {
         let mut hash_map = self.restrictions.unwrap_or_default();
         hash_map.insert(k, v);
         self.restrictions = ::std::option::Option::Some(hash_map);
@@ -132,7 +132,7 @@
     pub fn set_restrictions(
         mut self,
         input: ::std::option::Option<
-            ::std::collections::HashMap<crate::types::PackageGroupOriginRestrictionType, crate::types::PackageGroupOriginRestrictionMode>,
+            ::std::collections::HashMap<super::super::super::types::PackageGroupOriginRestrictionType, super::super::super::types::PackageGroupOriginRestrictionMode>,
         >,
     ) -> Self {
         self.restrictions = input;
@@ -142,7 +142,7 @@
     pub fn get_restrictions(
         &self,
     ) -> &::std::option::Option<
-        ::std::collections::HashMap<crate::types::PackageGroupOriginRestrictionType, crate::types::PackageGroupOriginRestrictionMode>,
+        ::std::collections::HashMap<super::super::super::types::PackageGroupOriginRestrictionType, super::super::super::types::PackageGroupOriginRestrictionMode>,
     > {
         &self.restrictions
     }
@@ -151,7 +151,7 @@
     /// To override the contents of this collection use [`set_add_allowed_repositories`](Self::set_add_allowed_repositories).
     ///
     /// <p>The repository name and restrictions to add to the allowed repository list of the specified package group.</p>
-    pub fn add_allowed_repositories(mut self, input: crate::types::PackageGroupAllowedRepository) -> Self {
+    pub fn add_allowed_repositories(mut self, input: super::super::super::types::PackageGroupAllowedRepository) -> Self {
         let mut v = self.add_allowed_repositories.unwrap_or_default();
         v.push(input);
         self.add_allowed_repositories = ::std::option::Option::Some(v);
@@ -160,13 +160,13 @@
     /// <p>The repository name and restrictions to add to the allowed repository list of the specified package group.</p>
     pub fn set_add_allowed_repositories(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>>,
     ) -> Self {
         self.add_allowed_repositories = input;
         self
     }
     /// <p>The repository name and restrictions to add to the allowed repository list of the specified package group.</p>
-    pub fn get_add_allowed_repositories(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>> {
+    pub fn get_add_allowed_repositories(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>> {
         &self.add_allowed_repositories
     }
     /// Appends an item to `remove_allowed_repositories`.
@@ -174,7 +174,7 @@
     /// To override the contents of this collection use [`set_remove_allowed_repositories`](Self::set_remove_allowed_repositories).
     ///
     /// <p>The repository name and restrictions to remove from the allowed repository list of the specified package group.</p>
-    pub fn remove_allowed_repositories(mut self, input: crate::types::PackageGroupAllowedRepository) -> Self {
+    pub fn remove_allowed_repositories(mut self, input: super::super::super::types::PackageGroupAllowedRepository) -> Self {
         let mut v = self.remove_allowed_repositories.unwrap_or_default();
         v.push(input);
         self.remove_allowed_repositories = ::std::option::Option::Some(v);
@@ -183,13 +183,13 @@
     /// <p>The repository name and restrictions to remove from the allowed repository list of the specified package group.</p>
     pub fn set_remove_allowed_repositories(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>>,
+        input: ::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>>,
     ) -> Self {
         self.remove_allowed_repositories = input;
         self
     }
     /// <p>The repository name and restrictions to remove from the allowed repository list of the specified package group.</p>
-    pub fn get_remove_allowed_repositories(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::PackageGroupAllowedRepository>> {
+    pub fn get_remove_allowed_repositories(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::PackageGroupAllowedRepository>> {
         &self.remove_allowed_repositories
     }
     /// Consumes the builder and constructs a [`UpdatePackageGroupOriginConfigurationInput`](crate::operation::update_package_group_origin_configuration::UpdatePackageGroupOriginConfigurationInput).
@@ -196,11 +196,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_package_group_origin_configuration::UpdatePackageGroupOriginConfigurationInput,
+        super::super::super::operation::update_package_group_origin_configuration::UpdatePackageGroupOriginConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::update_package_group_origin_configuration::UpdatePackageGroupOriginConfigurationInput {
+            super::super::super::operation::update_package_group_origin_configuration::UpdatePackageGroupOriginConfigurationInput {
                 domain: self.domain,
                 domain_owner: self.domain_owner,
                 package_group: self.package_group,
