--- original
+++ modified
@@ -46,7 +46,7 @@
     }
 }
 impl ::std::error::Error for InsufficientPermissionsException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::InsufficientPermissionsException {
+impl ::aws_types::request_id::RequestId for super::super::super::types::error::InsufficientPermissionsException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -59,8 +59,8 @@
 }
 impl InsufficientPermissionsException {
     /// Creates a new builder-style object to manufacture [`InsufficientPermissionsException`](crate::types::error::InsufficientPermissionsException).
-    pub fn builder() -> crate::types::error::builders::InsufficientPermissionsExceptionBuilder {
-        crate::types::error::builders::InsufficientPermissionsExceptionBuilder::default()
+    pub fn builder() -> super::super::super::types::error::builders::InsufficientPermissionsExceptionBuilder {
+        super::super::super::types::error::builders::InsufficientPermissionsExceptionBuilder::default()
     }
 }

@@ -98,8 +98,8 @@
         self
     }
     /// Consumes the builder and constructs a [`InsufficientPermissionsException`](crate::types::error::InsufficientPermissionsException).
-    pub fn build(self) -> crate::types::error::InsufficientPermissionsException {
-        crate::types::error::InsufficientPermissionsException {
+    pub fn build(self) -> super::super::super::types::error::InsufficientPermissionsException {
+        super::super::super::types::error::InsufficientPermissionsException {
             message: self.message,
             meta: self.meta.unwrap_or_default(),
         }
