--- original
+++ modified
@@ -26,7 +26,7 @@
     }
 }
 impl ::std::error::Error for InvalidS3KmsKeyArnException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::InvalidS3KmsKeyArnException {
+impl ::aws_types::request_id::RequestId for super::super::super::types::error::InvalidS3KmsKeyArnException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -39,8 +39,8 @@
 }
 impl InvalidS3KmsKeyArnException {
     /// Creates a new builder-style object to manufacture [`InvalidS3KmsKeyArnException`](crate::types::error::InvalidS3KmsKeyArnException).
-    pub fn builder() -> crate::types::error::builders::InvalidS3KmsKeyArnExceptionBuilder {
-        crate::types::error::builders::InvalidS3KmsKeyArnExceptionBuilder::default()
+    pub fn builder() -> super::super::super::types::error::builders::InvalidS3KmsKeyArnExceptionBuilder {
+        super::super::super::types::error::builders::InvalidS3KmsKeyArnExceptionBuilder::default()
     }
 }

@@ -78,8 +78,8 @@
         self
     }
     /// Consumes the builder and constructs a [`InvalidS3KmsKeyArnException`](crate::types::error::InvalidS3KmsKeyArnException).
-    pub fn build(self) -> crate::types::error::InvalidS3KmsKeyArnException {
-        crate::types::error::InvalidS3KmsKeyArnException {
+    pub fn build(self) -> super::super::super::types::error::InvalidS3KmsKeyArnException {
+        super::super::super::types::error::InvalidS3KmsKeyArnException {
             message: self.message,
             meta: self.meta.unwrap_or_default(),
         }
