--- original
+++ modified
@@ -42,7 +42,7 @@
     }
 }
 impl ::std::error::Error for InvalidRecordingGroupException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::InvalidRecordingGroupException {
+impl ::aws_types::request_id::RequestId for super::super::super::types::error::InvalidRecordingGroupException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -55,8 +55,8 @@
 }
 impl InvalidRecordingGroupException {
     /// Creates a new builder-style object to manufacture [`InvalidRecordingGroupException`](crate::types::error::InvalidRecordingGroupException).
-    pub fn builder() -> crate::types::error::builders::InvalidRecordingGroupExceptionBuilder {
-        crate::types::error::builders::InvalidRecordingGroupExceptionBuilder::default()
+    pub fn builder() -> super::super::super::types::error::builders::InvalidRecordingGroupExceptionBuilder {
+        super::super::super::types::error::builders::InvalidRecordingGroupExceptionBuilder::default()
     }
 }

@@ -94,8 +94,8 @@
         self
     }
     /// Consumes the builder and constructs a [`InvalidRecordingGroupException`](crate::types::error::InvalidRecordingGroupException).
-    pub fn build(self) -> crate::types::error::InvalidRecordingGroupException {
-        crate::types::error::InvalidRecordingGroupException {
+    pub fn build(self) -> super::super::super::types::error::InvalidRecordingGroupException {
+        super::super::super::types::error::InvalidRecordingGroupException {
             message: self.message,
             meta: self.meta.unwrap_or_default(),
         }
