--- original
+++ modified
@@ -6,7 +6,7 @@
     /// <p>The name of the configuration aggregator.</p>
     pub configuration_aggregator_name: ::std::option::Option<::std::string::String>,
     /// <p>A list of aggregate ResourceIdentifiers objects.</p>
-    pub resource_identifiers: ::std::option::Option<::std::vec::Vec<crate::types::AggregateResourceIdentifier>>,
+    pub resource_identifiers: ::std::option::Option<::std::vec::Vec<super::types::AggregateResourceIdentifier>>,
 }
 impl BatchGetAggregateResourceConfigInput {
     /// <p>The name of the configuration aggregator.</p>
@@ -16,14 +16,14 @@
     /// <p>A list of aggregate ResourceIdentifiers objects.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.resource_identifiers.is_none()`.
-    pub fn resource_identifiers(&self) -> &[crate::types::AggregateResourceIdentifier] {
+    pub fn resource_identifiers(&self) -> &[super::types::AggregateResourceIdentifier] {
         self.resource_identifiers.as_deref().unwrap_or_default()
     }
 }
 impl BatchGetAggregateResourceConfigInput {
     /// Creates a new builder-style object to manufacture [`BatchGetAggregateResourceConfigInput`](crate::operation::batch_get_aggregate_resource_config::BatchGetAggregateResourceConfigInput).
-    pub fn builder() -> crate::operation::batch_get_aggregate_resource_config::builders::BatchGetAggregateResourceConfigInputBuilder {
-        crate::operation::batch_get_aggregate_resource_config::builders::BatchGetAggregateResourceConfigInputBuilder::default()
+    pub fn builder() -> super::operation::batch_get_aggregate_resource_config::builders::BatchGetAggregateResourceConfigInputBuilder {
+        super::operation::batch_get_aggregate_resource_config::builders::BatchGetAggregateResourceConfigInputBuilder::default()
     }
 }

@@ -32,7 +32,7 @@
 #[non_exhaustive]
 pub struct BatchGetAggregateResourceConfigInputBuilder {
     pub(crate) configuration_aggregator_name: ::std::option::Option<::std::string::String>,
-    pub(crate) resource_identifiers: ::std::option::Option<::std::vec::Vec<crate::types::AggregateResourceIdentifier>>,
+    pub(crate) resource_identifiers: ::std::option::Option<::std::vec::Vec<super::types::AggregateResourceIdentifier>>,
 }
 impl BatchGetAggregateResourceConfigInputBuilder {
     /// <p>The name of the configuration aggregator.</p>
@@ -55,7 +55,7 @@
     /// To override the contents of this collection use [`set_resource_identifiers`](Self::set_resource_identifiers).
     ///
     /// <p>A list of aggregate ResourceIdentifiers objects.</p>
-    pub fn resource_identifiers(mut self, input: crate::types::AggregateResourceIdentifier) -> Self {
+    pub fn resource_identifiers(mut self, input: super::types::AggregateResourceIdentifier) -> Self {
         let mut v = self.resource_identifiers.unwrap_or_default();
         v.push(input);
         self.resource_identifiers = ::std::option::Option::Some(v);
@@ -62,12 +62,12 @@
         self
     }
     /// <p>A list of aggregate ResourceIdentifiers objects.</p>
-    pub fn set_resource_identifiers(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::AggregateResourceIdentifier>>) -> Self {
+    pub fn set_resource_identifiers(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::AggregateResourceIdentifier>>) -> Self {
         self.resource_identifiers = input;
         self
     }
     /// <p>A list of aggregate ResourceIdentifiers objects.</p>
-    pub fn get_resource_identifiers(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::AggregateResourceIdentifier>> {
+    pub fn get_resource_identifiers(&self) -> &::std::option::Option<::std::vec::Vec<super::types::AggregateResourceIdentifier>> {
         &self.resource_identifiers
     }
     /// Consumes the builder and constructs a [`BatchGetAggregateResourceConfigInput`](crate::operation::batch_get_aggregate_resource_config::BatchGetAggregateResourceConfigInput).
@@ -74,11 +74,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::batch_get_aggregate_resource_config::BatchGetAggregateResourceConfigInput,
+        super::operation::batch_get_aggregate_resource_config::BatchGetAggregateResourceConfigInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::batch_get_aggregate_resource_config::BatchGetAggregateResourceConfigInput {
+            super::operation::batch_get_aggregate_resource_config::BatchGetAggregateResourceConfigInput {
                 configuration_aggregator_name: self.configuration_aggregator_name,
                 resource_identifiers: self.resource_identifiers,
             },
