--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetScheduledQueryInput {
     /// Creates a new builder-style object to manufacture [`GetScheduledQueryInput`](crate::operation::get_scheduled_query::GetScheduledQueryInput).
-    pub fn builder() -> crate::operation::get_scheduled_query::builders::GetScheduledQueryInputBuilder {
-        crate::operation::get_scheduled_query::builders::GetScheduledQueryInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_scheduled_query::builders::GetScheduledQueryInputBuilder {
+        super::super::super::operation::get_scheduled_query::builders::GetScheduledQueryInputBuilder::default()
     }
 }

@@ -44,7 +44,7 @@
     /// Consumes the builder and constructs a [`GetScheduledQueryInput`](crate::operation::get_scheduled_query::GetScheduledQueryInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_scheduled_query::GetScheduledQueryInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_scheduled_query::GetScheduledQueryInput { identifier: self.identifier })
+    ) -> ::std::result::Result<super::super::super::operation::get_scheduled_query::GetScheduledQueryInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_scheduled_query::GetScheduledQueryInput { identifier: self.identifier })
     }
 }
