--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DescribeServiceJobInput {
     /// Creates a new builder-style object to manufacture [`DescribeServiceJobInput`](crate::operation::describe_service_job::DescribeServiceJobInput).
-    pub fn builder() -> crate::operation::describe_service_job::builders::DescribeServiceJobInputBuilder {
-        crate::operation::describe_service_job::builders::DescribeServiceJobInputBuilder::default()
+    pub fn builder() -> super::operation::describe_service_job::builders::DescribeServiceJobInputBuilder {
+        super::operation::describe_service_job::builders::DescribeServiceJobInputBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
     /// Consumes the builder and constructs a [`DescribeServiceJobInput`](crate::operation::describe_service_job::DescribeServiceJobInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::describe_service_job::DescribeServiceJobInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::describe_service_job::DescribeServiceJobInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::describe_service_job::DescribeServiceJobInput { job_id: self.job_id })
+        ::std::result::Result::Ok(super::operation::describe_service_job::DescribeServiceJobInput { job_id: self.job_id })
     }
 }
