--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On success, responds with [`CreatePlatformApplicationOutput`](crate::operation::create_platform_application::CreatePlatformApplicationOutput) with field(s):
     ///   - [`platform_application_arn(Option<String>)`](crate::operation::create_platform_application::CreatePlatformApplicationOutput::platform_application_arn): <p><code>PlatformApplicationArn</code> is returned.</p>
     /// - On failure, responds with [`SdkError<CreatePlatformApplicationError>`](crate::operation::create_platform_application::CreatePlatformApplicationError)
-    pub fn create_platform_application(&self) -> crate::operation::create_platform_application::builders::CreatePlatformApplicationFluentBuilder {
-        crate::operation::create_platform_application::builders::CreatePlatformApplicationFluentBuilder::new(self.handle.clone())
+    pub fn create_platform_application(&self) -> super::operation::create_platform_application::builders::CreatePlatformApplicationFluentBuilder {
+        super::operation::create_platform_application::builders::CreatePlatformApplicationFluentBuilder::new(self.handle.clone())
     }
 }
