--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On failure, responds with [`SdkError<DeleteSMSSandboxPhoneNumberError>`](crate::operation::delete_sms_sandbox_phone_number::DeleteSMSSandboxPhoneNumberError)
     pub fn delete_sms_sandbox_phone_number(
         &self,
-    ) -> crate::operation::delete_sms_sandbox_phone_number::builders::DeleteSMSSandboxPhoneNumberFluentBuilder {
-        crate::operation::delete_sms_sandbox_phone_number::builders::DeleteSMSSandboxPhoneNumberFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::delete_sms_sandbox_phone_number::builders::DeleteSMSSandboxPhoneNumberFluentBuilder {
+        super::operation::delete_sms_sandbox_phone_number::builders::DeleteSMSSandboxPhoneNumberFluentBuilder::new(self.handle.clone())
     }
 }
