--- original
+++ modified
@@ -9,7 +9,7 @@
     ///   - [`topics(Option<Vec::<Topic>>)`](crate::operation::list_topics::ListTopicsOutput::topics): <p>A list of topic ARNs.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_topics::ListTopicsOutput::next_token): <p>Token to pass along to the next <code>ListTopics</code> request. This element is returned if there are additional topics to retrieve.</p>
     /// - On failure, responds with [`SdkError<ListTopicsError>`](crate::operation::list_topics::ListTopicsError)
-    pub fn list_topics(&self) -> crate::operation::list_topics::builders::ListTopicsFluentBuilder {
-        crate::operation::list_topics::builders::ListTopicsFluentBuilder::new(self.handle.clone())
+    pub fn list_topics(&self) -> super::super::operation::list_topics::builders::ListTopicsFluentBuilder {
+        super::super::operation::list_topics::builders::ListTopicsFluentBuilder::new(self.handle.clone())
     }
 }
