--- original
+++ modified
@@ -8,7 +8,7 @@
     ///   - [`attribute_value(impl Into<String>)`](crate::operation::set_topic_attributes::builders::SetTopicAttributesFluentBuilder::attribute_value) / [`set_attribute_value(Option<String>)`](crate::operation::set_topic_attributes::builders::SetTopicAttributesFluentBuilder::set_attribute_value):<br>required: **false**<br><p>The new value for the attribute.</p><br>
     /// - On success, responds with [`SetTopicAttributesOutput`](crate::operation::set_topic_attributes::SetTopicAttributesOutput)
     /// - On failure, responds with [`SdkError<SetTopicAttributesError>`](crate::operation::set_topic_attributes::SetTopicAttributesError)
-    pub fn set_topic_attributes(&self) -> crate::operation::set_topic_attributes::builders::SetTopicAttributesFluentBuilder {
-        crate::operation::set_topic_attributes::builders::SetTopicAttributesFluentBuilder::new(self.handle.clone())
+    pub fn set_topic_attributes(&self) -> super::super::operation::set_topic_attributes::builders::SetTopicAttributesFluentBuilder {
+        super::super::operation::set_topic_attributes::builders::SetTopicAttributesFluentBuilder::new(self.handle.clone())
     }
 }
