--- original
+++ modified
@@ -11,7 +11,7 @@
     /// - On success, responds with [`SubscribeOutput`](crate::operation::subscribe::SubscribeOutput) with field(s):
     ///   - [`subscription_arn(Option<String>)`](crate::operation::subscribe::SubscribeOutput::subscription_arn): <p>The ARN of the subscription if it is confirmed, or the string "pending confirmation" if the subscription requires confirmation. However, if the API request parameter <code>ReturnSubscriptionArn</code> is true, then the value is always the subscription ARN, even if the subscription requires confirmation.</p>
     /// - On failure, responds with [`SdkError<SubscribeError>`](crate::operation::subscribe::SubscribeError)
-    pub fn subscribe(&self) -> crate::operation::subscribe::builders::SubscribeFluentBuilder {
-        crate::operation::subscribe::builders::SubscribeFluentBuilder::new(self.handle.clone())
+    pub fn subscribe(&self) -> super::super::operation::subscribe::builders::SubscribeFluentBuilder {
+        super::super::operation::subscribe::builders::SubscribeFluentBuilder::new(self.handle.clone())
     }
 }
