--- original
+++ modified
@@ -9,7 +9,7 @@
     /// - On failure, responds with [`SdkError<VerifySMSSandboxPhoneNumberError>`](crate::operation::verify_sms_sandbox_phone_number::VerifySMSSandboxPhoneNumberError)
     pub fn verify_sms_sandbox_phone_number(
         &self,
-    ) -> crate::operation::verify_sms_sandbox_phone_number::builders::VerifySMSSandboxPhoneNumberFluentBuilder {
-        crate::operation::verify_sms_sandbox_phone_number::builders::VerifySMSSandboxPhoneNumberFluentBuilder::new(self.handle.clone())
+    ) -> super::operation::verify_sms_sandbox_phone_number::builders::VerifySMSSandboxPhoneNumberFluentBuilder {
+        super::operation::verify_sms_sandbox_phone_number::builders::VerifySMSSandboxPhoneNumberFluentBuilder::new(self.handle.clone())
     }
 }
