--- original
+++ modified
@@ -44,8 +44,8 @@
 }
 impl ListRepositoriesInDomainInput {
     /// Creates a new builder-style object to manufacture [`ListRepositoriesInDomainInput`](crate::operation::list_repositories_in_domain::ListRepositoriesInDomainInput).
-    pub fn builder() -> crate::operation::list_repositories_in_domain::builders::ListRepositoriesInDomainInputBuilder {
-        crate::operation::list_repositories_in_domain::builders::ListRepositoriesInDomainInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_repositories_in_domain::builders::ListRepositoriesInDomainInputBuilder {
+        super::super::super::operation::list_repositories_in_domain::builders::ListRepositoriesInDomainInputBuilder::default()
     }
 }

@@ -150,10 +150,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_repositories_in_domain::ListRepositoriesInDomainInput,
+        super::super::super::operation::list_repositories_in_domain::ListRepositoriesInDomainInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_repositories_in_domain::ListRepositoriesInDomainInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_repositories_in_domain::ListRepositoriesInDomainInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             administrator_account: self.administrator_account,
