--- original
+++ modified
@@ -62,8 +62,8 @@
 }
 impl UntagResourceInput {
     /// Creates a new builder-style object to manufacture [`UntagResourceInput`](crate::operation::untag_resource::UntagResourceInput).
-    pub fn builder() -> crate::operation::untag_resource::builders::UntagResourceInputBuilder {
-        crate::operation::untag_resource::builders::UntagResourceInputBuilder::default()
+    pub fn builder() -> super::operation::untag_resource::builders::UntagResourceInputBuilder {
+        super::operation::untag_resource::builders::UntagResourceInputBuilder::default()
     }
 }

@@ -173,8 +173,8 @@
     /// Consumes the builder and constructs a [`UntagResourceInput`](crate::operation::untag_resource::UntagResourceInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::untag_resource::UntagResourceInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::untag_resource::UntagResourceInput {
+    ) -> ::std::result::Result<super::operation::untag_resource::UntagResourceInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::untag_resource::UntagResourceInput {
             resource_arn: self.resource_arn,
             tag_keys: self.tag_keys,
         })
