--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_server_certificate::GetServerCertificateOutput,
-    crate::operation::get_server_certificate::GetServerCertificateError,
+    super::super::operation::get_server_certificate::GetServerCertificateOutput,
+    super::super::operation::get_server_certificate::GetServerCertificateError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_server_certificate::GetServerCertificateError::unhandled(generic)),
+        None => return Err(super::super::operation::get_server_certificate::GetServerCertificateError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::get_server_certificate::GetServerCertificateError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::get_server_certificate::GetServerCertificateError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::get_server_certificate::GetServerCertificateError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::get_server_certificate::GetServerCertificateError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,7 +50,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_server_certificate::GetServerCertificateError::generic(generic),
+        _ => super::super::operation::get_server_certificate::GetServerCertificateError::generic(generic),
     })
 }

@@ -60,16 +60,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_server_certificate::GetServerCertificateOutput,
-    crate::operation::get_server_certificate::GetServerCertificateError,
+    super::super::operation::get_server_certificate::GetServerCertificateOutput,
+    super::super::operation::get_server_certificate::GetServerCertificateError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_server_certificate::builders::GetServerCertificateOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_server_certificate::de_get_server_certificate(_response_body, output)
-            .map_err(crate::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
+        let mut output = super::super::operation::get_server_certificate::builders::GetServerCertificateOutputBuilder::default();
+        output = super::super::protocol_serde::shape_get_server_certificate::de_get_server_certificate(_response_body, output)
+            .map_err(super::super::operation::get_server_certificate::GetServerCertificateError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_server_certificate_output_output_correct_errors(output).build()
+        super::super::serde_util::get_server_certificate_output_output_correct_errors(output).build()
     })
 }

@@ -76,9 +76,9 @@
 #[allow(unused_mut)]
 pub fn de_get_server_certificate(
     inp: &[u8],
-    mut builder: crate::operation::get_server_certificate::builders::GetServerCertificateOutputBuilder,
+    mut builder: super::super::operation::get_server_certificate::builders::GetServerCertificateOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::get_server_certificate::builders::GetServerCertificateOutputBuilder,
+    super::super::operation::get_server_certificate::builders::GetServerCertificateOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -106,7 +106,7 @@
             s if s.matches("ServerCertificate") /* ServerCertificate com.amazonaws.iam.synthetic#GetServerCertificateOutput$ServerCertificate */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_server_certificate::de_server_certificate(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_server_certificate::de_server_certificate(&mut tag, depth + 1)
                         ?
                     )
                 ;
