--- original
+++ modified
@@ -5,12 +5,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementOutput,
-    crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError,
+    super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementOutput,
+    super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body).map_err(
-        crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled,
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body).map_err(
+        super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled,
     )?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
@@ -18,7 +18,7 @@
         match generic.code() {
             Some(code) => code,
             None => return Err(
-                crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled(
+                super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled(
                     generic,
                 ),
             ),
@@ -26,13 +26,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AccountNotManagementOrDelegatedAdministratorException" => crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::AccountNotManagementOrDelegatedAdministratorException({
+        "AccountNotManagementOrDelegatedAdministratorException" => super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::AccountNotManagementOrDelegatedAdministratorException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::AccountNotManagementOrDelegatedAdministratorExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_account_not_management_or_delegated_administrator_exception::de_account_not_management_or_delegated_administrator_exception_xml_err(_response_body, output).map_err(crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::super::types::error::builders::AccountNotManagementOrDelegatedAdministratorExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_account_not_management_or_delegated_administrator_exception::de_account_not_management_or_delegated_administrator_exception_xml_err(_response_body, output).map_err(super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -42,13 +42,13 @@
                                                         }
             tmp
         }),
-        "OrganizationNotFoundException" => crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::OrganizationNotFoundException({
+        "OrganizationNotFoundException" => super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::OrganizationNotFoundException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationNotFoundExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_not_found_exception::de_organization_not_found_exception_xml_err(_response_body, output).map_err(crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::super::types::error::builders::OrganizationNotFoundExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_organization_not_found_exception::de_organization_not_found_exception_xml_err(_response_body, output).map_err(super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -58,13 +58,13 @@
                                                         }
             tmp
         }),
-        "OrganizationNotInAllFeaturesModeException" => crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::OrganizationNotInAllFeaturesModeException({
+        "OrganizationNotInAllFeaturesModeException" => super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::OrganizationNotInAllFeaturesModeException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationNotInAllFeaturesModeExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_not_in_all_features_mode_exception::de_organization_not_in_all_features_mode_exception_xml_err(_response_body, output).map_err(crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::super::types::error::builders::OrganizationNotInAllFeaturesModeExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_organization_not_in_all_features_mode_exception::de_organization_not_in_all_features_mode_exception_xml_err(_response_body, output).map_err(super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -74,13 +74,13 @@
                                                         }
             tmp
         }),
-        "ServiceAccessNotEnabledException" => crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::ServiceAccessNotEnabledException({
+        "ServiceAccessNotEnabledException" => super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::ServiceAccessNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceAccessNotEnabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_service_access_not_enabled_exception::de_service_access_not_enabled_exception_xml_err(_response_body, output).map_err(crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::super::types::error::builders::ServiceAccessNotEnabledExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_service_access_not_enabled_exception::de_service_access_not_enabled_exception_xml_err(_response_body, output).map_err(super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -90,7 +90,7 @@
                                                         }
             tmp
         }),
-        _ => crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::generic(generic)
+        _ => super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::generic(generic)
     })
 }

@@ -100,19 +100,19 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementOutput,
-    crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError,
+    super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementOutput,
+    super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementOutputBuilder::default();
+        let mut output = super::super::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_disable_organizations_root_credentials_management::de_disable_organizations_root_credentials_management(
+            super::super::protocol_serde::shape_disable_organizations_root_credentials_management::de_disable_organizations_root_credentials_management(
                 _response_body,
                 output,
             )
             .map_err(
-                crate::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled,
+                super::super::operation::disable_organizations_root_credentials_management::DisableOrganizationsRootCredentialsManagementError::unhandled,
             )?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
@@ -122,9 +122,9 @@
 #[allow(unused_mut)]
 pub fn de_disable_organizations_root_credentials_management(
     inp: &[u8],
-    mut builder: crate::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementOutputBuilder,
+    mut builder: super::super::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementOutputBuilder,
+    super::super::operation::disable_organizations_root_credentials_management::builders::DisableOrganizationsRootCredentialsManagementOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -165,7 +165,7 @@
             s if s.matches("EnabledFeatures") /* EnabledFeatures com.amazonaws.iam.synthetic#DisableOrganizationsRootCredentialsManagementOutput$EnabledFeatures */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_features_list_type::de_features_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_features_list_type::de_features_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
