--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_signing_certificates::ListSigningCertificatesOutput,
-    crate::operation::list_signing_certificates::ListSigningCertificatesError,
+    super::operation::list_signing_certificates::ListSigningCertificatesOutput,
+    super::operation::list_signing_certificates::ListSigningCertificatesError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::list_signing_certificates::ListSigningCertificatesError::unhandled(
+            return Err(super::operation::list_signing_certificates::ListSigningCertificatesError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::list_signing_certificates::ListSigningCertificatesError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::list_signing_certificates::ListSigningCertificatesError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::list_signing_certificates::ListSigningCertificatesError::ServiceFailureException({
+        "ServiceFailure" => super::operation::list_signing_certificates::ListSigningCertificatesError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,7 +54,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_signing_certificates::ListSigningCertificatesError::generic(generic),
+        _ => super::operation::list_signing_certificates::ListSigningCertificatesError::generic(generic),
     })
 }

@@ -64,18 +64,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_signing_certificates::ListSigningCertificatesOutput,
-    crate::operation::list_signing_certificates::ListSigningCertificatesError,
+    super::operation::list_signing_certificates::ListSigningCertificatesOutput,
+    super::operation::list_signing_certificates::ListSigningCertificatesError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_signing_certificates::builders::ListSigningCertificatesOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_signing_certificates::de_list_signing_certificates(_response_body, output)
-            .map_err(crate::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
+        let mut output = super::operation::list_signing_certificates::builders::ListSigningCertificatesOutputBuilder::default();
+        output = super::protocol_serde::shape_list_signing_certificates::de_list_signing_certificates(_response_body, output)
+            .map_err(super::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_signing_certificates_output_output_correct_errors(output)
+        super::serde_util::list_signing_certificates_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?
+            .map_err(super::operation::list_signing_certificates::ListSigningCertificatesError::unhandled)?
     })
 }

@@ -82,9 +82,9 @@
 #[allow(unused_mut)]
 pub fn de_list_signing_certificates(
     inp: &[u8],
-    mut builder: crate::operation::list_signing_certificates::builders::ListSigningCertificatesOutputBuilder,
+    mut builder: super::operation::list_signing_certificates::builders::ListSigningCertificatesOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::list_signing_certificates::builders::ListSigningCertificatesOutputBuilder,
+    super::operation::list_signing_certificates::builders::ListSigningCertificatesOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -112,7 +112,7 @@
             s if s.matches("Certificates") /* Certificates com.amazonaws.iam.synthetic#ListSigningCertificatesOutput$Certificates */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_certificate_list_type::de_certificate_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_certificate_list_type::de_certificate_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
