--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_open_id_connect_providers::ListOpenIdConnectProvidersOutput,
-    crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError,
+    super::super::operation::list_open_id_connect_providers::ListOpenIdConnectProvidersOutput,
+    super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled(generic)),
+        None => return Err(super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ServiceFailure" => crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,7 +35,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::generic(generic),
+        _ => super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::generic(generic),
     })
 }

@@ -45,14 +45,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_open_id_connect_providers::ListOpenIdConnectProvidersOutput,
-    crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError,
+    super::super::operation::list_open_id_connect_providers::ListOpenIdConnectProvidersOutput,
+    super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_open_id_connect_providers::builders::ListOpenIdConnectProvidersOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_open_id_connect_providers::de_list_open_id_connect_providers(_response_body, output)
-            .map_err(crate::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled)?;
+        let mut output = super::super::operation::list_open_id_connect_providers::builders::ListOpenIdConnectProvidersOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_open_id_connect_providers::de_list_open_id_connect_providers(_response_body, output)
+            .map_err(super::super::operation::list_open_id_connect_providers::ListOpenIDConnectProvidersError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -61,9 +61,9 @@
 #[allow(unused_mut)]
 pub fn de_list_open_id_connect_providers(
     inp: &[u8],
-    mut builder: crate::operation::list_open_id_connect_providers::builders::ListOpenIdConnectProvidersOutputBuilder,
+    mut builder: super::super::operation::list_open_id_connect_providers::builders::ListOpenIdConnectProvidersOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::list_open_id_connect_providers::builders::ListOpenIdConnectProvidersOutputBuilder,
+    super::super::operation::list_open_id_connect_providers::builders::ListOpenIdConnectProvidersOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -91,7 +91,7 @@
             s if s.matches("OpenIDConnectProviderList") /* OpenIDConnectProviderList com.amazonaws.iam.synthetic#ListOpenIDConnectProvidersOutput$OpenIDConnectProviderList */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_open_id_connect_provider_list_type::de_open_id_connect_provider_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_open_id_connect_provider_list_type::de_open_id_connect_provider_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
