--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_policy_versions::ListPolicyVersionsOutput,
-    crate::operation::list_policy_versions::ListPolicyVersionsError,
+    super::operation::list_policy_versions::ListPolicyVersionsOutput,
+    super::operation::list_policy_versions::ListPolicyVersionsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_policy_versions::ListPolicyVersionsError::unhandled(generic)),
+        None => return Err(super::operation::list_policy_versions::ListPolicyVersionsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidInput" => crate::operation::list_policy_versions::ListPolicyVersionsError::InvalidInputException({
+        "InvalidInput" => super::operation::list_policy_versions::ListPolicyVersionsError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::list_policy_versions::ListPolicyVersionsError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::list_policy_versions::ListPolicyVersionsError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::list_policy_versions::ListPolicyVersionsError::ServiceFailureException({
+        "ServiceFailure" => super::operation::list_policy_versions::ListPolicyVersionsError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,7 +65,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_policy_versions::ListPolicyVersionsError::generic(generic),
+        _ => super::operation::list_policy_versions::ListPolicyVersionsError::generic(generic),
     })
 }

@@ -75,14 +75,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_policy_versions::ListPolicyVersionsOutput,
-    crate::operation::list_policy_versions::ListPolicyVersionsError,
+    super::operation::list_policy_versions::ListPolicyVersionsOutput,
+    super::operation::list_policy_versions::ListPolicyVersionsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_policy_versions::builders::ListPolicyVersionsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_policy_versions::de_list_policy_versions(_response_body, output)
-            .map_err(crate::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
+        let mut output = super::operation::list_policy_versions::builders::ListPolicyVersionsOutputBuilder::default();
+        output = super::protocol_serde::shape_list_policy_versions::de_list_policy_versions(_response_body, output)
+            .map_err(super::operation::list_policy_versions::ListPolicyVersionsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -91,8 +91,8 @@
 #[allow(unused_mut)]
 pub fn de_list_policy_versions(
     inp: &[u8],
-    mut builder: crate::operation::list_policy_versions::builders::ListPolicyVersionsOutputBuilder,
-) -> std::result::Result<crate::operation::list_policy_versions::builders::ListPolicyVersionsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
+    mut builder: super::operation::list_policy_versions::builders::ListPolicyVersionsOutputBuilder,
+) -> std::result::Result<super::operation::list_policy_versions::builders::ListPolicyVersionsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
 {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

@@ -119,7 +119,7 @@
             s if s.matches("Versions") /* Versions com.amazonaws.iam.synthetic#ListPolicyVersionsOutput$Versions */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_policy_document_version_list_type::de_policy_document_version_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_policy_document_version_list_type::de_policy_document_version_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
