--- original
+++ modified
@@ -5,29 +5,29 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusOutput,
-    crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError,
+    super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusOutput,
+    super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled(generic)),
+        None => return Err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InvalidInput" => {
-            crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::InvalidInputException({
+            super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::InvalidInputException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -38,13 +38,13 @@
             })
         }
         "NoSuchEntity" => {
-            crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::NoSuchEntityException({
+            super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::NoSuchEntityException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                    output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -55,13 +55,13 @@
             })
         }
         "ServiceFailure" => {
-            crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::ServiceFailureException({
+            super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::ServiceFailureException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                    output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -71,7 +71,7 @@
                 tmp
             })
         }
-        _ => crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::generic(generic),
+        _ => super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::generic(generic),
     })
 }

@@ -81,20 +81,20 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusOutput,
-    crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError,
+    super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusOutput,
+    super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusOutputBuilder::default();
+            super::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_get_service_linked_role_deletion_status::de_get_service_linked_role_deletion_status(_response_body, output)
-                .map_err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
+            super::protocol_serde::shape_get_service_linked_role_deletion_status::de_get_service_linked_role_deletion_status(_response_body, output)
+                .map_err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_service_linked_role_deletion_status_output_output_correct_errors(output)
+        super::serde_util::get_service_linked_role_deletion_status_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?
+            .map_err(super::operation::get_service_linked_role_deletion_status::GetServiceLinkedRoleDeletionStatusError::unhandled)?
     })
 }

@@ -101,9 +101,9 @@
 #[allow(unused_mut)]
 pub fn de_get_service_linked_role_deletion_status(
     inp: &[u8],
-    mut builder: crate::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusOutputBuilder,
+    mut builder: super::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusOutputBuilder,
+    super::operation::get_service_linked_role_deletion_status::builders::GetServiceLinkedRoleDeletionStatusOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -131,8 +131,8 @@
             s if s.matches("Status") /* Status com.amazonaws.iam.synthetic#GetServiceLinkedRoleDeletionStatusOutput$Status */ =>  {
                 let var_1 =
                     Some(
-                        Result::<crate::types::DeletionTaskStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::DeletionTaskStatusType::from(
+                        Result::<super::types::DeletionTaskStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::DeletionTaskStatusType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -145,7 +145,7 @@
             s if s.matches("Reason") /* Reason com.amazonaws.iam.synthetic#GetServiceLinkedRoleDeletionStatusOutput$Reason */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_deletion_task_failure_reason_type::de_deletion_task_failure_reason_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_deletion_task_failure_reason_type::de_deletion_task_failure_reason_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
