--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_statement(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::Statement, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::Statement, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::Statement::builder();
+    let mut builder = super::super::types::Statement::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("SourcePolicyId") /* SourcePolicyId com.amazonaws.iam#Statement$SourcePolicyId */ =>  {
@@ -27,8 +27,8 @@
             s if s.matches("SourcePolicyType") /* SourcePolicyType com.amazonaws.iam#Statement$SourcePolicyType */ =>  {
                 let var_2 =
                     Some(
-                        Result::<crate::types::PolicySourceType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::PolicySourceType::from(
+                        Result::<super::super::types::PolicySourceType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::PolicySourceType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -41,7 +41,7 @@
             s if s.matches("StartPosition") /* StartPosition com.amazonaws.iam#Statement$StartPosition */ =>  {
                 let var_3 =
                     Some(
-                        crate::protocol_serde::shape_position::de_position(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_position::de_position(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -51,7 +51,7 @@
             s if s.matches("EndPosition") /* EndPosition com.amazonaws.iam#Statement$EndPosition */ =>  {
                 let var_4 =
                     Some(
-                        crate::protocol_serde::shape_position::de_position(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_position::de_position(&mut tag, depth + 1)
                         ?
                     )
                 ;
