--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsOutput,
-    crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError,
+    super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsOutput,
+    super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled(generic)),
+        None => return Err(super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidInput" => crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::InvalidInputException({
+        "InvalidInput" => super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,7 +50,7 @@
             }
             tmp
         }),
-        _ => crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::generic(generic),
+        _ => super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::generic(generic),
     })
 }

@@ -60,16 +60,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsOutput,
-    crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError,
+    super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsOutput,
+    super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::generate_service_last_accessed_details::builders::GenerateServiceLastAccessedDetailsOutputBuilder::default();
+            super::super::operation::generate_service_last_accessed_details::builders::GenerateServiceLastAccessedDetailsOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_generate_service_last_accessed_details::de_generate_service_last_accessed_details(_response_body, output)
-                .map_err(crate::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
+            super::super::protocol_serde::shape_generate_service_last_accessed_details::de_generate_service_last_accessed_details(_response_body, output)
+                .map_err(super::super::operation::generate_service_last_accessed_details::GenerateServiceLastAccessedDetailsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -78,9 +78,9 @@
 #[allow(unused_mut)]
 pub fn de_generate_service_last_accessed_details(
     inp: &[u8],
-    mut builder: crate::operation::generate_service_last_accessed_details::builders::GenerateServiceLastAccessedDetailsOutputBuilder,
+    mut builder: super::super::operation::generate_service_last_accessed_details::builders::GenerateServiceLastAccessedDetailsOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::generate_service_last_accessed_details::builders::GenerateServiceLastAccessedDetailsOutputBuilder,
+    super::super::operation::generate_service_last_accessed_details::builders::GenerateServiceLastAccessedDetailsOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
