--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_role_tags::ListRoleTagsOutput, crate::operation::list_role_tags::ListRoleTagsError> {
+) -> std::result::Result<super::operation::list_role_tags::ListRoleTagsOutput, super::operation::list_role_tags::ListRoleTagsError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_role_tags::ListRoleTagsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_role_tags::ListRoleTagsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_role_tags::ListRoleTagsError::unhandled(generic)),
+        None => return Err(super::operation::list_role_tags::ListRoleTagsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::list_role_tags::ListRoleTagsError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::list_role_tags::ListRoleTagsError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_role_tags::ListRoleTagsError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_role_tags::ListRoleTagsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::list_role_tags::ListRoleTagsError::ServiceFailureException({
+        "ServiceFailure" => super::operation::list_role_tags::ListRoleTagsError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_role_tags::ListRoleTagsError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_role_tags::ListRoleTagsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,7 +47,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_role_tags::ListRoleTagsError::generic(generic),
+        _ => super::operation::list_role_tags::ListRoleTagsError::generic(generic),
     })
 }

@@ -56,16 +56,16 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_role_tags::ListRoleTagsOutput, crate::operation::list_role_tags::ListRoleTagsError> {
+) -> std::result::Result<super::operation::list_role_tags::ListRoleTagsOutput, super::operation::list_role_tags::ListRoleTagsError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_role_tags::builders::ListRoleTagsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_role_tags::de_list_role_tags(_response_body, output)
-            .map_err(crate::operation::list_role_tags::ListRoleTagsError::unhandled)?;
+        let mut output = super::operation::list_role_tags::builders::ListRoleTagsOutputBuilder::default();
+        output = super::protocol_serde::shape_list_role_tags::de_list_role_tags(_response_body, output)
+            .map_err(super::operation::list_role_tags::ListRoleTagsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_role_tags_output_output_correct_errors(output)
+        super::serde_util::list_role_tags_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_role_tags::ListRoleTagsError::unhandled)?
+            .map_err(super::operation::list_role_tags::ListRoleTagsError::unhandled)?
     })
 }

@@ -72,8 +72,8 @@
 #[allow(unused_mut)]
 pub fn de_list_role_tags(
     inp: &[u8],
-    mut builder: crate::operation::list_role_tags::builders::ListRoleTagsOutputBuilder,
-) -> std::result::Result<crate::operation::list_role_tags::builders::ListRoleTagsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::operation::list_role_tags::builders::ListRoleTagsOutputBuilder,
+) -> std::result::Result<super::operation::list_role_tags::builders::ListRoleTagsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -99,7 +99,7 @@
             s if s.matches("Tags") /* Tags com.amazonaws.iam.synthetic#ListRoleTagsOutput$Tags */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
