--- original
+++ modified
@@ -2,7 +2,7 @@
 #[allow(unused_mut)]
 pub fn ser_delegation_permission(
     mut writer: ::aws_smithy_query::QueryValueWriter,
-    input: &crate::types::DelegationPermission,
+    input: &super::types::DelegationPermission,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
     let mut scope_1 = writer.prefix("PolicyTemplateArn");
@@ -16,7 +16,7 @@
         for item_5 in var_4 {
             #[allow(unused_mut)]
             let mut entry_7 = list_6.entry();
-            crate::protocol_serde::shape_policy_parameter::ser_policy_parameter(entry_7, item_5)?;
+            super::protocol_serde::shape_policy_parameter::ser_policy_parameter(entry_7, item_5)?;
         }
         list_6.finish();
     }
@@ -27,12 +27,12 @@
 pub fn de_delegation_permission(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::DelegationPermission, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::DelegationPermission, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::DelegationPermission::builder();
+    let mut builder = super::types::DelegationPermission::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("PolicyTemplateArn") /* PolicyTemplateArn com.amazonaws.iam#DelegationPermission$PolicyTemplateArn */ =>  {
@@ -51,7 +51,7 @@
             s if s.matches("Parameters") /* Parameters com.amazonaws.iam#DelegationPermission$Parameters */ =>  {
                 let var_9 =
                     Some(
-                        crate::protocol_serde::shape_policy_parameter_list_type::de_policy_parameter_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_policy_parameter_list_type::de_policy_parameter_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
