--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_policy_tags::ListPolicyTagsOutput, crate::operation::list_policy_tags::ListPolicyTagsError> {
+) -> std::result::Result<super::super::operation::list_policy_tags::ListPolicyTagsOutput, super::super::operation::list_policy_tags::ListPolicyTagsError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled(generic)),
+        None => return Err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidInput" => crate::operation::list_policy_tags::ListPolicyTagsError::InvalidInputException({
+        "InvalidInput" => super::super::operation::list_policy_tags::ListPolicyTagsError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::list_policy_tags::ListPolicyTagsError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::list_policy_tags::ListPolicyTagsError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::list_policy_tags::ListPolicyTagsError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::list_policy_tags::ListPolicyTagsError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,7 +62,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_policy_tags::ListPolicyTagsError::generic(generic),
+        _ => super::super::operation::list_policy_tags::ListPolicyTagsError::generic(generic),
     })
 }

@@ -71,16 +71,16 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_policy_tags::ListPolicyTagsOutput, crate::operation::list_policy_tags::ListPolicyTagsError> {
+) -> std::result::Result<super::super::operation::list_policy_tags::ListPolicyTagsOutput, super::super::operation::list_policy_tags::ListPolicyTagsError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_policy_tags::builders::ListPolicyTagsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_policy_tags::de_list_policy_tags(_response_body, output)
-            .map_err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
+        let mut output = super::super::operation::list_policy_tags::builders::ListPolicyTagsOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_policy_tags::de_list_policy_tags(_response_body, output)
+            .map_err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_policy_tags_output_output_correct_errors(output)
+        super::super::serde_util::list_policy_tags_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_policy_tags::ListPolicyTagsError::unhandled)?
+            .map_err(super::super::operation::list_policy_tags::ListPolicyTagsError::unhandled)?
     })
 }

@@ -87,8 +87,8 @@
 #[allow(unused_mut)]
 pub fn de_list_policy_tags(
     inp: &[u8],
-    mut builder: crate::operation::list_policy_tags::builders::ListPolicyTagsOutputBuilder,
-) -> std::result::Result<crate::operation::list_policy_tags::builders::ListPolicyTagsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::super::operation::list_policy_tags::builders::ListPolicyTagsOutputBuilder,
+) -> std::result::Result<super::super::operation::list_policy_tags::builders::ListPolicyTagsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -114,7 +114,7 @@
             s if s.matches("Tags") /* Tags com.amazonaws.iam.synthetic#ListPolicyTagsOutput$Tags */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
