--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_user::CreateUserOutput, crate::operation::create_user::CreateUserError> {
+) -> std::result::Result<super::operation::create_user::CreateUserOutput, super::operation::create_user::CreateUserError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::create_user::CreateUserError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::create_user::CreateUserError::unhandled(generic)),
+        None => return Err(super::operation::create_user::CreateUserError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConcurrentModification" => crate::operation::create_user::CreateUserError::ConcurrentModificationException({
+        "ConcurrentModification" => super::operation::create_user::CreateUserError::ConcurrentModificationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
+                let mut output = super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                output = super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+                .map_err(super::operation::create_user::CreateUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,14 +35,14 @@
             }
             tmp
         }),
-        "EntityAlreadyExists" => crate::operation::create_user::CreateUserError::EntityAlreadyExistsException({
+        "EntityAlreadyExists" => super::operation::create_user::CreateUserError::EntityAlreadyExistsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
+                let mut output = super::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+                    super::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::create_user::CreateUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -51,13 +51,13 @@
             }
             tmp
         }),
-        "InvalidInput" => crate::operation::create_user::CreateUserError::InvalidInputException({
+        "InvalidInput" => super::operation::create_user::CreateUserError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_user::CreateUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -66,13 +66,13 @@
             }
             tmp
         }),
-        "LimitExceeded" => crate::operation::create_user::CreateUserError::LimitExceededException({
+        "LimitExceeded" => super::operation::create_user::CreateUserError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_user::CreateUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -81,13 +81,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::create_user::CreateUserError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::create_user::CreateUserError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_user::CreateUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -96,13 +96,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::create_user::CreateUserError::ServiceFailureException({
+        "ServiceFailure" => super::operation::create_user::CreateUserError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_user::CreateUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -111,7 +111,7 @@
             }
             tmp
         }),
-        _ => crate::operation::create_user::CreateUserError::generic(generic),
+        _ => super::operation::create_user::CreateUserError::generic(generic),
     })
 }

@@ -120,12 +120,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_user::CreateUserOutput, crate::operation::create_user::CreateUserError> {
+) -> std::result::Result<super::operation::create_user::CreateUserOutput, super::operation::create_user::CreateUserError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::create_user::builders::CreateUserOutputBuilder::default();
-        output = crate::protocol_serde::shape_create_user::de_create_user(_response_body, output)
-            .map_err(crate::operation::create_user::CreateUserError::unhandled)?;
+        let mut output = super::operation::create_user::builders::CreateUserOutputBuilder::default();
+        output = super::protocol_serde::shape_create_user::de_create_user(_response_body, output)
+            .map_err(super::operation::create_user::CreateUserError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -134,8 +134,8 @@
 #[allow(unused_mut)]
 pub fn de_create_user(
     inp: &[u8],
-    mut builder: crate::operation::create_user::builders::CreateUserOutputBuilder,
-) -> std::result::Result<crate::operation::create_user::builders::CreateUserOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::operation::create_user::builders::CreateUserOutputBuilder,
+) -> std::result::Result<super::operation::create_user::builders::CreateUserOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -161,7 +161,7 @@
             s if s.matches("User") /* User com.amazonaws.iam.synthetic#CreateUserOutput$User */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_user::de_user(&mut tag, depth + 1)
+                        super::protocol_serde::shape_user::de_user(&mut tag, depth + 1)
                         ?
                     )
                 ;
