--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_login_profile(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::LoginProfile, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::LoginProfile, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::LoginProfile::builder();
+    let mut builder = super::super::types::LoginProfile::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("UserName") /* UserName com.amazonaws.iam#LoginProfile$UserName */ =>  {
@@ -56,7 +56,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::login_profile_correct_errors(builder)
+    Ok(super::super::serde_util::login_profile_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
