--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_server_certificate::UpdateServerCertificateOutput,
-    crate::operation::update_server_certificate::UpdateServerCertificateError,
+    super::super::operation::update_server_certificate::UpdateServerCertificateOutput,
+    super::super::operation::update_server_certificate::UpdateServerCertificateError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::update_server_certificate::UpdateServerCertificateError::unhandled(
+            return Err(super::super::operation::update_server_certificate::UpdateServerCertificateError::unhandled(
                 generic,
             ))
         }
@@ -24,14 +24,14 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "EntityAlreadyExists" => crate::operation::update_server_certificate::UpdateServerCertificateError::EntityAlreadyExistsException({
+        "EntityAlreadyExists" => super::super::operation::update_server_certificate::UpdateServerCertificateError::EntityAlreadyExistsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
+                    super::super::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
+                        .map_err(super::super::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -40,13 +40,13 @@
             }
             tmp
         }),
-        "LimitExceeded" => crate::operation::update_server_certificate::UpdateServerCertificateError::LimitExceededException({
+        "LimitExceeded" => super::super::operation::update_server_certificate::UpdateServerCertificateError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
+                let mut output = super::super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -55,13 +55,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::update_server_certificate::UpdateServerCertificateError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::update_server_certificate::UpdateServerCertificateError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,13 +70,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::update_server_certificate::UpdateServerCertificateError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::update_server_certificate::UpdateServerCertificateError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::update_server_certificate::UpdateServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -85,7 +85,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_server_certificate::UpdateServerCertificateError::generic(generic),
+        _ => super::super::operation::update_server_certificate::UpdateServerCertificateError::generic(generic),
     })
 }

@@ -95,12 +95,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_server_certificate::UpdateServerCertificateOutput,
-    crate::operation::update_server_certificate::UpdateServerCertificateError,
+    super::super::operation::update_server_certificate::UpdateServerCertificateOutput,
+    super::super::operation::update_server_certificate::UpdateServerCertificateError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_server_certificate::builders::UpdateServerCertificateOutputBuilder::default();
+        let mut output = super::super::operation::update_server_certificate::builders::UpdateServerCertificateOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
