--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::untag_policy::UntagPolicyOutput, crate::operation::untag_policy::UntagPolicyError> {
+) -> std::result::Result<super::super::operation::untag_policy::UntagPolicyOutput, super::super::operation::untag_policy::UntagPolicyError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::untag_policy::UntagPolicyError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::untag_policy::UntagPolicyError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::untag_policy::UntagPolicyError::unhandled(generic)),
+        None => return Err(super::super::operation::untag_policy::UntagPolicyError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConcurrentModification" => crate::operation::untag_policy::UntagPolicyError::ConcurrentModificationException({
+        "ConcurrentModification" => super::super::operation::untag_policy::UntagPolicyError::ConcurrentModificationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
+                let mut output = super::super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::untag_policy::UntagPolicyError::unhandled)?;
+                .map_err(super::super::operation::untag_policy::UntagPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InvalidInput" => crate::operation::untag_policy::UntagPolicyError::InvalidInputException({
+        "InvalidInput" => super::super::operation::untag_policy::UntagPolicyError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_policy::UntagPolicyError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_policy::UntagPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::untag_policy::UntagPolicyError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::untag_policy::UntagPolicyError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_policy::UntagPolicyError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_policy::UntagPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,13 +65,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::untag_policy::UntagPolicyError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::untag_policy::UntagPolicyError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::untag_policy::UntagPolicyError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::untag_policy::UntagPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -80,7 +80,7 @@
             }
             tmp
         }),
-        _ => crate::operation::untag_policy::UntagPolicyError::generic(generic),
+        _ => super::super::operation::untag_policy::UntagPolicyError::generic(generic),
     })
 }

@@ -89,10 +89,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::untag_policy::UntagPolicyOutput, crate::operation::untag_policy::UntagPolicyError> {
+) -> std::result::Result<super::super::operation::untag_policy::UntagPolicyOutput, super::super::operation::untag_policy::UntagPolicyError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::untag_policy::builders::UntagPolicyOutputBuilder::default();
+        let mut output = super::super::operation::untag_policy::builders::UntagPolicyOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
