--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::upload_server_certificate::UploadServerCertificateOutput,
-    crate::operation::upload_server_certificate::UploadServerCertificateError,
+    super::operation::upload_server_certificate::UploadServerCertificateOutput,
+    super::operation::upload_server_certificate::UploadServerCertificateError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled(
+            return Err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled(
                 generic,
             ))
         }
@@ -24,16 +24,16 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConcurrentModification" => crate::operation::upload_server_certificate::UploadServerCertificateError::ConcurrentModificationException({
+        "ConcurrentModification" => super::operation::upload_server_certificate::UploadServerCertificateError::ConcurrentModificationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
+                let mut output = super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                output = super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -42,14 +42,14 @@
             }
             tmp
         }),
-        "EntityAlreadyExists" => crate::operation::upload_server_certificate::UploadServerCertificateError::EntityAlreadyExistsException({
+        "EntityAlreadyExists" => super::operation::upload_server_certificate::UploadServerCertificateError::EntityAlreadyExistsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
+                let mut output = super::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                    super::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -58,13 +58,13 @@
             }
             tmp
         }),
-        "InvalidInput" => crate::operation::upload_server_certificate::UploadServerCertificateError::InvalidInputException({
+        "InvalidInput" => super::operation::upload_server_certificate::UploadServerCertificateError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -73,13 +73,13 @@
             }
             tmp
         }),
-        "KeyPairMismatch" => crate::operation::upload_server_certificate::UploadServerCertificateError::KeyPairMismatchException({
+        "KeyPairMismatch" => super::operation::upload_server_certificate::UploadServerCertificateError::KeyPairMismatchException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KeyPairMismatchExceptionBuilder::default();
-                output = crate::protocol_serde::shape_key_pair_mismatch_exception::de_key_pair_mismatch_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                let mut output = super::types::error::builders::KeyPairMismatchExceptionBuilder::default();
+                output = super::protocol_serde::shape_key_pair_mismatch_exception::de_key_pair_mismatch_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -88,13 +88,13 @@
             }
             tmp
         }),
-        "LimitExceeded" => crate::operation::upload_server_certificate::UploadServerCertificateError::LimitExceededException({
+        "LimitExceeded" => super::operation::upload_server_certificate::UploadServerCertificateError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -103,14 +103,14 @@
             }
             tmp
         }),
-        "MalformedCertificate" => crate::operation::upload_server_certificate::UploadServerCertificateError::MalformedCertificateException({
+        "MalformedCertificate" => super::operation::upload_server_certificate::UploadServerCertificateError::MalformedCertificateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::MalformedCertificateExceptionBuilder::default();
+                let mut output = super::types::error::builders::MalformedCertificateExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_malformed_certificate_exception::de_malformed_certificate_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                    super::protocol_serde::shape_malformed_certificate_exception::de_malformed_certificate_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -119,13 +119,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::upload_server_certificate::UploadServerCertificateError::ServiceFailureException({
+        "ServiceFailure" => super::operation::upload_server_certificate::UploadServerCertificateError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -134,7 +134,7 @@
             }
             tmp
         }),
-        _ => crate::operation::upload_server_certificate::UploadServerCertificateError::generic(generic),
+        _ => super::operation::upload_server_certificate::UploadServerCertificateError::generic(generic),
     })
 }

@@ -144,14 +144,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::upload_server_certificate::UploadServerCertificateOutput,
-    crate::operation::upload_server_certificate::UploadServerCertificateError,
+    super::operation::upload_server_certificate::UploadServerCertificateOutput,
+    super::operation::upload_server_certificate::UploadServerCertificateError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::upload_server_certificate::builders::UploadServerCertificateOutputBuilder::default();
-        output = crate::protocol_serde::shape_upload_server_certificate::de_upload_server_certificate(_response_body, output)
-            .map_err(crate::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
+        let mut output = super::operation::upload_server_certificate::builders::UploadServerCertificateOutputBuilder::default();
+        output = super::protocol_serde::shape_upload_server_certificate::de_upload_server_certificate(_response_body, output)
+            .map_err(super::operation::upload_server_certificate::UploadServerCertificateError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -160,9 +160,9 @@
 #[allow(unused_mut)]
 pub fn de_upload_server_certificate(
     inp: &[u8],
-    mut builder: crate::operation::upload_server_certificate::builders::UploadServerCertificateOutputBuilder,
+    mut builder: super::operation::upload_server_certificate::builders::UploadServerCertificateOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::upload_server_certificate::builders::UploadServerCertificateOutputBuilder,
+    super::operation::upload_server_certificate::builders::UploadServerCertificateOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -190,7 +190,7 @@
             s if s.matches("ServerCertificateMetadata") /* ServerCertificateMetadata com.amazonaws.iam.synthetic#UploadServerCertificateOutput$ServerCertificateMetadata */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_server_certificate_metadata::de_server_certificate_metadata(&mut tag, depth + 1)
+                        super::protocol_serde::shape_server_certificate_metadata::de_server_certificate_metadata(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -200,7 +200,7 @@
             s if s.matches("Tags") /* Tags com.amazonaws.iam.synthetic#UploadServerCertificateOutput$Tags */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
