--- original
+++ modified
@@ -5,12 +5,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementOutput,
-    crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError,
+    super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementOutput,
+    super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
@@ -17,7 +17,7 @@
         Some(code) => code,
         None => {
             return Err(
-                crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled(
+                super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled(
                     generic,
                 ),
             )
@@ -26,13 +26,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "AccountNotManagementOrDelegatedAdministratorException" => crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::AccountNotManagementOrDelegatedAdministratorException({
+        "AccountNotManagementOrDelegatedAdministratorException" => super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::AccountNotManagementOrDelegatedAdministratorException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::AccountNotManagementOrDelegatedAdministratorExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_account_not_management_or_delegated_administrator_exception::de_account_not_management_or_delegated_administrator_exception_xml_err(_response_body, output).map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::types::error::builders::AccountNotManagementOrDelegatedAdministratorExceptionBuilder::default();
+                    output = super::protocol_serde::shape_account_not_management_or_delegated_administrator_exception::de_account_not_management_or_delegated_administrator_exception_xml_err(_response_body, output).map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -42,13 +42,13 @@
                                                         }
             tmp
         }),
-        "CallerIsNotManagementAccountException" => crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::CallerIsNotManagementAccountException({
+        "CallerIsNotManagementAccountException" => super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::CallerIsNotManagementAccountException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CallerIsNotManagementAccountExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_caller_is_not_management_account_exception::de_caller_is_not_management_account_exception_xml_err(_response_body, output).map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::types::error::builders::CallerIsNotManagementAccountExceptionBuilder::default();
+                    output = super::protocol_serde::shape_caller_is_not_management_account_exception::de_caller_is_not_management_account_exception_xml_err(_response_body, output).map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -58,13 +58,13 @@
                                                         }
             tmp
         }),
-        "OrganizationNotFoundException" => crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::OrganizationNotFoundException({
+        "OrganizationNotFoundException" => super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::OrganizationNotFoundException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationNotFoundExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_not_found_exception::de_organization_not_found_exception_xml_err(_response_body, output).map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::types::error::builders::OrganizationNotFoundExceptionBuilder::default();
+                    output = super::protocol_serde::shape_organization_not_found_exception::de_organization_not_found_exception_xml_err(_response_body, output).map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -74,13 +74,13 @@
                                                         }
             tmp
         }),
-        "OrganizationNotInAllFeaturesModeException" => crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::OrganizationNotInAllFeaturesModeException({
+        "OrganizationNotInAllFeaturesModeException" => super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::OrganizationNotInAllFeaturesModeException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OrganizationNotInAllFeaturesModeExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_organization_not_in_all_features_mode_exception::de_organization_not_in_all_features_mode_exception_xml_err(_response_body, output).map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::types::error::builders::OrganizationNotInAllFeaturesModeExceptionBuilder::default();
+                    output = super::protocol_serde::shape_organization_not_in_all_features_mode_exception::de_organization_not_in_all_features_mode_exception_xml_err(_response_body, output).map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -90,13 +90,13 @@
                                                         }
             tmp
         }),
-        "ServiceAccessNotEnabledException" => crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::ServiceAccessNotEnabledException({
+        "ServiceAccessNotEnabledException" => super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::ServiceAccessNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp =
                  {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceAccessNotEnabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_service_access_not_enabled_exception::de_service_access_not_enabled_exception_xml_err(_response_body, output).map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+                    let mut output = super::types::error::builders::ServiceAccessNotEnabledExceptionBuilder::default();
+                    output = super::protocol_serde::shape_service_access_not_enabled_exception::de_service_access_not_enabled_exception_xml_err(_response_body, output).map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 }
@@ -106,7 +106,7 @@
                                                         }
             tmp
         }),
-        _ => crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::generic(generic)
+        _ => super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::generic(generic)
     })
 }

@@ -116,17 +116,17 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementOutput,
-    crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError,
+    super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementOutput,
+    super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::enable_organizations_root_credentials_management::builders::EnableOrganizationsRootCredentialsManagementOutputBuilder::default();
-        output = crate::protocol_serde::shape_enable_organizations_root_credentials_management::de_enable_organizations_root_credentials_management(
+        let mut output = super::operation::enable_organizations_root_credentials_management::builders::EnableOrganizationsRootCredentialsManagementOutputBuilder::default();
+        output = super::protocol_serde::shape_enable_organizations_root_credentials_management::de_enable_organizations_root_credentials_management(
             _response_body,
             output,
         )
-        .map_err(crate::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
+        .map_err(super::operation::enable_organizations_root_credentials_management::EnableOrganizationsRootCredentialsManagementError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -135,9 +135,9 @@
 #[allow(unused_mut)]
 pub fn de_enable_organizations_root_credentials_management(
     inp: &[u8],
-    mut builder: crate::operation::enable_organizations_root_credentials_management::builders::EnableOrganizationsRootCredentialsManagementOutputBuilder,
+    mut builder: super::operation::enable_organizations_root_credentials_management::builders::EnableOrganizationsRootCredentialsManagementOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::enable_organizations_root_credentials_management::builders::EnableOrganizationsRootCredentialsManagementOutputBuilder,
+    super::operation::enable_organizations_root_credentials_management::builders::EnableOrganizationsRootCredentialsManagementOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -178,7 +178,7 @@
             s if s.matches("EnabledFeatures") /* EnabledFeatures com.amazonaws.iam.synthetic#EnableOrganizationsRootCredentialsManagementOutput$EnabledFeatures */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_features_list_type::de_features_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_features_list_type::de_features_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
