--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_account_aliases::ListAccountAliasesOutput,
-    crate::operation::list_account_aliases::ListAccountAliasesError,
+    super::super::operation::list_account_aliases::ListAccountAliasesOutput,
+    super::super::operation::list_account_aliases::ListAccountAliasesError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_account_aliases::ListAccountAliasesError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_account_aliases::ListAccountAliasesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_account_aliases::ListAccountAliasesError::unhandled(generic)),
+        None => return Err(super::super::operation::list_account_aliases::ListAccountAliasesError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ServiceFailure" => crate::operation::list_account_aliases::ListAccountAliasesError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::list_account_aliases::ListAccountAliasesError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_account_aliases::ListAccountAliasesError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_account_aliases::ListAccountAliasesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,7 +35,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_account_aliases::ListAccountAliasesError::generic(generic),
+        _ => super::super::operation::list_account_aliases::ListAccountAliasesError::generic(generic),
     })
 }

@@ -45,18 +45,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_account_aliases::ListAccountAliasesOutput,
-    crate::operation::list_account_aliases::ListAccountAliasesError,
+    super::super::operation::list_account_aliases::ListAccountAliasesOutput,
+    super::super::operation::list_account_aliases::ListAccountAliasesError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_account_aliases::builders::ListAccountAliasesOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_account_aliases::de_list_account_aliases(_response_body, output)
-            .map_err(crate::operation::list_account_aliases::ListAccountAliasesError::unhandled)?;
+        let mut output = super::super::operation::list_account_aliases::builders::ListAccountAliasesOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_account_aliases::de_list_account_aliases(_response_body, output)
+            .map_err(super::super::operation::list_account_aliases::ListAccountAliasesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_account_aliases_output_output_correct_errors(output)
+        super::super::serde_util::list_account_aliases_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_account_aliases::ListAccountAliasesError::unhandled)?
+            .map_err(super::super::operation::list_account_aliases::ListAccountAliasesError::unhandled)?
     })
 }

@@ -63,8 +63,8 @@
 #[allow(unused_mut)]
 pub fn de_list_account_aliases(
     inp: &[u8],
-    mut builder: crate::operation::list_account_aliases::builders::ListAccountAliasesOutputBuilder,
-) -> std::result::Result<crate::operation::list_account_aliases::builders::ListAccountAliasesOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
+    mut builder: super::super::operation::list_account_aliases::builders::ListAccountAliasesOutputBuilder,
+) -> std::result::Result<super::super::operation::list_account_aliases::builders::ListAccountAliasesOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError>
 {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

@@ -91,7 +91,7 @@
             s if s.matches("AccountAliases") /* AccountAliases com.amazonaws.iam.synthetic#ListAccountAliasesOutput$AccountAliases */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_account_alias_list_type::de_account_alias_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_account_alias_list_type::de_account_alias_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
