--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_policy_granting_service_access(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::PolicyGrantingServiceAccess, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::PolicyGrantingServiceAccess, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::PolicyGrantingServiceAccess::builder();
+    let mut builder = super::super::types::PolicyGrantingServiceAccess::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("PolicyName") /* PolicyName com.amazonaws.iam#PolicyGrantingServiceAccess$PolicyName */ =>  {
@@ -27,8 +27,8 @@
             s if s.matches("PolicyType") /* PolicyType com.amazonaws.iam#PolicyGrantingServiceAccess$PolicyType */ =>  {
                 let var_2 =
                     Some(
-                        Result::<crate::types::PolicyType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::PolicyType::from(
+                        Result::<super::super::types::PolicyType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::PolicyType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -54,8 +54,8 @@
             s if s.matches("EntityType") /* EntityType com.amazonaws.iam#PolicyGrantingServiceAccess$EntityType */ =>  {
                 let var_4 =
                     Some(
-                        Result::<crate::types::PolicyOwnerEntityType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::PolicyOwnerEntityType::from(
+                        Result::<super::super::types::PolicyOwnerEntityType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::PolicyOwnerEntityType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -81,7 +81,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::policy_granting_service_access_correct_errors(builder)
+    Ok(super::super::serde_util::policy_granting_service_access_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
