--- original
+++ modified
@@ -4,27 +4,27 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_group_policies::ListGroupPoliciesOutput, crate::operation::list_group_policies::ListGroupPoliciesError>
+) -> std::result::Result<super::super::operation::list_group_policies::ListGroupPoliciesOutput, super::super::operation::list_group_policies::ListGroupPoliciesError>
 {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_group_policies::ListGroupPoliciesError::unhandled(generic)),
+        None => return Err(super::super::operation::list_group_policies::ListGroupPoliciesError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::list_group_policies::ListGroupPoliciesError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::list_group_policies::ListGroupPoliciesError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -33,13 +33,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::list_group_policies::ListGroupPoliciesError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::list_group_policies::ListGroupPoliciesError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -48,7 +48,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_group_policies::ListGroupPoliciesError::generic(generic),
+        _ => super::super::operation::list_group_policies::ListGroupPoliciesError::generic(generic),
     })
 }

@@ -57,17 +57,17 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_group_policies::ListGroupPoliciesOutput, crate::operation::list_group_policies::ListGroupPoliciesError>
+) -> std::result::Result<super::super::operation::list_group_policies::ListGroupPoliciesOutput, super::super::operation::list_group_policies::ListGroupPoliciesError>
 {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_group_policies::builders::ListGroupPoliciesOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_group_policies::de_list_group_policies(_response_body, output)
-            .map_err(crate::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
+        let mut output = super::super::operation::list_group_policies::builders::ListGroupPoliciesOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_group_policies::de_list_group_policies(_response_body, output)
+            .map_err(super::super::operation::list_group_policies::ListGroupPoliciesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_group_policies_output_output_correct_errors(output)
+        super::super::serde_util::list_group_policies_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_group_policies::ListGroupPoliciesError::unhandled)?
+            .map_err(super::super::operation::list_group_policies::ListGroupPoliciesError::unhandled)?
     })
 }

@@ -74,8 +74,8 @@
 #[allow(unused_mut)]
 pub fn de_list_group_policies(
     inp: &[u8],
-    mut builder: crate::operation::list_group_policies::builders::ListGroupPoliciesOutputBuilder,
-) -> std::result::Result<crate::operation::list_group_policies::builders::ListGroupPoliciesOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::super::operation::list_group_policies::builders::ListGroupPoliciesOutputBuilder,
+) -> std::result::Result<super::super::operation::list_group_policies::builders::ListGroupPoliciesOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -101,7 +101,7 @@
             s if s.matches("PolicyNames") /* PolicyNames com.amazonaws.iam.synthetic#ListGroupPoliciesOutput$PolicyNames */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_policy_name_list_type::de_policy_name_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_policy_name_list_type::de_policy_name_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
