--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_organizations_access_report::GetOrganizationsAccessReportOutput,
-    crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError,
+    super::operation::get_organizations_access_report::GetOrganizationsAccessReportOutput,
+    super::operation::get_organizations_access_report::GetOrganizationsAccessReportError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled(generic)),
+        None => return Err(super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,7 +35,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::generic(generic),
+        _ => super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::generic(generic),
     })
 }

@@ -45,18 +45,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_organizations_access_report::GetOrganizationsAccessReportOutput,
-    crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError,
+    super::operation::get_organizations_access_report::GetOrganizationsAccessReportOutput,
+    super::operation::get_organizations_access_report::GetOrganizationsAccessReportError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_organizations_access_report::builders::GetOrganizationsAccessReportOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_organizations_access_report::de_get_organizations_access_report(_response_body, output)
-            .map_err(crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?;
+        let mut output = super::operation::get_organizations_access_report::builders::GetOrganizationsAccessReportOutputBuilder::default();
+        output = super::protocol_serde::shape_get_organizations_access_report::de_get_organizations_access_report(_response_body, output)
+            .map_err(super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_organizations_access_report_output_output_correct_errors(output)
+        super::serde_util::get_organizations_access_report_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?
+            .map_err(super::operation::get_organizations_access_report::GetOrganizationsAccessReportError::unhandled)?
     })
 }

@@ -63,9 +63,9 @@
 #[allow(unused_mut)]
 pub fn de_get_organizations_access_report(
     inp: &[u8],
-    mut builder: crate::operation::get_organizations_access_report::builders::GetOrganizationsAccessReportOutputBuilder,
+    mut builder: super::operation::get_organizations_access_report::builders::GetOrganizationsAccessReportOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::get_organizations_access_report::builders::GetOrganizationsAccessReportOutputBuilder,
+    super::operation::get_organizations_access_report::builders::GetOrganizationsAccessReportOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -93,8 +93,8 @@
             s if s.matches("JobStatus") /* JobStatus com.amazonaws.iam.synthetic#GetOrganizationsAccessReportOutput$JobStatus */ =>  {
                 let var_1 =
                     Some(
-                        Result::<crate::types::JobStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::JobStatusType::from(
+                        Result::<super::types::JobStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::JobStatusType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -165,7 +165,7 @@
             s if s.matches("AccessDetails") /* AccessDetails com.amazonaws.iam.synthetic#GetOrganizationsAccessReportOutput$AccessDetails */ =>  {
                 let var_6 =
                     Some(
-                        crate::protocol_serde::shape_access_details::de_access_details(&mut tag, depth + 1)
+                        super::protocol_serde::shape_access_details::de_access_details(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -203,7 +203,7 @@
             s if s.matches("ErrorDetails") /* ErrorDetails com.amazonaws.iam.synthetic#GetOrganizationsAccessReportOutput$ErrorDetails */ =>  {
                 let var_9 =
                     Some(
-                        crate::protocol_serde::shape_error_details::de_error_details(&mut tag, depth + 1)
+                        super::protocol_serde::shape_error_details::de_error_details(&mut tag, depth + 1)
                         ?
                     )
                 ;
