--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessOutput,
-    crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError,
+    super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessOutput,
+    super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled(generic)),
+        None => return Err(super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidInput" => crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::InvalidInputException({
+        "InvalidInput" => super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,7 +50,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::generic(generic),
+        _ => super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::generic(generic),
     })
 }

@@ -60,18 +60,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessOutput,
-    crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError,
+    super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessOutput,
+    super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_policies_granting_service_access::de_list_policies_granting_service_access(_response_body, output)
-            .map_err(crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
+        let mut output = super::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessOutputBuilder::default();
+        output = super::protocol_serde::shape_list_policies_granting_service_access::de_list_policies_granting_service_access(_response_body, output)
+            .map_err(super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_policies_granting_service_access_output_output_correct_errors(output)
+        super::serde_util::list_policies_granting_service_access_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?
+            .map_err(super::operation::list_policies_granting_service_access::ListPoliciesGrantingServiceAccessError::unhandled)?
     })
 }

@@ -78,9 +78,9 @@
 #[allow(unused_mut)]
 pub fn de_list_policies_granting_service_access(
     inp: &[u8],
-    mut builder: crate::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessOutputBuilder,
+    mut builder: super::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessOutputBuilder,
+    super::operation::list_policies_granting_service_access::builders::ListPoliciesGrantingServiceAccessOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -108,7 +108,7 @@
             s if s.matches("PoliciesGrantingServiceAccess") /* PoliciesGrantingServiceAccess com.amazonaws.iam.synthetic#ListPoliciesGrantingServiceAccessOutput$PoliciesGrantingServiceAccess */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_list_policy_granting_service_access_response_list_type::de_list_policy_granting_service_access_response_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_list_policy_granting_service_access_response_list_type::de_list_policy_granting_service_access_response_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
