--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_access_key_last_used(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::AccessKeyLastUsed, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::AccessKeyLastUsed, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::AccessKeyLastUsed::builder();
+    let mut builder = super::super::types::AccessKeyLastUsed::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("LastUsedDate") /* LastUsedDate com.amazonaws.iam#AccessKeyLastUsed$LastUsedDate */ =>  {
@@ -54,7 +54,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::access_key_last_used_correct_errors(builder)
+    Ok(super::super::serde_util::access_key_last_used_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
