--- original
+++ modified
@@ -2,7 +2,7 @@
 #[allow(unused_mut)]
 pub fn ser_policy_parameter(
     mut writer: ::aws_smithy_query::QueryValueWriter,
-    input: &crate::types::PolicyParameter,
+    input: &super::super::types::PolicyParameter,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     #[allow(unused_mut)]
     let mut scope_1 = writer.prefix("Name");
@@ -32,12 +32,12 @@
 pub fn de_policy_parameter(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::PolicyParameter, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::PolicyParameter, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::PolicyParameter::builder();
+    let mut builder = super::super::types::PolicyParameter::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Name") /* Name com.amazonaws.iam#PolicyParameter$Name */ =>  {
@@ -56,7 +56,7 @@
             s if s.matches("Values") /* Values com.amazonaws.iam#PolicyParameter$Values */ =>  {
                 let var_11 =
                     Some(
-                        crate::protocol_serde::shape_policy_parameter_values_list_type::de_policy_parameter_values_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_policy_parameter_values_list_type::de_policy_parameter_values_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -66,8 +66,8 @@
             s if s.matches("Type") /* Type com.amazonaws.iam#PolicyParameter$Type */ =>  {
                 let var_12 =
                     Some(
-                        Result::<crate::types::PolicyParameterTypeEnum, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::PolicyParameterTypeEnum::from(
+                        Result::<super::super::types::PolicyParameterTypeEnum, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::PolicyParameterTypeEnum::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
