--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_delegation_request(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::DelegationRequest, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::DelegationRequest, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::DelegationRequest::builder();
+    let mut builder = super::super::types::DelegationRequest::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("DelegationRequestId") /* DelegationRequestId com.amazonaws.iam#DelegationRequest$DelegationRequestId */ =>  {
@@ -66,7 +66,7 @@
             s if s.matches("Permissions") /* Permissions com.amazonaws.iam#DelegationRequest$Permissions */ =>  {
                 let var_5 =
                     Some(
-                        crate::protocol_serde::shape_delegation_permission::de_delegation_permission(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_delegation_permission::de_delegation_permission(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -89,7 +89,7 @@
             s if s.matches("RolePermissionRestrictionArns") /* RolePermissionRestrictionArns com.amazonaws.iam#DelegationRequest$RolePermissionRestrictionArns */ =>  {
                 let var_7 =
                     Some(
-                        crate::protocol_serde::shape_role_permission_restriction_arn_list_type::de_role_permission_restriction_arn_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_role_permission_restriction_arn_list_type::de_role_permission_restriction_arn_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -125,8 +125,8 @@
             s if s.matches("State") /* State com.amazonaws.iam#DelegationRequest$State */ =>  {
                 let var_10 =
                     Some(
-                        Result::<crate::types::StateType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::StateType::from(
+                        Result::<super::super::types::StateType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::StateType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
