--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_role(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::Role, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::Role, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::Role::builder();
+    let mut builder = super::types::Role::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Path") /* Path com.amazonaws.iam#Role$Path */ =>  {
@@ -121,7 +121,7 @@
             s if s.matches("PermissionsBoundary") /* PermissionsBoundary com.amazonaws.iam#Role$PermissionsBoundary */ =>  {
                 let var_9 =
                     Some(
-                        crate::protocol_serde::shape_attached_permissions_boundary::de_attached_permissions_boundary(&mut tag, depth + 1)
+                        super::protocol_serde::shape_attached_permissions_boundary::de_attached_permissions_boundary(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -131,7 +131,7 @@
             s if s.matches("Tags") /* Tags com.amazonaws.iam#Role$Tags */ =>  {
                 let var_10 =
                     Some(
-                        crate::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -141,7 +141,7 @@
             s if s.matches("RoleLastUsed") /* RoleLastUsed com.amazonaws.iam#Role$RoleLastUsed */ =>  {
                 let var_11 =
                     Some(
-                        crate::protocol_serde::shape_role_last_used::de_role_last_used(&mut tag, depth + 1)
+                        super::protocol_serde::shape_role_last_used::de_role_last_used(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -151,7 +151,7 @@
             s if s.matches("SourceRoleTemplate") /* SourceRoleTemplate com.amazonaws.iam#Role$SourceRoleTemplate */ =>  {
                 let var_12 =
                     Some(
-                        crate::protocol_serde::shape_source_role_template::de_source_role_template(&mut tag, depth + 1)
+                        super::protocol_serde::shape_source_role_template::de_source_role_template(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -161,7 +161,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::role_correct_errors(builder)
+    Ok(super::serde_util::role_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
