--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_groups::ListGroupsOutput, crate::operation::list_groups::ListGroupsError> {
+) -> std::result::Result<super::super::operation::list_groups::ListGroupsOutput, super::super::operation::list_groups::ListGroupsError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_groups::ListGroupsError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_groups::ListGroupsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_groups::ListGroupsError::unhandled(generic)),
+        None => return Err(super::super::operation::list_groups::ListGroupsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ServiceFailure" => crate::operation::list_groups::ListGroupsError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::list_groups::ListGroupsError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::list_groups::ListGroupsError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::list_groups::ListGroupsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,7 +32,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_groups::ListGroupsError::generic(generic),
+        _ => super::super::operation::list_groups::ListGroupsError::generic(generic),
     })
 }

@@ -41,16 +41,16 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_groups::ListGroupsOutput, crate::operation::list_groups::ListGroupsError> {
+) -> std::result::Result<super::super::operation::list_groups::ListGroupsOutput, super::super::operation::list_groups::ListGroupsError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_groups::builders::ListGroupsOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_groups::de_list_groups(_response_body, output)
-            .map_err(crate::operation::list_groups::ListGroupsError::unhandled)?;
+        let mut output = super::super::operation::list_groups::builders::ListGroupsOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_groups::de_list_groups(_response_body, output)
+            .map_err(super::super::operation::list_groups::ListGroupsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_groups_output_output_correct_errors(output)
+        super::super::serde_util::list_groups_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_groups::ListGroupsError::unhandled)?
+            .map_err(super::super::operation::list_groups::ListGroupsError::unhandled)?
     })
 }

@@ -57,8 +57,8 @@
 #[allow(unused_mut)]
 pub fn de_list_groups(
     inp: &[u8],
-    mut builder: crate::operation::list_groups::builders::ListGroupsOutputBuilder,
-) -> std::result::Result<crate::operation::list_groups::builders::ListGroupsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::super::operation::list_groups::builders::ListGroupsOutputBuilder,
+) -> std::result::Result<super::super::operation::list_groups::builders::ListGroupsOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -84,7 +84,7 @@
             s if s.matches("Groups") /* Groups com.amazonaws.iam.synthetic#ListGroupsOutput$Groups */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_group_list_type::de_group_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_group_list_type::de_group_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
