--- original
+++ modified
@@ -5,32 +5,32 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::create_open_id_connect_provider::CreateOpenIdConnectProviderOutput,
-    crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError,
+    super::super::operation::create_open_id_connect_provider::CreateOpenIdConnectProviderOutput,
+    super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled(generic)),
+        None => return Err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "ConcurrentModification" => {
-            crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::ConcurrentModificationException({
+            super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::ConcurrentModificationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
+                    let mut output = super::super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+                    .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -40,14 +40,14 @@
                 tmp
             })
         }
-        "EntityAlreadyExists" => crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::EntityAlreadyExistsException({
+        "EntityAlreadyExists" => super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::EntityAlreadyExistsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::EntityAlreadyExistsExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+                    super::super::protocol_serde::shape_entity_already_exists_exception::de_entity_already_exists_exception_xml_err(_response_body, output)
+                        .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -56,13 +56,13 @@
             }
             tmp
         }),
-        "InvalidInput" => crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::InvalidInputException({
+        "InvalidInput" => super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -71,13 +71,13 @@
             }
             tmp
         }),
-        "LimitExceeded" => crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::LimitExceededException({
+        "LimitExceeded" => super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+                let mut output = super::super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -87,17 +87,17 @@
             tmp
         }),
         "OpenIdIdpCommunicationError" => {
-            crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::OpenIdIdpCommunicationErrorException({
+            super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::OpenIdIdpCommunicationErrorException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OpenIdIdpCommunicationErrorExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::OpenIdIdpCommunicationErrorExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_open_id_idp_communication_error_exception::de_open_id_idp_communication_error_exception_xml_err(
+                        super::super::protocol_serde::shape_open_id_idp_communication_error_exception::de_open_id_idp_communication_error_exception_xml_err(
                             _response_body,
                             output,
                         )
-                        .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+                        .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -107,13 +107,13 @@
                 tmp
             })
         }
-        "ServiceFailure" => crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -122,7 +122,7 @@
             }
             tmp
         }),
-        _ => crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::generic(generic),
+        _ => super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::generic(generic),
     })
 }

@@ -132,14 +132,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::create_open_id_connect_provider::CreateOpenIdConnectProviderOutput,
-    crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError,
+    super::super::operation::create_open_id_connect_provider::CreateOpenIdConnectProviderOutput,
+    super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::create_open_id_connect_provider::builders::CreateOpenIdConnectProviderOutputBuilder::default();
-        output = crate::protocol_serde::shape_create_open_id_connect_provider::de_create_open_id_connect_provider(_response_body, output)
-            .map_err(crate::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
+        let mut output = super::super::operation::create_open_id_connect_provider::builders::CreateOpenIdConnectProviderOutputBuilder::default();
+        output = super::super::protocol_serde::shape_create_open_id_connect_provider::de_create_open_id_connect_provider(_response_body, output)
+            .map_err(super::super::operation::create_open_id_connect_provider::CreateOpenIDConnectProviderError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -148,9 +148,9 @@
 #[allow(unused_mut)]
 pub fn de_create_open_id_connect_provider(
     inp: &[u8],
-    mut builder: crate::operation::create_open_id_connect_provider::builders::CreateOpenIdConnectProviderOutputBuilder,
+    mut builder: super::super::operation::create_open_id_connect_provider::builders::CreateOpenIdConnectProviderOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::create_open_id_connect_provider::builders::CreateOpenIdConnectProviderOutputBuilder,
+    super::super::operation::create_open_id_connect_provider::builders::CreateOpenIdConnectProviderOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -191,7 +191,7 @@
             s if s.matches("Tags") /* Tags com.amazonaws.iam.synthetic#CreateOpenIDConnectProviderOutput$Tags */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_tag_list_type::de_tag_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
