--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::generate_credential_report::GenerateCredentialReportOutput,
-    crate::operation::generate_credential_report::GenerateCredentialReportError,
+    super::operation::generate_credential_report::GenerateCredentialReportOutput,
+    super::operation::generate_credential_report::GenerateCredentialReportError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::generate_credential_report::GenerateCredentialReportError::unhandled(
+            return Err(super::operation::generate_credential_report::GenerateCredentialReportError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "LimitExceeded" => crate::operation::generate_credential_report::GenerateCredentialReportError::LimitExceededException({
+        "LimitExceeded" => super::operation::generate_credential_report::GenerateCredentialReportError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::generate_credential_report::GenerateCredentialReportError::ServiceFailureException({
+        "ServiceFailure" => super::operation::generate_credential_report::GenerateCredentialReportError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,7 +54,7 @@
             }
             tmp
         }),
-        _ => crate::operation::generate_credential_report::GenerateCredentialReportError::generic(generic),
+        _ => super::operation::generate_credential_report::GenerateCredentialReportError::generic(generic),
     })
 }

@@ -64,14 +64,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::generate_credential_report::GenerateCredentialReportOutput,
-    crate::operation::generate_credential_report::GenerateCredentialReportError,
+    super::operation::generate_credential_report::GenerateCredentialReportOutput,
+    super::operation::generate_credential_report::GenerateCredentialReportError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::generate_credential_report::builders::GenerateCredentialReportOutputBuilder::default();
-        output = crate::protocol_serde::shape_generate_credential_report::de_generate_credential_report(_response_body, output)
-            .map_err(crate::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
+        let mut output = super::operation::generate_credential_report::builders::GenerateCredentialReportOutputBuilder::default();
+        output = super::protocol_serde::shape_generate_credential_report::de_generate_credential_report(_response_body, output)
+            .map_err(super::operation::generate_credential_report::GenerateCredentialReportError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -80,9 +80,9 @@
 #[allow(unused_mut)]
 pub fn de_generate_credential_report(
     inp: &[u8],
-    mut builder: crate::operation::generate_credential_report::builders::GenerateCredentialReportOutputBuilder,
+    mut builder: super::operation::generate_credential_report::builders::GenerateCredentialReportOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::generate_credential_report::builders::GenerateCredentialReportOutputBuilder,
+    super::operation::generate_credential_report::builders::GenerateCredentialReportOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -110,8 +110,8 @@
             s if s.matches("State") /* State com.amazonaws.iam.synthetic#GenerateCredentialReportOutput$State */ =>  {
                 let var_1 =
                     Some(
-                        Result::<crate::types::ReportStateType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ReportStateType::from(
+                        Result::<super::types::ReportStateType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ReportStateType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
