--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialOutput,
-    crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError,
+    super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialOutput,
+    super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::unhandled(generic)),
+        None => return Err(super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,7 +35,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::generic(generic),
+        _ => super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError::generic(generic),
     })
 }

@@ -45,12 +45,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialOutput,
-    crate::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError,
+    super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialOutput,
+    super::super::operation::update_service_specific_credential::UpdateServiceSpecificCredentialError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_service_specific_credential::builders::UpdateServiceSpecificCredentialOutputBuilder::default();
+        let mut output = super::super::operation::update_service_specific_credential::builders::UpdateServiceSpecificCredentialOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
