--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_access_key::CreateAccessKeyOutput, crate::operation::create_access_key::CreateAccessKeyError> {
+) -> std::result::Result<super::operation::create_access_key::CreateAccessKeyOutput, super::operation::create_access_key::CreateAccessKeyError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::create_access_key::CreateAccessKeyError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::create_access_key::CreateAccessKeyError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::create_access_key::CreateAccessKeyError::unhandled(generic)),
+        None => return Err(super::operation::create_access_key::CreateAccessKeyError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "LimitExceeded" => crate::operation::create_access_key::CreateAccessKeyError::LimitExceededException({
+        "LimitExceeded" => super::operation::create_access_key::CreateAccessKeyError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_access_key::CreateAccessKeyError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_access_key::CreateAccessKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::create_access_key::CreateAccessKeyError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::create_access_key::CreateAccessKeyError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_access_key::CreateAccessKeyError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_access_key::CreateAccessKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::create_access_key::CreateAccessKeyError::ServiceFailureException({
+        "ServiceFailure" => super::operation::create_access_key::CreateAccessKeyError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::create_access_key::CreateAccessKeyError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::create_access_key::CreateAccessKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,7 +62,7 @@
             }
             tmp
         }),
-        _ => crate::operation::create_access_key::CreateAccessKeyError::generic(generic),
+        _ => super::operation::create_access_key::CreateAccessKeyError::generic(generic),
     })
 }

@@ -71,14 +71,14 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_access_key::CreateAccessKeyOutput, crate::operation::create_access_key::CreateAccessKeyError> {
+) -> std::result::Result<super::operation::create_access_key::CreateAccessKeyOutput, super::operation::create_access_key::CreateAccessKeyError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::create_access_key::builders::CreateAccessKeyOutputBuilder::default();
-        output = crate::protocol_serde::shape_create_access_key::de_create_access_key(_response_body, output)
-            .map_err(crate::operation::create_access_key::CreateAccessKeyError::unhandled)?;
+        let mut output = super::operation::create_access_key::builders::CreateAccessKeyOutputBuilder::default();
+        output = super::protocol_serde::shape_create_access_key::de_create_access_key(_response_body, output)
+            .map_err(super::operation::create_access_key::CreateAccessKeyError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::create_access_key_output_output_correct_errors(output).build()
+        super::serde_util::create_access_key_output_output_correct_errors(output).build()
     })
 }

@@ -85,8 +85,8 @@
 #[allow(unused_mut)]
 pub fn de_create_access_key(
     inp: &[u8],
-    mut builder: crate::operation::create_access_key::builders::CreateAccessKeyOutputBuilder,
-) -> std::result::Result<crate::operation::create_access_key::builders::CreateAccessKeyOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::operation::create_access_key::builders::CreateAccessKeyOutputBuilder,
+) -> std::result::Result<super::operation::create_access_key::builders::CreateAccessKeyOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -112,7 +112,7 @@
             s if s.matches("AccessKey") /* AccessKey com.amazonaws.iam.synthetic#CreateAccessKeyOutput$AccessKey */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_access_key::de_access_key(&mut tag, depth + 1)
+                        super::protocol_serde::shape_access_key::de_access_key(&mut tag, depth + 1)
                         ?
                     )
                 ;
