--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_source_role_template(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::SourceRoleTemplate, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::SourceRoleTemplate, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::SourceRoleTemplate::builder();
+    let mut builder = super::super::types::SourceRoleTemplate::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("TemplateArn") /* TemplateArn com.amazonaws.iam#SourceRoleTemplate$TemplateArn */ =>  {
@@ -42,7 +42,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::source_role_template_correct_errors(builder)
+    Ok(super::super::serde_util::source_role_template_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
