--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_role_description::UpdateRoleDescriptionOutput,
-    crate::operation::update_role_description::UpdateRoleDescriptionError,
+    super::operation::update_role_description::UpdateRoleDescriptionOutput,
+    super::operation::update_role_description::UpdateRoleDescriptionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_role_description::UpdateRoleDescriptionError::unhandled(generic)),
+        None => return Err(super::operation::update_role_description::UpdateRoleDescriptionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::update_role_description::UpdateRoleDescriptionError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::update_role_description::UpdateRoleDescriptionError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::update_role_description::UpdateRoleDescriptionError::ServiceFailureException({
+        "ServiceFailure" => super::operation::update_role_description::UpdateRoleDescriptionError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "UnmodifiableEntity" => crate::operation::update_role_description::UpdateRoleDescriptionError::UnmodifiableEntityException({
+        "UnmodifiableEntity" => super::operation::update_role_description::UpdateRoleDescriptionError::UnmodifiableEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnmodifiableEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_unmodifiable_entity_exception::de_unmodifiable_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
+                let mut output = super::types::error::builders::UnmodifiableEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_unmodifiable_entity_exception::de_unmodifiable_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,7 +65,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_role_description::UpdateRoleDescriptionError::generic(generic),
+        _ => super::operation::update_role_description::UpdateRoleDescriptionError::generic(generic),
     })
 }

@@ -75,14 +75,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_role_description::UpdateRoleDescriptionOutput,
-    crate::operation::update_role_description::UpdateRoleDescriptionError,
+    super::operation::update_role_description::UpdateRoleDescriptionOutput,
+    super::operation::update_role_description::UpdateRoleDescriptionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_role_description::builders::UpdateRoleDescriptionOutputBuilder::default();
-        output = crate::protocol_serde::shape_update_role_description::de_update_role_description(_response_body, output)
-            .map_err(crate::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
+        let mut output = super::operation::update_role_description::builders::UpdateRoleDescriptionOutputBuilder::default();
+        output = super::protocol_serde::shape_update_role_description::de_update_role_description(_response_body, output)
+            .map_err(super::operation::update_role_description::UpdateRoleDescriptionError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -91,9 +91,9 @@
 #[allow(unused_mut)]
 pub fn de_update_role_description(
     inp: &[u8],
-    mut builder: crate::operation::update_role_description::builders::UpdateRoleDescriptionOutputBuilder,
+    mut builder: super::operation::update_role_description::builders::UpdateRoleDescriptionOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::update_role_description::builders::UpdateRoleDescriptionOutputBuilder,
+    super::operation::update_role_description::builders::UpdateRoleDescriptionOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -121,7 +121,7 @@
             s if s.matches("Role") /* Role com.amazonaws.iam.synthetic#UpdateRoleDescriptionOutput$Role */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_role::de_role(&mut tag, depth + 1)
+                        super::protocol_serde::shape_role::de_role(&mut tag, depth + 1)
                         ?
                     )
                 ;
