--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_list_policies_granting_service_access_entry(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::ListPoliciesGrantingServiceAccessEntry, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::ListPoliciesGrantingServiceAccessEntry, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::ListPoliciesGrantingServiceAccessEntry::builder();
+    let mut builder = super::types::ListPoliciesGrantingServiceAccessEntry::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("ServiceNamespace") /* ServiceNamespace com.amazonaws.iam#ListPoliciesGrantingServiceAccessEntry$ServiceNamespace */ =>  {
@@ -27,7 +27,7 @@
             s if s.matches("Policies") /* Policies com.amazonaws.iam#ListPoliciesGrantingServiceAccessEntry$Policies */ =>  {
                 let var_2 =
                     Some(
-                        crate::protocol_serde::shape_policy_granting_service_access_list_type::de_policy_granting_service_access_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_policy_granting_service_access_list_type::de_policy_granting_service_access_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
