--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_delegation_request::GetDelegationRequestOutput,
-    crate::operation::get_delegation_request::GetDelegationRequestError,
+    super::operation::get_delegation_request::GetDelegationRequestOutput,
+    super::operation::get_delegation_request::GetDelegationRequestError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_delegation_request::GetDelegationRequestError::unhandled(generic)),
+        None => return Err(super::operation::get_delegation_request::GetDelegationRequestError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::get_delegation_request::GetDelegationRequestError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::get_delegation_request::GetDelegationRequestError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::get_delegation_request::GetDelegationRequestError::ServiceFailureException({
+        "ServiceFailure" => super::operation::get_delegation_request::GetDelegationRequestError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,7 +50,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_delegation_request::GetDelegationRequestError::generic(generic),
+        _ => super::operation::get_delegation_request::GetDelegationRequestError::generic(generic),
     })
 }

@@ -60,14 +60,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_delegation_request::GetDelegationRequestOutput,
-    crate::operation::get_delegation_request::GetDelegationRequestError,
+    super::operation::get_delegation_request::GetDelegationRequestOutput,
+    super::operation::get_delegation_request::GetDelegationRequestError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_delegation_request::builders::GetDelegationRequestOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_delegation_request::de_get_delegation_request(_response_body, output)
-            .map_err(crate::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
+        let mut output = super::operation::get_delegation_request::builders::GetDelegationRequestOutputBuilder::default();
+        output = super::protocol_serde::shape_get_delegation_request::de_get_delegation_request(_response_body, output)
+            .map_err(super::operation::get_delegation_request::GetDelegationRequestError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -76,9 +76,9 @@
 #[allow(unused_mut)]
 pub fn de_get_delegation_request(
     inp: &[u8],
-    mut builder: crate::operation::get_delegation_request::builders::GetDelegationRequestOutputBuilder,
+    mut builder: super::operation::get_delegation_request::builders::GetDelegationRequestOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::get_delegation_request::builders::GetDelegationRequestOutputBuilder,
+    super::operation::get_delegation_request::builders::GetDelegationRequestOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -106,7 +106,7 @@
             s if s.matches("DelegationRequest") /* DelegationRequest com.amazonaws.iam.synthetic#GetDelegationRequestOutput$DelegationRequest */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_delegation_request::de_delegation_request(&mut tag, depth + 1)
+                        super::protocol_serde::shape_delegation_request::de_delegation_request(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -116,8 +116,8 @@
             s if s.matches("PermissionCheckStatus") /* PermissionCheckStatus com.amazonaws.iam.synthetic#GetDelegationRequestOutput$PermissionCheckStatus */ =>  {
                 let var_2 =
                     Some(
-                        Result::<crate::types::PermissionCheckStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::PermissionCheckStatusType::from(
+                        Result::<super::types::PermissionCheckStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::PermissionCheckStatusType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -130,8 +130,8 @@
             s if s.matches("PermissionCheckResult") /* PermissionCheckResult com.amazonaws.iam.synthetic#GetDelegationRequestOutput$PermissionCheckResult */ =>  {
                 let var_3 =
                     Some(
-                        Result::<crate::types::PermissionCheckResultType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::PermissionCheckResultType::from(
+                        Result::<super::types::PermissionCheckResultType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::PermissionCheckResultType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
