--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryOutput,
-    crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError,
+    super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryOutput,
+    super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled(generic)),
+        None => return Err(super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidInput" => crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::InvalidInputException({
+        "InvalidInput" => super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,14 +50,14 @@
             }
             tmp
         }),
-        "PolicyNotAttachable" => crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::PolicyNotAttachableException({
+        "PolicyNotAttachable" => super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::PolicyNotAttachableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PolicyNotAttachableExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::PolicyNotAttachableExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_policy_not_attachable_exception::de_policy_not_attachable_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
+                    super::super::protocol_serde::shape_policy_not_attachable_exception::de_policy_not_attachable_exception_xml_err(_response_body, output)
+                        .map_err(super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -66,13 +66,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -81,7 +81,7 @@
             }
             tmp
         }),
-        _ => crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::generic(generic),
+        _ => super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError::generic(generic),
     })
 }

@@ -91,12 +91,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryOutput,
-    crate::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError,
+    super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryOutput,
+    super::super::operation::put_user_permissions_boundary::PutUserPermissionsBoundaryError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::put_user_permissions_boundary::builders::PutUserPermissionsBoundaryOutputBuilder::default();
+        let mut output = super::super::operation::put_user_permissions_boundary::builders::PutUserPermissionsBoundaryOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
