--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_access_detail(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::AccessDetail, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::AccessDetail, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::AccessDetail::builder();
+    let mut builder = super::types::AccessDetail::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("ServiceName") /* ServiceName com.amazonaws.iam#AccessDetail$ServiceName */ =>  {
@@ -95,7 +95,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::access_detail_correct_errors(builder)
+    Ok(super::serde_util::access_detail_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
