--- original
+++ modified
@@ -5,29 +5,29 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationOutput,
-    crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError,
+    super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationOutput,
+    super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled(generic)),
+        None => return Err(super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "FeatureEnabled" => {
-            crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::FeatureEnabledException({
+            super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::FeatureEnabledException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::FeatureEnabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_feature_enabled_exception::de_feature_enabled_exception_xml_err(_response_body, output)
-                        .map_err(crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled)?;
+                    let mut output = super::types::error::builders::FeatureEnabledExceptionBuilder::default();
+                    output = super::protocol_serde::shape_feature_enabled_exception::de_feature_enabled_exception_xml_err(_response_body, output)
+                        .map_err(super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -37,7 +37,7 @@
                 tmp
             })
         }
-        _ => crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::generic(generic),
+        _ => super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::generic(generic),
     })
 }

@@ -47,16 +47,16 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationOutput,
-    crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError,
+    super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationOutput,
+    super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::enable_outbound_web_identity_federation::builders::EnableOutboundWebIdentityFederationOutputBuilder::default();
+            super::operation::enable_outbound_web_identity_federation::builders::EnableOutboundWebIdentityFederationOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_enable_outbound_web_identity_federation::de_enable_outbound_web_identity_federation(_response_body, output)
-                .map_err(crate::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled)?;
+            super::protocol_serde::shape_enable_outbound_web_identity_federation::de_enable_outbound_web_identity_federation(_response_body, output)
+                .map_err(super::operation::enable_outbound_web_identity_federation::EnableOutboundWebIdentityFederationError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -65,9 +65,9 @@
 #[allow(unused_mut)]
 pub fn de_enable_outbound_web_identity_federation(
     inp: &[u8],
-    mut builder: crate::operation::enable_outbound_web_identity_federation::builders::EnableOutboundWebIdentityFederationOutputBuilder,
+    mut builder: super::operation::enable_outbound_web_identity_federation::builders::EnableOutboundWebIdentityFederationOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::enable_outbound_web_identity_federation::builders::EnableOutboundWebIdentityFederationOutputBuilder,
+    super::operation::enable_outbound_web_identity_federation::builders::EnableOutboundWebIdentityFederationOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
