--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_service_last_accessed(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::ServiceLastAccessed, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::ServiceLastAccessed, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::ServiceLastAccessed::builder();
+    let mut builder = super::types::ServiceLastAccessed::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("ServiceName") /* ServiceName com.amazonaws.iam#ServiceLastAccessed$ServiceName */ =>  {
@@ -95,7 +95,7 @@
             s if s.matches("TrackedActionsLastAccessed") /* TrackedActionsLastAccessed com.amazonaws.iam#ServiceLastAccessed$TrackedActionsLastAccessed */ =>  {
                 let var_7 =
                     Some(
-                        crate::protocol_serde::shape_tracked_actions_last_accessed::de_tracked_actions_last_accessed(&mut tag, depth + 1)
+                        super::protocol_serde::shape_tracked_actions_last_accessed::de_tracked_actions_last_accessed(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -105,7 +105,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::service_last_accessed_correct_errors(builder)
+    Ok(super::serde_util::service_last_accessed_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
