--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_parameter_definition(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::ParameterDefinition, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::ParameterDefinition, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::ParameterDefinition::builder();
+    let mut builder = super::super::types::ParameterDefinition::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Name") /* Name com.amazonaws.iam#ParameterDefinition$Name */ =>  {
@@ -27,8 +27,8 @@
             s if s.matches("Type") /* Type com.amazonaws.iam#ParameterDefinition$Type */ =>  {
                 let var_2 =
                     Some(
-                        Result::<crate::types::ParameterTypeType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ParameterTypeType::from(
+                        Result::<super::super::types::ParameterTypeType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::ParameterTypeType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -110,7 +110,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::parameter_definition_correct_errors(builder)
+    Ok(super::super::serde_util::parameter_definition_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
