--- original
+++ modified
@@ -2,7 +2,7 @@
 pub fn de_service_specific_credentials_list_type(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<::std::vec::Vec<crate::types::ServiceSpecificCredentialMetadata>, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<::std::vec::Vec<super::super::types::ServiceSpecificCredentialMetadata>, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
@@ -11,7 +11,7 @@
         match tag.start_el() {
             s if s.matches("member") /* member com.amazonaws.iam#ServiceSpecificCredentialsListType$member */ =>  {
                 out.push(
-                    crate::protocol_serde::shape_service_specific_credential_metadata::de_service_specific_credential_metadata(&mut tag, depth + 1)
+                    super::super::protocol_serde::shape_service_specific_credential_metadata::de_service_specific_credential_metadata(&mut tag, depth + 1)
                     ?
                 );
             }
