--- original
+++ modified
@@ -5,12 +5,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesOutput,
-    crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError,
+    super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesOutput,
+    super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
@@ -17,7 +17,7 @@
         Some(code) => code,
         None => {
             return Err(
-                crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled(generic),
+                super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled(generic),
             )
         }
     };
@@ -25,13 +25,13 @@
     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InvalidInput" => {
-            crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::InvalidInputException(
+            super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::InvalidInputException(
                 {
                     #[allow(unused_mut)]
                     let mut tmp = {
                         #[allow(unused_mut)]
-                        let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                        output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output).map_err(crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
+                        let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                        output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output).map_err(super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
                         let output = output.meta(generic);
                         output.build()
                     };
@@ -43,13 +43,13 @@
             )
         }
         "NoSuchEntity" => {
-            crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::NoSuchEntityException(
+            super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::NoSuchEntityException(
                 {
                     #[allow(unused_mut)]
                     let mut tmp = {
                         #[allow(unused_mut)]
-                        let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                        output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output).map_err(crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
+                        let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                        output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output).map_err(super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
                         let output = output.meta(generic);
                         output.build()
                     };
@@ -60,7 +60,7 @@
                 },
             )
         }
-        _ => crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::generic(generic),
+        _ => super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::generic(generic),
     })
 }

@@ -70,21 +70,21 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesOutput,
-    crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError,
+    super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesOutput,
+    super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_service_last_accessed_details_with_entities::builders::GetServiceLastAccessedDetailsWithEntitiesOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_service_last_accessed_details_with_entities::de_get_service_last_accessed_details_with_entities(
+        let mut output = super::operation::get_service_last_accessed_details_with_entities::builders::GetServiceLastAccessedDetailsWithEntitiesOutputBuilder::default();
+        output = super::protocol_serde::shape_get_service_last_accessed_details_with_entities::de_get_service_last_accessed_details_with_entities(
             _response_body,
             output,
         )
-        .map_err(crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
+        .map_err(super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_service_last_accessed_details_with_entities_output_output_correct_errors(output)
+        super::serde_util::get_service_last_accessed_details_with_entities_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?
+            .map_err(super::operation::get_service_last_accessed_details_with_entities::GetServiceLastAccessedDetailsWithEntitiesError::unhandled)?
     })
 }

@@ -91,9 +91,9 @@
 #[allow(unused_mut)]
 pub fn de_get_service_last_accessed_details_with_entities(
     inp: &[u8],
-    mut builder: crate::operation::get_service_last_accessed_details_with_entities::builders::GetServiceLastAccessedDetailsWithEntitiesOutputBuilder,
+    mut builder: super::operation::get_service_last_accessed_details_with_entities::builders::GetServiceLastAccessedDetailsWithEntitiesOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::get_service_last_accessed_details_with_entities::builders::GetServiceLastAccessedDetailsWithEntitiesOutputBuilder,
+    super::operation::get_service_last_accessed_details_with_entities::builders::GetServiceLastAccessedDetailsWithEntitiesOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -121,8 +121,8 @@
             s if s.matches("JobStatus") /* JobStatus com.amazonaws.iam.synthetic#GetServiceLastAccessedDetailsWithEntitiesOutput$JobStatus */ =>  {
                 let var_1 =
                     Some(
-                        Result::<crate::types::JobStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::JobStatusType::from(
+                        Result::<super::types::JobStatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::JobStatusType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -163,7 +163,7 @@
             s if s.matches("EntityDetailsList") /* EntityDetailsList com.amazonaws.iam.synthetic#GetServiceLastAccessedDetailsWithEntitiesOutput$EntityDetailsList */ =>  {
                 let var_4 =
                     Some(
-                        crate::protocol_serde::shape_entity_details_list_type::de_entity_details_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_entity_details_list_type::de_entity_details_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -201,7 +201,7 @@
             s if s.matches("Error") /* Error com.amazonaws.iam.synthetic#GetServiceLastAccessedDetailsWithEntitiesOutput$Error */ =>  {
                 let var_7 =
                     Some(
-                        crate::protocol_serde::shape_error_details::de_error_details(&mut tag, depth + 1)
+                        super::protocol_serde::shape_error_details::de_error_details(&mut tag, depth + 1)
                         ?
                     )
                 ;
