--- original
+++ modified
@@ -2,7 +2,7 @@
 pub fn de_summary_map_type(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<::std::collections::HashMap<crate::types::SummaryKeyType, i32>, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<::std::collections::HashMap<super::types::SummaryKeyType, i32>, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
@@ -10,7 +10,7 @@
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("entry") => {
-                crate::protocol_serde::shape_summary_map_type::de_summary_map_type_entry(&mut tag, &mut out, depth)?;
+                super::protocol_serde::shape_summary_map_type::de_summary_map_type_entry(&mut tag, &mut out, depth)?;
             }
             _ => {}
         }
@@ -20,20 +20,20 @@

 pub fn de_summary_map_type_entry(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
-    out: &mut ::std::collections::HashMap<crate::types::SummaryKeyType, i32>,
+    out: &mut ::std::collections::HashMap<super::types::SummaryKeyType, i32>,
     depth: u32,
 ) -> ::std::result::Result<(), ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
-    let mut k: Option<crate::types::SummaryKeyType> = None;
+    let mut k: Option<super::types::SummaryKeyType> = None;
     let mut v: Option<i32> = None;
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("key") /* key com.amazonaws.iam#summaryMapType$key */ =>  {
                 k = Some(
-                    Result::<crate::types::SummaryKeyType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                        crate::types::SummaryKeyType::from(
+                    Result::<super::types::SummaryKeyType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                        super::types::SummaryKeyType::from(
                             ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                         )
                     )
