--- original
+++ modified
@@ -5,31 +5,31 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::send_delegation_token::SendDelegationTokenOutput,
-    crate::operation::send_delegation_token::SendDelegationTokenError,
+    super::operation::send_delegation_token::SendDelegationTokenOutput,
+    super::operation::send_delegation_token::SendDelegationTokenError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::send_delegation_token::SendDelegationTokenError::unhandled(generic)),
+        None => return Err(super::operation::send_delegation_token::SendDelegationTokenError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConcurrentModification" => crate::operation::send_delegation_token::SendDelegationTokenError::ConcurrentModificationException({
+        "ConcurrentModification" => super::operation::send_delegation_token::SendDelegationTokenError::ConcurrentModificationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
+                let mut output = super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                output = super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
+                .map_err(super::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -38,13 +38,13 @@
             }
             tmp
         }),
-        "InvalidInput" => crate::operation::send_delegation_token::SendDelegationTokenError::InvalidInputException({
+        "InvalidInput" => super::operation::send_delegation_token::SendDelegationTokenError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -53,13 +53,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::send_delegation_token::SendDelegationTokenError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::send_delegation_token::SendDelegationTokenError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,13 +68,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::send_delegation_token::SendDelegationTokenError::ServiceFailureException({
+        "ServiceFailure" => super::operation::send_delegation_token::SendDelegationTokenError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::send_delegation_token::SendDelegationTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -83,7 +83,7 @@
             }
             tmp
         }),
-        _ => crate::operation::send_delegation_token::SendDelegationTokenError::generic(generic),
+        _ => super::operation::send_delegation_token::SendDelegationTokenError::generic(generic),
     })
 }

@@ -93,12 +93,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::send_delegation_token::SendDelegationTokenOutput,
-    crate::operation::send_delegation_token::SendDelegationTokenError,
+    super::operation::send_delegation_token::SendDelegationTokenOutput,
+    super::operation::send_delegation_token::SendDelegationTokenError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::send_delegation_token::builders::SendDelegationTokenOutputBuilder::default();
+        let mut output = super::operation::send_delegation_token::builders::SendDelegationTokenOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
