--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyOutput,
-    crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError,
+    super::operation::update_account_password_policy::UpdateAccountPasswordPolicyOutput,
+    super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled(generic)),
+        None => return Err(super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "LimitExceeded" => crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::LimitExceededException({
+        "LimitExceeded" => super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -36,16 +36,16 @@
             tmp
         }),
         "MalformedPolicyDocument" => {
-            crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::MalformedPolicyDocumentException({
+            super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::MalformedPolicyDocumentException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::MalformedPolicyDocumentExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_malformed_policy_document_exception::de_malformed_policy_document_exception_xml_err(
+                    let mut output = super::types::error::builders::MalformedPolicyDocumentExceptionBuilder::default();
+                    output = super::protocol_serde::shape_malformed_policy_document_exception::de_malformed_policy_document_exception_xml_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
+                    .map_err(super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -55,13 +55,13 @@
                 tmp
             })
         }
-        "NoSuchEntity" => crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,13 +70,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::ServiceFailureException({
+        "ServiceFailure" => super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -85,7 +85,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::generic(generic),
+        _ => super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError::generic(generic),
     })
 }

@@ -95,12 +95,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyOutput,
-    crate::operation::update_account_password_policy::UpdateAccountPasswordPolicyError,
+    super::operation::update_account_password_policy::UpdateAccountPasswordPolicyOutput,
+    super::operation::update_account_password_policy::UpdateAccountPasswordPolicyError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_account_password_policy::builders::UpdateAccountPasswordPolicyOutputBuilder::default();
+        let mut output = super::operation::update_account_password_policy::builders::UpdateAccountPasswordPolicyOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
