--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_policy_version::GetPolicyVersionOutput, crate::operation::get_policy_version::GetPolicyVersionError> {
+) -> std::result::Result<super::super::operation::get_policy_version::GetPolicyVersionOutput, super::super::operation::get_policy_version::GetPolicyVersionError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_policy_version::GetPolicyVersionError::unhandled(generic)),
+        None => return Err(super::super::operation::get_policy_version::GetPolicyVersionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidInput" => crate::operation::get_policy_version::GetPolicyVersionError::InvalidInputException({
+        "InvalidInput" => super::super::operation::get_policy_version::GetPolicyVersionError::InvalidInputException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidInputExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidInputExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_input_exception::de_invalid_input_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::get_policy_version::GetPolicyVersionError::NoSuchEntityException({
+        "NoSuchEntity" => super::super::operation::get_policy_version::GetPolicyVersionError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
+                let mut output = super::super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::get_policy_version::GetPolicyVersionError::ServiceFailureException({
+        "ServiceFailure" => super::super::operation::get_policy_version::GetPolicyVersionError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::super::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,7 +62,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_policy_version::GetPolicyVersionError::generic(generic),
+        _ => super::super::operation::get_policy_version::GetPolicyVersionError::generic(generic),
     })
 }

@@ -71,12 +71,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_policy_version::GetPolicyVersionOutput, crate::operation::get_policy_version::GetPolicyVersionError> {
+) -> std::result::Result<super::super::operation::get_policy_version::GetPolicyVersionOutput, super::super::operation::get_policy_version::GetPolicyVersionError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_policy_version::builders::GetPolicyVersionOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_policy_version::de_get_policy_version(_response_body, output)
-            .map_err(crate::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
+        let mut output = super::super::operation::get_policy_version::builders::GetPolicyVersionOutputBuilder::default();
+        output = super::super::protocol_serde::shape_get_policy_version::de_get_policy_version(_response_body, output)
+            .map_err(super::super::operation::get_policy_version::GetPolicyVersionError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -85,8 +85,8 @@
 #[allow(unused_mut)]
 pub fn de_get_policy_version(
     inp: &[u8],
-    mut builder: crate::operation::get_policy_version::builders::GetPolicyVersionOutputBuilder,
-) -> std::result::Result<crate::operation::get_policy_version::builders::GetPolicyVersionOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
+    mut builder: super::super::operation::get_policy_version::builders::GetPolicyVersionOutputBuilder,
+) -> std::result::Result<super::super::operation::get_policy_version::builders::GetPolicyVersionOutputBuilder, ::aws_smithy_xml::decode::XmlDecodeError> {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;

     #[allow(unused_mut)]
@@ -112,7 +112,7 @@
             s if s.matches("PolicyVersion") /* PolicyVersion com.amazonaws.iam.synthetic#GetPolicyVersionOutput$PolicyVersion */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_policy_version::de_policy_version(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_policy_version::de_policy_version(&mut tag, depth + 1)
                         ?
                     )
                 ;
