--- original
+++ modified
@@ -5,32 +5,32 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_login_profile::UpdateLoginProfileOutput,
-    crate::operation::update_login_profile::UpdateLoginProfileError,
+    super::operation::update_login_profile::UpdateLoginProfileOutput,
+    super::operation::update_login_profile::UpdateLoginProfileError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled(generic)),
+        None => return Err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "EntityTemporarilyUnmodifiable" => crate::operation::update_login_profile::UpdateLoginProfileError::EntityTemporarilyUnmodifiableException({
+        "EntityTemporarilyUnmodifiable" => super::operation::update_login_profile::UpdateLoginProfileError::EntityTemporarilyUnmodifiableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EntityTemporarilyUnmodifiableExceptionBuilder::default();
+                let mut output = super::types::error::builders::EntityTemporarilyUnmodifiableExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_entity_temporarily_unmodifiable_exception::de_entity_temporarily_unmodifiable_exception_xml_err(
+                    super::protocol_serde::shape_entity_temporarily_unmodifiable_exception::de_entity_temporarily_unmodifiable_exception_xml_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
+                    .map_err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "LimitExceeded" => crate::operation::update_login_profile::UpdateLoginProfileError::LimitExceededException({
+        "LimitExceeded" => super::operation::update_login_profile::UpdateLoginProfileError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,13 +54,13 @@
             }
             tmp
         }),
-        "NoSuchEntity" => crate::operation::update_login_profile::UpdateLoginProfileError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::update_login_profile::UpdateLoginProfileError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -69,16 +69,16 @@
             }
             tmp
         }),
-        "PasswordPolicyViolation" => crate::operation::update_login_profile::UpdateLoginProfileError::PasswordPolicyViolationException({
+        "PasswordPolicyViolation" => super::operation::update_login_profile::UpdateLoginProfileError::PasswordPolicyViolationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PasswordPolicyViolationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_password_policy_violation_exception::de_password_policy_violation_exception_xml_err(
+                let mut output = super::types::error::builders::PasswordPolicyViolationExceptionBuilder::default();
+                output = super::protocol_serde::shape_password_policy_violation_exception::de_password_policy_violation_exception_xml_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
+                .map_err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -87,13 +87,13 @@
             }
             tmp
         }),
-        "ServiceFailure" => crate::operation::update_login_profile::UpdateLoginProfileError::ServiceFailureException({
+        "ServiceFailure" => super::operation::update_login_profile::UpdateLoginProfileError::ServiceFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_failure_exception::de_service_failure_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::update_login_profile::UpdateLoginProfileError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -102,7 +102,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_login_profile::UpdateLoginProfileError::generic(generic),
+        _ => super::operation::update_login_profile::UpdateLoginProfileError::generic(generic),
     })
 }

@@ -112,12 +112,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_login_profile::UpdateLoginProfileOutput,
-    crate::operation::update_login_profile::UpdateLoginProfileError,
+    super::operation::update_login_profile::UpdateLoginProfileOutput,
+    super::operation::update_login_profile::UpdateLoginProfileError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_login_profile::builders::UpdateLoginProfileOutputBuilder::default();
+        let mut output = super::operation::update_login_profile::builders::UpdateLoginProfileOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
