--- original
+++ modified
@@ -5,15 +5,15 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_virtual_mfa_devices::ListVirtualMfaDevicesOutput,
-    crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError,
+    super::super::operation::list_virtual_mfa_devices::ListVirtualMfaDevicesOutput,
+    super::super::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
-    Err(crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::generic(generic))
+    Err(super::super::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::generic(generic))
 }

 #[allow(clippy::unnecessary_wraps)]
@@ -22,18 +22,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::list_virtual_mfa_devices::ListVirtualMfaDevicesOutput,
-    crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError,
+    super::super::operation::list_virtual_mfa_devices::ListVirtualMfaDevicesOutput,
+    super::super::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_virtual_mfa_devices::builders::ListVirtualMfaDevicesOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_virtual_mfa_devices::de_list_virtual_mfa_devices(_response_body, output)
-            .map_err(crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::unhandled)?;
+        let mut output = super::super::operation::list_virtual_mfa_devices::builders::ListVirtualMfaDevicesOutputBuilder::default();
+        output = super::super::protocol_serde::shape_list_virtual_mfa_devices::de_list_virtual_mfa_devices(_response_body, output)
+            .map_err(super::super::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::list_virtual_mfa_devices_output_output_correct_errors(output)
+        super::super::serde_util::list_virtual_mfa_devices_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::unhandled)?
+            .map_err(super::super::operation::list_virtual_mfa_devices::ListVirtualMFADevicesError::unhandled)?
     })
 }

@@ -40,9 +40,9 @@
 #[allow(unused_mut)]
 pub fn de_list_virtual_mfa_devices(
     inp: &[u8],
-    mut builder: crate::operation::list_virtual_mfa_devices::builders::ListVirtualMfaDevicesOutputBuilder,
+    mut builder: super::super::operation::list_virtual_mfa_devices::builders::ListVirtualMfaDevicesOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::list_virtual_mfa_devices::builders::ListVirtualMfaDevicesOutputBuilder,
+    super::super::operation::list_virtual_mfa_devices::builders::ListVirtualMfaDevicesOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -70,7 +70,7 @@
             s if s.matches("VirtualMFADevices") /* VirtualMFADevices com.amazonaws.iam.synthetic#ListVirtualMFADevicesOutput$VirtualMFADevices */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_virtual_mfa_device_list_type::de_virtual_mfa_device_list_type(&mut tag, depth + 1)
+                        super::super::protocol_serde::shape_virtual_mfa_device_list_type::de_virtual_mfa_device_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
