--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialOutput,
-    crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError,
+    super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialOutput,
+    super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled(generic)),
+        None => return Err(super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "NoSuchEntity" => crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::NoSuchEntityException({
+        "NoSuchEntity" => super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::NoSuchEntityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoSuchEntityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
-                    .map_err(crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled)?;
+                let mut output = super::types::error::builders::NoSuchEntityExceptionBuilder::default();
+                output = super::protocol_serde::shape_no_such_entity_exception::de_no_such_entity_exception_xml_err(_response_body, output)
+                    .map_err(super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,7 +35,7 @@
             }
             tmp
         }),
-        _ => crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::generic(generic),
+        _ => super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::generic(generic),
     })
 }

@@ -45,14 +45,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialOutput,
-    crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError,
+    super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialOutput,
+    super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::reset_service_specific_credential::builders::ResetServiceSpecificCredentialOutputBuilder::default();
-        output = crate::protocol_serde::shape_reset_service_specific_credential::de_reset_service_specific_credential(_response_body, output)
-            .map_err(crate::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled)?;
+        let mut output = super::operation::reset_service_specific_credential::builders::ResetServiceSpecificCredentialOutputBuilder::default();
+        output = super::protocol_serde::shape_reset_service_specific_credential::de_reset_service_specific_credential(_response_body, output)
+            .map_err(super::operation::reset_service_specific_credential::ResetServiceSpecificCredentialError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -61,9 +61,9 @@
 #[allow(unused_mut)]
 pub fn de_reset_service_specific_credential(
     inp: &[u8],
-    mut builder: crate::operation::reset_service_specific_credential::builders::ResetServiceSpecificCredentialOutputBuilder,
+    mut builder: super::operation::reset_service_specific_credential::builders::ResetServiceSpecificCredentialOutputBuilder,
 ) -> std::result::Result<
-    crate::operation::reset_service_specific_credential::builders::ResetServiceSpecificCredentialOutputBuilder,
+    super::operation::reset_service_specific_credential::builders::ResetServiceSpecificCredentialOutputBuilder,
     ::aws_smithy_xml::decode::XmlDecodeError,
 > {
     let mut doc = ::aws_smithy_xml::decode::Document::try_from(inp)?;
@@ -91,7 +91,7 @@
             s if s.matches("ServiceSpecificCredential") /* ServiceSpecificCredential com.amazonaws.iam.synthetic#ResetServiceSpecificCredentialOutput$ServiceSpecificCredential */ =>  {
                 let var_1 =
                     Some(
-                        crate::protocol_serde::shape_service_specific_credential::de_service_specific_credential(&mut tag, depth + 1)
+                        super::protocol_serde::shape_service_specific_credential::de_service_specific_credential(&mut tag, depth + 1)
                         ?
                     )
                 ;
