--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_role_template_version(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::RoleTemplateVersion, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::RoleTemplateVersion, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::RoleTemplateVersion::builder();
+    let mut builder = super::types::RoleTemplateVersion::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("TemplateArn") /* TemplateArn com.amazonaws.iam#RoleTemplateVersion$TemplateArn */ =>  {
@@ -96,8 +96,8 @@
             s if s.matches("ManagedByType") /* ManagedByType com.amazonaws.iam#RoleTemplateVersion$ManagedByType */ =>  {
                 let var_7 =
                     Some(
-                        Result::<crate::types::ManagedByTypeType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::ManagedByTypeType::from(
+                        Result::<super::types::ManagedByTypeType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::types::ManagedByTypeType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -205,7 +205,7 @@
             s if s.matches("InlinePolicyTemplates") /* InlinePolicyTemplates com.amazonaws.iam#RoleTemplateVersion$InlinePolicyTemplates */ =>  {
                 let var_15 =
                     Some(
-                        crate::protocol_serde::shape_inline_policy_template_list_type::de_inline_policy_template_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_inline_policy_template_list_type::de_inline_policy_template_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -215,7 +215,7 @@
             s if s.matches("ManagedPolicyArns") /* ManagedPolicyArns com.amazonaws.iam#RoleTemplateVersion$ManagedPolicyArns */ =>  {
                 let var_16 =
                     Some(
-                        crate::protocol_serde::shape_managed_policy_arn_list_type::de_managed_policy_arn_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_managed_policy_arn_list_type::de_managed_policy_arn_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -238,7 +238,7 @@
             s if s.matches("ParametersDefinition") /* ParametersDefinition com.amazonaws.iam#RoleTemplateVersion$ParametersDefinition */ =>  {
                 let var_18 =
                     Some(
-                        crate::protocol_serde::shape_parameters_definition_list_type::de_parameters_definition_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_parameters_definition_list_type::de_parameters_definition_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
@@ -248,7 +248,7 @@
             s if s.matches("RoleTagsTemplate") /* RoleTagsTemplate com.amazonaws.iam#RoleTemplateVersion$RoleTagsTemplate */ =>  {
                 let var_19 =
                     Some(
-                        crate::protocol_serde::shape_tag_template_list_type::de_tag_template_list_type(&mut tag, depth + 1)
+                        super::protocol_serde::shape_tag_template_list_type::de_tag_template_list_type(&mut tag, depth + 1)
                         ?
                     )
                 ;
