--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_group(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::Group, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::types::Group, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::Group::builder();
+    let mut builder = super::types::Group::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("Path") /* Path com.amazonaws.iam#Group$Path */ =>  {
@@ -80,7 +80,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::group_correct_errors(builder)
+    Ok(super::serde_util::group_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
