--- original
+++ modified
@@ -3,12 +3,12 @@
 pub fn de_service_specific_credential(
     decoder: &mut ::aws_smithy_xml::decode::ScopedDecoder,
     depth: u32,
-) -> ::std::result::Result<crate::types::ServiceSpecificCredential, ::aws_smithy_xml::decode::XmlDecodeError> {
+) -> ::std::result::Result<super::super::types::ServiceSpecificCredential, ::aws_smithy_xml::decode::XmlDecodeError> {
     if depth >= 128u32 {
         return Err(::aws_smithy_xml::decode::XmlDecodeError::custom("maximum nesting depth exceeded"));
     }
     #[allow(unused_mut)]
-    let mut builder = crate::types::ServiceSpecificCredential::builder();
+    let mut builder = super::super::types::ServiceSpecificCredential::builder();
     while let Some(mut tag) = decoder.next_tag() {
         match tag.start_el() {
             s if s.matches("CreateDate") /* CreateDate com.amazonaws.iam#ServiceSpecificCredential$CreateDate */ =>  {
@@ -133,8 +133,8 @@
             s if s.matches("Status") /* Status com.amazonaws.iam#ServiceSpecificCredential$Status */ =>  {
                 let var_10 =
                     Some(
-                        Result::<crate::types::StatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
-                            crate::types::StatusType::from(
+                        Result::<super::super::types::StatusType, ::aws_smithy_xml::decode::XmlDecodeError>::Ok(
+                            super::super::types::StatusType::from(
                                 ::aws_smithy_xml::decode::try_data(&mut tag)?.as_ref()
                             )
                         )
@@ -147,7 +147,7 @@
             _ => {}
         }
     }
-    Ok(crate::serde_util::service_specific_credential_correct_errors(builder)
+    Ok(super::super::serde_util::service_specific_credential_correct_errors(builder)
         .build()
         .map_err(|_| ::aws_smithy_xml::decode::XmlDecodeError::custom("missing field"))?)
 }
