--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ChangePasswordInput {
     /// Creates a new builder-style object to manufacture [`ChangePasswordInput`](crate::operation::change_password::ChangePasswordInput).
-    pub fn builder() -> crate::operation::change_password::builders::ChangePasswordInputBuilder {
-        crate::operation::change_password::builders::ChangePasswordInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::change_password::builders::ChangePasswordInputBuilder {
+        super::super::super::operation::change_password::builders::ChangePasswordInputBuilder::default()
     }
 }

@@ -79,8 +79,8 @@
     /// Consumes the builder and constructs a [`ChangePasswordInput`](crate::operation::change_password::ChangePasswordInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::change_password::ChangePasswordInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::change_password::ChangePasswordInput {
+    ) -> ::std::result::Result<super::super::super::operation::change_password::ChangePasswordInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::change_password::ChangePasswordInput {
             old_password: self.old_password,
             new_password: self.new_password,
         })
