--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::create_custom_key_store::CreateCustomKeyStoreInput,
+        input: super::operation::create_custom_key_store::CreateCustomKeyStoreInput,
     ) -> ::std::result::Result<
-        crate::operation::create_custom_key_store::CreateCustomKeyStoreOutput,
+        super::operation::create_custom_key_store::CreateCustomKeyStoreOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::create_custom_key_store::CreateCustomKeyStoreError,
+            super::operation::create_custom_key_store::CreateCustomKeyStoreError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::create_custom_key_store::CreateCustomKeyStoreError>()
+                err.downcast::<super::operation::create_custom_key_store::CreateCustomKeyStoreError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::create_custom_key_store::CreateCustomKeyStoreOutput>()
+                .downcast::<super::operation::create_custom_key_store::CreateCustomKeyStoreOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::create_custom_key_store::CreateCustomKeyStoreInput,
+        input: super::operation::create_custom_key_store::CreateCustomKeyStoreInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("CreateCustomKeyStore")
                 .build()
                 .expect("required fields set"),
@@ -136,13 +136,13 @@
                 CreateCustomKeyStoreEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::create_custom_key_store::CreateCustomKeyStoreError,
+                super::operation::create_custom_key_store::CreateCustomKeyStoreError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::create_custom_key_store::CreateCustomKeyStoreError,
+                super::operation::create_custom_key_store::CreateCustomKeyStoreError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::create_custom_key_store::CreateCustomKeyStoreError,
+                super::operation::create_custom_key_store::CreateCustomKeyStoreError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -237,11 +237,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_create_custom_key_store::de_create_custom_key_store_http_error(status, headers, body)
+            super::protocol_serde::shape_create_custom_key_store::de_create_custom_key_store_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_create_custom_key_store::de_create_custom_key_store_http_response(status, headers, body)
+            super::protocol_serde::shape_create_custom_key_store::de_create_custom_key_store_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -254,16 +254,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::create_custom_key_store::CreateCustomKeyStoreInput>()
+            .downcast::<super::operation::create_custom_key_store::CreateCustomKeyStoreInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::create_custom_key_store::CreateCustomKeyStoreInput,
+                _input: &super::operation::create_custom_key_store::CreateCustomKeyStoreInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -272,7 +272,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::create_custom_key_store::CreateCustomKeyStoreInput,
+                input: &super::operation::create_custom_key_store::CreateCustomKeyStoreInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -288,7 +288,7 @@
             );
             builder
         };
-        let body = ::aws_smithy_types::body::SdkBody::from(crate::protocol_serde::shape_create_custom_key_store::ser_create_custom_key_store_input(
+        let body = ::aws_smithy_types::body::SdkBody::from(super::protocol_serde::shape_create_custom_key_store::ser_create_custom_key_store_input(
             &input,
         )?);
         if let Some(content_length) = body.content_length() {
@@ -322,7 +322,7 @@
             .downcast_ref::<CreateCustomKeyStoreInput>()
             .ok_or("failed to downcast to CreateCustomKeyStoreInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -346,7 +346,7 @@
 pub enum CreateCustomKeyStoreError {
     /// <p>The request was rejected because the specified CloudHSM cluster is already associated with an CloudHSM key store in the account, or it shares a backup history with an CloudHSM key store in the account. Each CloudHSM key store in the account must be associated with a different CloudHSM cluster.</p>
     /// <p>CloudHSM clusters that share a backup history have the same cluster certificate. To view the cluster certificate of an CloudHSM cluster, use the <a href="https://docs.aws.amazon.com/cloudhsm/latest/APIReference/API_DescribeClusters.html">DescribeClusters</a> operation.</p>
-    CloudHsmClusterInUseException(crate::types::error::CloudHsmClusterInUseException),
+    CloudHsmClusterInUseException(super::types::error::CloudHsmClusterInUseException),
     /// <p>The request was rejected because the associated CloudHSM cluster did not meet the configuration requirements for an CloudHSM key store.</p>
     /// <ul>
     /// <li>
@@ -358,40 +358,40 @@
     /// <p>For the <code>CreateCustomKeyStore</code>, <code>UpdateCustomKeyStore</code>, and <code>CreateKey</code> operations, the CloudHSM cluster must have at least two active HSMs, each in a different Availability Zone. For the <code>ConnectCustomKeyStore</code> operation, the CloudHSM must contain at least one active HSM.</p></li>
     /// </ul>
     /// <p>For information about the requirements for an CloudHSM cluster that is associated with an CloudHSM key store, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/create-keystore.html#before-keystore">Assemble the Prerequisites</a> in the <i>Key Management Service Developer Guide</i>. For information about creating a private subnet for an CloudHSM cluster, see <a href="https://docs.aws.amazon.com/cloudhsm/latest/userguide/create-subnets.html">Create a Private Subnet</a> in the <i>CloudHSM User Guide</i>. For information about cluster security groups, see <a href="https://docs.aws.amazon.com/cloudhsm/latest/userguide/configure-sg.html">Configure a Default Security Group</a> in the <i> <i>CloudHSM User Guide</i> </i>.</p>
-    CloudHsmClusterInvalidConfigurationException(crate::types::error::CloudHsmClusterInvalidConfigurationException),
+    CloudHsmClusterInvalidConfigurationException(super::types::error::CloudHsmClusterInvalidConfigurationException),
     /// <p>The request was rejected because the CloudHSM cluster associated with the CloudHSM key store is not active. Initialize and activate the cluster and try the command again. For detailed instructions, see <a href="https://docs.aws.amazon.com/cloudhsm/latest/userguide/getting-started.html">Getting Started</a> in the <i>CloudHSM User Guide</i>.</p>
-    CloudHsmClusterNotActiveException(crate::types::error::CloudHsmClusterNotActiveException),
+    CloudHsmClusterNotActiveException(super::types::error::CloudHsmClusterNotActiveException),
     /// <p>The request was rejected because KMS cannot find the CloudHSM cluster with the specified cluster ID. Retry the request with a different cluster ID.</p>
-    CloudHsmClusterNotFoundException(crate::types::error::CloudHsmClusterNotFoundException),
+    CloudHsmClusterNotFoundException(super::types::error::CloudHsmClusterNotFoundException),
     /// <p>The request was rejected because the specified custom key store name is already assigned to another custom key store in the account. Try again with a custom key store name that is unique in the account.</p>
-    CustomKeyStoreNameInUseException(crate::types::error::CustomKeyStoreNameInUseException),
+    CustomKeyStoreNameInUseException(super::types::error::CustomKeyStoreNameInUseException),
     /// <p>The request was rejected because the trust anchor certificate in the request to create an CloudHSM key store is not the trust anchor certificate for the specified CloudHSM cluster.</p>
     /// <p>When you <a href="https://docs.aws.amazon.com/cloudhsm/latest/userguide/initialize-cluster.html#sign-csr">initialize the CloudHSM cluster</a>, you create the trust anchor certificate and save it in the <code>customerCA.crt</code> file.</p>
-    IncorrectTrustAnchorException(crate::types::error::IncorrectTrustAnchorException),
+    IncorrectTrustAnchorException(super::types::error::IncorrectTrustAnchorException),
     /// <p>The request was rejected because an internal exception occurred. The request can be retried.</p>
-    KmsInternalException(crate::types::error::KmsInternalException),
+    KmsInternalException(super::types::error::KmsInternalException),
     /// <p>The request was rejected because a length constraint or quota was exceeded. For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/limits.html">Quotas</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    LimitExceededException(crate::types::error::LimitExceededException),
+    LimitExceededException(super::types::error::LimitExceededException),
     /// <p>The request was rejected because the proxy credentials failed to authenticate to the specified external key store proxy. The specified external key store proxy rejected a status request from KMS due to invalid credentials. This can indicate an error in the credentials or in the identification of the external key store proxy.</p>
-    XksProxyIncorrectAuthenticationCredentialException(crate::types::error::XksProxyIncorrectAuthenticationCredentialException),
+    XksProxyIncorrectAuthenticationCredentialException(super::types::error::XksProxyIncorrectAuthenticationCredentialException),
     /// <p>The request was rejected because the external key store proxy is not configured correctly. To identify the cause, see the error message that accompanies the exception.</p>
-    XksProxyInvalidConfigurationException(crate::types::error::XksProxyInvalidConfigurationException),
+    XksProxyInvalidConfigurationException(super::types::error::XksProxyInvalidConfigurationException),
     /// <p></p>
     /// <p>KMS cannot interpret the response it received from the external key store proxy. The problem might be a poorly constructed response, but it could also be a transient network issue. If you see this error repeatedly, report it to the proxy vendor.</p>
-    XksProxyInvalidResponseException(crate::types::error::XksProxyInvalidResponseException),
+    XksProxyInvalidResponseException(super::types::error::XksProxyInvalidResponseException),
     /// <p>The request was rejected because the <code>XksProxyUriEndpoint</code> is already associated with another external key store in this Amazon Web Services Region. To identify the cause, see the error message that accompanies the exception.</p>
-    XksProxyUriEndpointInUseException(crate::types::error::XksProxyUriEndpointInUseException),
+    XksProxyUriEndpointInUseException(super::types::error::XksProxyUriEndpointInUseException),
     /// <p>The request was rejected because the concatenation of the <code>XksProxyUriEndpoint</code> and <code>XksProxyUriPath</code> is already associated with another external key store in this Amazon Web Services Region. Each external key store in a Region must use a unique external key store proxy API address.</p>
-    XksProxyUriInUseException(crate::types::error::XksProxyUriInUseException),
+    XksProxyUriInUseException(super::types::error::XksProxyUriInUseException),
     /// <p>KMS was unable to reach the specified <code>XksProxyUriPath</code>. The path must be reachable before you create the external key store or update its settings.</p>
     /// <p>This exception is also thrown when the external key store proxy response to a <code>GetHealthStatus</code> request indicates that all external key manager instances are unavailable.</p>
-    XksProxyUriUnreachableException(crate::types::error::XksProxyUriUnreachableException),
+    XksProxyUriUnreachableException(super::types::error::XksProxyUriUnreachableException),
     /// <p>The request was rejected because the specified Amazon VPC endpoint service is already associated with another external key store in this Amazon Web Services Region. Each external key store in a Region must use a different Amazon VPC endpoint service.</p>
-    XksProxyVpcEndpointServiceInUseException(crate::types::error::XksProxyVpcEndpointServiceInUseException),
+    XksProxyVpcEndpointServiceInUseException(super::types::error::XksProxyVpcEndpointServiceInUseException),
     /// <p>The request was rejected because the Amazon VPC endpoint service configuration does not fulfill the requirements for an external key store. To identify the cause, see the error message that accompanies the exception and <a href="https://docs.aws.amazon.com/kms/latest/developerguide/vpc-connectivity.html#xks-vpc-requirements">review the requirements</a> for Amazon VPC endpoint service connectivity for an external key store.</p>
-    XksProxyVpcEndpointServiceInvalidConfigurationException(crate::types::error::XksProxyVpcEndpointServiceInvalidConfigurationException),
+    XksProxyVpcEndpointServiceInvalidConfigurationException(super::types::error::XksProxyVpcEndpointServiceInvalidConfigurationException),
     /// <p>The request was rejected because KMS could not find the specified VPC endpoint service. Use <code>DescribeCustomKeyStores</code> to verify the VPC endpoint service name for the external key store. Also, confirm that the <code>Allow principals</code> list for the VPC endpoint service includes the KMS service principal for the Region, such as <code>cks.kms.us-east-1.amazonaws.com</code>.</p>
-    XksProxyVpcEndpointServiceNotFoundException(crate::types::error::XksProxyVpcEndpointServiceNotFoundException),
+    XksProxyVpcEndpointServiceNotFoundException(super::types::error::XksProxyVpcEndpointServiceNotFoundException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -399,7 +399,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-CreateCustomKeyStoreError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl CreateCustomKeyStoreError {
     /// Creates the `CreateCustomKeyStoreError::Unhandled` variant from any error type.
@@ -406,7 +406,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -414,7 +414,7 @@

     /// Creates the `CreateCustomKeyStoreError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -609,21 +609,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::create_custom_key_store::CreateCustomKeyStoreError {
+impl ::aws_types::request_id::RequestId for super::operation::create_custom_key_store::CreateCustomKeyStoreError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::create_custom_key_store::_create_custom_key_store_input::CreateCustomKeyStoreInput;
+pub use super::operation::create_custom_key_store::_create_custom_key_store_input::CreateCustomKeyStoreInput;

-pub use crate::operation::create_custom_key_store::_create_custom_key_store_output::CreateCustomKeyStoreOutput;
+pub use super::operation::create_custom_key_store::_create_custom_key_store_output::CreateCustomKeyStoreOutput;

 mod _create_custom_key_store_input;

