--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::re_encrypt::ReEncryptInput,
+        input: super::operation::re_encrypt::ReEncryptInput,
     ) -> ::std::result::Result<
-        crate::operation::re_encrypt::ReEncryptOutput,
+        super::operation::re_encrypt::ReEncryptOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::re_encrypt::ReEncryptError,
+            super::operation::re_encrypt::ReEncryptError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::re_encrypt::ReEncryptError>()
+                err.downcast::<super::operation::re_encrypt::ReEncryptError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::re_encrypt::ReEncryptOutput>()
+                .downcast::<super::operation::re_encrypt::ReEncryptOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::re_encrypt::ReEncryptInput,
+        input: super::operation::re_encrypt::ReEncryptInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -66,8 +66,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::config::Config,
+        config_override: ::std::option::Option<super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -75,7 +75,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -96,7 +96,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::config::auth::Params::builder()
                 .operation_name("ReEncrypt")
                 .build()
                 .expect("required fields set"),
@@ -133,13 +133,13 @@
                 ReEncryptEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::re_encrypt::ReEncryptError,
+                super::operation::re_encrypt::ReEncryptError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::re_encrypt::ReEncryptError,
+                super::operation::re_encrypt::ReEncryptError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::re_encrypt::ReEncryptError,
+                super::operation::re_encrypt::ReEncryptError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -209,11 +209,11 @@
         let mut force_error = false;
         ::tracing::debug!(request_id = ?::aws_types::request_id::RequestId::request_id(response));
         let parse_result = if !success && status != 200 || force_error {
-            crate::protocol_serde::shape_re_encrypt::de_re_encrypt_http_error(status, headers, body)
+            super::protocol_serde::shape_re_encrypt::de_re_encrypt_http_error(status, headers, body)
         } else {
-            crate::protocol_serde::shape_re_encrypt::de_re_encrypt_http_response(status, headers, body)
+            super::protocol_serde::shape_re_encrypt::de_re_encrypt_http_response(status, headers, body)
         };
-        crate::protocol_serde::type_erase_result(parse_result)
+        super::protocol_serde::type_erase_result(parse_result)
     }
 }
 #[derive(Debug)]
@@ -225,15 +225,15 @@
         input: ::aws_smithy_runtime_api::client::interceptors::context::Input,
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
-        let input = input.downcast::<crate::operation::re_encrypt::ReEncryptInput>().expect("correct type");
+        let input = input.downcast::<super::operation::re_encrypt::ReEncryptInput>().expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::re_encrypt::ReEncryptInput,
+                _input: &super::operation::re_encrypt::ReEncryptInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -242,7 +242,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::re_encrypt::ReEncryptInput,
+                input: &super::operation::re_encrypt::ReEncryptInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -258,7 +258,7 @@
             );
             builder
         };
-        let body = ::aws_smithy_types::body::SdkBody::from(crate::protocol_serde::shape_re_encrypt::ser_re_encrypt_input(&input)?);
+        let body = ::aws_smithy_types::body::SdkBody::from(super::protocol_serde::shape_re_encrypt::ser_re_encrypt_input(&input)?);
         if let Some(content_length) = body.content_length() {
             let content_length = content_length.to_string();
             request_builder = _header_serialization_settings.set_default_header(request_builder, ::http_1x::header::CONTENT_LENGTH, &content_length);
@@ -290,7 +290,7 @@
             .downcast_ref::<ReEncryptInput>()
             .ok_or("failed to downcast to ReEncryptInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -313,18 +313,18 @@
 #[derive(::std::fmt::Debug)]
 pub enum ReEncryptError {
     /// <p>The system timed out while trying to fulfill the request. You can retry the request.</p>
-    DependencyTimeoutException(crate::types::error::DependencyTimeoutException),
+    DependencyTimeoutException(super::types::error::DependencyTimeoutException),
     /// <p>The request was rejected because the specified KMS key is not enabled.</p>
-    DisabledException(crate::types::error::DisabledException),
+    DisabledException(super::types::error::DisabledException),
     /// <p>The request was rejected because the DryRun parameter was specified.</p>
-    DryRunOperationException(crate::types::error::DryRunOperationException),
+    DryRunOperationException(super::types::error::DryRunOperationException),
     /// <p>The request was rejected because the specified KMS key cannot decrypt the data. The <code>KeyId</code> in a <code>Decrypt</code> request and the <code>SourceKeyId</code> in a <code>ReEncrypt</code> request must identify the same KMS key that was used to encrypt the ciphertext.</p>
-    IncorrectKeyException(crate::types::error::IncorrectKeyException),
+    IncorrectKeyException(super::types::error::IncorrectKeyException),
     /// <p>From the <code>Decrypt</code> or <code>ReEncrypt</code> operation, the request was rejected because the specified ciphertext, or additional authenticated data incorporated into the ciphertext, such as the encryption context, is corrupted, missing, or otherwise invalid.</p>
     /// <p>From the <code>ImportKeyMaterial</code> operation, the request was rejected because KMS could not decrypt the encrypted (wrapped) key material.</p>
-    InvalidCiphertextException(crate::types::error::InvalidCiphertextException),
+    InvalidCiphertextException(super::types::error::InvalidCiphertextException),
     /// <p>The request was rejected because the specified grant token is not valid.</p>
-    InvalidGrantTokenException(crate::types::error::InvalidGrantTokenException),
+    InvalidGrantTokenException(super::types::error::InvalidGrantTokenException),
     /// <p>The request was rejected for one of the following reasons:</p>
     /// <ul>
     /// <li>
@@ -334,11 +334,11 @@
     /// </ul>
     /// <p>For encrypting, decrypting, re-encrypting, and generating data keys, the <code>KeyUsage</code> must be <code>ENCRYPT_DECRYPT</code>. For signing and verifying messages, the <code>KeyUsage</code> must be <code>SIGN_VERIFY</code>. For generating and verifying message authentication codes (MACs), the <code>KeyUsage</code> must be <code>GENERATE_VERIFY_MAC</code>. For deriving key agreement secrets, the <code>KeyUsage</code> must be <code>KEY_AGREEMENT</code>. To find the <code>KeyUsage</code> of a KMS key, use the <code>DescribeKey</code> operation.</p>
     /// <p>To find the encryption or signing algorithms supported for a particular KMS key, use the <code>DescribeKey</code> operation.</p>
-    InvalidKeyUsageException(crate::types::error::InvalidKeyUsageException),
+    InvalidKeyUsageException(super::types::error::InvalidKeyUsageException),
     /// <p>The request was rejected because the specified KMS key was not available. You can retry the request.</p>
-    KeyUnavailableException(crate::types::error::KeyUnavailableException),
+    KeyUnavailableException(super::types::error::KeyUnavailableException),
     /// <p>The request was rejected because an internal exception occurred. The request can be retried.</p>
-    KmsInternalException(crate::types::error::KmsInternalException),
+    KmsInternalException(super::types::error::KmsInternalException),
     /// <p>The request was rejected because the state of the specified resource is not valid for this request.</p>
     /// <p>This exceptions means one of the following:</p>
     /// <ul>
@@ -348,9 +348,9 @@
     /// <li>
     /// <p>For cryptographic operations on KMS keys in custom key stores, this exception represents a general failure with many possible causes. To identify the cause, see the error message that accompanies the exception.</p></li>
     /// </ul>
-    KmsInvalidStateException(crate::types::error::KmsInvalidStateException),
+    KmsInvalidStateException(super::types::error::KmsInvalidStateException),
     /// <p>The request was rejected because the specified entity or resource could not be found.</p>
-    NotFoundException(crate::types::error::NotFoundException),
+    NotFoundException(super::types::error::NotFoundException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -358,7 +358,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-ReEncryptError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::error::sealed_unhandled::Unhandled),
 }
 impl ReEncryptError {
     /// Creates the `ReEncryptError::Unhandled` variant from any error type.
@@ -365,7 +365,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -373,7 +373,7 @@

     /// Creates the `ReEncryptError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -516,21 +516,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::re_encrypt::ReEncryptError {
+impl ::aws_types::request_id::RequestId for super::operation::re_encrypt::ReEncryptError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::re_encrypt::_re_encrypt_input::ReEncryptInput;
+pub use super::operation::re_encrypt::_re_encrypt_input::ReEncryptInput;

-pub use crate::operation::re_encrypt::_re_encrypt_output::ReEncryptOutput;
+pub use super::operation::re_encrypt::_re_encrypt_output::ReEncryptOutput;

 mod _re_encrypt_input;

