--- original
+++ modified
@@ -27,8 +27,8 @@
 }
 impl ListUserImportJobsInput {
     /// Creates a new builder-style object to manufacture [`ListUserImportJobsInput`](crate::operation::list_user_import_jobs::ListUserImportJobsInput).
-    pub fn builder() -> crate::operation::list_user_import_jobs::builders::ListUserImportJobsInputBuilder {
-        crate::operation::list_user_import_jobs::builders::ListUserImportJobsInputBuilder::default()
+    pub fn builder() -> super::operation::list_user_import_jobs::builders::ListUserImportJobsInputBuilder {
+        super::operation::list_user_import_jobs::builders::ListUserImportJobsInputBuilder::default()
     }
 }

@@ -88,9 +88,9 @@
     /// Consumes the builder and constructs a [`ListUserImportJobsInput`](crate::operation::list_user_import_jobs::ListUserImportJobsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_user_import_jobs::ListUserImportJobsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_user_import_jobs::ListUserImportJobsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_user_import_jobs::ListUserImportJobsInput {
+        ::std::result::Result::Ok(super::operation::list_user_import_jobs::ListUserImportJobsInput {
             user_pool_id: self.user_pool_id,
             max_results: self.max_results,
             pagination_token: self.pagination_token,
