--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetEmailTemplateInput {
     /// Creates a new builder-style object to manufacture [`GetEmailTemplateInput`](crate::operation::get_email_template::GetEmailTemplateInput).
-    pub fn builder() -> crate::operation::get_email_template::builders::GetEmailTemplateInputBuilder {
-        crate::operation::get_email_template::builders::GetEmailTemplateInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_email_template::builders::GetEmailTemplateInputBuilder {
+        super::super::super::operation::get_email_template::builders::GetEmailTemplateInputBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`GetEmailTemplateInput`](crate::operation::get_email_template::GetEmailTemplateInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_email_template::GetEmailTemplateInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_email_template::GetEmailTemplateInput {
+    ) -> ::std::result::Result<super::super::super::operation::get_email_template::GetEmailTemplateInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_email_template::GetEmailTemplateInput {
             template_name: self.template_name,
         })
     }
