--- original
+++ modified
@@ -44,8 +44,8 @@
 }
 impl ListBucketsInput {
     /// Creates a new builder-style object to manufacture [`ListBucketsInput`](crate::operation::list_buckets::ListBucketsInput).
-    pub fn builder() -> crate::operation::list_buckets::builders::ListBucketsInputBuilder {
-        crate::operation::list_buckets::builders::ListBucketsInputBuilder::default()
+    pub fn builder() -> super::operation::list_buckets::builders::ListBucketsInputBuilder {
+        super::operation::list_buckets::builders::ListBucketsInputBuilder::default()
     }
 }

@@ -134,8 +134,8 @@
         &self.bucket_region
     }
     /// Consumes the builder and constructs a [`ListBucketsInput`](crate::operation::list_buckets::ListBucketsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_buckets::ListBucketsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_buckets::ListBucketsInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_buckets::ListBucketsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_buckets::ListBucketsInput {
             max_buckets: self.max_buckets,
             continuation_token: self.continuation_token,
             prefix: self.prefix,
