--- original
+++ modified
@@ -8,13 +8,13 @@
     /// <p>The Batch job queue associated with the quota share. This can be the job queue name or ARN. A job queue must be in the <code>VALID</code> state before you can associate it with a quota share.</p>
     pub job_queue: ::std::option::Option<::std::string::String>,
     /// <p>A list that specifies the quantity and type of compute capacity allocated to the quota share.</p>
-    pub capacity_limits: ::std::option::Option<::std::vec::Vec<crate::types::QuotaShareCapacityLimit>>,
+    pub capacity_limits: ::std::option::Option<::std::vec::Vec<super::super::super::types::QuotaShareCapacityLimit>>,
     /// <p>Specifies whether a quota share reserves, lends, or both lends and borrows idle compute capacity.</p>
-    pub resource_sharing_configuration: ::std::option::Option<crate::types::QuotaShareResourceSharingConfiguration>,
+    pub resource_sharing_configuration: ::std::option::Option<super::super::super::types::QuotaShareResourceSharingConfiguration>,
     /// <p>Specifies the preemption behavior for jobs in a quota share.</p>
-    pub preemption_configuration: ::std::option::Option<crate::types::QuotaSharePreemptionConfiguration>,
+    pub preemption_configuration: ::std::option::Option<super::super::super::types::QuotaSharePreemptionConfiguration>,
     /// <p>The state of the quota share. If the quota share is <code>ENABLED</code>, it is able to accept jobs. If the quota share is <code>DISABLED</code>, new jobs won't be accepted but jobs already submitted can finish. The default state is <code>ENABLED</code>.</p>
-    pub state: ::std::option::Option<crate::types::QuotaShareState>,
+    pub state: ::std::option::Option<super::super::super::types::QuotaShareState>,
     /// <p>The tags that you apply to the quota share to help you categorize and organize your resources. Each tag consists of a key and an optional value. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/using-tags.html">Tagging your Batch resources</a> in <i>Batch User Guide</i>.</p>
     pub tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
@@ -30,19 +30,19 @@
     /// <p>A list that specifies the quantity and type of compute capacity allocated to the quota share.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.capacity_limits.is_none()`.
-    pub fn capacity_limits(&self) -> &[crate::types::QuotaShareCapacityLimit] {
+    pub fn capacity_limits(&self) -> &[super::super::super::types::QuotaShareCapacityLimit] {
         self.capacity_limits.as_deref().unwrap_or_default()
     }
     /// <p>Specifies whether a quota share reserves, lends, or both lends and borrows idle compute capacity.</p>
-    pub fn resource_sharing_configuration(&self) -> ::std::option::Option<&crate::types::QuotaShareResourceSharingConfiguration> {
+    pub fn resource_sharing_configuration(&self) -> ::std::option::Option<&super::super::super::types::QuotaShareResourceSharingConfiguration> {
         self.resource_sharing_configuration.as_ref()
     }
     /// <p>Specifies the preemption behavior for jobs in a quota share.</p>
-    pub fn preemption_configuration(&self) -> ::std::option::Option<&crate::types::QuotaSharePreemptionConfiguration> {
+    pub fn preemption_configuration(&self) -> ::std::option::Option<&super::super::super::types::QuotaSharePreemptionConfiguration> {
         self.preemption_configuration.as_ref()
     }
     /// <p>The state of the quota share. If the quota share is <code>ENABLED</code>, it is able to accept jobs. If the quota share is <code>DISABLED</code>, new jobs won't be accepted but jobs already submitted can finish. The default state is <code>ENABLED</code>.</p>
-    pub fn state(&self) -> ::std::option::Option<&crate::types::QuotaShareState> {
+    pub fn state(&self) -> ::std::option::Option<&super::super::super::types::QuotaShareState> {
         self.state.as_ref()
     }
     /// <p>The tags that you apply to the quota share to help you categorize and organize your resources. Each tag consists of a key and an optional value. For more information, see <a href="https://docs.aws.amazon.com/batch/latest/userguide/using-tags.html">Tagging your Batch resources</a> in <i>Batch User Guide</i>.</p>
@@ -52,8 +52,8 @@
 }
 impl CreateQuotaShareInput {
     /// Creates a new builder-style object to manufacture [`CreateQuotaShareInput`](crate::operation::create_quota_share::CreateQuotaShareInput).
-    pub fn builder() -> crate::operation::create_quota_share::builders::CreateQuotaShareInputBuilder {
-        crate::operation::create_quota_share::builders::CreateQuotaShareInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::create_quota_share::builders::CreateQuotaShareInputBuilder {
+        super::super::super::operation::create_quota_share::builders::CreateQuotaShareInputBuilder::default()
     }
 }

@@ -63,10 +63,10 @@
 pub struct CreateQuotaShareInputBuilder {
     pub(crate) quota_share_name: ::std::option::Option<::std::string::String>,
     pub(crate) job_queue: ::std::option::Option<::std::string::String>,
-    pub(crate) capacity_limits: ::std::option::Option<::std::vec::Vec<crate::types::QuotaShareCapacityLimit>>,
-    pub(crate) resource_sharing_configuration: ::std::option::Option<crate::types::QuotaShareResourceSharingConfiguration>,
-    pub(crate) preemption_configuration: ::std::option::Option<crate::types::QuotaSharePreemptionConfiguration>,
-    pub(crate) state: ::std::option::Option<crate::types::QuotaShareState>,
+    pub(crate) capacity_limits: ::std::option::Option<::std::vec::Vec<super::super::super::types::QuotaShareCapacityLimit>>,
+    pub(crate) resource_sharing_configuration: ::std::option::Option<super::super::super::types::QuotaShareResourceSharingConfiguration>,
+    pub(crate) preemption_configuration: ::std::option::Option<super::super::super::types::QuotaSharePreemptionConfiguration>,
+    pub(crate) state: ::std::option::Option<super::super::super::types::QuotaShareState>,
     pub(crate) tags: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
 }
 impl CreateQuotaShareInputBuilder {
@@ -105,7 +105,7 @@
     /// To override the contents of this collection use [`set_capacity_limits`](Self::set_capacity_limits).
     ///
     /// <p>A list that specifies the quantity and type of compute capacity allocated to the quota share.</p>
-    pub fn capacity_limits(mut self, input: crate::types::QuotaShareCapacityLimit) -> Self {
+    pub fn capacity_limits(mut self, input: super::super::super::types::QuotaShareCapacityLimit) -> Self {
         let mut v = self.capacity_limits.unwrap_or_default();
         v.push(input);
         self.capacity_limits = ::std::option::Option::Some(v);
@@ -112,56 +112,56 @@
         self
     }
     /// <p>A list that specifies the quantity and type of compute capacity allocated to the quota share.</p>
-    pub fn set_capacity_limits(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::QuotaShareCapacityLimit>>) -> Self {
+    pub fn set_capacity_limits(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::QuotaShareCapacityLimit>>) -> Self {
         self.capacity_limits = input;
         self
     }
     /// <p>A list that specifies the quantity and type of compute capacity allocated to the quota share.</p>
-    pub fn get_capacity_limits(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::QuotaShareCapacityLimit>> {
+    pub fn get_capacity_limits(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::QuotaShareCapacityLimit>> {
         &self.capacity_limits
     }
     /// <p>Specifies whether a quota share reserves, lends, or both lends and borrows idle compute capacity.</p>
     /// This field is required.
-    pub fn resource_sharing_configuration(mut self, input: crate::types::QuotaShareResourceSharingConfiguration) -> Self {
+    pub fn resource_sharing_configuration(mut self, input: super::super::super::types::QuotaShareResourceSharingConfiguration) -> Self {
         self.resource_sharing_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether a quota share reserves, lends, or both lends and borrows idle compute capacity.</p>
-    pub fn set_resource_sharing_configuration(mut self, input: ::std::option::Option<crate::types::QuotaShareResourceSharingConfiguration>) -> Self {
+    pub fn set_resource_sharing_configuration(mut self, input: ::std::option::Option<super::super::super::types::QuotaShareResourceSharingConfiguration>) -> Self {
         self.resource_sharing_configuration = input;
         self
     }
     /// <p>Specifies whether a quota share reserves, lends, or both lends and borrows idle compute capacity.</p>
-    pub fn get_resource_sharing_configuration(&self) -> &::std::option::Option<crate::types::QuotaShareResourceSharingConfiguration> {
+    pub fn get_resource_sharing_configuration(&self) -> &::std::option::Option<super::super::super::types::QuotaShareResourceSharingConfiguration> {
         &self.resource_sharing_configuration
     }
     /// <p>Specifies the preemption behavior for jobs in a quota share.</p>
     /// This field is required.
-    pub fn preemption_configuration(mut self, input: crate::types::QuotaSharePreemptionConfiguration) -> Self {
+    pub fn preemption_configuration(mut self, input: super::super::super::types::QuotaSharePreemptionConfiguration) -> Self {
         self.preemption_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the preemption behavior for jobs in a quota share.</p>
-    pub fn set_preemption_configuration(mut self, input: ::std::option::Option<crate::types::QuotaSharePreemptionConfiguration>) -> Self {
+    pub fn set_preemption_configuration(mut self, input: ::std::option::Option<super::super::super::types::QuotaSharePreemptionConfiguration>) -> Self {
         self.preemption_configuration = input;
         self
     }
     /// <p>Specifies the preemption behavior for jobs in a quota share.</p>
-    pub fn get_preemption_configuration(&self) -> &::std::option::Option<crate::types::QuotaSharePreemptionConfiguration> {
+    pub fn get_preemption_configuration(&self) -> &::std::option::Option<super::super::super::types::QuotaSharePreemptionConfiguration> {
         &self.preemption_configuration
     }
     /// <p>The state of the quota share. If the quota share is <code>ENABLED</code>, it is able to accept jobs. If the quota share is <code>DISABLED</code>, new jobs won't be accepted but jobs already submitted can finish. The default state is <code>ENABLED</code>.</p>
-    pub fn state(mut self, input: crate::types::QuotaShareState) -> Self {
+    pub fn state(mut self, input: super::super::super::types::QuotaShareState) -> Self {
         self.state = ::std::option::Option::Some(input);
         self
     }
     /// <p>The state of the quota share. If the quota share is <code>ENABLED</code>, it is able to accept jobs. If the quota share is <code>DISABLED</code>, new jobs won't be accepted but jobs already submitted can finish. The default state is <code>ENABLED</code>.</p>
-    pub fn set_state(mut self, input: ::std::option::Option<crate::types::QuotaShareState>) -> Self {
+    pub fn set_state(mut self, input: ::std::option::Option<super::super::super::types::QuotaShareState>) -> Self {
         self.state = input;
         self
     }
     /// <p>The state of the quota share. If the quota share is <code>ENABLED</code>, it is able to accept jobs. If the quota share is <code>DISABLED</code>, new jobs won't be accepted but jobs already submitted can finish. The default state is <code>ENABLED</code>.</p>
-    pub fn get_state(&self) -> &::std::option::Option<crate::types::QuotaShareState> {
+    pub fn get_state(&self) -> &::std::option::Option<super::super::super::types::QuotaShareState> {
         &self.state
     }
     /// Adds a key-value pair to `tags`.
@@ -187,8 +187,8 @@
     /// Consumes the builder and constructs a [`CreateQuotaShareInput`](crate::operation::create_quota_share::CreateQuotaShareInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_quota_share::CreateQuotaShareInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::create_quota_share::CreateQuotaShareInput {
+    ) -> ::std::result::Result<super::super::super::operation::create_quota_share::CreateQuotaShareInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::create_quota_share::CreateQuotaShareInput {
             quota_share_name: self.quota_share_name,
             job_queue: self.job_queue,
             capacity_limits: self.capacity_limits,
