--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::TermsType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::TermsType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::TermsTypeBuilder::default();
+            let mut builder = super::types::builders::TermsTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -52,7 +52,7 @@
                         "TermsSource" => {
                             builder = builder.set_terms_source(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::TermsSourceType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::TermsSourceType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -59,12 +59,12 @@
                         "Enforcement" => {
                             builder = builder.set_enforcement(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::TermsEnforcementType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::TermsEnforcementType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "Links" => {
-                            builder = builder.set_links(crate::protocol_serde::shape_links_type::de_links_type(tokens, _value, depth + 1)?);
+                            builder = builder.set_links(super::protocol_serde::shape_links_type::de_links_type(tokens, _value, depth + 1)?);
                         }
                         "CreationDate" => {
                             builder = builder.set_creation_date(::aws_smithy_json::deserialize::token::expect_timestamp_or_null(
@@ -87,7 +87,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::terms_type_correct_errors(builder).build().map_err(|err| {
+            Ok(Some(super::serde_util::terms_type_correct_errors(builder).build().map_err(|err| {
                 ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err)
             })?))
         }
