--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_devices::ListDevicesOutput, crate::operation::list_devices::ListDevicesError> {
+) -> std::result::Result<super::operation::list_devices::ListDevicesOutput, super::operation::list_devices::ListDevicesError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::list_devices::ListDevicesError::unhandled(generic)),
+        None => return Err(super::operation::list_devices::ListDevicesError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ForbiddenException" => crate::operation::list_devices::ListDevicesError::ForbiddenException({
+        "ForbiddenException" => super::operation::list_devices::ListDevicesError::ForbiddenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ForbiddenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::ForbiddenExceptionBuilder::default();
+                output = super::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::list_devices::ListDevicesError::InternalErrorException({
+        "InternalErrorException" => super::operation::list_devices::ListDevicesError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::list_devices::ListDevicesError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::list_devices::ListDevicesError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -63,12 +63,12 @@
             tmp
         }),
         "InvalidUserPoolConfigurationException" => {
-            crate::operation::list_devices::ListDevicesError::InvalidUserPoolConfigurationException({
+            super::operation::list_devices::ListDevicesError::InvalidUserPoolConfigurationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidUserPoolConfigurationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_user_pool_configuration_exception::de_invalid_user_pool_configuration_exception_json_err(_response_body, output).map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidUserPoolConfigurationExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_user_pool_configuration_exception::de_invalid_user_pool_configuration_exception_json_err(_response_body, output).map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -78,13 +78,13 @@
                 tmp
             })
         }
-        "NotAuthorizedException" => crate::operation::list_devices::ListDevicesError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::list_devices::ListDevicesError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,14 +93,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::list_devices::ListDevicesError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::list_devices::ListDevicesError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -109,16 +109,16 @@
             }
             tmp
         }),
-        "PasswordResetRequiredException" => crate::operation::list_devices::ListDevicesError::PasswordResetRequiredException({
+        "PasswordResetRequiredException" => super::operation::list_devices::ListDevicesError::PasswordResetRequiredException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
-                output = crate::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
+                let mut output = super::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
+                output = super::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -127,13 +127,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::list_devices::ListDevicesError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::list_devices::ListDevicesError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -142,13 +142,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::list_devices::ListDevicesError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::list_devices::ListDevicesError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -157,13 +157,13 @@
             }
             tmp
         }),
-        "UserNotConfirmedException" => crate::operation::list_devices::ListDevicesError::UserNotConfirmedException({
+        "UserNotConfirmedException" => super::operation::list_devices::ListDevicesError::UserNotConfirmedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotConfirmedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotConfirmedExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -172,13 +172,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::list_devices::ListDevicesError::UserNotFoundException({
+        "UserNotFoundException" => super::operation::list_devices::ListDevicesError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -187,7 +187,7 @@
             }
             tmp
         }),
-        _ => crate::operation::list_devices::ListDevicesError::generic(generic),
+        _ => super::operation::list_devices::ListDevicesError::generic(generic),
     })
 }

@@ -196,12 +196,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::list_devices::ListDevicesOutput, crate::operation::list_devices::ListDevicesError> {
+) -> std::result::Result<super::operation::list_devices::ListDevicesOutput, super::operation::list_devices::ListDevicesError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::list_devices::builders::ListDevicesOutputBuilder::default();
-        output = crate::protocol_serde::shape_list_devices::de_list_devices(_response_body, output)
-            .map_err(crate::operation::list_devices::ListDevicesError::unhandled)?;
+        let mut output = super::operation::list_devices::builders::ListDevicesOutputBuilder::default();
+        output = super::protocol_serde::shape_list_devices::de_list_devices(_response_body, output)
+            .map_err(super::operation::list_devices::ListDevicesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -208,11 +208,11 @@
 }

 pub fn ser_list_devices_input(
-    input: &crate::operation::list_devices::ListDevicesInput,
+    input: &super::operation::list_devices::ListDevicesInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_list_devices_input::ser_list_devices_input_input(&mut object, input)?;
+    super::protocol_serde::shape_list_devices_input::ser_list_devices_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -219,10 +219,10 @@

 pub(crate) fn de_list_devices(
     _value: &[u8],
-    mut builder: crate::operation::list_devices::builders::ListDevicesOutputBuilder,
-) -> ::std::result::Result<crate::operation::list_devices::builders::ListDevicesOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError>
+    mut builder: super::operation::list_devices::builders::ListDevicesOutputBuilder,
+) -> ::std::result::Result<super::operation::list_devices::builders::ListDevicesOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError>
 {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -232,7 +232,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Devices" => {
-                    builder = builder.set_devices(crate::protocol_serde::shape_device_list_type::de_device_list_type(
+                    builder = builder.set_devices(super::protocol_serde::shape_device_list_type::de_device_list_type(
                         tokens,
                         _value,
                         depth + 1,
