--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::AuthEventType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::AuthEventType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::AuthEventTypeBuilder::default();
+            let mut builder = super::types::builders::AuthEventTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,7 +31,7 @@
                         "EventType" => {
                             builder = builder.set_event_type(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::EventType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::EventType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -44,12 +44,12 @@
                         "EventResponse" => {
                             builder = builder.set_event_response(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::EventResponseType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::EventResponseType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "EventRisk" => {
-                            builder = builder.set_event_risk(crate::protocol_serde::shape_event_risk_type::de_event_risk_type(
+                            builder = builder.set_event_risk(super::protocol_serde::shape_event_risk_type::de_event_risk_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -57,7 +57,7 @@
                         }
                         "ChallengeResponses" => {
                             builder = builder.set_challenge_responses(
-                                crate::protocol_serde::shape_challenge_response_list_type::de_challenge_response_list_type(
+                                super::protocol_serde::shape_challenge_response_list_type::de_challenge_response_list_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -66,11 +66,11 @@
                         }
                         "EventContextData" => {
                             builder = builder.set_event_context_data(
-                                crate::protocol_serde::shape_event_context_data_type::de_event_context_data_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_event_context_data_type::de_event_context_data_type(tokens, _value, depth + 1)?,
                             );
                         }
                         "EventFeedback" => {
-                            builder = builder.set_event_feedback(crate::protocol_serde::shape_event_feedback_type::de_event_feedback_type(
+                            builder = builder.set_event_feedback(super::protocol_serde::shape_event_feedback_type::de_event_feedback_type(
                                 tokens,
                                 _value,
                                 depth + 1,
