--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::IdentityProviderType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::IdentityProviderType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::IdentityProviderTypeBuilder::default();
+            let mut builder = super::super::types::builders::IdentityProviderTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -39,12 +39,12 @@
                             "ProviderType" => {
                                 builder = builder.set_provider_type(
                                     ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                        .map(|s| s.to_unescaped().map(|u| crate::types::IdentityProviderTypeType::from(u.as_ref())))
+                                        .map(|s| s.to_unescaped().map(|u| super::super::types::IdentityProviderTypeType::from(u.as_ref())))
                                         .transpose()?,
                                 );
                             }
                             "ProviderDetails" => {
-                                builder = builder.set_provider_details(crate::protocol_serde::shape_provider_details_type::de_provider_details_type(
+                                builder = builder.set_provider_details(super::super::protocol_serde::shape_provider_details_type::de_provider_details_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -52,12 +52,12 @@
                             }
                             "AttributeMapping" => {
                                 builder = builder.set_attribute_mapping(
-                                    crate::protocol_serde::shape_attribute_mapping_type::de_attribute_mapping_type(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_attribute_mapping_type::de_attribute_mapping_type(tokens, _value, depth + 1)?,
                                 );
                             }
                             "IdpIdentifiers" => {
                                 builder = builder.set_idp_identifiers(
-                                    crate::protocol_serde::shape_idp_identifiers_list_type::de_idp_identifiers_list_type(tokens, _value, depth + 1)?,
+                                    super::super::protocol_serde::shape_idp_identifiers_list_type::de_idp_identifiers_list_type(tokens, _value, depth + 1)?,
                                 );
                             }
                             "LastModifiedDate" => {
