--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsOutput,
-    crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError,
+    super::operation::admin_list_user_auth_events::AdminListUserAuthEventsOutput,
+    super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled(
+            return Err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::InternalErrorException({
+        "InternalErrorException" => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,13 +54,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,16 +70,16 @@
             tmp
         }),
         "OperationNotEnabledException" => {
-            crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::OperationNotEnabledException({
+            super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::OperationNotEnabledException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(
+                    let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                    output = super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -89,13 +89,13 @@
                 tmp
             })
         }
-        "ResourceNotFoundException" => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -104,13 +104,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -119,13 +119,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::UserNotFoundException({
+        "UserNotFoundException" => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -135,16 +135,16 @@
             tmp
         }),
         "UserPoolAddOnNotEnabledException" => {
-            crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::UserPoolAddOnNotEnabledException({
+            super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::UserPoolAddOnNotEnabledException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::UserPoolAddOnNotEnabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_user_pool_add_on_not_enabled_exception::de_user_pool_add_on_not_enabled_exception_json_err(
+                    let mut output = super::types::error::builders::UserPoolAddOnNotEnabledExceptionBuilder::default();
+                    output = super::protocol_serde::shape_user_pool_add_on_not_enabled_exception::de_user_pool_add_on_not_enabled_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+                    .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -154,7 +154,7 @@
                 tmp
             })
         }
-        _ => crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::generic(generic),
+        _ => super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::generic(generic),
     })
 }

@@ -164,14 +164,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsOutput,
-    crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError,
+    super::operation::admin_list_user_auth_events::AdminListUserAuthEventsOutput,
+    super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::admin_list_user_auth_events::builders::AdminListUserAuthEventsOutputBuilder::default();
-        output = crate::protocol_serde::shape_admin_list_user_auth_events::de_admin_list_user_auth_events(_response_body, output)
-            .map_err(crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
+        let mut output = super::operation::admin_list_user_auth_events::builders::AdminListUserAuthEventsOutputBuilder::default();
+        output = super::protocol_serde::shape_admin_list_user_auth_events::de_admin_list_user_auth_events(_response_body, output)
+            .map_err(super::operation::admin_list_user_auth_events::AdminListUserAuthEventsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -178,11 +178,11 @@
 }

 pub fn ser_admin_list_user_auth_events_input(
-    input: &crate::operation::admin_list_user_auth_events::AdminListUserAuthEventsInput,
+    input: &super::operation::admin_list_user_auth_events::AdminListUserAuthEventsInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_admin_list_user_auth_events_input::ser_admin_list_user_auth_events_input_input(&mut object, input)?;
+    super::protocol_serde::shape_admin_list_user_auth_events_input::ser_admin_list_user_auth_events_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -189,12 +189,12 @@

 pub(crate) fn de_admin_list_user_auth_events(
     _value: &[u8],
-    mut builder: crate::operation::admin_list_user_auth_events::builders::AdminListUserAuthEventsOutputBuilder,
+    mut builder: super::operation::admin_list_user_auth_events::builders::AdminListUserAuthEventsOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::admin_list_user_auth_events::builders::AdminListUserAuthEventsOutputBuilder,
+    super::operation::admin_list_user_auth_events::builders::AdminListUserAuthEventsOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -204,7 +204,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "AuthEvents" => {
-                    builder = builder.set_auth_events(crate::protocol_serde::shape_auth_events_type::de_auth_events_type(
+                    builder = builder.set_auth_events(super::protocol_serde::shape_auth_events_type::de_auth_events_type(
                         tokens,
                         _value,
                         depth + 1,
