--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::admin_list_groups_for_user::AdminListGroupsForUserOutput,
-    crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError,
+    super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserOutput,
+    super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled(
+            return Err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::InternalErrorException({
+        "InternalErrorException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::InvalidParameterException({
+        "InvalidParameterException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,13 +54,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::NotAuthorizedException({
+        "NotAuthorizedException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -69,14 +69,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                    super::super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -85,13 +85,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -100,13 +100,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -115,13 +115,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::UserNotFoundException({
+        "UserNotFoundException" => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+                let mut output = super::super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -130,7 +130,7 @@
             }
             tmp
         }),
-        _ => crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::generic(generic),
+        _ => super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::generic(generic),
     })
 }

@@ -140,14 +140,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::admin_list_groups_for_user::AdminListGroupsForUserOutput,
-    crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError,
+    super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserOutput,
+    super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserOutputBuilder::default();
-        output = crate::protocol_serde::shape_admin_list_groups_for_user::de_admin_list_groups_for_user(_response_body, output)
-            .map_err(crate::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
+        let mut output = super::super::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserOutputBuilder::default();
+        output = super::super::protocol_serde::shape_admin_list_groups_for_user::de_admin_list_groups_for_user(_response_body, output)
+            .map_err(super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -154,11 +154,11 @@
 }

 pub fn ser_admin_list_groups_for_user_input(
-    input: &crate::operation::admin_list_groups_for_user::AdminListGroupsForUserInput,
+    input: &super::super::operation::admin_list_groups_for_user::AdminListGroupsForUserInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_admin_list_groups_for_user_input::ser_admin_list_groups_for_user_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_admin_list_groups_for_user_input::ser_admin_list_groups_for_user_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -165,12 +165,12 @@

 pub(crate) fn de_admin_list_groups_for_user(
     _value: &[u8],
-    mut builder: crate::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserOutputBuilder,
+    mut builder: super::super::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserOutputBuilder,
+    super::super::operation::admin_list_groups_for_user::builders::AdminListGroupsForUserOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -180,7 +180,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Groups" => {
-                    builder = builder.set_groups(crate::protocol_serde::shape_group_list_type::de_group_list_type(
+                    builder = builder.set_groups(super::super::protocol_serde::shape_group_list_type::de_group_list_type(
                         tokens,
                         _value,
                         depth + 1,
