--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ChallengeResponseType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ChallengeResponseType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ChallengeResponseTypeBuilder::default();
+            let mut builder = super::types::builders::ChallengeResponseTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -24,7 +24,7 @@
                         "ChallengeName" => {
                             builder = builder.set_challenge_name(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ChallengeName::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::ChallengeName::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -31,7 +31,7 @@
                         "ChallengeResponse" => {
                             builder = builder.set_challenge_response(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ChallengeResponse::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::ChallengeResponse::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
