--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::admin_get_user::AdminGetUserOutput, crate::operation::admin_get_user::AdminGetUserError> {
+) -> std::result::Result<super::operation::admin_get_user::AdminGetUserOutput, super::operation::admin_get_user::AdminGetUserError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::admin_get_user::AdminGetUserError::unhandled(generic)),
+        None => return Err(super::operation::admin_get_user::AdminGetUserError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::admin_get_user::AdminGetUserError::InternalErrorException({
+        "InternalErrorException" => super::operation::admin_get_user::AdminGetUserError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::admin_get_user::AdminGetUserError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::admin_get_user::AdminGetUserError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::admin_get_user::AdminGetUserError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::admin_get_user::AdminGetUserError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,14 +62,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::admin_get_user::AdminGetUserError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::admin_get_user::AdminGetUserError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -78,13 +78,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::admin_get_user::AdminGetUserError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::admin_get_user::AdminGetUserError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,13 +93,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::admin_get_user::AdminGetUserError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::admin_get_user::AdminGetUserError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,13 +108,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::admin_get_user::AdminGetUserError::UserNotFoundException({
+        "UserNotFoundException" => super::operation::admin_get_user::AdminGetUserError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -123,7 +123,7 @@
             }
             tmp
         }),
-        _ => crate::operation::admin_get_user::AdminGetUserError::generic(generic),
+        _ => super::operation::admin_get_user::AdminGetUserError::generic(generic),
     })
 }

@@ -132,25 +132,25 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::admin_get_user::AdminGetUserOutput, crate::operation::admin_get_user::AdminGetUserError> {
+) -> std::result::Result<super::operation::admin_get_user::AdminGetUserOutput, super::operation::admin_get_user::AdminGetUserError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::admin_get_user::builders::AdminGetUserOutputBuilder::default();
-        output = crate::protocol_serde::shape_admin_get_user::de_admin_get_user(_response_body, output)
-            .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?;
+        let mut output = super::operation::admin_get_user::builders::AdminGetUserOutputBuilder::default();
+        output = super::protocol_serde::shape_admin_get_user::de_admin_get_user(_response_body, output)
+            .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::admin_get_user_output_output_correct_errors(output)
+        super::serde_util::admin_get_user_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::admin_get_user::AdminGetUserError::unhandled)?
+            .map_err(super::operation::admin_get_user::AdminGetUserError::unhandled)?
     })
 }

 pub fn ser_admin_get_user_input(
-    input: &crate::operation::admin_get_user::AdminGetUserInput,
+    input: &super::operation::admin_get_user::AdminGetUserInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_admin_get_user_input::ser_admin_get_user_input_input(&mut object, input)?;
+    super::protocol_serde::shape_admin_get_user_input::ser_admin_get_user_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -157,12 +157,12 @@

 pub(crate) fn de_admin_get_user(
     _value: &[u8],
-    mut builder: crate::operation::admin_get_user::builders::AdminGetUserOutputBuilder,
+    mut builder: super::operation::admin_get_user::builders::AdminGetUserOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::admin_get_user::builders::AdminGetUserOutputBuilder,
+    super::operation::admin_get_user::builders::AdminGetUserOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -179,7 +179,7 @@
                     );
                 }
                 "UserAttributes" => {
-                    builder = builder.set_user_attributes(crate::protocol_serde::shape_attribute_list_type::de_attribute_list_type(
+                    builder = builder.set_user_attributes(super::protocol_serde::shape_attribute_list_type::de_attribute_list_type(
                         tokens,
                         _value,
                         depth + 1,
@@ -203,12 +203,12 @@
                 "UserStatus" => {
                     builder = builder.set_user_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::UserStatusType::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::UserStatusType::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "MFAOptions" => {
-                    builder = builder.set_mfa_options(crate::protocol_serde::shape_mfa_option_list_type::de_mfa_option_list_type(
+                    builder = builder.set_mfa_options(super::protocol_serde::shape_mfa_option_list_type::de_mfa_option_list_type(
                         tokens,
                         _value,
                         depth + 1,
@@ -223,7 +223,7 @@
                 }
                 "UserMFASettingList" => {
                     builder = builder.set_user_mfa_setting_list(
-                        crate::protocol_serde::shape_user_mfa_setting_list_type::de_user_mfa_setting_list_type(tokens, _value, depth + 1)?,
+                        super::protocol_serde::shape_user_mfa_setting_list_type::de_user_mfa_setting_list_type(tokens, _value, depth + 1)?,
                     );
                 }
                 _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
