--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ResourceServerType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ResourceServerType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ResourceServerTypeBuilder::default();
+            let mut builder = super::types::builders::ResourceServerTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -44,7 +44,7 @@
                         }
                         "Scopes" => {
                             builder = builder.set_scopes(
-                                crate::protocol_serde::shape_resource_server_scope_list_type::de_resource_server_scope_list_type(
+                                super::protocol_serde::shape_resource_server_scope_list_type::de_resource_server_scope_list_type(
                                     tokens,
                                     _value,
                                     depth + 1,
