--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_csv_header::GetCsvHeaderOutput, crate::operation::get_csv_header::GetCSVHeaderError> {
+) -> std::result::Result<super::operation::get_csv_header::GetCsvHeaderOutput, super::operation::get_csv_header::GetCSVHeaderError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled(generic)),
+        None => return Err(super::operation::get_csv_header::GetCSVHeaderError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::get_csv_header::GetCSVHeaderError::InternalErrorException({
+        "InternalErrorException" => super::operation::get_csv_header::GetCSVHeaderError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::get_csv_header::GetCSVHeaderError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::get_csv_header::GetCSVHeaderError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::get_csv_header::GetCSVHeaderError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::get_csv_header::GetCSVHeaderError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,14 +62,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::get_csv_header::GetCSVHeaderError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::get_csv_header::GetCSVHeaderError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -78,13 +78,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::get_csv_header::GetCSVHeaderError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::get_csv_header::GetCSVHeaderError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,13 +93,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::get_csv_header::GetCSVHeaderError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::get_csv_header::GetCSVHeaderError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,7 +108,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_csv_header::GetCSVHeaderError::generic(generic),
+        _ => super::operation::get_csv_header::GetCSVHeaderError::generic(generic),
     })
 }

@@ -117,12 +117,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_csv_header::GetCsvHeaderOutput, crate::operation::get_csv_header::GetCSVHeaderError> {
+) -> std::result::Result<super::operation::get_csv_header::GetCsvHeaderOutput, super::operation::get_csv_header::GetCSVHeaderError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_csv_header::builders::GetCsvHeaderOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_csv_header::de_get_csv_header(_response_body, output)
-            .map_err(crate::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
+        let mut output = super::operation::get_csv_header::builders::GetCsvHeaderOutputBuilder::default();
+        output = super::protocol_serde::shape_get_csv_header::de_get_csv_header(_response_body, output)
+            .map_err(super::operation::get_csv_header::GetCSVHeaderError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -129,11 +129,11 @@
 }

 pub fn ser_get_csv_header_input(
-    input: &crate::operation::get_csv_header::GetCsvHeaderInput,
+    input: &super::operation::get_csv_header::GetCsvHeaderInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_get_csv_header_input::ser_get_csv_header_input_input(&mut object, input)?;
+    super::protocol_serde::shape_get_csv_header_input::ser_get_csv_header_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -140,12 +140,12 @@

 pub(crate) fn de_get_csv_header(
     _value: &[u8],
-    mut builder: crate::operation::get_csv_header::builders::GetCsvHeaderOutputBuilder,
+    mut builder: super::operation::get_csv_header::builders::GetCsvHeaderOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_csv_header::builders::GetCsvHeaderOutputBuilder,
+    super::operation::get_csv_header::builders::GetCsvHeaderOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -162,7 +162,7 @@
                     );
                 }
                 "CSVHeader" => {
-                    builder = builder.set_csv_header(crate::protocol_serde::shape_list_of_string_types::de_list_of_string_types(
+                    builder = builder.set_csv_header(super::protocol_serde::shape_list_of_string_types::de_list_of_string_types(
                         tokens,
                         _value,
                         depth + 1,
