--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput,
-    crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
+    super::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput,
+    super::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled(
+            return Err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::InternalErrorException({
+        "InternalErrorException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -55,12 +55,12 @@
             tmp
         }),
         "InvalidUserPoolConfigurationException" => {
-            crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::InvalidUserPoolConfigurationException({
+            super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::InvalidUserPoolConfigurationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidUserPoolConfigurationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_user_pool_configuration_exception::de_invalid_user_pool_configuration_exception_json_err(_response_body, output).map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidUserPoolConfigurationExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_user_pool_configuration_exception::de_invalid_user_pool_configuration_exception_json_err(_response_body, output).map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -70,13 +70,13 @@
                 tmp
             })
         }
-        "NotAuthorizedException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -85,14 +85,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -101,13 +101,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -116,13 +116,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -131,13 +131,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::UserNotFoundException({
+        "UserNotFoundException" => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -146,7 +146,7 @@
             }
             tmp
         }),
-        _ => crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError::generic(generic),
+        _ => super::operation::admin_update_device_status::AdminUpdateDeviceStatusError::generic(generic),
     })
 }

@@ -156,12 +156,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput,
-    crate::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
+    super::operation::admin_update_device_status::AdminUpdateDeviceStatusOutput,
+    super::operation::admin_update_device_status::AdminUpdateDeviceStatusError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusOutputBuilder::default();
+        let mut output = super::operation::admin_update_device_status::builders::AdminUpdateDeviceStatusOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -168,11 +168,11 @@
 }

 pub fn ser_admin_update_device_status_input(
-    input: &crate::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
+    input: &super::operation::admin_update_device_status::AdminUpdateDeviceStatusInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_admin_update_device_status_input::ser_admin_update_device_status_input_input(&mut object, input)?;
+    super::protocol_serde::shape_admin_update_device_status_input::ser_admin_update_device_status_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
