--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::revoke_token::RevokeTokenOutput, crate::operation::revoke_token::RevokeTokenError> {
+) -> std::result::Result<super::super::operation::revoke_token::RevokeTokenOutput, super::super::operation::revoke_token::RevokeTokenError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::revoke_token::RevokeTokenError::unhandled(generic)),
+        None => return Err(super::super::operation::revoke_token::RevokeTokenError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ForbiddenException" => crate::operation::revoke_token::RevokeTokenError::ForbiddenException({
+        "ForbiddenException" => super::super::operation::revoke_token::RevokeTokenError::ForbiddenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ForbiddenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                let mut output = super::super::types::error::builders::ForbiddenExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::revoke_token::RevokeTokenError::InternalErrorException({
+        "InternalErrorException" => super::super::operation::revoke_token::RevokeTokenError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::revoke_token::RevokeTokenError::InvalidParameterException({
+        "InvalidParameterException" => super::super::operation::revoke_token::RevokeTokenError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,14 +62,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::revoke_token::RevokeTokenError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::super::operation::revoke_token::RevokeTokenError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                    super::super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -78,13 +78,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::revoke_token::RevokeTokenError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::revoke_token::RevokeTokenError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,13 +93,13 @@
             }
             tmp
         }),
-        "UnauthorizedException" => crate::operation::revoke_token::RevokeTokenError::UnauthorizedException({
+        "UnauthorizedException" => super::super::operation::revoke_token::RevokeTokenError::UnauthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnauthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_unauthorized_exception::de_unauthorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                let mut output = super::super::types::error::builders::UnauthorizedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_unauthorized_exception::de_unauthorized_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,14 +108,14 @@
             }
             tmp
         }),
-        "UnsupportedOperationException" => crate::operation::revoke_token::RevokeTokenError::UnsupportedOperationException({
+        "UnsupportedOperationException" => super::super::operation::revoke_token::RevokeTokenError::UnsupportedOperationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnsupportedOperationExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::UnsupportedOperationExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_unsupported_operation_exception::de_unsupported_operation_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                    super::super::protocol_serde::shape_unsupported_operation_exception::de_unsupported_operation_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -124,16 +124,16 @@
             }
             tmp
         }),
-        "UnsupportedTokenTypeException" => crate::operation::revoke_token::RevokeTokenError::UnsupportedTokenTypeException({
+        "UnsupportedTokenTypeException" => super::super::operation::revoke_token::RevokeTokenError::UnsupportedTokenTypeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnsupportedTokenTypeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_unsupported_token_type_exception::de_unsupported_token_type_exception_json_err(
+                let mut output = super::super::types::error::builders::UnsupportedTokenTypeExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_unsupported_token_type_exception::de_unsupported_token_type_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::revoke_token::RevokeTokenError::unhandled)?;
+                .map_err(super::super::operation::revoke_token::RevokeTokenError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -142,7 +142,7 @@
             }
             tmp
         }),
-        _ => crate::operation::revoke_token::RevokeTokenError::generic(generic),
+        _ => super::super::operation::revoke_token::RevokeTokenError::generic(generic),
     })
 }

@@ -151,10 +151,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::revoke_token::RevokeTokenOutput, crate::operation::revoke_token::RevokeTokenError> {
+) -> std::result::Result<super::super::operation::revoke_token::RevokeTokenOutput, super::super::operation::revoke_token::RevokeTokenError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::revoke_token::builders::RevokeTokenOutputBuilder::default();
+        let mut output = super::super::operation::revoke_token::builders::RevokeTokenOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -161,11 +161,11 @@
 }

 pub fn ser_revoke_token_input(
-    input: &crate::operation::revoke_token::RevokeTokenInput,
+    input: &super::super::operation::revoke_token::RevokeTokenInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_revoke_token_input::ser_revoke_token_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_revoke_token_input::ser_revoke_token_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
