--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_user_pool::CreateUserPoolOutput, crate::operation::create_user_pool::CreateUserPoolError> {
+) -> std::result::Result<super::operation::create_user_pool::CreateUserPoolOutput, super::operation::create_user_pool::CreateUserPoolError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::create_user_pool::CreateUserPoolError::unhandled(generic)),
+        None => return Err(super::operation::create_user_pool::CreateUserPoolError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "FeatureUnavailableInTierException" => crate::operation::create_user_pool::CreateUserPoolError::FeatureUnavailableInTierException({
+        "FeatureUnavailableInTierException" => super::operation::create_user_pool::CreateUserPoolError::FeatureUnavailableInTierException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::FeatureUnavailableInTierExceptionBuilder::default();
-                output = crate::protocol_serde::shape_feature_unavailable_in_tier_exception::de_feature_unavailable_in_tier_exception_json_err(
+                let mut output = super::types::error::builders::FeatureUnavailableInTierExceptionBuilder::default();
+                output = super::protocol_serde::shape_feature_unavailable_in_tier_exception::de_feature_unavailable_in_tier_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::create_user_pool::CreateUserPoolError::InternalErrorException({
+        "InternalErrorException" => super::operation::create_user_pool::CreateUserPoolError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -51,12 +51,12 @@
             tmp
         }),
         "InvalidEmailRoleAccessPolicyException" => {
-            crate::operation::create_user_pool::CreateUserPoolError::InvalidEmailRoleAccessPolicyException({
+            super::operation::create_user_pool::CreateUserPoolError::InvalidEmailRoleAccessPolicyException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidEmailRoleAccessPolicyExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_email_role_access_policy_exception::de_invalid_email_role_access_policy_exception_json_err(_response_body, output).map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidEmailRoleAccessPolicyExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_email_role_access_policy_exception::de_invalid_email_role_access_policy_exception_json_err(_response_body, output).map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -66,13 +66,13 @@
                 tmp
             })
         }
-        "InvalidParameterException" => crate::operation::create_user_pool::CreateUserPoolError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::create_user_pool::CreateUserPoolError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -81,16 +81,16 @@
             }
             tmp
         }),
-        "InvalidSmsRoleAccessPolicyException" => crate::operation::create_user_pool::CreateUserPoolError::InvalidSmsRoleAccessPolicyException({
+        "InvalidSmsRoleAccessPolicyException" => super::operation::create_user_pool::CreateUserPoolError::InvalidSmsRoleAccessPolicyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSmsRoleAccessPolicyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_sms_role_access_policy_exception::de_invalid_sms_role_access_policy_exception_json_err(
+                let mut output = super::types::error::builders::InvalidSmsRoleAccessPolicyExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_sms_role_access_policy_exception::de_invalid_sms_role_access_policy_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -100,12 +100,12 @@
             tmp
         }),
         "InvalidSmsRoleTrustRelationshipException" => {
-            crate::operation::create_user_pool::CreateUserPoolError::InvalidSmsRoleTrustRelationshipException({
+            super::operation::create_user_pool::CreateUserPoolError::InvalidSmsRoleTrustRelationshipException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidSmsRoleTrustRelationshipExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_sms_role_trust_relationship_exception::de_invalid_sms_role_trust_relationship_exception_json_err(_response_body, output).map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidSmsRoleTrustRelationshipExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_sms_role_trust_relationship_exception::de_invalid_sms_role_trust_relationship_exception_json_err(_response_body, output).map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -115,13 +115,13 @@
                 tmp
             })
         }
-        "LimitExceededException" => crate::operation::create_user_pool::CreateUserPoolError::LimitExceededException({
+        "LimitExceededException" => super::operation::create_user_pool::CreateUserPoolError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
+                    .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -130,13 +130,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::create_user_pool::CreateUserPoolError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::create_user_pool::CreateUserPoolError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -145,16 +145,16 @@
             }
             tmp
         }),
-        "TierChangeNotAllowedException" => crate::operation::create_user_pool::CreateUserPoolError::TierChangeNotAllowedException({
+        "TierChangeNotAllowedException" => super::operation::create_user_pool::CreateUserPoolError::TierChangeNotAllowedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TierChangeNotAllowedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_tier_change_not_allowed_exception::de_tier_change_not_allowed_exception_json_err(
+                let mut output = super::types::error::builders::TierChangeNotAllowedExceptionBuilder::default();
+                output = super::protocol_serde::shape_tier_change_not_allowed_exception::de_tier_change_not_allowed_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -163,13 +163,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::create_user_pool::CreateUserPoolError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::create_user_pool::CreateUserPoolError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -178,13 +178,13 @@
             }
             tmp
         }),
-        "UserPoolTaggingException" => crate::operation::create_user_pool::CreateUserPoolError::UserPoolTaggingException({
+        "UserPoolTaggingException" => super::operation::create_user_pool::CreateUserPoolError::UserPoolTaggingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserPoolTaggingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_pool_tagging_exception::de_user_pool_tagging_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+                let mut output = super::types::error::builders::UserPoolTaggingExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_pool_tagging_exception::de_user_pool_tagging_exception_json_err(_response_body, output)
+                    .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -193,7 +193,7 @@
             }
             tmp
         }),
-        _ => crate::operation::create_user_pool::CreateUserPoolError::generic(generic),
+        _ => super::operation::create_user_pool::CreateUserPoolError::generic(generic),
     })
 }

@@ -202,12 +202,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_user_pool::CreateUserPoolOutput, crate::operation::create_user_pool::CreateUserPoolError> {
+) -> std::result::Result<super::operation::create_user_pool::CreateUserPoolOutput, super::operation::create_user_pool::CreateUserPoolError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::create_user_pool::builders::CreateUserPoolOutputBuilder::default();
-        output = crate::protocol_serde::shape_create_user_pool::de_create_user_pool(_response_body, output)
-            .map_err(crate::operation::create_user_pool::CreateUserPoolError::unhandled)?;
+        let mut output = super::operation::create_user_pool::builders::CreateUserPoolOutputBuilder::default();
+        output = super::protocol_serde::shape_create_user_pool::de_create_user_pool(_response_body, output)
+            .map_err(super::operation::create_user_pool::CreateUserPoolError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -214,11 +214,11 @@
 }

 pub fn ser_create_user_pool_input(
-    input: &crate::operation::create_user_pool::CreateUserPoolInput,
+    input: &super::operation::create_user_pool::CreateUserPoolInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_create_user_pool_input::ser_create_user_pool_input_input(&mut object, input)?;
+    super::protocol_serde::shape_create_user_pool_input::ser_create_user_pool_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -225,12 +225,12 @@

 pub(crate) fn de_create_user_pool(
     _value: &[u8],
-    mut builder: crate::operation::create_user_pool::builders::CreateUserPoolOutputBuilder,
+    mut builder: super::operation::create_user_pool::builders::CreateUserPoolOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::create_user_pool::builders::CreateUserPoolOutputBuilder,
+    super::operation::create_user_pool::builders::CreateUserPoolOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -240,7 +240,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "UserPool" => {
-                    builder = builder.set_user_pool(crate::protocol_serde::shape_user_pool_type::de_user_pool_type(tokens, _value, depth + 1)?);
+                    builder = builder.set_user_pool(super::protocol_serde::shape_user_pool_type::de_user_pool_type(tokens, _value, depth + 1)?);
                 }
                 _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
             },
