--- original
+++ modified
@@ -5,31 +5,31 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_user_pool_domain::DeleteUserPoolDomainOutput,
-    crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError,
+    super::operation::delete_user_pool_domain::DeleteUserPoolDomainOutput,
+    super::operation::delete_user_pool_domain::DeleteUserPoolDomainError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled(generic)),
+        None => return Err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ConcurrentModificationException" => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::ConcurrentModificationException({
+        "ConcurrentModificationException" => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::ConcurrentModificationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ConcurrentModificationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_json_err(
+                let mut output = super::types::error::builders::ConcurrentModificationExceptionBuilder::default();
+                output = super::protocol_serde::shape_concurrent_modification_exception::de_concurrent_modification_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+                .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -38,13 +38,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::InternalErrorException({
+        "InternalErrorException" => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -53,13 +53,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,13 +68,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -83,14 +83,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -99,13 +99,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -114,7 +114,7 @@
             }
             tmp
         }),
-        _ => crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError::generic(generic),
+        _ => super::operation::delete_user_pool_domain::DeleteUserPoolDomainError::generic(generic),
     })
 }

@@ -124,12 +124,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_user_pool_domain::DeleteUserPoolDomainOutput,
-    crate::operation::delete_user_pool_domain::DeleteUserPoolDomainError,
+    super::operation::delete_user_pool_domain::DeleteUserPoolDomainOutput,
+    super::operation::delete_user_pool_domain::DeleteUserPoolDomainError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainOutputBuilder::default();
+        let mut output = super::operation::delete_user_pool_domain::builders::DeleteUserPoolDomainOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -136,11 +136,11 @@
 }

 pub fn ser_delete_user_pool_domain_input(
-    input: &crate::operation::delete_user_pool_domain::DeleteUserPoolDomainInput,
+    input: &super::operation::delete_user_pool_domain::DeleteUserPoolDomainInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_delete_user_pool_domain_input::ser_delete_user_pool_domain_input_input(&mut object, input)?;
+    super::protocol_serde::shape_delete_user_pool_domain_input::ser_delete_user_pool_domain_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
