--- original
+++ modified
@@ -5,18 +5,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::describe_user_pool_domain::DescribeUserPoolDomainOutput,
-    crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError,
+    super::operation::describe_user_pool_domain::DescribeUserPoolDomainOutput,
+    super::operation::describe_user_pool_domain::DescribeUserPoolDomainError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled(
+            return Err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled(
                 generic,
             ))
         }
@@ -24,13 +24,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::InternalErrorException({
+        "InternalErrorException" => super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -39,13 +39,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -54,13 +54,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -69,14 +69,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -85,13 +85,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -100,7 +100,7 @@
             }
             tmp
         }),
-        _ => crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::generic(generic),
+        _ => super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::generic(generic),
     })
 }

@@ -110,14 +110,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::describe_user_pool_domain::DescribeUserPoolDomainOutput,
-    crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError,
+    super::operation::describe_user_pool_domain::DescribeUserPoolDomainOutput,
+    super::operation::describe_user_pool_domain::DescribeUserPoolDomainError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::describe_user_pool_domain::builders::DescribeUserPoolDomainOutputBuilder::default();
-        output = crate::protocol_serde::shape_describe_user_pool_domain::de_describe_user_pool_domain(_response_body, output)
-            .map_err(crate::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
+        let mut output = super::operation::describe_user_pool_domain::builders::DescribeUserPoolDomainOutputBuilder::default();
+        output = super::protocol_serde::shape_describe_user_pool_domain::de_describe_user_pool_domain(_response_body, output)
+            .map_err(super::operation::describe_user_pool_domain::DescribeUserPoolDomainError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -124,11 +124,11 @@
 }

 pub fn ser_describe_user_pool_domain_input(
-    input: &crate::operation::describe_user_pool_domain::DescribeUserPoolDomainInput,
+    input: &super::operation::describe_user_pool_domain::DescribeUserPoolDomainInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_describe_user_pool_domain_input::ser_describe_user_pool_domain_input_input(&mut object, input)?;
+    super::protocol_serde::shape_describe_user_pool_domain_input::ser_describe_user_pool_domain_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -135,12 +135,12 @@

 pub(crate) fn de_describe_user_pool_domain(
     _value: &[u8],
-    mut builder: crate::operation::describe_user_pool_domain::builders::DescribeUserPoolDomainOutputBuilder,
+    mut builder: super::operation::describe_user_pool_domain::builders::DescribeUserPoolDomainOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::describe_user_pool_domain::builders::DescribeUserPoolDomainOutputBuilder,
+    super::operation::describe_user_pool_domain::builders::DescribeUserPoolDomainOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -150,7 +150,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "DomainDescription" => {
-                    builder = builder.set_domain_description(crate::protocol_serde::shape_domain_description_type::de_domain_description_type(
+                    builder = builder.set_domain_description(super::protocol_serde::shape_domain_description_type::de_domain_description_type(
                         tokens,
                         _value,
                         depth + 1,
