--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::admin_list_devices::AdminListDevicesOutput, crate::operation::admin_list_devices::AdminListDevicesError> {
+) -> std::result::Result<super::operation::admin_list_devices::AdminListDevicesOutput, super::operation::admin_list_devices::AdminListDevicesError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled(generic)),
+        None => return Err(super::operation::admin_list_devices::AdminListDevicesError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InternalErrorException" => crate::operation::admin_list_devices::AdminListDevicesError::InternalErrorException({
+        "InternalErrorException" => super::operation::admin_list_devices::AdminListDevicesError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::admin_list_devices::AdminListDevicesError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::admin_list_devices::AdminListDevicesError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -48,12 +48,12 @@
             tmp
         }),
         "InvalidUserPoolConfigurationException" => {
-            crate::operation::admin_list_devices::AdminListDevicesError::InvalidUserPoolConfigurationException({
+            super::operation::admin_list_devices::AdminListDevicesError::InvalidUserPoolConfigurationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidUserPoolConfigurationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_user_pool_configuration_exception::de_invalid_user_pool_configuration_exception_json_err(_response_body, output).map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                    let mut output = super::types::error::builders::InvalidUserPoolConfigurationExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_user_pool_configuration_exception::de_invalid_user_pool_configuration_exception_json_err(_response_body, output).map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -63,13 +63,13 @@
                 tmp
             })
         }
-        "NotAuthorizedException" => crate::operation::admin_list_devices::AdminListDevicesError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::admin_list_devices::AdminListDevicesError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -78,14 +78,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::admin_list_devices::AdminListDevicesError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::admin_list_devices::AdminListDevicesError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -94,13 +94,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::admin_list_devices::AdminListDevicesError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::admin_list_devices::AdminListDevicesError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -109,13 +109,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::admin_list_devices::AdminListDevicesError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::admin_list_devices::AdminListDevicesError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -124,7 +124,7 @@
             }
             tmp
         }),
-        _ => crate::operation::admin_list_devices::AdminListDevicesError::generic(generic),
+        _ => super::operation::admin_list_devices::AdminListDevicesError::generic(generic),
     })
 }

@@ -133,12 +133,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::admin_list_devices::AdminListDevicesOutput, crate::operation::admin_list_devices::AdminListDevicesError> {
+) -> std::result::Result<super::operation::admin_list_devices::AdminListDevicesOutput, super::operation::admin_list_devices::AdminListDevicesError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::admin_list_devices::builders::AdminListDevicesOutputBuilder::default();
-        output = crate::protocol_serde::shape_admin_list_devices::de_admin_list_devices(_response_body, output)
-            .map_err(crate::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
+        let mut output = super::operation::admin_list_devices::builders::AdminListDevicesOutputBuilder::default();
+        output = super::protocol_serde::shape_admin_list_devices::de_admin_list_devices(_response_body, output)
+            .map_err(super::operation::admin_list_devices::AdminListDevicesError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -145,11 +145,11 @@
 }

 pub fn ser_admin_list_devices_input(
-    input: &crate::operation::admin_list_devices::AdminListDevicesInput,
+    input: &super::operation::admin_list_devices::AdminListDevicesInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_admin_list_devices_input::ser_admin_list_devices_input_input(&mut object, input)?;
+    super::protocol_serde::shape_admin_list_devices_input::ser_admin_list_devices_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -156,12 +156,12 @@

 pub(crate) fn de_admin_list_devices(
     _value: &[u8],
-    mut builder: crate::operation::admin_list_devices::builders::AdminListDevicesOutputBuilder,
+    mut builder: super::operation::admin_list_devices::builders::AdminListDevicesOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::admin_list_devices::builders::AdminListDevicesOutputBuilder,
+    super::operation::admin_list_devices::builders::AdminListDevicesOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -171,7 +171,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Devices" => {
-                    builder = builder.set_devices(crate::protocol_serde::shape_device_list_type::de_device_list_type(
+                    builder = builder.set_devices(super::protocol_serde::shape_device_list_type::de_device_list_type(
                         tokens,
                         _value,
                         depth + 1,
