--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::change_password::ChangePasswordOutput, crate::operation::change_password::ChangePasswordError> {
+) -> std::result::Result<super::operation::change_password::ChangePasswordOutput, super::operation::change_password::ChangePasswordError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::change_password::ChangePasswordError::unhandled(generic)),
+        None => return Err(super::operation::change_password::ChangePasswordError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ForbiddenException" => crate::operation::change_password::ChangePasswordError::ForbiddenException({
+        "ForbiddenException" => super::operation::change_password::ChangePasswordError::ForbiddenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ForbiddenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::ForbiddenExceptionBuilder::default();
+                output = super::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::change_password::ChangePasswordError::InternalErrorException({
+        "InternalErrorException" => super::operation::change_password::ChangePasswordError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::change_password::ChangePasswordError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::change_password::ChangePasswordError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,13 +62,13 @@
             }
             tmp
         }),
-        "InvalidPasswordException" => crate::operation::change_password::ChangePasswordError::InvalidPasswordException({
+        "InvalidPasswordException" => super::operation::change_password::ChangePasswordError::InvalidPasswordException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidPasswordExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_password_exception::de_invalid_password_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidPasswordExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_password_exception::de_invalid_password_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -77,13 +77,13 @@
             }
             tmp
         }),
-        "LimitExceededException" => crate::operation::change_password::ChangePasswordError::LimitExceededException({
+        "LimitExceededException" => super::operation::change_password::ChangePasswordError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -92,13 +92,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::change_password::ChangePasswordError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::change_password::ChangePasswordError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -107,14 +107,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::change_password::ChangePasswordError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::change_password::ChangePasswordError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -124,12 +124,12 @@
             tmp
         }),
         "PasswordHistoryPolicyViolationException" => {
-            crate::operation::change_password::ChangePasswordError::PasswordHistoryPolicyViolationException({
+            super::operation::change_password::ChangePasswordError::PasswordHistoryPolicyViolationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::PasswordHistoryPolicyViolationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_password_history_policy_violation_exception::de_password_history_policy_violation_exception_json_err(_response_body, output).map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                    let mut output = super::types::error::builders::PasswordHistoryPolicyViolationExceptionBuilder::default();
+                    output = super::protocol_serde::shape_password_history_policy_violation_exception::de_password_history_policy_violation_exception_json_err(_response_body, output).map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -139,16 +139,16 @@
                 tmp
             })
         }
-        "PasswordResetRequiredException" => crate::operation::change_password::ChangePasswordError::PasswordResetRequiredException({
+        "PasswordResetRequiredException" => super::operation::change_password::ChangePasswordError::PasswordResetRequiredException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
-                output = crate::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
+                let mut output = super::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
+                output = super::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -157,13 +157,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::change_password::ChangePasswordError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::change_password::ChangePasswordError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -172,13 +172,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::change_password::ChangePasswordError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::change_password::ChangePasswordError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -187,13 +187,13 @@
             }
             tmp
         }),
-        "UserNotConfirmedException" => crate::operation::change_password::ChangePasswordError::UserNotConfirmedException({
+        "UserNotConfirmedException" => super::operation::change_password::ChangePasswordError::UserNotConfirmedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotConfirmedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotConfirmedExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -202,13 +202,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::change_password::ChangePasswordError::UserNotFoundException({
+        "UserNotFoundException" => super::operation::change_password::ChangePasswordError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::change_password::ChangePasswordError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::change_password::ChangePasswordError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -217,7 +217,7 @@
             }
             tmp
         }),
-        _ => crate::operation::change_password::ChangePasswordError::generic(generic),
+        _ => super::operation::change_password::ChangePasswordError::generic(generic),
     })
 }

@@ -226,10 +226,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::change_password::ChangePasswordOutput, crate::operation::change_password::ChangePasswordError> {
+) -> std::result::Result<super::operation::change_password::ChangePasswordOutput, super::operation::change_password::ChangePasswordError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::change_password::builders::ChangePasswordOutputBuilder::default();
+        let mut output = super::operation::change_password::builders::ChangePasswordOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -236,11 +236,11 @@
 }

 pub fn ser_change_password_input(
-    input: &crate::operation::change_password::ChangePasswordInput,
+    input: &super::operation::change_password::ChangePasswordInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_change_password_input::ser_change_password_input_input(&mut object, input)?;
+    super::protocol_serde::shape_change_password_input::ser_change_password_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
