--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::set_user_mfa_preference::SetUserMfaPreferenceOutput,
-    crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError,
+    super::super::operation::set_user_mfa_preference::SetUserMfaPreferenceOutput,
+    super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled(generic)),
+        None => return Err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ForbiddenException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::ForbiddenException({
+        "ForbiddenException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::ForbiddenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ForbiddenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::ForbiddenExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::InternalErrorException({
+        "InternalErrorException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::InvalidParameterException({
+        "InvalidParameterException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,13 +65,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::NotAuthorizedException({
+        "NotAuthorizedException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -80,14 +80,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                    super::super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -96,16 +96,16 @@
             }
             tmp
         }),
-        "PasswordResetRequiredException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::PasswordResetRequiredException({
+        "PasswordResetRequiredException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::PasswordResetRequiredException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
-                output = crate::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
+                let mut output = super::super::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -114,13 +114,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -129,13 +129,13 @@
             }
             tmp
         }),
-        "UserNotConfirmedException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::UserNotConfirmedException({
+        "UserNotConfirmedException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::UserNotConfirmedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotConfirmedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::UserNotConfirmedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -144,13 +144,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::UserNotFoundException({
+        "UserNotFoundException" => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
+                let mut output = super::super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -159,7 +159,7 @@
             }
             tmp
         }),
-        _ => crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError::generic(generic),
+        _ => super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError::generic(generic),
     })
 }

@@ -169,12 +169,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::set_user_mfa_preference::SetUserMfaPreferenceOutput,
-    crate::operation::set_user_mfa_preference::SetUserMFAPreferenceError,
+    super::super::operation::set_user_mfa_preference::SetUserMfaPreferenceOutput,
+    super::super::operation::set_user_mfa_preference::SetUserMFAPreferenceError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::set_user_mfa_preference::builders::SetUserMfaPreferenceOutputBuilder::default();
+        let mut output = super::super::operation::set_user_mfa_preference::builders::SetUserMfaPreferenceOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -181,11 +181,11 @@
 }

 pub fn ser_set_user_mfa_preference_input(
-    input: &crate::operation::set_user_mfa_preference::SetUserMfaPreferenceInput,
+    input: &super::super::operation::set_user_mfa_preference::SetUserMfaPreferenceInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_set_user_mfa_preference_input::ser_set_user_mfa_preference_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_set_user_mfa_preference_input::ser_set_user_mfa_preference_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
