--- original
+++ modified
@@ -5,28 +5,28 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_user_auth_factors::GetUserAuthFactorsOutput,
-    crate::operation::get_user_auth_factors::GetUserAuthFactorsError,
+    super::operation::get_user_auth_factors::GetUserAuthFactorsOutput,
+    super::operation::get_user_auth_factors::GetUserAuthFactorsError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled(generic)),
+        None => return Err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "ForbiddenException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::ForbiddenException({
+        "ForbiddenException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::ForbiddenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ForbiddenExceptionBuilder::default();
-                output = crate::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::ForbiddenExceptionBuilder::default();
+                output = super::protocol_serde::shape_forbidden_exception::de_forbidden_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "InternalErrorException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::InternalErrorException({
+        "InternalErrorException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::InternalErrorException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InternalErrorExceptionBuilder::default();
-                output = crate::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::InternalErrorExceptionBuilder::default();
+                output = super::protocol_serde::shape_internal_error_exception::de_internal_error_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "InvalidParameterException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::InvalidParameterException({
+        "InvalidParameterException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::InvalidParameterException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidParameterExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_exception::de_invalid_parameter_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,13 +65,13 @@
             }
             tmp
         }),
-        "NotAuthorizedException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::NotAuthorizedException({
+        "NotAuthorizedException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::NotAuthorizedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotAuthorizedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::NotAuthorizedExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_authorized_exception::de_not_authorized_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -80,14 +80,14 @@
             }
             tmp
         }),
-        "OperationNotEnabledException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::OperationNotEnabledException({
+        "OperationNotEnabledException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::OperationNotEnabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::OperationNotEnabledExceptionBuilder::default();
+                let mut output = super::types::error::builders::OperationNotEnabledExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                    super::protocol_serde::shape_operation_not_enabled_exception::de_operation_not_enabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -96,16 +96,16 @@
             }
             tmp
         }),
-        "PasswordResetRequiredException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::PasswordResetRequiredException({
+        "PasswordResetRequiredException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::PasswordResetRequiredException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
-                output = crate::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
+                let mut output = super::types::error::builders::PasswordResetRequiredExceptionBuilder::default();
+                output = super::protocol_serde::shape_password_reset_required_exception::de_password_reset_required_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -114,13 +114,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -129,13 +129,13 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -144,13 +144,13 @@
             }
             tmp
         }),
-        "UserNotConfirmedException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::UserNotConfirmedException({
+        "UserNotConfirmedException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::UserNotConfirmedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotConfirmedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotConfirmedExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_confirmed_exception::de_user_not_confirmed_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -159,13 +159,13 @@
             }
             tmp
         }),
-        "UserNotFoundException" => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::UserNotFoundException({
+        "UserNotFoundException" => super::operation::get_user_auth_factors::GetUserAuthFactorsError::UserNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UserNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+                let mut output = super::types::error::builders::UserNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_user_not_found_exception::de_user_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -174,7 +174,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_user_auth_factors::GetUserAuthFactorsError::generic(generic),
+        _ => super::operation::get_user_auth_factors::GetUserAuthFactorsError::generic(generic),
     })
 }

@@ -184,27 +184,27 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_user_auth_factors::GetUserAuthFactorsOutput,
-    crate::operation::get_user_auth_factors::GetUserAuthFactorsError,
+    super::operation::get_user_auth_factors::GetUserAuthFactorsOutput,
+    super::operation::get_user_auth_factors::GetUserAuthFactorsError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_user_auth_factors::builders::GetUserAuthFactorsOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_user_auth_factors::de_get_user_auth_factors(_response_body, output)
-            .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
+        let mut output = super::operation::get_user_auth_factors::builders::GetUserAuthFactorsOutputBuilder::default();
+        output = super::protocol_serde::shape_get_user_auth_factors::de_get_user_auth_factors(_response_body, output)
+            .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_user_auth_factors_output_output_correct_errors(output)
+        super::serde_util::get_user_auth_factors_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?
+            .map_err(super::operation::get_user_auth_factors::GetUserAuthFactorsError::unhandled)?
     })
 }

 pub fn ser_get_user_auth_factors_input(
-    input: &crate::operation::get_user_auth_factors::GetUserAuthFactorsInput,
+    input: &super::operation::get_user_auth_factors::GetUserAuthFactorsInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_get_user_auth_factors_input::ser_get_user_auth_factors_input_input(&mut object, input)?;
+    super::protocol_serde::shape_get_user_auth_factors_input::ser_get_user_auth_factors_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -211,12 +211,12 @@

 pub(crate) fn de_get_user_auth_factors(
     _value: &[u8],
-    mut builder: crate::operation::get_user_auth_factors::builders::GetUserAuthFactorsOutputBuilder,
+    mut builder: super::operation::get_user_auth_factors::builders::GetUserAuthFactorsOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_user_auth_factors::builders::GetUserAuthFactorsOutputBuilder,
+    super::operation::get_user_auth_factors::builders::GetUserAuthFactorsOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -241,12 +241,12 @@
                 }
                 "UserMFASettingList" => {
                     builder = builder.set_user_mfa_setting_list(
-                        crate::protocol_serde::shape_user_mfa_setting_list_type::de_user_mfa_setting_list_type(tokens, _value, depth + 1)?,
+                        super::protocol_serde::shape_user_mfa_setting_list_type::de_user_mfa_setting_list_type(tokens, _value, depth + 1)?,
                     );
                 }
                 "ConfiguredUserAuthFactors" => {
                     builder = builder.set_configured_user_auth_factors(
-                        crate::protocol_serde::shape_configured_user_auth_factors_list_type::de_configured_user_auth_factors_list_type(
+                        super::protocol_serde::shape_configured_user_auth_factors_list_type::de_configured_user_auth_factors_list_type(
                             tokens,
                             _value,
                             depth + 1,
