--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::UserPoolType>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::UserPoolType>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::UserPoolTypeBuilder::default();
+            let mut builder = super::types::builders::UserPoolTypeBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -36,7 +36,7 @@
                             );
                         }
                         "Policies" => {
-                            builder = builder.set_policies(crate::protocol_serde::shape_user_pool_policy_type::de_user_pool_policy_type(
+                            builder = builder.set_policies(super::protocol_serde::shape_user_pool_policy_type::de_user_pool_policy_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -45,12 +45,12 @@
                         "DeletionProtection" => {
                             builder = builder.set_deletion_protection(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::DeletionProtectionType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::DeletionProtectionType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "LambdaConfig" => {
-                            builder = builder.set_lambda_config(crate::protocol_serde::shape_lambda_config_type::de_lambda_config_type(
+                            builder = builder.set_lambda_config(super::protocol_serde::shape_lambda_config_type::de_lambda_config_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -59,7 +59,7 @@
                         "Status" => {
                             builder = builder.set_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::StatusType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::StatusType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -77,12 +77,12 @@
                         }
                         "SchemaAttributes" => {
                             builder = builder.set_schema_attributes(
-                                crate::protocol_serde::shape_schema_attributes_list_type::de_schema_attributes_list_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_schema_attributes_list_type::de_schema_attributes_list_type(tokens, _value, depth + 1)?,
                             );
                         }
                         "AutoVerifiedAttributes" => {
                             builder = builder.set_auto_verified_attributes(
-                                crate::protocol_serde::shape_verified_attributes_list_type::de_verified_attributes_list_type(
+                                super::protocol_serde::shape_verified_attributes_list_type::de_verified_attributes_list_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -91,12 +91,12 @@
                         }
                         "AliasAttributes" => {
                             builder = builder.set_alias_attributes(
-                                crate::protocol_serde::shape_alias_attributes_list_type::de_alias_attributes_list_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_alias_attributes_list_type::de_alias_attributes_list_type(tokens, _value, depth + 1)?,
                             );
                         }
                         "UsernameAttributes" => {
                             builder = builder.set_username_attributes(
-                                crate::protocol_serde::shape_username_attributes_list_type::de_username_attributes_list_type(
+                                super::protocol_serde::shape_username_attributes_list_type::de_username_attributes_list_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -126,7 +126,7 @@
                         }
                         "VerificationMessageTemplate" => {
                             builder = builder.set_verification_message_template(
-                                crate::protocol_serde::shape_verification_message_template_type::de_verification_message_template_type(
+                                super::protocol_serde::shape_verification_message_template_type::de_verification_message_template_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -142,7 +142,7 @@
                         }
                         "UserAttributeUpdateSettings" => {
                             builder = builder.set_user_attribute_update_settings(
-                                crate::protocol_serde::shape_user_attribute_update_settings_type::de_user_attribute_update_settings_type(
+                                super::protocol_serde::shape_user_attribute_update_settings_type::de_user_attribute_update_settings_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -152,13 +152,13 @@
                         "MfaConfiguration" => {
                             builder = builder.set_mfa_configuration(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::UserPoolMfaType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::UserPoolMfaType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "DeviceConfiguration" => {
                             builder = builder.set_device_configuration(
-                                crate::protocol_serde::shape_device_configuration_type::de_device_configuration_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_device_configuration_type::de_device_configuration_type(tokens, _value, depth + 1)?,
                             );
                         }
                         "EstimatedNumberOfUsers" => {
@@ -170,11 +170,11 @@
                         }
                         "EmailConfiguration" => {
                             builder = builder.set_email_configuration(
-                                crate::protocol_serde::shape_email_configuration_type::de_email_configuration_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_email_configuration_type::de_email_configuration_type(tokens, _value, depth + 1)?,
                             );
                         }
                         "SmsConfiguration" => {
-                            builder = builder.set_sms_configuration(crate::protocol_serde::shape_sms_configuration_type::de_sms_configuration_type(
+                            builder = builder.set_sms_configuration(super::protocol_serde::shape_sms_configuration_type::de_sms_configuration_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -181,7 +181,7 @@
                             )?);
                         }
                         "UserPoolTags" => {
-                            builder = builder.set_user_pool_tags(crate::protocol_serde::shape_user_pool_tags_type::de_user_pool_tags_type(
+                            builder = builder.set_user_pool_tags(super::protocol_serde::shape_user_pool_tags_type::de_user_pool_tags_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -217,7 +217,7 @@
                         }
                         "AdminCreateUserConfig" => {
                             builder = builder.set_admin_create_user_config(
-                                crate::protocol_serde::shape_admin_create_user_config_type::de_admin_create_user_config_type(
+                                super::protocol_serde::shape_admin_create_user_config_type::de_admin_create_user_config_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -225,7 +225,7 @@
                             );
                         }
                         "UserPoolAddOns" => {
-                            builder = builder.set_user_pool_add_ons(crate::protocol_serde::shape_user_pool_add_ons_type::de_user_pool_add_ons_type(
+                            builder = builder.set_user_pool_add_ons(super::protocol_serde::shape_user_pool_add_ons_type::de_user_pool_add_ons_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -233,7 +233,7 @@
                         }
                         "UsernameConfiguration" => {
                             builder = builder.set_username_configuration(
-                                crate::protocol_serde::shape_username_configuration_type::de_username_configuration_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_username_configuration_type::de_username_configuration_type(tokens, _value, depth + 1)?,
                             );
                         }
                         "Arn" => {
@@ -245,7 +245,7 @@
                         }
                         "AccountRecoverySetting" => {
                             builder = builder.set_account_recovery_setting(
-                                crate::protocol_serde::shape_account_recovery_setting_type::de_account_recovery_setting_type(
+                                super::protocol_serde::shape_account_recovery_setting_type::de_account_recovery_setting_type(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -255,12 +255,12 @@
                         "UserPoolTier" => {
                             builder = builder.set_user_pool_tier(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::UserPoolTierType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::UserPoolTierType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "KeyConfiguration" => {
-                            builder = builder.set_key_configuration(crate::protocol_serde::shape_key_configuration_type::de_key_configuration_type(
+                            builder = builder.set_key_configuration(super::protocol_serde::shape_key_configuration_type::de_key_configuration_type(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -268,7 +268,7 @@
                         }
                         "IssuerConfiguration" => {
                             builder = builder.set_issuer_configuration(
-                                crate::protocol_serde::shape_issuer_configuration_type::de_issuer_configuration_type(tokens, _value, depth + 1)?,
+                                super::protocol_serde::shape_issuer_configuration_type::de_issuer_configuration_type(tokens, _value, depth + 1)?,
                             );
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
