--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_alias::GetAliasOutput, crate::operation::get_alias::GetAliasError> {
+) -> std::result::Result<super::super::operation::get_alias::GetAliasOutput, super::super::operation::get_alias::GetAliasError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_alias::GetAliasError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::get_alias::GetAliasError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_alias::GetAliasError::unhandled(generic)),
+        None => return Err(super::super::operation::get_alias::GetAliasError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidParameterValueException" => crate::operation::get_alias::GetAliasError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::super::operation::get_alias::GetAliasError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::get_alias::GetAliasError::unhandled)?;
+                .map_err(super::super::operation::get_alias::GetAliasError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,13 +35,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::get_alias::GetAliasError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::get_alias::GetAliasError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_alias::GetAliasError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_alias::GetAliasError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -50,13 +50,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::get_alias::GetAliasError::ServiceException({
+        "ServiceException" => super::super::operation::get_alias::GetAliasError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_alias::GetAliasError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_alias::GetAliasError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -65,16 +65,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::get_alias::GetAliasError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::get_alias::GetAliasError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_alias::GetAliasError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::get_alias::GetAliasError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::get_alias::GetAliasError::unhandled("Failed to parse retryAfterSeconds from header `Retry-After")
+                    super::super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::super::operation::get_alias::GetAliasError::unhandled("Failed to parse retryAfterSeconds from header `Retry-After")
                     })?,
                 );
                 let output = output.meta(generic);
@@ -85,7 +85,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_alias::GetAliasError::generic(generic),
+        _ => super::super::operation::get_alias::GetAliasError::generic(generic),
     })
 }

@@ -94,12 +94,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::get_alias::GetAliasOutput, crate::operation::get_alias::GetAliasError> {
+) -> std::result::Result<super::super::operation::get_alias::GetAliasOutput, super::super::operation::get_alias::GetAliasError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_alias::builders::GetAliasOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_alias::de_get_alias(_response_body, output)
-            .map_err(crate::operation::get_alias::GetAliasError::unhandled)?;
+        let mut output = super::super::operation::get_alias::builders::GetAliasOutputBuilder::default();
+        output = super::super::protocol_serde::shape_get_alias::de_get_alias(_response_body, output)
+            .map_err(super::super::operation::get_alias::GetAliasError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -107,9 +107,9 @@

 pub(crate) fn de_get_alias(
     _value: &[u8],
-    mut builder: crate::operation::get_alias::builders::GetAliasOutputBuilder,
-) -> ::std::result::Result<crate::operation::get_alias::builders::GetAliasOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    mut builder: super::super::operation::get_alias::builders::GetAliasOutputBuilder,
+) -> ::std::result::Result<super::super::operation::get_alias::builders::GetAliasOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -156,7 +156,7 @@
                     }
                     "RoutingConfig" => {
                         builder = builder.set_routing_config(
-                            crate::protocol_serde::shape_alias_routing_configuration::de_alias_routing_configuration(tokens, _value, depth + 1)?,
+                            super::super::protocol_serde::shape_alias_routing_configuration::de_alias_routing_configuration(tokens, _value, depth + 1)?,
                         );
                     }
                     _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
