--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::CapacityProvider>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::CapacityProvider>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::CapacityProviderBuilder::default();
+            let mut builder = super::super::types::builders::CapacityProviderBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -31,13 +31,13 @@
                         "State" => {
                             builder = builder.set_state(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::CapacityProviderState::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::CapacityProviderState::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "VpcConfig" => {
                             builder = builder.set_vpc_config(
-                                crate::protocol_serde::shape_capacity_provider_vpc_config::de_capacity_provider_vpc_config(
+                                super::super::protocol_serde::shape_capacity_provider_vpc_config::de_capacity_provider_vpc_config(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -46,7 +46,7 @@
                         }
                         "PermissionsConfig" => {
                             builder = builder.set_permissions_config(
-                                crate::protocol_serde::shape_capacity_provider_permissions_config::de_capacity_provider_permissions_config(
+                                super::super::protocol_serde::shape_capacity_provider_permissions_config::de_capacity_provider_permissions_config(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -55,12 +55,12 @@
                         }
                         "InstanceRequirements" => {
                             builder = builder.set_instance_requirements(
-                                crate::protocol_serde::shape_instance_requirements::de_instance_requirements(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_instance_requirements::de_instance_requirements(tokens, _value, depth + 1)?,
                             );
                         }
                         "CapacityProviderScalingConfig" => {
                             builder = builder.set_capacity_provider_scaling_config(
-                                crate::protocol_serde::shape_capacity_provider_scaling_config::de_capacity_provider_scaling_config(
+                                super::super::protocol_serde::shape_capacity_provider_scaling_config::de_capacity_provider_scaling_config(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -82,7 +82,7 @@
                             );
                         }
                         "PropagateTags" => {
-                            builder = builder.set_propagate_tags(crate::protocol_serde::shape_propagate_tags::de_propagate_tags(
+                            builder = builder.set_propagate_tags(super::super::protocol_serde::shape_propagate_tags::de_propagate_tags(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -90,7 +90,7 @@
                         }
                         "TelemetryConfig" => {
                             builder = builder.set_telemetry_config(
-                                crate::protocol_serde::shape_capacity_provider_telemetry_config::de_capacity_provider_telemetry_config(
+                                super::super::protocol_serde::shape_capacity_provider_telemetry_config::de_capacity_provider_telemetry_config(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -106,7 +106,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::capacity_provider_correct_errors(builder).build().map_err(
+            Ok(Some(super::super::serde_util::capacity_provider_correct_errors(builder).build().map_err(
                 |err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err),
             )?))
         }
