--- original
+++ modified
@@ -5,32 +5,32 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_runtime_management_config::GetRuntimeManagementConfigOutput,
-    crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError,
+    super::operation::get_runtime_management_config::GetRuntimeManagementConfigOutput,
+    super::operation::get_runtime_management_config::GetRuntimeManagementConfigError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled(generic)),
+        None => return Err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "InvalidParameterValueException" => {
-            crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::InvalidParameterValueException({
+            super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
+                    .map_err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -40,13 +40,13 @@
                 tmp
             })
         }
-        "ResourceNotFoundException" => crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -55,13 +55,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::ServiceException({
+        "ServiceException" => super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,16 +70,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled(
+                    super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -92,7 +92,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::generic(generic),
+        _ => super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::generic(generic),
     })
 }

@@ -102,14 +102,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_runtime_management_config::GetRuntimeManagementConfigOutput,
-    crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError,
+    super::operation::get_runtime_management_config::GetRuntimeManagementConfigOutput,
+    super::operation::get_runtime_management_config::GetRuntimeManagementConfigError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_runtime_management_config::builders::GetRuntimeManagementConfigOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_runtime_management_config::de_get_runtime_management_config(_response_body, output)
-            .map_err(crate::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
+        let mut output = super::operation::get_runtime_management_config::builders::GetRuntimeManagementConfigOutputBuilder::default();
+        output = super::protocol_serde::shape_get_runtime_management_config::de_get_runtime_management_config(_response_body, output)
+            .map_err(super::operation::get_runtime_management_config::GetRuntimeManagementConfigError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -117,12 +117,12 @@

 pub(crate) fn de_get_runtime_management_config(
     _value: &[u8],
-    mut builder: crate::operation::get_runtime_management_config::builders::GetRuntimeManagementConfigOutputBuilder,
+    mut builder: super::operation::get_runtime_management_config::builders::GetRuntimeManagementConfigOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_runtime_management_config::builders::GetRuntimeManagementConfigOutputBuilder,
+    super::operation::get_runtime_management_config::builders::GetRuntimeManagementConfigOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -148,7 +148,7 @@
                 "UpdateRuntimeOn" => {
                     builder = builder.set_update_runtime_on(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::UpdateRuntimeOn::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::UpdateRuntimeOn::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
