--- original
+++ modified
@@ -5,29 +5,29 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessOutput,
-    crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError,
+    super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessOutput,
+    super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled(generic)),
+        None => return Err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "CallbackTimeoutException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::CallbackTimeoutException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::CallbackTimeoutException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CallbackTimeoutExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_callback_timeout_exception::de_callback_timeout_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+                    let mut output = super::types::error::builders::CallbackTimeoutExceptionBuilder::default();
+                    output = super::protocol_serde::shape_callback_timeout_exception::de_callback_timeout_exception_json_err(_response_body, output)
+                        .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -38,16 +38,16 @@
             })
         }
         "InvalidParameterValueException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::InvalidParameterValueException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+                    .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -58,15 +58,15 @@
             })
         }
         "KMSAccessDeniedException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsAccessDeniedException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsAccessDeniedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
+                    let mut output = super::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
+                        super::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
                             .map_err(
-                                crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled,
+                                super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled,
                             )?;
                     let output = output.meta(generic);
                     output.build()
@@ -78,13 +78,13 @@
             })
         }
         "KMSDisabledException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsDisabledException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsDisabledException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::KmsDisabledExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+                    let mut output = super::types::error::builders::KmsDisabledExceptionBuilder::default();
+                    output = super::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
+                        .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -95,15 +95,15 @@
             })
         }
         "KMSInvalidStateException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsInvalidStateException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsInvalidStateException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                    let mut output = super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                        super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
                             .map_err(
-                                crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled,
+                                super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled,
                             )?;
                     let output = output.meta(generic);
                     output.build()
@@ -115,13 +115,13 @@
             })
         }
         "KMSNotFoundException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsNotFoundException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::KmsNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::KmsNotFoundExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+                    let mut output = super::types::error::builders::KmsNotFoundExceptionBuilder::default();
+                    output = super::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
+                        .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -132,14 +132,14 @@
             })
         }
         "ResourceNotFoundException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::ResourceNotFoundException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::ResourceNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                    let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+                        super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                            .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -150,13 +150,13 @@
             })
         }
         "ServiceException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::ServiceException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::ServiceException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
+                    let mut output = super::types::error::builders::ServiceExceptionBuilder::default();
+                    output = super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                        .map_err(super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -167,19 +167,19 @@
             })
         }
         "TooManyRequestsException" => {
-            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::TooManyRequestsException({
+            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::TooManyRequestsException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                    let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                        super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
                             .map_err(
-                                crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled,
+                                super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled,
                             )?;
                     output = output.set_retry_after_seconds(
-                        crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                            crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled(
+                        super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                            super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::unhandled(
                                 "Failed to parse retryAfterSeconds from header `Retry-After",
                             )
                         })?,
@@ -193,7 +193,7 @@
                 tmp
             })
         }
-        _ => crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::generic(generic),
+        _ => super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError::generic(generic),
     })
 }

@@ -203,13 +203,13 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessOutput,
-    crate::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError,
+    super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessOutput,
+    super::operation::send_durable_execution_callback_success::SendDurableExecutionCallbackSuccessError,
 > {
     Ok({
         #[allow(unused_mut)]
         let mut output =
-            crate::operation::send_durable_execution_callback_success::builders::SendDurableExecutionCallbackSuccessOutputBuilder::default();
+            super::operation::send_durable_execution_callback_success::builders::SendDurableExecutionCallbackSuccessOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
