--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::SelfManagedKafkaEventSourceConfig>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::SelfManagedKafkaEventSourceConfig>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::SelfManagedKafkaEventSourceConfigBuilder::default();
+            let mut builder = super::types::builders::SelfManagedKafkaEventSourceConfigBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -30,7 +30,7 @@
                         }
                         "SchemaRegistryConfig" => {
                             builder = builder.set_schema_registry_config(
-                                crate::protocol_serde::shape_kafka_schema_registry_config::de_kafka_schema_registry_config(
+                                super::protocol_serde::shape_kafka_schema_registry_config::de_kafka_schema_registry_config(
                                     tokens,
                                     _value,
                                     depth + 1,
@@ -56,7 +56,7 @@

 pub fn ser_self_managed_kafka_event_source_config(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::SelfManagedKafkaEventSourceConfig,
+    input: &super::types::SelfManagedKafkaEventSourceConfig,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     if let Some(var_1) = &input.consumer_group_id {
         object.key("ConsumerGroupId").string(var_1.as_str());
@@ -64,7 +64,7 @@
     if let Some(var_2) = &input.schema_registry_config {
         #[allow(unused_mut)]
         let mut object_3 = object.key("SchemaRegistryConfig").start_object();
-        crate::protocol_serde::shape_kafka_schema_registry_config::ser_kafka_schema_registry_config(&mut object_3, var_2)?;
+        super::protocol_serde::shape_kafka_schema_registry_config::ser_kafka_schema_registry_config(&mut object_3, var_2)?;
         object_3.finish();
     }
     Ok(())
