--- original
+++ modified
@@ -5,31 +5,31 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_durable_execution::GetDurableExecutionOutput,
-    crate::operation::get_durable_execution::GetDurableExecutionError,
+    super::operation::get_durable_execution::GetDurableExecutionOutput,
+    super::operation::get_durable_execution::GetDurableExecutionError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled(generic)),
+        None => return Err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidParameterValueException" => crate::operation::get_durable_execution::GetDurableExecutionError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::operation::get_durable_execution::GetDurableExecutionError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -38,13 +38,13 @@
             }
             tmp
         }),
-        "KMSAccessDeniedException" => crate::operation::get_durable_execution::GetDurableExecutionError::KmsAccessDeniedException({
+        "KMSAccessDeniedException" => super::operation::get_durable_execution::GetDurableExecutionError::KmsAccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -53,13 +53,13 @@
             }
             tmp
         }),
-        "KMSDisabledException" => crate::operation::get_durable_execution::GetDurableExecutionError::KmsDisabledException({
+        "KMSDisabledException" => super::operation::get_durable_execution::GetDurableExecutionError::KmsDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::KmsDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -68,13 +68,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::get_durable_execution::GetDurableExecutionError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::operation::get_durable_execution::GetDurableExecutionError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -83,13 +83,13 @@
             }
             tmp
         }),
-        "KMSNotFoundException" => crate::operation::get_durable_execution::GetDurableExecutionError::KmsNotFoundException({
+        "KMSNotFoundException" => super::operation::get_durable_execution::GetDurableExecutionError::KmsNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::KmsNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -98,13 +98,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::get_durable_execution::GetDurableExecutionError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::get_durable_execution::GetDurableExecutionError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -113,13 +113,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::get_durable_execution::GetDurableExecutionError::ServiceException({
+        "ServiceException" => super::operation::get_durable_execution::GetDurableExecutionError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -128,16 +128,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::get_durable_execution::GetDurableExecutionError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::get_durable_execution::GetDurableExecutionError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::get_durable_execution::GetDurableExecutionError::unhandled(
+                    super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::operation::get_durable_execution::GetDurableExecutionError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -150,7 +150,7 @@
             }
             tmp
         }),
-        _ => crate::operation::get_durable_execution::GetDurableExecutionError::generic(generic),
+        _ => super::operation::get_durable_execution::GetDurableExecutionError::generic(generic),
     })
 }

@@ -160,29 +160,29 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::get_durable_execution::GetDurableExecutionOutput,
-    crate::operation::get_durable_execution::GetDurableExecutionError,
+    super::operation::get_durable_execution::GetDurableExecutionOutput,
+    super::operation::get_durable_execution::GetDurableExecutionError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::get_durable_execution::builders::GetDurableExecutionOutputBuilder::default();
-        output = crate::protocol_serde::shape_get_durable_execution::de_get_durable_execution(_response_body, output)
-            .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
+        let mut output = super::operation::get_durable_execution::builders::GetDurableExecutionOutputBuilder::default();
+        output = super::protocol_serde::shape_get_durable_execution::de_get_durable_execution(_response_body, output)
+            .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
-        crate::serde_util::get_durable_execution_output_output_correct_errors(output)
+        super::serde_util::get_durable_execution_output_output_correct_errors(output)
             .build()
-            .map_err(crate::operation::get_durable_execution::GetDurableExecutionError::unhandled)?
+            .map_err(super::operation::get_durable_execution::GetDurableExecutionError::unhandled)?
     })
 }

 pub(crate) fn de_get_durable_execution(
     _value: &[u8],
-    mut builder: crate::operation::get_durable_execution::builders::GetDurableExecutionOutputBuilder,
+    mut builder: super::operation::get_durable_execution::builders::GetDurableExecutionOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::get_durable_execution::builders::GetDurableExecutionOutputBuilder,
+    super::operation::get_durable_execution::builders::GetDurableExecutionOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -192,7 +192,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "DurableConfig" => {
-                    builder = builder.set_durable_config(crate::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_durable_config(super::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
                 }
                 "DurableExecutionArn" => {
                     builder = builder.set_durable_execution_arn(
@@ -215,7 +215,7 @@
                     )?);
                 }
                 "Error" => {
-                    builder = builder.set_error(crate::protocol_serde::shape_error_object::de_error_object(tokens, _value, depth + 1)?);
+                    builder = builder.set_error(super::protocol_serde::shape_error_object::de_error_object(tokens, _value, depth + 1)?);
                 }
                 "ExecutionDataIncluded" => {
                     builder = builder.set_execution_data_included(::aws_smithy_json::deserialize::token::expect_bool_or_null(tokens.next())?);
@@ -250,12 +250,12 @@
                 "Status" => {
                     builder = builder.set_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::ExecutionStatus::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::ExecutionStatus::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "TraceHeader" => {
-                    builder = builder.set_trace_header(crate::protocol_serde::shape_trace_header::de_trace_header(tokens, _value, depth + 1)?);
+                    builder = builder.set_trace_header(super::protocol_serde::shape_trace_header::de_trace_header(tokens, _value, depth + 1)?);
                 }
                 "Version" => {
                     builder = builder.set_version(
