--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ChainedInvokeStartedDetails>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::ChainedInvokeStartedDetails>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ChainedInvokeStartedDetailsBuilder::default();
+            let mut builder = super::super::types::builders::ChainedInvokeStartedDetailsBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -36,7 +36,7 @@
                             );
                         }
                         "Input" => {
-                            builder = builder.set_input(crate::protocol_serde::shape_event_input::de_event_input(tokens, _value, depth + 1)?);
+                            builder = builder.set_input(super::super::protocol_serde::shape_event_input::de_event_input(tokens, _value, depth + 1)?);
                         }
                         "ExecutedVersion" => {
                             builder = builder.set_executed_version(
@@ -62,7 +62,7 @@
                 }
             }
             Ok(Some(
-                crate::serde_util::chained_invoke_started_details_correct_errors(builder)
+                super::super::serde_util::chained_invoke_started_details_correct_errors(builder)
                     .build()
                     .map_err(|err| ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err))?,
             ))
