--- original
+++ modified
@@ -5,33 +5,33 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_function_configuration::UpdateFunctionConfigurationOutput,
-    crate::operation::update_function_configuration::UpdateFunctionConfigurationError,
+    super::operation::update_function_configuration::UpdateFunctionConfigurationOutput,
+    super::operation::update_function_configuration::UpdateFunctionConfigurationError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled(generic)),
+        None => return Err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "CodeSigningConfigNotFoundException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::CodeSigningConfigNotFoundException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::CodeSigningConfigNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
+                    let mut output = super::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
+                        super::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
                             _response_body,
                             output,
                         )
-                        .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                        .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -42,16 +42,16 @@
             })
         }
         "CodeVerificationFailedException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::CodeVerificationFailedException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::CodeVerificationFailedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CodeVerificationFailedExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_code_verification_failed_exception::de_code_verification_failed_exception_json_err(
+                    let mut output = super::types::error::builders::CodeVerificationFailedExceptionBuilder::default();
+                    output = super::protocol_serde::shape_code_verification_failed_exception::de_code_verification_failed_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                    .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -62,16 +62,16 @@
             })
         }
         "InvalidCodeSignatureException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::InvalidCodeSignatureException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::InvalidCodeSignatureException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidCodeSignatureExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_code_signature_exception::de_invalid_code_signature_exception_json_err(
+                    let mut output = super::types::error::builders::InvalidCodeSignatureExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_code_signature_exception::de_invalid_code_signature_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                    .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -82,16 +82,16 @@
             })
         }
         "InvalidParameterValueException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::InvalidParameterValueException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                    .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -102,14 +102,14 @@
             })
         }
         "PreconditionFailedException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::PreconditionFailedException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::PreconditionFailedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::PreconditionFailedExceptionBuilder::default();
+                    let mut output = super::types::error::builders::PreconditionFailedExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_precondition_failed_exception::de_precondition_failed_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                        super::protocol_serde::shape_precondition_failed_exception::de_precondition_failed_exception_json_err(_response_body, output)
+                            .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -120,14 +120,14 @@
             })
         }
         "ResourceConflictException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::ResourceConflictException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::ResourceConflictException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
+                    let mut output = super::types::error::builders::ResourceConflictExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                        super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                            .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -138,14 +138,14 @@
             })
         }
         "ResourceNotFoundException" => {
-            crate::operation::update_function_configuration::UpdateFunctionConfigurationError::ResourceNotFoundException({
+            super::operation::update_function_configuration::UpdateFunctionConfigurationError::ResourceNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                    let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                        super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                            .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -155,13 +155,13 @@
                 tmp
             })
         }
-        "ServiceException" => crate::operation::update_function_configuration::UpdateFunctionConfigurationError::ServiceException({
+        "ServiceException" => super::operation::update_function_configuration::UpdateFunctionConfigurationError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -170,16 +170,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::update_function_configuration::UpdateFunctionConfigurationError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::update_function_configuration::UpdateFunctionConfigurationError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled(
+                    super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -192,7 +192,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_function_configuration::UpdateFunctionConfigurationError::generic(generic),
+        _ => super::operation::update_function_configuration::UpdateFunctionConfigurationError::generic(generic),
     })
 }

@@ -202,14 +202,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_function_configuration::UpdateFunctionConfigurationOutput,
-    crate::operation::update_function_configuration::UpdateFunctionConfigurationError,
+    super::operation::update_function_configuration::UpdateFunctionConfigurationOutput,
+    super::operation::update_function_configuration::UpdateFunctionConfigurationError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_function_configuration::builders::UpdateFunctionConfigurationOutputBuilder::default();
-        output = crate::protocol_serde::shape_update_function_configuration::de_update_function_configuration(_response_body, output)
-            .map_err(crate::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
+        let mut output = super::operation::update_function_configuration::builders::UpdateFunctionConfigurationOutputBuilder::default();
+        output = super::protocol_serde::shape_update_function_configuration::de_update_function_configuration(_response_body, output)
+            .map_err(super::operation::update_function_configuration::UpdateFunctionConfigurationError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -216,11 +216,11 @@
 }

 pub fn ser_update_function_configuration_input(
-    input: &crate::operation::update_function_configuration::UpdateFunctionConfigurationInput,
+    input: &super::operation::update_function_configuration::UpdateFunctionConfigurationInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_update_function_configuration_input::ser_update_function_configuration_input_input(&mut object, input)?;
+    super::protocol_serde::shape_update_function_configuration_input::ser_update_function_configuration_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -227,12 +227,12 @@

 pub(crate) fn de_update_function_configuration(
     _value: &[u8],
-    mut builder: crate::operation::update_function_configuration::builders::UpdateFunctionConfigurationOutputBuilder,
+    mut builder: super::operation::update_function_configuration::builders::UpdateFunctionConfigurationOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::update_function_configuration::builders::UpdateFunctionConfigurationOutputBuilder,
+    super::operation::update_function_configuration::builders::UpdateFunctionConfigurationOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -242,7 +242,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Architectures" => {
-                    builder = builder.set_architectures(crate::protocol_serde::shape_architectures_list::de_architectures_list(
+                    builder = builder.set_architectures(super::protocol_serde::shape_architectures_list::de_architectures_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -250,7 +250,7 @@
                 }
                 "CapacityProviderConfig" => {
                     builder = builder.set_capacity_provider_config(
-                        crate::protocol_serde::shape_capacity_provider_config::de_capacity_provider_config(tokens, _value, depth + 1)?,
+                        super::protocol_serde::shape_capacity_provider_config::de_capacity_provider_config(tokens, _value, depth + 1)?,
                     );
                 }
                 "CodeSha256" => {
@@ -275,7 +275,7 @@
                     );
                 }
                 "DeadLetterConfig" => {
-                    builder = builder.set_dead_letter_config(crate::protocol_serde::shape_dead_letter_config::de_dead_letter_config(
+                    builder = builder.set_dead_letter_config(super::protocol_serde::shape_dead_letter_config::de_dead_letter_config(
                         tokens,
                         _value,
                         depth + 1,
@@ -289,10 +289,10 @@
                     );
                 }
                 "DurableConfig" => {
-                    builder = builder.set_durable_config(crate::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_durable_config(super::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
                 }
                 "Environment" => {
-                    builder = builder.set_environment(crate::protocol_serde::shape_environment_response::de_environment_response(
+                    builder = builder.set_environment(super::protocol_serde::shape_environment_response::de_environment_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -299,7 +299,7 @@
                     )?);
                 }
                 "EphemeralStorage" => {
-                    builder = builder.set_ephemeral_storage(crate::protocol_serde::shape_ephemeral_storage::de_ephemeral_storage(
+                    builder = builder.set_ephemeral_storage(super::protocol_serde::shape_ephemeral_storage::de_ephemeral_storage(
                         tokens,
                         _value,
                         depth + 1,
@@ -306,7 +306,7 @@
                     )?);
                 }
                 "FileSystemConfigs" => {
-                    builder = builder.set_file_system_configs(crate::protocol_serde::shape_file_system_config_list::de_file_system_config_list(
+                    builder = builder.set_file_system_configs(super::protocol_serde::shape_file_system_config_list::de_file_system_config_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -334,7 +334,7 @@
                     );
                 }
                 "ImageConfigResponse" => {
-                    builder = builder.set_image_config_response(crate::protocol_serde::shape_image_config_response::de_image_config_response(
+                    builder = builder.set_image_config_response(super::protocol_serde::shape_image_config_response::de_image_config_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -357,7 +357,7 @@
                 "LastUpdateStatus" => {
                     builder = builder.set_last_update_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::LastUpdateStatus::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::LastUpdateStatus::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -371,12 +371,12 @@
                 "LastUpdateStatusReasonCode" => {
                     builder = builder.set_last_update_status_reason_code(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::LastUpdateStatusReasonCode::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::LastUpdateStatusReasonCode::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "Layers" => {
-                    builder = builder.set_layers(crate::protocol_serde::shape_layers_reference_list::de_layers_reference_list(
+                    builder = builder.set_layers(super::protocol_serde::shape_layers_reference_list::de_layers_reference_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -383,7 +383,7 @@
                     )?);
                 }
                 "LoggingConfig" => {
-                    builder = builder.set_logging_config(crate::protocol_serde::shape_logging_config::de_logging_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_logging_config(super::protocol_serde::shape_logging_config::de_logging_config(tokens, _value, depth + 1)?);
                 }
                 "MasterArn" => {
                     builder = builder.set_master_arn(
@@ -402,7 +402,7 @@
                 "PackageType" => {
                     builder = builder.set_package_type(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::PackageType::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::PackageType::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -423,12 +423,12 @@
                 "Runtime" => {
                     builder = builder.set_runtime(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::Runtime::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::Runtime::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "RuntimeVersionConfig" => {
-                    builder = builder.set_runtime_version_config(crate::protocol_serde::shape_runtime_version_config::de_runtime_version_config(
+                    builder = builder.set_runtime_version_config(super::protocol_serde::shape_runtime_version_config::de_runtime_version_config(
                         tokens,
                         _value,
                         depth + 1,
@@ -449,7 +449,7 @@
                     );
                 }
                 "SnapStart" => {
-                    builder = builder.set_snap_start(crate::protocol_serde::shape_snap_start_response::de_snap_start_response(
+                    builder = builder.set_snap_start(super::protocol_serde::shape_snap_start_response::de_snap_start_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -458,7 +458,7 @@
                 "State" => {
                     builder = builder.set_state(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::State::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::State::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -472,12 +472,12 @@
                 "StateReasonCode" => {
                     builder = builder.set_state_reason_code(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::StateReasonCode::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::types::StateReasonCode::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "TenancyConfig" => {
-                    builder = builder.set_tenancy_config(crate::protocol_serde::shape_tenancy_config::de_tenancy_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_tenancy_config(super::protocol_serde::shape_tenancy_config::de_tenancy_config(tokens, _value, depth + 1)?);
                 }
                 "Timeout" => {
                     builder = builder.set_timeout(
@@ -487,7 +487,7 @@
                     );
                 }
                 "TracingConfig" => {
-                    builder = builder.set_tracing_config(crate::protocol_serde::shape_tracing_config_response::de_tracing_config_response(
+                    builder = builder.set_tracing_config(super::protocol_serde::shape_tracing_config_response::de_tracing_config_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -501,7 +501,7 @@
                     );
                 }
                 "VpcConfig" => {
-                    builder = builder.set_vpc_config(crate::protocol_serde::shape_vpc_config_response::de_vpc_config_response(
+                    builder = builder.set_vpc_config(super::protocol_serde::shape_vpc_config_response::de_vpc_config_response(
                         tokens,
                         _value,
                         depth + 1,
