--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::invoke_async::InvokeAsyncOutput, crate::operation::invoke_async::InvokeAsyncError> {
+) -> std::result::Result<super::operation::invoke_async::InvokeAsyncOutput, super::operation::invoke_async::InvokeAsyncError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::invoke_async::InvokeAsyncError::unhandled(generic)),
+        None => return Err(super::operation::invoke_async::InvokeAsyncError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "EC2AccessDeniedException" => crate::operation::invoke_async::InvokeAsyncError::Ec2AccessDeniedException({
+        "EC2AccessDeniedException" => super::operation::invoke_async::InvokeAsyncError::Ec2AccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2AccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_access_denied_exception::de_ec2_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::Ec2AccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_ec2_access_denied_exception::de_ec2_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "EC2ThrottledException" => crate::operation::invoke_async::InvokeAsyncError::Ec2ThrottledException({
+        "EC2ThrottledException" => super::operation::invoke_async::InvokeAsyncError::Ec2ThrottledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2ThrottledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_throttled_exception::de_ec2_throttled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::Ec2ThrottledExceptionBuilder::default();
+                output = super::protocol_serde::shape_ec2_throttled_exception::de_ec2_throttled_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "EC2UnexpectedException" => crate::operation::invoke_async::InvokeAsyncError::Ec2UnexpectedException({
+        "EC2UnexpectedException" => super::operation::invoke_async::InvokeAsyncError::Ec2UnexpectedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2UnexpectedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_unexpected_exception::de_ec2_unexpected_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::Ec2UnexpectedExceptionBuilder::default();
+                output = super::protocol_serde::shape_ec2_unexpected_exception::de_ec2_unexpected_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,13 +62,13 @@
             }
             tmp
         }),
-        "EFSIOException" => crate::operation::invoke_async::InvokeAsyncError::EfsioException({
+        "EFSIOException" => super::operation::invoke_async::InvokeAsyncError::EfsioException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsioExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efsio_exception::de_efsio_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::EfsioExceptionBuilder::default();
+                output = super::protocol_serde::shape_efsio_exception::de_efsio_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -77,16 +77,16 @@
             }
             tmp
         }),
-        "EFSMountConnectivityException" => crate::operation::invoke_async::InvokeAsyncError::EfsMountConnectivityException({
+        "EFSMountConnectivityException" => super::operation::invoke_async::InvokeAsyncError::EfsMountConnectivityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountConnectivityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_connectivity_exception::de_efs_mount_connectivity_exception_json_err(
+                let mut output = super::types::error::builders::EfsMountConnectivityExceptionBuilder::default();
+                output = super::protocol_serde::shape_efs_mount_connectivity_exception::de_efs_mount_connectivity_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -95,13 +95,13 @@
             }
             tmp
         }),
-        "EFSMountFailureException" => crate::operation::invoke_async::InvokeAsyncError::EfsMountFailureException({
+        "EFSMountFailureException" => super::operation::invoke_async::InvokeAsyncError::EfsMountFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_failure_exception::de_efs_mount_failure_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::EfsMountFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_efs_mount_failure_exception::de_efs_mount_failure_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -110,13 +110,13 @@
             }
             tmp
         }),
-        "EFSMountTimeoutException" => crate::operation::invoke_async::InvokeAsyncError::EfsMountTimeoutException({
+        "EFSMountTimeoutException" => super::operation::invoke_async::InvokeAsyncError::EfsMountTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_timeout_exception::de_efs_mount_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::EfsMountTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_efs_mount_timeout_exception::de_efs_mount_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -125,13 +125,13 @@
             }
             tmp
         }),
-        "ENILimitReachedException" => crate::operation::invoke_async::InvokeAsyncError::EniLimitReachedException({
+        "ENILimitReachedException" => super::operation::invoke_async::InvokeAsyncError::EniLimitReachedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EniLimitReachedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_eni_limit_reached_exception::de_eni_limit_reached_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::EniLimitReachedExceptionBuilder::default();
+                output = super::protocol_serde::shape_eni_limit_reached_exception::de_eni_limit_reached_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -140,16 +140,16 @@
             }
             tmp
         }),
-        "InvalidRequestContentException" => crate::operation::invoke_async::InvokeAsyncError::InvalidRequestContentException({
+        "InvalidRequestContentException" => super::operation::invoke_async::InvokeAsyncError::InvalidRequestContentException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRequestContentExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_request_content_exception::de_invalid_request_content_exception_json_err(
+                let mut output = super::types::error::builders::InvalidRequestContentExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_request_content_exception::de_invalid_request_content_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -158,13 +158,13 @@
             }
             tmp
         }),
-        "InvalidRuntimeException" => crate::operation::invoke_async::InvokeAsyncError::InvalidRuntimeException({
+        "InvalidRuntimeException" => super::operation::invoke_async::InvokeAsyncError::InvalidRuntimeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRuntimeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_runtime_exception::de_invalid_runtime_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidRuntimeExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_runtime_exception::de_invalid_runtime_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -173,16 +173,16 @@
             }
             tmp
         }),
-        "InvalidSecurityGroupIDException" => crate::operation::invoke_async::InvokeAsyncError::InvalidSecurityGroupIdException({
+        "InvalidSecurityGroupIDException" => super::operation::invoke_async::InvokeAsyncError::InvalidSecurityGroupIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSecurityGroupIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_security_group_id_exception::de_invalid_security_group_id_exception_json_err(
+                let mut output = super::types::error::builders::InvalidSecurityGroupIdExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_security_group_id_exception::de_invalid_security_group_id_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -191,13 +191,13 @@
             }
             tmp
         }),
-        "InvalidSubnetIDException" => crate::operation::invoke_async::InvokeAsyncError::InvalidSubnetIdException({
+        "InvalidSubnetIDException" => super::operation::invoke_async::InvokeAsyncError::InvalidSubnetIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSubnetIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_subnet_id_exception::de_invalid_subnet_id_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidSubnetIdExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_subnet_id_exception::de_invalid_subnet_id_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -206,13 +206,13 @@
             }
             tmp
         }),
-        "KMSAccessDeniedException" => crate::operation::invoke_async::InvokeAsyncError::KmsAccessDeniedException({
+        "KMSAccessDeniedException" => super::operation::invoke_async::InvokeAsyncError::KmsAccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -221,13 +221,13 @@
             }
             tmp
         }),
-        "KMSDisabledException" => crate::operation::invoke_async::InvokeAsyncError::KmsDisabledException({
+        "KMSDisabledException" => super::operation::invoke_async::InvokeAsyncError::KmsDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::KmsDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -236,13 +236,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::invoke_async::InvokeAsyncError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::operation::invoke_async::InvokeAsyncError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -251,13 +251,13 @@
             }
             tmp
         }),
-        "KMSNotFoundException" => crate::operation::invoke_async::InvokeAsyncError::KmsNotFoundException({
+        "KMSNotFoundException" => super::operation::invoke_async::InvokeAsyncError::KmsNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::KmsNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -266,13 +266,13 @@
             }
             tmp
         }),
-        "ModeNotSupportedException" => crate::operation::invoke_async::InvokeAsyncError::ModeNotSupportedException({
+        "ModeNotSupportedException" => super::operation::invoke_async::InvokeAsyncError::ModeNotSupportedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ModeNotSupportedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_mode_not_supported_exception::de_mode_not_supported_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::ModeNotSupportedExceptionBuilder::default();
+                output = super::protocol_serde::shape_mode_not_supported_exception::de_mode_not_supported_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -281,13 +281,13 @@
             }
             tmp
         }),
-        "ResourceConflictException" => crate::operation::invoke_async::InvokeAsyncError::ResourceConflictException({
+        "ResourceConflictException" => super::operation::invoke_async::InvokeAsyncError::ResourceConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceConflictExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -296,13 +296,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::invoke_async::InvokeAsyncError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::invoke_async::InvokeAsyncError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -311,16 +311,16 @@
             }
             tmp
         }),
-        "S3FilesMountConnectivityException" => crate::operation::invoke_async::InvokeAsyncError::S3FilesMountConnectivityException({
+        "S3FilesMountConnectivityException" => super::operation::invoke_async::InvokeAsyncError::S3FilesMountConnectivityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::S3FilesMountConnectivityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_s3_files_mount_connectivity_exception::de_s3_files_mount_connectivity_exception_json_err(
+                let mut output = super::types::error::builders::S3FilesMountConnectivityExceptionBuilder::default();
+                output = super::protocol_serde::shape_s3_files_mount_connectivity_exception::de_s3_files_mount_connectivity_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -329,16 +329,16 @@
             }
             tmp
         }),
-        "S3FilesMountFailureException" => crate::operation::invoke_async::InvokeAsyncError::S3FilesMountFailureException({
+        "S3FilesMountFailureException" => super::operation::invoke_async::InvokeAsyncError::S3FilesMountFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::S3FilesMountFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_s3_files_mount_failure_exception::de_s3_files_mount_failure_exception_json_err(
+                let mut output = super::types::error::builders::S3FilesMountFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_s3_files_mount_failure_exception::de_s3_files_mount_failure_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -347,16 +347,16 @@
             }
             tmp
         }),
-        "S3FilesMountTimeoutException" => crate::operation::invoke_async::InvokeAsyncError::S3FilesMountTimeoutException({
+        "S3FilesMountTimeoutException" => super::operation::invoke_async::InvokeAsyncError::S3FilesMountTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::S3FilesMountTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_s3_files_mount_timeout_exception::de_s3_files_mount_timeout_exception_json_err(
+                let mut output = super::types::error::builders::S3FilesMountTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_s3_files_mount_timeout_exception::de_s3_files_mount_timeout_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -365,13 +365,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::invoke_async::InvokeAsyncError::ServiceException({
+        "ServiceException" => super::operation::invoke_async::InvokeAsyncError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -380,16 +380,16 @@
             }
             tmp
         }),
-        "ServiceQuotaExceededException" => crate::operation::invoke_async::InvokeAsyncError::ServiceQuotaExceededException({
+        "ServiceQuotaExceededException" => super::operation::invoke_async::InvokeAsyncError::ServiceQuotaExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
+                let mut output = super::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -398,13 +398,13 @@
             }
             tmp
         }),
-        "SnapStartException" => crate::operation::invoke_async::InvokeAsyncError::SnapStartException({
+        "SnapStartException" => super::operation::invoke_async::InvokeAsyncError::SnapStartException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartExceptionBuilder::default();
-                output = crate::protocol_serde::shape_snap_start_exception::de_snap_start_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::SnapStartExceptionBuilder::default();
+                output = super::protocol_serde::shape_snap_start_exception::de_snap_start_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -413,14 +413,14 @@
             }
             tmp
         }),
-        "SnapStartNotReadyException" => crate::operation::invoke_async::InvokeAsyncError::SnapStartNotReadyException({
+        "SnapStartNotReadyException" => super::operation::invoke_async::InvokeAsyncError::SnapStartNotReadyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartNotReadyExceptionBuilder::default();
+                let mut output = super::types::error::builders::SnapStartNotReadyExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_snap_start_not_ready_exception::de_snap_start_not_ready_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                    super::protocol_serde::shape_snap_start_not_ready_exception::de_snap_start_not_ready_exception_json_err(_response_body, output)
+                        .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -430,12 +430,12 @@
             tmp
         }),
         "SnapStartRegenerationFailureException" => {
-            crate::operation::invoke_async::InvokeAsyncError::SnapStartRegenerationFailureException({
+            super::operation::invoke_async::InvokeAsyncError::SnapStartRegenerationFailureException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SnapStartRegenerationFailureExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_snap_start_regeneration_failure_exception::de_snap_start_regeneration_failure_exception_json_err(_response_body, output).map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                    let mut output = super::types::error::builders::SnapStartRegenerationFailureExceptionBuilder::default();
+                    output = super::protocol_serde::shape_snap_start_regeneration_failure_exception::de_snap_start_regeneration_failure_exception_json_err(_response_body, output).map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -445,13 +445,13 @@
                 tmp
             })
         }
-        "SnapStartTimeoutException" => crate::operation::invoke_async::InvokeAsyncError::SnapStartTimeoutException({
+        "SnapStartTimeoutException" => super::operation::invoke_async::InvokeAsyncError::SnapStartTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_snap_start_timeout_exception::de_snap_start_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                let mut output = super::types::error::builders::SnapStartTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_snap_start_timeout_exception::de_snap_start_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -461,12 +461,12 @@
             tmp
         }),
         "SubnetIPAddressLimitReachedException" => {
-            crate::operation::invoke_async::InvokeAsyncError::SubnetIpAddressLimitReachedException({
+            super::operation::invoke_async::InvokeAsyncError::SubnetIpAddressLimitReachedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SubnetIpAddressLimitReachedExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_subnet_ip_address_limit_reached_exception::de_subnet_ip_address_limit_reached_exception_json_err(_response_body, output).map_err(crate::operation::invoke_async::InvokeAsyncError::unhandled)?;
+                    let mut output = super::types::error::builders::SubnetIpAddressLimitReachedExceptionBuilder::default();
+                    output = super::protocol_serde::shape_subnet_ip_address_limit_reached_exception::de_subnet_ip_address_limit_reached_exception_json_err(_response_body, output).map_err(super::operation::invoke_async::InvokeAsyncError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -476,7 +476,7 @@
                 tmp
             })
         }
-        _ => crate::operation::invoke_async::InvokeAsyncError::generic(generic),
+        _ => super::operation::invoke_async::InvokeAsyncError::generic(generic),
     })
 }

@@ -485,10 +485,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::invoke_async::InvokeAsyncOutput, crate::operation::invoke_async::InvokeAsyncError> {
+) -> std::result::Result<super::operation::invoke_async::InvokeAsyncOutput, super::operation::invoke_async::InvokeAsyncError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::invoke_async::builders::InvokeAsyncOutputBuilder::default();
+        let mut output = super::operation::invoke_async::builders::InvokeAsyncOutputBuilder::default();
         output = output.set_status(Some(_response_status as _));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
