--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::invoke::InvokeOutput, crate::operation::invoke::InvokeError> {
+) -> std::result::Result<super::operation::invoke::InvokeOutput, super::operation::invoke::InvokeError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::invoke::InvokeError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::invoke::InvokeError::unhandled(generic)),
+        None => return Err(super::operation::invoke::InvokeError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "CodeArtifactUserDeletedException" => crate::operation::invoke::InvokeError::CodeArtifactUserDeletedException({
+        "CodeArtifactUserDeletedException" => super::operation::invoke::InvokeError::CodeArtifactUserDeletedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeArtifactUserDeletedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_code_artifact_user_deleted_exception::de_code_artifact_user_deleted_exception_json_err(
+                let mut output = super::types::error::builders::CodeArtifactUserDeletedExceptionBuilder::default();
+                output = super::protocol_serde::shape_code_artifact_user_deleted_exception::de_code_artifact_user_deleted_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,16 +35,16 @@
             }
             tmp
         }),
-        "CodeArtifactUserFailedException" => crate::operation::invoke::InvokeError::CodeArtifactUserFailedException({
+        "CodeArtifactUserFailedException" => super::operation::invoke::InvokeError::CodeArtifactUserFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeArtifactUserFailedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_code_artifact_user_failed_exception::de_code_artifact_user_failed_exception_json_err(
+                let mut output = super::types::error::builders::CodeArtifactUserFailedExceptionBuilder::default();
+                output = super::protocol_serde::shape_code_artifact_user_failed_exception::de_code_artifact_user_failed_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -53,16 +53,16 @@
             }
             tmp
         }),
-        "CodeArtifactUserPendingException" => crate::operation::invoke::InvokeError::CodeArtifactUserPendingException({
+        "CodeArtifactUserPendingException" => super::operation::invoke::InvokeError::CodeArtifactUserPendingException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeArtifactUserPendingExceptionBuilder::default();
-                output = crate::protocol_serde::shape_code_artifact_user_pending_exception::de_code_artifact_user_pending_exception_json_err(
+                let mut output = super::types::error::builders::CodeArtifactUserPendingExceptionBuilder::default();
+                output = super::protocol_serde::shape_code_artifact_user_pending_exception::de_code_artifact_user_pending_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -71,12 +71,12 @@
             }
             tmp
         }),
-        "DurableExecutionAlreadyStartedException" => crate::operation::invoke::InvokeError::DurableExecutionAlreadyStartedException({
+        "DurableExecutionAlreadyStartedException" => super::operation::invoke::InvokeError::DurableExecutionAlreadyStartedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DurableExecutionAlreadyStartedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_durable_execution_already_started_exception::de_durable_execution_already_started_exception_json_err(_response_body, output).map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::DurableExecutionAlreadyStartedExceptionBuilder::default();
+                output = super::protocol_serde::shape_durable_execution_already_started_exception::de_durable_execution_already_started_exception_json_err(_response_body, output).map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -85,13 +85,13 @@
             }
             tmp
         }),
-        "EC2AccessDeniedException" => crate::operation::invoke::InvokeError::Ec2AccessDeniedException({
+        "EC2AccessDeniedException" => super::operation::invoke::InvokeError::Ec2AccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2AccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_access_denied_exception::de_ec2_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::Ec2AccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_ec2_access_denied_exception::de_ec2_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -100,13 +100,13 @@
             }
             tmp
         }),
-        "EC2ThrottledException" => crate::operation::invoke::InvokeError::Ec2ThrottledException({
+        "EC2ThrottledException" => super::operation::invoke::InvokeError::Ec2ThrottledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2ThrottledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_throttled_exception::de_ec2_throttled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::Ec2ThrottledExceptionBuilder::default();
+                output = super::protocol_serde::shape_ec2_throttled_exception::de_ec2_throttled_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -115,13 +115,13 @@
             }
             tmp
         }),
-        "EC2UnexpectedException" => crate::operation::invoke::InvokeError::Ec2UnexpectedException({
+        "EC2UnexpectedException" => super::operation::invoke::InvokeError::Ec2UnexpectedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2UnexpectedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_unexpected_exception::de_ec2_unexpected_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::Ec2UnexpectedExceptionBuilder::default();
+                output = super::protocol_serde::shape_ec2_unexpected_exception::de_ec2_unexpected_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -130,13 +130,13 @@
             }
             tmp
         }),
-        "EFSIOException" => crate::operation::invoke::InvokeError::EfsioException({
+        "EFSIOException" => super::operation::invoke::InvokeError::EfsioException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsioExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efsio_exception::de_efsio_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::EfsioExceptionBuilder::default();
+                output = super::protocol_serde::shape_efsio_exception::de_efsio_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -145,16 +145,16 @@
             }
             tmp
         }),
-        "EFSMountConnectivityException" => crate::operation::invoke::InvokeError::EfsMountConnectivityException({
+        "EFSMountConnectivityException" => super::operation::invoke::InvokeError::EfsMountConnectivityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountConnectivityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_connectivity_exception::de_efs_mount_connectivity_exception_json_err(
+                let mut output = super::types::error::builders::EfsMountConnectivityExceptionBuilder::default();
+                output = super::protocol_serde::shape_efs_mount_connectivity_exception::de_efs_mount_connectivity_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -163,13 +163,13 @@
             }
             tmp
         }),
-        "EFSMountFailureException" => crate::operation::invoke::InvokeError::EfsMountFailureException({
+        "EFSMountFailureException" => super::operation::invoke::InvokeError::EfsMountFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_failure_exception::de_efs_mount_failure_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::EfsMountFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_efs_mount_failure_exception::de_efs_mount_failure_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -178,13 +178,13 @@
             }
             tmp
         }),
-        "EFSMountTimeoutException" => crate::operation::invoke::InvokeError::EfsMountTimeoutException({
+        "EFSMountTimeoutException" => super::operation::invoke::InvokeError::EfsMountTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_timeout_exception::de_efs_mount_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::EfsMountTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_efs_mount_timeout_exception::de_efs_mount_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -193,13 +193,13 @@
             }
             tmp
         }),
-        "ENILimitReachedException" => crate::operation::invoke::InvokeError::EniLimitReachedException({
+        "ENILimitReachedException" => super::operation::invoke::InvokeError::EniLimitReachedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EniLimitReachedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_eni_limit_reached_exception::de_eni_limit_reached_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::EniLimitReachedExceptionBuilder::default();
+                output = super::protocol_serde::shape_eni_limit_reached_exception::de_eni_limit_reached_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -208,13 +208,13 @@
             }
             tmp
         }),
-        "ENINotReadyException" => crate::operation::invoke::InvokeError::EniNotReadyException({
+        "ENINotReadyException" => super::operation::invoke::InvokeError::EniNotReadyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EniNotReadyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_eni_not_ready_exception::de_eni_not_ready_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::EniNotReadyExceptionBuilder::default();
+                output = super::protocol_serde::shape_eni_not_ready_exception::de_eni_not_ready_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -223,16 +223,16 @@
             }
             tmp
         }),
-        "InvalidParameterValueException" => crate::operation::invoke::InvokeError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::operation::invoke::InvokeError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -241,16 +241,16 @@
             }
             tmp
         }),
-        "InvalidRequestContentException" => crate::operation::invoke::InvokeError::InvalidRequestContentException({
+        "InvalidRequestContentException" => super::operation::invoke::InvokeError::InvalidRequestContentException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRequestContentExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_request_content_exception::de_invalid_request_content_exception_json_err(
+                let mut output = super::types::error::builders::InvalidRequestContentExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_request_content_exception::de_invalid_request_content_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -259,13 +259,13 @@
             }
             tmp
         }),
-        "InvalidRuntimeException" => crate::operation::invoke::InvokeError::InvalidRuntimeException({
+        "InvalidRuntimeException" => super::operation::invoke::InvokeError::InvalidRuntimeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRuntimeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_runtime_exception::de_invalid_runtime_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidRuntimeExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_runtime_exception::de_invalid_runtime_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -274,16 +274,16 @@
             }
             tmp
         }),
-        "InvalidSecurityGroupIDException" => crate::operation::invoke::InvokeError::InvalidSecurityGroupIdException({
+        "InvalidSecurityGroupIDException" => super::operation::invoke::InvokeError::InvalidSecurityGroupIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSecurityGroupIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_security_group_id_exception::de_invalid_security_group_id_exception_json_err(
+                let mut output = super::types::error::builders::InvalidSecurityGroupIdExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_security_group_id_exception::de_invalid_security_group_id_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -292,13 +292,13 @@
             }
             tmp
         }),
-        "InvalidSubnetIDException" => crate::operation::invoke::InvokeError::InvalidSubnetIdException({
+        "InvalidSubnetIDException" => super::operation::invoke::InvokeError::InvalidSubnetIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSubnetIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_subnet_id_exception::de_invalid_subnet_id_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidSubnetIdExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_subnet_id_exception::de_invalid_subnet_id_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -307,13 +307,13 @@
             }
             tmp
         }),
-        "InvalidZipFileException" => crate::operation::invoke::InvokeError::InvalidZipFileException({
+        "InvalidZipFileException" => super::operation::invoke::InvokeError::InvalidZipFileException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidZipFileExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_zip_file_exception::de_invalid_zip_file_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidZipFileExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_zip_file_exception::de_invalid_zip_file_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -322,13 +322,13 @@
             }
             tmp
         }),
-        "KMSAccessDeniedException" => crate::operation::invoke::InvokeError::KmsAccessDeniedException({
+        "KMSAccessDeniedException" => super::operation::invoke::InvokeError::KmsAccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -337,13 +337,13 @@
             }
             tmp
         }),
-        "KMSDisabledException" => crate::operation::invoke::InvokeError::KmsDisabledException({
+        "KMSDisabledException" => super::operation::invoke::InvokeError::KmsDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::KmsDisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -352,13 +352,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::invoke::InvokeError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::operation::invoke::InvokeError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -367,13 +367,13 @@
             }
             tmp
         }),
-        "KMSNotFoundException" => crate::operation::invoke::InvokeError::KmsNotFoundException({
+        "KMSNotFoundException" => super::operation::invoke::InvokeError::KmsNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::KmsNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -382,13 +382,13 @@
             }
             tmp
         }),
-        "ModeNotSupportedException" => crate::operation::invoke::InvokeError::ModeNotSupportedException({
+        "ModeNotSupportedException" => super::operation::invoke::InvokeError::ModeNotSupportedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ModeNotSupportedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_mode_not_supported_exception::de_mode_not_supported_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::ModeNotSupportedExceptionBuilder::default();
+                output = super::protocol_serde::shape_mode_not_supported_exception::de_mode_not_supported_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -397,14 +397,14 @@
             }
             tmp
         }),
-        "NoPublishedVersionException" => crate::operation::invoke::InvokeError::NoPublishedVersionException({
+        "NoPublishedVersionException" => super::operation::invoke::InvokeError::NoPublishedVersionException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoPublishedVersionExceptionBuilder::default();
+                let mut output = super::types::error::builders::NoPublishedVersionExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_no_published_version_exception::de_no_published_version_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                    super::protocol_serde::shape_no_published_version_exception::de_no_published_version_exception_json_err(_response_body, output)
+                        .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -413,14 +413,14 @@
             }
             tmp
         }),
-        "RecursiveInvocationException" => crate::operation::invoke::InvokeError::RecursiveInvocationException({
+        "RecursiveInvocationException" => super::operation::invoke::InvokeError::RecursiveInvocationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RecursiveInvocationExceptionBuilder::default();
+                let mut output = super::types::error::builders::RecursiveInvocationExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_recursive_invocation_exception::de_recursive_invocation_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                    super::protocol_serde::shape_recursive_invocation_exception::de_recursive_invocation_exception_json_err(_response_body, output)
+                        .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -429,13 +429,13 @@
             }
             tmp
         }),
-        "RequestTooLargeException" => crate::operation::invoke::InvokeError::RequestTooLargeException({
+        "RequestTooLargeException" => super::operation::invoke::InvokeError::RequestTooLargeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RequestTooLargeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_request_too_large_exception::de_request_too_large_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::RequestTooLargeExceptionBuilder::default();
+                output = super::protocol_serde::shape_request_too_large_exception::de_request_too_large_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -444,13 +444,13 @@
             }
             tmp
         }),
-        "ResourceConflictException" => crate::operation::invoke::InvokeError::ResourceConflictException({
+        "ResourceConflictException" => super::operation::invoke::InvokeError::ResourceConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceConflictExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -459,13 +459,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::invoke::InvokeError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::operation::invoke::InvokeError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -474,13 +474,13 @@
             }
             tmp
         }),
-        "ResourceNotReadyException" => crate::operation::invoke::InvokeError::ResourceNotReadyException({
+        "ResourceNotReadyException" => super::operation::invoke::InvokeError::ResourceNotReadyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotReadyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_ready_exception::de_resource_not_ready_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::ResourceNotReadyExceptionBuilder::default();
+                output = super::protocol_serde::shape_resource_not_ready_exception::de_resource_not_ready_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -489,16 +489,16 @@
             }
             tmp
         }),
-        "S3FilesMountConnectivityException" => crate::operation::invoke::InvokeError::S3FilesMountConnectivityException({
+        "S3FilesMountConnectivityException" => super::operation::invoke::InvokeError::S3FilesMountConnectivityException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::S3FilesMountConnectivityExceptionBuilder::default();
-                output = crate::protocol_serde::shape_s3_files_mount_connectivity_exception::de_s3_files_mount_connectivity_exception_json_err(
+                let mut output = super::types::error::builders::S3FilesMountConnectivityExceptionBuilder::default();
+                output = super::protocol_serde::shape_s3_files_mount_connectivity_exception::de_s3_files_mount_connectivity_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -507,16 +507,16 @@
             }
             tmp
         }),
-        "S3FilesMountFailureException" => crate::operation::invoke::InvokeError::S3FilesMountFailureException({
+        "S3FilesMountFailureException" => super::operation::invoke::InvokeError::S3FilesMountFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::S3FilesMountFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_s3_files_mount_failure_exception::de_s3_files_mount_failure_exception_json_err(
+                let mut output = super::types::error::builders::S3FilesMountFailureExceptionBuilder::default();
+                output = super::protocol_serde::shape_s3_files_mount_failure_exception::de_s3_files_mount_failure_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -525,16 +525,16 @@
             }
             tmp
         }),
-        "S3FilesMountTimeoutException" => crate::operation::invoke::InvokeError::S3FilesMountTimeoutException({
+        "S3FilesMountTimeoutException" => super::operation::invoke::InvokeError::S3FilesMountTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::S3FilesMountTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_s3_files_mount_timeout_exception::de_s3_files_mount_timeout_exception_json_err(
+                let mut output = super::types::error::builders::S3FilesMountTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_s3_files_mount_timeout_exception::de_s3_files_mount_timeout_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -543,12 +543,12 @@
             }
             tmp
         }),
-        "SerializedRequestEntityTooLargeException" => crate::operation::invoke::InvokeError::SerializedRequestEntityTooLargeException({
+        "SerializedRequestEntityTooLargeException" => super::operation::invoke::InvokeError::SerializedRequestEntityTooLargeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SerializedRequestEntityTooLargeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_serialized_request_entity_too_large_exception::de_serialized_request_entity_too_large_exception_json_err(_response_body, output).map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::SerializedRequestEntityTooLargeExceptionBuilder::default();
+                output = super::protocol_serde::shape_serialized_request_entity_too_large_exception::de_serialized_request_entity_too_large_exception_json_err(_response_body, output).map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -557,13 +557,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::invoke::InvokeError::ServiceException({
+        "ServiceException" => super::operation::invoke::InvokeError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -572,16 +572,16 @@
             }
             tmp
         }),
-        "ServiceQuotaExceededException" => crate::operation::invoke::InvokeError::ServiceQuotaExceededException({
+        "ServiceQuotaExceededException" => super::operation::invoke::InvokeError::ServiceQuotaExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
+                let mut output = super::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
+                output = super::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -590,13 +590,13 @@
             }
             tmp
         }),
-        "SnapStartException" => crate::operation::invoke::InvokeError::SnapStartException({
+        "SnapStartException" => super::operation::invoke::InvokeError::SnapStartException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartExceptionBuilder::default();
-                output = crate::protocol_serde::shape_snap_start_exception::de_snap_start_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::SnapStartExceptionBuilder::default();
+                output = super::protocol_serde::shape_snap_start_exception::de_snap_start_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -605,14 +605,14 @@
             }
             tmp
         }),
-        "SnapStartNotReadyException" => crate::operation::invoke::InvokeError::SnapStartNotReadyException({
+        "SnapStartNotReadyException" => super::operation::invoke::InvokeError::SnapStartNotReadyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartNotReadyExceptionBuilder::default();
+                let mut output = super::types::error::builders::SnapStartNotReadyExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_snap_start_not_ready_exception::de_snap_start_not_ready_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                    super::protocol_serde::shape_snap_start_not_ready_exception::de_snap_start_not_ready_exception_json_err(_response_body, output)
+                        .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -622,12 +622,12 @@
             tmp
         }),
         "SnapStartRegenerationFailureException" => {
-            crate::operation::invoke::InvokeError::SnapStartRegenerationFailureException({
+            super::operation::invoke::InvokeError::SnapStartRegenerationFailureException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SnapStartRegenerationFailureExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_snap_start_regeneration_failure_exception::de_snap_start_regeneration_failure_exception_json_err(_response_body, output).map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                    let mut output = super::types::error::builders::SnapStartRegenerationFailureExceptionBuilder::default();
+                    output = super::protocol_serde::shape_snap_start_regeneration_failure_exception::de_snap_start_regeneration_failure_exception_json_err(_response_body, output).map_err(super::operation::invoke::InvokeError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -637,13 +637,13 @@
                 tmp
             })
         }
-        "SnapStartTimeoutException" => crate::operation::invoke::InvokeError::SnapStartTimeoutException({
+        "SnapStartTimeoutException" => super::operation::invoke::InvokeError::SnapStartTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_snap_start_timeout_exception::de_snap_start_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::SnapStartTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_snap_start_timeout_exception::de_snap_start_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -653,12 +653,12 @@
             tmp
         }),
         "SubnetIPAddressLimitReachedException" => {
-            crate::operation::invoke::InvokeError::SubnetIpAddressLimitReachedException({
+            super::operation::invoke::InvokeError::SubnetIpAddressLimitReachedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SubnetIpAddressLimitReachedExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_subnet_ip_address_limit_reached_exception::de_subnet_ip_address_limit_reached_exception_json_err(_response_body, output).map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                    let mut output = super::types::error::builders::SubnetIpAddressLimitReachedExceptionBuilder::default();
+                    output = super::protocol_serde::shape_subnet_ip_address_limit_reached_exception::de_subnet_ip_address_limit_reached_exception_json_err(_response_body, output).map_err(super::operation::invoke::InvokeError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -668,16 +668,16 @@
                 tmp
             })
         }
-        "TooManyRequestsException" => crate::operation::invoke::InvokeError::TooManyRequestsException({
+        "TooManyRequestsException" => super::operation::invoke::InvokeError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                let mut output = super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::invoke::InvokeError::unhandled("Failed to parse retryAfterSeconds from header `Retry-After")
+                    super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::operation::invoke::InvokeError::unhandled("Failed to parse retryAfterSeconds from header `Retry-After")
                     })?,
                 );
                 let output = output.meta(generic);
@@ -688,16 +688,16 @@
             }
             tmp
         }),
-        "UnsupportedMediaTypeException" => crate::operation::invoke::InvokeError::UnsupportedMediaTypeException({
+        "UnsupportedMediaTypeException" => super::operation::invoke::InvokeError::UnsupportedMediaTypeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnsupportedMediaTypeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_unsupported_media_type_exception::de_unsupported_media_type_exception_json_err(
+                let mut output = super::types::error::builders::UnsupportedMediaTypeExceptionBuilder::default();
+                output = super::protocol_serde::shape_unsupported_media_type_exception::de_unsupported_media_type_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::invoke::InvokeError::unhandled)?;
+                .map_err(super::operation::invoke::InvokeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -706,7 +706,7 @@
             }
             tmp
         }),
-        _ => crate::operation::invoke::InvokeError::generic(generic),
+        _ => super::operation::invoke::InvokeError::generic(generic),
     })
 }

@@ -715,29 +715,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::invoke::InvokeOutput, crate::operation::invoke::InvokeError> {
+) -> std::result::Result<super::operation::invoke::InvokeOutput, super::operation::invoke::InvokeError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::invoke::builders::InvokeOutputBuilder::default();
+        let mut output = super::operation::invoke::builders::InvokeOutputBuilder::default();
         output = output.set_durable_execution_arn(
-            crate::protocol_serde::shape_invoke_output::de_durable_execution_arn_header(_response_headers).map_err(|_| {
-                crate::operation::invoke::InvokeError::unhandled("Failed to parse DurableExecutionArn from header `X-Amz-Durable-Execution-Arn")
+            super::protocol_serde::shape_invoke_output::de_durable_execution_arn_header(_response_headers).map_err(|_| {
+                super::operation::invoke::InvokeError::unhandled("Failed to parse DurableExecutionArn from header `X-Amz-Durable-Execution-Arn")
             })?,
         );
         output = output.set_executed_version(
-            crate::protocol_serde::shape_invoke_output::de_executed_version_header(_response_headers).map_err(|_| {
-                crate::operation::invoke::InvokeError::unhandled("Failed to parse ExecutedVersion from header `X-Amz-Executed-Version")
+            super::protocol_serde::shape_invoke_output::de_executed_version_header(_response_headers).map_err(|_| {
+                super::operation::invoke::InvokeError::unhandled("Failed to parse ExecutedVersion from header `X-Amz-Executed-Version")
             })?,
         );
         output = output.set_function_error(
-            crate::protocol_serde::shape_invoke_output::de_function_error_header(_response_headers)
-                .map_err(|_| crate::operation::invoke::InvokeError::unhandled("Failed to parse FunctionError from header `X-Amz-Function-Error"))?,
+            super::protocol_serde::shape_invoke_output::de_function_error_header(_response_headers)
+                .map_err(|_| super::operation::invoke::InvokeError::unhandled("Failed to parse FunctionError from header `X-Amz-Function-Error"))?,
         );
         output = output.set_log_result(
-            crate::protocol_serde::shape_invoke_output::de_log_result_header(_response_headers)
-                .map_err(|_| crate::operation::invoke::InvokeError::unhandled("Failed to parse LogResult from header `X-Amz-Log-Result"))?,
+            super::protocol_serde::shape_invoke_output::de_log_result_header(_response_headers)
+                .map_err(|_| super::operation::invoke::InvokeError::unhandled("Failed to parse LogResult from header `X-Amz-Log-Result"))?,
         );
-        output = output.set_payload(crate::protocol_serde::shape_invoke_output::de_payload_payload(_response_body)?);
+        output = output.set_payload(super::protocol_serde::shape_invoke_output::de_payload_payload(_response_body)?);
         output = output.set_status_code(Some(_response_status as _));
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
@@ -745,7 +745,7 @@
 }

 pub fn ser_invoke_headers(
-    input: &crate::operation::invoke::InvokeInput,
+    input: &super::operation::invoke::InvokeInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.invocation_type {
