--- original
+++ modified
@@ -5,33 +5,33 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_function_code::UpdateFunctionCodeOutput,
-    crate::operation::update_function_code::UpdateFunctionCodeError,
+    super::super::operation::update_function_code::UpdateFunctionCodeOutput,
+    super::super::operation::update_function_code::UpdateFunctionCodeError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled(generic)),
+        None => return Err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "CodeSigningConfigNotFoundException" => {
-            crate::operation::update_function_code::UpdateFunctionCodeError::CodeSigningConfigNotFoundException({
+            super::super::operation::update_function_code::UpdateFunctionCodeError::CodeSigningConfigNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
+                        super::super::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
                             _response_body,
                             output,
                         )
-                        .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                        .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -41,14 +41,14 @@
                 tmp
             })
         }
-        "CodeStorageExceededException" => crate::operation::update_function_code::UpdateFunctionCodeError::CodeStorageExceededException({
+        "CodeStorageExceededException" => super::super::operation::update_function_code::UpdateFunctionCodeError::CodeStorageExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeStorageExceededExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::CodeStorageExceededExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_code_storage_exceeded_exception::de_code_storage_exceeded_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                    super::super::protocol_serde::shape_code_storage_exceeded_exception::de_code_storage_exceeded_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -57,16 +57,16 @@
             }
             tmp
         }),
-        "CodeVerificationFailedException" => crate::operation::update_function_code::UpdateFunctionCodeError::CodeVerificationFailedException({
+        "CodeVerificationFailedException" => super::super::operation::update_function_code::UpdateFunctionCodeError::CodeVerificationFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeVerificationFailedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_code_verification_failed_exception::de_code_verification_failed_exception_json_err(
+                let mut output = super::super::types::error::builders::CodeVerificationFailedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_code_verification_failed_exception::de_code_verification_failed_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -75,16 +75,16 @@
             }
             tmp
         }),
-        "InvalidCodeSignatureException" => crate::operation::update_function_code::UpdateFunctionCodeError::InvalidCodeSignatureException({
+        "InvalidCodeSignatureException" => super::super::operation::update_function_code::UpdateFunctionCodeError::InvalidCodeSignatureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidCodeSignatureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_code_signature_exception::de_invalid_code_signature_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidCodeSignatureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_code_signature_exception::de_invalid_code_signature_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,16 +93,16 @@
             }
             tmp
         }),
-        "InvalidParameterValueException" => crate::operation::update_function_code::UpdateFunctionCodeError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::super::operation::update_function_code::UpdateFunctionCodeError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -111,14 +111,14 @@
             }
             tmp
         }),
-        "PreconditionFailedException" => crate::operation::update_function_code::UpdateFunctionCodeError::PreconditionFailedException({
+        "PreconditionFailedException" => super::super::operation::update_function_code::UpdateFunctionCodeError::PreconditionFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PreconditionFailedExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::PreconditionFailedExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_precondition_failed_exception::de_precondition_failed_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                    super::super::protocol_serde::shape_precondition_failed_exception::de_precondition_failed_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -127,13 +127,13 @@
             }
             tmp
         }),
-        "ResourceConflictException" => crate::operation::update_function_code::UpdateFunctionCodeError::ResourceConflictException({
+        "ResourceConflictException" => super::super::operation::update_function_code::UpdateFunctionCodeError::ResourceConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceConflictExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -142,13 +142,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::update_function_code::UpdateFunctionCodeError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::update_function_code::UpdateFunctionCodeError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -157,13 +157,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::update_function_code::UpdateFunctionCodeError::ServiceException({
+        "ServiceException" => super::super::operation::update_function_code::UpdateFunctionCodeError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -172,16 +172,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::update_function_code::UpdateFunctionCodeError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::update_function_code::UpdateFunctionCodeError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::update_function_code::UpdateFunctionCodeError::unhandled(
+                    super::super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -194,7 +194,7 @@
             }
             tmp
         }),
-        _ => crate::operation::update_function_code::UpdateFunctionCodeError::generic(generic),
+        _ => super::super::operation::update_function_code::UpdateFunctionCodeError::generic(generic),
     })
 }

@@ -204,14 +204,14 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::update_function_code::UpdateFunctionCodeOutput,
-    crate::operation::update_function_code::UpdateFunctionCodeError,
+    super::super::operation::update_function_code::UpdateFunctionCodeOutput,
+    super::super::operation::update_function_code::UpdateFunctionCodeError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::update_function_code::builders::UpdateFunctionCodeOutputBuilder::default();
-        output = crate::protocol_serde::shape_update_function_code::de_update_function_code(_response_body, output)
-            .map_err(crate::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
+        let mut output = super::super::operation::update_function_code::builders::UpdateFunctionCodeOutputBuilder::default();
+        output = super::super::protocol_serde::shape_update_function_code::de_update_function_code(_response_body, output)
+            .map_err(super::super::operation::update_function_code::UpdateFunctionCodeError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -218,11 +218,11 @@
 }

 pub fn ser_update_function_code_input(
-    input: &crate::operation::update_function_code::UpdateFunctionCodeInput,
+    input: &super::super::operation::update_function_code::UpdateFunctionCodeInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_update_function_code_input::ser_update_function_code_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_update_function_code_input::ser_update_function_code_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -229,12 +229,12 @@

 pub(crate) fn de_update_function_code(
     _value: &[u8],
-    mut builder: crate::operation::update_function_code::builders::UpdateFunctionCodeOutputBuilder,
+    mut builder: super::super::operation::update_function_code::builders::UpdateFunctionCodeOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::update_function_code::builders::UpdateFunctionCodeOutputBuilder,
+    super::super::operation::update_function_code::builders::UpdateFunctionCodeOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -244,7 +244,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Architectures" => {
-                    builder = builder.set_architectures(crate::protocol_serde::shape_architectures_list::de_architectures_list(
+                    builder = builder.set_architectures(super::super::protocol_serde::shape_architectures_list::de_architectures_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -252,7 +252,7 @@
                 }
                 "CapacityProviderConfig" => {
                     builder = builder.set_capacity_provider_config(
-                        crate::protocol_serde::shape_capacity_provider_config::de_capacity_provider_config(tokens, _value, depth + 1)?,
+                        super::super::protocol_serde::shape_capacity_provider_config::de_capacity_provider_config(tokens, _value, depth + 1)?,
                     );
                 }
                 "CodeSha256" => {
@@ -277,7 +277,7 @@
                     );
                 }
                 "DeadLetterConfig" => {
-                    builder = builder.set_dead_letter_config(crate::protocol_serde::shape_dead_letter_config::de_dead_letter_config(
+                    builder = builder.set_dead_letter_config(super::super::protocol_serde::shape_dead_letter_config::de_dead_letter_config(
                         tokens,
                         _value,
                         depth + 1,
@@ -291,10 +291,10 @@
                     );
                 }
                 "DurableConfig" => {
-                    builder = builder.set_durable_config(crate::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_durable_config(super::super::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
                 }
                 "Environment" => {
-                    builder = builder.set_environment(crate::protocol_serde::shape_environment_response::de_environment_response(
+                    builder = builder.set_environment(super::super::protocol_serde::shape_environment_response::de_environment_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -301,7 +301,7 @@
                     )?);
                 }
                 "EphemeralStorage" => {
-                    builder = builder.set_ephemeral_storage(crate::protocol_serde::shape_ephemeral_storage::de_ephemeral_storage(
+                    builder = builder.set_ephemeral_storage(super::super::protocol_serde::shape_ephemeral_storage::de_ephemeral_storage(
                         tokens,
                         _value,
                         depth + 1,
@@ -308,7 +308,7 @@
                     )?);
                 }
                 "FileSystemConfigs" => {
-                    builder = builder.set_file_system_configs(crate::protocol_serde::shape_file_system_config_list::de_file_system_config_list(
+                    builder = builder.set_file_system_configs(super::super::protocol_serde::shape_file_system_config_list::de_file_system_config_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -336,7 +336,7 @@
                     );
                 }
                 "ImageConfigResponse" => {
-                    builder = builder.set_image_config_response(crate::protocol_serde::shape_image_config_response::de_image_config_response(
+                    builder = builder.set_image_config_response(super::super::protocol_serde::shape_image_config_response::de_image_config_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -359,7 +359,7 @@
                 "LastUpdateStatus" => {
                     builder = builder.set_last_update_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::LastUpdateStatus::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::LastUpdateStatus::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -373,12 +373,12 @@
                 "LastUpdateStatusReasonCode" => {
                     builder = builder.set_last_update_status_reason_code(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::LastUpdateStatusReasonCode::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::LastUpdateStatusReasonCode::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "Layers" => {
-                    builder = builder.set_layers(crate::protocol_serde::shape_layers_reference_list::de_layers_reference_list(
+                    builder = builder.set_layers(super::super::protocol_serde::shape_layers_reference_list::de_layers_reference_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -385,7 +385,7 @@
                     )?);
                 }
                 "LoggingConfig" => {
-                    builder = builder.set_logging_config(crate::protocol_serde::shape_logging_config::de_logging_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_logging_config(super::super::protocol_serde::shape_logging_config::de_logging_config(tokens, _value, depth + 1)?);
                 }
                 "MasterArn" => {
                     builder = builder.set_master_arn(
@@ -404,7 +404,7 @@
                 "PackageType" => {
                     builder = builder.set_package_type(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::PackageType::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::PackageType::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -425,12 +425,12 @@
                 "Runtime" => {
                     builder = builder.set_runtime(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::Runtime::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::Runtime::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "RuntimeVersionConfig" => {
-                    builder = builder.set_runtime_version_config(crate::protocol_serde::shape_runtime_version_config::de_runtime_version_config(
+                    builder = builder.set_runtime_version_config(super::super::protocol_serde::shape_runtime_version_config::de_runtime_version_config(
                         tokens,
                         _value,
                         depth + 1,
@@ -451,7 +451,7 @@
                     );
                 }
                 "SnapStart" => {
-                    builder = builder.set_snap_start(crate::protocol_serde::shape_snap_start_response::de_snap_start_response(
+                    builder = builder.set_snap_start(super::super::protocol_serde::shape_snap_start_response::de_snap_start_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -460,7 +460,7 @@
                 "State" => {
                     builder = builder.set_state(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::State::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::State::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -474,12 +474,12 @@
                 "StateReasonCode" => {
                     builder = builder.set_state_reason_code(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::StateReasonCode::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::StateReasonCode::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "TenancyConfig" => {
-                    builder = builder.set_tenancy_config(crate::protocol_serde::shape_tenancy_config::de_tenancy_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_tenancy_config(super::super::protocol_serde::shape_tenancy_config::de_tenancy_config(tokens, _value, depth + 1)?);
                 }
                 "Timeout" => {
                     builder = builder.set_timeout(
@@ -489,7 +489,7 @@
                     );
                 }
                 "TracingConfig" => {
-                    builder = builder.set_tracing_config(crate::protocol_serde::shape_tracing_config_response::de_tracing_config_response(
+                    builder = builder.set_tracing_config(super::super::protocol_serde::shape_tracing_config_response::de_tracing_config_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -503,7 +503,7 @@
                     );
                 }
                 "VpcConfig" => {
-                    builder = builder.set_vpc_config(crate::protocol_serde::shape_vpc_config_response::de_vpc_config_response(
+                    builder = builder.set_vpc_config(super::super::protocol_serde::shape_vpc_config_response::de_vpc_config_response(
                         tokens,
                         _value,
                         depth + 1,
