--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::LoggingConfig>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::LoggingConfig>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::LoggingConfigBuilder::default();
+            let mut builder = super::types::builders::LoggingConfigBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -24,7 +24,7 @@
                         "LogFormat" => {
                             builder = builder.set_log_format(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::LogFormat::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::LogFormat::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -31,7 +31,7 @@
                         "ApplicationLogLevel" => {
                             builder = builder.set_application_log_level(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::ApplicationLogLevel::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::ApplicationLogLevel::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -38,7 +38,7 @@
                         "SystemLogLevel" => {
                             builder = builder.set_system_log_level(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::SystemLogLevel::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::types::SystemLogLevel::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -68,7 +68,7 @@

 pub fn ser_logging_config(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::LoggingConfig,
+    input: &super::types::LoggingConfig,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     if let Some(var_1) = &input.log_format {
         object.key("LogFormat").string(var_1.as_str());
