--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::Operation>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::Operation>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::OperationBuilder::default();
+            let mut builder = super::super::types::builders::OperationBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -45,7 +45,7 @@
                         "Type" => {
                             builder = builder.set_type(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::OperationType::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::OperationType::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
@@ -71,12 +71,12 @@
                         "Status" => {
                             builder = builder.set_status(
                                 ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                                    .map(|s| s.to_unescaped().map(|u| crate::types::OperationStatus::from(u.as_ref())))
+                                    .map(|s| s.to_unescaped().map(|u| super::super::types::OperationStatus::from(u.as_ref())))
                                     .transpose()?,
                             );
                         }
                         "ExecutionDetails" => {
-                            builder = builder.set_execution_details(crate::protocol_serde::shape_execution_details::de_execution_details(
+                            builder = builder.set_execution_details(super::super::protocol_serde::shape_execution_details::de_execution_details(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -83,7 +83,7 @@
                             )?);
                         }
                         "ContextDetails" => {
-                            builder = builder.set_context_details(crate::protocol_serde::shape_context_details::de_context_details(
+                            builder = builder.set_context_details(super::super::protocol_serde::shape_context_details::de_context_details(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -91,14 +91,14 @@
                         }
                         "StepDetails" => {
                             builder =
-                                builder.set_step_details(crate::protocol_serde::shape_step_details::de_step_details(tokens, _value, depth + 1)?);
+                                builder.set_step_details(super::super::protocol_serde::shape_step_details::de_step_details(tokens, _value, depth + 1)?);
                         }
                         "WaitDetails" => {
                             builder =
-                                builder.set_wait_details(crate::protocol_serde::shape_wait_details::de_wait_details(tokens, _value, depth + 1)?);
+                                builder.set_wait_details(super::super::protocol_serde::shape_wait_details::de_wait_details(tokens, _value, depth + 1)?);
                         }
                         "CallbackDetails" => {
-                            builder = builder.set_callback_details(crate::protocol_serde::shape_callback_details::de_callback_details(
+                            builder = builder.set_callback_details(super::super::protocol_serde::shape_callback_details::de_callback_details(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -106,7 +106,7 @@
                         }
                         "ChainedInvokeDetails" => {
                             builder = builder.set_chained_invoke_details(
-                                crate::protocol_serde::shape_chained_invoke_details::de_chained_invoke_details(tokens, _value, depth + 1)?,
+                                super::super::protocol_serde::shape_chained_invoke_details::de_chained_invoke_details(tokens, _value, depth + 1)?,
                             );
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
@@ -118,7 +118,7 @@
                     }
                 }
             }
-            Ok(Some(crate::serde_util::operation_correct_errors(builder).build().map_err(|err| {
+            Ok(Some(super::super::serde_util::operation_correct_errors(builder).build().map_err(|err| {
                 ::aws_smithy_json::deserialize::error::DeserializeError::custom_source("Response was invalid", err)
             })?))
         }
