--- original
+++ modified
@@ -3,8 +3,8 @@
 pub fn de_invoke_with_response_stream_http_response(
     response: &mut ::aws_smithy_runtime_api::http::Response,
 ) -> std::result::Result<
-    crate::operation::invoke_with_response_stream::InvokeWithResponseStreamOutput,
-    crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError,
+    super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamOutput,
+    super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError,
 > {
     let mut _response_body = ::aws_smithy_types::body::SdkBody::taken();
     std::mem::swap(&mut _response_body, response.body_mut());
@@ -14,21 +14,21 @@
     let _response_headers = response.headers();
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::invoke_with_response_stream::builders::InvokeWithResponseStreamOutputBuilder::default();
+        let mut output = super::super::operation::invoke_with_response_stream::builders::InvokeWithResponseStreamOutputBuilder::default();
         output = output.set_event_stream(Some(
-            crate::protocol_serde::shape_invoke_with_response_stream_output::de_event_stream_payload(_response_body)?,
+            super::super::protocol_serde::shape_invoke_with_response_stream_output::de_event_stream_payload(_response_body)?,
         ));
         output = output.set_executed_version(
-            crate::protocol_serde::shape_invoke_with_response_stream_output::de_executed_version_header(_response_headers).map_err(|_| {
-                crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
+            super::super::protocol_serde::shape_invoke_with_response_stream_output::de_executed_version_header(_response_headers).map_err(|_| {
+                super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
                     "Failed to parse ExecutedVersion from header `X-Amz-Executed-Version",
                 )
             })?,
         );
         output = output.set_response_stream_content_type(
-            crate::protocol_serde::shape_invoke_with_response_stream_output::de_response_stream_content_type_header(_response_headers).map_err(
+            super::super::protocol_serde::shape_invoke_with_response_stream_output::de_response_stream_content_type_header(_response_headers).map_err(
                 |_| {
-                    crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
+                    super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
                         "Failed to parse ResponseStreamContentType from header `Content-Type",
                     )
                 },
@@ -38,7 +38,7 @@
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output
             .build()
-            .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?
+            .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?
     })
 }

@@ -48,18 +48,18 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::invoke_with_response_stream::InvokeWithResponseStreamOutput,
-    crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError,
+    super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamOutput,
+    super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
         None => {
-            return Err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
+            return Err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
                 generic,
             ))
         }
@@ -67,13 +67,13 @@

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "EC2AccessDeniedException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::Ec2AccessDeniedException({
+        "EC2AccessDeniedException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::Ec2AccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2AccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_access_denied_exception::de_ec2_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::Ec2AccessDeniedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_ec2_access_denied_exception::de_ec2_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -82,13 +82,13 @@
             }
             tmp
         }),
-        "EC2ThrottledException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::Ec2ThrottledException({
+        "EC2ThrottledException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::Ec2ThrottledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2ThrottledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_throttled_exception::de_ec2_throttled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::Ec2ThrottledExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_ec2_throttled_exception::de_ec2_throttled_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -97,13 +97,13 @@
             }
             tmp
         }),
-        "EC2UnexpectedException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::Ec2UnexpectedException({
+        "EC2UnexpectedException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::Ec2UnexpectedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::Ec2UnexpectedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_ec2_unexpected_exception::de_ec2_unexpected_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::Ec2UnexpectedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_ec2_unexpected_exception::de_ec2_unexpected_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -112,13 +112,13 @@
             }
             tmp
         }),
-        "EFSIOException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsioException({
+        "EFSIOException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsioException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsioExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efsio_exception::de_efsio_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::EfsioExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_efsio_exception::de_efsio_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -128,16 +128,16 @@
             tmp
         }),
         "EFSMountConnectivityException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsMountConnectivityException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsMountConnectivityException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::EfsMountConnectivityExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_efs_mount_connectivity_exception::de_efs_mount_connectivity_exception_json_err(
+                    let mut output = super::super::types::error::builders::EfsMountConnectivityExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_efs_mount_connectivity_exception::de_efs_mount_connectivity_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -147,13 +147,13 @@
                 tmp
             })
         }
-        "EFSMountFailureException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsMountFailureException({
+        "EFSMountFailureException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsMountFailureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountFailureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_failure_exception::de_efs_mount_failure_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::EfsMountFailureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_efs_mount_failure_exception::de_efs_mount_failure_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -162,13 +162,13 @@
             }
             tmp
         }),
-        "EFSMountTimeoutException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsMountTimeoutException({
+        "EFSMountTimeoutException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EfsMountTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EfsMountTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_efs_mount_timeout_exception::de_efs_mount_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::EfsMountTimeoutExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_efs_mount_timeout_exception::de_efs_mount_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -177,13 +177,13 @@
             }
             tmp
         }),
-        "ENILimitReachedException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EniLimitReachedException({
+        "ENILimitReachedException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::EniLimitReachedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::EniLimitReachedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_eni_limit_reached_exception::de_eni_limit_reached_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::EniLimitReachedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_eni_limit_reached_exception::de_eni_limit_reached_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -193,16 +193,16 @@
             tmp
         }),
         "InvalidParameterValueException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidParameterValueException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -213,16 +213,16 @@
             })
         }
         "InvalidRequestContentException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidRequestContentException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidRequestContentException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidRequestContentExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_request_content_exception::de_invalid_request_content_exception_json_err(
+                    let mut output = super::super::types::error::builders::InvalidRequestContentExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_request_content_exception::de_invalid_request_content_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -232,13 +232,13 @@
                 tmp
             })
         }
-        "InvalidRuntimeException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidRuntimeException({
+        "InvalidRuntimeException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidRuntimeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidRuntimeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_runtime_exception::de_invalid_runtime_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidRuntimeExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_runtime_exception::de_invalid_runtime_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -248,16 +248,16 @@
             tmp
         }),
         "InvalidSecurityGroupIDException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidSecurityGroupIdException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidSecurityGroupIdException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidSecurityGroupIdExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_security_group_id_exception::de_invalid_security_group_id_exception_json_err(
+                    let mut output = super::super::types::error::builders::InvalidSecurityGroupIdExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_security_group_id_exception::de_invalid_security_group_id_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -267,13 +267,13 @@
                 tmp
             })
         }
-        "InvalidSubnetIDException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidSubnetIdException({
+        "InvalidSubnetIDException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidSubnetIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidSubnetIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_subnet_id_exception::de_invalid_subnet_id_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidSubnetIdExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_subnet_id_exception::de_invalid_subnet_id_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -282,13 +282,13 @@
             }
             tmp
         }),
-        "InvalidZipFileException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidZipFileException({
+        "InvalidZipFileException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::InvalidZipFileException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidZipFileExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_zip_file_exception::de_invalid_zip_file_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidZipFileExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_zip_file_exception::de_invalid_zip_file_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -297,13 +297,13 @@
             }
             tmp
         }),
-        "KMSAccessDeniedException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsAccessDeniedException({
+        "KMSAccessDeniedException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsAccessDeniedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsAccessDeniedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_access_denied_exception::de_kms_access_denied_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -312,13 +312,13 @@
             }
             tmp
         }),
-        "KMSDisabledException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsDisabledException({
+        "KMSDisabledException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsDisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsDisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsDisabledExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_disabled_exception::de_kms_disabled_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -327,13 +327,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -342,13 +342,13 @@
             }
             tmp
         }),
-        "KMSNotFoundException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsNotFoundException({
+        "KMSNotFoundException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::KmsNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_not_found_exception::de_kms_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -357,14 +357,14 @@
             }
             tmp
         }),
-        "NoPublishedVersionException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::NoPublishedVersionException({
+        "NoPublishedVersionException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::NoPublishedVersionException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NoPublishedVersionExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::NoPublishedVersionExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_no_published_version_exception::de_no_published_version_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    super::super::protocol_serde::shape_no_published_version_exception::de_no_published_version_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -374,16 +374,16 @@
             tmp
         }),
         "RecursiveInvocationException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::RecursiveInvocationException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::RecursiveInvocationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::RecursiveInvocationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_recursive_invocation_exception::de_recursive_invocation_exception_json_err(
+                    let mut output = super::super::types::error::builders::RecursiveInvocationExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_recursive_invocation_exception::de_recursive_invocation_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -393,13 +393,13 @@
                 tmp
             })
         }
-        "RequestTooLargeException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::RequestTooLargeException({
+        "RequestTooLargeException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::RequestTooLargeException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::RequestTooLargeExceptionBuilder::default();
-                output = crate::protocol_serde::shape_request_too_large_exception::de_request_too_large_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::RequestTooLargeExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_request_too_large_exception::de_request_too_large_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -408,13 +408,13 @@
             }
             tmp
         }),
-        "ResourceConflictException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ResourceConflictException({
+        "ResourceConflictException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ResourceConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceConflictExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -423,13 +423,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -438,13 +438,13 @@
             }
             tmp
         }),
-        "ResourceNotReadyException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ResourceNotReadyException({
+        "ResourceNotReadyException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ResourceNotReadyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotReadyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_ready_exception::de_resource_not_ready_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotReadyExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_ready_exception::de_resource_not_ready_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -454,16 +454,16 @@
             tmp
         }),
         "S3FilesMountConnectivityException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::S3FilesMountConnectivityException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::S3FilesMountConnectivityException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::S3FilesMountConnectivityExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_s3_files_mount_connectivity_exception::de_s3_files_mount_connectivity_exception_json_err(
+                    let mut output = super::super::types::error::builders::S3FilesMountConnectivityExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_s3_files_mount_connectivity_exception::de_s3_files_mount_connectivity_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -474,16 +474,16 @@
             })
         }
         "S3FilesMountFailureException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::S3FilesMountFailureException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::S3FilesMountFailureException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::S3FilesMountFailureExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_s3_files_mount_failure_exception::de_s3_files_mount_failure_exception_json_err(
+                    let mut output = super::super::types::error::builders::S3FilesMountFailureExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_s3_files_mount_failure_exception::de_s3_files_mount_failure_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -494,16 +494,16 @@
             })
         }
         "S3FilesMountTimeoutException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::S3FilesMountTimeoutException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::S3FilesMountTimeoutException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::S3FilesMountTimeoutExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_s3_files_mount_timeout_exception::de_s3_files_mount_timeout_exception_json_err(
+                    let mut output = super::super::types::error::builders::S3FilesMountTimeoutExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_s3_files_mount_timeout_exception::de_s3_files_mount_timeout_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -514,12 +514,12 @@
             })
         }
         "SerializedRequestEntityTooLargeException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SerializedRequestEntityTooLargeException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SerializedRequestEntityTooLargeException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SerializedRequestEntityTooLargeExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_serialized_request_entity_too_large_exception::de_serialized_request_entity_too_large_exception_json_err(_response_body, output).map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    let mut output = super::super::types::error::builders::SerializedRequestEntityTooLargeExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_serialized_request_entity_too_large_exception::de_serialized_request_entity_too_large_exception_json_err(_response_body, output).map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -529,13 +529,13 @@
                 tmp
             })
         }
-        "ServiceException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ServiceException({
+        "ServiceException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -545,16 +545,16 @@
             tmp
         }),
         "ServiceQuotaExceededException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ServiceQuotaExceededException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::ServiceQuotaExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
+                    let mut output = super::super::types::error::builders::ServiceQuotaExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_service_quota_exceeded_exception::de_service_quota_exceeded_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -564,13 +564,13 @@
                 tmp
             })
         }
-        "SnapStartException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartException({
+        "SnapStartException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartExceptionBuilder::default();
-                output = crate::protocol_serde::shape_snap_start_exception::de_snap_start_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::SnapStartExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_snap_start_exception::de_snap_start_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -579,14 +579,14 @@
             }
             tmp
         }),
-        "SnapStartNotReadyException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartNotReadyException({
+        "SnapStartNotReadyException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartNotReadyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartNotReadyExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::SnapStartNotReadyExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_snap_start_not_ready_exception::de_snap_start_not_ready_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    super::super::protocol_serde::shape_snap_start_not_ready_exception::de_snap_start_not_ready_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -596,12 +596,12 @@
             tmp
         }),
         "SnapStartRegenerationFailureException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartRegenerationFailureException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartRegenerationFailureException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SnapStartRegenerationFailureExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_snap_start_regeneration_failure_exception::de_snap_start_regeneration_failure_exception_json_err(_response_body, output).map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    let mut output = super::super::types::error::builders::SnapStartRegenerationFailureExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_snap_start_regeneration_failure_exception::de_snap_start_regeneration_failure_exception_json_err(_response_body, output).map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -611,13 +611,13 @@
                 tmp
             })
         }
-        "SnapStartTimeoutException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartTimeoutException({
+        "SnapStartTimeoutException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SnapStartTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::SnapStartTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_snap_start_timeout_exception::de_snap_start_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::SnapStartTimeoutExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_snap_start_timeout_exception::de_snap_start_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -627,12 +627,12 @@
             tmp
         }),
         "SubnetIPAddressLimitReachedException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SubnetIpAddressLimitReachedException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::SubnetIpAddressLimitReachedException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::SubnetIpAddressLimitReachedExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_subnet_ip_address_limit_reached_exception::de_subnet_ip_address_limit_reached_exception_json_err(_response_body, output).map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    let mut output = super::super::types::error::builders::SubnetIpAddressLimitReachedExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_subnet_ip_address_limit_reached_exception::de_subnet_ip_address_limit_reached_exception_json_err(_response_body, output).map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -642,16 +642,16 @@
                 tmp
             })
         }
-        "TooManyRequestsException" => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
+                    super::super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -665,16 +665,16 @@
             tmp
         }),
         "UnsupportedMediaTypeException" => {
-            crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::UnsupportedMediaTypeException({
+            super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::UnsupportedMediaTypeException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::UnsupportedMediaTypeExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_unsupported_media_type_exception::de_unsupported_media_type_exception_json_err(
+                    let mut output = super::super::types::error::builders::UnsupportedMediaTypeExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_unsupported_media_type_exception::de_unsupported_media_type_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
+                    .map_err(super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -684,12 +684,12 @@
                 tmp
             })
         }
-        _ => crate::operation::invoke_with_response_stream::InvokeWithResponseStreamError::generic(generic),
+        _ => super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamError::generic(generic),
     })
 }

 pub fn ser_invoke_with_response_stream_headers(
-    input: &crate::operation::invoke_with_response_stream::InvokeWithResponseStreamInput,
+    input: &super::super::operation::invoke_with_response_stream::InvokeWithResponseStreamInput,
     mut builder: ::http_1x::request::Builder,
 ) -> std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
     if let ::std::option::Option::Some(inner_1) = &input.log_type {
