--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::remove_permission::RemovePermissionOutput, crate::operation::remove_permission::RemovePermissionError> {
+) -> std::result::Result<super::super::operation::remove_permission::RemovePermissionOutput, super::super::operation::remove_permission::RemovePermissionError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::remove_permission::RemovePermissionError::unhandled(generic)),
+        None => return Err(super::super::operation::remove_permission::RemovePermissionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "InvalidParameterValueException" => crate::operation::remove_permission::RemovePermissionError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::super::operation::remove_permission::RemovePermissionError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+                .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,14 +35,14 @@
             }
             tmp
         }),
-        "PreconditionFailedException" => crate::operation::remove_permission::RemovePermissionError::PreconditionFailedException({
+        "PreconditionFailedException" => super::super::operation::remove_permission::RemovePermissionError::PreconditionFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PreconditionFailedExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::PreconditionFailedExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_precondition_failed_exception::de_precondition_failed_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+                    super::super::protocol_serde::shape_precondition_failed_exception::de_precondition_failed_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -51,13 +51,13 @@
             }
             tmp
         }),
-        "PublicPolicyException" => crate::operation::remove_permission::RemovePermissionError::PublicPolicyException({
+        "PublicPolicyException" => super::super::operation::remove_permission::RemovePermissionError::PublicPolicyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::PublicPolicyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_public_policy_exception::de_public_policy_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+                let mut output = super::super::types::error::builders::PublicPolicyExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_public_policy_exception::de_public_policy_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -66,13 +66,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::remove_permission::RemovePermissionError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::remove_permission::RemovePermissionError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -81,13 +81,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::remove_permission::RemovePermissionError::ServiceException({
+        "ServiceException" => super::super::operation::remove_permission::RemovePermissionError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -96,16 +96,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::remove_permission::RemovePermissionError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::remove_permission::RemovePermissionError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::remove_permission::RemovePermissionError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::remove_permission::RemovePermissionError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::remove_permission::RemovePermissionError::unhandled(
+                    super::super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::super::operation::remove_permission::RemovePermissionError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -118,7 +118,7 @@
             }
             tmp
         }),
-        _ => crate::operation::remove_permission::RemovePermissionError::generic(generic),
+        _ => super::super::operation::remove_permission::RemovePermissionError::generic(generic),
     })
 }

@@ -127,10 +127,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::remove_permission::RemovePermissionOutput, crate::operation::remove_permission::RemovePermissionError> {
+) -> std::result::Result<super::super::operation::remove_permission::RemovePermissionOutput, super::super::operation::remove_permission::RemovePermissionError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::remove_permission::builders::RemovePermissionOutputBuilder::default();
+        let mut output = super::super::operation::remove_permission::builders::RemovePermissionOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
