--- original
+++ modified
@@ -5,33 +5,33 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigOutput,
-    crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError,
+    super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigOutput,
+    super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError,
 > {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled(generic)),
+        None => return Err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "CodeSigningConfigNotFoundException" => {
-            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::CodeSigningConfigNotFoundException({
+            super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::CodeSigningConfigNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
+                        super::super::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
                             _response_body,
                             output,
                         )
-                        .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+                        .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -42,16 +42,16 @@
             })
         }
         "InvalidParameterValueException" => {
-            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::InvalidParameterValueException({
+            super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::InvalidParameterValueException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                    let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                         _response_body,
                         output,
                     )
-                    .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+                    .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -62,14 +62,14 @@
             })
         }
         "ResourceConflictException" => {
-            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::ResourceConflictException({
+            super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::ResourceConflictException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::ResourceConflictExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+                        super::super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                            .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -80,14 +80,14 @@
             })
         }
         "ResourceNotFoundException" => {
-            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::ResourceNotFoundException({
+            super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::ResourceNotFoundException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+                        super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                            .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -97,13 +97,13 @@
                 tmp
             })
         }
-        "ServiceException" => crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::ServiceException({
+        "ServiceException" => super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -113,17 +113,17 @@
             tmp
         }),
         "TooManyRequestsException" => {
-            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::TooManyRequestsException({
+            super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::TooManyRequestsException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                    let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
                     output =
-                        crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                            .map_err(crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
+                        super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                            .map_err(super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled)?;
                     output = output.set_retry_after_seconds(
-                        crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                            crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled(
+                        super::super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                            super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::unhandled(
                                 "Failed to parse retryAfterSeconds from header `Retry-After",
                             )
                         })?,
@@ -137,7 +137,7 @@
                 tmp
             })
         }
-        _ => crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::generic(generic),
+        _ => super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError::generic(generic),
     })
 }

@@ -147,12 +147,12 @@
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
 ) -> std::result::Result<
-    crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigOutput,
-    crate::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError,
+    super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigOutput,
+    super::super::operation::delete_function_code_signing_config::DeleteFunctionCodeSigningConfigError,
 > {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::delete_function_code_signing_config::builders::DeleteFunctionCodeSigningConfigOutputBuilder::default();
+        let mut output = super::super::operation::delete_function_code_signing_config::builders::DeleteFunctionCodeSigningConfigOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
