--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ImageConfigResponse>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::super::types::ImageConfigResponse>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ImageConfigResponseBuilder::default();
+            let mut builder = super::super::types::builders::ImageConfigResponseBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -23,10 +23,10 @@
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                         "ImageConfig" => {
                             builder =
-                                builder.set_image_config(crate::protocol_serde::shape_image_config::de_image_config(tokens, _value, depth + 1)?);
+                                builder.set_image_config(super::super::protocol_serde::shape_image_config::de_image_config(tokens, _value, depth + 1)?);
                         }
                         "Error" => {
-                            builder = builder.set_error(crate::protocol_serde::shape_image_config_error::de_image_config_error(
+                            builder = builder.set_error(super::super::protocol_serde::shape_image_config_error::de_image_config_error(
                                 tokens,
                                 _value,
                                 depth + 1,
