--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::FilterCriteria>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::FilterCriteria>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,13 +16,13 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::FilterCriteriaBuilder::default();
+            let mut builder = super::types::builders::FilterCriteriaBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
                     Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                         "Filters" => {
-                            builder = builder.set_filters(crate::protocol_serde::shape_filter_list::de_filter_list(tokens, _value, depth + 1)?);
+                            builder = builder.set_filters(super::protocol_serde::shape_filter_list::de_filter_list(tokens, _value, depth + 1)?);
                         }
                         _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
                     },
@@ -43,7 +43,7 @@

 pub fn ser_filter_criteria(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::FilterCriteria,
+    input: &super::types::FilterCriteria,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     if let Some(var_1) = &input.filters {
         let mut array_2 = object.key("Filters").start_array();
@@ -51,7 +51,7 @@
             {
                 #[allow(unused_mut)]
                 let mut object_4 = array_2.value().start_object();
-                crate::protocol_serde::shape_filter::ser_filter(&mut object_4, item_3)?;
+                super::protocol_serde::shape_filter::ser_filter(&mut object_4, item_3)?;
                 object_4.finish();
             }
         }
