--- original
+++ modified
@@ -4,29 +4,29 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_function::CreateFunctionOutput, crate::operation::create_function::CreateFunctionError> {
+) -> std::result::Result<super::super::operation::create_function::CreateFunctionOutput, super::super::operation::create_function::CreateFunctionError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::create_function::CreateFunctionError::unhandled(generic)),
+        None => return Err(super::super::operation::create_function::CreateFunctionError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "CodeSigningConfigNotFoundException" => crate::operation::create_function::CreateFunctionError::CodeSigningConfigNotFoundException({
+        "CodeSigningConfigNotFoundException" => super::super::operation::create_function::CreateFunctionError::CodeSigningConfigNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
+                let mut output = super::super::types::error::builders::CodeSigningConfigNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_code_signing_config_not_found_exception::de_code_signing_config_not_found_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -35,14 +35,14 @@
             }
             tmp
         }),
-        "CodeStorageExceededException" => crate::operation::create_function::CreateFunctionError::CodeStorageExceededException({
+        "CodeStorageExceededException" => super::super::operation::create_function::CreateFunctionError::CodeStorageExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeStorageExceededExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::CodeStorageExceededExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_code_storage_exceeded_exception::de_code_storage_exceeded_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                    super::super::protocol_serde::shape_code_storage_exceeded_exception::de_code_storage_exceeded_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -51,16 +51,16 @@
             }
             tmp
         }),
-        "CodeVerificationFailedException" => crate::operation::create_function::CreateFunctionError::CodeVerificationFailedException({
+        "CodeVerificationFailedException" => super::super::operation::create_function::CreateFunctionError::CodeVerificationFailedException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CodeVerificationFailedExceptionBuilder::default();
-                output = crate::protocol_serde::shape_code_verification_failed_exception::de_code_verification_failed_exception_json_err(
+                let mut output = super::super::types::error::builders::CodeVerificationFailedExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_code_verification_failed_exception::de_code_verification_failed_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -70,12 +70,12 @@
             tmp
         }),
         "FunctionVersionsPerCapacityProviderLimitExceededException" => {
-            crate::operation::create_function::CreateFunctionError::FunctionVersionsPerCapacityProviderLimitExceededException({
+            super::super::operation::create_function::CreateFunctionError::FunctionVersionsPerCapacityProviderLimitExceededException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::FunctionVersionsPerCapacityProviderLimitExceededExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_function_versions_per_capacity_provider_limit_exceeded_exception::de_function_versions_per_capacity_provider_limit_exceeded_exception_json_err(_response_body, output).map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                    let mut output = super::super::types::error::builders::FunctionVersionsPerCapacityProviderLimitExceededExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_function_versions_per_capacity_provider_limit_exceeded_exception::de_function_versions_per_capacity_provider_limit_exceeded_exception_json_err(_response_body, output).map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -85,16 +85,16 @@
                 tmp
             })
         }
-        "InvalidCodeSignatureException" => crate::operation::create_function::CreateFunctionError::InvalidCodeSignatureException({
+        "InvalidCodeSignatureException" => super::super::operation::create_function::CreateFunctionError::InvalidCodeSignatureException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidCodeSignatureExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_code_signature_exception::de_invalid_code_signature_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidCodeSignatureExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_code_signature_exception::de_invalid_code_signature_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -103,16 +103,16 @@
             }
             tmp
         }),
-        "InvalidParameterValueException" => crate::operation::create_function::CreateFunctionError::InvalidParameterValueException({
+        "InvalidParameterValueException" => super::super::operation::create_function::CreateFunctionError::InvalidParameterValueException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidParameterValueExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
+                let mut output = super::super::types::error::builders::InvalidParameterValueExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_parameter_value_exception::de_invalid_parameter_value_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -121,13 +121,13 @@
             }
             tmp
         }),
-        "ResourceConflictException" => crate::operation::create_function::CreateFunctionError::ResourceConflictException({
+        "ResourceConflictException" => super::super::operation::create_function::CreateFunctionError::ResourceConflictException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceConflictExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceConflictExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_conflict_exception::de_resource_conflict_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -136,13 +136,13 @@
             }
             tmp
         }),
-        "ResourceNotFoundException" => crate::operation::create_function::CreateFunctionError::ResourceNotFoundException({
+        "ResourceNotFoundException" => super::super::operation::create_function::CreateFunctionError::ResourceNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ResourceNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ResourceNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_resource_not_found_exception::de_resource_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -151,13 +151,13 @@
             }
             tmp
         }),
-        "ServiceException" => crate::operation::create_function::CreateFunctionError::ServiceException({
+        "ServiceException" => super::super::operation::create_function::CreateFunctionError::ServiceException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::ServiceExceptionBuilder::default();
-                output = crate::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                let mut output = super::super::types::error::builders::ServiceExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_service_exception::de_service_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -166,16 +166,16 @@
             }
             tmp
         }),
-        "TooManyRequestsException" => crate::operation::create_function::CreateFunctionError::TooManyRequestsException({
+        "TooManyRequestsException" => super::super::operation::create_function::CreateFunctionError::TooManyRequestsException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TooManyRequestsExceptionBuilder::default();
-                output = crate::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+                let mut output = super::super::types::error::builders::TooManyRequestsExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_too_many_requests_exception::de_too_many_requests_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
                 output = output.set_retry_after_seconds(
-                    crate::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
-                        crate::operation::create_function::CreateFunctionError::unhandled(
+                    super::super::protocol_serde::shape_too_many_requests_exception::de_retry_after_seconds_header(_response_headers).map_err(|_| {
+                        super::super::operation::create_function::CreateFunctionError::unhandled(
                             "Failed to parse retryAfterSeconds from header `Retry-After",
                         )
                     })?,
@@ -188,7 +188,7 @@
             }
             tmp
         }),
-        _ => crate::operation::create_function::CreateFunctionError::generic(generic),
+        _ => super::super::operation::create_function::CreateFunctionError::generic(generic),
     })
 }

@@ -197,12 +197,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_function::CreateFunctionOutput, crate::operation::create_function::CreateFunctionError> {
+) -> std::result::Result<super::super::operation::create_function::CreateFunctionOutput, super::super::operation::create_function::CreateFunctionError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::create_function::builders::CreateFunctionOutputBuilder::default();
-        output = crate::protocol_serde::shape_create_function::de_create_function(_response_body, output)
-            .map_err(crate::operation::create_function::CreateFunctionError::unhandled)?;
+        let mut output = super::super::operation::create_function::builders::CreateFunctionOutputBuilder::default();
+        output = super::super::protocol_serde::shape_create_function::de_create_function(_response_body, output)
+            .map_err(super::super::operation::create_function::CreateFunctionError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -209,11 +209,11 @@
 }

 pub fn ser_create_function_input(
-    input: &crate::operation::create_function::CreateFunctionInput,
+    input: &super::super::operation::create_function::CreateFunctionInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_create_function_input::ser_create_function_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_create_function_input::ser_create_function_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -220,12 +220,12 @@

 pub(crate) fn de_create_function(
     _value: &[u8],
-    mut builder: crate::operation::create_function::builders::CreateFunctionOutputBuilder,
+    mut builder: super::super::operation::create_function::builders::CreateFunctionOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::create_function::builders::CreateFunctionOutputBuilder,
+    super::super::operation::create_function::builders::CreateFunctionOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -235,7 +235,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "Architectures" => {
-                    builder = builder.set_architectures(crate::protocol_serde::shape_architectures_list::de_architectures_list(
+                    builder = builder.set_architectures(super::super::protocol_serde::shape_architectures_list::de_architectures_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -243,7 +243,7 @@
                 }
                 "CapacityProviderConfig" => {
                     builder = builder.set_capacity_provider_config(
-                        crate::protocol_serde::shape_capacity_provider_config::de_capacity_provider_config(tokens, _value, depth + 1)?,
+                        super::super::protocol_serde::shape_capacity_provider_config::de_capacity_provider_config(tokens, _value, depth + 1)?,
                     );
                 }
                 "CodeSha256" => {
@@ -268,7 +268,7 @@
                     );
                 }
                 "DeadLetterConfig" => {
-                    builder = builder.set_dead_letter_config(crate::protocol_serde::shape_dead_letter_config::de_dead_letter_config(
+                    builder = builder.set_dead_letter_config(super::super::protocol_serde::shape_dead_letter_config::de_dead_letter_config(
                         tokens,
                         _value,
                         depth + 1,
@@ -282,10 +282,10 @@
                     );
                 }
                 "DurableConfig" => {
-                    builder = builder.set_durable_config(crate::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_durable_config(super::super::protocol_serde::shape_durable_config::de_durable_config(tokens, _value, depth + 1)?);
                 }
                 "Environment" => {
-                    builder = builder.set_environment(crate::protocol_serde::shape_environment_response::de_environment_response(
+                    builder = builder.set_environment(super::super::protocol_serde::shape_environment_response::de_environment_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -292,7 +292,7 @@
                     )?);
                 }
                 "EphemeralStorage" => {
-                    builder = builder.set_ephemeral_storage(crate::protocol_serde::shape_ephemeral_storage::de_ephemeral_storage(
+                    builder = builder.set_ephemeral_storage(super::super::protocol_serde::shape_ephemeral_storage::de_ephemeral_storage(
                         tokens,
                         _value,
                         depth + 1,
@@ -299,7 +299,7 @@
                     )?);
                 }
                 "FileSystemConfigs" => {
-                    builder = builder.set_file_system_configs(crate::protocol_serde::shape_file_system_config_list::de_file_system_config_list(
+                    builder = builder.set_file_system_configs(super::super::protocol_serde::shape_file_system_config_list::de_file_system_config_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -327,7 +327,7 @@
                     );
                 }
                 "ImageConfigResponse" => {
-                    builder = builder.set_image_config_response(crate::protocol_serde::shape_image_config_response::de_image_config_response(
+                    builder = builder.set_image_config_response(super::super::protocol_serde::shape_image_config_response::de_image_config_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -350,7 +350,7 @@
                 "LastUpdateStatus" => {
                     builder = builder.set_last_update_status(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::LastUpdateStatus::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::LastUpdateStatus::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -364,12 +364,12 @@
                 "LastUpdateStatusReasonCode" => {
                     builder = builder.set_last_update_status_reason_code(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::LastUpdateStatusReasonCode::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::LastUpdateStatusReasonCode::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "Layers" => {
-                    builder = builder.set_layers(crate::protocol_serde::shape_layers_reference_list::de_layers_reference_list(
+                    builder = builder.set_layers(super::super::protocol_serde::shape_layers_reference_list::de_layers_reference_list(
                         tokens,
                         _value,
                         depth + 1,
@@ -376,7 +376,7 @@
                     )?);
                 }
                 "LoggingConfig" => {
-                    builder = builder.set_logging_config(crate::protocol_serde::shape_logging_config::de_logging_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_logging_config(super::super::protocol_serde::shape_logging_config::de_logging_config(tokens, _value, depth + 1)?);
                 }
                 "MasterArn" => {
                     builder = builder.set_master_arn(
@@ -395,7 +395,7 @@
                 "PackageType" => {
                     builder = builder.set_package_type(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::PackageType::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::PackageType::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -416,12 +416,12 @@
                 "Runtime" => {
                     builder = builder.set_runtime(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::Runtime::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::Runtime::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "RuntimeVersionConfig" => {
-                    builder = builder.set_runtime_version_config(crate::protocol_serde::shape_runtime_version_config::de_runtime_version_config(
+                    builder = builder.set_runtime_version_config(super::super::protocol_serde::shape_runtime_version_config::de_runtime_version_config(
                         tokens,
                         _value,
                         depth + 1,
@@ -442,7 +442,7 @@
                     );
                 }
                 "SnapStart" => {
-                    builder = builder.set_snap_start(crate::protocol_serde::shape_snap_start_response::de_snap_start_response(
+                    builder = builder.set_snap_start(super::super::protocol_serde::shape_snap_start_response::de_snap_start_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -451,7 +451,7 @@
                 "State" => {
                     builder = builder.set_state(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::State::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::State::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
@@ -465,12 +465,12 @@
                 "StateReasonCode" => {
                     builder = builder.set_state_reason_code(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::StateReasonCode::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::StateReasonCode::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
                 "TenancyConfig" => {
-                    builder = builder.set_tenancy_config(crate::protocol_serde::shape_tenancy_config::de_tenancy_config(tokens, _value, depth + 1)?);
+                    builder = builder.set_tenancy_config(super::super::protocol_serde::shape_tenancy_config::de_tenancy_config(tokens, _value, depth + 1)?);
                 }
                 "Timeout" => {
                     builder = builder.set_timeout(
@@ -480,7 +480,7 @@
                     );
                 }
                 "TracingConfig" => {
-                    builder = builder.set_tracing_config(crate::protocol_serde::shape_tracing_config_response::de_tracing_config_response(
+                    builder = builder.set_tracing_config(super::super::protocol_serde::shape_tracing_config_response::de_tracing_config_response(
                         tokens,
                         _value,
                         depth + 1,
@@ -494,7 +494,7 @@
                     );
                 }
                 "VpcConfig" => {
-                    builder = builder.set_vpc_config(crate::protocol_serde::shape_vpc_config_response::de_vpc_config_response(
+                    builder = builder.set_vpc_config(super::super::protocol_serde::shape_vpc_config_response::de_vpc_config_response(
                         tokens,
                         _value,
                         depth + 1,
