--- original
+++ modified
@@ -3,7 +3,7 @@
     tokens: &mut ::std::iter::Peekable<I>,
     _value: &'a [u8],
     depth: u32,
-) -> ::std::result::Result<Option<crate::types::ErrorObject>, ::aws_smithy_json::deserialize::error::DeserializeError>
+) -> ::std::result::Result<Option<super::types::ErrorObject>, ::aws_smithy_json::deserialize::error::DeserializeError>
 where
     I: Iterator<Item = Result<::aws_smithy_json::deserialize::Token<'a>, ::aws_smithy_json::deserialize::error::DeserializeError>>,
 {
@@ -16,7 +16,7 @@
         Some(::aws_smithy_json::deserialize::Token::ValueNull { .. }) => Ok(None),
         Some(::aws_smithy_json::deserialize::Token::StartObject { .. }) => {
             #[allow(unused_mut)]
-            let mut builder = crate::types::builders::ErrorObjectBuilder::default();
+            let mut builder = super::types::builders::ErrorObjectBuilder::default();
             loop {
                 match tokens.next().transpose()? {
                     Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
@@ -43,7 +43,7 @@
                             );
                         }
                         "StackTrace" => {
-                            builder = builder.set_stack_trace(crate::protocol_serde::shape_stack_trace_entries::de_stack_trace_entries(
+                            builder = builder.set_stack_trace(super::protocol_serde::shape_stack_trace_entries::de_stack_trace_entries(
                                 tokens,
                                 _value,
                                 depth + 1,
@@ -68,7 +68,7 @@

 pub fn ser_error_object(
     object: &mut ::aws_smithy_json::serialize::JsonObjectWriter,
-    input: &crate::types::ErrorObject,
+    input: &super::types::ErrorObject,
 ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::SerializationError> {
     if let Some(var_1) = &input.error_message {
         object.key("ErrorMessage").string(var_1.as_str());
