--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetExportJobInput {
     /// Creates a new builder-style object to manufacture [`GetExportJobInput`](crate::operation::get_export_job::GetExportJobInput).
-    pub fn builder() -> crate::operation::get_export_job::builders::GetExportJobInputBuilder {
-        crate::operation::get_export_job::builders::GetExportJobInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_export_job::builders::GetExportJobInputBuilder {
+        super::super::super::operation::get_export_job::builders::GetExportJobInputBuilder::default()
     }
 }

@@ -45,7 +45,7 @@
     /// Consumes the builder and constructs a [`GetExportJobInput`](crate::operation::get_export_job::GetExportJobInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_export_job::GetExportJobInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_export_job::GetExportJobInput { job_id: self.job_id })
+    ) -> ::std::result::Result<super::super::super::operation::get_export_job::GetExportJobInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_export_job::GetExportJobInput { job_id: self.job_id })
     }
 }
