--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl UnsubscribeInput {
     /// Creates a new builder-style object to manufacture [`UnsubscribeInput`](crate::operation::unsubscribe::UnsubscribeInput).
-    pub fn builder() -> crate::operation::unsubscribe::builders::UnsubscribeInputBuilder {
-        crate::operation::unsubscribe::builders::UnsubscribeInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::unsubscribe::builders::UnsubscribeInputBuilder {
+        super::super::super::operation::unsubscribe::builders::UnsubscribeInputBuilder::default()
     }
 }

@@ -43,8 +43,8 @@
         &self.subscription_arn
     }
     /// Consumes the builder and constructs a [`UnsubscribeInput`](crate::operation::unsubscribe::UnsubscribeInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::unsubscribe::UnsubscribeInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::unsubscribe::UnsubscribeInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::unsubscribe::UnsubscribeInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::unsubscribe::UnsubscribeInput {
             subscription_arn: self.subscription_arn,
         })
     }
