--- original
+++ modified
@@ -48,8 +48,8 @@
 }
 impl ListAliasesInput {
     /// Creates a new builder-style object to manufacture [`ListAliasesInput`](crate::operation::list_aliases::ListAliasesInput).
-    pub fn builder() -> crate::operation::list_aliases::builders::ListAliasesInputBuilder {
-        crate::operation::list_aliases::builders::ListAliasesInputBuilder::default()
+    pub fn builder() -> super::operation::list_aliases::builders::ListAliasesInputBuilder {
+        super::operation::list_aliases::builders::ListAliasesInputBuilder::default()
     }
 }

@@ -138,8 +138,8 @@
         &self.marker
     }
     /// Consumes the builder and constructs a [`ListAliasesInput`](crate::operation::list_aliases::ListAliasesInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_aliases::ListAliasesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_aliases::ListAliasesInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_aliases::ListAliasesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_aliases::ListAliasesInput {
             key_id: self.key_id,
             limit: self.limit,
             marker: self.marker,
