--- original
+++ modified
@@ -26,15 +26,15 @@
     /// <li>
     /// <p><code>KMS</code> - server-side encryption with KMS managed keys</p></li>
     /// </ul>
-    pub s3_sse_algorithm: ::std::option::Option<crate::types::S3SseAlgorithm>,
+    pub s3_sse_algorithm: ::std::option::Option<super::super::super::types::S3SseAlgorithm>,
     /// <p>The ID of the KMS managed key used to encrypt the S3 bucket where export data will be stored (if applicable).</p>
     pub s3_sse_kms_key_id: ::std::option::Option<::std::string::String>,
     /// <p>The format for the exported data. Valid values for <code>ExportFormat</code> are <code>DYNAMODB_JSON</code> or <code>ION</code>.</p>
-    pub export_format: ::std::option::Option<crate::types::ExportFormat>,
+    pub export_format: ::std::option::Option<super::super::super::types::ExportFormat>,
     /// <p>Choice of whether to execute as a full export or incremental export. Valid values are FULL_EXPORT or INCREMENTAL_EXPORT. The default value is FULL_EXPORT. If INCREMENTAL_EXPORT is provided, the IncrementalExportSpecification must also be used.</p>
-    pub export_type: ::std::option::Option<crate::types::ExportType>,
+    pub export_type: ::std::option::Option<super::super::super::types::ExportType>,
     /// <p>Optional object containing the parameters specific to an incremental export.</p>
-    pub incremental_export_specification: ::std::option::Option<crate::types::IncrementalExportSpecification>,
+    pub incremental_export_specification: ::std::option::Option<super::super::super::types::IncrementalExportSpecification>,
 }
 impl ExportTableToPointInTimeInput {
     /// <p>The Amazon Resource Name (ARN) associated with the table to export.</p>
@@ -72,7 +72,7 @@
     /// <li>
     /// <p><code>KMS</code> - server-side encryption with KMS managed keys</p></li>
     /// </ul>
-    pub fn s3_sse_algorithm(&self) -> ::std::option::Option<&crate::types::S3SseAlgorithm> {
+    pub fn s3_sse_algorithm(&self) -> ::std::option::Option<&super::super::super::types::S3SseAlgorithm> {
         self.s3_sse_algorithm.as_ref()
     }
     /// <p>The ID of the KMS managed key used to encrypt the S3 bucket where export data will be stored (if applicable).</p>
@@ -80,22 +80,22 @@
         self.s3_sse_kms_key_id.as_deref()
     }
     /// <p>The format for the exported data. Valid values for <code>ExportFormat</code> are <code>DYNAMODB_JSON</code> or <code>ION</code>.</p>
-    pub fn export_format(&self) -> ::std::option::Option<&crate::types::ExportFormat> {
+    pub fn export_format(&self) -> ::std::option::Option<&super::super::super::types::ExportFormat> {
         self.export_format.as_ref()
     }
     /// <p>Choice of whether to execute as a full export or incremental export. Valid values are FULL_EXPORT or INCREMENTAL_EXPORT. The default value is FULL_EXPORT. If INCREMENTAL_EXPORT is provided, the IncrementalExportSpecification must also be used.</p>
-    pub fn export_type(&self) -> ::std::option::Option<&crate::types::ExportType> {
+    pub fn export_type(&self) -> ::std::option::Option<&super::super::super::types::ExportType> {
         self.export_type.as_ref()
     }
     /// <p>Optional object containing the parameters specific to an incremental export.</p>
-    pub fn incremental_export_specification(&self) -> ::std::option::Option<&crate::types::IncrementalExportSpecification> {
+    pub fn incremental_export_specification(&self) -> ::std::option::Option<&super::super::super::types::IncrementalExportSpecification> {
         self.incremental_export_specification.as_ref()
     }
 }
 impl ExportTableToPointInTimeInput {
     /// Creates a new builder-style object to manufacture [`ExportTableToPointInTimeInput`](crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput).
-    pub fn builder() -> crate::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeInputBuilder {
-        crate::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeInputBuilder {
+        super::super::super::operation::export_table_to_point_in_time::builders::ExportTableToPointInTimeInputBuilder::default()
     }
 }

@@ -109,11 +109,11 @@
     pub(crate) s3_bucket: ::std::option::Option<::std::string::String>,
     pub(crate) s3_bucket_owner: ::std::option::Option<::std::string::String>,
     pub(crate) s3_prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) s3_sse_algorithm: ::std::option::Option<crate::types::S3SseAlgorithm>,
+    pub(crate) s3_sse_algorithm: ::std::option::Option<super::super::super::types::S3SseAlgorithm>,
     pub(crate) s3_sse_kms_key_id: ::std::option::Option<::std::string::String>,
-    pub(crate) export_format: ::std::option::Option<crate::types::ExportFormat>,
-    pub(crate) export_type: ::std::option::Option<crate::types::ExportType>,
-    pub(crate) incremental_export_specification: ::std::option::Option<crate::types::IncrementalExportSpecification>,
+    pub(crate) export_format: ::std::option::Option<super::super::super::types::ExportFormat>,
+    pub(crate) export_type: ::std::option::Option<super::super::super::types::ExportType>,
+    pub(crate) incremental_export_specification: ::std::option::Option<super::super::super::types::IncrementalExportSpecification>,
 }
 impl ExportTableToPointInTimeInputBuilder {
     /// <p>The Amazon Resource Name (ARN) associated with the table to export.</p>
@@ -221,7 +221,7 @@
     /// <li>
     /// <p><code>KMS</code> - server-side encryption with KMS managed keys</p></li>
     /// </ul>
-    pub fn s3_sse_algorithm(mut self, input: crate::types::S3SseAlgorithm) -> Self {
+    pub fn s3_sse_algorithm(mut self, input: super::super::super::types::S3SseAlgorithm) -> Self {
         self.s3_sse_algorithm = ::std::option::Option::Some(input);
         self
     }
@@ -232,7 +232,7 @@
     /// <li>
     /// <p><code>KMS</code> - server-side encryption with KMS managed keys</p></li>
     /// </ul>
-    pub fn set_s3_sse_algorithm(mut self, input: ::std::option::Option<crate::types::S3SseAlgorithm>) -> Self {
+    pub fn set_s3_sse_algorithm(mut self, input: ::std::option::Option<super::super::super::types::S3SseAlgorithm>) -> Self {
         self.s3_sse_algorithm = input;
         self
     }
@@ -243,7 +243,7 @@
     /// <li>
     /// <p><code>KMS</code> - server-side encryption with KMS managed keys</p></li>
     /// </ul>
-    pub fn get_s3_sse_algorithm(&self) -> &::std::option::Option<crate::types::S3SseAlgorithm> {
+    pub fn get_s3_sse_algorithm(&self) -> &::std::option::Option<super::super::super::types::S3SseAlgorithm> {
         &self.s3_sse_algorithm
     }
     /// <p>The ID of the KMS managed key used to encrypt the S3 bucket where export data will be stored (if applicable).</p>
@@ -261,45 +261,45 @@
         &self.s3_sse_kms_key_id
     }
     /// <p>The format for the exported data. Valid values for <code>ExportFormat</code> are <code>DYNAMODB_JSON</code> or <code>ION</code>.</p>
-    pub fn export_format(mut self, input: crate::types::ExportFormat) -> Self {
+    pub fn export_format(mut self, input: super::super::super::types::ExportFormat) -> Self {
         self.export_format = ::std::option::Option::Some(input);
         self
     }
     /// <p>The format for the exported data. Valid values for <code>ExportFormat</code> are <code>DYNAMODB_JSON</code> or <code>ION</code>.</p>
-    pub fn set_export_format(mut self, input: ::std::option::Option<crate::types::ExportFormat>) -> Self {
+    pub fn set_export_format(mut self, input: ::std::option::Option<super::super::super::types::ExportFormat>) -> Self {
         self.export_format = input;
         self
     }
     /// <p>The format for the exported data. Valid values for <code>ExportFormat</code> are <code>DYNAMODB_JSON</code> or <code>ION</code>.</p>
-    pub fn get_export_format(&self) -> &::std::option::Option<crate::types::ExportFormat> {
+    pub fn get_export_format(&self) -> &::std::option::Option<super::super::super::types::ExportFormat> {
         &self.export_format
     }
     /// <p>Choice of whether to execute as a full export or incremental export. Valid values are FULL_EXPORT or INCREMENTAL_EXPORT. The default value is FULL_EXPORT. If INCREMENTAL_EXPORT is provided, the IncrementalExportSpecification must also be used.</p>
-    pub fn export_type(mut self, input: crate::types::ExportType) -> Self {
+    pub fn export_type(mut self, input: super::super::super::types::ExportType) -> Self {
         self.export_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>Choice of whether to execute as a full export or incremental export. Valid values are FULL_EXPORT or INCREMENTAL_EXPORT. The default value is FULL_EXPORT. If INCREMENTAL_EXPORT is provided, the IncrementalExportSpecification must also be used.</p>
-    pub fn set_export_type(mut self, input: ::std::option::Option<crate::types::ExportType>) -> Self {
+    pub fn set_export_type(mut self, input: ::std::option::Option<super::super::super::types::ExportType>) -> Self {
         self.export_type = input;
         self
     }
     /// <p>Choice of whether to execute as a full export or incremental export. Valid values are FULL_EXPORT or INCREMENTAL_EXPORT. The default value is FULL_EXPORT. If INCREMENTAL_EXPORT is provided, the IncrementalExportSpecification must also be used.</p>
-    pub fn get_export_type(&self) -> &::std::option::Option<crate::types::ExportType> {
+    pub fn get_export_type(&self) -> &::std::option::Option<super::super::super::types::ExportType> {
         &self.export_type
     }
     /// <p>Optional object containing the parameters specific to an incremental export.</p>
-    pub fn incremental_export_specification(mut self, input: crate::types::IncrementalExportSpecification) -> Self {
+    pub fn incremental_export_specification(mut self, input: super::super::super::types::IncrementalExportSpecification) -> Self {
         self.incremental_export_specification = ::std::option::Option::Some(input);
         self
     }
     /// <p>Optional object containing the parameters specific to an incremental export.</p>
-    pub fn set_incremental_export_specification(mut self, input: ::std::option::Option<crate::types::IncrementalExportSpecification>) -> Self {
+    pub fn set_incremental_export_specification(mut self, input: ::std::option::Option<super::super::super::types::IncrementalExportSpecification>) -> Self {
         self.incremental_export_specification = input;
         self
     }
     /// <p>Optional object containing the parameters specific to an incremental export.</p>
-    pub fn get_incremental_export_specification(&self) -> &::std::option::Option<crate::types::IncrementalExportSpecification> {
+    pub fn get_incremental_export_specification(&self) -> &::std::option::Option<super::super::super::types::IncrementalExportSpecification> {
         &self.incremental_export_specification
     }
     /// Consumes the builder and constructs a [`ExportTableToPointInTimeInput`](crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput).
@@ -306,10 +306,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput,
+        super::super::super::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput {
+        ::std::result::Result::Ok(super::super::super::operation::export_table_to_point_in_time::ExportTableToPointInTimeInput {
             table_arn: self.table_arn,
             export_time: self.export_time,
             client_token: self.client_token,
