--- original
+++ modified
@@ -40,7 +40,7 @@
     }
 }
 impl ::std::error::Error for ModelErrorException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::ModelErrorException {
+impl ::aws_types::request_id::RequestId for super::types::error::ModelErrorException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -53,8 +53,8 @@
 }
 impl ModelErrorException {
     /// Creates a new builder-style object to manufacture [`ModelErrorException`](crate::types::error::ModelErrorException).
-    pub fn builder() -> crate::types::error::builders::ModelErrorExceptionBuilder {
-        crate::types::error::builders::ModelErrorExceptionBuilder::default()
+    pub fn builder() -> super::types::error::builders::ModelErrorExceptionBuilder {
+        super::types::error::builders::ModelErrorExceptionBuilder::default()
     }
 }

@@ -122,8 +122,8 @@
         self
     }
     /// Consumes the builder and constructs a [`ModelErrorException`](crate::types::error::ModelErrorException).
-    pub fn build(self) -> crate::types::error::ModelErrorException {
-        crate::types::error::ModelErrorException {
+    pub fn build(self) -> super::types::error::ModelErrorException {
+        super::types::error::ModelErrorException {
             message: self.message,
             original_status_code: self.original_status_code,
             resource_name: self.resource_name,
