--- original
+++ modified
@@ -30,7 +30,7 @@
     }
 }
 impl ::std::error::Error for ModelNotReadyException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::ModelNotReadyException {
+impl ::aws_types::request_id::RequestId for super::types::error::ModelNotReadyException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -43,8 +43,8 @@
 }
 impl ModelNotReadyException {
     /// Creates a new builder-style object to manufacture [`ModelNotReadyException`](crate::types::error::ModelNotReadyException).
-    pub fn builder() -> crate::types::error::builders::ModelNotReadyExceptionBuilder {
-        crate::types::error::builders::ModelNotReadyExceptionBuilder::default()
+    pub fn builder() -> super::types::error::builders::ModelNotReadyExceptionBuilder {
+        super::types::error::builders::ModelNotReadyExceptionBuilder::default()
     }
 }

@@ -82,8 +82,8 @@
         self
     }
     /// Consumes the builder and constructs a [`ModelNotReadyException`](crate::types::error::ModelNotReadyException).
-    pub fn build(self) -> crate::types::error::ModelNotReadyException {
-        crate::types::error::ModelNotReadyException {
+    pub fn build(self) -> super::types::error::ModelNotReadyException {
+        super::types::error::ModelNotReadyException {
             message: self.message,
             meta: self.meta.unwrap_or_default(),
         }
