--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl AddClientIdToOpenIdConnectProviderInput {
     /// Creates a new builder-style object to manufacture [`AddClientIdToOpenIdConnectProviderInput`](crate::operation::add_client_id_to_open_id_connect_provider::AddClientIdToOpenIdConnectProviderInput).
-    pub fn builder() -> crate::operation::add_client_id_to_open_id_connect_provider::builders::AddClientIdToOpenIdConnectProviderInputBuilder {
-        crate::operation::add_client_id_to_open_id_connect_provider::builders::AddClientIdToOpenIdConnectProviderInputBuilder::default()
+    pub fn builder() -> super::operation::add_client_id_to_open_id_connect_provider::builders::AddClientIdToOpenIdConnectProviderInputBuilder {
+        super::operation::add_client_id_to_open_id_connect_provider::builders::AddClientIdToOpenIdConnectProviderInputBuilder::default()
     }
 }

@@ -67,11 +67,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::add_client_id_to_open_id_connect_provider::AddClientIdToOpenIdConnectProviderInput,
+        super::operation::add_client_id_to_open_id_connect_provider::AddClientIdToOpenIdConnectProviderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::add_client_id_to_open_id_connect_provider::AddClientIdToOpenIdConnectProviderInput {
+            super::operation::add_client_id_to_open_id_connect_provider::AddClientIdToOpenIdConnectProviderInput {
                 open_id_connect_provider_arn: self.open_id_connect_provider_arn,
                 client_id: self.client_id,
             },
