--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteResourcePolicyInput {
     /// Creates a new builder-style object to manufacture [`DeleteResourcePolicyInput`](crate::operation::delete_resource_policy::DeleteResourcePolicyInput).
-    pub fn builder() -> crate::operation::delete_resource_policy::builders::DeleteResourcePolicyInputBuilder {
-        crate::operation::delete_resource_policy::builders::DeleteResourcePolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_resource_policy::builders::DeleteResourcePolicyInputBuilder {
+        super::super::super::operation::delete_resource_policy::builders::DeleteResourcePolicyInputBuilder::default()
     }
 }

@@ -65,9 +65,9 @@
     /// Consumes the builder and constructs a [`DeleteResourcePolicyInput`](crate::operation::delete_resource_policy::DeleteResourcePolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_resource_policy::DeleteResourcePolicyInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::delete_resource_policy::DeleteResourcePolicyInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::delete_resource_policy::DeleteResourcePolicyInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_resource_policy::DeleteResourcePolicyInput {
             resource_arn: self.resource_arn,
             revision_id: self.revision_id,
         })
