--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AbacStatus {
     /// <p>The ABAC status of the general purpose bucket.</p>
-    pub status: ::std::option::Option<crate::types::BucketAbacStatus>,
+    pub status: ::std::option::Option<super::types::BucketAbacStatus>,
 }
 impl AbacStatus {
     /// <p>The ABAC status of the general purpose bucket.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::BucketAbacStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::BucketAbacStatus> {
         self.status.as_ref()
     }
 }
 impl AbacStatus {
     /// Creates a new builder-style object to manufacture [`AbacStatus`](crate::types::AbacStatus).
-    pub fn builder() -> crate::types::builders::AbacStatusBuilder {
-        crate::types::builders::AbacStatusBuilder::default()
+    pub fn builder() -> super::types::builders::AbacStatusBuilder {
+        super::types::builders::AbacStatusBuilder::default()
     }
 }

@@ -24,25 +24,25 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AbacStatusBuilder {
-    pub(crate) status: ::std::option::Option<crate::types::BucketAbacStatus>,
+    pub(crate) status: ::std::option::Option<super::types::BucketAbacStatus>,
 }
 impl AbacStatusBuilder {
     /// <p>The ABAC status of the general purpose bucket.</p>
-    pub fn status(mut self, input: crate::types::BucketAbacStatus) -> Self {
+    pub fn status(mut self, input: super::types::BucketAbacStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The ABAC status of the general purpose bucket.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::BucketAbacStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::BucketAbacStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>The ABAC status of the general purpose bucket.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::BucketAbacStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::BucketAbacStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`AbacStatus`](crate::types::AbacStatus).
-    pub fn build(self) -> crate::types::AbacStatus {
-        crate::types::AbacStatus { status: self.status }
+    pub fn build(self) -> super::types::AbacStatus {
+        super::types::AbacStatus { status: self.status }
     }
 }
