--- original
+++ modified
@@ -5,26 +5,26 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AccessControlPolicy {
     /// <p>A list of grants.</p>
-    pub grants: ::std::option::Option<::std::vec::Vec<crate::types::Grant>>,
+    pub grants: ::std::option::Option<::std::vec::Vec<super::super::types::Grant>>,
     /// <p>Container for the bucket owner's display name and ID.</p>
-    pub owner: ::std::option::Option<crate::types::Owner>,
+    pub owner: ::std::option::Option<super::super::types::Owner>,
 }
 impl AccessControlPolicy {
     /// <p>A list of grants.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.grants.is_none()`.
-    pub fn grants(&self) -> &[crate::types::Grant] {
+    pub fn grants(&self) -> &[super::super::types::Grant] {
         self.grants.as_deref().unwrap_or_default()
     }
     /// <p>Container for the bucket owner's display name and ID.</p>
-    pub fn owner(&self) -> ::std::option::Option<&crate::types::Owner> {
+    pub fn owner(&self) -> ::std::option::Option<&super::super::types::Owner> {
         self.owner.as_ref()
     }
 }
 impl AccessControlPolicy {
     /// Creates a new builder-style object to manufacture [`AccessControlPolicy`](crate::types::AccessControlPolicy).
-    pub fn builder() -> crate::types::builders::AccessControlPolicyBuilder {
-        crate::types::builders::AccessControlPolicyBuilder::default()
+    pub fn builder() -> super::super::types::builders::AccessControlPolicyBuilder {
+        super::super::types::builders::AccessControlPolicyBuilder::default()
     }
 }

@@ -32,8 +32,8 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AccessControlPolicyBuilder {
-    pub(crate) grants: ::std::option::Option<::std::vec::Vec<crate::types::Grant>>,
-    pub(crate) owner: ::std::option::Option<crate::types::Owner>,
+    pub(crate) grants: ::std::option::Option<::std::vec::Vec<super::super::types::Grant>>,
+    pub(crate) owner: ::std::option::Option<super::super::types::Owner>,
 }
 impl AccessControlPolicyBuilder {
     /// Appends an item to `grants`.
@@ -41,7 +41,7 @@
     /// To override the contents of this collection use [`set_grants`](Self::set_grants).
     ///
     /// <p>A list of grants.</p>
-    pub fn grants(mut self, input: crate::types::Grant) -> Self {
+    pub fn grants(mut self, input: super::super::types::Grant) -> Self {
         let mut v = self.grants.unwrap_or_default();
         v.push(input);
         self.grants = ::std::option::Option::Some(v);
@@ -48,31 +48,31 @@
         self
     }
     /// <p>A list of grants.</p>
-    pub fn set_grants(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Grant>>) -> Self {
+    pub fn set_grants(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::Grant>>) -> Self {
         self.grants = input;
         self
     }
     /// <p>A list of grants.</p>
-    pub fn get_grants(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Grant>> {
+    pub fn get_grants(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::Grant>> {
         &self.grants
     }
     /// <p>Container for the bucket owner's display name and ID.</p>
-    pub fn owner(mut self, input: crate::types::Owner) -> Self {
+    pub fn owner(mut self, input: super::super::types::Owner) -> Self {
         self.owner = ::std::option::Option::Some(input);
         self
     }
     /// <p>Container for the bucket owner's display name and ID.</p>
-    pub fn set_owner(mut self, input: ::std::option::Option<crate::types::Owner>) -> Self {
+    pub fn set_owner(mut self, input: ::std::option::Option<super::super::types::Owner>) -> Self {
         self.owner = input;
         self
     }
     /// <p>Container for the bucket owner's display name and ID.</p>
-    pub fn get_owner(&self) -> &::std::option::Option<crate::types::Owner> {
+    pub fn get_owner(&self) -> &::std::option::Option<super::super::types::Owner> {
         &self.owner
     }
     /// Consumes the builder and constructs a [`AccessControlPolicy`](crate::types::AccessControlPolicy).
-    pub fn build(self) -> crate::types::AccessControlPolicy {
-        crate::types::AccessControlPolicy {
+    pub fn build(self) -> super::super::types::AccessControlPolicy {
+        super::super::types::AccessControlPolicy {
             grants: self.grants,
             owner: self.owner,
         }
