--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AccelerateConfiguration {
     /// <p>Specifies the transfer acceleration status of the bucket.</p>
-    pub status: ::std::option::Option<crate::types::BucketAccelerateStatus>,
+    pub status: ::std::option::Option<super::super::types::BucketAccelerateStatus>,
 }
 impl AccelerateConfiguration {
     /// <p>Specifies the transfer acceleration status of the bucket.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::BucketAccelerateStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::BucketAccelerateStatus> {
         self.status.as_ref()
     }
 }
 impl AccelerateConfiguration {
     /// Creates a new builder-style object to manufacture [`AccelerateConfiguration`](crate::types::AccelerateConfiguration).
-    pub fn builder() -> crate::types::builders::AccelerateConfigurationBuilder {
-        crate::types::builders::AccelerateConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::AccelerateConfigurationBuilder {
+        super::super::types::builders::AccelerateConfigurationBuilder::default()
     }
 }

@@ -24,25 +24,25 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AccelerateConfigurationBuilder {
-    pub(crate) status: ::std::option::Option<crate::types::BucketAccelerateStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::BucketAccelerateStatus>,
 }
 impl AccelerateConfigurationBuilder {
     /// <p>Specifies the transfer acceleration status of the bucket.</p>
-    pub fn status(mut self, input: crate::types::BucketAccelerateStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::BucketAccelerateStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the transfer acceleration status of the bucket.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::BucketAccelerateStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::BucketAccelerateStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>Specifies the transfer acceleration status of the bucket.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::BucketAccelerateStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::BucketAccelerateStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`AccelerateConfiguration`](crate::types::AccelerateConfiguration).
-    pub fn build(self) -> crate::types::AccelerateConfiguration {
-        crate::types::AccelerateConfiguration { status: self.status }
+    pub fn build(self) -> super::super::types::AccelerateConfiguration {
+        super::super::types::AccelerateConfiguration { status: self.status }
     }
 }
