--- original
+++ modified
@@ -5,11 +5,11 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub enum AnalyticsFilter {
     /// <p>A conjunction (logical AND) of predicates, which is used in evaluating an analytics filter. The operator must have at least two predicates.</p>
-    And(crate::types::AnalyticsAndOperator),
+    And(super::super::types::AnalyticsAndOperator),
     /// <p>The prefix to use when evaluating an analytics filter.</p>
     Prefix(::std::string::String),
     /// <p>The tag to use when evaluating an analytics filter.</p>
-    Tag(crate::types::Tag),
+    Tag(super::super::types::Tag),
     /// The `Unknown` variant represents cases where new union variant was received. Consider upgrading the SDK to the latest available version.
     /// An unknown enum variant
     ///
@@ -23,7 +23,7 @@
 impl AnalyticsFilter {
     /// Tries to convert the enum instance into [`And`](crate::types::AnalyticsFilter::And), extracting the inner [`AnalyticsAndOperator`](crate::types::AnalyticsAndOperator).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_and(&self) -> ::std::result::Result<&crate::types::AnalyticsAndOperator, &Self> {
+    pub fn as_and(&self) -> ::std::result::Result<&super::super::types::AnalyticsAndOperator, &Self> {
         if let AnalyticsFilter::And(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
@@ -49,7 +49,7 @@
     }
     /// Tries to convert the enum instance into [`Tag`](crate::types::AnalyticsFilter::Tag), extracting the inner [`Tag`](crate::types::Tag).
     /// Returns `Err(&Self)` if it can't be converted.
-    pub fn as_tag(&self) -> ::std::result::Result<&crate::types::Tag, &Self> {
+    pub fn as_tag(&self) -> ::std::result::Result<&super::super::types::Tag, &Self> {
         if let AnalyticsFilter::Tag(val) = &self {
             ::std::result::Result::Ok(val)
         } else {
