--- original
+++ modified
@@ -5,19 +5,19 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct AnnotationTableConfiguration {
     /// <p>The state of the annotation table. Valid values are <code>ENABLED</code> and <code>DISABLED</code>.</p>
-    pub configuration_state: crate::types::AnnotationConfigurationState,
+    pub configuration_state: super::super::types::AnnotationConfigurationState,
     /// <p>The encryption settings for an S3 Metadata journal table or inventory table configuration.</p>
-    pub encryption_configuration: ::std::option::Option<crate::types::MetadataTableEncryptionConfiguration>,
+    pub encryption_configuration: ::std::option::Option<super::super::types::MetadataTableEncryptionConfiguration>,
     /// <p>The ARN of the IAM role used to manage the annotation table.</p>
     pub role: ::std::option::Option<::std::string::String>,
 }
 impl AnnotationTableConfiguration {
     /// <p>The state of the annotation table. Valid values are <code>ENABLED</code> and <code>DISABLED</code>.</p>
-    pub fn configuration_state(&self) -> &crate::types::AnnotationConfigurationState {
+    pub fn configuration_state(&self) -> &super::super::types::AnnotationConfigurationState {
         &self.configuration_state
     }
     /// <p>The encryption settings for an S3 Metadata journal table or inventory table configuration.</p>
-    pub fn encryption_configuration(&self) -> ::std::option::Option<&crate::types::MetadataTableEncryptionConfiguration> {
+    pub fn encryption_configuration(&self) -> ::std::option::Option<&super::super::types::MetadataTableEncryptionConfiguration> {
         self.encryption_configuration.as_ref()
     }
     /// <p>The ARN of the IAM role used to manage the annotation table.</p>
@@ -27,8 +27,8 @@
 }
 impl AnnotationTableConfiguration {
     /// Creates a new builder-style object to manufacture [`AnnotationTableConfiguration`](crate::types::AnnotationTableConfiguration).
-    pub fn builder() -> crate::types::builders::AnnotationTableConfigurationBuilder {
-        crate::types::builders::AnnotationTableConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::AnnotationTableConfigurationBuilder {
+        super::super::types::builders::AnnotationTableConfigurationBuilder::default()
     }
 }

@@ -36,38 +36,38 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct AnnotationTableConfigurationBuilder {
-    pub(crate) configuration_state: ::std::option::Option<crate::types::AnnotationConfigurationState>,
-    pub(crate) encryption_configuration: ::std::option::Option<crate::types::MetadataTableEncryptionConfiguration>,
+    pub(crate) configuration_state: ::std::option::Option<super::super::types::AnnotationConfigurationState>,
+    pub(crate) encryption_configuration: ::std::option::Option<super::super::types::MetadataTableEncryptionConfiguration>,
     pub(crate) role: ::std::option::Option<::std::string::String>,
 }
 impl AnnotationTableConfigurationBuilder {
     /// <p>The state of the annotation table. Valid values are <code>ENABLED</code> and <code>DISABLED</code>.</p>
     /// This field is required.
-    pub fn configuration_state(mut self, input: crate::types::AnnotationConfigurationState) -> Self {
+    pub fn configuration_state(mut self, input: super::super::types::AnnotationConfigurationState) -> Self {
         self.configuration_state = ::std::option::Option::Some(input);
         self
     }
     /// <p>The state of the annotation table. Valid values are <code>ENABLED</code> and <code>DISABLED</code>.</p>
-    pub fn set_configuration_state(mut self, input: ::std::option::Option<crate::types::AnnotationConfigurationState>) -> Self {
+    pub fn set_configuration_state(mut self, input: ::std::option::Option<super::super::types::AnnotationConfigurationState>) -> Self {
         self.configuration_state = input;
         self
     }
     /// <p>The state of the annotation table. Valid values are <code>ENABLED</code> and <code>DISABLED</code>.</p>
-    pub fn get_configuration_state(&self) -> &::std::option::Option<crate::types::AnnotationConfigurationState> {
+    pub fn get_configuration_state(&self) -> &::std::option::Option<super::super::types::AnnotationConfigurationState> {
         &self.configuration_state
     }
     /// <p>The encryption settings for an S3 Metadata journal table or inventory table configuration.</p>
-    pub fn encryption_configuration(mut self, input: crate::types::MetadataTableEncryptionConfiguration) -> Self {
+    pub fn encryption_configuration(mut self, input: super::super::types::MetadataTableEncryptionConfiguration) -> Self {
         self.encryption_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>The encryption settings for an S3 Metadata journal table or inventory table configuration.</p>
-    pub fn set_encryption_configuration(mut self, input: ::std::option::Option<crate::types::MetadataTableEncryptionConfiguration>) -> Self {
+    pub fn set_encryption_configuration(mut self, input: ::std::option::Option<super::super::types::MetadataTableEncryptionConfiguration>) -> Self {
         self.encryption_configuration = input;
         self
     }
     /// <p>The encryption settings for an S3 Metadata journal table or inventory table configuration.</p>
-    pub fn get_encryption_configuration(&self) -> &::std::option::Option<crate::types::MetadataTableEncryptionConfiguration> {
+    pub fn get_encryption_configuration(&self) -> &::std::option::Option<super::super::types::MetadataTableEncryptionConfiguration> {
         &self.encryption_configuration
     }
     /// <p>The ARN of the IAM role used to manage the annotation table.</p>
@@ -87,8 +87,8 @@
     /// Consumes the builder and constructs a [`AnnotationTableConfiguration`](crate::types::AnnotationTableConfiguration).
     /// This method will fail if any of the following fields are not set:
     /// - [`configuration_state`](crate::types::builders::AnnotationTableConfigurationBuilder::configuration_state)
-    pub fn build(self) -> ::std::result::Result<crate::types::AnnotationTableConfiguration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::AnnotationTableConfiguration {
+    pub fn build(self) -> ::std::result::Result<super::super::types::AnnotationTableConfiguration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::AnnotationTableConfiguration {
             configuration_state: self.configuration_state.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "configuration_state",
