--- original
+++ modified
@@ -37,8 +37,8 @@
 }
 impl Bucket {
     /// Creates a new builder-style object to manufacture [`Bucket`](crate::types::Bucket).
-    pub fn builder() -> crate::types::builders::BucketBuilder {
-        crate::types::builders::BucketBuilder::default()
+    pub fn builder() -> super::types::builders::BucketBuilder {
+        super::types::builders::BucketBuilder::default()
     }
 }

@@ -115,8 +115,8 @@
         &self.bucket_arn
     }
     /// Consumes the builder and constructs a [`Bucket`](crate::types::Bucket).
-    pub fn build(self) -> crate::types::Bucket {
-        crate::types::Bucket {
+    pub fn build(self) -> super::types::Bucket {
+        super::types::Bucket {
             name: self.name,
             creation_date: self.creation_date,
             bucket_region: self.bucket_region,
