--- original
+++ modified
@@ -6,8 +6,8 @@
 pub struct ContinuationEvent {}
 impl ContinuationEvent {
     /// Creates a new builder-style object to manufacture [`ContinuationEvent`](crate::types::ContinuationEvent).
-    pub fn builder() -> crate::types::builders::ContinuationEventBuilder {
-        crate::types::builders::ContinuationEventBuilder::default()
+    pub fn builder() -> super::super::types::builders::ContinuationEventBuilder {
+        super::super::types::builders::ContinuationEventBuilder::default()
     }
 }

@@ -17,7 +17,7 @@
 pub struct ContinuationEventBuilder {}
 impl ContinuationEventBuilder {
     /// Consumes the builder and constructs a [`ContinuationEvent`](crate::types::ContinuationEvent).
-    pub fn build(self) -> crate::types::ContinuationEvent {
-        crate::types::ContinuationEvent {}
+    pub fn build(self) -> super::super::types::ContinuationEvent {
+        super::super::types::ContinuationEvent {}
     }
 }
