--- original
+++ modified
@@ -51,8 +51,8 @@
 }
 impl CorsRule {
     /// Creates a new builder-style object to manufacture [`CorsRule`](crate::types::CorsRule).
-    pub fn builder() -> crate::types::builders::CorsRuleBuilder {
-        crate::types::builders::CorsRuleBuilder::default()
+    pub fn builder() -> super::super::types::builders::CorsRuleBuilder {
+        super::super::types::builders::CorsRuleBuilder::default()
     }
 }

@@ -180,8 +180,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`allowed_methods`](crate::types::builders::CorsRuleBuilder::allowed_methods)
     /// - [`allowed_origins`](crate::types::builders::CorsRuleBuilder::allowed_origins)
-    pub fn build(self) -> ::std::result::Result<crate::types::CorsRule, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::CorsRule {
+    pub fn build(self) -> ::std::result::Result<super::super::types::CorsRule, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::CorsRule {
             id: self.id,
             allowed_headers: self.allowed_headers,
             allowed_methods: self.allowed_methods.ok_or_else(|| {
