--- original
+++ modified
@@ -13,7 +13,7 @@
     /// <li>
     /// <p><code>Use</code>: First line is a header, and you can use the header value to identify a column in an expression (<code>SELECT "name" FROM OBJECT</code>).</p></li>
     /// </ul>
-    pub file_header_info: ::std::option::Option<crate::types::FileHeaderInfo>,
+    pub file_header_info: ::std::option::Option<super::super::types::FileHeaderInfo>,
     /// <p>A single character used to indicate that a row should be ignored when the character is present at the start of that row. You can specify any character to indicate a comment line. The default character is <code>#</code>.</p>
     /// <p>Default: <code>#</code></p>
     pub comments: ::std::option::Option<::std::string::String>,
@@ -41,7 +41,7 @@
     /// <li>
     /// <p><code>Use</code>: First line is a header, and you can use the header value to identify a column in an expression (<code>SELECT "name" FROM OBJECT</code>).</p></li>
     /// </ul>
-    pub fn file_header_info(&self) -> ::std::option::Option<&crate::types::FileHeaderInfo> {
+    pub fn file_header_info(&self) -> ::std::option::Option<&super::super::types::FileHeaderInfo> {
         self.file_header_info.as_ref()
     }
     /// <p>A single character used to indicate that a row should be ignored when the character is present at the start of that row. You can specify any character to indicate a comment line. The default character is <code>#</code>.</p>
@@ -75,8 +75,8 @@
 }
 impl CsvInput {
     /// Creates a new builder-style object to manufacture [`CsvInput`](crate::types::CsvInput).
-    pub fn builder() -> crate::types::builders::CsvInputBuilder {
-        crate::types::builders::CsvInputBuilder::default()
+    pub fn builder() -> super::super::types::builders::CsvInputBuilder {
+        super::super::types::builders::CsvInputBuilder::default()
     }
 }

@@ -84,7 +84,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct CsvInputBuilder {
-    pub(crate) file_header_info: ::std::option::Option<crate::types::FileHeaderInfo>,
+    pub(crate) file_header_info: ::std::option::Option<super::super::types::FileHeaderInfo>,
     pub(crate) comments: ::std::option::Option<::std::string::String>,
     pub(crate) quote_escape_character: ::std::option::Option<::std::string::String>,
     pub(crate) record_delimiter: ::std::option::Option<::std::string::String>,
@@ -102,7 +102,7 @@
     /// <li>
     /// <p><code>Use</code>: First line is a header, and you can use the header value to identify a column in an expression (<code>SELECT "name" FROM OBJECT</code>).</p></li>
     /// </ul>
-    pub fn file_header_info(mut self, input: crate::types::FileHeaderInfo) -> Self {
+    pub fn file_header_info(mut self, input: super::super::types::FileHeaderInfo) -> Self {
         self.file_header_info = ::std::option::Option::Some(input);
         self
     }
@@ -115,7 +115,7 @@
     /// <li>
     /// <p><code>Use</code>: First line is a header, and you can use the header value to identify a column in an expression (<code>SELECT "name" FROM OBJECT</code>).</p></li>
     /// </ul>
-    pub fn set_file_header_info(mut self, input: ::std::option::Option<crate::types::FileHeaderInfo>) -> Self {
+    pub fn set_file_header_info(mut self, input: ::std::option::Option<super::super::types::FileHeaderInfo>) -> Self {
         self.file_header_info = input;
         self
     }
@@ -128,7 +128,7 @@
     /// <li>
     /// <p><code>Use</code>: First line is a header, and you can use the header value to identify a column in an expression (<code>SELECT "name" FROM OBJECT</code>).</p></li>
     /// </ul>
-    pub fn get_file_header_info(&self) -> &::std::option::Option<crate::types::FileHeaderInfo> {
+    pub fn get_file_header_info(&self) -> &::std::option::Option<super::super::types::FileHeaderInfo> {
         &self.file_header_info
     }
     /// <p>A single character used to indicate that a row should be ignored when the character is present at the start of that row. You can specify any character to indicate a comment line. The default character is <code>#</code>.</p>
@@ -228,8 +228,8 @@
         &self.allow_quoted_record_delimiter
     }
     /// Consumes the builder and constructs a [`CsvInput`](crate::types::CsvInput).
-    pub fn build(self) -> crate::types::CsvInput {
-        crate::types::CsvInput {
+    pub fn build(self) -> super::super::types::CsvInput {
+        super::super::types::CsvInput {
             file_header_info: self.file_header_info,
             comments: self.comments,
             quote_escape_character: self.quote_escape_character,
