--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct DeleteMarkerEntry {
     /// <p>The account that created the delete marker.</p>
-    pub owner: ::std::option::Option<crate::types::Owner>,
+    pub owner: ::std::option::Option<super::super::types::Owner>,
     /// <p>The object key.</p>
     pub key: ::std::option::Option<::std::string::String>,
     /// <p>Version ID of an object.</p>
@@ -17,7 +17,7 @@
 }
 impl DeleteMarkerEntry {
     /// <p>The account that created the delete marker.</p>
-    pub fn owner(&self) -> ::std::option::Option<&crate::types::Owner> {
+    pub fn owner(&self) -> ::std::option::Option<&super::super::types::Owner> {
         self.owner.as_ref()
     }
     /// <p>The object key.</p>
@@ -39,8 +39,8 @@
 }
 impl DeleteMarkerEntry {
     /// Creates a new builder-style object to manufacture [`DeleteMarkerEntry`](crate::types::DeleteMarkerEntry).
-    pub fn builder() -> crate::types::builders::DeleteMarkerEntryBuilder {
-        crate::types::builders::DeleteMarkerEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeleteMarkerEntryBuilder {
+        super::super::types::builders::DeleteMarkerEntryBuilder::default()
     }
 }

@@ -48,7 +48,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct DeleteMarkerEntryBuilder {
-    pub(crate) owner: ::std::option::Option<crate::types::Owner>,
+    pub(crate) owner: ::std::option::Option<super::super::types::Owner>,
     pub(crate) key: ::std::option::Option<::std::string::String>,
     pub(crate) version_id: ::std::option::Option<::std::string::String>,
     pub(crate) is_latest: ::std::option::Option<bool>,
@@ -56,17 +56,17 @@
 }
 impl DeleteMarkerEntryBuilder {
     /// <p>The account that created the delete marker.</p>
-    pub fn owner(mut self, input: crate::types::Owner) -> Self {
+    pub fn owner(mut self, input: super::super::types::Owner) -> Self {
         self.owner = ::std::option::Option::Some(input);
         self
     }
     /// <p>The account that created the delete marker.</p>
-    pub fn set_owner(mut self, input: ::std::option::Option<crate::types::Owner>) -> Self {
+    pub fn set_owner(mut self, input: ::std::option::Option<super::super::types::Owner>) -> Self {
         self.owner = input;
         self
     }
     /// <p>The account that created the delete marker.</p>
-    pub fn get_owner(&self) -> &::std::option::Option<crate::types::Owner> {
+    pub fn get_owner(&self) -> &::std::option::Option<super::super::types::Owner> {
         &self.owner
     }
     /// <p>The object key.</p>
@@ -126,8 +126,8 @@
         &self.last_modified
     }
     /// Consumes the builder and constructs a [`DeleteMarkerEntry`](crate::types::DeleteMarkerEntry).
-    pub fn build(self) -> crate::types::DeleteMarkerEntry {
-        crate::types::DeleteMarkerEntry {
+    pub fn build(self) -> super::super::types::DeleteMarkerEntry {
+        super::super::types::DeleteMarkerEntry {
             owner: self.owner,
             key: self.key,
             version_id: self.version_id,
