--- original
+++ modified
@@ -10,20 +10,20 @@
     /// <p>Indicates whether to replicate delete markers.</p><note>
     /// <p>Indicates whether to replicate delete markers.</p>
     /// </note>
-    pub status: ::std::option::Option<crate::types::DeleteMarkerReplicationStatus>,
+    pub status: ::std::option::Option<super::super::types::DeleteMarkerReplicationStatus>,
 }
 impl DeleteMarkerReplication {
     /// <p>Indicates whether to replicate delete markers.</p><note>
     /// <p>Indicates whether to replicate delete markers.</p>
     /// </note>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::DeleteMarkerReplicationStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::DeleteMarkerReplicationStatus> {
         self.status.as_ref()
     }
 }
 impl DeleteMarkerReplication {
     /// Creates a new builder-style object to manufacture [`DeleteMarkerReplication`](crate::types::DeleteMarkerReplication).
-    pub fn builder() -> crate::types::builders::DeleteMarkerReplicationBuilder {
-        crate::types::builders::DeleteMarkerReplicationBuilder::default()
+    pub fn builder() -> super::super::types::builders::DeleteMarkerReplicationBuilder {
+        super::super::types::builders::DeleteMarkerReplicationBuilder::default()
     }
 }

@@ -31,13 +31,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct DeleteMarkerReplicationBuilder {
-    pub(crate) status: ::std::option::Option<crate::types::DeleteMarkerReplicationStatus>,
+    pub(crate) status: ::std::option::Option<super::super::types::DeleteMarkerReplicationStatus>,
 }
 impl DeleteMarkerReplicationBuilder {
     /// <p>Indicates whether to replicate delete markers.</p><note>
     /// <p>Indicates whether to replicate delete markers.</p>
     /// </note>
-    pub fn status(mut self, input: crate::types::DeleteMarkerReplicationStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::DeleteMarkerReplicationStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
@@ -44,7 +44,7 @@
     /// <p>Indicates whether to replicate delete markers.</p><note>
     /// <p>Indicates whether to replicate delete markers.</p>
     /// </note>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::DeleteMarkerReplicationStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::DeleteMarkerReplicationStatus>) -> Self {
         self.status = input;
         self
     }
@@ -51,11 +51,11 @@
     /// <p>Indicates whether to replicate delete markers.</p><note>
     /// <p>Indicates whether to replicate delete markers.</p>
     /// </note>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::DeleteMarkerReplicationStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::DeleteMarkerReplicationStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`DeleteMarkerReplication`](crate::types::DeleteMarkerReplication).
-    pub fn build(self) -> crate::types::DeleteMarkerReplication {
-        crate::types::DeleteMarkerReplication { status: self.status }
+    pub fn build(self) -> super::super::types::DeleteMarkerReplication {
+        super::super::types::DeleteMarkerReplication { status: self.status }
     }
 }
