--- original
+++ modified
@@ -5,7 +5,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq)]
 pub struct Encryption {
     /// <p>The server-side encryption algorithm used when storing job results in Amazon S3 (for example, AES256, <code>aws:kms</code>).</p>
-    pub encryption_type: crate::types::ServerSideEncryption,
+    pub encryption_type: super::super::types::ServerSideEncryption,
     /// <p>If the encryption type is <code>aws:kms</code>, this optional value specifies the ID of the symmetric encryption customer managed key to use for encryption of job results. Amazon S3 only supports symmetric encryption KMS keys. For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/symmetric-asymmetric.html">Asymmetric keys in KMS</a> in the <i>Amazon Web Services Key Management Service Developer Guide</i>.</p>
     pub kms_key_id: ::std::option::Option<::std::string::String>,
     /// <p>If the encryption type is <code>aws:kms</code>, this optional value can be used to specify the encryption context for the restore results.</p>
@@ -13,7 +13,7 @@
 }
 impl Encryption {
     /// <p>The server-side encryption algorithm used when storing job results in Amazon S3 (for example, AES256, <code>aws:kms</code>).</p>
-    pub fn encryption_type(&self) -> &crate::types::ServerSideEncryption {
+    pub fn encryption_type(&self) -> &super::super::types::ServerSideEncryption {
         &self.encryption_type
     }
     /// <p>If the encryption type is <code>aws:kms</code>, this optional value specifies the ID of the symmetric encryption customer managed key to use for encryption of job results. Amazon S3 only supports symmetric encryption KMS keys. For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/symmetric-asymmetric.html">Asymmetric keys in KMS</a> in the <i>Amazon Web Services Key Management Service Developer Guide</i>.</p>
@@ -36,8 +36,8 @@
 }
 impl Encryption {
     /// Creates a new builder-style object to manufacture [`Encryption`](crate::types::Encryption).
-    pub fn builder() -> crate::types::builders::EncryptionBuilder {
-        crate::types::builders::EncryptionBuilder::default()
+    pub fn builder() -> super::super::types::builders::EncryptionBuilder {
+        super::super::types::builders::EncryptionBuilder::default()
     }
 }

@@ -45,7 +45,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default)]
 #[non_exhaustive]
 pub struct EncryptionBuilder {
-    pub(crate) encryption_type: ::std::option::Option<crate::types::ServerSideEncryption>,
+    pub(crate) encryption_type: ::std::option::Option<super::super::types::ServerSideEncryption>,
     pub(crate) kms_key_id: ::std::option::Option<::std::string::String>,
     pub(crate) kms_context: ::std::option::Option<::std::string::String>,
 }
@@ -52,17 +52,17 @@
 impl EncryptionBuilder {
     /// <p>The server-side encryption algorithm used when storing job results in Amazon S3 (for example, AES256, <code>aws:kms</code>).</p>
     /// This field is required.
-    pub fn encryption_type(mut self, input: crate::types::ServerSideEncryption) -> Self {
+    pub fn encryption_type(mut self, input: super::super::types::ServerSideEncryption) -> Self {
         self.encryption_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The server-side encryption algorithm used when storing job results in Amazon S3 (for example, AES256, <code>aws:kms</code>).</p>
-    pub fn set_encryption_type(mut self, input: ::std::option::Option<crate::types::ServerSideEncryption>) -> Self {
+    pub fn set_encryption_type(mut self, input: ::std::option::Option<super::super::types::ServerSideEncryption>) -> Self {
         self.encryption_type = input;
         self
     }
     /// <p>The server-side encryption algorithm used when storing job results in Amazon S3 (for example, AES256, <code>aws:kms</code>).</p>
-    pub fn get_encryption_type(&self) -> &::std::option::Option<crate::types::ServerSideEncryption> {
+    pub fn get_encryption_type(&self) -> &::std::option::Option<super::super::types::ServerSideEncryption> {
         &self.encryption_type
     }
     /// <p>If the encryption type is <code>aws:kms</code>, this optional value specifies the ID of the symmetric encryption customer managed key to use for encryption of job results. Amazon S3 only supports symmetric encryption KMS keys. For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/symmetric-asymmetric.html">Asymmetric keys in KMS</a> in the <i>Amazon Web Services Key Management Service Developer Guide</i>.</p>
@@ -96,8 +96,8 @@
     /// Consumes the builder and constructs a [`Encryption`](crate::types::Encryption).
     /// This method will fail if any of the following fields are not set:
     /// - [`encryption_type`](crate::types::builders::EncryptionBuilder::encryption_type)
-    pub fn build(self) -> ::std::result::Result<crate::types::Encryption, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Encryption {
+    pub fn build(self) -> ::std::result::Result<super::super::types::Encryption, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::Encryption {
             encryption_type: self.encryption_type.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "encryption_type",
