--- original
+++ modified
@@ -5,13 +5,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct FilterRule {
     /// <p>The object key name prefix or suffix identifying one or more objects to which the filtering rule applies. The maximum length is 1,024 characters. Overlapping prefixes and suffixes are not supported. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html">Configuring Event Notifications</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub name: ::std::option::Option<crate::types::FilterRuleName>,
+    pub name: ::std::option::Option<super::super::types::FilterRuleName>,
     /// <p>The value that the filter searches for in object key names.</p>
     pub value: ::std::option::Option<::std::string::String>,
 }
 impl FilterRule {
     /// <p>The object key name prefix or suffix identifying one or more objects to which the filtering rule applies. The maximum length is 1,024 characters. Overlapping prefixes and suffixes are not supported. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html">Configuring Event Notifications</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn name(&self) -> ::std::option::Option<&crate::types::FilterRuleName> {
+    pub fn name(&self) -> ::std::option::Option<&super::super::types::FilterRuleName> {
         self.name.as_ref()
     }
     /// <p>The value that the filter searches for in object key names.</p>
@@ -21,8 +21,8 @@
 }
 impl FilterRule {
     /// Creates a new builder-style object to manufacture [`FilterRule`](crate::types::FilterRule).
-    pub fn builder() -> crate::types::builders::FilterRuleBuilder {
-        crate::types::builders::FilterRuleBuilder::default()
+    pub fn builder() -> super::super::types::builders::FilterRuleBuilder {
+        super::super::types::builders::FilterRuleBuilder::default()
     }
 }

@@ -30,22 +30,22 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct FilterRuleBuilder {
-    pub(crate) name: ::std::option::Option<crate::types::FilterRuleName>,
+    pub(crate) name: ::std::option::Option<super::super::types::FilterRuleName>,
     pub(crate) value: ::std::option::Option<::std::string::String>,
 }
 impl FilterRuleBuilder {
     /// <p>The object key name prefix or suffix identifying one or more objects to which the filtering rule applies. The maximum length is 1,024 characters. Overlapping prefixes and suffixes are not supported. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html">Configuring Event Notifications</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn name(mut self, input: crate::types::FilterRuleName) -> Self {
+    pub fn name(mut self, input: super::super::types::FilterRuleName) -> Self {
         self.name = ::std::option::Option::Some(input);
         self
     }
     /// <p>The object key name prefix or suffix identifying one or more objects to which the filtering rule applies. The maximum length is 1,024 characters. Overlapping prefixes and suffixes are not supported. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html">Configuring Event Notifications</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn set_name(mut self, input: ::std::option::Option<crate::types::FilterRuleName>) -> Self {
+    pub fn set_name(mut self, input: ::std::option::Option<super::super::types::FilterRuleName>) -> Self {
         self.name = input;
         self
     }
     /// <p>The object key name prefix or suffix identifying one or more objects to which the filtering rule applies. The maximum length is 1,024 characters. Overlapping prefixes and suffixes are not supported. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html">Configuring Event Notifications</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn get_name(&self) -> &::std::option::Option<crate::types::FilterRuleName> {
+    pub fn get_name(&self) -> &::std::option::Option<super::super::types::FilterRuleName> {
         &self.name
     }
     /// <p>The value that the filter searches for in object key names.</p>
@@ -63,8 +63,8 @@
         &self.value
     }
     /// Consumes the builder and constructs a [`FilterRule`](crate::types::FilterRule).
-    pub fn build(self) -> crate::types::FilterRule {
-        crate::types::FilterRule {
+    pub fn build(self) -> super::super::types::FilterRule {
+        super::super::types::FilterRule {
             name: self.name,
             value: self.value,
         }
