--- original
+++ modified
@@ -9,9 +9,9 @@
     /// </important>
     pub prefix: ::std::option::Option<::std::string::String>,
     /// <p>A container of a key value name pair.</p>
-    pub tag: ::std::option::Option<crate::types::Tag>,
+    pub tag: ::std::option::Option<super::types::Tag>,
     /// <p>A conjunction (logical AND) of predicates, which is used in evaluating a metrics filter. The operator must have at least two predicates, and an object must match all of the predicates in order for the filter to apply.</p>
-    pub and: ::std::option::Option<crate::types::IntelligentTieringAndOperator>,
+    pub and: ::std::option::Option<super::types::IntelligentTieringAndOperator>,
 }
 impl IntelligentTieringFilter {
     /// <p>An object key name prefix that identifies the subset of objects to which the rule applies.</p><important>
@@ -21,18 +21,18 @@
         self.prefix.as_deref()
     }
     /// <p>A container of a key value name pair.</p>
-    pub fn tag(&self) -> ::std::option::Option<&crate::types::Tag> {
+    pub fn tag(&self) -> ::std::option::Option<&super::types::Tag> {
         self.tag.as_ref()
     }
     /// <p>A conjunction (logical AND) of predicates, which is used in evaluating a metrics filter. The operator must have at least two predicates, and an object must match all of the predicates in order for the filter to apply.</p>
-    pub fn and(&self) -> ::std::option::Option<&crate::types::IntelligentTieringAndOperator> {
+    pub fn and(&self) -> ::std::option::Option<&super::types::IntelligentTieringAndOperator> {
         self.and.as_ref()
     }
 }
 impl IntelligentTieringFilter {
     /// Creates a new builder-style object to manufacture [`IntelligentTieringFilter`](crate::types::IntelligentTieringFilter).
-    pub fn builder() -> crate::types::builders::IntelligentTieringFilterBuilder {
-        crate::types::builders::IntelligentTieringFilterBuilder::default()
+    pub fn builder() -> super::types::builders::IntelligentTieringFilterBuilder {
+        super::types::builders::IntelligentTieringFilterBuilder::default()
     }
 }

@@ -41,8 +41,8 @@
 #[non_exhaustive]
 pub struct IntelligentTieringFilterBuilder {
     pub(crate) prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) tag: ::std::option::Option<crate::types::Tag>,
-    pub(crate) and: ::std::option::Option<crate::types::IntelligentTieringAndOperator>,
+    pub(crate) tag: ::std::option::Option<super::types::Tag>,
+    pub(crate) and: ::std::option::Option<super::types::IntelligentTieringAndOperator>,
 }
 impl IntelligentTieringFilterBuilder {
     /// <p>An object key name prefix that identifies the subset of objects to which the rule applies.</p><important>
@@ -66,36 +66,36 @@
         &self.prefix
     }
     /// <p>A container of a key value name pair.</p>
-    pub fn tag(mut self, input: crate::types::Tag) -> Self {
+    pub fn tag(mut self, input: super::types::Tag) -> Self {
         self.tag = ::std::option::Option::Some(input);
         self
     }
     /// <p>A container of a key value name pair.</p>
-    pub fn set_tag(mut self, input: ::std::option::Option<crate::types::Tag>) -> Self {
+    pub fn set_tag(mut self, input: ::std::option::Option<super::types::Tag>) -> Self {
         self.tag = input;
         self
     }
     /// <p>A container of a key value name pair.</p>
-    pub fn get_tag(&self) -> &::std::option::Option<crate::types::Tag> {
+    pub fn get_tag(&self) -> &::std::option::Option<super::types::Tag> {
         &self.tag
     }
     /// <p>A conjunction (logical AND) of predicates, which is used in evaluating a metrics filter. The operator must have at least two predicates, and an object must match all of the predicates in order for the filter to apply.</p>
-    pub fn and(mut self, input: crate::types::IntelligentTieringAndOperator) -> Self {
+    pub fn and(mut self, input: super::types::IntelligentTieringAndOperator) -> Self {
         self.and = ::std::option::Option::Some(input);
         self
     }
     /// <p>A conjunction (logical AND) of predicates, which is used in evaluating a metrics filter. The operator must have at least two predicates, and an object must match all of the predicates in order for the filter to apply.</p>
-    pub fn set_and(mut self, input: ::std::option::Option<crate::types::IntelligentTieringAndOperator>) -> Self {
+    pub fn set_and(mut self, input: ::std::option::Option<super::types::IntelligentTieringAndOperator>) -> Self {
         self.and = input;
         self
     }
     /// <p>A conjunction (logical AND) of predicates, which is used in evaluating a metrics filter. The operator must have at least two predicates, and an object must match all of the predicates in order for the filter to apply.</p>
-    pub fn get_and(&self) -> &::std::option::Option<crate::types::IntelligentTieringAndOperator> {
+    pub fn get_and(&self) -> &::std::option::Option<super::types::IntelligentTieringAndOperator> {
         &self.and
     }
     /// Consumes the builder and constructs a [`IntelligentTieringFilter`](crate::types::IntelligentTieringFilter).
-    pub fn build(self) -> crate::types::IntelligentTieringFilter {
-        crate::types::IntelligentTieringFilter {
+    pub fn build(self) -> super::types::IntelligentTieringFilter {
+        super::types::IntelligentTieringFilter {
             prefix: self.prefix,
             tag: self.tag,
             and: self.and,
