--- original
+++ modified
@@ -8,11 +8,11 @@
     /// <p>The ID used to identify the S3 Intelligent-Tiering configuration.</p>
     pub id: ::std::string::String,
     /// <p>Specifies a bucket filter. The configuration only includes objects that meet the filter's criteria.</p>
-    pub filter: ::std::option::Option<crate::types::IntelligentTieringFilter>,
+    pub filter: ::std::option::Option<super::super::types::IntelligentTieringFilter>,
     /// <p>Specifies the status of the configuration.</p>
-    pub status: crate::types::IntelligentTieringStatus,
+    pub status: super::super::types::IntelligentTieringStatus,
     /// <p>Specifies the S3 Intelligent-Tiering storage class tier of the configuration.</p>
-    pub tierings: ::std::vec::Vec<crate::types::Tiering>,
+    pub tierings: ::std::vec::Vec<super::super::types::Tiering>,
 }
 impl IntelligentTieringConfiguration {
     /// <p>The ID used to identify the S3 Intelligent-Tiering configuration.</p>
@@ -21,15 +21,15 @@
         self.id.deref()
     }
     /// <p>Specifies a bucket filter. The configuration only includes objects that meet the filter's criteria.</p>
-    pub fn filter(&self) -> ::std::option::Option<&crate::types::IntelligentTieringFilter> {
+    pub fn filter(&self) -> ::std::option::Option<&super::super::types::IntelligentTieringFilter> {
         self.filter.as_ref()
     }
     /// <p>Specifies the status of the configuration.</p>
-    pub fn status(&self) -> &crate::types::IntelligentTieringStatus {
+    pub fn status(&self) -> &super::super::types::IntelligentTieringStatus {
         &self.status
     }
     /// <p>Specifies the S3 Intelligent-Tiering storage class tier of the configuration.</p>
-    pub fn tierings(&self) -> &[crate::types::Tiering] {
+    pub fn tierings(&self) -> &[super::super::types::Tiering] {
         use std::ops::Deref;
         self.tierings.deref()
     }
@@ -36,8 +36,8 @@
 }
 impl IntelligentTieringConfiguration {
     /// Creates a new builder-style object to manufacture [`IntelligentTieringConfiguration`](crate::types::IntelligentTieringConfiguration).
-    pub fn builder() -> crate::types::builders::IntelligentTieringConfigurationBuilder {
-        crate::types::builders::IntelligentTieringConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::IntelligentTieringConfigurationBuilder {
+        super::super::types::builders::IntelligentTieringConfigurationBuilder::default()
     }
 }

@@ -46,9 +46,9 @@
 #[non_exhaustive]
 pub struct IntelligentTieringConfigurationBuilder {
     pub(crate) id: ::std::option::Option<::std::string::String>,
-    pub(crate) filter: ::std::option::Option<crate::types::IntelligentTieringFilter>,
-    pub(crate) status: ::std::option::Option<crate::types::IntelligentTieringStatus>,
-    pub(crate) tierings: ::std::option::Option<::std::vec::Vec<crate::types::Tiering>>,
+    pub(crate) filter: ::std::option::Option<super::super::types::IntelligentTieringFilter>,
+    pub(crate) status: ::std::option::Option<super::super::types::IntelligentTieringStatus>,
+    pub(crate) tierings: ::std::option::Option<::std::vec::Vec<super::super::types::Tiering>>,
 }
 impl IntelligentTieringConfigurationBuilder {
     /// <p>The ID used to identify the S3 Intelligent-Tiering configuration.</p>
@@ -67,32 +67,32 @@
         &self.id
     }
     /// <p>Specifies a bucket filter. The configuration only includes objects that meet the filter's criteria.</p>
-    pub fn filter(mut self, input: crate::types::IntelligentTieringFilter) -> Self {
+    pub fn filter(mut self, input: super::super::types::IntelligentTieringFilter) -> Self {
         self.filter = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies a bucket filter. The configuration only includes objects that meet the filter's criteria.</p>
-    pub fn set_filter(mut self, input: ::std::option::Option<crate::types::IntelligentTieringFilter>) -> Self {
+    pub fn set_filter(mut self, input: ::std::option::Option<super::super::types::IntelligentTieringFilter>) -> Self {
         self.filter = input;
         self
     }
     /// <p>Specifies a bucket filter. The configuration only includes objects that meet the filter's criteria.</p>
-    pub fn get_filter(&self) -> &::std::option::Option<crate::types::IntelligentTieringFilter> {
+    pub fn get_filter(&self) -> &::std::option::Option<super::super::types::IntelligentTieringFilter> {
         &self.filter
     }
     /// <p>Specifies the status of the configuration.</p>
     /// This field is required.
-    pub fn status(mut self, input: crate::types::IntelligentTieringStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::IntelligentTieringStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies the status of the configuration.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::IntelligentTieringStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::IntelligentTieringStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>Specifies the status of the configuration.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::IntelligentTieringStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::IntelligentTieringStatus> {
         &self.status
     }
     /// Appends an item to `tierings`.
@@ -100,7 +100,7 @@
     /// To override the contents of this collection use [`set_tierings`](Self::set_tierings).
     ///
     /// <p>Specifies the S3 Intelligent-Tiering storage class tier of the configuration.</p>
-    pub fn tierings(mut self, input: crate::types::Tiering) -> Self {
+    pub fn tierings(mut self, input: super::super::types::Tiering) -> Self {
         let mut v = self.tierings.unwrap_or_default();
         v.push(input);
         self.tierings = ::std::option::Option::Some(v);
@@ -107,12 +107,12 @@
         self
     }
     /// <p>Specifies the S3 Intelligent-Tiering storage class tier of the configuration.</p>
-    pub fn set_tierings(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::Tiering>>) -> Self {
+    pub fn set_tierings(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::Tiering>>) -> Self {
         self.tierings = input;
         self
     }
     /// <p>Specifies the S3 Intelligent-Tiering storage class tier of the configuration.</p>
-    pub fn get_tierings(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::Tiering>> {
+    pub fn get_tierings(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::Tiering>> {
         &self.tierings
     }
     /// Consumes the builder and constructs a [`IntelligentTieringConfiguration`](crate::types::IntelligentTieringConfiguration).
@@ -120,8 +120,8 @@
     /// - [`id`](crate::types::builders::IntelligentTieringConfigurationBuilder::id)
     /// - [`status`](crate::types::builders::IntelligentTieringConfigurationBuilder::status)
     /// - [`tierings`](crate::types::builders::IntelligentTieringConfigurationBuilder::tierings)
-    pub fn build(self) -> ::std::result::Result<crate::types::IntelligentTieringConfiguration, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::IntelligentTieringConfiguration {
+    pub fn build(self) -> ::std::result::Result<super::super::types::IntelligentTieringConfiguration, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::IntelligentTieringConfiguration {
             id: self.id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "id",
