--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl InventoryFilter {
     /// Creates a new builder-style object to manufacture [`InventoryFilter`](crate::types::InventoryFilter).
-    pub fn builder() -> crate::types::builders::InventoryFilterBuilder {
-        crate::types::builders::InventoryFilterBuilder::default()
+    pub fn builder() -> super::super::types::builders::InventoryFilterBuilder {
+        super::super::types::builders::InventoryFilterBuilder::default()
     }
 }

@@ -46,8 +46,8 @@
     /// Consumes the builder and constructs a [`InventoryFilter`](crate::types::InventoryFilter).
     /// This method will fail if any of the following fields are not set:
     /// - [`prefix`](crate::types::builders::InventoryFilterBuilder::prefix)
-    pub fn build(self) -> ::std::result::Result<crate::types::InventoryFilter, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::InventoryFilter {
+    pub fn build(self) -> ::std::result::Result<super::super::types::InventoryFilter, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::InventoryFilter {
             prefix: self.prefix.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "prefix",
