--- original
+++ modified
@@ -31,8 +31,8 @@
 }
 impl NoncurrentVersionExpiration {
     /// Creates a new builder-style object to manufacture [`NoncurrentVersionExpiration`](crate::types::NoncurrentVersionExpiration).
-    pub fn builder() -> crate::types::builders::NoncurrentVersionExpirationBuilder {
-        crate::types::builders::NoncurrentVersionExpirationBuilder::default()
+    pub fn builder() -> super::super::types::builders::NoncurrentVersionExpirationBuilder {
+        super::super::types::builders::NoncurrentVersionExpirationBuilder::default()
     }
 }

@@ -85,8 +85,8 @@
         &self.newer_noncurrent_versions
     }
     /// Consumes the builder and constructs a [`NoncurrentVersionExpiration`](crate::types::NoncurrentVersionExpiration).
-    pub fn build(self) -> crate::types::NoncurrentVersionExpiration {
-        crate::types::NoncurrentVersionExpiration {
+    pub fn build(self) -> super::super::types::NoncurrentVersionExpiration {
+        super::super::types::NoncurrentVersionExpiration {
             noncurrent_days: self.noncurrent_days,
             newer_noncurrent_versions: self.newer_noncurrent_versions,
         }
