--- original
+++ modified
@@ -21,23 +21,23 @@
     /// </note>
     pub e_tag: ::std::option::Option<::std::string::String>,
     /// <p>The algorithm that was used to create a checksum of the object.</p>
-    pub checksum_algorithm: ::std::option::Option<::std::vec::Vec<crate::types::ChecksumAlgorithm>>,
+    pub checksum_algorithm: ::std::option::Option<::std::vec::Vec<super::super::types::ChecksumAlgorithm>>,
     /// <p>The checksum type that is used to calculate the object’s checksum value. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub checksum_type: ::std::option::Option<crate::types::ChecksumType>,
+    pub checksum_type: ::std::option::Option<super::super::types::ChecksumType>,
     /// <p>Size in bytes of the object</p>
     pub size: ::std::option::Option<i64>,
     /// <p>The class of storage used to store the object.</p><note>
     /// <p><b>Directory buckets</b> - Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub storage_class: ::std::option::Option<crate::types::ObjectStorageClass>,
+    pub storage_class: ::std::option::Option<super::super::types::ObjectStorageClass>,
     /// <p>The owner of the object</p><note>
     /// <p><b>Directory buckets</b> - The bucket owner is returned as the object owner.</p>
     /// </note>
-    pub owner: ::std::option::Option<crate::types::Owner>,
+    pub owner: ::std::option::Option<super::super::types::Owner>,
     /// <p>Specifies the restoration status of an object. Objects in certain storage classes must be restored before they can be retrieved. For more information about these storage classes and how to work with archived objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/archived-objects.html"> Working with archived objects</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub restore_status: ::std::option::Option<crate::types::RestoreStatus>,
+    pub restore_status: ::std::option::Option<super::super::types::RestoreStatus>,
 }
 impl Object {
     /// <p>The name that you assign to an object. You use the object key to retrieve the object.</p>
@@ -65,11 +65,11 @@
     /// <p>The algorithm that was used to create a checksum of the object.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.checksum_algorithm.is_none()`.
-    pub fn checksum_algorithm(&self) -> &[crate::types::ChecksumAlgorithm] {
+    pub fn checksum_algorithm(&self) -> &[super::super::types::ChecksumAlgorithm] {
         self.checksum_algorithm.as_deref().unwrap_or_default()
     }
     /// <p>The checksum type that is used to calculate the object’s checksum value. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn checksum_type(&self) -> ::std::option::Option<&crate::types::ChecksumType> {
+    pub fn checksum_type(&self) -> ::std::option::Option<&super::super::types::ChecksumType> {
         self.checksum_type.as_ref()
     }
     /// <p>Size in bytes of the object</p>
@@ -79,26 +79,26 @@
     /// <p>The class of storage used to store the object.</p><note>
     /// <p><b>Directory buckets</b> - Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn storage_class(&self) -> ::std::option::Option<&crate::types::ObjectStorageClass> {
+    pub fn storage_class(&self) -> ::std::option::Option<&super::super::types::ObjectStorageClass> {
         self.storage_class.as_ref()
     }
     /// <p>The owner of the object</p><note>
     /// <p><b>Directory buckets</b> - The bucket owner is returned as the object owner.</p>
     /// </note>
-    pub fn owner(&self) -> ::std::option::Option<&crate::types::Owner> {
+    pub fn owner(&self) -> ::std::option::Option<&super::super::types::Owner> {
         self.owner.as_ref()
     }
     /// <p>Specifies the restoration status of an object. Objects in certain storage classes must be restored before they can be retrieved. For more information about these storage classes and how to work with archived objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/archived-objects.html"> Working with archived objects</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn restore_status(&self) -> ::std::option::Option<&crate::types::RestoreStatus> {
+    pub fn restore_status(&self) -> ::std::option::Option<&super::super::types::RestoreStatus> {
         self.restore_status.as_ref()
     }
 }
 impl Object {
     /// Creates a new builder-style object to manufacture [`Object`](crate::types::Object).
-    pub fn builder() -> crate::types::builders::ObjectBuilder {
-        crate::types::builders::ObjectBuilder::default()
+    pub fn builder() -> super::super::types::builders::ObjectBuilder {
+        super::super::types::builders::ObjectBuilder::default()
     }
 }

@@ -109,12 +109,12 @@
     pub(crate) key: ::std::option::Option<::std::string::String>,
     pub(crate) last_modified: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) e_tag: ::std::option::Option<::std::string::String>,
-    pub(crate) checksum_algorithm: ::std::option::Option<::std::vec::Vec<crate::types::ChecksumAlgorithm>>,
-    pub(crate) checksum_type: ::std::option::Option<crate::types::ChecksumType>,
+    pub(crate) checksum_algorithm: ::std::option::Option<::std::vec::Vec<super::super::types::ChecksumAlgorithm>>,
+    pub(crate) checksum_type: ::std::option::Option<super::super::types::ChecksumType>,
     pub(crate) size: ::std::option::Option<i64>,
-    pub(crate) storage_class: ::std::option::Option<crate::types::ObjectStorageClass>,
-    pub(crate) owner: ::std::option::Option<crate::types::Owner>,
-    pub(crate) restore_status: ::std::option::Option<crate::types::RestoreStatus>,
+    pub(crate) storage_class: ::std::option::Option<super::super::types::ObjectStorageClass>,
+    pub(crate) owner: ::std::option::Option<super::super::types::Owner>,
+    pub(crate) restore_status: ::std::option::Option<super::super::types::RestoreStatus>,
 }
 impl ObjectBuilder {
     /// <p>The name that you assign to an object. You use the object key to retrieve the object.</p>
@@ -194,7 +194,7 @@
     /// To override the contents of this collection use [`set_checksum_algorithm`](Self::set_checksum_algorithm).
     ///
     /// <p>The algorithm that was used to create a checksum of the object.</p>
-    pub fn checksum_algorithm(mut self, input: crate::types::ChecksumAlgorithm) -> Self {
+    pub fn checksum_algorithm(mut self, input: super::super::types::ChecksumAlgorithm) -> Self {
         let mut v = self.checksum_algorithm.unwrap_or_default();
         v.push(input);
         self.checksum_algorithm = ::std::option::Option::Some(v);
@@ -201,26 +201,26 @@
         self
     }
     /// <p>The algorithm that was used to create a checksum of the object.</p>
-    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ChecksumAlgorithm>>) -> Self {
+    pub fn set_checksum_algorithm(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::types::ChecksumAlgorithm>>) -> Self {
         self.checksum_algorithm = input;
         self
     }
     /// <p>The algorithm that was used to create a checksum of the object.</p>
-    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ChecksumAlgorithm>> {
+    pub fn get_checksum_algorithm(&self) -> &::std::option::Option<::std::vec::Vec<super::super::types::ChecksumAlgorithm>> {
         &self.checksum_algorithm
     }
     /// <p>The checksum type that is used to calculate the object’s checksum value. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn checksum_type(mut self, input: crate::types::ChecksumType) -> Self {
+    pub fn checksum_type(mut self, input: super::super::types::ChecksumType) -> Self {
         self.checksum_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The checksum type that is used to calculate the object’s checksum value. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn set_checksum_type(mut self, input: ::std::option::Option<crate::types::ChecksumType>) -> Self {
+    pub fn set_checksum_type(mut self, input: ::std::option::Option<super::super::types::ChecksumType>) -> Self {
         self.checksum_type = input;
         self
     }
     /// <p>The checksum type that is used to calculate the object’s checksum value. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html">Checking object integrity</a> in the <i>Amazon S3 User Guide</i>.</p>
-    pub fn get_checksum_type(&self) -> &::std::option::Option<crate::types::ChecksumType> {
+    pub fn get_checksum_type(&self) -> &::std::option::Option<super::super::types::ChecksumType> {
         &self.checksum_type
     }
     /// <p>Size in bytes of the object</p>
@@ -240,7 +240,7 @@
     /// <p>The class of storage used to store the object.</p><note>
     /// <p><b>Directory buckets</b> - Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn storage_class(mut self, input: crate::types::ObjectStorageClass) -> Self {
+    pub fn storage_class(mut self, input: super::super::types::ObjectStorageClass) -> Self {
         self.storage_class = ::std::option::Option::Some(input);
         self
     }
@@ -247,7 +247,7 @@
     /// <p>The class of storage used to store the object.</p><note>
     /// <p><b>Directory buckets</b> - Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn set_storage_class(mut self, input: ::std::option::Option<crate::types::ObjectStorageClass>) -> Self {
+    pub fn set_storage_class(mut self, input: ::std::option::Option<super::super::types::ObjectStorageClass>) -> Self {
         self.storage_class = input;
         self
     }
@@ -254,13 +254,13 @@
     /// <p>The class of storage used to store the object.</p><note>
     /// <p><b>Directory buckets</b> - Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn get_storage_class(&self) -> &::std::option::Option<crate::types::ObjectStorageClass> {
+    pub fn get_storage_class(&self) -> &::std::option::Option<super::super::types::ObjectStorageClass> {
         &self.storage_class
     }
     /// <p>The owner of the object</p><note>
     /// <p><b>Directory buckets</b> - The bucket owner is returned as the object owner.</p>
     /// </note>
-    pub fn owner(mut self, input: crate::types::Owner) -> Self {
+    pub fn owner(mut self, input: super::super::types::Owner) -> Self {
         self.owner = ::std::option::Option::Some(input);
         self
     }
@@ -267,7 +267,7 @@
     /// <p>The owner of the object</p><note>
     /// <p><b>Directory buckets</b> - The bucket owner is returned as the object owner.</p>
     /// </note>
-    pub fn set_owner(mut self, input: ::std::option::Option<crate::types::Owner>) -> Self {
+    pub fn set_owner(mut self, input: ::std::option::Option<super::super::types::Owner>) -> Self {
         self.owner = input;
         self
     }
@@ -274,13 +274,13 @@
     /// <p>The owner of the object</p><note>
     /// <p><b>Directory buckets</b> - The bucket owner is returned as the object owner.</p>
     /// </note>
-    pub fn get_owner(&self) -> &::std::option::Option<crate::types::Owner> {
+    pub fn get_owner(&self) -> &::std::option::Option<super::super::types::Owner> {
         &self.owner
     }
     /// <p>Specifies the restoration status of an object. Objects in certain storage classes must be restored before they can be retrieved. For more information about these storage classes and how to work with archived objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/archived-objects.html"> Working with archived objects</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn restore_status(mut self, input: crate::types::RestoreStatus) -> Self {
+    pub fn restore_status(mut self, input: super::super::types::RestoreStatus) -> Self {
         self.restore_status = ::std::option::Option::Some(input);
         self
     }
@@ -287,7 +287,7 @@
     /// <p>Specifies the restoration status of an object. Objects in certain storage classes must be restored before they can be retrieved. For more information about these storage classes and how to work with archived objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/archived-objects.html"> Working with archived objects</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn set_restore_status(mut self, input: ::std::option::Option<crate::types::RestoreStatus>) -> Self {
+    pub fn set_restore_status(mut self, input: ::std::option::Option<super::super::types::RestoreStatus>) -> Self {
         self.restore_status = input;
         self
     }
@@ -294,12 +294,12 @@
     /// <p>Specifies the restoration status of an object. Objects in certain storage classes must be restored before they can be retrieved. For more information about these storage classes and how to work with archived objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/archived-objects.html"> Working with archived objects</a> in the <i>Amazon S3 User Guide</i>.</p><note>
     /// <p>This functionality is not supported for directory buckets. Directory buckets only support <code>EXPRESS_ONEZONE</code> (the S3 Express One Zone storage class) in Availability Zones and <code>ONEZONE_IA</code> (the S3 One Zone-Infrequent Access storage class) in Dedicated Local Zones.</p>
     /// </note>
-    pub fn get_restore_status(&self) -> &::std::option::Option<crate::types::RestoreStatus> {
+    pub fn get_restore_status(&self) -> &::std::option::Option<super::super::types::RestoreStatus> {
         &self.restore_status
     }
     /// Consumes the builder and constructs a [`Object`](crate::types::Object).
-    pub fn build(self) -> crate::types::Object {
-        crate::types::Object {
+    pub fn build(self) -> super::super::types::Object {
+        super::super::types::Object {
             key: self.key,
             last_modified: self.last_modified,
             e_tag: self.e_tag,
