--- original
+++ modified
@@ -5,18 +5,18 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ObjectLockLegalHold {
     /// <p>Indicates whether the specified object has a legal hold in place.</p>
-    pub status: ::std::option::Option<crate::types::ObjectLockLegalHoldStatus>,
+    pub status: ::std::option::Option<super::types::ObjectLockLegalHoldStatus>,
 }
 impl ObjectLockLegalHold {
     /// <p>Indicates whether the specified object has a legal hold in place.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::ObjectLockLegalHoldStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::types::ObjectLockLegalHoldStatus> {
         self.status.as_ref()
     }
 }
 impl ObjectLockLegalHold {
     /// Creates a new builder-style object to manufacture [`ObjectLockLegalHold`](crate::types::ObjectLockLegalHold).
-    pub fn builder() -> crate::types::builders::ObjectLockLegalHoldBuilder {
-        crate::types::builders::ObjectLockLegalHoldBuilder::default()
+    pub fn builder() -> super::types::builders::ObjectLockLegalHoldBuilder {
+        super::types::builders::ObjectLockLegalHoldBuilder::default()
     }
 }

@@ -24,25 +24,25 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ObjectLockLegalHoldBuilder {
-    pub(crate) status: ::std::option::Option<crate::types::ObjectLockLegalHoldStatus>,
+    pub(crate) status: ::std::option::Option<super::types::ObjectLockLegalHoldStatus>,
 }
 impl ObjectLockLegalHoldBuilder {
     /// <p>Indicates whether the specified object has a legal hold in place.</p>
-    pub fn status(mut self, input: crate::types::ObjectLockLegalHoldStatus) -> Self {
+    pub fn status(mut self, input: super::types::ObjectLockLegalHoldStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Indicates whether the specified object has a legal hold in place.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::ObjectLockLegalHoldStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::ObjectLockLegalHoldStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>Indicates whether the specified object has a legal hold in place.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::ObjectLockLegalHoldStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::ObjectLockLegalHoldStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`ObjectLockLegalHold`](crate::types::ObjectLockLegalHold).
-    pub fn build(self) -> crate::types::ObjectLockLegalHold {
-        crate::types::ObjectLockLegalHold { status: self.status }
+    pub fn build(self) -> super::types::ObjectLockLegalHold {
+        super::types::ObjectLockLegalHold { status: self.status }
     }
 }
