--- original
+++ modified
@@ -9,20 +9,20 @@
     /// <p>Specifies the partition date source for the partitioned prefix. <code>PartitionDateSource</code> can be <code>EventTime</code> or <code>DeliveryTime</code>.</p>
     /// <p>For <code>DeliveryTime</code>, the time in the log file names corresponds to the delivery time for the log files.</p>
     /// <p>For <code>EventTime</code>, The logs delivered are for a specific day only. The year, month, and day correspond to the day on which the event occurred, and the hour, minutes and seconds are set to 00 in the key.</p>
-    pub partition_date_source: ::std::option::Option<crate::types::PartitionDateSource>,
+    pub partition_date_source: ::std::option::Option<super::super::types::PartitionDateSource>,
 }
 impl PartitionedPrefix {
     /// <p>Specifies the partition date source for the partitioned prefix. <code>PartitionDateSource</code> can be <code>EventTime</code> or <code>DeliveryTime</code>.</p>
     /// <p>For <code>DeliveryTime</code>, the time in the log file names corresponds to the delivery time for the log files.</p>
     /// <p>For <code>EventTime</code>, The logs delivered are for a specific day only. The year, month, and day correspond to the day on which the event occurred, and the hour, minutes and seconds are set to 00 in the key.</p>
-    pub fn partition_date_source(&self) -> ::std::option::Option<&crate::types::PartitionDateSource> {
+    pub fn partition_date_source(&self) -> ::std::option::Option<&super::super::types::PartitionDateSource> {
         self.partition_date_source.as_ref()
     }
 }
 impl PartitionedPrefix {
     /// Creates a new builder-style object to manufacture [`PartitionedPrefix`](crate::types::PartitionedPrefix).
-    pub fn builder() -> crate::types::builders::PartitionedPrefixBuilder {
-        crate::types::builders::PartitionedPrefixBuilder::default()
+    pub fn builder() -> super::super::types::builders::PartitionedPrefixBuilder {
+        super::super::types::builders::PartitionedPrefixBuilder::default()
     }
 }

@@ -30,13 +30,13 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct PartitionedPrefixBuilder {
-    pub(crate) partition_date_source: ::std::option::Option<crate::types::PartitionDateSource>,
+    pub(crate) partition_date_source: ::std::option::Option<super::super::types::PartitionDateSource>,
 }
 impl PartitionedPrefixBuilder {
     /// <p>Specifies the partition date source for the partitioned prefix. <code>PartitionDateSource</code> can be <code>EventTime</code> or <code>DeliveryTime</code>.</p>
     /// <p>For <code>DeliveryTime</code>, the time in the log file names corresponds to the delivery time for the log files.</p>
     /// <p>For <code>EventTime</code>, The logs delivered are for a specific day only. The year, month, and day correspond to the day on which the event occurred, and the hour, minutes and seconds are set to 00 in the key.</p>
-    pub fn partition_date_source(mut self, input: crate::types::PartitionDateSource) -> Self {
+    pub fn partition_date_source(mut self, input: super::super::types::PartitionDateSource) -> Self {
         self.partition_date_source = ::std::option::Option::Some(input);
         self
     }
@@ -43,7 +43,7 @@
     /// <p>Specifies the partition date source for the partitioned prefix. <code>PartitionDateSource</code> can be <code>EventTime</code> or <code>DeliveryTime</code>.</p>
     /// <p>For <code>DeliveryTime</code>, the time in the log file names corresponds to the delivery time for the log files.</p>
     /// <p>For <code>EventTime</code>, The logs delivered are for a specific day only. The year, month, and day correspond to the day on which the event occurred, and the hour, minutes and seconds are set to 00 in the key.</p>
-    pub fn set_partition_date_source(mut self, input: ::std::option::Option<crate::types::PartitionDateSource>) -> Self {
+    pub fn set_partition_date_source(mut self, input: ::std::option::Option<super::super::types::PartitionDateSource>) -> Self {
         self.partition_date_source = input;
         self
     }
@@ -50,12 +50,12 @@
     /// <p>Specifies the partition date source for the partitioned prefix. <code>PartitionDateSource</code> can be <code>EventTime</code> or <code>DeliveryTime</code>.</p>
     /// <p>For <code>DeliveryTime</code>, the time in the log file names corresponds to the delivery time for the log files.</p>
     /// <p>For <code>EventTime</code>, The logs delivered are for a specific day only. The year, month, and day correspond to the day on which the event occurred, and the hour, minutes and seconds are set to 00 in the key.</p>
-    pub fn get_partition_date_source(&self) -> &::std::option::Option<crate::types::PartitionDateSource> {
+    pub fn get_partition_date_source(&self) -> &::std::option::Option<super::super::types::PartitionDateSource> {
         &self.partition_date_source
     }
     /// Consumes the builder and constructs a [`PartitionedPrefix`](crate::types::PartitionedPrefix).
-    pub fn build(self) -> crate::types::PartitionedPrefix {
-        crate::types::PartitionedPrefix {
+    pub fn build(self) -> super::super::types::PartitionedPrefix {
+        super::super::types::PartitionedPrefix {
             partition_date_source: self.partition_date_source,
         }
     }
