--- original
+++ modified
@@ -15,22 +15,22 @@
     #[deprecated]
     pub prefix: ::std::option::Option<::std::string::String>,
     /// <p>A filter that identifies the subset of objects to which the replication rule applies. A <code>Filter</code> must specify exactly one <code>Prefix</code>, <code>Tag</code>, or an <code>And</code> child element.</p>
-    pub filter: ::std::option::Option<crate::types::ReplicationRuleFilter>,
+    pub filter: ::std::option::Option<super::types::ReplicationRuleFilter>,
     /// <p>Specifies whether the rule is enabled.</p>
-    pub status: crate::types::ReplicationRuleStatus,
+    pub status: super::types::ReplicationRuleStatus,
     /// <p>A container that describes additional filters for identifying the source objects that you want to replicate. You can choose to enable or disable the replication of these objects. Currently, Amazon S3 supports only the filter that you can specify for objects created with server-side encryption using a customer managed key stored in Amazon Web Services Key Management Service (SSE-KMS).</p>
-    pub source_selection_criteria: ::std::option::Option<crate::types::SourceSelectionCriteria>,
+    pub source_selection_criteria: ::std::option::Option<super::types::SourceSelectionCriteria>,
     /// <p>Optional configuration to replicate existing source bucket objects.</p><note>
     /// <p>This parameter is no longer supported. To replicate existing objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-batch-replication-batch.html">Replicating existing objects with S3 Batch Replication</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// </note>
-    pub existing_object_replication: ::std::option::Option<crate::types::ExistingObjectReplication>,
+    pub existing_object_replication: ::std::option::Option<super::types::ExistingObjectReplication>,
     /// <p>A container for information about the replication destination and its configurations including enabling the S3 Replication Time Control (S3 RTC).</p>
-    pub destination: ::std::option::Option<crate::types::Destination>,
+    pub destination: ::std::option::Option<super::types::Destination>,
     /// <p>Specifies whether Amazon S3 replicates delete markers. If you specify a <code>Filter</code> in your replication configuration, you must also include a <code>DeleteMarkerReplication</code> element. If your <code>Filter</code> includes a <code>Tag</code> element, the <code>DeleteMarkerReplication</code> <code>Status</code> must be set to Disabled, because Amazon S3 does not support replicating delete markers for tag-based rules. For an example configuration, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-config-min-rule-config">Basic Rule Configuration</a>.</p>
     /// <p>For more information about delete marker replication, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/delete-marker-replication.html">Basic Rule Configuration</a>.</p><note>
     /// <p>If you are using an earlier version of the replication configuration, Amazon S3 handles replication of delete markers differently. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-backward-compat-considerations">Backward Compatibility</a>.</p>
     /// </note>
-    pub delete_marker_replication: ::std::option::Option<crate::types::DeleteMarkerReplication>,
+    pub delete_marker_replication: ::std::option::Option<super::types::DeleteMarkerReplication>,
 }
 impl ReplicationRule {
     /// <p>A unique identifier for the rule. The maximum value is 255 characters.</p>
@@ -50,25 +50,25 @@
         self.prefix.as_deref()
     }
     /// <p>A filter that identifies the subset of objects to which the replication rule applies. A <code>Filter</code> must specify exactly one <code>Prefix</code>, <code>Tag</code>, or an <code>And</code> child element.</p>
-    pub fn filter(&self) -> ::std::option::Option<&crate::types::ReplicationRuleFilter> {
+    pub fn filter(&self) -> ::std::option::Option<&super::types::ReplicationRuleFilter> {
         self.filter.as_ref()
     }
     /// <p>Specifies whether the rule is enabled.</p>
-    pub fn status(&self) -> &crate::types::ReplicationRuleStatus {
+    pub fn status(&self) -> &super::types::ReplicationRuleStatus {
         &self.status
     }
     /// <p>A container that describes additional filters for identifying the source objects that you want to replicate. You can choose to enable or disable the replication of these objects. Currently, Amazon S3 supports only the filter that you can specify for objects created with server-side encryption using a customer managed key stored in Amazon Web Services Key Management Service (SSE-KMS).</p>
-    pub fn source_selection_criteria(&self) -> ::std::option::Option<&crate::types::SourceSelectionCriteria> {
+    pub fn source_selection_criteria(&self) -> ::std::option::Option<&super::types::SourceSelectionCriteria> {
         self.source_selection_criteria.as_ref()
     }
     /// <p>Optional configuration to replicate existing source bucket objects.</p><note>
     /// <p>This parameter is no longer supported. To replicate existing objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-batch-replication-batch.html">Replicating existing objects with S3 Batch Replication</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// </note>
-    pub fn existing_object_replication(&self) -> ::std::option::Option<&crate::types::ExistingObjectReplication> {
+    pub fn existing_object_replication(&self) -> ::std::option::Option<&super::types::ExistingObjectReplication> {
         self.existing_object_replication.as_ref()
     }
     /// <p>A container for information about the replication destination and its configurations including enabling the S3 Replication Time Control (S3 RTC).</p>
-    pub fn destination(&self) -> ::std::option::Option<&crate::types::Destination> {
+    pub fn destination(&self) -> ::std::option::Option<&super::types::Destination> {
         self.destination.as_ref()
     }
     /// <p>Specifies whether Amazon S3 replicates delete markers. If you specify a <code>Filter</code> in your replication configuration, you must also include a <code>DeleteMarkerReplication</code> element. If your <code>Filter</code> includes a <code>Tag</code> element, the <code>DeleteMarkerReplication</code> <code>Status</code> must be set to Disabled, because Amazon S3 does not support replicating delete markers for tag-based rules. For an example configuration, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-config-min-rule-config">Basic Rule Configuration</a>.</p>
@@ -75,14 +75,14 @@
     /// <p>For more information about delete marker replication, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/delete-marker-replication.html">Basic Rule Configuration</a>.</p><note>
     /// <p>If you are using an earlier version of the replication configuration, Amazon S3 handles replication of delete markers differently. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-backward-compat-considerations">Backward Compatibility</a>.</p>
     /// </note>
-    pub fn delete_marker_replication(&self) -> ::std::option::Option<&crate::types::DeleteMarkerReplication> {
+    pub fn delete_marker_replication(&self) -> ::std::option::Option<&super::types::DeleteMarkerReplication> {
         self.delete_marker_replication.as_ref()
     }
 }
 impl ReplicationRule {
     /// Creates a new builder-style object to manufacture [`ReplicationRule`](crate::types::ReplicationRule).
-    pub fn builder() -> crate::types::builders::ReplicationRuleBuilder {
-        crate::types::builders::ReplicationRuleBuilder::default()
+    pub fn builder() -> super::types::builders::ReplicationRuleBuilder {
+        super::types::builders::ReplicationRuleBuilder::default()
     }
 }

@@ -93,12 +93,12 @@
     pub(crate) id: ::std::option::Option<::std::string::String>,
     pub(crate) priority: ::std::option::Option<i32>,
     pub(crate) prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) filter: ::std::option::Option<crate::types::ReplicationRuleFilter>,
-    pub(crate) status: ::std::option::Option<crate::types::ReplicationRuleStatus>,
-    pub(crate) source_selection_criteria: ::std::option::Option<crate::types::SourceSelectionCriteria>,
-    pub(crate) existing_object_replication: ::std::option::Option<crate::types::ExistingObjectReplication>,
-    pub(crate) destination: ::std::option::Option<crate::types::Destination>,
-    pub(crate) delete_marker_replication: ::std::option::Option<crate::types::DeleteMarkerReplication>,
+    pub(crate) filter: ::std::option::Option<super::types::ReplicationRuleFilter>,
+    pub(crate) status: ::std::option::Option<super::types::ReplicationRuleStatus>,
+    pub(crate) source_selection_criteria: ::std::option::Option<super::types::SourceSelectionCriteria>,
+    pub(crate) existing_object_replication: ::std::option::Option<super::types::ExistingObjectReplication>,
+    pub(crate) destination: ::std::option::Option<super::types::Destination>,
+    pub(crate) delete_marker_replication: ::std::option::Option<super::types::DeleteMarkerReplication>,
 }
 impl ReplicationRuleBuilder {
     /// <p>A unique identifier for the rule. The maximum value is 255 characters.</p>
@@ -156,52 +156,52 @@
         &self.prefix
     }
     /// <p>A filter that identifies the subset of objects to which the replication rule applies. A <code>Filter</code> must specify exactly one <code>Prefix</code>, <code>Tag</code>, or an <code>And</code> child element.</p>
-    pub fn filter(mut self, input: crate::types::ReplicationRuleFilter) -> Self {
+    pub fn filter(mut self, input: super::types::ReplicationRuleFilter) -> Self {
         self.filter = ::std::option::Option::Some(input);
         self
     }
     /// <p>A filter that identifies the subset of objects to which the replication rule applies. A <code>Filter</code> must specify exactly one <code>Prefix</code>, <code>Tag</code>, or an <code>And</code> child element.</p>
-    pub fn set_filter(mut self, input: ::std::option::Option<crate::types::ReplicationRuleFilter>) -> Self {
+    pub fn set_filter(mut self, input: ::std::option::Option<super::types::ReplicationRuleFilter>) -> Self {
         self.filter = input;
         self
     }
     /// <p>A filter that identifies the subset of objects to which the replication rule applies. A <code>Filter</code> must specify exactly one <code>Prefix</code>, <code>Tag</code>, or an <code>And</code> child element.</p>
-    pub fn get_filter(&self) -> &::std::option::Option<crate::types::ReplicationRuleFilter> {
+    pub fn get_filter(&self) -> &::std::option::Option<super::types::ReplicationRuleFilter> {
         &self.filter
     }
     /// <p>Specifies whether the rule is enabled.</p>
     /// This field is required.
-    pub fn status(mut self, input: crate::types::ReplicationRuleStatus) -> Self {
+    pub fn status(mut self, input: super::types::ReplicationRuleStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether the rule is enabled.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::ReplicationRuleStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::types::ReplicationRuleStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>Specifies whether the rule is enabled.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::ReplicationRuleStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::types::ReplicationRuleStatus> {
         &self.status
     }
     /// <p>A container that describes additional filters for identifying the source objects that you want to replicate. You can choose to enable or disable the replication of these objects. Currently, Amazon S3 supports only the filter that you can specify for objects created with server-side encryption using a customer managed key stored in Amazon Web Services Key Management Service (SSE-KMS).</p>
-    pub fn source_selection_criteria(mut self, input: crate::types::SourceSelectionCriteria) -> Self {
+    pub fn source_selection_criteria(mut self, input: super::types::SourceSelectionCriteria) -> Self {
         self.source_selection_criteria = ::std::option::Option::Some(input);
         self
     }
     /// <p>A container that describes additional filters for identifying the source objects that you want to replicate. You can choose to enable or disable the replication of these objects. Currently, Amazon S3 supports only the filter that you can specify for objects created with server-side encryption using a customer managed key stored in Amazon Web Services Key Management Service (SSE-KMS).</p>
-    pub fn set_source_selection_criteria(mut self, input: ::std::option::Option<crate::types::SourceSelectionCriteria>) -> Self {
+    pub fn set_source_selection_criteria(mut self, input: ::std::option::Option<super::types::SourceSelectionCriteria>) -> Self {
         self.source_selection_criteria = input;
         self
     }
     /// <p>A container that describes additional filters for identifying the source objects that you want to replicate. You can choose to enable or disable the replication of these objects. Currently, Amazon S3 supports only the filter that you can specify for objects created with server-side encryption using a customer managed key stored in Amazon Web Services Key Management Service (SSE-KMS).</p>
-    pub fn get_source_selection_criteria(&self) -> &::std::option::Option<crate::types::SourceSelectionCriteria> {
+    pub fn get_source_selection_criteria(&self) -> &::std::option::Option<super::types::SourceSelectionCriteria> {
         &self.source_selection_criteria
     }
     /// <p>Optional configuration to replicate existing source bucket objects.</p><note>
     /// <p>This parameter is no longer supported. To replicate existing objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-batch-replication-batch.html">Replicating existing objects with S3 Batch Replication</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// </note>
-    pub fn existing_object_replication(mut self, input: crate::types::ExistingObjectReplication) -> Self {
+    pub fn existing_object_replication(mut self, input: super::types::ExistingObjectReplication) -> Self {
         self.existing_object_replication = ::std::option::Option::Some(input);
         self
     }
@@ -208,7 +208,7 @@
     /// <p>Optional configuration to replicate existing source bucket objects.</p><note>
     /// <p>This parameter is no longer supported. To replicate existing objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-batch-replication-batch.html">Replicating existing objects with S3 Batch Replication</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// </note>
-    pub fn set_existing_object_replication(mut self, input: ::std::option::Option<crate::types::ExistingObjectReplication>) -> Self {
+    pub fn set_existing_object_replication(mut self, input: ::std::option::Option<super::types::ExistingObjectReplication>) -> Self {
         self.existing_object_replication = input;
         self
     }
@@ -215,22 +215,22 @@
     /// <p>Optional configuration to replicate existing source bucket objects.</p><note>
     /// <p>This parameter is no longer supported. To replicate existing objects, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-batch-replication-batch.html">Replicating existing objects with S3 Batch Replication</a> in the <i>Amazon S3 User Guide</i>.</p>
     /// </note>
-    pub fn get_existing_object_replication(&self) -> &::std::option::Option<crate::types::ExistingObjectReplication> {
+    pub fn get_existing_object_replication(&self) -> &::std::option::Option<super::types::ExistingObjectReplication> {
         &self.existing_object_replication
     }
     /// <p>A container for information about the replication destination and its configurations including enabling the S3 Replication Time Control (S3 RTC).</p>
     /// This field is required.
-    pub fn destination(mut self, input: crate::types::Destination) -> Self {
+    pub fn destination(mut self, input: super::types::Destination) -> Self {
         self.destination = ::std::option::Option::Some(input);
         self
     }
     /// <p>A container for information about the replication destination and its configurations including enabling the S3 Replication Time Control (S3 RTC).</p>
-    pub fn set_destination(mut self, input: ::std::option::Option<crate::types::Destination>) -> Self {
+    pub fn set_destination(mut self, input: ::std::option::Option<super::types::Destination>) -> Self {
         self.destination = input;
         self
     }
     /// <p>A container for information about the replication destination and its configurations including enabling the S3 Replication Time Control (S3 RTC).</p>
-    pub fn get_destination(&self) -> &::std::option::Option<crate::types::Destination> {
+    pub fn get_destination(&self) -> &::std::option::Option<super::types::Destination> {
         &self.destination
     }
     /// <p>Specifies whether Amazon S3 replicates delete markers. If you specify a <code>Filter</code> in your replication configuration, you must also include a <code>DeleteMarkerReplication</code> element. If your <code>Filter</code> includes a <code>Tag</code> element, the <code>DeleteMarkerReplication</code> <code>Status</code> must be set to Disabled, because Amazon S3 does not support replicating delete markers for tag-based rules. For an example configuration, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-config-min-rule-config">Basic Rule Configuration</a>.</p>
@@ -237,7 +237,7 @@
     /// <p>For more information about delete marker replication, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/delete-marker-replication.html">Basic Rule Configuration</a>.</p><note>
     /// <p>If you are using an earlier version of the replication configuration, Amazon S3 handles replication of delete markers differently. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-backward-compat-considerations">Backward Compatibility</a>.</p>
     /// </note>
-    pub fn delete_marker_replication(mut self, input: crate::types::DeleteMarkerReplication) -> Self {
+    pub fn delete_marker_replication(mut self, input: super::types::DeleteMarkerReplication) -> Self {
         self.delete_marker_replication = ::std::option::Option::Some(input);
         self
     }
@@ -245,7 +245,7 @@
     /// <p>For more information about delete marker replication, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/delete-marker-replication.html">Basic Rule Configuration</a>.</p><note>
     /// <p>If you are using an earlier version of the replication configuration, Amazon S3 handles replication of delete markers differently. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-backward-compat-considerations">Backward Compatibility</a>.</p>
     /// </note>
-    pub fn set_delete_marker_replication(mut self, input: ::std::option::Option<crate::types::DeleteMarkerReplication>) -> Self {
+    pub fn set_delete_marker_replication(mut self, input: ::std::option::Option<super::types::DeleteMarkerReplication>) -> Self {
         self.delete_marker_replication = input;
         self
     }
@@ -253,14 +253,14 @@
     /// <p>For more information about delete marker replication, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/delete-marker-replication.html">Basic Rule Configuration</a>.</p><note>
     /// <p>If you are using an earlier version of the replication configuration, Amazon S3 handles replication of delete markers differently. For more information, see <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html#replication-backward-compat-considerations">Backward Compatibility</a>.</p>
     /// </note>
-    pub fn get_delete_marker_replication(&self) -> &::std::option::Option<crate::types::DeleteMarkerReplication> {
+    pub fn get_delete_marker_replication(&self) -> &::std::option::Option<super::types::DeleteMarkerReplication> {
         &self.delete_marker_replication
     }
     /// Consumes the builder and constructs a [`ReplicationRule`](crate::types::ReplicationRule).
     /// This method will fail if any of the following fields are not set:
     /// - [`status`](crate::types::builders::ReplicationRuleBuilder::status)
-    pub fn build(self) -> ::std::result::Result<crate::types::ReplicationRule, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::ReplicationRule {
+    pub fn build(self) -> ::std::result::Result<super::types::ReplicationRule, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::ReplicationRule {
             id: self.id,
             priority: self.priority,
             prefix: self.prefix,
