--- original
+++ modified
@@ -10,7 +10,7 @@
     pub prefix: ::std::option::Option<::std::string::String>,
     /// <p>A container for specifying a tag key and value.</p>
     /// <p>The rule applies only to objects that have the tag in their tag set.</p>
-    pub tag: ::std::option::Option<crate::types::Tag>,
+    pub tag: ::std::option::Option<super::types::Tag>,
     /// <p>A container for specifying rule filters. The filters determine the subset of objects to which the rule applies. This element is required only if you specify more than one filter. For example:</p>
     /// <ul>
     /// <li>
@@ -18,7 +18,7 @@
     /// <li>
     /// <p>If you specify a filter based on multiple tags, wrap the <code>Tag</code> elements in an <code>And</code> tag.</p></li>
     /// </ul>
-    pub and: ::std::option::Option<crate::types::ReplicationRuleAndOperator>,
+    pub and: ::std::option::Option<super::types::ReplicationRuleAndOperator>,
 }
 impl ReplicationRuleFilter {
     /// <p>An object key name prefix that identifies the subset of objects to which the rule applies.</p><important>
@@ -29,7 +29,7 @@
     }
     /// <p>A container for specifying a tag key and value.</p>
     /// <p>The rule applies only to objects that have the tag in their tag set.</p>
-    pub fn tag(&self) -> ::std::option::Option<&crate::types::Tag> {
+    pub fn tag(&self) -> ::std::option::Option<&super::types::Tag> {
         self.tag.as_ref()
     }
     /// <p>A container for specifying rule filters. The filters determine the subset of objects to which the rule applies. This element is required only if you specify more than one filter. For example:</p>
@@ -39,14 +39,14 @@
     /// <li>
     /// <p>If you specify a filter based on multiple tags, wrap the <code>Tag</code> elements in an <code>And</code> tag.</p></li>
     /// </ul>
-    pub fn and(&self) -> ::std::option::Option<&crate::types::ReplicationRuleAndOperator> {
+    pub fn and(&self) -> ::std::option::Option<&super::types::ReplicationRuleAndOperator> {
         self.and.as_ref()
     }
 }
 impl ReplicationRuleFilter {
     /// Creates a new builder-style object to manufacture [`ReplicationRuleFilter`](crate::types::ReplicationRuleFilter).
-    pub fn builder() -> crate::types::builders::ReplicationRuleFilterBuilder {
-        crate::types::builders::ReplicationRuleFilterBuilder::default()
+    pub fn builder() -> super::types::builders::ReplicationRuleFilterBuilder {
+        super::types::builders::ReplicationRuleFilterBuilder::default()
     }
 }

@@ -55,8 +55,8 @@
 #[non_exhaustive]
 pub struct ReplicationRuleFilterBuilder {
     pub(crate) prefix: ::std::option::Option<::std::string::String>,
-    pub(crate) tag: ::std::option::Option<crate::types::Tag>,
-    pub(crate) and: ::std::option::Option<crate::types::ReplicationRuleAndOperator>,
+    pub(crate) tag: ::std::option::Option<super::types::Tag>,
+    pub(crate) and: ::std::option::Option<super::types::ReplicationRuleAndOperator>,
 }
 impl ReplicationRuleFilterBuilder {
     /// <p>An object key name prefix that identifies the subset of objects to which the rule applies.</p><important>
@@ -81,19 +81,19 @@
     }
     /// <p>A container for specifying a tag key and value.</p>
     /// <p>The rule applies only to objects that have the tag in their tag set.</p>
-    pub fn tag(mut self, input: crate::types::Tag) -> Self {
+    pub fn tag(mut self, input: super::types::Tag) -> Self {
         self.tag = ::std::option::Option::Some(input);
         self
     }
     /// <p>A container for specifying a tag key and value.</p>
     /// <p>The rule applies only to objects that have the tag in their tag set.</p>
-    pub fn set_tag(mut self, input: ::std::option::Option<crate::types::Tag>) -> Self {
+    pub fn set_tag(mut self, input: ::std::option::Option<super::types::Tag>) -> Self {
         self.tag = input;
         self
     }
     /// <p>A container for specifying a tag key and value.</p>
     /// <p>The rule applies only to objects that have the tag in their tag set.</p>
-    pub fn get_tag(&self) -> &::std::option::Option<crate::types::Tag> {
+    pub fn get_tag(&self) -> &::std::option::Option<super::types::Tag> {
         &self.tag
     }
     /// <p>A container for specifying rule filters. The filters determine the subset of objects to which the rule applies. This element is required only if you specify more than one filter. For example:</p>
@@ -103,7 +103,7 @@
     /// <li>
     /// <p>If you specify a filter based on multiple tags, wrap the <code>Tag</code> elements in an <code>And</code> tag.</p></li>
     /// </ul>
-    pub fn and(mut self, input: crate::types::ReplicationRuleAndOperator) -> Self {
+    pub fn and(mut self, input: super::types::ReplicationRuleAndOperator) -> Self {
         self.and = ::std::option::Option::Some(input);
         self
     }
@@ -114,7 +114,7 @@
     /// <li>
     /// <p>If you specify a filter based on multiple tags, wrap the <code>Tag</code> elements in an <code>And</code> tag.</p></li>
     /// </ul>
-    pub fn set_and(mut self, input: ::std::option::Option<crate::types::ReplicationRuleAndOperator>) -> Self {
+    pub fn set_and(mut self, input: ::std::option::Option<super::types::ReplicationRuleAndOperator>) -> Self {
         self.and = input;
         self
     }
@@ -125,12 +125,12 @@
     /// <li>
     /// <p>If you specify a filter based on multiple tags, wrap the <code>Tag</code> elements in an <code>And</code> tag.</p></li>
     /// </ul>
-    pub fn get_and(&self) -> &::std::option::Option<crate::types::ReplicationRuleAndOperator> {
+    pub fn get_and(&self) -> &::std::option::Option<super::types::ReplicationRuleAndOperator> {
         &self.and
     }
     /// Consumes the builder and constructs a [`ReplicationRuleFilter`](crate::types::ReplicationRuleFilter).
-    pub fn build(self) -> crate::types::ReplicationRuleFilter {
-        crate::types::ReplicationRuleFilter {
+    pub fn build(self) -> super::types::ReplicationRuleFilter {
+        super::types::ReplicationRuleFilter {
             prefix: self.prefix,
             tag: self.tag,
             and: self.and,
