--- original
+++ modified
@@ -25,8 +25,8 @@
 }
 impl S3TablesDestination {
     /// Creates a new builder-style object to manufacture [`S3TablesDestination`](crate::types::S3TablesDestination).
-    pub fn builder() -> crate::types::builders::S3TablesDestinationBuilder {
-        crate::types::builders::S3TablesDestinationBuilder::default()
+    pub fn builder() -> super::super::types::builders::S3TablesDestinationBuilder {
+        super::super::types::builders::S3TablesDestinationBuilder::default()
     }
 }

@@ -72,8 +72,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`table_bucket_arn`](crate::types::builders::S3TablesDestinationBuilder::table_bucket_arn)
     /// - [`table_name`](crate::types::builders::S3TablesDestinationBuilder::table_name)
-    pub fn build(self) -> ::std::result::Result<crate::types::S3TablesDestination, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::S3TablesDestination {
+    pub fn build(self) -> ::std::result::Result<super::super::types::S3TablesDestination, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::S3TablesDestination {
             table_bucket_arn: self.table_bucket_arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "table_bucket_arn",
