--- original
+++ modified
@@ -37,8 +37,8 @@
 }
 impl ScanRange {
     /// Creates a new builder-style object to manufacture [`ScanRange`](crate::types::ScanRange).
-    pub fn builder() -> crate::types::builders::ScanRangeBuilder {
-        crate::types::builders::ScanRangeBuilder::default()
+    pub fn builder() -> super::super::types::builders::ScanRangeBuilder {
+        super::super::types::builders::ScanRangeBuilder::default()
     }
 }

@@ -103,8 +103,8 @@
         &self.end
     }
     /// Consumes the builder and constructs a [`ScanRange`](crate::types::ScanRange).
-    pub fn build(self) -> crate::types::ScanRange {
-        crate::types::ScanRange {
+    pub fn build(self) -> super::super::types::ScanRange {
+        super::super::types::ScanRange {
             start: self.start,
             end: self.end,
         }
