--- original
+++ modified
@@ -6,8 +6,8 @@
 pub struct Sses3 {}
 impl Sses3 {
     /// Creates a new builder-style object to manufacture [`Sses3`](crate::types::Sses3).
-    pub fn builder() -> crate::types::builders::Sses3Builder {
-        crate::types::builders::Sses3Builder::default()
+    pub fn builder() -> super::types::builders::Sses3Builder {
+        super::types::builders::Sses3Builder::default()
     }
 }

@@ -17,7 +17,7 @@
 pub struct Sses3Builder {}
 impl Sses3Builder {
     /// Consumes the builder and constructs a [`Sses3`](crate::types::Sses3).
-    pub fn build(self) -> crate::types::Sses3 {
-        crate::types::Sses3 {}
+    pub fn build(self) -> super::types::Sses3 {
+        super::types::Sses3 {}
     }
 }
