--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl SsekmsEncryption {
     /// Creates a new builder-style object to manufacture [`SsekmsEncryption`](crate::types::SsekmsEncryption).
-    pub fn builder() -> crate::types::builders::SsekmsEncryptionBuilder {
-        crate::types::builders::SsekmsEncryptionBuilder::default()
+    pub fn builder() -> super::types::builders::SsekmsEncryptionBuilder {
+        super::types::builders::SsekmsEncryptionBuilder::default()
     }
 }

@@ -95,8 +95,8 @@
     /// Consumes the builder and constructs a [`SsekmsEncryption`](crate::types::SsekmsEncryption).
     /// This method will fail if any of the following fields are not set:
     /// - [`kms_key_arn`](crate::types::builders::SsekmsEncryptionBuilder::kms_key_arn)
-    pub fn build(self) -> ::std::result::Result<crate::types::SsekmsEncryption, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::SsekmsEncryption {
+    pub fn build(self) -> ::std::result::Result<super::types::SsekmsEncryption, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::SsekmsEncryption {
             kms_key_arn: self.kms_key_arn.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "kms_key_arn",
