--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The number of consecutive days of no access after which an object will be eligible to be transitioned to the corresponding tier. The minimum number of days specified for Archive Access tier must be at least 90 days and Deep Archive Access tier must be at least 180 days. The maximum can be up to 2 years (730 days).</p>
     pub days: i32,
     /// <p>S3 Intelligent-Tiering access tier. See <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html#sc-dynamic-data-access">Storage class for automatically optimizing frequently and infrequently accessed objects</a> for a list of access tiers in the S3 Intelligent-Tiering storage class.</p>
-    pub access_tier: crate::types::IntelligentTieringAccessTier,
+    pub access_tier: super::super::types::IntelligentTieringAccessTier,
 }
 impl Tiering {
     /// <p>The number of consecutive days of no access after which an object will be eligible to be transitioned to the corresponding tier. The minimum number of days specified for Archive Access tier must be at least 90 days and Deep Archive Access tier must be at least 180 days. The maximum can be up to 2 years (730 days).</p>
@@ -15,14 +15,14 @@
         self.days
     }
     /// <p>S3 Intelligent-Tiering access tier. See <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html#sc-dynamic-data-access">Storage class for automatically optimizing frequently and infrequently accessed objects</a> for a list of access tiers in the S3 Intelligent-Tiering storage class.</p>
-    pub fn access_tier(&self) -> &crate::types::IntelligentTieringAccessTier {
+    pub fn access_tier(&self) -> &super::super::types::IntelligentTieringAccessTier {
         &self.access_tier
     }
 }
 impl Tiering {
     /// Creates a new builder-style object to manufacture [`Tiering`](crate::types::Tiering).
-    pub fn builder() -> crate::types::builders::TieringBuilder {
-        crate::types::builders::TieringBuilder::default()
+    pub fn builder() -> super::super::types::builders::TieringBuilder {
+        super::super::types::builders::TieringBuilder::default()
     }
 }

@@ -31,7 +31,7 @@
 #[non_exhaustive]
 pub struct TieringBuilder {
     pub(crate) days: ::std::option::Option<i32>,
-    pub(crate) access_tier: ::std::option::Option<crate::types::IntelligentTieringAccessTier>,
+    pub(crate) access_tier: ::std::option::Option<super::super::types::IntelligentTieringAccessTier>,
 }
 impl TieringBuilder {
     /// <p>The number of consecutive days of no access after which an object will be eligible to be transitioned to the corresponding tier. The minimum number of days specified for Archive Access tier must be at least 90 days and Deep Archive Access tier must be at least 180 days. The maximum can be up to 2 years (730 days).</p>
@@ -51,17 +51,17 @@
     }
     /// <p>S3 Intelligent-Tiering access tier. See <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html#sc-dynamic-data-access">Storage class for automatically optimizing frequently and infrequently accessed objects</a> for a list of access tiers in the S3 Intelligent-Tiering storage class.</p>
     /// This field is required.
-    pub fn access_tier(mut self, input: crate::types::IntelligentTieringAccessTier) -> Self {
+    pub fn access_tier(mut self, input: super::super::types::IntelligentTieringAccessTier) -> Self {
         self.access_tier = ::std::option::Option::Some(input);
         self
     }
     /// <p>S3 Intelligent-Tiering access tier. See <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html#sc-dynamic-data-access">Storage class for automatically optimizing frequently and infrequently accessed objects</a> for a list of access tiers in the S3 Intelligent-Tiering storage class.</p>
-    pub fn set_access_tier(mut self, input: ::std::option::Option<crate::types::IntelligentTieringAccessTier>) -> Self {
+    pub fn set_access_tier(mut self, input: ::std::option::Option<super::super::types::IntelligentTieringAccessTier>) -> Self {
         self.access_tier = input;
         self
     }
     /// <p>S3 Intelligent-Tiering access tier. See <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html#sc-dynamic-data-access">Storage class for automatically optimizing frequently and infrequently accessed objects</a> for a list of access tiers in the S3 Intelligent-Tiering storage class.</p>
-    pub fn get_access_tier(&self) -> &::std::option::Option<crate::types::IntelligentTieringAccessTier> {
+    pub fn get_access_tier(&self) -> &::std::option::Option<super::super::types::IntelligentTieringAccessTier> {
         &self.access_tier
     }
     /// Consumes the builder and constructs a [`Tiering`](crate::types::Tiering).
@@ -68,8 +68,8 @@
     /// This method will fail if any of the following fields are not set:
     /// - [`days`](crate::types::builders::TieringBuilder::days)
     /// - [`access_tier`](crate::types::builders::TieringBuilder::access_tier)
-    pub fn build(self) -> ::std::result::Result<crate::types::Tiering, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::Tiering {
+    pub fn build(self) -> ::std::result::Result<super::super::types::Tiering, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::types::Tiering {
             days: self.days.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
                     "days",
