--- original
+++ modified
@@ -5,24 +5,24 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct VersioningConfiguration {
     /// <p>Specifies whether MFA delete is enabled in the bucket versioning configuration. This element is only returned if the bucket has been configured with MFA delete. If the bucket has never been so configured, this element is not returned.</p>
-    pub mfa_delete: ::std::option::Option<crate::types::MfaDelete>,
+    pub mfa_delete: ::std::option::Option<super::super::types::MfaDelete>,
     /// <p>The versioning state of the bucket.</p>
-    pub status: ::std::option::Option<crate::types::BucketVersioningStatus>,
+    pub status: ::std::option::Option<super::super::types::BucketVersioningStatus>,
 }
 impl VersioningConfiguration {
     /// <p>Specifies whether MFA delete is enabled in the bucket versioning configuration. This element is only returned if the bucket has been configured with MFA delete. If the bucket has never been so configured, this element is not returned.</p>
-    pub fn mfa_delete(&self) -> ::std::option::Option<&crate::types::MfaDelete> {
+    pub fn mfa_delete(&self) -> ::std::option::Option<&super::super::types::MfaDelete> {
         self.mfa_delete.as_ref()
     }
     /// <p>The versioning state of the bucket.</p>
-    pub fn status(&self) -> ::std::option::Option<&crate::types::BucketVersioningStatus> {
+    pub fn status(&self) -> ::std::option::Option<&super::super::types::BucketVersioningStatus> {
         self.status.as_ref()
     }
 }
 impl VersioningConfiguration {
     /// Creates a new builder-style object to manufacture [`VersioningConfiguration`](crate::types::VersioningConfiguration).
-    pub fn builder() -> crate::types::builders::VersioningConfigurationBuilder {
-        crate::types::builders::VersioningConfigurationBuilder::default()
+    pub fn builder() -> super::super::types::builders::VersioningConfigurationBuilder {
+        super::super::types::builders::VersioningConfigurationBuilder::default()
     }
 }

@@ -30,41 +30,41 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct VersioningConfigurationBuilder {
-    pub(crate) mfa_delete: ::std::option::Option<crate::types::MfaDelete>,
-    pub(crate) status: ::std::option::Option<crate::types::BucketVersioningStatus>,
+    pub(crate) mfa_delete: ::std::option::Option<super::super::types::MfaDelete>,
+    pub(crate) status: ::std::option::Option<super::super::types::BucketVersioningStatus>,
 }
 impl VersioningConfigurationBuilder {
     /// <p>Specifies whether MFA delete is enabled in the bucket versioning configuration. This element is only returned if the bucket has been configured with MFA delete. If the bucket has never been so configured, this element is not returned.</p>
-    pub fn mfa_delete(mut self, input: crate::types::MfaDelete) -> Self {
+    pub fn mfa_delete(mut self, input: super::super::types::MfaDelete) -> Self {
         self.mfa_delete = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether MFA delete is enabled in the bucket versioning configuration. This element is only returned if the bucket has been configured with MFA delete. If the bucket has never been so configured, this element is not returned.</p>
-    pub fn set_mfa_delete(mut self, input: ::std::option::Option<crate::types::MfaDelete>) -> Self {
+    pub fn set_mfa_delete(mut self, input: ::std::option::Option<super::super::types::MfaDelete>) -> Self {
         self.mfa_delete = input;
         self
     }
     /// <p>Specifies whether MFA delete is enabled in the bucket versioning configuration. This element is only returned if the bucket has been configured with MFA delete. If the bucket has never been so configured, this element is not returned.</p>
-    pub fn get_mfa_delete(&self) -> &::std::option::Option<crate::types::MfaDelete> {
+    pub fn get_mfa_delete(&self) -> &::std::option::Option<super::super::types::MfaDelete> {
         &self.mfa_delete
     }
     /// <p>The versioning state of the bucket.</p>
-    pub fn status(mut self, input: crate::types::BucketVersioningStatus) -> Self {
+    pub fn status(mut self, input: super::super::types::BucketVersioningStatus) -> Self {
         self.status = ::std::option::Option::Some(input);
         self
     }
     /// <p>The versioning state of the bucket.</p>
-    pub fn set_status(mut self, input: ::std::option::Option<crate::types::BucketVersioningStatus>) -> Self {
+    pub fn set_status(mut self, input: ::std::option::Option<super::super::types::BucketVersioningStatus>) -> Self {
         self.status = input;
         self
     }
     /// <p>The versioning state of the bucket.</p>
-    pub fn get_status(&self) -> &::std::option::Option<crate::types::BucketVersioningStatus> {
+    pub fn get_status(&self) -> &::std::option::Option<super::super::types::BucketVersioningStatus> {
         &self.status
     }
     /// Consumes the builder and constructs a [`VersioningConfiguration`](crate::types::VersioningConfiguration).
-    pub fn build(self) -> crate::types::VersioningConfiguration {
-        crate::types::VersioningConfiguration {
+    pub fn build(self) -> super::super::types::VersioningConfiguration {
+        super::super::types::VersioningConfiguration {
             mfa_delete: self.mfa_delete,
             status: self.status,
         }
