--- original
+++ modified
@@ -14,9 +14,9 @@
 }
 impl DisassociateSourceFromS3TableIntegrationInput {
     /// Creates a new builder-style object to manufacture [`DisassociateSourceFromS3TableIntegrationInput`](crate::operation::disassociate_source_from_s3_table_integration::DisassociateSourceFromS3TableIntegrationInput).
-    pub fn builder() -> crate::operation::disassociate_source_from_s3_table_integration::builders::DisassociateSourceFromS3TableIntegrationInputBuilder
+    pub fn builder() -> super::super::super::operation::disassociate_source_from_s3_table_integration::builders::DisassociateSourceFromS3TableIntegrationInputBuilder
     {
-        crate::operation::disassociate_source_from_s3_table_integration::builders::DisassociateSourceFromS3TableIntegrationInputBuilder::default()
+        super::super::super::operation::disassociate_source_from_s3_table_integration::builders::DisassociateSourceFromS3TableIntegrationInputBuilder::default()
     }
 }

@@ -46,11 +46,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::disassociate_source_from_s3_table_integration::DisassociateSourceFromS3TableIntegrationInput,
+        super::super::super::operation::disassociate_source_from_s3_table_integration::DisassociateSourceFromS3TableIntegrationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::disassociate_source_from_s3_table_integration::DisassociateSourceFromS3TableIntegrationInput {
+            super::super::super::operation::disassociate_source_from_s3_table_integration::DisassociateSourceFromS3TableIntegrationInput {
                 identifier: self.identifier,
             },
         )
