--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetTransformerInput {
     /// Creates a new builder-style object to manufacture [`GetTransformerInput`](crate::operation::get_transformer::GetTransformerInput).
-    pub fn builder() -> crate::operation::get_transformer::builders::GetTransformerInputBuilder {
-        crate::operation::get_transformer::builders::GetTransformerInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_transformer::builders::GetTransformerInputBuilder {
+        super::super::super::operation::get_transformer::builders::GetTransformerInputBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
     /// Consumes the builder and constructs a [`GetTransformerInput`](crate::operation::get_transformer::GetTransformerInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::get_transformer::GetTransformerInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::get_transformer::GetTransformerInput {
+    ) -> ::std::result::Result<super::super::super::operation::get_transformer::GetTransformerInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::get_transformer::GetTransformerInput {
             log_group_identifier: self.log_group_identifier,
         })
     }
