--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl UpdateLookupTableInput {
     /// Creates a new builder-style object to manufacture [`UpdateLookupTableInput`](crate::operation::update_lookup_table::UpdateLookupTableInput).
-    pub fn builder() -> crate::operation::update_lookup_table::builders::UpdateLookupTableInputBuilder {
-        crate::operation::update_lookup_table::builders::UpdateLookupTableInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_lookup_table::builders::UpdateLookupTableInputBuilder {
+        super::super::super::operation::update_lookup_table::builders::UpdateLookupTableInputBuilder::default()
     }
 }

@@ -138,8 +138,8 @@
     /// Consumes the builder and constructs a [`UpdateLookupTableInput`](crate::operation::update_lookup_table::UpdateLookupTableInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_lookup_table::UpdateLookupTableInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_lookup_table::UpdateLookupTableInput {
+    ) -> ::std::result::Result<super::super::super::operation::update_lookup_table::UpdateLookupTableInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_lookup_table::UpdateLookupTableInput {
             lookup_table_arn: self.lookup_table_arn,
             description: self.description,
             table_body: self.table_body,
