--- original
+++ modified
@@ -43,8 +43,8 @@
 }
 impl CreateLoginProfileInput {
     /// Creates a new builder-style object to manufacture [`CreateLoginProfileInput`](crate::operation::create_login_profile::CreateLoginProfileInput).
-    pub fn builder() -> crate::operation::create_login_profile::builders::CreateLoginProfileInputBuilder {
-        crate::operation::create_login_profile::builders::CreateLoginProfileInputBuilder::default()
+    pub fn builder() -> super::operation::create_login_profile::builders::CreateLoginProfileInputBuilder {
+        super::operation::create_login_profile::builders::CreateLoginProfileInputBuilder::default()
     }
 }

@@ -114,9 +114,9 @@
     /// Consumes the builder and constructs a [`CreateLoginProfileInput`](crate::operation::create_login_profile::CreateLoginProfileInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::create_login_profile::CreateLoginProfileInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::create_login_profile::CreateLoginProfileInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::create_login_profile::CreateLoginProfileInput {
+        ::std::result::Result::Ok(super::operation::create_login_profile::CreateLoginProfileInput {
             user_name: self.user_name,
             password: self.password,
             password_reset_required: self.password_reset_required,
