--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl UpdateDelegationRequestInput {
     /// Creates a new builder-style object to manufacture [`UpdateDelegationRequestInput`](crate::operation::update_delegation_request::UpdateDelegationRequestInput).
-    pub fn builder() -> crate::operation::update_delegation_request::builders::UpdateDelegationRequestInputBuilder {
-        crate::operation::update_delegation_request::builders::UpdateDelegationRequestInputBuilder::default()
+    pub fn builder() -> super::operation::update_delegation_request::builders::UpdateDelegationRequestInputBuilder {
+        super::operation::update_delegation_request::builders::UpdateDelegationRequestInputBuilder::default()
     }
 }

@@ -66,10 +66,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_delegation_request::UpdateDelegationRequestInput,
+        super::operation::update_delegation_request::UpdateDelegationRequestInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_delegation_request::UpdateDelegationRequestInput {
+        ::std::result::Result::Ok(super::operation::update_delegation_request::UpdateDelegationRequestInput {
             delegation_request_id: self.delegation_request_id,
             notes: self.notes,
         })
