--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>Specifies whether to enable the Deliverability dashboard. To enable the dashboard, set this value to <code>true</code>.</p>
     pub dashboard_enabled: ::std::option::Option<bool>,
     /// <p>An array of objects, one for each verified domain that you use to send email and enabled the Deliverability dashboard for.</p>
-    pub subscribed_domains: ::std::option::Option<::std::vec::Vec<crate::types::DomainDeliverabilityTrackingOption>>,
+    pub subscribed_domains: ::std::option::Option<::std::vec::Vec<super::super::super::types::DomainDeliverabilityTrackingOption>>,
 }
 impl PutDeliverabilityDashboardOptionInput {
     /// <p>Specifies whether to enable the Deliverability dashboard. To enable the dashboard, set this value to <code>true</code>.</p>
@@ -18,14 +18,14 @@
     /// <p>An array of objects, one for each verified domain that you use to send email and enabled the Deliverability dashboard for.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.subscribed_domains.is_none()`.
-    pub fn subscribed_domains(&self) -> &[crate::types::DomainDeliverabilityTrackingOption] {
+    pub fn subscribed_domains(&self) -> &[super::super::super::types::DomainDeliverabilityTrackingOption] {
         self.subscribed_domains.as_deref().unwrap_or_default()
     }
 }
 impl PutDeliverabilityDashboardOptionInput {
     /// Creates a new builder-style object to manufacture [`PutDeliverabilityDashboardOptionInput`](crate::operation::put_deliverability_dashboard_option::PutDeliverabilityDashboardOptionInput).
-    pub fn builder() -> crate::operation::put_deliverability_dashboard_option::builders::PutDeliverabilityDashboardOptionInputBuilder {
-        crate::operation::put_deliverability_dashboard_option::builders::PutDeliverabilityDashboardOptionInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_deliverability_dashboard_option::builders::PutDeliverabilityDashboardOptionInputBuilder {
+        super::super::super::operation::put_deliverability_dashboard_option::builders::PutDeliverabilityDashboardOptionInputBuilder::default()
     }
 }

@@ -34,7 +34,7 @@
 #[non_exhaustive]
 pub struct PutDeliverabilityDashboardOptionInputBuilder {
     pub(crate) dashboard_enabled: ::std::option::Option<bool>,
-    pub(crate) subscribed_domains: ::std::option::Option<::std::vec::Vec<crate::types::DomainDeliverabilityTrackingOption>>,
+    pub(crate) subscribed_domains: ::std::option::Option<::std::vec::Vec<super::super::super::types::DomainDeliverabilityTrackingOption>>,
 }
 impl PutDeliverabilityDashboardOptionInputBuilder {
     /// <p>Specifies whether to enable the Deliverability dashboard. To enable the dashboard, set this value to <code>true</code>.</p>
@@ -57,7 +57,7 @@
     /// To override the contents of this collection use [`set_subscribed_domains`](Self::set_subscribed_domains).
     ///
     /// <p>An array of objects, one for each verified domain that you use to send email and enabled the Deliverability dashboard for.</p>
-    pub fn subscribed_domains(mut self, input: crate::types::DomainDeliverabilityTrackingOption) -> Self {
+    pub fn subscribed_domains(mut self, input: super::super::super::types::DomainDeliverabilityTrackingOption) -> Self {
         let mut v = self.subscribed_domains.unwrap_or_default();
         v.push(input);
         self.subscribed_domains = ::std::option::Option::Some(v);
@@ -64,12 +64,12 @@
         self
     }
     /// <p>An array of objects, one for each verified domain that you use to send email and enabled the Deliverability dashboard for.</p>
-    pub fn set_subscribed_domains(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::DomainDeliverabilityTrackingOption>>) -> Self {
+    pub fn set_subscribed_domains(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::DomainDeliverabilityTrackingOption>>) -> Self {
         self.subscribed_domains = input;
         self
     }
     /// <p>An array of objects, one for each verified domain that you use to send email and enabled the Deliverability dashboard for.</p>
-    pub fn get_subscribed_domains(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::DomainDeliverabilityTrackingOption>> {
+    pub fn get_subscribed_domains(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::DomainDeliverabilityTrackingOption>> {
         &self.subscribed_domains
     }
     /// Consumes the builder and constructs a [`PutDeliverabilityDashboardOptionInput`](crate::operation::put_deliverability_dashboard_option::PutDeliverabilityDashboardOptionInput).
@@ -76,11 +76,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_deliverability_dashboard_option::PutDeliverabilityDashboardOptionInput,
+        super::super::super::operation::put_deliverability_dashboard_option::PutDeliverabilityDashboardOptionInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::put_deliverability_dashboard_option::PutDeliverabilityDashboardOptionInput {
+            super::super::super::operation::put_deliverability_dashboard_option::PutDeliverabilityDashboardOptionInput {
                 dashboard_enabled: self.dashboard_enabled,
                 subscribed_domains: self.subscribed_domains,
             },
