--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl DeleteUserPoolInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserPoolInput`](crate::operation::delete_user_pool::DeleteUserPoolInput).
-    pub fn builder() -> crate::operation::delete_user_pool::builders::DeleteUserPoolInputBuilder {
-        crate::operation::delete_user_pool::builders::DeleteUserPoolInputBuilder::default()
+    pub fn builder() -> super::operation::delete_user_pool::builders::DeleteUserPoolInputBuilder {
+        super::operation::delete_user_pool::builders::DeleteUserPoolInputBuilder::default()
     }
 }

@@ -45,8 +45,8 @@
     /// Consumes the builder and constructs a [`DeleteUserPoolInput`](crate::operation::delete_user_pool::DeleteUserPoolInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_user_pool::DeleteUserPoolInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::delete_user_pool::DeleteUserPoolInput {
+    ) -> ::std::result::Result<super::operation::delete_user_pool::DeleteUserPoolInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::delete_user_pool::DeleteUserPoolInput {
             user_pool_id: self.user_pool_id,
         })
     }
