--- original
+++ modified
@@ -40,8 +40,8 @@
 }
 impl DisassociateKmsKeyInput {
     /// Creates a new builder-style object to manufacture [`DisassociateKmsKeyInput`](crate::operation::disassociate_kms_key::DisassociateKmsKeyInput).
-    pub fn builder() -> crate::operation::disassociate_kms_key::builders::DisassociateKmsKeyInputBuilder {
-        crate::operation::disassociate_kms_key::builders::DisassociateKmsKeyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::disassociate_kms_key::builders::DisassociateKmsKeyInputBuilder {
+        super::super::super::operation::disassociate_kms_key::builders::DisassociateKmsKeyInputBuilder::default()
     }
 }

@@ -114,9 +114,9 @@
     /// Consumes the builder and constructs a [`DisassociateKmsKeyInput`](crate::operation::disassociate_kms_key::DisassociateKmsKeyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::disassociate_kms_key::DisassociateKmsKeyInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::disassociate_kms_key::DisassociateKmsKeyInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::disassociate_kms_key::DisassociateKmsKeyInput {
+        ::std::result::Result::Ok(super::super::super::operation::disassociate_kms_key::DisassociateKmsKeyInput {
             log_group_name: self.log_group_name,
             resource_identifier: self.resource_identifier,
         })
