--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl ListSyslogConfigurationsInput {
     /// Creates a new builder-style object to manufacture [`ListSyslogConfigurationsInput`](crate::operation::list_syslog_configurations::ListSyslogConfigurationsInput).
-    pub fn builder() -> crate::operation::list_syslog_configurations::builders::ListSyslogConfigurationsInputBuilder {
-        crate::operation::list_syslog_configurations::builders::ListSyslogConfigurationsInputBuilder::default()
+    pub fn builder() -> super::operation::list_syslog_configurations::builders::ListSyslogConfigurationsInputBuilder {
+        super::operation::list_syslog_configurations::builders::ListSyslogConfigurationsInputBuilder::default()
     }
 }

@@ -107,10 +107,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_syslog_configurations::ListSyslogConfigurationsInput,
+        super::operation::list_syslog_configurations::ListSyslogConfigurationsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_syslog_configurations::ListSyslogConfigurationsInput {
+        ::std::result::Result::Ok(super::operation::list_syslog_configurations::ListSyslogConfigurationsInput {
             log_group_identifier: self.log_group_identifier,
             vpc_endpoint_id: self.vpc_endpoint_id,
             next_token: self.next_token,
