--- original
+++ modified
@@ -22,8 +22,8 @@
 }
 impl TestMetricFilterInput {
     /// Creates a new builder-style object to manufacture [`TestMetricFilterInput`](crate::operation::test_metric_filter::TestMetricFilterInput).
-    pub fn builder() -> crate::operation::test_metric_filter::builders::TestMetricFilterInputBuilder {
-        crate::operation::test_metric_filter::builders::TestMetricFilterInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::test_metric_filter::builders::TestMetricFilterInputBuilder {
+        super::super::super::operation::test_metric_filter::builders::TestMetricFilterInputBuilder::default()
     }
 }

@@ -73,8 +73,8 @@
     /// Consumes the builder and constructs a [`TestMetricFilterInput`](crate::operation::test_metric_filter::TestMetricFilterInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::test_metric_filter::TestMetricFilterInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::test_metric_filter::TestMetricFilterInput {
+    ) -> ::std::result::Result<super::super::super::operation::test_metric_filter::TestMetricFilterInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::test_metric_filter::TestMetricFilterInput {
             filter_pattern: self.filter_pattern,
             log_event_messages: self.log_event_messages,
         })
