--- original
+++ modified
@@ -28,8 +28,8 @@
 }
 impl DeleteWebAuthnCredentialInput {
     /// Creates a new builder-style object to manufacture [`DeleteWebAuthnCredentialInput`](crate::operation::delete_web_authn_credential::DeleteWebAuthnCredentialInput).
-    pub fn builder() -> crate::operation::delete_web_authn_credential::builders::DeleteWebAuthnCredentialInputBuilder {
-        crate::operation::delete_web_authn_credential::builders::DeleteWebAuthnCredentialInputBuilder::default()
+    pub fn builder() -> super::operation::delete_web_authn_credential::builders::DeleteWebAuthnCredentialInputBuilder {
+        super::operation::delete_web_authn_credential::builders::DeleteWebAuthnCredentialInputBuilder::default()
     }
 }

@@ -75,10 +75,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_web_authn_credential::DeleteWebAuthnCredentialInput,
+        super::operation::delete_web_authn_credential::DeleteWebAuthnCredentialInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_web_authn_credential::DeleteWebAuthnCredentialInput {
+        ::std::result::Result::Ok(super::operation::delete_web_authn_credential::DeleteWebAuthnCredentialInput {
             access_token: self.access_token,
             credential_id: self.credential_id,
         })
