--- original
+++ modified
@@ -17,7 +17,7 @@
     /// <p>You can create and use KMS keys in your custom key stores only when its <code>ConnectionState</code> is <code>CONNECTED</code>.</p>
     /// <p>The <code>ConnectionState</code> value is <code>DISCONNECTED</code> only if the key store has never been connected or you use the <code>DisconnectCustomKeyStore</code> operation to disconnect it. If the value is <code>CONNECTED</code> but you are having trouble using the custom key store, make sure that the backing key store is reachable and active. For an CloudHSM key store, verify that its associated CloudHSM cluster is active and contains at least one active HSM. For an external key store, verify that the external key store proxy and external key manager are connected and enabled.</p>
     /// <p>A value of <code>FAILED</code> indicates that an attempt to connect was unsuccessful. The <code>ConnectionErrorCode</code> field in the response indicates the cause of the failure. For help resolving a connection failure, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/fix-keystore.html">Troubleshooting a custom key store</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub connection_state: ::std::option::Option<crate::types::ConnectionStateType>,
+    pub connection_state: ::std::option::Option<super::super::types::ConnectionStateType>,
     /// <p>Describes the connection error. This field appears in the response only when the <code>ConnectionState</code> is <code>FAILED</code>.</p>
     /// <p>Many failures can be resolved by updating the properties of the custom key store. To update a custom key store, disconnect it (<code>DisconnectCustomKeyStore</code>), correct the errors (<code>UpdateCustomKeyStore</code>), and try to connect again (<code>ConnectCustomKeyStore</code>). For additional help resolving these errors, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/fix-keystore.html#fix-keystore-failed">How to Fix a Connection Failure</a> in <i>Key Management Service Developer Guide</i>.</p>
     /// <p><b>All custom key stores:</b></p>
@@ -85,14 +85,14 @@
     /// <li>
     /// <p><code>XKS_VPC_ENDPOINT_SERVICE_NOT_FOUND</code> — KMS can't find the VPC endpoint service that it uses to communicate with the external key store proxy. Verify that the <code>XksProxyVpcEndpointServiceName</code> is correct and the KMS service principal has service consumer permissions on the Amazon VPC endpoint service.</p></li>
     /// </ul>
-    pub connection_error_code: ::std::option::Option<crate::types::ConnectionErrorCodeType>,
+    pub connection_error_code: ::std::option::Option<super::super::types::ConnectionErrorCodeType>,
     /// <p>The date and time when the custom key store was created.</p>
     pub creation_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>Indicates the type of the custom key store. <code>AWS_CLOUDHSM</code> indicates a custom key store backed by an CloudHSM cluster. <code>EXTERNAL_KEY_STORE</code> indicates a custom key store backed by an external key store proxy and external key manager outside of Amazon Web Services.</p>
-    pub custom_key_store_type: ::std::option::Option<crate::types::CustomKeyStoreType>,
+    pub custom_key_store_type: ::std::option::Option<super::super::types::CustomKeyStoreType>,
     /// <p>Configuration settings for the external key store proxy (XKS proxy). The external key store proxy translates KMS requests into a format that your external key manager can understand. The proxy configuration includes connection information that KMS requires.</p>
     /// <p>This field appears only when the <code>CustomKeyStoreType</code> is <code>EXTERNAL_KEY_STORE</code>.</p>
-    pub xks_proxy_configuration: ::std::option::Option<crate::types::XksProxyConfigurationType>,
+    pub xks_proxy_configuration: ::std::option::Option<super::super::types::XksProxyConfigurationType>,
 }
 impl CustomKeyStoresListEntry {
     /// <p>A unique identifier for the custom key store.</p>
@@ -116,7 +116,7 @@
     /// <p>You can create and use KMS keys in your custom key stores only when its <code>ConnectionState</code> is <code>CONNECTED</code>.</p>
     /// <p>The <code>ConnectionState</code> value is <code>DISCONNECTED</code> only if the key store has never been connected or you use the <code>DisconnectCustomKeyStore</code> operation to disconnect it. If the value is <code>CONNECTED</code> but you are having trouble using the custom key store, make sure that the backing key store is reachable and active. For an CloudHSM key store, verify that its associated CloudHSM cluster is active and contains at least one active HSM. For an external key store, verify that the external key store proxy and external key manager are connected and enabled.</p>
     /// <p>A value of <code>FAILED</code> indicates that an attempt to connect was unsuccessful. The <code>ConnectionErrorCode</code> field in the response indicates the cause of the failure. For help resolving a connection failure, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/fix-keystore.html">Troubleshooting a custom key store</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn connection_state(&self) -> ::std::option::Option<&crate::types::ConnectionStateType> {
+    pub fn connection_state(&self) -> ::std::option::Option<&super::super::types::ConnectionStateType> {
         self.connection_state.as_ref()
     }
     /// <p>Describes the connection error. This field appears in the response only when the <code>ConnectionState</code> is <code>FAILED</code>.</p>
@@ -186,7 +186,7 @@
     /// <li>
     /// <p><code>XKS_VPC_ENDPOINT_SERVICE_NOT_FOUND</code> — KMS can't find the VPC endpoint service that it uses to communicate with the external key store proxy. Verify that the <code>XksProxyVpcEndpointServiceName</code> is correct and the KMS service principal has service consumer permissions on the Amazon VPC endpoint service.</p></li>
     /// </ul>
-    pub fn connection_error_code(&self) -> ::std::option::Option<&crate::types::ConnectionErrorCodeType> {
+    pub fn connection_error_code(&self) -> ::std::option::Option<&super::super::types::ConnectionErrorCodeType> {
         self.connection_error_code.as_ref()
     }
     /// <p>The date and time when the custom key store was created.</p>
@@ -194,19 +194,19 @@
         self.creation_date.as_ref()
     }
     /// <p>Indicates the type of the custom key store. <code>AWS_CLOUDHSM</code> indicates a custom key store backed by an CloudHSM cluster. <code>EXTERNAL_KEY_STORE</code> indicates a custom key store backed by an external key store proxy and external key manager outside of Amazon Web Services.</p>
-    pub fn custom_key_store_type(&self) -> ::std::option::Option<&crate::types::CustomKeyStoreType> {
+    pub fn custom_key_store_type(&self) -> ::std::option::Option<&super::super::types::CustomKeyStoreType> {
         self.custom_key_store_type.as_ref()
     }
     /// <p>Configuration settings for the external key store proxy (XKS proxy). The external key store proxy translates KMS requests into a format that your external key manager can understand. The proxy configuration includes connection information that KMS requires.</p>
     /// <p>This field appears only when the <code>CustomKeyStoreType</code> is <code>EXTERNAL_KEY_STORE</code>.</p>
-    pub fn xks_proxy_configuration(&self) -> ::std::option::Option<&crate::types::XksProxyConfigurationType> {
+    pub fn xks_proxy_configuration(&self) -> ::std::option::Option<&super::super::types::XksProxyConfigurationType> {
         self.xks_proxy_configuration.as_ref()
     }
 }
 impl CustomKeyStoresListEntry {
     /// Creates a new builder-style object to manufacture [`CustomKeyStoresListEntry`](crate::types::CustomKeyStoresListEntry).
-    pub fn builder() -> crate::types::builders::CustomKeyStoresListEntryBuilder {
-        crate::types::builders::CustomKeyStoresListEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::CustomKeyStoresListEntryBuilder {
+        super::super::types::builders::CustomKeyStoresListEntryBuilder::default()
     }
 }

@@ -218,11 +218,11 @@
     pub(crate) custom_key_store_name: ::std::option::Option<::std::string::String>,
     pub(crate) cloud_hsm_cluster_id: ::std::option::Option<::std::string::String>,
     pub(crate) trust_anchor_certificate: ::std::option::Option<::std::string::String>,
-    pub(crate) connection_state: ::std::option::Option<crate::types::ConnectionStateType>,
-    pub(crate) connection_error_code: ::std::option::Option<crate::types::ConnectionErrorCodeType>,
+    pub(crate) connection_state: ::std::option::Option<super::super::types::ConnectionStateType>,
+    pub(crate) connection_error_code: ::std::option::Option<super::super::types::ConnectionErrorCodeType>,
     pub(crate) creation_date: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) custom_key_store_type: ::std::option::Option<crate::types::CustomKeyStoreType>,
-    pub(crate) xks_proxy_configuration: ::std::option::Option<crate::types::XksProxyConfigurationType>,
+    pub(crate) custom_key_store_type: ::std::option::Option<super::super::types::CustomKeyStoreType>,
+    pub(crate) xks_proxy_configuration: ::std::option::Option<super::super::types::XksProxyConfigurationType>,
 }
 impl CustomKeyStoresListEntryBuilder {
     /// <p>A unique identifier for the custom key store.</p>
@@ -288,7 +288,7 @@
     /// <p>You can create and use KMS keys in your custom key stores only when its <code>ConnectionState</code> is <code>CONNECTED</code>.</p>
     /// <p>The <code>ConnectionState</code> value is <code>DISCONNECTED</code> only if the key store has never been connected or you use the <code>DisconnectCustomKeyStore</code> operation to disconnect it. If the value is <code>CONNECTED</code> but you are having trouble using the custom key store, make sure that the backing key store is reachable and active. For an CloudHSM key store, verify that its associated CloudHSM cluster is active and contains at least one active HSM. For an external key store, verify that the external key store proxy and external key manager are connected and enabled.</p>
     /// <p>A value of <code>FAILED</code> indicates that an attempt to connect was unsuccessful. The <code>ConnectionErrorCode</code> field in the response indicates the cause of the failure. For help resolving a connection failure, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/fix-keystore.html">Troubleshooting a custom key store</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn connection_state(mut self, input: crate::types::ConnectionStateType) -> Self {
+    pub fn connection_state(mut self, input: super::super::types::ConnectionStateType) -> Self {
         self.connection_state = ::std::option::Option::Some(input);
         self
     }
@@ -296,7 +296,7 @@
     /// <p>You can create and use KMS keys in your custom key stores only when its <code>ConnectionState</code> is <code>CONNECTED</code>.</p>
     /// <p>The <code>ConnectionState</code> value is <code>DISCONNECTED</code> only if the key store has never been connected or you use the <code>DisconnectCustomKeyStore</code> operation to disconnect it. If the value is <code>CONNECTED</code> but you are having trouble using the custom key store, make sure that the backing key store is reachable and active. For an CloudHSM key store, verify that its associated CloudHSM cluster is active and contains at least one active HSM. For an external key store, verify that the external key store proxy and external key manager are connected and enabled.</p>
     /// <p>A value of <code>FAILED</code> indicates that an attempt to connect was unsuccessful. The <code>ConnectionErrorCode</code> field in the response indicates the cause of the failure. For help resolving a connection failure, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/fix-keystore.html">Troubleshooting a custom key store</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn set_connection_state(mut self, input: ::std::option::Option<crate::types::ConnectionStateType>) -> Self {
+    pub fn set_connection_state(mut self, input: ::std::option::Option<super::super::types::ConnectionStateType>) -> Self {
         self.connection_state = input;
         self
     }
@@ -304,7 +304,7 @@
     /// <p>You can create and use KMS keys in your custom key stores only when its <code>ConnectionState</code> is <code>CONNECTED</code>.</p>
     /// <p>The <code>ConnectionState</code> value is <code>DISCONNECTED</code> only if the key store has never been connected or you use the <code>DisconnectCustomKeyStore</code> operation to disconnect it. If the value is <code>CONNECTED</code> but you are having trouble using the custom key store, make sure that the backing key store is reachable and active. For an CloudHSM key store, verify that its associated CloudHSM cluster is active and contains at least one active HSM. For an external key store, verify that the external key store proxy and external key manager are connected and enabled.</p>
     /// <p>A value of <code>FAILED</code> indicates that an attempt to connect was unsuccessful. The <code>ConnectionErrorCode</code> field in the response indicates the cause of the failure. For help resolving a connection failure, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/fix-keystore.html">Troubleshooting a custom key store</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn get_connection_state(&self) -> &::std::option::Option<crate::types::ConnectionStateType> {
+    pub fn get_connection_state(&self) -> &::std::option::Option<super::super::types::ConnectionStateType> {
         &self.connection_state
     }
     /// <p>Describes the connection error. This field appears in the response only when the <code>ConnectionState</code> is <code>FAILED</code>.</p>
@@ -374,7 +374,7 @@
     /// <li>
     /// <p><code>XKS_VPC_ENDPOINT_SERVICE_NOT_FOUND</code> — KMS can't find the VPC endpoint service that it uses to communicate with the external key store proxy. Verify that the <code>XksProxyVpcEndpointServiceName</code> is correct and the KMS service principal has service consumer permissions on the Amazon VPC endpoint service.</p></li>
     /// </ul>
-    pub fn connection_error_code(mut self, input: crate::types::ConnectionErrorCodeType) -> Self {
+    pub fn connection_error_code(mut self, input: super::super::types::ConnectionErrorCodeType) -> Self {
         self.connection_error_code = ::std::option::Option::Some(input);
         self
     }
@@ -445,7 +445,7 @@
     /// <li>
     /// <p><code>XKS_VPC_ENDPOINT_SERVICE_NOT_FOUND</code> — KMS can't find the VPC endpoint service that it uses to communicate with the external key store proxy. Verify that the <code>XksProxyVpcEndpointServiceName</code> is correct and the KMS service principal has service consumer permissions on the Amazon VPC endpoint service.</p></li>
     /// </ul>
-    pub fn set_connection_error_code(mut self, input: ::std::option::Option<crate::types::ConnectionErrorCodeType>) -> Self {
+    pub fn set_connection_error_code(mut self, input: ::std::option::Option<super::super::types::ConnectionErrorCodeType>) -> Self {
         self.connection_error_code = input;
         self
     }
@@ -516,7 +516,7 @@
     /// <li>
     /// <p><code>XKS_VPC_ENDPOINT_SERVICE_NOT_FOUND</code> — KMS can't find the VPC endpoint service that it uses to communicate with the external key store proxy. Verify that the <code>XksProxyVpcEndpointServiceName</code> is correct and the KMS service principal has service consumer permissions on the Amazon VPC endpoint service.</p></li>
     /// </ul>
-    pub fn get_connection_error_code(&self) -> &::std::option::Option<crate::types::ConnectionErrorCodeType> {
+    pub fn get_connection_error_code(&self) -> &::std::option::Option<super::super::types::ConnectionErrorCodeType> {
         &self.connection_error_code
     }
     /// <p>The date and time when the custom key store was created.</p>
@@ -534,39 +534,39 @@
         &self.creation_date
     }
     /// <p>Indicates the type of the custom key store. <code>AWS_CLOUDHSM</code> indicates a custom key store backed by an CloudHSM cluster. <code>EXTERNAL_KEY_STORE</code> indicates a custom key store backed by an external key store proxy and external key manager outside of Amazon Web Services.</p>
-    pub fn custom_key_store_type(mut self, input: crate::types::CustomKeyStoreType) -> Self {
+    pub fn custom_key_store_type(mut self, input: super::super::types::CustomKeyStoreType) -> Self {
         self.custom_key_store_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>Indicates the type of the custom key store. <code>AWS_CLOUDHSM</code> indicates a custom key store backed by an CloudHSM cluster. <code>EXTERNAL_KEY_STORE</code> indicates a custom key store backed by an external key store proxy and external key manager outside of Amazon Web Services.</p>
-    pub fn set_custom_key_store_type(mut self, input: ::std::option::Option<crate::types::CustomKeyStoreType>) -> Self {
+    pub fn set_custom_key_store_type(mut self, input: ::std::option::Option<super::super::types::CustomKeyStoreType>) -> Self {
         self.custom_key_store_type = input;
         self
     }
     /// <p>Indicates the type of the custom key store. <code>AWS_CLOUDHSM</code> indicates a custom key store backed by an CloudHSM cluster. <code>EXTERNAL_KEY_STORE</code> indicates a custom key store backed by an external key store proxy and external key manager outside of Amazon Web Services.</p>
-    pub fn get_custom_key_store_type(&self) -> &::std::option::Option<crate::types::CustomKeyStoreType> {
+    pub fn get_custom_key_store_type(&self) -> &::std::option::Option<super::super::types::CustomKeyStoreType> {
         &self.custom_key_store_type
     }
     /// <p>Configuration settings for the external key store proxy (XKS proxy). The external key store proxy translates KMS requests into a format that your external key manager can understand. The proxy configuration includes connection information that KMS requires.</p>
     /// <p>This field appears only when the <code>CustomKeyStoreType</code> is <code>EXTERNAL_KEY_STORE</code>.</p>
-    pub fn xks_proxy_configuration(mut self, input: crate::types::XksProxyConfigurationType) -> Self {
+    pub fn xks_proxy_configuration(mut self, input: super::super::types::XksProxyConfigurationType) -> Self {
         self.xks_proxy_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Configuration settings for the external key store proxy (XKS proxy). The external key store proxy translates KMS requests into a format that your external key manager can understand. The proxy configuration includes connection information that KMS requires.</p>
     /// <p>This field appears only when the <code>CustomKeyStoreType</code> is <code>EXTERNAL_KEY_STORE</code>.</p>
-    pub fn set_xks_proxy_configuration(mut self, input: ::std::option::Option<crate::types::XksProxyConfigurationType>) -> Self {
+    pub fn set_xks_proxy_configuration(mut self, input: ::std::option::Option<super::super::types::XksProxyConfigurationType>) -> Self {
         self.xks_proxy_configuration = input;
         self
     }
     /// <p>Configuration settings for the external key store proxy (XKS proxy). The external key store proxy translates KMS requests into a format that your external key manager can understand. The proxy configuration includes connection information that KMS requires.</p>
     /// <p>This field appears only when the <code>CustomKeyStoreType</code> is <code>EXTERNAL_KEY_STORE</code>.</p>
-    pub fn get_xks_proxy_configuration(&self) -> &::std::option::Option<crate::types::XksProxyConfigurationType> {
+    pub fn get_xks_proxy_configuration(&self) -> &::std::option::Option<super::super::types::XksProxyConfigurationType> {
         &self.xks_proxy_configuration
     }
     /// Consumes the builder and constructs a [`CustomKeyStoresListEntry`](crate::types::CustomKeyStoresListEntry).
-    pub fn build(self) -> crate::types::CustomKeyStoresListEntry {
-        crate::types::CustomKeyStoresListEntry {
+    pub fn build(self) -> super::super::types::CustomKeyStoresListEntry {
+        super::super::types::CustomKeyStoresListEntry {
             custom_key_store_id: self.custom_key_store_id,
             custom_key_store_name: self.custom_key_store_name,
             cloud_hsm_cluster_id: self.cloud_hsm_cluster_id,
