--- original
+++ modified
@@ -21,8 +21,8 @@
 }
 impl KeyListEntry {
     /// Creates a new builder-style object to manufacture [`KeyListEntry`](crate::types::KeyListEntry).
-    pub fn builder() -> crate::types::builders::KeyListEntryBuilder {
-        crate::types::builders::KeyListEntryBuilder::default()
+    pub fn builder() -> super::super::types::builders::KeyListEntryBuilder {
+        super::super::types::builders::KeyListEntryBuilder::default()
     }
 }

@@ -63,8 +63,8 @@
         &self.key_arn
     }
     /// Consumes the builder and constructs a [`KeyListEntry`](crate::types::KeyListEntry).
-    pub fn build(self) -> crate::types::KeyListEntry {
-        crate::types::KeyListEntry {
+    pub fn build(self) -> super::super::types::KeyListEntry {
+        super::super::types::KeyListEntry {
             key_id: self.key_id,
             key_arn: self.key_arn,
         }
