--- original
+++ modified
@@ -93,7 +93,7 @@
     SymmetricDefault,
     /// `Unknown` contains new variants that have been added since this code was generated.
     #[deprecated(note = "Don't directly match on `Unknown`. See the docs on this enum for the correct way to handle unknown variants.")]
-    Unknown(crate::primitives::sealed_enum_unknown::UnknownVariantValue),
+    Unknown(super::super::primitives::sealed_enum_unknown::UnknownVariantValue),
 }
 impl ::std::convert::From<&str> for KeySpec {
     fn from(s: &str) -> Self {
@@ -115,7 +115,7 @@
             "RSA_4096" => KeySpec::Rsa4096,
             "SM2" => KeySpec::Sm2,
             "SYMMETRIC_DEFAULT" => KeySpec::SymmetricDefault,
-            other => KeySpec::Unknown(crate::primitives::sealed_enum_unknown::UnknownVariantValue(other.to_owned())),
+            other => KeySpec::Unknown(super::super::primitives::sealed_enum_unknown::UnknownVariantValue(other.to_owned())),
         }
     }
 }
@@ -182,10 +182,10 @@
     /// Parses the enum value while disallowing unknown variants.
     ///
     /// Unknown variants will result in an error.
-    pub fn try_parse(value: &str) -> ::std::result::Result<Self, crate::error::UnknownVariantError> {
+    pub fn try_parse(value: &str) -> ::std::result::Result<Self, super::super::error::UnknownVariantError> {
         match Self::from(value) {
             #[allow(deprecated)]
-            Self::Unknown(_) => ::std::result::Result::Err(crate::error::UnknownVariantError::new(value)),
+            Self::Unknown(_) => ::std::result::Result::Err(super::super::error::UnknownVariantError::new(value)),
             known => Ok(known),
         }
     }
