--- original
+++ modified
@@ -18,10 +18,10 @@
     /// <p>The description of the KMS key.</p>
     pub description: ::std::option::Option<::std::string::String>,
     /// <p>The <a href="https://docs.aws.amazon.com/kms/latest/developerguide/kms-cryptography.html#cryptographic-operations">cryptographic operations</a> for which you can use the KMS key.</p>
-    pub key_usage: ::std::option::Option<crate::types::KeyUsageType>,
+    pub key_usage: ::std::option::Option<super::types::KeyUsageType>,
     /// <p>The current status of the KMS key.</p>
     /// <p>For more information about how key state affects the use of a KMS key, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-state.html">Key states of KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub key_state: ::std::option::Option<crate::types::KeyState>,
+    pub key_state: ::std::option::Option<super::types::KeyState>,
     /// <p>The date and time after which KMS deletes this KMS key. This value is present only when the KMS key is scheduled for deletion, that is, when its <code>KeyState</code> is <code>PendingDeletion</code>.</p>
     /// <p>When the primary key in a multi-Region key is scheduled for deletion but still has replica keys, its key state is <code>PendingReplicaDeletion</code> and the length of its waiting period is displayed in the <code>PendingDeletionWindowInDays</code> field.</p>
     pub deletion_date: ::std::option::Option<::aws_smithy_types::DateTime>,
@@ -28,29 +28,29 @@
     /// <p>The earliest time at which any imported key material permanently associated with this KMS key expires. When a key material expires, KMS deletes the key material and the KMS key becomes unusable. This value is present only for KMS keys whose <code>Origin</code> is <code>EXTERNAL</code> and the <code>ExpirationModel</code> is <code>KEY_MATERIAL_EXPIRES</code>, otherwise this value is omitted.</p>
     pub valid_to: ::std::option::Option<::aws_smithy_types::DateTime>,
     /// <p>The source of the key material for the KMS key. When this value is <code>AWS_KMS</code>, KMS created the key material. When this value is <code>EXTERNAL</code>, the key material was imported or the KMS key doesn't have any key material. When this value is <code>AWS_CLOUDHSM</code>, the key material was created in the CloudHSM cluster associated with a custom key store.</p>
-    pub origin: ::std::option::Option<crate::types::OriginType>,
+    pub origin: ::std::option::Option<super::types::OriginType>,
     /// <p>A unique identifier for the <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-store-overview.html">custom key store</a> that contains the KMS key. This field is present only when the KMS key is created in a custom key store.</p>
     pub custom_key_store_id: ::std::option::Option<::std::string::String>,
     /// <p>The cluster ID of the CloudHSM cluster that contains the key material for the KMS key. When you create a KMS key in an CloudHSM <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-store-overview.html">custom key store</a>, KMS creates the key material for the KMS key in the associated CloudHSM cluster. This field is present only when the KMS key is created in an CloudHSM key store.</p>
     pub cloud_hsm_cluster_id: ::std::option::Option<::std::string::String>,
     /// <p>Specifies whether the KMS key's key material expires. This value is present only when <code>Origin</code> is <code>EXTERNAL</code>, otherwise this value is omitted.</p>
-    pub expiration_model: ::std::option::Option<crate::types::ExpirationModelType>,
+    pub expiration_model: ::std::option::Option<super::types::ExpirationModelType>,
     /// <p>The manager of the KMS key. KMS keys in your Amazon Web Services account are either customer managed or Amazon Web Services managed. For more information about the difference, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#kms_keys">KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub key_manager: ::std::option::Option<crate::types::KeyManagerType>,
+    pub key_manager: ::std::option::Option<super::types::KeyManagerType>,
     /// <p>Instead, use the <code>KeySpec</code> field.</p>
     /// <p>The <code>KeySpec</code> and <code>CustomerMasterKeySpec</code> fields have the same value. We recommend that you use the <code>KeySpec</code> field in your code. However, to avoid breaking changes, KMS supports both fields.</p>
     #[deprecated(note = "This field has been deprecated. Instead, use the KeySpec field.")]
-    pub customer_master_key_spec: ::std::option::Option<crate::types::CustomerMasterKeySpec>,
+    pub customer_master_key_spec: ::std::option::Option<super::types::CustomerMasterKeySpec>,
     /// <p>Describes the type of key material in the KMS key.</p>
-    pub key_spec: ::std::option::Option<crate::types::KeySpec>,
+    pub key_spec: ::std::option::Option<super::types::KeySpec>,
     /// <p>The encryption algorithms that the KMS key supports. You cannot use the KMS key with other encryption algorithms within KMS.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>ENCRYPT_DECRYPT</code>.</p>
-    pub encryption_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::EncryptionAlgorithmSpec>>,
+    pub encryption_algorithms: ::std::option::Option<::std::vec::Vec<super::types::EncryptionAlgorithmSpec>>,
     /// <p>The signing algorithms that the KMS key supports. You cannot use the KMS key with other signing algorithms within KMS.</p>
     /// <p>This field appears only when the <code>KeyUsage</code> of the KMS key is <code>SIGN_VERIFY</code>.</p>
-    pub signing_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::SigningAlgorithmSpec>>,
+    pub signing_algorithms: ::std::option::Option<::std::vec::Vec<super::types::SigningAlgorithmSpec>>,
     /// <p>The key agreement algorithm used to derive a shared secret.</p>
-    pub key_agreement_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::KeyAgreementAlgorithmSpec>>,
+    pub key_agreement_algorithms: ::std::option::Option<::std::vec::Vec<super::types::KeyAgreementAlgorithmSpec>>,
     /// <p>Indicates whether the KMS key is a multi-Region (<code>True</code>) or regional (<code>False</code>) key. This value is <code>True</code> for multi-Region primary and replica keys and <code>False</code> for regional KMS keys.</p>
     /// <p>For more information about multi-Region keys, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/multi-region-keys-overview.html">Multi-Region keys in KMS</a> in the <i>Key Management Service Developer Guide</i>.</p>
     pub multi_region: ::std::option::Option<bool>,
@@ -64,16 +64,16 @@
     /// <li>
     /// <p><code>ReplicaKeys</code> displays the key ARNs and Regions of all replica keys. This field includes the current KMS key if it is a replica key.</p></li>
     /// </ul>
-    pub multi_region_configuration: ::std::option::Option<crate::types::MultiRegionConfiguration>,
+    pub multi_region_configuration: ::std::option::Option<super::types::MultiRegionConfiguration>,
     /// <p>The waiting period before the primary key in a multi-Region key is deleted. This waiting period begins when the last of its replica keys is deleted. This value is present only when the <code>KeyState</code> of the KMS key is <code>PendingReplicaDeletion</code>. That indicates that the KMS key is the primary key in a multi-Region key, it is scheduled for deletion, and it still has existing replica keys.</p>
     /// <p>When a single-Region KMS key or a multi-Region replica key is scheduled for deletion, its deletion date is displayed in the <code>DeletionDate</code> field. However, when the primary key in a multi-Region key is scheduled for deletion, its waiting period doesn't begin until all of its replica keys are deleted. This value displays that waiting period. When the last replica key in the multi-Region key is deleted, the <code>KeyState</code> of the scheduled primary key changes from <code>PendingReplicaDeletion</code> to <code>PendingDeletion</code> and the deletion date appears in the <code>DeletionDate</code> field.</p>
     pub pending_deletion_window_in_days: ::std::option::Option<i32>,
     /// <p>The message authentication code (MAC) algorithm that the HMAC KMS key supports.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>GENERATE_VERIFY_MAC</code>.</p>
-    pub mac_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::MacAlgorithmSpec>>,
+    pub mac_algorithms: ::std::option::Option<::std::vec::Vec<super::types::MacAlgorithmSpec>>,
     /// <p>Information about the external key that is associated with a KMS key in an external key store.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/keystore-external.html#concept-external-key">External key</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub xks_key_configuration: ::std::option::Option<crate::types::XksKeyConfigurationType>,
+    pub xks_key_configuration: ::std::option::Option<super::types::XksKeyConfigurationType>,
     /// <p>Identifies the current key material. This value is present for symmetric encryption keys with <code>AWS_KMS</code> or <code>EXTERNAL</code> origin. These KMS keys support automatic or on-demand key rotation and can have multiple key materials associated with them. KMS uses the current key material for both encryption and decryption, and the non-current key material for decryption operations only.</p>
     pub current_key_material_id: ::std::option::Option<::std::string::String>,
 }
@@ -104,12 +104,12 @@
         self.description.as_deref()
     }
     /// <p>The <a href="https://docs.aws.amazon.com/kms/latest/developerguide/kms-cryptography.html#cryptographic-operations">cryptographic operations</a> for which you can use the KMS key.</p>
-    pub fn key_usage(&self) -> ::std::option::Option<&crate::types::KeyUsageType> {
+    pub fn key_usage(&self) -> ::std::option::Option<&super::types::KeyUsageType> {
         self.key_usage.as_ref()
     }
     /// <p>The current status of the KMS key.</p>
     /// <p>For more information about how key state affects the use of a KMS key, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-state.html">Key states of KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn key_state(&self) -> ::std::option::Option<&crate::types::KeyState> {
+    pub fn key_state(&self) -> ::std::option::Option<&super::types::KeyState> {
         self.key_state.as_ref()
     }
     /// <p>The date and time after which KMS deletes this KMS key. This value is present only when the KMS key is scheduled for deletion, that is, when its <code>KeyState</code> is <code>PendingDeletion</code>.</p>
@@ -122,7 +122,7 @@
         self.valid_to.as_ref()
     }
     /// <p>The source of the key material for the KMS key. When this value is <code>AWS_KMS</code>, KMS created the key material. When this value is <code>EXTERNAL</code>, the key material was imported or the KMS key doesn't have any key material. When this value is <code>AWS_CLOUDHSM</code>, the key material was created in the CloudHSM cluster associated with a custom key store.</p>
-    pub fn origin(&self) -> ::std::option::Option<&crate::types::OriginType> {
+    pub fn origin(&self) -> ::std::option::Option<&super::types::OriginType> {
         self.origin.as_ref()
     }
     /// <p>A unique identifier for the <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-store-overview.html">custom key store</a> that contains the KMS key. This field is present only when the KMS key is created in a custom key store.</p>
@@ -134,21 +134,21 @@
         self.cloud_hsm_cluster_id.as_deref()
     }
     /// <p>Specifies whether the KMS key's key material expires. This value is present only when <code>Origin</code> is <code>EXTERNAL</code>, otherwise this value is omitted.</p>
-    pub fn expiration_model(&self) -> ::std::option::Option<&crate::types::ExpirationModelType> {
+    pub fn expiration_model(&self) -> ::std::option::Option<&super::types::ExpirationModelType> {
         self.expiration_model.as_ref()
     }
     /// <p>The manager of the KMS key. KMS keys in your Amazon Web Services account are either customer managed or Amazon Web Services managed. For more information about the difference, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#kms_keys">KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn key_manager(&self) -> ::std::option::Option<&crate::types::KeyManagerType> {
+    pub fn key_manager(&self) -> ::std::option::Option<&super::types::KeyManagerType> {
         self.key_manager.as_ref()
     }
     /// <p>Instead, use the <code>KeySpec</code> field.</p>
     /// <p>The <code>KeySpec</code> and <code>CustomerMasterKeySpec</code> fields have the same value. We recommend that you use the <code>KeySpec</code> field in your code. However, to avoid breaking changes, KMS supports both fields.</p>
     #[deprecated(note = "This field has been deprecated. Instead, use the KeySpec field.")]
-    pub fn customer_master_key_spec(&self) -> ::std::option::Option<&crate::types::CustomerMasterKeySpec> {
+    pub fn customer_master_key_spec(&self) -> ::std::option::Option<&super::types::CustomerMasterKeySpec> {
         self.customer_master_key_spec.as_ref()
     }
     /// <p>Describes the type of key material in the KMS key.</p>
-    pub fn key_spec(&self) -> ::std::option::Option<&crate::types::KeySpec> {
+    pub fn key_spec(&self) -> ::std::option::Option<&super::types::KeySpec> {
         self.key_spec.as_ref()
     }
     /// <p>The encryption algorithms that the KMS key supports. You cannot use the KMS key with other encryption algorithms within KMS.</p>
@@ -155,7 +155,7 @@
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>ENCRYPT_DECRYPT</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.encryption_algorithms.is_none()`.
-    pub fn encryption_algorithms(&self) -> &[crate::types::EncryptionAlgorithmSpec] {
+    pub fn encryption_algorithms(&self) -> &[super::types::EncryptionAlgorithmSpec] {
         self.encryption_algorithms.as_deref().unwrap_or_default()
     }
     /// <p>The signing algorithms that the KMS key supports. You cannot use the KMS key with other signing algorithms within KMS.</p>
@@ -162,13 +162,13 @@
     /// <p>This field appears only when the <code>KeyUsage</code> of the KMS key is <code>SIGN_VERIFY</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.signing_algorithms.is_none()`.
-    pub fn signing_algorithms(&self) -> &[crate::types::SigningAlgorithmSpec] {
+    pub fn signing_algorithms(&self) -> &[super::types::SigningAlgorithmSpec] {
         self.signing_algorithms.as_deref().unwrap_or_default()
     }
     /// <p>The key agreement algorithm used to derive a shared secret.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.key_agreement_algorithms.is_none()`.
-    pub fn key_agreement_algorithms(&self) -> &[crate::types::KeyAgreementAlgorithmSpec] {
+    pub fn key_agreement_algorithms(&self) -> &[super::types::KeyAgreementAlgorithmSpec] {
         self.key_agreement_algorithms.as_deref().unwrap_or_default()
     }
     /// <p>Indicates whether the KMS key is a multi-Region (<code>True</code>) or regional (<code>False</code>) key. This value is <code>True</code> for multi-Region primary and replica keys and <code>False</code> for regional KMS keys.</p>
@@ -186,7 +186,7 @@
     /// <li>
     /// <p><code>ReplicaKeys</code> displays the key ARNs and Regions of all replica keys. This field includes the current KMS key if it is a replica key.</p></li>
     /// </ul>
-    pub fn multi_region_configuration(&self) -> ::std::option::Option<&crate::types::MultiRegionConfiguration> {
+    pub fn multi_region_configuration(&self) -> ::std::option::Option<&super::types::MultiRegionConfiguration> {
         self.multi_region_configuration.as_ref()
     }
     /// <p>The waiting period before the primary key in a multi-Region key is deleted. This waiting period begins when the last of its replica keys is deleted. This value is present only when the <code>KeyState</code> of the KMS key is <code>PendingReplicaDeletion</code>. That indicates that the KMS key is the primary key in a multi-Region key, it is scheduled for deletion, and it still has existing replica keys.</p>
@@ -198,12 +198,12 @@
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>GENERATE_VERIFY_MAC</code>.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.mac_algorithms.is_none()`.
-    pub fn mac_algorithms(&self) -> &[crate::types::MacAlgorithmSpec] {
+    pub fn mac_algorithms(&self) -> &[super::types::MacAlgorithmSpec] {
         self.mac_algorithms.as_deref().unwrap_or_default()
     }
     /// <p>Information about the external key that is associated with a KMS key in an external key store.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/keystore-external.html#concept-external-key">External key</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn xks_key_configuration(&self) -> ::std::option::Option<&crate::types::XksKeyConfigurationType> {
+    pub fn xks_key_configuration(&self) -> ::std::option::Option<&super::types::XksKeyConfigurationType> {
         self.xks_key_configuration.as_ref()
     }
     /// <p>Identifies the current key material. This value is present for symmetric encryption keys with <code>AWS_KMS</code> or <code>EXTERNAL</code> origin. These KMS keys support automatic or on-demand key rotation and can have multiple key materials associated with them. KMS uses the current key material for both encryption and decryption, and the non-current key material for decryption operations only.</p>
@@ -213,8 +213,8 @@
 }
 impl KeyMetadata {
     /// Creates a new builder-style object to manufacture [`KeyMetadata`](crate::types::KeyMetadata).
-    pub fn builder() -> crate::types::builders::KeyMetadataBuilder {
-        crate::types::builders::KeyMetadataBuilder::default()
+    pub fn builder() -> super::types::builders::KeyMetadataBuilder {
+        super::types::builders::KeyMetadataBuilder::default()
     }
 }

@@ -228,25 +228,25 @@
     pub(crate) creation_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) enabled: ::std::option::Option<bool>,
     pub(crate) description: ::std::option::Option<::std::string::String>,
-    pub(crate) key_usage: ::std::option::Option<crate::types::KeyUsageType>,
-    pub(crate) key_state: ::std::option::Option<crate::types::KeyState>,
+    pub(crate) key_usage: ::std::option::Option<super::types::KeyUsageType>,
+    pub(crate) key_state: ::std::option::Option<super::types::KeyState>,
     pub(crate) deletion_date: ::std::option::Option<::aws_smithy_types::DateTime>,
     pub(crate) valid_to: ::std::option::Option<::aws_smithy_types::DateTime>,
-    pub(crate) origin: ::std::option::Option<crate::types::OriginType>,
+    pub(crate) origin: ::std::option::Option<super::types::OriginType>,
     pub(crate) custom_key_store_id: ::std::option::Option<::std::string::String>,
     pub(crate) cloud_hsm_cluster_id: ::std::option::Option<::std::string::String>,
-    pub(crate) expiration_model: ::std::option::Option<crate::types::ExpirationModelType>,
-    pub(crate) key_manager: ::std::option::Option<crate::types::KeyManagerType>,
-    pub(crate) customer_master_key_spec: ::std::option::Option<crate::types::CustomerMasterKeySpec>,
-    pub(crate) key_spec: ::std::option::Option<crate::types::KeySpec>,
-    pub(crate) encryption_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::EncryptionAlgorithmSpec>>,
-    pub(crate) signing_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::SigningAlgorithmSpec>>,
-    pub(crate) key_agreement_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::KeyAgreementAlgorithmSpec>>,
+    pub(crate) expiration_model: ::std::option::Option<super::types::ExpirationModelType>,
+    pub(crate) key_manager: ::std::option::Option<super::types::KeyManagerType>,
+    pub(crate) customer_master_key_spec: ::std::option::Option<super::types::CustomerMasterKeySpec>,
+    pub(crate) key_spec: ::std::option::Option<super::types::KeySpec>,
+    pub(crate) encryption_algorithms: ::std::option::Option<::std::vec::Vec<super::types::EncryptionAlgorithmSpec>>,
+    pub(crate) signing_algorithms: ::std::option::Option<::std::vec::Vec<super::types::SigningAlgorithmSpec>>,
+    pub(crate) key_agreement_algorithms: ::std::option::Option<::std::vec::Vec<super::types::KeyAgreementAlgorithmSpec>>,
     pub(crate) multi_region: ::std::option::Option<bool>,
-    pub(crate) multi_region_configuration: ::std::option::Option<crate::types::MultiRegionConfiguration>,
+    pub(crate) multi_region_configuration: ::std::option::Option<super::types::MultiRegionConfiguration>,
     pub(crate) pending_deletion_window_in_days: ::std::option::Option<i32>,
-    pub(crate) mac_algorithms: ::std::option::Option<::std::vec::Vec<crate::types::MacAlgorithmSpec>>,
-    pub(crate) xks_key_configuration: ::std::option::Option<crate::types::XksKeyConfigurationType>,
+    pub(crate) mac_algorithms: ::std::option::Option<::std::vec::Vec<super::types::MacAlgorithmSpec>>,
+    pub(crate) xks_key_configuration: ::std::option::Option<super::types::XksKeyConfigurationType>,
     pub(crate) current_key_material_id: ::std::option::Option<::std::string::String>,
 }
 impl KeyMetadataBuilder {
@@ -336,34 +336,34 @@
         &self.description
     }
     /// <p>The <a href="https://docs.aws.amazon.com/kms/latest/developerguide/kms-cryptography.html#cryptographic-operations">cryptographic operations</a> for which you can use the KMS key.</p>
-    pub fn key_usage(mut self, input: crate::types::KeyUsageType) -> Self {
+    pub fn key_usage(mut self, input: super::types::KeyUsageType) -> Self {
         self.key_usage = ::std::option::Option::Some(input);
         self
     }
     /// <p>The <a href="https://docs.aws.amazon.com/kms/latest/developerguide/kms-cryptography.html#cryptographic-operations">cryptographic operations</a> for which you can use the KMS key.</p>
-    pub fn set_key_usage(mut self, input: ::std::option::Option<crate::types::KeyUsageType>) -> Self {
+    pub fn set_key_usage(mut self, input: ::std::option::Option<super::types::KeyUsageType>) -> Self {
         self.key_usage = input;
         self
     }
     /// <p>The <a href="https://docs.aws.amazon.com/kms/latest/developerguide/kms-cryptography.html#cryptographic-operations">cryptographic operations</a> for which you can use the KMS key.</p>
-    pub fn get_key_usage(&self) -> &::std::option::Option<crate::types::KeyUsageType> {
+    pub fn get_key_usage(&self) -> &::std::option::Option<super::types::KeyUsageType> {
         &self.key_usage
     }
     /// <p>The current status of the KMS key.</p>
     /// <p>For more information about how key state affects the use of a KMS key, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-state.html">Key states of KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn key_state(mut self, input: crate::types::KeyState) -> Self {
+    pub fn key_state(mut self, input: super::types::KeyState) -> Self {
         self.key_state = ::std::option::Option::Some(input);
         self
     }
     /// <p>The current status of the KMS key.</p>
     /// <p>For more information about how key state affects the use of a KMS key, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-state.html">Key states of KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn set_key_state(mut self, input: ::std::option::Option<crate::types::KeyState>) -> Self {
+    pub fn set_key_state(mut self, input: ::std::option::Option<super::types::KeyState>) -> Self {
         self.key_state = input;
         self
     }
     /// <p>The current status of the KMS key.</p>
     /// <p>For more information about how key state affects the use of a KMS key, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-state.html">Key states of KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn get_key_state(&self) -> &::std::option::Option<crate::types::KeyState> {
+    pub fn get_key_state(&self) -> &::std::option::Option<super::types::KeyState> {
         &self.key_state
     }
     /// <p>The date and time after which KMS deletes this KMS key. This value is present only when the KMS key is scheduled for deletion, that is, when its <code>KeyState</code> is <code>PendingDeletion</code>.</p>
@@ -398,17 +398,17 @@
         &self.valid_to
     }
     /// <p>The source of the key material for the KMS key. When this value is <code>AWS_KMS</code>, KMS created the key material. When this value is <code>EXTERNAL</code>, the key material was imported or the KMS key doesn't have any key material. When this value is <code>AWS_CLOUDHSM</code>, the key material was created in the CloudHSM cluster associated with a custom key store.</p>
-    pub fn origin(mut self, input: crate::types::OriginType) -> Self {
+    pub fn origin(mut self, input: super::types::OriginType) -> Self {
         self.origin = ::std::option::Option::Some(input);
         self
     }
     /// <p>The source of the key material for the KMS key. When this value is <code>AWS_KMS</code>, KMS created the key material. When this value is <code>EXTERNAL</code>, the key material was imported or the KMS key doesn't have any key material. When this value is <code>AWS_CLOUDHSM</code>, the key material was created in the CloudHSM cluster associated with a custom key store.</p>
-    pub fn set_origin(mut self, input: ::std::option::Option<crate::types::OriginType>) -> Self {
+    pub fn set_origin(mut self, input: ::std::option::Option<super::types::OriginType>) -> Self {
         self.origin = input;
         self
     }
     /// <p>The source of the key material for the KMS key. When this value is <code>AWS_KMS</code>, KMS created the key material. When this value is <code>EXTERNAL</code>, the key material was imported or the KMS key doesn't have any key material. When this value is <code>AWS_CLOUDHSM</code>, the key material was created in the CloudHSM cluster associated with a custom key store.</p>
-    pub fn get_origin(&self) -> &::std::option::Option<crate::types::OriginType> {
+    pub fn get_origin(&self) -> &::std::option::Option<super::types::OriginType> {
         &self.origin
     }
     /// <p>A unique identifier for the <a href="https://docs.aws.amazon.com/kms/latest/developerguide/key-store-overview.html">custom key store</a> that contains the KMS key. This field is present only when the KMS key is created in a custom key store.</p>
@@ -440,37 +440,37 @@
         &self.cloud_hsm_cluster_id
     }
     /// <p>Specifies whether the KMS key's key material expires. This value is present only when <code>Origin</code> is <code>EXTERNAL</code>, otherwise this value is omitted.</p>
-    pub fn expiration_model(mut self, input: crate::types::ExpirationModelType) -> Self {
+    pub fn expiration_model(mut self, input: super::types::ExpirationModelType) -> Self {
         self.expiration_model = ::std::option::Option::Some(input);
         self
     }
     /// <p>Specifies whether the KMS key's key material expires. This value is present only when <code>Origin</code> is <code>EXTERNAL</code>, otherwise this value is omitted.</p>
-    pub fn set_expiration_model(mut self, input: ::std::option::Option<crate::types::ExpirationModelType>) -> Self {
+    pub fn set_expiration_model(mut self, input: ::std::option::Option<super::types::ExpirationModelType>) -> Self {
         self.expiration_model = input;
         self
     }
     /// <p>Specifies whether the KMS key's key material expires. This value is present only when <code>Origin</code> is <code>EXTERNAL</code>, otherwise this value is omitted.</p>
-    pub fn get_expiration_model(&self) -> &::std::option::Option<crate::types::ExpirationModelType> {
+    pub fn get_expiration_model(&self) -> &::std::option::Option<super::types::ExpirationModelType> {
         &self.expiration_model
     }
     /// <p>The manager of the KMS key. KMS keys in your Amazon Web Services account are either customer managed or Amazon Web Services managed. For more information about the difference, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#kms_keys">KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn key_manager(mut self, input: crate::types::KeyManagerType) -> Self {
+    pub fn key_manager(mut self, input: super::types::KeyManagerType) -> Self {
         self.key_manager = ::std::option::Option::Some(input);
         self
     }
     /// <p>The manager of the KMS key. KMS keys in your Amazon Web Services account are either customer managed or Amazon Web Services managed. For more information about the difference, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#kms_keys">KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn set_key_manager(mut self, input: ::std::option::Option<crate::types::KeyManagerType>) -> Self {
+    pub fn set_key_manager(mut self, input: ::std::option::Option<super::types::KeyManagerType>) -> Self {
         self.key_manager = input;
         self
     }
     /// <p>The manager of the KMS key. KMS keys in your Amazon Web Services account are either customer managed or Amazon Web Services managed. For more information about the difference, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#kms_keys">KMS keys</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn get_key_manager(&self) -> &::std::option::Option<crate::types::KeyManagerType> {
+    pub fn get_key_manager(&self) -> &::std::option::Option<super::types::KeyManagerType> {
         &self.key_manager
     }
     /// <p>Instead, use the <code>KeySpec</code> field.</p>
     /// <p>The <code>KeySpec</code> and <code>CustomerMasterKeySpec</code> fields have the same value. We recommend that you use the <code>KeySpec</code> field in your code. However, to avoid breaking changes, KMS supports both fields.</p>
     #[deprecated(note = "This field has been deprecated. Instead, use the KeySpec field.")]
-    pub fn customer_master_key_spec(mut self, input: crate::types::CustomerMasterKeySpec) -> Self {
+    pub fn customer_master_key_spec(mut self, input: super::types::CustomerMasterKeySpec) -> Self {
         self.customer_master_key_spec = ::std::option::Option::Some(input);
         self
     }
@@ -477,7 +477,7 @@
     /// <p>Instead, use the <code>KeySpec</code> field.</p>
     /// <p>The <code>KeySpec</code> and <code>CustomerMasterKeySpec</code> fields have the same value. We recommend that you use the <code>KeySpec</code> field in your code. However, to avoid breaking changes, KMS supports both fields.</p>
     #[deprecated(note = "This field has been deprecated. Instead, use the KeySpec field.")]
-    pub fn set_customer_master_key_spec(mut self, input: ::std::option::Option<crate::types::CustomerMasterKeySpec>) -> Self {
+    pub fn set_customer_master_key_spec(mut self, input: ::std::option::Option<super::types::CustomerMasterKeySpec>) -> Self {
         self.customer_master_key_spec = input;
         self
     }
@@ -484,21 +484,21 @@
     /// <p>Instead, use the <code>KeySpec</code> field.</p>
     /// <p>The <code>KeySpec</code> and <code>CustomerMasterKeySpec</code> fields have the same value. We recommend that you use the <code>KeySpec</code> field in your code. However, to avoid breaking changes, KMS supports both fields.</p>
     #[deprecated(note = "This field has been deprecated. Instead, use the KeySpec field.")]
-    pub fn get_customer_master_key_spec(&self) -> &::std::option::Option<crate::types::CustomerMasterKeySpec> {
+    pub fn get_customer_master_key_spec(&self) -> &::std::option::Option<super::types::CustomerMasterKeySpec> {
         &self.customer_master_key_spec
     }
     /// <p>Describes the type of key material in the KMS key.</p>
-    pub fn key_spec(mut self, input: crate::types::KeySpec) -> Self {
+    pub fn key_spec(mut self, input: super::types::KeySpec) -> Self {
         self.key_spec = ::std::option::Option::Some(input);
         self
     }
     /// <p>Describes the type of key material in the KMS key.</p>
-    pub fn set_key_spec(mut self, input: ::std::option::Option<crate::types::KeySpec>) -> Self {
+    pub fn set_key_spec(mut self, input: ::std::option::Option<super::types::KeySpec>) -> Self {
         self.key_spec = input;
         self
     }
     /// <p>Describes the type of key material in the KMS key.</p>
-    pub fn get_key_spec(&self) -> &::std::option::Option<crate::types::KeySpec> {
+    pub fn get_key_spec(&self) -> &::std::option::Option<super::types::KeySpec> {
         &self.key_spec
     }
     /// Appends an item to `encryption_algorithms`.
@@ -507,7 +507,7 @@
     ///
     /// <p>The encryption algorithms that the KMS key supports. You cannot use the KMS key with other encryption algorithms within KMS.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>ENCRYPT_DECRYPT</code>.</p>
-    pub fn encryption_algorithms(mut self, input: crate::types::EncryptionAlgorithmSpec) -> Self {
+    pub fn encryption_algorithms(mut self, input: super::types::EncryptionAlgorithmSpec) -> Self {
         let mut v = self.encryption_algorithms.unwrap_or_default();
         v.push(input);
         self.encryption_algorithms = ::std::option::Option::Some(v);
@@ -515,13 +515,13 @@
     }
     /// <p>The encryption algorithms that the KMS key supports. You cannot use the KMS key with other encryption algorithms within KMS.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>ENCRYPT_DECRYPT</code>.</p>
-    pub fn set_encryption_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::EncryptionAlgorithmSpec>>) -> Self {
+    pub fn set_encryption_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::EncryptionAlgorithmSpec>>) -> Self {
         self.encryption_algorithms = input;
         self
     }
     /// <p>The encryption algorithms that the KMS key supports. You cannot use the KMS key with other encryption algorithms within KMS.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>ENCRYPT_DECRYPT</code>.</p>
-    pub fn get_encryption_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::EncryptionAlgorithmSpec>> {
+    pub fn get_encryption_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<super::types::EncryptionAlgorithmSpec>> {
         &self.encryption_algorithms
     }
     /// Appends an item to `signing_algorithms`.
@@ -530,7 +530,7 @@
     ///
     /// <p>The signing algorithms that the KMS key supports. You cannot use the KMS key with other signing algorithms within KMS.</p>
     /// <p>This field appears only when the <code>KeyUsage</code> of the KMS key is <code>SIGN_VERIFY</code>.</p>
-    pub fn signing_algorithms(mut self, input: crate::types::SigningAlgorithmSpec) -> Self {
+    pub fn signing_algorithms(mut self, input: super::types::SigningAlgorithmSpec) -> Self {
         let mut v = self.signing_algorithms.unwrap_or_default();
         v.push(input);
         self.signing_algorithms = ::std::option::Option::Some(v);
@@ -538,13 +538,13 @@
     }
     /// <p>The signing algorithms that the KMS key supports. You cannot use the KMS key with other signing algorithms within KMS.</p>
     /// <p>This field appears only when the <code>KeyUsage</code> of the KMS key is <code>SIGN_VERIFY</code>.</p>
-    pub fn set_signing_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::SigningAlgorithmSpec>>) -> Self {
+    pub fn set_signing_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::SigningAlgorithmSpec>>) -> Self {
         self.signing_algorithms = input;
         self
     }
     /// <p>The signing algorithms that the KMS key supports. You cannot use the KMS key with other signing algorithms within KMS.</p>
     /// <p>This field appears only when the <code>KeyUsage</code> of the KMS key is <code>SIGN_VERIFY</code>.</p>
-    pub fn get_signing_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::SigningAlgorithmSpec>> {
+    pub fn get_signing_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<super::types::SigningAlgorithmSpec>> {
         &self.signing_algorithms
     }
     /// Appends an item to `key_agreement_algorithms`.
@@ -552,7 +552,7 @@
     /// To override the contents of this collection use [`set_key_agreement_algorithms`](Self::set_key_agreement_algorithms).
     ///
     /// <p>The key agreement algorithm used to derive a shared secret.</p>
-    pub fn key_agreement_algorithms(mut self, input: crate::types::KeyAgreementAlgorithmSpec) -> Self {
+    pub fn key_agreement_algorithms(mut self, input: super::types::KeyAgreementAlgorithmSpec) -> Self {
         let mut v = self.key_agreement_algorithms.unwrap_or_default();
         v.push(input);
         self.key_agreement_algorithms = ::std::option::Option::Some(v);
@@ -559,12 +559,12 @@
         self
     }
     /// <p>The key agreement algorithm used to derive a shared secret.</p>
-    pub fn set_key_agreement_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::KeyAgreementAlgorithmSpec>>) -> Self {
+    pub fn set_key_agreement_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::KeyAgreementAlgorithmSpec>>) -> Self {
         self.key_agreement_algorithms = input;
         self
     }
     /// <p>The key agreement algorithm used to derive a shared secret.</p>
-    pub fn get_key_agreement_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::KeyAgreementAlgorithmSpec>> {
+    pub fn get_key_agreement_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<super::types::KeyAgreementAlgorithmSpec>> {
         &self.key_agreement_algorithms
     }
     /// <p>Indicates whether the KMS key is a multi-Region (<code>True</code>) or regional (<code>False</code>) key. This value is <code>True</code> for multi-Region primary and replica keys and <code>False</code> for regional KMS keys.</p>
@@ -594,7 +594,7 @@
     /// <li>
     /// <p><code>ReplicaKeys</code> displays the key ARNs and Regions of all replica keys. This field includes the current KMS key if it is a replica key.</p></li>
     /// </ul>
-    pub fn multi_region_configuration(mut self, input: crate::types::MultiRegionConfiguration) -> Self {
+    pub fn multi_region_configuration(mut self, input: super::types::MultiRegionConfiguration) -> Self {
         self.multi_region_configuration = ::std::option::Option::Some(input);
         self
     }
@@ -608,7 +608,7 @@
     /// <li>
     /// <p><code>ReplicaKeys</code> displays the key ARNs and Regions of all replica keys. This field includes the current KMS key if it is a replica key.</p></li>
     /// </ul>
-    pub fn set_multi_region_configuration(mut self, input: ::std::option::Option<crate::types::MultiRegionConfiguration>) -> Self {
+    pub fn set_multi_region_configuration(mut self, input: ::std::option::Option<super::types::MultiRegionConfiguration>) -> Self {
         self.multi_region_configuration = input;
         self
     }
@@ -622,7 +622,7 @@
     /// <li>
     /// <p><code>ReplicaKeys</code> displays the key ARNs and Regions of all replica keys. This field includes the current KMS key if it is a replica key.</p></li>
     /// </ul>
-    pub fn get_multi_region_configuration(&self) -> &::std::option::Option<crate::types::MultiRegionConfiguration> {
+    pub fn get_multi_region_configuration(&self) -> &::std::option::Option<super::types::MultiRegionConfiguration> {
         &self.multi_region_configuration
     }
     /// <p>The waiting period before the primary key in a multi-Region key is deleted. This waiting period begins when the last of its replica keys is deleted. This value is present only when the <code>KeyState</code> of the KMS key is <code>PendingReplicaDeletion</code>. That indicates that the KMS key is the primary key in a multi-Region key, it is scheduled for deletion, and it still has existing replica keys.</p>
@@ -648,7 +648,7 @@
     ///
     /// <p>The message authentication code (MAC) algorithm that the HMAC KMS key supports.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>GENERATE_VERIFY_MAC</code>.</p>
-    pub fn mac_algorithms(mut self, input: crate::types::MacAlgorithmSpec) -> Self {
+    pub fn mac_algorithms(mut self, input: super::types::MacAlgorithmSpec) -> Self {
         let mut v = self.mac_algorithms.unwrap_or_default();
         v.push(input);
         self.mac_algorithms = ::std::option::Option::Some(v);
@@ -656,30 +656,30 @@
     }
     /// <p>The message authentication code (MAC) algorithm that the HMAC KMS key supports.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>GENERATE_VERIFY_MAC</code>.</p>
-    pub fn set_mac_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::MacAlgorithmSpec>>) -> Self {
+    pub fn set_mac_algorithms(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::MacAlgorithmSpec>>) -> Self {
         self.mac_algorithms = input;
         self
     }
     /// <p>The message authentication code (MAC) algorithm that the HMAC KMS key supports.</p>
     /// <p>This value is present only when the <code>KeyUsage</code> of the KMS key is <code>GENERATE_VERIFY_MAC</code>.</p>
-    pub fn get_mac_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::MacAlgorithmSpec>> {
+    pub fn get_mac_algorithms(&self) -> &::std::option::Option<::std::vec::Vec<super::types::MacAlgorithmSpec>> {
         &self.mac_algorithms
     }
     /// <p>Information about the external key that is associated with a KMS key in an external key store.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/keystore-external.html#concept-external-key">External key</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn xks_key_configuration(mut self, input: crate::types::XksKeyConfigurationType) -> Self {
+    pub fn xks_key_configuration(mut self, input: super::types::XksKeyConfigurationType) -> Self {
         self.xks_key_configuration = ::std::option::Option::Some(input);
         self
     }
     /// <p>Information about the external key that is associated with a KMS key in an external key store.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/keystore-external.html#concept-external-key">External key</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn set_xks_key_configuration(mut self, input: ::std::option::Option<crate::types::XksKeyConfigurationType>) -> Self {
+    pub fn set_xks_key_configuration(mut self, input: ::std::option::Option<super::types::XksKeyConfigurationType>) -> Self {
         self.xks_key_configuration = input;
         self
     }
     /// <p>Information about the external key that is associated with a KMS key in an external key store.</p>
     /// <p>For more information, see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/keystore-external.html#concept-external-key">External key</a> in the <i>Key Management Service Developer Guide</i>.</p>
-    pub fn get_xks_key_configuration(&self) -> &::std::option::Option<crate::types::XksKeyConfigurationType> {
+    pub fn get_xks_key_configuration(&self) -> &::std::option::Option<super::types::XksKeyConfigurationType> {
         &self.xks_key_configuration
     }
     /// <p>Identifies the current key material. This value is present for symmetric encryption keys with <code>AWS_KMS</code> or <code>EXTERNAL</code> origin. These KMS keys support automatic or on-demand key rotation and can have multiple key materials associated with them. KMS uses the current key material for both encryption and decryption, and the non-current key material for decryption operations only.</p>
@@ -699,8 +699,8 @@
     /// Consumes the builder and constructs a [`KeyMetadata`](crate::types::KeyMetadata).
     /// This method will fail if any of the following fields are not set:
     /// - [`key_id`](crate::types::builders::KeyMetadataBuilder::key_id)
-    pub fn build(self) -> ::std::result::Result<crate::types::KeyMetadata, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::types::KeyMetadata {
+    pub fn build(self) -> ::std::result::Result<super::types::KeyMetadata, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::types::KeyMetadata {
             aws_account_id: self.aws_account_id,
             key_id: self.key_id.ok_or_else(|| {
                 ::aws_smithy_types::error::operation::BuildError::missing_field(
