--- original
+++ modified
@@ -16,8 +16,8 @@
 }
 impl DeletePolicyInput {
     /// Creates a new builder-style object to manufacture [`DeletePolicyInput`](crate::operation::delete_policy::DeletePolicyInput).
-    pub fn builder() -> crate::operation::delete_policy::builders::DeletePolicyInputBuilder {
-        crate::operation::delete_policy::builders::DeletePolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_policy::builders::DeletePolicyInputBuilder {
+        super::super::super::operation::delete_policy::builders::DeletePolicyInputBuilder::default()
     }
 }

@@ -49,7 +49,7 @@
     /// Consumes the builder and constructs a [`DeletePolicyInput`](crate::operation::delete_policy::DeletePolicyInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_policy::DeletePolicyInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::delete_policy::DeletePolicyInput { policy_arn: self.policy_arn })
+    ) -> ::std::result::Result<super::super::super::operation::delete_policy::DeletePolicyInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::delete_policy::DeletePolicyInput { policy_arn: self.policy_arn })
     }
 }
