--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::retire_grant::RetireGrantOutput, crate::operation::retire_grant::RetireGrantError> {
+) -> std::result::Result<super::super::operation::retire_grant::RetireGrantOutput, super::super::operation::retire_grant::RetireGrantError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::retire_grant::RetireGrantError::unhandled(generic)),
+        None => return Err(super::super::operation::retire_grant::RetireGrantError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "DependencyTimeoutException" => crate::operation::retire_grant::RetireGrantError::DependencyTimeoutException({
+        "DependencyTimeoutException" => super::super::operation::retire_grant::RetireGrantError::DependencyTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DependencyTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::DependencyTimeoutExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "DryRunOperationException" => crate::operation::retire_grant::RetireGrantError::DryRunOperationException({
+        "DryRunOperationException" => super::super::operation::retire_grant::RetireGrantError::DryRunOperationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DryRunOperationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dry_run_operation_exception::de_dry_run_operation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::DryRunOperationExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_dry_run_operation_exception::de_dry_run_operation_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "InvalidArnException" => crate::operation::retire_grant::RetireGrantError::InvalidArnException({
+        "InvalidArnException" => super::super::operation::retire_grant::RetireGrantError::InvalidArnException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidArnExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_arn_exception::de_invalid_arn_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidArnExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_arn_exception::de_invalid_arn_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,13 +62,13 @@
             }
             tmp
         }),
-        "InvalidGrantIdException" => crate::operation::retire_grant::RetireGrantError::InvalidGrantIdException({
+        "InvalidGrantIdException" => super::super::operation::retire_grant::RetireGrantError::InvalidGrantIdException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidGrantIdExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_grant_id_exception::de_invalid_grant_id_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidGrantIdExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_grant_id_exception::de_invalid_grant_id_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -77,14 +77,14 @@
             }
             tmp
         }),
-        "InvalidGrantTokenException" => crate::operation::retire_grant::RetireGrantError::InvalidGrantTokenException({
+        "InvalidGrantTokenException" => super::super::operation::retire_grant::RetireGrantError::InvalidGrantTokenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidGrantTokenExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::InvalidGrantTokenExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_invalid_grant_token_exception::de_invalid_grant_token_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                    super::super::protocol_serde::shape_invalid_grant_token_exception::de_invalid_grant_token_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,13 +93,13 @@
             }
             tmp
         }),
-        "KMSInternalException" => crate::operation::retire_grant::RetireGrantError::KmsInternalException({
+        "KMSInternalException" => super::super::operation::retire_grant::RetireGrantError::KmsInternalException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInternalExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsInternalExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,13 +108,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::retire_grant::RetireGrantError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::super::operation::retire_grant::RetireGrantError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -123,13 +123,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::retire_grant::RetireGrantError::NotFoundException({
+        "NotFoundException" => super::super::operation::retire_grant::RetireGrantError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::retire_grant::RetireGrantError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::retire_grant::RetireGrantError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -138,7 +138,7 @@
             }
             tmp
         }),
-        _ => crate::operation::retire_grant::RetireGrantError::generic(generic),
+        _ => super::super::operation::retire_grant::RetireGrantError::generic(generic),
     })
 }

@@ -147,10 +147,10 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::retire_grant::RetireGrantOutput, crate::operation::retire_grant::RetireGrantError> {
+) -> std::result::Result<super::super::operation::retire_grant::RetireGrantOutput, super::super::operation::retire_grant::RetireGrantError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::retire_grant::builders::RetireGrantOutputBuilder::default();
+        let mut output = super::super::operation::retire_grant::builders::RetireGrantOutputBuilder::default();
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -157,11 +157,11 @@
 }

 pub fn ser_retire_grant_input(
-    input: &crate::operation::retire_grant::RetireGrantInput,
+    input: &super::super::operation::retire_grant::RetireGrantInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_retire_grant_input::ser_retire_grant_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_retire_grant_input::ser_retire_grant_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
