--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::decrypt::DecryptOutput, crate::operation::decrypt::DecryptError> {
+) -> std::result::Result<super::super::operation::decrypt::DecryptOutput, super::super::operation::decrypt::DecryptError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::decrypt::DecryptError::unhandled(generic)),
+        None => return Err(super::super::operation::decrypt::DecryptError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "DependencyTimeoutException" => crate::operation::decrypt::DecryptError::DependencyTimeoutException({
+        "DependencyTimeoutException" => super::super::operation::decrypt::DecryptError::DependencyTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DependencyTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::DependencyTimeoutExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "DisabledException" => crate::operation::decrypt::DecryptError::DisabledException({
+        "DisabledException" => super::super::operation::decrypt::DecryptError::DisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_disabled_exception::de_disabled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::DisabledExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_disabled_exception::de_disabled_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "DryRunOperationException" => crate::operation::decrypt::DecryptError::DryRunOperationException({
+        "DryRunOperationException" => super::super::operation::decrypt::DecryptError::DryRunOperationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DryRunOperationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dry_run_operation_exception::de_dry_run_operation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::DryRunOperationExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_dry_run_operation_exception::de_dry_run_operation_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,13 +62,13 @@
             }
             tmp
         }),
-        "IncorrectKeyException" => crate::operation::decrypt::DecryptError::IncorrectKeyException({
+        "IncorrectKeyException" => super::super::operation::decrypt::DecryptError::IncorrectKeyException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::IncorrectKeyExceptionBuilder::default();
-                output = crate::protocol_serde::shape_incorrect_key_exception::de_incorrect_key_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::IncorrectKeyExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_incorrect_key_exception::de_incorrect_key_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -77,13 +77,13 @@
             }
             tmp
         }),
-        "InvalidCiphertextException" => crate::operation::decrypt::DecryptError::InvalidCiphertextException({
+        "InvalidCiphertextException" => super::super::operation::decrypt::DecryptError::InvalidCiphertextException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidCiphertextExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_ciphertext_exception::de_invalid_ciphertext_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidCiphertextExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_ciphertext_exception::de_invalid_ciphertext_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -92,14 +92,14 @@
             }
             tmp
         }),
-        "InvalidGrantTokenException" => crate::operation::decrypt::DecryptError::InvalidGrantTokenException({
+        "InvalidGrantTokenException" => super::super::operation::decrypt::DecryptError::InvalidGrantTokenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidGrantTokenExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::InvalidGrantTokenExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_invalid_grant_token_exception::de_invalid_grant_token_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                    super::super::protocol_serde::shape_invalid_grant_token_exception::de_invalid_grant_token_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,13 +108,13 @@
             }
             tmp
         }),
-        "InvalidKeyUsageException" => crate::operation::decrypt::DecryptError::InvalidKeyUsageException({
+        "InvalidKeyUsageException" => super::super::operation::decrypt::DecryptError::InvalidKeyUsageException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidKeyUsageExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_key_usage_exception::de_invalid_key_usage_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidKeyUsageExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_key_usage_exception::de_invalid_key_usage_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -123,13 +123,13 @@
             }
             tmp
         }),
-        "KeyUnavailableException" => crate::operation::decrypt::DecryptError::KeyUnavailableException({
+        "KeyUnavailableException" => super::super::operation::decrypt::DecryptError::KeyUnavailableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KeyUnavailableExceptionBuilder::default();
-                output = crate::protocol_serde::shape_key_unavailable_exception::de_key_unavailable_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::KeyUnavailableExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_key_unavailable_exception::de_key_unavailable_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -138,13 +138,13 @@
             }
             tmp
         }),
-        "KMSInternalException" => crate::operation::decrypt::DecryptError::KmsInternalException({
+        "KMSInternalException" => super::super::operation::decrypt::DecryptError::KmsInternalException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInternalExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsInternalExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -153,13 +153,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::decrypt::DecryptError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::super::operation::decrypt::DecryptError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -168,13 +168,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::decrypt::DecryptError::NotFoundException({
+        "NotFoundException" => super::super::operation::decrypt::DecryptError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+                let mut output = super::super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -183,7 +183,7 @@
             }
             tmp
         }),
-        _ => crate::operation::decrypt::DecryptError::generic(generic),
+        _ => super::super::operation::decrypt::DecryptError::generic(generic),
     })
 }

@@ -192,12 +192,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::decrypt::DecryptOutput, crate::operation::decrypt::DecryptError> {
+) -> std::result::Result<super::super::operation::decrypt::DecryptOutput, super::super::operation::decrypt::DecryptError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::decrypt::builders::DecryptOutputBuilder::default();
+        let mut output = super::super::operation::decrypt::builders::DecryptOutputBuilder::default();
         output =
-            crate::protocol_serde::shape_decrypt::de_decrypt(_response_body, output).map_err(crate::operation::decrypt::DecryptError::unhandled)?;
+            super::super::protocol_serde::shape_decrypt::de_decrypt(_response_body, output).map_err(super::super::operation::decrypt::DecryptError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -204,11 +204,11 @@
 }

 pub fn ser_decrypt_input(
-    input: &crate::operation::decrypt::DecryptInput,
+    input: &super::super::operation::decrypt::DecryptInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_decrypt_input::ser_decrypt_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_decrypt_input::ser_decrypt_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -215,9 +215,9 @@

 pub(crate) fn de_decrypt(
     _value: &[u8],
-    mut builder: crate::operation::decrypt::builders::DecryptOutputBuilder,
-) -> ::std::result::Result<crate::operation::decrypt::builders::DecryptOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    mut builder: super::super::operation::decrypt::builders::DecryptOutputBuilder,
+) -> ::std::result::Result<super::super::operation::decrypt::builders::DecryptOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -239,7 +239,7 @@
                 "EncryptionAlgorithm" => {
                     builder = builder.set_encryption_algorithm(
                         ::aws_smithy_json::deserialize::token::expect_string_or_null(tokens.next())?
-                            .map(|s| s.to_unescaped().map(|u| crate::types::EncryptionAlgorithmSpec::from(u.as_ref())))
+                            .map(|s| s.to_unescaped().map(|u| super::super::types::EncryptionAlgorithmSpec::from(u.as_ref())))
                             .transpose()?,
                     );
                 }
