--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_key::CreateKeyOutput, crate::operation::create_key::CreateKeyError> {
+) -> std::result::Result<super::super::operation::create_key::CreateKeyOutput, super::super::operation::create_key::CreateKeyError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+    let mut generic_builder = super::super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::create_key::CreateKeyError::unhandled(generic)),
+        None => return Err(super::super::operation::create_key::CreateKeyError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
         "CloudHsmClusterInvalidConfigurationException" => {
-            crate::operation::create_key::CreateKeyError::CloudHsmClusterInvalidConfigurationException({
+            super::super::operation::create_key::CreateKeyError::CloudHsmClusterInvalidConfigurationException({
                 #[allow(unused_mut)]
                 let mut tmp = {
                     #[allow(unused_mut)]
-                    let mut output = crate::types::error::builders::CloudHsmClusterInvalidConfigurationExceptionBuilder::default();
-                    output = crate::protocol_serde::shape_cloud_hsm_cluster_invalid_configuration_exception::de_cloud_hsm_cluster_invalid_configuration_exception_json_err(_response_body, output).map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                    let mut output = super::super::types::error::builders::CloudHsmClusterInvalidConfigurationExceptionBuilder::default();
+                    output = super::super::protocol_serde::shape_cloud_hsm_cluster_invalid_configuration_exception::de_cloud_hsm_cluster_invalid_configuration_exception_json_err(_response_body, output).map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                     let output = output.meta(generic);
                     output.build()
                 };
@@ -33,16 +33,16 @@
                 tmp
             })
         }
-        "CustomKeyStoreInvalidStateException" => crate::operation::create_key::CreateKeyError::CustomKeyStoreInvalidStateException({
+        "CustomKeyStoreInvalidStateException" => super::super::operation::create_key::CreateKeyError::CustomKeyStoreInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CustomKeyStoreInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_custom_key_store_invalid_state_exception::de_custom_key_store_invalid_state_exception_json_err(
+                let mut output = super::super::types::error::builders::CustomKeyStoreInvalidStateExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_custom_key_store_invalid_state_exception::de_custom_key_store_invalid_state_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -51,16 +51,16 @@
             }
             tmp
         }),
-        "CustomKeyStoreNotFoundException" => crate::operation::create_key::CreateKeyError::CustomKeyStoreNotFoundException({
+        "CustomKeyStoreNotFoundException" => super::super::operation::create_key::CreateKeyError::CustomKeyStoreNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::CustomKeyStoreNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_custom_key_store_not_found_exception::de_custom_key_store_not_found_exception_json_err(
+                let mut output = super::super::types::error::builders::CustomKeyStoreNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_custom_key_store_not_found_exception::de_custom_key_store_not_found_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -69,13 +69,13 @@
             }
             tmp
         }),
-        "DependencyTimeoutException" => crate::operation::create_key::CreateKeyError::DependencyTimeoutException({
+        "DependencyTimeoutException" => super::super::operation::create_key::CreateKeyError::DependencyTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DependencyTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                let mut output = super::super::types::error::builders::DependencyTimeoutExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -84,13 +84,13 @@
             }
             tmp
         }),
-        "InvalidArnException" => crate::operation::create_key::CreateKeyError::InvalidArnException({
+        "InvalidArnException" => super::super::operation::create_key::CreateKeyError::InvalidArnException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidArnExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_arn_exception::de_invalid_arn_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                let mut output = super::super::types::error::builders::InvalidArnExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_invalid_arn_exception::de_invalid_arn_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -99,13 +99,13 @@
             }
             tmp
         }),
-        "KMSInternalException" => crate::operation::create_key::CreateKeyError::KmsInternalException({
+        "KMSInternalException" => super::super::operation::create_key::CreateKeyError::KmsInternalException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInternalExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                let mut output = super::super::types::error::builders::KmsInternalExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -114,13 +114,13 @@
             }
             tmp
         }),
-        "LimitExceededException" => crate::operation::create_key::CreateKeyError::LimitExceededException({
+        "LimitExceededException" => super::super::operation::create_key::CreateKeyError::LimitExceededException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::LimitExceededExceptionBuilder::default();
-                output = crate::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                let mut output = super::super::types::error::builders::LimitExceededExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_limit_exceeded_exception::de_limit_exceeded_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -129,16 +129,16 @@
             }
             tmp
         }),
-        "MalformedPolicyDocumentException" => crate::operation::create_key::CreateKeyError::MalformedPolicyDocumentException({
+        "MalformedPolicyDocumentException" => super::super::operation::create_key::CreateKeyError::MalformedPolicyDocumentException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::MalformedPolicyDocumentExceptionBuilder::default();
-                output = crate::protocol_serde::shape_malformed_policy_document_exception::de_malformed_policy_document_exception_json_err(
+                let mut output = super::super::types::error::builders::MalformedPolicyDocumentExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_malformed_policy_document_exception::de_malformed_policy_document_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -147,13 +147,13 @@
             }
             tmp
         }),
-        "TagException" => crate::operation::create_key::CreateKeyError::TagException({
+        "TagException" => super::super::operation::create_key::CreateKeyError::TagException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::TagExceptionBuilder::default();
-                output = crate::protocol_serde::shape_tag_exception::de_tag_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                let mut output = super::super::types::error::builders::TagExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_tag_exception::de_tag_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -162,14 +162,14 @@
             }
             tmp
         }),
-        "UnsupportedOperationException" => crate::operation::create_key::CreateKeyError::UnsupportedOperationException({
+        "UnsupportedOperationException" => super::super::operation::create_key::CreateKeyError::UnsupportedOperationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::UnsupportedOperationExceptionBuilder::default();
+                let mut output = super::super::types::error::builders::UnsupportedOperationExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_unsupported_operation_exception::de_unsupported_operation_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                    super::super::protocol_serde::shape_unsupported_operation_exception::de_unsupported_operation_exception_json_err(_response_body, output)
+                        .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -178,16 +178,16 @@
             }
             tmp
         }),
-        "XksKeyAlreadyInUseException" => crate::operation::create_key::CreateKeyError::XksKeyAlreadyInUseException({
+        "XksKeyAlreadyInUseException" => super::super::operation::create_key::CreateKeyError::XksKeyAlreadyInUseException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::XksKeyAlreadyInUseExceptionBuilder::default();
-                output = crate::protocol_serde::shape_xks_key_already_in_use_exception::de_xks_key_already_in_use_exception_json_err(
+                let mut output = super::super::types::error::builders::XksKeyAlreadyInUseExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_xks_key_already_in_use_exception::de_xks_key_already_in_use_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -196,16 +196,16 @@
             }
             tmp
         }),
-        "XksKeyInvalidConfigurationException" => crate::operation::create_key::CreateKeyError::XksKeyInvalidConfigurationException({
+        "XksKeyInvalidConfigurationException" => super::super::operation::create_key::CreateKeyError::XksKeyInvalidConfigurationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::XksKeyInvalidConfigurationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_xks_key_invalid_configuration_exception::de_xks_key_invalid_configuration_exception_json_err(
+                let mut output = super::super::types::error::builders::XksKeyInvalidConfigurationExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_xks_key_invalid_configuration_exception::de_xks_key_invalid_configuration_exception_json_err(
                     _response_body,
                     output,
                 )
-                .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -214,13 +214,13 @@
             }
             tmp
         }),
-        "XksKeyNotFoundException" => crate::operation::create_key::CreateKeyError::XksKeyNotFoundException({
+        "XksKeyNotFoundException" => super::super::operation::create_key::CreateKeyError::XksKeyNotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::XksKeyNotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_xks_key_not_found_exception::de_xks_key_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+                let mut output = super::super::types::error::builders::XksKeyNotFoundExceptionBuilder::default();
+                output = super::super::protocol_serde::shape_xks_key_not_found_exception::de_xks_key_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -229,7 +229,7 @@
             }
             tmp
         }),
-        _ => crate::operation::create_key::CreateKeyError::generic(generic),
+        _ => super::super::operation::create_key::CreateKeyError::generic(generic),
     })
 }

@@ -238,12 +238,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::create_key::CreateKeyOutput, crate::operation::create_key::CreateKeyError> {
+) -> std::result::Result<super::super::operation::create_key::CreateKeyOutput, super::super::operation::create_key::CreateKeyError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::create_key::builders::CreateKeyOutputBuilder::default();
-        output = crate::protocol_serde::shape_create_key::de_create_key(_response_body, output)
-            .map_err(crate::operation::create_key::CreateKeyError::unhandled)?;
+        let mut output = super::super::operation::create_key::builders::CreateKeyOutputBuilder::default();
+        output = super::super::protocol_serde::shape_create_key::de_create_key(_response_body, output)
+            .map_err(super::super::operation::create_key::CreateKeyError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -250,11 +250,11 @@
 }

 pub fn ser_create_key_input(
-    input: &crate::operation::create_key::CreateKeyInput,
+    input: &super::super::operation::create_key::CreateKeyInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_create_key_input::ser_create_key_input_input(&mut object, input)?;
+    super::super::protocol_serde::shape_create_key_input::ser_create_key_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -261,9 +261,9 @@

 pub(crate) fn de_create_key(
     _value: &[u8],
-    mut builder: crate::operation::create_key::builders::CreateKeyOutputBuilder,
-) -> ::std::result::Result<crate::operation::create_key::builders::CreateKeyOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    mut builder: super::super::operation::create_key::builders::CreateKeyOutputBuilder,
+) -> ::std::result::Result<super::super::operation::create_key::builders::CreateKeyOutputBuilder, ::aws_smithy_json::deserialize::error::DeserializeError> {
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
@@ -273,7 +273,7 @@
             Some(::aws_smithy_json::deserialize::Token::EndObject { .. }) => break,
             Some(::aws_smithy_json::deserialize::Token::ObjectKey { key, .. }) => match key.to_unescaped()?.as_ref() {
                 "KeyMetadata" => {
-                    builder = builder.set_key_metadata(crate::protocol_serde::shape_key_metadata::de_key_metadata(tokens, _value, depth + 1)?);
+                    builder = builder.set_key_metadata(super::super::protocol_serde::shape_key_metadata::de_key_metadata(tokens, _value, depth + 1)?);
                 }
                 _ => ::aws_smithy_json::deserialize::token::skip_value(tokens)?,
             },
