--- original
+++ modified
@@ -4,26 +4,26 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::generate_data_key::GenerateDataKeyOutput, crate::operation::generate_data_key::GenerateDataKeyError> {
+) -> std::result::Result<super::operation::generate_data_key::GenerateDataKeyOutput, super::operation::generate_data_key::GenerateDataKeyError> {
     #[allow(unused_mut)]
-    let mut generic_builder = crate::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
-        .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+    let mut generic_builder = super::protocol_serde::parse_http_error_metadata(_response_status, _response_headers, _response_body)
+        .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
     generic_builder = ::aws_types::request_id::apply_request_id(generic_builder, _response_headers);
     let generic = generic_builder.build();
     let error_code = match generic.code() {
         Some(code) => code,
-        None => return Err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled(generic)),
+        None => return Err(super::operation::generate_data_key::GenerateDataKeyError::unhandled(generic)),
     };

     let _error_message = generic.message().map(|msg| msg.to_owned());
     Err(match error_code {
-        "DependencyTimeoutException" => crate::operation::generate_data_key::GenerateDataKeyError::DependencyTimeoutException({
+        "DependencyTimeoutException" => super::operation::generate_data_key::GenerateDataKeyError::DependencyTimeoutException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DependencyTimeoutExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::DependencyTimeoutExceptionBuilder::default();
+                output = super::protocol_serde::shape_dependency_timeout_exception::de_dependency_timeout_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -32,13 +32,13 @@
             }
             tmp
         }),
-        "DisabledException" => crate::operation::generate_data_key::GenerateDataKeyError::DisabledException({
+        "DisabledException" => super::operation::generate_data_key::GenerateDataKeyError::DisabledException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DisabledExceptionBuilder::default();
-                output = crate::protocol_serde::shape_disabled_exception::de_disabled_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::DisabledExceptionBuilder::default();
+                output = super::protocol_serde::shape_disabled_exception::de_disabled_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -47,13 +47,13 @@
             }
             tmp
         }),
-        "DryRunOperationException" => crate::operation::generate_data_key::GenerateDataKeyError::DryRunOperationException({
+        "DryRunOperationException" => super::operation::generate_data_key::GenerateDataKeyError::DryRunOperationException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::DryRunOperationExceptionBuilder::default();
-                output = crate::protocol_serde::shape_dry_run_operation_exception::de_dry_run_operation_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::DryRunOperationExceptionBuilder::default();
+                output = super::protocol_serde::shape_dry_run_operation_exception::de_dry_run_operation_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -62,14 +62,14 @@
             }
             tmp
         }),
-        "InvalidGrantTokenException" => crate::operation::generate_data_key::GenerateDataKeyError::InvalidGrantTokenException({
+        "InvalidGrantTokenException" => super::operation::generate_data_key::GenerateDataKeyError::InvalidGrantTokenException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidGrantTokenExceptionBuilder::default();
+                let mut output = super::types::error::builders::InvalidGrantTokenExceptionBuilder::default();
                 output =
-                    crate::protocol_serde::shape_invalid_grant_token_exception::de_invalid_grant_token_exception_json_err(_response_body, output)
-                        .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                    super::protocol_serde::shape_invalid_grant_token_exception::de_invalid_grant_token_exception_json_err(_response_body, output)
+                        .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -78,13 +78,13 @@
             }
             tmp
         }),
-        "InvalidKeyUsageException" => crate::operation::generate_data_key::GenerateDataKeyError::InvalidKeyUsageException({
+        "InvalidKeyUsageException" => super::operation::generate_data_key::GenerateDataKeyError::InvalidKeyUsageException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::InvalidKeyUsageExceptionBuilder::default();
-                output = crate::protocol_serde::shape_invalid_key_usage_exception::de_invalid_key_usage_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::InvalidKeyUsageExceptionBuilder::default();
+                output = super::protocol_serde::shape_invalid_key_usage_exception::de_invalid_key_usage_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -93,13 +93,13 @@
             }
             tmp
         }),
-        "KeyUnavailableException" => crate::operation::generate_data_key::GenerateDataKeyError::KeyUnavailableException({
+        "KeyUnavailableException" => super::operation::generate_data_key::GenerateDataKeyError::KeyUnavailableException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KeyUnavailableExceptionBuilder::default();
-                output = crate::protocol_serde::shape_key_unavailable_exception::de_key_unavailable_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::KeyUnavailableExceptionBuilder::default();
+                output = super::protocol_serde::shape_key_unavailable_exception::de_key_unavailable_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -108,13 +108,13 @@
             }
             tmp
         }),
-        "KMSInternalException" => crate::operation::generate_data_key::GenerateDataKeyError::KmsInternalException({
+        "KMSInternalException" => super::operation::generate_data_key::GenerateDataKeyError::KmsInternalException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInternalExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::KmsInternalExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_internal_exception::de_kms_internal_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -123,13 +123,13 @@
             }
             tmp
         }),
-        "KMSInvalidStateException" => crate::operation::generate_data_key::GenerateDataKeyError::KmsInvalidStateException({
+        "KMSInvalidStateException" => super::operation::generate_data_key::GenerateDataKeyError::KmsInvalidStateException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::KmsInvalidStateExceptionBuilder::default();
-                output = crate::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::KmsInvalidStateExceptionBuilder::default();
+                output = super::protocol_serde::shape_kms_invalid_state_exception::de_kms_invalid_state_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -138,13 +138,13 @@
             }
             tmp
         }),
-        "NotFoundException" => crate::operation::generate_data_key::GenerateDataKeyError::NotFoundException({
+        "NotFoundException" => super::operation::generate_data_key::GenerateDataKeyError::NotFoundException({
             #[allow(unused_mut)]
             let mut tmp = {
                 #[allow(unused_mut)]
-                let mut output = crate::types::error::builders::NotFoundExceptionBuilder::default();
-                output = crate::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
-                    .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+                let mut output = super::types::error::builders::NotFoundExceptionBuilder::default();
+                output = super::protocol_serde::shape_not_found_exception::de_not_found_exception_json_err(_response_body, output)
+                    .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
                 let output = output.meta(generic);
                 output.build()
             };
@@ -153,7 +153,7 @@
             }
             tmp
         }),
-        _ => crate::operation::generate_data_key::GenerateDataKeyError::generic(generic),
+        _ => super::operation::generate_data_key::GenerateDataKeyError::generic(generic),
     })
 }

@@ -162,12 +162,12 @@
     _response_status: u16,
     _response_headers: &::aws_smithy_runtime_api::http::Headers,
     _response_body: &[u8],
-) -> std::result::Result<crate::operation::generate_data_key::GenerateDataKeyOutput, crate::operation::generate_data_key::GenerateDataKeyError> {
+) -> std::result::Result<super::operation::generate_data_key::GenerateDataKeyOutput, super::operation::generate_data_key::GenerateDataKeyError> {
     Ok({
         #[allow(unused_mut)]
-        let mut output = crate::operation::generate_data_key::builders::GenerateDataKeyOutputBuilder::default();
-        output = crate::protocol_serde::shape_generate_data_key::de_generate_data_key(_response_body, output)
-            .map_err(crate::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
+        let mut output = super::operation::generate_data_key::builders::GenerateDataKeyOutputBuilder::default();
+        output = super::protocol_serde::shape_generate_data_key::de_generate_data_key(_response_body, output)
+            .map_err(super::operation::generate_data_key::GenerateDataKeyError::unhandled)?;
         output._set_request_id(::aws_types::request_id::RequestId::request_id(_response_headers).map(str::to_string));
         output.build()
     })
@@ -174,11 +174,11 @@
 }

 pub fn ser_generate_data_key_input(
-    input: &crate::operation::generate_data_key::GenerateDataKeyInput,
+    input: &super::operation::generate_data_key::GenerateDataKeyInput,
 ) -> ::std::result::Result<::aws_smithy_types::body::SdkBody, ::aws_smithy_types::error::operation::SerializationError> {
     let mut out = String::new();
     let mut object = ::aws_smithy_json::serialize::JsonObjectWriter::new(&mut out);
-    crate::protocol_serde::shape_generate_data_key_input::ser_generate_data_key_input_input(&mut object, input)?;
+    super::protocol_serde::shape_generate_data_key_input::ser_generate_data_key_input_input(&mut object, input)?;
     object.finish();
     Ok(::aws_smithy_types::body::SdkBody::from(out))
 }
@@ -185,12 +185,12 @@

 pub(crate) fn de_generate_data_key(
     _value: &[u8],
-    mut builder: crate::operation::generate_data_key::builders::GenerateDataKeyOutputBuilder,
+    mut builder: super::operation::generate_data_key::builders::GenerateDataKeyOutputBuilder,
 ) -> ::std::result::Result<
-    crate::operation::generate_data_key::builders::GenerateDataKeyOutputBuilder,
+    super::operation::generate_data_key::builders::GenerateDataKeyOutputBuilder,
     ::aws_smithy_json::deserialize::error::DeserializeError,
 > {
-    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(crate::protocol_serde::or_empty_doc(_value)).peekable();
+    let mut tokens_owned = ::aws_smithy_json::deserialize::json_token_iter(super::protocol_serde::or_empty_doc(_value)).peekable();
     let tokens = &mut tokens_owned;
     #[allow(unused_variables)]
     let depth = 0u32;
