--- original
+++ modified
@@ -42,8 +42,8 @@
 }
 impl UpdateAssumeRolePolicyInput {
     /// Creates a new builder-style object to manufacture [`UpdateAssumeRolePolicyInput`](crate::operation::update_assume_role_policy::UpdateAssumeRolePolicyInput).
-    pub fn builder() -> crate::operation::update_assume_role_policy::builders::UpdateAssumeRolePolicyInputBuilder {
-        crate::operation::update_assume_role_policy::builders::UpdateAssumeRolePolicyInputBuilder::default()
+    pub fn builder() -> super::operation::update_assume_role_policy::builders::UpdateAssumeRolePolicyInputBuilder {
+        super::operation::update_assume_role_policy::builders::UpdateAssumeRolePolicyInputBuilder::default()
     }
 }

@@ -122,10 +122,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_assume_role_policy::UpdateAssumeRolePolicyInput,
+        super::operation::update_assume_role_policy::UpdateAssumeRolePolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_assume_role_policy::UpdateAssumeRolePolicyInput {
+        ::std::result::Result::Ok(super::operation::update_assume_role_policy::UpdateAssumeRolePolicyInput {
             role_name: self.role_name,
             policy_document: self.policy_document,
         })
