--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListRolePoliciesInput {
     /// Creates a new builder-style object to manufacture [`ListRolePoliciesInput`](crate::operation::list_role_policies::ListRolePoliciesInput).
-    pub fn builder() -> crate::operation::list_role_policies::builders::ListRolePoliciesInputBuilder {
-        crate::operation::list_role_policies::builders::ListRolePoliciesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_role_policies::builders::ListRolePoliciesInputBuilder {
+        super::super::super::operation::list_role_policies::builders::ListRolePoliciesInputBuilder::default()
     }
 }

@@ -96,8 +96,8 @@
     /// Consumes the builder and constructs a [`ListRolePoliciesInput`](crate::operation::list_role_policies::ListRolePoliciesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_role_policies::ListRolePoliciesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_role_policies::ListRolePoliciesInput {
+    ) -> ::std::result::Result<super::super::super::operation::list_role_policies::ListRolePoliciesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_role_policies::ListRolePoliciesInput {
             role_name: self.role_name,
             marker: self.marker,
             max_items: self.max_items,
