--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl UpdateUserInput {
     /// Creates a new builder-style object to manufacture [`UpdateUserInput`](crate::operation::update_user::UpdateUserInput).
-    pub fn builder() -> crate::operation::update_user::builders::UpdateUserInputBuilder {
-        crate::operation::update_user::builders::UpdateUserInputBuilder::default()
+    pub fn builder() -> super::operation::update_user::builders::UpdateUserInputBuilder {
+        super::operation::update_user::builders::UpdateUserInputBuilder::default()
     }
 }

@@ -99,8 +99,8 @@
         &self.new_user_name
     }
     /// Consumes the builder and constructs a [`UpdateUserInput`](crate::operation::update_user::UpdateUserInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::update_user::UpdateUserInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_user::UpdateUserInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::update_user::UpdateUserInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::update_user::UpdateUserInput {
             user_name: self.user_name,
             new_path: self.new_path,
             new_user_name: self.new_user_name,
