--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl UpdateGroupInput {
     /// Creates a new builder-style object to manufacture [`UpdateGroupInput`](crate::operation::update_group::UpdateGroupInput).
-    pub fn builder() -> crate::operation::update_group::builders::UpdateGroupInputBuilder {
-        crate::operation::update_group::builders::UpdateGroupInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::update_group::builders::UpdateGroupInputBuilder {
+        super::super::super::operation::update_group::builders::UpdateGroupInputBuilder::default()
     }
 }

@@ -99,8 +99,8 @@
         &self.new_group_name
     }
     /// Consumes the builder and constructs a [`UpdateGroupInput`](crate::operation::update_group::UpdateGroupInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::update_group::UpdateGroupInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_group::UpdateGroupInput {
+    pub fn build(self) -> ::std::result::Result<super::super::super::operation::update_group::UpdateGroupInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::update_group::UpdateGroupInput {
             group_name: self.group_name,
             new_path: self.new_path,
             new_group_name: self.new_group_name,
