--- original
+++ modified
@@ -12,16 +12,16 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads. <code>PAY_PER_REQUEST</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/on-demand-capacity-mode.html">On-demand capacity mode</a>.</p></li>
     /// </ul>
-    pub global_table_billing_mode: ::std::option::Option<crate::types::BillingMode>,
+    pub global_table_billing_mode: ::std::option::Option<super::types::BillingMode>,
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException.</code></p>
     pub global_table_provisioned_write_capacity_units: ::std::option::Option<i64>,
     /// <p>Auto scaling settings for managing provisioned write capacity for the global table.</p>
-    pub global_table_provisioned_write_capacity_auto_scaling_settings_update: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+    pub global_table_provisioned_write_capacity_auto_scaling_settings_update: ::std::option::Option<super::types::AutoScalingSettingsUpdate>,
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     pub global_table_global_secondary_index_settings_update:
-        ::std::option::Option<::std::vec::Vec<crate::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>>,
+        ::std::option::Option<::std::vec::Vec<super::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>>,
     /// <p>Represents the settings for a global table in a Region that will be modified.</p>
-    pub replica_settings_update: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaSettingsUpdate>>,
+    pub replica_settings_update: ::std::option::Option<::std::vec::Vec<super::types::ReplicaSettingsUpdate>>,
 }
 impl UpdateGlobalTableSettingsInput {
     /// <p>The name of the global table</p>
@@ -35,7 +35,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads. <code>PAY_PER_REQUEST</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/on-demand-capacity-mode.html">On-demand capacity mode</a>.</p></li>
     /// </ul>
-    pub fn global_table_billing_mode(&self) -> ::std::option::Option<&crate::types::BillingMode> {
+    pub fn global_table_billing_mode(&self) -> ::std::option::Option<&super::types::BillingMode> {
         self.global_table_billing_mode.as_ref()
     }
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException.</code></p>
@@ -45,26 +45,26 @@
     /// <p>Auto scaling settings for managing provisioned write capacity for the global table.</p>
     pub fn global_table_provisioned_write_capacity_auto_scaling_settings_update(
         &self,
-    ) -> ::std::option::Option<&crate::types::AutoScalingSettingsUpdate> {
+    ) -> ::std::option::Option<&super::types::AutoScalingSettingsUpdate> {
         self.global_table_provisioned_write_capacity_auto_scaling_settings_update.as_ref()
     }
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.global_table_global_secondary_index_settings_update.is_none()`.
-    pub fn global_table_global_secondary_index_settings_update(&self) -> &[crate::types::GlobalTableGlobalSecondaryIndexSettingsUpdate] {
+    pub fn global_table_global_secondary_index_settings_update(&self) -> &[super::types::GlobalTableGlobalSecondaryIndexSettingsUpdate] {
         self.global_table_global_secondary_index_settings_update.as_deref().unwrap_or_default()
     }
     /// <p>Represents the settings for a global table in a Region that will be modified.</p>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.replica_settings_update.is_none()`.
-    pub fn replica_settings_update(&self) -> &[crate::types::ReplicaSettingsUpdate] {
+    pub fn replica_settings_update(&self) -> &[super::types::ReplicaSettingsUpdate] {
         self.replica_settings_update.as_deref().unwrap_or_default()
     }
 }
 impl UpdateGlobalTableSettingsInput {
     /// Creates a new builder-style object to manufacture [`UpdateGlobalTableSettingsInput`](crate::operation::update_global_table_settings::UpdateGlobalTableSettingsInput).
-    pub fn builder() -> crate::operation::update_global_table_settings::builders::UpdateGlobalTableSettingsInputBuilder {
-        crate::operation::update_global_table_settings::builders::UpdateGlobalTableSettingsInputBuilder::default()
+    pub fn builder() -> super::operation::update_global_table_settings::builders::UpdateGlobalTableSettingsInputBuilder {
+        super::operation::update_global_table_settings::builders::UpdateGlobalTableSettingsInputBuilder::default()
     }
 }

@@ -73,12 +73,12 @@
 #[non_exhaustive]
 pub struct UpdateGlobalTableSettingsInputBuilder {
     pub(crate) global_table_name: ::std::option::Option<::std::string::String>,
-    pub(crate) global_table_billing_mode: ::std::option::Option<crate::types::BillingMode>,
+    pub(crate) global_table_billing_mode: ::std::option::Option<super::types::BillingMode>,
     pub(crate) global_table_provisioned_write_capacity_units: ::std::option::Option<i64>,
-    pub(crate) global_table_provisioned_write_capacity_auto_scaling_settings_update: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+    pub(crate) global_table_provisioned_write_capacity_auto_scaling_settings_update: ::std::option::Option<super::types::AutoScalingSettingsUpdate>,
     pub(crate) global_table_global_secondary_index_settings_update:
-        ::std::option::Option<::std::vec::Vec<crate::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>>,
-    pub(crate) replica_settings_update: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaSettingsUpdate>>,
+        ::std::option::Option<::std::vec::Vec<super::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>>,
+    pub(crate) replica_settings_update: ::std::option::Option<::std::vec::Vec<super::types::ReplicaSettingsUpdate>>,
 }
 impl UpdateGlobalTableSettingsInputBuilder {
     /// <p>The name of the global table</p>
@@ -103,7 +103,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads. <code>PAY_PER_REQUEST</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/on-demand-capacity-mode.html">On-demand capacity mode</a>.</p></li>
     /// </ul>
-    pub fn global_table_billing_mode(mut self, input: crate::types::BillingMode) -> Self {
+    pub fn global_table_billing_mode(mut self, input: super::types::BillingMode) -> Self {
         self.global_table_billing_mode = ::std::option::Option::Some(input);
         self
     }
@@ -114,7 +114,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads. <code>PAY_PER_REQUEST</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/on-demand-capacity-mode.html">On-demand capacity mode</a>.</p></li>
     /// </ul>
-    pub fn set_global_table_billing_mode(mut self, input: ::std::option::Option<crate::types::BillingMode>) -> Self {
+    pub fn set_global_table_billing_mode(mut self, input: ::std::option::Option<super::types::BillingMode>) -> Self {
         self.global_table_billing_mode = input;
         self
     }
@@ -125,7 +125,7 @@
     /// <li>
     /// <p><code>PAY_PER_REQUEST</code> - We recommend using <code>PAY_PER_REQUEST</code> for unpredictable workloads. <code>PAY_PER_REQUEST</code> sets the billing mode to <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/on-demand-capacity-mode.html">On-demand capacity mode</a>.</p></li>
     /// </ul>
-    pub fn get_global_table_billing_mode(&self) -> &::std::option::Option<crate::types::BillingMode> {
+    pub fn get_global_table_billing_mode(&self) -> &::std::option::Option<super::types::BillingMode> {
         &self.global_table_billing_mode
     }
     /// <p>The maximum number of writes consumed per second before DynamoDB returns a <code>ThrottlingException.</code></p>
@@ -143,7 +143,7 @@
         &self.global_table_provisioned_write_capacity_units
     }
     /// <p>Auto scaling settings for managing provisioned write capacity for the global table.</p>
-    pub fn global_table_provisioned_write_capacity_auto_scaling_settings_update(mut self, input: crate::types::AutoScalingSettingsUpdate) -> Self {
+    pub fn global_table_provisioned_write_capacity_auto_scaling_settings_update(mut self, input: super::types::AutoScalingSettingsUpdate) -> Self {
         self.global_table_provisioned_write_capacity_auto_scaling_settings_update = ::std::option::Option::Some(input);
         self
     }
@@ -150,7 +150,7 @@
     /// <p>Auto scaling settings for managing provisioned write capacity for the global table.</p>
     pub fn set_global_table_provisioned_write_capacity_auto_scaling_settings_update(
         mut self,
-        input: ::std::option::Option<crate::types::AutoScalingSettingsUpdate>,
+        input: ::std::option::Option<super::types::AutoScalingSettingsUpdate>,
     ) -> Self {
         self.global_table_provisioned_write_capacity_auto_scaling_settings_update = input;
         self
@@ -158,7 +158,7 @@
     /// <p>Auto scaling settings for managing provisioned write capacity for the global table.</p>
     pub fn get_global_table_provisioned_write_capacity_auto_scaling_settings_update(
         &self,
-    ) -> &::std::option::Option<crate::types::AutoScalingSettingsUpdate> {
+    ) -> &::std::option::Option<super::types::AutoScalingSettingsUpdate> {
         &self.global_table_provisioned_write_capacity_auto_scaling_settings_update
     }
     /// Appends an item to `global_table_global_secondary_index_settings_update`.
@@ -166,7 +166,7 @@
     /// To override the contents of this collection use [`set_global_table_global_secondary_index_settings_update`](Self::set_global_table_global_secondary_index_settings_update).
     ///
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
-    pub fn global_table_global_secondary_index_settings_update(mut self, input: crate::types::GlobalTableGlobalSecondaryIndexSettingsUpdate) -> Self {
+    pub fn global_table_global_secondary_index_settings_update(mut self, input: super::types::GlobalTableGlobalSecondaryIndexSettingsUpdate) -> Self {
         let mut v = self.global_table_global_secondary_index_settings_update.unwrap_or_default();
         v.push(input);
         self.global_table_global_secondary_index_settings_update = ::std::option::Option::Some(v);
@@ -175,7 +175,7 @@
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     pub fn set_global_table_global_secondary_index_settings_update(
         mut self,
-        input: ::std::option::Option<::std::vec::Vec<crate::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>>,
+        input: ::std::option::Option<::std::vec::Vec<super::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>>,
     ) -> Self {
         self.global_table_global_secondary_index_settings_update = input;
         self
@@ -183,7 +183,7 @@
     /// <p>Represents the settings of a global secondary index for a global table that will be modified.</p>
     pub fn get_global_table_global_secondary_index_settings_update(
         &self,
-    ) -> &::std::option::Option<::std::vec::Vec<crate::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>> {
+    ) -> &::std::option::Option<::std::vec::Vec<super::types::GlobalTableGlobalSecondaryIndexSettingsUpdate>> {
         &self.global_table_global_secondary_index_settings_update
     }
     /// Appends an item to `replica_settings_update`.
@@ -191,7 +191,7 @@
     /// To override the contents of this collection use [`set_replica_settings_update`](Self::set_replica_settings_update).
     ///
     /// <p>Represents the settings for a global table in a Region that will be modified.</p>
-    pub fn replica_settings_update(mut self, input: crate::types::ReplicaSettingsUpdate) -> Self {
+    pub fn replica_settings_update(mut self, input: super::types::ReplicaSettingsUpdate) -> Self {
         let mut v = self.replica_settings_update.unwrap_or_default();
         v.push(input);
         self.replica_settings_update = ::std::option::Option::Some(v);
@@ -198,12 +198,12 @@
         self
     }
     /// <p>Represents the settings for a global table in a Region that will be modified.</p>
-    pub fn set_replica_settings_update(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::ReplicaSettingsUpdate>>) -> Self {
+    pub fn set_replica_settings_update(mut self, input: ::std::option::Option<::std::vec::Vec<super::types::ReplicaSettingsUpdate>>) -> Self {
         self.replica_settings_update = input;
         self
     }
     /// <p>Represents the settings for a global table in a Region that will be modified.</p>
-    pub fn get_replica_settings_update(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::ReplicaSettingsUpdate>> {
+    pub fn get_replica_settings_update(&self) -> &::std::option::Option<::std::vec::Vec<super::types::ReplicaSettingsUpdate>> {
         &self.replica_settings_update
     }
     /// Consumes the builder and constructs a [`UpdateGlobalTableSettingsInput`](crate::operation::update_global_table_settings::UpdateGlobalTableSettingsInput).
@@ -210,10 +210,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::update_global_table_settings::UpdateGlobalTableSettingsInput,
+        super::operation::update_global_table_settings::UpdateGlobalTableSettingsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::update_global_table_settings::UpdateGlobalTableSettingsInput {
+        ::std::result::Result::Ok(super::operation::update_global_table_settings::UpdateGlobalTableSettingsInput {
             global_table_name: self.global_table_name,
             global_table_billing_mode: self.global_table_billing_mode,
             global_table_provisioned_write_capacity_units: self.global_table_provisioned_write_capacity_units,
