--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteOrganizationConformancePackInput {
     /// Creates a new builder-style object to manufacture [`DeleteOrganizationConformancePackInput`](crate::operation::delete_organization_conformance_pack::DeleteOrganizationConformancePackInput).
-    pub fn builder() -> crate::operation::delete_organization_conformance_pack::builders::DeleteOrganizationConformancePackInputBuilder {
-        crate::operation::delete_organization_conformance_pack::builders::DeleteOrganizationConformancePackInputBuilder::default()
+    pub fn builder() -> super::operation::delete_organization_conformance_pack::builders::DeleteOrganizationConformancePackInputBuilder {
+        super::operation::delete_organization_conformance_pack::builders::DeleteOrganizationConformancePackInputBuilder::default()
     }
 }

@@ -45,11 +45,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_organization_conformance_pack::DeleteOrganizationConformancePackInput,
+        super::operation::delete_organization_conformance_pack::DeleteOrganizationConformancePackInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::delete_organization_conformance_pack::DeleteOrganizationConformancePackInput {
+            super::operation::delete_organization_conformance_pack::DeleteOrganizationConformancePackInput {
                 organization_conformance_pack_name: self.organization_conformance_pack_name,
             },
         )
