--- original
+++ modified
@@ -38,8 +38,8 @@
 }
 impl ListSubPackageGroupsInput {
     /// Creates a new builder-style object to manufacture [`ListSubPackageGroupsInput`](crate::operation::list_sub_package_groups::ListSubPackageGroupsInput).
-    pub fn builder() -> crate::operation::list_sub_package_groups::builders::ListSubPackageGroupsInputBuilder {
-        crate::operation::list_sub_package_groups::builders::ListSubPackageGroupsInputBuilder::default()
+    pub fn builder() -> super::operation::list_sub_package_groups::builders::ListSubPackageGroupsInputBuilder {
+        super::operation::list_sub_package_groups::builders::ListSubPackageGroupsInputBuilder::default()
     }
 }

@@ -129,9 +129,9 @@
     /// Consumes the builder and constructs a [`ListSubPackageGroupsInput`](crate::operation::list_sub_package_groups::ListSubPackageGroupsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_sub_package_groups::ListSubPackageGroupsInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_sub_package_groups::ListSubPackageGroupsInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_sub_package_groups::ListSubPackageGroupsInput {
+        ::std::result::Result::Ok(super::operation::list_sub_package_groups::ListSubPackageGroupsInput {
             domain: self.domain,
             domain_owner: self.domain_owner,
             package_group: self.package_group,
