--- original
+++ modified
@@ -23,8 +23,8 @@
 }
 impl ListEmailTemplatesInput {
     /// Creates a new builder-style object to manufacture [`ListEmailTemplatesInput`](crate::operation::list_email_templates::ListEmailTemplatesInput).
-    pub fn builder() -> crate::operation::list_email_templates::builders::ListEmailTemplatesInputBuilder {
-        crate::operation::list_email_templates::builders::ListEmailTemplatesInputBuilder::default()
+    pub fn builder() -> super::operation::list_email_templates::builders::ListEmailTemplatesInputBuilder {
+        super::operation::list_email_templates::builders::ListEmailTemplatesInputBuilder::default()
     }
 }

@@ -70,9 +70,9 @@
     /// Consumes the builder and constructs a [`ListEmailTemplatesInput`](crate::operation::list_email_templates::ListEmailTemplatesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_email_templates::ListEmailTemplatesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::list_email_templates::ListEmailTemplatesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::list_email_templates::ListEmailTemplatesInput {
+        ::std::result::Result::Ok(super::operation::list_email_templates::ListEmailTemplatesInput {
             next_token: self.next_token,
             page_size: self.page_size,
         })
