--- original
+++ modified
@@ -8,7 +8,7 @@
     /// <p>The name that you want to assign to the IdP. You can pass the identity provider name in the <code>identity_provider</code> query parameter of requests to the <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html">Authorize endpoint</a> to silently redirect to sign-in with the associated IdP.</p>
     pub provider_name: ::std::option::Option<::std::string::String>,
     /// <p>The type of IdP that you want to add. Amazon Cognito supports OIDC, SAML 2.0, Login With Amazon, Sign In With Apple, Google, and Facebook IdPs.</p>
-    pub provider_type: ::std::option::Option<crate::types::IdentityProviderTypeType>,
+    pub provider_type: ::std::option::Option<super::super::super::types::IdentityProviderTypeType>,
     /// <p>The scopes, URLs, and identifiers for your external identity provider. The following examples describe the provider detail keys for each IdP type. These values and their schema are subject to change. Social IdP <code>authorize_scopes</code> values must match the values listed here.</p>
     /// <dl>
     /// <dt>
@@ -73,7 +73,7 @@
         self.provider_name.as_deref()
     }
     /// <p>The type of IdP that you want to add. Amazon Cognito supports OIDC, SAML 2.0, Login With Amazon, Sign In With Apple, Google, and Facebook IdPs.</p>
-    pub fn provider_type(&self) -> ::std::option::Option<&crate::types::IdentityProviderTypeType> {
+    pub fn provider_type(&self) -> ::std::option::Option<&super::super::super::types::IdentityProviderTypeType> {
         self.provider_type.as_ref()
     }
     /// <p>The scopes, URLs, and identifiers for your external identity provider. The following examples describe the provider detail keys for each IdP type. These values and their schema are subject to change. Social IdP <code>authorize_scopes</code> values must match the values listed here.</p>
@@ -140,8 +140,8 @@
 }
 impl CreateIdentityProviderInput {
     /// Creates a new builder-style object to manufacture [`CreateIdentityProviderInput`](crate::operation::create_identity_provider::CreateIdentityProviderInput).
-    pub fn builder() -> crate::operation::create_identity_provider::builders::CreateIdentityProviderInputBuilder {
-        crate::operation::create_identity_provider::builders::CreateIdentityProviderInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::create_identity_provider::builders::CreateIdentityProviderInputBuilder {
+        super::super::super::operation::create_identity_provider::builders::CreateIdentityProviderInputBuilder::default()
     }
 }

@@ -151,7 +151,7 @@
 pub struct CreateIdentityProviderInputBuilder {
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
     pub(crate) provider_name: ::std::option::Option<::std::string::String>,
-    pub(crate) provider_type: ::std::option::Option<crate::types::IdentityProviderTypeType>,
+    pub(crate) provider_type: ::std::option::Option<super::super::super::types::IdentityProviderTypeType>,
     pub(crate) provider_details: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     pub(crate) attribute_mapping: ::std::option::Option<::std::collections::HashMap<::std::string::String, ::std::string::String>>,
     pub(crate) idp_identifiers: ::std::option::Option<::std::vec::Vec<::std::string::String>>,
@@ -189,17 +189,17 @@
     }
     /// <p>The type of IdP that you want to add. Amazon Cognito supports OIDC, SAML 2.0, Login With Amazon, Sign In With Apple, Google, and Facebook IdPs.</p>
     /// This field is required.
-    pub fn provider_type(mut self, input: crate::types::IdentityProviderTypeType) -> Self {
+    pub fn provider_type(mut self, input: super::super::super::types::IdentityProviderTypeType) -> Self {
         self.provider_type = ::std::option::Option::Some(input);
         self
     }
     /// <p>The type of IdP that you want to add. Amazon Cognito supports OIDC, SAML 2.0, Login With Amazon, Sign In With Apple, Google, and Facebook IdPs.</p>
-    pub fn set_provider_type(mut self, input: ::std::option::Option<crate::types::IdentityProviderTypeType>) -> Self {
+    pub fn set_provider_type(mut self, input: ::std::option::Option<super::super::super::types::IdentityProviderTypeType>) -> Self {
         self.provider_type = input;
         self
     }
     /// <p>The type of IdP that you want to add. Amazon Cognito supports OIDC, SAML 2.0, Login With Amazon, Sign In With Apple, Google, and Facebook IdPs.</p>
-    pub fn get_provider_type(&self) -> &::std::option::Option<crate::types::IdentityProviderTypeType> {
+    pub fn get_provider_type(&self) -> &::std::option::Option<super::super::super::types::IdentityProviderTypeType> {
         &self.provider_type
     }
     /// Adds a key-value pair to `provider_details`.
@@ -421,10 +421,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::create_identity_provider::CreateIdentityProviderInput,
+        super::super::super::operation::create_identity_provider::CreateIdentityProviderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::create_identity_provider::CreateIdentityProviderInput {
+        ::std::result::Result::Ok(super::super::super::operation::create_identity_provider::CreateIdentityProviderInput {
             user_pool_id: self.user_pool_id,
             provider_name: self.provider_name,
             provider_type: self.provider_type,
