--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl DeleteRemediationConfigurationInput {
     /// Creates a new builder-style object to manufacture [`DeleteRemediationConfigurationInput`](crate::operation::delete_remediation_configuration::DeleteRemediationConfigurationInput).
-    pub fn builder() -> crate::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationInputBuilder {
-        crate::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationInputBuilder {
+        super::super::super::operation::delete_remediation_configuration::builders::DeleteRemediationConfigurationInputBuilder::default()
     }
 }

@@ -66,10 +66,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_remediation_configuration::DeleteRemediationConfigurationInput,
+        super::super::super::operation::delete_remediation_configuration::DeleteRemediationConfigurationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_remediation_configuration::DeleteRemediationConfigurationInput {
+        ::std::result::Result::Ok(super::super::super::operation::delete_remediation_configuration::DeleteRemediationConfigurationInput {
             config_rule_name: self.config_rule_name,
             resource_type: self.resource_type,
         })
