--- original
+++ modified
@@ -7,7 +7,7 @@
     /// <p>The name of the table to be configured. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
     pub table_name: ::std::option::Option<::std::string::String>,
     /// <p>Represents the settings used to enable or disable Time to Live for the specified table.</p>
-    pub time_to_live_specification: ::std::option::Option<crate::types::TimeToLiveSpecification>,
+    pub time_to_live_specification: ::std::option::Option<super::types::TimeToLiveSpecification>,
 }
 impl UpdateTimeToLiveInput {
     /// <p>The name of the table to be configured. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
@@ -15,14 +15,14 @@
         self.table_name.as_deref()
     }
     /// <p>Represents the settings used to enable or disable Time to Live for the specified table.</p>
-    pub fn time_to_live_specification(&self) -> ::std::option::Option<&crate::types::TimeToLiveSpecification> {
+    pub fn time_to_live_specification(&self) -> ::std::option::Option<&super::types::TimeToLiveSpecification> {
         self.time_to_live_specification.as_ref()
     }
 }
 impl UpdateTimeToLiveInput {
     /// Creates a new builder-style object to manufacture [`UpdateTimeToLiveInput`](crate::operation::update_time_to_live::UpdateTimeToLiveInput).
-    pub fn builder() -> crate::operation::update_time_to_live::builders::UpdateTimeToLiveInputBuilder {
-        crate::operation::update_time_to_live::builders::UpdateTimeToLiveInputBuilder::default()
+    pub fn builder() -> super::operation::update_time_to_live::builders::UpdateTimeToLiveInputBuilder {
+        super::operation::update_time_to_live::builders::UpdateTimeToLiveInputBuilder::default()
     }
 }

@@ -31,7 +31,7 @@
 #[non_exhaustive]
 pub struct UpdateTimeToLiveInputBuilder {
     pub(crate) table_name: ::std::option::Option<::std::string::String>,
-    pub(crate) time_to_live_specification: ::std::option::Option<crate::types::TimeToLiveSpecification>,
+    pub(crate) time_to_live_specification: ::std::option::Option<super::types::TimeToLiveSpecification>,
 }
 impl UpdateTimeToLiveInputBuilder {
     /// <p>The name of the table to be configured. You can also provide the Amazon Resource Name (ARN) of the table in this parameter.</p>
@@ -51,24 +51,24 @@
     }
     /// <p>Represents the settings used to enable or disable Time to Live for the specified table.</p>
     /// This field is required.
-    pub fn time_to_live_specification(mut self, input: crate::types::TimeToLiveSpecification) -> Self {
+    pub fn time_to_live_specification(mut self, input: super::types::TimeToLiveSpecification) -> Self {
         self.time_to_live_specification = ::std::option::Option::Some(input);
         self
     }
     /// <p>Represents the settings used to enable or disable Time to Live for the specified table.</p>
-    pub fn set_time_to_live_specification(mut self, input: ::std::option::Option<crate::types::TimeToLiveSpecification>) -> Self {
+    pub fn set_time_to_live_specification(mut self, input: ::std::option::Option<super::types::TimeToLiveSpecification>) -> Self {
         self.time_to_live_specification = input;
         self
     }
     /// <p>Represents the settings used to enable or disable Time to Live for the specified table.</p>
-    pub fn get_time_to_live_specification(&self) -> &::std::option::Option<crate::types::TimeToLiveSpecification> {
+    pub fn get_time_to_live_specification(&self) -> &::std::option::Option<super::types::TimeToLiveSpecification> {
         &self.time_to_live_specification
     }
     /// Consumes the builder and constructs a [`UpdateTimeToLiveInput`](crate::operation::update_time_to_live::UpdateTimeToLiveInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_time_to_live::UpdateTimeToLiveInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::update_time_to_live::UpdateTimeToLiveInput {
+    ) -> ::std::result::Result<super::operation::update_time_to_live::UpdateTimeToLiveInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::update_time_to_live::UpdateTimeToLiveInput {
             table_name: self.table_name,
             time_to_live_specification: self.time_to_live_specification,
         })
