--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListUsersInput {
     /// Creates a new builder-style object to manufacture [`ListUsersInput`](crate::operation::list_users::ListUsersInput).
-    pub fn builder() -> crate::operation::list_users::builders::ListUsersInputBuilder {
-        crate::operation::list_users::builders::ListUsersInputBuilder::default()
+    pub fn builder() -> super::operation::list_users::builders::ListUsersInputBuilder {
+        super::operation::list_users::builders::ListUsersInputBuilder::default()
     }
 }

@@ -93,8 +93,8 @@
         &self.max_items
     }
     /// Consumes the builder and constructs a [`ListUsersInput`](crate::operation::list_users::ListUsersInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_users::ListUsersInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_users::ListUsersInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_users::ListUsersInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_users::ListUsersInput {
             path_prefix: self.path_prefix,
             marker: self.marker,
             max_items: self.max_items,
