--- original
+++ modified
@@ -24,8 +24,8 @@
 }
 impl UntagOpenIdConnectProviderInput {
     /// Creates a new builder-style object to manufacture [`UntagOpenIdConnectProviderInput`](crate::operation::untag_open_id_connect_provider::UntagOpenIdConnectProviderInput).
-    pub fn builder() -> crate::operation::untag_open_id_connect_provider::builders::UntagOpenIdConnectProviderInputBuilder {
-        crate::operation::untag_open_id_connect_provider::builders::UntagOpenIdConnectProviderInputBuilder::default()
+    pub fn builder() -> super::operation::untag_open_id_connect_provider::builders::UntagOpenIdConnectProviderInputBuilder {
+        super::operation::untag_open_id_connect_provider::builders::UntagOpenIdConnectProviderInputBuilder::default()
     }
 }

@@ -79,10 +79,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::untag_open_id_connect_provider::UntagOpenIdConnectProviderInput,
+        super::operation::untag_open_id_connect_provider::UntagOpenIdConnectProviderInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::untag_open_id_connect_provider::UntagOpenIdConnectProviderInput {
+        ::std::result::Result::Ok(super::operation::untag_open_id_connect_provider::UntagOpenIdConnectProviderInput {
             open_id_connect_provider_arn: self.open_id_connect_provider_arn,
             tag_keys: self.tag_keys,
         })
