--- original
+++ modified
@@ -40,8 +40,8 @@
 }
 impl UpdatePrimaryRegionInput {
     /// Creates a new builder-style object to manufacture [`UpdatePrimaryRegionInput`](crate::operation::update_primary_region::UpdatePrimaryRegionInput).
-    pub fn builder() -> crate::operation::update_primary_region::builders::UpdatePrimaryRegionInputBuilder {
-        crate::operation::update_primary_region::builders::UpdatePrimaryRegionInputBuilder::default()
+    pub fn builder() -> super::operation::update_primary_region::builders::UpdatePrimaryRegionInputBuilder {
+        super::operation::update_primary_region::builders::UpdatePrimaryRegionInputBuilder::default()
     }
 }

@@ -116,9 +116,9 @@
     /// Consumes the builder and constructs a [`UpdatePrimaryRegionInput`](crate::operation::update_primary_region::UpdatePrimaryRegionInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::update_primary_region::UpdatePrimaryRegionInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::operation::update_primary_region::UpdatePrimaryRegionInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::update_primary_region::UpdatePrimaryRegionInput {
+        ::std::result::Result::Ok(super::operation::update_primary_region::UpdatePrimaryRegionInput {
             key_id: self.key_id,
             primary_region: self.primary_region,
         })
