--- original
+++ modified
@@ -36,8 +36,8 @@
 }
 impl AddUserPoolClientSecretInput {
     /// Creates a new builder-style object to manufacture [`AddUserPoolClientSecretInput`](crate::operation::add_user_pool_client_secret::AddUserPoolClientSecretInput).
-    pub fn builder() -> crate::operation::add_user_pool_client_secret::builders::AddUserPoolClientSecretInputBuilder {
-        crate::operation::add_user_pool_client_secret::builders::AddUserPoolClientSecretInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::add_user_pool_client_secret::builders::AddUserPoolClientSecretInputBuilder {
+        super::super::super::operation::add_user_pool_client_secret::builders::AddUserPoolClientSecretInputBuilder::default()
     }
 }

@@ -98,10 +98,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::add_user_pool_client_secret::AddUserPoolClientSecretInput,
+        super::super::super::operation::add_user_pool_client_secret::AddUserPoolClientSecretInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::add_user_pool_client_secret::AddUserPoolClientSecretInput {
+        ::std::result::Result::Ok(super::super::super::operation::add_user_pool_client_secret::AddUserPoolClientSecretInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
             client_secret: self.client_secret,
