--- original
+++ modified
@@ -4,11 +4,11 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct ListLayerVersionsInput {
     /// <p>The compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architecture</a>.</p>
-    pub compatible_architecture: ::std::option::Option<crate::types::Architecture>,
+    pub compatible_architecture: ::std::option::Option<super::types::Architecture>,
     /// <p>A runtime identifier.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-deprecation-levels">Runtime use after deprecation</a>.</p>
     /// <p>For a list of all currently supported runtimes, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported">Supported runtimes</a>.</p>
-    pub compatible_runtime: ::std::option::Option<crate::types::Runtime>,
+    pub compatible_runtime: ::std::option::Option<super::types::Runtime>,
     /// <p>The name or Amazon Resource Name (ARN) of the layer.</p>
     pub layer_name: ::std::option::Option<::std::string::String>,
     /// <p>A pagination token returned by a previous call.</p>
@@ -18,13 +18,13 @@
 }
 impl ListLayerVersionsInput {
     /// <p>The compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architecture</a>.</p>
-    pub fn compatible_architecture(&self) -> ::std::option::Option<&crate::types::Architecture> {
+    pub fn compatible_architecture(&self) -> ::std::option::Option<&super::types::Architecture> {
         self.compatible_architecture.as_ref()
     }
     /// <p>A runtime identifier.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-deprecation-levels">Runtime use after deprecation</a>.</p>
     /// <p>For a list of all currently supported runtimes, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported">Supported runtimes</a>.</p>
-    pub fn compatible_runtime(&self) -> ::std::option::Option<&crate::types::Runtime> {
+    pub fn compatible_runtime(&self) -> ::std::option::Option<&super::types::Runtime> {
         self.compatible_runtime.as_ref()
     }
     /// <p>The name or Amazon Resource Name (ARN) of the layer.</p>
@@ -42,8 +42,8 @@
 }
 impl ListLayerVersionsInput {
     /// Creates a new builder-style object to manufacture [`ListLayerVersionsInput`](crate::operation::list_layer_versions::ListLayerVersionsInput).
-    pub fn builder() -> crate::operation::list_layer_versions::builders::ListLayerVersionsInputBuilder {
-        crate::operation::list_layer_versions::builders::ListLayerVersionsInputBuilder::default()
+    pub fn builder() -> super::operation::list_layer_versions::builders::ListLayerVersionsInputBuilder {
+        super::operation::list_layer_versions::builders::ListLayerVersionsInputBuilder::default()
     }
 }

@@ -51,8 +51,8 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct ListLayerVersionsInputBuilder {
-    pub(crate) compatible_architecture: ::std::option::Option<crate::types::Architecture>,
-    pub(crate) compatible_runtime: ::std::option::Option<crate::types::Runtime>,
+    pub(crate) compatible_architecture: ::std::option::Option<super::types::Architecture>,
+    pub(crate) compatible_runtime: ::std::option::Option<super::types::Runtime>,
     pub(crate) layer_name: ::std::option::Option<::std::string::String>,
     pub(crate) marker: ::std::option::Option<::std::string::String>,
     pub(crate) max_items: ::std::option::Option<i32>,
@@ -59,23 +59,23 @@
 }
 impl ListLayerVersionsInputBuilder {
     /// <p>The compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architecture</a>.</p>
-    pub fn compatible_architecture(mut self, input: crate::types::Architecture) -> Self {
+    pub fn compatible_architecture(mut self, input: super::types::Architecture) -> Self {
         self.compatible_architecture = ::std::option::Option::Some(input);
         self
     }
     /// <p>The compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architecture</a>.</p>
-    pub fn set_compatible_architecture(mut self, input: ::std::option::Option<crate::types::Architecture>) -> Self {
+    pub fn set_compatible_architecture(mut self, input: ::std::option::Option<super::types::Architecture>) -> Self {
         self.compatible_architecture = input;
         self
     }
     /// <p>The compatible <a href="https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html">instruction set architecture</a>.</p>
-    pub fn get_compatible_architecture(&self) -> &::std::option::Option<crate::types::Architecture> {
+    pub fn get_compatible_architecture(&self) -> &::std::option::Option<super::types::Architecture> {
         &self.compatible_architecture
     }
     /// <p>A runtime identifier.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-deprecation-levels">Runtime use after deprecation</a>.</p>
     /// <p>For a list of all currently supported runtimes, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported">Supported runtimes</a>.</p>
-    pub fn compatible_runtime(mut self, input: crate::types::Runtime) -> Self {
+    pub fn compatible_runtime(mut self, input: super::types::Runtime) -> Self {
         self.compatible_runtime = ::std::option::Option::Some(input);
         self
     }
@@ -82,7 +82,7 @@
     /// <p>A runtime identifier.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-deprecation-levels">Runtime use after deprecation</a>.</p>
     /// <p>For a list of all currently supported runtimes, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported">Supported runtimes</a>.</p>
-    pub fn set_compatible_runtime(mut self, input: ::std::option::Option<crate::types::Runtime>) -> Self {
+    pub fn set_compatible_runtime(mut self, input: ::std::option::Option<super::types::Runtime>) -> Self {
         self.compatible_runtime = input;
         self
     }
@@ -89,7 +89,7 @@
     /// <p>A runtime identifier.</p>
     /// <p>The following list includes deprecated runtimes. For more information, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-deprecation-levels">Runtime use after deprecation</a>.</p>
     /// <p>For a list of all currently supported runtimes, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported">Supported runtimes</a>.</p>
-    pub fn get_compatible_runtime(&self) -> &::std::option::Option<crate::types::Runtime> {
+    pub fn get_compatible_runtime(&self) -> &::std::option::Option<super::types::Runtime> {
         &self.compatible_runtime
     }
     /// <p>The name or Amazon Resource Name (ARN) of the layer.</p>
@@ -138,8 +138,8 @@
     /// Consumes the builder and constructs a [`ListLayerVersionsInput`](crate::operation::list_layer_versions::ListLayerVersionsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_layer_versions::ListLayerVersionsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_layer_versions::ListLayerVersionsInput {
+    ) -> ::std::result::Result<super::operation::list_layer_versions::ListLayerVersionsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_layer_versions::ListLayerVersionsInput {
             compatible_architecture: self.compatible_architecture,
             compatible_runtime: self.compatible_runtime,
             layer_name: self.layer_name,
