--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl PutBearerTokenAuthenticationInput {
     /// Creates a new builder-style object to manufacture [`PutBearerTokenAuthenticationInput`](crate::operation::put_bearer_token_authentication::PutBearerTokenAuthenticationInput).
-    pub fn builder() -> crate::operation::put_bearer_token_authentication::builders::PutBearerTokenAuthenticationInputBuilder {
-        crate::operation::put_bearer_token_authentication::builders::PutBearerTokenAuthenticationInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::put_bearer_token_authentication::builders::PutBearerTokenAuthenticationInputBuilder {
+        super::super::super::operation::put_bearer_token_authentication::builders::PutBearerTokenAuthenticationInputBuilder::default()
     }
 }

@@ -97,10 +97,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::put_bearer_token_authentication::PutBearerTokenAuthenticationInput,
+        super::super::super::operation::put_bearer_token_authentication::PutBearerTokenAuthenticationInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::put_bearer_token_authentication::PutBearerTokenAuthenticationInput {
+        ::std::result::Result::Ok(super::super::super::operation::put_bearer_token_authentication::PutBearerTokenAuthenticationInput {
             log_group_identifier: self.log_group_identifier,
             bearer_token_authentication_enabled: self.bearer_token_authentication_enabled,
         })
