--- original
+++ modified
@@ -30,8 +30,8 @@
 }
 impl ListQuotaSharesInput {
     /// Creates a new builder-style object to manufacture [`ListQuotaSharesInput`](crate::operation::list_quota_shares::ListQuotaSharesInput).
-    pub fn builder() -> crate::operation::list_quota_shares::builders::ListQuotaSharesInputBuilder {
-        crate::operation::list_quota_shares::builders::ListQuotaSharesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_quota_shares::builders::ListQuotaSharesInputBuilder {
+        super::super::super::operation::list_quota_shares::builders::ListQuotaSharesInputBuilder::default()
     }
 }

@@ -96,8 +96,8 @@
     /// Consumes the builder and constructs a [`ListQuotaSharesInput`](crate::operation::list_quota_shares::ListQuotaSharesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_quota_shares::ListQuotaSharesInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_quota_shares::ListQuotaSharesInput {
+    ) -> ::std::result::Result<super::super::super::operation::list_quota_shares::ListQuotaSharesInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_quota_shares::ListQuotaSharesInput {
             job_queue: self.job_queue,
             max_results: self.max_results,
             next_token: self.next_token,
