--- original
+++ modified
@@ -15,8 +15,8 @@
 }
 impl GetCustomVerificationEmailTemplateInput {
     /// Creates a new builder-style object to manufacture [`GetCustomVerificationEmailTemplateInput`](crate::operation::get_custom_verification_email_template::GetCustomVerificationEmailTemplateInput).
-    pub fn builder() -> crate::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateInputBuilder {
-        crate::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateInputBuilder {
+        super::super::super::operation::get_custom_verification_email_template::builders::GetCustomVerificationEmailTemplateInputBuilder::default()
     }
 }

@@ -46,11 +46,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_custom_verification_email_template::GetCustomVerificationEmailTemplateInput,
+        super::super::super::operation::get_custom_verification_email_template::GetCustomVerificationEmailTemplateInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::get_custom_verification_email_template::GetCustomVerificationEmailTemplateInput {
+            super::super::super::operation::get_custom_verification_email_template::GetCustomVerificationEmailTemplateInput {
                 template_name: self.template_name,
             },
         )
