--- original
+++ modified
@@ -68,8 +68,8 @@
 }
 impl ListGrantsInput {
     /// Creates a new builder-style object to manufacture [`ListGrantsInput`](crate::operation::list_grants::ListGrantsInput).
-    pub fn builder() -> crate::operation::list_grants::builders::ListGrantsInputBuilder {
-        crate::operation::list_grants::builders::ListGrantsInputBuilder::default()
+    pub fn builder() -> super::operation::list_grants::builders::ListGrantsInputBuilder {
+        super::operation::list_grants::builders::ListGrantsInputBuilder::default()
     }
 }

@@ -207,8 +207,8 @@
         &self.grantee_service_principal
     }
     /// Consumes the builder and constructs a [`ListGrantsInput`](crate::operation::list_grants::ListGrantsInput).
-    pub fn build(self) -> ::std::result::Result<crate::operation::list_grants::ListGrantsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_grants::ListGrantsInput {
+    pub fn build(self) -> ::std::result::Result<super::operation::list_grants::ListGrantsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::operation::list_grants::ListGrantsInput {
             limit: self.limit,
             marker: self.marker,
             key_id: self.key_id,
