--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl DeleteContactListInput {
     /// Creates a new builder-style object to manufacture [`DeleteContactListInput`](crate::operation::delete_contact_list::DeleteContactListInput).
-    pub fn builder() -> crate::operation::delete_contact_list::builders::DeleteContactListInputBuilder {
-        crate::operation::delete_contact_list::builders::DeleteContactListInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::delete_contact_list::builders::DeleteContactListInputBuilder {
+        super::super::super::operation::delete_contact_list::builders::DeleteContactListInputBuilder::default()
     }
 }

@@ -44,8 +44,8 @@
     /// Consumes the builder and constructs a [`DeleteContactListInput`](crate::operation::delete_contact_list::DeleteContactListInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::delete_contact_list::DeleteContactListInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::delete_contact_list::DeleteContactListInput {
+    ) -> ::std::result::Result<super::super::super::operation::delete_contact_list::DeleteContactListInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::delete_contact_list::DeleteContactListInput {
             contact_list_name: self.contact_list_name,
         })
     }
