--- original
+++ modified
@@ -26,7 +26,7 @@
     ///   - [`performance_config(Option<PerformanceConfiguration>)`](crate::operation::converse::ConverseOutput::performance_config): <p>Model performance settings for the request.</p>
     ///   - [`service_tier(Option<ServiceTier>)`](crate::operation::converse::ConverseOutput::service_tier): <p>Specifies the processing tier configuration used for serving the request.</p>
     /// - On failure, responds with [`SdkError<ConverseError>`](crate::operation::converse::ConverseError)
-    pub fn converse(&self) -> crate::operation::converse::builders::ConverseFluentBuilder {
-        crate::operation::converse::builders::ConverseFluentBuilder::new(self.handle.clone())
+    pub fn converse(&self) -> super::super::operation::converse::builders::ConverseFluentBuilder {
+        super::super::operation::converse::builders::ConverseFluentBuilder::new(self.handle.clone())
     }
 }
