--- original
+++ modified
@@ -8,7 +8,7 @@
     /// - On success, responds with [`CountTokensOutput`](crate::operation::count_tokens::CountTokensOutput) with field(s):
     ///   - [`input_tokens(i32)`](crate::operation::count_tokens::CountTokensOutput::input_tokens): <p>The number of tokens in the provided input according to the specified model's tokenization rules. This count represents the number of input tokens that would be processed if the same input were sent to the model in an inference request. Use this value to estimate costs and ensure your inputs stay within model token limits.</p>
     /// - On failure, responds with [`SdkError<CountTokensError>`](crate::operation::count_tokens::CountTokensError)
-    pub fn count_tokens(&self) -> crate::operation::count_tokens::builders::CountTokensFluentBuilder {
-        crate::operation::count_tokens::builders::CountTokensFluentBuilder::new(self.handle.clone())
+    pub fn count_tokens(&self) -> super::operation::count_tokens::builders::CountTokensFluentBuilder {
+        super::operation::count_tokens::builders::CountTokensFluentBuilder::new(self.handle.clone())
     }
 }
