--- original
+++ modified
@@ -19,7 +19,7 @@
     ///   - [`performance_config_latency(Option<PerformanceConfigLatency>)`](crate::operation::invoke_model::InvokeModelOutput::performance_config_latency): <p>Model performance settings for the request.</p>
     ///   - [`service_tier(Option<ServiceTierType>)`](crate::operation::invoke_model::InvokeModelOutput::service_tier): <p>Specifies the processing tier type used for serving the request.</p>
     /// - On failure, responds with [`SdkError<InvokeModelError>`](crate::operation::invoke_model::InvokeModelError)
-    pub fn invoke_model(&self) -> crate::operation::invoke_model::builders::InvokeModelFluentBuilder {
-        crate::operation::invoke_model::builders::InvokeModelFluentBuilder::new(self.handle.clone())
+    pub fn invoke_model(&self) -> super::operation::invoke_model::builders::InvokeModelFluentBuilder {
+        super::operation::invoke_model::builders::InvokeModelFluentBuilder::new(self.handle.clone())
     }
 }
