--- original
+++ modified
@@ -10,11 +10,11 @@
     }
     pub(crate) async fn orchestrate(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
+        input: super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
     ) -> ::std::result::Result<
-        crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamOutput,
+        super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamOutput,
         ::aws_smithy_runtime_api::client::result::SdkError<
-            crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
+            super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >,
     > {
@@ -23,7 +23,7 @@
             ::aws_smithy_runtime_api::client::orchestrator::HttpResponse,
         >| {
             err.map_service_error(|err| {
-                err.downcast::<crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError>()
+                err.downcast::<super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError>()
                     .expect("correct error type")
             })
         };
@@ -33,7 +33,7 @@
         let output = context.finalize().map_err(map_err)?;
         ::std::result::Result::Ok(
             output
-                .downcast::<crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamOutput>()
+                .downcast::<super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamOutput>()
                 .expect("correct output type"),
         )
     }
@@ -40,7 +40,7 @@

     pub(crate) async fn orchestrate_with_stop_point(
         runtime_plugins: &::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        input: crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
+        input: super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
         stop_point: ::aws_smithy_runtime::client::orchestrator::StopPoint,
     ) -> ::std::result::Result<
         ::aws_smithy_runtime_api::client::interceptors::context::InterceptorContext,
@@ -72,8 +72,8 @@

     pub(crate) fn operation_runtime_plugins(
         client_runtime_plugins: ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins,
-        client_config: &crate::config::Config,
-        config_override: ::std::option::Option<crate::config::Builder>,
+        client_config: &super::super::config::Config,
+        config_override: ::std::option::Option<super::super::config::Builder>,
     ) -> ::aws_smithy_runtime_api::client::runtime_plugin::RuntimePlugins {
         let mut runtime_plugins = client_runtime_plugins.with_operation_plugin(Self::new());

@@ -81,7 +81,7 @@
             for plugin in config_override.runtime_plugins.iter().cloned() {
                 runtime_plugins = runtime_plugins.with_operation_plugin(plugin);
             }
-            runtime_plugins = runtime_plugins.with_operation_plugin(crate::config::ConfigOverrideRuntimePlugin::new(
+            runtime_plugins = runtime_plugins.with_operation_plugin(super::super::config::ConfigOverrideRuntimePlugin::new(
                 config_override,
                 client_config.config.clone(),
                 &client_config.runtime_components,
@@ -102,7 +102,7 @@
         ));

         cfg.store_put(::aws_smithy_runtime_api::client::auth::AuthSchemeOptionResolverParams::new(
-            crate::config::auth::Params::builder()
+            super::super::config::auth::Params::builder()
                 .operation_name("InvokeModelWithBidirectionalStream")
                 .build()
                 .expect("required fields set"),
@@ -140,13 +140,13 @@
                 InvokeModelWithBidirectionalStreamEndpointParamsInterceptor,
             ))
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::TransientErrorClassifier::<
-                crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
+                super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
             >::new())
             .with_retry_classifier(::aws_smithy_runtime::client::retries::classifiers::ModeledAsRetryableClassifier::<
-                crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
+                super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
             >::new())
             .with_retry_classifier(::aws_runtime::retries::classifiers::AwsErrorCodeClassifier::<
-                crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
+                super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError,
             >::new());

         ::std::borrow::Cow::Owned(rcb)
@@ -211,8 +211,8 @@
         if (!response.status().is_success() && response.status().as_u16() != 200) || force_error {
             return ::std::option::Option::None;
         }
-        ::std::option::Option::Some(crate::protocol_serde::type_erase_result(
-            crate::protocol_serde::shape_invoke_model_with_bidirectional_stream::de_invoke_model_with_bidirectional_stream_http_response(response),
+        ::std::option::Option::Some(super::super::protocol_serde::type_erase_result(
+            super::super::protocol_serde::shape_invoke_model_with_bidirectional_stream::de_invoke_model_with_bidirectional_stream_http_response(response),
         ))
     }

@@ -223,8 +223,8 @@
     ) -> ::aws_smithy_runtime_api::client::interceptors::context::OutputOrError {
         // For streaming operations, we only hit this case if its an error
         let body = response.body().bytes().expect("body loaded");
-        crate::protocol_serde::type_erase_result(
-            crate::protocol_serde::shape_invoke_model_with_bidirectional_stream::de_invoke_model_with_bidirectional_stream_http_error(
+        super::super::protocol_serde::type_erase_result(
+            super::super::protocol_serde::shape_invoke_model_with_bidirectional_stream::de_invoke_model_with_bidirectional_stream_http_error(
                 response.status().as_u16(),
                 response.headers(),
                 body,
@@ -242,16 +242,16 @@
         _cfg: &mut ::aws_smithy_types::config_bag::ConfigBag,
     ) -> ::std::result::Result<::aws_smithy_runtime_api::client::orchestrator::HttpRequest, ::aws_smithy_runtime_api::box_error::BoxError> {
         let input = input
-            .downcast::<crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput>()
+            .downcast::<super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput>()
             .expect("correct type");
         let _header_serialization_settings = _cfg
-            .load::<crate::serialization_settings::HeaderSerializationSettings>()
+            .load::<super::super::serialization_settings::HeaderSerializationSettings>()
             .cloned()
             .unwrap_or_default();
         let mut request_builder = {
             #[allow(clippy::uninlined_format_args)]
             fn uri_base(
-                _input: &crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
+                _input: &super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
                 output: &mut ::std::string::String,
             ) -> ::std::result::Result<(), ::aws_smithy_types::error::operation::BuildError> {
                 use ::std::fmt::Write as _;
@@ -271,7 +271,7 @@
             }
             #[allow(clippy::unnecessary_wraps)]
             fn update_http_builder(
-                input: &crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
+                input: &super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
                 builder: ::http_1x::request::Builder,
             ) -> ::std::result::Result<::http_1x::request::Builder, ::aws_smithy_types::error::operation::BuildError> {
                 let mut uri = ::std::string::String::new();
@@ -284,8 +284,8 @@
             builder
         };
         let body = ::aws_smithy_types::body::SdkBody::from({
-            let error_marshaller = crate::event_stream_serde::InvokeModelWithBidirectionalStreamInputErrorMarshaller::new();
-            let marshaller = crate::event_stream_serde::InvokeModelWithBidirectionalStreamInputMarshaller::new();
+            let error_marshaller = super::super::event_stream_serde::InvokeModelWithBidirectionalStreamInputErrorMarshaller::new();
+            let marshaller = super::super::event_stream_serde::InvokeModelWithBidirectionalStreamInputMarshaller::new();

             let (signer, signer_sender) = ::aws_smithy_eventstream::frame::DeferredSigner::new();
             _cfg.interceptor_state().store_put(signer_sender);
@@ -326,7 +326,7 @@
             .downcast_ref::<InvokeModelWithBidirectionalStreamInput>()
             .ok_or("failed to downcast to InvokeModelWithBidirectionalStreamInput")?;

-        let params = crate::config::endpoint::Params::builder()
+        let params = super::super::config::endpoint::Params::builder()
             .set_region(cfg.load::<::aws_types::region::Region>().map(|r| r.as_ref().to_owned()))
             .set_use_dual_stack(cfg.load::<::aws_types::endpoint_config::UseDualStack>().map(|ty| ty.0))
             .set_use_fips(cfg.load::<::aws_types::endpoint_config::UseFips>().map(|ty| ty.0))
@@ -349,27 +349,27 @@
 #[derive(::std::fmt::Debug)]
 pub enum InvokeModelWithBidirectionalStreamError {
     /// <p>The request is denied because you do not have sufficient permissions to perform the requested action. For troubleshooting this error, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html#ts-access-denied">AccessDeniedException</a> in the Amazon Bedrock User Guide</p>
-    AccessDeniedException(crate::types::error::AccessDeniedException),
+    AccessDeniedException(super::super::types::error::AccessDeniedException),
     /// <p>An internal server error occurred. For troubleshooting this error, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html#ts-internal-failure">InternalFailure</a> in the Amazon Bedrock User Guide</p>
-    InternalServerException(crate::types::error::InternalServerException),
+    InternalServerException(super::super::types::error::InternalServerException),
     /// <p>The request failed due to an error while processing the model.</p>
-    ModelErrorException(crate::types::error::ModelErrorException),
+    ModelErrorException(super::super::types::error::ModelErrorException),
     /// <p>The model specified in the request is not ready to serve inference requests. The AWS SDK will automatically retry the operation up to 5 times. For information about configuring automatic retries, see <a href="https://docs.aws.amazon.com/sdkref/latest/guide/feature-retry-behavior.html">Retry behavior</a> in the <i>AWS SDKs and Tools</i> reference guide.</p>
-    ModelNotReadyException(crate::types::error::ModelNotReadyException),
+    ModelNotReadyException(super::super::types::error::ModelNotReadyException),
     /// <p>An error occurred while streaming the response. Retry your request.</p>
-    ModelStreamErrorException(crate::types::error::ModelStreamErrorException),
+    ModelStreamErrorException(super::super::types::error::ModelStreamErrorException),
     /// <p>The request took too long to process. Processing time exceeded the model timeout length.</p>
-    ModelTimeoutException(crate::types::error::ModelTimeoutException),
+    ModelTimeoutException(super::super::types::error::ModelTimeoutException),
     /// <p>The specified resource ARN was not found. For troubleshooting this error, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html#ts-resource-not-found">ResourceNotFound</a> in the Amazon Bedrock User Guide</p>
-    ResourceNotFoundException(crate::types::error::ResourceNotFoundException),
+    ResourceNotFoundException(super::super::types::error::ResourceNotFoundException),
     /// <p>Your request exceeds the service quota for your account. You can view your quotas at <a href="https://docs.aws.amazon.com/servicequotas/latest/userguide/gs-request-quota.html">Viewing service quotas</a>. You can resubmit your request later.</p>
-    ServiceQuotaExceededException(crate::types::error::ServiceQuotaExceededException),
+    ServiceQuotaExceededException(super::super::types::error::ServiceQuotaExceededException),
     /// <p>The service isn't currently available. For troubleshooting this error, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html#ts-service-unavailable">ServiceUnavailable</a> in the Amazon Bedrock User Guide</p>
-    ServiceUnavailableException(crate::types::error::ServiceUnavailableException),
+    ServiceUnavailableException(super::super::types::error::ServiceUnavailableException),
     /// <p>Your request was denied due to exceeding the account quotas for <i>Amazon Bedrock</i>. For troubleshooting this error, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html#ts-throttling-exception">ThrottlingException</a> in the Amazon Bedrock User Guide</p>
-    ThrottlingException(crate::types::error::ThrottlingException),
+    ThrottlingException(super::super::types::error::ThrottlingException),
     /// <p>The input fails to satisfy the constraints specified by <i>Amazon Bedrock</i>. For troubleshooting this error, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html#ts-validation-error">ValidationError</a> in the Amazon Bedrock User Guide</p>
-    ValidationException(crate::types::error::ValidationException),
+    ValidationException(super::super::types::error::ValidationException),
     /// An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).
     #[deprecated(note = "Matching `Unhandled` directly is not forwards compatible. Instead, match using a \
     variable wildcard pattern and check `.code()`:
@@ -377,7 +377,7 @@
     &nbsp;&nbsp;&nbsp;`err if err.code() == Some(\"SpecificExceptionCode\") => { /* handle the error */ }`
      \
     See [`ProvideErrorMetadata`](#impl-ProvideErrorMetadata-for-InvokeModelWithBidirectionalStreamError) for what information is available for the error.")]
-    Unhandled(crate::error::sealed_unhandled::Unhandled),
+    Unhandled(super::super::error::sealed_unhandled::Unhandled),
 }
 impl InvokeModelWithBidirectionalStreamError {
     /// Creates the `InvokeModelWithBidirectionalStreamError::Unhandled` variant from any error type.
@@ -384,7 +384,7 @@
     pub fn unhandled(
         err: impl ::std::convert::Into<::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.into(),
             meta: ::std::default::Default::default(),
         })
@@ -392,7 +392,7 @@

     /// Creates the `InvokeModelWithBidirectionalStreamError::Unhandled` variant from an [`ErrorMetadata`](::aws_smithy_types::error::ErrorMetadata).
     pub fn generic(err: ::aws_smithy_types::error::ErrorMetadata) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source: err.clone().into(),
             meta: err,
         })
@@ -538,21 +538,21 @@
         source: ::std::boxed::Box<dyn ::std::error::Error + ::std::marker::Send + ::std::marker::Sync + 'static>,
         meta: ::std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
     ) -> Self {
-        Self::Unhandled(crate::error::sealed_unhandled::Unhandled {
+        Self::Unhandled(super::super::error::sealed_unhandled::Unhandled {
             source,
             meta: meta.unwrap_or_default(),
         })
     }
 }
-impl ::aws_types::request_id::RequestId for crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError {
+impl ::aws_types::request_id::RequestId for super::super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamError {
     fn request_id(&self) -> Option<&str> {
         self.meta().request_id()
     }
 }

-pub use crate::operation::invoke_model_with_bidirectional_stream::_invoke_model_with_bidirectional_stream_input::InvokeModelWithBidirectionalStreamInput;
+pub use super::super::operation::invoke_model_with_bidirectional_stream::_invoke_model_with_bidirectional_stream_input::InvokeModelWithBidirectionalStreamInput;

-pub use crate::operation::invoke_model_with_bidirectional_stream::_invoke_model_with_bidirectional_stream_output::InvokeModelWithBidirectionalStreamOutput;
+pub use super::super::operation::invoke_model_with_bidirectional_stream::_invoke_model_with_bidirectional_stream_output::InvokeModelWithBidirectionalStreamOutput;

 mod _invoke_model_with_bidirectional_stream_input;

