--- original
+++ modified
@@ -20,8 +20,8 @@
 }
 impl ListContactListsInput {
     /// Creates a new builder-style object to manufacture [`ListContactListsInput`](crate::operation::list_contact_lists::ListContactListsInput).
-    pub fn builder() -> crate::operation::list_contact_lists::builders::ListContactListsInputBuilder {
-        crate::operation::list_contact_lists::builders::ListContactListsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_contact_lists::builders::ListContactListsInputBuilder {
+        super::super::super::operation::list_contact_lists::builders::ListContactListsInputBuilder::default()
     }
 }

@@ -64,8 +64,8 @@
     /// Consumes the builder and constructs a [`ListContactListsInput`](crate::operation::list_contact_lists::ListContactListsInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::list_contact_lists::ListContactListsInput, ::aws_smithy_types::error::operation::BuildError> {
-        ::std::result::Result::Ok(crate::operation::list_contact_lists::ListContactListsInput {
+    ) -> ::std::result::Result<super::super::super::operation::list_contact_lists::ListContactListsInput, ::aws_smithy_types::error::operation::BuildError> {
+        ::std::result::Result::Ok(super::super::super::operation::list_contact_lists::ListContactListsInput {
             page_size: self.page_size,
             next_token: self.next_token,
         })
