--- original
+++ modified
@@ -7,7 +7,7 @@
     ///   - [`receipt_handle(impl Into<String>)`](crate::operation::delete_message::builders::DeleteMessageFluentBuilder::receipt_handle) / [`set_receipt_handle(Option<String>)`](crate::operation::delete_message::builders::DeleteMessageFluentBuilder::set_receipt_handle):<br>required: **true**<br><p>The receipt handle associated with the message to delete.</p><br>
     /// - On success, responds with [`DeleteMessageOutput`](crate::operation::delete_message::DeleteMessageOutput)
     /// - On failure, responds with [`SdkError<DeleteMessageError>`](crate::operation::delete_message::DeleteMessageError)
-    pub fn delete_message(&self) -> crate::operation::delete_message::builders::DeleteMessageFluentBuilder {
-        crate::operation::delete_message::builders::DeleteMessageFluentBuilder::new(self.handle.clone())
+    pub fn delete_message(&self) -> super::super::operation::delete_message::builders::DeleteMessageFluentBuilder {
+        super::super::operation::delete_message::builders::DeleteMessageFluentBuilder::new(self.handle.clone())
     }
 }
