--- original
+++ modified
@@ -11,7 +11,7 @@
     ///   - [`queue_urls(Option<Vec::<String>>)`](crate::operation::list_queues::ListQueuesOutput::queue_urls): <p>A list of queue URLs, up to 1,000 entries, or the value of <code>MaxResults</code> that you sent in the request.</p>
     ///   - [`next_token(Option<String>)`](crate::operation::list_queues::ListQueuesOutput::next_token): <p>Pagination token to include in the next request. Token value is <code>null</code> if there are no additional results to request, or if you did not set <code>MaxResults</code> in the request.</p>
     /// - On failure, responds with [`SdkError<ListQueuesError>`](crate::operation::list_queues::ListQueuesError)
-    pub fn list_queues(&self) -> crate::operation::list_queues::builders::ListQueuesFluentBuilder {
-        crate::operation::list_queues::builders::ListQueuesFluentBuilder::new(self.handle.clone())
+    pub fn list_queues(&self) -> super::operation::list_queues::builders::ListQueuesFluentBuilder {
+        super::operation::list_queues::builders::ListQueuesFluentBuilder::new(self.handle.clone())
     }
 }
