--- original
+++ modified
@@ -7,7 +7,7 @@
     /// - On success, responds with [`ListQueueTagsOutput`](crate::operation::list_queue_tags::ListQueueTagsOutput) with field(s):
     ///   - [`tags(Option<HashMap::<String, String>>)`](crate::operation::list_queue_tags::ListQueueTagsOutput::tags): <p>The list of all tags added to the specified queue.</p>
     /// - On failure, responds with [`SdkError<ListQueueTagsError>`](crate::operation::list_queue_tags::ListQueueTagsError)
-    pub fn list_queue_tags(&self) -> crate::operation::list_queue_tags::builders::ListQueueTagsFluentBuilder {
-        crate::operation::list_queue_tags::builders::ListQueueTagsFluentBuilder::new(self.handle.clone())
+    pub fn list_queue_tags(&self) -> super::operation::list_queue_tags::builders::ListQueueTagsFluentBuilder {
+        super::operation::list_queue_tags::builders::ListQueueTagsFluentBuilder::new(self.handle.clone())
     }
 }
