--- original
+++ modified
@@ -36,7 +36,7 @@
     }
 }
 impl ::std::error::Error for DataAlreadyAcceptedException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::DataAlreadyAcceptedException {
+impl ::aws_types::request_id::RequestId for super::types::error::DataAlreadyAcceptedException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -49,8 +49,8 @@
 }
 impl DataAlreadyAcceptedException {
     /// Creates a new builder-style object to manufacture [`DataAlreadyAcceptedException`](crate::types::error::DataAlreadyAcceptedException).
-    pub fn builder() -> crate::types::error::builders::DataAlreadyAcceptedExceptionBuilder {
-        crate::types::error::builders::DataAlreadyAcceptedExceptionBuilder::default()
+    pub fn builder() -> super::types::error::builders::DataAlreadyAcceptedExceptionBuilder {
+        super::types::error::builders::DataAlreadyAcceptedExceptionBuilder::default()
     }
 }

@@ -103,8 +103,8 @@
         self
     }
     /// Consumes the builder and constructs a [`DataAlreadyAcceptedException`](crate::types::error::DataAlreadyAcceptedException).
-    pub fn build(self) -> crate::types::error::DataAlreadyAcceptedException {
-        crate::types::error::DataAlreadyAcceptedException {
+    pub fn build(self) -> super::types::error::DataAlreadyAcceptedException {
+        super::types::error::DataAlreadyAcceptedException {
             expected_sequence_token: self.expected_sequence_token,
             message: self.message,
             meta: self.meta.unwrap_or_default(),
