--- original
+++ modified
@@ -6,7 +6,7 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
 pub struct MalformedQueryException {
     /// <p>Reserved.</p>
-    pub query_compile_error: ::std::option::Option<crate::types::QueryCompileError>,
+    pub query_compile_error: ::std::option::Option<super::super::super::types::QueryCompileError>,
     #[allow(missing_docs)] // documentation missing in model
     pub message: ::std::option::Option<::std::string::String>,
     pub(crate) meta: ::aws_smithy_types::error::ErrorMetadata,
@@ -13,7 +13,7 @@
 }
 impl MalformedQueryException {
     /// <p>Reserved.</p>
-    pub fn query_compile_error(&self) -> ::std::option::Option<&crate::types::QueryCompileError> {
+    pub fn query_compile_error(&self) -> ::std::option::Option<&super::super::super::types::QueryCompileError> {
         self.query_compile_error.as_ref()
     }
 }
@@ -35,7 +35,7 @@
     }
 }
 impl ::std::error::Error for MalformedQueryException {}
-impl ::aws_types::request_id::RequestId for crate::types::error::MalformedQueryException {
+impl ::aws_types::request_id::RequestId for super::super::super::types::error::MalformedQueryException {
     fn request_id(&self) -> Option<&str> {
         use ::aws_smithy_types::error::metadata::ProvideErrorMetadata;
         self.meta().request_id()
@@ -48,8 +48,8 @@
 }
 impl MalformedQueryException {
     /// Creates a new builder-style object to manufacture [`MalformedQueryException`](crate::types::error::MalformedQueryException).
-    pub fn builder() -> crate::types::error::builders::MalformedQueryExceptionBuilder {
-        crate::types::error::builders::MalformedQueryExceptionBuilder::default()
+    pub fn builder() -> super::super::super::types::error::builders::MalformedQueryExceptionBuilder {
+        super::super::super::types::error::builders::MalformedQueryExceptionBuilder::default()
     }
 }

@@ -57,23 +57,23 @@
 #[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::default::Default, ::std::fmt::Debug)]
 #[non_exhaustive]
 pub struct MalformedQueryExceptionBuilder {
-    pub(crate) query_compile_error: ::std::option::Option<crate::types::QueryCompileError>,
+    pub(crate) query_compile_error: ::std::option::Option<super::super::super::types::QueryCompileError>,
     pub(crate) message: ::std::option::Option<::std::string::String>,
     meta: std::option::Option<::aws_smithy_types::error::ErrorMetadata>,
 }
 impl MalformedQueryExceptionBuilder {
     /// <p>Reserved.</p>
-    pub fn query_compile_error(mut self, input: crate::types::QueryCompileError) -> Self {
+    pub fn query_compile_error(mut self, input: super::super::super::types::QueryCompileError) -> Self {
         self.query_compile_error = ::std::option::Option::Some(input);
         self
     }
     /// <p>Reserved.</p>
-    pub fn set_query_compile_error(mut self, input: ::std::option::Option<crate::types::QueryCompileError>) -> Self {
+    pub fn set_query_compile_error(mut self, input: ::std::option::Option<super::super::super::types::QueryCompileError>) -> Self {
         self.query_compile_error = input;
         self
     }
     /// <p>Reserved.</p>
-    pub fn get_query_compile_error(&self) -> &::std::option::Option<crate::types::QueryCompileError> {
+    pub fn get_query_compile_error(&self) -> &::std::option::Option<super::super::super::types::QueryCompileError> {
         &self.query_compile_error
     }
     #[allow(missing_docs)] // documentation missing in model
@@ -102,8 +102,8 @@
         self
     }
     /// Consumes the builder and constructs a [`MalformedQueryException`](crate::types::error::MalformedQueryException).
-    pub fn build(self) -> crate::types::error::MalformedQueryException {
-        crate::types::error::MalformedQueryException {
+    pub fn build(self) -> super::super::super::types::error::MalformedQueryException {
+        super::super::super::types::error::MalformedQueryException {
             query_compile_error: self.query_compile_error,
             message: self.message,
             meta: self.meta.unwrap_or_default(),
