--- original
+++ modified
@@ -34,8 +34,8 @@
 }
 impl GetFunctionConcurrencyInput {
     /// Creates a new builder-style object to manufacture [`GetFunctionConcurrencyInput`](crate::operation::get_function_concurrency::GetFunctionConcurrencyInput).
-    pub fn builder() -> crate::operation::get_function_concurrency::builders::GetFunctionConcurrencyInputBuilder {
-        crate::operation::get_function_concurrency::builders::GetFunctionConcurrencyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_function_concurrency::builders::GetFunctionConcurrencyInputBuilder {
+        super::super::super::operation::get_function_concurrency::builders::GetFunctionConcurrencyInputBuilder::default()
     }
 }

@@ -95,10 +95,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_function_concurrency::GetFunctionConcurrencyInput,
+        super::super::super::operation::get_function_concurrency::GetFunctionConcurrencyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_function_concurrency::GetFunctionConcurrencyInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_function_concurrency::GetFunctionConcurrencyInput {
             function_name: self.function_name,
         })
     }
