--- original
+++ modified
@@ -7,8 +7,8 @@
     pub model_id: ::std::option::Option<::std::string::String>,
     /// <p>The prompt and inference parameters in the format specified in the <code>BidirectionalInputPayloadPart</code> in the header. You must provide the body in JSON format. To see the format and content of the request and response bodies for different models, refer to <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html">Inference parameters</a>. For more information, see <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/api-methods-run.html">Run inference</a> in the Bedrock User Guide.</p>
     pub body: ::aws_smithy_http::event_stream::EventStreamSender<
-        crate::types::InvokeModelWithBidirectionalStreamInput,
-        crate::types::error::InvokeModelWithBidirectionalStreamInputError,
+        super::types::InvokeModelWithBidirectionalStreamInput,
+        super::types::error::InvokeModelWithBidirectionalStreamInputError,
     >,
 }
 impl InvokeModelWithBidirectionalStreamInput {
@@ -20,8 +20,8 @@
     pub fn body(
         &self,
     ) -> &::aws_smithy_http::event_stream::EventStreamSender<
-        crate::types::InvokeModelWithBidirectionalStreamInput,
-        crate::types::error::InvokeModelWithBidirectionalStreamInputError,
+        super::types::InvokeModelWithBidirectionalStreamInput,
+        super::types::error::InvokeModelWithBidirectionalStreamInputError,
     > {
         &self.body
     }
@@ -28,8 +28,8 @@
 }
 impl InvokeModelWithBidirectionalStreamInput {
     /// Creates a new builder-style object to manufacture [`InvokeModelWithBidirectionalStreamInput`](crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput).
-    pub fn builder() -> crate::operation::invoke_model_with_bidirectional_stream::builders::InvokeModelWithBidirectionalStreamInputBuilder {
-        crate::operation::invoke_model_with_bidirectional_stream::builders::InvokeModelWithBidirectionalStreamInputBuilder::default()
+    pub fn builder() -> super::operation::invoke_model_with_bidirectional_stream::builders::InvokeModelWithBidirectionalStreamInputBuilder {
+        super::operation::invoke_model_with_bidirectional_stream::builders::InvokeModelWithBidirectionalStreamInputBuilder::default()
     }
 }

@@ -40,8 +40,8 @@
     pub(crate) model_id: ::std::option::Option<::std::string::String>,
     pub(crate) body: ::std::option::Option<
         ::aws_smithy_http::event_stream::EventStreamSender<
-            crate::types::InvokeModelWithBidirectionalStreamInput,
-            crate::types::error::InvokeModelWithBidirectionalStreamInputError,
+            super::types::InvokeModelWithBidirectionalStreamInput,
+            super::types::error::InvokeModelWithBidirectionalStreamInputError,
         >,
     >,
 }
@@ -66,8 +66,8 @@
     pub fn body(
         mut self,
         input: ::aws_smithy_http::event_stream::EventStreamSender<
-            crate::types::InvokeModelWithBidirectionalStreamInput,
-            crate::types::error::InvokeModelWithBidirectionalStreamInputError,
+            super::types::InvokeModelWithBidirectionalStreamInput,
+            super::types::error::InvokeModelWithBidirectionalStreamInputError,
         >,
     ) -> Self {
         self.body = ::std::option::Option::Some(input);
@@ -78,8 +78,8 @@
         mut self,
         input: ::std::option::Option<
             ::aws_smithy_http::event_stream::EventStreamSender<
-                crate::types::InvokeModelWithBidirectionalStreamInput,
-                crate::types::error::InvokeModelWithBidirectionalStreamInputError,
+                super::types::InvokeModelWithBidirectionalStreamInput,
+                super::types::error::InvokeModelWithBidirectionalStreamInputError,
             >,
         >,
     ) -> Self {
@@ -91,8 +91,8 @@
         &self,
     ) -> &::std::option::Option<
         ::aws_smithy_http::event_stream::EventStreamSender<
-            crate::types::InvokeModelWithBidirectionalStreamInput,
-            crate::types::error::InvokeModelWithBidirectionalStreamInputError,
+            super::types::InvokeModelWithBidirectionalStreamInput,
+            super::types::error::InvokeModelWithBidirectionalStreamInputError,
         >,
     > {
         &self.body
@@ -103,11 +103,11 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
+        super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
         ::std::result::Result::Ok(
-            crate::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput {
+            super::operation::invoke_model_with_bidirectional_stream::InvokeModelWithBidirectionalStreamInput {
                 model_id: self.model_id,
                 body: self.body.ok_or_else(|| {
                     ::aws_smithy_types::error::operation::BuildError::missing_field(
