--- original
+++ modified
@@ -32,8 +32,8 @@
 }
 impl GetDurableExecutionStateInput {
     /// Creates a new builder-style object to manufacture [`GetDurableExecutionStateInput`](crate::operation::get_durable_execution_state::GetDurableExecutionStateInput).
-    pub fn builder() -> crate::operation::get_durable_execution_state::builders::GetDurableExecutionStateInputBuilder {
-        crate::operation::get_durable_execution_state::builders::GetDurableExecutionStateInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_durable_execution_state::builders::GetDurableExecutionStateInputBuilder {
+        super::super::super::operation::get_durable_execution_state::builders::GetDurableExecutionStateInputBuilder::default()
     }
 }

@@ -109,10 +109,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_durable_execution_state::GetDurableExecutionStateInput,
+        super::super::super::operation::get_durable_execution_state::GetDurableExecutionStateInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_durable_execution_state::GetDurableExecutionStateInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_durable_execution_state::GetDurableExecutionStateInput {
             durable_execution_arn: self.durable_execution_arn,
             checkpoint_token: self.checkpoint_token,
             marker: self.marker,
