--- original
+++ modified
@@ -51,7 +51,7 @@
     /// <p>This legacy option creates an attribute with a <code>dev:</code> prefix. You can only set the value of a developer-only attribute with administrative IAM credentials.</p>
     /// </dd>
     /// </dl>
-    pub custom_attributes: ::std::option::Option<::std::vec::Vec<crate::types::SchemaAttributeType>>,
+    pub custom_attributes: ::std::option::Option<::std::vec::Vec<super::super::super::types::SchemaAttributeType>>,
 }
 impl AddCustomAttributesInput {
     /// <p>The ID of the user pool where you want to add custom attributes.</p>
@@ -105,14 +105,14 @@
     /// </dl>
     ///
     /// If no value was sent for this field, a default will be set. If you want to determine if no value was sent, use `.custom_attributes.is_none()`.
-    pub fn custom_attributes(&self) -> &[crate::types::SchemaAttributeType] {
+    pub fn custom_attributes(&self) -> &[super::super::super::types::SchemaAttributeType] {
         self.custom_attributes.as_deref().unwrap_or_default()
     }
 }
 impl AddCustomAttributesInput {
     /// Creates a new builder-style object to manufacture [`AddCustomAttributesInput`](crate::operation::add_custom_attributes::AddCustomAttributesInput).
-    pub fn builder() -> crate::operation::add_custom_attributes::builders::AddCustomAttributesInputBuilder {
-        crate::operation::add_custom_attributes::builders::AddCustomAttributesInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::add_custom_attributes::builders::AddCustomAttributesInputBuilder {
+        super::super::super::operation::add_custom_attributes::builders::AddCustomAttributesInputBuilder::default()
     }
 }

@@ -121,7 +121,7 @@
 #[non_exhaustive]
 pub struct AddCustomAttributesInputBuilder {
     pub(crate) user_pool_id: ::std::option::Option<::std::string::String>,
-    pub(crate) custom_attributes: ::std::option::Option<::std::vec::Vec<crate::types::SchemaAttributeType>>,
+    pub(crate) custom_attributes: ::std::option::Option<::std::vec::Vec<super::super::super::types::SchemaAttributeType>>,
 }
 impl AddCustomAttributesInputBuilder {
     /// <p>The ID of the user pool where you want to add custom attributes.</p>
@@ -188,7 +188,7 @@
     /// <p>This legacy option creates an attribute with a <code>dev:</code> prefix. You can only set the value of a developer-only attribute with administrative IAM credentials.</p>
     /// </dd>
     /// </dl>
-    pub fn custom_attributes(mut self, input: crate::types::SchemaAttributeType) -> Self {
+    pub fn custom_attributes(mut self, input: super::super::super::types::SchemaAttributeType) -> Self {
         let mut v = self.custom_attributes.unwrap_or_default();
         v.push(input);
         self.custom_attributes = ::std::option::Option::Some(v);
@@ -239,7 +239,7 @@
     /// <p>This legacy option creates an attribute with a <code>dev:</code> prefix. You can only set the value of a developer-only attribute with administrative IAM credentials.</p>
     /// </dd>
     /// </dl>
-    pub fn set_custom_attributes(mut self, input: ::std::option::Option<::std::vec::Vec<crate::types::SchemaAttributeType>>) -> Self {
+    pub fn set_custom_attributes(mut self, input: ::std::option::Option<::std::vec::Vec<super::super::super::types::SchemaAttributeType>>) -> Self {
         self.custom_attributes = input;
         self
     }
@@ -288,15 +288,15 @@
     /// <p>This legacy option creates an attribute with a <code>dev:</code> prefix. You can only set the value of a developer-only attribute with administrative IAM credentials.</p>
     /// </dd>
     /// </dl>
-    pub fn get_custom_attributes(&self) -> &::std::option::Option<::std::vec::Vec<crate::types::SchemaAttributeType>> {
+    pub fn get_custom_attributes(&self) -> &::std::option::Option<::std::vec::Vec<super::super::super::types::SchemaAttributeType>> {
         &self.custom_attributes
     }
     /// Consumes the builder and constructs a [`AddCustomAttributesInput`](crate::operation::add_custom_attributes::AddCustomAttributesInput).
     pub fn build(
         self,
-    ) -> ::std::result::Result<crate::operation::add_custom_attributes::AddCustomAttributesInput, ::aws_smithy_types::error::operation::BuildError>
+    ) -> ::std::result::Result<super::super::super::operation::add_custom_attributes::AddCustomAttributesInput, ::aws_smithy_types::error::operation::BuildError>
     {
-        ::std::result::Result::Ok(crate::operation::add_custom_attributes::AddCustomAttributesInput {
+        ::std::result::Result::Ok(super::super::super::operation::add_custom_attributes::AddCustomAttributesInput {
             user_pool_id: self.user_pool_id,
             custom_attributes: self.custom_attributes,
         })
