--- original
+++ modified
@@ -36,8 +36,8 @@
 }
 impl ListUserPoolClientSecretsInput {
     /// Creates a new builder-style object to manufacture [`ListUserPoolClientSecretsInput`](crate::operation::list_user_pool_client_secrets::ListUserPoolClientSecretsInput).
-    pub fn builder() -> crate::operation::list_user_pool_client_secrets::builders::ListUserPoolClientSecretsInputBuilder {
-        crate::operation::list_user_pool_client_secrets::builders::ListUserPoolClientSecretsInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::list_user_pool_client_secrets::builders::ListUserPoolClientSecretsInputBuilder {
+        super::super::super::operation::list_user_pool_client_secrets::builders::ListUserPoolClientSecretsInputBuilder::default()
     }
 }

@@ -98,10 +98,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::list_user_pool_client_secrets::ListUserPoolClientSecretsInput,
+        super::super::super::operation::list_user_pool_client_secrets::ListUserPoolClientSecretsInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::list_user_pool_client_secrets::ListUserPoolClientSecretsInput {
+        ::std::result::Result::Ok(super::super::super::operation::list_user_pool_client_secrets::ListUserPoolClientSecretsInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
             next_token: self.next_token,
