--- original
+++ modified
@@ -36,8 +36,8 @@
 }
 impl DeleteUserPoolClientSecretInput {
     /// Creates a new builder-style object to manufacture [`DeleteUserPoolClientSecretInput`](crate::operation::delete_user_pool_client_secret::DeleteUserPoolClientSecretInput).
-    pub fn builder() -> crate::operation::delete_user_pool_client_secret::builders::DeleteUserPoolClientSecretInputBuilder {
-        crate::operation::delete_user_pool_client_secret::builders::DeleteUserPoolClientSecretInputBuilder::default()
+    pub fn builder() -> super::operation::delete_user_pool_client_secret::builders::DeleteUserPoolClientSecretInputBuilder {
+        super::operation::delete_user_pool_client_secret::builders::DeleteUserPoolClientSecretInputBuilder::default()
     }
 }

@@ -99,10 +99,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::delete_user_pool_client_secret::DeleteUserPoolClientSecretInput,
+        super::operation::delete_user_pool_client_secret::DeleteUserPoolClientSecretInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::delete_user_pool_client_secret::DeleteUserPoolClientSecretInput {
+        ::std::result::Result::Ok(super::operation::delete_user_pool_client_secret::DeleteUserPoolClientSecretInput {
             user_pool_id: self.user_pool_id,
             client_id: self.client_id,
             client_secret_id: self.client_secret_id,
