--- original
+++ modified
@@ -14,8 +14,8 @@
 }
 impl GetDeliveryDestinationPolicyInput {
     /// Creates a new builder-style object to manufacture [`GetDeliveryDestinationPolicyInput`](crate::operation::get_delivery_destination_policy::GetDeliveryDestinationPolicyInput).
-    pub fn builder() -> crate::operation::get_delivery_destination_policy::builders::GetDeliveryDestinationPolicyInputBuilder {
-        crate::operation::get_delivery_destination_policy::builders::GetDeliveryDestinationPolicyInputBuilder::default()
+    pub fn builder() -> super::super::super::operation::get_delivery_destination_policy::builders::GetDeliveryDestinationPolicyInputBuilder {
+        super::super::super::operation::get_delivery_destination_policy::builders::GetDeliveryDestinationPolicyInputBuilder::default()
     }
 }

@@ -45,10 +45,10 @@
     pub fn build(
         self,
     ) -> ::std::result::Result<
-        crate::operation::get_delivery_destination_policy::GetDeliveryDestinationPolicyInput,
+        super::super::super::operation::get_delivery_destination_policy::GetDeliveryDestinationPolicyInput,
         ::aws_smithy_types::error::operation::BuildError,
     > {
-        ::std::result::Result::Ok(crate::operation::get_delivery_destination_policy::GetDeliveryDestinationPolicyInput {
+        ::std::result::Result::Ok(super::super::super::operation::get_delivery_destination_policy::GetDeliveryDestinationPolicyInput {
             delivery_destination_name: self.delivery_destination_name,
         })
     }
