--- original
+++ modified
@@ -20,8 +20,8 @@
     response_headers: &::aws_smithy_runtime_api::http::Headers,
     response_body: &[u8],
 ) -> ::std::result::Result<::aws_smithy_types::error::metadata::Builder, ::aws_smithy_json::deserialize::error::DeserializeError> {
-    let mut builder = crate::json_errors::parse_error_metadata(response_body, response_headers)?;
-    if let Some((error_code, error_type)) = crate::aws_query_compatible_errors::parse_aws_query_compatible_error(response_headers) {
+    let mut builder = super::json_errors::parse_error_metadata(response_body, response_headers)?;
+    if let Some((error_code, error_type)) = super::aws_query_compatible_errors::parse_aws_query_compatible_error(response_headers) {
         builder = builder.code(error_code);
         builder = builder.custom("type", error_type);
     }
